<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NMETH_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nmeth---401">NMETH - 401</h2>
<ul>
<li><details>
<summary>
(2023). Author correction: Mapping the genomic landscape of
CRISPR–Cas9 cleavage. <em>NMETH</em>, <em>20</em>(12), 2068. (<a
href="https://doi.org/10.1038/s41592-023-02114-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Cameron, Peter and Fuller, Chris K. and Donohoue, Paul D. and Jones, Brittnee N. and Thompson, Matthew S. and Carter, Matthew M. and Gradia, Scott and Vidal, Bastien and Garner, Elizabeth and Slorach, Euan M. and Lau, Elaine and Banh, Lynda M. and Lied, Alexandra M. and Edwards, Leslie S. and Settle, Alexander H. and Capurso, Daniel and Llaca, Victor and Deschamps, Stéphane and Cigan, Mark and Young, Joshua K. and May, Andrew P.},
  doi          = {10.1038/s41592-023-02114-4},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2068},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Mapping the genomic landscape of CRISPR–Cas9 cleavage},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Comprehensive benchmarking and guidelines of mosaic variant
calling strategies. <em>NMETH</em>, <em>20</em>(12), 2058–2067. (<a
href="https://doi.org/10.1038/s41592-023-02043-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapid advances in sequencing and analysis technologies have enabled the accurate detection of diverse forms of genomic variants represented as heterozygous, homozygous and mosaic mutations. However, the best practices for mosaic variant calling remain disorganized owing to the technical and conceptual difficulties faced in evaluation. Here we present our benchmark of 11 feasible mosaic variant detection approaches based on a systematically designed whole-exome-level reference standard that mimics mosaic samples, supported by 354,258 control positive mosaic single-nucleotide variants and insertion-deletion mutations and 33,111,725 control negatives. We identified not only the best practice for mosaic variant detection but also the condition-dependent strengths and weaknesses of the current methods. Furthermore, feature-level evaluation and their combinatorial usage across multiple algorithms direct the way for immediate to prolonged improvements in mosaic variant detection. Our results will guide researchers in selecting suitable calling algorithms and suggest future strategies for developers. A benchmarking study evaluates 11 mosaic single-nucleotide variant and insertion-deletion mutation detection approaches.},
  archive      = {J_NMETH},
  author       = {Ha, Yoo-Jin and Kang, Seungseok and Kim, Jisoo and Kim, Junhan and Jo, Se-Young and Kim, Sangwoo},
  doi          = {10.1038/s41592-023-02043-2},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2058-2067},
  shortjournal = {Nat. Meth.},
  title        = {Comprehensive benchmarking and guidelines of mosaic variant calling strategies},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Next-generation MRI scanner designed for
ultra-high-resolution human brain imaging at 7 tesla. <em>NMETH</em>,
<em>20</em>(12), 2048–2057. (<a
href="https://doi.org/10.1038/s41592-023-02068-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To increase granularity in human neuroimaging science, we designed and built a next-generation 7 Tesla magnetic resonance imaging scanner to reach ultra-high resolution by implementing several advances in hardware. To improve spatial encoding and increase the image signal-to-noise ratio, we developed a head-only asymmetric gradient coil (200 mT m−1, 900 T m−1s−1) with an additional third layer of windings. We integrated a 128-channel receiver system with 64- and 96-channel receiver coil arrays to boost signal in the cerebral cortex while reducing g-factor noise to enable higher accelerations. A 16-channel transmit system reduced power deposition and improved image uniformity. The scanner routinely performs functional imaging studies at 0.35–0.45 mm isotropic spatial resolution to reveal cortical layer functional activity, achieves high angular resolution in diffusion imaging and reduces acquisition time for both functional and structural imaging. A combination of hardware developments has increased the achievable spatial resolution in 7 Tesla human neuroimaging to about 0.4 mm.},
  archive      = {J_NMETH},
  author       = {Feinberg, David A. and Beckett, Alexander J. S. and Vu, An T. and Stockmann, Jason and Huber, Laurentius and Ma, Samantha and Ahn, Sinyeob and Setsompop, Kawin and Cao, Xiaozhi and Park, Suhyung and Liu, Chunlei and Wald, Lawrence L. and Polimeni, Jonathan R. and Mareyam, Azma and Gruber, Bernhard and Stirnberg, Rüdiger and Liao, Congyu and Yacoub, Essa and Davids, Mathias and Bell, Paul and Rummert, Elmar and Koehler, Michael and Potthast, Andreas and Gonzalez-Insua, Ignacio and Stocker, Stefan and Gunamony, Shajan and Dietz, Peter},
  doi          = {10.1038/s41592-023-02068-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2048-2057},
  shortjournal = {Nat. Meth.},
  title        = {Next-generation MRI scanner designed for ultra-high-resolution human brain imaging at 7 tesla},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). In vitro modeling of the human dopaminergic system using
spatially arranged ventral midbrain–striatum–cortex assembloids.
<em>NMETH</em>, <em>20</em>(12), 2034–2047. (<a
href="https://doi.org/10.1038/s41592-023-02080-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ventral midbrain dopaminergic neurons project to the striatum as well as the cortex and are involved in movement control and reward-related cognition. In Parkinson’s disease, nigrostriatal midbrain dopaminergic neurons degenerate and cause typical Parkinson’s disease motor-related impairments, while the dysfunction of mesocorticolimbic midbrain dopaminergic neurons is implicated in addiction and neuropsychiatric disorders. Study of the development and selective neurodegeneration of the human dopaminergic system, however, has been limited due to the lack of an appropriate model and access to human material. Here, we have developed a human in vitro model that recapitulates key aspects of dopaminergic innervation of the striatum and cortex. These spatially arranged ventral midbrain–striatum–cortical organoids (MISCOs) can be used to study dopaminergic neuron maturation, innervation and function with implications for cell therapy and addiction research. We detail protocols for growing ventral midbrain, striatal and cortical organoids and describe how they fuse in a linear manner when placed in custom embedding molds. We report the formation of functional long-range dopaminergic connections to striatal and cortical tissues in MISCOs, and show that injected, ventral midbrain-patterned progenitors can mature and innervate the tissue. Using these assembloids, we examine dopaminergic circuit perturbations and show that chronic cocaine treatment causes long-lasting morphological, functional and transcriptional changes that persist upon drug withdrawal. Thus, our method opens new avenues to investigate human dopaminergic cell transplantation and circuitry reconstruction as well as the effect of drugs on the human dopaminergic system. MISCOs are spatially arranged ventral midbrain–striatum–cortical organoids that enable investigations into the human dopaminergic system.},
  archive      = {J_NMETH},
  author       = {Reumann, Daniel and Krauditsch, Christian and Novatchkova, Maria and Sozzi, Edoardo and Wong, Sakurako Nagumo and Zabolocki, Michael and Priouret, Marthe and Doleschall, Balint and Ritzau-Reid, Kaja I. and Piber, Marielle and Morassut, Ilaria and Fieseler, Charles and Fiorenzano, Alessandro and Stevens, Molly M. and Zimmer, Manuel and Bardy, Cedric and Parmar, Malin and Knoblich, Jürgen A.},
  doi          = {10.1038/s41592-023-02080-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2034-2047},
  shortjournal = {Nat. Meth.},
  title        = {In vitro modeling of the human dopaminergic system using spatially arranged ventral midbrain–striatum–cortex assembloids},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reconstruction of dynamic mammary mini gland in vitro for
normal physiology and oncogenesis. <em>NMETH</em>, <em>20</em>(12),
2021–2033. (<a
href="https://doi.org/10.1038/s41592-023-02039-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organoid culture has been extensively exploited for normal tissue reconstruction and disease modeling. However, it is still challenging to establish organoids that mimic in vivo-like architecture, size and function under homeostatic conditions. Here we describe the development of a long-term adult stem cell-derived mammary mini gland culture system that supports robust three-dimensional outgrowths recapitulating the morphology, scale, cellular context and transcriptional heterogeneity of the normal mammary gland. The self-organization ability of stem cells and the stability of the outgrowths were determined by a coordinated combination of extracellular matrix, environmental signals and dynamic physiological cycles. We show that these mini glands were hormone responsive and could recapitulate the entire postnatal mammary development including puberty, estrus cycle, lactation and involution. We also observed that these mini glands maintained the presence of mammary stem cells and could also recapitulate the fate transition from embryonic bipotency to postnatal unipotency in lineage tracing assays. In addition, upon induction of oncogene expression in the mini glands, we observed tumor initiation in vitro and in vivo in a mouse model. Together, this study provides an experimental system that can support a dynamic miniature mammary gland for the study of physiologically relevant, complex biological processes. A dynamic, hormone-responsive mammary gland organoid system that can mimic complex physiological processes in vitro.},
  archive      = {J_NMETH},
  author       = {Yuan, Lei and Xie, Shaofang and Bai, Huiru and Liu, Xiaoqin and Cai, Pei and Lu, Jing and Wang, Chunhui and Lin, Zuobao and Li, Shuying and Guo, Yajing and Cai, Shang},
  doi          = {10.1038/s41592-023-02039-y},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2021-2033},
  shortjournal = {Nat. Meth.},
  title        = {Reconstruction of dynamic mammary mini gland in vitro for normal physiology and oncogenesis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multi-layered maps of neuropil with segmentation-guided
contrastive learning. <em>NMETH</em>, <em>20</em>(12), 2011–2020. (<a
href="https://doi.org/10.1038/s41592-023-02059-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Maps of the nervous system that identify individual cells along with their type, subcellular components and connectivity have the potential to elucidate fundamental organizational principles of neural circuits. Nanometer-resolution imaging of brain tissue provides the necessary raw data, but inferring cellular and subcellular annotation layers is challenging. We present segmentation-guided contrastive learning of representations (SegCLR), a self-supervised machine learning technique that produces representations of cells directly from 3D imagery and segmentations. When applied to volumes of human and mouse cortex, SegCLR enables accurate classification of cellular subcompartments and achieves performance equivalent to a supervised approach while requiring 400-fold fewer labeled examples. SegCLR also enables inference of cell types from fragments as small as 10 μm, which enhances the utility of volumes in which many neurites are truncated at boundaries. Finally, SegCLR enables exploration of layer 5 pyramidal cell subtypes and automated large-scale analysis of synaptic partners in mouse visual cortex. SegCLR automatically annotates segmented electron microscopy datasets of the brain with information such as cellular subcompartments and cell types, using a self-supervised contrastive learning approach.},
  archive      = {J_NMETH},
  author       = {Dorkenwald, Sven and Li, Peter H. and Januszewski, Michał and Berger, Daniel R. and Maitin-Shepard, Jeremy and Bodor, Agnes L. and Collman, Forrest and Schneider-Mizell, Casey M. and da Costa, Nuno Maçarico and Lichtman, Jeff W. and Jain, Viren},
  doi          = {10.1038/s41592-023-02059-8},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2011-2020},
  shortjournal = {Nat. Meth.},
  title        = {Multi-layered maps of neuropil with segmentation-guided contrastive learning},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Uncovering developmental time and tempo using deep learning.
<em>NMETH</em>, <em>20</em>(12), 2000–2010. (<a
href="https://doi.org/10.1038/s41592-023-02083-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {During animal development, embryos undergo complex morphological changes over time. Differences in developmental tempo between species are emerging as principal drivers of evolutionary novelty, but accurate description of these processes is very challenging. To address this challenge, we present here an automated and unbiased deep learning approach to analyze the similarity between embryos of different timepoints. Calculation of similarities across stages resulted in complex phenotypic fingerprints, which carry characteristic information about developmental time and tempo. Using this approach, we were able to accurately stage embryos, quantitatively determine temperature-dependent developmental tempo, detect naturally occurring and induced changes in the developmental progression of individual embryos, and derive staging atlases for several species de novo in an unsupervised manner. Our approach allows us to quantify developmental time and tempo objectively and provides a standardized way to analyze early embryogenesis. A deep learning-based method that uses microscopy images to stage embryos and analyze developmental time.},
  archive      = {J_NMETH},
  author       = {Toulany, Nikan and Morales-Navarrete, Hernán and Čapek, Daniel and Grathwohl, Jannis and Ünalan, Murat and Müller, Patrick},
  doi          = {10.1038/s41592-023-02083-8},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {2000-2010},
  shortjournal = {Nat. Meth.},
  title        = {Uncovering developmental time and tempo using deep learning},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Embryo mechanics cartography: Inference of 3D force atlases
from fluorescence microscopy. <em>NMETH</em>, <em>20</em>(12),
1989–1999. (<a
href="https://doi.org/10.1038/s41592-023-02084-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tissue morphogenesis results from a tight interplay between gene expression, biochemical signaling and mechanics. Although sequencing methods allow the generation of cell-resolved spatiotemporal maps of gene expression, creating similar maps of cell mechanics in three-dimensional (3D) developing tissues has remained a real challenge. Exploiting the foam-like arrangement of cells, we propose a robust end-to-end computational method called ‘foambryo’ to infer spatiotemporal atlases of cellular forces from fluorescence microscopy images of cell membranes. Our method generates precise 3D meshes of cells’ geometry and successively predicts relative cell surface tensions and pressures. We validate it with 3D foam simulations, study its noise sensitivity and prove its biological relevance in mouse, ascidian and worm embryos. 3D force inference allows us to recover mechanical features identified previously, but also predicts new ones, unveiling potential new insights on the spatiotemporal regulation of cell mechanics in developing embryos. Our code is freely available and paves the way for unraveling the unknown mechanochemical feedbacks that control embryo and tissue morphogenesis. Foambryo is an analysis pipeline for three-dimensional force-inference measurements in developing embryos.},
  archive      = {J_NMETH},
  author       = {Ichbiah, Sacha and Delbary, Fabrice and McDougall, Alex and Dumollard, Rémi and Turlier, Hervé},
  doi          = {10.1038/s41592-023-02084-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1989-1999},
  shortjournal = {Nat. Meth.},
  title        = {Embryo mechanics cartography: Inference of 3D force atlases from fluorescence microscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). TubULAR: Tracking in toto deformations of dynamic tissues
via constrained maps. <em>NMETH</em>, <em>20</em>(12), 1980–1988. (<a
href="https://doi.org/10.1038/s41592-023-02081-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A common motif in biology is the arrangement of cells into tubes, which further transform into complex shapes. Traditionally, analysis of dynamic tissues has relied on inspecting static snapshots, live imaging of cross-sections or tracking isolated cells in three dimensions. However, capturing the interplay between in-plane and out-of-plane behaviors requires following the full surface as it deforms and integrating cell-scale motions into collective, tissue-scale deformations. Here, we present an analysis framework that builds in toto maps of tissue deformations by following tissue parcels in a static material frame of reference. Our approach then relates in-plane and out-of-plane behaviors and decomposes complex deformation maps into elementary contributions. The tube-like surface Lagrangian analysis resource (TubULAR) provides an open-source implementation accessible either as a standalone toolkit or as an extension of the ImSAnE package used in the developmental biology community. We demonstrate our approach by analyzing shape change in the embryonic Drosophila midgut and beating zebrafish heart. The method naturally generalizes to in vitro and synthetic systems and provides ready access to the mechanical mechanisms relating genetic patterning to organ shape change. TubULAR is an in toto tissue cartography method for mapping complex dynamic surfaces},
  archive      = {J_NMETH},
  author       = {Mitchell, Noah P. and Cislo, Dillon J.},
  doi          = {10.1038/s41592-023-02081-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1980-1988},
  shortjournal = {Nat. Meth.},
  title        = {TubULAR: Tracking in toto deformations of dynamic tissues via constrained maps},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pulsed stimulated brillouin microscopy enables
high-sensitivity mechanical imaging of live and fragile biological
specimens. <em>NMETH</em>, <em>20</em>(12), 1971–1979. (<a
href="https://doi.org/10.1038/s41592-023-02054-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brillouin microscopy is an emerging optical elastography technique capable of assessing mechanical properties of biological samples in a three-dimensional, all-optical and noncontact fashion. The typically weak Brillouin scattering signal can be substantially enhanced via a stimulated Brillouin scattering (SBS) process; however, current implementations require high pump powers, which prohibit applications to photosensitive or live imaging of biological samples. Here we present a pulsed SBS scheme that takes advantage of the nonlinearity of the pump–probe interaction. In particular, we show that the required pump laser power can be decreased ~20-fold without affecting the signal levels or spectral precision. We demonstrate the low phototoxicity and high specificity of our pulsed SBS approach by imaging, with subcellular detail, sensitive single cells, zebrafish larvae, mouse embryos and adult Caenorhabditis elegans. Furthermore, our method permits observing the mechanics of organoids and C. elegans embryos over time, opening up further possibilities for the field of mechanobiology. A pulsed illumination scheme renders stimulated Brillouin microscopy less phototoxic and allows imaging of the mechanical properties of sensitive samples such as single cells, Caenorhabditis elegans embryos, zebrafish larvae and organoids.},
  archive      = {J_NMETH},
  author       = {Yang, Fan and Bevilacqua, Carlo and Hambura, Sebastian and Neves, Ana and Gopalan, Anusha and Watanabe, Koki and Govendir, Matt and Bernabeu, Maria and Ellenberg, Jan and Diz-Muñoz, Alba and Köhler, Simone and Rapti, Georgia and Jechlinger, Martin and Prevedel, Robert},
  doi          = {10.1038/s41592-023-02054-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1971-1979},
  shortjournal = {Nat. Meth.},
  title        = {Pulsed stimulated brillouin microscopy enables high-sensitivity mechanical imaging of live and fragile biological specimens},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Bio-friendly long-term subcellular dynamic recording by
self-supervised image enhancement microscopy. <em>NMETH</em>,
<em>20</em>(12), 1957–1970. (<a
href="https://doi.org/10.1038/s41592-023-02058-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescence microscopy has become an indispensable tool for revealing the dynamic regulation of cells and organelles. However, stochastic noise inherently restricts optical interrogation quality and exacerbates observation fidelity when balancing the joint demands of high frame rate, long-term recording and low phototoxicity. Here we propose DeepSeMi, a self-supervised-learning-based denoising framework capable of increasing signal-to-noise ratio by over 12 dB across various conditions. With the introduction of newly designed eccentric blind-spot convolution filters, DeepSeMi effectively denoises images with no loss of spatiotemporal resolution. In combination with confocal microscopy, DeepSeMi allows for recording organelle interactions in four colors at high frame rates across tens of thousands of frames, monitoring migrasomes and retractosomes over a half day, and imaging ultra-phototoxicity-sensitive Dictyostelium cells over thousands of frames. Through comprehensive validations across various samples and instruments, we prove DeepSeMi to be a versatile and biocompatible tool for breaking the shot-noise limit. DeepSeMi is a self-supervised denoising framework that can enhance SNR over 12 dB across diverse samples and imaging modalities. DeepSeMi enables extended longitudinal imaging of subcellular dynamics with high spatiotemporal resolution.},
  archive      = {J_NMETH},
  author       = {Zhang, Guoxun and Li, Xiaopeng and Zhang, Yuanlong and Han, Xiaofei and Li, Xinyang and Yu, Jinqiang and Liu, Boqi and Wu, Jiamin and Yu, Li and Dai, Qionghai},
  doi          = {10.1038/s41592-023-02058-9},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1957-1970},
  shortjournal = {Nat. Meth.},
  title        = {Bio-friendly long-term subcellular dynamic recording by self-supervised image enhancement microscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-fidelity 3D live-cell nanoscopy through data-driven
enhanced super-resolution radial fluctuation. <em>NMETH</em>,
<em>20</em>(12), 1949–1956. (<a
href="https://doi.org/10.1038/s41592-023-02057-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Live-cell super-resolution microscopy enables the imaging of biological structure dynamics below the diffraction limit. Here we present enhanced super-resolution radial fluctuations (eSRRF), substantially improving image fidelity and resolution compared to the original SRRF method. eSRRF incorporates automated parameter optimization based on the data itself, giving insight into the trade-off between resolution and fidelity. We demonstrate eSRRF across a range of imaging modalities and biological systems. Notably, we extend eSRRF to three dimensions by combining it with multifocus microscopy. This realizes live-cell volumetric super-resolution imaging with an acquisition speed of ~1 volume per second. eSRRF provides an accessible super-resolution approach, maximizing information extraction across varied experimental conditions while minimizing artifacts. Its optimal parameter prediction strategy is generalizable, moving toward unbiased and optimized analyses in super-resolution microscopy. Enhanced super-resolution radial fluctuations (eSRRF) offers improved image fidelity and resolution compared to the popular SRRF method and further enables volumetric live-cell super-resolution imaging at high speeds.},
  archive      = {J_NMETH},
  author       = {Laine, Romain F. and Heil, Hannah S. and Coelho, Simao and Nixon-Abell, Jonathon and Jimenez, Angélique and Wiesner, Theresa and Martínez, Damián and Galgani, Tommaso and Régnier, Louise and Stubb, Aki and Follain, Gautier and Webster, Samantha and Goyette, Jesse and Dauphin, Aurelien and Salles, Audrey and Culley, Siân and Jacquemet, Guillaume and Hajj, Bassam and Leterrier, Christophe and Henriques, Ricardo},
  doi          = {10.1038/s41592-023-02057-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1949-1956},
  shortjournal = {Nat. Meth.},
  title        = {High-fidelity 3D live-cell nanoscopy through data-driven enhanced super-resolution radial fluctuation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). DBlink: Dynamic localization microscopy in super
spatiotemporal resolution via deep learning. <em>NMETH</em>,
<em>20</em>(12), 1939–1948. (<a
href="https://doi.org/10.1038/s41592-023-01966-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule localization microscopy (SMLM) has revolutionized biological imaging, improving the spatial resolution of traditional microscopes by an order of magnitude. However, SMLM techniques require long acquisition times, typically a few minutes, to yield a single super-resolved image, because they depend on accumulation of many localizations over thousands of recorded frames. Hence, the capability of SMLM to observe dynamics at high temporal resolution has always been limited. In this work, we present DBlink, a deep-learning-based method for super spatiotemporal resolution reconstruction from SMLM data. The input to DBlink is a recorded video of SMLM data and the output is a super spatiotemporal resolution video reconstruction. We use a convolutional neural network combined with a bidirectional long short-term memory network architecture, designed for capturing long-term dependencies between different input frames. We demonstrate DBlink performance on simulated filaments and mitochondria-like structures, on experimental SMLM data under controlled motion conditions and on live-cell dynamic SMLM. DBlink’s spatiotemporal interpolation constitutes an important advance in super-resolution imaging of dynamic processes in live cells. DBlink uses deep learning to capture long-term dependencies between different frames in single-molecule localization microscopy data, yielding super spatiotemporal resolution videos of fast dynamic processes in living cells.},
  archive      = {J_NMETH},
  author       = {Saguy, Alon and Alalouf, Onit and Opatovski, Nadav and Jang, Soohyen and Heilemann, Mike and Shechtman, Yoav},
  doi          = {10.1038/s41592-023-01966-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1939-1948},
  shortjournal = {Nat. Meth.},
  title        = {DBlink: Dynamic localization microscopy in super spatiotemporal resolution via deep learning},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fluorescence to measure light intensity. <em>NMETH</em>,
<em>20</em>(12), 1930–1938. (<a
href="https://doi.org/10.1038/s41592-023-02063-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the need for quantitative measurements of light intensity across many scientific disciplines, existing technologies for measuring light dose at the sample of a fluorescence microscope cannot simultaneously retrieve light intensity along with spatial distribution over a wide range of wavelengths and intensities. To address this limitation, we developed two rapid and straightforward protocols that use organic dyes and fluorescent proteins as actinometers. The first protocol relies on molecular systems whose fluorescence intensity decays and/or rises in a monoexponential fashion when constant light is applied. The second protocol relies on a broad-absorbing photochemically inert fluorophore to back-calculate the light intensity from one wavelength to another. As a demonstration of their use, the protocols are applied to quantitatively characterize the spatial distribution of light of various fluorescence imaging systems, and to calibrate illumination of commercially available instruments and light sources. Two methods for fluorescence-based actinometry using organic dyes and photoconvertible fluorescent proteins enable rapid and precise measurement of light intensity at the sample in fluorescence microscopes.},
  archive      = {J_NMETH},
  author       = {Lahlou, Aliénor and Tehrani, Hessam Sepasi and Coghill, Ian and Shpinov, Yuriy and Mandal, Mrinal and Plamont, Marie-Aude and Aujard, Isabelle and Niu, Yuxi and Nedbal, Ladislav and Lazár, Dusan and Mahou, Pierre and Supatto, Willy and Beaurepaire, Emmanuel and Eisenmann, Isabelle and Desprat, Nicolas and Croquette, Vincent and Jeanneret, Raphaël and Le Saux, Thomas and Jullien, Ludovic},
  doi          = {10.1038/s41592-023-02063-y},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1930-1938},
  shortjournal = {Nat. Meth.},
  title        = {Fluorescence to measure light intensity},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Adaptable, turn-on maturation (ATOM) fluorescent biosensors
for multiplexed detection in cells. <em>NMETH</em>, <em>20</em>(12),
1920–1929. (<a
href="https://doi.org/10.1038/s41592-023-02065-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A grand challenge in biosensor design is to develop a single-molecule, fluorescent protein-based platform that can be easily adapted to recognize targets of choice. Here, we created a family of adaptable, turn-on maturation (ATOM) biosensors consisting of a monobody (circularly permuted at one of two positions) or a nanobody (circularly permuted at one of three positions) inserted into a fluorescent protein at one of three surface loops. Multiplexed imaging of live human cells coexpressing cyan, yellow and red ATOM sensors detected biosensor targets that were specifically localized to various subcellular compartments. Fluorescence activation involved ligand-dependent chromophore maturation with turn-on ratios of up to 62-fold in cells and 100-fold in vitro. Endoplasmic reticulum- and mitochondria-localized ATOM sensors detected ligands that were targeted to those organelles. The ATOM design was validated with three monobodies and one nanobody inserted into distinct fluorescent proteins, suggesting that customized ATOM sensors can be generated quickly. Adaptable, turn-on maturation (ATOM) biosensors use monobody or nanobody targeting to control fluorescent protein maturation for fluorescence in the presence of target biomolecules, enabling bright and specific cellular biosensing.},
  archive      = {J_NMETH},
  author       = {Sekhon, Harsimranjit and Ha, Jeung-Hoi and Presti, Maria F. and Procopio, Spencer B. and Jarvis, Ava R. and Mirsky, Paige O. and John, Anna M. and Loh, Stewart N.},
  doi          = {10.1038/s41592-023-02065-w},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1920-1929},
  shortjournal = {Nat. Meth.},
  title        = {Adaptable, turn-on maturation (ATOM) fluorescent biosensors for multiplexed detection in cells},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). NextPYP: A comprehensive and scalable platform for
characterizing protein variability in situ using single-particle
cryo-electron tomography. <em>NMETH</em>, <em>20</em>(12), 1909–1919.
(<a href="https://doi.org/10.1038/s41592-023-02045-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-particle cryo-electron tomography is an emerging technique capable of determining the structure of proteins imaged within the native context of cells at molecular resolution. While high-throughput techniques for sample preparation and tilt-series acquisition are beginning to provide sufficient data to allow structural studies of proteins at physiological concentrations, the complex data analysis pipeline and the demanding storage and computational requirements pose major barriers for the development and broader adoption of this technology. Here, we present a scalable, end-to-end framework for single-particle cryo-electron tomography data analysis from on-the-fly pre-processing of tilt series to high-resolution refinement and classification, which allows efficient analysis and visualization of datasets with hundreds of tilt series and hundreds of thousands of particles. We validate our approach using in vitro and cellular datasets, demonstrating its effectiveness at achieving high-resolution and revealing conformational heterogeneity in situ. The framework is made available through an intuitive and easy-to-use computer application, nextPYP ( http://nextpyp.app ). nextPYP is a turn-key framework for single-particle cryo-electron tomography that streamlines complex data analysis pipelines, from pre-processing of tilt series to high-resolution refinement, for efficient analysis and visualization of large datasets.},
  archive      = {J_NMETH},
  author       = {Liu, Hsuan-Fu and Zhou, Ye and Huang, Qinwen and Piland, Jonathan and Jin, Weisheng and Mandel, Justin and Du, Xiaochen and Martin, Jeffrey and Bartesaghi, Alberto},
  doi          = {10.1038/s41592-023-02045-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1909-1919},
  shortjournal = {Nat. Meth.},
  title        = {NextPYP: A comprehensive and scalable platform for characterizing protein variability in situ using single-particle cryo-electron tomography},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetically encoded multimeric tags for subcellular protein
localization in cryo-EM. <em>NMETH</em>, <em>20</em>(12), 1900–1908. (<a
href="https://doi.org/10.1038/s41592-023-02053-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron tomography (cryo-ET) allows for label-free high-resolution imaging of macromolecular assemblies in their native cellular context. However, the localization of macromolecules of interest in tomographic volumes can be challenging. Here we present a ligand-inducible labeling strategy for intracellular proteins based on fluorescent, 25-nm-sized, genetically encoded multimeric particles (GEMs). The particles exhibit recognizable structural signatures, enabling their automated detection in cryo-ET data by convolutional neural networks. The coupling of GEMs to green fluorescent protein-tagged macromolecules of interest is triggered by addition of a small-molecule ligand, allowing for time-controlled labeling to minimize disturbance to native protein function. We demonstrate the applicability of GEMs for subcellular-level localization of endogenous and overexpressed proteins across different organelles in human cells using cryo-correlative fluorescence and cryo-ET imaging. We describe means for quantifying labeling specificity and efficiency, and for systematic optimization for rare and abundant protein targets, with emphasis on assessing the potential effects of labeling on protein function. Genetically encoded multimeric particles (GEMs) are 25-nm tags with recognizable structural signatures, which can be used to label specific proteins in mammalian cells to facilitate their subcellular localization in cryo-ET.},
  archive      = {J_NMETH},
  author       = {Fung, Herman K. H. and Hayashi, Yuki and Salo, Veijo T. and Babenko, Anastasiia and Zagoriy, Ievgeniia and Brunner, Andreas and Ellenberg, Jan and Müller, Christoph W. and Cuylen-Haering, Sara and Mahamid, Julia},
  doi          = {10.1038/s41592-023-02053-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1900-1908},
  shortjournal = {Nat. Meth.},
  title        = {Genetically encoded multimeric tags for subcellular protein localization in cryo-EM},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). RNA molecular recording with an engineered RNA deaminase.
<em>NMETH</em>, <em>20</em>(12), 1887–1899. (<a
href="https://doi.org/10.1038/s41592-023-02046-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA deaminases are powerful tools for base editing and RNA molecular recording. However, the enzymes used in currently available RNA molecular recorders such as TRIBE, DART or STAMP have limitations due to RNA structure and sequence dependence. We designed a platform for directed evolution of RNA molecular recorders. We engineered an RNA A-to-I deaminase (an RNA adenosine base editor, rABE) that has high activity, low bias and low background. Using rABE, we present REMORA (RNA-encoded molecular recording in adenosines), wherein deamination by rABE writes a molecular record of RNA–protein interactions. By combining rABE with the C-to-U deaminase APOBEC1 and long-read RNA sequencing, we measured binding by two RNA-binding proteins on single messenger RNAs. Orthogonal RNA molecular recording of mammalian Pumilio proteins PUM1 and PUM2 shows that PUM1 competes with PUM2 for a subset of sites in cells. Furthermore, we identify transcript isoform-specific RNA–protein interactions driven by isoform changes distal to the binding site. The genetically encodable RNA deaminase rABE enables single-molecule identification of RNA–protein interactions with cell type specificity. An engineered RNA A-to-I deaminase (rABE) offers low sequence bias, high activity and low background for REMORA (RNA-encoded molecular recording in adenosines) and enables improved molecular recording of RNA–protein interactions.},
  archive      = {J_NMETH},
  author       = {Lin, Yizhu and Kwok, Samentha and Hein, Abigail E. and Thai, Bao Quoc and Alabi, Yewande and Ostrowski, Megan S. and Wu, Ke and Floor, Stephen N.},
  doi          = {10.1038/s41592-023-02046-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1887-1899},
  shortjournal = {Nat. Meth.},
  title        = {RNA molecular recording with an engineered RNA deaminase},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cardinal v.3: A versatile open-source software for mass
spectrometry imaging analysis. <em>NMETH</em>, <em>20</em>(12),
1883–1886. (<a
href="https://doi.org/10.1038/s41592-023-02070-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cardinal v.3 is an open-source software for reproducible analysis of mass spectrometry imaging experiments. A major update from its previous versions, Cardinal v.3 supports most mass spectrometry imaging workflows. Its analytical capabilities include advanced data processing such as mass recalibration, advanced statistical analyses such as single-ion segmentation and rough annotation-based classification, and memory-efficient analyses of large-scale multitissue experiments. Cardinal v.3 is an open-source software for reproducible analysis of mass spectrometry imaging experiments, and includes data processing features such as mass recalibration, statistical analyses such as single-ion segmentation and rough annotation-based classification, and analyses of large-scale multitissue experiments.},
  archive      = {J_NMETH},
  author       = {Bemis, Kylie Ariel and Föll, Melanie Christine and Guo, Dan and Lakkimsetty, Sai Srikanth and Vitek, Olga},
  doi          = {10.1038/s41592-023-02070-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1883-1886},
  shortjournal = {Nat. Meth.},
  title        = {Cardinal v.3: A versatile open-source software for mass spectrometry imaging analysis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fluorescent actinometers for fast and simple quantitative
measurement of light intensity. <em>NMETH</em>, <em>20</em>(12),
1881–1882. (<a
href="https://doi.org/10.1038/s41592-023-02062-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescent actinometers enable the measurement of light intensity even in the depths of samples and over wide ranges of wavelengths and intensities. We introduce two protocols to quantitatively characterize the spatial distribution of light of various fluorescence imaging systems and to calibrate the illumination of commercially available instruments and light sources.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02062-z},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1881-1882},
  shortjournal = {Nat. Meth.},
  title        = {Fluorescent actinometers for fast and simple quantitative measurement of light intensity},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Modeling the complexity of mammary gland in vitro.
<em>NMETH</em>, <em>20</em>(12), 1879–1880. (<a
href="https://doi.org/10.1038/s41592-023-02064-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new method models mammary gland in vitro during physiological and pathological conditions.},
  archive      = {J_NMETH},
  author       = {Fioramonti, Marco and Blanpain, Cédric},
  doi          = {10.1038/s41592-023-02064-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1879-1880},
  shortjournal = {Nat. Meth.},
  title        = {Modeling the complexity of mammary gland in vitro},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Movies tell us how tissues develop a 3D shape.
<em>NMETH</em>, <em>20</em>(12), 1877–1878. (<a
href="https://doi.org/10.1038/s41592-023-02102-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two new methods enable the study of the mechanics of tissue development.},
  archive      = {J_NMETH},
  author       = {Graner, François},
  doi          = {10.1038/s41592-023-02102-8},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1877-1878},
  shortjournal = {Nat. Meth.},
  title        = {Movies tell us how tissues develop a 3D shape},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetically encoded shape probes for cryogenic electron
tomography. <em>NMETH</em>, <em>20</em>(12), 1874–1876. (<a
href="https://doi.org/10.1038/s41592-023-02044-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A class of protein-based molecular shape probes move us closer toward the goal of a general, genetically encoded tagging system for cryogenic electron tomography.},
  archive      = {J_NMETH},
  author       = {Taraska, Justin W.},
  doi          = {10.1038/s41592-023-02044-1},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1874-1876},
  shortjournal = {Nat. Meth.},
  title        = {Genetically encoded shape probes for cryogenic electron tomography},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Better detection for localization microscopy.
<em>NMETH</em>, <em>20</em>(12), 1873. (<a
href="https://doi.org/10.1038/s41592-023-02132-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-023-02132-2},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1873},
  shortjournal = {Nat. Meth.},
  title        = {Better detection for localization microscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). GWAS and eQTL disparity. <em>NMETH</em>, <em>20</em>(12),
1873. (<a href="https://doi.org/10.1038/s41592-023-02133-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-023-02133-1},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1873},
  shortjournal = {Nat. Meth.},
  title        = {GWAS and eQTL disparity},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Greater diversity for lineage tracing. <em>NMETH</em>,
<em>20</em>(12), 1872. (<a
href="https://doi.org/10.1038/s41592-023-02131-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DARLIN enables the generation of a massive diversity of barcodes for in vivo lineage tracing and the combination with single-cell multi-omics measurements.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-023-02131-3},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1872},
  shortjournal = {Nat. Meth.},
  title        = {Greater diversity for lineage tracing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023q). Spatially resolved multiomics. <em>NMETH</em>,
<em>20</em>(12), 1871. (<a
href="https://doi.org/10.1038/s41592-023-02110-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatially resolved multimodal omics offers a collective way to capture molecular information in complex tissues.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-023-02110-8},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1871},
  shortjournal = {Nat. Meth.},
  title        = {Spatially resolved multiomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023j). Synthetic tissue environments. <em>NMETH</em>,
<em>20</em>(12), 1871. (<a
href="https://doi.org/10.1038/s41592-023-02111-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial ECMs enable recapitulation of tissue microenvironments.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-023-02111-7},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1871},
  shortjournal = {Nat. Meth.},
  title        = {Synthetic tissue environments},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Nanopores for sequencing proteins. <em>NMETH</em>,
<em>20</em>(12), 1870. (<a
href="https://doi.org/10.1038/s41592-023-02108-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Developments in nanopore-based peptide detection and sequencing show promise of a breakthrough.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-023-02108-2},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1870},
  shortjournal = {Nat. Meth.},
  title        = {Nanopores for sequencing proteins},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Imaging across scales. <em>NMETH</em>, <em>20</em>(12),
1870. (<a href="https://doi.org/10.1038/s41592-023-02109-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New twists on established methods and multimodal imaging are poised to bridge gaps between cellular and organismal imaging.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-023-02109-1},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1870},
  shortjournal = {Nat. Meth.},
  title        = {Imaging across scales},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Human brain mapping. <em>NMETH</em>, <em>20</em>(12), 1869.
(<a href="https://doi.org/10.1038/s41592-023-02106-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-resolution connectomics of the human brain is the next frontier in neuroscience.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-023-02106-4},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1869},
  shortjournal = {Nat. Meth.},
  title        = {Human brain mapping},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). From genome structure to function. <em>NMETH</em>,
<em>20</em>(12), 1869. (<a
href="https://doi.org/10.1038/s41592-023-02107-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in profiling chromosome structure help reveal the regulatory roles of gene organization.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-023-02107-3},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1869},
  shortjournal = {Nat. Meth.},
  title        = {From genome structure to function},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023l). Visual proteomics. <em>NMETH</em>, <em>20</em>(12), 1868.
(<a href="https://doi.org/10.1038/s41592-023-02104-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances will enable proteome-scale structure determination in cells.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-023-02104-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1868},
  shortjournal = {Nat. Meth.},
  title        = {Visual proteomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023l). Large models for genomics. <em>NMETH</em>, <em>20</em>(12),
1868. (<a href="https://doi.org/10.1038/s41592-023-02105-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models are learning the language of genomics.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-023-02105-5},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1868},
  shortjournal = {Nat. Meth.},
  title        = {Large models for genomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). “Ceci n’est pas un embryon?” The ethics of human embryo
model research. <em>NMETH</em>, <em>20</em>(12), 1863–1867. (<a
href="https://doi.org/10.1038/s41592-023-02066-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Increasingly advanced in vitro stem-cell-derived human embryo models raise novel ethical questions and shed a light on long-standing questions regarding research on human embryos.},
  archive      = {J_NMETH},
  author       = {de Graeff, Nienke and De Proost, Lien and Munsie, Megan},
  doi          = {10.1038/s41592-023-02066-9},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1863-1867},
  shortjournal = {Nat. Meth.},
  title        = {‘Ceci n’est pas un embryon?’ the ethics of human embryo model research},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mapping morphogenesis and mechanics in embryo models.
<em>NMETH</em>, <em>20</em>(12), 1859–1862. (<a
href="https://doi.org/10.1038/s41592-023-02079-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent methodological advances in measurements of geometry and forces in the early embryo and its models are enabling a deeper understanding of the complex interplay of genetics, mechanics and geometry during development.},
  archive      = {J_NMETH},
  author       = {Liu, Zong-Yuan and Trivedi, Vikas and Heemskerk, Idse},
  doi          = {10.1038/s41592-023-02079-4},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1859-1862},
  shortjournal = {Nat. Meth.},
  title        = {Mapping morphogenesis and mechanics in embryo models},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The in vitro culture of mammalian embryos. <em>NMETH</em>,
<em>20</em>(12), 1855–1858. (<a
href="https://doi.org/10.1038/s41592-023-02071-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advancements in methods that enable in vitro culture of mammalian embryos have become an essential way of investigating mammalian early embryonic development and modeling developmental and pregnancy-related disorders. Here, we discuss the recent method development in this space and analyze current challenges and future directions.},
  archive      = {J_NMETH},
  author       = {Wu, Xulun and Zhai, Jinglei and Li, Qian and Wang, Hongmei},
  doi          = {10.1038/s41592-023-02071-y},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1855-1858},
  shortjournal = {Nat. Meth.},
  title        = {The in vitro culture of mammalian embryos},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Tracing developmental lineages. <em>NMETH</em>,
<em>20</em>(12), 1852–1854. (<a
href="https://doi.org/10.1038/s41592-023-02067-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {I discuss how recent advancements in lineage tracing methods have enabled rare insight into the cell fate and trajectory during development.},
  archive      = {J_NMETH},
  author       = {Raj, Bushra},
  doi          = {10.1038/s41592-023-02067-8},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1852-1854},
  shortjournal = {Nat. Meth.},
  title        = {Tracing developmental lineages},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). How developmental cell atlases inform stem cell embryo
models. <em>NMETH</em>, <em>20</em>(12), 1849–1851. (<a
href="https://doi.org/10.1038/s41592-023-02072-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vivo developmental atlases provide a crucial reference for the new class of stem-cell-derived human embryo models, helping accelerate insights into the mechanisms of human development.},
  archive      = {J_NMETH},
  author       = {Haniffa, Muzlifah and Maartens, Aidan and Teichmann, Sarah A.},
  doi          = {10.1038/s41592-023-02072-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1849-1851},
  shortjournal = {Nat. Meth.},
  title        = {How developmental cell atlases inform stem cell embryo models},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The evolution of embryo models. <em>NMETH</em>,
<em>20</em>(12), 1844–1848. (<a
href="https://doi.org/10.1038/s41592-023-02077-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The creation of multiple stem-cell-derived models of mammalian embryogenesis is opening many new doors to study human development and brings a need for scientists to demonstrate responsible dialog over the associated ethical issues.},
  archive      = {J_NMETH},
  author       = {Zernicka-Goetz, Magdalena},
  doi          = {10.1038/s41592-023-02077-6},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1844-1848},
  shortjournal = {Nat. Meth.},
  title        = {The evolution of embryo models},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). How to build a virtual embryo. <em>NMETH</em>,
<em>20</em>(12), 1838–1843. (<a
href="https://doi.org/10.1038/s41592-023-02094-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Research with human embryos and embryo models, this year’s Method of the Year, can be fraught. In contrast, digital embryos could be studied, even perturbed, in computational what-happens-when experiments.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-02094-5},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1838-1843},
  shortjournal = {Nat. Meth.},
  title        = {How to build a virtual embryo},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). METLIN-CCS: An ion mobility spectrometry collision cross
section database. <em>NMETH</em>, <em>20</em>(12), 1836–1837. (<a
href="https://doi.org/10.1038/s41592-023-02078-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Baker, Erin S. and Hoang, Corey and Uritboonthai, Winnie and Heyman, Heino M. and Pratt, Brian and MacCoss, Michael and MacLean, Brendan and Plumb, Robert and Aisporna, Aries and Siuzdak, Gary},
  doi          = {10.1038/s41592-023-02078-5},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1836-1837},
  shortjournal = {Nat. Meth.},
  title        = {METLIN-CCS: An ion mobility spectrometry collision cross section database},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chromoscope: Interactive multiscale visualization for
structural variation in human genomes. <em>NMETH</em>, <em>20</em>(12),
1834–1835. (<a
href="https://doi.org/10.1038/s41592-023-02056-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {L’Yi, Sehi and Maziec, Dominika and Stevens, Victoria and Manz, Trevor and Veit, Alexander and Berselli, Michele and Park, Peter J. and Głodzik, Dominik and Gehlenborg, Nils},
  doi          = {10.1038/s41592-023-02056-x},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1834-1835},
  shortjournal = {Nat. Meth.},
  title        = {Chromoscope: Interactive multiscale visualization for structural variation in human genomes},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Method of the year 2023: Methods for modeling development.
<em>NMETH</em>, <em>20</em>(12), 1831–1832. (<a
href="https://doi.org/10.1038/s41592-023-02134-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vitro embryo models supported by methods development in adjacent fields have revolutionized our understanding of embryogenesis.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02134-0},
  journal      = {Nature Methods},
  month        = {12},
  number       = {12},
  pages        = {1831-1832},
  shortjournal = {Nat. Meth.},
  title        = {Method of the year 2023: Methods for modeling development},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023o). Sizing a lab. <em>NMETH</em>, <em>20</em>(12), 1833. (<a
href="https://doi.org/10.1038/s41592-023-02089-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Labs range in size, and no one size is likely to fit all. Here some researchers share their lab size experiences and preferences.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-02089-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {12},
  pages        = {1833},
  shortjournal = {Nat. Meth.},
  title        = {Sizing a lab},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Learning single-cell perturbation
responses using neural optimal transport. <em>NMETH</em>,
<em>20</em>(11), 1830. (<a
href="https://doi.org/10.1038/s41592-023-02073-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Bunne, Charlotte and Stark, Stefan G. and Gut, Gabriele and del Castillo, Jacobo Sarabia and Levesque, Mitch and Lehmann, Kjong-Van and Pelkmans, Lucas and Krause, Andreas and Rätsch, Gunnar},
  doi          = {10.1038/s41592-023-02073-w},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1830},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Learning single-cell perturbation responses using neural optimal transport},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Waxholm space atlas of the rat brain: A 3D atlas supporting
data analysis and integration. <em>NMETH</em>, <em>20</em>(11),
1822–1829. (<a
href="https://doi.org/10.1038/s41592-023-02034-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Volumetric brain atlases are increasingly used to integrate and analyze diverse experimental neuroscience data acquired from animal models, but until recently a publicly available digital atlas with complete coverage of the rat brain has been missing. Here we present an update of the Waxholm Space rat brain atlas, a comprehensive open-access volumetric atlas resource. This brain atlas features annotations of 222 structures, of which 112 are new and 57 revised compared to previous versions. It provides a detailed map of the cerebral cortex, hippocampal region, striatopallidal areas, midbrain dopaminergic system, thalamic cell groups, the auditory system and main fiber tracts. We document the criteria underlying the annotations and demonstrate how the atlas with related tools and workflows can be used to support interpretation, integration, analysis and dissemination of experimental rat brain data. An updated version of the Waxholm Space atlas of the rat brain includes more detailed annotations of several brain regions, including the cortex, striatopallidal region, midbrain and thalamus, expanding the previous version with 112 new and 57 revised structures.},
  archive      = {J_NMETH},
  author       = {Kleven, Heidi and Bjerke, Ingvild E. and Clascá, Francisco and Groenewegen, Henk J. and Bjaalie, Jan G. and Leergaard, Trygve B.},
  doi          = {10.1038/s41592-023-02034-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1822-1829},
  shortjournal = {Nat. Meth.},
  title        = {Waxholm space atlas of the rat brain: A 3D atlas supporting data analysis and integration},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Benchmarking long-read RNA-sequencing analysis tools using
in silico mixtures. <em>NMETH</em>, <em>20</em>(11), 1810–1821. (<a
href="https://doi.org/10.1038/s41592-023-02026-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The lack of benchmark data sets with inbuilt ground-truth makes it challenging to compare the performance of existing long-read isoform detection and differential expression analysis workflows. Here, we present a benchmark experiment using two human lung adenocarcinoma cell lines that were each profiled in triplicate together with synthetic, spliced, spike-in RNAs (sequins). Samples were deeply sequenced on both Illumina short-read and Oxford Nanopore Technologies long-read platforms. Alongside the ground-truth available via the sequins, we created in silico mixture samples to allow performance assessment in the absence of true positives or true negatives. Our results show that StringTie2 and bambu outperformed other tools from the six isoform detection tools tested, DESeq2, edgeR and limma-voom were best among the five differential transcript expression tools tested and there was no clear front-runner for performing differential transcript usage analysis between the five tools compared, which suggests further methods development is needed for this application. This analysis leverages experimentally sequenced data and in silico mixtures to simulate transcript expression differences, which enables a performance assessment of long-read tools developed for isoform detection, differential transcript expression analysis and differential transcript usage analysis.},
  archive      = {J_NMETH},
  author       = {Dong, Xueyi and Du, Mei R. M. and Gouil, Quentin and Tian, Luyi and Jabbari, Jafar S. and Bowden, Rory and Baldoni, Pedro L. and Chen, Yunshun and Smyth, Gordon K. and Amarasinghe, Shanika L. and Law, Charity W. and Ritchie, Matthew E.},
  doi          = {10.1038/s41592-023-02026-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1810-1821},
  shortjournal = {Nat. Meth.},
  title        = {Benchmarking long-read RNA-sequencing analysis tools using in silico mixtures},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fatigue-resistant hydrogel optical fibers enable peripheral
nerve optogenetics during locomotion. <em>NMETH</em>, <em>20</em>(11),
1802–1809. (<a
href="https://doi.org/10.1038/s41592-023-02020-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We develop soft and stretchable fatigue-resistant hydrogel optical fibers that enable optogenetic modulation of peripheral nerves in naturally behaving animals during persistent locomotion. The formation of polymeric nanocrystalline domains within the hydrogels yields fibers with low optical losses of 1.07 dB cm−1, Young’s modulus of 1.6 MPa, stretchability of 200\% and fatigue strength of 1.4 MPa against 30,000 stretch cycles. The hydrogel fibers permitted light delivery to the sciatic nerve, optogenetically activating hindlimb muscles in Thy1::ChR2 mice during 6-week voluntary wheel running assays while experiencing repeated deformation. The fibers additionally enabled optical inhibition of pain hypersensitivity in an inflammatory model in TRPV1::NpHR mice over an 8-week period. Our hydrogel fibers offer a motion-adaptable and robust solution to peripheral nerve optogenetics, facilitating the investigation of somatosensation. Flexible and fatigue-resistant optical fibers made from hydrogel allow optogenetic manipulations in the periphery in freely behaving mice.},
  archive      = {J_NMETH},
  author       = {Liu, Xinyue and Rao, Siyuan and Chen, Weixuan and Felix, Kayla and Ni, Jiahua and Sahasrabudhe, Atharva and Lin, Shaoting and Wang, Qianbin and Liu, Yuanyuan and He, Zhigang and Xu, Jingyi and Huang, Sizhe and Hong, Eunji and Yau, Todd and Anikeeva, Polina and Zhao, Xuanhe},
  doi          = {10.1038/s41592-023-02020-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1802-1809},
  shortjournal = {Nat. Meth.},
  title        = {Fatigue-resistant hydrogel optical fibers enable peripheral nerve optogenetics during locomotion},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Crystal ribcage: A platform for probing real-time lung
function at cellular resolution. <em>NMETH</em>, <em>20</em>(11),
1790–1801. (<a
href="https://doi.org/10.1038/s41592-023-02004-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the dynamic pathogenesis and treatment response in pulmonary diseases requires probing the lung at cellular resolution in real time. Despite advances in intravital imaging, optical imaging of the lung during active respiration and circulation has remained challenging. Here, we introduce the crystal ribcage: a transparent ribcage that allows multiscale optical imaging of the functioning lung from whole-organ to single-cell level. It enables the modulation of lung biophysics and immunity through intravascular, intrapulmonary, intraparenchymal and optogenetic interventions, and it preserves the three-dimensional architecture, air–liquid interface, cellular diversity and respiratory–circulatory functions of the lung. Utilizing these capabilities on murine models of pulmonary pathologies we probed remodeling of respiratory–circulatory functions at the single-alveolus and capillary levels during disease progression. The crystal ribcage and its broad applications presented here will facilitate further studies of nearly any pulmonary disease as well as lead to the identification of new targets for treatment strategies. A fabricated transparent ribcage facilitates real-time analysis of whole-lung function down to cellular resolution.},
  archive      = {J_NMETH},
  author       = {Banerji, Rohin and Grifno, Gabrielle N. and Shi, Linzheng and Smolen, Dylan and LeBourdais, Rob and Muhvich, Johnathan and Eberman, Cate and Hiller, Bradley E. and Lee, Jisu and Regan, Kathryn and Zheng, Siyi and Zhang, Sue and Jiang, John and Raslan, Ahmed A. and Breda, Julia C. and Pihl, Riley and Traber, Katrina and Mazzilli, Sarah and Ligresti, Giovanni and Mizgerd, Joseph P. and Suki, Béla and Nia, Hadi T.},
  doi          = {10.1038/s41592-023-02004-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1790-1801},
  shortjournal = {Nat. Meth.},
  title        = {Crystal ribcage: A platform for probing real-time lung function at cellular resolution},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Hydrogel-based molecular tension fluorescence microscopy for
investigating receptor-mediated rigidity sensing. <em>NMETH</em>,
<em>20</em>(11), 1780–1789. (<a
href="https://doi.org/10.1038/s41592-023-02037-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extracellular matrix (ECM) rigidity serves as a crucial mechanical cue impacting diverse biological processes. However, understanding the molecular mechanisms of rigidity sensing has been limited by the spatial resolution and force sensitivity of current cellular force measurement techniques. Here we developed a method to functionalize DNA tension probes on soft hydrogel surfaces in a controllable and reliable manner, enabling molecular tension fluorescence microscopy for rigidity sensing studies. Our findings showed that fibroblasts respond to substrate rigidity by recruiting more force-bearing integrins and modulating integrin sampling frequency of the ECM, rather than simply overloading the existing integrin–ligand bonds, to promote focal adhesion maturation. We also demonstrated that ECM rigidity positively regulates the pN force of T cell receptor–ligand bond and T cell receptor mechanical sampling frequency, promoting T cell activation. Thus, hydrogel-based molecular tension fluorescence microscopy implemented on a standard confocal microscope provides a simple and effective means to explore detailed molecular force information for rigidity-dependent biological processes. Hydrogel surfaces modified with DNA tension probes can explore extracellular rigidity sensing at the molecular level.},
  archive      = {J_NMETH},
  author       = {Wang, Wenxu and Chen, Wei and Wu, Chaoyang and Zhang, Chen and Feng, Jingjing and Liu, Pengxiang and Hu, Yuru and Li, Hongyun and Sun, Feng and Jiang, Kai and Zhang, Xinghua and Liu, Zheng},
  doi          = {10.1038/s41592-023-02037-0},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1780-1789},
  shortjournal = {Nat. Meth.},
  title        = {Hydrogel-based molecular tension fluorescence microscopy for investigating receptor-mediated rigidity sensing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Causal identification of single-cell experimental
perturbation effects with CINEMA-OT. <em>NMETH</em>, <em>20</em>(11),
1769–1779. (<a
href="https://doi.org/10.1038/s41592-023-02040-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advancements in single-cell technologies allow characterization of experimental perturbations at single-cell resolution. While methods have been developed to analyze such experiments, the application of a strict causal framework has not yet been explored for the inference of treatment effects at the single-cell level. Here we present a causal-inference-based approach to single-cell perturbation analysis, termed CINEMA-OT (causal independent effect module attribution + optimal transport). CINEMA-OT separates confounding sources of variation from perturbation effects to obtain an optimal transport matching that reflects counterfactual cell pairs. These cell pairs represent causal perturbation responses permitting a number of novel analyses, such as individual treatment-effect analysis, response clustering, attribution analysis, and synergy analysis. We benchmark CINEMA-OT on an array of treatment-effect estimation tasks for several simulated and real datasets and show that it outperforms other single-cell perturbation analysis methods. Finally, we perform CINEMA-OT analysis of two newly generated datasets: (1) rhinovirus and cigarette-smoke-exposed airway organoids, and (2) combinatorial cytokine stimulation of immune cells. In these experiments, CINEMA-OT reveals potential mechanisms by which cigarette-smoke exposure dulls the airway antiviral response, as well as the logic that governs chemokine secretion and peripheral immune cell recruitment. CINEMA-OT is a framework that enables accurate causal inference of the effects of perturbation experiments at the single-cell level.},
  archive      = {J_NMETH},
  author       = {Dong, Mingze and Wang, Bao and Wei, Jessica and de O. Fonseca, Antonio H. and Perry, Curtis J. and Frey, Alexander and Ouerghi, Feriel and Foxman, Ellen F. and Ishizuka, Jeffrey J. and Dhodapkar, Rahul M. and van Dijk, David},
  doi          = {10.1038/s41592-023-02040-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1769-1779},
  shortjournal = {Nat. Meth.},
  title        = {Causal identification of single-cell experimental perturbation effects with CINEMA-OT},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Learning single-cell perturbation responses using neural
optimal transport. <em>NMETH</em>, <em>20</em>(11), 1759–1768. (<a
href="https://doi.org/10.1038/s41592-023-01969-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding and predicting molecular responses in single cells upon chemical, genetic or mechanical perturbations is a core question in biology. Obtaining single-cell measurements typically requires the cells to be destroyed. This makes learning heterogeneous perturbation responses challenging as we only observe unpaired distributions of perturbed or non-perturbed cells. Here we leverage the theory of optimal transport and the recent advent of input convex neural architectures to present CellOT, a framework for learning the response of individual cells to a given perturbation by mapping these unpaired distributions. CellOT outperforms current methods at predicting single-cell drug responses, as profiled by scRNA-seq and a multiplexed protein-imaging technology. Further, we illustrate that CellOT generalizes well on unseen settings by (1) predicting the scRNA-seq responses of holdout patients with lupus exposed to interferon-β and patients with glioblastoma to panobinostat; (2) inferring lipopolysaccharide responses across different species; and (3) modeling the hematopoietic developmental trajectories of different subpopulations. CellOT combines the benefits of optimal transport and input convex neural architectures to directly learn and uncover maps between control and perturbed cell states at the single-cell level.},
  archive      = {J_NMETH},
  author       = {Bunne, Charlotte and Stark, Stefan G. and Gut, Gabriele and del Castillo, Jacobo Sarabia and Levesque, Mitch and Lehmann, Kjong-Van and Pelkmans, Lucas and Krause, Andreas and Rätsch, Gunnar},
  doi          = {10.1038/s41592-023-01969-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1759-1768},
  shortjournal = {Nat. Meth.},
  title        = {Learning single-cell perturbation responses using neural optimal transport},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Deep learning-driven adaptive optics for single-molecule
localization microscopy. <em>NMETH</em>, <em>20</em>(11), 1748–1758. (<a
href="https://doi.org/10.1038/s41592-023-02029-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The inhomogeneous refractive indices of biological tissues blur and distort single-molecule emission patterns generating image artifacts and decreasing the achievable resolution of single-molecule localization microscopy (SMLM). Conventional sensorless adaptive optics methods rely on iterative mirror changes and image-quality metrics. However, these metrics result in inconsistent metric responses and thus fundamentally limit their efficacy for aberration correction in tissues. To bypass iterative trial-then-evaluate processes, we developed deep learning-driven adaptive optics for SMLM to allow direct inference of wavefront distortion and near real-time compensation. Our trained deep neural network monitors the individual emission patterns from single-molecule experiments, infers their shared wavefront distortion, feeds the estimates through a dynamic filter and drives a deformable mirror to compensate sample-induced aberrations. We demonstrated that our method simultaneously estimates and compensates 28 wavefront deformation shapes and improves the resolution and fidelity of three-dimensional SMLM through &amp;gt;130-µm-thick brain tissue specimens. A deep learning approach bypasses iterative trials associated with sensorless adaptive optics to compensate for wavefront deformations when imaging biological specimens, enabling improved deep tissue localization microscopy.},
  archive      = {J_NMETH},
  author       = {Zhang, Peiyi and Ma, Donghan and Cheng, Xi and Tsai, Andy P. and Tang, Yu and Gao, Hao-Cheng and Fang, Li and Bi, Cheng and Landreth, Gary E. and Chubykin, Alexander A. and Huang, Fang},
  doi          = {10.1038/s41592-023-02029-0},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1748-1758},
  shortjournal = {Nat. Meth.},
  title        = {Deep learning-driven adaptive optics for single-molecule localization microscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CryoREAD: De novo structure modeling for nucleic acids in
cryo-EM maps using deep learning. <em>NMETH</em>, <em>20</em>(11),
1739–1747. (<a
href="https://doi.org/10.1038/s41592-023-02032-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA and RNA play fundamental roles in various cellular processes, where their three-dimensional structures provide information critical to understanding the molecular mechanisms of their functions. Although an increasing number of nucleic acid structures and their complexes with proteins are determined by cryogenic electron microscopy (cryo-EM), structure modeling for DNA and RNA remains challenging particularly when the map is determined at a resolution coarser than atomic level. Moreover, computational methods for nucleic acid structure modeling are relatively scarce. Here, we present CryoREAD, a fully automated de novo DNA/RNA atomic structure modeling method using deep learning. CryoREAD identifies phosphate, sugar and base positions in a cryo-EM map using deep learning, which are traced and modeled into a three-dimensional structure. When tested on cryo-EM maps determined at 2.0 to 5.0 Å resolution, CryoREAD built substantially more accurate models than existing methods. We also applied the method to cryo-EM maps of biomolecular complexes in severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Few methods for three-dimensional structure modeling of nucleic acids from cryo-EM data exist. CryoREAD, a fully automated DNA/RNA atomic structure modeling method based on deep learning, was developed to fill this gap.},
  archive      = {J_NMETH},
  author       = {Wang, Xiao and Terashi, Genki and Kihara, Daisuke},
  doi          = {10.1038/s41592-023-02032-5},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1739-1747},
  shortjournal = {Nat. Meth.},
  title        = {CryoREAD: De novo structure modeling for nucleic acids in cryo-EM maps using deep learning},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). OPUS-DSD: Deep structural disentanglement for cryo-EM
single-particle analysis. <em>NMETH</em>, <em>20</em>(11), 1729–1738.
(<a href="https://doi.org/10.1038/s41592-023-02031-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron microscopy (cryo-EM) captures snapshots of dynamic macromolecules, collectively illustrating the involved structural landscapes. This provides an exciting opportunity to explore the structural variations of macromolecules under study. However, traditional cryo-EM single-particle analysis often yields static structures. Here we describe OPUS-DSD, an algorithm capable of efficiently reconstructing the structural landscape embedded in cryo-EM data. OPUS-DSD uses a three-dimensional convolutional encoder–decoder architecture trained with cryo-EM images, thereby encoding structural variations into a smooth and easily analyzable low-dimension space. This space can be traversed to reconstruct continuous dynamics or clustered to identify distinct conformations. OPUS-DSD can offer meaningful insights into the structural variations of macromolecules, filling in the gaps left by traditional cryo-EM structural determination, and potentially improves the reconstruction resolution by reliably clustering similar particles within the dataset. These functionalities are especially relevant to the study of highly dynamic biological systems. OPUS-DSD is available at https://github.com/alncat/opusDSD . OPUS-DSD is a neural network-based algorithm that reconstructs distinct conformations or continuous dynamics of the macromolecular structural landscape, starting from single-particle cryo-EM data.},
  archive      = {J_NMETH},
  author       = {Luo, Zhenwei and Ni, Fengyun and Wang, Qinghua and Ma, Jianpeng},
  doi          = {10.1038/s41592-023-02031-6},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1729-1738},
  shortjournal = {Nat. Meth.},
  title        = {OPUS-DSD: Deep structural disentanglement for cryo-EM single-particle analysis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Compact engineered human mechanosensitive transactivation
modules enable potent and versatile synthetic transcriptional control.
<em>NMETH</em>, <em>20</em>(11), 1716–1728. (<a
href="https://doi.org/10.1038/s41592-023-02036-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Engineered transactivation domains (TADs) combined with programmable DNA binding platforms have revolutionized synthetic transcriptional control. Despite recent progress in programmable CRISPR–Cas-based transactivation (CRISPRa) technologies, the TADs used in these systems often contain poorly tolerated elements and/or are prohibitively large for many applications. Here, we defined and optimized minimal TADs built from human mechanosensitive transcription factors. We used these components to construct potent and compact multipartite transactivation modules (MSN, NMS and eN3x9) and to build the CRISPR–dCas9 recruited enhanced activation module (CRISPR-DREAM) platform. We found that CRISPR-DREAM was specific and robust across mammalian cell types, and efficiently stimulated transcription from diverse regulatory loci. We also showed that MSN and NMS were portable across Type I, II and V CRISPR systems, transcription activator-like effectors and zinc finger proteins. Further, as proofs of concept, we used dCas9-NMS to efficiently reprogram human fibroblasts into induced pluripotent stem cells and demonstrated that mechanosensitive transcription factor TADs are efficacious and well tolerated in therapeutically important primary human cell types. Finally, we leveraged the compact and potent features of these engineered TADs to build dual and all-in-one CRISPRa AAV systems. Altogether, these compact human TADs, fusion modules and delivery architectures should be valuable for synthetic transcriptional control in biomedical applications. Synthetic CRISPR-based transactivation domains engineered using mechanosensitive transcription factors enable robust transcriptional control.},
  archive      = {J_NMETH},
  author       = {Mahata, Barun and Cabrera, Alan and Brenner, Daniel A. and Guerra-Resendez, Rosa Selenia and Li, Jing and Goell, Jacob and Wang, Kaiyuan and Guo, Yannie and Escobar, Mario and Parthasarathy, Abinand Krishna and Szadowski, Hailey and Bedford, Guy and Reed, Daniel R. and Kim, Sunghwan and Hilton, Isaac B.},
  doi          = {10.1038/s41592-023-02036-1},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1716-1728},
  shortjournal = {Nat. Meth.},
  title        = {Compact engineered human mechanosensitive transactivation modules enable potent and versatile synthetic transcriptional control},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Streamlined and sensitive mono- and di-ribosome profiling in
yeast and human cells. <em>NMETH</em>, <em>20</em>(11), 1704–1715. (<a
href="https://doi.org/10.1038/s41592-023-02028-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ribosome profiling has unveiled diverse regulation and perturbations of translation through a transcriptome-wide survey of ribosome occupancy, read out by sequencing of ribosome-protected messenger RNA fragments. Generation of ribosome footprints and their conversion into sequencing libraries is technically demanding and sensitive to biases that distort the representation of physiological ribosome occupancy. We address these challenges by producing ribosome footprints with P1 nuclease rather than RNase I and replacing RNA ligation with ordered two-template relay, a single-tube protocol for sequencing library preparation that incorporates adaptors by reverse transcription. Our streamlined approach reduced sequence bias and enhanced enrichment of ribosome footprints relative to ribosomal RNA. Furthermore, P1 nuclease preserved distinct juxtaposed ribosome complexes informative about yeast and human ribosome fates during translation initiation, stalling and termination. Our optimized methods for mRNA footprint generation and capture provide a richer translatome profile with low input and fewer technical challenges. A comprehensive redevelopment of the ribosome profiling workflow involves improved nuclease treatment and sequencing library preparation, enabling richer and more accurate translatome profiling with lower input and fewer technical hurdles.},
  archive      = {J_NMETH},
  author       = {Ferguson, Lucas and Upton, Heather E. and Pimentel, Sydney C. and Mok, Amanda and Lareau, Liana F. and Collins, Kathleen and Ingolia, Nicholas T.},
  doi          = {10.1038/s41592-023-02028-1},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1704-1715},
  shortjournal = {Nat. Meth.},
  title        = {Streamlined and sensitive mono- and di-ribosome profiling in yeast and human cells},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A new bayesian factor analysis method improves detection of
genes and biological processes affected by perturbations in single-cell
CRISPR screening. <em>NMETH</em>, <em>20</em>(11), 1693–1703. (<a
href="https://doi.org/10.1038/s41592-023-02017-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clustered regularly interspaced short palindromic repeats (CRISPR) screening coupled with single-cell RNA sequencing has emerged as a powerful tool to characterize the effects of genetic perturbations on the whole transcriptome at a single-cell level. However, due to its sparsity and complex structure, analysis of single-cell CRISPR screening data is challenging. In particular, standard differential expression analysis methods are often underpowered to detect genes affected by CRISPR perturbations. We developed a statistical method for such data, called guided sparse factor analysis (GSFA). GSFA infers latent factors that represent coregulated genes or gene modules; by borrowing information from these factors, it infers the effects of genetic perturbations on individual genes. We demonstrated through extensive simulation studies that GSFA detects perturbation effects with much higher power than state-of-the-art methods. Using single-cell CRISPR data from human CD8+ T cells and neural progenitor cells, we showed that GSFA identified biologically relevant gene modules and specific genes affected by CRISPR perturbations, many of which were missed by existing methods, providing new insights into the functions of genes involved in T cell activation and neurodevelopment. Guided sparse factor analysis (GSFA) is a powerful statistical framework to detect changes in gene expression as a result of perturbations in single-cell CRISPR screening.},
  archive      = {J_NMETH},
  author       = {Zhou, Yifan and Luo, Kaixuan and Liang, Lifan and Chen, Mengjie and He, Xin},
  doi          = {10.1038/s41592-023-02017-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1693-1703},
  shortjournal = {Nat. Meth.},
  title        = {A new bayesian factor analysis method improves detection of genes and biological processes affected by perturbations in single-cell CRISPR screening},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Population-level integration of single-cell datasets enables
multi-scale analysis across samples. <em>NMETH</em>, <em>20</em>(11),
1683–1692. (<a
href="https://doi.org/10.1038/s41592-023-02035-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The increasing generation of population-level single-cell atlases has the potential to link sample metadata with cellular data. Constructing such references requires integration of heterogeneous cohorts with varying metadata. Here we present single-cell population level integration (scPoli), an open-world learner that incorporates generative models to learn sample and cell representations for data integration, label transfer and reference mapping. We applied scPoli on population-level atlases of lung and peripheral blood mononuclear cells, the latter consisting of 7.8 million cells across 2,375 samples. We demonstrate that scPoli can explain sample-level biological and technical variations using sample embeddings revealing genes associated with batch effects and biological effects. scPoli is further applicable to single-cell sequencing assay for transposase-accessible chromatin and cross-species datasets, offering insights into chromatin accessibility and comparative genomics. We envision scPoli becoming an important tool for population-level single-cell data integration facilitating atlas use but also interpretation by means of multi-scale analyses. By learning representations for both cells and various condition covariates, scPoli facilitates atlas-level integration and analysis of single-cell genomics datasets with improved interpretability.},
  archive      = {J_NMETH},
  author       = {De Donno, Carlo and Hediyeh-Zadeh, Soroor and Moinfar, Amir Ali and Wagenstetter, Marco and Zappia, Luke and Lotfollahi, Mohammad and Theis, Fabian J.},
  doi          = {10.1038/s41592-023-02035-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1683-1692},
  shortjournal = {Nat. Meth.},
  title        = {Population-level integration of single-cell datasets enables multi-scale analysis across samples},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Microbial-enrichment method enables high-throughput
metagenomic characterization from host-rich samples. <em>NMETH</em>,
<em>20</em>(11), 1672–1682. (<a
href="https://doi.org/10.1038/s41592-023-02025-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Host–microbe interactions have been linked to health and disease states through the use of microbial taxonomic profiling, mostly via 16S ribosomal RNA gene sequencing. However, many mechanistic insights remain elusive, in part because studying the genomes of microbes associated with mammalian tissue is difficult due to the high ratio of host to microbial DNA in such samples. Here we describe a microbial-enrichment method (MEM), which we demonstrate on a wide range of sample types, including saliva, stool, intestinal scrapings, and intestinal mucosal biopsies. MEM enabled high-throughput characterization of microbial metagenomes from human intestinal biopsies by reducing host DNA more than 1,000-fold with minimal microbial community changes (roughly 90\% of taxa had no significant differences between MEM-treated and untreated control groups). Shotgun sequencing of MEM-treated human intestinal biopsies enabled characterization of both high- and low-abundance microbial taxa, pathways and genes longitudinally along the gastrointestinal tract. We report the construction of metagenome-assembled genomes directly from human intestinal biopsies for bacteria and archaea at relative abundances as low as 1\%. Analysis of metagenome-assembled genomes reveals distinct subpopulation structures between the small and large intestine for some taxa. MEM opens a path for the microbiome field to acquire deeper insights into host–microbe interactions by enabling in-depth characterization of host-tissue-associated microbial communities. This work introduces microbial-enrichment methodology (MEM) that enables removal of host DNA in human intestinal biopsies and characterization of low-abundance microbial taxa down to 1\%.},
  archive      = {J_NMETH},
  author       = {Wu-Woods, Natalie J. and Barlow, Jacob T. and Trigodet, Florian and Shaw, Dustin G. and Romano, Anna E. and Jabri, Bana and Eren, A. Murat and Ismagilov, Rustem F.},
  doi          = {10.1038/s41592-023-02025-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1672-1682},
  shortjournal = {Nat. Meth.},
  title        = {Microbial-enrichment method enables high-throughput metagenomic characterization from host-rich samples},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Molecular mechanocytometry using tension-activated cell
tagging. <em>NMETH</em>, <em>20</em>(11), 1666–1671. (<a
href="https://doi.org/10.1038/s41592-023-02030-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Flow cytometry is used routinely to measure single-cell gene expression by staining cells with fluorescent antibodies and nucleic acids. Here, we present tension-activated cell tagging (TaCT) to label cells fluorescently based on the magnitude of molecular force transmitted through cell adhesion receptors. As a proof-of-concept, we analyzed fibroblasts and mouse platelets after TaCT using conventional flow cytometry. Tension-activated cell tagging (TaCT) is a new method that uses flow cytometry to sort mechanically active cells based on the forces generated by their surface adhesion receptors.},
  archive      = {J_NMETH},
  author       = {Ma, Rong and Rashid, Sk Aysha and Velusamy, Arventh and Deal, Brendan R. and Chen, Wenchun and Petrich, Brian and Li, Renhao and Salaita, Khalid},
  doi          = {10.1038/s41592-023-02030-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1666-1671},
  shortjournal = {Nat. Meth.},
  title        = {Molecular mechanocytometry using tension-activated cell tagging},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fast and robust metagenomic sequence comparison through
sparse chaining with skani. <em>NMETH</em>, <em>20</em>(11), 1661–1665.
(<a href="https://doi.org/10.1038/s41592-023-02018-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sequence comparison tools for metagenome-assembled genomes (MAGs) struggle with high-volume or low-quality data. We present skani ( https://github.com/bluenote-1577/skani ), a method for determining average nucleotide identity (ANI) via sparse approximate alignments. skani outperforms FastANI in accuracy and speed (&amp;gt;20× faster) for fragmented, incomplete MAGs. skani can query genomes against &amp;gt;65,000 prokaryotic genomes in seconds and 6 GB memory. skani unlocks higher-resolution insights for extensive, noisy metagenomic datasets. skani achieves fast calculation of average nucleotide identity (ANI) between metagenome-assembled genomes (MAGs), with improved robustness against incomplete and fragmented MAGs.},
  archive      = {J_NMETH},
  author       = {Shaw, Jim and Yu, Yun William},
  doi          = {10.1038/s41592-023-02018-3},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1661-1665},
  shortjournal = {Nat. Meth.},
  title        = {Fast and robust metagenomic sequence comparison through sparse chaining with skani},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Artificial intelligence-enabled quantitative phase imaging
methods for life sciences. <em>NMETH</em>, <em>20</em>(11), 1645–1660.
(<a href="https://doi.org/10.1038/s41592-023-02041-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantitative phase imaging, integrated with artificial intelligence, allows for the rapid and label-free investigation of the physiology and pathology of biological systems. This review presents the principles of various two-dimensional and three-dimensional label-free phase imaging techniques that exploit refractive index as an intrinsic optical imaging contrast. In particular, we discuss artificial intelligence-based analysis methodologies for biomedical studies including image enhancement, segmentation of cellular or subcellular structures, classification of types of biological samples and image translation to furnish subcellular and histochemical information from label-free phase images. We also discuss the advantages and challenges of artificial intelligence-enabled quantitative phase imaging analyses, summarize recent notable applications in the life sciences, and cover the potential of this field for basic and industrial research in the life sciences. This Perspective introduces advances in quantitative phase imaging and artificial intelligence-based image analysis and further describes how the two technologies intersect and synergize to enable biomedical research.},
  archive      = {J_NMETH},
  author       = {Park, Juyeon and Bai, Bijie and Ryu, DongHun and Liu, Tairan and Lee, Chungha and Luo, Yi and Lee, Mahn Jae and Huang, Luzhe and Shin, Jeongwon and Zhang, Yijie and Ryu, Dongmin and Li, Yuzhu and Kim, Geon and Min, Hyun-seok and Ozcan, Aydogan and Park, YongKeun},
  doi          = {10.1038/s41592-023-02041-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1645-1660},
  shortjournal = {Nat. Meth.},
  title        = {Artificial intelligence-enabled quantitative phase imaging methods for life sciences},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evaluating long-read RNA-sequencing analysis tools with in
silico mixtures. <em>NMETH</em>, <em>20</em>(11), 1643–1644. (<a
href="https://doi.org/10.1038/s41592-023-02027-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We conducted a comprehensive long-read RNA sequencing (RNA-seq) benchmarking experiment by combining spike-ins and in silico mixtures to establish a ground-truth dataset. We used long- and short-read RNA-seq technology to deeply sequence samples and compared the performance of a range of analysis tools on these data.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02027-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1643-1644},
  shortjournal = {Nat. Meth.},
  title        = {Evaluating long-read RNA-sequencing analysis tools with in silico mixtures},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Hydrogel fibers that enable optogenetic pain inhibition
during locomotion. <em>NMETH</em>, <em>20</em>(11), 1641–1642. (<a
href="https://doi.org/10.1038/s41592-023-02022-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed fatigue-resistant hydrogel optical fibers through the controlled growth of polymeric nanocrystalline domains to enable light delivery to peripheral nerves during locomotion. The hydrogel fibers withstand locomotion strain across more than 30,000 fiber stretch cycles and enable the optogenetic inhibition of pain hypersensitivity in naturally behaving mice.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02022-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1641-1642},
  shortjournal = {Nat. Meth.},
  title        = {Hydrogel fibers that enable optogenetic pain inhibition during locomotion},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Neural optimal transport predicts perturbation responses at
the single-cell level. <em>NMETH</em>, <em>20</em>(11), 1639–1640. (<a
href="https://doi.org/10.1038/s41592-023-01968-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed CellOT, a tool that integrates optimal transport with input convex neural networks to predict molecular responses of individual cells to various perturbations. By learning a map between the unpaired distributions of unperturbed and perturbed cells, CellOT outperforms current methods and generalizes the inference of treatment outcomes in unobserved cell types and patients.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01968-y},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1639-1640},
  shortjournal = {Nat. Meth.},
  title        = {Neural optimal transport predicts perturbation responses at the single-cell level},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CryoREAD provides fully automated DNA–RNA structure modeling
for cryo-EM maps. <em>NMETH</em>, <em>20</em>(11), 1637–1638. (<a
href="https://doi.org/10.1038/s41592-023-02033-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CryoREAD automatically builds DNA–RNA atomic structure from cryo-EM maps. Backbone accuracy is typically &amp;gt;85\% and the method is applicable for maps with RNA-only, DNA-only and DNA–RNA–protein complex structures. CryoREAD uses deep learning to identify structure information and subsequently construct the 3D structure of nucleic acids.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02033-4},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1637-1638},
  shortjournal = {Nat. Meth.},
  title        = {CryoREAD provides fully automated DNA–RNA structure modeling for cryo-EM maps},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Combining compact human protein domains with CRISPR systems
for robust gene activation. <em>NMETH</em>, <em>20</em>(11), 1635–1636.
(<a href="https://doi.org/10.1038/s41592-023-02038-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we developed synthetic transactivation domains (TADs) built from human mechanosensitive transcription factors (MTFs). By linking MTF TAD segments together, we engineered compact and potent multipartite transcriptional activation modules. We then harnessed these modules to create a CRISPR activation system, which we termed the dCas9 recruited enhanced activation module (CRISPR-DREAM).},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02038-z},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1635-1636},
  shortjournal = {Nat. Meth.},
  title        = {Combining compact human protein domains with CRISPR systems for robust gene activation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Skani enables accurate and efficient genome comparison for
modern metagenomic datasets. <em>NMETH</em>, <em>20</em>(11), 1633–1634.
(<a href="https://doi.org/10.1038/s41592-023-02019-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern high-throughput metagenomics is producing hundreds of thousands of metagenome-assembled genomes (MAGs), which is overwhelming traditional sequence-similarity search methods. We present a computational method, skani, that efficiently compares MAGs on a terabyte scale while being robust to the inherent noise in MAGs, enabling larger and more accurate analyses.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02019-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1633-1634},
  shortjournal = {Nat. Meth.},
  title        = {Skani enables accurate and efficient genome comparison for modern metagenomic datasets},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Real-time imaging of dynamic tissues. <em>NMETH</em>,
<em>20</em>(11), 1631–1632. (<a
href="https://doi.org/10.1038/s41592-023-02047-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A combined modality enables real-time imaging of mouse lungs and spans whole-organ to cellular scales.},
  archive      = {J_NMETH},
  author       = {Nichols, Joan E. and Azar, Sasha R.},
  doi          = {10.1038/s41592-023-02047-y},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1631-1632},
  shortjournal = {Nat. Meth.},
  title        = {Real-time imaging of dynamic tissues},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Factoring single-cell perturbations. <em>NMETH</em>,
<em>20</em>(11), 1629–1630. (<a
href="https://doi.org/10.1038/s41592-023-02002-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {GSFA is a statistical model to automatically detect latent factors (or gene modules) in single-cell CRISPR screening.},
  archive      = {J_NMETH},
  author       = {Song, Bicna and Li, Wei},
  doi          = {10.1038/s41592-023-02002-x},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1629-1630},
  shortjournal = {Nat. Meth.},
  title        = {Factoring single-cell perturbations},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A sound solution for deep-brain imaging. <em>NMETH</em>,
<em>20</em>(11), 1623–1628. (<a
href="https://doi.org/10.1038/s41592-023-02055-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultrasound-based modalities are revealing the brain’s inner workings with steadily increasing speed, resolution and depth.},
  archive      = {J_NMETH},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41592-023-02055-y},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1623-1628},
  shortjournal = {Nat. Meth.},
  title        = {A sound solution for deep-brain imaging},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Modeling morphogenesis. <em>NMETH</em>, <em>20</em>(11),
1622. (<a href="https://doi.org/10.1038/s41592-023-02075-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-023-02075-8},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1622},
  shortjournal = {Nat. Meth.},
  title        = {Modeling morphogenesis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Making sense of missense. <em>NMETH</em>, <em>20</em>(11),
1622. (<a href="https://doi.org/10.1038/s41592-023-02076-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-023-02076-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1622},
  shortjournal = {Nat. Meth.},
  title        = {Making sense of missense},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Capturing subcellular metal ion dynamics. <em>NMETH</em>,
<em>20</em>(11), 1621. (<a
href="https://doi.org/10.1038/s41592-023-02074-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Newly developed DNA devices can be used to study intracellular sodium and potassium ion levels with organellar resolution.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-023-02074-9},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1621},
  shortjournal = {Nat. Meth.},
  title        = {Capturing subcellular metal ion dynamics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Extending support for mouse data in the molecular signatures
database (MSigDB). <em>NMETH</em>, <em>20</em>(11), 1619–1620. (<a
href="https://doi.org/10.1038/s41592-023-02014-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Castanza, Anthony S. and Recla, Jill M. and Eby, David and Thorvaldsdóttir, Helga and Bult, Carol J. and Mesirov, Jill P.},
  doi          = {10.1038/s41592-023-02014-7},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1619-1620},
  shortjournal = {Nat. Meth.},
  title        = {Extending support for mouse data in the molecular signatures database (MSigDB)},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023v). When you lose a mentor or a colleague. <em>NMETH</em>,
<em>20</em>(11), 1618. (<a
href="https://doi.org/10.1038/s41592-023-02051-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The heartbreak of losing a mentor or a colleague to illness can be part of a researcher’s life.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-02051-2},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1618},
  shortjournal = {Nat. Meth.},
  title        = {When you lose a mentor or a colleague},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Art appreciation. <em>NMETH</em>, <em>20</em>(11), 1617. (<a
href="https://doi.org/10.1038/s41592-023-02093-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This month we continue our annual celebration of beautiful images.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02093-6},
  journal      = {Nature Methods},
  month        = {11},
  number       = {11},
  pages        = {1617},
  shortjournal = {Nat. Meth.},
  title        = {Art appreciation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Quantitative assessment of near-infrared fluorescent
proteins. <em>NMETH</em>, <em>20</em>(10), 1605–1616. (<a
href="https://doi.org/10.1038/s41592-023-01975-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent progress in fluorescent protein development has generated a large diversity of near-infrared fluorescent proteins (NIR FPs), which are rapidly becoming popular probes for a variety of imaging applications. However, the diversity of NIR FPs poses a challenge for end-users in choosing the optimal one for a given application. Here we conducted a systematic and quantitative assessment of intracellular brightness, photostability, oligomeric state, chemical stability and cytotoxicity of 22 NIR FPs in cultured mammalian cells and primary mouse neurons and identified a set of top-performing FPs including emiRFP670, miRFP680, miRFP713 and miRFP720, which can cover a majority of imaging applications. The top-performing proteins were further validated for in vivo imaging of neurons in Caenorhabditis elegans, zebrafish, and mice as well as in mice liver. We also assessed the applicability of the selected NIR FPs for multicolor imaging of fusions, expansion microscopy and two-photon imaging. This Registered Report describes an extensive comparison of 22 near-infrared fluorescent proteins in vitro, in cultured mammalian cells, and in model animals, clarifying top performers in diverse biological settings.},
  archive      = {J_NMETH},
  author       = {Zhang, Hanbin and Papadaki, Stavrini and Sun, Xiaoting and Wang, Xinyue and Drobizhev, Mikhail and Yao, Luxia and Rehbock, Michel and Köster, Reinhard W. and Wu, Lianfeng and Namikawa, Kazuhiko and Piatkevich, Kiryl D.},
  doi          = {10.1038/s41592-023-01975-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1605-1616},
  shortjournal = {Nat. Meth.},
  title        = {Quantitative assessment of near-infrared fluorescent proteins},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). D-LMBmap: A fully automated deep-learning pipeline for
whole-brain profiling of neural circuitry. <em>NMETH</em>,
<em>20</em>(10), 1593–1604. (<a
href="https://doi.org/10.1038/s41592-023-01998-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent proliferation and integration of tissue-clearing methods and light-sheet fluorescence microscopy has created new opportunities to achieve mesoscale three-dimensional whole-brain connectivity mapping with exceptionally high throughput. With the rapid generation of large, high-quality imaging datasets, downstream analysis is becoming the major technical bottleneck for mesoscale connectomics. Current computational solutions are labor intensive with limited applications because of the exhaustive manual annotation and heavily customized training. Meanwhile, whole-brain data analysis always requires combining multiple packages and secondary development by users. To address these challenges, we developed D-LMBmap, an end-to-end package providing an integrated workflow containing three modules based on deep-learning algorithms for whole-brain connectivity mapping: axon segmentation, brain region segmentation and whole-brain registration. D-LMBmap does not require manual annotation for axon segmentation and achieves quantitative analysis of whole-brain projectome in a single workflow with superior accuracy for multiple cell types in all of the modalities tested. D-LMBmap is a fully automated pipeline for mesoscale connectomics including deep-learning modules for axon segmentation, brain region segmentation and whole-brain registration. D-LMBmap works accurately across cell types and modalities.},
  archive      = {J_NMETH},
  author       = {Li, Zhongyu and Shang, Zengyi and Liu, Jingyi and Zhen, Haotian and Zhu, Entao and Zhong, Shilin and Sturgess, Robyn N. and Zhou, Yitian and Hu, Xuemeng and Zhao, Xingyue and Wu, Yi and Li, Peiqi and Lin, Rui and Ren, Jing},
  doi          = {10.1038/s41592-023-01998-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1593-1604},
  shortjournal = {Nat. Meth.},
  title        = {D-LMBmap: A fully automated deep-learning pipeline for whole-brain profiling of neural circuitry},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Statistically unbiased prediction enables accurate denoising
of voltage imaging data. <em>NMETH</em>, <em>20</em>(10), 1581–1592. (<a
href="https://doi.org/10.1038/s41592-023-02005-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we report SUPPORT (statistically unbiased prediction utilizing spatiotemporal information in imaging data), a self-supervised learning method for removing Poisson–Gaussian noise in voltage imaging data. SUPPORT is based on the insight that a pixel value in voltage imaging data is highly dependent on its spatiotemporal neighboring pixels, even when its temporally adjacent frames alone do not provide useful information for statistical prediction. Such dependency is captured and used by a convolutional neural network with a spatiotemporal blind spot to accurately denoise voltage imaging data in which the existence of the action potential in a time frame cannot be inferred by the information in other frames. Through simulations and experiments, we show that SUPPORT enables precise denoising of voltage imaging data and other types of microscopy image while preserving the underlying dynamics within the scene. Statistically unbiased prediction utilizing spatiotemporal information in imaging data (SUPPORT) is a self-supervised deep learning approach to accurately denoise voltage and calcium imaging data while preserving true dynamic signals.},
  archive      = {J_NMETH},
  author       = {Eom, Minho and Han, Seungjae and Park, Pojeong and Kim, Gyuri and Cho, Eun-Seo and Sim, Jueun and Lee, Kang-Han and Kim, Seonghoon and Tian, He and Böhm, Urs L. and Lowet, Eric and Tseng, Hua-an and Choi, Jieun and Lucia, Stephani Edwina and Ryu, Seung Hyun and Rózsa, Márton and Chang, Sunghoe and Kim, Pilhan and Han, Xue and Piatkevich, Kiryl D. and Choi, Myunghwan and Kim, Cheol-Hee and Cohen, Adam E. and Chang, Jae-Byum and Yoon, Young-Gyu},
  doi          = {10.1038/s41592-023-02005-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1581-1592},
  shortjournal = {Nat. Meth.},
  title        = {Statistically unbiased prediction enables accurate denoising of voltage imaging data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). GelMap: Intrinsic calibration and deformation mapping for
expansion microscopy. <em>NMETH</em>, <em>20</em>(10), 1573–1580. (<a
href="https://doi.org/10.1038/s41592-023-02001-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Expansion microscopy (ExM) is a powerful technique to overcome the diffraction limit of light microscopy by physically expanding biological specimen in three dimensions. Nonetheless, using ExM for quantitative or diagnostic applications requires robust quality control methods to precisely determine expansion factors and to map deformations due to anisotropic expansion. Here we present GelMap, a flexible workflow to introduce a fluorescent grid into pre-expanded hydrogels that scales with expansion and reports deformations. We demonstrate that GelMap can be used to precisely determine the local expansion factor and to correct for deformations without the use of cellular reference structures or pre-expansion ground-truth images. Moreover, we show that GelMap aids sample navigation for correlative uses of expansion microscopy. Finally, we show that GelMap is compatible with expansion of tissue and can be readily implemented as a quality control step into existing ExM workflows. The GelMap workflow adds a fluorescent grid into samples before expansion, allowing for precise determination of expansion factor and subsequent deformation correction in ExM. GelMap works with diverse samples and expansion methods.},
  archive      = {J_NMETH},
  author       = {Damstra, Hugo G. J. and Passmore, Josiah B. and Serweta, Albert K. and Koutlas, Ioannis and Burute, Mithila and Meye, Frank J. and Akhmanova, Anna and Kapitein, Lukas C.},
  doi          = {10.1038/s41592-023-02001-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1573-1580},
  shortjournal = {Nat. Meth.},
  title        = {GelMap: Intrinsic calibration and deformation mapping for expansion microscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Large stokes shift fluorescent RNAs for dual-emission
fluorescence and bioluminescence imaging in live cells. <em>NMETH</em>,
<em>20</em>(10), 1563–1572. (<a
href="https://doi.org/10.1038/s41592-023-01997-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescent RNAs, aptamers that bind and activate small fluorogenic dyes, have provided a particularly attractive approach to visualizing RNAs in live cells. However, the simultaneous imaging of multiple RNAs remains challenging due to a lack of bright and stable fluorescent RNAs with bio-orthogonality and suitable spectral properties. Here, we develop the Clivias, a series of small, monomeric and stable orange-to-red fluorescent RNAs with large Stokes shifts of up to 108 nm, enabling the simple and robust imaging of RNA with minimal perturbation of the target RNA’s localization and functionality. In combination with Pepper fluorescent RNAs, the Clivias enable the single-excitation two-emission dual-color imaging of cellular RNAs and genomic loci. Clivias can also be used to detect RNA–protein interactions by bioluminescent imaging both in live cells and in vivo. We believe that these large Stokes shift fluorescent RNAs will be useful tools for the tracking and quantification of multiple RNAs in diverse biological processes. The Clivias are a series of small, monomeric fluorescent RNAs that emit with a large Stokes shift in the orange–red. They enable multiplexed RNA imaging in live cells and BRET-based detection of protein–RNA interactions in mice.},
  archive      = {J_NMETH},
  author       = {Jiang, Li and Xie, Xin and Su, Ni and Zhang, Dasheng and Chen, Xianjun and Xu, Xiaochen and Zhang, Bibi and Huang, Kaiyi and Yu, Jingwei and Fang, Mengyue and Bao, Bingkun and Zuo, Fangting and Yang, Lipeng and Zhang, Rui and Li, Huiwen and Huang, Xinyi and Chen, Zhengda and Zeng, Qingmei and Liu, Renmei and Lin, Qiuning and Zhao, Yuzheng and Ren, Aiming and Zhu, Linyong and Yang, Yi},
  doi          = {10.1038/s41592-023-01997-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1563-1572},
  shortjournal = {Nat. Meth.},
  title        = {Large stokes shift fluorescent RNAs for dual-emission fluorescence and bioluminescence imaging in live cells},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A fluorogenic chemically induced dimerization technology for
controlling, imaging and sensing protein proximity. <em>NMETH</em>,
<em>20</em>(10), 1553–1562. (<a
href="https://doi.org/10.1038/s41592-023-01988-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular tools enabling the control and observation of the proximity of proteins are essential for studying the functional role of physical distance between two proteins. Here we present CATCHFIRE (chemically assisted tethering of chimera by fluorogenic-induced recognition), a chemically induced proximity technology with intrinsic fluorescence imaging and sensing capabilities. CATCHFIRE relies on genetic fusion to small dimerizing domains that interact upon addition of fluorogenic inducers of proximity that fluoresce upon formation of the ternary assembly, allowing real-time monitoring of the chemically induced proximity. CATCHFIRE is rapid and fully reversible and allows the control and tracking of protein localization, protein trafficking, organelle transport and cellular processes, opening new avenues for studying or controlling biological processes with high spatiotemporal resolution. Its fluorogenic nature allows the design of a new class of biosensors for the study of processes such as signal transduction and apoptosis. CATCHFIRE (chemically assisted tethering of chimera by fluorogenic-induced recognition) tools are small tags that can chemically dimerize with turn-on fluorescence, enabling simultaneous control and visualization of protein proximity.},
  archive      = {J_NMETH},
  author       = {Bottone, Sara and Joliot, Octave and Cakil, Zeyneb Vildan and El Hajji, Lina and Rakotoarison, Louise-Marie and Boncompain, Gaelle and Perez, Franck and Gautier, Arnaud},
  doi          = {10.1038/s41592-023-01988-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1553-1562},
  shortjournal = {Nat. Meth.},
  title        = {A fluorogenic chemically induced dimerization technology for controlling, imaging and sensing protein proximity},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatiotemporal, optogenetic control of gene expression in
organoids. <em>NMETH</em>, <em>20</em>(10), 1544–1552. (<a
href="https://doi.org/10.1038/s41592-023-01986-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organoids derived from stem cells have become an increasingly important tool for studying human development and modeling disease. However, methods are still needed to control and study spatiotemporal patterns of gene expression in organoids. Here we combined optogenetics and gene perturbation technologies to activate or knock-down RNA of target genes in programmable spatiotemporal patterns. To illustrate the usefulness of our approach, we locally activated Sonic Hedgehog (SHH) signaling in an organoid model for human neurodevelopment. Spatial and single-cell transcriptomic analyses showed that this local induction was sufficient to generate stereotypically patterned organoids and revealed new insights into SHH’s contribution to gene regulation in neurodevelopment. With this study, we propose optogenetic perturbations in combination with spatial transcriptomics as a powerful technology to reprogram and study cell fates and tissue patterning in organoids. A workflow combining optogenetic perturbations with spatial transcriptomics to program spatiotemporal gene expression patterns in organoids.},
  archive      = {J_NMETH},
  author       = {Legnini, Ivano and Emmenegger, Lisa and Zappulo, Alessandra and Rybak-Wolf, Agnieszka and Wurmus, Ricardo and Martinez, Anna Oliveras and Jara, Cledi Cerda and Boltengagen, Anastasiya and Hessler, Talé and Mastrobuoni, Guido and Kempa, Stefan and Zinzen, Robert and Woehler, Andrew and Rajewsky, Nikolaus},
  doi          = {10.1038/s41592-023-01986-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1544-1552},
  shortjournal = {Nat. Meth.},
  title        = {Spatiotemporal, optogenetic control of gene expression in organoids},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Correlative montage parallel array cryo-tomography for in
situ structural cell biology. <em>NMETH</em>, <em>20</em>(10),
1537–1543. (<a
href="https://doi.org/10.1038/s41592-023-01999-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Imaging large fields of view while preserving high-resolution structural information remains a challenge in low-dose cryo-electron tomography. Here we present robust tools for montage parallel array cryo-tomography (MPACT) tailored for vitrified specimens. The combination of correlative cryo-fluorescence microscopy, focused-ion-beam milling, substrate micropatterning, and MPACT supports studies that contextually define the three-dimensional architecture of cells. To further extend the flexibility of MPACT, tilt series may be processed in their entirety or as individual tiles suitable for sub-tomogram averaging, enabling efficient data processing and analysis. Montage parallel array cryo-tomography adopts principles of montage tomography via regular array beam-image-shift montage acquisition and is robust for imaging large fields of view while retaining high-resolution structural information in cryo-electron tomography.},
  archive      = {J_NMETH},
  author       = {Yang, Jie E. and Larson, Matthew R. and Sibert, Bryan S. and Kim, Joseph Y. and Parrell, Daniel and Sanchez, Juan C. and Pappas, Victoria and Kumar, Anil and Cai, Kai and Thompson, Keith and Wright, Elizabeth R.},
  doi          = {10.1038/s41592-023-01999-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1537-1543},
  shortjournal = {Nat. Meth.},
  title        = {Correlative montage parallel array cryo-tomography for in situ structural cell biology},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatial single-cell mass spectrometry defines zonation of
the hepatocyte proteome. <em>NMETH</em>, <em>20</em>(10), 1530–1536. (<a
href="https://doi.org/10.1038/s41592-023-02007-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell proteomics by mass spectrometry is emerging as a powerful and unbiased method for the characterization of biological heterogeneity. So far, it has been limited to cultured cells, whereas an expansion of the method to complex tissues would greatly enhance biological insights. Here we describe single-cell Deep Visual Proteomics (scDVP), a technology that integrates high-content imaging, laser microdissection and multiplexed mass spectrometry. scDVP resolves the context-dependent, spatial proteome of murine hepatocytes at a current depth of 1,700 proteins from a cell slice. Half of the proteome was differentially regulated in a spatial manner, with protein levels changing dramatically in proximity to the central vein. We applied machine learning to proteome classes and images, which subsequently inferred the spatial proteome from imaging data alone. scDVP is applicable to healthy and diseased tissues and complements other spatial proteomics and spatial omics technologies. Single-cell Deep Visual Proteomics integrates imaging, cell segmentation, laser microdissection and multiplexed mass spectrometry for spatial single-cell proteomics measurements in complex tissues.},
  archive      = {J_NMETH},
  author       = {Rosenberger, Florian A. and Thielert, Marvin and Strauss, Maximilian T. and Schweizer, Lisa and Ammar, Constantin and Mädler, Sophia C. and Metousis, Andreas and Skowronek, Patricia and Wahle, Maria and Madden, Katherine and Gote-Schniering, Janine and Semenova, Anna and Schiller, Herbert B. and Rodriguez, Edwin and Nordmann, Thierry M. and Mund, Andreas and Mann, Matthias},
  doi          = {10.1038/s41592-023-02007-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1530-1536},
  shortjournal = {Nat. Meth.},
  title        = {Spatial single-cell mass spectrometry defines zonation of the hepatocyte proteome},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A computational framework for the inference of protein
complex remodeling from whole-proteome measurements. <em>NMETH</em>,
<em>20</em>(10), 1523–1529. (<a
href="https://doi.org/10.1038/s41592-023-02011-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein complexes are responsible for the enactment of most cellular functions. For the protein complex to form and function, its subunits often need to be present at defined quantitative ratios. Typically, global changes in protein complex composition are assessed with experimental approaches that tend to be time consuming. Here, we have developed a computational algorithm for the detection of altered protein complexes based on the systematic assessment of subunit ratios from quantitative proteomic measurements. We applied it to measurements from breast cancer cell lines and patient biopsies and were able to identify strong remodeling of HDAC2 epigenetic complexes in more aggressive forms of cancer. The presented algorithm is available as an R package and enables the inference of changes in protein complex states by extracting functionally relevant information from bottom-up proteomic datasets. AlteredPQR is a software tool, available as an R package, to infer remodeling of protein functional modules from whole-cell or tissue lysate proteomic measurements.},
  archive      = {J_NMETH},
  author       = {Buljan, Marija and Banaei-Esfahani, Amir and Blattmann, Peter and Meier-Abt, Fabienne and Shao, Wenguang and Vitek, Olga and Tang, Hua and Aebersold, Ruedi},
  doi          = {10.1038/s41592-023-02011-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1523-1529},
  shortjournal = {Nat. Meth.},
  title        = {A computational framework for the inference of protein complex remodeling from whole-proteome measurements},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). EzMechanism: An automated tool to propose catalytic
mechanisms of enzyme reactions. <em>NMETH</em>, <em>20</em>(10),
1516–1522. (<a
href="https://doi.org/10.1038/s41592-023-02006-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over the years, hundreds of enzyme reaction mechanisms have been studied using experimental and simulation methods. This rich literature on biological catalysis is now ripe for use as the foundation of new knowledge-based approaches to investigate enzyme mechanisms. Here, we present a tool able to automatically infer mechanistic paths for a given three-dimensional active site and enzyme reaction, based on a set of catalytic rules compiled from the Mechanism and Catalytic Site Atlas, a database of enzyme mechanisms. EzMechanism (pronounced as ‘Easy’ Mechanism) is available to everyone through a web user interface. When studying a mechanism, EzMechanism facilitates and improves the generation of hypotheses, by making sure that relevant information is considered, as derived from the literature on both related and unrelated enzymes. We validated EzMechanism on a set of 62 enzymes and have identified paths for further improvement, including the need for additional and more generic catalytic rules. EzMechanism is a tool for automated prediction of the catalytic mechanisms of enzymes using their three-dimensional structures and chemical reactions as input.},
  archive      = {J_NMETH},
  author       = {Ribeiro, Antonio J. M. and Riziotis, Ioannis G. and Tyzack, Jonathan D. and Borkakoti, Neera and Thornton, Janet M.},
  doi          = {10.1038/s41592-023-02006-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1516-1522},
  shortjournal = {Nat. Meth.},
  title        = {EzMechanism: An automated tool to propose catalytic mechanisms of enzyme reactions},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Recovery of missing single-cell RNA-sequencing data with
optimized transcriptomic references. <em>NMETH</em>, <em>20</em>(10),
1506–1515. (<a
href="https://doi.org/10.1038/s41592-023-02003-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA-sequencing (scRNA-seq) is an indispensable tool for characterizing cellular diversity and generating hypotheses throughout biology. Droplet-based scRNA-seq datasets often lack expression data for genes that can be detected with other methods. Here we show that the observed sensitivity deficits stem from three sources: (1) poor annotation of 3′ gene ends; (2) issues with intronic read incorporation; and (3) gene overlap-derived read loss. We show that missing gene expression data can be recovered by optimizing the reference transcriptome for scRNA-seq through recovering false intergenic reads, implementing a hybrid pre-mRNA mapping strategy and resolving gene overlaps. We demonstrate, with a diverse collection of mouse and human tissue data, that reference optimization can substantially improve cellular profiling resolution and reveal missing cell types and marker genes. Our findings argue that transcriptomic references need to be optimized for scRNA-seq analysis and warrant a reanalysis of previously published datasets and cell atlases. This paper presents an improved approach for mapping single-cell RNA-seq reads with optimized transcriptomic references, which markedly recovers previously missing gene expression data.},
  archive      = {J_NMETH},
  author       = {Pool, Allan-Hermann and Poldsam, Helen and Chen, Sisi and Thomson, Matt and Oka, Yuki},
  doi          = {10.1038/s41592-023-02003-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1506-1515},
  shortjournal = {Nat. Meth.},
  title        = {Recovery of missing single-cell RNA-sequencing data with optimized transcriptomic references},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). ScNanoHi-c: A single-cell long-read concatemer sequencing
method to reveal high-order chromatin structures within individual
cells. <em>NMETH</em>, <em>20</em>(10), 1493–1505. (<a
href="https://doi.org/10.1038/s41592-023-01978-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The high-order three-dimensional (3D) organization of regulatory genomic elements provides a topological basis for gene regulation, but it remains unclear how multiple regulatory elements across the mammalian genome interact within an individual cell. To address this, herein, we developed scNanoHi-C, which applies Nanopore long-read sequencing to explore genome-wide proximal high-order chromatin contacts within individual cells. We show that scNanoHi-C can reliably and effectively profile 3D chromatin structures and distinguish structure subtypes among individual cells. This method could also be used to detect genomic variations, including copy-number variations and structural variations, as well as to scaffold the de novo assembly of single-cell genomes. Notably, our results suggest that extensive high-order chromatin structures exist in active chromatin regions across the genome, and multiway interactions between enhancers and their target promoters were systematically identified within individual cells. Altogether, scNanoHi-C offers new opportunities to investigate high-order 3D genome structures at the single-cell level. scNanoHi-C combines Nanopore long-read sequencing with a proximity-ligation-based Hi-C protocol to profile high-order genome structures in individual cells, enabling the capture of multiway interactions among enhancers and promoters.},
  archive      = {J_NMETH},
  author       = {Li, Wen and Lu, Jiansen and Lu, Ping and Gao, Yun and Bai, Yichen and Chen, Kexuan and Su, Xinjie and Li, Mengyao and Liu, Jun’e and Chen, Yijun and Wen, Lu and Tang, Fuchou},
  doi          = {10.1038/s41592-023-01978-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1493-1505},
  shortjournal = {Nat. Meth.},
  title        = {ScNanoHi-C: A single-cell long-read concatemer sequencing method to reveal high-order chromatin structures within individual cells},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Scalable nanopore sequencing of human genomes provides a
comprehensive view of haplotype-resolved variation and methylation.
<em>NMETH</em>, <em>20</em>(10), 1483–1492. (<a
href="https://doi.org/10.1038/s41592-023-01993-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read sequencing technologies substantially overcome the limitations of short-reads but have not been considered as a feasible replacement for population-scale projects, being a combination of too expensive, not scalable enough or too error-prone. Here we develop an efficient and scalable wet lab and computational protocol, Napu, for Oxford Nanopore Technologies long-read sequencing that seeks to address those limitations. We applied our protocol to cell lines and brain tissue samples as part of a pilot project for the National Institutes of Health Center for Alzheimer’s and Related Dementias. Using a single PromethION flow cell, we can detect single nucleotide polymorphisms with F1-score comparable to Illumina short-read sequencing. Small indel calling remains difficult within homopolymers and tandem repeats, but achieves good concordance to Illumina indel calls elsewhere. Further, we can discover structural variants with F1-score on par with state-of-the-art de novo assembly methods. Our protocol phases small and structural variants at megabase scales and produces highly accurate, haplotype-specific methylation calls. This work introduces a wet lab and computational pipeline, Napu, for small variant calling and de novo assembly of Nanopore sequencing data, which leads to comparable performances to short-read sequencing and allows for large-scale long-read sequencing projects.},
  archive      = {J_NMETH},
  author       = {Kolmogorov, Mikhail and Billingsley, Kimberley J. and Mastoras, Mira and Meredith, Melissa and Monlong, Jean and Lorig-Roach, Ryan and Asri, Mobin and Alvarez Jerez, Pilar and Malik, Laksh and Dewan, Ramita and Reed, Xylena and Genner, Rylee M. and Daida, Kensuke and Behera, Sairam and Shafin, Kishwar and Pesout, Trevor and Prabakaran, Jeshuwin and Carnevali, Paolo and Yang, Jianzhi and Rhie, Arang and Scholz, Sonja W. and Traynor, Bryan J. and Miga, Karen H. and Jain, Miten and Timp, Winston and Phillippy, Adam M. and Chaisson, Mark and Sedlazeck, Fritz J. and Blauwendraat, Cornelis and Paten, Benedict},
  doi          = {10.1038/s41592-023-01993-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1483-1492},
  shortjournal = {Nat. Meth.},
  title        = {Scalable nanopore sequencing of human genomes provides a comprehensive view of haplotype-resolved variation and methylation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rapid droplet-based mixing for single-molecule spectroscopy.
<em>NMETH</em>, <em>20</em>(10), 1479–1482. (<a
href="https://doi.org/10.1038/s41592-023-01995-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Probing non-equilibrium dynamics with single-molecule spectroscopy is important for dissecting biomolecular mechanisms. However, existing microfluidic rapid-mixing systems for this purpose are incompatible with surface-adhesive biomolecules, exhibit undesirable flow dispersion and are often demanding to fabricate. Here we introduce droplet-based microfluidic mixing for single-molecule spectroscopy to overcome these limitations in a wide range of applications. We demonstrate its robust functionality with binding kinetics of even very surface-adhesive proteins on the millisecond timescale. Droplet-based microfluidics enable rapid mixing with millisecond dead times and allow single-molecule measurements of non-equilibrium binding kinetics on even challenging, strongly adsorptive samples, such as intrinsically disordered proteins.},
  archive      = {J_NMETH},
  author       = {Yang, Tianjin and Buholzer, Karin J. and Sottini, Andrea and Cao, Xiaobao and deMello, Andrew and Nettels, Daniel and Schuler, Benjamin},
  doi          = {10.1038/s41592-023-01995-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1479-1482},
  shortjournal = {Nat. Meth.},
  title        = {Rapid droplet-based mixing for single-molecule spectroscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Flash entropy search to query all mass spectral libraries in
real time. <em>NMETH</em>, <em>20</em>(10), 1475–1478. (<a
href="https://doi.org/10.1038/s41592-023-02012-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Public repositories of metabolomics mass spectra encompass more than 1 billion entries. With open search, dot product or entropy similarity, comparisons of a single tandem mass spectrometry spectrum take more than 8 h. Flash entropy search speeds up calculations more than 10,000 times to query 1 billion spectra in less than 2 s, without loss in accuracy. It benefits from using multiple threads and GPU calculations. This algorithm can fully exploit large spectral libraries with little memory overhead for any mass spectrometry laboratory. Flash entropy speeds up mass spectral similarity searches 10,000-fold, including for open search, hybrid search and neutral loss search.},
  archive      = {J_NMETH},
  author       = {Li, Yuanyue and Fiehn, Oliver},
  doi          = {10.1038/s41592-023-02012-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1475-1478},
  shortjournal = {Nat. Meth.},
  title        = {Flash entropy search to query all mass spectral libraries in real time},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Principles and challenges of modeling temporal and spatial
omics data. <em>NMETH</em>, <em>20</em>(10), 1462–1474. (<a
href="https://doi.org/10.1038/s41592-023-01992-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Studies with temporal or spatial resolution are crucial to understand the molecular dynamics and spatial dependencies underlying a biological process or system. With advances in high-throughput omic technologies, time- and space-resolved molecular measurements at scale are increasingly accessible, providing new opportunities to study the role of timing or structure in a wide range of biological questions. At the same time, analyses of the data being generated in the context of spatiotemporal studies entail new challenges that need to be considered, including the need to account for temporal and spatial dependencies and compare them across different scales, biological samples or conditions. In this Review, we provide an overview of common principles and challenges in the analysis of temporal and spatial omics data. We discuss statistical concepts to model temporal and spatial dependencies and highlight opportunities for adapting existing analysis methods to data with temporal and spatial dimensions. This Review discusses statistical and computational strategies for analyzing various spatial and temporal omics data types, with an emphasis on the common modeling principles.},
  archive      = {J_NMETH},
  author       = {Velten, Britta and Stegle, Oliver},
  doi          = {10.1038/s41592-023-01992-y},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1462-1474},
  shortjournal = {Nat. Meth.},
  title        = {Principles and challenges of modeling temporal and spatial omics data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mapping deformations and increasing quantitative accuracy in
expansion microscopy. <em>NMETH</em>, <em>20</em>(10), 1460–1461. (<a
href="https://doi.org/10.1038/s41592-023-02015-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce GelMap, a flexible workflow for reporting deformations and anisotropy in expansion microscopy. By intrinsically calibrating the expansion hydrogel using a fluorescent grid that scales with expansion and deforms with anisotropy, GelMap enables the reliable quantification of expansion factors and correction of deformations.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02015-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1460-1461},
  shortjournal = {Nat. Meth.},
  title        = {Mapping deformations and increasing quantitative accuracy in expansion microscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Capturing detailed cellular landscapes by montage
cryo-electron tomography. <em>NMETH</em>, <em>20</em>(10), 1458–1459.
(<a href="https://doi.org/10.1038/s41592-023-02000-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To capture expansive, seamless fields of view from frozen hydrated specimens by cryo-electron tomography, we developed methods for the collection and processing of montage data. This approach enables rapid acquisition of contiguous regions of specimens using a montaged tilt series collection scheme.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02000-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1458-1459},
  shortjournal = {Nat. Meth.},
  title        = {Capturing detailed cellular landscapes by montage cryo-electron tomography},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). ScNanoHi-c uncovers single-cell high-order chromatin
structures and gene regulation. <em>NMETH</em>, <em>20</em>(10),
1456–1457. (<a
href="https://doi.org/10.1038/s41592-023-01979-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Leveraging nanopore long-read sequencing, scNanoHi-C identifies multiway interactions between enhancers and their target promoters within a single cell. Compared with short-read-based single-cell Hi-C or population-based multiway sequencing methods, scNanoHi-C offers new opportunities to investigate the heterogeneities of single-cell gene regulation networks mediated by high-order 3D chromatin structures.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01979-9},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1456-1457},
  shortjournal = {Nat. Meth.},
  title        = {ScNanoHi-C uncovers single-cell high-order chromatin structures and gene regulation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fluorogenic chemically induced dimerization. <em>NMETH</em>,
<em>20</em>(10), 1454–1455. (<a
href="https://doi.org/10.1038/s41592-023-01989-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new chemically induced dimerization (CID) pair exhibits fluorescence upon dimerization for the first time. Moreover, the CID pair is small and offers easily reversible dimerization that can be repeated multiple times.},
  archive      = {J_NMETH},
  author       = {Wang, Wenjing and Shen, Jiaqi},
  doi          = {10.1038/s41592-023-01989-7},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1454-1455},
  shortjournal = {Nat. Meth.},
  title        = {Fluorogenic chemically induced dimerization},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Greening the lab. <em>NMETH</em>, <em>20</em>(10),
1449–1453. (<a
href="https://doi.org/10.1038/s41592-023-02024-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Making biological research more sustainable requires an accurate assessment of its environmental impact, both at the bench and on the computer.},
  archive      = {J_NMETH},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41592-023-02024-5},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1449-1453},
  shortjournal = {Nat. Meth.},
  title        = {Greening the lab},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023p). Resolving epigenetic base ambiguity. <em>NMETH</em>,
<em>20</em>(10), 1448. (<a
href="https://doi.org/10.1038/s41592-023-02049-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-023-02049-w},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1448},
  shortjournal = {Nat. Meth.},
  title        = {Resolving epigenetic base ambiguity},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023k). Large model predicts variant effects. <em>NMETH</em>,
<em>20</em>(10), 1448. (<a
href="https://doi.org/10.1038/s41592-023-02050-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-023-02050-3},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1448},
  shortjournal = {Nat. Meth.},
  title        = {Large model predicts variant effects},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Droplet-based power sources for modulating neural activity.
<em>NMETH</em>, <em>20</em>(10), 1447. (<a
href="https://doi.org/10.1038/s41592-023-02048-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Miniaturized power sources consisting of a few droplets can generate enough power to modulate neuronal activity.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-023-02048-x},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1447},
  shortjournal = {Nat. Meth.},
  title        = {Droplet-based power sources for modulating neural activity},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). ScNodes: A correlation and processing toolkit for
super-resolution fluorescence and electron microscopy. <em>NMETH</em>,
<em>20</em>(10), 1445–1446. (<a
href="https://doi.org/10.1038/s41592-023-01991-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Last, Mart G. F. and Voortman, Lenard M. and Sharp, Thomas H.},
  doi          = {10.1038/s41592-023-01991-z},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1445-1446},
  shortjournal = {Nat. Meth.},
  title        = {ScNodes: A correlation and processing toolkit for super-resolution fluorescence and electron microscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Napari-imagej: ImageJ ecosystem access from napari.
<em>NMETH</em>, <em>20</em>(10), 1443–1444. (<a
href="https://doi.org/10.1038/s41592-023-01990-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Selzer, Gabriel J. and Rueden, Curtis T. and Hiner, Mark C. and Evans, Edward L. and Harrington, Kyle I. S. and Eliceiri, Kevin W.},
  doi          = {10.1038/s41592-023-01990-0},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1443-1444},
  shortjournal = {Nat. Meth.},
  title        = {Napari-imagej: ImageJ ecosystem access from napari},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chlamydomonas reinhardtii: A model for photosynthesis and so
much more. <em>NMETH</em>, <em>20</em>(10), 1441–1442. (<a
href="https://doi.org/10.1038/s41592-023-02023-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The green alga Chlamydomonas reinhardtii is a useful reference organism for studying photosynthesis, cilia and the cell cycle. Like many other algae, it exhibits daily rhythms in gene expression and behavior that are in sync with the rising and setting of the sun.},
  archive      = {J_NMETH},
  author       = {Dupuis, Sunnyjoy and Merchant, Sabeeha S.},
  doi          = {10.1038/s41592-023-02023-6},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1441-1442},
  shortjournal = {Nat. Meth.},
  title        = {Chlamydomonas reinhardtii: A model for photosynthesis and so much more},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023u). When housing is insecure. <em>NMETH</em>, <em>20</em>(10),
1440. (<a href="https://doi.org/10.1038/s41592-023-02013-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Science needs diversity, but barriers such as housing insecurity can hinder access to education and training. To lobby for change, students put their own experiences to work.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-02013-8},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1440},
  shortjournal = {Nat. Meth.},
  title        = {When housing is insecure},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Registered reports: What we’ve learned so far.
<em>NMETH</em>, <em>20</em>(10), 1439. (<a
href="https://doi.org/10.1038/s41592-023-02052-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Methods is proud to publish our very first Registered Report in this issue. Here, we reflect on what we have learned since introducing this article type.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02052-1},
  journal      = {Nature Methods},
  month        = {10},
  number       = {10},
  pages        = {1439},
  shortjournal = {Nat. Meth.},
  title        = {Registered reports: What we’ve learned so far},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Open-3DSIM: An open-source
three-dimensional structured illumination microscopy reconstruction
platform. <em>NMETH</em>, <em>20</em>(9), 1437. (<a
href="https://doi.org/10.1038/s41592-023-01996-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Cao, Ruijie and Li, Yaning and Chen, Xin and Ge, Xichuan and Li, Meiqi and Guan, Meiling and Hou, Yiwei and Fu, Yunzhe and Xu, Xinzhu and Leterrier, Christophe and Jiang, Shan and Gao, Baoxiang and Xi, Peng},
  doi          = {10.1038/s41592-023-01996-8},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1437},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: open-3DSIM: an open-source three-dimensional structured illumination microscopy reconstruction platform},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Sensitive multicolor indicators for monitoring
norepinephrine in vivo. <em>NMETH</em>, <em>20</em>(9), 1426–1436. (<a
href="https://doi.org/10.1038/s41592-023-01959-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetically encoded indicators engineered from G-protein-coupled receptors are important tools that enable high-resolution in vivo neuromodulator imaging. Here, we introduce a family of sensitive multicolor norepinephrine (NE) indicators, which includes nLightG (green) and nLightR (red). These tools report endogenous NE release in vitro, ex vivo and in vivo with improved sensitivity, ligand selectivity and kinetics, as well as a distinct pharmacological profile compared with previous state-of-the-art GRABNE indicators. Using in vivo multisite fiber photometry recordings of nLightG, we could simultaneously monitor optogenetically evoked NE release in the mouse locus coeruleus and hippocampus. Two-photon imaging of nLightG revealed locomotion and reward-related NE transients in the dorsal CA1 area of the hippocampus. Thus, the sensitive NE indicators introduced here represent an important addition to the current repertoire of indicators and provide the means for a thorough investigation of the NE system. Red and green genetically encoded indicators for norepinephrine have been developed and employed to monitor norepinephrine during locomotion and reward behavior in mice. The strategy used for generating these indicators also produced indicators for other neuromodulators.},
  archive      = {J_NMETH},
  author       = {Kagiampaki, Zacharoula and Rohner, Valentin and Kiss, Cedric and Curreli, Sebastiano and Dieter, Alexander and Wilhelm, Maria and Harada, Masaya and Duss, Sian N. and Dernic, Jan and Bhat, Musadiq A. and Zhou, Xuehan and Ravotto, Luca and Ziebarth, Tim and Wasielewski, Laura Moreno and Sönmez, Latife and Benke, Dietmar and Weber, Bruno and Bohacek, Johannes and Reiner, Andreas and Wiegert, J. Simon and Fellin, Tommaso and Patriarchi, Tommaso},
  doi          = {10.1038/s41592-023-01959-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1426-1436},
  shortjournal = {Nat. Meth.},
  title        = {Sensitive multicolor indicators for monitoring norepinephrine in vivo},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). FIOLA: An accelerated pipeline for fluorescence imaging
online analysis. <em>NMETH</em>, <em>20</em>(9), 1417–1425. (<a
href="https://doi.org/10.1038/s41592-023-01964-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical microscopy methods such as calcium and voltage imaging enable fast activity readout of large neuronal populations using light. However, the lack of corresponding advances in online algorithms has slowed progress in retrieving information about neural activity during or shortly after an experiment. This gap not only prevents the execution of real-time closed-loop experiments, but also hampers fast experiment–analysis–theory turnover for high-throughput imaging modalities. Reliable extraction of neural activity from fluorescence imaging frames at speeds compatible with indicator dynamics and imaging modalities poses a challenge. We therefore developed FIOLA, a framework for fluorescence imaging online analysis that extracts neuronal activity from calcium and voltage imaging movies at speeds one order of magnitude faster than state-of-the-art methods. FIOLA exploits algorithms optimized for parallel processing on GPUs and CPUs. We demonstrate reliable and scalable performance of FIOLA on both simulated and real calcium and voltage imaging datasets. Finally, we present an online experimental scenario to provide guidance in setting FIOLA parameters and to highlight the trade-offs of our approach. FIOLA is a pipeline for processing calcium or voltage imaging data. Its advantages include the fast speed and online processing.},
  archive      = {J_NMETH},
  author       = {Cai, Changjia and Dong, Cynthia and Friedrich, Johannes and Rozsa, Marton and Pnevmatikakis, Eftychios A. and Giovannucci, Andrea},
  doi          = {10.1038/s41592-023-01964-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1417-1425},
  shortjournal = {Nat. Meth.},
  title        = {FIOLA: An accelerated pipeline for fluorescence imaging online analysis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Voltage-seq: All-optical postsynaptic connectome-guided
single-cell transcriptomics. <em>NMETH</em>, <em>20</em>(9), 1409–1416.
(<a href="https://doi.org/10.1038/s41592-023-01965-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the routing of neuronal information requires the functional characterization of connections. Neuronal projections recruit large postsynaptic ensembles with distinct postsynaptic response types (PRTs). PRT is typically probed by low-throughput whole-cell electrophysiology and is not a selection criterion for single-cell RNA-sequencing (scRNA-seq). To overcome these limitations and target neurons based on specific PRTs for soma harvesting and subsequent scRNA-seq, we created Voltage-Seq. We established all-optical voltage imaging and recorded the PRT of 8,347 neurons in the mouse periaqueductal gray (PAG) evoked by the optogenetic activation of ventromedial hypothalamic (VMH) terminals. PRTs were classified and spatially resolved in the entire VMH-PAG connectome. We built an onsite analysis tool named VoltView to navigate soma harvesting towards target PRTs guided by a classifier that used the VMH-PAG connectome database as a reference. We demonstrated Voltage-seq by locating VMH-driven γ-aminobutyric acid-ergic neurons in the PAG, guided solely by the onsite classification in VoltView. Voltage-Seq combines voltage imaging, optogenetics and single-cell RNA-seq for high-throughput analysis of functional and transcriptomic properties of neurons in situ.},
  archive      = {J_NMETH},
  author       = {Csillag, Veronika and Bizzozzero, Marianne Hiriart and Noble, J. C. and Reinius, Björn and Fuzik, János},
  doi          = {10.1038/s41592-023-01965-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1409-1416},
  shortjournal = {Nat. Meth.},
  title        = {Voltage-seq: All-optical postsynaptic connectome-guided single-cell transcriptomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Time-resolved cryo-EM using a combination of droplet
microfluidics with on-demand jetting. <em>NMETH</em>, <em>20</em>(9),
1400–1408. (<a
href="https://doi.org/10.1038/s41592-023-01967-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-particle cryogenic electron microscopy (cryo-EM) allows reconstruction of high-resolution structures of proteins in different conformations. Protein function often involves transient functional conformations, which can be resolved using time-resolved cryo-EM (trEM). In trEM, reactions are arrested after a defined delay time by rapid vitrification of protein solution on the EM grid. Despite the increasing interest in trEM among the cryo-EM community, making trEM samples with a time resolution below 100 ms remains challenging. Here we report the design and the realization of a time-resolved cryo-plunger that combines a droplet-based microfluidic mixer with a laser-induced generator of microjets that allows rapid reaction initiation and plunge-freezing of cryo-EM grids. Using this approach, a time resolution of 5 ms was achieved and the protein density map was reconstructed to a resolution of 2.1 Å. trEM experiments on GroEL:GroES chaperonin complex resolved the kinetics of the complex formation and visualized putative short-lived conformations of GroEL–ATP complex. This article describes a time-resolved cryo-plunger that combines a droplet-based microfluidic mixer with a laser-induced generator of microjets that allows rapid reaction initiation and plunge-freezing of cryo-EM grids. A time resolution of 5 ms was achieved using this approach.},
  archive      = {J_NMETH},
  author       = {Torino, Stefania and Dhurandhar, Mugdha and Stroobants, Annelore and Claessens, Raf and Efremov, Rouslan G.},
  doi          = {10.1038/s41592-023-01967-z},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1400-1408},
  shortjournal = {Nat. Meth.},
  title        = {Time-resolved cryo-EM using a combination of droplet microfluidics with on-demand jetting},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Efficient high-precision homology-directed repair-dependent
genome editing by HDRobust. <em>NMETH</em>, <em>20</em>(9), 1388–1399.
(<a href="https://doi.org/10.1038/s41592-023-01949-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Homology-directed repair (HDR), a method for repair of DNA double-stranded breaks can be leveraged for the precise introduction of mutations supplied by synthetic DNA donors, but remains limited by low efficiency and off-target effects. In this study, we report HDRobust, a high-precision method that, via the combined transient inhibition of nonhomologous end joining and microhomology-mediated end joining, resulted in the induction of point mutations by HDR in up to 93\% (median 60\%, s.e.m. 3) of chromosomes in populations of cells. We found that, using this method, insertions, deletions and rearrangements at the target site, as well as unintended changes at other genomic sites, were largely abolished. We validated this approach for 58 different target sites and showed that it allows efficient correction of pathogenic mutations in cells derived from patients suffering from anemia, sickle cell disease and thrombophilia. HDRobust is a high-precision tool for homology-directed repair-mediated single-nucleotide editing.},
  archive      = {J_NMETH},
  author       = {Riesenberg, Stephan and Kanis, Philipp and Macak, Dominik and Wollny, Damian and Düsterhöft, Dorothee and Kowalewski, Johannes and Helmbrecht, Nelly and Maricic, Tomislav and Pääbo, Svante},
  doi          = {10.1038/s41592-023-01949-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1388-1399},
  shortjournal = {Nat. Meth.},
  title        = {Efficient high-precision homology-directed repair-dependent genome editing by HDRobust},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Alignment of spatial genomics data using deep gaussian
processes. <em>NMETH</em>, <em>20</em>(9), 1379–1387. (<a
href="https://doi.org/10.1038/s41592-023-01972-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatially resolved genomic technologies have allowed us to study the physical organization of cells and tissues, and promise an understanding of local interactions between cells. However, it remains difficult to precisely align spatial observations across slices, samples, scales, individuals and technologies. Here, we propose a probabilistic model that aligns spatially-resolved samples onto a known or unknown common coordinate system (CCS) with respect to phenotypic readouts (for example, gene expression). Our method, Gaussian Process Spatial Alignment (GPSA), consists of a two-layer Gaussian process: the first layer maps observed samples’ spatial locations onto a CCS, and the second layer maps from the CCS to the observed readouts. Our approach enables complex downstream spatially aware analyses that are impossible or inaccurate with unaligned data, including an analysis of variance, creation of a dense three-dimensional (3D) atlas from sparse two-dimensional (2D) slices or association tests across data modalities. Gaussian Process Spatial Alignment (GPSA) aligns multiple spatially resolved genomics and histology datasets and improves downstream analysis.},
  archive      = {J_NMETH},
  author       = {Jones, Andrew and Townes, F. William and Li, Didong and Engelhardt, Barbara E.},
  doi          = {10.1038/s41592-023-01972-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1379-1387},
  shortjournal = {Nat. Meth.},
  title        = {Alignment of spatial genomics data using deep gaussian processes},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dictys: Dynamic gene regulatory network dissects
developmental continuum with single-cell multiomics. <em>NMETH</em>,
<em>20</em>(9), 1368–1378. (<a
href="https://doi.org/10.1038/s41592-023-01971-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene regulatory networks (GRNs) are key determinants of cell function and identity and are dynamically rewired during development and disease. Despite decades of advancement, challenges remain in GRN inference, including dynamic rewiring, causal inference, feedback loop modeling and context specificity. To address these challenges, we develop Dictys, a dynamic GRN inference and analysis method that leverages multiomic single-cell assays of chromatin accessibility and gene expression, context-specific transcription factor footprinting, stochastic process network and efficient probabilistic modeling of single-cell RNA-sequencing read counts. Dictys improves GRN reconstruction accuracy and reproducibility and enables the inference and comparative analysis of context-specific and dynamic GRNs across developmental contexts. Dictys’ network analyses recover unique insights in human blood and mouse skin development with cell-type-specific and dynamic GRNs. Its dynamic network visualizations enable time-resolved discovery and investigation of developmental driver transcription factors and their regulated targets. Dictys is available as a free, open-source and user-friendly Python package. By probabilistic modeling of gene regulation and expression kinetics, Dictys infers dynamic and context-specific gene regulatory networks using single-cell multiomics data.},
  archive      = {J_NMETH},
  author       = {Wang, Lingfei and Trasanidis, Nikolaos and Wu, Ting and Dong, Guanlan and Hu, Michael and Bauer, Daniel E. and Pinello, Luca},
  doi          = {10.1038/s41592-023-01971-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1368-1378},
  shortjournal = {Nat. Meth.},
  title        = {Dictys: Dynamic gene regulatory network dissects developmental continuum with single-cell multiomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). SCENIC+: Single-cell multiomic inference of enhancers and
gene regulatory networks. <em>NMETH</em>, <em>20</em>(9), 1355–1367. (<a
href="https://doi.org/10.1038/s41592-023-01938-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Joint profiling of chromatin accessibility and gene expression in individual cells provides an opportunity to decipher enhancer-driven gene regulatory networks (GRNs). Here we present a method for the inference of enhancer-driven GRNs, called SCENIC+. SCENIC+ predicts genomic enhancers along with candidate upstream transcription factors (TFs) and links these enhancers to candidate target genes. To improve both recall and precision of TF identification, we curated and clustered a motif collection with more than 30,000 motifs. We benchmarked SCENIC+ on diverse datasets from different species, including human peripheral blood mononuclear cells, ENCODE cell lines, melanoma cell states and Drosophila retinal development. Next, we exploit SCENIC+ predictions to study conserved TFs, enhancers and GRNs between human and mouse cell types in the cerebral cortex. Finally, we use SCENIC+ to study the dynamics of gene regulation along differentiation trajectories and the effect of TF perturbations on cell state. SCENIC+ is available at scenicplus.readthedocs.io . SCENIC+ is a comprehensive toolbox for inferring and analyzing enhancer-driven gene regulatory networks using single-cell multiomic data.},
  archive      = {J_NMETH},
  author       = {Bravo González-Blas, Carmen and De Winter, Seppe and Hulselmans, Gert and Hecker, Nikolai and Matetovici, Irina and Christiaens, Valerie and Poovathingal, Suresh and Wouters, Jasper and Aibar, Sara and Aerts, Stein},
  doi          = {10.1038/s41592-023-01938-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1355-1367},
  shortjournal = {Nat. Meth.},
  title        = {SCENIC+: Single-cell multiomic inference of enhancers and gene regulatory networks},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). UniAligner: A parameter-free framework for fast sequence
alignment. <em>NMETH</em>, <em>20</em>(9), 1346–1354. (<a
href="https://doi.org/10.1038/s41592-023-01970-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Even though the recent advances in ‘complete genomics’ revealed the previously inaccessible genomic regions, analysis of variations in centromeres and other extra-long tandem repeats (ETRs) faces an algorithmic challenge since there are currently no tools for accurate sequence comparison of ETRs. Counterintuitively, the classical alignment approaches, such as the Smith–Waterman algorithm, fail to construct biologically adequate alignments of ETRs. We present UniAligner—the parameter-free sequence alignment algorithm with sequence-dependent alignment scoring that automatically changes for any pair of compared sequences. UniAligner prioritizes matches of rare substrings that are more likely to be relevant to the evolutionary relationship between two sequences. We apply UniAligner to estimate the mutation rates in human centromeres, and quantify the extremely high rate of large duplications and deletions in centromeres. This high rate suggests that centromeres may represent some of the most rapidly evolving regions of the human genome with respect to their structural organization. Compared to other sequences, extra-long tandem repeats, such as centromeres and immunoglobulin loci, are more difficult to align. This study presents UniAligner, a computational method for efficiently and accurately aligning extra-long tandem repeats, facilitating analysis of their variation and evolution.},
  archive      = {J_NMETH},
  author       = {Bzikadze, Andrey V. and Pevzner, Pavel A.},
  doi          = {10.1038/s41592-023-01970-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1346-1354},
  shortjournal = {Nat. Meth.},
  title        = {UniAligner: A parameter-free framework for fast sequence alignment},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Isolating salient variations of interest in single-cell data
with contrastiveVI. <em>NMETH</em>, <em>20</em>(9), 1336–1345. (<a
href="https://doi.org/10.1038/s41592-023-01955-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell datasets are routinely collected to investigate changes in cellular state between control cells and the corresponding cells in a treatment condition, such as exposure to a drug or infection by a pathogen. To better understand heterogeneity in treatment response, it is desirable to deconvolve variations enriched in treated cells from those shared with controls. However, standard computational models of single-cell data are not designed to explicitly separate these variations. Here, we introduce contrastive variational inference (contrastiveVI; https://github.com/suinleelab/contrastiveVI ), a framework for deconvolving variations in treatment–control single-cell RNA sequencing (scRNA-seq) datasets into shared and treatment-specific latent variables. Using three treatment–control scRNA-seq datasets, we apply contrastiveVI to perform a variety of analysis tasks, including visualization, clustering and differential expression testing. We find that contrastiveVI consistently achieves results that agree with known ground truths and often highlights subtle phenomena that may be difficult to ascertain with standard workflows. We conclude by generalizing contrastiveVI to accommodate joint transcriptome and surface protein measurements. contrastiveVI separates single-cell gene expression variations uniquely related to treatment conditions from those shared with control conditions using a probabilistic latent variable model.},
  archive      = {J_NMETH},
  author       = {Weinberger, Ethan and Lin, Chris and Lee, Su-In},
  doi          = {10.1038/s41592-023-01955-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1336-1345},
  shortjournal = {Nat. Meth.},
  title        = {Isolating salient variations of interest in single-cell data with contrastiveVI},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Unsupervised removal of systematic background noise from
droplet-based single-cell experiments using CellBender. <em>NMETH</em>,
<em>20</em>(9), 1323–1335. (<a
href="https://doi.org/10.1038/s41592-023-01943-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Droplet-based single-cell assays, including single-cell RNA sequencing (scRNA-seq), single-nucleus RNA sequencing (snRNA-seq) and cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq), generate considerable background noise counts, the hallmark of which is nonzero counts in cell-free droplets and off-target gene expression in unexpected cell types. Such systematic background noise can lead to batch effects and spurious differential gene expression results. Here we develop a deep generative model based on the phenomenology of noise generation in droplet-based assays. The proposed model accurately distinguishes cell-containing droplets from cell-free droplets, learns the background noise profile and provides noise-free quantification in an end-to-end fashion. We implement this approach in the scalable and robust open-source software package CellBender. Analysis of simulated data demonstrates that CellBender operates near the theoretically optimal denoising limit. Extensive evaluations using real datasets and experimental benchmarks highlight enhanced concordance between droplet-based single-cell data and established gene expression patterns, while the learned background noise profile provides evidence of degraded or uncaptured cell types. Using a deep generative model, CellBender models and denoises droplet-based single-cell data and improves multiple downstream analyses.},
  archive      = {J_NMETH},
  author       = {Fleming, Stephen J. and Chaffin, Mark D. and Arduini, Alessandro and Akkad, Amer-Denis and Banks, Eric and Marioni, John C. and Philippakis, Anthony A. and Ellinor, Patrick T. and Babadi, Mehrtash},
  doi          = {10.1038/s41592-023-01943-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1323-1335},
  shortjournal = {Nat. Meth.},
  title        = {Unsupervised removal of systematic background noise from droplet-based single-cell experiments using CellBender},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multielement z-tag imaging by x-ray fluorescence microscopy
for next-generation multiplex imaging. <em>NMETH</em>, <em>20</em>(9),
1310–1322. (<a
href="https://doi.org/10.1038/s41592-023-01977-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapid, highly multiplexed, nondestructive imaging that spans the molecular to the supra-cellular scale would be a powerful tool for tissue analysis. However, the physical constraints of established imaging methods limit the simultaneous improvement of these parameters. Whole-organism to atomic-level imaging is possible with tissue-penetrant, picometer-wavelength X-rays. To enable highly multiplexed X-ray imaging, we developed multielement Z-tag X-ray fluorescence (MEZ-XRF) that can operate at kHz speeds when combined with signal amplification by exchange reaction (SABER)-amplified Z-tag reagents. We demonstrated parallel imaging of 20 Z-tag or SABER Z-tag reagents at subcellular resolution in cell lines and multiple human tissues. We benchmarked MEZ-XRF against imaging mass cytometry and demonstrated the nondestructive multiscale repeat imaging capabilities of MEZ-XRF with rapid tissue overview scans, followed by slower, more sensitive imaging of low-abundance markers such as immune checkpoint proteins. The unique multiscale, nondestructive nature of MEZ-XRF, combined with SABER Z-tags for high sensitivity or enhanced speed, enables highly multiplexed bioimaging across biological scales. Multielement Z-tag X-ray fluorescence (MEZ-XRF) offers a new avenue for nondestructive and highly multiplexed tissue imaging and operates from the nanometer to whole-tissue scale, unlocking new biological observations.},
  archive      = {J_NMETH},
  author       = {Strotton, Merrick and Hosogane, Tsuyoshi and di Michiel, Marco and Moch, Holger and Varga, Zsuzsanna and Bodenmiller, Bernd},
  doi          = {10.1038/s41592-023-01977-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1310-1322},
  shortjournal = {Nat. Meth.},
  title        = {Multielement Z-tag imaging by X-ray fluorescence microscopy for next-generation multiplex imaging},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). DNA-barcoded signal amplification for imaging mass cytometry
enables sensitive and highly multiplexed tissue imaging. <em>NMETH</em>,
<em>20</em>(9), 1304–1309. (<a
href="https://doi.org/10.1038/s41592-023-01976-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Imaging mass cytometry (IMC) is a highly multiplexed, antibody-based imaging method that captures heterogeneous spatial protein expression patterns at subcellular resolution. Here we report the extension of IMC to low-abundance markers through incorporation of the DNA-based signal amplification by exchange reaction, immuno-SABER. We applied SABER-IMC to image the tumor immune microenvironment in human melanoma by simultaneous imaging of 18 markers with immuno-SABER and 20 markers without amplification. SABER-IMC enabled the identification of immune cell phenotypic markers, such as T cell co-receptors and their ligands, that are not detectable with IMC. SABER-IMC combines DNA-based signal amplification by exchange reaction (SABER) with imaging mass cytometry (IMC) to enable simultaneous and highly multiplexed marker detection, even of low-abundance markers not detectable with IMC alone.},
  archive      = {J_NMETH},
  author       = {Hosogane, Tsuyoshi and Casanova, Ruben and Bodenmiller, Bernd},
  doi          = {10.1038/s41592-023-01976-y},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1304-1309},
  shortjournal = {Nat. Meth.},
  title        = {DNA-barcoded signal amplification for imaging mass cytometry enables sensitive and highly multiplexed tissue imaging},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Minimum information guidelines for experiments structurally
characterizing intrinsically disordered protein regions. <em>NMETH</em>,
<em>20</em>(9), 1291–1303. (<a
href="https://doi.org/10.1038/s41592-023-01915-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An unambiguous description of an experiment, and the subsequent biological observation, is vital for accurate data interpretation. Minimum information guidelines define the fundamental complement of data that can support an unambiguous conclusion based on experimental observations. We present the Minimum Information About Disorder Experiments (MIADE) guidelines to define the parameters required for the wider scientific community to understand the findings of an experiment studying the structural properties of intrinsically disordered regions (IDRs). MIADE guidelines provide recommendations for data producers to describe the results of their experiments at source, for curators to annotate experimental data to community resources and for database developers maintaining community resources to disseminate the data. The MIADE guidelines will improve the interpretability of experimental results for data consumers, facilitate direct data submission, simplify data curation, improve data exchange among repositories and standardize the dissemination of the key metadata on an IDR experiment by IDR data sources. This Perspective introduces the Minimum Information About Disorder Experiments guidelines, which provide a community consensus on the minimum information required to appropriately describe metadata on experimentally and computationally derived structural state(s) of intrinsically disordered proteins or regions.},
  archive      = {J_NMETH},
  author       = {Mészáros, Bálint and Hatos, András and Palopoli, Nicolas and Quaglia, Federica and Salladini, Edoardo and Van Roey, Kim and Arthanari, Haribabu and Dosztányi, Zsuzsanna and Felli, Isabella C. and Fischer, Patrick D. and Hoch, Jeffrey C. and Jeffries, Cy M. and Longhi, Sonia and Maiani, Emiliano and Orchard, Sandra and Pancsa, Rita and Papaleo, Elena and Pierattelli, Roberta and Piovesan, Damiano and Pritisanac, Iva and Tenorio, Luiggi and Viennet, Thibault and Tompa, Peter and Vranken, Wim and Tosatto, Silvio C. E. and Davey, Norman E.},
  doi          = {10.1038/s41592-023-01915-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1291-1303},
  shortjournal = {Nat. Meth.},
  title        = {Minimum information guidelines for experiments structurally characterizing intrinsically disordered protein regions},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ultrafast nLight indicators for sensitive and specific in
vivo imaging of norepinephrine. <em>NMETH</em>, <em>20</em>(9),
1289–1290. (<a
href="https://doi.org/10.1038/s41592-023-01960-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed, characterized and validated nLight sensors, a new family of genetically encoded green and red fluorescent norepinephrine indicators based on an alpha-1 adrenergic receptor. nLight probes can detect norepinephrine in living animals with superior sensitivity, ligand specificity and temporal resolution as compared with previous tools.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01960-6},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1289-1290},
  shortjournal = {Nat. Meth.},
  title        = {Ultrafast nLight indicators for sensitive and specific in vivo imaging of norepinephrine},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Deep-learning-based isolation of perturbation-induced
variations in single-cell data. <em>NMETH</em>, <em>20</em>(9),
1287–1288. (<a
href="https://doi.org/10.1038/s41592-023-01956-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell perturbation screens are routinely conducted to study the effects of different perturbations on cellular state, yet such studies are easily confounded by nuisance sources of variation shared with control cells. We present a deep learning method that isolates perturbation-specific sources of variation, enabling a better understanding of the perturbation’s effects.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01956-2},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1287-1288},
  shortjournal = {Nat. Meth.},
  title        = {Deep-learning-based isolation of perturbation-induced variations in single-cell data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CellBender removes technical artifacts from single-cell RNA
sequencing data. <em>NMETH</em>, <em>20</em>(9), 1285–1286. (<a
href="https://doi.org/10.1038/s41592-023-01946-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The conversion of biological molecules into digital signals through sequencing is a complex process that often generates substantial systematic background noise. This noise can obscure important biological insights. However, by precisely identifying and removing this noise, we can bring the true signal into focus and eliminate misleading results from downstream analyses.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01946-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1285-1286},
  shortjournal = {Nat. Meth.},
  title        = {CellBender removes technical artifacts from single-cell RNA sequencing data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dissecting gene regulation with multimodal sequencing.
<em>NMETH</em>, <em>20</em>(9), 1282–1284. (<a
href="https://doi.org/10.1038/s41592-023-01957-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recently proposed computational approaches explore casual links between chromatin and transcriptional changes that are provided by single-cell multimodal sequencing to bridge the knowledge gap in transcriptional regulatory control.},
  archive      = {J_NMETH},
  author       = {Costa, Ivan G.},
  doi          = {10.1038/s41592-023-01957-1},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1282-1284},
  shortjournal = {Nat. Meth.},
  title        = {Dissecting gene regulation with multimodal sequencing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Advancing multiplexed imaging for enhanced tissue complexity
analysis. <em>NMETH</em>, <em>20</em>(9), 1280–1281. (<a
href="https://doi.org/10.1038/s41592-023-01935-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed spatial immunophenotyping has advanced our understanding of tissues in the context of homeostasis and disease. Two studies now provide additional tools to overcome challenges with multiplexed imaging: one procedure amplifies the detection of low-abundance antigens by integrating SABER and IMC technologies, and the other is an X-ray-based method that enables the non-destructive multiplexed detection of antigens in tissues at scalable resolution and speed.},
  archive      = {J_NMETH},
  author       = {Ijsselsteijn, Marieke E. and de Miranda, Noel F. C. C.},
  doi          = {10.1038/s41592-023-01935-7},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1280-1281},
  shortjournal = {Nat. Meth.},
  title        = {Advancing multiplexed imaging for enhanced tissue complexity analysis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A tough ask: High-efficiency, large-cargo prime editing.
<em>NMETH</em>, <em>20</em>(9), 1275–1279. (<a
href="https://doi.org/10.1038/s41592-023-01985-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers like the gene-editing method called prime editing for its precision and versatility. As they ask the method to do more, many factors shape what happens next.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01985-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1275-1279},
  shortjournal = {Nat. Meth.},
  title        = {A tough ask: High-efficiency, large-cargo prime editing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023o). Prefabricated inks promote heart bioprinting.
<em>NMETH</em>, <em>20</em>(9), 1274. (<a
href="https://doi.org/10.1038/s41592-023-02008-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-023-02008-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1274},
  shortjournal = {Nat. Meth.},
  title        = {Prefabricated inks promote heart bioprinting},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Ancestry inference using machine learning. <em>NMETH</em>,
<em>20</em>(9), 1274. (<a
href="https://doi.org/10.1038/s41592-023-02009-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-023-02009-4},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1274},
  shortjournal = {Nat. Meth.},
  title        = {Ancestry inference using machine learning},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A closer look at chromatin. <em>NMETH</em>, <em>20</em>(9),
1273. (<a href="https://doi.org/10.1038/s41592-023-02010-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An expansion microscopy technique called ChromExM offers detailed views into the organization chromatin and associated gene expression machinery in embryos.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-023-02010-x},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1273},
  shortjournal = {Nat. Meth.},
  title        = {A closer look at chromatin},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Addgene AAV data hub: A platform for sharing AAV
experimental data. <em>NMETH</em>, <em>20</em>(9), 1271–1272. (<a
href="https://doi.org/10.1038/s41592-023-01963-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Nasse, Jason S. and Leeson, Rachel and Zirkle, Heather and Bourges-Waldegg, Daniela and Pyhtila, Brook and Fan, Melina},
  doi          = {10.1038/s41592-023-01963-3},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1271-1272},
  shortjournal = {Nat. Meth.},
  title        = {Addgene AAV data hub: A platform for sharing AAV experimental data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Setting standards for stem cells. <em>NMETH</em>,
<em>20</em>(9), 1267. (<a
href="https://doi.org/10.1038/s41592-023-02016-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New traceability and reporting standards aim to improve transparency in stem cell research.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-02016-5},
  journal      = {Nature Methods},
  month        = {9},
  number       = {9},
  pages        = {1267},
  shortjournal = {Nat. Meth.},
  title        = {Setting standards for stem cells},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Convolutional neural networks. <em>NMETH</em>,
<em>20</em>(9), 1269–1270. (<a
href="https://doi.org/10.1038/s41592-023-01973-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature uses only the longest threads to weave her patterns, so that each small piece of her fabric reveals the organization of the entire tapestry. —Richard Feynman},
  archive      = {J_NMETH},
  author       = {Derry, Alexander and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-023-01973-1},
  journal      = {Nature Methods},
  month        = {8},
  number       = {9},
  pages        = {1269-1270},
  shortjournal = {Nat. Meth.},
  title        = {Convolutional neural networks},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Happy labiversary. <em>NMETH</em>, <em>20</em>(9), 1268.
(<a href="https://doi.org/10.1038/s41592-023-01984-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {They started their lab mid-COVID. The new principal investigator adventure is a roller-coaster ride with some unexpected surprises.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01984-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {9},
  pages        = {1268},
  shortjournal = {Nat. Meth.},
  title        = {Happy labiversary},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: Points of view: Color blindness.
<em>NMETH</em>, <em>20</em>(8), 1266. (<a
href="https://doi.org/10.1038/s41592-023-01974-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Wong, Bang},
  doi          = {10.1038/s41592-023-01974-0},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1266},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: points of view: color blindness},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dense 4D nanoscale reconstruction of living brain tissue.
<em>NMETH</em>, <em>20</em>(8), 1256–1265. (<a
href="https://doi.org/10.1038/s41592-023-01936-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Three-dimensional (3D) reconstruction of living brain tissue down to an individual synapse level would create opportunities for decoding the dynamics and structure–function relationships of the brain’s complex and dense information processing network; however, this has been hindered by insufficient 3D resolution, inadequate signal-to-noise ratio and prohibitive light burden in optical imaging, whereas electron microscopy is inherently static. Here we solved these challenges by developing an integrated optical/machine-learning technology, LIONESS (live information-optimized nanoscopy enabling saturated segmentation). This leverages optical modifications to stimulated emission depletion microscopy in comprehensively, extracellularly labeled tissue and previous information on sample structure via machine learning to simultaneously achieve isotropic super-resolution, high signal-to-noise ratio and compatibility with living tissue. This allows dense deep-learning-based instance segmentation and 3D reconstruction at a synapse level, incorporating molecular, activity and morphodynamic information. LIONESS opens up avenues for studying the dynamic functional (nano-)architecture of living brain tissue. A combination of gentle stimulated emission depletion microscopy imaging and deep-learning-based improvements in signal-to-noise ratio enables high-resolution reconstruction of neuronal architecture in living tissue.},
  archive      = {J_NMETH},
  author       = {Velicky, Philipp and Miguel, Eder and Michalska, Julia M. and Lyudchik, Julia and Wei, Donglai and Lin, Zudi and Watson, Jake F. and Troidl, Jakob and Beyer, Johanna and Ben-Simon, Yoav and Sommer, Christoph and Jahr, Wiebke and Cenameri, Alban and Broichhagen, Johannes and Grant, Seth G. N. and Jonas, Peter and Novarino, Gaia and Pfister, Hanspeter and Bickel, Bernd and Danzl, Johann G.},
  doi          = {10.1038/s41592-023-01936-6},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1256-1265},
  shortjournal = {Nat. Meth.},
  title        = {Dense 4D nanoscale reconstruction of living brain tissue},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Comprehensive spatiotemporal mapping of single-cell lineages
in developing mouse brain by CRISPR-based barcoding. <em>NMETH</em>,
<em>20</em>(8), 1244–1255. (<a
href="https://doi.org/10.1038/s41592-023-01947-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A fundamental interest in developmental neuroscience lies in the ability to map the complete single-cell lineages within the brain. To this end, we developed a CRISPR editing-based lineage-specific tracing (CREST) method for clonal tracing in Cre mice. We then used two complementary strategies based on CREST to map single-cell lineages in developing mouse ventral midbrain (vMB). By applying snapshotting CREST (snapCREST), we constructed a spatiotemporal lineage landscape of developing vMB and identified six progenitor archetypes that could represent the principal clonal fates of individual vMB progenitors and three distinct clonal lineages in the floor plate that specified glutamatergic, dopaminergic or both neurons. We further created pandaCREST (progenitor and derivative associating CREST) to associate the transcriptomes of progenitor cells in vivo with their differentiation potentials. We identified multiple origins of dopaminergic neurons and demonstrated that a transcriptome-defined progenitor type comprises heterogeneous progenitors, each with distinct clonal fates and molecular signatures. Therefore, the CREST method and strategies allow comprehensive single-cell lineage analysis that could offer new insights into the molecular programs underlying neural specification. CREST is a dual-recorder single-cell lineage tracing method for spatiotemporal mapping of the mouse brain.},
  archive      = {J_NMETH},
  author       = {Xie, Lianshun and Liu, Hengxin and You, Zhiwen and Wang, Luyue and Li, Yiwen and Zhang, Xinyue and Ji, Xiaoshan and He, Hui and Yuan, Tingli and Zheng, Wenping and Wu, Ziyan and Xiong, Man and Wei, Wu and Chen, Yuejun},
  doi          = {10.1038/s41592-023-01947-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1244-1255},
  shortjournal = {Nat. Meth.},
  title        = {Comprehensive spatiotemporal mapping of single-cell lineages in developing mouse brain by CRISPR-based barcoding},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). SCS: Cell segmentation for high-resolution spatial
transcriptomics. <em>NMETH</em>, <em>20</em>(8), 1237–1243. (<a
href="https://doi.org/10.1038/s41592-023-01939-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomics promises to greatly improve our understanding of tissue organization and cell–cell interactions. While most current platforms for spatial transcriptomics only offer multi-cellular resolution, with 10–15 cells per spot, recent technologies provide a much denser spot placement leading to subcellular resolution. A key challenge for these newer methods is cell segmentation and the assignment of spots to cells. Traditional image-based segmentation methods are limited and do not make full use of the information profiled by spatial transcriptomics. Here we present subcellular spatial transcriptomics cell segmentation (SCS), which combines imaging data with sequencing data to improve cell segmentation accuracy. SCS assigns spots to cells by adaptively learning the position of each spot relative to the center of its cell using a transformer neural network. SCS was tested on two new subcellular spatial transcriptomics technologies and outperformed traditional image-based segmentation methods. SCS achieved better accuracy, identified more cells and provided more realistic cell size estimation. Subcellular analysis of RNAs using SCS spot assignments provides information on RNA localization and further supports the segmentation results. Subcellular spatial transcriptomics cell segmentation (SCS) combines information from stained images and sequencing data to improve cell segmentation in high-resolution spatial transcriptomics data.},
  archive      = {J_NMETH},
  author       = {Chen, Hao and Li, Dongshunyi and Bar-Joseph, Ziv},
  doi          = {10.1038/s41592-023-01939-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1237-1243},
  shortjournal = {Nat. Meth.},
  title        = {SCS: Cell segmentation for high-resolution spatial transcriptomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CherryML: Scalable maximum likelihood estimation of
phylogenetic models. <em>NMETH</em>, <em>20</em>(8), 1232–1236. (<a
href="https://doi.org/10.1038/s41592-023-01917-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phylogenetic models of molecular evolution are central to numerous biological applications spanning diverse timescales, from hundreds of millions of years involving orthologous proteins to just tens of days relating to single cells within an organism. A fundamental problem in these applications is estimating model parameters, for which maximum likelihood estimation is typically employed. Unfortunately, maximum likelihood estimation is a computationally expensive task, in some cases prohibitively so. To address this challenge, we here introduce CherryML, a broadly applicable method that achieves several orders of magnitude speedup by using a quantized composite likelihood over cherries in the trees. The massive speedup offered by our method should enable researchers to consider more complex and biologically realistic models than previously possible. Here we demonstrate CherryML’s utility by applying it to estimate a general 400 × 400 rate matrix for residue–residue coevolution at contact sites in three-dimensional protein structures; we estimate that using current state-of-the-art methods such as the expectation-maximization algorithm for the same task would take &amp;gt;100,000 times longer. CherryML is a method to scale up maximum likelihood estimation for general phylogenetic models of molecular evolution, providing several orders of magnitude speedup over traditional methods.},
  archive      = {J_NMETH},
  author       = {Prillo, Sebastian and Deng, Yun and Boyeau, Pierre and Li, Xingyu and Chen, Po-Yen and Song, Yun S.},
  doi          = {10.1038/s41592-023-01917-9},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1232-1236},
  shortjournal = {Nat. Meth.},
  title        = {CherryML: Scalable maximum likelihood estimation of phylogenetic models},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). MultiVI: Deep generative model for the integration of
multimodal data. <em>NMETH</em>, <em>20</em>(8), 1222–1231. (<a
href="https://doi.org/10.1038/s41592-023-01909-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Jointly profiling the transcriptome, chromatin accessibility and other molecular properties of single cells offers a powerful way to study cellular diversity. Here we present MultiVI, a probabilistic model to analyze such multiomic data and leverage it to enhance single-modality datasets. MultiVI creates a joint representation that allows an analysis of all modalities included in the multiomic input data, even for cells for which one or more modalities are missing. It is available at scvi-tools.org . By learning a joint representation using deep generative modeling, MultiVI integrates multimodal and single-modality single-cell datasets, which enhances multiple functionalities.},
  archive      = {J_NMETH},
  author       = {Ashuach, Tal and Gabitto, Mariano I. and Koodli, Rohan V. and Saldi, Giuseppe-Antonio and Jordan, Michael I. and Yosef, Nir},
  doi          = {10.1038/s41592-023-01909-9},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1222-1231},
  shortjournal = {Nat. Meth.},
  title        = {MultiVI: Deep generative model for the integration of multimodal data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multiscale analysis of pangenomes enables improved
representation of genomic diversity for repetitive and clinically
relevant genes. <em>NMETH</em>, <em>20</em>(8), 1213–1221. (<a
href="https://doi.org/10.1038/s41592-023-01914-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advancements in sequencing technologies and assembly methods enable the regular production of high-quality genome assemblies characterizing complex regions. However, challenges remain in efficiently interpreting variation at various scales, from smaller tandem repeats to megabase rearrangements, across many human genomes. We present a PanGenome Research Tool Kit (PGR-TK) enabling analyses of complex pangenome structural and haplotype variation at multiple scales. We apply the graph decomposition methods in PGR-TK to the class II major histocompatibility complex demonstrating the importance of the human pangenome for analyzing complicated regions. Moreover, we investigate the Y-chromosome genes, DAZ1/DAZ2/DAZ3/DAZ4, of which structural variants have been linked to male infertility, and X-chromosome genes OPN1LW and OPN1MW linked to eye disorders. We further showcase PGR-TK across 395 complex repetitive medically important genes. This highlights the power of PGR-TK to resolve complex variation in regions of the genome that were previously too complex to analyze. The PanGenome Research Tool Kit (PGR-TK) achieves flexible and scalable representation, visualization and analysis of genomic variation using pangenome graphs.},
  archive      = {J_NMETH},
  author       = {Chin, Chen-Shan and Behera, Sairam and Khalak, Asif and Sedlazeck, Fritz J. and Sudmant, Peter H. and Wagner, Justin and Zook, Justin M.},
  doi          = {10.1038/s41592-023-01914-y},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1213-1221},
  shortjournal = {Nat. Meth.},
  title        = {Multiscale analysis of pangenomes enables improved representation of genomic diversity for repetitive and clinically relevant genes},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CheckM2: A rapid, scalable and accurate tool for assessing
microbial genome quality using machine learning. <em>NMETH</em>,
<em>20</em>(8), 1203–1212. (<a
href="https://doi.org/10.1038/s41592-023-01940-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in sequencing technologies and bioinformatics tools have dramatically increased the recovery rate of microbial genomes from metagenomic data. Assessing the quality of metagenome-assembled genomes (MAGs) is a critical step before downstream analysis. Here, we present CheckM2, an improved method of predicting genome quality of MAGs using machine learning. Using synthetic and experimental data, we demonstrate that CheckM2 outperforms existing tools in both accuracy and computational speed. In addition, CheckM2’s database can be rapidly updated with new high-quality reference genomes, including taxa represented only by a single genome. We also show that CheckM2 accurately predicts genome quality for MAGs from novel lineages, even for those with reduced genome size (for example, Patescibacteria and the DPANN superphylum). CheckM2 provides accurate genome quality predictions across bacterial and archaeal lineages, giving increased confidence when inferring biological conclusions from MAGs. This work presents CheckM2, which is a machine learning-based tool to predict genome quality of isolate, single-cell and metagenome-assembled genomes.},
  archive      = {J_NMETH},
  author       = {Chklovski, Alex and Parks, Donovan H. and Woodcroft, Ben J. and Tyson, Gene W.},
  doi          = {10.1038/s41592-023-01940-w},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1203-1212},
  shortjournal = {Nat. Meth.},
  title        = {CheckM2: A rapid, scalable and accurate tool for assessing microbial genome quality using machine learning},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Significance analysis for clustering with single-cell
RNA-sequencing data. <em>NMETH</em>, <em>20</em>(8), 1196–1202. (<a
href="https://doi.org/10.1038/s41592-023-01933-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unsupervised clustering of single-cell RNA-sequencing data enables the identification of distinct cell populations. However, the most widely used clustering algorithms are heuristic and do not formally account for statistical uncertainty. We find that not addressing known sources of variability in a statistically rigorous manner can lead to overconfidence in the discovery of novel cell types. Here we extend a previous method, significance of hierarchical clustering, to propose a model-based hypothesis testing approach that incorporates significance analysis into the clustering algorithm and permits statistical evaluation of clusters as distinct cell populations. We also adapt this approach to permit statistical assessment on the clusters reported by any algorithm. Finally, we extend these approaches to account for batch structure. We benchmarked our approach against popular clustering workflows, demonstrating improved performance. To show practical utility, we applied our approach to the Human Lung Cell Atlas and an atlas of the mouse cerebellar cortex, identifying several cases of over-clustering and recapitulating experimentally validated cell type definitions. This study presents a significance analysis framework for evaluating single-cell clusters. Application of the method detects cases of over-clustering in reported single-cell RNA-sequencing analysis results.},
  archive      = {J_NMETH},
  author       = {Grabski, Isabella N. and Street, Kelly and Irizarry, Rafael A.},
  doi          = {10.1038/s41592-023-01933-9},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1196-1202},
  shortjournal = {Nat. Meth.},
  title        = {Significance analysis for clustering with single-cell RNA-sequencing data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Context-aware transcript quantification from long-read
RNA-seq data with bambu. <em>NMETH</em>, <em>20</em>(8), 1187–1195. (<a
href="https://doi.org/10.1038/s41592-023-01908-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most approaches to transcript quantification rely on fixed reference annotations; however, the transcriptome is dynamic and depending on the context, such static annotations contain inactive isoforms for some genes, whereas they are incomplete for others. Here we present Bambu, a method that performs machine-learning-based transcript discovery to enable quantification specific to the context of interest using long-read RNA-sequencing. To identify novel transcripts, Bambu estimates the novel discovery rate, which replaces arbitrary per-sample thresholds with a single, interpretable, precision-calibrated parameter. Bambu retains the full-length and unique read counts, enabling accurate quantification in presence of inactive isoforms. Compared to existing methods for transcript discovery, Bambu achieves greater precision without sacrificing sensitivity. We show that context-aware annotations improve quantification for both novel and known transcripts. We apply Bambu to quantify isoforms from repetitive HERVH-LTR7 retrotransposons in human embryonic stem cells, demonstrating the ability for context-specific transcript expression analysis. Leveraging long-read RNA-seq data and machine learning, Bambu facilitates accurate transcript discovery and quantification.},
  archive      = {J_NMETH},
  author       = {Chen, Ying and Sim, Andre and Wan, Yuk Kei and Yeo, Keith and Lee, Joseph Jing Xian and Ling, Min Hao and Love, Michael I. and Göke, Jonathan},
  doi          = {10.1038/s41592-023-01908-w},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1187-1195},
  shortjournal = {Nat. Meth.},
  title        = {Context-aware transcript quantification from long-read RNA-seq data with bambu},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Open-3DSIM: An open-source three-dimensional structured
illumination microscopy reconstruction platform. <em>NMETH</em>,
<em>20</em>(8), 1183–1186. (<a
href="https://doi.org/10.1038/s41592-023-01958-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Open-3DSIM is an open-source reconstruction platform for three-dimensional structured illumination microscopy. We demonstrate its superior performance for artifact suppression and high-fidelity reconstruction relative to other algorithms on various specimens and over a range of signal-to-noise levels. Open-3DSIM also offers the capacity to extract dipole orientation, paving a new avenue for interpreting subcellular structures in six dimensions (xyzθλt). The platform is available as MATLAB code, a Fiji plugin and an Exe application to maximize user-friendliness. Open-3DSIM is a versatile open-source software for high-fidelity reconstruction of three-dimensional structured illumination microscopy data (with polarization). It is available in three convenient forms for user-friendly and customizable applications.},
  archive      = {J_NMETH},
  author       = {Cao, Ruijie and Li, Yaning and Chen, Xin and Ge, Xichuan and Li, Meiqi and Guan, Meiling and Hou, Yiwei and Fu, Yunzhe and Xu, Xinzhu and Leterrier, Christophe and Jiang, Shan and Gao, Baoxiang and Xi, Peng},
  doi          = {10.1038/s41592-023-01958-0},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1183-1186},
  shortjournal = {Nat. Meth.},
  title        = {Open-3DSIM: An open-source three-dimensional structured illumination microscopy reconstruction platform},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Expansion spatial transcriptomics. <em>NMETH</em>,
<em>20</em>(8), 1179–1182. (<a
href="https://doi.org/10.1038/s41592-023-01911-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Capture array-based spatial transcriptomics methods have been widely used to resolve gene expression in tissues; however, their spatial resolution is limited by the density of the array. Here we present expansion spatial transcriptomics to overcome this limitation by clearing and expanding tissue prior to capturing the entire polyadenylated transcriptome with an enhanced protocol. This approach enables us to achieve higher spatial resolution while retaining high library quality, which we demonstrate using mouse brain samples. Expansion spatial transcriptomics (Ex-ST) combines the power of tissue expansion with an improved RNA capture protocol for carrying out capture array-based spatial transcriptomics at high spatial resolution.},
  archive      = {J_NMETH},
  author       = {Fan, Yuhang and Andrusivová, Žaneta and Wu, Yunming and Chai, Chew and Larsson, Ludvig and He, Mengxiao and Luo, Liqun and Lundeberg, Joakim and Wang, Bo},
  doi          = {10.1038/s41592-023-01911-1},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1179-1182},
  shortjournal = {Nat. Meth.},
  title        = {Expansion spatial transcriptomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Organ mapping antibody panels: A community resource for
standardized multiplexed tissue imaging. <em>NMETH</em>, <em>20</em>(8),
1174–1178. (<a
href="https://doi.org/10.1038/s41592-023-01846-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiplexed antibody-based imaging enables the detailed characterization of molecular and cellular organization in tissues. Advances in the field now allow high-parameter data collection (&amp;gt;60 targets); however, considerable expertise and capital are needed to construct the antibody panels employed by these methods. Organ mapping antibody panels are community-validated resources that save time and money, increase reproducibility, accelerate discovery and support the construction of a Human Reference Atlas. Organ Mapping Antibody Panels are a community-led initiative to create standardized antibody panels for multiplexed spatial imaging.},
  archive      = {J_NMETH},
  author       = {Quardokus, Ellen M. and Saunders, Diane C. and McDonough, Elizabeth and Hickey, John W. and Werlein, Christopher and Surrette, Christine and Rajbhandari, Presha and Casals, Anna Martinez and Tian, Hua and Lowery, Lisa and Neumann, Elizabeth K. and Björklund, Frida and Neelakantan, Taruna V. and Croteau, Josh and Wiblin, Anne E. and Fisher, Jeremy and Livengood, April J. and Dowell, Karen G. and Silverstein, Jonathan C. and Spraggins, Jeffrey M. and Pryhuber, Gloria S. and Deutsch, Gail and Ginty, Fiona and Nolan, Garry P. and Melov, Simon and Jonigk, Danny and Caldwell, Michael A. and Vlachos, Ioannis S. and Muller, Werner and Gehlenborg, Nils and Stockwell, Brent R. and Lundberg, Emma and Snyder, Michael P. and Germain, Ronald N. and Camarillo, Jeannie M. and Kelleher, Neil L. and Börner, Katy and Radtke, Andrea J.},
  doi          = {10.1038/s41592-023-01846-7},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1174-1178},
  shortjournal = {Nat. Meth.},
  title        = {Organ mapping antibody panels: A community resource for standardized multiplexed tissue imaging},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A comparison of single-coverage and multi-coverage
metagenomic binning reveals extensive hidden contamination.
<em>NMETH</em>, <em>20</em>(8), 1170–1173. (<a
href="https://doi.org/10.1038/s41592-023-01934-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metagenomic binning has revolutionized the study of uncultured microorganisms. Here we compare single- and multi-coverage binning on the same set of samples, and demonstrate that multi-coverage binning produces better results than single-coverage binning and identifies contaminant contigs and chimeric bins that other approaches miss. While resource expensive, multi-coverage binning is a superior approach and should always be performed over single-coverage binning. This study shows, when analyzing multi-sample metagenomic datasets, the multi-coverage binning approach outperforms the single-coverage binning alternative in generating bins with higher quality and less contamination.},
  archive      = {J_NMETH},
  author       = {Mattock, Jennifer and Watson, Mick},
  doi          = {10.1038/s41592-023-01934-8},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1170-1173},
  shortjournal = {Nat. Meth.},
  title        = {A comparison of single-coverage and multi-coverage metagenomic binning reveals extensive hidden contamination},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Large-scale benchmarking of circRNA detection tools reveals
large differences in sensitivity but not in precision. <em>NMETH</em>,
<em>20</em>(8), 1159–1169. (<a
href="https://doi.org/10.1038/s41592-023-01944-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The detection of circular RNA molecules (circRNAs) is typically based on short-read RNA sequencing data processed using computational tools. Numerous such tools have been developed, but a systematic comparison with orthogonal validation is missing. Here, we set up a circRNA detection tool benchmarking study, in which 16 tools detected more than 315,000 unique circRNAs in three deeply sequenced human cell types. Next, 1,516 predicted circRNAs were validated using three orthogonal methods. Generally, tool-specific precision is high and similar (median of 98.8\%, 96.3\% and 95.5\% for qPCR, RNase R and amplicon sequencing, respectively) whereas the sensitivity and number of predicted circRNAs (ranging from 1,372 to 58,032) are the most significant differentiators. Of note, precision values are lower when evaluating low-abundance circRNAs. We also show that the tools can be used complementarily to increase detection sensitivity. Finally, we offer recommendations for future circRNA detection and validation. This study describes benchmarking and validation of computational tools for detecting circRNAs, finding most to be highly precise with variations in sensitivity and total detection. The study also finds over 315,000 putative human circRNAs.},
  archive      = {J_NMETH},
  author       = {Vromman, Marieke and Anckaert, Jasper and Bortoluzzi, Stefania and Buratin, Alessia and Chen, Chia-Ying and Chu, Qinjie and Chuang, Trees-Juen and Dehghannasiri, Roozbeh and Dieterich, Christoph and Dong, Xin and Flicek, Paul and Gaffo, Enrico and Gu, Wanjun and He, Chunjiang and Hoffmann, Steve and Izuogu, Osagie and Jackson, Michael S. and Jakobi, Tobias and Lai, Eric C. and Nuytens, Justine and Salzman, Julia and Santibanez-Koref, Mauro and Stadler, Peter and Thas, Olivier and Vanden Eynde, Eveline and Verniers, Kimberly and Wen, Guoxia and Westholm, Jakub and Yang, Li and Ye, Chu-Yu and Yigit, Nurten and Yuan, Guo-Hua and Zhang, Jinyang and Zhao, Fangqing and Vandesompele, Jo and Volders, Pieter-Jan},
  doi          = {10.1038/s41592-023-01944-6},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1159-1169},
  shortjournal = {Nat. Meth.},
  title        = {Large-scale benchmarking of circRNA detection tools reveals large differences in sensitivity but not in precision},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A survey of algorithms for the detection of genomic
structural variants from long-read sequencing data. <em>NMETH</em>,
<em>20</em>(8), 1143–1158. (<a
href="https://doi.org/10.1038/s41592-023-01932-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As long-read sequencing technologies are becoming increasingly popular, a number of methods have been developed for the discovery and analysis of structural variants (SVs) from long reads. Long reads enable detection of SVs that could not be previously detected from short-read sequencing, but computational methods must adapt to the unique challenges and opportunities presented by long-read sequencing. Here, we summarize over 50 long-read-based methods for SV detection, genotyping and visualization, and discuss how new telomere-to-telomere genome assemblies and pangenome efforts can improve the accuracy and drive the development of SV callers in the future. This Review provides an overview of computational methods recently developed for detecting and analyzing structural variants using long-read sequencing data.},
  archive      = {J_NMETH},
  author       = {Ahsan, Mian Umair and Liu, Qian and Perdomo, Jonathan Elliot and Fang, Li and Wang, Kai},
  doi          = {10.1038/s41592-023-01932-w},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1143-1158},
  shortjournal = {Nat. Meth.},
  title        = {A survey of algorithms for the detection of genomic structural variants from long-read sequencing data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). LIONESS enables 4D nanoscale reconstruction of living brain
tissue. <em>NMETH</em>, <em>20</em>(8), 1141–1142. (<a
href="https://doi.org/10.1038/s41592-023-01937-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed LIONESS, a technology that leverages improvements to optical super-resolution microscopy and prior information on sample structure via machine learning to overcome the limitations (in 3D-resolution, signal-to-noise ratio and light exposure) of optical microscopy of living biological specimens. LIONESS enables dense reconstruction of living brain tissue and morphodynamics visualization at the nanoscale.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01937-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1141-1142},
  shortjournal = {Nat. Meth.},
  title        = {LIONESS enables 4D nanoscale reconstruction of living brain tissue},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A method to map single-cell lineages in the mouse brain by
CRISPR-based barcoding. <em>NMETH</em>, <em>20</em>(8), 1139–1140. (<a
href="https://doi.org/10.1038/s41592-023-01948-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed CREST (CRISPR editing-based lineage-specific tracing) to enable high-throughput mapping of single-cell lineages in any Cre lineage of interest in mice. In addition, we delineated a comprehensive lineage landscape of the developing mouse ventral midbrain, revealing novel differentiation trajectories and molecular programs underlying neural specification.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01948-2},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1139-1140},
  shortjournal = {Nat. Meth.},
  title        = {A method to map single-cell lineages in the mouse brain by CRISPR-based barcoding},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Machine learning improves genome quality prediction across
the microbial tree of life. <em>NMETH</em>, <em>20</em>(8), 1137–1138.
(<a href="https://doi.org/10.1038/s41592-023-01941-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CheckM2 is a tool that applies machine learning to evaluate the quality of genomes from metagenomic data. CheckM2 is faster and more accurate than existing methods, and it outperforms them when applied to novel lineages and lineages with reduced genome sizes, such as Patescibacteria and the DPANN superphylum.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01941-9},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1137-1138},
  shortjournal = {Nat. Meth.},
  title        = {Machine learning improves genome quality prediction across the microbial tree of life},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Circular RNA detection pipelines yield divergent sets of
circular RNAs. <em>NMETH</em>, <em>20</em>(8), 1135–1136. (<a
href="https://doi.org/10.1038/s41592-023-01945-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A decade ago, the first bioinformatics pipelines to detect circular RNA molecules based on short-read sequencing data were published. Here, we show that dozens of such circular RNA detection tools differ vastly in their sensitivity but not in their specificity.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01945-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1135-1136},
  shortjournal = {Nat. Meth.},
  title        = {Circular RNA detection pipelines yield divergent sets of circular RNAs},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023p). Soil researchers dig deeper into dirt’s complexity.
<em>NMETH</em>, <em>20</em>(8), 1131–1134. (<a
href="https://doi.org/10.1038/s41592-023-01962-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To study soil with a view to ecosystems, researchers iterate between experiments in the lab, the field and controlled environments.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01962-4},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1131-1134},
  shortjournal = {Nat. Meth.},
  title        = {Soil researchers dig deeper into dirt’s complexity},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Nanopore-based detection of phosphorylation.
<em>NMETH</em>, <em>20</em>(8), 1130. (<a
href="https://doi.org/10.1038/s41592-023-01981-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-023-01981-1},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1130},
  shortjournal = {Nat. Meth.},
  title        = {Nanopore-based detection of phosphorylation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A peek into early human embryogenesis. <em>NMETH</em>,
<em>20</em>(8), 1130. (<a
href="https://doi.org/10.1038/s41592-023-01982-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-023-01982-0},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1130},
  shortjournal = {Nat. Meth.},
  title        = {A peek into early human embryogenesis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023j). In situ sequencing of localized translation.
<em>NMETH</em>, <em>20</em>(8), 1129. (<a
href="https://doi.org/10.1038/s41592-023-01980-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ribosome-bound mRNA mapping (RIBOmap) enables three-dimensional in situ detection of spatially resolved protein synthesis in single cells.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-023-01980-2},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1129},
  shortjournal = {Nat. Meth.},
  title        = {In situ sequencing of localized translation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The end game: Respecting major sources of population
diversity. <em>NMETH</em>, <em>20</em>(8), 1122–1128. (<a
href="https://doi.org/10.1038/s41592-023-01812-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human neuroscience is enjoying burgeoning population data resources: large-scale cohorts with thousands of participant profiles of gene expression, brain scanning and sociodemographic measures. The depth of phenotyping puts us in a better position than ever to fully embrace major sources of population diversity as effects of interest to illuminate mechanisms underlying brain health.},
  archive      = {J_NMETH},
  author       = {Kopal, Jakub and Uddin, Lucina Q. and Bzdok, Danilo},
  doi          = {10.1038/s41592-023-01812-3},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1122-1128},
  shortjournal = {Nat. Meth.},
  title        = {The end game: Respecting major sources of population diversity},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Distributed-something: Scripts to leverage AWS storage and
computing for distributed workflows at scale. <em>NMETH</em>,
<em>20</em>(8), 1120–1121. (<a
href="https://doi.org/10.1038/s41592-023-01918-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Weisbart, Erin and Cimini, Beth A.},
  doi          = {10.1038/s41592-023-01918-8},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1120-1121},
  shortjournal = {Nat. Meth.},
  title        = {Distributed-something: Scripts to leverage AWS storage and computing for distributed workflows at scale},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The axolotl. <em>NMETH</em>, <em>20</em>(8), 1117–1119. (<a
href="https://doi.org/10.1038/s41592-023-01961-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Originally from central Mexico and now found worldwide, the axolotl is a prominent model organism in numerous fields of research.},
  archive      = {J_NMETH},
  author       = {Sandoval-Guzmán, Tatiana},
  doi          = {10.1038/s41592-023-01961-5},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1117-1119},
  shortjournal = {Nat. Meth.},
  title        = {The axolotl},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Let’s talk about diversity in human neuroscience.
<em>NMETH</em>, <em>20</em>(8), 1115. (<a
href="https://doi.org/10.1038/s41592-023-01983-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Diversity in human neuroscience studies is an important and recurring topic of discussion. Though slow, progress is being made.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01983-z},
  journal      = {Nature Methods},
  month        = {8},
  number       = {8},
  pages        = {1115},
  shortjournal = {Nat. Meth.},
  title        = {Let’s talk about diversity in human neuroscience},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). How to make double affiliations work. <em>NMETH</em>,
<em>20</em>(8), 1116. (<a
href="https://doi.org/10.1038/s41592-023-01954-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Some researchers have the good fortune of two academic affiliations. Sometimes these affiliations are not exactly within easy commuting distance.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01954-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {8},
  pages        = {1116},
  shortjournal = {Nat. Meth.},
  title        = {How to make double affiliations work},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A positively tuned voltage indicator for extended electrical
recordings in the brain. <em>NMETH</em>, <em>20</em>(7), 1104–1113. (<a
href="https://doi.org/10.1038/s41592-023-01913-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetically encoded voltage indicators (GEVIs) enable optical recording of electrical signals in the brain, providing subthreshold sensitivity and temporal resolution not possible with calcium indicators. However, one- and two-photon voltage imaging over prolonged periods with the same GEVI has not yet been demonstrated. Here, we report engineering of ASAP family GEVIs to enhance photostability by inversion of the fluorescence–voltage relationship. Two of the resulting GEVIs, ASAP4b and ASAP4e, respond to 100-mV depolarizations with ≥180\% fluorescence increases, compared with the 50\% fluorescence decrease of the parental ASAP3. With standard microscopy equipment, ASAP4e enables single-trial detection of spikes in mice over the course of minutes. Unlike GEVIs previously used for one-photon voltage recordings, ASAP4b and ASAP4e also perform well under two-photon illumination. By imaging voltage and calcium simultaneously, we show that ASAP4b and ASAP4e can identify place cells and detect voltage spikes with better temporal resolution than commonly used calcium indicators. Thus, ASAP4b and ASAP4e extend the capabilities of voltage imaging to standard one- and two-photon microscopes while improving the duration of voltage recordings. The ASAP4 family of genetically encoded voltage indicators allows recording of action potentials and subthreshold activity with either one- or two-photon microscopy over extended periods of time.},
  archive      = {J_NMETH},
  author       = {Evans, S. Wenceslao and Shi, Dong-Qing and Chavarha, Mariya and Plitt, Mark H. and Taxidis, Jiannis and Madruga, Blake and Fan, Jiang Lan and Hwang, Fuu-Jiun and van Keulen, Siri C. and Suomivuori, Carl-Mikael and Pang, Michelle M. and Su, Sharon and Lee, Sungmoo and Hao, Yukun A. and Zhang, Guofeng and Jiang, Dongyun and Pradhan, Lagnajeet and Roth, Richard H. and Liu, Yu and Dorian, Conor C. and Reese, Austin L. and Negrean, Adrian and Losonczy, Attila and Makinson, Christopher D. and Wang, Sui and Clandinin, Thomas R. and Dror, Ron O. and Ding, Jun B. and Ji, Na and Golshani, Peyman and Giocomo, Lisa M. and Bi, Guo-Qiang and Lin, Michael Z.},
  doi          = {10.1038/s41592-023-01913-z},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1104-1113},
  shortjournal = {Nat. Meth.},
  title        = {A positively tuned voltage indicator for extended electrical recordings in the brain},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-speed low-light in vivo two-photon voltage imaging of
large neuronal populations. <em>NMETH</em>, <em>20</em>(7), 1095–1103.
(<a href="https://doi.org/10.1038/s41592-023-01820-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Monitoring spiking activity across large neuronal populations at behaviorally relevant timescales is critical for understanding neural circuit function. Unlike calcium imaging, voltage imaging requires kilohertz sampling rates that reduce fluorescence detection to near shot-noise levels. High-photon flux excitation can overcome photon-limited shot noise, but photobleaching and photodamage restrict the number and duration of simultaneously imaged neurons. We investigated an alternative approach aimed at low two-photon flux, which is voltage imaging below the shot-noise limit. This framework involved developing positive-going voltage indicators with improved spike detection (SpikeyGi and SpikeyGi2); a two-photon microscope (‘SMURF’) for kilohertz frame rate imaging across a 0.4 mm × 0.4 mm field of view; and a self-supervised denoising algorithm (DeepVID) for inferring fluorescence from shot-noise-limited signals. Through these combined advances, we achieved simultaneous high-speed deep-tissue imaging of more than 100 densely labeled neurons over 1 hour in awake behaving mice. This demonstrates a scalable approach for voltage imaging across increasing neuronal populations. A suite of tools including positive-going voltage indicators, a high-speed two-photon microscope, and denoising software enables prolonged imaging of electrical activity in neurons with limited toxicity.},
  archive      = {J_NMETH},
  author       = {Platisa, Jelena and Ye, Xin and Ahrens, Allison M. and Liu, Chang and Chen, Ichun Anderson and Davison, Ian G. and Tian, Lei and Pieribone, Vincent A. and Chen, Jerry L.},
  doi          = {10.1038/s41592-023-01820-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1095-1103},
  shortjournal = {Nat. Meth.},
  title        = {High-speed low-light in vivo two-photon voltage imaging of large neuronal populations},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Video-based pooled screening yields improved far-red
genetically encoded voltage indicators. <em>NMETH</em>, <em>20</em>(7),
1082–1094. (<a
href="https://doi.org/10.1038/s41592-022-01743-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Video-based screening of pooled libraries is a powerful approach for directed evolution of biosensors because it enables selection along multiple dimensions simultaneously from large libraries. Here we develop a screening platform, Photopick, which achieves precise phenotype-activated photoselection over a large field of view (2.3 × 2.3 mm, containing &amp;gt;103 cells, per shot). We used the Photopick platform to evolve archaerhodopsin-derived genetically encoded voltage indicators (GEVIs) with improved signal-to-noise ratio (QuasAr6a) and kinetics (QuasAr6b). These GEVIs gave improved signals in cultured neurons and in live mouse brains. By combining targeted in vivo optogenetic stimulation with high-precision voltage imaging, we characterized inhibitory synaptic coupling between individual cortical NDNF (neuron-derived neurotrophic factor) interneurons, and excitatory electrical synapses between individual hippocampal parvalbumin neurons. The QuasAr6 GEVIs are powerful tools for all-optical electrophysiology and the Photopick approach could be adapted to evolve a broad range of biosensors. The Photopick platform, which can be used for phenotype-activated cell selection, was used to develop the improved voltage sensors QuasAr6a and QuasAr6b. These GEVIs offer improved signals and are useful for all-optical electrophysiology.},
  archive      = {J_NMETH},
  author       = {Tian, He and Davis, Hunter C. and Wong-Campos, J. David and Park, Pojeong and Fan, Linlin Z. and Gmeiner, Benjamin and Begum, Shahinoor and Werley, Christopher A. and Borja, Gabriel B. and Upadhyay, Hansini and Shah, Himali and Jacques, Jane and Qi, Yitong and Parot, Vicente and Deisseroth, Karl and Cohen, Adam E.},
  doi          = {10.1038/s41592-022-01743-5},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1082-1094},
  shortjournal = {Nat. Meth.},
  title        = {Video-based pooled screening yields improved far-red genetically encoded voltage indicators},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Hardwiring tissue-specific AAV transduction in mice through
engineered receptor expression. <em>NMETH</em>, <em>20</em>(7),
1070–1081. (<a
href="https://doi.org/10.1038/s41592-023-01896-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of transgenic mouse models that express genes of interest in specific cell types has transformed our understanding of basic biology and disease. However, generating these models is time- and resource-intensive. Here we describe a model system, SELective Expression and Controlled Transduction In Vivo (SELECTIV), that enables efficient and specific expression of transgenes by coupling adeno-associated virus (AAV) vectors with Cre-inducible overexpression of the multi-serotype AAV receptor, AAVR. We demonstrate that transgenic AAVR overexpression greatly increases the efficiency of transduction of many diverse cell types, including muscle stem cells, which are normally refractory to AAV transduction. Superior specificity is achieved by combining Cre-mediated AAVR overexpression with whole-body knockout of endogenous Aavr, which is demonstrated in heart cardiomyocytes, liver hepatocytes and cholinergic neurons. The enhanced efficacy and exquisite specificity of SELECTIV has broad utility in development of new mouse model systems and expands the use of AAV for gene delivery in vivo. SELECTIV is an efficient method for tissue-specific AAV-mediated transgene expression in mice mediated by AAV receptor overexpression.},
  archive      = {J_NMETH},
  author       = {Zengel, James and Wang, Yu Xin and Seo, Jai Woong and Ning, Ke and Hamilton, James N. and Wu, Bo and Raie, Marina and Holbrook, Colin and Su, Shiqi and Clements, Derek R. and Pillay, Sirika and Puschnik, Andreas S. and Winslow, Monte M. and Idoyaga, Juliana and Nagamine, Claude M. and Sun, Yang and Mahajan, Vinit B. and Ferrara, Katherine W. and Blau, Helen M. and Carette, Jan E.},
  doi          = {10.1038/s41592-023-01896-x},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1070-1081},
  shortjournal = {Nat. Meth.},
  title        = {Hardwiring tissue-specific AAV transduction in mice through engineered receptor expression},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Learning consistent subcellular landmarks to quantify
changes in multiplexed protein maps. <em>NMETH</em>, <em>20</em>(7),
1058–1069. (<a
href="https://doi.org/10.1038/s41592-023-01894-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Highly multiplexed imaging holds enormous promise for understanding how spatial context shapes the activity of the genome and its products at multiple length scales. Here, we introduce a deep learning framework called CAMPA (Conditional Autoencoder for Multiplexed Pixel Analysis), which uses a conditional variational autoencoder to learn representations of molecular pixel profiles that are consistent across heterogeneous cell populations and experimental perturbations. Clustering these pixel-level representations identifies consistent subcellular landmarks, which can be quantitatively compared in terms of their size, shape, molecular composition and relative spatial organization. Using high-resolution multiplexed immunofluorescence, this reveals how subcellular organization changes upon perturbation of RNA synthesis, RNA processing or cell size, and uncovers links between the molecular composition of membraneless organelles and cell-to-cell variability in bulk RNA synthesis rates. By capturing interpretable cellular phenotypes, we anticipate that CAMPA will greatly accelerate the systematic mapping of multiscale atlases of biological organization to identify the rules by which context shapes physiology and disease. CAMPA (Conditional Autoencoder for Multiplexed Pixel Analysis) learns representations of molecular pixel profiles from multiplexed images that can be clustered to quantify subcellular landmarks and capture interpretable cellular phenotypes.},
  archive      = {J_NMETH},
  author       = {Spitzer, Hannah and Berry, Scott and Donoghoe, Mark and Pelkmans, Lucas and Theis, Fabian J.},
  doi          = {10.1038/s41592-023-01894-z},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1058-1069},
  shortjournal = {Nat. Meth.},
  title        = {Learning consistent subcellular landmarks to quantify changes in multiplexed protein maps},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Simultaneous profiling of spatial gene expression and
chromatin accessibility during mouse brain development. <em>NMETH</em>,
<em>20</em>(7), 1048–1057. (<a
href="https://doi.org/10.1038/s41592-023-01884-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The brain is a complex tissue whose function relies on coordinated anatomical and molecular features. However, the molecular annotation of the spatial organization of the brain is currently insufficient. Here, we describe microfluidic indexing-based spatial assay for transposase-accessible chromatin and RNA-sequencing (MISAR-seq), a method for spatially resolved joint profiling of chromatin accessibility and gene expression. By applying MISAR-seq to the developing mouse brain, we study tissue organization and spatiotemporal regulatory logics during mouse brain development. MISAR-seq combines spatial-ATAC-seq and RNA-seq for spatial profiling of both chromatin accessibility and gene expression, as demonstrated in the developing mouse brain.},
  archive      = {J_NMETH},
  author       = {Jiang, Fuqing and Zhou, Xin and Qian, Yingying and Zhu, Miao and Wang, Li and Li, Zhuxia and Shen, Qingmei and Wang, Minhan and Qu, Fangfang and Cui, Guizhong and Chen, Kai and Peng, Guangdun},
  doi          = {10.1038/s41592-023-01884-1},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1048-1057},
  shortjournal = {Nat. Meth.},
  title        = {Simultaneous profiling of spatial gene expression and chromatin accessibility during mouse brain development},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multiplex-GAM: Genome-wide identification of chromatin
contacts yields insights overlooked by hi-c. <em>NMETH</em>,
<em>20</em>(7), 1037–1047. (<a
href="https://doi.org/10.1038/s41592-023-01903-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Technology for measuring 3D genome topology is increasingly important for studying gene regulation, for genome assembly and for mapping of genome rearrangements. Hi-C and other ligation-based methods have become routine but have specific biases. Here, we develop multiplex-GAM, a faster and more affordable version of genome architecture mapping (GAM), a ligation-free technique that maps chromatin contacts genome-wide. We perform a detailed comparison of multiplex-GAM and Hi-C using mouse embryonic stem cells. When examining the strongest contacts detected by either method, we find that only one-third of these are shared. The strongest contacts specifically found in GAM often involve ‘active’ regions, including many transcribed genes and super-enhancers, whereas in Hi-C they more often contain ‘inactive’ regions. Our work shows that active genomic regions are involved in extensive complex contacts that are currently underestimated in ligation-based approaches, and highlights the need for orthogonal advances in genome-wide contact mapping technologies. Multiplex-genome architecture mapping (multiplex-GAM) enables rapid, unbiased, ligation-free mapping of genome-wide chromatin interactions.},
  archive      = {J_NMETH},
  author       = {Beagrie, Robert A. and Thieme, Christoph J. and Annunziatella, Carlo and Baugher, Catherine and Zhang, Yingnan and Schueler, Markus and Kukalev, Alexander and Kempfer, Rieke and Chiariello, Andrea M. and Bianco, Simona and Li, Yichao and Davis, Trenton and Scialdone, Antonio and Welch, Lonnie R. and Nicodemi, Mario and Pombo, Ana},
  doi          = {10.1038/s41592-023-01903-1},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1037-1047},
  shortjournal = {Nat. Meth.},
  title        = {Multiplex-GAM: Genome-wide identification of chromatin contacts yields insights overlooked by hi-C},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Development of miniature base editors using engineered IscB
nickase. <em>NMETH</em>, <em>20</em>(7), 1029–1036. (<a
href="https://doi.org/10.1038/s41592-023-01898-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As a miniature RNA-guided endonuclease, IscB is presumed to be the ancestor of Cas9 and to share similar functions. IscB is less than half the size of Cas9 and thus more suitable for in vivo delivery. However, the poor editing efficiency of IscB in eukaryotic cells limits its in vivo applications. Here we describe the engineering of OgeuIscB and its corresponding ωRNA to develop an IscB system that is highly efficient in mammalian systems, named enIscB. By fusing enIscB with T5 exonuclease (T5E), we found enIscB-T5E exhibited comparable targeting efficiency to SpG Cas9 while showing reduced chromosome translocation effects in human cells. Furthermore, by fusing cytosine or adenosine deaminase with enIscB nickase, we generated miniature IscB-derived base editors (miBEs), exhibiting robust editing efficiency (up to 92\%) to induce DNA base conversions. Overall, our work establishes enIscB-T5E and miBEs as versatile tools for genome editing. The miniature RNA-guided endonuclease IscB and its ωRNA were engineered for efficient gene editing in mammalian cells. Fusions of ‘enIscB’ to T5 exonuclease and cytosine or adenosine deaminase yield versatile tools for genome engineering.},
  archive      = {J_NMETH},
  author       = {Han, Dingyi and Xiao, Qingquan and Wang, Yifan and Zhang, Hainan and Dong, Xue and Li, Guoling and Kong, Xiangfeng and Wang, Shihao and Song, Jinhui and Zhang, Weihong and Zhou, Jingxing and Bi, Lanting and Yuan, Yuan and Shi, Linyu and Zhong, Na and Yang, Hui and Zhou, Yingsi},
  doi          = {10.1038/s41592-023-01898-9},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1029-1036},
  shortjournal = {Nat. Meth.},
  title        = {Development of miniature base editors using engineered IscB nickase},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). ReX: An integrative tool for quantifying and optimizing
measurement reliability for the study of individual differences.
<em>NMETH</em>, <em>20</em>(7), 1025–1028. (<a
href="https://doi.org/10.1038/s41592-023-01901-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterizing multifaceted individual differences in brain function using neuroimaging is central to biomarker discovery in neuroscience. We provide an integrative toolbox, Reliability eXplorer (ReX), to facilitate the examination of individual variation and reliability as well as the effective direction for optimization of measuring individual differences in biomarker discovery. We also illustrate gradient flows, a two-dimensional field map-based approach to identifying and representing the most effective direction for optimization when measuring individual differences, which is implemented in ReX. The Reliability eXplorer is a tool for assessing measurement reliability in neuroimaging studies.},
  archive      = {J_NMETH},
  author       = {Xu, Ting and Kiar, Gregory and Cho, Jae Wook and Bridgeford, Eric W. and Nikolaidis, Aki and Vogelstein, Joshua T. and Milham, Michael P.},
  doi          = {10.1038/s41592-023-01901-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1025-1028},
  shortjournal = {Nat. Meth.},
  title        = {ReX: An integrative tool for quantifying and optimizing measurement reliability for the study of individual differences},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). New measures of anisotropy of cryo-EM maps. <em>NMETH</em>,
<em>20</em>(7), 1021–1024. (<a
href="https://doi.org/10.1038/s41592-023-01874-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We propose two new measures of resolution anisotropy for cryogenic electron microscopy maps: Fourier shell occupancy (FSO), and the Bingham test (BT). FSO varies from 1 to 0, with 1 representing perfect isotropy, and lower values indicating increasing anisotropy. The threshold FSO = 0.5 occurs at Fourier shell correlation resolution. BT is a hypothesis test that complements the FSO to ensure the existence of anisotropy. FSO and BT allow visualization of resolution anisotropy. We illustrate their use with different experimental cryogenic electron microscopy maps. This paper proposes two new anisotropy metrics—the Fourier shell occupancy and the Bingham test—that can be used to understand the quality of cryogenic electron microscopy maps.},
  archive      = {J_NMETH},
  author       = {Vilas, Jose-Luis and Tagare, Hemant D.},
  doi          = {10.1038/s41592-023-01874-3},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1021-1024},
  shortjournal = {Nat. Meth.},
  title        = {New measures of anisotropy of cryo-EM maps},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The cell tracking challenge: 10 years of objective
benchmarking. <em>NMETH</em>, <em>20</em>(7), 1010–1020. (<a
href="https://doi.org/10.1038/s41592-023-01879-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Cell Tracking Challenge is an ongoing benchmarking initiative that has become a reference in cell segmentation and tracking algorithm development. Here, we present a significant number of improvements introduced in the challenge since our 2017 report. These include the creation of a new segmentation-only benchmark, the enrichment of the dataset repository with new datasets that increase its diversity and complexity, and the creation of a silver standard reference corpus based on the most competitive results, which will be of particular interest for data-hungry deep learning-based strategies. Furthermore, we present the up-to-date cell segmentation and tracking leaderboards, an in-depth analysis of the relationship between the performance of the state-of-the-art methods and the properties of the datasets and annotations, and two novel, insightful studies about the generalizability and the reusability of top-performing methods. These studies provide critical practical conclusions for both developers and users of traditional and machine learning-based cell segmentation and tracking algorithms. This updated analysis of the Cell Tracking Challenge explores how algorithms for cell segmentation and tracking in both 2D and 3D have advanced in recent years, pointing users to high-performing tools and developers to open challenges.},
  archive      = {J_NMETH},
  author       = {Maška, Martin and Ulman, Vladimír and Delgado-Rodriguez, Pablo and Gómez-de-Mariscal, Estibaliz and Nečasová, Tereza and Guerrero Peña, Fidel A. and Ren, Tsang Ing and Meyerowitz, Elliot M. and Scherr, Tim and Löffler, Katharina and Mikut, Ralf and Guo, Tianqi and Wang, Yin and Allebach, Jan P. and Bao, Rina and Al-Shakarji, Noor M. and Rahmon, Gani and Toubal, Imad Eddine and Palaniappan, Kannappan and Lux, Filip and Matula, Petr and Sugawara, Ko and Magnusson, Klas E. G. and Aho, Layton and Cohen, Andrew R. and Arbelle, Assaf and Ben-Haim, Tal and Raviv, Tammy Riklin and Isensee, Fabian and Jäger, Paul F. and Maier-Hein, Klaus H. and Zhu, Yanming and Ederra, Cristina and Urbiola, Ainhoa and Meijering, Erik and Cunha, Alexandre and Muñoz-Barrutia, Arrate and Kozubek, Michal and Ortiz-de-Solórzano, Carlos},
  doi          = {10.1038/s41592-023-01879-y},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {1010-1020},
  shortjournal = {Nat. Meth.},
  title        = {The cell tracking challenge: 10 years of objective benchmarking},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Massively parallel evaluation and computational prediction
of the activities and specificities of 17 small cas9s. <em>NMETH</em>,
<em>20</em>(7), 999–1009. (<a
href="https://doi.org/10.1038/s41592-023-01875-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recently, various small Cas9 orthologs and variants have been reported for use in in vivo delivery applications. Although small Cas9s are particularly suited for this purpose, selecting the most optimal small Cas9 for use at a specific target sequence continues to be challenging. Here, to this end, we have systematically compared the activities of 17 small Cas9s for thousands of target sequences. For each small Cas9, we have characterized the protospacer adjacent motif and determined optimal single guide RNA expression formats and scaffold sequence. High-throughput comparative analyses revealed distinct high- and low-activity groups of small Cas9s. We also developed DeepSmallCas9, a set of computational models predicting the activities of the small Cas9s at matched and mismatched target sequences. Together, this analysis and these computational models provide a useful guide for researchers to select the most suitable small Cas9 for specific applications. A comparison of 17 small Cas9s for optimal target-specific genome editing.},
  archive      = {J_NMETH},
  author       = {Seo, Sang-Yeon and Min, Seonwoo and Lee, Sungtae and Seo, Jung Hwa and Park, Jinman and Kim, Hui Kwon and Song, Myungjae and Baek, Dawoon and Cho, Sung-Rae and Kim, Hyongbum Henry},
  doi          = {10.1038/s41592-023-01875-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {999-1009},
  shortjournal = {Nat. Meth.},
  title        = {Massively parallel evaluation and computational prediction of the activities and specificities of 17 small cas9s},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A flexible system for tissue-specific gene expression in
mice using adeno-associated virus. <em>NMETH</em>, <em>20</em>(7),
997–998. (<a href="https://doi.org/10.1038/s41592-023-01897-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell type-specific transgene expression in mice has broad utility in biomedical research. We developed a versatile system for in vivo transgene delivery using adeno-associated virus (AAV). Efficient and tissue-specific transgene expression is achieved by regulating the expression of the gene encoding the AAV receptor, thereby precisely targeting AAV to the cell type of interest.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01897-w},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {997-998},
  shortjournal = {Nat. Meth.},
  title        = {A flexible system for tissue-specific gene expression in mice using adeno-associated virus},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Deciphering subcellular organization with multiplexed
imaging and deep learning. <em>NMETH</em>, <em>20</em>(7), 995–996. (<a
href="https://doi.org/10.1038/s41592-023-01895-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our study introduces conditional autoencoder for multiplexed pixel analysis (CAMPA), a deep-learning framework that uses highly multiplexed imaging to identify consistent subcellular landmarks across heterogeneous cell populations and experimental perturbations. Generating interpretable cellular phenotypes revealed links between subcellular organization and perturbations of RNA production, RNA processing and cell size.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01895-y},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {995-996},
  shortjournal = {Nat. Meth.},
  title        = {Deciphering subcellular organization with multiplexed imaging and deep learning},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome architecture mapping detects transcriptionally
active, multiway chromatin contacts. <em>NMETH</em>, <em>20</em>(7),
993–994. (<a href="https://doi.org/10.1038/s41592-023-01905-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome architecture mapping (GAM) enables understanding of 3D genome structure in the nucleus. We directly compared multiplex-GAM and Hi-C data and found that local chromatin interactions were generally detected by both methods, but active genomic regions rich in enhancers that established higher-order contacts were preferentially detected by GAM.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01905-z},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {993-994},
  shortjournal = {Nat. Meth.},
  title        = {Genome architecture mapping detects transcriptionally active, multiway chromatin contacts},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Lighting up action potentials with fast and bright voltage
sensors. <em>NMETH</em>, <em>20</em>(7), 990–992. (<a
href="https://doi.org/10.1038/s41592-023-01928-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Three groundbreaking studies have created a new generation of genetically encoded voltage indicators, empowering us to tackle a host of questions on our path toward understanding the brain.},
  archive      = {J_NMETH},
  author       = {Andreoni, Alessio and Tian, Lin},
  doi          = {10.1038/s41592-023-01928-6},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {990-992},
  shortjournal = {Nat. Meth.},
  title        = {Lighting up action potentials with fast and bright voltage sensors},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023r). To share is to be a scientist. <em>NMETH</em>,
<em>20</em>(7), 984–989. (<a
href="https://doi.org/10.1038/s41592-023-01927-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Wrangling big data is now part of being a biomedical scientist, and mandates on data sharing have entered the scene. Mandates can alter behavior, but data sharing also needs incentives and shifts in science culture.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01927-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {984-989},
  shortjournal = {Nat. Meth.},
  title        = {To share is to be a scientist},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Diving deeper in the 3D genome. <em>NMETH</em>,
<em>20</em>(7), 983. (<a
href="https://doi.org/10.1038/s41592-023-01952-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-023-01952-6},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {983},
  shortjournal = {Nat. Meth.},
  title        = {Diving deeper in the 3D genome},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Speedier protein structure search. <em>NMETH</em>,
<em>20</em>(7), 983. (<a
href="https://doi.org/10.1038/s41592-023-01953-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-023-01953-5},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {983},
  shortjournal = {Nat. Meth.},
  title        = {Speedier protein structure search},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Ex utero development of primate embryos. <em>NMETH</em>,
<em>20</em>(7), 982. (<a
href="https://doi.org/10.1038/s41592-023-01951-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two independent studies demonstrate in vitro development of early cynomolgus monkey embryos.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-023-01951-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {982},
  shortjournal = {Nat. Meth.},
  title        = {Ex utero development of primate embryos},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Unveiling the vision: Exploring the potential of image
analysis in africa. <em>NMETH</em>, <em>20</em>(7), 979–981. (<a
href="https://doi.org/10.1038/s41592-023-01907-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we discuss the prospects of bioimage analysis in the context of the African research landscape as well as challenges faced in the development of bioimage analysis in countries on the continent. We also speculate about potential approaches and areas of focus to overcome these challenges and thus build the communities, infrastructure and initiatives that are required to grow image analysis in African research.},
  archive      = {J_NMETH},
  author       = {Rahmoon, Mai Atef and Simegn, Gizeaddis Lamesgin and William, Wasswa and Reiche, Michael A.},
  doi          = {10.1038/s41592-023-01907-x},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {979-981},
  shortjournal = {Nat. Meth.},
  title        = {Unveiling the vision: Exploring the potential of image analysis in africa},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The twenty questions of bioimage object analysis.
<em>NMETH</em>, <em>20</em>(7), 976–978. (<a
href="https://doi.org/10.1038/s41592-023-01919-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The language used by microscopists who wish to find and measure objects in an image often differs in critical ways from that used by computer scientists who create tools to help them do this, making communication hard across disciplines. This work proposes a set of standardized questions that can guide analyses and shows how it can improve the future of bioimage analysis as a whole by making image analysis workflows and tools more FAIR (findable, accessible, interoperable and reusable).},
  archive      = {J_NMETH},
  author       = {Cimini, Beth A. and Eliceiri, Kevin W.},
  doi          = {10.1038/s41592-023-01919-7},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {976-978},
  shortjournal = {Nat. Meth.},
  title        = {The twenty questions of bioimage object analysis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Using AI in bioimage analysis to elevate the rate of
scientific discovery as a community. <em>NMETH</em>, <em>20</em>(7),
973–975. (<a href="https://doi.org/10.1038/s41592-023-01929-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The future of bioimage analysis is increasingly defined by the development and use of tools that rely on deep learning and artificial intelligence (AI). For this trend to continue in a way most useful for stimulating scientific progress, it will require our multidisciplinary community to work together, establish FAIR (findable, accessible, interoperable and reusable) data sharing and deliver usable and reproducible analytical tools.},
  archive      = {J_NMETH},
  author       = {Nogare, Damian Dalle and Hartley, Matthew and Deschamps, Joran and Ellenberg, Jan and Jug, Florian},
  doi          = {10.1038/s41592-023-01929-5},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {973-975},
  shortjournal = {Nat. Meth.},
  title        = {Using AI in bioimage analysis to elevate the rate of scientific discovery as a community},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Towards effective adoption of novel image analysis methods.
<em>NMETH</em>, <em>20</em>(7), 971–972. (<a
href="https://doi.org/10.1038/s41592-023-01910-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The bridging of domains such as deep learning-driven image analysis and biology brings exciting promises of previously impossible discoveries as well as perils of misinterpretation and misapplication. We encourage continual communication between method developers and application scientists that emphases likely pitfalls and provides validation tools in conjunction with new techniques.},
  archive      = {J_NMETH},
  author       = {Lambert, Talley and Waters, Jennifer},
  doi          = {10.1038/s41592-023-01910-2},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {971-972},
  shortjournal = {Nat. Meth.},
  title        = {Towards effective adoption of novel image analysis methods},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). When seeing is not believing: Application-appropriate
validation matters for quantitative bioimage analysis. <em>NMETH</em>,
<em>20</em>(7), 968–970. (<a
href="https://doi.org/10.1038/s41592-023-01881-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A key step toward biologically interpretable analysis of microscopy image-based assays is rigorous quantitative validation with metrics appropriate for the particular application in use. Here we describe this challenge for both classical and modern deep learning-based image analysis approaches and discuss possible solutions for automating and streamlining the validation process in the next five to ten years.},
  archive      = {J_NMETH},
  author       = {Chen, Jianxu and Viana, Matheus P. and Rafelski, Susanne M.},
  doi          = {10.1038/s41592-023-01881-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {968-970},
  shortjournal = {Nat. Meth.},
  title        = {When seeing is not believing: Application-appropriate validation matters for quantitative bioimage analysis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Phasor plots and the future of spectral and lifetime
imaging. <em>NMETH</em>, <em>20</em>(7), 965–967. (<a
href="https://doi.org/10.1038/s41592-023-01906-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {I share my opinions on the benefits of and bottlenecks for hyperspectral and time-resolved imaging. I also discuss current and future perspectives for analyzing these types of data using the phasor approach.},
  archive      = {J_NMETH},
  author       = {Malacrida, Leonel},
  doi          = {10.1038/s41592-023-01906-y},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {965-967},
  shortjournal = {Nat. Meth.},
  title        = {Phasor plots and the future of spectral and lifetime imaging},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Smart microscopes of the future. <em>NMETH</em>,
<em>20</em>(7), 962–964. (<a
href="https://doi.org/10.1038/s41592-023-01912-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We dream of a future where light microscopes have new capabilities: language-guided image acquisition, automatic image analysis based on extensive prior training from biologist experts, and language-guided image analysis for custom analyses. Most capabilities have reached the proof-of-principle stage, but implementation would be accelerated by efforts to gather appropriate training sets and make user-friendly interfaces.},
  archive      = {J_NMETH},
  author       = {Carpenter, Anne E. and Cimini, Beth A. and Eliceiri, Kevin W.},
  doi          = {10.1038/s41592-023-01912-0},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {962-964},
  shortjournal = {Nat. Meth.},
  title        = {Smart microscopes of the future},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Challenges and opportunities in bioimage analysis.
<em>NMETH</em>, <em>20</em>(7), 958–961. (<a
href="https://doi.org/10.1038/s41592-023-01900-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advanced imaging techniques provide holistic observations of complicated biological phenomena across multiple scales while posing great challenges to data analysis. We summarize recent advances and trends in bioimage analysis, discuss current challenges toward better applicability, and envisage new possibilities.},
  archive      = {J_NMETH},
  author       = {Li, Xinyang and Zhang, Yuanlong and Wu, Jiamin and Dai, Qionghai},
  doi          = {10.1038/s41592-023-01900-4},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {958-961},
  shortjournal = {Nat. Meth.},
  title        = {Challenges and opportunities in bioimage analysis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Scaling biological discovery at the interface of deep
learning and cellular imaging. <em>NMETH</em>, <em>20</em>(7), 956–957.
(<a href="https://doi.org/10.1038/s41592-023-01931-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Concurrent advances in imaging technologies and deep learning have transformed the nature and scale of data that can now be collected with imaging. Here we discuss the progress that has been made and outline potential research directions at the intersection of deep learning and imaging-based measurements of living systems.},
  archive      = {J_NMETH},
  author       = {Schwartz, Morgan and Israel, Uriah and Wang, Xuefei (Julie) and Laubscher, Emily and Yu, Changhua and Dilip, Rohit and Li, Qilin and Mari, Joud and Soro, Johnathon and Yu, Kevin and Pradhan, Elora and Ates, Ada and Gallandt, Danielle and Barnowski, Ross and Pao, Edward and Van Valen, David},
  doi          = {10.1038/s41592-023-01931-x},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {956-957},
  shortjournal = {Nat. Meth.},
  title        = {Scaling biological discovery at the interface of deep learning and cellular imaging},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Towards foundation models of biological image segmentation.
<em>NMETH</em>, <em>20</em>(7), 953–955. (<a
href="https://doi.org/10.1038/s41592-023-01885-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the ever-evolving landscape of biological imaging technology, it is crucial to develop foundation models capable of adapting to various imaging modalities and tackling complex segmentation tasks.},
  archive      = {J_NMETH},
  author       = {Ma, Jun and Wang, Bo},
  doi          = {10.1038/s41592-023-01885-0},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {953-955},
  shortjournal = {Nat. Meth.},
  title        = {Towards foundation models of biological image segmentation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The future of bioimage analysis: A dialog between mind and
machine. <em>NMETH</em>, <em>20</em>(7), 951–952. (<a
href="https://doi.org/10.1038/s41592-023-01930-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The field of bioimage analysis is poised for a major transformation, owing to advancements in imaging technologies and artificial intelligence. The emergence of multimodal foundation models — which are akin to large language models (such as ChatGPT) but are capable of comprehending and processing biological images — holds great potential for ushering in a revolutionary era in bioimage analysis.},
  archive      = {J_NMETH},
  author       = {Royer, Loïc A.},
  doi          = {10.1038/s41592-023-01930-y},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {951-952},
  shortjournal = {Nat. Meth.},
  title        = {The future of bioimage analysis: A dialog between mind and machine},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The cavefish astyanax mexicanus. <em>NMETH</em>,
<em>20</em>(7), 948–950. (<a
href="https://doi.org/10.1038/s41592-023-01916-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Astyanax mexicanus exists in two forms: a surface form that is abundantly distributed throughout freshwater bodies in Middle America and a blind subterranean form endemic to caves in northeastern Mexico. These diverse fish populations have become a vertebrate model for investigating the genetic basis of environmental adaptation.},
  archive      = {J_NMETH},
  author       = {Rohner, Nicolas},
  doi          = {10.1038/s41592-023-01916-w},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {948-950},
  shortjournal = {Nat. Meth.},
  title        = {The cavefish astyanax mexicanus},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Grant-writing rituals. <em>NMETH</em>, <em>20</em>(7), 947.
(<a href="https://doi.org/10.1038/s41592-023-01920-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Everyone has their own methods to address the time-consuming and challenging task of grant-writing.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01920-0},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {947},
  shortjournal = {Nat. Meth.},
  title        = {Grant-writing rituals},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). What’s next for bioimage analysis? <em>NMETH</em>,
<em>20</em>(7), 945–946. (<a
href="https://doi.org/10.1038/s41592-023-01950-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advanced bioimage analysis tools are poised to disrupt the way in which microscopy images are acquired and analyzed. This Focus issue shares the hopes and opinions of experts on the near and distant future of image analysis.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01950-8},
  journal      = {Nature Methods},
  month        = {7},
  number       = {7},
  pages        = {945-946},
  shortjournal = {Nat. Meth.},
  title        = {What’s next for bioimage analysis?},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cross-modality supervised image restoration enables
nanoscale tracking of synaptic plasticity in living mice.
<em>NMETH</em>, <em>20</em>(6), 935–944. (<a
href="https://doi.org/10.1038/s41592-023-01871-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Learning is thought to involve changes in glutamate receptors at synapses, submicron structures that mediate communication between neurons in the central nervous system. Due to their small size and high density, synapses are difficult to resolve in vivo, limiting our ability to directly relate receptor dynamics to animal behavior. Here we developed a combination of computational and biological methods to overcome these challenges. First, we trained a deep-learning image-restoration algorithm that combines the advantages of ex vivo super-resolution and in vivo imaging modalities to overcome limitations specific to each optical system. When applied to in vivo images from transgenic mice expressing fluorescently labeled glutamate receptors, this restoration algorithm super-resolved synapses, enabling the tracking of behavior-associated synaptic plasticity with high spatial resolution. This method demonstrates the capabilities of image enhancement to learn from ex vivo data and imaging techniques to improve in vivo imaging resolution. XTC is a supervised deep-learning-based image-restoration approach that is trained with images from different modalities and applied to an in vivo modality with no ground truth. XTC’s capabilities are demonstrated in synapse tracking in the mouse brain.},
  archive      = {J_NMETH},
  author       = {Xu, Yu Kang T. and Graves, Austin R. and Coste, Gabrielle I. and Huganir, Richard L. and Bergles, Dwight E. and Charles, Adam S. and Sulam, Jeremias},
  doi          = {10.1038/s41592-023-01871-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {935-944},
  shortjournal = {Nat. Meth.},
  title        = {Cross-modality supervised image restoration enables nanoscale tracking of synaptic plasticity in living mice},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Glutamate indicators with improved activation kinetics and
localization for imaging synaptic transmission. <em>NMETH</em>,
<em>20</em>(6), 925–934. (<a
href="https://doi.org/10.1038/s41592-023-01863-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The fluorescent glutamate indicator iGluSnFR enables imaging of neurotransmission with genetic and molecular specificity. However, existing iGluSnFR variants exhibit low in vivo signal-to-noise ratios, saturating activation kinetics and exclusion from postsynaptic densities. Using a multiassay screen in bacteria, soluble protein and cultured neurons, we generated variants with improved signal-to-noise ratios and kinetics. We developed surface display constructs that improve iGluSnFR’s nanoscopic localization to postsynapses. The resulting indicator iGluSnFR3 exhibits rapid nonsaturating activation kinetics and reports synaptic glutamate release with decreased saturation and increased specificity versus extrasynaptic signals in cultured neurons. Simultaneous imaging and electrophysiology at individual boutons in mouse visual cortex showed that iGluSnFR3 transients report single action potentials with high specificity. In vibrissal sensory cortex layer 4, we used iGluSnFR3 to characterize distinct patterns of touch-evoked feedforward input from thalamocortical boutons and both feedforward and recurrent input onto L4 cortical neuron dendritic spines. iGluSnFR variants with improved signal-to-noise ratios and targeting to postsynaptic sites have been developed, enabling the analysis of glutamatergic neurotransmission in vivo as illustrated in the mouse visual and somatosensory cortex.},
  archive      = {J_NMETH},
  author       = {Aggarwal, Abhi and Liu, Rui and Chen, Yang and Ralowicz, Amelia J. and Bergerson, Samuel J. and Tomaska, Filip and Mohar, Boaz and Hanson, Timothy L. and Hasseman, Jeremy P. and Reep, Daniel and Tsegaye, Getahun and Yao, Pantong and Ji, Xiang and Kloos, Marinus and Walpita, Deepika and Patel, Ronak and Mohr, Manuel A. and Tillberg, Paul W. and Looger, Loren L. and Marvin, Jonathan S. and Hoppa, Michael B. and Konnerth, Arthur and Kleinfeld, David and Schreiter, Eric R. and Podgorski, Kaspar},
  doi          = {10.1038/s41592-023-01863-6},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {925-934},
  shortjournal = {Nat. Meth.},
  title        = {Glutamate indicators with improved activation kinetics and localization for imaging synaptic transmission},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Engineering of NEMO as calcium indicators with large
dynamics and high sensitivity. <em>NMETH</em>, <em>20</em>(6), 918–924.
(<a href="https://doi.org/10.1038/s41592-023-01852-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetically encoded calcium indicators (GECIs) are indispensable tools for real-time monitoring of intracellular calcium signals and cellular activities in living organisms. Current GECIs face the challenge of suboptimal peak signal-to-baseline ratio (SBR) with limited resolution for reporting subtle calcium transients. We report herein the development of a suite of calcium sensors, designated NEMO, with fast kinetics and wide dynamic ranges (&amp;gt;100-fold). NEMO indicators report Ca2+ transients with peak SBRs around 20-fold larger than the top-of-the-range GCaMP6 series. NEMO sensors further enable the quantification of absolution calcium concentration with ratiometric or photochromic imaging. Compared with GCaMP6s, NEMOs could detect single action potentials in neurons with a peak SBR two times higher and a median peak SBR four times larger in vivo, thereby outperforming most existing state-of-the-art GECIs. Given their high sensitivity and resolution to report intracellular Ca2+ signals, NEMO sensors may find broad applications in monitoring neuronal activities and other Ca2+-modulated physiological processes in both mammals and plants. The NEMO series of genetically encoded calcium indicators report calcium activity in neuronal and non-neuronal cells with high signal-to-baseline ratio, which is shown in neuronal culture, slice preparations, in vivo and in planta.},
  archive      = {J_NMETH},
  author       = {Li, Jia and Shang, Ziwei and Chen, Jia-Hui and Gu, Wenjia and Yao, Li and Yang, Xin and Sun, Xiaowen and Wang, Liuqing and Wang, Tianlu and Liu, Siyao and Li, Jiajing and Hou, Tingting and Xing, Dajun and Gill, Donald L. and Li, Jiejie and Wang, Shi-Qiang and Hou, Lijuan and Zhou, Yubin and Tang, Ai-Hui and Zhang, Xiaohui and Wang, Youjun},
  doi          = {10.1038/s41592-023-01852-9},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {918-924},
  shortjournal = {Nat. Meth.},
  title        = {Engineering of NEMO as calcium indicators with large dynamics and high sensitivity},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Engineered allostery in light-regulated LOV-turbo enables
precise spatiotemporal control of proximity labeling in living cells.
<em>NMETH</em>, <em>20</em>(6), 908–917. (<a
href="https://doi.org/10.1038/s41592-023-01880-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The incorporation of light-responsive domains into engineered proteins has enabled control of protein localization, interactions and function with light. We integrated optogenetic control into proximity labeling, a cornerstone technique for high-resolution proteomic mapping of organelles and interactomes in living cells. Through structure-guided screening and directed evolution, we installed the light-sensitive LOV domain into the proximity labeling enzyme TurboID to rapidly and reversibly control its labeling activity with low-power blue light. ‘LOV-Turbo’ works in multiple contexts and dramatically reduces background in biotin-rich environments such as neurons. We used LOV-Turbo for pulse-chase labeling to discover proteins that traffic between endoplasmic reticulum, nuclear and mitochondrial compartments under cellular stress. We also showed that instead of external light, LOV-Turbo can be activated by bioluminescence resonance energy transfer from luciferase, enabling interaction-dependent proximity labeling. Overall, LOV-Turbo increases the spatial and temporal precision of proximity labeling, expanding the scope of experimental questions that can be addressed with proximity labeling. The light-sensitive LOV domain was engineered into the TurboID enzyme, creating ‘LOV-Turbo’. LOV-Turbo enables optogenetic control over proximity labeling, increasing the spatiotemporal precision of this technique.},
  archive      = {J_NMETH},
  author       = {Lee, Song-Yi and Cheah, Joleen S. and Zhao, Boxuan and Xu, Charles and Roh, Heegwang and Kim, Christina K. and Cho, Kelvin F. and Udeshi, Namrata D. and Carr, Steven A. and Ting, Alice Y.},
  doi          = {10.1038/s41592-023-01880-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {908-917},
  shortjournal = {Nat. Meth.},
  title        = {Engineered allostery in light-regulated LOV-turbo enables precise spatiotemporal control of proximity labeling in living cells},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome-wide profiling of prime editor off-target sites in
vitro and in vivo using PE-tag. <em>NMETH</em>, <em>20</em>(6), 898–907.
(<a href="https://doi.org/10.1038/s41592-023-01859-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editors have a broad range of potential research and clinical applications. However, methods to delineate their genome-wide editing activities have generally relied on indirect genome-wide editing assessments or the computational prediction of near-cognate sequences. Here we describe a genome-wide approach for the identification of potential prime editor off-target sites, which we call PE-tag. This method relies on the attachment or insertion of an amplification tag at sites of prime editor activity to allow their identification. PE-tag enables genome-wide profiling of off-target sites in vitro using extracted genomic DNA, in mammalian cell lines and in the adult mouse liver. PE-tag components can be delivered in a variety of formats for off-target site detection. Our studies are consistent with the high specificity previously described for prime editor systems, but we find that off-target editing rates are influenced by prime editing guide RNA design. PE-tag represents an accessible, rapid and sensitive approach for the genome-wide identification of prime editor activity and the evaluation of prime editor safety. PE-tag is a genome-wide approach to map the off-target effects of in vitro and in vivo prime editing.},
  archive      = {J_NMETH},
  author       = {Liang, Shun-Qing and Liu, Pengpeng and Ponnienselvan, Karthikeyan and Suresh, Sneha and Chen, Zexiang and Kramme, Christian and Chatterjee, Pranam and Zhu, Lihua Julie and Sontheimer, Erik J. and Xue, Wen and Wolfe, Scot A.},
  doi          = {10.1038/s41592-023-01859-2},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {898-907},
  shortjournal = {Nat. Meth.},
  title        = {Genome-wide profiling of prime editor off-target sites in vitro and in vivo using PE-tag},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Direct determination of oligomeric organization of integral
membrane proteins and lipids from intact customizable bilayer.
<em>NMETH</em>, <em>20</em>(6), 891–897. (<a
href="https://doi.org/10.1038/s41592-023-01864-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Hierarchical organization of integral membrane proteins (IMP) and lipids at the membrane is essential for regulating myriad downstream signaling. A quantitative understanding of these processes requires both detections of oligomeric organization of IMPs and lipids directly from intact membranes and determination of key membrane components and properties that regulate them. Addressing this, we have developed a platform that enables native mass spectrometry (nMS) analysis of IMP–lipid complexes directly from intact and customizable lipid membranes. Both the lipid composition and membrane properties (such as curvature, tension, and fluidity) of these bilayers can be precisely customized to a target membrane. Subsequent direct nMS analysis of these intact proteolipid vesicles can yield the oligomeric states of the embedded IMPs, identify bound lipids, and determine the membrane properties that can regulate the observed IMP–lipid organization. Applying this method, we show how lipid binding regulates neurotransmitter release and how membrane composition regulates the functional oligomeric state of a transporter. Editor summary: A native-mass-spectrometry-based approach analyzes integral membrane protein–lipid complexes directly from near-physiological membrane conditions, providing information about protein oligomeric states, lipid identities, and membrane properties.},
  archive      = {J_NMETH},
  author       = {Panda, Aniruddha and Giska, Fabian and Duncan, Anna L. and Welch, Alexander J. and Brown, Caroline and McAllister, Rachel and Hariharan, Parameswaran and Goder, Jean N. D. and Coleman, Jeff and Ramakrishnan, Sathish and Pincet, Frédéric and Guan, Lan and Krishnakumar, Shyam and Rothman, James E. and Gupta, Kallol},
  doi          = {10.1038/s41592-023-01864-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {891-897},
  shortjournal = {Nat. Meth.},
  title        = {Direct determination of oligomeric organization of integral membrane proteins and lipids from intact customizable bilayer},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). BUDDY: Molecular formula discovery via bottom-up MS/MS
interrogation. <em>NMETH</em>, <em>20</em>(6), 881–890. (<a
href="https://doi.org/10.1038/s41592-023-01850-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A substantial fraction of metabolic features remains undetermined in mass spectrometry (MS)-based metabolomics, and molecular formula annotation is the starting point for unraveling their chemical identities. Here we present bottom-up tandem MS (MS/MS) interrogation, a method for de novo formula annotation. Our approach prioritizes MS/MS-explainable formula candidates, implements machine-learned ranking and offers false discovery rate estimation. Compared with the mathematically exhaustive formula enumeration, our approach shrinks the formula candidate space by 42.8\% on average. Method benchmarking on annotation accuracy was systematically carried out on reference MS/MS libraries and real metabolomics datasets. Applied on 155,321 recurrent unidentified spectra, our approach confidently annotated &amp;gt;5,000 novel molecular formulae absent from chemical databases. Beyond the level of individual metabolic features, we combined bottom-up MS/MS interrogation with global optimization to refine formula annotations while revealing peak interrelationships. This approach allowed the systematic annotation of 37 fatty acid amide molecules in human fecal data. All bioinformatics pipelines are available in a standalone software, BUDDY ( https://github.com/HuanLab/BUDDY ). BUDDY is a bottom-up tandem MS (MS/MS) interrogation method for de novo molecular formula annotation with significance estimation.},
  archive      = {J_NMETH},
  author       = {Xing, Shipei and Shen, Sam and Xu, Banghua and Li, Xiaoxiao and Huan, Tao},
  doi          = {10.1038/s41592-023-01850-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {881-890},
  shortjournal = {Nat. Meth.},
  title        = {BUDDY: Molecular formula discovery via bottom-up MS/MS interrogation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). TomoTwin: Generalized 3D localization of macromolecules in
cryo-electron tomograms with structural data mining. <em>NMETH</em>,
<em>20</em>(6), 871–880. (<a
href="https://doi.org/10.1038/s41592-023-01878-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryogenic-electron tomography enables the visualization of cellular environments in extreme detail, however, tools to analyze the full amount of information contained within these densely packed volumes are still needed. Detailed analysis of macromolecules through subtomogram averaging requires particles to first be localized within the tomogram volume, a task complicated by several factors including a low signal to noise ratio and crowding of the cellular space. Available methods for this task suffer either from being error prone or requiring manual annotation of training data. To assist in this crucial particle picking step, we present TomoTwin: an open source general picking model for cryogenic-electron tomograms based on deep metric learning. By embedding tomograms in an information-rich, high-dimensional space that separates macromolecules according to their three-dimensional structure, TomoTwin allows users to identify proteins in tomograms de novo without manually creating training data or retraining the network to locate new proteins. TomoTwin is a deep metric learning-based particle picking method for cryo-electron tomograms. TomoTwin obviates the need for annotating training data and retraining a picking model for each protein.},
  archive      = {J_NMETH},
  author       = {Rice, Gavin and Wagner, Thorsten and Stabrin, Markus and Sitsel, Oleg and Prumbaum, Daniel and Raunser, Stefan},
  doi          = {10.1038/s41592-023-01878-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {871-880},
  shortjournal = {Nat. Meth.},
  title        = {TomoTwin: Generalized 3D localization of macromolecules in cryo-electron tomograms with structural data mining},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). 3DFlex: Determining structure and motion of flexible
proteins from cryo-EM. <em>NMETH</em>, <em>20</em>(6), 860–870. (<a
href="https://doi.org/10.1038/s41592-023-01853-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modeling flexible macromolecules is one of the foremost challenges in single-particle cryogenic-electron microscopy (cryo-EM), with the potential to illuminate fundamental questions in structural biology. We introduce Three-Dimensional Flexible Refinement (3DFlex), a motion-based neural network model for continuous molecular heterogeneity for cryo-EM data. 3DFlex exploits knowledge that conformational variability of a protein is often the result of physical processes that transport density over space and tend to preserve local geometry. From two-dimensional image data, 3DFlex enables the determination of high-resolution 3D density, and provides an explicit model of a flexible protein’s motion over its conformational landscape. Experimentally, for large molecular machines (tri-snRNP spliceosome complex, translocating ribosome) and small flexible proteins (TRPV1 ion channel, αVβ8 integrin, SARS-CoV-2 spike), 3DFlex learns nonrigid molecular motions while resolving details of moving secondary structure elements. 3DFlex can improve 3D density resolution beyond the limits of existing methods because particle images contribute coherent signal over the conformational landscape. 3D Flexible Refinement (3DFlex) is a generative neural network model for continuous molecular heterogeneity for cryo-EM data that can be used to determine the structure and motion of flexible biomolecules. It enables visualization of nonrigid motion and improves 3D structure resolution by aggregating information from particle images spanning the conformational landscape of the target molecule.},
  archive      = {J_NMETH},
  author       = {Punjani, Ali and Fleet, David J.},
  doi          = {10.1038/s41592-023-01853-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {860-870},
  shortjournal = {Nat. Meth.},
  title        = {3DFlex: Determining structure and motion of flexible proteins from cryo-EM},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Nano-DMS-MaP allows isoform-specific RNA structure
determination. <em>NMETH</em>, <em>20</em>(6), 849–859. (<a
href="https://doi.org/10.1038/s41592-023-01862-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide measurements of RNA structure can be obtained using reagents that react with unpaired bases, leading to adducts that can be identified by mutational profiling on next-generation sequencing machines. One drawback of these experiments is that short sequencing reads can rarely be mapped to specific transcript isoforms. Consequently, information is acquired as a population average in regions that are shared between transcripts, thus blurring the underlying structural landscape. Here, we present nanopore dimethylsulfate mutational profiling (Nano-DMS-MaP)—a method that exploits long-read sequencing to provide isoform-resolved structural information of highly similar RNA molecules. We demonstrate the value of Nano-DMS-MaP by resolving the complex structural landscape of human immunodeficiency virus-1 transcripts in infected cells. We show that unspliced and spliced transcripts have distinct structures at the packaging site within the common 5′ untranslated region, likely explaining why spliced viral RNAs are excluded from viral particles. Thus, Nano-DMS-MaP is a straightforward method to resolve biologically important transcript-specific RNA structures that were previously hidden in short-read ensemble analyses. Nano-DMS-MaP combines the power of DMS mutational profiling and long-read nanopore sequencing to resolve structural differences among RNA isoforms, revealing the structural landscape of HIV-1 transcripts in cells.},
  archive      = {J_NMETH},
  author       = {Bohn, Patrick and Gribling-Burrer, Anne-Sophie and Ambi, Uddhav B. and Smyth, Redmond P.},
  doi          = {10.1038/s41592-023-01862-7},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {849-859},
  shortjournal = {Nat. Meth.},
  title        = {Nano-DMS-MaP allows isoform-specific RNA structure determination},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fast and efficient template-mediated synthesis of genetic
variants. <em>NMETH</em>, <em>20</em>(6), 841–848. (<a
href="https://doi.org/10.1038/s41592-023-01868-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efficient methods for the generation of specific mutations enable the study of functional variations in natural populations and lead to advances in genetic engineering applications. Here, we present a new approach, mutagenesis by template-guided amplicon assembly (MEGAA), for the rapid construction of kilobase-sized DNA variants. With this method, many mutations can be generated at a time to a DNA template at more than 90\% efficiency per target in a predictable manner. We devised a robust and iterative protocol for an open-source laboratory automation robot that enables desktop production and long-read sequencing validation of variants. Using this system, we demonstrated the construction of 31 natural SARS-CoV2 spike gene variants and 10 recoded Escherichia coli genome fragments, with each 4 kb region containing up to 150 mutations. Furthermore, 125 defined combinatorial adeno-associated virus-2 cap gene variants were easily built using the system, which exhibited viral packaging enhancements of up to 10-fold compared with wild type. Thus, the MEGAA platform enables generation of multi-site sequence variants quickly, cheaply, and in a scalable manner for diverse applications in biotechnology. MEGAA, an efficient template-guided method to synthesize high-fidelity DNA variants},
  archive      = {J_NMETH},
  author       = {Liu, Liyuan and Huang, Yiming and Wang, Harris H.},
  doi          = {10.1038/s41592-023-01868-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {841-848},
  shortjournal = {Nat. Meth.},
  title        = {Fast and efficient template-mediated synthesis of genetic variants},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-contrast en bloc staining of mouse whole-brain and
human brain samples for EM-based connectomics. <em>NMETH</em>,
<em>20</em>(6), 836–840. (<a
href="https://doi.org/10.1038/s41592-023-01866-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Connectomes of human cortical gray matter require high-contrast homogeneously stained samples sized at least 2 mm on a side, and a mouse whole-brain connectome requires samples sized at least 5–10 mm on a side. Here we report en bloc staining and embedding protocols for these and other applications, removing a key obstacle for connectomic analyses at the mammalian whole-brain level. For EM-based connectomics applications, a staining protocol for large tissue samples in the range of a centimeter has been developed, which avoids artifacts common with established protocols.},
  archive      = {J_NMETH},
  author       = {Song, Kun and Feng, Zhihui and Helmstaedter, Moritz},
  doi          = {10.1038/s41592-023-01866-3},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {836-840},
  shortjournal = {Nat. Meth.},
  title        = {High-contrast en bloc staining of mouse whole-brain and human brain samples for EM-based connectomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). BigNeuron: A resource to benchmark and predict performance
of algorithms for automated tracing of neurons in light microscopy
datasets. <em>NMETH</em>, <em>20</em>(6), 824–835. (<a
href="https://doi.org/10.1038/s41592-023-01848-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {BigNeuron is an open community bench-testing platform with the goal of setting open standards for accurate and fast automatic neuron tracing. We gathered a diverse set of image volumes across several species that is representative of the data obtained in many neuroscience laboratories interested in neuron tracing. Here, we report generated gold standard manual annotations for a subset of the available imaging datasets and quantified tracing quality for 35 automatic tracing algorithms. The goal of generating such a hand-curated diverse dataset is to advance the development of tracing algorithms and enable generalizable benchmarking. Together with image quality features, we pooled the data in an interactive web application that enables users and developers to perform principal component analysis, t-distributed stochastic neighbor embedding, correlation and clustering, visualization of imaging and tracing data, and benchmarking of automatic tracing algorithms in user-defined data subsets. The image quality metrics explain most of the variance in the data, followed by neuromorphological features related to neuron size. We observed that diverse algorithms can provide complementary information to obtain accurate results and developed a method to iteratively combine methods and generate consensus reconstructions. The consensus trees obtained provide estimates of the neuron structure ground truth that typically outperform single algorithms in noisy datasets. However, specific algorithms may outperform the consensus tree strategy in specific imaging conditions. Finally, to aid users in predicting the most accurate automatic tracing results without manual annotations for comparison, we used support vector machine regression to predict reconstruction quality given an image volume and a set of automatic tracings. This resource describes a collection of neurons from a variety of light microscopy-based datasets, which can serve as a gold standard for testing automated tracing algorithms, as shown by comparison of the performance of 35 algorithms.},
  archive      = {J_NMETH},
  author       = {Manubens-Gil, Linus and Zhou, Zhi and Chen, Hanbo and Ramanathan, Arvind and Liu, Xiaoxiao and Liu, Yufeng and Bria, Alessandro and Gillette, Todd and Ruan, Zongcai and Yang, Jian and Radojević, Miroslav and Zhao, Ting and Cheng, Li and Qu, Lei and Liu, Siqi and Bouchard, Kristofer E. and Gu, Lin and Cai, Weidong and Ji, Shuiwang and Roysam, Badrinath and Wang, Ching-Wei and Yu, Hongchuan and Sironi, Amos and Iascone, Daniel Maxim and Zhou, Jie and Bas, Erhan and Conde-Sousa, Eduardo and Aguiar, Paulo and Li, Xiang and Li, Yujie and Nanda, Sumit and Wang, Yuan and Muresan, Leila and Fua, Pascal and Ye, Bing and He, Hai-yan and Staiger, Jochen F. and Peter, Manuel and Cox, Daniel N. and Simonneau, Michel and Oberlaender, Marcel and Jefferis, Gregory and Ito, Kei and Gonzalez-Bellido, Paloma and Kim, Jinhyun and Rubel, Edwin and Cline, Hollis T. and Zeng, Hongkui and Nern, Aljoscha and Chiang, Ann-Shyn and Yao, Jianhua and Roskams, Jane and Livesey, Rick and Stevens, Janine and Liu, Tianming and Dang, Chinh and Guo, Yike and Zhong, Ning and Tourassi, Georgia and Hill, Sean and Hawrylycz, Michael and Koch, Christof and Meijering, Erik and Ascoli, Giorgio A. and Peng, Hanchuan},
  doi          = {10.1038/s41592-023-01848-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {824-835},
  shortjournal = {Nat. Meth.},
  title        = {BigNeuron: A resource to benchmark and predict performance of algorithms for automated tracing of neurons in light microscopy datasets},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). EmbryoNet: Using deep learning to link embryonic phenotypes
to signaling pathways. <em>NMETH</em>, <em>20</em>(6), 815–823. (<a
href="https://doi.org/10.1038/s41592-023-01873-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evolutionarily conserved signaling pathways are essential for early embryogenesis, and reducing or abolishing their activity leads to characteristic developmental defects. Classification of phenotypic defects can identify the underlying signaling mechanisms, but this requires expert knowledge and the classification schemes have not been standardized. Here we use a machine learning approach for automated phenotyping to train a deep convolutional neural network, EmbryoNet, to accurately identify zebrafish signaling mutants in an unbiased manner. Combined with a model of time-dependent developmental trajectories, this approach identifies and classifies with high precision phenotypic defects caused by loss of function of the seven major signaling pathways relevant for vertebrate development. Our classification algorithms have wide applications in developmental biology and robustly identify signaling defects in evolutionarily distant species. Furthermore, using automated phenotyping in high-throughput drug screens, we show that EmbryoNet can resolve the mechanism of action of pharmaceutical substances. As part of this work, we freely provide more than 2 million images that were used to train and test EmbryoNet. EmbryoNet is an automated approach to the phenotyping of developing embryos that surpasses experts in terms of speed, accuracy and sensitivity. A large annotated image dataset of zebrafish, medaka and stickleback development rounds out this resource.},
  archive      = {J_NMETH},
  author       = {Čapek, Daniel and Safroshkin, Matvey and Morales-Navarrete, Hernán and Toulany, Nikan and Arutyunov, Grigory and Kurzbach, Anica and Bihler, Johanna and Hagauer, Julia and Kick, Sebastian and Jones, Felicity and Jordan, Ben and Müller, Patrick},
  doi          = {10.1038/s41592-023-01873-4},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {815-823},
  shortjournal = {Nat. Meth.},
  title        = {EmbryoNet: Using deep learning to link embryonic phenotypes to signaling pathways},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Machine learning in rare disease. <em>NMETH</em>,
<em>20</em>(6), 803–814. (<a
href="https://doi.org/10.1038/s41592-023-01886-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-throughput profiling methods (such as genomics or imaging) have accelerated basic research and made deep molecular characterization of patient samples routine. These approaches provide a rich portrait of genes, molecular pathways and cell types involved in disease phenotypes. Machine learning (ML) can be a useful tool for extracting disease-relevant patterns from high-dimensional datasets. However, depending upon the complexity of the biological question, machine learning often requires many samples to identify recurrent and biologically meaningful patterns. Rare diseases are inherently limited in clinical cases, leading to few samples to study. In this Perspective, we outline the challenges and emerging solutions for using ML for small sample sets, specifically in rare diseases. Advances in ML methods for rare diseases are likely to be informative for applications beyond rare diseases for which few samples exist with high-dimensional data. We propose that the method community prioritize the development of ML techniques for rare disease research. This Perspective discusses how machine learning can help in studying rare diseases using various emerging approaches.},
  archive      = {J_NMETH},
  author       = {Banerjee, Jineta and Taroni, Jaclyn N. and Allaway, Robert J. and Prasad, Deepashree Venkatesh and Guinney, Justin and Greene, Casey},
  doi          = {10.1038/s41592-023-01886-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {803-814},
  shortjournal = {Nat. Meth.},
  title        = {Machine learning in rare disease},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Capturing prime editor off-target sites within the genome.
<em>NMETH</em>, <em>20</em>(6), 801–802. (<a
href="https://doi.org/10.1038/s41592-023-01860-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editing systems hold tremendous promise for the precise correction of pathogenic mutations. We developed a method to tag sequences modified by a prime editor to evaluate its genome-wide precision for therapeutic applications.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01860-9},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {801-802},
  shortjournal = {Nat. Meth.},
  title        = {Capturing prime editor off-target sites within the genome},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Accurate determination of molecular formulae using tandem
mass spectrometry. <em>NMETH</em>, <em>20</em>(6), 799–800. (<a
href="https://doi.org/10.1038/s41592-023-01851-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We highlight the BUDDY software, which was developed to accurately determine the molecular formulae of unknown chemicals in mass spectrometry data. BUDDY is a bottom-up approach that shows superior annotation performance on reference spectra and experimental datasets. Incorporation of global peak annotation could enable BUDDY to refine formula annotations and reveal feature interrelationships.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01851-w},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {799-800},
  shortjournal = {Nat. Meth.},
  title        = {Accurate determination of molecular formulae using tandem mass spectrometry},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mapping the motion and structure of flexible proteins from
cryo-EM data. <em>NMETH</em>, <em>20</em>(6), 797–798. (<a
href="https://doi.org/10.1038/s41592-023-01883-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A deep learning algorithm maps out the continuous conformational changes of flexible protein molecules from single-particle cryo-electron microscopy images, allowing the visualization of the conformational landscape of a protein with improved resolution of its moving parts.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01883-2},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {797-798},
  shortjournal = {Nat. Meth.},
  title        = {Mapping the motion and structure of flexible proteins from cryo-EM data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Highly efficient and rapid generation of genetic variants.
<em>NMETH</em>, <em>20</em>(6), 795–796. (<a
href="https://doi.org/10.1038/s41592-023-01869-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new mutagenesis platform enables the fast, cost-efficient and automatable production of defined multi-site sequence variants for a wide range of applications. Demonstrations of this method included the generation of SARS-CoV-2 spike gene variants, DNA fragments for large-scale genome engineering, and adeno-associated virus 2 (AAV2) cap genes with improved packaging capacity.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01869-0},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {795-796},
  shortjournal = {Nat. Meth.},
  title        = {Highly efficient and rapid generation of genetic variants},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). AI identifies developmental defects and drug mechanisms in
embryos. <em>NMETH</em>, <em>20</em>(6), 793–794. (<a
href="https://doi.org/10.1038/s41592-023-01872-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed EmbryoNet, a deep learning tool that can automatically identify and classify developmental defects caused by perturbations of signaling pathways in vertebrate embryos. The tool could help to elucidate the mechanisms of action of pharmaceuticals, potentially transforming the drug discovery process.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01872-5},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {793-794},
  shortjournal = {Nat. Meth.},
  title        = {AI identifies developmental defects and drug mechanisms in embryos},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Identifying the structures of individual RNA isoforms inside
cells. <em>NMETH</em>, <em>20</em>(6), 791–792. (<a
href="https://doi.org/10.1038/s41592-023-01904-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nano-DMS-MaP focuses in on the structures of individual RNA isoforms, enabling direct examination of the structural diversity of different RNAs inside cells.},
  archive      = {J_NMETH},
  author       = {Kenyon, Julia C. and Mak, Johnson},
  doi          = {10.1038/s41592-023-01904-0},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {791-792},
  shortjournal = {Nat. Meth.},
  title        = {Identifying the structures of individual RNA isoforms inside cells},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023t). Welcome to the nursery. <em>NMETH</em>, <em>20</em>(6),
786–790. (<a href="https://doi.org/10.1038/s41592-023-01902-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As they pursue their puzzles of interest, scientists are inventive caretakers of the organisms in their keep, especially the young.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01902-2},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {786-790},
  shortjournal = {Nat. Meth.},
  title        = {Welcome to the nursery},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Protein language models guide directed antibody evolution.
<em>NMETH</em>, <em>20</em>(6), 785. (<a
href="https://doi.org/10.1038/s41592-023-01924-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-023-01924-w},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {785},
  shortjournal = {Nat. Meth.},
  title        = {Protein language models guide directed antibody evolution},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023k). Tracing lymphocyte development. <em>NMETH</em>,
<em>20</em>(6), 785. (<a
href="https://doi.org/10.1038/s41592-023-01925-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-023-01925-9},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {785},
  shortjournal = {Nat. Meth.},
  title        = {Tracing lymphocyte development},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). In situ electro-sequencing. <em>NMETH</em>, <em>20</em>(6),
784. (<a href="https://doi.org/10.1038/s41592-023-01922-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-023-01922-y},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {784},
  shortjournal = {Nat. Meth.},
  title        = {In situ electro-sequencing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Assembly-free phylogenetic trees. <em>NMETH</em>,
<em>20</em>(6), 784. (<a
href="https://doi.org/10.1038/s41592-023-01923-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-023-01923-x},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {784},
  shortjournal = {Nat. Meth.},
  title        = {Assembly-free phylogenetic trees},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Capturing hyperspectral images. <em>NMETH</em>,
<em>20</em>(6), 783. (<a
href="https://doi.org/10.1038/s41592-023-01921-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A single-shot hyperspectral phasor camera (SHy-Cam) enables fast, multiplexed volumetric imaging.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-023-01921-z},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {783},
  shortjournal = {Nat. Meth.},
  title        = {Capturing hyperspectral images},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Volume EM: A quiet revolution takes shape. <em>NMETH</em>,
<em>20</em>(6), 777–782. (<a
href="https://doi.org/10.1038/s41592-023-01861-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Volume electron microscopy (vEM) is a group of techniques that reveal the 3D ultrastructure of cells and tissues through continuous depths of at least 1 micrometer. A burgeoning grassroots community effort is fast building the profile and revealing the impact of vEM technology in the life sciences and clinical research.},
  archive      = {J_NMETH},
  author       = {Collinson, Lucy M. and Bosch, Carles and Bullen, Anwen and Burden, Jemima J. and Carzaniga, Raffaella and Cheng, Cheng and Darrow, Michele C. and Fletcher, Georgina and Johnson, Errin and Narayan, Kedar and Peddie, Christopher J. and Winn, Martyn and Wood, Charles and Patwardhan, Ardan and Kleywegt, Gerard J. and Verkade, Paul},
  doi          = {10.1038/s41592-023-01861-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {777-782},
  shortjournal = {Nat. Meth.},
  title        = {Volume EM: A quiet revolution takes shape},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). DAQ-score database: Assessment of map–model compatibility
for protein structure models from cryo-EM maps. <em>NMETH</em>,
<em>20</em>(6), 775–776. (<a
href="https://doi.org/10.1038/s41592-023-01876-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Nakamura, Tsukasa and Wang, Xiao and Terashi, Genki and Kihara, Daisuke},
  doi          = {10.1038/s41592-023-01876-1},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {775-776},
  shortjournal = {Nat. Meth.},
  title        = {DAQ-score database: Assessment of map–model compatibility for protein structure models from cryo-EM maps},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023q). The LGBTQ+ job hunt. <em>NMETH</em>, <em>20</em>(6), 774.
(<a href="https://doi.org/10.1038/s41592-023-01882-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Job-hunting is never easy, and more tasks get added on for members of the LGBTQ+ community as they search for welcoming environments.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01882-3},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {774},
  shortjournal = {Nat. Meth.},
  title        = {The LGBTQ+ job hunt},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Show us your resources. <em>NMETH</em>, <em>20</em>(6), 773.
(<a href="https://doi.org/10.1038/s41592-023-01926-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Methods is not just about methods. Resources too are important components of the research toolbox.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01926-8},
  journal      = {Nature Methods},
  month        = {6},
  number       = {6},
  pages        = {773},
  shortjournal = {Nat. Meth.},
  title        = {Show us your resources},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Julia for biologists. <em>NMETH</em>,
<em>20</em>(5), 771. (<a
href="https://doi.org/10.1038/s41592-023-01887-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Roesch, Elisabeth and Greener, Joe G. and MacLean, Adam L. and Nassar, Huda and Rackauckas, Christopher and Holy, Timothy E. and Stumpf, Michael P. H.},
  doi          = {10.1038/s41592-023-01887-y},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {771},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Julia for biologists},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Publisher correction: ELI trifocal microscope: A precise
system to prepare target cryo-lamellae for in situ cryo-ET study.
<em>NMETH</em>, <em>20</em>(5), 770. (<a
href="https://doi.org/10.1038/s41592-023-01867-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Li, Shuoguo and Wang, Ziyan and Jia, Xing and Niu, Tongxin and Zhang, Jianguo and Yin, Guoliang and Zhang, Xiaoyun and Zhu, Yun and Ji, Gang and Sun, Fei},
  doi          = {10.1038/s41592-023-01867-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {770},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: ELI trifocal microscope: a precise system to prepare target cryo-lamellae for in situ cryo-ET study},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Publisher correction: Rapid biomechanical imaging at low
irradiation level via dual line-scanning brillouin microscopy.
<em>NMETH</em>, <em>20</em>(5), 770. (<a
href="https://doi.org/10.1038/s41592-023-01877-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Zhang, Jitao and Nikolic, Milos and Tanner, Kandice and Scarcelli, Giuliano},
  doi          = {10.1038/s41592-023-01877-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {770},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Rapid biomechanical imaging at low irradiation level via dual line-scanning brillouin microscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Neural engineering with photons as synaptic transmitters.
<em>NMETH</em>, <em>20</em>(5), 761–769. (<a
href="https://doi.org/10.1038/s41592-023-01836-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neuronal computation is achieved through connections of individual neurons into a larger network. To expand the repertoire of endogenous cellular communication, we developed a synthetic, photon-assisted synaptic transmission (PhAST) system. PhAST is based on luciferases and channelrhodopsins that enable the transmission of a neuronal state across space, using photons as neurotransmitters. PhAST overcomes synaptic barriers and rescues the behavioral deficit of a glutamate mutant with conditional, calcium-triggered photon emission between two neurons of the Caenorhabditis elegans nociceptive avoidance circuit. To demonstrate versatility and flexibility, we generated de novo synaptic transmission between two unconnected cells in a sexually dimorphic neuronal circuit, suppressed endogenous nocifensive response through activation of an anion channelrhodopsin and switched attractive to aversive behavior in an olfactory circuit. Finally, we applied PhAST to dissect the calcium dynamics of the temporal pattern generator in a motor circuit for ovipositioning. In summary, we established photon-based synaptic transmission that facilitates the modification of animal behavior. PhAST is a technology for establishing de novo or modulating synaptic transmission in a light-dependent manner in C. elegans. By combining a calcium-dependent luciferase on pre-synapses with channelrhodopsin on post-synapses, light serves as a synthetic neurotransmitter.},
  archive      = {J_NMETH},
  author       = {Porta-de-la-Riva, Montserrat and Gonzalez, Adriana Carolina and Sanfeliu-Cerdán, Neus and Karimi, Shadi and Malaiwong, Nawaphat and Pidde, Aleksandra and Morales-Curiel, Luis-Felipe and Fernandez, Pablo and González-Bolívar, Sara and Hurth, Cedric and Krieg, Michael},
  doi          = {10.1038/s41592-023-01836-9},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {761-769},
  shortjournal = {Nat. Meth.},
  title        = {Neural engineering with photons as synaptic transmitters},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-resolution line-scan brillouin microscopy for live
imaging of mechanical properties during embryo development.
<em>NMETH</em>, <em>20</em>(5), 755–760. (<a
href="https://doi.org/10.1038/s41592-023-01822-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brillouin microscopy can assess mechanical properties of biological samples in a three-dimensional (3D), all-optical and hence non-contact fashion, but its weak signals often lead to long imaging times and require an illumination dosage harmful for living organisms. Here, we present a high-resolution line-scanning Brillouin microscope for multiplexed and hence fast 3D imaging of dynamic biological processes with low phototoxicity. The improved background suppression and resolution, in combination with fluorescence light-sheet imaging, enables the visualization of the mechanical properties of cells and tissues over space and time in living organism models such as fruit flies, ascidians and mouse embryos. Line-scan Brillouin microscopy enables fast 3D imaging of mechanical properties with low phototoxicity, as shown for Drosophila and mouse embryos, as well as ascidians.},
  archive      = {J_NMETH},
  author       = {Bevilacqua, Carlo and Gomez, Juan Manuel and Fiuza, Ulla-Maj and Chan, Chii Jou and Wang, Ling and Hambura, Sebastian and Eguren, Manuel and Ellenberg, Jan and Diz-Muñoz, Alba and Leptin, Maria and Prevedel, Robert},
  doi          = {10.1038/s41592-023-01822-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {755-760},
  shortjournal = {Nat. Meth.},
  title        = {High-resolution line-scan brillouin microscopy for live imaging of mechanical properties during embryo development},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rapid detection of neurons in widefield calcium imaging
datasets after training with synthetic data. <em>NMETH</em>,
<em>20</em>(5), 747–754. (<a
href="https://doi.org/10.1038/s41592-023-01838-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Widefield microscopy can provide optical access to multi-millimeter fields of view and thousands of neurons in mammalian brains at video rate. However, tissue scattering and background contamination results in signal deterioration, making the extraction of neuronal activity challenging, laborious and time consuming. Here we present our deep-learning-based widefield neuron finder (DeepWonder), which is trained by simulated functional recordings and effectively works on experimental data to achieve high-fidelity neuronal extraction. Equipped with systematic background contribution priors, DeepWonder conducts neuronal inference with an order-of-magnitude-faster speed and improved accuracy compared with alternative approaches. DeepWonder removes background contaminations and is computationally efficient. Specifically, DeepWonder accomplishes 50-fold signal-to-background ratio enhancement when processing terabytes-scale cortex-wide functional recordings, with over 14,000 neurons extracted in 17 h. DeepWonder removes background signals from widefield calcium recordings and enables accurate and efficient neuronal segmentation with high throughput.},
  archive      = {J_NMETH},
  author       = {Zhang, Yuanlong and Zhang, Guoxun and Han, Xiaofei and Wu, Jiamin and Li, Ziwei and Li, Xinyang and Xiao, Guihua and Xie, Hao and Fang, Lu and Dai, Qionghai},
  doi          = {10.1038/s41592-023-01838-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {747-754},
  shortjournal = {Nat. Meth.},
  title        = {Rapid detection of neurons in widefield calcium imaging datasets after training with synthetic data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Virtual-scanning light-field microscopy for robust snapshot
high-resolution volumetric imaging. <em>NMETH</em>, <em>20</em>(5),
735–746. (<a href="https://doi.org/10.1038/s41592-023-01839-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-speed three-dimensional (3D) intravital imaging in animals is useful for studying transient subcellular interactions and functions in health and disease. Light-field microscopy (LFM) provides a computational solution for snapshot 3D imaging with low phototoxicity but is restricted by low resolution and reconstruction artifacts induced by optical aberrations, motion and noise. Here, we propose virtual-scanning LFM (VsLFM), a physics-based deep learning framework to increase the resolution of LFM up to the diffraction limit within a snapshot. By constructing a 40 GB high-resolution scanning LFM dataset across different species, we exploit physical priors between phase-correlated angular views to address the frequency aliasing problem. This enables us to bypass hardware scanning and associated motion artifacts. Here, we show that VsLFM achieves ultrafast 3D imaging of diverse processes such as the beating heart in embryonic zebrafish, voltage activity in Drosophila brains and neutrophil migration in the mouse liver at up to 500 volumes per second. Virtual-scanning light-field microscopy (VsLFM) uses a physics-based deep learning model to improve the quality and speed of LFM, reducing motion artifacts and enabling challenging demonstrations such as fast 3D voltage imaging in Drosophila.},
  archive      = {J_NMETH},
  author       = {Lu, Zhi and Liu, Yu and Jin, Manchang and Luo, Xin and Yue, Huanjing and Wang, Zian and Zuo, Siqing and Zeng, Yunmin and Fan, Jiaqi and Pang, Yanwei and Wu, Jiamin and Yang, Jingyu and Dai, Qionghai},
  doi          = {10.1038/s41592-023-01839-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {735-746},
  shortjournal = {Nat. Meth.},
  title        = {Virtual-scanning light-field microscopy for robust snapshot high-resolution volumetric imaging},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Time-resolved assessment of single-cell protein secretion by
sequencing. <em>NMETH</em>, <em>20</em>(5), 723–734. (<a
href="https://doi.org/10.1038/s41592-023-01841-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Secreted proteins play critical roles in cellular communication. Methods enabling concurrent measurement of cellular protein secretion, phenotypes and transcriptomes are still unavailable. Here we describe time-resolved assessment of protein secretion from single cells by sequencing (TRAPS-seq). Released proteins are trapped onto the cell surface and probed by oligonucleotide-barcoded antibodies before being simultaneously sequenced with transcriptomes in single cells. We demonstrate that TRAPS-seq helps unravel the phenotypic and transcriptional determinants of the secretion of pleiotropic TH1 cytokines (IFNγ, IL-2 and TNF) in activated T cells. In addition, we show that TRAPS-seq can be used to track the secretion of multiple cytokines over time, uncovering unique molecular signatures that govern the dynamics of single-cell cytokine secretions. Our results revealed that early central memory T cells with CD45RA expression (TCMRA) are important in both the production and maintenance of polyfunctional cytokines. TRAPS-seq presents a unique tool for seamless integration of secretomics measurements with multi-omics profiling in single cells. TRAPS-seq is a sequencing-based method to profile dynamic protein secretion in single cells.},
  archive      = {J_NMETH},
  author       = {Wu, Tongjin and Womersley, Howard John and Wang, Jiehao Ray and Scolnick, Jonathan and Cheow, Lih Feng},
  doi          = {10.1038/s41592-023-01841-y},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {723-734},
  shortjournal = {Nat. Meth.},
  title        = {Time-resolved assessment of single-cell protein secretion by sequencing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Prioritized mass spectrometry increases the depth,
sensitivity and data completeness of single-cell proteomics.
<em>NMETH</em>, <em>20</em>(5), 714–722. (<a
href="https://doi.org/10.1038/s41592-023-01830-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Major aims of single-cell proteomics include increasing the consistency, sensitivity and depth of protein quantification, especially for proteins and modifications of biological interest. Here, to simultaneously advance all these aims, we developed prioritized Single-Cell ProtEomics (pSCoPE). pSCoPE consistently analyzes thousands of prioritized peptides across all single cells (thus increasing data completeness) while maximizing instrument time spent analyzing identifiable peptides, thus increasing proteome depth. These strategies increased the sensitivity, data completeness and proteome coverage over twofold. The gains enabled quantifying protein variation in untreated and lipopolysaccharide-treated primary macrophages. Within each condition, proteins covaried within functional sets, including phagosome maturation and proton transport, similarly across both treatment conditions. This covariation is coupled to phenotypic variability in endocytic activity. pSCoPE also enabled quantifying proteolytic products, suggesting a gradient of cathepsin activities within a treatment condition. pSCoPE is freely available and widely applicable, especially for analyzing proteins of interest without sacrificing proteome coverage. Support for pSCoPE is available at http://scp.slavovlab.net/pSCoPE . Prioritized Single-Cell ProtEomics (pSCoPE) introduces the concept of using priority levels that define the temporal order of peptide analysis for single-cell proteomic analysis. Prioritized data acquisition aims to simultaneously optimize the consistency, sensitivity, depth and accuracy of protein quantification.},
  archive      = {J_NMETH},
  author       = {Huffman, R. Gray and Leduc, Andrew and Wichmann, Christoph and Di Gioia, Marco and Borriello, Francesco and Specht, Harrison and Derks, Jason and Khan, Saad and Khoury, Luke and Emmott, Edward and Petelski, Aleksandra A. and Perlman, David H. and Cox, Jürgen and Zanoni, Ivan and Slavov, Nikolai},
  doi          = {10.1038/s41592-023-01830-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {714-722},
  shortjournal = {Nat. Meth.},
  title        = {Prioritized mass spectrometry increases the depth, sensitivity and data completeness of single-cell proteomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Improving the sensitivity of in vivo CRISPR off-target
detection with DISCOVER-seq+. <em>NMETH</em>, <em>20</em>(5), 706–713.
(<a href="https://doi.org/10.1038/s41592-023-01840-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Discovery of off-target CRISPR–Cas activity in patient-derived cells and animal models is crucial for genome editing applications, but currently exhibits low sensitivity. We demonstrate that inhibition of DNA-dependent protein kinase catalytic subunit accumulates the repair protein MRE11 at CRISPR–Cas-targeted sites, enabling high-sensitivity mapping of off-target sites to positions of MRE11 binding using chromatin immunoprecipitation followed by sequencing. This technique, termed DISCOVER-Seq+, discovered up to fivefold more CRISPR off-target sites in immortalized cell lines, primary human cells and mice compared with previous methods. We demonstrate applicability to ex vivo knock-in of a cancer-directed transgenic T cell receptor in primary human T cells and in vivo adenovirus knock-out of cardiovascular risk gene PCSK9 in mice. Thus, DISCOVER-Seq+ is, to our knowledge, the most sensitive method to-date for discovering off-target genome editing in vivo. DISCOVER-Seq+ is a high-sensitivity method for the detection of off-target effects of CRISPR genome editing.},
  archive      = {J_NMETH},
  author       = {Zou, Roger S. and Liu, Yang and Gaido, Oscar E. Reyes and Konig, Maximilian F. and Mog, Brian J. and Shen, Leo L. and Aviles-Vazquez, Franklin and Marin-Gonzalez, Alberto and Ha, Taekjip},
  doi          = {10.1038/s41592-023-01840-z},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {706-713},
  shortjournal = {Nat. Meth.},
  title        = {Improving the sensitivity of in vivo CRISPR off-target detection with DISCOVER-seq+},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatiotemporally resolved transcriptomics reveals the
subcellular RNA kinetic landscape. <em>NMETH</em>, <em>20</em>(5),
695–705. (<a href="https://doi.org/10.1038/s41592-023-01829-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatiotemporal regulation of the cellular transcriptome is crucial for proper protein expression and cellular function. However, the intricate subcellular dynamics of RNA remain obscured due to the limitations of existing transcriptomics methods. Here, we report TEMPOmap—a method that uncovers subcellular RNA profiles across time and space at the single-cell level. TEMPOmap integrates pulse-chase metabolic labeling with highly multiplexed three-dimensional in situ sequencing to simultaneously profile the age and location of individual RNA molecules. Using TEMPOmap, we constructed the subcellular RNA kinetic landscape in various human cells from transcription and translocation to degradation. Clustering analysis of RNA kinetic parameters across single cells revealed ‘kinetic gene clusters’ whose expression patterns were shaped by multistep kinetic sculpting. Importantly, these kinetic gene clusters are functionally segregated, suggesting that subcellular RNA kinetics are differentially regulated in a cell-state- and cell-type-dependent manner. Spatiotemporally resolved transcriptomics provides a gateway to uncovering new spatiotemporal gene regulation principles. TEMPOmap combines pulse-chase metabolic labeling with multiplexed three-dimensional in situ sequencing to simultaneously profile the age and subcellular location of individual RNA molecules from thousands of genes to reveal RNA kinetic landscapes.},
  archive      = {J_NMETH},
  author       = {Ren, Jingyi and Zhou, Haowen and Zeng, Hu and Wang, Connie Kangni and Huang, Jiahao and Qiu, Xiaojie and Sui, Xin and Li, Qiang and Wu, Xunwei and Lin, Zuwan and Lo, Jennifer A. and Maher, Kamal and He, Yichun and Tang, Xin and Lam, Judson and Chen, Hongyu and Li, Brian and Fisher, David E. and Liu, Jia and Wang, Xiao},
  doi          = {10.1038/s41592-023-01829-8},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {695-705},
  shortjournal = {Nat. Meth.},
  title        = {Spatiotemporally resolved transcriptomics reveals the subcellular RNA kinetic landscape},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Photoselective sequencing: Microscopically guided genomic
measurements with subcellular resolution. <em>NMETH</em>,
<em>20</em>(5), 686–694. (<a
href="https://doi.org/10.1038/s41592-023-01845-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In biological systems, spatial organization and function are interconnected. Here we present photoselective sequencing, a new method for genomic and epigenomic profiling within morphologically distinct regions. Starting with an intact biological specimen, photoselective sequencing uses targeted illumination to selectively unblock a photocaged fragment library, restricting the sequencing-based readout to microscopically identified spatial regions. We validate photoselective sequencing by measuring the chromatin accessibility profiles of fluorescently labeled cell types within the mouse brain and comparing with published data. Furthermore, by combining photoselective sequencing with a computational strategy for decomposing bulk accessibility profiles, we find that the oligodendrocyte-lineage-cell population is relatively enriched for oligodendrocyte-progenitor cells in the cortex versus the corpus callosum. Finally, we leverage photoselective sequencing at the subcellular scale to identify features of chromatin that are correlated with positioning at the nuclear periphery. These results collectively demonstrate that photoselective sequencing is a flexible and generalizable platform for exploring the interplay of spatial structures with genomic and epigenomic properties. Photoselective sequencing combines targeted illumination and photocaged fragment libraries to enable the spatial analysis of genomic sequence and chromatin accessibility profiles with subcellular resolution in the context of complex tissues.},
  archive      = {J_NMETH},
  author       = {Mangiameli, Sarah M. and Chen, Haiqi and Earl, Andrew S. and Dobkin, Julie A. and Lesman, Daniel and Buenrostro, Jason D. and Chen, Fei},
  doi          = {10.1038/s41592-023-01845-8},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {686-694},
  shortjournal = {Nat. Meth.},
  title        = {Photoselective sequencing: Microscopically guided genomic measurements with subcellular resolution},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). In vivo photopharmacology with a caged mu opioid receptor
agonist drives rapid changes in behavior. <em>NMETH</em>,
<em>20</em>(5), 682–685. (<a
href="https://doi.org/10.1038/s41592-023-01819-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Photoactivatable drugs and peptides can drive quantitative studies into receptor signaling with high spatiotemporal precision, yet few are compatible with behavioral studies in mammals. We developed CNV-Y-DAMGO—a caged derivative of the mu opioid receptor-selective peptide agonist DAMGO. Photoactivation in the mouse ventral tegmental area produced an opioid-dependent increase in locomotion within seconds of illumination. These results demonstrate the power of in vivo photopharmacology for dynamic studies into animal behavior. CNV-Y-DAMGO enables photopharmacological manipulations of mu opioid receptor signaling in behaving mice with high temporal resolution.},
  archive      = {J_NMETH},
  author       = {Ma, Xiang and Johnson, Desiree A. and He, Xinyi Jenny and Layden, Aryanna E. and McClain, Shannan P. and Yung, Jean C. and Rizzo, Arianna and Bonaventura, Jordi and Banghart, Matthew R.},
  doi          = {10.1038/s41592-023-01819-w},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {682-685},
  shortjournal = {Nat. Meth.},
  title        = {In vivo photopharmacology with a caged mu opioid receptor agonist drives rapid changes in behavior},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Rapid biomechanical imaging at low irradiation level via
dual line-scanning brillouin microscopy. <em>NMETH</em>, <em>20</em>(5),
677–681. (<a href="https://doi.org/10.1038/s41592-023-01816-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brillouin microscopy is a technique for mechanical characterization of biological material without contact at high three-dimensional resolution. Here, we introduce dual line-scanning Brillouin microscopy (dLSBM), which improves acquisition speed and reduces irradiation dose by more than one order of magnitude with selective illumination and single-shot analysis of hundreds of points along the incident beam axis. Using tumor spheroids, we demonstrate the ability to capture the sample response to rapid mechanical perturbations as well as the spatially resolved evolution of the mechanical properties in growing spheroids. A dual line-scanning approach increases imaging speed and reduces illumination levels in Brillouin microscopy, which is demonstrated on spheroids.},
  archive      = {J_NMETH},
  author       = {Zhang, Jitao and Nikolic, Milos and Tanner, Kandice and Scarcelli, Giuliano},
  doi          = {10.1038/s41592-023-01816-z},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {677-681},
  shortjournal = {Nat. Meth.},
  title        = {Rapid biomechanical imaging at low irradiation level via dual line-scanning brillouin microscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). CD-CODE: Crowdsourcing condensate database and encyclopedia.
<em>NMETH</em>, <em>20</em>(5), 673–676. (<a
href="https://doi.org/10.1038/s41592-023-01831-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The discovery of biomolecular condensates transformed our understanding of intracellular compartmentalization of molecules. To integrate interdisciplinary scientific knowledge about the function and composition of biomolecular condensates, we developed the crowdsourcing condensate database and encyclopedia ( cd-code.org ). CD-CODE is a community-editable platform, which includes a database of biomolecular condensates based on the literature, an encyclopedia of relevant scientific terms and a crowdsourcing web application. Our platform will accelerate the discovery and validation of biomolecular condensates, and facilitate efforts to understand their role in disease and as therapeutic targets. Crowdsourcing condensate database and encyclopedia is a community-editable platform for verified biomolecular condensates and their protein constituents. It also provides an encyclopedia for the scientific terms used in condensate biology and a crowdsourcing web application.},
  archive      = {J_NMETH},
  author       = {Rostam, Nadia and Ghosh, Soumyadeep and Chow, Chi Fung Willis and Hadarovich, Anna and Landerer, Cedric and Ghosh, Rajat and Moon, HongKee and Hersemann, Lena and Mitrea, Diana M. and Klein, Isaac A. and Hyman, Anthony A. and Toth-Petroczy, Agnes},
  doi          = {10.1038/s41592-023-01831-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {673-676},
  shortjournal = {Nat. Meth.},
  title        = {CD-CODE: Crowdsourcing condensate database and encyclopedia},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Comparison of transformations for single-cell RNA-seq data.
<em>NMETH</em>, <em>20</em>(5), 665–672. (<a
href="https://doi.org/10.1038/s41592-023-01814-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The count table, a numeric matrix of genes × cells, is the basic input data structure in the analysis of single-cell RNA-sequencing data. A common preprocessing step is to adjust the counts for variable sampling efficiency and to transform them so that the variance is similar across the dynamic range. These steps are intended to make subsequent application of generic statistical methods more palatable. Here, we describe four transformation approaches based on the delta method, model residuals, inferred latent expression state and factor analysis. We compare their strengths and weaknesses and find that the latter three have appealing theoretical properties; however, in benchmarks using simulated and real-world data, it turns out that a rather simple approach, namely, the logarithm with a pseudo-count followed by principal-component analysis, performs as well or better than the more sophisticated alternatives. This result highlights limitations of current theoretical analysis as assessed by bottom-line performance benchmarks. This paper compares different transformation approaches for analysis of single-cell RNA-sequencing data and provides recommendations for method selection.},
  archive      = {J_NMETH},
  author       = {Ahlmann-Eltze, Constantin and Huber, Wolfgang},
  doi          = {10.1038/s41592-023-01814-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {665-672},
  shortjournal = {Nat. Meth.},
  title        = {Comparison of transformations for single-cell RNA-seq data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Julia for biologists. <em>NMETH</em>, <em>20</em>(5),
655–664. (<a href="https://doi.org/10.1038/s41592-023-01832-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Major computational challenges exist in relation to the collection, curation, processing and analysis of large genomic and imaging datasets, as well as the simulation of larger and more realistic models in systems biology. Here we discuss how a relative newcomer among programming languages—Julia—is poised to meet the current and emerging demands in the computational biosciences and beyond. Speed, flexibility, a thriving package ecosystem and readability are major factors that make high-performance computing and data analysis available to an unprecedented degree. We highlight how Julia’s design is already enabling new ways of analyzing biological data and systems, and we provide a list of resources that can facilitate the transition into Julian computing. This Perspective introduces biologists interested in computational approaches to the benefits of the Julia programming language for meeting current and future computational demands.},
  archive      = {J_NMETH},
  author       = {Roesch, Elisabeth and Greener, Joe G. and MacLean, Adam L. and Nassar, Huda and Rackauckas, Christopher and Holy, Timothy E. and Stumpf, Michael P. H.},
  doi          = {10.1038/s41592-023-01832-z},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {655-664},
  shortjournal = {Nat. Meth.},
  title        = {Julia for biologists},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Photon-based communication between two neurons in
caenorhabditis elegans. <em>NMETH</em>, <em>20</em>(5), 653–654. (<a
href="https://doi.org/10.1038/s41592-023-01837-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cells exchange information with one another using secreted chemicals as data carriers. We developed an all-optogenetic synaptic transmission system that replaced a chemical neurotransmitter with emitted photons. This system enabled synthetic signaling between unconnected neurons and the generation of prosthetic synaptic circuits.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01837-8},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {653-654},
  shortjournal = {Nat. Meth.},
  title        = {Photon-based communication between two neurons in caenorhabditis elegans},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell measurement of dynamic protein secretion and
transcriptome. <em>NMETH</em>, <em>20</em>(5), 651–652. (<a
href="https://doi.org/10.1038/s41592-023-01842-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unlike cell surface proteins, secreted proteins are difficult to quantify and trace back to individual cells. We show that the capture of secreted proteins onto their source cell surfaces using an affinity matrix enables simultaneous measurement of protein secretion, cell surface proteins and transcriptomics in thousands of cells at single-cell resolution.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01842-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {651-652},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell measurement of dynamic protein secretion and transcriptome},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Extending the sensitivity, consistency and depth of
single-cell proteomics. <em>NMETH</em>, <em>20</em>(5), 649–650. (<a
href="https://doi.org/10.1038/s41592-023-01786-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Simultaneous maximization of sensitivity, data completeness and throughput in mass-spectrometry proteomics often necessitates trade-offs. To mitigate these trade-offs, we introduce a prioritization algorithm that achieves high sensitivity and data completeness while maximizing throughput. With prioritized single-cell proteomics (pSCoPE), we consistently and accurately quantify proteins and their post-translational modifications in single macrophages and link them to endocytic activity.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01786-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {649-650},
  shortjournal = {Nat. Meth.},
  title        = {Extending the sensitivity, consistency and depth of single-cell proteomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Photoselective sequencing harnesses microscopy to guide
genomic analyses. <em>NMETH</em>, <em>20</em>(5), 647–648. (<a
href="https://doi.org/10.1038/s41592-023-01849-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Photoselective sequencing is a new method for genomic and epigenomic profiling within specific regions of a biological specimen that are chosen using light microscopy. This combination of spatial and sequencing information preserves the connections between genomic and environmental properties and deepens our understanding of structure–function relationships in cells and tissues.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01849-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {647-648},
  shortjournal = {Nat. Meth.},
  title        = {Photoselective sequencing harnesses microscopy to guide genomic analyses},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Deep-brain photoactivation of an opioid peptide shapes mouse
behavior within seconds. <em>NMETH</em>, <em>20</em>(5), 645–646. (<a
href="https://doi.org/10.1038/s41592-023-01821-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Light-activated drugs and signaling molecules have therapeutic potential and are valuable experimental tools. Photoactivation of a mu opioid receptor agonist in the mouse brain rapidly triggered pain relief and locomotion, demonstrating that in vivo photopharmacology can drive dynamic studies into animal behavior.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01821-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {645-646},
  shortjournal = {Nat. Meth.},
  title        = {Deep-brain photoactivation of an opioid peptide shapes mouse behavior within seconds},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Line-scanning speeds up brillouin microscopy.
<em>NMETH</em>, <em>20</em>(5), 643–644. (<a
href="https://doi.org/10.1038/s41592-023-01843-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two new Brillouin microscopes leverage line-scanning to overcome previous limitations of the technique, enabling fast imaging, with low phototoxicity, of mechanical properties in living embryos of model organisms and tumor spheroids.},
  archive      = {J_NMETH},
  author       = {Khalilgharibi, Nargess and Paci, Giulia and Mao, Yanlan},
  doi          = {10.1038/s41592-023-01843-w},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {643-644},
  shortjournal = {Nat. Meth.},
  title        = {Line-scanning speeds up brillouin microscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Discovering CRISPR–cas off-target breaks. <em>NMETH</em>,
<em>20</em>(5), 641–642. (<a
href="https://doi.org/10.1038/s41592-023-01847-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DISCOVER-seq+ enables the efficient detection of CRISPR–Cas off-target breaks in primary cells and tissues.},
  archive      = {J_NMETH},
  author       = {Jiménez, Carlos and Crosetto, Nicola},
  doi          = {10.1038/s41592-023-01847-6},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {641-642},
  shortjournal = {Nat. Meth.},
  title        = {Discovering CRISPR–Cas off-target breaks},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). IMAGE-seq and you shall find. <em>NMETH</em>,
<em>20</em>(5), 639–640. (<a
href="https://doi.org/10.1038/s41592-023-01835-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integration of single-cell molecular profiling with cellular spatial localization has remained an elusive goal. Image-seq leverages high-resolution microscopy to spatially resolve and isolate viable bone marrow and leukemia cells for subsequent state-of-the art, single-cell transcriptomics.},
  archive      = {J_NMETH},
  author       = {Chute, John P. and Sasine, Joshua P.},
  doi          = {10.1038/s41592-023-01835-w},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {639-640},
  shortjournal = {Nat. Meth.},
  title        = {IMAGE-seq and you shall find},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023s). Tuning in to epigenetic cross-talk. <em>NMETH</em>,
<em>20</em>(5), 634–638. (<a
href="https://doi.org/10.1038/s41592-023-01870-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chemical modifications to DNA, histones and RNA make changes happen. Scientists are exploring ways to track these modifications and how they interact.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01870-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {634-638},
  shortjournal = {Nat. Meth.},
  title        = {Tuning in to epigenetic cross-talk},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). A new member of the spatial omics family. <em>NMETH</em>,
<em>20</em>(5), 633. (<a
href="https://doi.org/10.1038/s41592-023-01889-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Hua, Hui},
  doi          = {10.1038/s41592-023-01889-w},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {633},
  shortjournal = {Nat. Meth.},
  title        = {A new member of the spatial omics family},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Crosslinking mass spectrometry data bolster protein
structure prediction. <em>NMETH</em>, <em>20</em>(5), 633. (<a
href="https://doi.org/10.1038/s41592-023-01890-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-023-01890-3},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {633},
  shortjournal = {Nat. Meth.},
  title        = {Crosslinking mass spectrometry data bolster protein structure prediction},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Machine learning for MHC II specificities. <em>NMETH</em>,
<em>20</em>(5), 632. (<a
href="https://doi.org/10.1038/s41592-023-01891-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-023-01891-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {632},
  shortjournal = {Nat. Meth.},
  title        = {Machine learning for MHC II specificities},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Imaging behavior with a camera array. <em>NMETH</em>,
<em>20</em>(5), 632. (<a
href="https://doi.org/10.1038/s41592-023-01892-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-023-01892-1},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {632},
  shortjournal = {Nat. Meth.},
  title        = {Imaging behavior with a camera array},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023k). Rethinking microscope objectives. <em>NMETH</em>,
<em>20</em>(5), 631. (<a
href="https://doi.org/10.1038/s41592-023-01888-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A microscope objective inspired by the Schmidt telescope offers a large field of view, high numerical aperture, long working distance and compatibility with all homogeneous immersion media for versatile bioimaging.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-023-01888-x},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {631},
  shortjournal = {Nat. Meth.},
  title        = {Rethinking microscope objectives},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). PoCA: A software platform for point cloud data visualization
and quantification. <em>NMETH</em>, <em>20</em>(5), 629–630. (<a
href="https://doi.org/10.1038/s41592-023-01811-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Levet, Florian and Sibarita, Jean-Baptiste},
  doi          = {10.1038/s41592-023-01811-4},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {629-630},
  shortjournal = {Nat. Meth.},
  title        = {PoCA: A software platform for point cloud data visualization and quantification},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A standardized ontology for naming tRNA-derived RNAs based
on molecular origin. <em>NMETH</em>, <em>20</em>(5), 627–628. (<a
href="https://doi.org/10.1038/s41592-023-01813-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Holmes, Andrew D. and Chan, Patricia P. and Chen, Qi and Ivanov, Pavel and Drouard, Laurence and Polacek, Norbert and Kay, Mark A. and Lowe, Todd M.},
  doi          = {10.1038/s41592-023-01813-2},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {627-628},
  shortjournal = {Nat. Meth.},
  title        = {A standardized ontology for naming tRNA-derived RNAs based on molecular origin},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Languages in the lab. <em>NMETH</em>, <em>20</em>(5), 626.
(<a href="https://doi.org/10.1038/s41592-023-01854-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Members of a lab often have a varied language background. This rich language diversity leads to lab dynamics that take mindful handling.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01854-7},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {626},
  shortjournal = {Nat. Meth.},
  title        = {Languages in the lab},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Scoping out nature methods. <em>NMETH</em>, <em>20</em>(5),
625. (<a href="https://doi.org/10.1038/s41592-023-01893-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Methods welcomes manuscript submissions that describe new technology, tool and methodology developments across the spectrum of basic biology research. Here, we clarify our scope and highlight some areas of interest.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01893-0},
  journal      = {Nature Methods},
  month        = {5},
  number       = {5},
  pages        = {625},
  shortjournal = {Nat. Meth.},
  title        = {Scoping out nature methods},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Publisher correction: SODB facilitates comprehensive
exploration of spatial omics data. <em>NMETH</em>, <em>20</em>(4), 623.
(<a href="https://doi.org/10.1038/s41592-023-01844-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Yuan, Zhiyuan and Pan, Wentao and Zhao, Xuan and Zhao, Fangyuan and Xu, Zhimeng and Li, Xiu and Zhao, Yi and Zhang, Michael Q. and Yao, Jianhua},
  doi          = {10.1038/s41592-023-01844-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {623},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: SODB facilitates comprehensive exploration of spatial omics data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Miniature three-photon microscopy maximized for scattered
fluorescence collection. <em>NMETH</em>, <em>20</em>(4), 617–622. (<a
href="https://doi.org/10.1038/s41592-023-01777-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In deep-tissue multiphoton microscopy, diffusion and scattering of fluorescent photons, rather than ballistic emanation from the focal point, have been a confounding factor. Here we report on a 2.17-g miniature three-photon microscope (m3PM) with a configuration that maximizes fluorescence collection when imaging in highly scattering regimes. We demonstrate its capability by imaging calcium activity throughout the entire cortex and dorsal hippocampal CA1, up to 1.2 mm depth, at a safe laser power. It also enables the detection of sensorimotor behavior-correlated activities of layer 6 neurons in the posterior parietal cortex in freely moving mice during single-pellet reaching tasks. Thus, m3PM-empowered imaging allows the study of neural mechanisms in deep cortex and subcortical structures, like the dorsal hippocampus and dorsal striatum, in freely behaving animals. A three-photon miniature microscope with optimized light-collection efficiency facilitates imaging of neuronal activity throughout the cortex, as well as in the hippocampus, in freely moving mice.},
  archive      = {J_NMETH},
  author       = {Zhao, Chunzhu and Chen, Shiyuan and Zhang, Lifeng and Zhang, Dong and Wu, Runlong and Hu, Yanhui and Zeng, Fengqingyang and Li, Yijun and Wu, Dakun and Yu, Fei and Zhang, Yunfeng and Zhang, Jue and Chen, Liangyi and Wang, Aimin and Cheng, Heping},
  doi          = {10.1038/s41592-023-01777-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {617-622},
  shortjournal = {Nat. Meth.},
  title        = {Miniature three-photon microscopy maximized for scattered fluorescence collection},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A three-photon head-mounted microscope for imaging all
layers of visual cortex in freely moving mice. <em>NMETH</em>,
<em>20</em>(4), 610–616. (<a
href="https://doi.org/10.1038/s41592-022-01688-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in head-mounted microscopes have enabled imaging of neuronal activity using genetic tools in freely moving mice but these microscopes are restricted to recording in minimally lit arenas and imaging upper cortical layers. Here we built a 2-g, three-photon excitation-based microscope, containing a z-drive that enabled access to all cortical layers while mice freely behaved in a fully lit environment. The microscope had on-board photon detectors, robust to environmental light, and the arena lighting was timed to the end of each line-scan, enabling functional imaging of activity from cortical layer 4 and layer 6 neurons expressing jGCaMP7f in mice roaming a fully lit or dark arena. By comparing the neuronal activity measured from populations in these layers we show that activity in cortical layer 4 and layer 6 is differentially modulated by lit and dark conditions during free exploration. A lightweight three-photon miniature microscope allows imaging neuronal activity throughout the cortex in freely moving mice.},
  archive      = {J_NMETH},
  author       = {Klioutchnikov, Alexandr and Wallace, Damian J. and Sawinski, Juergen and Voit, Kay-Michael and Groemping, Yvonne and Kerr, Jason N. D.},
  doi          = {10.1038/s41592-022-01688-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {610-616},
  shortjournal = {Nat. Meth.},
  title        = {A three-photon head-mounted microscope for imaging all layers of visual cortex in freely moving mice},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mesoscale volumetric light-field (MesoLF) imaging of
neuroactivity across cortical areas at 18 hz. <em>NMETH</em>,
<em>20</em>(4), 600–609. (<a
href="https://doi.org/10.1038/s41592-023-01789-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Various implementations of mesoscopes provide optical access for calcium imaging across multi-millimeter fields of view in the mammalian brain; however, capturing the activity of the neuronal population within such fields of view near-simultaneously and in a volumetric fashion has remained challenging as approaches for imaging scattering brain tissues typically are based on sequential acquisition. Here we present a modular, mesoscale light-field (MesoLF) imaging hardware and software solution that allows recording from thousands of neurons within volumes of ⌀ 4 × 0.2 mm, located at up to 350 µm depth in the mouse cortex, at 18 volumes per second and an effective voxel rate of ~40 megavoxels per second. Using our optical design and computational approach we show recording of ~10,000 neurons across multiple cortical areas in mice using workstation-grade computing resources. Light-field microscopy is extended to mesoscale fields of view, allowing calcium imaging of thousands of neurons at a high frame rate and high spatial resolution.},
  archive      = {J_NMETH},
  author       = {Nöbauer, Tobias and Zhang, Yuanlong and Kim, Hyewon and Vaziri, Alipasha},
  doi          = {10.1038/s41592-023-01789-z},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {600-609},
  shortjournal = {Nat. Meth.},
  title        = {Mesoscale volumetric light-field (MesoLF) imaging of neuroactivity across cortical areas at 18 hz},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Femtosecond laser microdissection for isolation of
regenerating c. Elegans neurons for single-cell RNA sequencing.
<em>NMETH</em>, <em>20</em>(4), 590–599. (<a
href="https://doi.org/10.1038/s41592-023-01804-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our understanding of nerve regeneration can be enhanced by delineating its underlying molecular activities at single-neuron resolution in model organisms such as Caenorhabditis elegans. Existing cell isolation techniques cannot isolate neurons with specific regeneration phenotypes from C. elegans. We present femtosecond laser microdissection (fs-LM), a single-cell isolation method that dissects specific cells directly from living tissue by leveraging the micrometer-scale precision of fs-laser ablation. We show that fs-LM facilitates sensitive and specific gene expression profiling by single-cell RNA sequencing (scRNA-seq), while mitigating the stress-related transcriptional artifacts induced by tissue dissociation. scRNA-seq of fs-LM isolated regenerating neurons revealed transcriptional programs that are correlated with either successful or failed regeneration in wild-type and dlk-1 (0) animals, respectively. This method also allowed studying heterogeneity displayed by the same type of neuron and found gene modules with expression patterns correlated with axon regrowth rate. Our results establish fs-LM as a spatially resolved single-cell isolation method for phenotype-to-genotype mapping. Femtosecond laser microdissection enables transcriptomic analyses of single neurons based on their phenotype.},
  archive      = {J_NMETH},
  author       = {Zhao, Peisen and Mondal, Sudip and Martin, Chris and DuPlissis, Andrew and Chizari, Shahab and Ma, Ke-Yue and Maiya, Rajani and Messing, Robert O. and Jiang, Ning and Ben-Yakar, Adela},
  doi          = {10.1038/s41592-023-01804-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {590-599},
  shortjournal = {Nat. Meth.},
  title        = {Femtosecond laser microdissection for isolation of regenerating c. elegans neurons for single-cell RNA sequencing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Combining long-term circuit mapping and network
transcriptomics with SiR-n2c. <em>NMETH</em>, <em>20</em>(4), 580–589.
(<a href="https://doi.org/10.1038/s41592-023-01787-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An exciting frontier in circuit neuroscience lies at the intersection between neural network mapping and single-cell genomics. Monosynaptic rabies viruses provide a promising platform for the merger of circuit mapping methods with -omics approaches. However, three key limitations have hindered the extraction of physiologically meaningful gene expression profiles from rabies-mapped circuits: inherent viral cytotoxicity, high viral immunogenicity and virus-induced alteration of cellular transcriptional regulation. These factors alter the transcriptional and translational profiles of infected neurons and their neighboring cells. To overcome these limitations we applied a self-inactivating genomic modification to the less immunogenic rabies strain, CVS-N2c, to generate a self-inactivating CVS-N2c rabies virus (SiR-N2c). SiR-N2c not only eliminates undesired cytotoxic effects but also substantially reduces gene expression alterations in infected neurons and dampens the recruitment of innate and acquired immune responses, thus enabling open-ended interventions on neural networks and their genetic characterization using single-cell genomic approaches. A self-inactivating variant of the CVS-N2c rabies virus enables both retrograde viral tracing and transcriptomic analyses, thereby allowing a combination of circuit mapping and molecular studies.},
  archive      = {J_NMETH},
  author       = {Lee, Hassal and Ciabatti, Ernesto and González-Rueda, Ana and Williams, Elena and Nugent, Fiona and Mookerjee, Souradip and Morgese, Fabio and Tripodi, Marco},
  doi          = {10.1038/s41592-023-01787-1},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {580-589},
  shortjournal = {Nat. Meth.},
  title        = {Combining long-term circuit mapping and network transcriptomics with SiR-n2c},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). ERnet: A tool for the semantic segmentation and quantitative
analysis of endoplasmic reticulum topology. <em>NMETH</em>,
<em>20</em>(4), 569–579. (<a
href="https://doi.org/10.1038/s41592-023-01815-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to quantify structural changes of the endoplasmic reticulum (ER) is crucial for understanding the structure and function of this organelle. However, the rapid movement and complex topology of ER networks make this challenging. Here, we construct a state-of-the-art semantic segmentation method that we call ERnet for the automatic classification of sheet and tubular ER domains inside individual cells. Data are skeletonized and represented by connectivity graphs, enabling precise and efficient quantification of network connectivity. ERnet generates metrics on topology and integrity of ER structures and quantifies structural change in response to genetic or metabolic manipulation. We validate ERnet using data obtained by various ER-imaging methods from different cell types as well as ground truth images of synthetic ER structures. ERnet can be deployed in an automatic high-throughput and unbiased fashion and identifies subtle changes in ER phenotypes that may inform on disease progression and response to therapy. ERnet is a deep learning-based software tool for automatic segmentation and classification of structures in the endoplasmic reticulum. ERnet is compatible with many fluorescence imaging modalities and can uncover subtle phenotypic changes.},
  archive      = {J_NMETH},
  author       = {Lu, Meng and Christensen, Charles N. and Weber, Jana M. and Konno, Tasuku and Läubli, Nino F. and Scherer, Katharina M. and Avezov, Edward and Lio, Pietro and Lapkin, Alexei A. and Kaminski Schierle, Gabriele S. and Kaminski, Clemens F.},
  doi          = {10.1038/s41592-023-01815-0},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {569-579},
  shortjournal = {Nat. Meth.},
  title        = {ERnet: A tool for the semantic segmentation and quantitative analysis of endoplasmic reticulum topology},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cue: A deep-learning framework for structural variant
discovery and genotyping. <em>NMETH</em>, <em>20</em>(4), 559–568. (<a
href="https://doi.org/10.1038/s41592-023-01799-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural variants (SVs) are a major driver of genetic diversity and disease in the human genome and their discovery is imperative to advances in precision medicine. Existing SV callers rely on hand-engineered features and heuristics to model SVs, which cannot scale to the vast diversity of SVs nor fully harness the information available in sequencing datasets. Here we propose an extensible deep-learning framework, Cue, to call and genotype SVs that can learn complex SV abstractions directly from the data. At a high level, Cue converts alignments to images that encode SV-informative signals and uses a stacked hourglass convolutional neural network to predict the type, genotype and genomic locus of the SVs captured in each image. We show that Cue outperforms the state of the art in the detection of several classes of SVs on synthetic and real short-read data and that it can be easily extended to other sequencing platforms, while achieving competitive performance. Cue achieves versatile and performant structural variant calling and genotyping using a deep-learning approach.},
  archive      = {J_NMETH},
  author       = {Popic, Victoria and Rohlicek, Chris and Cunial, Fabio and Hajirasouliha, Iman and Meleshko, Dmitry and Garimella, Kiran and Maheshwari, Anant},
  doi          = {10.1038/s41592-023-01799-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {559-568},
  shortjournal = {Nat. Meth.},
  title        = {Cue: A deep-learning framework for structural variant discovery and genotyping},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). SVDSS: Structural variation discovery in hard-to-call
genomic regions using sample-specific strings from accurate long reads.
<em>NMETH</em>, <em>20</em>(4), 550–558. (<a
href="https://doi.org/10.1038/s41592-022-01674-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structural variants (SVs) account for a large amount of sequence variability across genomes and play an important role in human genomics and precision medicine. Despite intense efforts over the years, the discovery of SVs in individuals remains challenging due to the diploid and highly repetitive structure of the human genome, and by the presence of SVs that vastly exceed sequencing read lengths. However, the recent introduction of low-error long-read sequencing technologies such as PacBio HiFi may finally enable these barriers to be overcome. Here we present SV discovery with sample-specific strings (SVDSS)—a method for discovery of SVs from long-read sequencing technologies (for example, PacBio HiFi) that combines and effectively leverages mapping-free, mapping-based and assembly-based methodologies for overall superior SV discovery performance. Our experiments on several human samples show that SVDSS outperforms state-of-the-art mapping-based methods for discovery of insertion and deletion SVs in PacBio HiFi reads and achieves notable improvements in calling SVs in repetitive regions of the genome. SVDSS combines multiple strategies to improve detection of structural variation using low-error long-read sequencing data.},
  archive      = {J_NMETH},
  author       = {Denti, Luca and Khorsand, Parsoa and Bonizzoni, Paola and Hormozdiari, Fereydoun and Chikhi, Rayan},
  doi          = {10.1038/s41592-022-01674-1},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {550-558},
  shortjournal = {Nat. Meth.},
  title        = {SVDSS: Structural variation discovery in hard-to-call genomic regions using sample-specific strings from accurate long reads},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). An optical design enabling lightweight and large
field-of-view head-mounted microscopes. <em>NMETH</em>, <em>20</em>(4),
546–549. (<a href="https://doi.org/10.1038/s41592-023-01806-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we present a fluorescence microscope light path that enables imaging, during free behavior, of thousands of neurons in mice and hundreds of neurons in juvenile songbirds. The light path eliminates traditional illumination optics, allowing for head-mounted microscopes that have both a lower weight and a larger field of view (FOV) than previously possible. Using this light path, we designed two microscopes: one optimized for FOV (~4 mm FOV; 1.4 g), and the other optimized for weight (1.0 mm FOV; 1.0 g). Two miniature microscopes with innovative light paths are described and applied to imaging of juvenile zebra finches and mice.},
  archive      = {J_NMETH},
  author       = {Scherrer, Joseph R. and Lynch, Galen F. and Zhang, Jie J. and Fee, Michale S.},
  doi          = {10.1038/s41592-023-01806-1},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {546-549},
  shortjournal = {Nat. Meth.},
  title        = {An optical design enabling lightweight and large field-of-view head-mounted microscopes},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). MScarlet3: A brilliant and fast-maturing red fluorescent
protein. <em>NMETH</em>, <em>20</em>(4), 541–545. (<a
href="https://doi.org/10.1038/s41592-023-01809-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report the evolution of mScarlet3, a cysteine-free monomeric red fluorescent protein with fast and complete maturation, as well as record brightness, quantum yield (75\%) and fluorescence lifetime (4.0 ns). The mScarlet3 crystal structure reveals a barrel rigidified at one of its heads by a large hydrophobic patch of internal residues. mScarlet3 behaves well as a fusion tag, displays no apparent cytotoxicity and it surpasses existing red fluorescent proteins as a Förster resonance energy transfer acceptor and as a reporter in transient expression systems. Two mScarlet variants with high brightness and fast maturation times have been evolved. These variants behave favorably as fusion tags and Förster resonance energy transfer acceptors.},
  archive      = {J_NMETH},
  author       = {Gadella, Theodorus W. J. and van Weeren, Laura and Stouthamer, Jente and Hink, Mark A. and Wolters, Anouk H. G. and Giepmans, Ben N. G. and Aumonier, Sylvain and Dupuy, Jérôme and Royant, Antoine},
  doi          = {10.1038/s41592-023-01809-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {541-545},
  shortjournal = {Nat. Meth.},
  title        = {MScarlet3: A brilliant and fast-maturing red fluorescent protein},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Outbreak.info research library: A standardized, searchable
platform to discover and explore COVID-19 resources. <em>NMETH</em>,
<em>20</em>(4), 536–540. (<a
href="https://doi.org/10.1038/s41592-023-01770-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Outbreak.info Research Library is a standardized, searchable interface of coronavirus disease 2019 (COVID-19) and severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) publications, clinical trials, datasets, protocols and other resources, built with a reusable framework. We developed a rigorous schema to enforce consistency across different sources and resource types and linked related resources. Researchers can quickly search the latest research across data repositories, regardless of resource type or repository location, via a search interface, public application programming interface (API) and R package. Outbreak.info Research Library collects and integrates COVID-19 information from various repositories, enabling one-stop search for publications, clinical trials, datasets and other resources.},
  archive      = {J_NMETH},
  author       = {Tsueng, Ginger and Mullen, Julia L. and Alkuzweny, Manar and Cano, Marco and Rush, Benjamin and Haag, Emily and Lin, Jason and Welzel, Dylan J. and Zhou, Xinghua and Qian, Zhongchao and Latif, Alaa Abdel and Hufbauer, Emory and Zeller, Mark and Andersen, Kristian G. and Wu, Chunlei and Su, Andrew I. and Gangavarapu, Karthik and Hughes, Laura D.},
  doi          = {10.1038/s41592-023-01770-w},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {536-540},
  shortjournal = {Nat. Meth.},
  title        = {Outbreak.info research library: A standardized, searchable platform to discover and explore COVID-19 resources},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reliability and accuracy of single-molecule FRET studies for
characterization of structural dynamics and distances in proteins.
<em>NMETH</em>, <em>20</em>(4), 523–535. (<a
href="https://doi.org/10.1038/s41592-023-01807-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule Förster-resonance energy transfer (smFRET) experiments allow the study of biomolecular structure and dynamics in vitro and in vivo. We performed an international blind study involving 19 laboratories to assess the uncertainty of FRET experiments for proteins with respect to the measured FRET efficiency histograms, determination of distances, and the detection and quantification of structural dynamics. Using two protein systems with distinct conformational changes and dynamics, we obtained an uncertainty of the FRET efficiency ≤0.06, corresponding to an interdye distance precision of ≤2 Å and accuracy of ≤5 Å. We further discuss the limits for detecting fluctuations in this distance range and how to identify dye perturbations. Our work demonstrates the ability of smFRET experiments to simultaneously measure distances and avoid the averaging of conformational dynamics for realistic protein systems, highlighting its importance in the expanding toolbox of integrative structural biology. An international blind study confirms that smFRET measurements on dynamic proteins are highly reproducible across instruments, analysis procedures and timescales, further highlighting the promise of smFRET for dynamic structural biology.},
  archive      = {J_NMETH},
  author       = {Agam, Ganesh and Gebhardt, Christian and Popara, Milana and Mächtel, Rebecca and Folz, Julian and Ambrose, Benjamin and Chamachi, Neharika and Chung, Sang Yoon and Craggs, Timothy D. and de Boer, Marijn and Grohmann, Dina and Ha, Taekjip and Hartmann, Andreas and Hendrix, Jelle and Hirschfeld, Verena and Hübner, Christian G. and Hugel, Thorsten and Kammerer, Dominik and Kang, Hyun-Seo and Kapanidis, Achillefs N. and Krainer, Georg and Kramm, Kevin and Lemke, Edward A. and Lerner, Eitan and Margeat, Emmanuel and Martens, Kirsten and Michaelis, Jens and Mitra, Jaba and Moya Muñoz, Gabriel G. and Quast, Robert B. and Robb, Nicole C. and Sattler, Michael and Schlierf, Michael and Schneider, Jonathan and Schröder, Tim and Sefer, Anna and Tan, Piau Siong and Thurn, Johann and Tinnefeld, Philip and van Noort, John and Weiss, Shimon and Wendler, Nicolas and Zijlstra, Niels and Barth, Anders and Seidel, Claus A. M. and Lamb, Don C. and Cordes, Thorben},
  doi          = {10.1038/s41592-023-01807-0},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {523-535},
  shortjournal = {Nat. Meth.},
  title        = {Reliability and accuracy of single-molecule FRET studies for characterization of structural dynamics and distances in proteins},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Outbreak.info genomic reports: Scalable and dynamic
surveillance of SARS-CoV-2 variants and mutations. <em>NMETH</em>,
<em>20</em>(4), 512–522. (<a
href="https://doi.org/10.1038/s41592-023-01769-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In response to the emergence of SARS-CoV-2 variants of concern, the global scientific community, through unprecedented effort, has sequenced and shared over 11 million genomes through GISAID, as of May 2022. This extraordinarily high sampling rate provides a unique opportunity to track the evolution of the virus in near real-time. Here, we present outbreak.info , a platform that currently tracks over 40 million combinations of Pango lineages and individual mutations, across over 7,000 locations, to provide insights for researchers, public health officials and the general public. We describe the interpretable visualizations available in our web application, the pipelines that enable the scalable ingestion of heterogeneous sources of SARS-CoV-2 variant data and the server infrastructure that enables widespread data dissemination via a high-performance API that can be accessed using an R package. We show how outbreak.info can be used for genomic surveillance and as a hypothesis-generation tool to understand the ongoing pandemic at varying geographic and temporal scales. The outbreak.info genomic reports provides comprehensive and detailed information about SARS-CoV-2 lineages and mutations worldwide, which facilitates near real-time genomic surveillance.},
  archive      = {J_NMETH},
  author       = {Gangavarapu, Karthik and Latif, Alaa Abdel and Mullen, Julia L. and Alkuzweny, Manar and Hufbauer, Emory and Tsueng, Ginger and Haag, Emily and Zeller, Mark and Aceves, Christine M. and Zaiets, Karina and Cano, Marco and Zhou, Xinghua and Qian, Zhongchao and Sattler, Rachel and Matteson, Nathaniel L. and Levy, Joshua I. and Lee, Raphael T. C. and Freitas, Lucas and Maurer-Stroh, Sebastian and Suchard, Marc A. and Wu, Chunlei and Su, Andrew I. and Andersen, Kristian G. and Hughes, Laura D.},
  doi          = {10.1038/s41592-023-01769-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {512-522},
  shortjournal = {Nat. Meth.},
  title        = {Outbreak.info genomic reports: Scalable and dynamic surveillance of SARS-CoV-2 variants and mutations},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cryo-electron tomography on focused ion beam lamellae
transforms structural cell biology. <em>NMETH</em>, <em>20</em>(4),
499–511. (<a href="https://doi.org/10.1038/s41592-023-01783-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryogenic electron microscopy and data processing enable the determination of structures of isolated macromolecules to near-atomic resolution. However, these data do not provide structural information in the cellular environment where macromolecules perform their native functions, and vital molecular interactions can be lost during the isolation process. Cryogenic focused ion beam (FIB) fabrication generates thin lamellae of cellular samples and tissues, enabling structural studies on the near-native cellular interior and its surroundings by cryogenic electron tomography (cryo-ET). Cellular cryo-ET benefits from the technological developments in electron microscopes, detectors and data processing, and more in situ structures are being obtained and at increasingly higher resolution. In this Review, we discuss recent studies employing cryo-ET on FIB-generated lamellae and the technological developments in ultrarapid sample freezing, FIB fabrication of lamellae, tomography, data processing and correlative light and electron microscopy that have enabled these studies. Finally, we explore the future of cryo-ET in terms of both methods development and biological application. This Review describes advances in cryogenic electron tomography on focused ion beam lamellae, highlighting the key benefits of this technology for in situ structural biology and discussing important future directions.},
  archive      = {J_NMETH},
  author       = {Berger, Casper and Premaraj, Navya and Ravelli, Raimond B. G. and Knoops, Kèvin and López-Iglesias, Carmen and Peters, Peter J.},
  doi          = {10.1038/s41592-023-01783-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {499-511},
  shortjournal = {Nat. Meth.},
  title        = {Cryo-electron tomography on focused ion beam lamellae transforms structural cell biology},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A brilliant monomeric red fluorescent protein combining high
brightness and fast maturation. <em>NMETH</em>, <em>20</em>(4), 497–498.
(<a href="https://doi.org/10.1038/s41592-023-01810-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We evolved the brilliant monomeric red fluorescent protein mScarlet3 using a multiparameter screening approach. Owing to a newly engineered hydrophobic patch inside its β-barrel structure, mScarlet3 combines a high quantum yield and high fluorescence lifetime with fast and complete maturation. Consequently, mScarlet3 performs well as a fusion tag in live-cell imaging.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01810-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {497-498},
  shortjournal = {Nat. Meth.},
  title        = {A brilliant monomeric red fluorescent protein combining high brightness and fast maturation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Deep brain imaging on the move. <em>NMETH</em>,
<em>20</em>(4), 495–496. (<a
href="https://doi.org/10.1038/s41592-023-01808-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New three-photon miniature microscopes open the study of neuronal networks to those deep in the brains of behaving animals.},
  archive      = {J_NMETH},
  author       = {Lecoq, Jérôme A. and Boehringer, Roman and Grewe, Benjamin F.},
  doi          = {10.1038/s41592-023-01808-z},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {495-496},
  shortjournal = {Nat. Meth.},
  title        = {Deep brain imaging on the move},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Linking connectome with transcriptome using a
self-inactivating rabies virus. <em>NMETH</em>, <em>20</em>(4), 493–494.
(<a href="https://doi.org/10.1038/s41592-023-01803-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A self-inactivating rabies virus tracer allows long-term monitoring and transcriptomic profiling of synaptically connected cells.},
  archive      = {J_NMETH},
  author       = {Yao, Shenqin and Levi, Boaz P.},
  doi          = {10.1038/s41592-023-01803-4},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {493-494},
  shortjournal = {Nat. Meth.},
  title        = {Linking connectome with transcriptome using a self-inactivating rabies virus},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Facilitating genome structural variation analysis.
<em>NMETH</em>, <em>20</em>(4), 491–492. (<a
href="https://doi.org/10.1038/s41592-023-01767-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although structural variation is less explored than single-nucleotide variation, recent studies have shown it to be associated with several human diseases. Three fresh computational methods might help to elucidate this inadequately understood part of our genetic makeup.},
  archive      = {J_NMETH},
  author       = {Sikic, Mile},
  doi          = {10.1038/s41592-023-01767-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {491-492},
  shortjournal = {Nat. Meth.},
  title        = {Facilitating genome structural variation analysis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Tracking SARS-CoV-2 variants and resources. <em>NMETH</em>,
<em>20</em>(4), 489–490. (<a
href="https://doi.org/10.1038/s41592-023-01833-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Outbreak.info empowers real-time variant monitoring and tracing of associated publications and resources during the ‘infodemic’ of SARS-CoV-2.},
  archive      = {J_NMETH},
  author       = {Oude Munnink, Bas B. and Koopmans, Marion},
  doi          = {10.1038/s41592-023-01833-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {489-490},
  shortjournal = {Nat. Meth.},
  title        = {Tracking SARS-CoV-2 variants and resources},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023l). Researchers dig into cancer niches. <em>NMETH</em>,
<em>20</em>(4), 484–488. (<a
href="https://doi.org/10.1038/s41592-023-01834-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The tumor microenvironment is a source of cancer’s strength. Multi-omics and new spatial techniques stand to find ways to weaken it.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01834-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {484-488},
  shortjournal = {Nat. Meth.},
  title        = {Researchers dig into cancer niches},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Predicted protein structures expand the CATH database.
<em>NMETH</em>, <em>20</em>(4), 483. (<a
href="https://doi.org/10.1038/s41592-023-01857-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-023-01857-4},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {483},
  shortjournal = {Nat. Meth.},
  title        = {Predicted protein structures expand the CATH database},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Bat–virus entanglements. <em>NMETH</em>, <em>20</em>(4),
483. (<a href="https://doi.org/10.1038/s41592-023-01858-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-023-01858-3},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {483},
  shortjournal = {Nat. Meth.},
  title        = {Bat–virus entanglements},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). In situ generation of electrodes. <em>NMETH</em>,
<em>20</em>(4), 482. (<a
href="https://doi.org/10.1038/s41592-023-01855-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-023-01855-6},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {482},
  shortjournal = {Nat. Meth.},
  title        = {In situ generation of electrodes},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Combined sequencing of genomes and epigenomes.
<em>NMETH</em>, <em>20</em>(4), 482. (<a
href="https://doi.org/10.1038/s41592-023-01856-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-023-01856-5},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {482},
  shortjournal = {Nat. Meth.},
  title        = {Combined sequencing of genomes and epigenomes},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Profiling the epigenetic landscape of the antigen receptor
repertoire: The missing epi-immunogenomics data. <em>NMETH</em>,
<em>20</em>(4), 477–481. (<a
href="https://doi.org/10.1038/s41592-022-01723-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-resolution sequencing methods that capture the epigenetic landscape within the T cell receptor (TCR) gene loci are pivotal for a fundamental understanding of the epigenetic regulatory mechanisms of the TCR repertoire. In our opinion, filling the gaps in our understanding of the epigenetic mechanisms regulating the TCR repertoire will benefit the development of strategies that can modulate the TCR repertoire composition by leveraging the dynamic nature of epigenetic modifications.},
  archive      = {J_NMETH},
  author       = {Aburajab, Rayyan and Pospiech, Mateusz and Alachkar, Houda},
  doi          = {10.1038/s41592-022-01723-9},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {477-481},
  shortjournal = {Nat. Meth.},
  title        = {Profiling the epigenetic landscape of the antigen receptor repertoire: The missing epi-immunogenomics data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). MoBIE: A fiji plugin for sharing and exploration of
multi-modal cloud-hosted big image data. <em>NMETH</em>, <em>20</em>(4),
475–476. (<a href="https://doi.org/10.1038/s41592-023-01776-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Pape, Constantin and Meechan, Kimberly and Moreva, Ekaterina and Schorb, Martin and Chiaruttini, Nicolas and Zinchenko, Valentyna and Martinez Vergara, Hernando and Mizzon, Giulia and Moore, Josh and Arendt, Detlev and Kreshuk, Anna and Schwab, Yannick and Tischer, Christian},
  doi          = {10.1038/s41592-023-01776-4},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {475-476},
  shortjournal = {Nat. Meth.},
  title        = {MoBIE: A fiji plugin for sharing and exploration of multi-modal cloud-hosted big image data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). We need a plan d. <em>NMETH</em>, <em>20</em>(4), 473–474.
(<a href="https://doi.org/10.1038/s41592-023-01817-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Sever, Richard},
  doi          = {10.1038/s41592-023-01817-y},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {473-474},
  shortjournal = {Nat. Meth.},
  title        = {We need a plan d},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023n). Science while parenting. <em>NMETH</em>, <em>20</em>(4),
472. (<a href="https://doi.org/10.1038/s41592-023-01818-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It takes juggling expertise to be an academic and a parent.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01818-x},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {472},
  shortjournal = {Nat. Meth.},
  title        = {Science while parenting},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Data sharing is the future. <em>NMETH</em>, <em>20</em>(4),
471. (<a href="https://doi.org/10.1038/s41592-023-01865-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We examine data sharing practices and explore possible future directions.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01865-4},
  journal      = {Nature Methods},
  month        = {4},
  number       = {4},
  pages        = {471},
  shortjournal = {Nat. Meth.},
  title        = {Data sharing is the future},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: CrY2H-seq: A massively multiplexed assay
for deep-coverage interactome mapping. <em>NMETH</em>, <em>20</em>(3),
469. (<a href="https://doi.org/10.1038/s41592-023-01768-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Wanamaker, Shelly A. and Garza, Renee M. and MacWilliams, Andrew and Nery, Joseph R. and Bartlett, Anna and Castanon, Rosa and Goubil, Adeline and Feeney, Joseph and O’Malley, Ronan and Huang, Shao-shan C. and Zhang, Zhuzhu Z. and Galli, Mary and Ecker, Joseph R.},
  doi          = {10.1038/s41592-023-01768-4},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {469},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: CrY2H-seq: a massively multiplexed assay for deep-coverage interactome mapping},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Deep-tissue SWIR imaging using
rationally designed small red-shifted near-infrared fluorescent protein.
<em>NMETH</em>, <em>20</em>(3), 469. (<a
href="https://doi.org/10.1038/s41592-023-01798-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Oliinyk, Olena S. and Ma, Chenshuo and Pletnev, Sergei and Baloban, Mikhail and Taboada, Carlos and Sheng, Huaxin and Yao, Junjie and Verkhusha, Vladislav V.},
  doi          = {10.1038/s41592-023-01798-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {469},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Deep-tissue SWIR imaging using rationally designed small red-shifted near-infrared fluorescent protein},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Video-based pooled screening yields
improved far-red genetically encoded voltage indicators. <em>NMETH</em>,
<em>20</em>(3), 469. (<a
href="https://doi.org/10.1038/s41592-023-01805-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tian, He and Davis, Hunter C. and Wong-Campos, J. David and Park, Pojeong and Fan, Linlin Z. and Gmeiner, Benjamin and Begum, Shahinoor and Werley, Christopher A. and Borja, Gabriel B. and Upadhyay, Hansini and Shah, Himali and Jacques, Jane and Qi, Yitong and Parot, Vicente and Deisseroth, Karl and Cohen, Adam E.},
  doi          = {10.1038/s41592-023-01805-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {469},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Video-based pooled screening yields improved far-red genetically encoded voltage indicators},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Field-dependent deep learning enables high-throughput
whole-cell 3D super-resolution imaging. <em>NMETH</em>, <em>20</em>(3),
459–468. (<a href="https://doi.org/10.1038/s41592-023-01775-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule localization microscopy in a typical wide-field setup has been widely used for investigating subcellular structures with super resolution; however, field-dependent aberrations restrict the field of view (FOV) to only tens of micrometers. Here, we present a deep-learning method for precise localization of spatially variant point emitters (FD-DeepLoc) over a large FOV covering the full chip of a modern sCMOS camera. Using a graphic processing unit-based vectorial point spread function (PSF) fitter, we can fast and accurately model the spatially variant PSF of a high numerical aperture objective in the entire FOV. Combined with deformable mirror-based optimal PSF engineering, we demonstrate high-accuracy three-dimensional single-molecule localization microscopy over a volume of ~180 × 180 × 5 μm3, allowing us to image mitochondria and nuclear pore complexes in entire cells in a single imaging cycle without hardware scanning; a 100-fold increase in throughput compared to the state of the art. FD-DeepLoc uses field-dependent deep learning for precise localization of spatially variant point emitters over the full chip of a modern sCMOS camera, enabling fast and high-throughput volumetric localization microscopy.},
  archive      = {J_NMETH},
  author       = {Fu, Shuang and Shi, Wei and Luo, Tingdan and He, Yingchuan and Zhou, Lulu and Yang, Jie and Yang, Zhichao and Liu, Jiadong and Liu, Xiaotian and Guo, Zhiyong and Yang, Chengyu and Liu, Chao and Huang, Zhen-li and Ries, Jonas and Zhang, Mingjie and Xi, Peng and Jin, Dayong and Li, Yiming},
  doi          = {10.1038/s41592-023-01775-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {459-468},
  shortjournal = {Nat. Meth.},
  title        = {Field-dependent deep learning enables high-throughput whole-cell 3D super-resolution imaging},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Super-resolution SRS microscopy with a-PoD. <em>NMETH</em>,
<em>20</em>(3), 448–458. (<a
href="https://doi.org/10.1038/s41592-023-01779-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stimulated Raman scattering (SRS) offers the ability to image metabolic dynamics with high signal-to-noise ratio. However, its spatial resolution is limited by the numerical aperture of the imaging objective and the scattering cross-section of molecules. To achieve super-resolved SRS imaging, we developed a deconvolution algorithm, adaptive moment estimation (Adam) optimization-based pointillism deconvolution (A-PoD) and demonstrated a spatial resolution of lower than 59 nm on the membrane of a single lipid droplet (LD). We applied A-PoD to spatially correlated multiphoton fluorescence imaging and deuterium oxide (D2O)-probed SRS (DO-SRS) imaging from diverse samples to compare nanoscopic distributions of proteins and lipids in cells and subcellular organelles. We successfully differentiated newly synthesized lipids in LDs using A-PoD-coupled DO-SRS. The A-PoD-enhanced DO-SRS imaging method was also applied to reveal metabolic changes in brain samples from Drosophila on different diets. This new approach allows us to quantitatively measure the nanoscopic colocalization of biomolecules and metabolic dynamics in organelles. Adam optimization-based pointillism deconvolution (A-PoD) is a broadly applicable super-resolution deconvolution algorithm. A-PoD-coupled SRS microscopy reveals heterogeneous metabolic activity in subcellular structures like lipid droplets.},
  archive      = {J_NMETH},
  author       = {Jang, Hongje and Li, Yajuan and Fung, Anthony A. and Bagheri, Pegah and Hoang, Khang and Skowronska-Krawczyk, Dorota and Chen, Xiaoping and Wu, Jane Y. and Bintu, Bogdan and Shi, Lingyan},
  doi          = {10.1038/s41592-023-01779-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {448-458},
  shortjournal = {Nat. Meth.},
  title        = {Super-resolution SRS microscopy with A-PoD},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Self-supervised machine learning pushes the sensitivity
limit in label-free detection of single proteins below 10 kDa.
<em>NMETH</em>, <em>20</em>(3), 442–447. (<a
href="https://doi.org/10.1038/s41592-023-01778-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Interferometric scattering (iSCAT) microscopy is a label-free optical method capable of detecting single proteins, localizing their binding positions with nanometer precision, and measuring their mass. In the ideal case, iSCAT is limited by shot noise such that collection of more photons should extend its detection sensitivity to biomolecules of arbitrarily low mass. However, a number of technical noise sources combined with speckle-like background fluctuations have restricted the detection limit in iSCAT. Here, we show that an unsupervised machine learning isolation forest algorithm for anomaly detection pushes the mass sensitivity limit by a factor of 4 to below 10 kDa. We implement this scheme both with a user-defined feature matrix and a self-supervised FastDVDNet and validate our results with correlative fluorescence images recorded in total internal reflection mode. Our work opens the door to optical investigations of small traces of biomolecules and disease markers such as α-synuclein, chemokines and cytokines. An unsupervised machine learning approach for anomaly detection, implemented as both a user-defined feature matrix and a self-supervised deep neural network, improves the mass sensitivity of iSCAT by a factor of 4 to below 10 kDa.},
  archive      = {J_NMETH},
  author       = {Dahmardeh, Mahyar and Mirzaalian Dastjerdi, Houman and Mazal, Hisham and Köstler, Harald and Sandoghdar, Vahid},
  doi          = {10.1038/s41592-023-01778-2},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {442-447},
  shortjournal = {Nat. Meth.},
  title        = {Self-supervised machine learning pushes the sensitivity limit in label-free detection of single proteins below 10 kDa},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Engineering of bidirectional, cyanobacteriochrome-based
light-inducible dimers (BICYCL)s. <em>NMETH</em>, <em>20</em>(3),
432–441. (<a href="https://doi.org/10.1038/s41592-023-01764-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optogenetic tools for controlling protein–protein interactions (PPIs) have been developed from a small number of photosensory modules that respond to a limited selection of wavelengths. Cyanobacteriochrome (CBCR) GAF domain variants respond to an unmatched array of colors; however, their natural molecular mechanisms of action cannot easily be exploited for optogenetic control of PPIs. Here we developed bidirectional, cyanobacteriochrome-based light-inducible dimers (BICYCL)s by engineering synthetic light-dependent interactors for a red/green GAF domain. The systematic approach enables the future engineering of the broad chromatic palette of CBCRs for optogenetics use. BICYCLs are among the smallest optogenetic tools for controlling PPIs and enable either green-ON/red-OFF (BICYCL-Red) or red-ON/green-OFF (BICYCL-Green) control with up to 800-fold state selectivity. The access to green wavelengths creates new opportunities for multiplexing with existing tools. We demonstrate the utility of BICYCLs for controlling protein subcellular localization and transcriptional processes in mammalian cells and for multiplexing with existing blue-light tools. Bidirectional, cyanobacteriochrome-based light-inducible dimers (BICYCL)s enable optogenetic control of protein–protein interactions with green and red light, allowing multiplexing with existing blue light-controlled tools.},
  archive      = {J_NMETH},
  author       = {Jang, Jaewan and Tang, Kun and Youn, Jeffrey and McDonald, Sherin and Beyer, Hannes M. and Zurbriggen, Matias D. and Uppalapati, Maruti and Woolley, G. Andrew},
  doi          = {10.1038/s41592-023-01764-8},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {432-441},
  shortjournal = {Nat. Meth.},
  title        = {Engineering of bidirectional, cyanobacteriochrome-based light-inducible dimers (BICYCL)s},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). In silico tissue generation and power analysis for spatial
omics. <em>NMETH</em>, <em>20</em>(3), 424–431. (<a
href="https://doi.org/10.1038/s41592-023-01766-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As spatially resolved multiplex profiling of RNA and proteins becomes more prominent, it is increasingly important to understand the statistical power available to test specific hypotheses when designing and interpreting such experiments. Ideally, it would be possible to create an oracle that predicts sampling requirements for generalized spatial experiments. However, the unknown number of relevant spatial features and the complexity of spatial data analysis make this challenging. Here, we enumerate multiple parameters of interest that should be considered in the design of a properly powered spatial omics study. We introduce a method for tunable in silico tissue (IST) generation and use it with spatial profiling data sets to construct an exploratory computational framework for spatial power analysis. Finally, we demonstrate that our framework can be applied across diverse spatial data modalities and tissues of interest. While we demonstrate ISTs in the context of spatial power analysis, these simulated tissues have other potential use cases, including spatial method benchmarking and optimization. This paper presents a statistical framework for power analysis of spatial omics studies, facilitated by an in silico tissue-generation method.},
  archive      = {J_NMETH},
  author       = {Baker, Ethan A. G. and Schapiro, Denis and Dumitrascu, Bianca and Vickovic, Sanja and Regev, Aviv},
  doi          = {10.1038/s41592-023-01766-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {424-431},
  shortjournal = {Nat. Meth.},
  title        = {In silico tissue generation and power analysis for spatial omics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Optimizing multiplexed imaging experimental design through
tissue spatial segregation estimation. <em>NMETH</em>, <em>20</em>(3),
418–423. (<a href="https://doi.org/10.1038/s41592-022-01692-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in multiplexed imaging methods allow simultaneous detection of dozens of proteins and hundreds of RNAs, enabling deep spatial characterization of both healthy and diseased tissues. Parameters for the design of optimal multiplex imaging studies, especially those estimating how much area has to be imaged to capture all cell phenotype clusters, are lacking. Here, using a spatial transcriptomic atlas of healthy and tumor human tissues, we developed a statistical framework that determines the number and area of fields of view necessary to accurately identify all cell phenotypes that are part of a tissue. Using this strategy on imaging mass cytometry data, we identified a measurement of tissue spatial segregation that enables optimal experimental design. This strategy will enable an improved design of multiplexed imaging studies. A statistical approach for optimal design of multiplexed imaging studies has been developed. It determines experimental parameters that facilitate cell phenotype identification.},
  archive      = {J_NMETH},
  author       = {Bost, Pierre and Schulz, Daniel and Engler, Stefanie and Wasserfall, Clive and Bodenmiller, Bernd},
  doi          = {10.1038/s41592-022-01692-z},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {418-423},
  shortjournal = {Nat. Meth.},
  title        = {Optimizing multiplexed imaging experimental design through tissue spatial segregation estimation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Jasmine and iris: Population-scale structural variant
comparison and analysis. <em>NMETH</em>, <em>20</em>(3), 408–417. (<a
href="https://doi.org/10.1038/s41592-022-01753-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The availability of long reads is revolutionizing studies of structural variants (SVs). However, because SVs vary across individuals and are discovered through imprecise read technologies and methods, they can be difficult to compare. Addressing this, we present Jasmine and Iris ( https://github.com/mkirsche/Jasmine/ ), for fast and accurate SV refinement, comparison and population analysis. Using an SV proximity graph, Jasmine outperforms six widely used comparison methods, including reducing the rate of Mendelian discordance in trio datasets by more than fivefold, and reveals a set of high-confidence de novo SVs confirmed by multiple technologies. We also present a unified callset of 122,813 SVs and 82,379 indels from 31 samples of diverse ancestry sequenced with long reads. We genotype these variants in 1,317 samples from the 1000 Genomes Project and the Genotype-Tissue Expression project with DNA and RNA-sequencing data and assess their widespread impact on gene expression, including within medically relevant genes. An optimized pipeline for improved inference and analysis of structural variants (SVs) has been developed, which uses Iris for refining breakpoints and sequences, and Jasmine for comparing SV calls at population scale.},
  archive      = {J_NMETH},
  author       = {Kirsche, Melanie and Prabhu, Gautam and Sherman, Rachel and Ni, Bohan and Battle, Alexis and Aganezov, Sergey and Schatz, Michael C.},
  doi          = {10.1038/s41592-022-01753-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {408-417},
  shortjournal = {Nat. Meth.},
  title        = {Jasmine and iris: Population-scale structural variant comparison and analysis},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A modular architecture for organizing, processing and
sharing neurophysiology data. <em>NMETH</em>, <em>20</em>(3), 403–407.
(<a href="https://doi.org/10.1038/s41592-022-01742-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We describe an architecture for organizing, integrating and sharing neurophysiology data within a single laboratory or across a group of collaborators. It comprises a database linking data files to metadata and electronic laboratory notes; a module collecting data from multiple laboratories into one location; a protocol for searching and sharing data and a module for automatic analyses that populates a website. These modules can be used together or individually, by single laboratories or worldwide collaborations. A modular architecture for managing and sharing electrophysiology, behavior, colony management and other data has been built to support individual laboratories or large consortia.},
  archive      = {J_NMETH},
  author       = {Bonacchi, Niccolò and Chapuis, Gaelle A. and Churchland, Anne K. and DeWitt, Eric E. J. and Faulkner, Mayo and Harris, Kenneth D. and Huntenburg, Julia M. and Hunter, Max and Laranjeira, Inês C. and Rossant, Cyrille and Sasaki, Maho and Schartner, Michael M. and Shen, Shan and Steinmetz, Nicholas A. and Walker, Edgar Y. and West, Steven J. and Winter, Olivier and Wells, Miles J.},
  doi          = {10.1038/s41592-022-01742-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {403-407},
  shortjournal = {Nat. Meth.},
  title        = {A modular architecture for organizing, processing and sharing neurophysiology data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). EnzymeML: Seamless data flow and modeling of enzymatic data.
<em>NMETH</em>, <em>20</em>(3), 400–402. (<a
href="https://doi.org/10.1038/s41592-022-01763-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The design of biocatalytic reaction systems is highly complex owing to the dependency of the estimated kinetic parameters on the enzyme, the reaction conditions, and the modeling method. Consequently, reproducibility of enzymatic experiments and reusability of enzymatic data are challenging. We developed the XML-based markup language EnzymeML to enable storage and exchange of enzymatic data such as reaction conditions, the time course of the substrate and the product, kinetic parameters and the kinetic model, thus making enzymatic data findable, accessible, interoperable and reusable (FAIR). The feasibility and usefulness of the EnzymeML toolbox is demonstrated in six scenarios, for which data and metadata of different enzymatic reactions are collected and analyzed. EnzymeML serves as a seamless communication channel between experimental platforms, electronic lab notebooks, tools for modeling of enzyme kinetics, publication platforms and enzymatic reaction databases. EnzymeML is open and transparent, and invites the community to contribute. All documents and codes are freely available at https://enzymeml.org . EnzymeML is an XML-based markup language that enables FAIR (findable, accessible, interoperable and reusable) storage and exchange of enzymatic data such as reaction conditions, the time course of the substrate and the product, kinetic parameters and the kinetic model.},
  archive      = {J_NMETH},
  author       = {Lauterbach, Simone and Dienhart, Hannah and Range, Jan and Malzacher, Stephan and Spöring, Jan-Dirk and Rother, Dörte and Pinto, Maria Filipa and Martins, Pedro and Lagerman, Colton E. and Bommarius, Andreas S. and Høst, Amalie Vang and Woodley, John M. and Ngubane, Sandile and Kudanga, Tukayi and Bergmann, Frank T. and Rohwer, Johann M. and Iglezakis, Dorothea and Weidemann, Andreas and Wittig, Ulrike and Kettner, Carsten and Swainston, Neil and Schnell, Santiago and Pleiss, Jürgen},
  doi          = {10.1038/s41592-022-01763-1},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {400-402},
  shortjournal = {Nat. Meth.},
  title        = {EnzymeML: Seamless data flow and modeling of enzymatic data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). SODB facilitates comprehensive exploration of spatial omics
data. <em>NMETH</em>, <em>20</em>(3), 387–399. (<a
href="https://doi.org/10.1038/s41592-023-01773-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial omics technologies generate wealthy but highly complex datasets. Here we present Spatial Omics DataBase (SODB), a web-based platform providing both rich data resources and a suite of interactive data analytical modules. SODB currently maintains &amp;gt;2,400 experiments from &amp;gt;25 spatial omics technologies, which are freely accessible as a unified data format compatible with various computational packages. SODB also provides multiple interactive data analytical modules, especially a unique module, Spatial Omics View (SOView). We conduct comprehensive statistical analyses and illustrate the utility of both basic and advanced analytical modules using multiple spatial omics datasets. We demonstrate SOView utility with brain spatial transcriptomics data and recover known anatomical structures. We further delineate functional tissue domains with associated marker genes that were obscured when analyzed using previous methods. We finally show how SODB may efficiently facilitate computational method development. The SODB website is https://gene.ai.tencent.com/SpatialOmics/ . The command-line package is available at https://pysodb.readthedocs.io/en/latest/ . Spatial Omics DataBase (SODB) is a web-based platform for sharing and analyzing &amp;gt;25 types of spatial omics data. SODB promotes data reuse, offers novel visualization tools and can streamline development of omics analysis tools.},
  archive      = {J_NMETH},
  author       = {Yuan, Zhiyuan and Pan, Wentao and Zhao, Xuan and Zhao, Fangyuan and Xu, Zhimeng and Li, Xiu and Zhao, Yi and Zhang, Michael Q. and Yao, Jianhua},
  doi          = {10.1038/s41592-023-01773-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {387-399},
  shortjournal = {Nat. Meth.},
  title        = {SODB facilitates comprehensive exploration of spatial omics data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Initial recommendations for performing, benchmarking and
reporting single-cell proteomics experiments. <em>NMETH</em>,
<em>20</em>(3), 375–386. (<a
href="https://doi.org/10.1038/s41592-023-01785-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analyzing proteins from single cells by tandem mass spectrometry (MS) has recently become technically feasible. While such analysis has the potential to accurately quantify thousands of proteins across thousands of single cells, the accuracy and reproducibility of the results may be undermined by numerous factors affecting experimental design, sample preparation, data acquisition and data analysis. We expect that broadly accepted community guidelines and standardized metrics will enhance rigor, data quality and alignment between laboratories. Here we propose best practices, quality controls and data-reporting recommendations to assist in the broad adoption of reliable quantitative workflows for single-cell proteomics. Resources and discussion forums are available at https://single-cell.net/guidelines . A community of researchers working in the emerging field of single-cell proteomics propose best-practice experimental and computational recommendations and reporting guidelines for studies analyzing proteins from single cells by mass spectrometry.},
  archive      = {J_NMETH},
  author       = {Gatto, Laurent and Aebersold, Ruedi and Cox, Juergen and Demichev, Vadim and Derks, Jason and Emmott, Edward and Franks, Alexander M. and Ivanov, Alexander R. and Kelly, Ryan T. and Khoury, Luke and Leduc, Andrew and MacCoss, Michael J. and Nemes, Peter and Perlman, David H. and Petelski, Aleksandra A. and Rose, Christopher M. and Schoof, Erwin M. and Van Eyk, Jennifer and Vanderaa, Christophe and Yates, John R. and Slavov, Nikolai},
  doi          = {10.1038/s41592-023-01785-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {375-386},
  shortjournal = {Nat. Meth.},
  title        = {Initial recommendations for performing, benchmarking and reporting single-cell proteomics experiments},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell proteomics enabled by next-generation sequencing
or mass spectrometry. <em>NMETH</em>, <em>20</em>(3), 363–374. (<a
href="https://doi.org/10.1038/s41592-023-01791-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the last decade, single-cell RNA sequencing routinely performed on large numbers of single cells has greatly advanced our understanding of the underlying heterogeneity of complex biological systems. Technological advances have also enabled protein measurements, further contributing to the elucidation of cell types and states present in complex tissues. Recently, there have been independent advances in mass spectrometric techniques bringing us one step closer to characterizing single-cell proteomes. Here we discuss the challenges of detecting proteins in single cells by both mass spectrometry and sequencing-based methods. We review the state of the art for these techniques and propose that there is a space for technological advancements and complementary approaches that maximize the advantages of both classes of technologies. This Review covers the state of the art in applying mass spectrometry- or next-generation sequencing-based techniques for single-cell proteomics analysis, offering suggestions for maximizing the advantages of both approaches.},
  archive      = {J_NMETH},
  author       = {Bennett, Hayley M. and Stephenson, William and Rose, Christopher M. and Darmanis, Spyros},
  doi          = {10.1038/s41592-023-01791-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {363-374},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell proteomics enabled by next-generation sequencing or mass spectrometry},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A deconvolution algorithm to achieve super-resolution
stimulated raman scattering imaging. <em>NMETH</em>, <em>20</em>(3),
361–362. (<a href="https://doi.org/10.1038/s41592-023-01780-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stimulated Raman scattering (SRS) microscopy has the capability to simultaneously visualize the spatial distribution of different biomolecules, but it remains challenging to reach super-resolution. To achieve this goal, a deconvolution algorithm, A-PoD, was developed and combined with SRS microscopy, enabling examination of nanoscopic biomolecular distribution and subcellular metabolic activity in cells and tissues.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01780-8},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {361-362},
  shortjournal = {Nat. Meth.},
  title        = {A deconvolution algorithm to achieve super-resolution stimulated raman scattering imaging},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatial omics DataBase (SODB): Increasing accessibility to
spatial omics data. <em>NMETH</em>, <em>20</em>(3), 359–360. (<a
href="https://doi.org/10.1038/s41592-023-01772-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SODB is a large-scale platform providing rich data resources, efficient data exporting, and interactive analytical modules for spatial omics research.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01772-8},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {359-360},
  shortjournal = {Nat. Meth.},
  title        = {Spatial omics DataBase (SODB): Increasing accessibility to spatial omics data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Controlling cellular activities with light. <em>NMETH</em>,
<em>20</em>(3), 357–358. (<a
href="https://doi.org/10.1038/s41592-022-01745-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New red–green optogenetic dimerizers enable the switching on and off of gene expression with high spatiotemporal precision.},
  archive      = {J_NMETH},
  author       = {Wu, Yao-Wen},
  doi          = {10.1038/s41592-022-01745-3},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {357-358},
  shortjournal = {Nat. Meth.},
  title        = {Controlling cellular activities with light},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Designing spatial transcriptomic experiments.
<em>NMETH</em>, <em>20</em>(3), 355–356. (<a
href="https://doi.org/10.1038/s41592-023-01801-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optimal design of spatial transcriptomic experiments allows statistical evaluation of the impact of various biological and technological features on the discovery of cell phenotypes.},
  archive      = {J_NMETH},
  author       = {Righelli, Dario and Sottosanti, Andrea and Risso, Davide},
  doi          = {10.1038/s41592-023-01801-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {355-356},
  shortjournal = {Nat. Meth.},
  title        = {Designing spatial transcriptomic experiments},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023k). Proteomics sets up single-cell and single-molecule
solutions. <em>NMETH</em>, <em>20</em>(3), 350–354. (<a
href="https://doi.org/10.1038/s41592-023-01781-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New instruments are emerging in single-cell and single-molecule proteomics. This will offer labs a choice of technologies.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01781-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {350-354},
  shortjournal = {Nat. Meth.},
  title        = {Proteomics sets up single-cell and single-molecule solutions},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). SynNotch tuned for tension. <em>NMETH</em>, <em>20</em>(3),
349. (<a href="https://doi.org/10.1038/s41592-023-01826-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-023-01826-x},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {349},
  shortjournal = {Nat. Meth.},
  title        = {SynNotch tuned for tension},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). From GWAS to single-cell MPRA. <em>NMETH</em>,
<em>20</em>(3), 349. (<a
href="https://doi.org/10.1038/s41592-023-01827-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Hua, Hui},
  doi          = {10.1038/s41592-023-01827-w},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {349},
  shortjournal = {Nat. Meth.},
  title        = {From GWAS to single-cell MPRA},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Contrast enhancement for cryo-ET. <em>NMETH</em>,
<em>20</em>(3), 348. (<a
href="https://doi.org/10.1038/s41592-023-01824-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-023-01824-z},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {348},
  shortjournal = {Nat. Meth.},
  title        = {Contrast enhancement for cryo-ET},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023n). Microbial metabolic reconstruction. <em>NMETH</em>,
<em>20</em>(3), 348. (<a
href="https://doi.org/10.1038/s41592-023-01825-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-023-01825-y},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {348},
  shortjournal = {Nat. Meth.},
  title        = {Microbial metabolic reconstruction},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). Modeling symmetry breaking. <em>NMETH</em>, <em>20</em>(3),
347. (<a href="https://doi.org/10.1038/s41592-023-01823-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A bioengineering and machine learning-guided approach models human axial development and somitogenesis in vitro.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-023-01823-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {347},
  shortjournal = {Nat. Meth.},
  title        = {Modeling symmetry breaking},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Sampling the proteome by emerging single-molecule and mass
spectrometry methods. <em>NMETH</em>, <em>20</em>(3), 339–346. (<a
href="https://doi.org/10.1038/s41592-023-01802-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian cells have about 30,000 times as many protein molecules as mRNA molecules, which has major implications in the development of proteomics technologies. We discuss strategies that have been helpful for counting billions of protein molecules by liquid chromatography–tandem mass spectrometry and suggest that these strategies can benefit single-molecule methods, especially in mitigating the challenges posed by the wide dynamic range of the proteome.},
  archive      = {J_NMETH},
  author       = {MacCoss, Michael J. and Alfaro, Javier Antonio and Faivre, Danielle A. and Wu, Christine C. and Wanunu, Meni and Slavov, Nikolai},
  doi          = {10.1038/s41592-023-01802-5},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {339-346},
  shortjournal = {Nat. Meth.},
  title        = {Sampling the proteome by emerging single-molecule and mass spectrometry methods},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Not if but when nanopore protein sequencing meets
single-cell proteomics. <em>NMETH</em>, <em>20</em>(3), 336–338. (<a
href="https://doi.org/10.1038/s41592-023-01800-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The nanopore community is stepping toward a new frontier of single-molecule protein sequencing. Here, we offer our opinions on the unique potential for this emerging technology, with a focus on single-cell proteomics, and some challenges that must be overcome to realize it.},
  archive      = {J_NMETH},
  author       = {Motone, Keisuke and Nivala, Jeff},
  doi          = {10.1038/s41592-023-01800-7},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {336-338},
  shortjournal = {Nat. Meth.},
  title        = {Not if but when nanopore protein sequencing meets single-cell proteomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Subcellular omics: A new frontier pushing the limits of
resolution, complexity and throughput. <em>NMETH</em>, <em>20</em>(3),
331–335. (<a href="https://doi.org/10.1038/s41592-023-01788-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We argue that the study of single-cell subcellular organelle omics is needed to understand and regulate cell function. This requires and is being enabled by new technology development.},
  archive      = {J_NMETH},
  author       = {Eberwine, James and Kim, Junhyong and Anafi, Ron C. and Brem, Steven and Bucan, Maja and Fisher, Stephen A. and Grady, M. Sean and Herr, Amy E. and Issadore, David and Jeong, Hyejoong and Kim, HyunBum and Lee, Daeyeon and Rubakhin, Stanislav and Sul, Jai-Yoon and Sweedler, Jonathan V. and Wolf, John A. and Zaret, Kenneth S. and Zou, James},
  doi          = {10.1038/s41592-023-01788-0},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {331-335},
  shortjournal = {Nat. Meth.},
  title        = {Subcellular omics: A new frontier pushing the limits of resolution, complexity and throughput},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mass spectrometry imaging: The rise of spatially resolved
single-cell omics. <em>NMETH</em>, <em>20</em>(3), 327–330. (<a
href="https://doi.org/10.1038/s41592-023-01774-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Increasing evidence suggests that the spatial distribution of biomolecules within cells is a critical component in deciphering single-cell molecular heterogeneity. State-of-the-art single-cell MS imaging is uniquely capable of localizing biomolecules within cells, providing a dimension of information beyond what is currently available through in-depth omics investigations.},
  archive      = {J_NMETH},
  author       = {Zhang, Hua and Delafield, Daniel G. and Li, Lingjun},
  doi          = {10.1038/s41592-023-01774-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {327-330},
  shortjournal = {Nat. Meth.},
  title        = {Mass spectrometry imaging: The rise of spatially resolved single-cell omics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Tackling tumor complexity with single-cell proteomics.
<em>NMETH</em>, <em>20</em>(3), 324–326. (<a
href="https://doi.org/10.1038/s41592-023-01784-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of mass spectrometry-based single-cell proteomics technologies opens unique opportunities to understand the functional crosstalk between cells that drive tumor development.},
  archive      = {J_NMETH},
  author       = {Geiger, Tamar},
  doi          = {10.1038/s41592-023-01784-4},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {324-326},
  shortjournal = {Nat. Meth.},
  title        = {Tackling tumor complexity with single-cell proteomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Making single-cell proteomics biologically relevant.
<em>NMETH</em>, <em>20</em>(3), 320–323. (<a
href="https://doi.org/10.1038/s41592-023-01771-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent technological advances in mass spectrometry promise to add single-cell proteomics to the biologist’s toolbox. Here we discuss the current status and what is needed for this exciting technology to lead to biological insight — alone or as a complement to other omics technologies.},
  archive      = {J_NMETH},
  author       = {Rosenberger, Florian A. and Thielert, Marvin and Mann, Matthias},
  doi          = {10.1038/s41592-023-01771-9},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {320-323},
  shortjournal = {Nat. Meth.},
  title        = {Making single-cell proteomics biologically relevant},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023m). Science meets art. <em>NMETH</em>, <em>20</em>(3), 319. (<a
href="https://doi.org/10.1038/s41592-023-01782-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It’s a creative, perhaps wild and sometimes challenging encounter. But some labs make a habit of inviting artists in.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-023-01782-6},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {319},
  shortjournal = {Nat. Meth.},
  title        = {Science meets art},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell proteomics: Challenges and prospects.
<em>NMETH</em>, <em>20</em>(3), 317–318. (<a
href="https://doi.org/10.1038/s41592-023-01828-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell proteomics is a challenging goal and an area of rapid methods development. This Focus issue highlights the many paths toward high-throughput, high-sensitivity measurements.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01828-9},
  journal      = {Nature Methods},
  month        = {3},
  number       = {3},
  pages        = {317-318},
  shortjournal = {Nat. Meth.},
  title        = {Single-cell proteomics: Challenges and prospects},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Publisher correction: Nascent ribo-seq measures ribosomal
loading time and reveals kinetic impact on ribosome density.
<em>NMETH</em>, <em>20</em>(2), 316. (<a
href="https://doi.org/10.1038/s41592-023-01765-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Schott, Johanna and Reitter, Sonja and Lindner, Doris and Grosser, Jan and Bruer, Marius and Shenoy, Anjana and Geiger, Tamar and Mathes, Arthur and Dobreva, Gergana and Stoecklin, Georg},
  doi          = {10.1038/s41592-023-01765-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {316},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Nascent ribo-seq measures ribosomal loading time and reveals kinetic impact on ribosome density},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Robust single-cell matching and multimodal analysis using
shared and distinct features. <em>NMETH</em>, <em>20</em>(2), 304–315.
(<a href="https://doi.org/10.1038/s41592-022-01709-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to align individual cellular information from multiple experimental sources is fundamental for a systems-level understanding of biological processes. However, currently available tools are mainly designed for single-cell transcriptomics matching and integration, and generally rely on a large number of shared features across datasets for cell matching. This approach underperforms when applied to single-cell proteomic datasets due to the limited number of parameters simultaneously accessed and lack of shared markers across these experiments. Here, we introduce a cell-matching algorithm, matching with partial overlap (MARIO) that accounts for both shared and distinct features, while consisting of vital filtering steps to avoid suboptimal matching. MARIO accurately matches and integrates data from different single-cell proteomic and multimodal methods, including spatial techniques and has cross-species capabilities. MARIO robustly matched tissue macrophages identified from COVID-19 lung autopsies via codetection by indexing imaging to macrophages recovered from COVID-19 bronchoalveolar lavage fluid by cellular indexing of transcriptomes and epitopes by sequencing, revealing unique immune responses within the lung microenvironment of patients with COVID. MARIO is a robust tool for accurately matching multimodal single-cell datasets.},
  archive      = {J_NMETH},
  author       = {Zhu, Bokai and Chen, Shuxiao and Bai, Yunhao and Chen, Han and Liao, Guanrui and Mukherjee, Nilanjan and Vazquez, Gustavo and McIlwain, David R. and Tzankov, Alexandar and Lee, Ivan T. and Matter, Matthias S. and Goltsev, Yury and Ma, Zongming and Nolan, Garry P. and Jiang, Sizun},
  doi          = {10.1038/s41592-022-01709-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {304-315},
  shortjournal = {Nat. Meth.},
  title        = {Robust single-cell matching and multimodal analysis using shared and distinct features},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Local shape descriptors for neuron segmentation.
<em>NMETH</em>, <em>20</em>(2), 295–303. (<a
href="https://doi.org/10.1038/s41592-022-01711-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present an auxiliary learning task for the problem of neuron segmentation in electron microscopy volumes. The auxiliary task consists of the prediction of local shape descriptors (LSDs), which we combine with conventional voxel-wise direct neighbor affinities for neuron boundary detection. The shape descriptors capture local statistics about the neuron to be segmented, such as diameter, elongation, and direction. On a study comparing several existing methods across various specimen, imaging techniques, and resolutions, auxiliary learning of LSDs consistently increases segmentation accuracy of affinity-based methods over a range of metrics. Furthermore, the addition of LSDs promotes affinity-based segmentation methods to be on par with the current state of the art for neuron segmentation (flood-filling networks), while being two orders of magnitudes more efficient—a critical requirement for the processing of future petabyte-sized datasets. During segmentation of neurons in electron microscopy datasets, auxiliary learning via the prediction of local shape descriptors increases efficiency, which is important for the processing of datasets of ever-increasing size.},
  archive      = {J_NMETH},
  author       = {Sheridan, Arlo and Nguyen, Tri M. and Deb, Diptodip and Lee, Wei-Chung Allen and Saalfeld, Stephan and Turaga, Srinivas C. and Manor, Uri and Funke, Jan},
  doi          = {10.1038/s41592-022-01711-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {295-303},
  shortjournal = {Nat. Meth.},
  title        = {Local shape descriptors for neuron segmentation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Convolutional networks for supervised mining of molecular
patterns within cellular context. <em>NMETH</em>, <em>20</em>(2),
284–294. (<a href="https://doi.org/10.1038/s41592-022-01746-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron tomograms capture a wealth of structural information on the molecular constituents of cells and tissues. We present DeePiCt (deep picker in context), an open-source deep-learning framework for supervised segmentation and macromolecular complex localization in cryo-electron tomography. To train and benchmark DeePiCt on experimental data, we comprehensively annotated 20 tomograms of Schizosaccharomyces pombe for ribosomes, fatty acid synthases, membranes, nuclear pore complexes, organelles, and cytosol. By comparing DeePiCt to state-of-the-art approaches on this dataset, we show its unique ability to identify low-abundance and low-density complexes. We use DeePiCt to study compositionally distinct subpopulations of cellular ribosomes, with emphasis on their contextual association with mitochondria and the endoplasmic reticulum. Finally, applying pre-trained networks to a HeLa cell tomogram demonstrates that DeePiCt achieves high-quality predictions in unseen datasets from different biological species in a matter of minutes. The comprehensively annotated experimental data and pre-trained networks are provided for immediate use by the community. DeePiCt (deep picker in context) is a versatile, open-source deep-learning framework for supervised segmentation and localization of subcellular organelles and biomolecular complexes in cryo-electron tomography.},
  archive      = {J_NMETH},
  author       = {de Teresa-Trueba, Irene and Goetz, Sara K. and Mattausch, Alexander and Stojanovska, Frosina and Zimmerli, Christian E. and Toro-Nahuelpan, Mauricio and Cheng, Dorothy W. C. and Tollervey, Fergus and Pape, Constantin and Beck, Martin and Diz-Muñoz, Alba and Kreshuk, Anna and Mahamid, Julia and Zaugg, Judith B.},
  doi          = {10.1038/s41592-022-01746-2},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {284-294},
  shortjournal = {Nat. Meth.},
  title        = {Convolutional networks for supervised mining of molecular patterns within cellular context},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). ELI trifocal microscope: A precise system to prepare target
cryo-lamellae for in situ cryo-ET study. <em>NMETH</em>, <em>20</em>(2),
276–283. (<a href="https://doi.org/10.1038/s41592-022-01748-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron tomography (cryo-ET) has become a powerful approach to study the high-resolution structure of cellular macromolecular machines in situ. However, the current correlative cryo-fluorescence and electron microscopy lacks sufficient accuracy and efficiency to precisely prepare cryo-lamellae of target locations for subsequent cryo-ET. Here we describe a precise cryogenic fabrication system, ELI-TriScope, which sets electron (E), light (L) and ion (I) beams at the same focal point to achieve accurate and efficient preparation of a target cryo-lamella. ELI-TriScope uses a commercial dual-beam scanning electron microscope modified to incorporate a cryo-holder-based transfer system and embed an optical imaging system just underneath the vitrified specimen. Cryo-focused ion beam milling can be accurately navigated by monitoring the real-time fluorescence signal of the target molecule. Using ELI-TriScope, we prepared a batch of cryo-lamellae of HeLa cells targeting the centrosome with a success rate of ~91\% and discovered new in situ structural features of the human centrosome by cryo-ET. The ELI-TriScope advances cryo-CLEM by focusing light, electron and ion beams on cryopreserved samples for markedly improved preparation of cryo-lamellae containing target structures.},
  archive      = {J_NMETH},
  author       = {Li, Shuoguo and Wang, Ziyan and Jia, Xing and Niu, Tongxin and Zhang, Jianguo and Yin, Guoliang and Zhang, Xiaoyun and Zhu, Yun and Ji, Gang and Sun, Fei},
  doi          = {10.1038/s41592-022-01748-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {276-283},
  shortjournal = {Nat. Meth.},
  title        = {ELI trifocal microscope: A precise system to prepare target cryo-lamellae for in situ cryo-ET study},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Integrated multimodality microscope for accurate and
efficient target-guided cryo-lamellae preparation. <em>NMETH</em>,
<em>20</em>(2), 268–275. (<a
href="https://doi.org/10.1038/s41592-022-01749-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron tomography (cryo-ET) is a revolutionary technique for resolving the structure of subcellular organelles and macromolecular complexes in their cellular context. However, the application of the cryo-ET is hampered by the sample preparation step. Performing cryo-focused ion beam milling at an arbitrary position on the sample is inefficient, and the target of interest is not guaranteed to be preserved when thinning the cell from several micrometers to less than 300 nm thick. Here, we report a cryogenic correlated light, ion and electron microscopy (cryo-CLIEM) technique that is capable of preparing cryo-lamellae under the guidance of three-dimensional confocal imaging. Moreover, we demonstrate a workflow to preselect and preserve nanoscale target regions inside the finished cryo-lamellae. By successfully preparing cryo-lamellae that contain a single centriole or contact sites between subcellular organelles, we show that this approach is generally applicable, and shall help in innovating more applications of cryo-ET. Cryogenic correlated light, ion and electron microscopy (cryo-CLIEM) integrates three-dimensional confocal microscopy with focused ion beam–scanning electron microscopy for efficient preparation of lamellae containing target structures for in situ structural biology with cryo-electron tomography.},
  archive      = {J_NMETH},
  author       = {Li, Weixing and Lu, Jing and Xiao, Ke and Zhou, Maoge and Li, Yuanyuan and Zhang, Xiang and Li, Zhixun and Gu, Lusheng and Xu, Xiaojun and Guo, Qiang and Xu, Tao and Ji, Wei},
  doi          = {10.1038/s41592-022-01749-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {268-275},
  shortjournal = {Nat. Meth.},
  title        = {Integrated multimodality microscope for accurate and efficient target-guided cryo-lamellae preparation},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A framework for evaluating the performance of SMLM cluster
analysis algorithms. <em>NMETH</em>, <em>20</em>(2), 259–267. (<a
href="https://doi.org/10.1038/s41592-022-01750-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-molecule localization microscopy (SMLM) generates data in the form of coordinates of localized fluorophores. Cluster analysis is an attractive route for extracting biologically meaningful information from such data and has been widely applied. Despite a range of cluster analysis algorithms, there exists no consensus framework for the evaluation of their performance. Here, we use a systematic approach based on two metrics to score the success of clustering algorithms in simulated conditions mimicking experimental data. We demonstrate the framework using seven diverse analysis algorithms: DBSCAN, ToMATo, KDE, FOCAL, CAML, ClusterViSu and SR-Tesseler. Given that the best performer depended on the underlying distribution of localizations, we demonstrate an analysis pipeline based on statistical similarity measures that enables the selection of the most appropriate algorithm, and the optimized analysis parameters for real SMLM data. We propose that these standard simulated conditions, metrics and analysis pipeline become the basis for future analysis algorithm development and evaluation. This analysis compares the performance of seven algorithms for cluster analysis of single-molecule localization microscopy data. The results provide a framework for comparing these types of methods and point users to the best tools.},
  archive      = {J_NMETH},
  author       = {Nieves, Daniel J. and Pike, Jeremy A. and Levet, Florian and Williamson, David J. and Baragilly, Mohammed and Oloketuyi, Sandra and de Marco, Ario and Griffié, Juliette and Sage, Daniel and Cohen, Edward A. K. and Sibarita, Jean-Baptiste and Heilemann, Mike and Owen, Dylan M.},
  doi          = {10.1038/s41592-022-01750-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {259-267},
  shortjournal = {Nat. Meth.},
  title        = {A framework for evaluating the performance of SMLM cluster analysis algorithms},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). HyU: Hybrid unmixing for longitudinal in vivo imaging of low
signal-to-noise fluorescence. <em>NMETH</em>, <em>20</em>(2), 248–258.
(<a href="https://doi.org/10.1038/s41592-022-01751-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The expansion of fluorescence bioimaging toward more complex systems and geometries requires analytical tools capable of spanning widely varying timescales and length scales, cleanly separating multiple fluorescent labels and distinguishing these labels from background autofluorescence. Here we meet these challenging objectives for multispectral fluorescence microscopy, combining hyperspectral phasors and linear unmixing to create Hybrid Unmixing (HyU). HyU is efficient and robust, capable of quantitative signal separation even at low illumination levels. In dynamic imaging of developing zebrafish embryos and in mouse tissue, HyU was able to cleanly and efficiently unmix multiple fluorescent labels, even in demanding volumetric timelapse imaging settings. HyU permits high dynamic range imaging, allowing simultaneous imaging of bright exogenous labels and dim endogenous labels. This enables coincident studies of tagged components, cellular behaviors and cellular metabolism within the same specimen, providing more accurate insights into the orchestrated complexity of biological systems. Hybrid Unmixing offers enhanced imaging of multiplexed fluorescence labels, enabling longitudinal imaging of multiple fluorescent signals with reduced illumination intensities.},
  archive      = {J_NMETH},
  author       = {Chiang, Hsiao Ju and Koo, Daniel E. S. and Kitano, Masahiro and Burkitt, Sean and Unruh, Jay R. and Zavaleta, Cristina and Trinh, Le A. and Fraser, Scott E. and Cutrale, Francesco},
  doi          = {10.1038/s41592-022-01751-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {248-258},
  shortjournal = {Nat. Meth.},
  title        = {HyU: Hybrid unmixing for longitudinal in vivo imaging of low signal-to-noise fluorescence},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Haplotype-aware pantranscriptome analyses using spliced
pangenome graphs. <em>NMETH</em>, <em>20</em>(2), 239–247. (<a
href="https://doi.org/10.1038/s41592-022-01731-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pangenomics is emerging as a powerful computational paradigm in bioinformatics. This field uses population-level genome reference structures, typically consisting of a sequence graph, to mitigate reference bias and facilitate analyses that were challenging with previous reference-based methods. In this work, we extend these methods into transcriptomics to analyze sequencing data using the pantranscriptome: a population-level transcriptomic reference. Our toolchain, which consists of additions to the VG toolkit and a standalone tool, RPVG, can construct spliced pangenome graphs, map RNA sequencing data to these graphs, and perform haplotype-aware expression quantification of transcripts in a pantranscriptome. We show that this workflow improves accuracy over state-of-the-art RNA sequencing mapping methods, and that it can efficiently quantify haplotype-specific transcript expression without needing to characterize the haplotypes of a sample beforehand. A computational pipeline for haplotype-aware pantranscriptome analysis has been developed, which enables spliced pangenome graph construction, RNA sequencing data alignment, and estimation of haplotype-specific transcript expression levels.},
  archive      = {J_NMETH},
  author       = {Sibbesen, Jonas A. and Eizenga, Jordan M. and Novak, Adam M. and Sirén, Jouni and Chang, Xian and Garrison, Erik and Paten, Benedict},
  doi          = {10.1038/s41592-022-01731-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {239-247},
  shortjournal = {Nat. Meth.},
  title        = {Haplotype-aware pantranscriptome analyses using spliced pangenome graphs},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Nonnegative spatial factorization applied to spatial
genomics. <em>NMETH</em>, <em>20</em>(2), 229–238. (<a
href="https://doi.org/10.1038/s41592-022-01687-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nonnegative matrix factorization (NMF) is widely used to analyze high-dimensional count data because, in contrast to real-valued alternatives such as factor analysis, it produces an interpretable parts-based representation. However, in applications such as spatial transcriptomics, NMF fails to incorporate known structure between observations. Here, we present nonnegative spatial factorization (NSF), a spatially-aware probabilistic dimension reduction model based on transformed Gaussian processes that naturally encourages sparsity and scales to tens of thousands of observations. NSF recovers ground truth factors more accurately than real-valued alternatives such as MEFISTO in simulations, and has lower out-of-sample prediction error than probabilistic NMF on three spatial transcriptomics datasets from mouse brain and liver. Since not all patterns of gene expression have spatial correlations, we also propose a hybrid extension of NSF that combines spatial and nonspatial components, enabling quantification of spatial importance for both observations and features. A TensorFlow implementation of NSF is available from https://github.com/willtownes/nsf-paper . This paper presents nonnegative spatial factorization, a general framework for spatially aware and interpretable dimension reduction for high-dimensional spatial data, and its application to spatial transcriptomics analysis.},
  archive      = {J_NMETH},
  author       = {Townes, F. William and Engelhardt, Barbara E.},
  doi          = {10.1038/s41592-022-01687-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {229-238},
  shortjournal = {Nat. Meth.},
  title        = {Nonnegative spatial factorization applied to spatial genomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Screening cell–cell communication in spatial transcriptomics
via collective optimal transport. <em>NMETH</em>, <em>20</em>(2),
218–228. (<a href="https://doi.org/10.1038/s41592-022-01728-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial transcriptomic technologies and spatially annotated single-cell RNA sequencing datasets provide unprecedented opportunities to dissect cell–cell communication (CCC). However, incorporation of the spatial information and complex biochemical processes required in the reconstruction of CCC remains a major challenge. Here, we present COMMOT (COMMunication analysis by Optimal Transport) to infer CCC in spatial transcriptomics, which accounts for the competition between different ligand and receptor species as well as spatial distances between cells. A collective optimal transport method is developed to handle complex molecular interactions and spatial constraints. Furthermore, we introduce downstream analysis tools to infer spatial signaling directionality and genes regulated by signaling using machine learning models. We apply COMMOT to simulation data and eight spatial datasets acquired with five different technologies to show its effectiveness and robustness in identifying spatial CCC in data with varying spatial resolutions and gene coverages. Finally, COMMOT identifies new CCCs during skin morphogenesis in a case study of human epidermal development. This work presents a computational framework, COMMOT, to spatially infer cell–cell communication from transcriptomics data based on a variant of optimal transport (OT).},
  archive      = {J_NMETH},
  author       = {Cang, Zixuan and Zhao, Yanxiang and Almet, Axel A. and Stabell, Adam and Ramos, Raul and Plikus, Maksim V. and Atwood, Scott X. and Nie, Qing},
  doi          = {10.1038/s41592-022-01728-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {218-228},
  shortjournal = {Nat. Meth.},
  title        = {Screening cell–cell communication in spatial transcriptomics via collective optimal transport},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). LILAC: Enhanced actin imaging with an optogenetic lifeact.
<em>NMETH</em>, <em>20</em>(2), 214–217. (<a
href="https://doi.org/10.1038/s41592-022-01761-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lifeact is a popular peptide-based label of actin filaments in live cells. We have designed an improved Lifeact variant, LILAC, that binds to actin in light using the LOV2 protein. Light control allows the user to modulate actin labeling, enabling image analysis that leverages modulation for an enhanced view of F-actin dynamics in cells. Furthermore, the tool reduces actin perturbations and cell sickness caused by Lifeact overexpression. LILAC is a photoactivatable version of Lifeact, a tool for labeling F-actin. LILAC can help avoid cytotoxicity, which is sometimes associated with the use of Lifeact.},
  archive      = {J_NMETH},
  author       = {Kroll, Kourtney L. and French, Alexander R. and Sosnick, Tobin R. and Rock, Ronald S.},
  doi          = {10.1038/s41592-022-01761-3},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {214-217},
  shortjournal = {Nat. Meth.},
  title        = {LILAC: Enhanced actin imaging with an optogenetic lifeact},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). AlphaFill: Enriching AlphaFold models with ligands and
cofactors. <em>NMETH</em>, <em>20</em>(2), 205–213. (<a
href="https://doi.org/10.1038/s41592-022-01685-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence-based protein structure prediction approaches have had a transformative effect on biomolecular sciences. The predicted protein models in the AlphaFold protein structure database, however, all lack coordinates for small molecules, essential for molecular structure or function: hemoglobin lacks bound heme; zinc-finger motifs lack zinc ions essential for structural integrity and metalloproteases lack metal ions needed for catalysis. Ligands important for biological function are absent too; no ADP or ATP is bound to any of the ATPases or kinases. Here we present AlphaFill, an algorithm that uses sequence and structure similarity to ‘transplant’ such ‘missing’ small molecules and ions from experimentally determined structures to predicted protein models. The algorithm was successfully validated against experimental structures. A total of 12,029,789 transplants were performed on 995,411 AlphaFold models and are available together with associated validation metrics in the alphafill.eu databank, a resource to help scientists make new hypotheses and design targeted experiments. The AlphaFill algorithm transplants missing small molecules and ions from experimentally determined structures to predicted protein models in the AlphaFold protein structure database. All AlphaFill entries are available for visual inspection and download through the AlphaFill website.},
  archive      = {J_NMETH},
  author       = {Hekkelman, Maarten L. and de Vries, Ida and Joosten, Robbie P. and Perrakis, Anastassis},
  doi          = {10.1038/s41592-022-01685-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {205-213},
  shortjournal = {Nat. Meth.},
  title        = {AlphaFill: Enriching AlphaFold models with ligands and cofactors},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Guiding the choice of informatics software and tools for
lipidomics research applications. <em>NMETH</em>, <em>20</em>(2),
193–204. (<a href="https://doi.org/10.1038/s41592-022-01710-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Progress in mass spectrometry lipidomics has led to a rapid proliferation of studies across biology and biomedicine. These generate extremely large raw datasets requiring sophisticated solutions to support automated data processing. To address this, numerous software tools have been developed and tailored for specific tasks. However, for researchers, deciding which approach best suits their application relies on ad hoc testing, which is inefficient and time consuming. Here we first review the data processing pipeline, summarizing the scope of available tools. Next, to support researchers, LIPID MAPS provides an interactive online portal listing open-access tools with a graphical user interface. This guides users towards appropriate solutions within major areas in data processing, including (1) lipid-oriented databases, (2) mass spectrometry data repositories, (3) analysis of targeted lipidomics datasets, (4) lipid identification and (5) quantification from untargeted lipidomics datasets, (6) statistical analysis and visualization, and (7) data integration solutions. Detailed descriptions of functions and requirements are provided to guide customized data analysis workflows. This Perspective discusses available software tools for lipidomics data analysis and provides a web-based Lipidomics Tools Guide to help guide the choice of these tools, organized by the major tasks for lipidomics research.},
  archive      = {J_NMETH},
  author       = {Ni, Zhixu and Wölk, Michele and Jukes, Geoff and Mendivelso Espinosa, Karla and Ahrends, Robert and Aimo, Lucila and Alvarez-Jarreta, Jorge and Andrews, Simon and Andrews, Robert and Bridge, Alan and Clair, Geremy C. and Conroy, Matthew J. and Fahy, Eoin and Gaud, Caroline and Goracci, Laura and Hartler, Jürgen and Hoffmann, Nils and Kopczyinki, Dominik and Korf, Ansgar and Lopez-Clavijo, Andrea F. and Malik, Adnan and Ackerman, Jacobo Miranda and Molenaar, Martijn R. and O’Donovan, Claire and Pluskal, Tomáš and Shevchenko, Andrej and Slenter, Denise and Siuzdak, Gary and Kutmon, Martina and Tsugawa, Hiroshi and Willighagen, Egon L. and Xia, Jianguo and O’Donnell, Valerie B. and Fedorova, Maria},
  doi          = {10.1038/s41592-022-01710-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {193-204},
  shortjournal = {Nat. Meth.},
  title        = {Guiding the choice of informatics software and tools for lipidomics research applications},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Confident multimodal analysis of single cells across
platforms and species. <em>NMETH</em>, <em>20</em>(2), 191–192. (<a
href="https://doi.org/10.1038/s41592-022-01713-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Alignment of single-cell proteomics data across platforms is difficult when different data sets contain limited shared features, as is typical in single-cell assays with antibody readouts. Therefore, we developed matching with partial overlap (MARIO) to enable confident and accurate matching for multimodal data integration and cross-species analysis.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01713-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {191-192},
  shortjournal = {Nat. Meth.},
  title        = {Confident multimodal analysis of single cells across platforms and species},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Artificial intelligence gives neuron reconstruction a
performance boost. <em>NMETH</em>, <em>20</em>(2), 189–190. (<a
href="https://doi.org/10.1038/s41592-022-01712-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed an advanced deep learning approach called local shape descriptors (LSDs) to enable analysis of large electron microscopy datasets with increased efficiency. This technique will speed processing of future petabyte-sized datasets and democratize connectomics research by enabling these analyses using modest computational infrastructure available to most laboratories.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01712-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {189-190},
  shortjournal = {Nat. Meth.},
  title        = {Artificial intelligence gives neuron reconstruction a performance boost},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Parts-based decomposition of spatial genomics data finds
distinct tissue regions. <em>NMETH</em>, <em>20</em>(2), 187–188. (<a
href="https://doi.org/10.1038/s41592-022-01725-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dimension reduction is a cornerstone of exploratory data analysis; however, traditional methods fail to preserve the spatial context of spatial genomics data. In this work, we develop a nonnegative spatial factorization (NSF) model that allows interpretable, parts-based decomposition of spatial single-cell count data. NSF allows label-free annotation of regions of interest in spatial genomics data and identifies genes and cells that can be used to define those regions.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01725-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {187-188},
  shortjournal = {Nat. Meth.},
  title        = {Parts-based decomposition of spatial genomics data finds distinct tissue regions},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A mathematical method and software for spatially mapping
intercellular communication. <em>NMETH</em>, <em>20</em>(2), 185–186.
(<a href="https://doi.org/10.1038/s41592-022-01729-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Communication between cells is crucial for coordinated cellular functions in multicellular organisms. We present an optimal transport theory-based tool to infer cell–cell communication networks, spatial signaling directions and downstream targets in multicellular systems from spatial gene expression data.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01729-3},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {185-186},
  shortjournal = {Nat. Meth.},
  title        = {A mathematical method and software for spatially mapping intercellular communication},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Seeing the wood for the trees. <em>NMETH</em>,
<em>20</em>(2), 183–184. (<a
href="https://doi.org/10.1038/s41592-022-01741-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A deep learning approach called DeepPiCt facilitates segmentation and macromolecular identification in the cellular jungle of electron cryotomography data.},
  archive      = {J_NMETH},
  author       = {Smith, Olivia E. R. and Bharat, Tanmay A. M.},
  doi          = {10.1038/s41592-022-01741-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {183-184},
  shortjournal = {Nat. Meth.},
  title        = {Seeing the wood for the trees},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Actin in action. <em>NMETH</em>, <em>20</em>(2), 178–182.
(<a href="https://doi.org/10.1038/s41592-022-01762-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To gain insight into cell function, researchers are tracking the cytoskeleton and its parts, such as actin. They combine methods, find new trackers and validate them.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01762-2},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {178-182},
  shortjournal = {Nat. Meth.},
  title        = {Actin in action},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). The chinese human connectome project. <em>NMETH</em>,
<em>20</em>(2), 177. (<a
href="https://doi.org/10.1038/s41592-023-01796-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-023-01796-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {177},
  shortjournal = {Nat. Meth.},
  title        = {The chinese human connectome project},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). ENTERing the world of immune cells. <em>NMETH</em>,
<em>20</em>(2), 177. (<a
href="https://doi.org/10.1038/s41592-023-01797-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-023-01797-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {177},
  shortjournal = {Nat. Meth.},
  title        = {ENTERing the world of immune cells},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Giant assembly of DNA-origami slats. <em>NMETH</em>,
<em>20</em>(2), 176. (<a
href="https://doi.org/10.1038/s41592-023-01794-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-023-01794-2},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {176},
  shortjournal = {Nat. Meth.},
  title        = {Giant assembly of DNA-origami slats},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). Structure prediction for orphan proteins. <em>NMETH</em>,
<em>20</em>(2), 176. (<a
href="https://doi.org/10.1038/s41592-023-01795-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-023-01795-1},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {176},
  shortjournal = {Nat. Meth.},
  title        = {Structure prediction for orphan proteins},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023j). Next-generation expansion microscopy. <em>NMETH</em>,
<em>20</em>(2), 175. (<a
href="https://doi.org/10.1038/s41592-023-01793-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new twist on expansion microscopy called Magnify uses a mechanically sturdy gel to simultaneously anchor and expand diverse biological samples for super-resolution imaging.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-023-01793-3},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {175},
  shortjournal = {Nat. Meth.},
  title        = {Next-generation expansion microscopy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A cell’s growth rings. <em>NMETH</em>, <em>20</em>(2), 174.
(<a href="https://doi.org/10.1038/s41592-023-01792-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two research groups independently develop tools to record the history of cellular dynamics.},
  archive      = {J_NMETH},
  author       = {Hua, Hui},
  doi          = {10.1038/s41592-023-01792-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {174},
  shortjournal = {Nat. Meth.},
  title        = {A cell’s growth rings},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Protein structure prediction has reached the
single-structure frontier. <em>NMETH</em>, <em>20</em>(2), 170–173. (<a
href="https://doi.org/10.1038/s41592-022-01760-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dramatic advances in protein structure prediction have sparked debate as to whether the problem of predicting structure from sequence is solved or not. Here, I argue that AlphaFold2 and its peers are currently limited by the fact that they predict only a single structure, instead of a structural distribution, and that this realization is crucial for the next generation of structure prediction algorithms.},
  archive      = {J_NMETH},
  author       = {Lane, Thomas J.},
  doi          = {10.1038/s41592-022-01760-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {170-173},
  shortjournal = {Nat. Meth.},
  title        = {Protein structure prediction has reached the single-structure frontier},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). JIPipe: Visual batch processing for ImageJ. <em>NMETH</em>,
<em>20</em>(2), 168–169. (<a
href="https://doi.org/10.1038/s41592-022-01744-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Gerst, Ruman and Cseresnyés, Zoltán and Figge, Marc Thilo},
  doi          = {10.1038/s41592-022-01744-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {168-169},
  shortjournal = {Nat. Meth.},
  title        = {JIPipe: Visual batch processing for ImageJ},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023j). Navigating two-body challenges. <em>NMETH</em>,
<em>20</em>(2), 164. (<a
href="https://doi.org/10.1038/s41592-022-01758-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In academia, job-hunting couples can face difficult two-body challenges. But many universities offer options.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01758-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {164},
  shortjournal = {Nat. Meth.},
  title        = {Navigating two-body challenges},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). AlphaFold and beyond. <em>NMETH</em>, <em>20</em>(2), 163.
(<a href="https://doi.org/10.1038/s41592-023-01790-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A great deal has happened in the protein structure prediction field since Nature Methods selected this topic as our Method of the Year 2021. Here’s a quick, non-comprehensive update.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-023-01790-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {163},
  shortjournal = {Nat. Meth.},
  title        = {AlphaFold and beyond},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Neural networks primer. <em>NMETH</em>, <em>20</em>(2),
165–167. (<a href="https://doi.org/10.1038/s41592-022-01747-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature is often hidden, sometimes overcome, seldom extinguished. —Francis Bacon},
  archive      = {J_NMETH},
  author       = {Derry, Alexander and Krzywinski, Martin and Altman, Naomi},
  doi          = {10.1038/s41592-022-01747-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {2},
  pages        = {165-167},
  shortjournal = {Nat. Meth.},
  title        = {Neural networks primer},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Author correction: Eavesdropping on extracellular vesicles.
<em>NMETH</em>, <em>20</em>(1), 162. (<a
href="https://doi.org/10.1038/s41592-022-01752-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41592-022-01752-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {162},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Eavesdropping on extracellular vesicles},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: scBasset: Sequence-based modeling of
single-cell ATAC-seq using convolutional neural networks.
<em>NMETH</em>, <em>20</em>(1), 162. (<a
href="https://doi.org/10.1038/s41592-022-01754-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Yuan, Han and Kelley, David R.},
  doi          = {10.1038/s41592-022-01754-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {162},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: scBasset: sequence-based modeling of single-cell ATAC-seq using convolutional neural networks},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Bioprinted 3D outer retina barrier uncovers RPE-dependent
choroidal phenotype in advanced macular degeneration. <em>NMETH</em>,
<em>20</em>(1), 149–161. (<a
href="https://doi.org/10.1038/s41592-022-01701-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Age-related macular degeneration (AMD), a leading cause of blindness, initiates in the outer-blood-retina-barrier (oBRB) formed by the retinal pigment epithelium (RPE), Bruch’s membrane, and choriocapillaris. The mechanisms of AMD initiation and progression remain poorly understood owing to the lack of physiologically relevant human oBRB models. To this end, we engineered a native-like three-dimensional (3D) oBRB tissue (3D-oBRB) by bioprinting endothelial cells, pericytes, and fibroblasts on the basal side of a biodegradable scaffold and establishing an RPE monolayer on top. In this 3D-oBRB model, a fully-polarized RPE monolayer provides barrier resistance, induces choriocapillaris fenestration, and supports the formation of Bruch’s-membrane-like structure by inducing changes in gene expression in cells of the choroid. Complement activation in the 3D-oBRB triggers dry AMD phenotypes (including subRPE lipid-rich deposits called drusen and choriocapillaris degeneration), and HIF-α stabilization or STAT3 overactivation induce choriocapillaris neovascularization and type-I wet AMD phenotype. The 3D-oBRB provides a physiologically relevant model to studying RPE–choriocapillaris interactions under healthy and diseased conditions. A physiologically relevant 3D model of the outer-blood-retina-barrier recapitulates healthy and disease phenotypes.},
  archive      = {J_NMETH},
  author       = {Song, Min Jae and Quinn, Russ and Nguyen, Eric and Hampton, Christopher and Sharma, Ruchi and Park, Tea Soon and Koster, Céline and Voss, Ty and Tristan, Carlos and Weber, Claire and Singh, Anju and Dejene, Roba and Bose, Devika and Chen, Yu-Chi and Derr, Paige and Derr, Kristy and Michael, Sam and Barone, Francesca and Chen, Guibin and Boehm, Manfred and Maminishkis, Arvydas and Singec, Ilyas and Ferrer, Marc and Bharti, Kapil},
  doi          = {10.1038/s41592-022-01701-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {149-161},
  shortjournal = {Nat. Meth.},
  title        = {Bioprinted 3D outer retina barrier uncovers RPE-dependent choroidal phenotype in advanced macular degeneration},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Maximum-likelihood model fitting for quantitative analysis
of SMLM data. <em>NMETH</em>, <em>20</em>(1), 139–148. (<a
href="https://doi.org/10.1038/s41592-022-01676-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantitative data analysis is important for any single-molecule localization microscopy (SMLM) workflow to extract biological insights from the coordinates of the single fluorophores. However, current approaches are restricted to simple geometries or require identical structures. Here, we present LocMoFit (Localization Model Fit), an open-source framework to fit an arbitrary model to localization coordinates. It extracts meaningful parameters from individual structures and can select the most suitable model. In addition to analyzing complex, heterogeneous and dynamic structures for in situ structural biology, we demonstrate how LocMoFit can assemble multi-protein distribution maps of six nuclear pore components, calculate single-particle averages without any assumption about geometry or symmetry, and perform a time-resolved reconstruction of the highly dynamic endocytic process from static snapshots. We provide extensive simulation and visualization routines to validate the robustness of LocMoFit and tutorials to enable any user to increase the information content they can extract from their SMLM data. Localization Model Fit (LocMoFit) is an open-source tool for extracting meaningful parameters from individual structures in localization microscopy data. The framework was used for quantitative analysis of diverse biological structures.},
  archive      = {J_NMETH},
  author       = {Wu, Yu-Le and Hoess, Philipp and Tschanz, Aline and Matti, Ulf and Mund, Markus and Ries, Jonas},
  doi          = {10.1038/s41592-022-01676-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {139-148},
  shortjournal = {Nat. Meth.},
  title        = {Maximum-likelihood model fitting for quantitative analysis of SMLM data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Parallel cryo electron tomography on in situ lamellae.
<em>NMETH</em>, <em>20</em>(1), 131–138. (<a
href="https://doi.org/10.1038/s41592-022-01690-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In situ cryo electron tomography of cryo focused ion beam milled samples has emerged in recent years as a powerful technique for structural studies of macromolecular complexes in their native cellular environment. However, the possibilities for recording tomographic tilt series in a high-throughput manner are limited, in part by the lamella-shaped samples. Here we utilize a geometrical sample model and optical image shift to record tens of tilt series in parallel, thereby saving time and gaining access to sample areas conventionally used for tracking specimen movement. The parallel cryo electron tomography (PACE-tomo) method achieves a throughput faster than 5 min per tilt series and allows for the collection of sample areas that were previously unreachable, thus maximizing the amount of data from each lamella. Performance testing with ribosomes in vitro and in situ on state-of-the-art and general-purpose microscopes demonstrated the high throughput and quality of PACE-tomo. The parallel cryo electron tomography (PACE-tomo) method increases the throughput on in situ samples by parallelizing acquisition. It maximizes the usable sample area on individual lamellae without compromising data quality.},
  archive      = {J_NMETH},
  author       = {Eisenstein, Fabian and Yanagisawa, Haruaki and Kashihara, Hiroka and Kikkawa, Masahide and Tsukita, Sachiko and Danev, Radostin},
  doi          = {10.1038/s41592-022-01690-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {131-138},
  shortjournal = {Nat. Meth.},
  title        = {Parallel cryo electron tomography on in situ lamellae},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Uniform thin ice on ultraflat graphene for high-resolution
cryo-EM. <em>NMETH</em>, <em>20</em>(1), 123–130. (<a
href="https://doi.org/10.1038/s41592-022-01693-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cryo-electron microscopy (cryo-EM) visualizes the atomic structure of macromolecules that are embedded in vitrified thin ice at their close-to-native state. However, the homogeneity of ice thickness, a key factor to ensure high image quality, is poorly controlled during specimen preparation and has become one of the main challenges for high-resolution cryo-EM. Here we found that the uniformity of thin ice relies on the surface flatness of the supporting film, and developed a method to use ultraflat graphene (UFG) as the support for cryo-EM specimen preparation to achieve better control of vitreous ice thickness. We show that the uniform thin ice on UFG improves the image quality of vitrified specimens. Using such a method we successfully determined the three-dimensional structures of hemoglobin (64 kDa), α-fetoprotein (67 kDa) with no symmetry, and streptavidin (52 kDa) at a resolution of 3.5 Å, 2.6 Å and 2.2 Å, respectively. Furthermore, our results demonstrate the potential of UFG for the fields of cryo-electron tomography and structure-based drug discovery. This paper shows that the uniformity of vitreous ice thickness relies on the surface flatness of the supporting film, and presents a method to use ultraflat graphene as the support for cryo-EM specimen preparation.},
  archive      = {J_NMETH},
  author       = {Zheng, Liming and Liu, Nan and Gao, Xiaoyin and Zhu, Wenqing and Liu, Kun and Wu, Cang and Yan, Rui and Zhang, Jincan and Gao, Xin and Yao, Yating and Deng, Bing and Xu, Jie and Lu, Ye and Liu, Zhongmin and Li, Mengsen and Wei, Xiaoding and Wang, Hong-Wei and Peng, Hailin},
  doi          = {10.1038/s41592-022-01693-y},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {123-130},
  shortjournal = {Nat. Meth.},
  title        = {Uniform thin ice on ultraflat graphene for high-resolution cryo-EM},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A general method for chemogenetic control of peptide
function. <em>NMETH</em>, <em>20</em>(1), 112–122. (<a
href="https://doi.org/10.1038/s41592-022-01697-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Natural or engineered peptides serve important biological functions. A general approach to achieve chemical-dependent activation of short peptides will be valuable for spatial and temporal control of cellular processes. Here we present a pair of chemically activated protein domains (CAPs) for controlling the accessibility of both the N- and C-terminal portion of a peptide. CAPs were developed through directed evolution of an FK506-binding protein. By fusing a peptide to one or both CAPs, the function of the peptide is blocked until a small molecule displaces them from the FK506-binding protein ligand-binding site. We demonstrate that CAPs are generally applicable to a range of short peptides, including a protease cleavage site, a dimerization-inducing heptapeptide, a nuclear localization signal peptide, and an opioid peptide, with a chemical dependence up to 156-fold. We show that the CAPs system can be utilized in cell cultures and multiple organs in living animals. Chemically activated protein domains are chemogenetic tools that inhibit the activity of short peptides in the absence of a small molecule. Their versatility was demonstrated on a range of peptides and in both cells and mice.},
  archive      = {J_NMETH},
  author       = {Shen, Jiaqi and Geng, Lequn and Li, Xingyu and Emery, Catherine and Kroning, Kayla and Shingles, Gwendolyn and Lee, Kerry and Heyden, Matthias and Li, Peng and Wang, Wenjing},
  doi          = {10.1038/s41592-022-01697-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {112-122},
  shortjournal = {Nat. Meth.},
  title        = {A general method for chemogenetic control of peptide function},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Deep embedding and alignment of protein sequences.
<em>NMETH</em>, <em>20</em>(1), 104–111. (<a
href="https://doi.org/10.1038/s41592-022-01700-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protein sequence alignment is a key component of most bioinformatics pipelines to study the structures and functions of proteins. Aligning highly divergent sequences remains, however, a difficult task that current algorithms often fail to perform accurately, leaving many proteins or open reading frames poorly annotated. Here we leverage recent advances in deep learning for language modeling and differentiable programming to propose DEDAL (deep embedding and differentiable alignment), a flexible model to align protein sequences and detect homologs. DEDAL is a machine learning-based model that learns to align sequences by observing large datasets of raw protein sequences and of correct alignments. Once trained, we show that DEDAL improves by up to two- or threefold the alignment correctness over existing methods on remote homologs and better discriminates remote homologs from evolutionarily unrelated sequences, paving the way to improvements on many downstream tasks relying on sequence alignment in structural and functional genomics. DEDAL is a deep learning-based protein sequence alignment method that improves the quality of predicted alignment for remote homologs and better discriminates remote homologs from evolutionarily unrelated sequences.},
  archive      = {J_NMETH},
  author       = {Llinares-López, Felipe and Berthet, Quentin and Blondel, Mathieu and Teboul, Olivier and Vert, Jean-Philippe},
  doi          = {10.1038/s41592-022-01700-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {104-111},
  shortjournal = {Nat. Meth.},
  title        = {Deep embedding and alignment of protein sequences},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Virus-assisted directed evolution of enhanced suppressor
tRNAs in mammalian cells. <em>NMETH</em>, <em>20</em>(1), 95–103. (<a
href="https://doi.org/10.1038/s41592-022-01706-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Site-specific incorporation of unnatural amino acids (Uaas) in living cells relies on engineered aminoacyl-transfer RNA synthetase–tRNA pairs borrowed from a distant domain of life. Such heterologous suppressor tRNAs often have poor intrinsic activity, presumably due to suboptimal interaction with a non-native translation system. This limitation can be addressed in Escherichia coli using directed evolution. However, no suitable selection system is currently available to do the same in mammalian cells. Here we report virus-assisted directed evolution of tRNAs (VADER) in mammalian cells, which uses a double-sieve selection scheme to facilitate single-step enrichment of active yet orthogonal tRNA mutants from naive libraries. Using VADER we developed improved mutants of Methanosarcina mazei pyrrolysyl-tRNA, as well as a bacterial tyrosyl-tRNA. We also show that the higher activity of the most efficient mutant pyrrolysyl-tRNA is specific for mammalian cells, alluding to an improved interaction with the unique mammalian translation apparatus. Virus-assisted directed evolution of tRNAs (VADER) enables selection of improved suppressor tRNAs in mammalian cells. Improved pyrrolysyl- and tyrosyl-tRNA variants enhance unnatural amino acid incorporation in mammalian systems.},
  archive      = {J_NMETH},
  author       = {Jewel, Delilah and Kelemen, Rachel E. and Huang, Rachel L. and Zhu, Zeyu and Sundaresh, Bharathi and Cao, Xiaofu and Malley, Kaitlin and Huang, Zeyi and Pasha, Muhammad and Anthony, Jon and van Opijnen, Tim and Chatterjee, Abhishek},
  doi          = {10.1038/s41592-022-01706-w},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {95-103},
  shortjournal = {Nat. Meth.},
  title        = {Virus-assisted directed evolution of enhanced suppressor tRNAs in mammalian cells},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Efficient combinatorial targeting of RNA transcripts in
single cells with cas13 RNA perturb-seq. <em>NMETH</em>, <em>20</em>(1),
86–94. (<a href="https://doi.org/10.1038/s41592-022-01705-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pooled CRISPR screens coupled with single-cell RNA-sequencing have enabled systematic interrogation of gene function and regulatory networks. Here, we introduce Cas13 RNA Perturb-seq (CaRPool-seq), which leverages the RNA-targeting CRISPR–Cas13d system and enables efficient combinatorial perturbations alongside multimodal single-cell profiling. CaRPool-seq encodes multiple perturbations on a cleavable CRISPR array that is associated with a detectable barcode sequence, allowing for the simultaneous targeting of multiple genes. We compared CaRPool-seq to existing Cas9-based methods, highlighting its unique strength to efficiently profile combinatorially perturbed cells. Finally, we apply CaRPool-seq to perform multiplexed combinatorial perturbations of myeloid differentiation regulators in an acute myeloid leukemia (AML) model system and identify extensive interactions between different chromatin regulators that can enhance or suppress AML differentiation phenotypes. This work introduces CaRPool-seq that utilizes the RNA-targeting CRISPR–Cas13d system to perform combinatorial perturbations in single-cell screens.},
  archive      = {J_NMETH},
  author       = {Wessels, Hans-Hermann and Méndez-Mancilla, Alejandro and Hao, Yuhan and Papalexi, Efthymia and Mauck, William M. and Lu, Lu and Morris, John A. and Mimitou, Eleni P. and Smibert, Peter and Sanjana, Neville E. and Satija, Rahul},
  doi          = {10.1038/s41592-022-01705-x},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {86-94},
  shortjournal = {Nat. Meth.},
  title        = {Efficient combinatorial targeting of RNA transcripts in single cells with cas13 RNA perturb-seq},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Nano3P-seq: Transcriptome-wide analysis of gene expression
and tail dynamics using end-capture nanopore cDNA sequencing.
<em>NMETH</em>, <em>20</em>(1), 75–85. (<a
href="https://doi.org/10.1038/s41592-022-01714-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA polyadenylation plays a central role in RNA maturation, fate, and stability. In response to developmental cues, polyA tail lengths can vary, affecting the translation efficiency and stability of mRNAs. Here we develop Nanopore 3′ end-capture sequencing (Nano3P-seq), a method that relies on nanopore cDNA sequencing to simultaneously quantify RNA abundance, tail composition, and tail length dynamics at per-read resolution. By employing a template-switching-based sequencing protocol, Nano3P-seq can sequence RNA molecule from its 3′ end, regardless of its polyadenylation status, without the need for PCR amplification or ligation of RNA adapters. We demonstrate that Nano3P-seq provides quantitative estimates of RNA abundance and tail lengths, and captures a wide diversity of RNA biotypes. We find that, in addition to mRNA and long non-coding RNA, polyA tails can be identified in 16S mitochondrial ribosomal RNA in both mouse and zebrafish models. Moreover, we show that mRNA tail lengths are dynamically regulated during vertebrate embryogenesis at an isoform-specific level, correlating with mRNA decay. Finally, we demonstrate the ability of Nano3P-seq in capturing non-A bases within polyA tails of various lengths, and reveal their distribution during vertebrate embryogenesis. Overall, Nano3P-seq is a simple and robust method for accurately estimating transcript levels, tail lengths, and tail composition heterogeneity in individual reads, with minimal library preparation biases, both in the coding and non-coding transcriptome. Nano3P-seq presents a nanopore-based sequencing tool to profile polyA-tailed and non-polyA-tailed transcripts, as well as capture polyA tail length and composition.},
  archive      = {J_NMETH},
  author       = {Begik, Oguzhan and Diensthuber, Gregor and Liu, Huanle and Delgado-Tejedor, Anna and Kontur, Cassandra and Niazi, Adnan Muhammad and Valen, Eivind and Giraldez, Antonio J. and Beaudoin, Jean-Denis and Mattick, John S. and Novoa, Eva Maria},
  doi          = {10.1038/s41592-022-01714-w},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {75-85},
  shortjournal = {Nat. Meth.},
  title        = {Nano3P-seq: Transcriptome-wide analysis of gene expression and tail dynamics using end-capture nanopore cDNA sequencing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Deep-tissue SWIR imaging using rationally designed small
red-shifted near-infrared fluorescent protein. <em>NMETH</em>,
<em>20</em>(1), 70–74. (<a
href="https://doi.org/10.1038/s41592-022-01683-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Applying rational design, we developed 17 kDa cyanobacteriochrome-based near-infrared (NIR-I) fluorescent protein, miRFP718nano. miRFP718nano efficiently binds endogenous biliverdin chromophore and brightly fluoresces in mammalian cells and tissues. miRFP718nano has maximal emission at 718 nm and an emission tail in the short-wave infrared (SWIR) region, allowing deep-penetrating off-peak fluorescence imaging in vivo. The miRFP718nano structure reveals the molecular basis of its red shift. We demonstrate superiority of miRFP718nano-enabled SWIR imaging over NIR-I imaging of microbes in the mouse digestive tract, mammalian cells injected into the mouse mammary gland and NF-kB activity in a mouse model of liver inflammation. miRFP718nano is a rationally designed small near-infrared fluorescent protein with an emission tail that extends into the short-wave infrared range for improved multiplexed and deep-tissue imaging applications.},
  archive      = {J_NMETH},
  author       = {Oliinyk, Olena S. and Ma, Chenshuo and Pletnev, Sergei and Baloban, Mikhail and Taboada, Carlos and Sheng, Huaxin and Yao, Junjie and Verkhusha, Vladislav V.},
  doi          = {10.1038/s41592-022-01683-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {70-74},
  shortjournal = {Nat. Meth.},
  title        = {Deep-tissue SWIR imaging using rationally designed small red-shifted near-infrared fluorescent protein},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multiplexed transcriptome discovery of RNA-binding protein
binding sites by antibody-barcode eCLIP. <em>NMETH</em>, <em>20</em>(1),
65–69. (<a href="https://doi.org/10.1038/s41592-022-01708-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ultraviolet crosslinking and immunoprecipitation (CLIP) methodologies enable the identification of RNA binding sites of RNA-binding proteins (RBPs). Despite improvements in the library preparation of RNA fragments, the enhanced CLIP (eCLIP) protocol requires 4 days of hands-on time and lacks the ability to process several RBPs in parallel. We present a new method termed antibody-barcode eCLIP that utilizes DNA-barcoded antibodies and proximity ligation of the DNA oligonucleotides to RBP-protected RNA fragments to interrogate several RBPs simultaneously. We observe performance comparable with that of eCLIP with the advantage of dramatically increased scaling while maintaining the same material requirement of a single eCLIP experiment. Antibody-barcode eCLIP (ABC) uses proximity ligation to couple DNA-barcoded antibodies to RNA-binding protein (RBP)-protected RNA fragments for multiplexed eCLIP. ABC can be used to interrogate several RBPs in a single tube with results on par with eCLIP.},
  archive      = {J_NMETH},
  author       = {Lorenz, Daniel A. and Her, Hsuan-Lin and Shen, Kylie A. and Rothamel, Katie and Hutt, Kasey R. and Nojadera, Allan C. and Bruns, Stephanie C. and Manakov, Sergei A. and Yee, Brian A. and Chapman, Karen B. and Yeo, Gene W.},
  doi          = {10.1038/s41592-022-01708-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {65-69},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexed transcriptome discovery of RNA-binding protein binding sites by antibody-barcode eCLIP},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multifaceted atlases of the human brain in its infancy.
<em>NMETH</em>, <em>20</em>(1), 55–64. (<a
href="https://doi.org/10.1038/s41592-022-01703-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brain atlases are spatial references for integrating, processing, and analyzing brain features gathered from different individuals, sources, and scales. Here we introduce a collection of joint surface–volume atlases that chart postnatal development of the human brain in a spatiotemporally dense manner from two weeks to two years of age. Our month-specific atlases chart normative patterns and capture key traits of early brain development and are therefore conducive to identifying aberrations from normal developmental trajectories. These atlases will enhance our understanding of early structural and functional development by facilitating the mapping of diverse features of the infant brain to a common reference frame for precise multifaceted quantification of cortical and subcortical changes. This Resource presents surface and volume atlases of human brain development during early infancy, at monthly intervals.},
  archive      = {J_NMETH},
  author       = {Ahmad, Sahar and Wu, Ye and Wu, Zhengwang and Thung, Kim-Han and Liu, Siyuan and Lin, Weili and Li, Gang and Wang, Li and Yap, Pew-Thian},
  doi          = {10.1038/s41592-022-01703-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {55-64},
  shortjournal = {Nat. Meth.},
  title        = {Multifaceted atlases of the human brain in its infancy},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reply to: Assessment of 3D MINFLUX data for quantitative
structural biology in cells. <em>NMETH</em>, <em>20</em>(1), 52–54. (<a
href="https://doi.org/10.1038/s41592-022-01695-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Gwosch, Klaus C. and Balzarotti, Francisco and Pape, Jasmin K. and Hoess, Philipp and Ellenberg, Jan and Ries, Jonas and Matti, Ulf and Schmidt, Roman and Sahl, Steffen J. and Hell, Stefan W.},
  doi          = {10.1038/s41592-022-01695-w},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {52-54},
  shortjournal = {Nat. Meth.},
  title        = {Reply to: Assessment of 3D MINFLUX data for quantitative structural biology in cells},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Assessment of 3D MINFLUX data for quantitative structural
biology in cells. <em>NMETH</em>, <em>20</em>(1), 48–51. (<a
href="https://doi.org/10.1038/s41592-022-01694-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Prakash, Kirti and Curd, Alistair P.},
  doi          = {10.1038/s41592-022-01694-x},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {48-51},
  shortjournal = {Nat. Meth.},
  title        = {Assessment of 3D MINFLUX data for quantitative structural biology in cells},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). 3D vascularized eye tissue models age-related macular
degeneration. <em>NMETH</em>, <em>20</em>(1), 46–47. (<a
href="https://doi.org/10.1038/s41592-022-01702-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We engineered a 3D outer-blood-retina-barrier (3D-oBRB) with a fully polarized retinal pigment epithelium (RPE) monolayer on top of a Bruch’s membrane and a fenestrated choriocapillaris network. This 3D-oBRB tissue faithfully recapitulates RPE– choriocapillaris interactions, dry age-related macular degeneration (AMD) phenotypes (including sub-RPE drusen deposits and choriocapillaris degeneration) and the wet AMD phenotype of choriocapillaris neovascularization.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01702-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {46-47},
  shortjournal = {Nat. Meth.},
  title        = {3D vascularized eye tissue models age-related macular degeneration},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). LocMoFit quantifies cellular structures in super-resolution
data. <em>NMETH</em>, <em>20</em>(1), 44–45. (<a
href="https://doi.org/10.1038/s41592-022-01696-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Localization Model Fit (LocMoFit) is a tool that enables fitting of super-resolution microscopy data to an arbitrary geometric model. The fit extracts quantitative parameters of individual cellular structures, which can be used to investigate dynamic and heterogenous protein assemblies and to create average protein distribution maps.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01696-9},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {44-45},
  shortjournal = {Nat. Meth.},
  title        = {LocMoFit quantifies cellular structures in super-resolution data},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fast cryo-electron tomography data acquisition to determine
molecular structures in situ. <em>NMETH</em>, <em>20</em>(1), 42–43. (<a
href="https://doi.org/10.1038/s41592-022-01691-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To accelerate data acquisition for in situ cryo-electron tomography, we created a method that takes into consideration sample geometry for the robust prediction of sample movement while the microscope stage is tilted. This approach enabled the parallel collection of tens to hundreds of tilt series.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01691-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {42-43},
  shortjournal = {Nat. Meth.},
  title        = {Fast cryo-electron tomography data acquisition to determine molecular structures in situ},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Deep-learning language models help to improve protein
sequence alignment. <em>NMETH</em>, <em>20</em>(1), 40–41. (<a
href="https://doi.org/10.1038/s41592-022-01707-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We trained DEDAL, an algorithm based on deep-learning language models, to generate pairwise alignments of protein sequences taking into account the sequence-specific context of amino acid substitutions or gaps. DEDAL improved the alignment correctness on remote homologs by up to threefold and the discrimination of remote homologs from evolutionarily unrelated sequences.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01707-9},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {40-41},
  shortjournal = {Nat. Meth.},
  title        = {Deep-learning language models help to improve protein sequence alignment},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Digital brain atlases reveal postnatal development to 2
years of age in human infants. <em>NMETH</em>, <em>20</em>(1), 38–39.
(<a href="https://doi.org/10.1038/s41592-022-01704-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {During the first two years of postnatal development, the human brain undergoes rapid, pronounced changes in size, shape and content. Using high-resolution MRI, we constructed month-to-month atlases of infants 2 weeks to 2 years old, capturing key spatiotemporal traits of early brain development in terms of cortical geometries and tissue properties.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01704-y},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {38-39},
  shortjournal = {Nat. Meth.},
  title        = {Digital brain atlases reveal postnatal development to 2 years of age in human infants},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). CRISPR inspirations from nature. <em>NMETH</em>,
<em>20</em>(1), 37. (<a
href="https://doi.org/10.1038/s41592-022-01756-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Hua, Hui},
  doi          = {10.1038/s41592-022-01756-0},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {37},
  shortjournal = {Nat. Meth.},
  title        = {CRISPR inspirations from nature},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). Nanobodies made versatile. <em>NMETH</em>, <em>20</em>(1),
37. (<a href="https://doi.org/10.1038/s41592-022-01757-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-022-01757-z},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {37},
  shortjournal = {Nat. Meth.},
  title        = {Nanobodies made versatile},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). Tumors and their microbiome. <em>NMETH</em>,
<em>20</em>(1), 36. (<a
href="https://doi.org/10.1038/s41592-022-01755-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatially resolved and single-cell transcriptomics provide insights into tumor heterogeneity and tumor–microbiome interactions.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-022-01755-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {36},
  shortjournal = {Nat. Meth.},
  title        = {Tumors and their microbiome},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Fluorescence in structural biology. <em>NMETH</em>,
<em>20</em>(1), 35. (<a
href="https://doi.org/10.1038/s41592-022-01738-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in fluorescence microscopy and spectroscopy show their promise for applications that complement in situ structural biology methods like cryoelectron tomography.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-022-01738-2},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {35},
  shortjournal = {Nat. Meth.},
  title        = {Fluorescence in structural biology},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Immunomechanics. <em>NMETH</em>, <em>20</em>(1), 35. (<a
href="https://doi.org/10.1038/s41592-022-01739-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mechanoimmunology methods help dissect the mechanical forces within the immune system.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-022-01739-1},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {35},
  shortjournal = {Nat. Meth.},
  title        = {Immunomechanics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Genomics data integration. <em>NMETH</em>, <em>20</em>(1),
34. (<a href="https://doi.org/10.1038/s41592-022-01736-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data integration methods weave genomics data into holistic pictures of biological systems.},
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-022-01736-4},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {34},
  shortjournal = {Nat. Meth.},
  title        = {Genomics data integration},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Reproducibility in MRI. <em>NMETH</em>, <em>20</em>(1), 34.
(<a href="https://doi.org/10.1038/s41592-022-01737-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Strategies to overcome poor reproducibility in MRI studies are needed.},
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-022-01737-3},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {34},
  shortjournal = {Nat. Meth.},
  title        = {Reproducibility in MRI},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023m). Loading large DNA cargoes. <em>NMETH</em>, <em>20</em>(1),
33. (<a href="https://doi.org/10.1038/s41592-022-01734-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR systems can be leveraged to direct recombinases to integrate gene-sized DNA sequences.},
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-022-01734-6},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {33},
  shortjournal = {Nat. Meth.},
  title        = {Loading large DNA cargoes},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Annotating unknown metabolites. <em>NMETH</em>,
<em>20</em>(1), 33. (<a
href="https://doi.org/10.1038/s41592-022-01735-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational approaches are pushing the limits of unknown metabolite annotation.},
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-022-01735-5},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {33},
  shortjournal = {Nat. Meth.},
  title        = {Annotating unknown metabolites},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Exploring the embryo. <em>NMETH</em>, <em>20</em>(1), 32.
(<a href="https://doi.org/10.1038/s41592-022-01732-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In vitro models and single-cell atlases help uncover the mysteries of human embryogenesis.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-022-01732-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {32},
  shortjournal = {Nat. Meth.},
  title        = {Exploring the embryo},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). A light switch for targeted genomics. <em>NMETH</em>,
<em>20</em>(1), 32. (<a
href="https://doi.org/10.1038/s41592-022-01733-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The combination of microscopy, targeted illumination and single-cell sequencing is driving applications from direct evolution to spatial omics.},
  archive      = {J_NMETH},
  author       = {Strack, Rita},
  doi          = {10.1038/s41592-022-01733-7},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {32},
  shortjournal = {Nat. Meth.},
  title        = {A light switch for targeted genomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Long-read metagenomics paves the way toward a complete
microbial tree of life. <em>NMETH</em>, <em>20</em>(1), 30–31. (<a
href="https://doi.org/10.1038/s41592-022-01726-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read sequencing has made closed microbial genomes a routine task, and the dramatic increase in quality and quantity now paves the way to a complete microbial tree of life through genome-centric metagenomics.},
  archive      = {J_NMETH},
  author       = {Albertsen, Mads},
  doi          = {10.1038/s41592-022-01726-6},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {30-31},
  shortjournal = {Nat. Meth.},
  title        = {Long-read metagenomics paves the way toward a complete microbial tree of life},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Long-read sequencing in the era of epigenomics and
epitranscriptomics. <em>NMETH</em>, <em>20</em>(1), 25–29. (<a
href="https://doi.org/10.1038/s41592-022-01724-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As long-read sequencing technologies continue to advance, the possibility of obtaining maps of DNA and RNA modifications at single-molecule resolution has become a reality. Here we highlight the opportunities and challenges posed by the use of long-read sequencing technologies to study epigenetic and epitranscriptomic marks and how this will affect the way in which we approach the study of health and disease states.},
  archive      = {J_NMETH},
  author       = {Lucas, Morghan C. and Novoa, Eva Maria},
  doi          = {10.1038/s41592-022-01724-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {25-29},
  shortjournal = {Nat. Meth.},
  title        = {Long-read sequencing in the era of epigenomics and epitranscriptomics},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The variables on RNA molecules: Concert or cacophony?
Answers in long-read sequencing. <em>NMETH</em>, <em>20</em>(1), 20–24.
(<a href="https://doi.org/10.1038/s41592-022-01715-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read sequencing has become a widely employed technology that enables a comprehensive view of RNA transcripts. Here, we discuss the importance of long-read sequencing in interpreting the variables along RNA molecules, such as polyadenylation sites, transcription start sites, splice sites and other RNA modifications. In addition, we highlight the history of short-read and long-read technologies and their advantages and disadvantages, as well as future directions in the field.},
  archive      = {J_NMETH},
  author       = {Foord, Careen and Hsu, Justine and Jarroux, Julien and Hu, Wen and Belchikov, Natan and Pollard, Shaun and He, Yi and Joglekar, Anoushka and Tilgner, Hagen U.},
  doi          = {10.1038/s41592-022-01715-9},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {20-24},
  shortjournal = {Nat. Meth.},
  title        = {The variables on RNA molecules: Concert or cacophony? answers in long-read sequencing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Comprehensive variant discovery in the era of complete human
reference genomes. <em>NMETH</em>, <em>20</em>(1), 17–19. (<a
href="https://doi.org/10.1038/s41592-022-01740-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advances in long-read sequencing technologies have broadened our understanding of genetic variation in the human population, uncovered new complex structural variants and offered an opportunity to elucidate new variant associations with disease.},
  archive      = {J_NMETH},
  author       = {Cechova, Monika and Miga, Karen H.},
  doi          = {10.1038/s41592-022-01740-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {17-19},
  shortjournal = {Nat. Meth.},
  title        = {Comprehensive variant discovery in the era of complete human reference genomes},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Approaching complete genomes, transcriptomes and epi-omes
with accurate long-read sequencing. <em>NMETH</em>, <em>20</em>(1),
12–16. (<a href="https://doi.org/10.1038/s41592-022-01716-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The year 2022 will be remembered as the turning point for accurate long-read sequencing, which now establishes the gold standard for speed and accuracy at competitive costs. We discuss the key bioinformatics techniques needed to power long reads across application areas and close with our vision for long-read sequencing over the coming years.},
  archive      = {J_NMETH},
  author       = {Kovaka, Sam and Ou, Shujun and Jenike, Katharine M. and Schatz, Michael C.},
  doi          = {10.1038/s41592-022-01716-8},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {12-16},
  shortjournal = {Nat. Meth.},
  title        = {Approaching complete genomes, transcriptomes and epi-omes with accurate long-read sequencing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). Method of the year: Long-read sequencing. <em>NMETH</em>,
<em>20</em>(1), 6–11. (<a
href="https://doi.org/10.1038/s41592-022-01730-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To large-scale projects and individual labs, long-read sequencing has delivered new vistas and long wish lists for this technology’s future.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01730-w},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {6-11},
  shortjournal = {Nat. Meth.},
  title        = {Method of the year: Long-read sequencing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The planarian flatworm schmidtea mediterranea.
<em>NMETH</em>, <em>20</em>(1), 3–5. (<a
href="https://doi.org/10.1038/s41592-022-01727-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Planaria are a group of worms within the phylum Platyhelminthes (flatworms). Many species, including Schmidtea mediterranea, have the ability to regenerate their body from small pieces of tissue and are easy to keep in the laboratory, which makes them a prime model system for studying whole-body regeneration.},
  archive      = {J_NMETH},
  author       = {Drees, Leonard and Rink, Jochen C.},
  doi          = {10.1038/s41592-022-01727-5},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {3-5},
  shortjournal = {Nat. Meth.},
  title        = {The planarian flatworm schmidtea mediterranea},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). How to stay social. <em>NMETH</em>, <em>20</em>(1), 2. (<a
href="https://doi.org/10.1038/s41592-022-01722-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many scientists are active on social media, especially Twitter. The social media world is changing, but these researchers want to stay socially connected.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-022-01722-w},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {2},
  shortjournal = {Nat. Meth.},
  title        = {How to stay social},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Method of the year 2022: Long-read sequencing.
<em>NMETH</em>, <em>20</em>(1), 1. (<a
href="https://doi.org/10.1038/s41592-022-01759-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Long-read sequencing powers a more complete reading of genomic information.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-022-01759-x},
  journal      = {Nature Methods},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Meth.},
  title        = {Method of the year 2022: Long-read sequencing},
  volume       = {20},
  year         = {2023},
}
</textarea>
</details></li>
</ul>

</body>
</html>
