<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NG_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="ng---271">NG - 271</h2>
<ul>
<li><details>
<summary>
(2022). Multiparametric and accurate functional analysis of genetic
sequence variants using CRISPR-select. <em>NG</em>, <em>54</em>(12),
1983–1993. (<a
href="https://doi.org/10.1038/s41588-022-01224-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determining the functional role of thousands of genetic sequence variants (mutations) associated with genetic diseases is a major challenge. Here we present clustered regularly interspaced short palindromic repeat (CRISPR)-SelectTIME, CRISPR-SelectSPACE and CRISPR-SelectSTATE, a set of flexible knock-in assays that introduce a genetic variant in a cell population and track its absolute frequencies relative to an internal, neutral control mutation as a function of time, space or a cell state measurable by flow cytometry. Phenotypically, CRISPR-Select can thereby determine, for example, pathogenicity, drug responsiveness/resistance or in vivo tumor promotion by a specific variant. Mechanistically, CRISPR-Select can dissect how the variant elicits the phenotype by causally linking the variant to motility/invasiveness or any cell state or biochemical process with a flow cytometry marker. The method is applicable to organoids, nontransformed or cancer cell lines. It is accurate, quantitative, fast and simple and works in single-well or 96-well higher throughput format. CRISPR-Select provides a versatile functional variant assay for research, diagnostics and drug development for genetic disorders. CRISPR-Select is a quantitative assay for the functional impact of genetic variants, including pathogenicity, drug response, oncogenicity, cell motility and other cell states.},
  archive      = {J_NG},
  author       = {Niu, Yiyuan and Ferreira Azevedo, Catarina A. and Li, Xin and Kamali, Elahe and Haagen Nielsen, Ole and Storgaard Sørensen, Claus and Frödin, Morten},
  doi          = {10.1038/s41588-022-01224-7},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1983-1993},
  shortjournal = {Nat. Genet.},
  title        = {Multiparametric and accurate functional analysis of genetic sequence variants using CRISPR-select},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Antagonistic control of seed dormancy in rice by two bHLH
transcription factors. <em>NG</em>, <em>54</em>(12), 1972–1982. (<a
href="https://doi.org/10.1038/s41588-022-01240-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Preharvest sprouting (PHS) due to lack of seed dormancy seriously threatens crop production worldwide. As a complex quantitative trait, breeding of crop cultivars with suitable seed dormancy is hindered by limited useful regulatory genes. Here by repeatable phenotypic characterization of fixed recombinant individuals, we report a quantitative genetic locus, Seed Dormancy 6 (SD6), from aus-type rice, encoding a basic helix-loop-helix (bHLH) transcription factor, which underlies the natural variation of seed dormancy. SD6 and another bHLH factor inducer of C-repeat binding factors expression 2 (ICE2) function antagonistically in controlling seed dormancy by directly regulating the ABA catabolism gene ABA8OX3, and indirectly regulating the ABA biosynthesis gene NCED2 via OsbHLH048, in a temperature-dependent manner. The weak-dormancy allele of SD6 is common in cultivated rice but undergoes negative selection in wild rice. Notably, by genome editing SD6 and its wheat homologs, we demonstrated that SD6 is a useful breeding target for alleviating PHS in cereals under field conditions. SD6, which encodes a bHLH transcription factor, along with another bHLH factor ICE2, antagonistically controls rice seed dormancy by regulating ABA catabolism and biosynthesis genes in a temperature-dependent manner.},
  archive      = {J_NG},
  author       = {Xu, Fan and Tang, Jiuyou and Wang, Shengxing and Cheng, Xi and Wang, Hongru and Ou, Shujun and Gao, Shaopei and Li, Boshu and Qian, Yangwen and Gao, Caixia and Chu, Chengcai},
  doi          = {10.1038/s41588-022-01240-7},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1972-1982},
  shortjournal = {Nat. Genet.},
  title        = {Antagonistic control of seed dormancy in rice by two bHLH transcription factors},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genomic innovation and regulatory rewiring during evolution
of the cotton genus gossypium. <em>NG</em>, <em>54</em>(12), 1959–1971.
(<a href="https://doi.org/10.1038/s41588-022-01237-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phenotypic diversity and evolutionary innovation ultimately trace to variation in genomic sequence and rewiring of regulatory networks. Here, we constructed a pan-genome of the Gossypium genus using ten representative diploid genomes. We document the genomic evolutionary history and the impact of lineage-specific transposon amplification on differential genome composition. The pan-3D genome reveals evolutionary connections between transposon-driven genome size variation and both higher-order chromatin structure reorganization and the rewiring of chromatin interactome. We linked changes in chromatin structures to phenotypic differences in cotton fiber and identified regulatory variations that decode the genetic basis of fiber length, the latter enabled by sequencing 1,005 transcriptomes during fiber development. We showcase how pan-genomic, pan-3D genomic and genetic regulatory data serve as a resource for delineating the evolutionary basis of spinnable cotton fiber. Our work provides insights into the evolution of genome organization and regulation and will inform cotton improvement by enabling regulome-based approaches. Pan-genome and pan-3D genome analyses of the Gossypium genus reveal evolutionary relationships among transposon-driven genome expansion and chromatin topology innovation and regulatory variations for cotton fiber development.},
  archive      = {J_NG},
  author       = {Wang, Maojun and Li, Jianying and Qi, Zhengyang and Long, Yuexuan and Pei, Liuling and Huang, Xianhui and Grover, Corrinne E. and Du, Xiongming and Xia, Chunjiao and Wang, Pengcheng and Liu, Zhenping and You, Jiaqi and Tian, Xuehan and Ma, Yizan and Wang, Ruipeng and Chen, Xinyuan and He, Xin and Fang, David D. and Sun, Yuqiang and Tu, Lili and Jin, Shuangxia and Zhu, Longfu and Wendel, Jonathan F. and Zhang, Xianlong},
  doi          = {10.1038/s41588-022-01237-2},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1959-1971},
  shortjournal = {Nat. Genet.},
  title        = {Genomic innovation and regulatory rewiring during evolution of the cotton genus gossypium},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Prrx1 marks stem cells for bone, white adipose tissue and
dermis in adult mice. <em>NG</em>, <em>54</em>(12), 1946–1958. (<a
href="https://doi.org/10.1038/s41588-022-01227-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Specialized connective tissues, including bone and adipose tissues, control various physiological activities, including mineral and energy homeostasis. However, the identity of stem cells maintaining these tissues throughout adulthood remains elusive. By conducting genetic lineage tracing and cell depletion experiments in newly generated knock-in Cre/CreERT2 lines, we show here that rare Prrx1-expressing cells act as stem cells for bone, white adipose tissue and dermis in adult mice, which are indispensable for the homeostasis and repair of these tissues. Single-cell profiling reveals the cycling and multipotent nature of Prrx1-expressing cells and the stemness of these cells is further validated by transplantation assays. Moreover, we identify the cell surface markers for Prrx1-expressing stem cells and show that the activities of these stem cells are regulated by Wnt signaling. These findings expand our knowledge of connective tissue homeostasis/regeneration and may help improve stem-cell-based therapies. Prrx1-expressing can act as stem cells for bone, white adipose tissue and dermis in adult mice. These cells have a key role in tissue homeostasis and repair and are regulated by Wnt signaling.},
  archive      = {J_NG},
  author       = {Liu, Huijuan and Li, Ping and Zhang, Shaoyang and Xiang, Jinnan and Yang, Ruichen and Liu, Jiajia and Shafiquzzaman, Md and Biswas, Soma and Wei, Zhanying and Zhang, Zhenlin and Zhou, Xin and Yin, Feng and Xie, Yangli and Goff, Stephen P. and Chen, Lin and Li, Baojie},
  doi          = {10.1038/s41588-022-01227-4},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1946-1958},
  shortjournal = {Nat. Genet.},
  title        = {Prrx1 marks stem cells for bone, white adipose tissue and dermis in adult mice},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Retrotransposon activation during drosophila metamorphosis
conditions adult antiviral responses. <em>NG</em>, <em>54</em>(12),
1933–1945. (<a
href="https://doi.org/10.1038/s41588-022-01214-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Retrotransposons are one type of mobile genetic element that abundantly reside in the genomes of nearly all animals. Their uncontrolled activation is linked to sterility, cancer and other pathologies, thereby being largely considered detrimental. Here we report that, within a specific time window of development, retrotransposon activation can license the host’s immune system for future antiviral responses. We found that the mdg4 (also known as Gypsy) retrotransposon selectively becomes active during metamorphosis at the Drosophila pupal stage. At this stage, mdg4 activation educates the host’s innate immune system by inducing the systemic antiviral function of the nuclear factor-κB protein Relish in a dSTING-dependent manner. Consequently, adult flies with mdg4, Relish or dSTING silenced at the pupal stage are unable to clear exogenous viruses and succumb to viral infection. Altogether, our data reveal that hosts can establish a protective antiviral response that endows a long-term benefit in pathogen warfare due to the developmental activation of mobile genetic elements. Expression of mdg4 retrotransposons during Drosophila metamorphosis activates the antiviral NF-κB factor Relish. Silencing of mdg4 or Relish at the pupal stage leads to an inability to clear exogenous viruses in adulthood.},
  archive      = {J_NG},
  author       = {Wang, Lu and Tracy, Lauren and Su, Weijia and Yang, Fu and Feng, Yu and Silverman, Neal and Zhang, Z. Z. Zhao},
  doi          = {10.1038/s41588-022-01214-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1933-1945},
  shortjournal = {Nat. Genet.},
  title        = {Retrotransposon activation during drosophila metamorphosis conditions adult antiviral responses},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Enhancer–promoter interactions and transcription are largely
maintained upon acute loss of CTCF, cohesin, WAPL or YY1. <em>NG</em>,
<em>54</em>(12), 1919–1932. (<a
href="https://doi.org/10.1038/s41588-022-01223-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It remains unclear why acute depletion of CTCF (CCCTC-binding factor) and cohesin only marginally affects expression of most genes despite substantially perturbing three-dimensional (3D) genome folding at the level of domains and structural loops. To address this conundrum, we used high-resolution Micro-C and nascent transcript profiling in mouse embryonic stem cells. We find that enhancer–promoter (E–P) interactions are largely insensitive to acute (3-h) depletion of CTCF, cohesin or WAPL. YY1 has been proposed as a structural regulator of E–P loops, but acute YY1 depletion also had minimal effects on E–P loops, transcription and 3D genome folding. Strikingly, live-cell, single-molecule imaging revealed that cohesin depletion reduced transcription factor (TF) binding to chromatin. Thus, although CTCF, cohesin, WAPL or YY1 is not required for the short-term maintenance of most E–P interactions and gene expression, our results suggest that cohesin may facilitate TFs to search for and bind their targets more efficiently. Micro-C and nascent transcript profiling in mouse embryonic stem cells shows that enhancer–promoter interactions are robust to acute depletion of CTCF, cohesin, WAPL or YY1. Live-cell, single-molecule imaging shows that cohesin depletion reduces transcription factor binding to chromatin.},
  archive      = {J_NG},
  author       = {Hsieh, Tsung-Han S. and Cattoglio, Claudia and Slobodyanyuk, Elena and Hansen, Anders S. and Darzacq, Xavier and Tjian, Robert},
  doi          = {10.1038/s41588-022-01223-8},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1919-1932},
  shortjournal = {Nat. Genet.},
  title        = {Enhancer–promoter interactions and transcription are largely maintained upon acute loss of CTCF, cohesin, WAPL or YY1},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cohesin and CTCF control the dynamics of chromosome folding.
<em>NG</em>, <em>54</em>(12), 1907–1918. (<a
href="https://doi.org/10.1038/s41588-022-01232-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In mammals, interactions between sequences within topologically associating domains enable control of gene expression across large genomic distances. Yet it is unknown how frequently such contacts occur, how long they last and how they depend on the dynamics of chromosome folding and loop extrusion activity of cohesin. By imaging chromosomal locations at high spatial and temporal resolution in living cells, we show that interactions within topologically associating domains are transient and occur frequently during the course of a cell cycle. Interactions become more frequent and longer in the presence of convergent CTCF sites, resulting in suppression of variability in chromosome folding across time. Supported by physical models of chromosome dynamics, our data suggest that CTCF-anchored loops last around 10 min. Our results show that long-range transcriptional regulation might rely on transient physical proximity, and that cohesin and CTCF stabilize highly dynamic chromosome structures, facilitating selected subsets of chromosomal interactions. Live-cell imaging shows that interactions within topologically associating domains are transient and frequent throughout the cell cycle. Convergent CTCF sites regulate the frequency and duration of interactions, which last a few minutes on average.},
  archive      = {J_NG},
  author       = {Mach, Pia and Kos, Pavel I. and Zhan, Yinxiu and Cramard, Julie and Gaudin, Simon and Tünnermann, Jana and Marchi, Edoardo and Eglinger, Jan and Zuin, Jessica and Kryzhanovska, Mariya and Smallwood, Sebastien and Gelman, Laurent and Roth, Gregory and Nora, Elphège P. and Tiana, Guido and Giorgetti, Luca},
  doi          = {10.1038/s41588-022-01232-7},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1907-1918},
  shortjournal = {Nat. Genet.},
  title        = {Cohesin and CTCF control the dynamics of chromosome folding},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Evidence that direct inhibition of transcription factor
binding is the prevailing mode of gene and repeat repression by DNA
methylation. <em>NG</em>, <em>54</em>(12), 1895–1906. (<a
href="https://doi.org/10.1038/s41588-022-01241-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cytosine methylation efficiently silences CpG-rich regulatory regions of genes and repeats in mammalian genomes. To what extent this entails direct inhibition of transcription factor (TF) binding versus indirect inhibition via recruitment of methyl-CpG-binding domain (MBD) proteins is unclear. Here we show that combinatorial genetic deletions of all four proteins with functional MBDs in mouse embryonic stem cells, derived neurons or a human cell line do not reactivate genes or repeats with methylated promoters. These do, however, become activated by methylation-restricted TFs if DNA methylation is removed. We identify several causal TFs in neurons, including ONECUT1, which is methylation sensitive only at a motif variant. Rampantly upregulated retrotransposons in methylation-free neurons feature a CRE motif, which activates them in the absence of DNA methylation via methylation-sensitive binding of CREB1. Our study reveals methylation-sensitive TFs in vivo and argues that direct inhibition, rather than indirect repression by the tested MBD proteins, is the prevailing mechanism of methylation-mediated repression at regulatory regions and repeats. Co-deletion of all proteins with a functional methyl-CpG-binding domain does not impact genes or repeats silenced by DNA methylation. Analysis of chromatin accessibility and sequence motifs identifies transcription factors that appear sensitive to DNA methylation.},
  archive      = {J_NG},
  author       = {Kaluscha, Sebastian and Domcke, Silvia and Wirbelauer, Christiane and Stadler, Michael B. and Durdu, Sevi and Burger, Lukas and Schübeler, Dirk},
  doi          = {10.1038/s41588-022-01241-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1895-1906},
  shortjournal = {Nat. Genet.},
  title        = {Evidence that direct inhibition of transcription factor binding is the prevailing mode of gene and repeat repression by DNA methylation},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The landscape of tumor cell states and spatial organization
in h3-K27M mutant diffuse midline glioma across age and location.
<em>NG</em>, <em>54</em>(12), 1881–1894. (<a
href="https://doi.org/10.1038/s41588-022-01236-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Histone 3 lysine27-to-methionine (H3-K27M) mutations most frequently occur in diffuse midline gliomas (DMGs) of the childhood pons but are also increasingly recognized in adults. Their potential heterogeneity at different ages and midline locations is vastly understudied. Here, through dissecting the single-cell transcriptomic, epigenomic and spatial architectures of a comprehensive cohort of patient H3-K27M DMGs, we delineate how age and anatomical location shape glioma cell-intrinsic and -extrinsic features in light of the shared driver mutation. We show that stem-like oligodendroglial precursor-like cells, present across all clinico-anatomical groups, display varying levels of maturation dependent on location. We reveal a previously underappreciated relationship between mesenchymal cancer cell states and age, linked to age-dependent differences in the immune microenvironment. Further, we resolve the spatial organization of H3-K27M DMG cell populations and identify a mitotic oligodendroglial-lineage niche. Collectively, our study provides a powerful framework for rational modeling and therapeutic interventions. Single-cell RNA-seq and ATAC-seq, combined with spatial transcriptomics, identify age- and location-related cellular dynamics of diffuse midline gliomas, such as variable oligodendrocyte precursor-like tumor stem cell populations and increased mesenchymal states with age.},
  archive      = {J_NG},
  author       = {Liu, Ilon and Jiang, Li and Samuelsson, Erik R. and Marco Salas, Sergio and Beck, Alexander and Hack, Olivia A. and Jeong, Daeun and Shaw, McKenzie L. and Englinger, Bernhard and LaBelle, Jenna and Mire, Hafsa M. and Madlener, Sibylle and Mayr, Lisa and Quezada, Michael A. and Trissal, Maria and Panditharatna, Eshini and Ernst, Kati J. and Vogelzang, Jayne and Gatesman, Taylor A. and Halbert, Matthew E. and Palova, Hana and Pokorna, Petra and Sterba, Jaroslav and Slaby, Ondrej and Geyeregger, Rene and Diaz, Aaron and Findlay, Izac J. and Dun, Matthew D. and Resnick, Adam and Suvà, Mario L. and Jones, David T. W. and Agnihotri, Sameer and Svedlund, Jessica and Koschmann, Carl and Haberler, Christine and Czech, Thomas and Slavc, Irene and Cotter, Jennifer A. and Ligon, Keith L. and Alexandrescu, Sanda and Yung, W. K. Alfred and Arrillaga-Romany, Isabel and Gojo, Johannes and Monje, Michelle and Nilsson, Mats and Filbin, Mariella G.},
  doi          = {10.1038/s41588-022-01236-3},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1881-1894},
  shortjournal = {Nat. Genet.},
  title        = {The landscape of tumor cell states and spatial organization in h3-K27M mutant diffuse midline glioma across age and location},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). K27M in canonical and noncanonical h3 variants occurs in
distinct oligodendroglial cell lineages in brain midline gliomas.
<em>NG</em>, <em>54</em>(12), 1865–1880. (<a
href="https://doi.org/10.1038/s41588-022-01205-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Canonical (H3.1/H3.2) and noncanonical (H3.3) histone 3 K27M-mutant gliomas have unique spatiotemporal distributions, partner alterations and molecular profiles. The contribution of the cell of origin to these differences has been challenging to uncouple from the oncogenic reprogramming induced by the mutation. Here, we perform an integrated analysis of 116 tumors, including single-cell transcriptome and chromatin accessibility, 3D chromatin architecture and epigenomic profiles, and show that K27M-mutant gliomas faithfully maintain chromatin configuration at developmental genes consistent with anatomically distinct oligodendrocyte precursor cells (OPCs). H3.3K27M thalamic gliomas map to prosomere 2-derived lineages. In turn, H3.1K27M ACVR1-mutant pontine gliomas uniformly mirror early ventral NKX6-1+/SHH-dependent brainstem OPCs, whereas H3.3K27M gliomas frequently resemble dorsal PAX3+/BMP-dependent progenitors. Our data suggest a context-specific vulnerability in H3.1K27M-mutant SHH-dependent ventral OPCs, which rely on acquisition of ACVR1 mutations to drive aberrant BMP signaling required for oncogenesis. The unifying action of K27M mutations is to restrict H3K27me3 at PRC2 landing sites, whereas other epigenetic changes are mainly contingent on the cell of origin chromatin state and cycling rate. Comprehensive genomic profiling of human H3K27M mutant diffuse midline gliomas, combining single-cell RNA sequencing and ATAC sequencing with bulk ChIP sequencing and other data, proposes distinct oligodendrocyte populations as potential cells of origin.},
  archive      = {J_NG},
  author       = {Jessa, Selin and Mohammadnia, Abdulshakour and Harutyunyan, Ashot S. and Hulswit, Maud and Varadharajan, Srinidhi and Lakkis, Hussein and Kabir, Nisha and Bashardanesh, Zahedeh and Hébert, Steven and Faury, Damien and Vladoiu, Maria C. and Worme, Samantha and Coutelier, Marie and Krug, Brian and Faria Andrade, Augusto and Pathania, Manav and Bajic, Andrea and Weil, Alexander G. and Ellezam, Benjamin and Atkinson, Jeffrey and Dudley, Roy W. R. and Farmer, Jean-Pierre and Perreault, Sebastien and Garcia, Benjamin A. and Larouche, Valérie and Blanchette, Mathieu and Garzia, Livia and Bhaduri, Aparna and Ligon, Keith L. and Bandopadhayay, Pratiti and Taylor, Michael D. and Mack, Stephen C. and Jabado, Nada and Kleinman, Claudia L.},
  doi          = {10.1038/s41588-022-01205-w},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1865-1880},
  shortjournal = {Nat. Genet.},
  title        = {K27M in canonical and noncanonical h3 variants occurs in distinct oligodendroglial cell lineages in brain midline gliomas},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The genomic and immune landscape of long-term survivors of
high-grade serous ovarian cancer. <em>NG</em>, <em>54</em>(12),
1853–1864. (<a
href="https://doi.org/10.1038/s41588-022-01230-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fewer than half of all patients with advanced-stage high-grade serous ovarian cancers (HGSCs) survive more than five years after diagnosis, but those who have an exceptionally long survival could provide insights into tumor biology and therapeutic approaches. We analyzed 60 patients with advanced-stage HGSC who survived more than 10 years after diagnosis using whole-genome sequencing, transcriptome and methylome profiling of their primary tumor samples, comparing this data to 66 short- or moderate-term survivors. Tumors of long-term survivors were more likely to have multiple alterations in genes associated with DNA repair and more frequent somatic variants resulting in an increased predicted neoantigen load. Patients clustered into survival groups based on genomic and immune cell signatures, including three subsets of patients with BRCA1 alterations with distinctly different outcomes. Specific combinations of germline and somatic gene alterations, tumor cell phenotypes and differential immune responses appear to contribute to long-term survival in HGSC. A genomic and transcriptomic analysis identifies molecular features associated with long-term survival in ovarian cancer. Exceptional survival was heterogeneous across the cohort, suggesting that it is likely the function of multiple cell-intrinsic and microenvironmental factors working in combination.},
  archive      = {J_NG},
  author       = {Garsed, Dale W. and Pandey, Ahwan and Fereday, Sian and Kennedy, Catherine J. and Takahashi, Kazuaki and Alsop, Kathryn and Hamilton, Phineas T. and Hendley, Joy and Chiew, Yoke-Eng and Traficante, Nadia and Provan, Pamela and Ariyaratne, Dinuka and Au-Yeung, George and Bateman, Nicholas W. and Bowes, Leanne and Brand, Alison and Christie, Elizabeth L. and Cunningham, Julie M. and Friedlander, Michael and Grout, Bronwyn and Harnett, Paul and Hung, Jillian and McCauley, Bryan and McNally, Orla and Piskorz, Anna M. and Saner, Flurina A. M. and Vierkant, Robert A. and Wang, Chen and Winham, Stacey J. and Pharoah, Paul D. P. and Brenton, James D. and Conrads, Thomas P. and Maxwell, George L. and Ramus, Susan J. and Pearce, Celeste Leigh and Pike, Malcolm C. and Nelson, Brad H. and Goode, Ellen L. and DeFazio, Anna and Bowtell, David D. L.},
  doi          = {10.1038/s41588-022-01230-9},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1853-1864},
  shortjournal = {Nat. Genet.},
  title        = {The genomic and immune landscape of long-term survivors of high-grade serous ovarian cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Loss of NECTIN1 triggers melanoma dissemination upon local
IGF1 depletion. <em>NG</em>, <em>54</em>(12), 1839–1852. (<a
href="https://doi.org/10.1038/s41588-022-01191-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer genetics has uncovered many tumor-suppressor and oncogenic pathways, but few alterations have revealed mechanisms involved in tumor spreading. Here, we examined the role of the third most significant chromosomal deletion in human melanoma that inactivates the adherens junction gene NECTIN1 in 55% of cases. We found that NECTIN1 loss stimulates melanoma cell migration in vitro and spreading in vivo in both zebrafish and human tumors specifically in response to decreased IGF1 signaling. In human melanoma biopsy specimens, adherens junctions were seen exclusively in areas with low IGF1 levels, but not in NECTIN1-deficient tumors. Our study establishes NECTIN1 as a major determinant of melanoma dissemination and uncovers a genetic control of the response to microenvironmental signals. Loss of NECTIN1 is a frequent event in human melanoma and is associated with metastasis. NECTIN1 depletion modulates a switch from cell–cell adhesion to cell–matrix adhesion in response to low levels of IGF1 signaling from the microenvironment.},
  archive      = {J_NG},
  author       = {Ablain, Julien and Al Mahi, Amira and Rothschild, Harriet and Prasad, Meera and Aires, Sophie and Yang, Song and Dokukin, Maxim E. and Xu, Shuyun and Dang, Michelle and Sokolov, Igor and Lian, Christine G. and Zon, Leonard I.},
  doi          = {10.1038/s41588-022-01191-z},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1839-1852},
  shortjournal = {Nat. Genet.},
  title        = {Loss of NECTIN1 triggers melanoma dissemination upon local IGF1 depletion},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The NALCN channel regulates metastasis and nonmalignant cell
dissemination. <em>NG</em>, <em>54</em>(12), 1827–1838. (<a
href="https://doi.org/10.1038/s41588-022-01182-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We identify the sodium leak channel non-selective protein (NALCN) as a key regulator of cancer metastasis and nonmalignant cell dissemination. Among 10,022 human cancers, NALCN loss-of-function mutations were enriched in gastric and colorectal cancers. Deletion of Nalcn from gastric, intestinal or pancreatic adenocarcinomas in mice did not alter tumor incidence, but markedly increased the number of circulating tumor cells (CTCs) and metastases. Treatment of these mice with gadolinium—a NALCN channel blocker—similarly increased CTCs and metastases. Deletion of Nalcn from mice that lacked oncogenic mutations and never developed cancer caused shedding of epithelial cells into the blood at levels equivalent to those seen in tumor-bearing animals. These cells trafficked to distant organs to form normal structures including lung epithelium, and kidney glomeruli and tubules. Thus, NALCN regulates cell shedding from solid tissues independent of cancer, divorcing this process from tumorigenesis and unmasking a potential new target for antimetastatic therapies. The ion channel NALCN regulates cell shedding in mice and enhances metastasis in mouse models of cancer. Disseminated cells without oncogenic mutations form normal structures at secondary sites, suggesting that cell shedding is a physiological process that is hijacked during tumorigenesis.},
  archive      = {J_NG},
  author       = {Rahrmann, Eric P. and Shorthouse, David and Jassim, Amir and Hu, Linda P. and Ortiz, Mariaestela and Mahler-Araujo, Betania and Vogel, Peter and Paez-Ribes, Marta and Fatemi, Atefeh and Hannon, Gregory J. and Iyer, Radhika and Blundon, Jay A. and Lourenço, Filipe C. and Kay, Jonathan and Nazarian, Rosalynn M. and Hall, Benjamin A. and Zakharenko, Stanislav S. and Winton, Douglas J. and Zhu, Liqin and Gilbertson, Richard J.},
  doi          = {10.1038/s41588-022-01182-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1827-1838},
  shortjournal = {Nat. Genet.},
  title        = {The NALCN channel regulates metastasis and nonmalignant cell dissemination},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Novel genetic loci associated with osteoarthritis in
multi-ancestry analyses in the million veteran program and UK biobank.
<em>NG</em>, <em>54</em>(12), 1816–1826. (<a
href="https://doi.org/10.1038/s41588-022-01221-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Osteoarthritis is a common progressive joint disease. As no effective medical interventions are available, osteoarthritis often progresses to the end stage, in which only surgical options such as total joint replacement are available. A more thorough understanding of genetic influences of osteoarthritis is essential to develop targeted personalized approaches to treatment, ideally long before the end stage is reached. To date, there have been no large multiancestry genetic studies of osteoarthritis. Here, we leveraged the unique resources of 484,374 participants in the Million Veteran Program and UK Biobank to address this gap. Analyses included participants of European, African, Asian and Hispanic descent. We discovered osteoarthritis-associated genetic variation at 10 loci and replicated findings from previous osteoarthritis studies. We also present evidence that some osteoarthritis-associated regions are robust to population ancestry. Drug repurposing analyses revealed enrichment of targets of several medication classes and provide potential insight into the etiology of beneficial effects of antiepileptics on osteoarthritis pain. Multiancestry genome-wide association analyses in the Million Veteran Program and UK Biobank identify new risk loci for osteoarthritis. Drug repurposing analyses yield potential insights into the effects of antiepileptics on osteoarthritis pain.},
  archive      = {J_NG},
  author       = {McDonald, Merry-Lynn N. and Lakshman Kumar, Preeti and Srinivasasainagendra, Vinodh and Nair, Ashwathy and Rocco, Alison P. and Wilson, Ava C. and Chiles, Joe W. and Richman, Joshua S. and Pinson, Sarah A. and Dennis, Richard A. and Jagadale, Vivek and Brown, Cynthia J. and Pyarajan, Saiju and Tiwari, Hemant K. and Bamman, Marcas M. and Singh, Jasvinder A.},
  doi          = {10.1038/s41588-022-01221-w},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1816-1826},
  shortjournal = {Nat. Genet.},
  title        = {Novel genetic loci associated with osteoarthritis in multi-ancestry analyses in the million veteran program and UK biobank},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Discovery and systematic characterization of risk variants
and genes for coronary artery disease in over a million participants.
<em>NG</em>, <em>54</em>(12), 1803–1815. (<a
href="https://doi.org/10.1038/s41588-022-01233-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The discovery of genetic loci associated with complex diseases has outpaced the elucidation of mechanisms of disease pathogenesis. Here we conducted a genome-wide association study (GWAS) for coronary artery disease (CAD) comprising 181,522 cases among 1,165,690 participants of predominantly European ancestry. We detected 241 associations, including 30 new loci. Cross-ancestry meta-analysis with a Japanese GWAS yielded 38 additional new loci. We prioritized likely causal variants using functionally informed fine-mapping, yielding 42 associations with less than five variants in the 95% credible set. Similarity-based clustering suggested roles for early developmental processes, cell cycle signaling and vascular cell migration and proliferation in the pathogenesis of CAD. We prioritized 220 candidate causal genes, combining eight complementary approaches, including 123 supported by three or more approaches. Using CRISPR–Cas9, we experimentally validated the effect of an enhancer in MYO9B, which appears to mediate CAD risk by regulating vascular cell motility. Our analysis identifies and systematically characterizes &amp;gt;250 risk loci for CAD to inform experimental interrogation of putative causal mechanisms for CAD. Genome-wide association analyses, functionally informed fine-mapping and complementary gene prioritization approaches identify new risk loci and candidate effector genes for CAD.},
  archive      = {J_NG},
  author       = {Aragam, Krishna G. and Jiang, Tao and Goel, Anuj and Kanoni, Stavroula and Wolford, Brooke N. and Atri, Deepak S. and Weeks, Elle M. and Wang, Minxian and Hindy, George and Zhou, Wei and Grace, Christopher and Roselli, Carolina and Marston, Nicholas A. and Kamanu, Frederick K. and Surakka, Ida and Venegas, Loreto Muñoz and Sherliker, Paul and Koyama, Satoshi and Ishigaki, Kazuyoshi and Åsvold, Bjørn O. and Brown, Michael R. and Brumpton, Ben and de Vries, Paul S. and Giannakopoulou, Olga and Giardoglou, Panagiota and Gudbjartsson, Daniel F. and Güldener, Ulrich and Haider, Syed M. Ijlal and Helgadottir, Anna and Ibrahim, Maysson and Kastrati, Adnan and Kessler, Thorsten and Kyriakou, Theodosios and Konopka, Tomasz and Li, Ling and Ma, Lijiang and Meitinger, Thomas and Mucha, Sören and Munz, Matthias and Murgia, Federico and Nielsen, Jonas B. and Nöthen, Markus M. and Pang, Shichao and Reinberger, Tobias and Schnitzler, Gavin and Smedley, Damian and Thorleifsson, Gudmar and von Scheidt, Moritz and Ulirsch, Jacob C. and Arnar, David O. and Burtt, Noël P. and Costanzo, Maria C. and Flannick, Jason and Ito, Kaoru and Jang, Dong-Keun and Kamatani, Yoichiro and Khera, Amit V. and Komuro, Issei and Kullo, Iftikhar J. and Lotta, Luca A. and Nelson, Christopher P. and Roberts, Robert and Thorgeirsson, Gudmundur and Thorsteinsdottir, Unnur and Webb, Thomas R. and Baras, Aris and Björkegren, Johan L. M. and Boerwinkle, Eric and Dedoussis, George and Holm, Hilma and Hveem, Kristian and Melander, Olle and Morrison, Alanna C. and Orho-Melander, Marju and Rallidis, Loukianos S. and Ruusalepp, Arno and Sabatine, Marc S. and Stefansson, Kari and Zalloua, Pierre and Ellinor, Patrick T. and Farrall, Martin and Danesh, John and Ruff, Christian T. and Finucane, Hilary K. and Hopewell, Jemma C. and Clarke, Robert and Gupta, Rajat M. and Erdmann, Jeanette and Samani, Nilesh J. and Schunkert, Heribert and Watkins, Hugh and Willer, Cristen J. and Deloukas, Panos and Kathiresan, Sekar and Butterworth, Adam S.},
  doi          = {10.1038/s41588-022-01233-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1803-1815},
  shortjournal = {Nat. Genet.},
  title        = {Discovery and systematic characterization of risk variants and genes for coronary artery disease in over a million participants},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Exploring the genetic overlap between twelve psychiatric
disorders. <em>NG</em>, <em>54</em>(12), 1795–1802. (<a
href="https://doi.org/10.1038/s41588-022-01245-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The widespread comorbidity among psychiatric disorders demonstrated in epidemiological studies1–5 is mirrored by non-zero, positive genetic correlations from large-scale genetic studies6–10. To identify shared biological processes underpinning this observed phenotypic and genetic covariance and enhance molecular characterization of general psychiatric disorder liability11–13, we used several strategies aimed at uncovering pleiotropic, that is, cross-trait-associated, single-nucleotide polymorphisms (SNPs), genes and biological pathways. We conducted cross-trait meta-analysis on 12 psychiatric disorders to identify pleiotropic SNPs. The meta-analytic signal was driven by schizophrenia, hampering interpretation and joint biological characterization of the cross-trait meta-analytic signal. Subsequent pairwise comparisons of psychiatric disorders identified substantial pleiotropic overlap, but mainly among pairs of psychiatric disorders, and mainly at less stringent P-value thresholds. Only annotations related to evolutionarily conserved genomic regions were significant for multiple (9 out of 12) psychiatric disorders. Overall, identification of shared biological mechanisms remains challenging due to variation in power and genetic architecture between psychiatric disorders. Cross-trait meta-analysis on 12 psychiatric disorders identifies the genetic overlap between pairs of disorders and highlights the challenges of cross-trait genetic research.},
  archive      = {J_NG},
  author       = {Romero, Cato and Werme, Josefin and Jansen, Philip R. and Gelernter, Joel and Stein, Murray B. and Levey, Daniel and Polimanti, Renato and de Leeuw, Christiaan and Posthuma, Danielle and Nagel, Mats and van der Sluis, Sophie},
  doi          = {10.1038/s41588-022-01245-2},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1795-1802},
  shortjournal = {Nat. Genet.},
  title        = {Exploring the genetic overlap between twelve psychiatric disorders},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Exome sequencing identifies rare damaging variants in ATP8B4
and ABCA1 as risk factors for alzheimer’s disease. <em>NG</em>,
<em>54</em>(12), 1786–1794. (<a
href="https://doi.org/10.1038/s41588-022-01208-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Alzheimer’s disease (AD), the leading cause of dementia, has an estimated heritability of approximately 70%1. The genetic component of AD has been mainly assessed using genome-wide association studies, which do not capture the risk contributed by rare variants2. Here, we compared the gene-based burden of rare damaging variants in exome sequencing data from 32,558 individuals—16,036 AD cases and 16,522 controls. Next to variants in TREM2, SORL1 and ABCA7, we observed a significant association of rare, predicted damaging variants in ATP8B4 and ABCA1 with AD risk, and a suggestive signal in ADAM10. Additionally, the rare-variant burden in RIN3, CLU, ZCWPW1 and ACE highlighted these genes as potential drivers of respective AD-genome-wide association study loci. Variants associated with the strongest effect on AD risk, in particular loss-of-function variants, are enriched in early-onset AD cases. Our results provide additional evidence for a major role for amyloid-β precursor protein processing, amyloid-β aggregation, lipid metabolism and microglial function in AD. Multistage gene burden analysis in exome sequencing data from 32,558 individuals identifies rare damaging variants in ATP8B4 and ABCA1 as risk factors for Alzheimer’s disease.},
  archive      = {J_NG},
  author       = {Holstege, Henne and Hulsman, Marc and Charbonnier, Camille and Grenier-Boley, Benjamin and Quenez, Olivier and Grozeva, Detelina and van Rooij, Jeroen G. J. and Sims, Rebecca and Ahmad, Shahzad and Amin, Najaf and Norsworthy, Penny J. and Dols-Icardo, Oriol and Hummerich, Holger and Kawalia, Amit and Amouyel, Philippe and Beecham, Gary W. and Berr, Claudine and Bis, Joshua C. and Boland, Anne and Bossù, Paola and Bouwman, Femke and Bras, Jose and Campion, Dominique and Cochran, J. Nicholas and Daniele, Antonio and Dartigues, Jean-François and Debette, Stéphanie and Deleuze, Jean-François and Denning, Nicola and DeStefano, Anita L. and Farrer, Lindsay A. and Fernández, Maria Victoria and Fox, Nick C. and Galimberti, Daniela and Genin, Emmanuelle and Gille, Johan J. P. and Le Guen, Yann and Guerreiro, Rita and Haines, Jonathan L. and Holmes, Clive and Ikram, M. Arfan and Ikram, M. Kamran and Jansen, Iris E. and Kraaij, Robert and Lathrop, Marc and Lemstra, Afina W. and Lleó, Alberto and Luckcuck, Lauren and Mannens, Marcel M. A. M. and Marshall, Rachel and Martin, Eden R. and Masullo, Carlo and Mayeux, Richard and Mecocci, Patrizia and Meggy, Alun and Mol, Merel O. and Morgan, Kevin and Myers, Richard M. and Nacmias, Benedetta and Naj, Adam C. and Napolioni, Valerio and Pasquier, Florence and Pastor, Pau and Pericak-Vance, Margaret A. and Raybould, Rachel and Redon, Richard and Reinders, Marcel J. T. and Richard, Anne-Claire and Riedel-Heller, Steffi G. and Rivadeneira, Fernando and Rousseau, Stéphane and Ryan, Natalie S. and Saad, Salha and Sanchez-Juan, Pascual and Schellenberg, Gerard D. and Scheltens, Philip and Schott, Jonathan M. and Seripa, Davide and Seshadri, Sudha and Sie, Daoud and Sistermans, Erik A. and Sorbi, Sandro and van Spaendonk, Resie and Spalletta, Gianfranco and Tesi, Niccolo’ and Tijms, Betty and Uitterlinden, André G. and van der Lee, Sven J. and Visser, Pieter Jelle and Wagner, Michael and Wallon, David and Wang, Li-San and Zarea, Aline and Clarimon, Jordi and van Swieten, John C. and Greicius, Michael D. and Yokoyama, Jennifer S. and Cruchaga, Carlos and Hardy, John and Ramirez, Alfredo and Mead, Simon and van der Flier, Wiesje M. and van Duijn, Cornelia M. and Williams, Julie and Nicolas, Gaël and Bellenguez, Céline and Lambert, Jean-Charles},
  doi          = {10.1038/s41588-022-01208-7},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1786-1794},
  shortjournal = {Nat. Genet.},
  title        = {Exome sequencing identifies rare damaging variants in ATP8B4 and ABCA1 as risk factors for alzheimer’s disease},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Tumor aneuploidy predicts survival following immunotherapy
across multiple cancers. <em>NG</em>, <em>54</em>(12), 1782–1785. (<a
href="https://doi.org/10.1038/s41588-022-01235-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumor mutational burden (TMB) has emerged as a promising biomarker of immunotherapy response across multiple cancer types; however, clinical outcomes among patients with low TMB tumors are heterogeneous. Herein, we demonstrate that tumor aneuploidy provides independent prognostic value among patients with lower TMB (&amp;lt;80th percentile) tumors treated with immunotherapy. A higher aneuploidy score is associated with poor prognosis following immunotherapy among tumors with low TMB, but not those with high TMB. Importantly, aneuploidy scores can be calculated from existing clinical targeted sequencing infrastructure, facilitating deployment of aneuploidy scores as a clinical biomarker. Reanalysis of the immunotherapy-treated MSK-IMPACT cohort finds that chromosomal arm-level aneuploidy scores are an independent predictor of survival for patients with low tumor mutational burden.},
  archive      = {J_NG},
  author       = {Spurr, Liam F. and Weichselbaum, Ralph R. and Pitroda, Sean P.},
  doi          = {10.1038/s41588-022-01235-4},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1782-1785},
  shortjournal = {Nat. Genet.},
  title        = {Tumor aneuploidy predicts survival following immunotherapy across multiple cancers},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Beating the odds: Molecular characteristics of long-term
survivors of ovarian cancer. <em>NG</em>, <em>54</em>(12), 1780–1781.
(<a href="https://doi.org/10.1038/s41588-022-01234-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-grade serous ovarian cancer, the most common form of the disease, is often fatal. This study investigated the genomic and immune characteristics of tumors from women who survived more than 10 years after their initial diagnosis, and compared them with short-term and moderate-term survivors.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01234-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1780-1781},
  shortjournal = {Nat. Genet.},
  title        = {Beating the odds: Molecular characteristics of long-term survivors of ovarian cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A versatile functional assay for genetic variants in human
disease. <em>NG</em>, <em>54</em>(12), 1778–1779. (<a
href="https://doi.org/10.1038/s41588-022-01226-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a CRISPR-based functional assay for genetic sequence variants found in human disease, probing their effects on cell proliferation, survival, motility and any physiological or pathological process measurable by fluorescence-activated cell sorting (FACS). The assay accurately assessed variant pathogenicity, drug responsiveness or resistance and mechanistic role in disease, in vitro and in vivo.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01226-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1778-1779},
  shortjournal = {Nat. Genet.},
  title        = {A versatile functional assay for genetic variants in human disease},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). NECTIN1 is a melanoma metastasis suppressor gene.
<em>NG</em>, <em>54</em>(12), 1776–1777. (<a
href="https://doi.org/10.1038/s41588-022-01229-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Few genetic alterations have been linked to metastasis, during which cancer cells acquire abnormal migratory behavior. A new study sheds light on how loss of NECTIN1 leads to melanoma dissemination after local depletion of IGF1.},
  archive      = {J_NG},
  author       = {Barcelo, Jaume and Sanz-Moreno, Victoria},
  doi          = {10.1038/s41588-022-01229-2},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1776-1777},
  shortjournal = {Nat. Genet.},
  title        = {NECTIN1 is a melanoma metastasis suppressor gene},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Skip to the end: Metastasis before tumorigenesis.
<em>NG</em>, <em>54</em>(12), 1774–1775. (<a
href="https://doi.org/10.1038/s41588-022-01228-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations in the sodium channel NALCN promote epithelial cell shedding and dissemination independent of oncogenic transformation. This observation suggests that metastasis may not uniformly represent the end stage of carcinogenesis but can occur before oncogenic transformation.},
  archive      = {J_NG},
  author       = {Nobre, Ana Rita and Boire, Adrienne},
  doi          = {10.1038/s41588-022-01228-3},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1774-1775},
  shortjournal = {Nat. Genet.},
  title        = {Skip to the end: Metastasis before tumorigenesis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Two bHLH transcription factors affect sprouting by
regulating the level of ABA. <em>NG</em>, <em>54</em>(12), 1772–1773.
(<a href="https://doi.org/10.1038/s41588-022-01238-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For plants, the decision to germinate is a gamble on their subsequent survival. Xu et al. have now discovered a mechanism that determines the choice between germination and dormancy by regulating the level of ABA.},
  archive      = {J_NG},
  author       = {Yoshida, Hideki and Matsuoka, Makoto},
  doi          = {10.1038/s41588-022-01238-1},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1772-1773},
  shortjournal = {Nat. Genet.},
  title        = {Two bHLH transcription factors affect sprouting by regulating the level of ABA},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Defining the cell of origin for diffuse midline gliomas.
<em>NG</em>, <em>54</em>(12), 1770–1771. (<a
href="https://doi.org/10.1038/s41588-022-01231-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {K27M mutation of histone H3 has been identified as a driver event in diffuse midline glioma. Two studies used comprehensive multi-model single-cell genomic, epigenomic and chromatin structure analysis to characterize the cell of origin and find a distinct etiology of H3K27M between pontine and thalamic tumors, and show that pontine gliomas harbor more immature oligodendrocyte-precursor-like cells.},
  archive      = {J_NG},
  author       = {Li, Xiao-Nan},
  doi          = {10.1038/s41588-022-01231-8},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1770-1771},
  shortjournal = {Nat. Genet.},
  title        = {Defining the cell of origin for diffuse midline gliomas},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Finding causal genes underlying risk for coronary artery
disease. <em>NG</em>, <em>54</em>(12), 1768–1769. (<a
href="https://doi.org/10.1038/s41588-022-01094-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Previous genome-wide association studies of coronary artery disease (CAD) have discovered multiple susceptibility loci but have largely failed to uncover causal genes. A new study identifies hundreds of likely causal genes underlying the genetic risk for CAD.},
  archive      = {J_NG},
  author       = {Auer, Paul L.},
  doi          = {10.1038/s41588-022-01094-z},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1768-1769},
  shortjournal = {Nat. Genet.},
  title        = {Finding causal genes underlying risk for coronary artery disease},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Endogenous retrovirus expression during fruitfly
metamorphosis enhances adult viral immunity. <em>NG</em>,
<em>54</em>(12), 1765–1767. (<a
href="https://doi.org/10.1038/s41588-022-01209-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The evolutionary benefits of ongoing endogenous retrovirus (ERV) mobility in animals are largely unknown. A new paradigm suggests activation of the ERV family mdg4 at the fruitfly pupal stage may bolster adult antiviral defenses.},
  archive      = {J_NG},
  author       = {Botto, Juan M. and Faulkner, Geoffrey J.},
  doi          = {10.1038/s41588-022-01209-6},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1765-1767},
  shortjournal = {Nat. Genet.},
  title        = {Endogenous retrovirus expression during fruitfly metamorphosis enhances adult viral immunity},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Human height-associated variants. <em>NG</em>,
<em>54</em>(12), 1764. (<a
href="https://doi.org/10.1038/s41588-022-01263-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-022-01263-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1764},
  shortjournal = {Nat. Genet.},
  title        = {Human height-associated variants},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Global biobank meta-analysis. <em>NG</em>, <em>54</em>(12),
1764. (<a href="https://doi.org/10.1038/s41588-022-01264-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-022-01264-z},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1764},
  shortjournal = {Nat. Genet.},
  title        = {Global biobank meta-analysis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Black death and the evolution of immunity. <em>NG</em>,
<em>54</em>(12), 1764. (<a
href="https://doi.org/10.1038/s41588-022-01265-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-022-01265-y},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1764},
  shortjournal = {Nat. Genet.},
  title        = {Black death and the evolution of immunity},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Butterfly wing pattern diversity. <em>NG</em>,
<em>54</em>(12), 1764. (<a
href="https://doi.org/10.1038/s41588-022-01266-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-022-01266-x},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1764},
  shortjournal = {Nat. Genet.},
  title        = {Butterfly wing pattern diversity},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mapping the arab genome. <em>NG</em>, <em>54</em>(12),
1761–1763. (<a
href="https://doi.org/10.1038/s41588-022-01239-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Qatar Genome Program was established to interrogate the genomics and genetics of populations in the Middle East. Improving precision medicine strategies and building long-term research capacity are both key aims of the initiative.},
  archive      = {J_NG},
  author       = {Mbarek, Hamdi and Ismail, Said I.},
  doi          = {10.1038/s41588-022-01239-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1761-1763},
  shortjournal = {Nat. Genet.},
  title        = {Mapping the arab genome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). GATTACA is still pertinent 25 years later. <em>NG</em>,
<em>54</em>(12), 1758–1760. (<a
href="https://doi.org/10.1038/s41588-022-01242-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It has been 25 years since the release of GATTACA, a film that tells the story of a credible near future in which society’s inequalities, formerly associated with race and class, have been replaced with new prejudices based on genetic determinism. Here we compare GATTACA’s fictional technologies with reality’s state of the art, assessing the legal protections afforded in today’s society against GATTACA’s dystopian future in which personal freedom and privacy rights are substantially curtailed by genomic innovations. We further discuss how GATTACA’s prescient forewarnings are still relevant today in light of the current trajectory of genomic science and technology.},
  archive      = {J_NG},
  author       = {Greenbaum, Dov and Gerstein, Mark},
  doi          = {10.1038/s41588-022-01242-5},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1758-1760},
  shortjournal = {Nat. Genet.},
  title        = {GATTACA is still pertinent 25 years later},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetic futurism. <em>NG</em>, <em>54</em>(12), 1757. (<a
href="https://doi.org/10.1038/s41588-022-01271-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale genotyping and phenotyping efforts, including biobanks, have revolutionized our understanding of the genetic architecture of human traits and diseases. Years of ever-larger genome-wide association studies (GWAS) have produced a catalog of genetic variants that contribute to complex traits. A corollary of this research has been the development of personalized polygenic scores (PGS) or polygenic risk scores (PRS).},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01271-0},
  journal      = {Nature Genetics},
  month        = {12},
  number       = {12},
  pages        = {1757},
  shortjournal = {Nat. Genet.},
  title        = {Genetic futurism},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Publisher correction: SAIGE-GENE+ improves the efficiency
and accuracy of set-based rare variant association tests. <em>NG</em>,
<em>54</em>(11), 1755. (<a
href="https://doi.org/10.1038/s41588-022-01220-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Zhou, Wei and Bi, Wenjian and Zhao, Zhangchen and Dey, Kushal K. and Jagadeesh, Karthik A. and Karczewski, Konrad J. and Daly, Mark J. and Neale, Benjamin M. and Lee, Seunggeun},
  doi          = {10.1038/s41588-022-01220-x},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1755},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: SAIGE-GENE+ improves the efficiency and accuracy of set-based rare variant association tests},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Targeted profiling of human extrachromosomal DNA by
CRISPR-CATCH. <em>NG</em>, <em>54</em>(11), 1746–1754. (<a
href="https://doi.org/10.1038/s41588-022-01190-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extrachromosomal DNA (ecDNA) is a common mode of oncogene amplification but is challenging to analyze. Here, we adapt CRISPR-CATCH, in vitro CRISPR-Cas9 treatment and pulsed field gel electrophoresis of agarose-entrapped genomic DNA, previously developed for bacterial chromosome segments, to isolate megabase-sized human ecDNAs. We demonstrate strong enrichment of ecDNA molecules containing EGFR, FGFR2 and MYC from human cancer cells and NRAS ecDNA from human metastatic melanoma with acquired therapeutic resistance. Targeted enrichment of ecDNA versus chromosomal DNA enabled phasing of genetic variants, identified the presence of an EGFRvIII mutation exclusively on ecDNAs and supported an excision model of ecDNA genesis in a glioblastoma model. CRISPR-CATCH followed by nanopore sequencing enabled single-molecule ecDNA methylation profiling and revealed hypomethylation of the EGFR promoter on ecDNAs. We distinguished heterogeneous ecDNA species within the same sample by size and sequence with base-pair resolution and discovered functionally specialized ecDNAs that amplify select enhancers or oncogene-coding sequences. CRISPR-CATCH is used to isolate extrachromosomal DNA (ecDNA) molecules containing oncogenes from human cancer cells. CRISPR-CATCH followed by nanopore sequencing allows for methylation profiling, highlighting differences from the native chromosomal loci.},
  archive      = {J_NG},
  author       = {Hung, King L. and Luebeck, Jens and Dehkordi, Siavash R. and Colón, Caterina I. and Li, Rui and Wong, Ivy Tsz-Lo and Coruh, Ceyda and Dharanipragada, Prashanthi and Lomeli, Shirley H. and Weiser, Natasha E. and Moriceau, Gatien and Zhang, Xiao and Bailey, Chris and Houlahan, Kathleen E. and Yang, Wenting and González, Rocío Chamorro and Swanton, Charles and Curtis, Christina and Jamal-Hanjani, Mariam and Henssen, Anton G. and Law, Julie A. and Greenleaf, William J. and Lo, Roger S. and Mischel, Paul S. and Bafna, Vineet and Chang, Howard Y.},
  doi          = {10.1038/s41588-022-01190-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1746-1754},
  shortjournal = {Nat. Genet.},
  title        = {Targeted profiling of human extrachromosomal DNA by CRISPR-CATCH},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome sequencing reveals evidence of adaptive variation in
the genus zea. <em>NG</em>, <em>54</em>(11), 1736–1745. (<a
href="https://doi.org/10.1038/s41588-022-01184-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Maize is a globally valuable commodity and one of the most extensively studied genetic model organisms. However, we know surprisingly little about the extent and potential utility of the genetic variation found in wild relatives of maize. Here, we characterize a high-density genomic variation map from 744 genomes encompassing maize and all wild taxa of the genus Zea, identifying over 70 million single-nucleotide polymorphisms. The variation map reveals evidence of selection within taxa displaying novel adaptations. We focus on adaptive alleles in highland teosinte and temperate maize, highlighting the key role of flowering-time-related pathways in their adaptation. To show the utility of variants in these data, we generate mutant alleles for two flowering-time candidate genes. This work provides an extensive sampling of the genetic diversity of Zea, resolving questions on evolution and identifying adaptive variants for direct use in modern breeding. A high-density genomic variation map from 744 genomes encompassing maize and all wild taxa of the genus Zea reveals evidence of adaptive variation and provides a genus-wide resource of genetic diversity in Zea.},
  archive      = {J_NG},
  author       = {Chen, Lu and Luo, Jingyun and Jin, Minliang and Yang, Ning and Liu, Xiangguo and Peng, Yong and Li, Wenqiang and Phillips, Alyssa and Cameron, Brenda and Bernal, Julio S. and Rellán-Álvarez, Rubén and Sawers, Ruairidh J. H. and Liu, Qing and Yin, Yuejia and Ye, Xinnan and Yan, Jiali and Zhang, Qinghua and Zhang, Xiaoting and Wu, Shenshen and Gui, Songtao and Wei, Wenjie and Wang, Yuebin and Luo, Yun and Jiang, Chenglin and Deng, Min and Jin, Min and Jian, Liumei and Yu, Yanhui and Zhang, Maolin and Yang, Xiaohong and Hufford, Matthew B. and Fernie, Alisdair R. and Warburton, Marilyn L. and Ross-Ibarra, Jeffrey and Yan, Jianbing},
  doi          = {10.1038/s41588-022-01184-y},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1736-1745},
  shortjournal = {Nat. Genet.},
  title        = {Genome sequencing reveals evidence of adaptive variation in the genus zea},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Histone h3 proline 16 hydroxylation regulates mammalian gene
expression. <em>NG</em>, <em>54</em>(11), 1721–1735. (<a
href="https://doi.org/10.1038/s41588-022-01212-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Histone post-translational modifications (PTMs) are important for regulating various DNA-templated processes. Here, we report the existence of a histone PTM in mammalian cells, namely histone H3 with hydroxylation of proline at residue 16 (H3P16oh), which is catalyzed by the proline hydroxylase EGLN2. We show that H3P16oh enhances direct binding of KDM5A to its substrate, histone H3 with trimethylation at the fourth lysine residue (H3K4me3), resulting in enhanced chromatin recruitment of KDM5A and a corresponding decrease of H3K4me3 at target genes. Genome- and transcriptome-wide analyses show that the EGLN2–KDM5A axis regulates target gene expression in mammalian cells. Specifically, our data demonstrate repression of the WNT pathway negative regulator DKK1 through the EGLN2-H3P16oh-KDM5A pathway to promote WNT/β-catenin signaling in triple-negative breast cancer (TNBC). This study characterizes a regulatory mark in the histone code and reveals a role for H3P16oh in regulating mammalian gene expression. EGLN2 hydroxylates histone H3 at proline 16, enhancing the binding of KDM5A to H3K4me3. Genome- and transcriptome-wide analyses show that the EGLN2–KDM5A axis regulates target gene expression in mammalian cells.},
  archive      = {J_NG},
  author       = {Liu, Xijuan and Wang, Jun and Boyer, Joshua A. and Gong, Weida and Zhao, Shuai and Xie, Ling and Wu, Qiong and Zhang, Cheng and Jain, Kanishk and Guo, Yiran and Rodriguez, Javier and Li, Mingjie and Uryu, Hidetaka and Liao, Chengheng and Hu, Lianxin and Zhou, Jin and Shi, Xiaobing and Tsai, Yi-Hsuan and Yan, Qin and Luo, Weibo and Chen, Xian and Strahl, Brian D. and von Kriegsheim, Alex and Zhang, Qi and Wang, Gang Greg and Baldwin, Albert S. and Zhang, Qing},
  doi          = {10.1038/s41588-022-01212-x},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1721-1735},
  shortjournal = {Nat. Genet.},
  title        = {Histone h3 proline 16 hydroxylation regulates mammalian gene expression},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep learning of cross-species single-cell landscapes
identifies conserved regulatory programs underlying cell types.
<em>NG</em>, <em>54</em>(11), 1711–1720. (<a
href="https://doi.org/10.1038/s41588-022-01197-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite extensive efforts to generate and analyze reference genomes, genetic models to predict gene regulation and cell fate decisions are lacking for most species. Here, we generated whole-body single-cell transcriptomic landscapes of zebrafish, Drosophila and earthworm. We then integrated cell landscapes from eight representative metazoan species to study gene regulation across evolution. Using these uniformly constructed cross-species landscapes, we developed a deep-learning-based strategy, Nvwa, to predict gene expression and identify regulatory sequences at the single-cell level. We systematically compared cell-type-specific transcription factors to reveal conserved genetic regulation in vertebrates and invertebrates. Our work provides a valuable resource and offers a new strategy for studying regulatory grammar in diverse biological systems. Microwell-seq is used to generate single-cell, whole-body expression landscapes of zebrafish, Drosophila and earthworm. A deep-learning model, Nvwa, predicts gene expression from DNA sequence and identifies regulatory programs shared across eight species.},
  archive      = {J_NG},
  author       = {Li, Jiaqi and Wang, Jingjing and Zhang, Peijing and Wang, Renying and Mei, Yuqing and Sun, Zhongyi and Fei, Lijiang and Jiang, Mengmeng and Ma, Lifeng and E, Weigao and Chen, Haide and Wang, Xinru and Fu, Yuting and Wu, Hanyu and Liu, Daiyuan and Wang, Xueyi and Li, Jingyu and Guo, Qile and Liao, Yuan and Yu, Chengxuan and Jia, Danmei and Wu, Jian and He, Shibo and Liu, Huanju and Ma, Jun and Lei, Kai and Chen, Jiming and Han, Xiaoping and Guo, Guoji},
  doi          = {10.1038/s41588-022-01197-7},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1711-1720},
  shortjournal = {Nat. Genet.},
  title        = {Deep learning of cross-species single-cell landscapes identifies conserved regulatory programs underlying cell types},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DNA sequence and chromatin modifiers cooperate to confer
epigenetic bistability at imprinting control regions. <em>NG</em>,
<em>54</em>(11), 1702–1710. (<a
href="https://doi.org/10.1038/s41588-022-01210-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic imprinting is regulated by parental-specific DNA methylation of imprinting control regions (ICRs). Despite an identical DNA sequence, ICRs can exist in two distinct epigenetic states that are memorized throughout unlimited cell divisions and reset during germline formation. Here, we systematically study the genetic and epigenetic determinants of this epigenetic bistability. By iterative integration of ICRs and related DNA sequences to an ectopic location in the mouse genome, we first identify the DNA sequence features required for maintenance of epigenetic states in embryonic stem cells. The autonomous regulatory properties of ICRs further enabled us to create DNA-methylation-sensitive reporters and to screen for key components involved in regulating their epigenetic memory. Besides DNMT1, UHRF1 and ZFP57, we identify factors that prevent switching from methylated to unmethylated states and show that two of these candidates, ATF7IP and ZMYM2, are important for the stability of DNA and H3K9 methylation at ICRs in embryonic stem cells. Ectopic imprinting control regions (ICRs) recapitulate chromatin states of endogenous imprinted loci in mouse embryonic stem cells. ATF7IP and ZMYM2 regulate epigenetic memory at ICRs.},
  archive      = {J_NG},
  author       = {Butz, Stefan and Schmolka, Nina and Karemaker, Ino D. and Villaseñor, Rodrigo and Schwarz, Isabel and Domcke, Silvia and Uijttewaal, Esther C. H. and Jude, Julian and Lienert, Florian and Krebs, Arnaud R. and de Wagenaar, Nathalie P. and Bao, Xue and Zuber, Johannes and Elling, Ulrich and Schübeler, Dirk and Baubec, Tuncay},
  doi          = {10.1038/s41588-022-01210-z},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1702-1710},
  shortjournal = {Nat. Genet.},
  title        = {DNA sequence and chromatin modifiers cooperate to confer epigenetic bistability at imprinting control regions},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Adult human kidney organoids originate from CD24+ cells and
represent an advanced model for adult polycystic kidney disease.
<em>NG</em>, <em>54</em>(11), 1690–1701. (<a
href="https://doi.org/10.1038/s41588-022-01202-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adult kidney organoids have been described as strictly tubular epithelia and termed tubuloids. While the cellular origin of tubuloids has remained elusive, here we report that they originate from a distinct CD24+ epithelial subpopulation. Long-term-cultured CD24+ cell-derived tubuloids represent a functional human kidney tubule. We show that kidney tubuloids can be used to model the most common inherited kidney disease, namely autosomal dominant polycystic kidney disease (ADPKD), reconstituting the phenotypic hallmark of this disease with cyst formation. Single-cell RNA sequencing of CRISPR–Cas9 gene-edited PKD1- and PKD2-knockout tubuloids and human ADPKD and control tissue shows similarities in upregulation of disease-driving genes. Furthermore, in a proof of concept, we demonstrate that tolvaptan, the only approved drug for ADPKD, has a significant effect on cyst size in tubuloids but no effect on a pluripotent stem cell-derived model. Thus, tubuloids are derived from a tubular epithelial subpopulation and represent an advanced system for ADPKD disease modeling. Adult kidney organoids, or tubuloids, originate from CD24+ epithelial cells. Tubuloids represent a functional kidney tubule and can be used to model autosomal dominant polycystic kidney disease and study drug response.},
  archive      = {J_NG},
  author       = {Xu, Yaoxian and Kuppe, Christoph and Perales-Patón, Javier and Hayat, Sikander and Kranz, Jennifer and Abdallah, Ali T. and Nagai, James and Li, Zhijian and Peisker, Fabian and Saritas, Turgay and Halder, Maurice and Menzel, Sylvia and Hoeft, Konrad and Kenter, Annegien and Kim, Hyojin and van Roeyen, Claudia R. C. and Lehrke, Michael and Moellmann, Julia and Speer, Thimoteus and Buhl, Eva M. and Hoogenboezem, Remco and Boor, Peter and Jansen, Jitske and Knopp, Cordula and Kurth, Ingo and Smeets, Bart and Bindels, Eric and Reinders, Marlies E. J. and Baan, Carla and Gribnau, Joost and Hoorn, Ewout J. and Steffens, Joachim and Huber, Tobias B. and Costa, Ivan and Floege, Jürgen and Schneider, Rebekka K. and Saez-Rodriguez, Julio and Freedman, Benjamin S. and Kramann, Rafael},
  doi          = {10.1038/s41588-022-01202-z},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1690-1701},
  shortjournal = {Nat. Genet.},
  title        = {Adult human kidney organoids originate from CD24+ cells and represent an advanced model for adult polycystic kidney disease},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Whole-genome sequencing of chronic lymphocytic leukemia
identifies subgroups with distinct biological and clinical features.
<em>NG</em>, <em>54</em>(11), 1675–1689. (<a
href="https://doi.org/10.1038/s41588-022-01211-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The value of genome-wide over targeted driver analyses for predicting clinical outcomes of cancer patients is debated. Here, we report the whole-genome sequencing of 485 chronic lymphocytic leukemia patients enrolled in clinical trials as part of the United Kingdom’s 100,000 Genomes Project. We identify an extended catalog of recurrent coding and noncoding genetic mutations that represents a source for future studies and provide the most complete high-resolution map of structural variants, copy number changes and global genome features including telomere length, mutational signatures and genomic complexity. We demonstrate the relationship of these features with clinical outcome and show that integration of 186 distinct recurrent genomic alterations defines five genomic subgroups that associate with response to therapy, refining conventional outcome prediction. While requiring independent validation, our findings highlight the potential of whole-genome sequencing to inform future risk stratification in chronic lymphocytic leukemia. Whole-genome sequencing of chronic lymphocytic leukemia from 485 patients identifies putative coding and noncoding drivers of disease. Genomically defined subgroups show distinct clinical and biological characteristics.},
  archive      = {J_NG},
  author       = {Robbe, Pauline and Ridout, Kate E. and Vavoulis, Dimitrios V. and Dréau, Helene and Kinnersley, Ben and Denny, Nicholas and Chubb, Daniel and Appleby, Niamh and Cutts, Anthony and Cornish, Alex J. and Lopez-Pascua, Laura and Clifford, Ruth and Burns, Adam and Stamatopoulos, Basile and Cabes, Maite and Alsolami, Reem and Antoniou, Pavlos and Oates, Melanie and Cavalieri, Doriane and Gibson, Jane and Prabhu, Anika V. and Schwessinger, Ron and Jennings, Daisy and James, Terena and Maheswari, Uma and Duran-Ferrer, Martí and Carninci, Piero and Knight, Samantha J. L. and Månsson, Robert and Hughes, Jim and Davies, James and Ross, Mark and Bentley, David and Strefford, Jonathan C. and Devereux, Stephen and Pettitt, Andrew R. and Hillmen, Peter and Caulfield, Mark J. and Houlston, Richard S. and Martín-Subero, José I. and Schuh, Anna},
  doi          = {10.1038/s41588-022-01211-y},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1675-1689},
  shortjournal = {Nat. Genet.},
  title        = {Whole-genome sequencing of chronic lymphocytic leukemia identifies subgroups with distinct biological and clinical features},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Molecular map of chronic lymphocytic leukemia and its impact
on outcome. <em>NG</em>, <em>54</em>(11), 1664–1674. (<a
href="https://doi.org/10.1038/s41588-022-01140-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent advances in cancer characterization have consistently revealed marked heterogeneity, impeding the completion of integrated molecular and clinical maps for each malignancy. Here, we focus on chronic lymphocytic leukemia (CLL), a B cell neoplasm with variable natural history that is conventionally categorized into two subtypes distinguished by extent of somatic mutations in the heavy-chain variable region of immunoglobulin genes (IGHV). To build the ‘CLL map,’ we integrated genomic, transcriptomic and epigenomic data from 1,148 patients. We identified 202 candidate genetic drivers of CLL (109 new) and refined the characterization of IGHV subtypes, which revealed distinct genomic landscapes and leukemogenic trajectories. Discovery of new gene expression subtypes further subcategorized this neoplasm and proved to be independent prognostic factors. Clinical outcomes were associated with a combination of genetic, epigenetic and gene expression features, further advancing our prognostic paradigm. Overall, this work reveals fresh insights into CLL oncogenesis and prognostication. A genomic, transcriptomic and epigenomic analysis of chronic lymphocytic leukemia identifies genetic drivers and molecular subtypes associated with clinical outcomes.},
  archive      = {J_NG},
  author       = {Knisbacher, Binyamin A. and Lin, Ziao and Hahn, Cynthia K. and Nadeu, Ferran and Duran-Ferrer, Martí and Stevenson, Kristen E. and Tausch, Eugen and Delgado, Julio and Barbera-Mourelle, Alex and Taylor-Weiner, Amaro and Bousquets-Muñoz, Pablo and Diaz-Navarro, Ander and Dunford, Andrew and Anand, Shankara and Kretzmer, Helene and Gutierrez-Abril, Jesus and López-Tamargo, Sara and Fernandes, Stacey M. and Sun, Clare and Sivina, Mariela and Rassenti, Laura Z. and Schneider, Christof and Li, Shuqiang and Parida, Laxmi and Meissner, Alexander and Aguet, François and Burger, Jan A. and Wiestner, Adrian and Kipps, Thomas J. and Brown, Jennifer R. and Hallek, Michael and Stewart, Chip and Neuberg, Donna S. and Martín-Subero, José I. and Puente, Xose S. and Stilgenbauer, Stephan and Wu, Catherine J. and Campo, Elias and Getz, Gad},
  doi          = {10.1038/s41588-022-01140-w},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1664-1674},
  shortjournal = {Nat. Genet.},
  title        = {Molecular map of chronic lymphocytic leukemia and its impact on outcome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multiomics study of nonalcoholic fatty liver disease.
<em>NG</em>, <em>54</em>(11), 1652–1663. (<a
href="https://doi.org/10.1038/s41588-022-01199-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nonalcoholic fatty liver (NAFL) and its sequelae are growing health problems. We performed a genome-wide association study of NAFL, cirrhosis and hepatocellular carcinoma, and integrated the findings with expression and proteomic data. For NAFL, we utilized 9,491 clinical cases and proton density fat fraction extracted from 36,116 liver magnetic resonance images. We identified 18 sequence variants associated with NAFL and 4 with cirrhosis, and found rare, protective, predicted loss-of-function variants in MTARC1 and GPAM, underscoring them as potential drug targets. We leveraged messenger RNA expression, splicing and predicted coding effects to identify 16 putative causal genes, of which many are implicated in lipid metabolism. We analyzed levels of 4,907 plasma proteins in 35,559 Icelanders and 1,459 proteins in 47,151 UK Biobank participants, identifying multiple proteins involved in disease pathogenesis. We show that proteomics can discriminate between NAFL and cirrhosis. The present study provides insights into the development of noninvasive evaluation of NAFL and new therapeutic options. Genome-wide analyses identify loci associated with nonalcoholic fatty liver disease, including rare, protective loss-of-function variants in MTARC1 and GPAM. Plasma proteomic analyses provide insight into proteins involved in disease pathogenesis.},
  archive      = {J_NG},
  author       = {Sveinbjornsson, Gardar and Ulfarsson, Magnus O. and Thorolfsdottir, Rosa B. and Jonsson, Benedikt A. and Einarsson, Eythor and Gunnlaugsson, Gylfi and Rognvaldsson, Solvi and Arnar, David O. and Baldvinsson, Magnus and Bjarnason, Ragnar G. and Eiriksdottir, Thjodbjorg and Erikstrup, Christian and Ferkingstad, Egil and Halldorsson, Gisli H. and Helgason, Hannes and Helgadottir, Anna and Hindhede, Lotte and Hjorleifsson, Grimur and Jones, David and Knowlton, Kirk U. and Lund, Sigrun H. and Melsted, Pall and Norland, Kristjan and Olafsson, Isleifur and Olafsson, Sigurdur and Oskarsson, Gudjon R. and Ostrowski, Sisse Rye and Pedersen, Ole Birger and Snaebjarnarson, Auðunn S. and Sigurdsson, Emil and Steinthorsdottir, Valgerdur and Schwinn, Michael and Thorgeirsson, Gudmundur and Thorleifsson, Gudmar and Jonsdottir, Ingileif and Bundgaard, Henning and Nadauld, Lincoln and Bjornsson, Einar S. and Rulifson, Ingrid C. and Rafnar, Thorunn and Norddahl, Gudmundur L. and Thorsteinsdottir, Unnur and Sulem, Patrick and Gudbjartsson, Daniel F. and Holm, Hilma and Stefansson, Kari},
  doi          = {10.1038/s41588-022-01199-5},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1652-1663},
  shortjournal = {Nat. Genet.},
  title        = {Multiomics study of nonalcoholic fatty liver disease},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multi-ancestry genome-wide association analyses identify
novel genetic mechanisms in rheumatoid arthritis. <em>NG</em>,
<em>54</em>(11), 1640–1651. (<a
href="https://doi.org/10.1038/s41588-022-01213-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rheumatoid arthritis (RA) is a highly heritable complex disease with unknown etiology. Multi-ancestry genetic research of RA promises to improve power to detect genetic signals, fine-mapping resolution and performances of polygenic risk scores (PRS). Here, we present a large-scale genome-wide association study (GWAS) of RA, which includes 276,020 samples from five ancestral groups. We conducted a multi-ancestry meta-analysis and identified 124 loci (P &amp;lt; 5 × 10−8), of which 34 are novel. Candidate genes at the novel loci suggest essential roles of the immune system (for example, TNIP2 and TNFRSF11A) and joint tissues (for example, WISP1) in RA etiology. Multi-ancestry fine-mapping identified putatively causal variants with biological insights (for example, LEF1). Moreover, PRS based on multi-ancestry GWAS outperformed PRS based on single-ancestry GWAS and had comparable performance between populations of European and East Asian ancestries. Our study provides several insights into the etiology of RA and improves the genetic predictability of RA. Multi-ancestry genome-wide association analyses identify 124 risk loci for rheumatoid arthritis, of which 34 are novel. A polygenic risk score based on multi-ancestry data showed comparable performance between populations of European and East Asian ancestries.},
  archive      = {J_NG},
  author       = {Ishigaki, Kazuyoshi and Sakaue, Saori and Terao, Chikashi and Luo, Yang and Sonehara, Kyuto and Yamaguchi, Kensuke and Amariuta, Tiffany and Too, Chun Lai and Laufer, Vincent A. and Scott, Ian C. and Viatte, Sebastien and Takahashi, Meiko and Ohmura, Koichiro and Murasawa, Akira and Hashimoto, Motomu and Ito, Hiromu and Hammoudeh, Mohammed and Emadi, Samar Al and Masri, Basel K. and Halabi, Hussein and Badsha, Humeira and Uthman, Imad W. and Wu, Xin and Lin, Li and Li, Ting and Plant, Darren and Barton, Anne and Orozco, Gisela and Verstappen, Suzanne M. M. and Bowes, John and MacGregor, Alexander J. and Honda, Suguru and Koido, Masaru and Tomizuka, Kohei and Kamatani, Yoichiro and Tanaka, Hiroaki and Tanaka, Eiichi and Suzuki, Akari and Maeda, Yuichi and Yamamoto, Kenichi and Miyawaki, Satoru and Xie, Gang and Zhang, Jinyi and Amos, Christopher I. and Keystone, Edward and Wolbink, Gertjan and van der Horst-Bruinsma, Irene and Cui, Jing and Liao, Katherine P. and Carroll, Robert J. and Lee, Hye-Soon and Bang, So-Young and Siminovitch, Katherine A. and de Vries, Niek and Alfredsson, Lars and Rantapää-Dahlqvist, Solbritt and Karlson, Elizabeth W. and Bae, Sang-Cheol and Kimberly, Robert P. and Edberg, Jeffrey C. and Mariette, Xavier and Huizinga, Tom and Dieudé, Philippe and Schneider, Matthias and Kerick, Martin and Denny, Joshua C. and Matsuda, Koichi and Matsuo, Keitaro and Mimori, Tsuneyo and Matsuda, Fumihiko and Fujio, Keishi and Tanaka, Yoshiya and Kumanogoh, Atsushi and Traylor, Matthew and Lewis, Cathryn M. and Eyre, Stephen and Xu, Huji and Saxena, Richa and Arayssi, Thurayya and Kochi, Yuta and Ikari, Katsunori and Harigai, Masayoshi and Gregersen, Peter K. and Yamamoto, Kazuhiko and Louis Bridges, S. and Padyukov, Leonid and Martin, Javier and Klareskog, Lars and Okada, Yukinori and Raychaudhuri, Soumya},
  doi          = {10.1038/s41588-022-01213-w},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1640-1651},
  shortjournal = {Nat. Genet.},
  title        = {Multi-ancestry genome-wide association analyses identify novel genetic mechanisms in rheumatoid arthritis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Statistical and functional convergence of common and rare
genetic influences on autism at chromosome 16p. <em>NG</em>,
<em>54</em>(11), 1630–1639. (<a
href="https://doi.org/10.1038/s41588-022-01203-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The canonical paradigm for converting genetic association to mechanism involves iteratively mapping individual associations to the proximal genes through which they act. In contrast, in the present study we demonstrate the feasibility of extracting biological insights from a very large region of the genome and leverage this strategy to study the genetic influences on autism. Using a new statistical approach, we identified the 33-Mb p-arm of chromosome 16 (16p) as harboring the greatest excess of autism’s common polygenic influences. The region also includes the mechanistically cryptic and autism-associated 16p11.2 copy number variant. Analysis of RNA-sequencing data revealed that both the common polygenic influences within 16p and the 16p11.2 deletion were associated with decreased average gene expression across 16p. The transcriptional effects of the rare deletion and diffuse common variation were correlated at the level of individual genes and analysis of Hi-C data revealed patterns of chromatin contact that may explain this transcriptional convergence. These results reflect a new approach for extracting biological insight from genetic association data and suggest convergence of common and rare genetic influences on autism at 16p. Common polygenic variation at chromosome 16p and rare recurrent deletions of 16p11.2 influencing autism risk are associated with reduced expression of genes throughout the 16p region, suggesting functional convergence of rare and common variant effects.},
  archive      = {J_NG},
  author       = {Weiner, Daniel J. and Ling, Emi and Erdin, Serkan and Tai, Derek J. C. and Yadav, Rachita and Grove, Jakob and Fu, Jack M. and Nadig, Ajay and Carey, Caitlin E. and Baya, Nikolas and Bybjerg-Grauholm, Jonas and Berretta, Sabina and Macosko, Evan Z. and Sebat, Jonathan and O’Connor, Luke J. and Hougaard, David M. and Børglum, Anders D. and Talkowski, Michael E. and McCarroll, Steven A. and Robinson, Elise B.},
  doi          = {10.1038/s41588-022-01203-y},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1630-1639},
  shortjournal = {Nat. Genet.},
  title        = {Statistical and functional convergence of common and rare genetic influences on autism at chromosome 16p},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Discovery of 42 genome-wide significant loci associated with
dyslexia. <em>NG</em>, <em>54</em>(11), 1621–1629. (<a
href="https://doi.org/10.1038/s41588-022-01192-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reading and writing are crucial life skills but roughly one in ten children are affected by dyslexia, which can persist into adulthood. Family studies of dyslexia suggest heritability up to 70%, yet few convincing genetic markers have been found. Here we performed a genome-wide association study of 51,800 adults self-reporting a dyslexia diagnosis and 1,087,070 controls and identified 42 independent genome-wide significant loci: 15 in genes linked to cognitive ability/educational attainment, and 27 new and potentially more specific to dyslexia. We validated 23 loci (13 new) in independent cohorts of Chinese and European ancestry. Genetic etiology of dyslexia was similar between sexes, and genetic covariance with many traits was found, including ambidexterity, but not neuroanatomical measures of language-related circuitry. Dyslexia polygenic scores explained up to 6% of variance in reading traits, and might in future contribute to earlier identification and remediation of dyslexia. Genome-wide analysis of self-reported dyslexia identifies 42 associated loci, including 27 not previously associated with cognitive traits. Dyslexia shows genetic correlation with ambidexterity but not neuroanatomical measures of language-related circuitry.},
  archive      = {J_NG},
  author       = {Doust, Catherine and Fontanillas, Pierre and Eising, Else and Gordon, Scott D. and Wang, Zhengjun and Alagöz, Gökberk and Molz, Barbara and Pourcain, Beate St and Francks, Clyde and Marioni, Riccardo E. and Zhao, Jingjing and Paracchini, Silvia and Talcott, Joel B. and Monaco, Anthony P. and Stein, John F. and Gruen, Jeffrey R. and Olson, Richard K. and Willcutt, Erik G. and DeFries, John C. and Pennington, Bruce F. and Smith, Shelley D. and Wright, Margaret J. and Martin, Nicholas G. and Auton, Adam and Bates, Timothy C. and Fisher, Simon E. and Luciano, Michelle},
  doi          = {10.1038/s41588-022-01192-y},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1621-1629},
  shortjournal = {Nat. Genet.},
  title        = {Discovery of 42 genome-wide significant loci associated with dyslexia},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Non-coding variants disrupting a tissue-specific regulatory
element in HK1 cause congenital hyperinsulinism. <em>NG</em>,
<em>54</em>(11), 1615–1620. (<a
href="https://doi.org/10.1038/s41588-022-01204-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression is tightly regulated, with many genes exhibiting cell-specific silencing when their protein product would disrupt normal cellular function1. This silencing is largely controlled by non-coding elements, and their disruption might cause human disease2. We performed gene-agnostic screening of the non-coding regions to discover new molecular causes of congenital hyperinsulinism. This identified 14 non-coding de novo variants affecting a 42-bp conserved region encompassed by a regulatory element in intron 2 of the hexokinase 1 gene (HK1). HK1 is widely expressed across all tissues except in the liver and pancreatic beta cells and is thus termed a ‘disallowed gene’ in these specific tissues. We demonstrated that the variants result in a loss of repression of HK1 in pancreatic beta cells, thereby causing insulin secretion and congenital hyperinsulinism. Using epigenomic data accessed from public repositories, we demonstrated that these variants reside within a regulatory region that we determine to be critical for cell-specific silencing. Importantly, this has revealed a disease mechanism for non-coding variants that cause inappropriate expression of a disallowed gene. De novo variants altering a conserved region in an intron of HK1 cause congenital hyperinsulinism by perturbing the activity of a putative cell type-specific regulatory element.},
  archive      = {J_NG},
  author       = {Wakeling, Matthew N. and Owens, Nick D. L. and Hopkinson, Jessica R. and Johnson, Matthew B. and Houghton, Jayne A. L. and Dastamani, Antonia and Flaxman, Christine S. and Wyatt, Rebecca C. and Hewat, Thomas I. and Hopkins, Jasmin J. and Laver, Thomas W. and van Heugten, Rachel and Weedon, Michael N. and De Franco, Elisa and Patel, Kashyap A. and Ellard, Sian and Morgan, Noel G. and Cheesman, Edmund and Banerjee, Indraneel and Hattersley, Andrew T. and Dunne, Mark J. and Richardson, Sarah J. and Flanagan, Sarah E.},
  doi          = {10.1038/s41588-022-01204-x},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1615-1620},
  shortjournal = {Nat. Genet.},
  title        = {Non-coding variants disrupting a tissue-specific regulatory element in HK1 cause congenital hyperinsulinism},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A combined polygenic score of 21,293 rare and 22 common
variants improves diabetes diagnosis based on hemoglobin A1C levels.
<em>NG</em>, <em>54</em>(11), 1609–1614. (<a
href="https://doi.org/10.1038/s41588-022-01200-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polygenic scores (PGSs) combine the effects of common genetic variants1,2 to predict risk or treatment strategies for complex diseases3–7. Adding rare variation to PGSs has largely unknown benefits and is methodically challenging. Here, we developed a method for constructing rare variant PGSs and applied it to calculate genetically modified hemoglobin A1C thresholds for type 2 diabetes (T2D) diagnosis7–10. The resultant rare variant PGS is highly polygenic (21,293 variants across 154 genes), depends on ultra-rare variants (72.7% observed in fewer than three people) and identifies significantly more undiagnosed T2D cases than expected by chance (odds ratio = 2.71; P = 1.51 × 10−6). A PGS combining common and rare variants is expected to identify 4.9 million misdiagnosed T2D cases in the United States—nearly 1.5-fold more than the common variant PGS alone. These results provide a method for constructing complex trait PGSs from rare variants and suggest that rare variants will augment common variants in precision medicine approaches for common disease. This study presents a method for constructing complex trait polygenic scores from rare variants and shows that a polygenic score combining common and rare variants improves the accuracy of diagnosis for type 2 diabetes based on hemoglobin A1C levels.},
  archive      = {J_NG},
  author       = {Dornbos, Peter and Koesterer, Ryan and Ruttenburg, Andrew and Nguyen, Trang and Cole, Joanne B. and Leong, Aaron and Meigs, James B. and Florez, Jose C. and Rotter, Jerome I. and Udler, Miriam S. and Flannick, Jason},
  doi          = {10.1038/s41588-022-01200-1},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1609-1614},
  shortjournal = {Nat. Genet.},
  title        = {A combined polygenic score of 21,293 rare and 22 common variants improves diabetes diagnosis based on hemoglobin A1C levels},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Addressing the benefits of inhibiting APOBEC3-dependent
mutagenesis in cancer. <em>NG</em>, <em>54</em>(11), 1599–1608. (<a
href="https://doi.org/10.1038/s41588-022-01196-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutational signatures associated with apolipoprotein B mRNA-editing enzyme catalytic polypeptide-like (APOBEC)3 cytosine deaminase activity have been found in over half of cancer types, including some therapy-resistant and metastatic tumors. Driver mutations can occur in APOBEC3-favored sequence contexts, suggesting that mutagenesis by APOBEC3 enzymes may drive cancer evolution. The APOBEC3-mediated signatures are often detected in subclonal branches of tumor phylogenies and are acquired in cancer cell lines over long periods of time, indicating that APOBEC3 mutagenesis can be ongoing in cancer. Collectively, these and other observations have led to the proposal that APOBEC3 mutagenesis represents a disease-modifying process that could be inhibited to limit tumor heterogeneity, metastasis and drug resistance. However, critical aspects of APOBEC3 biology in cancer and in healthy tissues have not been clearly defined, limiting well-grounded predictions regarding the benefits of inhibiting APOBEC3 mutagenesis in different settings in cancer. We discuss the relevant mechanistic gaps and strategies to address them to investigate whether inhibiting APOBEC3 mutagenesis may confer clinical benefits in cancer. This Perspective addresses next steps to investigate the predictions that inhibition of APOBEC3-mediated mutagenesis may limit tumor heterogeneity, metastasis and drug resistance in a broad range of cancer types by highlighting gaps in our understanding of APOBEC3 biology in cancer and in healthy tissues and strategies to address them.},
  archive      = {J_NG},
  author       = {Petljak, Mia and Green, Abby M. and Maciejowski, John and Weitzman, Matthew D.},
  doi          = {10.1038/s41588-022-01196-8},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1599-1608},
  shortjournal = {Nat. Genet.},
  title        = {Addressing the benefits of inhibiting APOBEC3-dependent mutagenesis in cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Disrupted β-cell-specific gene silencing causes congenital
hyperinsulinism. <em>NG</em>, <em>54</em>(11), 1597–1598. (<a
href="https://doi.org/10.1038/s41588-022-01206-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome sequencing and analysis of public epigenomic data enabled the identification of disease-causing variants in a non-coding regulatory region of hexokinase 1 (HK1) in individuals with congenital hyperinsulinism. These variants caused inappropriate HK1 expression within pancreatic β-cells, which led to increased insulin secretion and hypoglycemia.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01206-9},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1597-1598},
  shortjournal = {Nat. Genet.},
  title        = {Disrupted β-cell-specific gene silencing causes congenital hyperinsulinism},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deciphering single-cell transcriptional programs across
species. <em>NG</em>, <em>54</em>(11), 1595–1596. (<a
href="https://doi.org/10.1038/s41588-022-01198-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to predict gene-expression landscapes at single-cell resolution has long been a challenge in the field of genomics. We mapped whole-body single-cell transcriptomic landscapes of zebrafish, Drosophila, and earthworm using Microwell-seq. We propose the first sequence-based model, Nvwa, that can predict gene expression at single-cell resolution directly from genomic sequences.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01198-6},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1595-1596},
  shortjournal = {Nat. Genet.},
  title        = {Deciphering single-cell transcriptional programs across species},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Adult human kidney organoids: Cellular origin and disease
modeling. <em>NG</em>, <em>54</em>(11), 1593–1594. (<a
href="https://doi.org/10.1038/s41588-022-01207-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adult human kidney organoids or tubuloids are derived from an epithelial CD24+ subpopulation in the proximal nephron and can be utilized for advanced disease modeling of the most common hereditary kidney disease: autosomal dominant polycystic kidney disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01207-8},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1593-1594},
  shortjournal = {Nat. Genet.},
  title        = {Adult human kidney organoids: Cellular origin and disease modeling},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Profiling oncogenic extra-chromosomal DNA in cancer.
<em>NG</em>, <em>54</em>(11), 1591–1592. (<a
href="https://doi.org/10.1038/s41588-022-01193-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer cells frequently amplify oncogenes on DNA molecules outside of chromosomes — extrachromosomal DNA. A technique adapted for isolation of extrachromosomal DNA, termed CRISPR-CATCH, enables analyses of its genetic and epigenetic compositions, which provides insights into its origin, structural diversity and mechanism of oncogene activation in cancer.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01193-x},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1591-1592},
  shortjournal = {Nat. Genet.},
  title        = {Profiling oncogenic extra-chromosomal DNA in cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Adaptive variation in wild relatives of modern maize has
potential use in breeding. <em>NG</em>, <em>54</em>(11), 1589–1590. (<a
href="https://doi.org/10.1038/s41588-022-01186-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a high-resolution genomic variation map that greatly expands the sequence information for maize and its wild relatives in the Zea genus. Population genetics of Zea spp. provide a vast trove of adaptive alleles that are absent in maize, with the potential for accelerating future breeding by reintroducing genetic diversity.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01186-w},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1589-1590},
  shortjournal = {Nat. Genet.},
  title        = {Adaptive variation in wild relatives of modern maize has potential use in breeding},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Rare and common autism risk variants converge across 16p.
<em>NG</em>, <em>54</em>(11), 1587–1588. (<a
href="https://doi.org/10.1038/s41588-022-01219-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic risk factors for autism include both rare and common variants. A study shows that rare copy number variants and common variants across 16p that contribute to autism risk functionally converge to downregulate the expression of a large group of neuronally expressed genes in the 16p subtelomeric region.},
  archive      = {J_NG},
  author       = {Won, Hyejung and Huguet, Guillaume and Jacquemont, Sébastien},
  doi          = {10.1038/s41588-022-01219-4},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1587-1588},
  shortjournal = {Nat. Genet.},
  title        = {Rare and common autism risk variants converge across 16p},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chromatin oxygen sensing by histone h3 prolyl hydroxylation.
<em>NG</em>, <em>54</em>(11), 1585–1586. (<a
href="https://doi.org/10.1038/s41588-022-01201-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study identifies prolyl hydroxylation of histone H3 as a signal for the recruitment of KDM5A, altering H3K4me3 and gene expression. H3P16oh is independent of the HIF hypoxia-sensing pathway and provides a further layer of complexity to oxygen-sensitive chromatin modifications.},
  archive      = {J_NG},
  author       = {Nathan, James A.},
  doi          = {10.1038/s41588-022-01201-0},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1585-1586},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin oxygen sensing by histone h3 prolyl hydroxylation},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Genetics of healthy life years. <em>NG</em>,
<em>54</em>(11), 1584. (<a
href="https://doi.org/10.1038/s41588-022-01215-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-022-01215-8},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1584},
  shortjournal = {Nat. Genet.},
  title        = {Genetics of healthy life years},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). TP53-dependent genomic instability. <em>NG</em>,
<em>54</em>(11), 1584. (<a
href="https://doi.org/10.1038/s41588-022-01216-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-022-01216-7},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1584},
  shortjournal = {Nat. Genet.},
  title        = {TP53-dependent genomic instability},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Improved RNA-seq normalization. <em>NG</em>,
<em>54</em>(11), 1584. (<a
href="https://doi.org/10.1038/s41588-022-01217-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-022-01217-6},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1584},
  shortjournal = {Nat. Genet.},
  title        = {Improved RNA-seq normalization},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Somatic recombination of repetitive elements. <em>NG</em>,
<em>54</em>(11), 1584. (<a
href="https://doi.org/10.1038/s41588-022-01218-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-022-01218-5},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1584},
  shortjournal = {Nat. Genet.},
  title        = {Somatic recombination of repetitive elements},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The dynseq browser track shows context-specific features at
nucleotide resolution. <em>NG</em>, <em>54</em>(11), 1581–1583. (<a
href="https://doi.org/10.1038/s41588-022-01194-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-throughput experimental platforms have revolutionized the ability to profile biochemical and functional properties of biological sequences such as DNA, RNA and proteins. By collating several data modalities with customizable tracks rendered using intuitive visualizations, genome browsers enable an interactive and interpretable exploration of diverse types of genome profiling experiments and derived annotations. However, existing genome browser tracks are not well suited for intuitive visualization of high-resolution DNA sequence features such as transcription factor motifs. Typically, motif instances in regulatory DNA sequences are visualized as BED-based annotation tracks, which highlight the genomic coordinates of the motif instances but do not expose their specific sequences. Instead, a genome sequence track needs to be cross-referenced with the BED track to identify sequences of motif hits. Even so, quantitative information about the motif instances such as affinity or conservation as well as differences in base resolution from the consensus motif are not immediately apparent. This makes interpretation slow and challenging. This problem is compounded when analyzing several cellular states and/or molecular readouts (such as ATAC-seq and ChIP–seq) simultaneously, as coordinates of enriched regions (peaks) and the set of active transcription factor motifs vary across cell states.},
  archive      = {J_NG},
  author       = {Nair, Surag and Barrett, Arjun and Li, Daofeng and Raney, Brian J. and Lee, Brian T. and Kerpedjiev, Peter and Ramalingam, Vivekanandan and Pampari, Anusri and Lekschas, Fritz and Wang, Ting and Haeussler, Maximilian and Kundaje, Anshul},
  doi          = {10.1038/s41588-022-01194-w},
  journal      = {Nature Genetics},
  month        = {11},
  number       = {11},
  pages        = {1581-1583},
  shortjournal = {Nat. Genet.},
  title        = {The dynseq browser track shows context-specific features at nucleotide resolution},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Polygenic enrichment distinguishes disease associations of
individual cells in single-cell RNA-seq data. <em>NG</em>,
<em>54</em>(10), 1572–1580. (<a
href="https://doi.org/10.1038/s41588-022-01167-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell RNA sequencing (scRNA-seq) provides unique insights into the pathology and cellular origin of disease. We introduce single-cell disease relevance score (scDRS), an approach that links scRNA-seq with polygenic disease risk at single-cell resolution, independent of annotated cell types. scDRS identifies cells exhibiting excess expression across disease-associated genes implicated by genome-wide association studies (GWASs). We applied scDRS to 74 diseases/traits and 1.3 million single-cell gene-expression profiles across 31 tissues/organs. Cell-type-level results broadly recapitulated known cell-type–disease associations. Individual-cell-level results identified subpopulations of disease-associated cells not captured by existing cell-type labels, including T cell subpopulations associated with inflammatory bowel disease, partially characterized by their effector-like states; neuron subpopulations associated with schizophrenia, partially characterized by their spatial locations; and hepatocyte subpopulations associated with triglyceride levels, partially characterized by their higher ploidy levels. Genes whose expression was correlated with the scDRS score across cells (reflecting coexpression with GWAS disease-associated genes) were strongly enriched for gold-standard drug target and Mendelian disease genes. scDRS associates individual cells in scRNA-seq with disease by scoring single-cell transcriptomes using GWAS gene signatures. Applied to 74 GWAS and 1.3 million single-cell profiles, scDRS identifies specific cellular subpopulations associated with these diseases.},
  archive      = {J_NG},
  author       = {Zhang, Martin Jinye and Hou, Kangcheng and Dey, Kushal K. and Sakaue, Saori and Jagadeesh, Karthik A. and Weinand, Kathryn and Taychameekiatchai, Aris and Rao, Poorvi and Pisco, Angela Oliveira and Zou, James and Wang, Bruce and Gandal, Michael and Raychaudhuri, Soumya and Pasaniuc, Bogdan and Price, Alkes L.},
  doi          = {10.1038/s41588-022-01167-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1572-1580},
  shortjournal = {Nat. Genet.},
  title        = {Polygenic enrichment distinguishes disease associations of individual cells in single-cell RNA-seq data},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell genome sequencing of human neurons identifies
somatic point mutation and indel enrichment in regulatory elements.
<em>NG</em>, <em>54</em>(10), 1564–1571. (<a
href="https://doi.org/10.1038/s41588-022-01180-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate somatic mutation detection from single-cell DNA sequencing is challenging due to amplification-related artifacts. To reduce this artifact burden, an improved amplification technique, primary template-directed amplification (PTA), was recently introduced. We analyzed whole-genome sequencing data from 52 PTA-amplified single neurons using SCAN2, a new genotyper we developed to leverage mutation signatures and allele balance in identifying somatic single-nucleotide variants (SNVs) and small insertions and deletions (indels) in PTA data. Our analysis confirms an increase in nonclonal somatic mutation in single neurons with age, but revises the estimated rate of this accumulation to 16 SNVs per year. We also identify artifacts in other amplification methods. Most importantly, we show that somatic indels increase by at least three per year per neuron and are enriched in functional regions of the genome such as enhancers and promoters. Our data suggest that indels in gene-regulatory elements have a considerable effect on genome integrity in human neurons. Single-cell DNA sequencing data are generated from human neurons using primary template-directed amplification and analyzed using SCAN2, an improved genotyping tool. Indels are enriched in neuronal regulatory elements and may be deleterious.},
  archive      = {J_NG},
  author       = {Luquette, Lovelace J. and Miller, Michael B. and Zhou, Zinan and Bohrson, Craig L. and Zhao, Yifan and Jin, Hu and Gulhan, Doga and Ganz, Javier and Bizzotto, Sara and Kirkham, Samantha and Hochepied, Tino and Libert, Claude and Galor, Alon and Kim, Junho and Lodato, Michael A. and Garaycoechea, Juan I. and Gawad, Charles and West, Jay and Walsh, Christopher A. and Park, Peter J.},
  doi          = {10.1038/s41588-022-01180-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1564-1571},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell genome sequencing of human neurons identifies somatic point mutation and indel enrichment in regulatory elements},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Improved pea reference genome and pan-genome highlight
genomic features and evolutionary characteristics. <em>NG</em>,
<em>54</em>(10), 1553–1563. (<a
href="https://doi.org/10.1038/s41588-022-01172-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complete and accurate reference genomes and annotations provide fundamental resources for functional genomics and crop breeding. Here we report a de novo assembly and annotation of a pea cultivar ZW6 with contig N50 of 8.98 Mb, which features a 243-fold increase in contig length and evident improvements in the continuity and quality of sequence in complex repeat regions compared with the existing one. Genome diversity of 118 cultivated and wild pea demonstrated that Pisum abyssinicum is a separate species different from P. fulvum and P. sativum within Pisum. Quantitative trait locus analyses uncovered two known Mendel’s genes related to stem length (Le/le) and seed shape (R/r) as well as some candidate genes for pod form studied by Mendel. A pan-genome of 116 pea accessions was constructed, and pan-genes preferred in P. abyssinicum and P. fulvum showed distinct functional enrichment, indicating the potential value of them as pea breeding resources in the future. A high-quality genome assembly of pea cultivar ZW6 and pan-genome analyses provide insights into pea genome evolution and domestication as well as genomic resources for pea improvement.},
  archive      = {J_NG},
  author       = {Yang, Tao and Liu, Rong and Luo, Yingfeng and Hu, Songnian and Wang, Dong and Wang, Chenyu and Pandey, Manish K. and Ge, Song and Xu, Quanle and Li, Nana and Li, Guan and Huang, Yuning and Saxena, Rachit K. and Ji, Yishan and Li, Mengwei and Yan, Xin and He, Yuhua and Liu, Yujiao and Wang, Xuejun and Xiang, Chao and Varshney, Rajeev K. and Ding, Hanfeng and Gao, Shenghan and Zong, Xuxiao},
  doi          = {10.1038/s41588-022-01172-2},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1553-1563},
  shortjournal = {Nat. Genet.},
  title        = {Improved pea reference genome and pan-genome highlight genomic features and evolutionary characteristics},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genomics-informed prebreeding unlocks the diversity in
genebanks for wheat improvement. <em>NG</em>, <em>54</em>(10),
1544–1552. (<a
href="https://doi.org/10.1038/s41588-022-01189-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The great efforts spent in the maintenance of past diversity in genebanks are rationalized by the potential role of plant genetic resources (PGR) in future crop improvement—a concept whose practical implementation has fallen short of expectations. Here, we implement a genomics-informed prebreeding strategy for wheat improvement that does not discriminate against nonadapted germplasm. We collect and analyze dense genetic profiles for a large winter wheat collection and evaluate grain yield and resistance to yellow rust (YR) in bespoke core sets. Breeders already profit from wild introgressions but PGR still offer useful, yet unused, diversity. Potential donors of resistance sources not yet deployed in breeding were detected, while the prebreeding contribution of PGR to yield was estimated through ‘Elite × PGR’ F1 crosses. Genomic prediction within and across genebanks identified the best parents to be used in crosses with elite cultivars whose advanced progenies can outyield current wheat varieties in multiple field trials. Implementation of a genomics-informed prebreeding strategy in a global winter wheat collection enhances the use of genebank accessions and uncovers the value of genetic resources for wheat improvement.},
  archive      = {J_NG},
  author       = {Schulthess, Albert W. and Kale, Sandip M. and Liu, Fang and Zhao, Yusheng and Philipp, Norman and Rembe, Maximilian and Jiang, Yong and Beukert, Ulrike and Serfling, Albrecht and Himmelbach, Axel and Fuchs, Jörg and Oppermann, Markus and Weise, Stephan and Boeven, Philipp H. G. and Schacht, Johannes and Longin, C. Friedrich H. and Kollers, Sonja and Pfeiffer, Nina and Korzun, Viktor and Lange, Matthias and Scholz, Uwe and Stein, Nils and Mascher, Martin and Reif, Jochen C.},
  doi          = {10.1038/s41588-022-01189-7},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1544-1552},
  shortjournal = {Nat. Genet.},
  title        = {Genomics-informed prebreeding unlocks the diversity in genebanks for wheat improvement},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Gain-of-function mutations in KCNK3 cause a developmental
disorder with sleep apnea. <em>NG</em>, <em>54</em>(10), 1534–1543. (<a
href="https://doi.org/10.1038/s41588-022-01185-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sleep apnea is a common disorder that represents a global public health burden. KCNK3 encodes TASK-1, a K+ channel implicated in the control of breathing, but its link with sleep apnea remains poorly understood. Here we describe a new developmental disorder with associated sleep apnea (developmental delay with sleep apnea, or DDSA) caused by rare de novo gain-of-function mutations in KCNK3. The mutations cluster around the ‘X-gate’, a gating motif that controls channel opening, and produce overactive channels that no longer respond to inhibition by G-protein-coupled receptor pathways. However, despite their defective X-gating, these mutant channels can still be inhibited by a range of known TASK channel inhibitors. These results not only highlight an important new role for TASK-1 K+ channels and their link with sleep apnea but also identify possible therapeutic strategies. Heterozygous de novo gain-of-function mutations in KCNK3, which encodes the two-pore-domain K+ channel TASK-1, cause a channelopathy characterized by developmental delay with sleep apnea.},
  archive      = {J_NG},
  author       = {Sörmann, Janina and Schewe, Marcus and Proks, Peter and Jouen-Tachoire, Thibault and Rao, Shanlin and Riel, Elena B. and Agre, Katherine E. and Begtrup, Amber and Dean, John and Descartes, Maria and Fischer, Jan and Gardham, Alice and Lahner, Carrie and Mark, Paul R. and Muppidi, Srikanth and Pichurin, Pavel N. and Porrmann, Joseph and Schallner, Jens and Smith, Kirstin and Straub, Volker and Vasudevan, Pradeep and Willaert, Rebecca and Carpenter, Elisabeth P. and Rödström, Karin E. J. and Hahn, Michael G. and Müller, Thomas and Baukrowitz, Thomas and Hurles, Matthew E. and Wright, Caroline F. and Tucker, Stephen J.},
  doi          = {10.1038/s41588-022-01185-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1534-1543},
  shortjournal = {Nat. Genet.},
  title        = {Gain-of-function mutations in KCNK3 cause a developmental disorder with sleep apnea},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The evolutionary dynamics of extrachromosomal DNA in human
cancers. <em>NG</em>, <em>54</em>(10), 1527–1533. (<a
href="https://doi.org/10.1038/s41588-022-01177-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Oncogene amplification on extrachromosomal DNA (ecDNA) is a common event, driving aggressive tumor growth, drug resistance and shorter survival. Currently, the impact of nonchromosomal oncogene inheritance—random identity by descent—is poorly understood. Also unclear is the impact of ecDNA on somatic variation and selection. Here integrating theoretical models of random segregation, unbiased image analysis, CRISPR-based ecDNA tagging with live-cell imaging and CRISPR-C, we demonstrate that random ecDNA inheritance results in extensive intratumoral ecDNA copy number heterogeneity and rapid adaptation to metabolic stress and targeted treatment. Observed ecDNAs benefit host cell survival or growth and can change within a single cell cycle. ecDNA inheritance can predict, a priori, some of the aggressive features of ecDNA-containing cancers. These properties are facilitated by the ability of ecDNA to rapidly adapt genomes in a way that is not possible through chromosomal oncogene amplification. These results show how the nonchromosomal random inheritance pattern of ecDNA contributes to poor outcomes for patients with cancer. Random segregation of extrachromosomal DNA contributes to intratumoral heterogeneity and facilitates the rapid adaptation of human tumor cells to anticancer drugs.},
  archive      = {J_NG},
  author       = {Lange, Joshua T. and Rose, John C. and Chen, Celine Y. and Pichugin, Yuriy and Xie, Liangqi and Tang, Jun and Hung, King L. and Yost, Kathryn E. and Shi, Quanming and Erb, Marcella L. and Rajkumar, Utkrisht and Wu, Sihan and Taschner-Mandl, Sabine and Bernkopf, Marie and Swanton, Charles and Liu, Zhe and Huang, Weini and Chang, Howard Y. and Bafna, Vineet and Henssen, Anton G. and Werner, Benjamin and Mischel, Paul S.},
  doi          = {10.1038/s41588-022-01177-x},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1527-1533},
  shortjournal = {Nat. Genet.},
  title        = {The evolutionary dynamics of extrachromosomal DNA in human cancers},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell multi-omics of human clonal hematopoiesis
reveals that DNMT3A r882 mutations perturb early progenitor states
through selective hypomethylation. <em>NG</em>, <em>54</em>(10),
1514–1526. (<a
href="https://doi.org/10.1038/s41588-022-01179-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Somatic mutations in cancer genes have been detected in clonal expansions across healthy human tissue, including in clonal hematopoiesis. However, because mutated and wild-type cells are admixed, we have limited ability to link genotypes with phenotypes. To overcome this limitation, we leveraged multi-modality single-cell sequencing, capturing genotype, transcriptomes and methylomes in progenitors from individuals with DNMT3A R882 mutated clonal hematopoiesis. DNMT3A mutations result in myeloid over lymphoid bias, and an expansion of immature myeloid progenitors primed toward megakaryocytic–erythroid fate, with dysregulated expression of lineage and leukemia stem cell markers. Mutated DNMT3A leads to preferential hypomethylation of polycomb repressive complex 2 targets and a specific CpG flanking motif. Notably, the hypomethylation motif is enriched in binding motifs of key hematopoietic transcription factors, serving as a potential mechanistic link between DNMT3A mutations and aberrant transcriptional phenotypes. Thus, single-cell multi-omics paves the road to defining the downstream consequences of mutations that drive clonal mosaicism. Multi-modality single-cell sequencing determines genotype, transcriptome and methylome information in cells from individuals with DNMT3A R882 mutated clonal hematopoiesis, allowing for the comparison of mutant and wild-type cells from the same individuals.},
  archive      = {J_NG},
  author       = {Nam, Anna S. and Dusaj, Neville and Izzo, Franco and Murali, Rekha and Myers, Robert M. and Mouhieddine, Tarek H. and Sotelo, Jesus and Benbarche, Salima and Waarts, Michael and Gaiti, Federico and Tahri, Sabrin and Levine, Ross and Abdel-Wahab, Omar and Godley, Lucy A. and Chaligne, Ronan and Ghobrial, Irene and Landau, Dan A.},
  doi          = {10.1038/s41588-022-01179-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1514-1526},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell multi-omics of human clonal hematopoiesis reveals that DNMT3A r882 mutations perturb early progenitor states through selective hypomethylation},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Long-range phasing of dynamic, tissue-specific and
allele-specific regulatory elements. <em>NG</em>, <em>54</em>(10),
1504–1513. (<a
href="https://doi.org/10.1038/s41588-022-01188-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epigenomic maps identify gene regulatory elements by their chromatin state. However, prevailing short-read sequencing methods cannot effectively distinguish alleles, evaluate the interdependence of elements in a locus or capture single-molecule dynamics. Here, we apply targeted nanopore sequencing to profile chromatin accessibility and DNA methylation on contiguous ~100-kb DNA molecules that span loci relevant to development, immunity and imprinting. We detect promoters, enhancers, insulators and transcription factor footprints on single molecules based on exogenous GpC methylation. We infer relationships among dynamic elements within immune loci, and order successive remodeling events during T cell stimulation. Finally, we phase primary sequence and regulatory elements across the H19/IGF2 locus, uncovering primate-specific features. These include a segmental duplication that stabilizes the imprinting control region and a noncanonical enhancer that drives biallelic IGF2 expression in specific contexts. Our study advances emerging strategies for phasing gene regulatory landscapes and reveals a mechanism that overrides IGF2 imprinting in human cells. Nanopore sequencing is used to profile chromatin accessibility and DNA methylation on DNA molecules over 100 kb. Phasing analysis at the H19/IGF2 locus identifies a primate-specific enhancer driving biallelic IGF2 expression in specific cellular contexts.},
  archive      = {J_NG},
  author       = {Battaglia, Sofia and Dong, Kevin and Wu, Jingyi and Chen, Zeyu and Najm, Fadi J. and Zhang, Yuanyuan and Moore, Molly M. and Hecht, Vivian and Shoresh, Noam and Bernstein, Bradley E.},
  doi          = {10.1038/s41588-022-01188-8},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1504-1513},
  shortjournal = {Nat. Genet.},
  title        = {Long-range phasing of dynamic, tissue-specific and allele-specific regulatory elements},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Population-level variation in enhancer expression identifies
disease mechanisms in the human brain. <em>NG</em>, <em>54</em>(10),
1493–1503. (<a
href="https://doi.org/10.1038/s41588-022-01170-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identification of risk variants for neuropsychiatric diseases within enhancers underscores the importance of understanding population-level variation in enhancer function in the human brain. Besides regulating tissue-specific and cell-type-specific transcription of target genes, enhancers themselves can be transcribed. By jointly analyzing large-scale cell-type-specific transcriptome and regulome data, we cataloged 30,795 neuronal and 23,265 non-neuronal candidate transcribed enhancers. Examination of the transcriptome in 1,382 brain samples identified robust expression of transcribed enhancers. We explored gene–enhancer coordination and found that enhancer-linked genes are strongly implicated in neuropsychiatric disease. We identified expression quantitative trait loci (eQTLs) for both genes and enhancers and found that enhancer eQTLs mediate a substantial fraction of neuropsychiatric trait heritability. Inclusion of enhancer eQTLs in transcriptome-wide association studies enhanced functional interpretation of disease loci. Overall, our study characterizes the gene–enhancer regulome and genetic mechanisms in the human cortex in both healthy and diseased states. Analyses of population-level variation in gene and enhancer expression in the human brain characterize the gene–enhancer regulome and the regulatory mechanisms of transcribed enhancers in neuropsychiatric diseases.},
  archive      = {J_NG},
  author       = {Dong, Pengfei and Hoffman, Gabriel E. and Apontes, Pasha and Bendl, Jaroslav and Rahman, Samir and Fernando, Michael B. and Zeng, Biao and Vicari, James M. and Zhang, Wen and Girdhar, Kiran and Townsley, Kayla G. and Misir, Ruth and Brennand, Kristen J. and Haroutunian, Vahram and Voloudakis, Georgios and Fullard, John F. and Roussos, Panos},
  doi          = {10.1038/s41588-022-01170-4},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1493-1503},
  shortjournal = {Nat. Genet.},
  title        = {Population-level variation in enhancer expression identifies disease mechanisms in the human brain},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identifying disease-critical cell types and cellular
processes by integrating single-cell RNA-sequencing and human genetics.
<em>NG</em>, <em>54</em>(10), 1479–1492. (<a
href="https://doi.org/10.1038/s41588-022-01187-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies provide a powerful means of identifying loci and genes contributing to disease, but in many cases, the related cell types/states through which genes confer disease risk remain unknown. Deciphering such relationships is important for identifying pathogenic processes and developing therapeutics. In the present study, we introduce sc-linker, a framework for integrating single-cell RNA-sequencing, epigenomic SNP-to-gene maps and genome-wide association study summary statistics to infer the underlying cell types and processes by which genetic variants influence disease. The inferred disease enrichments recapitulated known biology and highlighted notable cell–disease relationships, including γ-aminobutyric acid-ergic neurons in major depressive disorder, a disease-dependent M-cell program in ulcerative colitis and a disease-specific complement cascade process in multiple sclerosis. In autoimmune disease, both healthy and disease-dependent immune cell-type programs were associated, whereas only disease-dependent epithelial cell programs were prominent, suggesting a role in disease response rather than initiation. Our framework provides a powerful approach for identifying the cell types and cellular processes by which genetic variants influence disease. The sc-linker is an analysis framework that combines genome-wide association study summary statistics, epigenomics and single-cell transcriptomes to identify disease-critical cell types and cellular processes across tissues and states.},
  archive      = {J_NG},
  author       = {Jagadeesh, Karthik A. and Dey, Kushal K. and Montoro, Daniel T. and Mohan, Rahul and Gazal, Steven and Engreitz, Jesse M. and Xavier, Ramnik J. and Price, Alkes L. and Regev, Aviv},
  doi          = {10.1038/s41588-022-01187-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1479-1492},
  shortjournal = {Nat. Genet.},
  title        = {Identifying disease-critical cell types and cellular processes by integrating single-cell RNA-sequencing and human genetics},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identification of shared and differentiating genetic
architecture for autism spectrum disorder, attention-deficit
hyperactivity disorder and case subgroups. <em>NG</em>, <em>54</em>(10),
1470–1478. (<a
href="https://doi.org/10.1038/s41588-022-01171-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Attention-deficit hyperactivity disorder (ADHD) and autism spectrum disorder (ASD) are highly heritable neurodevelopmental conditions, with considerable overlap in their genetic etiology. We dissected their shared and distinct genetic etiology by cross-disorder analyses of large datasets. We identified seven loci shared by the disorders and five loci differentiating them. All five differentiating loci showed opposite allelic directions in the two disorders and significant associations with other traits, including educational attainment, neuroticism and regional brain volume. Integration with brain transcriptome data enabled us to identify and prioritize several significantly associated genes. The shared genomic fraction contributing to both disorders was strongly correlated with other psychiatric phenotypes, whereas the differentiating portion was correlated most strongly with cognitive traits. Additional analyses revealed that individuals diagnosed with both ASD and ADHD were double-loaded with genetic predispositions for both disorders and showed distinctive patterns of genetic association with other traits compared with the ASD-only and ADHD-only subgroups. These results provide insights into the biological foundation of the development of one or both conditions and of the factors driving psychopathology discriminatively toward either ADHD or ASD. Cross-disorder genetic association analyses identify five loci differentiating attention-deficit hyperactivity disorder (ADHD) and autism spectrum disorder (ASD). Individuals diagnosed with both ADHD and ASD are double-loaded with genetic risk for both disorders.},
  archive      = {J_NG},
  author       = {Mattheisen, Manuel and Grove, Jakob and Als, Thomas D. and Martin, Joanna and Voloudakis, Georgios and Meier, Sandra and Demontis, Ditte and Bendl, Jaroslav and Walters, Raymond and Carey, Caitlin E. and Rosengren, Anders and Strom, Nora I. and Hauberg, Mads Engel and Zeng, Biao and Hoffman, Gabriel and Zhang, Wen and Bybjerg-Grauholm, Jonas and Bækvad-Hansen, Marie and Agerbo, Esben and Cormand, Bru and Nordentoft, Merete and Werge, Thomas and Mors, Ole and Hougaard, David M. and Buxbaum, Joseph D. and Faraone, Stephen V. and Franke, Barbara and Dalsgaard, Søren and Mortensen, Preben B. and Robinson, Elise B. and Roussos, Panos and Neale, Benjamin M. and Daly, Mark J. and Børglum, Anders D.},
  doi          = {10.1038/s41588-022-01171-3},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1470-1478},
  shortjournal = {Nat. Genet.},
  title        = {Identification of shared and differentiating genetic architecture for autism spectrum disorder, attention-deficit hyperactivity disorder and case subgroups},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). SAIGE-GENE+ improves the efficiency and accuracy of
set-based rare variant association tests. <em>NG</em>, <em>54</em>(10),
1466–1469. (<a
href="https://doi.org/10.1038/s41588-022-01178-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several biobanks, including UK Biobank (UKBB), are generating large-scale sequencing data. An existing method, SAIGE-GENE, performs well when testing variants with minor allele frequency (MAF) ≤ 1%, but inflation is observed in variance component set-based tests when restricting to variants with MAF ≤ 0.1% or 0.01%. Here, we propose SAIGE-GENE+ with greatly improved type I error control and computational efficiency to facilitate rare variant tests in large-scale data. We further show that incorporating multiple MAF cutoffs and functional annotations can improve power and thus uncover new gene–phenotype associations. In the analysis of UKBB whole exome sequencing data for 30 quantitative and 141 binary traits, SAIGE-GENE+ identified 551 gene–phenotype associations. SAIGE-GENE+ performs set-based rare variant association tests with improved type 1 error control and computational efficiency by collapsing ultra-rare variants and conducting multiple tests corresponding to different minor allele frequency cutoffs and annotations.},
  archive      = {J_NG},
  author       = {Zhou, Wei and Bi, Wenjian and Zhao, Zhangchen and Dey, Kushal K. and Jagadeesh, Karthik A. and Karczewski, Konrad J. and Daly, Mark J. and Neale, Benjamin M. and Lee, Seunggeun},
  doi          = {10.1038/s41588-022-01178-w},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1466-1469},
  shortjournal = {Nat. Genet.},
  title        = {SAIGE-GENE+ improves the efficiency and accuracy of set-based rare variant association tests},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ten challenges for clinical translation in psychiatric
genetics. <em>NG</em>, <em>54</em>(10), 1457–1465. (<a
href="https://doi.org/10.1038/s41588-022-01174-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies have identified hundreds of robust genetic associations underlying psychiatric disorders and provided important biological insights into disease onset and progression. There is optimism that genetic findings will pave the way to precision psychiatry by facilitating the development of more effective treatments and the identification of groups of patients that these treatments should be targeted toward. However, there are several challenges that must be addressed before genetic findings can be translated into the clinic. In this Perspective, we highlight ten challenges for the field of psychiatric genetics, focused on the robust and generalizable detection of genetic risk factors, improved definition and assessment of psychopathology and achieving better clinical indicators. We discuss recent advancements in the field that will improve the explanatory and predictive power of genetic data and ultimately contribute to improving the management and treatment of patients with a psychiatric disorder. This Perspective outlines ten challenges facing translation of psychiatric genetic findings into clinical practice.},
  archive      = {J_NG},
  author       = {Derks, Eske M. and Thorp, Jackson G. and Gerring, Zachary F.},
  doi          = {10.1038/s41588-022-01174-0},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1457-1465},
  shortjournal = {Nat. Genet.},
  title        = {Ten challenges for clinical translation in psychiatric genetics},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deciphering the evolutionary dynamics of extrachromosomal
DNA in human cancer. <em>NG</em>, <em>54</em>(10), 1455–1456. (<a
href="https://doi.org/10.1038/s41588-022-01183-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Oncogenes commonly amplify on circular extrachromosomal DNA (ecDNA) molecules in cancer. We show that ecDNA shapes each of the foundational principles of Darwinian evolution — random inheritance by descent, enhanced variation through random segregation, and selection — and thereby promotes rapid genome change, treatment resistance and poor outcomes for patients with cancer.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01183-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1455-1456},
  shortjournal = {Nat. Genet.},
  title        = {Deciphering the evolutionary dynamics of extrachromosomal DNA in human cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Somatic mosaicism rewires transcriptional and epigenetic
states in clonal hematopoiesis. <em>NG</em>, <em>54</em>(10), 1453–1454.
(<a href="https://doi.org/10.1038/s41588-022-01181-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multi-modal single-cell sequencing enables mapping of mutant and wild-type human hematopoietic stem and progenitor cells within the same person, to define cellular phenotypic and epigenetic perturbations associated with clonal hematopoiesis.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01181-1},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1453-1454},
  shortjournal = {Nat. Genet.},
  title        = {Somatic mosaicism rewires transcriptional and epigenetic states in clonal hematopoiesis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genomic insights into TASK-1 reveal functional roles in
sleep apnea. <em>NG</em>, <em>54</em>(10), 1451–1452. (<a
href="https://doi.org/10.1038/s41588-022-01195-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {KCNK3 mutations identified in sleep apnea probands affect TASK-1 X-gate function. These changes lead to an increase in potassium current and open probability, as well as impaired sensitivity to G-protein-coupled receptor inhibitors.},
  archive      = {J_NG},
  author       = {Simonson, Tatum S. and Moya, Esteban A. and Malhotra, Atul},
  doi          = {10.1038/s41588-022-01195-9},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1451-1452},
  shortjournal = {Nat. Genet.},
  title        = {Genomic insights into TASK-1 reveal functional roles in sleep apnea},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multi-omics on our multitudes. <em>NG</em>, <em>54</em>(10),
1449–1450. (<a
href="https://doi.org/10.1038/s41588-022-01175-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clonal expansion of DNMT3A-mutant hematopoietic stem cells is a risk factor for myeloid malignancies and other morbidities. A new study uses multi-modal single-cell genomics to characterize the myeloid differentiation bias of DNMT3A-mutated clones, and finds preferential hypomethylation of binding motifs for key transcriptional regulators.},
  archive      = {J_NG},
  author       = {Voit, Richard A. and Sankaran, Vijay G.},
  doi          = {10.1038/s41588-022-01175-z},
  journal      = {Nature Genetics},
  month        = {10},
  number       = {10},
  pages        = {1449-1450},
  shortjournal = {Nat. Genet.},
  title        = {Multi-omics on our multitudes},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: From mendel to quantitative genetics in
the genome era: The scientific legacy of w. G. hill. <em>NG</em>,
<em>54</em>(9), 1448. (<a
href="https://doi.org/10.1038/s41588-022-01160-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Charlesworth, Brian and Goddard, Michael E. and Meyer, Karin and Visscher, Peter M. and Weir, Bruce S. and Wray, Naomi R.},
  doi          = {10.1038/s41588-022-01160-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1448},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: from mendel to quantitative genetics in the genome era: the scientific legacy of w. g. hill},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Effect of host genetics on the gut
microbiome in 7,738 participants of the dutch microbiome project.
<em>NG</em>, <em>54</em>(9), 1448. (<a
href="https://doi.org/10.1038/s41588-022-01164-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Lopera-Maya, Esteban A. and Kurilshikov, Alexander and van der Graaf, Adriaan and Hu, Shixian and Andreu-Sánchez, Sergio and Chen, Lianmin and Vila, Arnau Vich and Gacesa, Ranko and Sinha, Trishla and Collij, Valerie and Klaassen, Marjiolein A. Y. and Bolte, Laura A. and Gois, Milla F. Brandao and Neerincx, Pieter B. T. and Swertz, Morris A. and Harmsen, Hermie J. M. and Wijmenga, Cisca and Fu, Jingyuan and Weersma, Rinse K. and Zhernakova, Alexandra and Sanna, Serena},
  doi          = {10.1038/s41588-022-01164-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1448},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Effect of host genetics on the gut microbiome in 7,738 participants of the dutch microbiome project},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Systematic identification of cell-fate
regulatory programs using a single-cell atlas of mouse development.
<em>NG</em>, <em>54</em>(9), 1448. (<a
href="https://doi.org/10.1038/s41588-022-01176-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fei, Lijiang and Chen, Haide and Ma, Lifeng and E, Weigao and Wang, Renying and Fang, Xing and Zhou, Ziming and Sun, Huiyu and Wang, Jingjing and Jiang, Mengmeng and Wang, Xinru and Yu, Chengxuan and Mei, Yuqing and Jia, Danmei and Zhang, Tingyue and Han, Xiaoping and Guo, Guoji},
  doi          = {10.1038/s41588-022-01176-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1448},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Systematic identification of cell-fate regulatory programs using a single-cell atlas of mouse development},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A multi-tissue atlas of regulatory variants in cattle.
<em>NG</em>, <em>54</em>(9), 1438–1447. (<a
href="https://doi.org/10.1038/s41588-022-01153-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterization of genetic regulatory variants acting on livestock gene expression is essential for interpreting the molecular mechanisms underlying traits of economic value and for increasing the rate of genetic gain through artificial selection. Here we build a Cattle Genotype–Tissue Expression atlas (CattleGTEx) as part of the pilot phase of the Farm animal GTEx (FarmGTEx) project for the research community based on 7,180 publicly available RNA-sequencing (RNA-seq) samples. We describe the transcriptomic landscape of more than 100 tissues/cell types and report hundreds of thousands of genetic associations with gene expression and alternative splicing for 23 distinct tissues. We evaluate the tissue-sharing patterns of these genetic regulatory effects, and functionally annotate them using multiomics data. Finally, we link gene expression in different tissues to 43 economically important traits using both transcriptome-wide association and colocalization analyses to decipher the molecular regulatory mechanisms underpinning such agronomic traits in cattle. The cattle Genotype–Tissue Expression atlas of expression and splicing QTLs is generated from 7,180 uniformly re-processed RNA-seq samples. Integration with GWAS identifies candidate genes and variants associated with economically important traits.},
  archive      = {J_NG},
  author       = {Liu, Shuli and Gao, Yahui and Canela-Xandri, Oriol and Wang, Sheng and Yu, Ying and Cai, Wentao and Li, Bingjie and Xiang, Ruidong and Chamberlain, Amanda J. and Pairo-Castineira, Erola and D’Mellow, Kenton and Rawlik, Konrad and Xia, Charley and Yao, Yuelin and Navarro, Pau and Rocha, Dominique and Li, Xiujin and Yan, Ze and Li, Congjun and Rosen, Benjamin D. and Van Tassell, Curtis P. and Vanraden, Paul M. and Zhang, Shengli and Ma, Li and Cole, John B. and Liu, George E. and Tenesa, Albert and Fang, Lingzhao},
  doi          = {10.1038/s41588-022-01153-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1438-1447},
  shortjournal = {Nat. Genet.},
  title        = {A multi-tissue atlas of regulatory variants in cattle},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). RNA m6A regulates transcription via DNA demethylation and
chromatin accessibility. <em>NG</em>, <em>54</em>(9), 1427–1437. (<a
href="https://doi.org/10.1038/s41588-022-01173-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptional regulation, which integrates chromatin accessibility, transcription factors and epigenetic modifications, is crucial for establishing and maintaining cell identity. The interplay between different epigenetic modifications and its contribution to transcriptional regulation remains elusive. Here, we show that METTL3-mediated RNA N6-methyladenosine (m6A) formation leads to DNA demethylation in nearby genomic loci in normal and cancer cells, which is mediated by the interaction between m6A reader FXR1 and DNA 5-methylcytosine dioxygenase TET1. Upon recognizing RNA m6A, FXR1 recruits TET1 to genomic loci to demethylate DNA, leading to reprogrammed chromatin accessibility and gene transcription. Therefore, we have characterized a regulatory mechanism of chromatin accessibility and gene transcription mediated by RNA m6A formation coupled with DNA demethylation, highlighting the importance of the crosstalk between RNA m6A and DNA modification in physiologic and pathogenic process. METTL3-dependent RNA N6-methyladenosine (m6A) deposition can lead to DNA demethylation. The m6A reader FXR1 recruits DNA 5-methylcytosine dioxygenase TET1 to chromatin, which is linked to chromatin accessibility and gene transcription.},
  archive      = {J_NG},
  author       = {Deng, Shuang and Zhang, Jialiang and Su, Jiachun and Zuo, Zhixiang and Zeng, Lingxing and Liu, Kaijing and Zheng, Yanfen and Huang, Xudong and Bai, Ruihong and Zhuang, Lisha and Ye, Ying and Li, Mei and Pan, Ling and Deng, Junge and Wu, Guandi and Li, Rui and Zhang, Shaoping and Wu, Chen and Lin, Dongxin and Chen, Jianjun and Zheng, Jian},
  doi          = {10.1038/s41588-022-01173-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1427-1437},
  shortjournal = {Nat. Genet.},
  title        = {RNA m6A regulates transcription via DNA demethylation and chromatin accessibility},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). HIC2 controls developmental hemoglobin switching by
repressing BCL11A transcription. <em>NG</em>, <em>54</em>(9), 1417–1426.
(<a href="https://doi.org/10.1038/s41588-022-01152-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The fetal-to-adult switch in hemoglobin production is a model of developmental gene control with relevance to the treatment of hemoglobinopathies. The expression of transcription factor BCL11A, which represses fetal β-type globin (HBG) genes in adult erythroid cells, is predominantly controlled at the transcriptional level but the underlying mechanism is unclear. We identify HIC2 as a repressor of BCL11A transcription. HIC2 and BCL11A are reciprocally expressed during development. Forced expression of HIC2 in adult erythroid cells inhibits BCL11A transcription and induces HBG expression. HIC2 binds to erythroid BCL11A enhancers to reduce chromatin accessibility and binding of transcription factor GATA1, diminishing enhancer activity and enhancer–promoter contacts. DNA-binding and crystallography studies reveal direct steric hindrance as one mechanism by which HIC2 inhibits GATA1 binding at a critical BCL11A enhancer. Conversely, loss of HIC2 in fetal erythroblasts increases enhancer accessibility, GATA1 binding and BCL11A transcription. HIC2 emerges as an evolutionarily conserved regulator of hemoglobin switching via developmental control of BCL11A. HIC2 regulates the fetal-to-adult hemoglobin switch. It inactivates an enhancer of the BCL11A gene, a fetal globin repressor, by reducing chromatin accessibility and displacing the transcription factor GATA1.},
  archive      = {J_NG},
  author       = {Huang, Peng and Peslak, Scott A. and Ren, Ren and Khandros, Eugene and Qin, Kunhua and Keller, Cheryl A. and Giardine, Belinda and Bell, Henry W. and Lan, Xianjiang and Sharma, Malini and Horton, John R. and Abdulmalik, Osheiza and Chou, Stella T. and Shi, Junwei and Crossley, Merlin and Hardison, Ross C. and Cheng, Xiaodong and Blobel, Gerd A.},
  doi          = {10.1038/s41588-022-01152-6},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1417-1426},
  shortjournal = {Nat. Genet.},
  title        = {HIC2 controls developmental hemoglobin switching by repressing BCL11A transcription},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Substantial somatic genomic variation and selection for BCOR
mutations in human induced pluripotent stem cells. <em>NG</em>,
<em>54</em>(9), 1406–1416. (<a
href="https://doi.org/10.1038/s41588-022-01147-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We explored human induced pluripotent stem cells (hiPSCs) derived from different tissues to gain insights into genomic integrity at single-nucleotide resolution. We used genome sequencing data from two large hiPSC repositories involving 696 hiPSCs and daughter subclones. We find ultraviolet light (UV)-related damage in ~72% of skin fibroblast-derived hiPSCs (F-hiPSCs), occasionally resulting in substantial mutagenesis (up to 15 mutations per megabase). We demonstrate remarkable genomic heterogeneity between independent F-hiPSC clones derived during the same round of reprogramming due to oligoclonal fibroblast populations. In contrast, blood-derived hiPSCs (B-hiPSCs) had fewer mutations and no UV damage but a high prevalence of acquired BCOR mutations (26.9% of lines). We reveal strong selection pressure for BCOR mutations in F-hiPSCs and B-hiPSCs and provide evidence that they arise in vitro. Directed differentiation of hiPSCs and RNA sequencing showed that BCOR mutations have functional consequences. Our work strongly suggests that detailed nucleotide-resolution characterization is essential before using hiPSCs. Sequencing of human induced pluripotent stem cell lines highlights pervasive mutagenesis, heterogeneity between clones derived from the same individual during a single reprogramming experiment and positive selection for acquired mutations in BCOR.},
  archive      = {J_NG},
  author       = {Rouhani, Foad J. and Zou, Xueqing and Danecek, Petr and Badja, Cherif and Amarante, Tauanne Dias and Koh, Gene and Wu, Qianxin and Memari, Yasin and Durbin, Richard and Martincorena, Inigo and Bassett, Andrew R. and Gaffney, Daniel and Nik-Zainal, Serena},
  doi          = {10.1038/s41588-022-01147-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1406-1416},
  shortjournal = {Nat. Genet.},
  title        = {Substantial somatic genomic variation and selection for BCOR mutations in human induced pluripotent stem cells},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Spatially restricted drivers and transitional cell
populations cooperate with the microenvironment in untreated and
chemo-resistant pancreatic cancer. <em>NG</em>, <em>54</em>(9),
1390–1405. (<a
href="https://doi.org/10.1038/s41588-022-01157-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pancreatic ductal adenocarcinoma is a lethal disease with limited treatment options and poor survival. We studied 83 spatial samples from 31 patients (11 treatment-naïve and 20 treated) using single-cell/nucleus RNA sequencing, bulk-proteogenomics, spatial transcriptomics and cellular imaging. Subpopulations of tumor cells exhibited signatures of proliferation, KRAS signaling, cell stress and epithelial-to-mesenchymal transition. Mapping mutations and copy number events distinguished tumor populations from normal and transitional cells, including acinar-to-ductal metaplasia and pancreatic intraepithelial neoplasia. Pathology-assisted deconvolution of spatial transcriptomic data identified tumor and transitional subpopulations with distinct histological features. We showed coordinated expression of TIGIT in exhausted and regulatory T cells and Nectin in tumor cells. Chemo-resistant samples contain a threefold enrichment of inflammatory cancer-associated fibroblasts that upregulate metallothioneins. Our study reveals a deeper understanding of the intricate substructure of pancreatic ductal adenocarcinoma tumors that could help improve therapy for patients with this disease. A multi-omic analysis of pancreatic cancer identifies spatially resolved, heterogeneous cell populations including transitional cell types. Analysis of primary samples identifies treatment-related changes in cross-talk between tumor and stromal cells.},
  archive      = {J_NG},
  author       = {Cui Zhou, Daniel and Jayasinghe, Reyka G. and Chen, Siqi and Herndon, John M. and Iglesia, Michael D. and Navale, Pooja and Wendl, Michael C. and Caravan, Wagma and Sato, Kazuhito and Storrs, Erik and Mo, Chia-Kuei and Liu, Jingxian and Southard-Smith, Austin N. and Wu, Yige and Naser Al Deen, Nataly and Baer, John M. and Fulton, Robert S. and Wyczalkowski, Matthew A. and Liu, Ruiyang and Fronick, Catrina C. and Fulton, Lucinda A. and Shinkle, Andrew and Thammavong, Lisa and Zhu, Houxiang and Sun, Hua and Wang, Liang-Bo and Li, Yize and Zuo, Chong and McMichael, Joshua F. and Davies, Sherri R. and Appelbaum, Elizabeth L. and Robbins, Keenan J. and Chasnoff, Sara E. and Yang, Xiaolu and Reeb, Ashley N. and Oh, Clara and Serasanambati, Mamatha and Lal, Preet and Varghese, Rajees and Mashl, Jay R. and Ponce, Jennifer and Terekhanova, Nadezhda V. and Yao, Lijun and Wang, Fang and Chen, Lijun and Schnaubelt, Michael and Lu, Rita Jui-Hsien and Schwarz, Julie K. and Puram, Sidharth V. and Kim, Albert H. and Song, Sheng-Kwei and Shoghi, Kooresh I. and Lau, Ken S. and Ju, Tao and Chen, Ken and Chatterjee, Deyali and Hawkins, William G. and Zhang, Hui and Achilefu, Samuel and Chheda, Milan G. and Oh, Stephen T. and Gillanders, William E. and Chen, Feng and DeNardo, David G. and Fields, Ryan C. and Ding, Li},
  doi          = {10.1038/s41588-022-01157-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1390-1405},
  shortjournal = {Nat. Genet.},
  title        = {Spatially restricted drivers and transitional cell populations cooperate with the microenvironment in untreated and chemo-resistant pancreatic cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The genomic landscape of pediatric acute lymphoblastic
leukemia. <em>NG</em>, <em>54</em>(9), 1376–1389. (<a
href="https://doi.org/10.1038/s41588-022-01159-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Acute lymphoblastic leukemia (ALL) is the most common childhood cancer. Here, using whole-genome, exome and transcriptome sequencing of 2,754 childhood patients with ALL, we find that, despite a generally low mutation burden, ALL cases harbor a median of four putative somatic driver alterations per sample, with 376 putative driver genes identified varying in prevalence across ALL subtypes. Most samples harbor at least one rare gene alteration, including 70 putative cancer driver genes associated with ubiquitination, SUMOylation, noncoding transcripts and other functions. In hyperdiploid B-ALL, chromosomal gains are acquired early and synchronously before ultraviolet-induced mutation. By contrast, ultraviolet-induced mutations precede chromosomal gains in B-ALL cases with intrachromosomal amplification of chromosome 21. We also demonstrate the prognostic significance of genetic alterations within subtypes. Intriguingly, DUX4- and KMT2A-rearranged subtypes separate into CEBPA/FLT3- or NFATC4-expressing subgroups with potential clinical implications. Together, these results deepen understanding of the ALL genomic landscape and associated outcomes. A genomic and transcriptomic analysis of 2,754 childhood acute lymphoblastic leukemias identifies 376 putative driver genes, and associations between disease subtypes and prognosis.},
  archive      = {J_NG},
  author       = {Brady, Samuel W. and Roberts, Kathryn G. and Gu, Zhaohui and Shi, Lei and Pounds, Stanley and Pei, Deqing and Cheng, Cheng and Dai, Yunfeng and Devidas, Meenakshi and Qu, Chunxu and Hill, Ashley N. and Payne-Turner, Debbie and Ma, Xiaotu and Iacobucci, Ilaria and Baviskar, Pradyuamna and Wei, Lei and Arunachalam, Sasi and Hagiwara, Kohei and Liu, Yanling and Flasch, Diane A. and Liu, Yu and Parker, Matthew and Chen, Xiaolong and Elsayed, Abdelrahman H. and Pathak, Omkar and Li, Yongjin and Fan, Yiping and Michael, J. Robert and Rusch, Michael and Wilkinson, Mark R. and Foy, Scott and Hedges, Dale J. and Newman, Scott and Zhou, Xin and Wang, Jian and Reilly, Colleen and Sioson, Edgar and Rice, Stephen V. and Pastor Loyola, Victor and Wu, Gang and Rampersaud, Evadnie and Reshmi, Shalini C. and Gastier-Foster, Julie and Guidry Auvil, Jaime M. and Gesuwan, Patee and Smith, Malcolm A. and Winick, Naomi and Carroll, Andrew J. and Heerema, Nyla A. and Harvey, Richard C. and Willman, Cheryl L. and Larsen, Eric and Raetz, Elizabeth A. and Borowitz, Michael J. and Wood, Brent L. and Carroll, William L. and Zweidler-McKay, Patrick A. and Rabin, Karen R. and Mattano, Leonard A. and Maloney, Kelly W. and Winter, Stuart S. and Burke, Michael J. and Salzer, Wanda and Dunsmore, Kimberly P. and Angiolillo, Anne L. and Crews, Kristine R. and Downing, James R. and Jeha, Sima and Pui, Ching-Hon and Evans, William E. and Yang, Jun J. and Relling, Mary V. and Gerhard, Daniela S. and Loh, Mignon L. and Hunger, Stephen P. and Zhang, Jinghui and Mullighan, Charles G.},
  doi          = {10.1038/s41588-022-01159-z},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1376-1389},
  shortjournal = {Nat. Genet.},
  title        = {The genomic landscape of pediatric acute lymphoblastic leukemia},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetic determinants of chromatin reveal prostate cancer
risk mediated by context-dependent gene regulation. <em>NG</em>,
<em>54</em>(9), 1364–1375. (<a
href="https://doi.org/10.1038/s41588-022-01168-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many genetic variants affect disease risk by altering context-dependent gene regulation. Such variants are difficult to study mechanistically using current methods that link genetic variation to steady-state gene expression levels, such as expression quantitative trait loci (eQTLs). To address this challenge, we developed the cistrome-wide association study (CWAS), a framework for identifying genotypic and allele-specific effects on chromatin that are also associated with disease. In prostate cancer, CWAS identified regulatory elements and androgen receptor-binding sites that explained the association at 52 of 98 known prostate cancer risk loci and discovered 17 additional risk loci. CWAS implicated key developmental transcription factors in prostate cancer risk that are overlooked by eQTL-based approaches due to context-dependent gene regulation. We experimentally validated associations and demonstrated the extensibility of CWAS to additional epigenomic datasets and phenotypes, including response to prostate cancer treatment. CWAS is a powerful and biologically interpretable paradigm for studying variants that influence traits by affecting transcriptional regulation. Cistrome-wide association study (CWAS) is an approach for nominating variants that impact traits through effects on chromatin state. CWAS performed on 307 prostate cistromes identifies candidate loci for prostate cancer and androgen-related traits.},
  archive      = {J_NG},
  author       = {Baca, Sylvan C. and Singler, Cassandra and Zacharia, Soumya and Seo, Ji-Heui and Morova, Tunc and Hach, Faraz and Ding, Yi and Schwarz, Tommer and Huang, Chia-Chi Flora and Anderson, Jacob and Fay, André P. and Kalita, Cynthia and Groha, Stefan and Pomerantz, Mark M. and Wang, Victoria and Linder, Simon and Sweeney, Christopher J. and Zwart, Wilbert and Lack, Nathan A. and Pasaniuc, Bogdan and Takeda, David Y. and Gusev, Alexander and Freedman, Matthew L.},
  doi          = {10.1038/s41588-022-01168-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1364-1375},
  shortjournal = {Nat. Genet.},
  title        = {Genetic determinants of chromatin reveal prostate cancer risk mediated by context-dependent gene regulation},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetic control of RNA splicing and its distinct role in
complex trait variation. <em>NG</em>, <em>54</em>(9), 1355–1363. (<a
href="https://doi.org/10.1038/s41588-022-01154-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most genetic variants identified from genome-wide association studies (GWAS) in humans are noncoding, indicating their role in gene regulation. Previous studies have shown considerable links of GWAS signals to expression quantitative trait loci (eQTLs) but the links to other genetic regulatory mechanisms, such as splicing QTLs (sQTLs), are underexplored. Here, we introduce an sQTL mapping method, ﻿testing for heterogeneity between isoform-eQTL effects (THISTLE), with improved power over competing methods. Applying THISTLE together with a complementary sQTL mapping strategy to brain transcriptomic (n = 2,865) and genotype data, we identified 12,794 genes with cis-sQTLs at P &amp;lt; 5 × 10−8, approximately 61% of which were distinct from eQTLs. Integrating the sQTL data into GWAS for 12 brain-related complex traits (including diseases), we identified 244 genes associated with the traits through cis-sQTLs, approximately 61% of which could not be discovered using the corresponding eQTL data. Our study demonstrates the distinct role of most sQTLs in the genetic regulation of transcription and complex trait variation. A powerful method for splicing quantitative trait loci (sQTL) mapping, THISTLE, is presented and applied to a collection of 2,865 brain samples. Integration with GWAS identifies 244 genes associated via cis-sQTLs, of which 61% were not identified using expression QTLs.},
  archive      = {J_NG},
  author       = {Qi, Ting and Wu, Yang and Fang, Hailing and Zhang, Futao and Liu, Shouye and Zeng, Jian and Yang, Jian},
  doi          = {10.1038/s41588-022-01154-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1355-1363},
  shortjournal = {Nat. Genet.},
  title        = {Genetic control of RNA splicing and its distinct role in complex trait variation},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Gene–environment correlations across geographic regions
affect genome-wide association studies. <em>NG</em>, <em>54</em>(9),
1345–1354. (<a
href="https://doi.org/10.1038/s41588-022-01158-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene–environment correlations affect associations between genetic variants and complex traits in genome-wide association studies (GWASs). Here we showed in up to 43,516 British siblings that educational attainment polygenic scores capture gene–environment correlations, and that migration extends these gene–environment correlations beyond the family to broader geographic regions. We then ran GWASs on 56 complex traits in up to 254,387 British individuals. Controlling for geographic regions significantly decreased the heritability for socioeconomic status (SES)-related traits, most strongly for educational attainment and income. For most traits, controlling for regions significantly reduced genetic correlations with educational attainment and income, most significantly for body mass index/body fat, sedentary behavior and substance use, consistent with gene–environment correlations related to regional socio-economic differences. The effects of controlling for birthplace and current address suggest both passive and active sources of gene–environment correlations. Our results show that the geographic clustering of DNA and SES introduces gene–environment correlations that affect GWAS results. Analyses of gene–environment correlations across geographic regions for 56 complex traits in UK Biobank suggest that both passive and active sources of gene–environment correlation affect genetic association signals.},
  archive      = {J_NG},
  author       = {Abdellaoui, Abdel and Dolan, Conor V. and Verweij, Karin J. H. and Nivard, Michel G.},
  doi          = {10.1038/s41588-022-01158-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1345-1354},
  shortjournal = {Nat. Genet.},
  title        = {Gene–environment correlations across geographic regions affect genome-wide association studies},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome-wide association analyses of physical activity and
sedentary behavior provide insights into underlying mechanisms and roles
in disease prevention. <em>NG</em>, <em>54</em>(9), 1332–1344. (<a
href="https://doi.org/10.1038/s41588-022-01165-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although physical activity and sedentary behavior are moderately heritable, little is known about the mechanisms that influence these traits. Combining data for up to 703,901 individuals from 51 studies in a multi-ancestry meta-analysis of genome-wide association studies yields 99 loci that associate with self-reported moderate-to-vigorous intensity physical activity during leisure time (MVPA), leisure screen time (LST) and/or sedentary behavior at work. Loci associated with LST are enriched for genes whose expression in skeletal muscle is altered by resistance training. A missense variant in ACTN3 makes the alpha-actinin-3 filaments more flexible, resulting in lower maximal force in isolated type IIA muscle fibers, and possibly protection from exercise-induced muscle damage. Finally, Mendelian randomization analyses show that beneficial effects of lower LST and higher MVPA on several risk factors and diseases are mediated or confounded by body mass index (BMI). Our results provide insights into physical activity mechanisms and its role in disease prevention. Multi-ancestry meta-analyses of genome-wide association studies for self-reported physical activity during leisure time, leisure screen time, sedentary commuting and sedentary behavior at work identify 99 loci associated with at least one of these traits.},
  archive      = {J_NG},
  author       = {Wang, Zhe and Emmerich, Andrew and Pillon, Nicolas J. and Moore, Tim and Hemerich, Daiane and Cornelis, Marilyn C. and Mazzaferro, Eugenia and Broos, Siacia and Ahluwalia, Tarunveer S. and Bartz, Traci M. and Bentley, Amy R. and Bielak, Lawrence F. and Chong, Mike and Chu, Audrey Y. and Berry, Diane and Dorajoo, Rajkumar and Dueker, Nicole D. and Kasbohm, Elisa and Feenstra, Bjarke and Feitosa, Mary F. and Gieger, Christian and Graff, Mariaelisa and Hall, Leanne M. and Haller, Toomas and Hartwig, Fernando P. and Hillis, David A. and Huikari, Ville and Heard-Costa, Nancy and Holzapfel, Christina and Jackson, Anne U. and Johansson, Åsa and Jørgensen, Anja Moltke and Kaakinen, Marika A. and Karlsson, Robert and Kerr, Kathleen F. and Kim, Boram and Koolhaas, Chantal M. and Kutalik, Zoltan and Lagou, Vasiliki and Lind, Penelope A. and Lorentzon, Mattias and Lyytikäinen, Leo-Pekka and Mangino, Massimo and Metzendorf, Christoph and Monroe, Kristine R. and Pacolet, Alexander and Pérusse, Louis and Pool, Rene and Richmond, Rebecca C. and Rivera, Natalia V. and Robiou-du-Pont, Sebastien and Schraut, Katharina E. and Schulz, Christina-Alexandra and Stringham, Heather M. and Tanaka, Toshiko and Teumer, Alexander and Turman, Constance and van der Most, Peter J. and Vanmunster, Mathias and van Rooij, Frank J. A. and van Vliet-Ostaptchouk, Jana V. and Zhang, Xiaoshuai and Zhao, Jing-Hua and Zhao, Wei and Balkhiyarova, Zhanna and Balslev-Harder, Marie N. and Baumeister, Sebastian E. and Beilby, John and Blangero, John and Boomsma, Dorret I. and Brage, Soren and Braund, Peter S. and Brody, Jennifer A. and Bruinenberg, Marcel and Ekelund, Ulf and Liu, Ching-Ti and Cole, John W. and Collins, Francis S. and Cupples, L. Adrienne and Esko, Tõnu and Enroth, Stefan and Faul, Jessica D. and Fernandez-Rhodes, Lindsay and Fohner, Alison E. and Franco, Oscar H. and Galesloot, Tessel E. and Gordon, Scott D. and Grarup, Niels and Hartman, Catharina A. and Heiss, Gerardo and Hui, Jennie and Illig, Thomas and Jago, Russell and James, Alan and Joshi, Peter K. and Jung, Taeyeong and Kähönen, Mika and Kilpeläinen, Tuomas O. and Koh, Woon-Puay and Kolcic, Ivana and Kraft, Peter P. and Kuusisto, Johanna and Launer, Lenore J. and Li, Aihua and Linneberg, Allan and Luan, Jian’an and Vidal, Pedro Marques and Medland, Sarah E. and Milaneschi, Yuri and Moscati, Arden and Musk, Bill and Nelson, Christopher P. and Nolte, Ilja M. and Pedersen, Nancy L. and Peters, Annette and Peyser, Patricia A. and Power, Christine and Raitakari, Olli T. and Reedik, Mägi and Reiner, Alex P. and Ridker, Paul M. and Rudan, Igor and Ryan, Kathy and Sarzynski, Mark A. and Scott, Laura J. and Scott, Robert A. and Sidney, Stephen and Siggeirsdottir, Kristin and Smith, Albert V. and Smith, Jennifer A. and Sonestedt, Emily and Strøm, Marin and Tai, E. Shyong and Teo, Koon K. and Thorand, Barbara and Tönjes, Anke and Tremblay, Angelo and Uitterlinden, Andre G. and Vangipurapu, Jagadish and van Schoor, Natasja and Völker, Uwe and Willemsen, Gonneke and Williams, Kayleen and Wong, Quenna and Xu, Huichun and Young, Kristin L. and Yuan, Jian Min and Zillikens, M. Carola and Zonderman, Alan B. and Ameur, Adam and Bandinelli, Stefania and Bis, Joshua C. and Boehnke, Michael and Bouchard, Claude and Chasman, Daniel I. and Smith, George Davey and de Geus, Eco J. C. and Deldicque, Louise and Dörr, Marcus and Evans, Michele K. and Ferrucci, Luigi and Fornage, Myriam and Fox, Caroline and Garland, Theodore and Gudnason, Vilmundur and Gyllensten, Ulf and Hansen, Torben and Hayward, Caroline and Horta, Bernardo L. and Hyppönen, Elina and Jarvelin, Marjo-Riitta and Johnson, W. Craig and Kardia, Sharon L. R. and Kiemeney, Lambertus A. and Laakso, Markku and Langenberg, Claudia and Lehtimäki, Terho and Marchand, Loic Le and Magnusson, Patrik K. E. and Martin, Nicholas G. and Melbye, Mads and Metspalu, Andres and Meyre, David and North, Kari E. and Ohlsson, Claes and Oldehinkel, Albertine J. and Orho-Melander, Marju and Pare, Guillaume and Park, Taesung and Pedersen, Oluf and Penninx, Brenda W. J. H. and Pers, Tune H. and Polasek, Ozren and Prokopenko, Inga and Rotimi, Charles N. and Samani, Nilesh J. and Sim, Xueling and Snieder, Harold and Sørensen, Thorkild I. A. and Spector, Tim D. and Timpson, Nicholas J. and van Dam, Rob M. and van der Velde, Nathalie and van Duijn, Cornelia M. and Vollenweider, Peter and Völzke, Henry and Voortman, Trudy and Waeber, Gérard and Wareham, Nicholas J. and Weir, David R. and Wichmann, Heinz-Erich and Wilson, James F. and Hevener, Andrea L. and Krook, Anna and Zierath, Juleen R. and Thomis, Martine A. I. and Loos, Ruth J. F. and Hoed, Marcel den},
  doi          = {10.1038/s41588-022-01165-1},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1332-1344},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association analyses of physical activity and sedentary behavior provide insights into underlying mechanisms and roles in disease prevention},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Rare coding variation provides insight into the genetic
architecture and phenotypic context of autism. <em>NG</em>,
<em>54</em>(9), 1320–1331. (<a
href="https://doi.org/10.1038/s41588-022-01104-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Some individuals with autism spectrum disorder (ASD) carry functional mutations rarely observed in the general population. We explored the genes disrupted by these variants from joint analysis of protein-truncating variants (PTVs), missense variants and copy number variants (CNVs) in a cohort of 63,237 individuals. We discovered 72 genes associated with ASD at false discovery rate (FDR) ≤ 0.001 (185 at FDR ≤ 0.05). De novo PTVs, damaging missense variants and CNVs represented 57.5%, 21.1% and 8.44% of association evidence, while CNVs conferred greatest relative risk. Meta-analysis with cohorts ascertained for developmental delay (DD) (n = 91,605) yielded 373 genes associated with ASD/DD at FDR ≤ 0.001 (664 at FDR ≤ 0.05), some of which differed in relative frequency of mutation between ASD and DD cohorts. The DD-associated genes were enriched in transcriptomes of progenitor and immature neuronal cells, whereas genes showing stronger evidence in ASD were more enriched in maturing neurons and overlapped with schizophrenia-associated genes, emphasizing that these neuropsychiatric disorders may share common pathways to risk. Analysis of rare protein-truncating, damaging missense and copy number variants from exome sequencing of 63,237 individuals identifies 72 genes associated with autism spectrum disorder.},
  archive      = {J_NG},
  author       = {Fu, Jack M. and Satterstrom, F. Kyle and Peng, Minshi and Brand, Harrison and Collins, Ryan L. and Dong, Shan and Wamsley, Brie and Klei, Lambertus and Wang, Lily and Hao, Stephanie P. and Stevens, Christine R. and Cusick, Caroline and Babadi, Mehrtash and Banks, Eric and Collins, Brett and Dodge, Sheila and Gabriel, Stacey B. and Gauthier, Laura and Lee, Samuel K. and Liang, Lindsay and Ljungdahl, Alicia and Mahjani, Behrang and Sloofman, Laura and Smirnov, Andrey N. and Barbosa, Mafalda and Betancur, Catalina and Brusco, Alfredo and Chung, Brian H. Y. and Cook, Edwin H. and Cuccaro, Michael L. and Domenici, Enrico and Ferrero, Giovanni Battista and Gargus, J. Jay and Herman, Gail E. and Hertz-Picciotto, Irva and Maciel, Patricia and Manoach, Dara S. and Passos-Bueno, Maria Rita and Persico, Antonio M. and Renieri, Alessandra and Sutcliffe, James S. and Tassone, Flora and Trabetti, Elisabetta and Campos, Gabriele and Cardaropoli, Simona and Carli, Diana and Chan, Marcus C. Y. and Fallerini, Chiara and Giorgio, Elisa and Girardi, Ana Cristina and Hansen-Kiss, Emily and Lee, So Lun and Lintas, Carla and Ludena, Yunin and Nguyen, Rachel and Pavinato, Lisa and Pericak-Vance, Margaret and Pessah, Isaac N. and Schmidt, Rebecca J. and Smith, Moyra and Costa, Claudia I. S. and Trajkova, Slavica and Wang, Jaqueline Y. T. and Yu, Mullin H. C. and Cutler, David J. and De Rubeis, Silvia and Buxbaum, Joseph D. and Daly, Mark J. and Devlin, Bernie and Roeder, Kathryn and Sanders, Stephan J. and Talkowski, Michael E.},
  doi          = {10.1038/s41588-022-01104-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1320-1331},
  shortjournal = {Nat. Genet.},
  title        = {Rare coding variation provides insight into the genetic architecture and phenotypic context of autism},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Integrating de novo and inherited variants in 42,607 autism
cases identifies mutations in new moderate-risk genes. <em>NG</em>,
<em>54</em>(9), 1305–1319. (<a
href="https://doi.org/10.1038/s41588-022-01148-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To capture the full spectrum of genetic risk for autism, we performed a two-stage analysis of rare de novo and inherited coding variants in 42,607 autism cases, including 35,130 new cases recruited online by SPARK. We identified 60 genes with exome-wide significance (P &amp;lt; 2.5 × 10−6), including five new risk genes (NAV3, ITSN1, MARK2, SCAF1 and HNRNPUL2). The association of NAV3 with autism risk is primarily driven by rare inherited loss-of-function (LoF) variants, with an estimated relative risk of 4, consistent with moderate effect. Autistic individuals with LoF variants in the four moderate-risk genes (NAV3, ITSN1, SCAF1 and HNRNPUL2; n = 95) have less cognitive impairment than 129 autistic individuals with LoF variants in highly penetrant genes (CHD8, SCN2A, ADNP, FOXP1 and SHANK3) (59% vs 88%, P = 1.9 × 10−6). Power calculations suggest that much larger numbers of autism cases are needed to identify additional moderate-risk genes. An integrated analysis of de novo and inherited coding variants in 42,607 individuals with autism spectrum disorder identifies 60 risk genes of which five have not previously been associated with neurodevelopmental disorders.},
  archive      = {J_NG},
  author       = {Zhou, Xueya and Feliciano, Pamela and Shu, Chang and Wang, Tianyun and Astrovskaya, Irina and Hall, Jacob B. and Obiajulu, Joseph U. and Wright, Jessica R. and Murali, Shwetha C. and Xu, Simon Xuming and Brueggeman, Leo and Thomas, Taylor R. and Marchenko, Olena and Fleisch, Christopher and Barns, Sarah D. and Snyder, LeeAnne Green and Han, Bing and Chang, Timothy S. and Turner, Tychele N. and Harvey, William T. and Nishida, Andrew and O’Roak, Brian J. and Geschwind, Daniel H. and Michaelson, Jacob J. and Volfovsky, Natalia and Eichler, Evan E. and Shen, Yufeng and Chung, Wendy K.},
  doi          = {10.1038/s41588-022-01148-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1305-1319},
  shortjournal = {Nat. Genet.},
  title        = {Integrating de novo and inherited variants in 42,607 autism cases identifies mutations in new moderate-risk genes},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetic correlates of phenotypic heterogeneity in autism.
<em>NG</em>, <em>54</em>(9), 1293–1304. (<a
href="https://doi.org/10.1038/s41588-022-01072-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The substantial phenotypic heterogeneity in autism limits our understanding of its genetic etiology. To address this gap, here we investigated genetic differences between autistic individuals (nmax = 12,893) based on core and associated features of autism, co-occurring developmental disabilities and sex. We conducted a comprehensive factor analysis of core autism features in autistic individuals and identified six factors. Common genetic variants were associated with the core factors, but de novo variants were not. We found that higher autism polygenic scores (PGS) were associated with lower likelihood of co-occurring developmental disabilities in autistic individuals. Furthermore, in autistic individuals without co-occurring intellectual disability (ID), autism PGS are overinherited by autistic females compared to males. Finally, we observed higher SNP heritability for autistic males and for autistic individuals without ID. Deeper phenotypic characterization will be critical in determining how the complex underlying genetics shape cognition, behavior and co-occurring conditions in autism. Comprehensive factor analysis of core diagnostic features provides insights into the complex genetic architecture underlying phenotypic heterogeneity in autism.},
  archive      = {J_NG},
  author       = {Warrier, Varun and Zhang, Xinhe and Reed, Patrick and Havdahl, Alexandra and Moore, Tyler M. and Cliquet, Freddy and Leblond, Claire S. and Rolland, Thomas and Rosengren, Anders and Rowitch, David H. and Hurles, Matthew E. and Geschwind, Daniel H. and Børglum, Anders D. and Robinson, Elise B. and Grove, Jakob and Martin, Hilary C. and Bourgeron, Thomas and Baron-Cohen, Simon},
  doi          = {10.1038/s41588-022-01072-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1293-1304},
  shortjournal = {Nat. Genet.},
  title        = {Genetic correlates of phenotypic heterogeneity in autism},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). A phenotypic spectrum of autism is attributable to the
combined effects of rare variants, polygenic risk and sex. <em>NG</em>,
<em>54</em>(9), 1284–1292. (<a
href="https://doi.org/10.1038/s41588-022-01064-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genetic etiology of autism spectrum disorder (ASD) is multifactorial, but how combinations of genetic factors determine risk is unclear. In a large family sample, we show that genetic loads of rare and polygenic risk are inversely correlated in cases and greater in females than in males, consistent with a liability threshold that differs by sex. De novo mutations (DNMs), rare inherited variants and polygenic scores were associated with various dimensions of symptom severity in children and parents. Parental age effects on risk for ASD in offspring were attributable to a combination of genetic mechanisms, including DNMs that accumulate in the paternal germline and inherited risk that influences behavior in parents. Genes implicated by rare variants were enriched in excitatory and inhibitory neurons compared with genes implicated by common variants. Our results suggest that a phenotypic spectrum of ASD is attributable to a spectrum of genetic factors that impact different neurodevelopmental processes. Integrated analyses in a large collection of families provide insights into the combined effects of rare variants and polygenic risk on autism spectrum disorder.},
  archive      = {J_NG},
  author       = {Antaki, Danny and Guevara, James and Maihofer, Adam X. and Klein, Marieke and Gujral, Madhusudan and Grove, Jakob and Carey, Caitlin E. and Hong, Oanh and Arranz, Maria J. and Hervas, Amaia and Corsello, Christina and Vaux, Keith K. and Muotri, Alysson R. and Iakoucheva, Lilia M. and Courchesne, Eric and Pierce, Karen and Gleeson, Joseph G. and Robinson, Elise B. and Nievergelt, Caroline M. and Sebat, Jonathan},
  doi          = {10.1038/s41588-022-01064-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1284-1292},
  shortjournal = {Nat. Genet.},
  title        = {A phenotypic spectrum of autism is attributable to the combined effects of rare variants, polygenic risk and sex},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Large-scale sequencing identifies multiple genes and rare
variants associated with crohn’s disease susceptibility. <em>NG</em>,
<em>54</em>(9), 1275–1283. (<a
href="https://doi.org/10.1038/s41588-022-01156-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWASs) have identified hundreds of loci associated with Crohn’s disease (CD). However, as with all complex diseases, robust identification of the genes dysregulated by noncoding variants typically driving GWAS discoveries has been challenging. Here, to complement GWASs and better define actionable biological targets, we analyzed sequence data from more than 30,000 patients with CD and 80,000 population controls. We directly implicate ten genes in general onset CD for the first time to our knowledge via association to coding variation, four of which lie within established CD GWAS loci. In nine instances, a single coding variant is significantly associated, and in the tenth, ATG4C, we see additionally a significantly increased burden of very rare coding variants in CD cases. In addition to reiterating the central role of innate and adaptive immune cells as well as autophagy in CD pathogenesis, these newly associated genes highlight the emerging role of mesenchymal cells in the development and maintenance of intestinal inflammation. Large-scale sequence-based analyses identify novel risk variants and susceptibility genes for Crohn’s disease, and implicate mesenchymal cell-mediated intestinal homeostasis in disease etiology.},
  archive      = {J_NG},
  author       = {Sazonovs, Aleksejs and Stevens, Christine R. and Venkataraman, Guhan R. and Yuan, Kai and Avila, Brandon and Abreu, Maria T. and Ahmad, Tariq and Allez, Matthieu and Ananthakrishnan, Ashwin N. and Atzmon, Gil and Baras, Aris and Barrett, Jeffrey C. and Barzilai, Nir and Beaugerie, Laurent and Beecham, Ashley and Bernstein, Charles N. and Bitton, Alain and Bokemeyer, Bernd and Chan, Andrew and Chung, Daniel and Cleynen, Isabelle and Cosnes, Jacques and Cutler, David J. and Daly, Allan and Damas, Oriana M. and Datta, Lisa W. and Dawany, Noor and Devoto, Marcella and Dodge, Sheila and Ellinghaus, Eva and Fachal, Laura and Farkkila, Martti and Faubion, William and Ferreira, Manuel and Franchimont, Denis and Gabriel, Stacey B. and Ge, Tian and Georges, Michel and Gettler, Kyle and Giri, Mamta and Glaser, Benjamin and Goerg, Siegfried and Goyette, Philippe and Graham, Daniel and Hämäläinen, Eija and Haritunians, Talin and Heap, Graham A. and Hiltunen, Mikko and Hoeppner, Marc and Horowitz, Julie E. and Irving, Peter and Iyer, Vivek and Jalas, Chaim and Kelsen, Judith and Khalili, Hamed and Kirschner, Barbara S. and Kontula, Kimmo and Koskela, Jukka T. and Kugathasan, Subra and Kupcinskas, Juozas and Lamb, Christopher A. and Laudes, Matthias and Lévesque, Chloé and Levine, Adam P. and Lewis, James D. and Liefferinckx, Claire and Loescher, Britt-Sabina and Louis, Edouard and Mansfield, John and May, Sandra and McCauley, Jacob L. and Mengesha, Emebet and Mni, Myriam and Moayyedi, Paul and Moran, Christopher J. and Newberry, Rodney D. and O’Charoen, Sirimon and Okou, David T. and Oldenburg, Bas and Ostrer, Harry and Palotie, Aarno and Paquette, Jean and Pekow, Joel and Peter, Inga and Pierik, Marieke J. and Ponsioen, Cyriel Y. and Pontikos, Nikolas and Prescott, Natalie and Pulver, Ann E. and Rahmouni, Souad and Rice, Daniel L. and Saavalainen, Päivi and Sands, Bruce and Sartor, R. Balfour and Schiff, Elena R. and Schreiber, Stefan and Schumm, L. Philip and Segal, Anthony W. and Seksik, Philippe and Shawky, Rasha and Sheikh, Shehzad Z. and Silverberg, Mark S. and Simmons, Alison and Skeiceviciene, Jurgita and Sokol, Harry and Solomonson, Matthew and Somineni, Hari and Sun, Dylan and Targan, Stephan and Turner, Dan and Uhlig, Holm H. and van der Meulen, Andrea E. and Vermeire, Séverine and Verstockt, Sare and Voskuil, Michiel D. and Winter, Harland S. and Young, Justine and Duerr, Richard H. and Franke, Andre and Brant, Steven R. and Cho, Judy and Weersma, Rinse K. and Parkes, Miles and Xavier, Ramnik J. and Rivas, Manuel A. and Rioux, John D. and McGovern, Dermot P. B. and Huang, Hailiang and Anderson, Carl A. and Daly, Mark J.},
  doi          = {10.1038/s41588-022-01156-2},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1275-1283},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale sequencing identifies multiple genes and rare variants associated with crohn’s disease susceptibility},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The CattleGTEx atlas reveals regulatory mechanisms
underlying complex traits. <em>NG</em>, <em>54</em>(9), 1273–1274. (<a
href="https://doi.org/10.1038/s41588-022-01155-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We generated a cattle genotype–tissue expression atlas (CattleGTEx) as part of the farm animal GTEx project. Analysis of 7,180 public RNA-sequencing samples revealed genetic variants that regulate the transcriptome across 23 distinct bovine tissues. Integrating these data with GWAS advances our understanding of the fundamental molecular mechanisms that underpin complex traits in cattle.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01155-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1273-1274},
  shortjournal = {Nat. Genet.},
  title        = {The CattleGTEx atlas reveals regulatory mechanisms underlying complex traits},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Large-scale genomic analysis of human iPSCs identifies
recurrent somatic driver mutations. <em>NG</em>, <em>54</em>(9),
1271–1272. (<a
href="https://doi.org/10.1038/s41588-022-01169-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genetic landscape of human induced pluripotent stem cells (iPSCs) is strongly influenced by the somatic cells of origin, and mutational signatures directly reflect pre-reprogramming and post-reprogramming mutagenic processes. BCOR mutations are recurrent and have functional consequences for the differentiation capacity of iPSCs.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01169-x},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1271-1272},
  shortjournal = {Nat. Genet.},
  title        = {Large-scale genomic analysis of human iPSCs identifies recurrent somatic driver mutations},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Transitioning cancer cells and the surrounding tumor
environment in pancreatic cancer. <em>NG</em>, <em>54</em>(9),
1269–1270. (<a
href="https://doi.org/10.1038/s41588-022-01162-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In our study of 83 tissue samples from 31 patients with pancreatic cancer receiving different treatment regimens, we directly identified normal cells transitioning to a tumor state. Single-cell technologies enabled us to observe how the tumor and the surrounding environment work together to create a treatment-resistant niche.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01162-4},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1269-1270},
  shortjournal = {Nat. Genet.},
  title        = {Transitioning cancer cells and the surrounding tumor environment in pancreatic cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Generating a genetic road map of childhood acute
lymphoblastic leukemia. <em>NG</em>, <em>54</em>(9), 1267–1268. (<a
href="https://doi.org/10.1038/s41588-022-01161-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We comprehensively define the genomic landscape of pediatric acute lymphoblastic leukemia (ALL) in the largest cohort assembled to date, and identify new driver genes and biological pathways that are targeted by genetic alteration. These findings serve as a road map to improve our understanding of disease development and identify therapeutic targets in pediatric ALL.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01161-5},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1267-1268},
  shortjournal = {Nat. Genet.},
  title        = {Generating a genetic road map of childhood acute lymphoblastic leukemia},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genes influence complex traits through environments that
vary between geographic regions. <em>NG</em>, <em>54</em>(9), 1265–1266.
(<a href="https://doi.org/10.1038/s41588-022-01163-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Migration can increase regional differences in both DNA variants and environmental factors that are associated with socio-economic status. Without controlling for geography, associations between genes and complex traits will therefore include effects of environmental differences between high-income and low-income regions.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01163-3},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1265-1266},
  shortjournal = {Nat. Genet.},
  title        = {Genes influence complex traits through environments that vary between geographic regions},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Diverse mutations in autism-related genes and their
expression in the developing brain. <em>NG</em>, <em>54</em>(9),
1263–1264. (<a
href="https://doi.org/10.1038/s41588-022-01114-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Across &amp;gt;150,000 individuals, we identified hundreds of genes associated with autism spectrum disorder (ASD) and atypical neurodevelopment. Most ASD-related genes were also associated with developmental delay. However, increased mutation rates in ASD and shared genetic risk with schizophrenia was observed for some genes, many of which are enriched in developing neurons.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01114-y},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1263-1264},
  shortjournal = {Nat. Genet.},
  title        = {Diverse mutations in autism-related genes and their expression in the developing brain},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). When RNA methylation meets DNA methylation. <em>NG</em>,
<em>54</em>(9), 1261–1262. (<a
href="https://doi.org/10.1038/s41588-022-01166-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this issue, Deng and colleagues show that the RNA m6A modification reader FXR1 recruits DNA 5-methylcytosine dioxygenase TET1 to genomic regions in order to demethylate DNA, which highlights a crosstalk between RNA methylation and DNA methylation in chromatin.},
  archive      = {J_NG},
  author       = {Xu, Wenqi and Shen, Hongjie},
  doi          = {10.1038/s41588-022-01166-0},
  journal      = {Nature Genetics},
  month        = {9},
  number       = {9},
  pages        = {1261-1262},
  shortjournal = {Nat. Genet.},
  title        = {When RNA methylation meets DNA methylation},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Single-nucleus chromatin accessibility
profiling highlights regulatory mechanisms of coronary artery disease
risk. <em>NG</em>, <em>54</em>(8), 1259. (<a
href="https://doi.org/10.1038/s41588-022-01142-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Turner, Adam W. and Hu, Shengen Shawn and Mosquera, Jose Verdezoto and Ma, Wei Feng and Hodonsky, Chani J. and Wong, Doris and Auguste, Gaëlle and Song, Yipei and Sol-Church, Katia and Farber, Emily and Kundu, Soumya and Kundaje, Anshul and Lopez, Nicolas G. and Ma, Lijiang and Ghosh, Saikat Kumar B. and Onengut-Gumuscu, Suna and Ashley, Euan A. and Quertermous, Thomas and Finn, Aloke V. and Leeper, Nicholas J. and Kovacic, Jason C. and Björkegren, Johan L. M. and Zang, Chongzhi and Miller, Clint L.},
  doi          = {10.1038/s41588-022-01142-8},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1259},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Single-nucleus chromatin accessibility profiling highlights regulatory mechanisms of coronary artery disease risk},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Improving polygenic prediction in
ancestrally diverse populations. <em>NG</em>, <em>54</em>(8), 1259. (<a
href="https://doi.org/10.1038/s41588-022-01144-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Ruan, Yunfeng and Lin, Yen-Feng and Feng, Yen-Chen Anne and Chen, Chia-Yen and Lam, Max and Guo, Zhenglin and He, Lin and Sawa, Akira and Martin, Alicia R. and Qin, Shengying and Huang, Hailiang and Ge, Tian},
  doi          = {10.1038/s41588-022-01144-6},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1259},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Improving polygenic prediction in ancestrally diverse populations},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: A phenotypic spectrum of autism is
attributable to the combined effects of rare variants, polygenic risk
and sex. <em>NG</em>, <em>54</em>(8), 1259. (<a
href="https://doi.org/10.1038/s41588-022-01145-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Antaki, Danny and Guevara, James and Maihofer, Adam X. and Klein, Marieke and Gujral, Madhusudan and Grove, Jakob and Carey, Caitlin E. and Hong, Oanh and Arranz, Maria J. and Hervas, Amaia and Corsello, Christina and Vaux, Keith K. and Muotri, Alysson R. and Iakoucheva, Lilia M. and Courchesne, Eric and Pierce, Karen and Gleeson, Joseph G. and Robinson, Elise B. and Nievergelt, Caroline M. and Sebat, Jonathan},
  doi          = {10.1038/s41588-022-01145-5},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1259},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: A phenotypic spectrum of autism is attributable to the combined effects of rare variants, polygenic risk and sex},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reference genome assemblies reveal the origin and evolution
of allohexaploid oat. <em>NG</em>, <em>54</em>(8), 1248–1258. (<a
href="https://doi.org/10.1038/s41588-022-01127-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Common oat (Avena sativa) is an important cereal crop serving as a valuable source of forage and human food. Although reference genomes of many important crops have been generated, such work in oat has lagged behind, primarily owing to its large, repeat-rich polyploid genome. Here, using Oxford Nanopore ultralong sequencing and Hi-C technologies, we have generated a reference-quality genome assembly of hulless common oat, comprising 21 pseudomolecules with a total length of 10.76 Gb and contig N50 of 75.27 Mb. We also produced genome assemblies for diploid and tetraploid Avena ancestors, which enabled the identification of oat subgenomes and provided insights into oat chromosomal evolution. The origin of hexaploid oat is inferred from whole-genome sequencing, chloroplast genomes and transcriptome assemblies of different Avena species. These findings and the high-quality reference genomes presented here will facilitate the full use of crop genetic resources to accelerate oat improvement. A reference-quality genome assembly of hexaploid oat variety ‘Sanfensan’ and genome assemblies of its diploid and tetraploid Avena ancestors provide insights into the evolutionary history of allohexaploid oat.},
  archive      = {J_NG},
  author       = {Peng, Yuanying and Yan, Honghai and Guo, Laichun and Deng, Cao and Wang, Chunlong and Wang, Yubo and Kang, Lipeng and Zhou, Pingping and Yu, Kaiquan and Dong, Xiaolong and Liu, Xiaomeng and Sun, Zongyi and Peng, Yun and Zhao, Jun and Deng, Di and Xu, Yinghong and Li, Ying and Jiang, Qiantao and Li, Yan and Wei, Liming and Wang, Jirui and Ma, Jian and Hao, Ming and Li, Wei and Kang, Houyang and Peng, Zhengsong and Liu, Dengcai and Jia, Jizeng and Zheng, Youliang and Ma, Tao and Wei, Yuming and Lu, Fei and Ren, Changzhong},
  doi          = {10.1038/s41588-022-01127-7},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1248-1258},
  shortjournal = {Nat. Genet.},
  title        = {Reference genome assemblies reveal the origin and evolution of allohexaploid oat},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Hijacking of transcriptional condensates by endogenous
retroviruses. <em>NG</em>, <em>54</em>(8), 1238–1247. (<a
href="https://doi.org/10.1038/s41588-022-01132-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most endogenous retroviruses (ERVs) in mammals are incapable of retrotransposition; therefore, why ERV derepression is associated with lethality during early development has been a mystery. Here, we report that rapid and selective degradation of the heterochromatin adapter protein TRIM28 triggers dissociation of transcriptional condensates from loci encoding super-enhancer (SE)-driven pluripotency genes and their association with transcribed ERV loci in murine embryonic stem cells. Knockdown of ERV RNAs or forced expression of SE-enriched transcription factors rescued condensate localization at SEs in TRIM28-degraded cells. In a biochemical reconstitution system, ERV RNA facilitated partitioning of RNA polymerase II and the Mediator coactivator into phase-separated droplets. In TRIM28 knockout mouse embryos, single-cell RNA-seq analysis revealed specific depletion of pluripotent lineages. We propose that coding and noncoding nascent RNAs, including those produced by retrotransposons, may facilitate ‘hijacking’ of transcriptional condensates in various developmental and disease contexts. TRIM28 depletion in embryonic stem cells disconnects transcriptional condensates from super-enhancers, which is rescued by knockdown of endogenous retroviruses.},
  archive      = {J_NG},
  author       = {Asimi, Vahid and Sampath Kumar, Abhishek and Niskanen, Henri and Riemenschneider, Christina and Hetzel, Sara and Naderi, Julian and Fasching, Nina and Popitsch, Niko and Du, Manyu and Kretzmer, Helene and Smith, Zachary D. and Weigert, Raha and Walther, Maria and Mamde, Sainath and Meierhofer, David and Wittler, Lars and Buschow, René and Timmermann, Bernd and Cisse, Ibrahim I. and Ameres, Stefan L. and Meissner, Alexander and Hnisz, Denes},
  doi          = {10.1038/s41588-022-01132-w},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1238-1247},
  shortjournal = {Nat. Genet.},
  title        = {Hijacking of transcriptional condensates by endogenous retroviruses},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Origin and function of activated fibroblast states during
zebrafish heart regeneration. <em>NG</em>, <em>54</em>(8), 1227–1237.
(<a href="https://doi.org/10.1038/s41588-022-01129-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The adult zebrafish heart has a high capacity for regeneration following injury. However, the composition of the regenerative niche has remained largely elusive. Here, we dissected the diversity of activated cell states in the regenerating zebrafish heart based on single-cell transcriptomics and spatiotemporal analysis. We observed the emergence of several transient cell states with fibroblast characteristics following injury, and we outlined the proregenerative function of collagen-12-expressing fibroblasts. To understand the cascade of events leading to heart regeneration, we determined the origin of these cell states by high-throughput lineage tracing. We found that activated fibroblasts were derived from two separate sources: the epicardium and the endocardium. Mechanistically, we determined Wnt signalling as a regulator of the endocardial fibroblast response. In summary, our work identifies specialized activated fibroblast cell states that contribute to heart regeneration, thereby opening up possible approaches to modulating the regenerative capacity of the vertebrate heart. Single-cell RNA sequencing and spatiotemporal analysis of the regenerating zebrafish heart identify transient proregenerative fibroblast-like cells that are derived from the epicardium and the endocardium. Wnt signalling regulates the endocardial fibroblast response.},
  archive      = {J_NG},
  author       = {Hu, Bo and Lelek, Sara and Spanjaard, Bastiaan and El-Sammak, Hadil and Simões, Mariana Guedes and Mintcheva, Janita and Aliee, Hananeh and Schäfer, Ronny and Meyer, Alexander M. and Theis, Fabian and Stainier, Didier Y. R. and Panáková, Daniela and Junker, Jan Philipp},
  doi          = {10.1038/s41588-022-01129-5},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1227-1237},
  shortjournal = {Nat. Genet.},
  title        = {Origin and function of activated fibroblast states during zebrafish heart regeneration},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Loss of FOCAD, operating via the SKI messenger RNA
surveillance pathway, causes a pediatric syndrome with liver cirrhosis.
<em>NG</em>, <em>54</em>(8), 1214–1226. (<a
href="https://doi.org/10.1038/s41588-022-01120-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cirrhosis is usually a late-onset and life-threatening disease characterized by fibrotic scarring and inflammation that disrupts liver architecture and function. While it is typically the result of alcoholism or hepatitis viral infection in adults, its etiology in infants is much less understood. In this study, we report 14 children from ten unrelated families presenting with a syndromic form of pediatric liver cirrhosis. By genome/exome sequencing, we found recessive variants in FOCAD segregating with the disease. Zebrafish lacking focad phenocopied the human disease, revealing a signature of altered messenger RNA (mRNA) degradation processes in the liver. Using patient’s primary cells and CRISPR-Cas9-mediated inactivation in human hepatic cell lines, we found that FOCAD deficiency compromises the SKI mRNA surveillance pathway by reducing the levels of the RNA helicase SKIC2 and its cofactor SKIC3. FOCAD knockout hepatocytes exhibited lowered albumin expression and signs of persistent injury accompanied by CCL2 overproduction. Our results reveal the importance of FOCAD in maintaining liver homeostasis and disclose a possible therapeutic intervention point via inhibition of the CCL2/CCR2 signaling axis. Biallelic loss-of-function variants in FOCAD cause a syndromic form of pediatric liver disease by compromising the SKI messenger RNA surveillance pathway.},
  archive      = {J_NG},
  author       = {Moreno Traspas, Ricardo and Teoh, Tze Shin and Wong, Pui-Mun and Maier, Michael and Chia, Crystal Y. and Lay, Kenneth and Ali, Nur Ain and Larson, Austin and Al Mutairi, Fuad and Al-Sannaa, Nouriya Abbas and Faqeih, Eissa Ali and Alfadhel, Majid and Cheema, Huma Arshad and Dupont, Juliette and Bézieau, Stéphane and Isidor, Bertrand and Low, Dorrain Yanwen and Wang, Yulan and Tan, Grace and Lai, Poh San and Piloquet, Hugues and Joubert, Madeleine and Kayserili, Hulya and Kripps, Kimberly A. and Nahas, Shareef A. and Wartchow, Eric P. and Warren, Mikako and Bhavani, Gandham SriLakshmi and Dasouki, Majed and Sandoval, Renata and Carvalho, Elisa and Ramos, Luiza and Porta, Gilda and Wu, Bin and Lashkari, Harsha Prasada and AlSaleem, Badr and BaAbbad, Raeda M. and Abreu Ferrão, Anabela Natália and Karageorgou, Vasiliki and Ordonez-Herrera, Natalia and Khan, Suliman and Bauer, Peter and Cogne, Benjamin and Bertoli-Avella, Aida M. and Vincent, Marie and Girisha, Katta Mohan and Reversade, Bruno},
  doi          = {10.1038/s41588-022-01120-0},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1214-1226},
  shortjournal = {Nat. Genet.},
  title        = {Loss of FOCAD, operating via the SKI messenger RNA surveillance pathway, causes a pediatric syndrome with liver cirrhosis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). YAP induces an oncogenic transcriptional program through
TET1-mediated epigenetic remodeling in liver growth and tumorigenesis.
<em>NG</em>, <em>54</em>(8), 1202–1213. (<a
href="https://doi.org/10.1038/s41588-022-01119-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epigenetic remodeling is essential for oncogene-induced cellular transformation and malignancy. In contrast to histone post-translational modifications, how DNA methylation is remodeled by oncogenic signaling remains poorly understood. The oncoprotein YAP, a coactivator of the TEAD transcription factors mediating Hippo signaling, is widely activated in human cancers. Here, we identify the 5-methylcytosine dioxygenase TET1 as a direct YAP target and a master regulator that coordinates the genome-wide epigenetic and transcriptional reprogramming of YAP target genes in the liver. YAP activation induces the expression of TET1, which physically interacts with TEAD to cause regional DNA demethylation, histone H3K27 acetylation and chromatin opening in YAP target genes to facilitate transcriptional activation. Loss of TET1 not only reverses YAP-induced epigenetic and transcriptional changes but also suppresses YAP-induced hepatomegaly and tumorigenesis. These findings exemplify how oncogenic signaling regulates the site specificity of DNA demethylation to promote tumorigenesis and implicate TET1 as a potential target for modulating YAP signaling in physiology and disease. YAP upregulates TET1, which physically interacts with TEAD1/4 to demethylate DNA at YAP target genes in the liver. Loss of TET1 reverses YAP-induced chromatin and transcriptional changes and suppresses YAP-induced hepatomegaly and tumorigenesis.},
  archive      = {J_NG},
  author       = {Wu, Bo-Kuan and Mei, Szu-Chieh and Chen, Elizabeth H. and Zheng, Yonggang and Pan, Duojia},
  doi          = {10.1038/s41588-022-01119-7},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1202-1213},
  shortjournal = {Nat. Genet.},
  title        = {YAP induces an oncogenic transcriptional program through TET1-mediated epigenetic remodeling in liver growth and tumorigenesis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cancer cell states recur across tumor types and form
specific interactions with the tumor microenvironment. <em>NG</em>,
<em>54</em>(8), 1192–1201. (<a
href="https://doi.org/10.1038/s41588-022-01141-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transcriptional heterogeneity among malignant cells of a tumor has been studied in individual cancer types and shown to be organized into cancer cell states; however, it remains unclear to what extent these states span tumor types, constituting general features of cancer. Here, we perform a pan-cancer single-cell RNA-sequencing analysis across 15 cancer types and identify a catalog of gene modules whose expression defines recurrent cancer cell states including ‘stress’, ‘interferon response’, ‘epithelial-mesenchymal transition’, ‘metal response’, ‘basal’ and ‘ciliated’. Spatial transcriptomic analysis linked the interferon response in cancer cells to T cells and macrophages in the tumor microenvironment. Using mouse models, we further found that induction of the interferon response module varies by tumor location and is diminished upon elimination of lymphocytes. Our work provides a framework for studying how cancer cell states interact with the tumor microenvironment to form organized systems capable of immune evasion, drug resistance and metastasis. Pan-cancer single-cell and spatial transcriptomic profiling identifies recurrent gene modules that underlie a continuum of cancer cell states. Tumor microenvironment influences the occurrence of these states.},
  archive      = {J_NG},
  author       = {Barkley, Dalia and Moncada, Reuben and Pour, Maayan and Liberman, Deborah A. and Dryg, Ian and Werba, Gregor and Wang, Wei and Baron, Maayan and Rao, Anjali and Xia, Bo and França, Gustavo S. and Weil, Alejandro and Delair, Deborah F. and Hajdu, Cristina and Lund, Amanda W. and Osman, Iman and Yanai, Itai},
  doi          = {10.1038/s41588-022-01141-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1192-1201},
  shortjournal = {Nat. Genet.},
  title        = {Cancer cell states recur across tumor types and form specific interactions with the tumor microenvironment},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-nucleus and spatial transcriptome profiling of
pancreatic cancer identifies multicellular dynamics associated with
neoadjuvant treatment. <em>NG</em>, <em>54</em>(8), 1178–1191. (<a
href="https://doi.org/10.1038/s41588-022-01134-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pancreatic ductal adenocarcinoma (PDAC) is a highly lethal and treatment-refractory cancer. Molecular stratification in pancreatic cancer remains rudimentary and does not yet inform clinical management or therapeutic development. Here, we construct a high-resolution molecular landscape of the cellular subtypes and spatial communities that compose PDAC using single-nucleus RNA sequencing and whole-transcriptome digital spatial profiling (DSP) of 43 primary PDAC tumor specimens that either received neoadjuvant therapy or were treatment naive. We uncovered recurrent expression programs across malignant cells and fibroblasts, including a newly identified neural-like progenitor malignant cell program that was enriched after chemotherapy and radiotherapy and associated with poor prognosis in independent cohorts. Integrating spatial and cellular profiles revealed three multicellular communities with distinct contributions from malignant, fibroblast and immune subtypes: classical, squamoid-basaloid and treatment enriched. Our refined molecular and cellular taxonomy can provide a framework for stratification in clinical trials and serve as a roadmap for therapeutic targeting of specific cellular phenotypes and multicellular interactions. Single-nucleus and spatial, whole-transcriptome profiling of 43 pancreatic adenocarcinomas provides a refined molecular and cellular classification, highlighting a new neoadjuvant treatment-associated neural-like progenitor tumor cell state.},
  archive      = {J_NG},
  author       = {Hwang, William L. and Jagadeesh, Karthik A. and Guo, Jimmy A. and Hoffman, Hannah I. and Yadollahpour, Payman and Reeves, Jason W. and Mohan, Rahul and Drokhlyansky, Eugene and Van Wittenberghe, Nicholas and Ashenberg, Orr and Farhi, Samouil L. and Schapiro, Denis and Divakar, Prajan and Miller, Eric and Zollinger, Daniel R. and Eng, George and Schenkel, Jason M. and Su, Jennifer and Shiau, Carina and Yu, Patrick and Freed-Pastor, William A. and Abbondanza, Domenic and Mehta, Arnav and Gould, Joshua and Lambden, Conner and Porter, Caroline B. M. and Tsankov, Alexander and Dionne, Danielle and Waldman, Julia and Cuoco, Michael S. and Nguyen, Lan and Delorey, Toni and Phillips, Devan and Barth, Jaimie L. and Kem, Marina and Rodrigues, Clifton and Ciprani, Debora and Roldan, Jorge and Zelga, Piotr and Jorgji, Vjola and Chen, Jonathan H. and Ely, Zackery and Zhao, Daniel and Fuhrman, Kit and Fropf, Robin and Beechem, Joseph M. and Loeffler, Jay S. and Ryan, David P. and Weekes, Colin D. and Ferrone, Cristina R. and Qadan, Motaz and Aryee, Martin J. and Jain, Rakesh K. and Neuberg, Donna S. and Wo, Jennifer Y. and Hong, Theodore S. and Xavier, Ramnik and Aguirre, Andrew J. and Rozenblatt-Rosen, Orit and Mino-Kenudson, Mari and Castillo, Carlos Fernandez-del and Liss, Andrew S. and Ting, David T. and Jacks, Tyler and Regev, Aviv},
  doi          = {10.1038/s41588-022-01134-8},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1178-1191},
  shortjournal = {Nat. Genet.},
  title        = {Single-nucleus and spatial transcriptome profiling of pancreatic cancer identifies multicellular dynamics associated with neoadjuvant treatment},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Cross-ancestry genome-wide meta-analysis of 61,047 cases and
947,237 controls identifies new susceptibility loci contributing to lung
cancer. <em>NG</em>, <em>54</em>(8), 1167–1177. (<a
href="https://doi.org/10.1038/s41588-022-01115-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To identify new susceptibility loci to lung cancer among diverse populations, we performed cross-ancestry genome-wide association studies in European, East Asian and African populations and discovered five loci that have not been previously reported. We replicated 26 signals and identified 10 new lead associations from previously reported loci. Rare-variant associations tended to be specific to populations, but even common-variant associations influencing smoking behavior, such as those with CHRNA5 and CYP2A6, showed population specificity. Fine-mapping and expression quantitative trait locus colocalization nominated several candidate variants and susceptibility genes such as IRF4 and FUBP1. DNA damage assays of prioritized genes in lung fibroblasts indicated that a subset of these genes, including the pleiotropic gene IRF4, potentially exert effects by promoting endogenous DNA damage. A cross-ancestry genome-wide association meta-analysis of lung cancer including 61,047 cases and 947,237 controls identifies five new cross-ancestry susceptibility loci and highlights ancestry-specific effects of common and rare variants on lung cancer risk.},
  archive      = {J_NG},
  author       = {Byun, Jinyoung and Han, Younghun and Li, Yafang and Xia, Jun and Long, Erping and Choi, Jiyeon and Xiao, Xiangjun and Zhu, Meng and Zhou, Wen and Sun, Ryan and Bossé, Yohan and Song, Zhuoyi and Schwartz, Ann and Lusk, Christine and Rafnar, Thorunn and Stefansson, Kari and Zhang, Tongwu and Zhao, Wei and Pettit, Rowland W. and Liu, Yanhong and Li, Xihao and Zhou, Hufeng and Walsh, Kyle M. and Gorlov, Ivan and Gorlova, Olga and Zhu, Dakai and Rosenberg, Susan M. and Pinney, Susan and Bailey-Wilson, Joan E. and Mandal, Diptasri and de Andrade, Mariza and Gaba, Colette and Willey, James C. and You, Ming and Anderson, Marshall and Wiencke, John K. and Albanes, Demetrius and Lam, Stephan and Tardon, Adonina and Chen, Chu and Goodman, Gary and Bojeson, Stig and Brenner, Hermann and Landi, Maria Teresa and Chanock, Stephen J. and Johansson, Mattias and Muley, Thomas and Risch, Angela and Wichmann, H.-Erich and Bickeböller, Heike and Christiani, David C. and Rennert, Gad and Arnold, Susanne and Field, John K. and Shete, Sanjay and Le Marchand, Loic and Melander, Olle and Brunnstrom, Hans and Liu, Geoffrey and Andrew, Angeline S. and Kiemeney, Lambertus A. and Shen, Hongbing and Zienolddiny, Shanbeh and Grankvist, Kjell and Johansson, Mikael and Caporaso, Neil and Cox, Angela and Hong, Yun-Chul and Yuan, Jian-Min and Lazarus, Philip and Schabath, Matthew B. and Aldrich, Melinda C. and Patel, Alpa and Lan, Qing and Rothman, Nathaniel and Taylor, Fiona and Kachuri, Linda and Witte, John S. and Sakoda, Lori C. and Spitz, Margaret and Brennan, Paul and Lin, Xihong and McKay, James and Hung, Rayjean J. and Amos, Christopher I.},
  doi          = {10.1038/s41588-022-01115-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1167-1177},
  shortjournal = {Nat. Genet.},
  title        = {Cross-ancestry genome-wide meta-analysis of 61,047 cases and 947,237 controls identifies new susceptibility loci contributing to lung cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome-wide analyses of 200,453 individuals yield new
insights into the causes and consequences of clonal hematopoiesis.
<em>NG</em>, <em>54</em>(8), 1155–1166. (<a
href="https://doi.org/10.1038/s41588-022-01121-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clonal hematopoiesis (CH), the clonal expansion of a blood stem cell and its progeny driven by somatic driver mutations, affects over a third of people, yet remains poorly understood. Here we analyze genetic data from 200,453 UK Biobank participants to map the landscape of inherited predisposition to CH, increasing the number of germline associations with CH in European-ancestry populations from 4 to 14. Genes at new loci implicate DNA damage repair (PARP1, ATM, CHEK2), hematopoietic stem cell migration/homing (CD164) and myeloid oncogenesis (SETBP1). Several associations were CH-subtype-specific including variants at TCL1A and CD164 that had opposite associations with DNMT3A- versus TET2-mutant CH, the two most common CH subtypes, proposing key roles for these two loci in CH development. Mendelian randomization analyses showed that smoking and longer leukocyte telomere length are causal risk factors for CH and that genetic predisposition to CH increases risks of myeloproliferative neoplasia, nonhematological malignancies, atrial fibrillation and blood epigenetic ageing. Analysis of whole-exome sequencing data from 200,453 UK Biobank participants identifies loci associated with clonal hematopoiesis and highlights causal links between clonal hematopoiesis and other traits.},
  archive      = {J_NG},
  author       = {Kar, Siddhartha P. and Quiros, Pedro M. and Gu, Muxin and Jiang, Tao and Mitchell, Jonathan and Langdon, Ryan and Iyer, Vivek and Barcena, Clea and Vijayabaskar, M. S. and Fabre, Margarete A. and Carter, Paul and Petrovski, Slavé and Burgess, Stephen and Vassiliou, George S.},
  doi          = {10.1038/s41588-022-01121-z},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1155-1166},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide analyses of 200,453 individuals yield new insights into the causes and consequences of clonal hematopoiesis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetics of the human microglia regulome refines alzheimer’s
disease risk loci. <em>NG</em>, <em>54</em>(8), 1145–1154. (<a
href="https://doi.org/10.1038/s41588-022-01149-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microglia are brain myeloid cells that play a critical role in neuroimmunity and the etiology of Alzheimer’s disease (AD), yet our understanding of how the genetic regulatory landscape controls microglial function and contributes to AD is limited. Here, we performed transcriptome and chromatin accessibility profiling in primary human microglia from 150 donors to identify genetically driven variation and cell-specific enhancer–promoter (E-P) interactions. Integrative fine-mapping analysis identified putative regulatory mechanisms for 21 AD risk loci, of which 18 were refined to a single gene, including 3 new candidate risk genes (KCNN4, FIBP and LRRC25). Transcription factor regulatory networks captured AD risk variation and identified SPI1 as a key putative regulator of microglia expression and AD risk. This comprehensive resource capturing variation in the human microglia regulome provides insights into the etiology of neurodegenerative disease. Transcriptomic and epigenomic profiling of human microglia identifies putative gene regulatory mechanisms for 21 Alzheimer’s disease (AD) risk loci. SPI1/PU.1 is nominated as a key regulator of microglia gene expression and AD risk.},
  archive      = {J_NG},
  author       = {Kosoy, Roman and Fullard, John F. and Zeng, Biao and Bendl, Jaroslav and Dong, Pengfei and Rahman, Samir and Kleopoulos, Steven P. and Shao, Zhiping and Girdhar, Kiran and Humphrey, Jack and de Paiva Lopes, Katia and Charney, Alexander W. and Kopell, Brian H. and Raj, Towfique and Bennett, David and Kellner, Christopher P. and Haroutunian, Vahram and Hoffman, Gabriel E. and Roussos, Panos},
  doi          = {10.1038/s41588-022-01149-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1145-1154},
  shortjournal = {Nat. Genet.},
  title        = {Genetics of the human microglia regulome refines alzheimer’s disease risk loci},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Systematic discovery and perturbation of regulatory genes in
human t cells reveals the architecture of immune networks. <em>NG</em>,
<em>54</em>(8), 1133–1144. (<a
href="https://doi.org/10.1038/s41588-022-01106-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene regulatory networks ensure that important genes are expressed at precise levels. When gene expression is sufficiently perturbed, it can lead to disease. To understand how gene expression disruptions percolate through a network, we must first map connections between regulatory genes and their downstream targets. However, we lack comprehensive knowledge of the upstream regulators of most genes. Here, we developed an approach for systematic discovery of upstream regulators of critical immune factors—IL2RA, IL-2 and CTLA4—in primary human T cells. Then, we mapped the network of the target genes of these regulators and putative cis-regulatory elements using CRISPR perturbations, RNA-seq and ATAC-seq. These regulators form densely interconnected networks with extensive feedback loops. Furthermore, this network is enriched for immune-associated disease variants and genes. These results provide insight into how immune-associated disease genes are regulated in T cells and broader principles about the structure of human gene regulatory networks. Pooled loss-of-function CRISPR screens in primary human T cells identify upstream regulators of IL2RA, IL-2 and CTLA4. Individual knockouts of 24 regulators of IL2RA define a central network enriched for genes associated with immune-mediated diseases.},
  archive      = {J_NG},
  author       = {Freimer, Jacob W. and Shaked, Oren and Naqvi, Sahin and Sinnott-Armstrong, Nasa and Kathiria, Arwa and Garrido, Christian M. and Chen, Amy F. and Cortez, Jessica T. and Greenleaf, William J. and Pritchard, Jonathan K. and Marson, Alexander},
  doi          = {10.1038/s41588-022-01106-y},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1133-1144},
  shortjournal = {Nat. Genet.},
  title        = {Systematic discovery and perturbation of regulatory genes in human t cells reveals the architecture of immune networks},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome-wide meta-analysis of insomnia prioritizes genes
associated with metabolic and psychiatric pathways. <em>NG</em>,
<em>54</em>(8), 1125–1132. (<a
href="https://doi.org/10.1038/s41588-022-01124-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Insomnia is a heritable, highly prevalent sleep disorder for which no sufficient treatment currently exists. Previous genome-wide association studies with up to 1.3 million subjects identified over 200 associated loci. This extreme polygenicity suggested that many more loci remain to be discovered. The current study almost doubled the sample size to 593,724 cases and 1,771,286 controls, thereby increasing statistical power, and identified 554 risk loci (including 364 novel loci). To capitalize on this large number of loci, we propose a novel strategy to prioritize genes using external biological resources and functional interactions between genes across risk loci. Of all 3,898 genes naively implicated from the risk loci, we prioritize 289 and find brain-tissue expression specificity and enrichment in specific gene sets of synaptic signaling functions and neuronal differentiation. We show that this novel gene prioritization strategy yields specific hypotheses on underlying mechanisms of insomnia that would have been missed by traditional approaches. Genome-wide association meta-analysis of insomnia in 593,724 cases and 1,771,286 controls identifies 554 risk loci and implicates specific biological pathways through gene prioritization.},
  archive      = {J_NG},
  author       = {Watanabe, Kyoko and Jansen, Philip R. and Savage, Jeanne E. and Nandakumar, Priyanka and Wang, Xin and Hinds, David A. and Gelernter, Joel and Levey, Daniel F. and Polimanti, Renato and Stein, Murray B. and Van Someren, Eus J. W. and Smit, August B. and Posthuma, Danielle},
  doi          = {10.1038/s41588-022-01124-w},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1125-1132},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide meta-analysis of insomnia prioritizes genes associated with metabolic and psychiatric pathways},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Differences in the genetic architecture of common and rare
variants in childhood, persistent and late-diagnosed attention-deficit
hyperactivity disorder. <em>NG</em>, <em>54</em>(8), 1117–1124. (<a
href="https://doi.org/10.1038/s41588-022-01143-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Attention-deficit hyperactivity disorder (ADHD) is a neurodevelopmental disorder with onset in childhood (childhood ADHD); two-thirds of affected individuals continue to have ADHD in adulthood (persistent ADHD), and sometimes ADHD is diagnosed in adulthood (late-diagnosed ADHD). We evaluated genetic differences among childhood (n = 14,878), persistent (n = 1,473) and late-diagnosed (n = 6,961) ADHD cases alongside 38,303 controls, and rare variant differences in 7,650 ADHD cases and 8,649 controls. We identified four genome-wide significant loci for childhood ADHD and one for late-diagnosed ADHD. We found increased polygenic scores for ADHD in persistent ADHD compared with the other two groups. Childhood ADHD had higher genetic overlap with hyperactivity and autism compared with late-diagnosed ADHD and the highest burden of rare protein-truncating variants in evolutionarily constrained genes. Late-diagnosed ADHD had a larger genetic overlap with depression than childhood ADHD and no increased burden in rare protein-truncating variants. Overall, these results suggest a genetic influence on age at first ADHD diagnosis, persistence of ADHD and the different comorbidity patterns among the groups. Analyses of the polygenic architecture of childhood, persistent and late-diagnosed attention-deficit hyperactivity disorder (ADHD) in a Danish population-based case–cohort sample identify differences among ADHD subgroups with respect to common and rare variants.},
  archive      = {J_NG},
  author       = {Rajagopal, Veera M. and Duan, Jinjie and Vilar-Ribó, Laura and Grove, Jakob and Zayats, Tetyana and Ramos-Quiroga, J. Antoni and Satterstrom, F. Kyle and Artigas, María Soler and Bybjerg-Grauholm, Jonas and Bækvad-Hansen, Marie and Als, Thomas D. and Rosengren, Anders and Daly, Mark J. and Neale, Benjamin M. and Nordentoft, Merete and Werge, Thomas and Mors, Ole and Hougaard, David M. and Mortensen, Preben B. and Ribasés, Marta and Børglum, Anders D. and Demontis, Ditte},
  doi          = {10.1038/s41588-022-01143-7},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1117-1124},
  shortjournal = {Nat. Genet.},
  title        = {Differences in the genetic architecture of common and rare variants in childhood, persistent and late-diagnosed attention-deficit hyperactivity disorder},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetic regulation of OAS1 nonsense-mediated decay underlies
association with COVID-19 hospitalization in patients of european and
african ancestries. <em>NG</em>, <em>54</em>(8), 1103–1116. (<a
href="https://doi.org/10.1038/s41588-022-01113-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The chr12q24.13 locus encoding OAS1–OAS3 antiviral proteins has been associated with coronavirus disease 2019 (COVID-19) susceptibility. Here, we report genetic, functional and clinical insights into this locus in relation to COVID-19 severity. In our analysis of patients of European (n = 2,249) and African (n = 835) ancestries with hospitalized versus nonhospitalized COVID-19, the risk of hospitalized disease was associated with a common OAS1 haplotype, which was also associated with reduced severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) clearance in a clinical trial with pegIFN-λ1. Bioinformatic analyses and in vitro studies reveal the functional contribution of two associated OAS1 exonic variants comprising the risk haplotype. Derived human-specific alleles rs10774671-A and rs1131454 -A decrease OAS1 protein abundance through allele-specific regulation of splicing and nonsense-mediated decay (NMD). We conclude that decreased OAS1 expression due to a common haplotype contributes to COVID-19 severity. Our results provide insight into molecular mechanisms through which early treatment with interferons could accelerate SARS-CoV-2 clearance and mitigate against severe COVID-19. Genetic and functional studies implicate allele-specific regulation of OAS1 splicing and nonsense-mediated decay in COVID-19 severity. The OAS1 risk haplotype is also associated with reduced SARS-CoV-2 clearance in a clinical trial with pegIFN-λ1.},
  archive      = {J_NG},
  author       = {Banday, A. Rouf and Stanifer, Megan L. and Florez-Vargas, Oscar and Onabajo, Olusegun O. and Papenberg, Brenen W. and Zahoor, Muhammad A. and Mirabello, Lisa and Ring, Timothy J. and Lee, Chia-Han and Albert, Paul S. and Andreakos, Evangelos and Arons, Evgeny and Barsh, Greg and Biesecker, Leslie G. and Boyle, David L. and Brahier, Mark S. and Burnett-Hartman, Andrea and Carrington, Mary and Chang, Euijin and Choe, Pyoeng Gyun and Chisholm, Rex L. and Colli, Leandro M. and Dalgard, Clifton L. and Dude, Carolynn M. and Edberg, Jeff and Erdmann, Nathan and Feigelson, Heather S. and Fonseca, Benedito A. and Firestein, Gary S. and Gehring, Adam J. and Guo, Cuncai and Ho, Michelle and Holland, Steven and Hutchinson, Amy A. and Im, Hogune and Irby, Les’Shon and Ison, Michael G. and Joseph, Naima T. and Kim, Hong Bin and Kreitman, Robert J. and Korf, Bruce R. and Lipkin, Steven M. and Mahgoub, Siham M. and Mohammed, Iman and Paschoalini, Guilherme L. and Pacheco, Jennifer A. and Peluso, Michael J. and Rader, Daniel J. and Redden, David T. and Ritchie, Marylyn D. and Rosenblum, Brooke and Ross, M. Elizabeth and Anna, Hanaisa P. Sant and Savage, Sharon A. and Sharma, Sudha and Siouti, Eleni and Smith, Alicia K. and Triantafyllia, Vasiliki and Vargas, Joselin M. and Vargas, Jose D. and Verma, Anurag and Vij, Vibha and Wesemann, Duane R. and Yeager, Meredith and Yu, Xu and Zhang, Yu and Boulant, Steeve and Chanock, Stephen J. and Feld, Jordan J. and Prokunina-Olsson, Ludmila},
  doi          = {10.1038/s41588-022-01113-z},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1103-1116},
  shortjournal = {Nat. Genet.},
  title        = {Genetic regulation of OAS1 nonsense-mediated decay underlies association with COVID-19 hospitalization in patients of european and african ancestries},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bidirectional genome-wide CRISPR screens reveal host factors
regulating SARS-CoV-2, MERS-CoV and seasonal HCoVs. <em>NG</em>,
<em>54</em>(8), 1090–1102. (<a
href="https://doi.org/10.1038/s41588-022-01110-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR knockout (KO) screens have identified host factors regulating severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) replication. Here, we conducted a meta-analysis of these screens, which showed a high level of cell-type specificity of the identified hits, highlighting the necessity of additional models to uncover the full landscape of host factors. Thus, we performed genome-wide KO and activation screens in Calu-3 lung cells and KO screens in Caco-2 colorectal cells, followed by secondary screens in four human cell lines. This revealed host-dependency factors, including AP1G1 adaptin and ATP8B1 flippase, as well as inhibitors, including mucins. Interestingly, some of the identified genes also modulate Middle East respiratory syndrome coronavirus (MERS-CoV) and seasonal human coronavirus (HCoV) (HCoV-NL63 and HCoV-229E) replication. Moreover, most genes had an impact on viral entry, with AP1G1 likely regulating TMPRSS2 activity at the plasma membrane. These results demonstrate the value of multiple cell models and perturbational modalities for understanding SARS-CoV-2 replication and provide a list of potential targets for therapeutic interventions. Genome-wide CRISPR knockout and activation screens in Calu-3 lung cells and Caco-2 colorectal cells identify host factors regulating SARS-CoV-2, MERS-CoV and seasonal HCoVs.},
  archive      = {J_NG},
  author       = {Rebendenne, Antoine and Roy, Priyanka and Bonaventure, Boris and Chaves Valadão, Ana Luiza and Desmarets, Lowiese and Arnaud-Arnould, Mary and Rouillé, Yves and Tauziet, Marine and Giovannini, Donatella and Touhami, Jawida and Lee, Yenarae and DeWeirdt, Peter and Hegde, Mudra and Urbach, Serge and Koulali, Khadija El and de Gracia, Francisco Garcia and McKellar, Joe and Dubuisson, Jean and Wencker, Mélanie and Belouzard, Sandrine and Moncorgé, Olivier and Doench, John G. and Goujon, Caroline},
  doi          = {10.1038/s41588-022-01110-2},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1090-1102},
  shortjournal = {Nat. Genet.},
  title        = {Bidirectional genome-wide CRISPR screens reveal host factors regulating SARS-CoV-2, MERS-CoV and seasonal HCoVs},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome-wide bidirectional CRISPR screens identify mucins as
host factors modulating SARS-CoV-2 infection. <em>NG</em>,
<em>54</em>(8), 1078–1089. (<a
href="https://doi.org/10.1038/s41588-022-01131-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) causes a range of symptoms in infected individuals, from mild respiratory illness to acute respiratory distress syndrome. A systematic understanding of host factors influencing viral infection is critical to elucidate SARS-CoV-2–host interactions and the progression of Coronavirus disease 2019 (COVID-19). Here, we conducted genome-wide CRISPR knockout and activation screens in human lung epithelial cells with endogenous expression of the SARS-CoV-2 entry factors ACE2 and TMPRSS2. We uncovered proviral and antiviral factors across highly interconnected host pathways, including clathrin transport, inflammatory signaling, cell-cycle regulation, and transcriptional and epigenetic regulation. We further identified mucins, a family of high molecular weight glycoproteins, as a prominent viral restriction network that inhibits SARS-CoV-2 infection in vitro and in murine models. These mucins also inhibit infection of diverse respiratory viruses. This functional landscape of SARS-CoV-2 host factors provides a physiologically relevant starting point for new host-directed therapeutics and highlights airway mucins as a host defense mechanism. Genome-wide CRISPR knockout and activation screens in human lung epithelial cells with endogenous expression of the SARS-CoV-2 entry factors ACE2 and TMPRSS2 identify mucins as key host factors restricting viral infection.},
  archive      = {J_NG},
  author       = {Biering, Scott B. and Sarnik, Sylvia A. and Wang, Eleanor and Zengel, James R. and Leist, Sarah R. and Schäfer, Alexandra and Sathyan, Varun and Hawkins, Padraig and Okuda, Kenichi and Tau, Cyrus and Jangid, Aditya R. and Duffy, Connor V. and Wei, Jin and Gilmore, Rodney C. and Alfajaro, Mia Madel and Strine, Madison S. and Nguyenla, Xammy and Van Dis, Erik and Catamura, Carmelle and Yamashiro, Livia H. and Belk, Julia A. and Begeman, Adam and Stark, Jessica C. and Shon, D. Judy and Fox, Douglas M. and Ezzatpour, Shahrzad and Huang, Emily and Olegario, Nico and Rustagi, Arjun and Volmer, Allison S. and Livraghi-Butrico, Alessandra and Wehri, Eddie and Behringer, Richard R. and Cheon, Dong-Joo and Schaletzky, Julia and Aguilar, Hector C. and Puschnik, Andreas S. and Button, Brian and Pinsky, Benjamin A. and Blish, Catherine A. and Baric, Ralph S. and O’Neal, Wanda K. and Bertozzi, Carolyn R. and Wilen, Craig B. and Boucher, Richard C. and Carette, Jan E. and Stanley, Sarah A. and Harris, Eva and Konermann, Silvana and Hsu, Patrick D.},
  doi          = {10.1038/s41588-022-01131-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1078-1089},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide bidirectional CRISPR screens identify mucins as host factors modulating SARS-CoV-2 infection},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Transient activated fibroblasts contribute to zebrafish
heart regeneration. <em>NG</em>, <em>54</em>(8), 1076–1077. (<a
href="https://doi.org/10.1038/s41588-022-01130-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the zebrafish heart, several transient fibroblast types appear after injury. High-throughput lineage tracing revealed that injury-responsive fibroblasts are derived from two distinct lineage origins: the epicardium and the endocardium. Targeted cell-type-specific depletion showed that at least one fibroblast type has a critical role in heart regeneration.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01130-y},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1076-1077},
  shortjournal = {Nat. Genet.},
  title        = {Transient activated fibroblasts contribute to zebrafish heart regeneration},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome origin and evolution of common oat. <em>NG</em>,
<em>54</em>(8), 1074–1075. (<a
href="https://doi.org/10.1038/s41588-022-01133-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Assemblies of hexaploid cultivated oat, and of close relatives of its diploid and tetraploid progenitors, have revealed its polyploid formation and subgenome evolution. These high-quality oat reference genomes will facilitate the discovery of candidate genes that underlie beneficial traits such as hulless grain and disease resistance.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01133-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1074-1075},
  shortjournal = {Nat. Genet.},
  title        = {Genome origin and evolution of common oat},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). New genetic loci associated with the risk of clonal
hematopoiesis. <em>NG</em>, <em>54</em>(8), 1072–1073. (<a
href="https://doi.org/10.1038/s41588-022-01125-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic and phenotypic analyses of data from over 400,000 participants in the UK Biobank identified 10 new loci associated with the development of clonal hematopoiesis and implicated DNA damage, oncogene signaling, telomere maintenance and blood cell homing in its pathogenesis. These findings can help to decipher the pathogenesis of clonal hematopoiesis and develop therapeutic approaches.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01125-9},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1072-1073},
  shortjournal = {Nat. Genet.},
  title        = {New genetic loci associated with the risk of clonal hematopoiesis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mapping the regulators of key immune genes and disease
networks. <em>NG</em>, <em>54</em>(8), 1070–1071. (<a
href="https://doi.org/10.1038/s41588-022-01107-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Systematic CRISPR screens in primary human T cells uncovered the upstream regulators of crucial immune genes — IL2RA, IL2 and CTLA4. Then, using RNA-seq and ATAC-seq in knockout T cells, we mapped the downstream target genes and non-coding cis-regulatory elements of key regulators, thereby revealing a regulatory network enriched for immune disease-associated genes.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01107-x},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1070-1071},
  shortjournal = {Nat. Genet.},
  title        = {Mapping the regulators of key immune genes and disease networks},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Endogenous retroviruses steer transcriptional condensates
away from pluripotency. <em>NG</em>, <em>54</em>(8), 1068–1069. (<a
href="https://doi.org/10.1038/s41588-022-01111-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The impact of endogenous retrovirus silencing during mammalian development is poorly understood. A new study shows that their abnormal reactivation in pluripotent cells dismantles key gene regulatory networks by perturbing transcriptional condensates linked to super-enhancer function.},
  archive      = {J_NG},
  author       = {Grow, Edward J.},
  doi          = {10.1038/s41588-022-01111-1},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1068-1069},
  shortjournal = {Nat. Genet.},
  title        = {Endogenous retroviruses steer transcriptional condensates away from pluripotency},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mapping the cancer cell states conserved across solid
tumors. <em>NG</em>, <em>54</em>(8), 1066–1067. (<a
href="https://doi.org/10.1038/s41588-022-01151-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study uses single-cell and spatial transcriptomics to provide a systematic characterization of the recurrent gene-expression programs that control neoplastic cell states in diverse cancers.},
  archive      = {J_NG},
  author       = {Roden, Daniel and Swarbrick, Alexander},
  doi          = {10.1038/s41588-022-01151-7},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1066-1067},
  shortjournal = {Nat. Genet.},
  title        = {Mapping the cancer cell states conserved across solid tumors},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Supporting undiagnosed participants when clinical genomics
studies end. <em>NG</em>, <em>54</em>(8), 1063–1065. (<a
href="https://doi.org/10.1038/s41588-022-01150-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many large research initiatives have cumulatively enrolled thousands of patients with a range of complex medical issues but no clear genetic etiology. However, it is unclear how researchers, institutions and funders should manage the data and relationships with those participants who remain undiagnosed when these studies end. In this Comment, we outline the current literature relevant to post-study obligations in clinical genomics research and discuss the application of current guidelines to research with undiagnosed participants.},
  archive      = {J_NG},
  author       = {Halley, Meghan C. and Ashley, Euan A. and Tabor, Holly K.},
  doi          = {10.1038/s41588-022-01150-8},
  journal      = {Nature Genetics},
  month        = {8},
  number       = {8},
  pages        = {1063-1065},
  shortjournal = {Nat. Genet.},
  title        = {Supporting undiagnosed participants when clinical genomics studies end},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Publisher correction: UTX-mediated enhancer and chromatin
remodeling suppresses myeloid leukemogenesis through noncatalytic
inverse regulation of ETS and GATA programs. <em>NG</em>,
<em>54</em>(7), 1062. (<a
href="https://doi.org/10.1038/s41588-022-01060-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Gozdecka, Malgorzata and Meduri, Eshwar and Mazan, Milena and Tzelepis, Konstantinos and Dudek, Monika and Knights, Andrew J. and Pardo, Mercedes and Yu, Lu and Choudhary, Jyoti S. and Metzakopian, Emmanouil and Iyer, Vivek and Yun, Haiyang and Park, Naomi and Varela, Ignacio and Bautista, Ruben and Collord, Grace and Dovey, Oliver and Garyfallos, Dimitrios A. and De Braekeleer, Etienne and Kondo, Saki and Cooper, Jonathan and Göttgens, Berthold and Bullinger, Lars and Northcott, Paul A. and Adams, David and Vassiliou, George S. and Huntly, Brian J. P.},
  doi          = {10.1038/s41588-022-01060-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1062},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: UTX-mediated enhancer and chromatin remodeling suppresses myeloid leukemogenesis through noncatalytic inverse regulation of ETS and GATA programs},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: A genome-wide association study with
1,126,563 individuals identifies new risk loci for alzheimer’s disease.
<em>NG</em>, <em>54</em>(7), 1062. (<a
href="https://doi.org/10.1038/s41588-022-01126-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Wightman, Douglas P. and Jansen, Iris E. and Savage, Jeanne E. and Shadrin, Alexey A. and Bahrami, Shahram and Holland, Dominic and Rongve, Arvid and Børte, Sigrid and Winsvold, Bendik S. and Drange, Ole Kristian and Martinsen, Amy E. and Skogholt, Anne Heidi and Willer, Cristen and Bråthen, Geir and Bosnes, Ingunn and Nielsen, Jonas Bille and Fritsche, Lars G. and Thomas, Laurent F. and Pedersen, Linda M. and Gabrielsen, Maiken E. and Johnsen, Marianne Bakke and Meisingset, Tore Wergeland and Zhou, Wei and Proitsi, Petroula and Hodges, Angela and Dobson, Richard and Velayudhan, Latha and Heilbron, Karl and Auton, Adam and Sealock, Julia M. and Davis, Lea K. and Pedersen, Nancy L. and Reynolds, Chandra A. and Karlsson, Ida K. and Magnusson, Sigurdur and Stefansson, Hreinn and Thordardottir, Steinunn and Jonsson, Palmi V. and Snaedal, Jon and Zettergren, Anna and Skoog, Ingmar and Kern, Silke and Waern, Margda and Zetterberg, Henrik and Blennow, Kaj and Stordal, Eystein and Hveem, Kristian and Zwart, John-Anker and Athanasiu, Lavinia and Selnes, Per and Saltvedt, Ingvild and Sando, Sigrid B. and Ulstein, Ingun and Djurovic, Srdjan and Fladby, Tormod and Aarsland, Dag and Selbæk, Geir and Ripke, Stephan and Stefansson, Kari and Andreassen, Ole A. and Posthuma, Danielle},
  doi          = {10.1038/s41588-022-01126-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1062},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: A genome-wide association study with 1,126,563 individuals identifies new risk loci for alzheimer’s disease},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Systematic identification of cell-fate regulatory programs
using a single-cell atlas of mouse development. <em>NG</em>,
<em>54</em>(7), 1051–1061. (<a
href="https://doi.org/10.1038/s41588-022-01118-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Waddington’s epigenetic landscape is a metaphor frequently used to illustrate cell differentiation. Recent advances in single-cell genomics are altering our understanding of the Waddington landscape, yet the molecular mechanisms of cell-fate decisions remain poorly understood. We constructed a cell landscape of mouse lineage differentiation during development at the single-cell level and described both lineage-common and lineage-specific regulatory programs during cell-type maturation. We also found lineage-common regulatory programs that are broadly active during the development of invertebrates and vertebrates. In particular, we identified Xbp1 as an evolutionarily conserved regulator of cell-fate determinations across different species. We demonstrated that Xbp1 transcriptional regulation is important for the stabilization of the gene-regulatory networks for a wide range of mouse cell types. Our results offer genetic and molecular insights into cellular gene-regulatory programs and will serve as a basis for further advancing the understanding of cell-fate decisions. Single-cell RNA-sequencing of seven mouse developmental stages identifies lineage-specific and shared regulatory programs controlling cell-fate decisions. Cross-species analysis associates differentiation potency with ribosomal protein gene expression.},
  archive      = {J_NG},
  author       = {Fei, Lijiang and Chen, Haide and Ma, Lifeng and E, Weigao and Wang, Renying and Fang, Xing and Zhou, Ziming and Sun, Huiyu and Wang, Jingjing and Jiang, Mengmeng and Wang, Xinru and Yu, Chengxuan and Mei, Yuqing and Jia, Danmei and Zhang, Tingyue and Han, Xiaoping and Guo, Guoji},
  doi          = {10.1038/s41588-022-01118-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1051-1061},
  shortjournal = {Nat. Genet.},
  title        = {Systematic identification of cell-fate regulatory programs using a single-cell atlas of mouse development},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multiomic atlas with functional stratification and
developmental dynamics of zebrafish cis-regulatory elements.
<em>NG</em>, <em>54</em>(7), 1037–1050. (<a
href="https://doi.org/10.1038/s41588-022-01089-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zebrafish, a popular organism for studying embryonic development and for modeling human diseases, has so far lacked a systematic functional annotation program akin to those in other animal models. To address this, we formed the international DANIO-CODE consortium and created a central repository to store and process zebrafish developmental functional genomic data. Our data coordination center ( https://danio-code.zfin.org ) combines a total of 1,802 sets of unpublished and re-analyzed published genomic data, which we used to improve existing annotations and show its utility in experimental design. We identified over 140,000 cis-regulatory elements throughout development, including classes with distinct features dependent on their activity in time and space. We delineated the distinct distance topology and chromatin features between regulatory elements active during zygotic genome activation and those active during organogenesis. Finally, we matched regulatory elements and epigenomic landscapes between zebrafish and mouse and predicted functional relationships between them beyond sequence similarity, thus extending the utility of zebrafish developmental genomics to mammals. The DANIO-CODE consortium leverages a large-scale multiomic dataset to improve zebrafish genome annotation. They identify ~140,000 cis-regulatory elements throughout development and perform a comparison with the mouse regulatory landscape.},
  archive      = {J_NG},
  author       = {Baranasic, Damir and Hörtenhuber, Matthias and Balwierz, Piotr J. and Zehnder, Tobias and Mukarram, Abdul Kadir and Nepal, Chirag and Várnai, Csilla and Hadzhiev, Yavor and Jimenez-Gonzalez, Ada and Li, Nan and Wragg, Joseph and D’Orazio, Fabio M. and Relic, Dorde and Pachkov, Mikhail and Díaz, Noelia and Hernández-Rodríguez, Benjamín and Chen, Zelin and Stoiber, Marcus and Dong, Michaël and Stevens, Irene and Ross, Samuel E. and Eagle, Anne and Martin, Ryan and Obasaju, Oluwapelumi and Rastegar, Sepand and McGarvey, Alison C. and Kopp, Wolfgang and Chambers, Emily and Wang, Dennis and Kim, Hyejeong R. and Acemel, Rafael D. and Naranjo, Silvia and Łapiński, Maciej and Chong, Vanessa and Mathavan, Sinnakaruppan and Peers, Bernard and Sauka-Spengler, Tatjana and Vingron, Martin and Carninci, Piero and Ohler, Uwe and Lacadie, Scott Allen and Burgess, Shawn M. and Winata, Cecilia and van Eeden, Freek and Vaquerizas, Juan M. and Gómez-Skarmeta, José Luis and Onichtchouk, Daria and Brown, Ben James and Bogdanovic, Ozren and van Nimwegen, Erik and Westerfield, Monte and Wardle, Fiona C. and Daub, Carsten O. and Lenhard, Boris and Müller, Ferenc},
  doi          = {10.1038/s41588-022-01089-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1037-1050},
  shortjournal = {Nat. Genet.},
  title        = {Multiomic atlas with functional stratification and developmental dynamics of zebrafish cis-regulatory elements},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). In vivo dissection of a clustered-CTCF domain boundary
reveals developmental principles of regulatory insulation. <em>NG</em>,
<em>54</em>(7), 1026–1036. (<a
href="https://doi.org/10.1038/s41588-022-01117-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Vertebrate genomes organize into topologically associating domains, delimited by boundaries that insulate regulatory elements from nontarget genes. However, how boundary function is established is not well understood. Here, we combine genome-wide analyses and transgenic mouse assays to dissect the regulatory logic of clustered-CCCTC-binding factor (CTCF) boundaries in vivo, interrogating their function at multiple levels: chromatin interactions, transcription and phenotypes. Individual CTCF binding site (CBS) deletions revealed that the characteristics of specific sites can outweigh other factors such as CBS number and orientation. Combined deletions demonstrated that CBSs cooperate redundantly and provide boundary robustness. We show that divergent CBS signatures are not strictly required for effective insulation and that chromatin loops formed by nonconvergently oriented sites could be mediated by a loop interference mechanism. Further, we observe that insulation strength constitutes a quantitative modulator of gene expression and phenotypes. Our results highlight the modular nature of boundaries and their control over developmental processes. Genetically dissecting the Epha4–Pax3 topological boundary in mice shows that divergent CTCF binding sites (CBSs) are not essential for insulation and that chromatin loops in nonconvergently oriented CBSs can be driven by a loop interference mechanism.},
  archive      = {J_NG},
  author       = {Anania, Chiara and Acemel, Rafael D. and Jedamzick, Johanna and Bolondi, Adriano and Cova, Giulia and Brieske, Norbert and Kühn, Ralf and Wittler, Lars and Real, Francisca M. and Lupiáñez, Darío G.},
  doi          = {10.1038/s41588-022-01117-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1026-1036},
  shortjournal = {Nat. Genet.},
  title        = {In vivo dissection of a clustered-CTCF domain boundary reveals developmental principles of regulatory insulation},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DeepLoop robustly maps chromatin interactions from sparse
allele-resolved or single-cell hi-c data at kilobase resolution.
<em>NG</em>, <em>54</em>(7), 1013–1025. (<a
href="https://doi.org/10.1038/s41588-022-01116-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mapping chromatin loops from noisy Hi-C heatmaps remains a major challenge. Here we present DeepLoop, which performs rigorous bias correction followed by deep-learning-based signal enhancement for robust chromatin interaction mapping from low-depth Hi-C data. DeepLoop enables loop-resolution, single-cell Hi-C analysis. It also achieves a cross-platform convergence between different Hi-C protocols and micrococcal nuclease (micro-C). DeepLoop allowed us to map the genetic and epigenetic determinants of allele-specific chromatin interactions in the human genome. We nominate new loci with allele-specific interactions governed by imprinting or allelic DNA methylation. We also discovered that, in the inactivated X chromosome (Xi), local loops at the DXZ4 ‘megadomain’ boundary escape X-inactivation but the FIRRE ‘superloop’ locus does not. Importantly, DeepLoop can pinpoint heterozygous single-nucleotide polymorphisms and large structure variants that cause allelic chromatin loops, many of which rewire enhancers with transcription consequences. Taken together, DeepLoop expands the use of Hi-C to provide loop-resolution insights into the genetics of the three-dimensional genome. DeepLoop is a modular Hi-C processing workflow that enables kilobase-resolution analysis of sparse data. Reanalysis of published data demonstrates that DeepLoop can identify allele-specific chromatin loops and large heterozygous structural variants.},
  archive      = {J_NG},
  author       = {Zhang, Shanshan and Plummer, Dylan and Lu, Leina and Cui, Jian and Xu, Wanying and Wang, Miao and Liu, Xiaoxiao and Prabhakar, Nachiketh and Shrinet, Jatin and Srinivasan, Divyaa and Fraser, Peter and Li, Yan and Li, Jing and Jin, Fulai},
  doi          = {10.1038/s41588-022-01116-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {1013-1025},
  shortjournal = {Nat. Genet.},
  title        = {DeepLoop robustly maps chromatin interactions from sparse allele-resolved or single-cell hi-C data at kilobase resolution},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Functional landscapes of POLE and POLD1 mutations in
checkpoint blockade-dependent antitumor immunity. <em>NG</em>,
<em>54</em>(7), 996–1012. (<a
href="https://doi.org/10.1038/s41588-022-01108-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Defects in pathways governing genomic fidelity have been linked to improved response to immune checkpoint blockade therapy (ICB). Pathogenic POLE/POLD1 mutations can cause hypermutation, yet how diverse mutations in POLE/POLD1 influence antitumor immunity following ICB is unclear. Here, we comprehensively determined the effect of POLE/POLD1 mutations in ICB and elucidated the mechanistic impact of these mutations on tumor immunity. Murine syngeneic tumors harboring Pole/Pold1 functional mutations displayed enhanced antitumor immunity and were sensitive to ICB. Patients with POLE/POLD1 mutated tumors harboring telltale mutational signatures respond better to ICB than patients harboring wild-type or signature-negative tumors. A mutant POLE/D1 function-associated signature-based model outperformed several traditional approaches for identifying POLE/POLD1 mutated patients that benefit from ICB. Strikingly, the spectrum of mutational signatures correlates with the biochemical features of neoantigens. Alterations that cause POLE/POLD1 function-associated signatures generate T cell receptor (TCR)-contact residues with increased hydrophobicity, potentially facilitating T cell recognition. Altogether, the functional landscapes of POLE/POLD1 mutations shape immunotherapy efficacy. An analysis of POLE and POLD1 mutations distinguishes driver mutations from passengers and explores their functionality. Driver mutations are associated with specific mutational signatures and correlate with immune checkpoint blockage response.},
  archive      = {J_NG},
  author       = {Ma, Xiaoxiao and Riaz, Nadeem and Samstein, Robert M. and Lee, Mark and Makarov, Vladimir and Valero, Cristina and Chowell, Diego and Kuo, Fengshen and Hoen, Douglas and Fitzgerald, Conall W. R. and Jiang, Hui and Alektiar, Jonathan and Alban, Tyler J. and Juric, Ivan and Parthasarathy, Prerana Bangalore and Zhao, Yu and Sabio, Erich Y. and Verma, Richa and Srivastava, Raghvendra M. and Vuong, Lynda and Yang, Wei and Zhang, Xiao and Wang, Jingming and Chu, Lawrence K. and Wang, Stephen L. and Kelly, Daniel W. and Pei, Xin and Chen, Jiapeng and Yaeger, Rona and Zamarin, Dmitriy and Zehir, Ahmet and Gönen, Mithat and Morris, Luc G. T. and Chan, Timothy A.},
  doi          = {10.1038/s41588-022-01108-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {996-1012},
  shortjournal = {Nat. Genet.},
  title        = {Functional landscapes of POLE and POLD1 mutations in checkpoint blockade-dependent antitumor immunity},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell analyses define a continuum of cell state and
composition changes in the malignant transformation of polyps to
colorectal cancer. <em>NG</em>, <em>54</em>(7), 985–995. (<a
href="https://doi.org/10.1038/s41588-022-01088-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To chart cell composition and cell state changes that occur during the transformation of healthy colon to precancerous adenomas to colorectal cancer (CRC), we generated single-cell chromatin accessibility profiles and single-cell transcriptomes from 1,000 to 10,000 cells per sample for 48 polyps, 27 normal tissues and 6 CRCs collected from patients with or without germline APC mutations. A large fraction of polyp and CRC cells exhibit a stem-like phenotype, and we define a continuum of epigenetic and transcriptional changes occurring in these stem-like cells as they progress from homeostasis to CRC. Advanced polyps contain increasing numbers of stem-like cells, regulatory T cells and a subtype of pre-cancer-associated fibroblasts. In the cancerous state, we observe T cell exhaustion, RUNX1-regulated cancer-associated fibroblasts and increasing accessibility associated with HNF4A motifs in epithelia. DNA methylation changes in sporadic CRC are strongly anti-correlated with accessibility changes along this continuum, further identifying regulatory markers for molecular staging of polyps. Single-cell ATAC-seq and RNA-seq profiling traces the transformation of healthy colon to precancerous adenomas to colorectal cancer (CRC). A large proportion of polyp and CRC cells show a stem-like phenotype.},
  archive      = {J_NG},
  author       = {Becker, Winston R. and Nevins, Stephanie A. and Chen, Derek C. and Chiu, Roxanne and Horning, Aaron M. and Guha, Tuhin K. and Laquindanum, Rozelle and Mills, Meredith and Chaib, Hassan and Ladabaum, Uri and Longacre, Teri and Shen, Jeanne and Esplin, Edward D. and Kundaje, Anshul and Ford, James M. and Curtis, Christina and Snyder, Michael P. and Greenleaf, William J.},
  doi          = {10.1038/s41588-022-01088-x},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {985-995},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell analyses define a continuum of cell state and composition changes in the malignant transformation of polyps to colorectal cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A modified fluctuation-test framework characterizes the
population dynamics and mutation rate of colorectal cancer persister
cells. <em>NG</em>, <em>54</em>(7), 976–984. (<a
href="https://doi.org/10.1038/s41588-022-01105-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Compelling evidence shows that cancer persister cells represent a major limit to the long-term efficacy of targeted therapies. However, the phenotype and population dynamics of cancer persister cells remain unclear. We developed a quantitative framework to study persisters by combining experimental characterization and mathematical modeling. We found that, in colorectal cancer, a fraction of persisters slowly replicates. Clinically approved targeted therapies induce a switch to drug-tolerant persisters and a temporary 7- to 50-fold increase of their mutation rate, thus increasing the number of persister-derived resistant cells. These findings reveal that treatment may influence persistence and mutability in cancer cells and pinpoint inhibition of error-prone DNA polymerases as a strategy to restrict tumor recurrence. A modified fluctuation test applied to colorectal cancer cells shows that EGFR/BRAF inhibitor-induced persisters slowly proliferate and have an increased mutation rate. Error-prone DNA polymerases are identified as potential targets to avoid tumor recurrence following treatment with these drugs.},
  archive      = {J_NG},
  author       = {Russo, Mariangela and Pompei, Simone and Sogari, Alberto and Corigliano, Mattia and Crisafulli, Giovanni and Puliafito, Alberto and Lamba, Simona and Erriquez, Jessica and Bertotti, Andrea and Gherardi, Marco and Di Nicolantonio, Federica and Bardelli, Alberto and Cosentino Lagomarsino, Marco},
  doi          = {10.1038/s41588-022-01105-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {976-984},
  shortjournal = {Nat. Genet.},
  title        = {A modified fluctuation-test framework characterizes the population dynamics and mutation rate of colorectal cancer persister cells},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell and bulk transcriptome sequencing identifies two
epithelial tumor cell states and refines the consensus molecular
classification of colorectal cancer. <em>NG</em>, <em>54</em>(7),
963–975. (<a href="https://doi.org/10.1038/s41588-022-01100-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The consensus molecular subtype (CMS) classification of colorectal cancer is based on bulk transcriptomics. The underlying epithelial cell diversity remains unclear. We analyzed 373,058 single-cell transcriptomes from 63 patients, focusing on 49,155 epithelial cells. We identified a pervasive genetic and transcriptomic dichotomy of malignant cells, based on distinct gene expression, DNA copy number and gene regulatory network. We recapitulated these subtypes in bulk transcriptomes from 3,614 patients. The two intrinsic subtypes, iCMS2 and iCMS3, refine CMS. iCMS3 comprises microsatellite unstable (MSI-H) cancers and one-third of microsatellite-stable (MSS) tumors. iCMS3 MSS cancers are transcriptomically more similar to MSI-H cancers than to other MSS cancers. CMS4 cancers had either iCMS2 or iCMS3 epithelium; the latter had the worst prognosis. We defined the intrinsic epithelial axis of colorectal cancer and propose a refined ‘IMF’ classification with five subtypes, combining intrinsic epithelial subtype (I), microsatellite instability status (M) and fibrosis (F). A single-cell transcriptomic analysis of 63 patients with colorectal cancer classifies tumor cells into two epithelial subtypes. An improved tumor classification based on epithelial subtype, microsatellite stability and fibrosis reveals differences in pathway activation and metastasis.},
  archive      = {J_NG},
  author       = {Joanito, Ignasius and Wirapati, Pratyaksha and Zhao, Nancy and Nawaz, Zahid and Yeo, Grace and Lee, Fiona and Eng, Christine L. P. and Macalinao, Dominique Camat and Kahraman, Merve and Srinivasan, Harini and Lakshmanan, Vairavan and Verbandt, Sara and Tsantoulis, Petros and Gunn, Nicole and Venkatesh, Prasanna Nori and Poh, Zhong Wee and Nahar, Rahul and Oh, Hsueh Ling Janice and Loo, Jia Min and Chia, Shumei and Cheow, Lih Feng and Cheruba, Elsie and Wong, Michael Thomas and Kua, Lindsay and Chua, Clarinda and Nguyen, Andy and Golovan, Justin and Gan, Anna and Lim, Wan-Jun and Guo, Yu Amanda and Yap, Choon Kong and Tay, Brenda and Hong, Yourae and Chong, Dawn Qingqing and Chok, Aik-Yong and Park, Woong-Yang and Han, Shuting and Chang, Mei Huan and Seow-En, Isaac and Fu, Cherylin and Mathew, Ronnie and Toh, Ee-Lin and Hong, Lewis Z. and Skanderup, Anders Jacobsen and DasGupta, Ramanuj and Ong, Chin-Ann Johnny and Lim, Kiat Hon and Tan, Emile K. W. and Koo, Si-Lin and Leow, Wei Qiang and Tejpar, Sabine and Prabhakar, Shyam and Tan, Iain Beehuat},
  doi          = {10.1038/s41588-022-01100-4},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {963-975},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell and bulk transcriptome sequencing identifies two epithelial tumor cell states and refines the consensus molecular classification of colorectal cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Epigenomic and transcriptomic analyses define core cell
types, genes and targetable mechanisms for kidney disease. <em>NG</em>,
<em>54</em>(7), 950–962. (<a
href="https://doi.org/10.1038/s41588-022-01097-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {More than 800 million people suffer from kidney disease, yet the mechanism of kidney dysfunction is poorly understood. In the present study, we define the genetic association with kidney function in 1.5 million individuals and identify 878 (126 new) loci. We map the genotype effect on the methylome in 443 kidneys, transcriptome in 686 samples and single-cell open chromatin in 57,229 kidney cells. Heritability analysis reveals that methylation variation explains a larger fraction of heritability than gene expression. We present a multi-stage prioritization strategy and prioritize target genes for 87% of kidney function loci. We highlight key roles of proximal tubules and metabolism in kidney function regulation. Furthermore, the causal role of SLC47A1 in kidney disease is defined in mice with genetic loss of Slc47a1 and in human individuals carrying loss-of-function variants. Our findings emphasize the key role of bulk and single-cell epigenomic information in translating genome-wide association studies into identifying causal genes, cellular origins and mechanisms of complex traits. Genome-wide analyses identify hundreds of loci associated with kidney function. Integrated analyses of expression, methylation and single-cell open chromatin and expression data derived from human kidney samples prioritize genes and mechanisms underlying renal disease.},
  archive      = {J_NG},
  author       = {Liu, Hongbo and Doke, Tomohito and Guo, Dong and Sheng, Xin and Ma, Ziyuan and Park, Joseph and Vy, Ha My T. and Nadkarni, Girish N. and Abedini, Amin and Miao, Zhen and Palmer, Matthew and Voight, Benjamin F. and Li, Hongzhe and Brown, Christopher D. and Ritchie, Marylyn D. and Shu, Yan and Susztak, Katalin},
  doi          = {10.1038/s41588-022-01097-w},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {950-962},
  shortjournal = {Nat. Genet.},
  title        = {Epigenomic and transcriptomic analyses define core cell types, genes and targetable mechanisms for kidney disease},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A sequence-based global map of regulatory activity for
deciphering human genetics. <em>NG</em>, <em>54</em>(7), 940–949. (<a
href="https://doi.org/10.1038/s41588-022-01102-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Epigenomic profiling has enabled large-scale identification of regulatory elements, yet we still lack a systematic mapping from any sequence or variant to regulatory activities. We address this challenge with Sei, a framework for integrating human genetics data with sequence information to discover the regulatory basis of traits and diseases. Sei learns a vocabulary of regulatory activities, called sequence classes, using a deep learning model that predicts 21,907 chromatin profiles across &amp;gt;1,300 cell lines and tissues. Sequence classes provide a global classification and quantification of sequence and variant effects based on diverse regulatory activities, such as cell type-specific enhancer functions. These predictions are supported by tissue-specific expression, expression quantitative trait loci and evolutionary constraint data. Furthermore, sequence classes enable characterization of the tissue-specific, regulatory architecture of complex traits and generate mechanistic hypotheses for individual regulatory pathogenic mutations. We provide Sei as a resource to elucidate the regulatory basis of human health and disease. Sei is a new framework for integrating human genetics data with a sequence-based mapping of predicted regulatory activities to elucidate mechanisms contributing to complex traits and diseases.},
  archive      = {J_NG},
  author       = {Chen, Kathleen M. and Wong, Aaron K. and Troyanskaya, Olga G. and Zhou, Jian},
  doi          = {10.1038/s41588-022-01102-2},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {940-949},
  shortjournal = {Nat. Genet.},
  title        = {A sequence-based global map of regulatory activity for deciphering human genetics},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). From mendel to quantitative genetics in the genome era: The
scientific legacy of w. G. hill. <em>NG</em>, <em>54</em>(7), 934–939.
(<a href="https://doi.org/10.1038/s41588-022-01103-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The quantitative geneticist W. G. (‘Bill’) Hill, awardee of the 2018 Darwin Medal of the Royal Society and the 2019 Mendel Medal of the Genetics Society (United Kingdom), died on 17 December 2021 at the age of 81 years. Here, we pay tribute to his multiple key scientific contributions, which span population and evolutionary genetics, animal and plant breeding and human genetics. We discuss his theoretical research on the role of linkage disequilibrium (LD) and mutational variance in the response to selection, the origin of the widely used LD metric r2 in genomic association studies, the genetic architecture of complex traits, the quantification of the variation in realized relationships given a pedigree relationship and much more. We demonstrate that basic theoretical research in quantitative and statistical genetics has led to profound insights into the genetics and evolution of complex traits and made predictions that were subsequently empirically validated, often decades later. This Perspective celebrates the scientific legacy of W. G. Hill and highlights his various important contributions to the field of genetics.},
  archive      = {J_NG},
  author       = {Charlesworth, Brian and Goddard, Michael E. and Meyer, Karin and Visscher, Peter M. and Weir, Bruce S. and Wray, Naomi R.},
  doi          = {10.1038/s41588-022-01103-1},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {934-939},
  shortjournal = {Nat. Genet.},
  title        = {From mendel to quantitative genetics in the genome era: The scientific legacy of w. g. hill},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). How did mendel arrive at his discoveries? <em>NG</em>,
<em>54</em>(7), 926–933. (<a
href="https://doi.org/10.1038/s41588-022-01109-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There are few historical records concerning Gregor Johann Mendel and his work, so theories abound concerning his motivation. These theories range from Fisher’s view that Mendel was testing a fully formed previous theory of inheritance to Olby’s view that Mendel was not interested in inheritance at all, whereas textbooks often state his motivation was to understand inheritance. In this Perspective, we review current ideas about how Mendel arrived at his discoveries and then discuss an alternative scenario based on recently discovered historical sources that support the suggestion that Mendel’s fundamental research on the inheritance of traits emerged from an applied plant breeding program. Mendel recognized the importance of the new cell theory; understanding of the formation of reproductive cells and the process of fertilization explained his segregation ratios. This interest was probably encouraged by his friendship with Johann Nave, whose untimely death preceded Mendel’s first 1865 lecture by a few months. This year is the 200th anniversary of Mendel’s birth, presenting a timely opportunity to revisit the events in his life that led him to undertake his seminal research. We review existing ideas on how Mendel made his discoveries, before presenting more recent evidence. In celebration of the 200th anniversary of Gregor Mendel’s birth, this Perspective discusses the historical context of Mendel’s discoveries and the importance of these insights in shaping the field of genetics.},
  archive      = {J_NG},
  author       = {van Dijk, Peter J. and Jessop, Adrienne P. and Ellis, T. H. Noel},
  doi          = {10.1038/s41588-022-01109-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {926-933},
  shortjournal = {Nat. Genet.},
  title        = {How did mendel arrive at his discoveries?},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identification of two intrinsic epithelial subtypes of
colorectal cancer. <em>NG</em>, <em>54</em>(7), 924–925. (<a
href="https://doi.org/10.1038/s41588-022-01101-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By integrating single-cell and bulk transcriptomic analyses, we found that malignant cells belong to two major intrinsic epithelial subtypes. We propose a refined, three-tiered classification of colorectal cancer subtypes based on intrinsic epithelial subtypes, microsatellite instability status and the presence of fibrosis.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01101-3},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {924-925},
  shortjournal = {Nat. Genet.},
  title        = {Identification of two intrinsic epithelial subtypes of colorectal cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Linking genetic variants to kidney disease via the
epigenome. <em>NG</em>, <em>54</em>(7), 922–923. (<a
href="https://doi.org/10.1038/s41588-022-01098-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The largest GWAS for kidney function so far provided the starting point for integrated multi-stage annotation of genetic loci. Whole kidney and single-cell epigenomic information is crucial for translating GWAS information to the identification of causal genes and pathogenetic (and potentially targetable) cellular and molecular mechanisms of kidney disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01098-9},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {922-923},
  shortjournal = {Nat. Genet.},
  title        = {Linking genetic variants to kidney disease via the epigenome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Lineage tracing in lung cancer. <em>NG</em>,
<em>54</em>(7), 921. (<a
href="https://doi.org/10.1038/s41588-022-01135-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Danovi, Safia},
  doi          = {10.1038/s41588-022-01135-7},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {921},
  shortjournal = {Nat. Genet.},
  title        = {Lineage tracing in lung cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Mapping genetic variants to cellular contexts. <em>NG</em>,
<em>54</em>(7), 921. (<a
href="https://doi.org/10.1038/s41588-022-01136-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Fletcher, Michael},
  doi          = {10.1038/s41588-022-01136-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {921},
  shortjournal = {Nat. Genet.},
  title        = {Mapping genetic variants to cellular contexts},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Potato genome diversity. <em>NG</em>, <em>54</em>(7), 921.
(<a href="https://doi.org/10.1038/s41588-022-01137-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Li, Wei},
  doi          = {10.1038/s41588-022-01137-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {921},
  shortjournal = {Nat. Genet.},
  title        = {Potato genome diversity},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). FTO regulates LINE1 in early development. <em>NG</em>,
<em>54</em>(7), 921. (<a
href="https://doi.org/10.1038/s41588-022-01138-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Faial, Tiago},
  doi          = {10.1038/s41588-022-01138-4},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {921},
  shortjournal = {Nat. Genet.},
  title        = {FTO regulates LINE1 in early development},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022c). Keeping the beat. <em>NG</em>, <em>54</em>(7), 921. (<a
href="https://doi.org/10.1038/s41588-022-01139-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Vogan, Kyle},
  doi          = {10.1038/s41588-022-01139-3},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {921},
  shortjournal = {Nat. Genet.},
  title        = {Keeping the beat},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Graph pangenomes find missing heritability. <em>NG</em>,
<em>54</em>(7), 919–920. (<a
href="https://doi.org/10.1038/s41588-022-01099-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of association studies to identify candidate genes for complex biological traits in plants has been challenging due to a reliance on single reference genomes, leading to missing heritability. Graphical pangenomes and the identification of causal variants help overcome this and provide an important advance for crop breeding.},
  archive      = {J_NG},
  author       = {Edwards, David and Batley, Jacqueline},
  doi          = {10.1038/s41588-022-01099-8},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {919-920},
  shortjournal = {Nat. Genet.},
  title        = {Graph pangenomes find missing heritability},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Decoding the zebrafish genome. <em>NG</em>, <em>54</em>(7),
917–919. (<a href="https://doi.org/10.1038/s41588-022-01080-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A large-scale collaborative effort now provides a comprehensive annotation of functional non-coding elements in the zebrafish genome. This work serves as an essential foundation for future studies to understand how gene regulatory circuits control embryonic development.},
  archive      = {J_NG},
  author       = {Lawson, Nathan D.},
  doi          = {10.1038/s41588-022-01080-5},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {917-919},
  shortjournal = {Nat. Genet.},
  title        = {Decoding the zebrafish genome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Automated sequence-based annotation and interpretation of
the human genome. <em>NG</em>, <em>54</em>(7), 916–917. (<a
href="https://doi.org/10.1038/s41588-022-01123-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A machine-learning model produces summarized sequence representations of genomic regulatory activity, and provides a functional view of regulatory DNA variation in the human genome, with the aim of better understanding the role of sequence variation in health and disease.},
  archive      = {J_NG},
  author       = {Kundaje, Anshul and Meuleman, Wouter},
  doi          = {10.1038/s41588-022-01123-x},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {916-917},
  shortjournal = {Nat. Genet.},
  title        = {Automated sequence-based annotation and interpretation of the human genome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deciphering the regulatory logic of a chromatin domain
boundary. <em>NG</em>, <em>54</em>(7), 914–915. (<a
href="https://doi.org/10.1038/s41588-022-01086-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The boundaries of chromatin domains have an important role in genome organization and regulation. A comprehensive genetic dissection of a domain boundary in vivo provides insights into how boundary elements function and cooperate to mediate insulation between chromatin domains.},
  archive      = {J_NG},
  author       = {Oudelaar, A. Marieke},
  doi          = {10.1038/s41588-022-01086-z},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {914-915},
  shortjournal = {Nat. Genet.},
  title        = {Deciphering the regulatory logic of a chromatin domain boundary},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Democratizing plant genomics to accelerate global food
production. <em>NG</em>, <em>54</em>(7), 911–913. (<a
href="https://doi.org/10.1038/s41588-022-01122-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Building on the fundamental discoveries of Mendel, plant genomics has had a major role in advancing the genetic improvement of crops worldwide, particularly in developed economies where the technologies are easily accessible. From cumbersome to more miniaturized high-throughput sequencing technologies, the field continues to evolve, providing vast opportunities for studying plant genomes with varying levels of complexity and potential real-life applications.},
  archive      = {J_NG},
  author       = {Ibe, Carol Nkechi},
  doi          = {10.1038/s41588-022-01122-y},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {911-913},
  shortjournal = {Nat. Genet.},
  title        = {Democratizing plant genomics to accelerate global food production},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). 30 years of progress from positional cloning to precision
genome editing. <em>NG</em>, <em>54</em>(7), 908–910. (<a
href="https://doi.org/10.1038/s41588-022-01128-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Thirty years ago, I had the privilege of launching Nature Genetics, the first spin-off journal bearing the famous Nature logo. Spurred on by the Human Genome Project, there were high hopes for the new journal and indeed the future of human genetics. But there was little expectation that we would launch a science publishing franchise of more than 30 sister journals — or be able to therapeutically rewrite the faulty genomes of patients. Here, I reflect on the humble beginnings of Nature Genetics and 30 years of progress in genetics.},
  archive      = {J_NG},
  author       = {Davies, Kevin},
  doi          = {10.1038/s41588-022-01128-6},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {908-910},
  shortjournal = {Nat. Genet.},
  title        = {30 years of progress from positional cloning to precision genome editing},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mendel, memories and meaning. <em>NG</em>, <em>54</em>(7),
907. (<a href="https://doi.org/10.1038/s41588-022-01146-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this issue of Nature Genetics, we celebrate the legacy of Gregor Mendel, who was born 200 years ago. We also note the 30th anniversary of the launch of Nature Genetics. The convergence of these two milestones helps us to look back on how far the genetics field has come, and also to look to the future to see where we are heading.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01146-4},
  journal      = {Nature Genetics},
  month        = {7},
  number       = {7},
  pages        = {907},
  shortjournal = {Nat. Genet.},
  title        = {Mendel, memories and meaning},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Discovery of a genetic module
essential for assigning left–right asymmetry in humans and ancestral
vertebrates. <em>NG</em>, <em>54</em>(6), 906. (<a
href="https://doi.org/10.1038/s41588-022-01053-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Szenker-Ravi, Emmanuelle and Ott, Tim and Khatoo, Muznah and Moreau de Bellaing, Anne and Goh, Wei Xuan and Chong, Yan Ling and Beckers, Anja and Kannesan, Darshini and Louvel, Guillaume and Anujan, Priyanka and Ravi, Vydianathan and Bonnard, Carine and Moutton, Sébastien and Schoen, Patric and Fradin, Mélanie and Colin, Estelle and Megarbane, André and Daou, Linda and Chehab, Ghassan and Di Filippo, Sylvie and Rooryck, Caroline and Deleuze, Jean-François and Boland, Anne and Arribard, Nicolas and Eker, Rukiye and Tohari, Sumanty and Ng, Alvin Yu-Jin and Rio, Marlène and Lim, Chun Teck and Eisenhaber, Birgit and Eisenhaber, Frank and Venkatesh, Byrappa and Amiel, Jeanne and Crollius, Hugues Roest and Gordon, Christopher T. and Gossler, Achim and Roy, Sudipto and Attie-Bitach, Tania and Blum, Martin and Bouvagnet, Patrice and Reversade, Bruno},
  doi          = {10.1038/s41588-022-01053-8},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {906},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Discovery of a genetic module essential for assigning left–right asymmetry in humans and ancestral vertebrates},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Indirect paths from genetics to
education. <em>NG</em>, <em>54</em>(6), 906. (<a
href="https://doi.org/10.1038/s41588-022-01092-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Schork, Andrew J. and Peterson, Roseann E. and Dahl, Andrew and Cai, Na and Kendler, Kenneth S.},
  doi          = {10.1038/s41588-022-01092-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {906},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Indirect paths from genetics to education},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Publisher correction: Dual function NFI factors control
fetal hemoglobin silencing in adult erythroid cells. <em>NG</em>,
<em>54</em>(6), 906. (<a
href="https://doi.org/10.1038/s41588-022-01112-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Qin, Kunhua and Huang, Peng and Feng, Ruopeng and Keller, Cheryl A. and Peslak, Scott A. and Khandros, Eugene and Saari, Megan S. and Lan, Xianjiang and Mayuranathan, Thiyagaraj and Doerfler, Phillip A. and Abdulmalik, Osheiza and Giardine, Belinda and Chou, Stella T. and Shi, Junwei and Hardison, Ross C. and Weiss, Mitchell J. and Blobel, Gerd A.},
  doi          = {10.1038/s41588-022-01112-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {906},
  shortjournal = {Nat. Genet.},
  title        = {Publisher correction: Dual function NFI factors control fetal hemoglobin silencing in adult erythroid cells},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mendelian imputation of parental genotypes improves
estimates of direct genetic effects. <em>NG</em>, <em>54</em>(6),
897–905. (<a href="https://doi.org/10.1038/s41588-022-01085-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Effects estimated by genome-wide association studies (GWASs) include effects of alleles in an individual on that individual (direct genetic effects), indirect genetic effects (for example, effects of alleles in parents on offspring through the environment) and bias from confounding. Within-family genetic variation is random, enabling unbiased estimation of direct genetic effects when parents are genotyped. However, parental genotypes are often missing. We introduce a method that imputes missing parental genotypes and estimates direct genetic effects. Our method, implemented in the software package snipar (single-nucleotide imputation of parents), gives more precise estimates of direct genetic effects than existing approaches. Using 39,614 individuals from the UK Biobank with at least one genotyped sibling/parent, we estimate the correlation between direct genetic effects and effects from standard GWASs for nine phenotypes, including educational attainment (r = 0.739, standard error (s.e.) = 0.086) and cognitive ability (r = 0.490, s.e. = 0.086). Our results demonstrate substantial confounding bias in standard GWASs for some phenotypes. snipar is a software package for imputing missing parental genotypes and estimating direct genetic effects. Application to UK Biobank data shows that effects estimated by standard genome-wide association study designs have confounding bias for some phenotypes.},
  archive      = {J_NG},
  author       = {Young, Alexander I. and Nehzati, Seyed Moeen and Benonisdottir, Stefania and Okbay, Aysu and Jayashankar, Hariharan and Lee, Chanwook and Cesarini, David and Benjamin, Daniel J. and Turley, Patrick and Kong, Augustine},
  doi          = {10.1038/s41588-022-01085-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {897-905},
  shortjournal = {Nat. Genet.},
  title        = {Mendelian imputation of parental genotypes improves estimates of direct genetic effects},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genomic insights into the recent chromosome reduction of
autopolyploid sugarcane saccharum spontaneum. <em>NG</em>,
<em>54</em>(6), 885–896. (<a
href="https://doi.org/10.1038/s41588-022-01084-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Saccharum spontaneum is a founding Saccharum species and exhibits wide variation in ploidy levels. We have assembled a high-quality autopolyploid genome of S. spontaneum Np-X (2n = 4x = 40) into 40 pseudochromosomes across 10 homologous groups, that better elucidates recent chromosome reduction and polyploidization that occurred circa 1.5 million years ago (Mya). One paleo-duplicated chromosomal pair in Saccharum, NpChr5 and NpChr8, underwent fission followed by fusion accompanied by centromeric split around 0.80 Mya. We inferred that Np-X, with x = 10, most likely represents the ancestral karyotype, from which x = 9 and x = 8 evolved. Resequencing of 102 S. spontaneum accessions revealed that S. spontaneum originated in northern India from an x = 10 ancestor, which then radiated into four major groups across the Indian subcontinent, China, and Southeast Asia. Our study suggests new directions for accelerating sugarcane improvement and expands our knowledge of the evolution of autopolyploids. A high-quality autopolyploid genome of Saccharum spontaneum accession Np-X and resequencing of 102 accessions provide insights into the recent chromosome reduction and polyploidization in Saccharum.},
  archive      = {J_NG},
  author       = {Zhang, Qing and Qi, Yiying and Pan, Haoran and Tang, Haibao and Wang, Gang and Hua, Xiuting and Wang, Yongjun and Lin, Lianyu and Li, Zhen and Li, Yihan and Yu, Fan and Yu, Zehuai and Huang, Yongji and Wang, Tianyou and Ma, Panpan and Dou, Meijie and Sun, Zongyi and Wang, Yibin and Wang, Hengbo and Zhang, Xingtan and Yao, Wei and Wang, Yuntong and Liu, Xinlong and Wang, Maojun and Wang, Jianping and Deng, Zuhu and Xu, Jingsheng and Yang, Qinghui and Liu, ZhongJian and Chen, Baoshan and Zhang, Muqing and Ming, Ray and Zhang, Jisen},
  doi          = {10.1038/s41588-022-01084-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {885-896},
  shortjournal = {Nat. Genet.},
  title        = {Genomic insights into the recent chromosome reduction of autopolyploid sugarcane saccharum spontaneum},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Dual function NFI factors control fetal hemoglobin
silencing in adult erythroid cells. <em>NG</em>, <em>54</em>(6),
874–884. (<a href="https://doi.org/10.1038/s41588-022-01076-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanisms by which the fetal-type β-globin-like genes HBG1 and HBG2 are silenced in adult erythroid precursor cells remain a fundamental question in human biology and have therapeutic relevance to sickle cell disease and β-thalassemia. Here, we identify via a CRISPR–Cas9 genetic screen two members of the NFI transcription factor family—NFIA and NFIX—as HBG1/2 repressors. NFIA and NFIX are expressed at elevated levels in adult erythroid cells compared with fetal cells, and function cooperatively to repress HBG1/2 in cultured cells and in human-to-mouse xenotransplants. Genomic profiling, genome editing and DNA binding assays demonstrate that the potent concerted activity of NFIA and NFIX is explained in part by their ability to stimulate the expression of BCL11A, a known silencer of the HBG1/2 genes, and in part by directly repressing the HBG1/2 genes. Thus, NFI factors emerge as versatile regulators of the fetal-to-adult switch in β-globin production. NFIA and NFIX directly repress the expression of fetal-type β-globin-like genes HBG1 and HBG2 in adult erythroid cells, and also do it indirectly through the upregulation of BCL11A.},
  archive      = {J_NG},
  author       = {Qin, Kunhua and Huang, Peng and Feng, Ruopeng and Keller, Cheryl A. and Peslak, Scott A. and Khandros, Eugene and Saari, Megan S. and Lan, Xianjiang and Mayuranathan, Thiyagaraj and Doerfler, Phillip A. and Abdulmalik, Osheiza and Giardine, Belinda and Chou, Stella T. and Shi, Junwei and Hardison, Ross C. and Weiss, Mitchell J. and Blobel, Gerd A.},
  doi          = {10.1038/s41588-022-01076-1},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {874-884},
  shortjournal = {Nat. Genet.},
  title        = {Dual function NFI factors control fetal hemoglobin silencing in adult erythroid cells},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). SMARCE1 deficiency generates a targetable mSWI/SNF
dependency in clear cell meningioma. <em>NG</em>, <em>54</em>(6),
861–873. (<a href="https://doi.org/10.1038/s41588-022-01077-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian SWI/SNF (mSWI/SNF) ATP-dependent chromatin remodeling complexes establish and maintain chromatin accessibility and gene expression, and are frequently perturbed in cancer. Clear cell meningioma (CCM), an aggressive tumor of the central nervous system, is uniformly driven by loss of SMARCE1, an integral subunit of the mSWI/SNF core. Here, we identify a structural role for SMARCE1 in selectively stabilizing the canonical BAF (cBAF) complex core–ATPase module interaction. In CCM, cBAF complexes fail to stabilize on chromatin, reducing enhancer accessibility, and residual core module components increase the formation of BRD9-containing non-canonical BAF (ncBAF) complexes. Combined attenuation of cBAF function and increased ncBAF complex activity generates the CCM-specific gene expression signature, which is distinct from that of NF2-mutated meningiomas. Importantly, SMARCE1-deficient cells exhibit heightened sensitivity to small-molecule inhibition of ncBAF complexes. These data inform the function of a previously elusive SWI/SNF subunit and suggest potential therapeutic approaches for intractable SMARCE1-deficient CCM tumors. SMARCE1 loss destabilizes the canonical BAF complex and increases the formation of BRD9-containing non-canonical (ncBAF) complexes. SMARCE1-deficient cells, which are a model for clear cell meningioma, are sensitive to ncBAF complex inhibition.},
  archive      = {J_NG},
  author       = {St. Pierre, Roodolph and Collings, Clayton K. and Samé Guerra, Daniel D. and Widmer, Christian J. and Bolonduro, Olubusayo and Mashtalir, Nazar and Sankar, Akshay and Liang, Yu and Bi, Wenya Linda and Gerkes, Erica H. and Ramesh, Vijaya and Qi, Jun and Smith, Miriam J. and Meredith, David M. and Kadoch, Cigall},
  doi          = {10.1038/s41588-022-01077-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {861-873},
  shortjournal = {Nat. Genet.},
  title        = {SMARCE1 deficiency generates a targetable mSWI/SNF dependency in clear cell meningioma},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genomic analysis defines clonal relationships of ductal
carcinoma in situ and recurrent invasive breast cancer. <em>NG</em>,
<em>54</em>(6), 850–860. (<a
href="https://doi.org/10.1038/s41588-022-01082-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ductal carcinoma in situ (DCIS) is the most common form of preinvasive breast cancer and, despite treatment, a small fraction (5–10%) of DCIS patients develop subsequent invasive disease. A fundamental biologic question is whether the invasive disease arises from tumor cells in the initial DCIS or represents new unrelated disease. To address this question, we performed genomic analyses on the initial DCIS lesion and paired invasive recurrent tumors in 95 patients together with single-cell DNA sequencing in a subset of cases. Our data show that in 75% of cases the invasive recurrence was clonally related to the initial DCIS, suggesting that tumor cells were not eliminated during the initial treatment. Surprisingly, however, 18% were clonally unrelated to the DCIS, representing new independent lineages and 7% of cases were ambiguous. This knowledge is essential for accurate risk evaluation of DCIS, treatment de-escalation strategies and the identification of predictive biomarkers. A genomic analysis of ductal carcinoma in situ (DCIS) samples with matched ipsilateral invasive breast cancer recurring later shows that around 18% of tumors were unrelated to the DCIS, and had distinct clonal origins.},
  archive      = {J_NG},
  author       = {Lips, Esther H. and Kumar, Tapsi and Megalios, Anargyros and Visser, Lindy L. and Sheinman, Michael and Fortunato, Angelo and Shah, Vandna and Hoogstraat, Marlous and Sei, Emi and Mallo, Diego and Roman-Escorza, Maria and Ahmed, Ahmed A. and Xu, Mingchu and van den Belt-Dusebout, Alexandra W. and Brugman, Wim and Casasent, Anna K. and Clements, Karen and Davies, Helen R. and Fu, Liping and Grigoriadis, Anita and Hardman, Timothy M. and King, Lorraine M. and Krete, Marielle and Kristel, Petra and de Maaker, Michiel and Maley, Carlo C. and Marks, Jeffrey R. and Menegaz, Brian A. and Mulder, Lennart and Nieboer, Frank and Nowinski, Salpie and Pinder, Sarah and Quist, Jelmar and Salinas-Souza, Carolina and Schaapveld, Michael and Schmidt, Marjanka K. and Shaaban, Abeer M. and Shami, Rana and Sridharan, Mathini and Zhang, John and Stobart, Hilary and Collyar, Deborah and Nik-Zainal, Serena and Wessels, Lodewyk F. A. and Hwang, E. Shelley and Navin, Nicholas E. and Futreal, P. Andrew and Thompson, Alastair M. and Wesseling, Jelle and Sawyer, Elinor J.},
  doi          = {10.1038/s41588-022-01082-3},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {850-860},
  shortjournal = {Nat. Genet.},
  title        = {Genomic analysis defines clonal relationships of ductal carcinoma in situ and recurrent invasive breast cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Allelic imbalance of chromatin accessibility in cancer
identifies candidate causal risk variants and their mechanisms.
<em>NG</em>, <em>54</em>(6), 837–849. (<a
href="https://doi.org/10.1038/s41588-022-01075-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While many germline cancer risk variants have been identified through genome-wide association studies (GWAS), the mechanisms by which these variants operate remain largely unknown. Here we used 406 cancer ATAC-Seq samples across 23 cancer types to identify 7,262 germline allele-specific accessibility QTLs (as-aQTLs). Cancer as-aQTLs had stronger enrichment for cancer risk heritability (up to 145 fold) than any other functional annotation across seven cancer GWAS. Most cancer as-aQTLs directly altered transcription factor (TF) motifs and exhibited differential TF binding and gene expression in functional screens. To connect as-aQTLs to putative risk mechanisms, we introduced the regulome-wide associations study (RWAS). RWAS identified genetically associated accessible peaks at &amp;gt;70% of known breast and prostate loci and discovered new risk loci in all examined cancer types. Integrating as-aQTL discovery, motif analysis and RWAS identified candidate causal regulatory elements and their probable upstream regulators. Our work establishes cancer as-aQTLs and RWAS analysis as powerful tools to study the genetic architecture of cancer risk. Allelic imbalance analysis applied to ATAC-Seq data from 23 cancer types identifies 7,262 allele-specific accessibility quantitative trait loci, which are enriched for cancer risk heritability and altered transcription factor binding motifs.},
  archive      = {J_NG},
  author       = {Grishin, Dennis and Gusev, Alexander},
  doi          = {10.1038/s41588-022-01075-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {837-849},
  shortjournal = {Nat. Genet.},
  title        = {Allelic imbalance of chromatin accessibility in cancer identifies candidate causal risk variants and their mechanisms},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Combining SNP-to-gene linking strategies to identify disease
genes and assess disease omnigenicity. <em>NG</em>, <em>54</em>(6),
827–836. (<a href="https://doi.org/10.1038/s41588-022-01087-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Disease-associated single-nucleotide polymorphisms (SNPs) generally do not implicate target genes, as most disease SNPs are regulatory. Many SNP-to-gene (S2G) linking strategies have been developed to link regulatory SNPs to the genes that they regulate in cis. Here, we developed a heritability-based framework for evaluating and combining different S2G strategies to optimize their informativeness for common disease risk. Our optimal combined S2G strategy (cS2G) included seven constituent S2G strategies and achieved a precision of 0.75 and a recall of 0.33, more than doubling the recall of any individual strategy. We applied cS2G to fine-mapping results for 49 UK Biobank diseases/traits to predict 5,095 causal SNP–gene-disease triplets (with S2G-derived functional interpretation) with high confidence. We further applied cS2G to provide an empirical assessment of disease omnigenicity; we determined that the top 1% of genes explained roughly half of the SNP heritability linked to all genes and that gene-level architectures vary with variant allele frequency. A heritability-based framework for evaluation of SNP-to-gene linking methods is used to construct an optimal, combined approach and applied to 49 traits. Analysis of trait omnigenicity suggests gene-level architecture varies depending on variant frequency.},
  archive      = {J_NG},
  author       = {Gazal, Steven and Weissbrod, Omer and Hormozdiari, Farhad and Dey, Kushal K. and Nasser, Joseph and Jagadeesh, Karthik A. and Weiner, Daniel J. and Shi, Huwenbo and Fulco, Charles P. and O’Connor, Luke J. and Pasaniuc, Bogdan and Engreitz, Jesse M. and Price, Alkes L.},
  doi          = {10.1038/s41588-022-01087-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {827-836},
  shortjournal = {Nat. Genet.},
  title        = {Combining SNP-to-gene linking strategies to identify disease genes and assess disease omnigenicity},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Immune disease risk variants regulate gene expression
dynamics during CD4+ t cell activation. <em>NG</em>, <em>54</em>(6),
817–826. (<a href="https://doi.org/10.1038/s41588-022-01066-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {During activation, T cells undergo extensive gene expression changes that shape the properties of cells to exert their effector function. Understanding the regulation of this process could help explain how genetic variants predispose to immune diseases. Here, we mapped genetic effects on gene expression (expression quantitative trait loci (eQTLs)) using single-cell transcriptomics. We profiled 655,349 CD4+ T cells, capturing transcriptional states of unstimulated cells and three time points of cell activation in 119 healthy individuals. This identified 38 cell clusters, including transient clusters that were only present at individual time points of activation. We found 6,407 genes whose expression was correlated with genetic variation, of which 2,265 (35%) were dynamically regulated during activation. Furthermore, 127 genes were regulated by variants associated with immune-mediated diseases, with significant enrichment for dynamic effects. Our results emphasize the importance of studying context-specific gene expression regulation and provide insights into the mechanisms underlying genetic susceptibility to immune-mediated diseases. Single-cell RNA sequencing of CD4+ naive and memory T cells from 119 individuals generates an expression quantitative trait locus (eQTL) map during T cell activation, identifying 6,407 eQTL genes, including 2,265 that are dynamically regulated.},
  archive      = {J_NG},
  author       = {Soskic, Blagoje and Cano-Gamez, Eddie and Smyth, Deborah J. and Ambridge, Kirsty and Ke, Ziying and Matte, Julie C. and Bossini-Castillo, Lara and Kaplanis, Joanna and Ramirez-Navarro, Lucia and Lorenc, Anna and Nakic, Nikolina and Esparza-Gordillo, Jorge and Rowan, Wendy and Wille, David and Tough, David F. and Bronson, Paola G. and Trynka, Gosia},
  doi          = {10.1038/s41588-022-01066-3},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {817-826},
  shortjournal = {Nat. Genet.},
  title        = {Immune disease risk variants regulate gene expression dynamics during CD4+ t cell activation},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Single-nucleus chromatin accessibility profiling highlights
regulatory mechanisms of coronary artery disease risk. <em>NG</em>,
<em>54</em>(6), 804–816. (<a
href="https://doi.org/10.1038/s41588-022-01069-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coronary artery disease (CAD) is a complex inflammatory disease involving genetic influences across cell types. Genome-wide association studies have identified over 200 loci associated with CAD, where the majority of risk variants reside in noncoding DNA sequences impacting cis-regulatory elements. Here, we applied single-nucleus assay for transposase-accessible chromatin with sequencing to profile 28,316 nuclei across coronary artery segments from 41 patients with varying stages of CAD, which revealed 14 distinct cellular clusters. We mapped ~320,000 accessible sites across all cells, identified cell-type-specific elements and transcription factors, and prioritized functional CAD risk variants. We identified elements in smooth muscle cell transition states (for example, fibromyocytes) and functional variants predicted to alter smooth muscle cell- and macrophage-specific regulation of MRAS (3q22) and LIPA (10q23), respectively. We further nominated key driver transcription factors such as PRDM16 and TBX2. Together, this single-nucleus atlas provides a critical step towards interpreting regulatory mechanisms across the continuum of CAD risk. Single-nucleus ATAC-seq characterization of chromatin accessibility in human coronary artery disease samples identifies cell-type- and state-specific regulatory mechanisms underlying disease risk, highlighting the roles of TBX2 and PRDM16.},
  archive      = {J_NG},
  author       = {Turner, Adam W. and Hu, Shengen Shawn and Mosquera, Jose Verdezoto and Ma, Wei Feng and Hodonsky, Chani J. and Wong, Doris and Auguste, Gaëlle and Song, Yipei and Sol-Church, Katia and Farber, Emily and Kundu, Soumya and Kundaje, Anshul and Lopez, Nicolas G. and Ma, Lijiang and Ghosh, Saikat Kumar B. and Onengut-Gumuscu, Suna and Ashley, Euan A. and Quertermous, Thomas and Finn, Aloke V. and Leeper, Nicholas J. and Kovacic, Jason C. and Björkegren, Johan L. M. and Zang, Chongzhi and Miller, Clint L.},
  doi          = {10.1038/s41588-022-01069-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {804-816},
  shortjournal = {Nat. Genet.},
  title        = {Single-nucleus chromatin accessibility profiling highlights regulatory mechanisms of coronary artery disease risk},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetic analysis of right heart structure and function in
40,000 people. <em>NG</em>, <em>54</em>(6), 792–803. (<a
href="https://doi.org/10.1038/s41588-022-01090-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Congenital heart diseases often involve maldevelopment of the evolutionarily recent right heart chamber. To gain insight into right heart structure and function, we fine-tuned deep learning models to recognize the right atrium, right ventricle and pulmonary artery, measuring right heart structures in 40,000 individuals from the UK Biobank with magnetic resonance imaging. Genome-wide association studies identified 130 distinct loci associated with at least one right heart measurement, of which 72 were not associated with left heart structures. Loci were found near genes previously linked with congenital heart disease, including NKX2-5, TBX5/TBX3, WNT9B and GATA4. A genome-wide polygenic predictor of right ventricular ejection fraction was associated with incident dilated cardiomyopathy (hazard ratio, 1.33 per standard deviation; P = 7.1 × 10−13) and remained significant after accounting for a left ventricular polygenic score. Harnessing deep learning to perform large-scale cardiac phenotyping, our results yield insights into the genetic determinants of right heart structure and function. Genome-wide analyses of cardiac magnetic resonance imaging data identify loci associated with right heart structure and function. A polygenic predictor of right ventricular ejection fraction is associated with dilated cardiomyopathy risk.},
  archive      = {J_NG},
  author       = {Pirruccello, James P. and Di Achille, Paolo and Nauffal, Victor and Nekoui, Mahan and Friedman, Samuel F. and Klarqvist, Marcus D. R. and Chaffin, Mark D. and Weng, Lu-Chen and Cunningham, Jonathan W. and Khurshid, Shaan and Roselli, Carolina and Lin, Honghuang and Koyama, Satoshi and Ito, Kaoru and Kamatani, Yoichiro and Komuro, Issei and Jurgens, Sean J. and Benjamin, Emelia J. and Batra, Puneet and Natarajan, Pradeep and Ng, Kenney and Hoffmann, Udo and Lubitz, Steven A. and Ho, Jennifer E. and Lindsay, Mark E. and Philippakis, Anthony A. and Ellinor, Patrick T.},
  doi          = {10.1038/s41588-022-01090-3},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {792-803},
  shortjournal = {Nat. Genet.},
  title        = {Genetic analysis of right heart structure and function in 40,000 people},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome-wide association analysis reveals insights into the
genetic architecture of right ventricular structure and function.
<em>NG</em>, <em>54</em>(6), 783–791. (<a
href="https://doi.org/10.1038/s41588-022-01083-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Right ventricular (RV) structure and function influence the morbidity and mortality from coronary artery disease (CAD), dilated cardiomyopathy (DCM), pulmonary hypertension and heart failure. Little is known about the genetic basis of RV measurements. Here we perform genome-wide association analyses of four clinically relevant RV phenotypes (RV end-diastolic volume, RV end-systolic volume, RV stroke volume, RV ejection fraction) from cardiovascular magnetic resonance images, using a state-of-the-art deep learning algorithm in 29,506 UK Biobank participants. We identify 25 unique loci associated with at least one RV phenotype at P &amp;lt; 2.27 ×10−8, 17 of which are validated in a combined meta-analysis (n = 41,830). Several candidate genes overlap with Mendelian cardiomyopathy genes and are involved in cardiac muscle contraction and cellular adhesion. The RV polygenic risk scores (PRSs) are associated with DCM and CAD. The findings substantially advance our understanding of the genetic underpinning of RV measurements. Genome-wide analyses of cardiovascular magnetic resonance images identify variants associated with right ventricular structure and function. Polygenic scores for these traits are associated with dilated cardiomyopathy and coronary artery disease.},
  archive      = {J_NG},
  author       = {Aung, Nay and Vargas, Jose D. and Yang, Chaojie and Fung, Kenneth and Sanghvi, Mihir M. and Piechnik, Stefan K. and Neubauer, Stefan and Manichaikul, Ani and Rotter, Jerome I. and Taylor, Kent D. and Lima, Joao A. C. and Bluemke, David A. and Kawut, Steven M. and Petersen, Steffen E. and Munroe, Patricia B.},
  doi          = {10.1038/s41588-022-01083-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {783-791},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association analysis reveals insights into the genetic architecture of right ventricular structure and function},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). High heritability of ascending aortic diameter and
trans-ancestry prediction of thoracic aortic disease. <em>NG</em>,
<em>54</em>(6), 772–782. (<a
href="https://doi.org/10.1038/s41588-022-01070-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enlargement of the aorta is an important risk factor for aortic aneurysm and dissection, a leading cause of morbidity in the developed world. Here we performed automated extraction of ascending aortic diameter from cardiac magnetic resonance images of 36,021 individuals from the UK Biobank, followed by genome-wide association. We identified lead variants across 41 loci, including genes related to cardiovascular development (HAND2, TBX20) and Mendelian forms of thoracic aortic disease (ELN, FBN1). A polygenic score significantly predicted prevalent risk of thoracic aortic aneurysm and the need for surgical intervention for patients with thoracic aneurysm across multiple ancestries within the UK Biobank, FinnGen, the Penn Medicine Biobank and the Million Veterans Program (MVP). Additionally, we highlight the primary causal role of blood pressure in reducing aortic dilation using Mendelian randomization. Overall, our findings provide a roadmap for using genetic determinants of human anatomy to understand cardiovascular development while improving prediction of diseases of the thoracic aorta. Trans-ancestry genome-wide analyses identify multiple loci associated with ascending aortic diameter. A polygenic score constructed from these loci predicted prevalent risk of thoracic aortic aneurysm in independent populations.},
  archive      = {J_NG},
  author       = {Tcheandjieu, Catherine and Xiao, Ke and Tejeda, Helio and Lynch, Julie A. and Ruotsalainen, Sanni and Bellomo, Tiffany and Palnati, Madhuri and Judy, Renae and Klarin, Derek and Kember, Rachel L. and Verma, Shefali and Palotie, Aarno and Daly, Mark and Ritchie, Marylyn and Rader, Daniel J. and Rivas, Manuel A. and Assimes, Themistocles and Tsao, Philip and Damrauer, Scott and Priest, James R.},
  doi          = {10.1038/s41588-022-01070-7},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {772-782},
  shortjournal = {Nat. Genet.},
  title        = {High heritability of ascending aortic diameter and trans-ancestry prediction of thoracic aortic disease},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A multiancestry genome-wide association study of unexplained
chronic ALT elevation as a proxy for nonalcoholic fatty liver disease
with histological and radiological validation. <em>NG</em>,
<em>54</em>(6), 761–771. (<a
href="https://doi.org/10.1038/s41588-022-01078-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nonalcoholic fatty liver disease (NAFLD) is a growing cause of chronic liver disease. Using a proxy NAFLD definition of chronic elevation of alanine aminotransferase (cALT) levels without other liver diseases, we performed a multiancestry genome-wide association study (GWAS) in the Million Veteran Program (MVP) including 90,408 cALT cases and 128,187 controls. Seventy-seven loci exceeded genome-wide significance, including 25 without prior NAFLD or alanine aminotransferase associations, with one additional locus identified in European American-only and two in African American-only analyses (P &amp;lt; 5 × 10−8). External replication in histology-defined NAFLD cohorts (7,397 cases and 56,785 controls) or radiologic imaging cohorts (n = 44,289) replicated 17 single-nucleotide polymorphisms (SNPs) (P &amp;lt; 6.5 × 10−4), of which 9 were new (TRIB1, PPARG, MTTP, SERPINA1, FTO, IL1RN, COBLL1, APOH and IFI30). Pleiotropy analysis showed that 61 of 77 multiancestry and all 17 replicated SNPs were jointly associated with metabolic and/or inflammatory traits, revealing a complex model of genetic architecture. Our approach integrating cALT, histology and imaging reveals new insights into genetic liability to NAFLD. A multiancestry genome-wide association study of chronic alanine aminotransferase elevation identifies candidate risk loci for nonalcoholic fatty liver disease, with replication in external cohorts defined by histology or imaging.},
  archive      = {J_NG},
  author       = {Vujkovic, Marijana and Ramdas, Shweta and Lorenz, Kim M. and Guo, Xiuqing and Darlay, Rebecca and Cordell, Heather J. and He, Jing and Gindin, Yevgeniy and Chung, Chuhan and Myers, Robert P. and Schneider, Carolin V. and Park, Joseph and Lee, Kyung Min and Serper, Marina and Carr, Rotonya M. and Kaplan, David E. and Haas, Mary E. and MacLean, Matthew T. and Witschey, Walter R. and Zhu, Xiang and Tcheandjieu, Catherine and Kember, Rachel L. and Kranzler, Henry R. and Verma, Anurag and Giri, Ayush and Klarin, Derek M. and Sun, Yan V. and Huang, Jie and Huffman, Jennifer E. and Creasy, Kate Townsend and Hand, Nicholas J. and Liu, Ching-Ti and Long, Michelle T. and Yao, Jie and Budoff, Matthew and Tan, Jingyi and Li, Xiaohui and Lin, Henry J. and Chen, Yii-Der Ida and Taylor, Kent D. and Chang, Ruey-Kang and Krauss, Ronald M. and Vilarinho, Silvia and Brancale, Joseph and Nielsen, Jonas B. and Locke, Adam E. and Jones, Marcus B. and Verweij, Niek and Baras, Aris and Reddy, K. Rajender and Neuschwander-Tetri, Brent A. and Schwimmer, Jeffrey B. and Sanyal, Arun J. and Chalasani, Naga and Ryan, Kathleen A. and Mitchell, Braxton D. and Gill, Dipender and Wells, Andrew D. and Manduchi, Elisabetta and Saiman, Yedidya and Mahmud, Nadim and Miller, Donald R. and Reaven, Peter D. and Phillips, Lawrence S. and Muralidhar, Sumitra and DuVall, Scott L. and Lee, Jennifer S. and Assimes, Themistocles L. and Pyarajan, Saiju and Cho, Kelly and Edwards, Todd L. and Damrauer, Scott M. and Wilson, Peter W. and Gaziano, J. Michael and O’Donnell, Christopher J. and Khera, Amit V. and Grant, Struan F. A. and Brown, Christopher D. and Tsao, Philip S. and Saleheen, Danish and Lotta, Luca A. and Bastarache, Lisa and Anstee, Quentin M. and Daly, Ann K. and Meigs, James B. and Rotter, Jerome I. and Lynch, Julie A. and Rader, Daniel J. and Voight, Benjamin F. and Chang, Kyong-Mi},
  doi          = {10.1038/s41588-022-01078-z},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {761-771},
  shortjournal = {Nat. Genet.},
  title        = {A multiancestry genome-wide association study of unexplained chronic ALT elevation as a proxy for nonalcoholic fatty liver disease with histological and radiological validation},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Histone editing elucidates the functional roles of H3K27
methylation and acetylation in mammals. <em>NG</em>, <em>54</em>(6),
754–760. (<a href="https://doi.org/10.1038/s41588-022-01091-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Posttranslational modifications of histones (PTMs) are associated with specific chromatin and gene expression states1,2. Although studies in Drosophila melanogaster have revealed phenotypic associations between chromatin-modifying enzymes and their histone substrates, comparable studies in mammalian models do not exist3–5. Here, we use CRISPR base editing in mouse embryonic stem cells (mESCs) to address the regulatory role of lysine 27 of histone H3 (H3K27), a substrate for Polycomb repressive complex 2 (PRC2)-mediated methylation and CBP/EP300-mediated acetylation6,7. By generating pan-H3K27R (pK27R) mutant mESCs, where all 28 alleles of H3.1, H3.2 and H3.3 have been mutated, we demonstrate similarity in transcription patterns of genes and differentiation to PRC2-null mutants. Moreover, H3K27 acetylation is not essential for gene derepression linked to loss of H3K27 methylation, or de novo activation of genes during cell-fate transition to epiblast-like cells (EpiLCs). In conclusion, our results show that H3K27 is an essential substrate for PRC2 in mESCs, whereas other PTMs in addition to H3K27 acetylation are likely involved in mediating CBP/EP300 function. Our work demonstrates the feasibility of large-scale multicopy gene editing to interrogate histone PTM function in mammalian cells. CRISPR editing of all 28 alleles encoding histones H3.1, H3.2 and H3.3 in mouse embryonic stem cells (mESCs) generates pan-H3K27R mutant mESCs, which are transcriptionally similar to PRC2-null mESCs. H3K27 acetylation is dispensable for gene derepression in mESCs and for gene activation in epiblast-like cells.},
  archive      = {J_NG},
  author       = {Sankar, Aditya and Mohammad, Faizaan and Sundaramurthy, Arun Kumar and Wang, Hua and Lerdrup, Mads and Tatar, Tulin and Helin, Kristian},
  doi          = {10.1038/s41588-022-01091-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {754-760},
  shortjournal = {Nat. Genet.},
  title        = {Histone editing elucidates the functional roles of H3K27 methylation and acetylation in mammals},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A molecular map of t cell activation gives insights into
immune disease. <em>NG</em>, <em>54</em>(6), 752–753. (<a
href="https://doi.org/10.1038/s41588-022-01067-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study using single-cell transcriptomics and mapping of expression quantitative trait loci (eQTLs) in a dynamic model of CD4+ T cell activation reveals novel, context-specific eQTLs linked to genes associated with immune diseases such as inflammatory bowel disease.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01067-2},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {752-753},
  shortjournal = {Nat. Genet.},
  title        = {A molecular map of t cell activation gives insights into immune disease},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chromatin profiling of coronary artery illuminates genetic
risk for heart disease. <em>NG</em>, <em>54</em>(6), 750–751. (<a
href="https://doi.org/10.1038/s41588-022-01029-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A large-scale single-nucleus chromatin accessibility profiling study in coronary artery samples from patients with coronary artery disease generated a landscape of the regulatory activity during the disease. These data highlight cell type-specific gene programs that can improve the interpretation of human genome-wide association studies findings for cardiovascular diseases.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01029-8},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {750-751},
  shortjournal = {Nat. Genet.},
  title        = {Chromatin profiling of coronary artery illuminates genetic risk for heart disease},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). One step closer to linking GWAS SNPs with the right genes.
<em>NG</em>, <em>54</em>(6), 748–749. (<a
href="https://doi.org/10.1038/s41588-022-01093-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study highlights a strategy to link SNPs implicated in human complex traits and diseases with probable causal genes. This method prioritizes genes for functional characterization and helps address questions about the architecture of human phenotypes.},
  archive      = {J_NG},
  author       = {Lettre, Guillaume},
  doi          = {10.1038/s41588-022-01093-0},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {748-749},
  shortjournal = {Nat. Genet.},
  title        = {One step closer to linking GWAS SNPs with the right genes},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Histone gene editing probes functions of H3K27 modifications
in mammals. <em>NG</em>, <em>54</em>(6), 746–747. (<a
href="https://doi.org/10.1038/s41588-022-01096-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study employs CRISPR–Cas9-based base editing for simultaneous mutagenesis of all copies of histone H3 genes in mammals, highlighting the functional importance of H3K27me3 for Polycomb-mediated gene silencing and the dispensability of H3K27ac in transcriptional activation.},
  archive      = {J_NG},
  author       = {Scacchetti, Alessandro and Bonasio, Roberto},
  doi          = {10.1038/s41588-022-01096-x},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {746-747},
  shortjournal = {Nat. Genet.},
  title        = {Histone gene editing probes functions of H3K27 modifications in mammals},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Increasing diversity in genomics requires investment in
equitable partnerships and capacity building. <em>NG</em>,
<em>54</em>(6), 740–745. (<a
href="https://doi.org/10.1038/s41588-022-01095-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Calls for diversity in genomics have motivated new global research collaborations across institutions with highly imbalanced resources. We describe practical lessons we have learned so far from designing multidisciplinary international research and capacity-building programs that prioritize equity in two intertwined programs — the NeuroGAP-Psychosis research study and GINGER training program — spanning institutions in Ethiopia, Kenya, South Africa, Uganda and the United States.},
  archive      = {J_NG},
  author       = {Martin, Alicia R. and Stroud, Rocky E. and Abebe, Tamrat and Akena, Dickens and Alemayehu, Melkam and Atwoli, Lukoye and Chapman, Sinéad B. and Flowers, Katelyn and Gelaye, Bizu and Gichuru, Stella and Kariuki, Symon M. and Kinyanjui, Sam and Korte, Kristina J. and Koen, Nastassja and Koenen, Karestan C. and Newton, Charles R. J. C. and Olivares, Ana Maria and Pollock, Sam and Post, Kristianna and Singh, Ilina and Stein, Dan J. and Teferra, Solomon and Zingela, Zukiswa and Chibnik, Lori B.},
  doi          = {10.1038/s41588-022-01095-y},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {740-745},
  shortjournal = {Nat. Genet.},
  title        = {Increasing diversity in genomics requires investment in equitable partnerships and capacity building},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The missing diversity in human epigenomic studies.
<em>NG</em>, <em>54</em>(6), 737–739. (<a
href="https://doi.org/10.1038/s41588-022-01081-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent work has highlighted a lack of diversity in genomic studies. However, less attention has been given to epigenomics. Here, we show that epigenomic studies are lacking in diversity and propose several solutions to address this problem.},
  archive      = {J_NG},
  author       = {Breeze, Charles E. and Beck, Stephan and Berndt, Sonja I. and Franceschini, Nora},
  doi          = {10.1038/s41588-022-01081-4},
  journal      = {Nature Genetics},
  month        = {6},
  number       = {6},
  pages        = {737-739},
  shortjournal = {Nat. Genet.},
  title        = {The missing diversity in human epigenomic studies},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Author correction: Genome-wide association analyses
identify new brugada syndrome risk loci and highlight a new mechanism of
sodium channel regulation in disease susceptibility. <em>NG</em>,
<em>54</em>(5), 735. (<a
href="https://doi.org/10.1038/s41588-022-01079-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Barc, Julien and Tadros, Rafik and Glinge, Charlotte and Chiang, David Y. and Jouni, Mariam and Simonet, Floriane and Jurgens, Sean J. and Baudic, Manon and Nicastro, Michele and Potet, Franck and Offerhaus, Joost A. and Walsh, Roddy and Choi, Seung Hoan and Verkerk, Arie O. and Mizusawa, Yuka and Anys, Soraya and Minois, Damien and Arnaud, Marine and Duchateau, Josselin and Wijeyeratne, Yanushi D. and Muir, Alison and Papadakis, Michael and Castelletti, Silvia and Torchio, Margherita and Ortuño, Cristina Gil and Lacunza, Javier and Giachino, Daniela F. and Cerrato, Natascia and Martins, Raphaël P. and Campuzano, Oscar and Van Dooren, Sonia and Thollet, Aurélie and Kyndt, Florence and Mazzanti, Andrea and Clémenty, Nicolas and Bisson, Arnaud and Corveleyn, Anniek and Stallmeyer, Birgit and Dittmann, Sven and Saenen, Johan and Noël, Antoine and Honarbakhsh, Shohreh and Rudic, Boris and Marzak, Halim and Rowe, Matthew K. and Federspiel, Claire and Le Page, Sophie and Placide, Leslie and Milhem, Antoine and Barajas-Martinez, Hector and Beckmann, Britt-Maria and Krapels, Ingrid P. and Steinfurt, Johannes and Winkel, Bo Gregers and Jabbari, Reza and Shoemaker, Moore B. and Boukens, Bas J. and Škorić-Milosavljević, Doris and Bikker, Hennie and Manevy, Federico and Lichtner, Peter and Ribasés, Marta and Meitinger, Thomas and Müller-Nurasyid, Martina and Veldink, Jan H. and van den Berg, Leonard H. and Van Damme, Philip and Cusi, Daniele and Lanzani, Chiara and Rigade, Sidwell and Charpentier, Eric and Baron, Estelle and Bonnaud, Stéphanie and Lecointe, Simon and Donnart, Audrey and Le Marec, Hervé and Chatel, Stéphanie and Karakachoff, Matilde and Bézieau, Stéphane and London, Barry and Tfelt-Hansen, Jacob and Roden, Dan and Odening, Katja E. and Cerrone, Marina and Chinitz, Larry A. and Volders, Paul G. and van de Berg, Maarten P. and Laurent, Gabriel and Faivre, Laurence and Antzelevitch, Charles and Kääb, Stefan and Arnaout, Alain Al and Dupuis, Jean-Marc and Pasquie, Jean-Luc and Billon, Olivier and Roberts, Jason D. and Jesel, Laurence and Borggrefe, Martin and Lambiase, Pier D. and Mansourati, Jacques and Loeys, Bart and Leenhardt, Antoine and Guicheney, Pascale and Maury, Philippe and Schulze-Bahr, Eric and Robyns, Tomas and Breckpot, Jeroen and Babuty, Dominique and Priori, Silvia G. and Napolitano, Carlo and de Asmundis, Carlo and Brugada, Pedro and Brugada, Ramon and Arbelo, Elena and Brugada, Josep and Mabo, Philippe and Behar, Nathalie and Giustetto, Carla and Molina, Maria Sabater and Gimeno, Juan R. and Hasdemir, Can and Schwartz, Peter J. and Crotti, Lia and McKeown, Pascal P. and Sharma, Sanjay and Behr, Elijah R. and Haissaguerre, Michel and Sacher, Frédéric and Rooryck, Caroline and Tan, Hanno L. and Remme, Carol A. and Postema, Pieter G. and Delmar, Mario and Ellinor, Patrick T. and Lubitz, Steven A. and Gourraud, Jean-Baptiste and Tanck, Michael W. and George, Alfred L. and MacRae, Calum A. and Burridge, Paul W. and Dina, Christian and Probst, Vincent and Wilde, Arthur A. and Schott, Jean-Jacques and Redon, Richard and Bezzina, Connie R.},
  doi          = {10.1038/s41588-022-01079-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {735},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Genome-wide association analyses identify new brugada syndrome risk loci and highlight a new mechanism of sodium channel regulation in disease susceptibility},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sequence-based modeling of three-dimensional genome
architecture from kilobase to chromosome scale. <em>NG</em>,
<em>54</em>(5), 725–734. (<a
href="https://doi.org/10.1038/s41588-022-01065-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To learn how genomic sequence influences multiscale three-dimensional (3D) genome architecture, this manuscript presents a sequence-based deep-learning approach, Orca, that predicts directly from sequence the 3D genome architecture from kilobase to whole-chromosome scale. Orca captures the sequence dependencies of structures including chromatin compartments and topologically associating domains, as well as diverse types of interactions from CTCF-mediated to enhancer–promoter interactions and Polycomb-mediated interactions with cell-type specificity. Orca enables various applications including predicting structural variant effects on multiscale genome organization and it recapitulated effects of experimentally studied variants at varying sizes (300 bp to 90 Mb). Moreover, Orca enables in silico virtual screens to probe the sequence basis of 3D genome organization at different scales. At the submegabase scale, it predicted specific transcription factor motifs underlying cell-type-specific genome interactions. At the compartment scale, virtual screens of sequence activities suggest a model for the sequence basis of chromatin compartments with a prominent role of transcription start sites. Orca is a sequence-based deep-learning algorithm that predicts 3D genome architecture from kilobase to whole-chromosome scale, including the impact of structural variants. In silico modeling identifies a putative sequence basis for chromatin compartment formation.},
  archive      = {J_NG},
  author       = {Zhou, Jian},
  doi          = {10.1038/s41588-022-01065-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {725-734},
  shortjournal = {Nat. Genet.},
  title        = {Sequence-based modeling of three-dimensional genome architecture from kilobase to chromosome scale},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). SunUp and sunset genomes revealed impact of particle
bombardment mediated transformation and domestication history in papaya.
<em>NG</em>, <em>54</em>(5), 715–724. (<a
href="https://doi.org/10.1038/s41588-022-01068-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transgenic papaya is widely publicized for controlling papaya ringspot virus. However, the impact of particle bombardment on the genome remains unknown. The transgenic SunUp and its progenitor Sunset genomes were assembled into 351.5 and 350.3 Mb in nine chromosomes, respectively. We identified a 1.64 Mb insertion containing three transgenic insertions in SunUp chromosome 5, consisting of 52 nuclear-plastid, 21 nuclear-mitochondrial and 1 nuclear genomic fragments. A 591.9 kb fragment in chromosome 5 was translocated into the 1.64 Mb insertion. We assembled a gapless 9.8 Mb hermaphrodite-specific region of the Yh chromosome and its 6.0 Mb X counterpart. Resequencing 86 genomes revealed three distinct groups, validating their geographic origin and breeding history. We identified 147 selective sweeps and defined the essential role of zeta-carotene desaturase in carotenoid accumulation during domestication. Our findings elucidated the impact of particle bombardment and improved our understanding of sex chromosomes and domestication to expedite papaya improvement. Genome assemblies for the SunUp transgenic papaya and its progenitor Sunset identify three transgene-insertions in SunUp. Resequencing of 86 papaya genomes highlights the impacts of breeding and geographic origin.},
  archive      = {J_NG},
  author       = {Yue, Jingjing and VanBuren, Robert and Liu, Juan and Fang, Jingping and Zhang, Xingtan and Liao, Zhenyang and Wai, Ching Man and Xu, Xiuming and Chen, Shuai and Zhang, Shengchen and Ma, Xiaokai and Ma, Yaying and Yu, Hongying and Lin, Jing and Zhou, Ping and Huang, Yongji and Deng, Ban and Deng, Fang and Zhao, Xiaobing and Yan, Hansong and Fatima, Mahpara and Zerpa-Catanho, Dessireé and Zhang, Xiaodan and Lin, Zhicong and Yang, Mei and Chen, Nancy J. and Mora-Newcomer, Eric and Quesada-Rojas, Patricia and Bogantes, Antonio and Jiménez, Víctor M. and Tang, Haibao and Zhang, Jisen and Wang, Ming-Li and Paull, Robert E. and Yu, Qingyi and Ming, Ray},
  doi          = {10.1038/s41588-022-01068-1},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {715-724},
  shortjournal = {Nat. Genet.},
  title        = {SunUp and sunset genomes revealed impact of particle bombardment mediated transformation and domestication history in papaya},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Systematic characterization of gene function in the
photosynthetic alga chlamydomonas reinhardtii. <em>NG</em>,
<em>54</em>(5), 705–714. (<a
href="https://doi.org/10.1038/s41588-022-01052-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most genes in photosynthetic organisms remain functionally uncharacterized. Here, using a barcoded mutant library of the model eukaryotic alga Chlamydomonas reinhardtii, we determined the phenotypes of more than 58,000 mutants under more than 121 different environmental growth conditions and chemical treatments. A total of 59% of genes are represented by at least one mutant that showed a phenotype, providing clues to the functions of thousands of genes. Mutant phenotypic profiles place uncharacterized genes into functional pathways such as DNA repair, photosynthesis, the CO2-concentrating mechanism and ciliogenesis. We illustrate the value of this resource by validating phenotypes and gene functions, including three new components of an actin cytoskeleton defense pathway. The data also inform phenotype discovery in land plants; mutants in Arabidopsis thaliana genes exhibit phenotypes similar to those we observed in their Chlamydomonas homologs. We anticipate that this resource will guide the functional characterization of genes across the tree of life. Systematic phenotyping of 58,101 mutants of the model eukaryotic alga Chlamydomonas reinhardtii under 121 environmental and chemical stress conditions provides a large resource for characterizing gene function.},
  archive      = {J_NG},
  author       = {Fauser, Friedrich and Vilarrasa-Blasi, Josep and Onishi, Masayuki and Ramundo, Silvia and Patena, Weronika and Millican, Matthew and Osaki, Jacqueline and Philp, Charlotte and Nemeth, Matthew and Salomé, Patrice A. and Li, Xiaobo and Wakao, Setsuko and Kim, Rick G. and Kaye, Yuval and Grossman, Arthur R. and Niyogi, Krishna K. and Merchant, Sabeeha S. and Cutler, Sean R. and Walter, Peter and Dinneny, José R. and Jonikas, Martin C. and Jinkerson, Robert E.},
  doi          = {10.1038/s41588-022-01052-9},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {705-714},
  shortjournal = {Nat. Genet.},
  title        = {Systematic characterization of gene function in the photosynthetic alga chlamydomonas reinhardtii},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genomic selection and genetic architecture of agronomic
traits during modern rapeseed breeding. <em>NG</em>, <em>54</em>(5),
694–704. (<a href="https://doi.org/10.1038/s41588-022-01055-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapeseed (Brassica napus L.) is an important oil-producing crop for the world. Its adaptation, yield and quality have been considerably improved in recent decades, but the genomic basis underlying successful breeding selection remains unclear. Hence, we conducted a comprehensive genomic assessment of rapeseed in the breeding process based on the whole-genome resequencing of 418 diverse rapeseed accessions. We unraveled the genomic basis for the selection of adaptation and agronomic traits. Genome-wide association studies identified 628 associated loci-related causative candidate genes for 56 agronomically important traits, including plant architecture and yield traits. Furthermore, we uncovered nonsynonymous mutations in plausible candidate genes for agronomic traits with significant differences in allele frequency distributions across the improvement process, including the ribosome recycling factor (BnRRF) gene for seed weight. This study provides insights into the genomic basis for improving rapeseed varieties and a valuable genomic resource for genome-assisted rapeseed breeding. Whole-genome resequencing of 418 diverse rapeseed accessions and genome-wide association analyses for 56 agronomic traits provide insights into the genetic architecture and artificial selection or adaptation during modern rapeseed breeding.},
  archive      = {J_NG},
  author       = {Hu, Jihong and Chen, Biyun and Zhao, Jing and Zhang, Fugui and Xie, Ting and Xu, Kun and Gao, Guizhen and Yan, Guixin and Li, Hongge and Li, Lixia and Ji, Gaoxiang and An, Hong and Li, Hao and Huang, Qian and Zhang, Meili and Wu, Jinfeng and Song, Weilin and Zhang, Xiaojun and Luo, Yujie and Chris Pires, J. and Batley, Jacqueline and Tian, Shilin and Wu, Xiaoming},
  doi          = {10.1038/s41588-022-01055-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {694-704},
  shortjournal = {Nat. Genet.},
  title        = {Genomic selection and genetic architecture of agronomic traits during modern rapeseed breeding},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome-wide analysis of cis-regulatory changes underlying
metabolic adaptation of cavefish. <em>NG</em>, <em>54</em>(5), 684–693.
(<a href="https://doi.org/10.1038/s41588-022-01049-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cis-regulatory changes are key drivers of adaptative evolution. However, their contribution to the metabolic adaptation of organisms is not well understood. Here, we used a unique vertebrate model, Astyanax mexicanus—different morphotypes of which survive in nutrient-rich surface and nutrient-deprived cave waters—to uncover gene regulatory networks underlying metabolic adaptation. We performed genome-wide epigenetic profiling in the liver tissues of Astyanax and found that many of the identified cis-regulatory elements (CREs) have genetically diverged and have differential chromatin features between surface and cave morphotypes, while retaining remarkably similar regulatory signatures between independently derived cave populations. One such CRE in the hpdb gene harbors a genomic deletion in cavefish that abolishes IRF2 repressor binding and derepresses enhancer activity in reporter assays. Selection of this mutation in multiple independent cave populations supports its importance in cave adaptation, and provides novel molecular insights into the evolutionary trade-off between loss of pigmentation and adaptation to food-deprived caves. Genome-wide epigenomic profiling of liver tissue from one surface and two independent cave populations of Astyanax mexicanus sheds light on regulatory changes underlying metabolic adaptations to the nutrient-deprived cave environment.},
  archive      = {J_NG},
  author       = {Krishnan, Jaya and Seidel, Chris W. and Zhang, Ning and Singh, Narendra Pratap and VanCampen, Jake and Peuß, Robert and Xiong, Shaolei and Kenzior, Alexander and Li, Hua and Conaway, Joan W. and Rohner, Nicolas},
  doi          = {10.1038/s41588-022-01049-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {684-693},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide analysis of cis-regulatory changes underlying metabolic adaptation of cavefish},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). HOXB13 suppresses de novo lipogenesis through HDAC3-mediated
epigenetic reprogramming in prostate cancer. <em>NG</em>,
<em>54</em>(5), 670–683. (<a
href="https://doi.org/10.1038/s41588-022-01045-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {HOXB13, a homeodomain transcription factor, critically regulates androgen receptor (AR) activities and androgen-dependent prostate cancer (PCa) growth. However, its functions in AR-independent contexts remain elusive. Here we report HOXB13 interaction with histone deacetylase HDAC3, which is disrupted by the HOXB13 G84E mutation that has been associated with early-onset PCa. Independently of AR, HOXB13 recruits HDAC3 to lipogenic enhancers to catalyze histone deacetylation and suppress lipogenic regulators such as fatty acid synthase. Analysis of human tissues reveals that the HOXB13 gene is hypermethylated and downregulated in approximately 30% of metastatic castration-resistant PCa. HOXB13 loss or G84E mutation leads to lipid accumulation in PCa cells, thereby promoting cell motility and xenograft tumor metastasis, which is mitigated by pharmaceutical inhibition of fatty acid synthase. In summary, we present evidence that HOXB13 recruits HDAC3 to suppress de novo lipogenesis and inhibit tumor metastasis and that lipogenic pathway inhibitors may be useful to treat HOXB13-low PCa. HOXB13 suppresses a lipogenic transcriptional program in prostate cancer (PCa) through HDAC3 recruitment to enhancers. Loss of HOXB13 leads to lipid accumulation in PCa cells, promoting cell motility in vitro and xenograft tumor metastasis in vivo.},
  archive      = {J_NG},
  author       = {Lu, Xiaodong and Fong, Ka-wing and Gritsina, Galina and Wang, Fang and Baca, Sylvan C. and Brea, Lourdes T. and Berchuck, Jacob E. and Spisak, Sandor and Ross, Jenny and Morrissey, Colm and Corey, Eva and Chandel, Navdeep S. and Catalona, William J. and Yang, Ximing and Freedman, Matthew L. and Zhao, Jonathan C. and Yu, Jindan},
  doi          = {10.1038/s41588-022-01045-8},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {670-683},
  shortjournal = {Nat. Genet.},
  title        = {HOXB13 suppresses de novo lipogenesis through HDAC3-mediated epigenetic reprogramming in prostate cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Breast tumor microenvironment structures are associated with
genomic features and clinical outcome. <em>NG</em>, <em>54</em>(5),
660–669. (<a href="https://doi.org/10.1038/s41588-022-01041-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The functions of the tumor microenvironment (TME) are orchestrated by precise spatial organization of specialized cells, yet little is known about the multicellular structures that form within the TME. Here we systematically mapped TME structures in situ using imaging mass cytometry and multitiered spatial analysis of 693 breast tumors linked to genomic and clinical data. We identified ten recurrent TME structures that varied by vascular content, stromal quiescence versus activation, and leukocyte composition. These TME structures had distinct enrichment patterns among breast cancer subtypes, and some were associated with genomic profiles indicative of immune escape. Regulatory and dysfunctional T cells co-occurred in large ‘suppressed expansion’ structures. These structures were characterized by high cellular diversity, proliferating cells and enrichment for BRCA1 and CASP8 mutations and predicted poor outcome in estrogen-receptor-positive disease. The multicellular structures revealed here link conserved spatial organization to local TME function and could improve patient stratification. Imaging mass cytometry profiling of 693 breast tumors identified 10 recurrent tumor microenvironment spatial structures. These structures were enriched in different molecular subtypes and can be associated with genomic profiles and outcomes.},
  archive      = {J_NG},
  author       = {Danenberg, Esther and Bardwell, Helen and Zanotelli, Vito R. T. and Provenzano, Elena and Chin, Suet-Feung and Rueda, Oscar M. and Green, Andrew and Rakha, Emad and Aparicio, Samuel and Ellis, Ian O. and Bodenmiller, Bernd and Caldas, Carlos and Ali, H. Raza},
  doi          = {10.1038/s41588-022-01041-y},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {660-669},
  shortjournal = {Nat. Genet.},
  title        = {Breast tumor microenvironment structures are associated with genomic features and clinical outcome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Meningioma DNA methylation groups identify biological
drivers and therapeutic vulnerabilities. <em>NG</em>, <em>54</em>(5),
649–659. (<a href="https://doi.org/10.1038/s41588-022-01061-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Meningiomas are the most common primary intracranial tumors. There are no effective medical therapies for meningioma patients, and new treatments have been encumbered by limited understanding of meningioma biology. Here, we use DNA methylation profiling on 565 meningiomas integrated with genetic, transcriptomic, biochemical, proteomic and single-cell approaches to show meningiomas are composed of three DNA methylation groups with distinct clinical outcomes, biological drivers and therapeutic vulnerabilities. Merlin-intact meningiomas (34%) have the best outcomes and are distinguished by NF2/Merlin regulation of susceptibility to cytotoxic therapy. Immune-enriched meningiomas (38%) have intermediate outcomes and are distinguished by immune infiltration, HLA expression and lymphatic vessels. Hypermitotic meningiomas (28%) have the worst outcomes and are distinguished by convergent genetic and epigenetic mechanisms driving the cell cycle and resistance to cytotoxic therapy. To translate these findings into clinical practice, we show cytostatic cell cycle inhibitors attenuate meningioma growth in cell culture, organoids, xenografts and patients. DNA methylation profiling of 565 meningiomas highlights three groups associated with distinct molecular, clinical and therapeutic features.},
  archive      = {J_NG},
  author       = {Choudhury, Abrar and Magill, Stephen T. and Eaton, Charlotte D. and Prager, Briana C. and Chen, William C. and Cady, Martha A. and Seo, Kyounghee and Lucas, Calixto-Hope G. and Casey-Clyde, Tim J. and Vasudevan, Harish N. and Liu, S. John and Villanueva-Meyer, Javier E. and Lam, Tai-Chung and Pu, Jenny Kan-Suen and Li, Lai-Fung and Leung, Gilberto Ka-Kit and Swaney, Danielle L. and Zhang, Michael Y. and Chan, Jason W. and Qiu, Zhixin and Martin, Michael V. and Susko, Matthew S. and Braunstein, Steve E. and Bush, Nancy Ann Oberheim and Schulte, Jessica D. and Butowski, Nicholas and Sneed, Penny K. and Berger, Mitchel S. and Krogan, Nevan J. and Perry, Arie and Phillips, Joanna J. and Solomon, David A. and Costello, Joseph F. and McDermott, Michael W. and Rich, Jeremy N. and Raleigh, David R.},
  doi          = {10.1038/s41588-022-01061-8},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {649-659},
  shortjournal = {Nat. Genet.},
  title        = {Meningioma DNA methylation groups identify biological drivers and therapeutic vulnerabilities},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CCL22 mutations drive natural killer cell
lymphoproliferative disease by deregulating microenvironmental
crosstalk. <em>NG</em>, <em>54</em>(5), 637–648. (<a
href="https://doi.org/10.1038/s41588-022-01059-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chronic lymphoproliferative disorder of natural killer cells (CLPD-NK) is characterized by clonal expansion of natural killer (NK) cells where the underlying genetic mechanisms are incompletely understood. In the present study, we report somatic mutations in the chemokine gene CCL22 as the hallmark of a distinct subset of CLPD-NK. CCL22 mutations were enriched at highly conserved residues, mutually exclusive of STAT3 mutations and associated with gene expression programs that resembled normal CD16dim/CD56bright NK cells. Mechanistically, the mutations resulted in ligand-biased chemokine receptor signaling, with decreased internalization of the G-protein-coupled receptor (GPCR) for CCL22, CCR4, via impaired β-arrestin recruitment. This resulted in increased cell chemotaxis in vitro, bidirectional crosstalk with the hematopoietic microenvironment and enhanced NK cell proliferation in vivo in transgenic human IL-15 mice. Somatic CCL22 mutations illustrate a unique mechanism of tumor formation in which gain-of-function chemokine mutations promote tumorigenesis by biased GPCR signaling and dysregulation of microenvironmental crosstalk. Genomic and transcriptomic analyses of chronic lymphoproliferative disorder of natural killer cells identifies somatic gain-of-function mutations in the chemokine gene CCL22 with cell-extrinsic effects. Mutations caused biased signaling downstream of the G-protein-coupled receptor for CCL22 and deregulated interactions with the hematopoietic microenvironment.},
  archive      = {J_NG},
  author       = {Baer, Constance and Kimura, Shunsuke and Rana, Mitra S. and Kleist, Andrew B. and Flerlage, Tim and Feith, David J. and Chockley, Peter and Walter, Wencke and Meggendorfer, Manja and Olson, Thomas L. and Cheon, HeeJin and Olson, Kristine C. and Ratan, Aakrosh and Mueller, Martha-Lena and Foran, James M. and Janke, Laura J. and Qu, Chunxu and Porter, Shaina N. and Pruett-Miller, Shondra M. and Kalathur, Ravi C. and Haferlach, Claudia and Kern, Wolfgang and Paietta, Elisabeth and Thomas, Paul G. and Babu, M. Madan and Loughran, Thomas P. and Iacobucci, Ilaria and Haferlach, Torsten and Mullighan, Charles G.},
  doi          = {10.1038/s41588-022-01059-2},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {637-648},
  shortjournal = {Nat. Genet.},
  title        = {CCL22 mutations drive natural killer cell lymphoproliferative disease by deregulating microenvironmental crosstalk},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The disordered n-terminal domain of DNMT3A recognizes
H2AK119ub and is required for postnatal development. <em>NG</em>,
<em>54</em>(5), 625–636. (<a
href="https://doi.org/10.1038/s41588-022-01063-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA methyltransferase 3a (DNMT3A) plays a crucial role during mammalian development. Two isoforms of DNMT3A are differentially expressed from stem cells to somatic tissues, but their individual functions remain largely uncharacterized. Here we report that the long isoform DNMT3A1, but not the short DNMT3A2, is essential for mouse postnatal development. DNMT3A1 binds to and regulates bivalent neurodevelopmental genes in the brain. Strikingly, Dnmt3a1 knockout perinatal lethality could be partially rescued by DNMT3A1 restoration in the nervous system. We further show that the intrinsically disordered N terminus of DNMT3A1 is required for normal development and DNA methylation at DNMT3A1-enriched regions. Mechanistically, a ubiquitin-interacting motif embedded in a putative α-helix within the N terminus binds to mono-ubiquitinated histone H2AK119, probably mediating recruitment of DNMT3A1 to Polycomb-regulated regions. These data demonstrate an isoform-specific role for DNMT3A1 in mouse postnatal development and reveal the N terminus as a necessary regulatory domain for DNMT3A1 chromatin occupancy and functions in the nervous system. The long isoform of DNMT3A is essential for mouse postnatal development and regulates bivalent genes in the brain, likely via a PRC1-mediated mechanism.},
  archive      = {J_NG},
  author       = {Gu, Tianpeng and Hao, Dapeng and Woo, Junsung and Huang, Teng-Wei and Guo, Lei and Lin, Xueqiu and Guzman, Anna G. and Tovy, Ayala and Rosas, Carina and Jeong, Mira and Zhou, Yubin and Deneen, Benjamin and Huang, Yun and Li, Wei and Goodell, Margaret A.},
  doi          = {10.1038/s41588-022-01063-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {625-636},
  shortjournal = {Nat. Genet.},
  title        = {The disordered N-terminal domain of DNMT3A recognizes H2AK119ub and is required for postnatal development},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DeepSTARR predicts enhancer activity from DNA sequence and
enables the de novo design of synthetic enhancers. <em>NG</em>,
<em>54</em>(5), 613–624. (<a
href="https://doi.org/10.1038/s41588-022-01048-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enhancer sequences control gene expression and comprise binding sites (motifs) for different transcription factors (TFs). Despite extensive genetic and computational studies, the relationship between DNA sequence and regulatory activity is poorly understood, and de novo enhancer design has been challenging. Here, we built a deep-learning model, DeepSTARR, to quantitatively predict the activities of thousands of developmental and housekeeping enhancers directly from DNA sequence in Drosophila melanogaster S2 cells. The model learned relevant TF motifs and higher-order syntax rules, including functionally nonequivalent instances of the same TF motif that are determined by motif-flanking sequence and intermotif distances. We validated these rules experimentally and demonstrated that they can be generalized to humans by testing more than 40,000 wildtype and mutant Drosophila and human enhancers. Finally, we designed and functionally validated synthetic enhancers with desired activities de novo. A deep-learning model called DeepSTARR quantitatively predicts enhancer activity on the basis of DNA sequence. The model learns relevant motifs and syntax rules, allowing for the design of synthetic enhancers with specific strengths.},
  archive      = {J_NG},
  author       = {de Almeida, Bernardo P. and Reiter, Franziska and Pagani, Michaela and Stark, Alexander},
  doi          = {10.1038/s41588-022-01048-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {613-624},
  shortjournal = {Nat. Genet.},
  title        = {DeepSTARR predicts enhancer activity from DNA sequence and enables the de novo design of synthetic enhancers},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Prioritization of autoimmune disease-associated genetic
variants that perturb regulatory element activity in t cells.
<em>NG</em>, <em>54</em>(5), 603–612. (<a
href="https://doi.org/10.1038/s41588-022-01056-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-wide association studies (GWASs) have uncovered hundreds of autoimmune disease-associated loci; however, the causal genetic variants within each locus are mostly unknown. Here, we perform high-throughput allele-specific reporter assays to prioritize disease-associated variants for five autoimmune diseases. By examining variants that both promote allele-specific reporter expression and are located in accessible chromatin, we identify 60 putatively causal variants that enrich for statistically fine-mapped variants by up to 57.8-fold. We introduced the risk allele of a prioritized variant (rs72928038) into a human T cell line and deleted the orthologous sequence in mice, both resulting in reduced BACH2 expression. Naive CD8 T cells from mice containing the deletion had reduced expression of genes that suppress activation and maintain stemness and, upon acute viral infection, displayed greater propensity to become effector T cells. Our results represent an example of an effective approach for prioritizing variants and studying their physiologically relevant effects. Integrating methods that assess allele-specific regulatory activity and chromatin accessibility in T cells identifies putative causal variants for five autoimmune diseases.},
  archive      = {J_NG},
  author       = {Mouri, Kousuke and Guo, Michael H. and de Boer, Carl G. and Lissner, Michelle M. and Harten, Ingrid A. and Newby, Gregory A. and DeBerg, Hannah A. and Platt, Winona F. and Gentili, Matteo and Liu, David R. and Campbell, Daniel J. and Hacohen, Nir and Tewhey, Ryan and Ray, John P.},
  doi          = {10.1038/s41588-022-01056-5},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {603-612},
  shortjournal = {Nat. Genet.},
  title        = {Prioritization of autoimmune disease-associated genetic variants that perturb regulatory element activity in t cells},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Plasma proteome analyses in individuals of european and
african ancestry identify cis-pQTLs and models for proteome-wide
association studies. <em>NG</em>, <em>54</em>(5), 593–602. (<a
href="https://doi.org/10.1038/s41588-022-01051-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Improved understanding of genetic regulation of the proteome can facilitate identification of the causal mechanisms for complex traits. We analyzed data on 4,657 plasma proteins from 7,213 European American (EA) and 1,871 African American (AA) individuals from the Atherosclerosis Risk in Communities study, and further replicated findings on 467 AA individuals from the African American Study of Kidney Disease and Hypertension study. Here, we identified 2,004 proteins in EA and 1,618 in AA, with most overlapping, which showed associations with common variants in cis-regions. Availability of AA samples led to smaller credible sets and notable number of population-specific cis-protein quantitative trait loci. Elastic Net produced powerful models for protein prediction in both populations. An application of proteome-wide association studies to serum urate and gout implicated several proteins, including IL1RN, revealing the promise of the drug anakinra to treat acute gout flares. Our study demonstrates the value of large and diverse ancestry study to investigate the genetic mechanisms of molecular phenotypes and their relationship with complex traits. Analyses of cis-genetic regulation of the plasma proteome in European and African American populations lead to the identification of shared and unique cis-protein quantitative trait loci and models for proteome-wide association studies of complex traits.},
  archive      = {J_NG},
  author       = {Zhang, Jingning and Dutta, Diptavo and Köttgen, Anna and Tin, Adrienne and Schlosser, Pascal and Grams, Morgan E. and Harvey, Benjamin and Yu, Bing and Boerwinkle, Eric and Coresh, Josef and Chatterjee, Nilanjan},
  doi          = {10.1038/s41588-022-01051-w},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {593-602},
  shortjournal = {Nat. Genet.},
  title        = {Plasma proteome analyses in individuals of european and african ancestry identify cis-pQTLs and models for proteome-wide association studies},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Within-sibship genome-wide association analyses decrease
bias in estimates of direct genetic effects. <em>NG</em>,
<em>54</em>(5), 581–592. (<a
href="https://doi.org/10.1038/s41588-022-01062-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Estimates from genome-wide association studies (GWAS) of unrelated individuals capture effects of inherited variation (direct effects), demography (population stratification, assortative mating) and relatives (indirect genetic effects). Family-based GWAS designs can control for demographic and indirect genetic effects, but large-scale family datasets have been lacking. We combined data from 178,086 siblings from 19 cohorts to generate population (between-family) and within-sibship (within-family) GWAS estimates for 25 phenotypes. Within-sibship GWAS estimates were smaller than population estimates for height, educational attainment, age at first birth, number of children, cognitive ability, depressive symptoms and smoking. Some differences were observed in downstream SNP heritability, genetic correlations and Mendelian randomization analyses. For example, the within-sibship genetic correlation between educational attainment and body mass index attenuated towards zero. In contrast, analyses of most molecular phenotypes (for example, low-density lipoprotein-cholesterol) were generally consistent. We also found within-sibship evidence of polygenic adaptation on taller height. Here, we illustrate the importance of family-based GWAS data for phenotypes influenced by demographic and indirect genetic effects. Within-sibship genome-wide association analyses using data from 178,076 siblings illustrate differences between population-based and within-sibship GWAS estimates for phenotypes influenced by demographic and indirect genetic effects.},
  archive      = {J_NG},
  author       = {Howe, Laurence J. and Nivard, Michel G. and Morris, Tim T. and Hansen, Ailin F. and Rasheed, Humaira and Cho, Yoonsu and Chittoor, Geetha and Ahlskog, Rafael and Lind, Penelope A. and Palviainen, Teemu and van der Zee, Matthijs D. and Cheesman, Rosa and Mangino, Massimo and Wang, Yunzhang and Li, Shuai and Klaric, Lucija and Ratliff, Scott M. and Bielak, Lawrence F. and Nygaard, Marianne and Giannelis, Alexandros and Willoughby, Emily A. and Reynolds, Chandra A. and Balbona, Jared V. and Andreassen, Ole A. and Ask, Helga and Baras, Aris and Bauer, Christopher R. and Boomsma, Dorret I. and Campbell, Archie and Campbell, Harry and Chen, Zhengming and Christofidou, Paraskevi and Corfield, Elizabeth and Dahm, Christina C. and Dokuru, Deepika R. and Evans, Luke M. and de Geus, Eco J. C. and Giddaluru, Sudheer and Gordon, Scott D. and Harden, K. Paige and Hill, W. David and Hughes, Amanda and Kerr, Shona M. and Kim, Yongkang and Kweon, Hyeokmoon and Latvala, Antti and Lawlor, Deborah A. and Li, Liming and Lin, Kuang and Magnus, Per and Magnusson, Patrik K. E. and Mallard, Travis T. and Martikainen, Pekka and Mills, Melinda C. and Njølstad, Pål Rasmus and Overton, John D. and Pedersen, Nancy L. and Porteous, David J. and Reid, Jeffrey and Silventoinen, Karri and Southey, Melissa C. and Stoltenberg, Camilla and Tucker-Drob, Elliot M. and Wright, Margaret J. and Hewitt, John K. and Keller, Matthew C. and Stallings, Michael C. and Lee, James J. and Christensen, Kaare and Kardia, Sharon L. R. and Peyser, Patricia A. and Smith, Jennifer A. and Wilson, James F. and Hopper, John L. and Hägg, Sara and Spector, Tim D. and Pingault, Jean-Baptiste and Plomin, Robert and Havdahl, Alexandra and Bartels, Meike and Martin, Nicholas G. and Oskarsson, Sven and Justice, Anne E. and Millwood, Iona Y. and Hveem, Kristian and Naess, Øyvind and Willer, Cristen J. and Åsvold, Bjørn Olav and Koellinger, Philipp D. and Kaprio, Jaakko and Medland, Sarah E. and Walters, Robin G. and Benjamin, Daniel J. and Turley, Patrick and Evans, David M. and Davey Smith, George and Hayward, Caroline and Brumpton, Ben and Hemani, Gibran and Davies, Neil M.},
  doi          = {10.1038/s41588-022-01062-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {581-592},
  shortjournal = {Nat. Genet.},
  title        = {Within-sibship genome-wide association analyses decrease bias in estimates of direct genetic effects},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Improving polygenic prediction in ancestrally diverse
populations. <em>NG</em>, <em>54</em>(5), 573–580. (<a
href="https://doi.org/10.1038/s41588-022-01054-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polygenic risk scores (PRS) have attenuated cross-population predictive performance. As existing genome-wide association studies (GWAS) have been conducted predominantly in individuals of European descent, the limited transferability of PRS reduces their clinical value in non-European populations, and may exacerbate healthcare disparities. Recent efforts to level ancestry imbalance in genomic research have expanded the scale of non-European GWAS, although most remain underpowered. Here, we present a new PRS construction method, PRS-CSx, which improves cross-population polygenic prediction by integrating GWAS summary statistics from multiple populations. PRS-CSx couples genetic effects across populations via a shared continuous shrinkage (CS) prior, enabling more accurate effect size estimation by sharing information between summary statistics and leveraging linkage disequilibrium diversity across discovery samples, while inheriting computational efficiency and robustness from PRS-CS. We show that PRS-CSx outperforms alternative methods across traits with a wide range of genetic architectures, cross-population genetic overlaps and discovery GWAS sample sizes in simulations, and improves the prediction of quantitative traits and schizophrenia risk in non-European populations. PRS-CSx is a polygenic risk score construction method that improves cross-population polygenic prediction by integrating GWAS summary statistics from multiple populations.},
  archive      = {J_NG},
  author       = {Ruan, Yunfeng and Lin, Yen-Feng and Feng, Yen-Chen Anne and Chen, Chia-Yen and Lam, Max and Guo, Zhenglin and He, Lin and Sawa, Akira and Martin, Alicia R. and Qin, Shengying and Huang, Hailiang and Ge, Tian},
  doi          = {10.1038/s41588-022-01054-7},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {573-580},
  shortjournal = {Nat. Genet.},
  title        = {Improving polygenic prediction in ancestrally diverse populations},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multi-ancestry genetic study of type 2 diabetes highlights
the power of diverse populations for discovery and translation.
<em>NG</em>, <em>54</em>(5), 560–572. (<a
href="https://doi.org/10.1038/s41588-022-01058-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We assembled an ancestrally diverse collection of genome-wide association studies (GWAS) of type 2 diabetes (T2D) in 180,834 affected individuals and 1,159,055 controls (48.9% non-European descent) through the Diabetes Meta-Analysis of Trans-Ethnic association studies (DIAMANTE) Consortium. Multi-ancestry GWAS meta-analysis identified 237 loci attaining stringent genome-wide significance (P &amp;lt; 5 × 10−9), which were delineated to 338 distinct association signals. Fine-mapping of these signals was enhanced by the increased sample size and expanded population diversity of the multi-ancestry meta-analysis, which localized 54.4% of T2D associations to a single variant with &amp;gt;50% posterior probability. This improved fine-mapping enabled systematic assessment of candidate causal genes and molecular mechanisms through which T2D associations are mediated, laying the foundations for functional investigations. Multi-ancestry genetic risk scores enhanced transferability of T2D prediction across diverse populations. Our study provides a step toward more effective clinical translation of T2D GWAS to improve global health for all, irrespective of genetic background. Genome-wide association and fine-mapping analyses in ancestrally diverse populations implicate candidate causal genes and mechanisms underlying type 2 diabetes. Trans-ancestry genetic risk scores enhance transferability across populations.},
  archive      = {J_NG},
  author       = {Mahajan, Anubha and Spracklen, Cassandra N. and Zhang, Weihua and Ng, Maggie C. Y. and Petty, Lauren E. and Kitajima, Hidetoshi and Yu, Grace Z. and Rüeger, Sina and Speidel, Leo and Kim, Young Jin and Horikoshi, Momoko and Mercader, Josep M. and Taliun, Daniel and Moon, Sanghoon and Kwak, Soo-Heon and Robertson, Neil R. and Rayner, Nigel W. and Loh, Marie and Kim, Bong-Jo and Chiou, Joshua and Miguel-Escalada, Irene and della Briotta Parolo, Pietro and Lin, Kuang and Bragg, Fiona and Preuss, Michael H. and Takeuchi, Fumihiko and Nano, Jana and Guo, Xiuqing and Lamri, Amel and Nakatochi, Masahiro and Scott, Robert A. and Lee, Jung-Jin and Huerta-Chagoya, Alicia and Graff, Mariaelisa and Chai, Jin-Fang and Parra, Esteban J. and Yao, Jie and Bielak, Lawrence F. and Tabara, Yasuharu and Hai, Yang and Steinthorsdottir, Valgerdur and Cook, James P. and Kals, Mart and Grarup, Niels and Schmidt, Ellen M. and Pan, Ian and Sofer, Tamar and Wuttke, Matthias and Sarnowski, Chloe and Gieger, Christian and Nousome, Darryl and Trompet, Stella and Long, Jirong and Sun, Meng and Tong, Lin and Chen, Wei-Min and Ahmad, Meraj and Noordam, Raymond and Lim, Victor J. Y. and Tam, Claudia H. T. and Joo, Yoonjung Yoonie and Chen, Chien-Hsiun and Raffield, Laura M. and Lecoeur, Cécile and Prins, Bram Peter and Nicolas, Aude and Yanek, Lisa R. and Chen, Guanjie and Jensen, Richard A. and Tajuddin, Salman and Kabagambe, Edmond K. and An, Ping and Xiang, Anny H. and Choi, Hyeok Sun and Cade, Brian E. and Tan, Jingyi and Flanagan, Jack and Abaitua, Fernando and Adair, Linda S. and Adeyemo, Adebowale and Aguilar-Salinas, Carlos A. and Akiyama, Masato and Anand, Sonia S. and Bertoni, Alain and Bian, Zheng and Bork-Jensen, Jette and Brandslund, Ivan and Brody, Jennifer A. and Brummett, Chad M. and Buchanan, Thomas A. and Canouil, Mickaël and Chan, Juliana C. N. and Chang, Li-Ching and Chee, Miao-Li and Chen, Ji and Chen, Shyh-Huei and Chen, Yuan-Tsong and Chen, Zhengming and Chuang, Lee-Ming and Cushman, Mary and Das, Swapan K. and de Silva, H. Janaka and Dedoussis, George and Dimitrov, Latchezar and Doumatey, Ayo P. and Du, Shufa and Duan, Qing and Eckardt, Kai-Uwe and Emery, Leslie S. and Evans, Daniel S. and Evans, Michele K. and Fischer, Krista and Floyd, James S. and Ford, Ian and Fornage, Myriam and Franco, Oscar H. and Frayling, Timothy M. and Freedman, Barry I. and Fuchsberger, Christian and Genter, Pauline and Gerstein, Hertzel C. and Giedraitis, Vilmantas and González-Villalpando, Clicerio and González-Villalpando, Maria Elena and Goodarzi, Mark O. and Gordon-Larsen, Penny and Gorkin, David and Gross, Myron and Guo, Yu and Hackinger, Sophie and Han, Sohee and Hattersley, Andrew T. and Herder, Christian and Howard, Annie-Green and Hsueh, Willa and Huang, Mengna and Huang, Wei and Hung, Yi-Jen and Hwang, Mi Yeong and Hwu, Chii-Min and Ichihara, Sahoko and Ikram, Mohammad Arfan and Ingelsson, Martin and Islam, Md Tariqul and Isono, Masato and Jang, Hye-Mi and Jasmine, Farzana and Jiang, Guozhi and Jonas, Jost B. and Jørgensen, Marit E. and Jørgensen, Torben and Kamatani, Yoichiro and Kandeel, Fouad R. and Kasturiratne, Anuradhani and Katsuya, Tomohiro and Kaur, Varinderpal and Kawaguchi, Takahisa and Keaton, Jacob M. and Kho, Abel N. and Khor, Chiea-Chuen and Kibriya, Muhammad G. and Kim, Duk-Hwan and Kohara, Katsuhiko and Kriebel, Jennifer and Kronenberg, Florian and Kuusisto, Johanna and Läll, Kristi and Lange, Leslie A. and Lee, Myung-Shik and Lee, Nanette R. and Leong, Aaron and Li, Liming and Li, Yun and Li-Gao, Ruifang and Ligthart, Symen and Lindgren, Cecilia M. and Linneberg, Allan and Liu, Ching-Ti and Liu, Jianjun and Locke, Adam E. and Louie, Tin and Luan, Jian’an and Luk, Andrea O. and Luo, Xi and Lv, Jun and Lyssenko, Valeriya and Mamakou, Vasiliki and Mani, K. Radha and Meitinger, Thomas and Metspalu, Andres and Morris, Andrew D. and Nadkarni, Girish N. and Nadler, Jerry L. and Nalls, Michael A. and Nayak, Uma and Nongmaithem, Suraj S. and Ntalla, Ioanna and Okada, Yukinori and Orozco, Lorena and Patel, Sanjay R. and Pereira, Mark A. and Peters, Annette and Pirie, Fraser J. and Porneala, Bianca and Prasad, Gauri and Preissl, Sebastian and Rasmussen-Torvik, Laura J. and Reiner, Alexander P. and Roden, Michael and Rohde, Rebecca and Roll, Kathryn and Sabanayagam, Charumathi and Sander, Maike and Sandow, Kevin and Sattar, Naveed and Schönherr, Sebastian and Schurmann, Claudia and Shahriar, Mohammad and Shi, Jinxiu and Shin, Dong Mun and Shriner, Daniel and Smith, Jennifer A. and So, Wing Yee and Stančáková, Alena and Stilp, Adrienne M. and Strauch, Konstantin and Suzuki, Ken and Takahashi, Atsushi and Taylor, Kent D. and Thorand, Barbara and Thorleifsson, Gudmar and Thorsteinsdottir, Unnur and Tomlinson, Brian and Torres, Jason M. and Tsai, Fuu-Jen and Tuomilehto, Jaakko and Tusie-Luna, Teresa and Udler, Miriam S. and Valladares-Salgado, Adan and van Dam, Rob M. and van Klinken, Jan B. and Varma, Rohit and Vujkovic, Marijana and Wacher-Rodarte, Niels and Wheeler, Eleanor and Whitsel, Eric A. and Wickremasinghe, Ananda R. and van Dijk, Ko Willems and Witte, Daniel R. and Yajnik, Chittaranjan S. and Yamamoto, Ken and Yamauchi, Toshimasa and Yengo, Loïc and Yoon, Kyungheon and Yu, Canqing and Yuan, Jian-Min and Yusuf, Salim and Zhang, Liang and Zheng, Wei and Raffel, Leslie J. and Igase, Michiya and Ipp, Eli and Redline, Susan and Cho, Yoon Shin and Lind, Lars and Province, Michael A. and Hanis, Craig L. and Peyser, Patricia A. and Ingelsson, Erik and Zonderman, Alan B. and Psaty, Bruce M. and Wang, Ya-Xing and Rotimi, Charles N. and Becker, Diane M. and Matsuda, Fumihiko and Liu, Yongmei and Zeggini, Eleftheria and Yokota, Mitsuhiro and Rich, Stephen S. and Kooperberg, Charles and Pankow, James S. and Engert, James C. and Chen, Yii-Der Ida and Froguel, Philippe and Wilson, James G. and Sheu, Wayne H. H. and Kardia, Sharon L. R. and Wu, Jer-Yuarn and Hayes, M. Geoffrey and Ma, Ronald C. W. and Wong, Tien-Yin and Groop, Leif and Mook-Kanamori, Dennis O. and Chandak, Giriraj R. and Collins, Francis S. and Bharadwaj, Dwaipayan and Paré, Guillaume and Sale, Michèle M. and Ahsan, Habibul and Motala, Ayesha A. and Shu, Xiao-Ou and Park, Kyong-Soo and Jukema, J. Wouter and Cruz, Miguel and McKean-Cowdin, Roberta and Grallert, Harald and Cheng, Ching-Yu and Bottinger, Erwin P. and Dehghan, Abbas and Tai, E-Shyong and Dupuis, Josée and Kato, Norihiro and Laakso, Markku and Köttgen, Anna and Koh, Woon-Puay and Palmer, Colin N. A. and Liu, Simin and Abecasis, Goncalo and Kooner, Jaspal S. and Loos, Ruth J. F. and North, Kari E. and Haiman, Christopher A. and Florez, Jose C. and Saleheen, Danish and Hansen, Torben and Pedersen, Oluf and Mägi, Reedik and Langenberg, Claudia and Wareham, Nicholas J. and Maeda, Shiro and Kadowaki, Takashi and Lee, Juyoung and Millwood, Iona Y. and Walters, Robin G. and Stefansson, Kari and Myers, Simon R. and Ferrer, Jorge and Gaulton, Kyle J. and Meigs, James B. and Mohlke, Karen L. and Gloyn, Anna L. and Bowden, Donald W. and Below, Jennifer E. and Chambers, John C. and Sim, Xueling and Boehnke, Michael and Rotter, Jerome I. and McCarthy, Mark I. and Morris, Andrew P.},
  doi          = {10.1038/s41588-022-01058-3},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {560-572},
  shortjournal = {Nat. Genet.},
  title        = {Multi-ancestry genetic study of type 2 diabetes highlights the power of diverse populations for discovery and translation},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetic architecture of 11 major psychiatric disorders at
biobehavioral, functional genomic and molecular genetic levels of
analysis. <em>NG</em>, <em>54</em>(5), 548–559. (<a
href="https://doi.org/10.1038/s41588-022-01057-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We interrogate the joint genetic architecture of 11 major psychiatric disorders at biobehavioral, functional genomic and molecular genetic levels of analysis. We identify four broad factors (neurodevelopmental, compulsive, psychotic and internalizing) that underlie genetic correlations among the disorders and test whether these factors adequately explain their genetic correlations with biobehavioral traits. We introduce stratified genomic structural equation modeling, which we use to identify gene sets that disproportionately contribute to genetic risk sharing. This includes protein-truncating variant-intolerant genes expressed in excitatory and GABAergic brain cells that are enriched for genetic overlap across disorders with psychotic features. Multivariate association analyses detect 152 (20 new) independent loci that act on the individual factors and identify nine loci that act heterogeneously across disorders within a factor. Despite moderate-to-high genetic correlations across all 11 disorders, we find little utility of a single dimension of genetic risk across psychiatric disorders either at the level of biobehavioral correlates or at the level of individual variants. Joint analysis of 11 major psychiatric disorders identifies four broad factor underlying genetic correlations among the disorders. Association analyses detect 152 loci acting on these factors and identify 9 loci that act heterogeneously across disorders.},
  archive      = {J_NG},
  author       = {Grotzinger, Andrew D. and Mallard, Travis T. and Akingbuwa, Wonuola A. and Ip, Hill F. and Adams, Mark J. and Lewis, Cathryn M. and McIntosh, Andrew M. and Grove, Jakob and Dalsgaard, Søren and Lesch, Klaus-Peter and Strom, Nora and Meier, Sandra M. and Mattheisen, Manuel and Børglum, Anders D. and Mors, Ole and Breen, Gerome and Lee, Phil H. and Kendler, Kenneth S. and Smoller, Jordan W. and Tucker-Drob, Elliot M. and Nivard, Michel G.},
  doi          = {10.1038/s41588-022-01057-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {548-559},
  shortjournal = {Nat. Genet.},
  title        = {Genetic architecture of 11 major psychiatric disorders at biobehavioral, functional genomic and molecular genetic levels of analysis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Exome sequencing in bipolar disorder identifies AKAP11 as a
risk gene shared with schizophrenia. <em>NG</em>, <em>54</em>(5),
541–547. (<a href="https://doi.org/10.1038/s41588-022-01034-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report results from the Bipolar Exome (BipEx) collaboration analysis of whole-exome sequencing of 13,933 patients with bipolar disorder (BD) matched with 14,422 controls. We find an excess of ultra-rare protein-truncating variants (PTVs) in patients with BD among genes under strong evolutionary constraint in both major BD subtypes. We find enrichment of ultra-rare PTVs within genes implicated from a recent schizophrenia exome meta-analysis (SCHEMA; 24,248 cases and 97,322 controls) and among binding targets of CHD8. Genes implicated from genome-wide association studies (GWASs) of BD, however, are not significantly enriched for ultra-rare PTVs. Combining gene-level results with SCHEMA, AKAP11 emerges as a definitive risk gene (odds ratio (OR) = 7.06, P = 2.83 × 10−9). At the protein level, AKAP-11 interacts with GSK3B, the hypothesized target of lithium, a primary treatment for BD. Our results lend support to BD’s polygenicity, demonstrating a role for rare coding variation as a significant risk factor in BD etiology. Exome sequencing analysis of 13,933 individuals with bipolar disorder finds enrichment of ultra-rare protein-truncating variants in constrained genes. Combined analysis with schizophrenia exome data identifies AKAP11 as a risk gene for both disorders.},
  archive      = {J_NG},
  author       = {Palmer, Duncan S. and Howrigan, Daniel P. and Chapman, Sinéad B. and Adolfsson, Rolf and Bass, Nick and Blackwood, Douglas and Boks, Marco P. M. and Chen, Chia-Yen and Churchhouse, Claire and Corvin, Aiden P. and Craddock, Nicholas and Curtis, David and Di Florio, Arianna and Dickerson, Faith and Freimer, Nelson B. and Goes, Fernando S. and Jia, Xiaoming and Jones, Ian and Jones, Lisa and Jonsson, Lina and Kahn, Rene S. and Landén, Mikael and Locke, Adam E. and McIntosh, Andrew M. and McQuillin, Andrew and Morris, Derek W. and O’Donovan, Michael C. and Ophoff, Roel A. and Owen, Michael J. and Pedersen, Nancy L. and Posthuma, Danielle and Reif, Andreas and Risch, Neil and Schaefer, Catherine and Scott, Laura and Singh, Tarjinder and Smoller, Jordan W. and Solomonson, Matthew and Clair, David St. and Stahl, Eli A. and Vreeker, Annabel and Walters, James T. R. and Wang, Weiqing and Watts, Nicholas A. and Yolken, Robert and Zandi, Peter P. and Neale, Benjamin M.},
  doi          = {10.1038/s41588-022-01034-x},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {541-547},
  shortjournal = {Nat. Genet.},
  title        = {Exome sequencing in bipolar disorder identifies AKAP11 as a risk gene shared with schizophrenia},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genomic analyses reveal domestication history and transgenic
insertions in papaya. <em>NG</em>, <em>54</em>(5), 539–540. (<a
href="https://doi.org/10.1038/s41588-022-01030-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genome of the SunUp transgenic papaya cultivar includes a complex 1.64-Mb insertion that contains 3 transgenic fragments integrated with 61 nuclear genome fragments from the progenitor Sunset cultivar and 13 organelle genome fragments. Population genomic analyses yielded 147 selective sweeps during papaya domestication, which include essential genes that are involved in fruit flesh color formation and sugar content.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01030-1},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {539-540},
  shortjournal = {Nat. Genet.},
  title        = {Genomic analyses reveal domestication history and transgenic insertions in papaya},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DNMT3A binds ubiquitinated histones to regulate bivalent
genes. <em>NG</em>, <em>54</em>(5), 537–538. (<a
href="https://doi.org/10.1038/s41588-022-01073-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study demonstrates that the disordered N-terminal domain of DNMT3A1 binds PRC1-catalyzed H2AK119ub, targeting DNA methylation to bivalent promoters in mouse brain cortical cells. Methylation around bivalent genes is critical for mouse postnatal development, and could be equally important in other cell types and in disease.},
  archive      = {J_NG},
  author       = {Parry, Aled J. and Reik, Wolf},
  doi          = {10.1038/s41588-022-01073-4},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {537-538},
  shortjournal = {Nat. Genet.},
  title        = {DNMT3A binds ubiquitinated histones to regulate bivalent genes},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Promoting the genomic revolution in africa through the
nigerian 100K genome project. <em>NG</em>, <em>54</em>(5), 531–536. (<a
href="https://doi.org/10.1038/s41588-022-01071-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To leverage the genetic diversity in Nigeria, we established the Non-Communicable Diseases Genetic Heritage Study (NCD-GHS) consortium to help produce a comprehensive catalog of human genetic variation in Nigeria and assess the burden and etiological characteristics of non-communicable diseases in 100,000 adults in Nigeria.},
  archive      = {J_NG},
  author       = {Fatumo, Segun and Yakubu, Aminu and Oyedele, Olubukunola and Popoola, Jumi and Attipoe, Delali Attiogbe and Eze-Echesi, Golibe and Modibbo, Fatima Z and Ado-Wanka, Nabila and Salako, Omolola and Nashiru, Oyekanmi and Salako, Babatunde L and O’Dushlaine, Colm and Ene-Obong, Abasi},
  doi          = {10.1038/s41588-022-01071-6},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {531-536},
  shortjournal = {Nat. Genet.},
  title        = {Promoting the genomic revolution in africa through the nigerian 100K genome project},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Polygenic risk scores for CARDINAL study. <em>NG</em>,
<em>54</em>(5), 527–530. (<a
href="https://doi.org/10.1038/s41588-022-01074-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Cardiometabolic Disorders in African-Ancestry Populations (CARDINAL) study site is a well-powered, first-of-its-kind resource for developing, refining and validating methods for research into polygenic risk scores that accounts for local ancestry, to improve risk prediction in diverse populations.},
  archive      = {J_NG},
  author       = {Adebamowo, Clement A. and Adeyemo, Adebowale and Ashaye, Adeyinka and Akpa, Onoja M. and Chikowore, Tinashe and Choudhury, Ananyo and Fakim, Yasmina J. and Fatumo, Segun and Hanchard, Neil and Hauser, Michael and Mitchell, Braxton and Mulder, Nicola and Ofori-Acquah, Solomon F. and Owolabi, Mayowa and Ramsay, Michèle and Tayo, Bamidele and VasanthKumar, Archana Bhavani and Zhang, Yuji and Adebamowo, Sally N.},
  doi          = {10.1038/s41588-022-01074-3},
  journal      = {Nature Genetics},
  month        = {5},
  number       = {5},
  pages        = {527-530},
  shortjournal = {Nat. Genet.},
  title        = {Polygenic risk scores for CARDINAL study},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Pangenome-based genome inference allows efficient and
accurate genotyping across a wide spectrum of variant classes.
<em>NG</em>, <em>54</em>(4), 518–525. (<a
href="https://doi.org/10.1038/s41588-022-01043-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Typical genotyping workflows map reads to a reference genome before identifying genetic variants. Generating such alignments introduces reference biases and comes with substantial computational burden. Furthermore, short-read lengths limit the ability to characterize repetitive genomic regions, which are particularly challenging for fast k-mer-based genotypers. In the present study, we propose a new algorithm, PanGenie, that leverages a haplotype-resolved pangenome reference together with k-mer counts from short-read sequencing data to genotype a wide spectrum of genetic variation—a process we refer to as genome inference. Compared with mapping-based approaches, PanGenie is more than 4 times faster at 30-fold coverage and achieves better genotype concordances for almost all variant types and coverages tested. Improvements are especially pronounced for large insertions (≥50 bp) and variants in repetitive regions, enabling the inclusion of these classes of variants in genome-wide association studies. PanGenie efficiently leverages the increasing amount of haplotype-resolved assemblies to unravel the functional impact of previously inaccessible variants while being faster compared with alignment-based workflows. PanGenie is an alignment-free, k-mer-based tool that utilizes a haplotype-resolved pangenome reference to genotype a wide range of variants.},
  archive      = {J_NG},
  author       = {Ebler, Jana and Ebert, Peter and Clarke, Wayne E. and Rausch, Tobias and Audano, Peter A. and Houwaart, Torsten and Mao, Yafei and Korbel, Jan O. and Eichler, Evan E. and Zody, Michael C. and Dilthey, Alexander T. and Marschall, Tobias},
  doi          = {10.1038/s41588-022-01043-w},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {518-525},
  shortjournal = {Nat. Genet.},
  title        = {Pangenome-based genome inference allows efficient and accurate genotyping across a wide spectrum of variant classes},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Common variants contribute to intrinsic human brain
functional networks. <em>NG</em>, <em>54</em>(4), 508–517. (<a
href="https://doi.org/10.1038/s41588-022-01039-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human brain forms functional networks of correlated activity, which have been linked with both cognitive and clinical outcomes. However, the genetic variants affecting brain function are largely unknown. Here, we used resting-state functional magnetic resonance images from 47,276 individuals to discover and validate common genetic variants influencing intrinsic brain activity. We identified 45 new genetic regions associated with brain functional signatures (P &amp;lt; 2.8 × 10−11), including associations to the central executive, default mode, and salience networks involved in the triple-network model of psychopathology. A number of brain activity-associated loci colocalized with brain disorders (e.g., the APOE ε4 locus with Alzheimer’s disease). Variation in brain function was genetically correlated with brain disorders, such as major depressive disorder and schizophrenia. Together, our study provides a step forward in understanding the genetic architecture of brain functional networks and their genetic links to brain-related complex traits and disorders. Association analyses using resting-state functional magnetic resonance images identify common genetic variants influencing intrinsic brain activity. Variation in brain function is genetically correlated with several neuropsychiatric traits.},
  archive      = {J_NG},
  author       = {Zhao, Bingxin and Li, Tengfei and Smith, Stephen M. and Xiong, Di and Wang, Xifeng and Yang, Yue and Luo, Tianyou and Zhu, Ziliang and Shan, Yue and Matoba, Nana and Sun, Quan and Yang, Yuchen and Hauberg, Mads E. and Bendl, Jaroslav and Fullard, John F. and Roussos, Panagiotis and Lin, Weili and Li, Yun and Stein, Jason L. and Zhu, Hongtu},
  doi          = {10.1038/s41588-022-01039-6},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {508-517},
  shortjournal = {Nat. Genet.},
  title        = {Common variants contribute to intrinsic human brain functional networks},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Global landscape of SARS-CoV-2 genomic surveillance and data
sharing. <em>NG</em>, <em>54</em>(4), 499–507. (<a
href="https://doi.org/10.1038/s41588-022-01033-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic surveillance has shaped our understanding of severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) variants. We performed a global landscape analysis on SARS-CoV-2 genomic surveillance and genomic data using a collection of country-specific data. Here, we characterize increasing circulation of the Alpha variant in early 2021, subsequently replaced by the Delta variant around May 2021. SARS-CoV-2 genomic surveillance and sequencing availability varied markedly across countries, with 45 countries performing a high level of routine genomic surveillance and 96 countries with a high availability of SARS-CoV-2 sequencing. We also observed a marked heterogeneity of sequencing percentage, sequencing technologies, turnaround time and completeness of released metadata across regions and income groups. A total of 37% of countries with explicit reporting on variants shared less than half of their sequences of variants of concern (VOCs) in public repositories. Our findings indicate an urgent need to increase timely and full sharing of sequences, the standardization of metadata files and support for countries with limited sequencing and bioinformatics capacity. Analyses on the global diversity of SARS-CoV-2 genomic surveillance across 118 countries and the extent of public availability of genomic data provide evidence to better inform SARS-CoV-2 surveillance policy.},
  archive      = {J_NG},
  author       = {Chen, Zhiyuan and Azman, Andrew S. and Chen, Xinhua and Zou, Junyi and Tian, Yuyang and Sun, Ruijia and Xu, Xiangyanyu and Wu, Yani and Lu, Wanying and Ge, Shijia and Zhao, Zeyao and Yang, Juan and Leung, Daniel T. and Domman, Daryl B. and Yu, Hongjie},
  doi          = {10.1038/s41588-022-01033-y},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {499-507},
  shortjournal = {Nat. Genet.},
  title        = {Global landscape of SARS-CoV-2 genomic surveillance and data sharing},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Single-cell analysis of somatic mutations in human bronchial
epithelial cells in relation to aging and smoking. <em>NG</em>,
<em>54</em>(4), 492–498. (<a
href="https://doi.org/10.1038/s41588-022-01035-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although lung cancer risk among smokers is dependent on smoking dose, it remains unknown if this increased risk reflects an increased rate of somatic mutation accumulation in normal lung cells. Here, we applied single-cell whole-genome sequencing of proximal bronchial basal cells from 33 participants aged between 11 and 86 years with smoking histories varying from never-smoking to 116 pack-years. We found an increase in the frequency of single-nucleotide variants and small insertions and deletions with chronological age in never-smokers, with mutation frequencies significantly elevated among smokers. When plotted against smoking pack-years, mutations followed the linear increase in cancer risk until about 23 pack-years, after which no further increase in mutation frequency was observed, pointing toward individual selection for mutation avoidance. Known lung cancer-defined mutation signatures tracked with both age and smoking. No significant enrichment for somatic mutations in lung cancer driver genes was observed. Single-cell whole-genome sequencing of proximal bronchial basal cells shows that somatic mutations accumulate with age and at a higher level in smokers compared to never-smokers. Mutation frequencies increased with smoking dose but then plateaued, suggesting intrinsic mechanisms to limit mutation burden.},
  archive      = {J_NG},
  author       = {Huang, Zhenqiu and Sun, Shixiang and Lee, Moonsook and Maslov, Alexander Y. and Shi, Miao and Waldman, Spencer and Marsh, Ava and Siddiqui, Taha and Dong, Xiao and Peter, Yakov and Sadoughi, Ali and Shah, Chirag and Ye, Kenny and Spivack, Simon D. and Vijg, Jan},
  doi          = {10.1038/s41588-022-01035-w},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {492-498},
  shortjournal = {Nat. Genet.},
  title        = {Single-cell analysis of somatic mutations in human bronchial epithelial cells in relation to aging and smoking},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). BRD2 compartmentalizes the accessible genome. <em>NG</em>,
<em>54</em>(4), 481–491. (<a
href="https://doi.org/10.1038/s41588-022-01044-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian chromosomes are organized into megabase-sized compartments that are further subdivided into topologically associating domains (TADs). While the formation of TADs is dependent on cohesin, the mechanism behind compartmentalization remains enigmatic. Here, we show that the bromodomain and extraterminal (BET) family scaffold protein BRD2 promotes spatial mixing and compartmentalization of active chromatin after cohesin loss. This activity is independent of transcription but requires BRD2 to recognize acetylated targets through its double bromodomain and interact with binding partners with its low-complexity domain. Notably, genome compartmentalization mediated by BRD2 is antagonized on the one hand by cohesin and on the other hand by the BET homolog protein BRD4, both of which inhibit BRD2 binding to chromatin. Polymer simulation of our data supports a BRD2–cohesin interplay model of nuclear topology, in which genome compartmentalization results from a competition between loop extrusion and chromatin-state-specific affinity interactions. BRD2 facilitates mixing and compartmentalization of active chromatin upon cohesin depletion. BRD2’s function is counteracted by cohesin and BRD4.},
  archive      = {J_NG},
  author       = {Xie, Liangqi and Dong, Peng and Qi, Yifeng and Hsieh, Tsung-Han S. and English, Brian P. and Jung, SeolKyoung and Chen, Xingqi and De Marzio, Margherita and Casellas, Rafael and Chang, Howard Y. and Zhang, Bin and Tjian, Robert and Liu, Zhe},
  doi          = {10.1038/s41588-022-01044-9},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {481-491},
  shortjournal = {Nat. Genet.},
  title        = {BRD2 compartmentalizes the accessible genome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DNMT3A-dependent DNA methylation is required for
spermatogonial stem cells to commit to spermatogenesis. <em>NG</em>,
<em>54</em>(4), 469–480. (<a
href="https://doi.org/10.1038/s41588-022-01040-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA methylation plays a critical role in spermatogenesis, as evidenced by the male sterility of DNA methyltransferase (DNMT) mutant mice. Here, we report a division of labor in the establishment of the methylation landscape of male germ cells and its functions in spermatogenesis. Although DNMT3C is essential for preventing retrotransposons from interfering with meiosis, DNMT3A broadly methylates the genome (with the exception of DNMT3C-dependent retrotransposons) and controls spermatogonial stem cell (SSC) plasticity. By reconstructing developmental trajectories through single-cell RNA sequencing and profiling chromatin states, we found that Dnmt3A mutant SSCs can only self-renew and no longer differentiate in association with spurious enhancer activation that enforces an irreversible stem cell gene program. Our findings therefore highlight a key function of DNA methylation in male fertility: the epigenetic programming of SSC commitment to differentiation and lifelong spermatogenesis supply. In the male germline, DNMT3C methylates retrotransposons whereas DNMT3A globally methylates the genome and regulates spermatogonial stem cell (SSC) plasticity. Single-cell RNA-sequencing analysis shows that Dnmt3A mutant SSCs self-renew but do not differentiate.},
  archive      = {J_NG},
  author       = {Dura, Mathilde and Teissandier, Aurélie and Armand, Mélanie and Barau, Joan and Lapoujade, Clémentine and Fouchet, Pierre and Bonneville, Lorraine and Schulz, Mathieu and Weber, Michael and Baudrin, Laura G. and Lameiras, Sonia and Bourc’his, Deborah},
  doi          = {10.1038/s41588-022-01040-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {469-480},
  shortjournal = {Nat. Genet.},
  title        = {DNMT3A-dependent DNA methylation is required for spermatogonial stem cells to commit to spermatogenesis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). H3K27me3 conditions chemotolerance in triple-negative breast
cancer. <em>NG</em>, <em>54</em>(4), 459–468. (<a
href="https://doi.org/10.1038/s41588-022-01047-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The persistence of cancer cells resistant to therapy remains a major clinical challenge. In triple-negative breast cancer, resistance to chemotherapy results in the highest recurrence risk among breast cancer subtypes. The drug-tolerant state seems largely defined by nongenetic features, but the underlying mechanisms are poorly understood. Here, by monitoring epigenomes, transcriptomes and lineages with single-cell resolution, we show that the repressive histone mark H3K27me3 (trimethylation of histone H3 at lysine 27) regulates cell fate at the onset of chemotherapy. We report that a persister expression program is primed with both H3K4me3 (trimethylation of histone H3 at lysine 4) and H3K27me3 in unchallenged cells, with H3K27me3 being the lock to its transcriptional activation. We further demonstrate that depleting H3K27me3 enhances the potential of cancer cells to tolerate chemotherapy. Conversely, preventing H3K27me3 demethylation simultaneously to chemotherapy inhibits the transition to a drug-tolerant state, and delays tumor recurrence in vivo. Our results highlight how chromatin landscapes shape the potential of cancer cells to respond to initial therapy. Single-cell multiomic profiling of triple-negative breast cancer samples treated with capecitabine shows that H3K27me3 regulates a persister cell state. Blocking H3K27me3 demethylation inhibits the transition to a drug-tolerant state and delays tumor recurrence.},
  archive      = {J_NG},
  author       = {Marsolier, Justine and Prompsy, Pacôme and Durand, Adeline and Lyne, Anne-Marie and Landragin, Camille and Trouchet, Amandine and Bento, Sabrina Tenreira and Eisele, Almut and Foulon, Sophie and Baudre, Léa and Grosselin, Kevin and Bohec, Mylène and Baulande, Sylvain and Dahmani, Ahmed and Sourd, Laura and Letouzé, Eric and Salomon, Anne-Vincent and Marangoni, Elisabetta and Perié, Leïla and Vallot, Céline},
  doi          = {10.1038/s41588-022-01047-6},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {459-468},
  shortjournal = {Nat. Genet.},
  title        = {H3K27me3 conditions chemotolerance in triple-negative breast cancer},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Leveraging fine-mapping and multipopulation training data to
improve cross-population polygenic risk scores. <em>NG</em>,
<em>54</em>(4), 450–458. (<a
href="https://doi.org/10.1038/s41588-022-01036-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polygenic risk scores suffer reduced accuracy in non-European populations, exacerbating health disparities. We propose PolyPred, a method that improves cross-population polygenic risk scores by combining two predictors: a new predictor that leverages functionally informed fine-mapping to estimate causal effects (instead of tagging effects), addressing linkage disequilibrium differences, and BOLT-LMM, a published predictor. When a large training sample is available in the non-European target population, we propose PolyPred+, which further incorporates the non-European training data. We applied PolyPred to 49 diseases/traits in four UK Biobank populations using UK Biobank British training data, and observed relative improvements versus BOLT-LMM ranging from +7% in south Asians to +32% in Africans, consistent with simulations. We applied PolyPred+ to 23 diseases/traits in UK Biobank east Asians using both UK Biobank British and Biobank Japan training data, and observed improvements of +24% versus BOLT-LMM and +12% versus PolyPred. Summary statistics-based analogs of PolyPred and PolyPred+ attained similar improvements. PolyPred and PolyPred+ methods that leverage fine-mapping and non-European training data significantly improve cross-population polygenic prediction accuracy when applied to diseases and complex traits in UK Biobank populations.},
  archive      = {J_NG},
  author       = {Weissbrod, Omer and Kanai, Masahiro and Shi, Huwenbo and Gazal, Steven and Peyrot, Wouter J. and Khera, Amit V. and Okada, Yukinori and Martin, Alicia R. and Finucane, Hilary K. and Price, Alkes L.},
  doi          = {10.1038/s41588-022-01036-9},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {450-458},
  shortjournal = {Nat. Genet.},
  title        = {Leveraging fine-mapping and multipopulation training data to improve cross-population polygenic risk scores},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Polygenic prediction of educational attainment within and
between families from genome-wide association analyses in 3 million
individuals. <em>NG</em>, <em>54</em>(4), 437–449. (<a
href="https://doi.org/10.1038/s41588-022-01016-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We conduct a genome-wide association study (GWAS) of educational attainment (EA) in a sample of ~3 million individuals and identify 3,952 approximately uncorrelated genome-wide-significant single-nucleotide polymorphisms (SNPs). A genome-wide polygenic predictor, or polygenic index (PGI), explains 12–16% of EA variance and contributes to risk prediction for ten diseases. Direct effects (i.e., controlling for parental PGIs) explain roughly half the PGI’s magnitude of association with EA and other phenotypes. The correlation between mate-pair PGIs is far too large to be consistent with phenotypic assortment alone, implying additional assortment on PGI-associated factors. In an additional GWAS of dominance deviations from the additive model, we identify no genome-wide-significant SNPs, and a separate X-chromosome additive GWAS identifies 57. A genome-wide association study in ~3 million individuals identifies 3,952 independent variants associated with educational attainment. A polygenic index explains 12–16% of variance for this trait and contributes to risk prediction for ten diseases.},
  archive      = {J_NG},
  author       = {Okbay, Aysu and Wu, Yeda and Wang, Nancy and Jayashankar, Hariharan and Bennett, Michael and Nehzati, Seyed Moeen and Sidorenko, Julia and Kweon, Hyeokmoon and Goldman, Grant and Gjorgjieva, Tamara and Jiang, Yunxuan and Hicks, Barry and Tian, Chao and Hinds, David A. and Ahlskog, Rafael and Magnusson, Patrik K. E. and Oskarsson, Sven and Hayward, Caroline and Campbell, Archie and Porteous, David J. and Freese, Jeremy and Herd, Pamela and Watson, Chelsea and Jala, Jonathan and Conley, Dalton and Koellinger, Philipp D. and Johannesson, Magnus and Laibson, David and Meyer, Michelle N. and Lee, James J. and Kong, Augustine and Yengo, Loic and Cesarini, David and Turley, Patrick and Visscher, Peter M. and Beauchamp, Jonathan P. and Benjamin, Daniel J. and Young, Alexander I.},
  doi          = {10.1038/s41588-022-01016-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {437-449},
  shortjournal = {Nat. Genet.},
  title        = {Polygenic prediction of educational attainment within and between families from genome-wide association analyses in 3 million individuals},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). New insights into the genetic etiology of alzheimer’s
disease and related dementias. <em>NG</em>, <em>54</em>(4), 412–436. (<a
href="https://doi.org/10.1038/s41588-022-01024-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterization of the genetic landscape of Alzheimer’s disease (AD) and related dementias (ADD) provides a unique opportunity for a better understanding of the associated pathophysiological processes. We performed a two-stage genome-wide association study totaling 111,326 clinically diagnosed/‘proxy’ AD cases and 677,663 controls. We found 75 risk loci, of which 42 were new at the time of analysis. Pathway enrichment analyses confirmed the involvement of amyloid/tau pathways and highlighted microglia implication. Gene prioritization in the new loci identified 31 genes that were suggestive of new genetically associated processes, including the tumor necrosis factor alpha pathway through the linear ubiquitin chain assembly complex. We also built a new genetic risk score associated with the risk of future AD/dementia or progression from mild cognitive impairment to AD/dementia. The improvement in prediction led to a 1.6- to 1.9-fold increase in AD risk from the lowest to the highest decile, in addition to effects of age and the APOE ε4 allele. Meta-analysis of genome-wide association studies on Alzheimer’s disease and related dementias identifies new loci and enables generation of a new genetic risk score associated with the risk of future Alzheimer’s disease and dementia.},
  archive      = {J_NG},
  author       = {Bellenguez, Céline and Küçükali, Fahri and Jansen, Iris E. and Kleineidam, Luca and Moreno-Grau, Sonia and Amin, Najaf and Naj, Adam C. and Campos-Martin, Rafael and Grenier-Boley, Benjamin and Andrade, Victor and Holmans, Peter A. and Boland, Anne and Damotte, Vincent and van der Lee, Sven J. and Costa, Marcos R. and Kuulasmaa, Teemu and Yang, Qiong and de Rojas, Itziar and Bis, Joshua C. and Yaqub, Amber and Prokic, Ivana and Chapuis, Julien and Ahmad, Shahzad and Giedraitis, Vilmantas and Aarsland, Dag and Garcia-Gonzalez, Pablo and Abdelnour, Carla and Alarcón-Martín, Emilio and Alcolea, Daniel and Alegret, Montserrat and Alvarez, Ignacio and Álvarez, Victoria and Armstrong, Nicola J. and Tsolaki, Anthoula and Antúnez, Carmen and Appollonio, Ildebrando and Arcaro, Marina and Archetti, Silvana and Pastor, Alfonso Arias and Arosio, Beatrice and Athanasiu, Lavinia and Bailly, Henri and Banaj, Nerisa and Baquero, Miquel and Barral, Sandra and Beiser, Alexa and Pastor, Ana Belén and Below, Jennifer E. and Benchek, Penelope and Benussi, Luisa and Berr, Claudine and Besse, Céline and Bessi, Valentina and Binetti, Giuliano and Bizarro, Alessandra and Blesa, Rafael and Boada, Mercè and Boerwinkle, Eric and Borroni, Barbara and Boschi, Silvia and Bossù, Paola and Bråthen, Geir and Bressler, Jan and Bresner, Catherine and Brodaty, Henry and Brookes, Keeley J. and Brusco, Luis Ignacio and Buiza-Rueda, Dolores and Bûrger, Katharina and Burholt, Vanessa and Bush, William S. and Calero, Miguel and Cantwell, Laura B. and Chene, Geneviève and Chung, Jaeyoon and Cuccaro, Michael L. and Carracedo, Ángel and Cecchetti, Roberta and Cervera-Carles, Laura and Charbonnier, Camille and Chen, Hung-Hsin and Chillotti, Caterina and Ciccone, Simona and Claassen, Jurgen A. H. R. and Clark, Christopher and Conti, Elisa and Corma-Gómez, Anaïs and Costantini, Emanuele and Custodero, Carlo and Daian, Delphine and Dalmasso, Maria Carolina and Daniele, Antonio and Dardiotis, Efthimios and Dartigues, Jean-François and de Deyn, Peter Paul and de Paiva Lopes, Katia and de Witte, Lot D. and Debette, Stéphanie and Deckert, Jürgen and del Ser, Teodoro and Denning, Nicola and DeStefano, Anita and Dichgans, Martin and Diehl-Schmid, Janine and Diez-Fairen, Mónica and Rossi, Paolo Dionigi and Djurovic, Srdjan and Duron, Emmanuelle and Düzel, Emrah and Dufouil, Carole and Eiriksdottir, Gudny and Engelborghs, Sebastiaan and Escott-Price, Valentina and Espinosa, Ana and Ewers, Michael and Faber, Kelley M. and Fabrizio, Tagliavini and Nielsen, Sune Fallgaard and Fardo, David W. and Farotti, Lucia and Fenoglio, Chiara and Fernández-Fuertes, Marta and Ferrari, Raffaele and Ferreira, Catarina B. and Ferri, Evelyn and Fin, Bertrand and Fischer, Peter and Fladby, Tormod and Fließbach, Klaus and Fongang, Bernard and Fornage, Myriam and Fortea, Juan and Foroud, Tatiana M. and Fostinelli, Silvia and Fox, Nick C. and Franco-Macías, Emlio and Bullido, María J. and Frank-García, Ana and Froelich, Lutz and Fulton-Howard, Brian and Galimberti, Daniela and García-Alberca, Jose Maria and García-González, Pablo and Garcia-Madrona, Sebastian and Garcia-Ribas, Guillermo and Ghidoni, Roberta and Giegling, Ina and Giorgio, Giaccone and Goate, Alison M. and Goldhardt, Oliver and Gomez-Fonseca, Duber and González-Pérez, Antonio and Graff, Caroline and Grande, Giulia and Green, Emma and Grimmer, Timo and Grünblatt, Edna and Grunin, Michelle and Gudnason, Vilmundur and Guetta-Baranes, Tamar and Haapasalo, Annakaisa and Hadjigeorgiou, Georgios and Haines, Jonathan L. and Hamilton-Nelson, Kara L. and Hampel, Harald and Hanon, Olivier and Hardy, John and Hartmann, Annette M. and Hausner, Lucrezia and Harwood, Janet and Heilmann-Heimbach, Stefanie and Helisalmi, Seppo and Heneka, Michael T. and Hernández, Isabel and Herrmann, Martin J. and Hoffmann, Per and Holmes, Clive and Holstege, Henne and Vilas, Raquel Huerto and Hulsman, Marc and Humphrey, Jack and Biessels, Geert Jan and Jian, Xueqiu and Johansson, Charlotte and Jun, Gyungah R. and Kastumata, Yuriko and Kauwe, John and Kehoe, Patrick G. and Kilander, Lena and Ståhlbom, Anne Kinhult and Kivipelto, Miia and Koivisto, Anne and Kornhuber, Johannes and Kosmidis, Mary H. and Kukull, Walter A. and Kuksa, Pavel P. and Kunkle, Brian W. and Kuzma, Amanda B. and Lage, Carmen and Laukka, Erika J. and Launer, Lenore and Lauria, Alessandra and Lee, Chien-Yueh and Lehtisalo, Jenni and Lerch, Ondrej and Lleó, Alberto and Longstreth, William and Lopez, Oscar and de Munain, Adolfo Lopez and Love, Seth and Löwemark, Malin and Luckcuck, Lauren and Lunetta, Kathryn L. and Ma, Yiyi and Macías, Juan and MacLeod, Catherine A. and Maier, Wolfgang and Mangialasche, Francesca and Spallazzi, Marco and Marquié, Marta and Marshall, Rachel and Martin, Eden R. and Montes, Angel Martín and Rodríguez, Carmen Martínez and Masullo, Carlo and Mayeux, Richard and Mead, Simon and Mecocci, Patrizia and Medina, Miguel and Meggy, Alun and Mehrabian, Shima and Mendoza, Silvia and Menéndez-González, Manuel and Mir, Pablo and Moebus, Susanne and Mol, Merel and Molina-Porcel, Laura and Montrreal, Laura and Morelli, Laura and Moreno, Fermin and Morgan, Kevin and Mosley, Thomas and Nöthen, Markus M. and Muchnik, Carolina and Mukherjee, Shubhabrata and Nacmias, Benedetta and Ngandu, Tiia and Nicolas, Gael and Nordestgaard, Børge G. and Olaso, Robert and Orellana, Adelina and Orsini, Michela and Ortega, Gemma and Padovani, Alessandro and Paolo, Caffarra and Papenberg, Goran and Parnetti, Lucilla and Pasquier, Florence and Pastor, Pau and Peloso, Gina and Pérez-Cordón, Alba and Pérez-Tur, Jordi and Pericard, Pierre and Peters, Oliver and Pijnenburg, Yolande A. L. and Pineda, Juan A. and Piñol-Ripoll, Gerard and Pisanu, Claudia and Polak, Thomas and Popp, Julius and Posthuma, Danielle and Priller, Josef and Puerta, Raquel and Quenez, Olivier and Quintela, Inés and Thomassen, Jesper Qvist and Rábano, Alberto and Rainero, Innocenzo and Rajabli, Farid and Ramakers, Inez and Real, Luis M. and Reinders, Marcel J. T. and Reitz, Christiane and Reyes-Dumeyer, Dolly and Ridge, Perry and Riedel-Heller, Steffi and Riederer, Peter and Roberto, Natalia and Rodriguez-Rodriguez, Eloy and Rongve, Arvid and Allende, Irene Rosas and Rosende-Roca, Maitée and Royo, Jose Luis and Rubino, Elisa and Rujescu, Dan and Sáez, María Eugenia and Sakka, Paraskevi and Saltvedt, Ingvild and Sanabria, Ángela and Sánchez-Arjona, María Bernal and Sanchez-Garcia, Florentino and Juan, Pascual Sánchez and Sánchez-Valle, Raquel and Sando, Sigrid B. and Sarnowski, Chloé and Satizabal, Claudia L. and Scamosci, Michela and Scarmeas, Nikolaos and Scarpini, Elio and Scheltens, Philip and Scherbaum, Norbert and Scherer, Martin and Schmid, Matthias and Schneider, Anja and Schott, Jonathan M. and Selbæk, Geir and Seripa, Davide and Serrano, Manuel and Sha, Jin and Shadrin, Alexey A. and Skrobot, Olivia and Slifer, Susan and Snijders, Gijsje J. L. and Soininen, Hilkka and Solfrizzi, Vincenzo and Solomon, Alina and Song, Yeunjoo and Sorbi, Sandro and Sotolongo-Grau, Oscar and Spalletta, Gianfranco and Spottke, Annika and Squassina, Alessio and Stordal, Eystein and Tartan, Juan Pablo and Tárraga, Lluís and Tesí, Niccolo and Thalamuthu, Anbupalam and Thomas, Tegos and Tosto, Giuseppe and Traykov, Latchezar and Tremolizzo, Lucio and Tybjærg-Hansen, Anne and Uitterlinden, Andre and Ullgren, Abbe and Ulstein, Ingun and Valero, Sergi and Valladares, Otto and Broeckhoven, Christine Van and Vance, Jeffery and Vardarajan, Badri N. and van der Lugt, Aad and Dongen, Jasper Van and van Rooij, Jeroen and van Swieten, John and Vandenberghe, Rik and Verhey, Frans and Vidal, Jean-Sébastien and Vogelgsang, Jonathan and Vyhnalek, Martin and Wagner, Michael and Wallon, David and Wang, Li-San and Wang, Ruiqi and Weinhold, Leonie and Wiltfang, Jens and Windle, Gill and Woods, Bob and Yannakoulia, Mary and Zare, Habil and Zhao, Yi and Zhang, Xiaoling and Zhu, Congcong and Zulaica, Miren and Farrer, Lindsay A. and Psaty, Bruce M. and Ghanbari, Mohsen and Raj, Towfique and Sachdev, Perminder and Mather, Karen and Jessen, Frank and Ikram, M. Arfan and de Mendonça, Alexandre and Hort, Jakub and Tsolaki, Magda and Pericak-Vance, Margaret A. and Amouyel, Philippe and Williams, Julie and Frikke-Schmidt, Ruth and Clarimon, Jordi and Deleuze, Jean-François and Rossi, Giacomina and Seshadri, Sudha and Andreassen, Ole A. and Ingelsson, Martin and Hiltunen, Mikko and Sleegers, Kristel and Schellenberg, Gerard D. and van Duijn, Cornelia M. and Sims, Rebecca and van der Flier, Wiesje M. and Ruiz, Agustín and Ramirez, Alfredo and Lambert, Jean-Charles},
  doi          = {10.1038/s41588-022-01024-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {412-436},
  shortjournal = {Nat. Genet.},
  title        = {New insights into the genetic etiology of alzheimer’s disease and related dementias},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetic variants underlying differences in facial morphology
in east asian and european populations. <em>NG</em>, <em>54</em>(4),
403–411. (<a href="https://doi.org/10.1038/s41588-022-01038-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Facial morphology—a conspicuous feature of human appearance—is highly heritable. Previous studies on the genetic basis of facial morphology were performed mainly in European-ancestry cohorts (EUR). Applying a data-driven phenotyping and multivariate genome-wide scanning protocol to a large collection of three-dimensional facial images of individuals with East Asian ancestry (EAS), we identified 244 variants in 166 loci (62 new) associated with typical-range facial variation. A newly proposed polygenic shape analysis indicates that the effects of the variants on facial shape in EAS can be generalized to EUR. Based on this, we further identified 13 variants related to differences between facial shape in EUR and EAS populations. Evolutionary analyses suggest that the difference in nose shape between EUR and EAS populations is caused by a directional selection, due mainly to a local adaptation in Europeans. Our results illustrate the underlying genetic basis for facial differences across populations. A genome-wide study based on three-dimensional facial images identifies new loci associated with facial morphology in East Asian populations. Comparison with data from European populations identifies variants related to facial shape differences between these populations.},
  archive      = {J_NG},
  author       = {Zhang, Manfei and Wu, Sijie and Du, Siyuan and Qian, Wei and Chen, Jieyi and Qiao, Lu and Yang, Yajun and Tan, Jingze and Yuan, Ziyu and Peng, Qianqian and Liu, Yu and Navarro, Nicolas and Tang, Kun and Ruiz-Linares, Andrés and Wang, Jiucun and Claes, Peter and Jin, Li and Li, Jiarui and Wang, Sijia},
  doi          = {10.1038/s41588-022-01038-7},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {403-411},
  shortjournal = {Nat. Genet.},
  title        = {Genetic variants underlying differences in facial morphology in east asian and european populations},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). HLA autoimmune risk alleles restrict the hypervariable
region of t cell receptors. <em>NG</em>, <em>54</em>(4), 393–402. (<a
href="https://doi.org/10.1038/s41588-022-01032-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polymorphisms in the human leukocyte antigen (HLA) genes strongly influence autoimmune disease risk. HLA risk alleles may influence thymic selection to increase the frequency of T cell receptors (TCRs) reactive to autoantigens (central hypothesis). However, research in human autoimmunity has provided little evidence supporting the central hypothesis. Here we investigated the influence of HLA alleles on TCR composition at the highly diverse complementarity determining region 3 (CDR3), which confers antigen recognition. We observed unexpectedly strong HLA–CDR3 associations. The strongest association was found at HLA-DRB1 amino acid position 13, the position that mediates genetic risk for multiple autoimmune diseases. We identified multiple CDR3 amino acid features enriched by HLA risk alleles. Moreover, the CDR3 features promoted by the HLA risk alleles are more enriched in candidate pathogenic TCRs than control TCRs (for example, citrullinated epitope-specific TCRs in patients with rheumatoid arthritis). Together, these results provide genetic evidence supporting the central hypothesis. Genetic analyses identify associations between T cell antigen receptor composition at complementarity-determining region 3 and specific HLA alleles, including position 13 of HLA-DRB1, which mediates risk for multiple autoimmune diseases.},
  archive      = {J_NG},
  author       = {Ishigaki, Kazuyoshi and Lagattuta, Kaitlyn A. and Luo, Yang and James, Eddie A. and Buckner, Jane H. and Raychaudhuri, Soumya},
  doi          = {10.1038/s41588-022-01032-z},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {393-402},
  shortjournal = {Nat. Genet.},
  title        = {HLA autoimmune risk alleles restrict the hypervariable region of t cell receptors},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome-wide analysis provides genetic evidence that ACE2
influences COVID-19 risk and yields risk scores associated with severe
disease. <em>NG</em>, <em>54</em>(4), 382–392. (<a
href="https://doi.org/10.1038/s41588-021-01006-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) enters human host cells via angiotensin-converting enzyme 2 (ACE2) and causes coronavirus disease 2019 (COVID-19). Here, through a genome-wide association study, we identify a variant (rs190509934, minor allele frequency 0.2–2%) that downregulates ACE2 expression by 37% (P = 2.7 × 10−8) and reduces the risk of SARS-CoV-2 infection by 40% (odds ratio = 0.60, P = 4.5 × 10−13), providing human genetic evidence that ACE2 expression levels influence COVID-19 risk. We also replicate the associations of six previously reported risk variants, of which four were further associated with worse outcomes in individuals infected with the virus (in/near LZTFL1, MHC, DPP9 and IFNAR2). Lastly, we show that common variants define a risk score that is strongly associated with severe disease among cases and modestly improves the prediction of disease severity relative to demographic and clinical factors alone. Genome-wide meta-analysis of SARS-CoV-2 susceptibility and severity phenotypes in up to 756,646 samples identifies a rare protective variant proximal to ACE2. A 6-SNP genetic risk score provides additional predictive power when added to known risk factors.},
  archive      = {J_NG},
  author       = {Horowitz, Julie E. and Kosmicki, Jack A. and Damask, Amy and Sharma, Deepika and Roberts, Genevieve H. L. and Justice, Anne E. and Banerjee, Nilanjana and Coignet, Marie V. and Yadav, Ashish and Leader, Joseph B. and Marcketta, Anthony and Park, Danny S. and Lanche, Rouel and Maxwell, Evan and Knight, Spencer C. and Bai, Xiaodong and Guturu, Harendra and Sun, Dylan and Baltzell, Asher and Kury, Fabricio S. P. and Backman, Joshua D. and Girshick, Ahna R. and O’Dushlaine, Colm and McCurdy, Shannon R. and Partha, Raghavendran and Mansfield, Adam J. and Turissini, David A. and Li, Alexander H. and Zhang, Miao and Mbatchou, Joelle and Watanabe, Kyoko and Gurski, Lauren and McCarthy, Shane E. and Kang, Hyun M. and Dobbyn, Lee and Stahl, Eli and Verma, Anurag and Sirugo, Giorgio and Ritchie, Marylyn D. and Jones, Marcus and Balasubramanian, Suganthi and Siminovitch, Katherine and Salerno, William J. and Shuldiner, Alan R. and Rader, Daniel J. and Mirshahi, Tooraj and Locke, Adam E. and Marchini, Jonathan and Overton, John D. and Carey, David J. and Habegger, Lukas and Cantor, Michael N. and Rand, Kristin A. and Hong, Eurie L. and Reid, Jeffrey G. and Ball, Catherine A. and Baras, Aris and Abecasis, Gonçalo R. and Ferreira, Manuel A. R.},
  doi          = {10.1038/s41588-021-01006-7},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {382-392},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide analysis provides genetic evidence that ACE2 influences COVID-19 risk and yields risk scores associated with severe disease},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Expanded COVID-19 phenotype definitions reveal distinct
patterns of genetic association and protective effects. <em>NG</em>,
<em>54</em>(4), 374–381. (<a
href="https://doi.org/10.1038/s41588-022-01042-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multiple COVID-19 genome-wide association studies (GWASs) have identified reproducible genetic associations indicating that there is a genetic component to susceptibility and severity risk. To complement these studies, we collected deep coronavirus disease 2019 (COVID-19) phenotype data from a survey of 736,723 AncestryDNA research participants. With these data, we defined eight phenotypes related to COVID-19 outcomes: four phenotypes that align with previously studied COVID-19 definitions and four ‘expanded’ phenotypes that focus on susceptibility given exposure, mild clinical manifestations and an aggregate score of symptom severity. We performed a replication analysis of 12 previously reported COVID-19 genetic associations with all eight phenotypes in a trans-ancestry meta-analysis of AncestryDNA research participants. In this analysis, we show distinct patterns of association at the 12 loci with the eight outcomes that we assessed. We also performed a genome-wide discovery analysis of all eight phenotypes, which did not yield new genome-wide significant loci but did suggest that three of the four ‘﻿expanded’﻿ COVID-19 phenotypes have enhanced power to capture protective genetic associations relative to the previously studied phenotypes. Thus, we conclude that continued large-scale ascertainment of deep COVID-19 phenotype data would likely represent a boon for COVID-19 therapeutic target identification. GWASs based on self-reported phenotypes in 736,723 individuals show distinct associations between risk loci and eight COVID-19 outcomes, suggesting differences in genetic susceptibility to infection upon exposure and severe and symptomatic disease.},
  archive      = {J_NG},
  author       = {Roberts, Genevieve H. L. and Partha, Raghavendran and Rhead, Brooke and Knight, Spencer C. and Park, Danny S. and Coignet, Marie V. and Zhang, Miao and Berkowitz, Nathan and Turrisini, David A. and Gaddis, Michael and McCurdy, Shannon R. and Pavlovic, Milos and Ruiz, Luong and Sass, Chodon and Haug Baltzell, Asher K. and Guturu, Harendra and Girshick, Ahna R. and Ball, Catherine A. and Hong, Eurie L. and Rand, Kristin A.},
  doi          = {10.1038/s41588-022-01042-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {374-381},
  shortjournal = {Nat. Genet.},
  title        = {Expanded COVID-19 phenotype definitions reveal distinct patterns of genetic association and protective effects},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Indirect paths from genetics to education. <em>NG</em>,
<em>54</em>(4), 372–373. (<a
href="https://doi.org/10.1038/s41588-021-00999-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The largest genetic study of educational attainment (EA) so far combines gene mapping and family analyses to show that genetic associations with EA and its health benefits may be mostly indirect. As such, future genetic studies of human social and behavioral traits must include diversity in population, demographic and environmental contexts.},
  archive      = {J_NG},
  author       = {Schork, Andrew J. and Peterson, Roseann E. and Dahl, Andrew and Cai, Na and Kendler, Kenneth S.},
  doi          = {10.1038/s41588-021-00999-5},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {372-373},
  shortjournal = {Nat. Genet.},
  title        = {Indirect paths from genetics to education},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). BET proteins loop and compartmentalize the 3D genome.
<em>NG</em>, <em>54</em>(4), 370–371. (<a
href="https://doi.org/10.1038/s41588-022-01031-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chromosomes are shaped by an interplay between loop extrusion and compartmentalization. Two new studies demonstrate that bromodomain and extraterminal domain (BET) proteins contribute to both processes, with BRD4 facilitating one process and surprisingly inhibiting the other.},
  archive      = {J_NG},
  author       = {Eagen, Kyle P.},
  doi          = {10.1038/s41588-022-01031-0},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {370-371},
  shortjournal = {Nat. Genet.},
  title        = {BET proteins loop and compartmentalize the 3D genome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Understanding COVID-19 through genome-wide association
studies. <em>NG</em>, <em>54</em>(4), 368–369. (<a
href="https://doi.org/10.1038/s41588-021-00985-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Defining the most appropriate phenotypes in genome-wide association studies of COVID-19 is challenging, and two new publications demonstrate how case-control definitions critically determine outcomes and downstream clinical utility of findings.},
  archive      = {J_NG},
  author       = {Karlsen, Tom H.},
  doi          = {10.1038/s41588-021-00985-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {368-369},
  shortjournal = {Nat. Genet.},
  title        = {Understanding COVID-19 through genome-wide association studies},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome-edited crops for improved food security of
smallholder farmers. <em>NG</em>, <em>54</em>(4), 364–367. (<a
href="https://doi.org/10.1038/s41588-022-01046-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Widespread enthusiasm about potential contributions of genome-edited crops to address climate change, food security, nutrition and health, environmental sustainability and diversification of agriculture is dampened by concerns about the associated risks. Analysis of the top seven risks of genome-edited crops finds that the scientific risks are comparable to those of accepted, past and current breeding methods, but failure to address regulatory, legal and trade framework, and the granting of social license, squanders the potential benefits.},
  archive      = {J_NG},
  author       = {Pixley, Kevin V. and Falck-Zepeda, Jose B. and Paarlberg, Robert L. and Phillips, Peter W. B. and Slamet-Loedin, Inez H. and Dhugga, Kanwarpal S. and Campos, Hugo and Gutterson, Neal},
  doi          = {10.1038/s41588-022-01046-7},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {364-367},
  shortjournal = {Nat. Genet.},
  title        = {Genome-edited crops for improved food security of smallholder farmers},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Exploring the epigenetics of resilience. <em>NG</em>,
<em>54</em>(4), 363. (<a
href="https://doi.org/10.1038/s41588-022-01050-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To do good science, we need to include diverse perspectives, work across disciplines and think outside the box while reminding ourselves that our goal as scientists is to serve humanity. I am sharing my story to encourage others to trust their gut feelings and to have the courage to see what everyone sees, but think what no one has thought.},
  archive      = {J_NG},
  author       = {Dajani, Rana},
  doi          = {10.1038/s41588-022-01050-x},
  journal      = {Nature Genetics},
  month        = {4},
  number       = {4},
  pages        = {363},
  shortjournal = {Nat. Genet.},
  title        = {Exploring the epigenetics of resilience},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Common and rare variant association
analyses in amyotrophic lateral sclerosis identify 15 risk loci with
distinct genetic architectures and neuron-specific biology. <em>NG</em>,
<em>54</em>(3), 361. (<a
href="https://doi.org/10.1038/s41588-022-01020-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {van Rheenen, Wouter and van der Spek, Rick A. A. and Bakker, Mark K. and van Vugt, Joke J. F. A. and Hop, Paul J. and Zwamborn, Ramona A. J. and de Klein, Niek and Westra, Harm-Jan and Bakker, Olivier B. and Deelen, Patrick and Shireby, Gemma and Hannon, Eilis and Moisse, Matthieu and Baird, Denis and Restuadi, Restuadi and Dolzhenko, Egor and Dekker, Annelot M. and Gawor, Klara and Westeneng, Henk-Jan and Tazelaar, Gijs H. P. and van Eijk, Kristel R. and Kooyman, Maarten and Byrne, Ross P. and Doherty, Mark and Heverin, Mark and Al Khleifat, Ahmad and Iacoangeli, Alfredo and Shatunov, Aleksey and Ticozzi, Nicola and Cooper-Knock, Johnathan and Smith, Bradley N. and Gromicho, Marta and Chandran, Siddharthan and Pal, Suvankar and Morrison, Karen E. and Shaw, Pamela J. and Hardy, John and Orrell, Richard W. and Sendtner, Michael and Meyer, Thomas and Başak, Nazli and van der Kooi, Anneke J. and Ratti, Antonia and Fogh, Isabella and Gellera, Cinzia and Lauria, Giuseppe and Corti, Stefania and Cereda, Cristina and Sproviero, Daisy and D’Alfonso, Sandra and Sorarù, Gianni and Siciliano, Gabriele and Filosto, Massimiliano and Padovani, Alessandro and Chiò, Adriano and Calvo, Andrea and Moglia, Cristina and Brunetti, Maura and Canosa, Antonio and Grassano, Maurizio and Beghi, Ettore and Pupillo, Elisabetta and Logroscino, Giancarlo and Nefussy, Beatrice and Osmanovic, Alma and Nordin, Angelica and Lerner, Yossef and Zabari, Michal and Gotkine, Marc and Baloh, Robert H. and Bell, Shaughn and Vourc’h, Patrick and Corcia, Philippe and Couratier, Philippe and Millecamps, Stéphanie and Meininger, Vincent and Salachas, François and Mora Pardina, Jesus S. and Assialioui, Abdelilah and Rojas-García, Ricardo and Dion, Patrick A. and Ross, Jay P. and Ludolph, Albert C. and Weishaupt, Jochen H. and Brenner, David and Freischmidt, Axel and Bensimon, Gilbert and Brice, Alexis and Durr, Alexandra and Payan, Christine A. M. and Saker-Delye, Safa and Wood, Nicholas W. and Topp, Simon and Rademakers, Rosa and Tittmann, Lukas and Lieb, Wolfgang and Franke, Andre and Ripke, Stephan and Braun, Alice and Kraft, Julia and Whiteman, David C. and Olsen, Catherine M. and Uitterlinden, Andre G. and Hofman, Albert and Rietschel, Marcella and Cichon, Sven and Nöthen, Markus M. and Amouyel, Philippe and Traynor, Bryan J. and Singleton, Andrew B. and Mitne Neto, Miguel and Cauchi, Ruben J. and Ophoff, Roel A. and Wiedau-Pazos, Martina and Lomen-Hoerth, Catherine and van Deerlin, Vivianna M. and Grosskreutz, Julian and Roediger, Annekathrin and Gaur, Nayana and Jörk, Alexander and Barthel, Tabea and Theele, Erik and Ilse, Benjamin and Stubendorff, Beatrice and Witte, Otto W. and Steinbach, Robert and Hübner, Christian A. and Graff, Caroline and Brylev, Lev and Fominykh, Vera and Demeshonok, Vera and Ataulina, Anastasia and Rogelj, Boris and Koritnik, Blaž and Zidar, Janez and Ravnik-Glavač, Metka and Glavač, Damjan and Stević, Zorica and Drory, Vivian and Povedano, Monica and Blair, Ian P. and Kiernan, Matthew C. and Benyamin, Beben and Henderson, Robert D. and Furlong, Sarah and Mathers, Susan and McCombe, Pamela A. and Needham, Merrilee and Ngo, Shyuan T. and Nicholson, Garth A. and Pamphlett, Roger and Rowe, Dominic B. and Steyn, Frederik J. and Williams, Kelly L. and Mather, Karen A. and Sachdev, Perminder S. and Henders, Anjali K. and Wallace, Leanne and de Carvalho, Mamede and Pinto, Susana and Petri, Susanne and Weber, Markus and Rouleau, Guy A. and Silani, Vincenzo and Curtis, Charles J. and Breen, Gerome and Glass, Jonathan D. and Brown, Robert H. and Landers, John E. and Shaw, Christopher E. and Andersen, Peter M. and Groen, Ewout J. N. and van Es, Michael A. and Pasterkamp, R. Jeroen and Fan, Dongsheng and Garton, Fleur C. and McRae, Allan F. and Davey Smith, George and Gaunt, Tom R. and Eberle, Michael A. and Mill, Jonathan and McLaughlin, Russell L. and Hardiman, Orla and Kenna, Kevin P. and Wray, Naomi R. and Tsai, Ellen and Runz, Heiko and Franke, Lude and Al-Chalabi, Ammar and Van Damme, Philip and van den Berg, Leonard H. and Veldink, Jan H.},
  doi          = {10.1038/s41588-022-01020-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {361},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Common and rare variant association analyses in amyotrophic lateral sclerosis identify 15 risk loci with distinct genetic architectures and neuron-specific biology},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: EMPReSS: Standardized phenotype screens
for functional annotation of the mouse genome. <em>NG</em>,
<em>54</em>(3), 358–360. (<a
href="https://doi.org/10.1038/s41588-022-01019-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01019-w},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {358-360},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: EMPReSS: standardized phenotype screens for functional annotation of the mouse genome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). GestaltMatcher facilitates rare disease matching using
facial phenotype descriptors. <em>NG</em>, <em>54</em>(3), 349–357. (<a
href="https://doi.org/10.1038/s41588-021-01010-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many monogenic disorders cause a characteristic facial morphology. Artificial intelligence can support physicians in recognizing these patterns by associating facial phenotypes with the underlying syndrome through training on thousands of patient photographs. However, this ‘supervised’ approach means that diagnoses are only possible if the disorder was part of the training set. To improve recognition of ultra-rare disorders, we developed GestaltMatcher, an encoder for portraits that is based on a deep convolutional neural network. Photographs of 17,560 patients with 1,115 rare disorders were used to define a Clinical Face Phenotype Space, in which distances between cases define syndromic similarity. Here we show that patients can be matched to others with the same molecular diagnosis even when the disorder was not included in the training set. Together with mutation data, GestaltMatcher could not only accelerate the clinical diagnosis of patients with ultra-rare disorders and facial dysmorphism but also enable the delineation of new phenotypes. GestaltMatcher uses a deep convolutional neural network to improve recognition of rare disorders based on facial morphology. The framework detects similarities among patients with previously unseen syndromes, aiding discovery of new disease genes.},
  archive      = {J_NG},
  author       = {Hsieh, Tzung-Chien and Bar-Haim, Aviram and Moosa, Shahida and Ehmke, Nadja and Gripp, Karen W. and Pantel, Jean Tori and Danyel, Magdalena and Mensah, Martin Atta and Horn, Denise and Rosnev, Stanislav and Fleischer, Nicole and Bonini, Guilherme and Hustinx, Alexander and Schmid, Alexander and Knaus, Alexej and Javanmardi, Behnam and Klinkhammer, Hannah and Lesmann, Hellen and Sivalingam, Sugirthan and Kamphans, Tom and Meiswinkel, Wolfgang and Ebstein, Frédéric and Krüger, Elke and Küry, Sébastien and Bézieau, Stéphane and Schmidt, Axel and Peters, Sophia and Engels, Hartmut and Mangold, Elisabeth and Kreiß, Martina and Cremer, Kirsten and Perne, Claudia and Betz, Regina C. and Bender, Tim and Grundmann-Hauser, Kathrin and Haack, Tobias B. and Wagner, Matias and Brunet, Theresa and Bentzen, Heidi Beate and Averdunk, Luisa and Coetzer, Kimberly Christine and Lyon, Gholson J. and Spielmann, Malte and Schaaf, Christian P. and Mundlos, Stefan and Nöthen, Markus M. and Krawitz, Peter M.},
  doi          = {10.1038/s41588-021-01010-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {349-357},
  shortjournal = {Nat. Genet.},
  title        = {GestaltMatcher facilitates rare disease matching using facial phenotype descriptors},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chromosome-scale and haplotype-resolved genome assembly of a
tetraploid potato cultivar. <em>NG</em>, <em>54</em>(3), 342–348. (<a
href="https://doi.org/10.1038/s41588-022-01015-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Potato is the most widely produced tuber crop worldwide. However, reconstructing the four haplotypes of its autotetraploid genome remained an unsolved challenge. Here, we report the 3.1 Gb haplotype-resolved (at 99.6% precision), chromosome-scale assembly of the potato cultivar ‘Otava’ based on high-quality long reads, single-cell sequencing of 717 pollen genomes and Hi-C data. Unexpectedly, ~50% of the genome was identical-by-descent due to recent inbreeding, which was contrasted by highly abundant structural rearrangements involving ~20% of the genome. Among 38,214 genes, only 54% were present in all four haplotypes with an average of 3.2 copies per gene. Taking the leaf transcriptome as an example, 11% of the genes were differently expressed in at least one haplotype, where 25% of them were likely regulated through allele-specific DNA methylation. Our work sheds light on the recent breeding history of potato, the functional organization of its tetraploid genome and has the potential to strengthen the future of genomics-assisted breeding. Haplotype-resolved genome assembly of the tetraploid potato cultivar ‘Otava’ sheds light on functional organization of the tetraploid genome and provides the potential for genomics-assisted breeding.},
  archive      = {J_NG},
  author       = {Sun, Hequan and Jiao, Wen-Biao and Krause, Kristin and Campoy, José A. and Goel, Manish and Folz-Donahue, Kat and Kukat, Christian and Huettel, Bruno and Schneeberger, Korbinian},
  doi          = {10.1038/s41588-022-01015-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {342-348},
  shortjournal = {Nat. Genet.},
  title        = {Chromosome-scale and haplotype-resolved genome assembly of a tetraploid potato cultivar},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Systematic reconstruction of cellular trajectories across
mouse embryogenesis. <em>NG</em>, <em>54</em>(3), 328–341. (<a
href="https://doi.org/10.1038/s41588-022-01018-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mammalian embryogenesis is characterized by rapid cellular proliferation and diversification. Within a few weeks, a single-cell zygote gives rise to millions of cells expressing a panoply of molecular programs. Although intensively studied, a comprehensive delineation of the major cellular trajectories that comprise mammalian development in vivo remains elusive. Here, we set out to integrate several single-cell RNA-sequencing (scRNA-seq) datasets that collectively span mouse gastrulation and organogenesis, supplemented with new profiling of ~150,000 nuclei from approximately embryonic day 8.5 (E8.5) embryos staged in one-somite increments. Overall, we define cell states at each of 19 successive stages spanning E3.5 to E13.5 and heuristically connect them to their pseudoancestors and pseudodescendants. Although constructed through automated procedures, the resulting directed acyclic graph (TOME (trajectories of mammalian embryogenesis)) is largely consistent with our contemporary understanding of mammalian development. We leverage TOME to systematically nominate transcription factors (TFs) as candidate regulators of each cell type’s specification, as well as ‘cell-type homologs’ across vertebrate evolution. Integrative analysis of single-cell RNA-sequencing datasets across mouse gastrulation and organogenesis identifies cell states and trajectories at successive developmental stages, along with transcription factors that could potentially mediate lineage choices.},
  archive      = {J_NG},
  author       = {Qiu, Chengxiang and Cao, Junyue and Martin, Beth K. and Li, Tony and Welsh, Ian C. and Srivatsan, Sanjay and Huang, Xingfan and Calderon, Diego and Noble, William Stafford and Disteche, Christine M. and Murray, Stephen A. and Spielmann, Malte and Moens, Cecilia B. and Trapnell, Cole and Shendure, Jay},
  doi          = {10.1038/s41588-022-01018-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {328-341},
  shortjournal = {Nat. Genet.},
  title        = {Systematic reconstruction of cellular trajectories across mouse embryogenesis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). DNA replication fork speed underlies cell fate changes and
promotes reprogramming. <em>NG</em>, <em>54</em>(3), 318–327. (<a
href="https://doi.org/10.1038/s41588-022-01023-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Totipotency emerges in early embryogenesis, but its molecular underpinnings remain poorly characterized. In the present study, we employed DNA fiber analysis to investigate how pluripotent stem cells are reprogrammed into totipotent-like 2-cell-like cells (2CLCs). We show that totipotent cells of the early mouse embryo have slow DNA replication fork speed and that 2CLCs recapitulate this feature, suggesting that fork speed underlies the transition to a totipotent-like state. 2CLCs emerge concomitant with DNA replication and display changes in replication timing (RT), particularly during the early S-phase. RT changes occur prior to 2CLC emergence, suggesting that RT may predispose to gene expression changes and consequent reprogramming of cell fate. Slowing down replication fork speed experimentally induces 2CLCs. In vivo, slowing fork speed improves the reprogramming efficiency of somatic cell nuclear transfer. Our data suggest that fork speed regulates cellular plasticity and that remodeling of replication features leads to changes in cell fate and reprogramming. Totipotent cells in mouse embryos and 2-cell-like cells have slow DNA replication fork speed. Perturbations that slow replication fork speed promote 2-cell-like cell emergence and improve somatic cell nuclear transfer reprogramming and formation of induced pluripotent stem cell colonies.},
  archive      = {J_NG},
  author       = {Nakatani, Tsunetoshi and Lin, Jiangwei and Ji, Fei and Ettinger, Andreas and Pontabry, Julien and Tokoro, Mikiko and Altamirano-Pacheco, Luis and Fiorentino, Jonathan and Mahammadov, Elmir and Hatano, Yu and Van Rechem, Capucine and Chakraborty, Damayanti and Ruiz-Morales, Elias R. and Arguello Pascualli, Paola Y. and Scialdone, Antonio and Yamagata, Kazuo and Whetstine, Johnathan R. and Sadreyev, Ruslan I. and Torres-Padilla, Maria-Elena},
  doi          = {10.1038/s41588-022-01023-0},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {318-327},
  shortjournal = {Nat. Genet.},
  title        = {DNA replication fork speed underlies cell fate changes and promotes reprogramming},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Transcriptional kinetics and molecular functions of long
noncoding RNAs. <em>NG</em>, <em>54</em>(3), 306–317. (<a
href="https://doi.org/10.1038/s41588-022-01014-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An increasing number of long noncoding RNAs (lncRNAs) have experimentally confirmed functions, yet little is known about their transcriptional dynamics and it is challenging to determine their regulatory effects. Here, we used allele-sensitive single-cell RNA sequencing to demonstrate that, compared to messenger RNAs, lncRNAs have twice as long duration between two transcriptional bursts. Additionally, we observed increased cell-to-cell variability in lncRNA expression due to lower frequency bursting producing larger numbers of RNA molecules. Exploiting heterogeneity in asynchronously growing cells, we identified and experimentally validated lncRNAs with cell state-specific functions involved in cell cycle progression and apoptosis. Finally, we identified cis-functioning lncRNAs and showed that knockdown of these lncRNAs modulated the nearby protein-coding gene’s transcriptional burst frequency or size. In summary, we identified distinct transcriptional regulation of lncRNAs and demonstrated a role for lncRNAs in the regulation of mRNA transcriptional bursting. Allele-sensitive single-cell RNA sequencing analysis of long noncoding RNA (lncRNA) transcriptional kinetics shows that their lower expression compared to mRNA is due to lower burst frequencies and highlights cell-state-specific functions for several lncRNAs.},
  archive      = {J_NG},
  author       = {Johnsson, Per and Ziegenhain, Christoph and Hartmanis, Leonard and Hendriks, Gert-Jan and Hagemann-Jensen, Michael and Reinius, Björn and Sandberg, Rickard},
  doi          = {10.1038/s41588-022-01014-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {306-317},
  shortjournal = {Nat. Genet.},
  title        = {Transcriptional kinetics and molecular functions of long noncoding RNAs},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Prediction of histone post-translational modification
patterns based on nascent transcription data. <em>NG</em>,
<em>54</em>(3), 295–305. (<a
href="https://doi.org/10.1038/s41588-022-01026-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The role of histone modifications in transcription remains incompletely understood. Here, we examine the relationship between histone modifications and transcription using experimental perturbations combined with sensitive machine-learning tools. Transcription predicted the variation in active histone marks and complex chromatin states, like bivalent promoters, down to single-nucleosome resolution and at an accuracy that rivaled the correspondence between independent ChIP-seq experiments. Blocking transcription rapidly removed two punctate marks, H3K4me3 and H3K27ac, from chromatin indicating that transcription is required for active histone modifications. Transcription was also required for maintenance of H3K27me3, consistent with a role for RNA in recruiting PRC2. A subset of DNase-I-hypersensitive sites were refractory to prediction, precluding models where transcription initiates pervasively at any open chromatin. Our results, in combination with past literature, support a model in which active histone modifications serve a supportive, rather than an essential regulatory, role in transcription. A machine-learning tool can predict the distribution of histone post-translational modifications using nascent transcription data. Inhibiting transcription impacts H3K4me3, H3K27ac and H3K27me3 dynamics.},
  archive      = {J_NG},
  author       = {Wang, Zhong and Chivu, Alexandra G. and Choate, Lauren A. and Rice, Edward J. and Miller, Donald C. and Chu, Tinyi and Chou, Shao-Pei and Kingsley, Nicole B. and Petersen, Jessica L. and Finno, Carrie J. and Bellone, Rebecca R. and Antczak, Douglas F. and Lis, John T. and Danko, Charles G.},
  doi          = {10.1038/s41588-022-01026-x},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {295-305},
  shortjournal = {Nat. Genet.},
  title        = {Prediction of histone post-translational modification patterns based on nascent transcription data},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sequence determinants of human gene regulatory elements.
<em>NG</em>, <em>54</em>(3), 283–294. (<a
href="https://doi.org/10.1038/s41588-021-01009-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA can determine where and when genes are expressed, but the full set of sequence determinants that control gene expression is unknown. Here, we measured the transcriptional activity of DNA sequences that represent an ~100 times larger sequence space than the human genome using massively parallel reporter assays (MPRAs). Machine learning models revealed that transcription factors (TFs) generally act in an additive manner with weak grammar and that most enhancers increase expression from a promoter by a mechanism that does not appear to involve specific TF–TF interactions. The enhancers themselves can be classified into three types: classical, closed chromatin and chromatin dependent. We also show that few TFs are strongly active in a cell, with most activities being similar between cell types. Individual TFs can have multiple gene regulatory activities, including chromatin opening and enhancing, promoting and determining transcription start site (TSS) activity, consistent with the view that the TF binding motif is the key atomic unit of gene expression. Analysis of massively parallel reporter assays measuring the transcriptional activity of DNA sequences indicates that most transcription factor (TF) activity is additive and does not rely on specific TF–TF interactions. Individual TFs can have different gene regulatory activities.},
  archive      = {J_NG},
  author       = {Sahu, Biswajyoti and Hartonen, Tuomo and Pihlajamaa, Päivi and Wei, Bei and Dave, Kashyap and Zhu, Fangjie and Kaasinen, Eevi and Lidschreiber, Katja and Lidschreiber, Michael and Daub, Carsten O. and Cramer, Patrick and Kivioja, Teemu and Taipale, Jussi},
  doi          = {10.1038/s41588-021-01009-4},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {283-294},
  shortjournal = {Nat. Genet.},
  title        = {Sequence determinants of human gene regulatory elements},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An integrated framework for local genetic correlation
analysis. <em>NG</em>, <em>54</em>(3), 274–282. (<a
href="https://doi.org/10.1038/s41588-022-01017-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic correlation (rg) analysis is used to identify phenotypes that may have a shared genetic basis. Traditionally, rg is studied globally, considering only the average of the shared signal across the genome, although this approach may fail when the rg is confined to particular genomic regions or in opposing directions at different loci. Current tools for local rg analysis are restricted to analysis of two phenotypes. Here we introduce LAVA, an integrated framework for local rg analysis that, in addition to testing the standard bivariate local rgs between two phenotypes, can evaluate local heritabilities and analyze conditional genetic relations between several phenotypes using partial correlation and multiple regression. Applied to 25 behavioral and health phenotypes, we show considerable heterogeneity in the bivariate local rgs across the genome, which is often masked by the global rg patterns, and demonstrate how our conditional approaches can elucidate more complex, multivariate genetic relations. LAVA estimates multivariate local genetic relations, which enables conditional genetic analyses. Application to behavioral and health traits identifies local genetic heterogeneity and provides insights into genetic mediation and confounding.},
  archive      = {J_NG},
  author       = {Werme, Josefin and van der Sluis, Sophie and Posthuma, Danielle and de Leeuw, Christiaan A.},
  doi          = {10.1038/s41588-022-01017-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {274-282},
  shortjournal = {Nat. Genet.},
  title        = {An integrated framework for local genetic correlation analysis},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Assessing the contribution of rare variants to complex trait
heritability from whole-genome sequence data. <em>NG</em>,
<em>54</em>(3), 263–273. (<a
href="https://doi.org/10.1038/s41588-021-00997-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analyses of data from genome-wide association studies on unrelated individuals have shown that, for human traits and diseases, approximately one-third to two-thirds of heritability is captured by common SNPs. However, it is not known whether the remaining heritability is due to the imperfect tagging of causal variants by common SNPs, in particular whether the causal variants are rare, or whether it is overestimated due to bias in inference from pedigree data. Here we estimated heritability for height and body mass index (BMI) from whole-genome sequence data on 25,465 unrelated individuals of European ancestry. The estimated heritability was 0.68 (standard error 0.10) for height and 0.30 (standard error 0.10) for body mass index. Low minor allele frequency variants in low linkage disequilibrium (LD) with neighboring variants were enriched for heritability, to a greater extent for protein-altering variants, consistent with negative selection. Our results imply that rare variants, in particular those in regions of low linkage disequilibrium, are a major source of the still missing heritability of complex traits and disease. Analysis of whole-genome sequences of 25,465 individuals of European ancestry shows that rare variants contribute substantially to the heritability of height and body mass index.},
  archive      = {J_NG},
  author       = {Wainschtein, Pierrick and Jain, Deepti and Zheng, Zhili and Cupples, L. Adrienne and Shadyab, Aladdin H. and McKnight, Barbara and Shoemaker, Benjamin M. and Mitchell, Braxton D. and Psaty, Bruce M. and Kooperberg, Charles and Liu, Ching-Ti and Albert, Christine M. and Roden, Dan and Chasman, Daniel I. and Darbar, Dawood and Lloyd-Jones, Donald M. and Arnett, Donna K. and Regan, Elizabeth A. and Boerwinkle, Eric and Rotter, Jerome I. and O’Connell, Jeffrey R. and Yanek, Lisa R. and de Andrade, Mariza and Allison, Matthew A. and McDonald, Merry-Lynn N. and Chung, Mina K. and Fornage, Myriam and Chami, Nathalie and Smith, Nicholas L. and Ellinor, Patrick T. and Vasan, Ramachandran S. and Mathias, Rasika A. and Loos, Ruth J. F. and Rich, Stephen S. and Lubitz, Steven A. and Heckbert, Susan R. and Redline, Susan and Guo, Xiuqing and Chen, Y. -D Ida and Laurie, Cecelia A. and Hernandez, Ryan D. and McGarvey, Stephen T. and Goddard, Michael E. and Laurie, Cathy C. and North, Kari E. and Lange, Leslie A. and Weir, Bruce S. and Yengo, Loic and Yang, Jian and Visscher, Peter M.},
  doi          = {10.1038/s41588-021-00997-7},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {263-273},
  shortjournal = {Nat. Genet.},
  title        = {Assessing the contribution of rare variants to complex trait heritability from whole-genome sequence data},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetic associations at regulatory phenotypes improve
fine-mapping of causal variants for 12 immune-mediated diseases.
<em>NG</em>, <em>54</em>(3), 251–262. (<a
href="https://doi.org/10.1038/s41588-022-01025-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The resolution of causal genetic variants informs understanding of disease biology. We used regulatory quantitative trait loci (QTLs) from the BLUEPRINT, GTEx and eQTLGen projects to fine-map putative causal variants for 12 immune-mediated diseases. We identify 340 unique loci that colocalize with high posterior probability (≥98%) with regulatory QTLs and apply Bayesian frameworks to fine-map associations at each locus. We show that fine-mapping credible sets derived from regulatory QTLs are smaller compared to disease summary statistics. Further, they are enriched for more functionally interpretable candidate causal variants and for putatively causal insertion/deletion (INDEL) polymorphisms. Finally, we use massively parallel reporter assays to evaluate candidate causal variants at the ITGA4 locus associated with inflammatory bowel disease. Overall, our findings suggest that fine-mapping applied to disease-colocalizing regulatory QTLs can enhance the discovery of putative causal disease variants and enhance insights into the underlying causal genes and molecular mechanisms. Colocalization of regulatory quantitative trait loci from the BLUEPRINT project with genetic association data aids fine-mapping of putative causal variants for 12 immune-mediated diseases.},
  archive      = {J_NG},
  author       = {Kundu, Kousik and Tardaguila, Manuel and Mann, Alice L. and Watt, Stephen and Ponstingl, Hannes and Vasquez, Louella and Von Schiller, Dominique and Morrell, Nicholas W. and Stegle, Oliver and Pastinen, Tomi and Sawcer, Stephen J. and Anderson, Carl A. and Walter, Klaudia and Soranzo, Nicole},
  doi          = {10.1038/s41588-022-01025-y},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {251-262},
  shortjournal = {Nat. Genet.},
  title        = {Genetic associations at regulatory phenotypes improve fine-mapping of causal variants for 12 immune-mediated diseases},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Analysis of rare genetic variation underlying
cardiometabolic diseases and traits among 200,000 individuals in the UK
biobank. <em>NG</em>, <em>54</em>(3), 240–250. (<a
href="https://doi.org/10.1038/s41588-021-01011-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cardiometabolic diseases are the leading cause of death worldwide. Despite a known genetic component, our understanding of these diseases remains incomplete. Here, we analyzed the contribution of rare variants to 57 diseases and 26 cardiometabolic traits, using data from 200,337 UK Biobank participants with whole-exome sequencing. We identified 57 gene-based associations, with broad replication of novel signals in Geisinger MyCode. There was a striking risk associated with mutations in known Mendelian disease genes, including MYBPC3, LDLR, GCK, PKD1 and TTN. Many genes showed independent convergence of rare and common variant evidence, including an association between GIGYF1 and type 2 diabetes. We identified several large effect associations for height and 18 unique genes associated with blood lipid or glucose levels. Finally, we found that between 1.0% and 2.4% of participants carried rare potentially pathogenic variants for cardiometabolic disorders. These findings may facilitate studies aimed at therapeutics and screening of these common disorders. Analysis of whole-exome sequencing data from over 200,000 individuals in the UK Biobank provides new insights into the contribution of rare variants to cardiometabolic diseases and traits.},
  archive      = {J_NG},
  author       = {Jurgens, Sean J. and Choi, Seung Hoan and Morrill, Valerie N. and Chaffin, Mark and Pirruccello, James P. and Halford, Jennifer L. and Weng, Lu-Chen and Nauffal, Victor and Roselli, Carolina and Hall, Amelia W. and Oetjens, Matthew T. and Lagerman, Braxton and vanMaanen, David P. and Aragam, Krishna G. and Lunetta, Kathryn L. and Haggerty, Christopher M. and Lubitz, Steven A. and Ellinor, Patrick T.},
  doi          = {10.1038/s41588-021-01011-w},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {240-250},
  shortjournal = {Nat. Genet.},
  title        = {Analysis of rare genetic variation underlying cardiometabolic diseases and traits among 200,000 individuals in the UK biobank},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Genome-wide association analyses identify new brugada
syndrome risk loci and highlight a new mechanism of sodium channel
regulation in disease susceptibility. <em>NG</em>, <em>54</em>(3),
232–239. (<a href="https://doi.org/10.1038/s41588-021-01007-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brugada syndrome (BrS) is a cardiac arrhythmia disorder associated with sudden death in young adults. With the exception of SCN5A, encoding the cardiac sodium channel NaV1.5, susceptibility genes remain largely unknown. Here we performed a genome-wide association meta-analysis comprising 2,820 unrelated cases with BrS and 10,001 controls, and identified 21 association signals at 12 loci (10 new). Single nucleotide polymorphism (SNP)-heritability estimates indicate a strong polygenic influence. Polygenic risk score analyses based on the 21 susceptibility variants demonstrate varying cumulative contribution of common risk alleles among different patient subgroups, as well as genetic associations with cardiac electrical traits and disorders in the general population. The predominance of cardiac transcription factor loci indicates that transcriptional regulation is a key feature of BrS pathogenesis. Furthermore, functional studies conducted on MAPRE2, encoding the microtubule plus-end binding protein EB2, point to microtubule-related trafficking effects on NaV1.5 expression as a new underlying molecular mechanism. Taken together, these findings broaden our understanding of the genetic architecture of BrS and provide new insights into its molecular underpinnings. Genome-wide association analyses identify new susceptibility loci for Brugada syndrome. Functional studies implicate microtubule-related trafficking effects on sodium channel expression as an underlying molecular mechanism.},
  archive      = {J_NG},
  author       = {Barc, Julien and Tadros, Rafik and Glinge, Charlotte and Chiang, David Y. and Jouni, Mariam and Simonet, Floriane and Jurgens, Sean J. and Baudic, Manon and Nicastro, Michele and Potet, Franck and Offerhaus, Joost A. and Walsh, Roddy and Choi, Seung Hoan and Verkerk, Arie O. and Mizusawa, Yuka and Anys, Soraya and Minois, Damien and Arnaud, Marine and Duchateau, Josselin and Wijeyeratne, Yanushi D. and Muir, Alison and Papadakis, Michael and Castelletti, Silvia and Torchio, Margherita and Ortuño, Cristina Gil and Lacunza, Javier and Giachino, Daniela F. and Cerrato, Natascia and Martins, Raphaël P. and Campuzano, Oscar and Van Dooren, Sonia and Thollet, Aurélie and Kyndt, Florence and Mazzanti, Andrea and Clémenty, Nicolas and Bisson, Arnaud and Corveleyn, Anniek and Stallmeyer, Birgit and Dittmann, Sven and Saenen, Johan and Noël, Antoine and Honarbakhsh, Shohreh and Rudic, Boris and Marzak, Halim and Rowe, Matthew K. and Federspiel, Claire and Le Page, Sophie and Placide, Leslie and Milhem, Antoine and Barajas-Martinez, Hector and Beckmann, Britt-Maria and Krapels, Ingrid P. and Steinfurt, Johannes and Winkel, Bo Gregers and Jabbari, Reza and Shoemaker, Moore B. and Boukens, Bas J. and Škorić-Milosavljević, Doris and Bikker, Hennie and Manevy, Federico and Lichtner, Peter and Ribasés, Marta and Meitinger, Thomas and Müller-Nurasyid, Martina and Veldink, Jan H. and van den Berg, Leonard H. and Van Damme, Philip and Cusi, Daniele and Lanzani, Chiara and Rigade, Sidwell and Charpentier, Eric and Baron, Estelle and Bonnaud, Stéphanie and Lecointe, Simon and Donnart, Audrey and Le Marec, Hervé and Chatel, Stéphanie and Karakachoff, Matilde and Bézieau, Stéphane and London, Barry and Tfelt-Hansen, Jacob and Roden, Dan and Odening, Katja E. and Cerrone, Marina and Chinitz, Larry A. and Volders, Paul G. and van de Berg, Maarten P. and Laurent, Gabriel and Faivre, Laurence and Antzelevitch, Charles and Kääb, Stefan and Arnaout, Alain Al and Dupuis, Jean-Marc and Pasquie, Jean-Luc and Billon, Olivier and Roberts, Jason D. and Jesel, Laurence and Borggrefe, Martin and Lambiase, Pier D. and Mansourati, Jacques and Loeys, Bart and Leenhardt, Antoine and Guicheney, Pascale and Maury, Philippe and Schulze-Bahr, Eric and Robyns, Tomas and Breckpot, Jeroen and Babuty, Dominique and Priori, Silvia G. and Napolitano, Carlo and de Asmundis, Carlo and Brugada, Pedro and Brugada, Ramon and Arbelo, Elena and Brugada, Josep and Mabo, Philippe and Behar, Nathalie and Giustetto, Carla and Molina, Maria Sabater and Gimeno, Juan R. and Hasdemir, Can and Schwartz, Peter J. and Crotti, Lia and McKeown, Pascal P. and Sharma, Sanjay and Behr, Elijah R. and Haissaguerre, Michel and Sacher, Frédéric and Rooryck, Caroline and Tan, Hanno L. and Remme, Carol A. and Postema, Pieter G. and Delmar, Mario and Ellinor, Patrick T. and Lubitz, Steven A. and Gourraud, Jean-Baptiste and Tanck, Michael W. and George, Alfred L. and MacRae, Calum A. and Burridge, Paul W. and Dina, Christian and Probst, Vincent and Wilde, Arthur A. and Schott, Jean-Jacques and Redon, Richard and Bezzina, Connie R.},
  doi          = {10.1038/s41588-021-01007-6},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {232-239},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide association analyses identify new brugada syndrome risk loci and highlight a new mechanism of sodium channel regulation in disease susceptibility},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Long-read genome sequencing of bread wheat facilitates
disease resistance gene cloning. <em>NG</em>, <em>54</em>(3), 227–231.
(<a href="https://doi.org/10.1038/s41588-022-01022-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cloning of agronomically important genes from large, complex crop genomes remains challenging. Here we generate a 14.7 gigabase chromosome-scale assembly of the South African bread wheat (Triticum aestivum) cultivar Kariega by combining high-fidelity long reads, optical mapping and chromosome conformation capture. The resulting assembly is an order of magnitude more contiguous than previous wheat assemblies. Kariega shows durable resistance to the devastating fungal stripe rust disease1. We identified the race-specific disease resistance gene Yr27, which encodes an intracellular immune receptor, to be a major contributor to this resistance. Yr27 is allelic to the leaf rust resistance gene Lr13; the Yr27 and Lr13 proteins show 97% sequence identity2,3. Our results demonstrate the feasibility of generating chromosome-scale wheat assemblies to clone genes, and exemplify that highly similar alleles of a single-copy gene can confer resistance to different pathogens, which might provide a basis for engineering Yr27 alleles with multiple recognition specificities in the future. Chromosome-scale genome assembly of the South African bread wheat (Triticum aestivum) cultivar Kariega facilitates the cloning of the stripe rust resistance gene Yr27.},
  archive      = {J_NG},
  author       = {Athiyannan, Naveenkumar and Abrouk, Michael and Boshoff, Willem H. P. and Cauet, Stéphane and Rodde, Nathalie and Kudrna, David and Mohammed, Nahed and Bettgenhaeuser, Jan and Botha, Kirsty S. and Derman, Shannon S. and Wing, Rod A. and Prins, Renée and Krattinger, Simon G.},
  doi          = {10.1038/s41588-022-01022-1},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {227-231},
  shortjournal = {Nat. Genet.},
  title        = {Long-read genome sequencing of bread wheat facilitates disease resistance gene cloning},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Discovering missing heritability in whole-genome sequencing
data. <em>NG</em>, <em>54</em>(3), 224–226. (<a
href="https://doi.org/10.1038/s41588-022-01012-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The gap between heritability estimates from twin studies and those from genotyping array data has puzzled researchers for over a decade. New research suggests that much of the ‘missing’ heritability is due to rare variants that can only be captured by whole-genome sequencing (WGS) data.},
  archive      = {J_NG},
  author       = {Young, Alexander I.},
  doi          = {10.1038/s41588-022-01012-3},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {224-226},
  shortjournal = {Nat. Genet.},
  title        = {Discovering missing heritability in whole-genome sequencing data},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Nascent transcription as a predictor and driver of histone
modifications. <em>NG</em>, <em>54</em>(3), 223–224. (<a
href="https://doi.org/10.1038/s41588-021-01004-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new study demonstrates that profiles of nascent RNA accurately predict genomic patterns of histone modifications and chromatin state. Consistent with that, active histone marks are revealed to reflect transcription activity, rather than preceding or directing gene output.},
  archive      = {J_NG},
  author       = {Adelman, Karen},
  doi          = {10.1038/s41588-021-01004-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {223-224},
  shortjournal = {Nat. Genet.},
  title        = {Nascent transcription as a predictor and driver of histone modifications},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A call for an integrated approach to improve efficiency,
equity and sustainability in rare disease research in the united states.
<em>NG</em>, <em>54</em>(3), 219–222. (<a
href="https://doi.org/10.1038/s41588-022-01027-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To build a more efficient, equitable and sustainable approach to rare disease research in the United States, we must prioritize integrated research infrastructure and approaches that focus on understanding connections across rare diseases.},
  archive      = {J_NG},
  author       = {Halley, Meghan C. and Smith, Hadley Stevens and Ashley, Euan A. and Goldenberg, Aaron J. and Tabor, Holly K.},
  doi          = {10.1038/s41588-022-01027-w},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {219-222},
  shortjournal = {Nat. Genet.},
  title        = {A call for an integrated approach to improve efficiency, equity and sustainability in rare disease research in the united states},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A genetic research story of giving back and returning to the
country of a thousand hills. <em>NG</em>, <em>54</em>(3), 216–218. (<a
href="https://doi.org/10.1038/s41588-022-01028-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Would genetics research be a priority for Rwanda while the country was rebuilding just after the 1994 genocide against Tutsi? This was a question that I needed to consider. Sometimes, it is very hard to make the best choice for your career in a new scientific discipline when you have no role models and the only way forward is to start from scratch. Later, however, you can look back on what you have accomplished with surprise, and pride, when you see all your efforts paying off. Here I tell the story of my journey in genetics research, from rebuilding a country after trauma to facing our current COVID-19 pandemic challenges.},
  archive      = {J_NG},
  author       = {Mutesa, Léon},
  doi          = {10.1038/s41588-022-01028-9},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {216-218},
  shortjournal = {Nat. Genet.},
  title        = {A genetic research story of giving back and returning to the country of a thousand hills},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Rare diseases, common challenges. <em>NG</em>,
<em>54</em>(3), 215. (<a
href="https://doi.org/10.1038/s41588-022-01037-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The genetics community has a particularly important part to play in accelerating rare disease research and contributing to improving diagnosis and treatment. Innovations in sequencing technology and machine learning approaches have positively affected diagnostic success, but more coordinated efforts are needed to move towards effective therapies or even cures for these important, and sometimes overlooked, class of diseases.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01037-8},
  journal      = {Nature Genetics},
  month        = {3},
  number       = {3},
  pages        = {215},
  shortjournal = {Nat. Genet.},
  title        = {Rare diseases, common challenges},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Gain-of-function variants in SYK cause
immune dysregulation and systemic inflammation in humans and mice.
<em>NG</em>, <em>54</em>(2), 213. (<a
href="https://doi.org/10.1038/s41588-021-00998-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Wang, Lin and Aschenbrenner, Dominik and Zeng, Zhiyang and Cao, Xiya and Mayr, Daniel and Mehta, Meera and Capitani, Melania and Warner, Neil and Pan, Jie and Wang, Liren and Li, Qi and Zuo, Tao and Cohen-Kedar, Sarit and Lu, Jiawei and Ardy, Rico Chandra and Mulder, Daniel J. and Dissanayake, Dilan and Peng, Kaiyue and Huang, Zhiheng and Li, Xiaoqin and Wang, Yuesheng and Wang, Xiaobing and Li, Shuchao and Bullers, Samuel and Gammage, Anís N. and Warnatz, Klaus and Schiefer, Ana-Iris and Krivan, Gergely and Goda, Vera and Kahr, Walter H. A. and Lemaire, Mathieu and Lu, Chien-Yi and Siddiqui, Iram and Surette, Michael G. and Kotlarz, Daniel and Engelhardt, Karin R. and Griffin, Helen R. and Rottapel, Robert and Decaluwe, Hélène and Laxer, Ronald M. and Proietti, Michele and Hambleton, Sophie and Elcombe, Suzanne and Guo, Cong-Hui and Grimbacher, Bodo and Dotan, Iris and Ng, Siew C. and Freeman, Spencer A. and Snapper, Scott B. and Klein, Christoph and Boztug, Kaan and Huang, Ying and Li, Dali and Uhlig, Holm H. and Muise, Aleixo M.},
  doi          = {10.1038/s41588-021-00998-6},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {213},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Gain-of-function variants in SYK cause immune dysregulation and systemic inflammation in humans and mice},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Author correction: Mutations in PYCR1 cause cutis laxa with
progeroid features. <em>NG</em>, <em>54</em>(2), 213. (<a
href="https://doi.org/10.1038/s41588-022-01013-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NG},
  author       = {Reversade, Bruno and Escande-Beillard, Nathalie and Dimopoulou, Aikaterini and Fischer, Björn and Chng, Serene C. and Li, Yun and Shboul, Mohammad and Tham, Puay-Yoke and Kayserili, Hülya and Al-Gazali, Lihadh and Shahwan, Monzer and Brancati, Francesco and Lee, Hane and O’Connor, Brian D. and Kegler, Mareen Schmidt-von and Merriman, Barry and Nelson, Stanley F. and Masri, Amira and Alkazaleh, Fawaz and Guerra, Deanna and Ferrari, Paola and Nanda, Arti and Rajab, Anna and Markie, David and Gray, Mary and Nelson, John and Grix, Arthur and Sommer, Annemarie and Savarirayan, Ravi and Janecke, Andreas R. and Steichen, Elisabeth and Sillence, David and Haußer, Ingrid and Budde, Birgit and Nürnberg, Gudrun and Nürnberg, Peter and Seemann, Petra and Kunkel, Désirée and Zambruno, Giovanna and Dallapiccola, Bruno and Schuelke, Markus and Robertson, Stephen and Hamamy, Hanan and Wollnik, Bernd and Van Maldergem, Lionel and Mundlos, Stefan and Kornak, Uwe},
  doi          = {10.1038/s41588-022-01013-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {213},
  shortjournal = {Nat. Genet.},
  title        = {Author correction: Mutations in PYCR1 cause cutis laxa with progeroid features},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CRISPR and biochemical screens identify MAZ as a cofactor in
CTCF-mediated insulation at hox clusters. <em>NG</em>, <em>54</em>(2),
202–212. (<a href="https://doi.org/10.1038/s41588-021-01008-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CCCTC-binding factor (CTCF) is critical to three-dimensional genome organization. Upon differentiation, CTCF insulates active and repressed genes within Hox gene clusters. We conducted a genome-wide CRISPR knockout (KO) screen to identify genes required for CTCF-boundary activity at the HoxA cluster, complemented by biochemical approaches. Among the candidates, we identified Myc-associated zinc-finger protein (MAZ) as a cofactor in CTCF insulation. MAZ colocalizes with CTCF at chromatin borders and, similar to CTCF, interacts with the cohesin subunit RAD21. MAZ KO disrupts gene expression and local contacts within topologically associating domains. Similar to CTCF motif deletions, MAZ motif deletions lead to derepression of posterior Hox genes immediately after CTCF boundaries upon differentiation, giving rise to homeotic transformations in mouse. Thus, MAZ is a factor contributing to appropriate insulation, gene expression and genomic architecture during development. Genome-wide screens identify several genes, including MAZ, required for CTCF-mediated insulation. MAZ interacts with cohesin, and MAZ motif deletions derepress posterior Hox gene expression, leading to homeotic transformations in mouse.},
  archive      = {J_NG},
  author       = {Ortabozkoyun, Havva and Huang, Pin-Yao and Cho, Hyunwoo and Narendra, Varun and LeRoy, Gary and Gonzalez-Buendia, Edgar and Skok, Jane A. and Tsirigos, Aristotelis and Mazzoni, Esteban O. and Reinberg, Danny},
  doi          = {10.1038/s41588-021-01008-5},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {202-212},
  shortjournal = {Nat. Genet.},
  title        = {CRISPR and biochemical screens identify MAZ as a cofactor in CTCF-mediated insulation at hox clusters},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A chromosomal loop anchor mediates bacterial genome
organization. <em>NG</em>, <em>54</em>(2), 194–201. (<a
href="https://doi.org/10.1038/s41588-021-00988-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nucleoprotein complexes play an integral role in genome organization of both eukaryotes and prokaryotes. Apart from their role in locally structuring and compacting DNA, several complexes are known to influence global organization by mediating long-range anchored chromosomal loop formation leading to spatial segregation of large sections of DNA. Such megabase-range interactions are ubiquitous in eukaryotes, but have not been demonstrated in prokaryotes. Here, using a genome-wide sedimentation-based approach, we found that a transcription factor, Rok, forms large nucleoprotein complexes in the bacterium Bacillus subtilis. Using chromosome conformation capture and live-imaging of DNA loci, we show that these complexes robustly interact with each other over large distances. Importantly, these Rok-dependent long-range interactions lead to anchored chromosomal loop formation, thereby spatially isolating large sections of DNA, as previously observed for insulator proteins in eukaryotes. Hi-C and live-imaging data show that nucleoprotein complexes containing the transcription factor Rok interact over long distances in the bacterium Bacillus subtilis. Rok-dependent interactions contribute to anchored chromosomal loop formation.},
  archive      = {J_NG},
  author       = {Dugar, Gaurav and Hofmann, Andreas and Heermann, Dieter W. and Hamoen, Leendert W.},
  doi          = {10.1038/s41588-021-00988-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {194-201},
  shortjournal = {Nat. Genet.},
  title        = {A chromosomal loop anchor mediates bacterial genome organization},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). LINE1 are spliced in non-canonical transcript variants to
regulate t cell quiescence and exhaustion. <em>NG</em>, <em>54</em>(2),
180–193. (<a href="https://doi.org/10.1038/s41588-021-00989-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How gene expression is controlled to preserve human T cell quiescence is poorly understood. Here we show that non-canonical splicing variants containing long interspersed nuclear element 1 (LINE1) enforce naive CD4+ T cell quiescence. LINE1-containing transcripts are derived from CD4+ T cell-specific genes upregulated during T cell activation. In naive CD4+ T cells, LINE1-containing transcripts are regulated by the transcription factor IRF4 and kept at chromatin by nucleolin; these transcripts act in cis, hampering levels of histone 3 (H3) lysine 36 trimethyl (H3K36me3) and stalling gene expression. T cell activation induces LINE1-containing transcript downregulation by the splicing suppressor PTBP1 and promotes expression of the corresponding protein-coding genes by the elongating factor GTF2F1 through mTORC1. Dysfunctional T cells, exhausted in vitro or tumor-infiltrating lymphocytes (TILs), accumulate LINE1-containing transcripts at chromatin. Remarkably, depletion of LINE1-containing transcripts restores TIL effector function. Our study identifies a role for LINE1 elements in maintaining T cell quiescence and suggests that an abundance of LINE1-containing transcripts is critical for T cell effector function and exhaustion. Non-canonical transcripts containing LINE1 transposable elements maintain naive CD4+ T cell quiescence and interfere with gene expression in cis. LINE1-containing transcripts are downregulated upon T cell activation.},
  archive      = {J_NG},
  author       = {Marasca, Federica and Sinha, Shruti and Vadalà, Rebecca and Polimeni, Benedetto and Ranzani, Valeria and Paraboschi, Elvezia Maria and Burattin, Filippo Vittorio and Ghilotti, Marco and Crosti, Mariacristina and Negri, Maria Luce and Campagnoli, Susanna and Notarbartolo, Samuele and Sartore-Bianchi, Andrea and Siena, Salvatore and Prati, Daniele and Montini, Giovanni and Viale, Giuseppe and Torre, Olga and Harari, Sergio and Grifantini, Renata and Soldà, Giulia and Biffo, Stefano and Abrignani, Sergio and Bodega, Beatrice},
  doi          = {10.1038/s41588-021-00989-7},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {180-193},
  shortjournal = {Nat. Genet.},
  title        = {LINE1 are spliced in non-canonical transcript variants to regulate t cell quiescence and exhaustion},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Noncoding genetic variation in GATA3 increases acute
lymphoblastic leukemia risk through local and global changes in
chromatin conformation. <em>NG</em>, <em>54</em>(2), 170–179. (<a
href="https://doi.org/10.1038/s41588-021-00993-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inherited noncoding genetic variants confer significant disease susceptibility to childhood acute lymphoblastic leukemia (ALL) but the molecular processes linking germline polymorphisms with somatic lesions in this cancer are poorly understood. Through targeted sequencing in 5,008 patients, we identified a key regulatory germline variant in GATA3 associated with Philadelphia chromosome-like ALL (Ph-like ALL). Using CRISPR–Cas9 editing and samples from patients with Ph-like ALL, we showed that this variant activated a strong enhancer that upregulated GATA3 transcription. This, in turn, reshaped global chromatin accessibility and three-dimensional genome organization, including regions proximal to the ALL oncogene CRLF2. Finally, we showed that GATA3 directly regulated CRLF2 and potentiated the JAK–STAT oncogenic effects during leukemogenesis. Taken together, we provide evidence for a distinct mechanism by which a germline noncoding variant contributes to oncogene activation, epigenetic regulation and three-dimensional genome reprogramming. A germline variant associated with acute lymphoblastic leukemia activates an enhancer element resulting in increased GATA3 expression, altered chromatin accessibility and changes in three-dimensional genome organization.},
  archive      = {J_NG},
  author       = {Yang, Hongbo and Zhang, Hui and Luan, Yu and Liu, Tingting and Yang, Wentao and Roberts, Kathryn G. and Qian, Mao-xiang and Zhang, Bo and Yang, Wenjian and Perez-Andreu, Virginia and Xu, Jie and Iyyanki, Sriranga and Kuang, Da and Stasiak, Lena A. and Reshmi, Shalini C. and Gastier-Foster, Julie and Smith, Colton and Pui, Ching-Hon and Evans, William E. and Hunger, Stephen P. and Platanias, Leonidas C. and Relling, Mary V. and Mullighan, Charles G. and Loh, Mignon L. and Yue, Feng and Yang, Jun J.},
  doi          = {10.1038/s41588-021-00993-x},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {170-179},
  shortjournal = {Nat. Genet.},
  title        = {Noncoding genetic variation in GATA3 increases acute lymphoblastic leukemia risk through local and global changes in chromatin conformation},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multi-ancestry eQTL meta-analysis of human brain identifies
candidate causal variants for brain-related traits. <em>NG</em>,
<em>54</em>(2), 161–169. (<a
href="https://doi.org/10.1038/s41588-021-00987-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While large-scale, genome-wide association studies (GWAS) have identified hundreds of loci associated with brain-related traits, identification of the variants, genes and molecular mechanisms underlying these traits remains challenging. Integration of GWAS with expression quantitative trait loci (eQTLs) and identification of shared genetic architecture have been widely adopted to nominate genes and candidate causal variants. However, this approach is limited by sample size, statistical power and linkage disequilibrium. We developed the multivariate multiple QTL approach and performed a large-scale, multi-ancestry eQTL meta-analysis to increase power and fine-mapping resolution. Analysis of 3,983 RNA-sequenced samples from 2,119 donors, including 474 non-European individuals, yielded an effective sample size of 3,154. Joint statistical fine-mapping of eQTL and GWAS identified 329 variant–trait pairs for 24 brain-related traits driven by 204 unique candidate causal variants for 189 unique genes. This integrative analysis identifies candidate causal variants and elucidates potential regulatory mechanisms for genes underlying schizophrenia, bipolar disorder and Alzheimer’s disease. A multi-ancestry expression quantitative trait locus meta-analysis of 3,983 RNA-seq samples from 2,119 donors using the multivariate multiple QTL (mmQTL) approach characterizes the genetics of gene expression in the human brain and identifies candidate causal variants for brain-related traits.},
  archive      = {J_NG},
  author       = {Zeng, Biao and Bendl, Jaroslav and Kosoy, Roman and Fullard, John F. and Hoffman, Gabriel E. and Roussos, Panos},
  doi          = {10.1038/s41588-021-00987-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {161-169},
  shortjournal = {Nat. Genet.},
  title        = {Multi-ancestry eQTL meta-analysis of human brain identifies candidate causal variants for brain-related traits},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genome-wide analysis of 102,084 migraine cases identifies
123 risk loci and subtype-specific risk alleles. <em>NG</em>,
<em>54</em>(2), 152–160. (<a
href="https://doi.org/10.1038/s41588-021-00990-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Migraine affects over a billion individuals worldwide but its genetic underpinning remains largely unknown. Here, we performed a genome-wide association study of 102,084 migraine cases and 771,257 controls and identified 123 loci, of which 86 are previously unknown. These loci provide an opportunity to evaluate shared and distinct genetic components in the two main migraine subtypes: migraine with aura and migraine without aura. Stratification of the risk loci using 29,679 cases with subtype information indicated three risk variants that seem specific for migraine with aura (in HMOX2, CACNA1A and MPPED2), two that seem specific for migraine without aura (near SPINK2 and near FECH) and nine that increase susceptibility for migraine regardless of subtype. The new risk loci include genes encoding recent migraine-specific drug targets, namely calcitonin gene-related peptide (CALCA/CALCB) and serotonin 1F receptor (HTR1F). Overall, genomic annotations among migraine-associated variants were enriched in both vascular and central nervous system tissue/cell types, supporting unequivocally that neurovascular mechanisms underlie migraine pathophysiology. Genome-wide association analyses identify 123 susceptibility loci for migraine and implicate neurovascular mechanisms in its pathophysiology. Subtype analyses highlight risk loci specific for migraine with or without aura in addition to shared risk variants.},
  archive      = {J_NG},
  author       = {Hautakangas, Heidi and Winsvold, Bendik S. and Ruotsalainen, Sanni E. and Bjornsdottir, Gyda and Harder, Aster V. E. and Kogelman, Lisette J. A. and Thomas, Laurent F. and Noordam, Raymond and Benner, Christian and Gormley, Padhraig and Artto, Ville and Banasik, Karina and Bjornsdottir, Anna and Boomsma, Dorret I. and Brumpton, Ben M. and Burgdorf, Kristoffer Sølvsten and Buring, Julie E. and Chalmer, Mona Ameri and de Boer, Irene and Dichgans, Martin and Erikstrup, Christian and Färkkilä, Markus and Garbrielsen, Maiken Elvestad and Ghanbari, Mohsen and Hagen, Knut and Häppölä, Paavo and Hottenga, Jouke-Jan and Hrafnsdottir, Maria G. and Hveem, Kristian and Johnsen, Marianne Bakke and Kähönen, Mika and Kristoffersen, Espen S. and Kurth, Tobias and Lehtimäki, Terho and Lighart, Lannie and Magnusson, Sigurdur H. and Malik, Rainer and Pedersen, Ole Birger and Pelzer, Nadine and Penninx, Brenda W. J. H. and Ran, Caroline and Ridker, Paul M. and Rosendaal, Frits R. and Sigurdardottir, Gudrun R. and Skogholt, Anne Heidi and Sveinsson, Olafur A. and Thorgeirsson, Thorgeir E. and Ullum, Henrik and Vijfhuizen, Lisanne S. and Widén, Elisabeth and van Dijk, Ko Willems and Aromaa, Arpo and Belin, Andrea Carmine and Freilinger, Tobias and Ikram, M. Arfan and Järvelin, Marjo-Riitta and Raitakari, Olli T. and Terwindt, Gisela M. and Kallela, Mikko and Wessman, Maija and Olesen, Jes and Chasman, Daniel I. and Nyholt, Dale R. and Stefánsson, Hreinn and Stefansson, Kari and van den Maagdenberg, Arn M. J. M. and Hansen, Thomas Folkmann and Ripatti, Samuli and Zwart, John-Anker and Palotie, Aarno and Pirinen, Matti},
  doi          = {10.1038/s41588-021-00990-0},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {152-160},
  shortjournal = {Nat. Genet.},
  title        = {Genome-wide analysis of 102,084 migraine cases identifies 123 risk loci and subtype-specific risk alleles},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Effect of host genetics on the gut microbiome in 7,738
participants of the dutch microbiome project. <em>NG</em>,
<em>54</em>(2), 143–151. (<a
href="https://doi.org/10.1038/s41588-021-00992-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Host genetics are known to influence the gut microbiome, yet their role remains poorly understood. To robustly characterize these effects, we performed a genome-wide association study of 207 taxa and 205 pathways representing microbial composition and function in 7,738 participants of the Dutch Microbiome Project. Two robust, study-wide significant (P &amp;lt; 1.89 × 10−10) signals near the LCT and ABO genes were found to be associated with multiple microbial taxa and pathways and were replicated in two independent cohorts. The LCT locus associations seemed modulated by lactose intake, whereas those at ABO could be explained by participant secretor status determined by their FUT2 genotype. Twenty-two other loci showed suggestive evidence (P &amp;lt; 5 × 10−8) of association with microbial taxa and pathways. At a more lenient threshold, the number of loci we identified strongly correlated with trait heritability, suggesting that much larger sample sizes are needed to elucidate the remaining effects of host genetics on the gut microbiome. A genome-wide association study of 207 taxa and 205 pathways representing gut microbial composition and function from 7,738 individuals of the Dutch Microbiome Project identifies genetic associations at the LCT and ABO loci.},
  archive      = {J_NG},
  author       = {Lopera-Maya, Esteban A. and Kurilshikov, Alexander and van der Graaf, Adriaan and Hu, Shixian and Andreu-Sánchez, Sergio and Chen, Lianmin and Vila, Arnau Vich and Gacesa, Ranko and Sinha, Trishla and Collij, Valerie and Klaassen, Marjiolein A. Y. and Bolte, Laura A. and Gois, Milla F. Brandao and Neerincx, Pieter B. T. and Swertz, Morris A. and Harmsen, Hermie J. M. and Wijmenga, Cisca and Fu, Jingyuan and Weersma, Rinse K. and Zhernakova, Alexandra and Sanna, Serena},
  doi          = {10.1038/s41588-021-00992-y},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {143-151},
  shortjournal = {Nat. Genet.},
  title        = {Effect of host genetics on the gut microbiome in 7,738 participants of the dutch microbiome project},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Combined effects of host genetics and diet on human gut
microbiota and incident disease in a single population cohort.
<em>NG</em>, <em>54</em>(2), 134–142. (<a
href="https://doi.org/10.1038/s41588-021-00991-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human genetic variation affects the gut microbiota through a complex combination of environmental and host factors. Here we characterize genetic variations associated with microbial abundances in a single large-scale population-based cohort of 5,959 genotyped individuals with matched gut microbial metagenomes, and dietary and health records (prevalent and follow-up). We identified 567 independent SNP–taxon associations. Variants at the LCT locus associated with Bifidobacterium and other taxa, but they differed according to dairy intake. Furthermore, levels of Faecalicatena lactaris associated with ABO, and suggested preferential utilization of secreted blood antigens as energy source in the gut. Enterococcus faecalis levels associated with variants in the MED13L locus, which has been linked to colorectal cancer. Mendelian randomization analysis indicated a potential causal effect of Morganella on major depressive disorder, consistent with observational incident disease analysis. Overall, we identify and characterize the intricate nature of host–microbiota interactions and their association with disease. Genome-wide association analysis of gut microbial taxa in a single homogenous population-based cohort of 5,959 Finnish individuals identifies 567 independent SNP–taxon associations, including strong associations with LCT, ABO and MED13L.},
  archive      = {J_NG},
  author       = {Qin, Youwen and Havulinna, Aki S. and Liu, Yang and Jousilahti, Pekka and Ritchie, Scott C. and Tokolyi, Alex and Sanders, Jon G. and Valsta, Liisa and Brożyńska, Marta and Zhu, Qiyun and Tripathi, Anupriya and Vázquez-Baeza, Yoshiki and Loomba, Rohit and Cheng, Susan and Jain, Mohit and Niiranen, Teemu and Lahti, Leo and Knight, Rob and Salomaa, Veikko and Inouye, Michael and Méric, Guillaume},
  doi          = {10.1038/s41588-021-00991-z},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {134-142},
  shortjournal = {Nat. Genet.},
  title        = {Combined effects of host genetics and diet on human gut microbiota and incident disease in a single population cohort},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Biallelic mutations in cancer genomes reveal local
mutational determinants. <em>NG</em>, <em>54</em>(2), 128–133. (<a
href="https://doi.org/10.1038/s41588-021-01005-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The infinite sites model of molecular evolution posits that every position in the genome is mutated at most once1. By restricting the number of possible mutation histories, haplotypes and alleles, it forms a cornerstone of tumor phylogenetic analysis2 and is often implied when calling, phasing and interpreting variants3,4 or studying the mutational landscape as a whole5. Here we identify 18,295 biallelic mutations, where the same base is mutated independently on both parental copies, in 559 (21%) bulk sequencing samples from the Pan-Cancer Analysis of Whole Genomes study. Biallelic mutations reveal ultraviolet light damage hotspots at E26 transformation-specific (ETS) and nuclear factor of activated T cells (NFAT) binding sites, and hypermutable motifs in POLE-mutant and other cancers. We formulate recommendations for variant calling and provide frameworks to model and detect biallelic mutations. These results highlight the need for accurate models of mutation rates and tumor evolution, as well as their inference from sequencing data. Common mutational analyses assume that every base in the genome can only mutate once. Here, the authors report multiple violations of this infinite sites model in whole-genome data across a range of tumor types.},
  archive      = {J_NG},
  author       = {Demeulemeester, Jonas and Dentro, Stefan C. and Gerstung, Moritz and Van Loo, Peter},
  doi          = {10.1038/s41588-021-01005-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {128-133},
  shortjournal = {Nat. Genet.},
  title        = {Biallelic mutations in cancer genomes reveal local mutational determinants},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Multi-ancestry fine mapping implicates OAS1 splicing in risk
of severe COVID-19. <em>NG</em>, <em>54</em>(2), 125–127. (<a
href="https://doi.org/10.1038/s41588-021-00996-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The OAS1/2/3 cluster has been identified as a risk locus for severe COVID-19 among individuals of European ancestry, with a protective haplotype of approximately 75 kilobases (kb) derived from Neanderthals in the chromosomal region 12q24.13. This haplotype contains a splice variant of OAS1, which occurs in people of African ancestry independently of gene flow from Neanderthals. Using trans-ancestry fine-mapping approaches in 20,779 hospitalized cases, we demonstrate that this splice variant is likely to be the SNP responsible for the association at this locus, thus strongly implicating OAS1 as an effector gene influencing COVID-19 severity. Multi-ancestry fine-mapping of the OAS1/2/3 region shows that a splice site variant in OAS1 is likely responsible for the association of this locus with the risk of severe COVID-19.},
  archive      = {J_NG},
  author       = {Huffman, Jennifer E. and Butler-Laporte, Guillaume and Khan, Atlas and Pairo-Castineira, Erola and Drivas, Theodore G. and Peloso, Gina M. and Nakanishi, Tomoko and Ganna, Andrea and Verma, Anurag and Baillie, J. Kenneth and Kiryluk, Krzysztof and Richards, J. Brent and Zeberg, Hugo},
  doi          = {10.1038/s41588-021-00996-8},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {125-127},
  shortjournal = {Nat. Genet.},
  title        = {Multi-ancestry fine mapping implicates OAS1 splicing in risk of severe COVID-19},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The UGT2A1/UGT2A2 locus is associated with COVID-19-related
loss of smell or taste. <em>NG</em>, <em>54</em>(2), 121–124. (<a
href="https://doi.org/10.1038/s41588-021-00986-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using online surveys, we collected data regarding COVID-19-related loss of smell or taste from 69,841 individuals. We performed a multi-ancestry genome-wide association study and identified a genome-wide significant locus in the vicinity of the UGT2A1 and UGT2A2 genes. Both genes are expressed in the olfactory epithelium and play a role in metabolizing odorants. These findings provide a genetic link to the biological mechanisms underlying COVID-19-related loss of smell or taste. Multi-ancestry genome-wide analyses identify variants near UGT2A1 and UGT2A2 associated with COVID-19-related loss of smell or taste. Both genes are expressed in the olfactory epithelium and play a role in metabolizing odorants.},
  archive      = {J_NG},
  author       = {Shelton, Janie F. and Shastri, Anjali J. and Fletez-Brant, Kipper and Aslibekyan, Stella and Auton, Adam},
  doi          = {10.1038/s41588-021-00986-w},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {121-124},
  shortjournal = {Nat. Genet.},
  title        = {The UGT2A1/UGT2A2 locus is associated with COVID-19-related loss of smell or taste},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The threat of programmed DNA damage to neuronal genome
integrity and plasticity. <em>NG</em>, <em>54</em>(2), 115–120. (<a
href="https://doi.org/10.1038/s41588-021-01001-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The neuronal genome is particularly sensitive to loss or attenuation of DNA repair, and many neurological diseases ensue when DNA repair is impaired. It is well-established that the neuronal genome is subjected to stochastic DNA damage, most likely because of extensive oxidative stress in the brain. However, recent studies have identified unexpected high levels of ‘programmed’ DNA breakage in neurons, which we propose arise during physiological DNA metabolic processes intrinsic to neuronal development, differentiation and maintenance. The role of programmed DNA breaks in normal neuronal physiology and disease remains relatively unexplored thus far. However, bulk and single-cell sequencing analyses of neurodegenerative diseases have revealed age-related somatic mutational signatures that are enriched in regulatory regions of the genome. Here, we explore a paradigm of DNA repair in neurons, in which the genome is safeguarded from erroneous impacts of programmed genome breakage intrinsic to normal neuronal function. Normal cellular processes can cause DNA breaks which become substrates for the cell’s DNA repair machinery. Focusing on neurons, this Perspective article explores the role of this ‘programmed’ DNA damage and its repair in health, ageing and neurodegenerative disease.},
  archive      = {J_NG},
  author       = {Caldecott, Keith W. and Ward, Michael E. and Nussenzweig, André},
  doi          = {10.1038/s41588-021-01001-y},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {115-120},
  shortjournal = {Nat. Genet.},
  title        = {The threat of programmed DNA damage to neuronal genome integrity and plasticity},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The genomic and spatial mobility of extrachromosomal DNA and
its implications for cancer therapy. <em>NG</em>, <em>54</em>(2),
107–114. (<a href="https://doi.org/10.1038/s41588-021-01000-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extrachromosomal DNA (ecDNA) amplification has been observed in at least 30 different cancer types and is associated with worse patient outcomes. This has been linked to increased oncogene dosage because both oncogenes and associated enhancers can occupy ecDNA. New data challenge the view that only oncogene dosage is affected by ecDNA, and raises the possibility that ecDNA could disrupt genome-wide gene expression. Recent investigations suggest that ecDNA localizes to specialized nuclear bodies (hubs) in which they can act in trans as ectopic enhancers for genes on other ecDNA or chromosomes. Moreover, ecDNA can reintegrate into the genome, possibly further disrupting the gene regulatory landscape in tumor cells. In this Perspective, we discuss the emerging properties of ecDNA and highlight promising avenues to exploit this new knowledge for the development of ecDNA-directed therapies for cancer. This Perspective explores the spatial and genomic mobility of extrachromosomal DNA within the cell and proposes how these properties may be harnessed for therapeutic benefit.},
  archive      = {J_NG},
  author       = {van Leen, Eric and Brückner, Lotte and Henssen, Anton G.},
  doi          = {10.1038/s41588-021-01000-z},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {107-114},
  shortjournal = {Nat. Genet.},
  title        = {The genomic and spatial mobility of extrachromosomal DNA and its implications for cancer therapy},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Challenges and future directions for studying effects of
host genetics on the gut microbiome. <em>NG</em>, <em>54</em>(2),
100–106. (<a href="https://doi.org/10.1038/s41588-021-00983-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human gut microbiome is a complex ecosystem that is involved in its host’s metabolism, immunity and health. Although interindividual variations in gut microbial composition are mainly driven by environmental factors, some gut microorganisms are heritable and thus can be influenced by host genetics. In the past 5 years, 12 microbial genome-wide association studies (mbGWAS) with &amp;gt;1,000 participants have been published, yet only a few genetic loci have been consistently confirmed across multiple studies. Here we discuss the state of the art for mbGWAS, focusing on current challenges such as the heterogeneity of microbiome measurements and power issues, and we elaborate on potential future directions for genetic analysis of the microbiome. This Perspective discusses the analytical issues concerning heterogeneity and power encountered in microbial genome-wide association studies and highlights potential future directions for genetic analysis of the microbiome.},
  archive      = {J_NG},
  author       = {Sanna, Serena and Kurilshikov, Alexander and van der Graaf, Adriaan and Fu, Jingyuan and Zhernakova, Alexandra},
  doi          = {10.1038/s41588-021-00983-z},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {100-106},
  shortjournal = {Nat. Genet.},
  title        = {Challenges and future directions for studying effects of host genetics on the gut microbiome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Walking the LINEs hidden in the dark matter of the genome.
<em>NG</em>, <em>54</em>(2), 98–99. (<a
href="https://doi.org/10.1038/s41588-021-01003-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The function of transposable elements present in mammalian genomes remains an enigma. In this issue, Bodega, Abrignani and colleagues show that LINE1-containing transcripts are key regulators of T cell effector function and exhaustion.},
  archive      = {J_NG},
  author       = {Lusic, Marina and Mhlanga, Musa M.},
  doi          = {10.1038/s41588-021-01003-w},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {98-99},
  shortjournal = {Nat. Genet.},
  title        = {Walking the LINEs hidden in the dark matter of the genome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). MAZ regulates chromatin domain insulation and genome
architecture. <em>NG</em>, <em>54</em>(2), 96–97. (<a
href="https://doi.org/10.1038/s41588-021-00995-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Similar to CTCF, MAZ insulates repressed posterior Hoxa genes from the spreading of anterior active regulatory cues during motoneuron differentiation. This discovery provides new perspectives to understand chromatin organization and insulation.},
  archive      = {J_NG},
  author       = {Darbellay, Fabrice and Rouco, Raquel and Andrey, Guillaume},
  doi          = {10.1038/s41588-021-00995-9},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {96-97},
  shortjournal = {Nat. Genet.},
  title        = {MAZ regulates chromatin domain insulation and genome architecture},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Our genes, our microbes. <em>NG</em>, <em>54</em>(2), 95.
(<a href="https://doi.org/10.1038/s41588-022-01021-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In recent years, large-scale genomic studies have been performed in attempts to determine how genetic variation in the human host influences the gut microbiome. As microbiome traits are very heterogeneous, new analytical approaches are needed to move this field forward. By using genetic tools, there is a huge opportunity to enrich our understanding of the complex link between humans and our intimately associated microbial species.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-022-01021-2},
  journal      = {Nature Genetics},
  month        = {2},
  number       = {2},
  pages        = {95},
  shortjournal = {Nat. Genet.},
  title        = {Our genes, our microbes},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A PARTHENOGENESIS allele from apomictic dandelion can induce
egg cell division without fertilization in lettuce. <em>NG</em>,
<em>54</em>(1), 84–93. (<a
href="https://doi.org/10.1038/s41588-021-00984-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Apomixis, the clonal formation of seeds, is a rare yet widely distributed trait in flowering plants. We have isolated the PARTHENOGENESIS (PAR) gene from apomictic dandelion that triggers embryo development in unfertilized egg cells. PAR encodes a K2-2 zinc finger, EAR-domain protein. Unlike the recessive sexual alleles, the dominant PAR allele is expressed in egg cells and has a miniature inverted-repeat transposable element (MITE) transposon insertion in the promoter. The MITE-containing promoter can invoke a homologous gene from sexual lettuce to complement dandelion LOSS OF PARTHENOGENESIS mutants. A similar MITE is also present in the promoter of the PAR gene in apomictic forms of hawkweed, suggesting a case of parallel evolution. Heterologous expression of dandelion PAR in lettuce egg cells induced haploid embryo-like structures in the absence of fertilization. Sexual PAR alleles are expressed in pollen, suggesting that the gene product releases a block on embryogenesis after fertilization in sexual species while in apomictic species PAR expression triggers embryogenesis in the absence of fertilization. The PARTHENOGENESIS (PAR) gene is identified in apomictic dandelion. A dominant allele has a MITE transposon insertion similar to that found in apomictic hawkweed. Expression of dandelion PAR in lettuce induces embryo-like structures without fertilization.},
  archive      = {J_NG},
  author       = {Underwood, Charles J. and Vijverberg, Kitty and Rigola, Diana and Okamoto, Shunsuke and Oplaat, Carla and Camp, Rik H. M. Op den and Radoeva, Tatyana and Schauer, Stephen E. and Fierens, Joke and Jansen, Kim and Mansveld, Sandra and Busscher, Marco and Xiong, Wei and Datema, Erwin and Nijbroek, Koen and Blom, Evert-Jan and Bicknell, Ross and Catanach, Andrew and Erasmuson, Sylvia and Winefield, Christopher and van Tunen, Arjen J. and Prins, Marcel and Schranz, M. Eric and van Dijk, Peter J.},
  doi          = {10.1038/s41588-021-00984-y},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {84-93},
  shortjournal = {Nat. Genet.},
  title        = {A PARTHENOGENESIS allele from apomictic dandelion can induce egg cell division without fertilization in lettuce},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Two divergent haplotypes from a highly heterozygous lychee
genome suggest independent domestication events for early and
late-maturing cultivars. <em>NG</em>, <em>54</em>(1), 73–83. (<a
href="https://doi.org/10.1038/s41588-021-00971-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lychee is an exotic tropical fruit with a distinct flavor. The genome of cultivar ‘Feizixiao’ was assembled into 15 pseudochromosomes, totaling ~470 Mb. High heterozygosity (2.27%) resulted in two complete haplotypic assemblies. A total of 13,517 allelic genes (42.4%) were differentially expressed in diverse tissues. Analyses of 72 resequenced lychee accessions revealed two independent domestication events. The extremely early maturing cultivars preferentially aligned to one haplotype were domesticated from a wild population in Yunnan, whereas the late-maturing cultivars that mapped mostly to the second haplotype were domesticated independently from a wild population in Hainan. Early maturing cultivars were probably developed in Guangdong via hybridization between extremely early maturing cultivar and late-maturing cultivar individuals. Variable deletions of a 3.7 kb region encompassed by a pair of CONSTANS-like genes probably regulate fruit maturation differences among lychee cultivars. These genomic resources provide insights into the natural history of lychee domestication and will accelerate the improvement of lychee and related crops. Two divergent haplotypes from a highly heterozygous lychee genome of the cultivar ‘Feizixiao’ and resequencing of 72 lychee accessions provide insights into the genome evolution and domestication history of lychee.},
  archive      = {J_NG},
  author       = {Hu, Guibing and Feng, Junting and Xiang, Xu and Wang, Jiabao and Salojärvi, Jarkko and Liu, Chengming and Wu, Zhenxian and Zhang, Jisen and Liang, Xinming and Jiang, Zide and Liu, Wei and Ou, Liangxi and Li, Jiawei and Fan, Guangyi and Mai, Yingxiao and Chen, Chengjie and Zhang, Xingtan and Zheng, Jiakun and Zhang, Yanqing and Peng, Hongxiang and Yao, Lixian and Wai, Ching Man and Luo, Xinping and Fu, Jiaxin and Tang, Haibao and Lan, Tianying and Lai, Biao and Sun, Jinhua and Wei, Yongzan and Li, Huanling and Chen, Jiezhen and Huang, Xuming and Yan, Qian and Liu, Xin and McHale, Leah K. and Rolling, William and Guyot, Romain and Sankoff, David and Zheng, Chunfang and Albert, Victor A. and Ming, Ray and Chen, Houbin and Xia, Rui and Li, Jianguo},
  doi          = {10.1038/s41588-021-00971-3},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {73-83},
  shortjournal = {Nat. Genet.},
  title        = {Two divergent haplotypes from a highly heterozygous lychee genome suggest independent domestication events for early and late-maturing cultivars},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Discovery of a genetic module essential for assigning
left–right asymmetry in humans and ancestral vertebrates. <em>NG</em>,
<em>54</em>(1), 62–72. (<a
href="https://doi.org/10.1038/s41588-021-00970-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The vertebrate left–right axis is specified during embryogenesis by a transient organ: the left–right organizer (LRO). Species including fish, amphibians, rodents and humans deploy motile cilia in the LRO to break bilateral symmetry, while reptiles, birds, even-toed mammals and cetaceans are believed to have LROs without motile cilia. We searched for genes whose loss during vertebrate evolution follows this pattern and identified five genes encoding extracellular proteins, including a putative protease with hitherto unknown functions that we named ciliated left–right organizer metallopeptidase (CIROP). Here, we show that CIROP is specifically expressed in ciliated LROs. In zebrafish and Xenopus, CIROP is required solely on the left side, downstream of the leftward flow, but upstream of DAND5, the first asymmetrically expressed gene. We further ascertained 21 human patients with loss-of-function CIROP mutations presenting with recessive situs anomalies. Our findings posit the existence of an ancestral genetic module that has twice disappeared during vertebrate evolution but remains essential for distinguishing left from right in humans. Phylogenomic and genetic analyses identify an ancestral module of genes expressed specifically in ciliated left–right organizer tissue and required for left–right axis specification in humans and certain vertebrates.},
  archive      = {J_NG},
  author       = {Szenker-Ravi, Emmanuelle and Ott, Tim and Khatoo, Muznah and Moreau de Bellaing, Anne and Goh, Wei Xuan and Chong, Yan Ling and Beckers, Anja and Kannesan, Darshini and Louvel, Guillaume and Anujan, Priyanka and Ravi, Vydianathan and Bonnard, Carine and Moutton, Sébastien and Schoen, Patric and Fradin, Mélanie and Colin, Estelle and Megarbane, André and Daou, Linda and Chehab, Ghassan and Di Filippo, Sylvie and Rooryck, Caroline and Deleuze, Jean-François and Boland, Anne and Arribard, Nicolas and Eker, Rukiye and Tohari, Sumanty and Ng, Alvin Yu-Jin and Rio, Marlène and Lim, Chun Teck and Eisenhaber, Birgit and Eisenhaber, Frank and Venkatesh, Byrappa and Amiel, Jeanne and Crollius, Hugues Roest and Gordon, Christopher T. and Gossler, Achim and Roy, Sudipto and Attie-Bitach, Tania and Blum, Martin and Bouvagnet, Patrice and Reversade, Bruno},
  doi          = {10.1038/s41588-021-00970-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {62-72},
  shortjournal = {Nat. Genet.},
  title        = {Discovery of a genetic module essential for assigning left–right asymmetry in humans and ancestral vertebrates},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mendelian randomization analyses support causal
relationships between blood metabolites and the gut microbiome.
<em>NG</em>, <em>54</em>(1), 52–61. (<a
href="https://doi.org/10.1038/s41588-021-00968-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The gut microbiome has been implicated in a variety of physiological states, but controversy over causality remains unresolved. Here, we performed bidirectional Mendelian randomization analyses on 3,432 Chinese individuals with whole-genome, whole-metagenome, anthropometric and blood metabolic trait data. We identified 58 causal relationships between the gut microbiome and blood metabolites, and replicated 43 of them. Increased relative abundances of fecal Oscillibacter and Alistipes were causally linked to decreased triglyceride concentration. Conversely, blood metabolites such as glutamic acid appeared to decrease fecal Oxalobacter, and members of Proteobacteria were influenced by metabolites such as 5-methyltetrahydrofolic acid, alanine, glutamate and selenium. Two-sample Mendelian randomization with data from Biobank Japan partly corroborated results with triglyceride and with uric acid, and also provided causal support for published fecal bacterial markers for cancer and cardiovascular diseases. This study illustrates the value of human genetic information to help prioritize gut microbial features for mechanistic and clinical studies. Bidirectional Mendelian randomization analyses in 3,432 Chinese individuals identify putative causal relationships between the gut microbiome and blood metabolite levels.},
  archive      = {J_NG},
  author       = {Liu, Xiaomin and Tong, Xin and Zou, Yuanqiang and Lin, Xiaoqian and Zhao, Hui and Tian, Liu and Jie, Zhuye and Wang, Qi and Zhang, Zhe and Lu, Haorong and Xiao, Liang and Qiu, Xuemei and Zi, Jin and Wang, Rong and Xu, Xun and Yang, Huanming and Wang, Jian and Zong, Yang and Liu, Weibin and Hou, Yong and Zhu, Shida and Jia, Huijue and Zhang, Tao},
  doi          = {10.1038/s41588-021-00968-y},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {52-61},
  shortjournal = {Nat. Genet.},
  title        = {Mendelian randomization analyses support causal relationships between blood metabolites and the gut microbiome},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep learning enables genetic analysis of the human thoracic
aorta. <em>NG</em>, <em>54</em>(1), 40–51. (<a
href="https://doi.org/10.1038/s41588-021-00962-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enlargement or aneurysm of the aorta predisposes to dissection, an important cause of sudden death. We trained a deep learning model to evaluate the dimensions of the ascending and descending thoracic aorta in 4.6 million cardiac magnetic resonance images from the UK Biobank. We then conducted genome-wide association studies in 39,688 individuals, identifying 82 loci associated with ascending and 47 with descending thoracic aortic diameter, of which 14 loci overlapped. Transcriptome-wide analyses, rare-variant burden tests and human aortic single nucleus RNA sequencing prioritized genes including SVIL, which was strongly associated with descending aortic diameter. A polygenic score for ascending aortic diameter was associated with thoracic aortic aneurysm in 385,621 UK Biobank participants (hazard ratio = 1.43 per s.d., confidence interval 1.32–1.54, P = 3.3 × 10−20). Our results illustrate the potential for rapidly defining quantitative traits with deep learning, an approach that can be broadly applied to biomedical images. Genome-wide association analyses identify variants associated with thoracic aortic diameter. A polygenic score for ascending aortic diameter was associated with a diagnosis of thoracic aortic aneurysm in independent samples.},
  archive      = {J_NG},
  author       = {Pirruccello, James P. and Chaffin, Mark D. and Chou, Elizabeth L. and Fleming, Stephen J. and Lin, Honghuang and Nekoui, Mahan and Khurshid, Shaan and Friedman, Samuel F. and Bick, Alexander G. and Arduini, Alessandro and Weng, Lu-Chen and Choi, Seung Hoan and Akkad, Amer-Denis and Batra, Puneet and Tucker, Nathan R. and Hall, Amelia W. and Roselli, Carolina and Benjamin, Emelia J. and Vellarikkal, Shamsudheen K. and Gupta, Rajat M. and Stegmann, Christian M. and Juric, Dejan and Stone, James R. and Vasan, Ramachandran S. and Ho, Jennifer E. and Hoffmann, Udo and Lubitz, Steven A. and Philippakis, Anthony A. and Lindsay, Mark E. and Ellinor, Patrick T.},
  doi          = {10.1038/s41588-021-00962-4},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {40-51},
  shortjournal = {Nat. Genet.},
  title        = {Deep learning enables genetic analysis of the human thoracic aorta},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Large uncertainty in individual polygenic risk score
estimation impacts PRS-based risk stratification. <em>NG</em>,
<em>54</em>(1), 30–39. (<a
href="https://doi.org/10.1038/s41588-021-00961-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although the cohort-level accuracy of polygenic risk scores (PRSs)—estimates of genetic value at the individual level—has been widely assessed, uncertainty in PRSs remains underexplored. In the present study, we show that Bayesian PRS methods can estimate the variance of an individual’s PRS and can yield well-calibrated credible intervals via posterior sampling. For 13 real traits in the UK Biobank (n = 291,273 unrelated ‘white British’), we observe large variances in individual PRS estimates which impact interpretation of PRS-based stratification; averaging across traits, only 0.8% (s.d. = 1.6%) of individuals with PRS point estimates in the top decile have corresponding 95% credible intervals fully contained in the top decile. We provide an analytical estimator for the expectation of individual PRS variance as a function of SNP heritability, number of causal SNPs and sample size. Our results showcase the importance of incorporating uncertainty in individual PRS estimates into subsequent analyses. Analysis of real traits in the UK Biobank demonstrates that large uncertainty in polygenic risk score (PRS) estimates at the individual level impacts the interpretation of subsequent analyses such as PRS-based stratification.},
  archive      = {J_NG},
  author       = {Ding, Yi and Hou, Kangcheng and Burch, Kathryn S. and Lapinska, Sandra and Privé, Florian and Vilhjálmsson, Bjarni and Sankararaman, Sriram and Pasaniuc, Bogdan},
  doi          = {10.1038/s41588-021-00961-5},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {30-39},
  shortjournal = {Nat. Genet.},
  title        = {Large uncertainty in individual polygenic risk score estimation impacts PRS-based risk stratification},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetic variation influencing DNA methylation provides
insights into molecular mechanisms regulating genomic function.
<em>NG</em>, <em>54</em>(1), 18–29. (<a
href="https://doi.org/10.1038/s41588-021-00969-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We determined the relationships between DNA sequence variation and DNA methylation using blood samples from 3,799 Europeans and 3,195 South Asians. We identify 11,165,559 SNP–CpG associations (methylation quantitative trait loci (meQTL), P &amp;lt; 10−14), including 467,915 meQTL that operate in trans. The meQTL are enriched for functionally relevant characteristics, including shared chromatin state, High-throuhgput chromosome conformation interaction, and association with gene expression, metabolic variation and clinical traits. We use molecular interaction and colocalization analyses to identify multiple nuclear regulatory pathways linking meQTL loci to phenotypic variation, including UBASH3B (body mass index), NFKBIE (rheumatoid arthritis), MGA (blood pressure) and COMMD7 (white cell counts). For rs6511961 , chromatin immunoprecipitation followed by sequencing (ChIP–seq) validates zinc finger protein (ZNF)333 as the likely trans acting effector protein. Finally, we used interaction analyses to identify population- and lineage-specific meQTL, including rs174548 in FADS1, with the strongest effect in CD8+ T cells, thus linking fatty acid metabolism with immune dysregulation and asthma. Our study advances understanding of the potential pathways linking genetic variation to human phenotype. Genome-wide association analyses of DNA methylation in peripheral blood from 3,799 Europeans and 3,195 South Asians identify unique SNP–CpG associations (meQTL), providing insights into molecular mechanisms and the potential links to phenotypic variation.},
  archive      = {J_NG},
  author       = {Hawe, Johann S. and Wilson, Rory and Schmid, Katharina T. and Zhou, Li and Lakshmanan, Lakshmi Narayanan and Lehne, Benjamin C. and Kühnel, Brigitte and Scott, William R. and Wielscher, Matthias and Yew, Yik Weng and Baumbach, Clemens and Lee, Dominic P. and Marouli, Eirini and Bernard, Manon and Pfeiffer, Liliane and Matías-García, Pamela R. and Autio, Matias I. and Bourgeois, Stephane and Herder, Christian and Karhunen, Ville and Meitinger, Thomas and Prokisch, Holger and Rathmann, Wolfgang and Roden, Michael and Sebert, Sylvain and Shin, Jean and Strauch, Konstantin and Zhang, Weihua and Tan, Wilson L. W. and Hauck, Stefanie M. and Merl-Pham, Juliane and Grallert, Harald and Barbosa, Eudes G. V. and Illig, Thomas and Peters, Annette and Paus, Tomas and Pausova, Zdenka and Deloukas, Panos and Foo, Roger S. Y. and Jarvelin, Marjo-Riitta and Kooner, Jaspal S. and Loh, Marie and Heinig, Matthias and Gieger, Christian and Waldenberger, Melanie and Chambers, John C.},
  doi          = {10.1038/s41588-021-00969-x},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {18-29},
  shortjournal = {Nat. Genet.},
  title        = {Genetic variation influencing DNA methylation provides insights into molecular mechanisms regulating genomic function},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Genetic analysis of the human microglial transcriptome
across brain regions, aging and disease pathologies. <em>NG</em>,
<em>54</em>(1), 4–17. (<a
href="https://doi.org/10.1038/s41588-021-00976-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microglia have emerged as important players in brain aging and pathology. To understand how genetic risk for neurological and psychiatric disorders is related to microglial function, large transcriptome studies are essential. Here we describe the transcriptome analysis of 255 primary human microglial samples isolated at autopsy from multiple brain regions of 100 individuals. We performed systematic analyses to investigate various aspects of microglial heterogeneities, including brain region and aging. We mapped expression and splicing quantitative trait loci and showed that many neurological disease susceptibility loci are mediated through gene expression or splicing in microglia. Fine-mapping of these loci nominated candidate causal variants that are within microglia-specific enhancers, finding associations with microglial expression of USP6NL for Alzheimer’s disease and P2RY12 for Parkinson’s disease. We have built the most comprehensive catalog to date of genetic effects on the microglial transcriptome and propose candidate functional variants in neurological and psychiatric disorders. Transcriptomic analyses of 255 primary human microglial samples from 100 individuals highlight brain region, age, sex and disease states as sources of microglial heterogeneity. Molecular quantitative trait locus analyses implicate variants involved in neurological diseases through effects on gene expression and splicing.},
  archive      = {J_NG},
  author       = {Lopes, Katia de Paiva and Snijders, Gijsje J. L. and Humphrey, Jack and Allan, Amanda and Sneeboer, Marjolein A. M. and Navarro, Elisa and Schilder, Brian M. and Vialle, Ricardo A. and Parks, Madison and Missall, Roy and van Zuiden, Welmoed and Gigase, Frederieke A. J. and Kübler, Raphael and van Berlekom, Amber Berdenis and Hicks, Emily M. and Bӧttcher, Chotima and Priller, Josef and Kahn, René S. and de Witte, Lot D. and Raj, Towfique},
  doi          = {10.1038/s41588-021-00976-y},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {4-17},
  shortjournal = {Nat. Genet.},
  title        = {Genetic analysis of the human microglial transcriptome across brain regions, aging and disease pathologies},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The regulatory network architecture of cardiometabolic
diseases. <em>NG</em>, <em>54</em>(1), 2–3. (<a
href="https://doi.org/10.1038/s41588-021-00994-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complex disease definitions often represent descriptive umbrella terms of symptoms rather than mechanistic entities. A new study shows how network-based approaches can help identify the mechanisms that link genes, cells, tissues and organs in cardiovascular diseases.},
  archive      = {J_NG},
  author       = {Schmidt, Harald H. H. W. and Menche, Jörg},
  doi          = {10.1038/s41588-021-00994-w},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {2-3},
  shortjournal = {Nat. Genet.},
  title        = {The regulatory network architecture of cardiometabolic diseases},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A very mendelian year. <em>NG</em>, <em>54</em>(1), 1. (<a
href="https://doi.org/10.1038/s41588-021-01002-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We kick off 2022 by noting that the 200th anniversary of Gregor Mendel’s birth will be on 20 July 2022. We look forward to celebrating this milestone and reflecting on how far the genetics field has developed in 200 years. Here are some of the things that we are most excited about for 2022.},
  archive      = {J_NG},
  doi          = {10.1038/s41588-021-01002-x},
  journal      = {Nature Genetics},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Genet.},
  title        = {A very mendelian year},
  volume       = {54},
  year         = {2022},
}
</textarea>
</details></li>
</ul>

</body>
</html>
