<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATCS_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natcs---197">NATCS - 197</h2>
<ul>
<li><details>
<summary>
(2023). Spatial redundancy transformer for self-supervised
fluorescence image denoising. <em>NATCS</em>, <em>3</em>(12), 1067–1080.
(<a href="https://doi.org/10.1038/s43588-023-00568-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fluorescence imaging with high signal-to-noise ratios has become the foundation of accurate visualization and analysis of biological phenomena. However, the inevitable noise poses a formidable challenge to imaging sensitivity. Here we provide the spatial redundancy denoising transformer (SRDTrans) to remove noise from fluorescence images in a self-supervised manner. First, a sampling strategy based on spatial redundancy is proposed to extract adjacent orthogonal training pairs, which eliminates the dependence on high imaging speed. Second, we designed a lightweight spatiotemporal transformer architecture to capture long-range dependencies and high-resolution features at low computational cost. SRDTrans can restore high-frequency information without producing oversmoothed structures and distorted fluorescence traces. Finally, we demonstrate the state-of-the-art denoising performance of SRDTrans on single-molecule localization microscopy and two-photon volumetric calcium imaging. SRDTrans does not contain any assumptions about the imaging process and the sample, thus can be easily extended to various imaging modalities and biological applications. SRDTrans is a self-supervised denoising method for fluorescence images powered by spatial redundancy sampling and a dedicated transformer network that achieves good performance on fast dynamics and various imaging modalities.},
  archive      = {J_NATCS},
  author       = {Li, Xinyang and Hu, Xiaowan and Chen, Xingye and Fan, Jiaqi and Zhao, Zhifeng and Wu, Jiamin and Wang, Haoqian and Dai, Qionghai},
  doi          = {10.1038/s43588-023-00568-2},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1067-1080},
  shortjournal = {Nat. Comput. Sci},
  title        = {Spatial redundancy transformer for self-supervised fluorescence image denoising},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Interpretable neural architecture search and transfer
learning for understanding CRISPR–Cas9 off-target enzymatic reactions.
<em>NATCS</em>, <em>3</em>(12), 1056–1066. (<a
href="https://doi.org/10.1038/s43588-023-00569-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Finely tuned enzymatic pathways control cellular processes, and their dysregulation can lead to disease. Developing predictive and interpretable models for these pathways is challenging because of the complexity of the pathways and of the cellular and genomic contexts. Here we introduce Elektrum, a deep learning framework that addresses these challenges with data-driven and biophysically interpretable models for determining the kinetics of biochemical systems. First, it uses in vitro kinetic assays to rapidly hypothesize an ensemble of high-quality kinetically interpretable neural networks (KINNs) that predict reaction rates. It then employs a transfer learning step, where the KINNs are inserted as intermediary layers into deeper convolutional neural networks, fine-tuning the predictions for reaction-dependent in vivo outcomes. We apply Elektrum to predict CRISPR–Cas9 off-target editing probabilities and demonstrate that Elektrum achieves improved performance, regularizes neural network architectures and maintains physical interpretability. Developing predictive mechanistic models in biology is challenging. Elektrum uses neural architecture search, kinetic models and transfer learning to discover CRISPR–Cas9 cleavage kinetics, achieving high performance and biophysical interpretability.},
  archive      = {J_NATCS},
  author       = {Zhang, Zijun and Lamson, Adam R. and Shelley, Michael and Troyanskaya, Olga},
  doi          = {10.1038/s43588-023-00569-1},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1056-1066},
  shortjournal = {Nat. Comput. Sci},
  title        = {Interpretable neural architecture search and transfer learning for understanding CRISPR–Cas9 off-target enzymatic reactions},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Accurate transition state generation with an object-aware
equivariant elementary reaction diffusion model. <em>NATCS</em>,
<em>3</em>(12), 1045–1055. (<a
href="https://doi.org/10.1038/s43588-023-00563-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transition state search is key in chemistry for elucidating reaction mechanisms and exploring reaction networks. The search for accurate 3D transition state structures, however, requires numerous computationally intensive quantum chemistry calculations due to the complexity of potential energy surfaces. Here we developed an object-aware SE(3) equivariant diffusion model that satisfies all physical symmetries and constraints for generating sets of structures—reactant, transition state and product—in an elementary reaction. Provided reactant and product, this model generates a transition state structure in seconds instead of hours, which is typically required when performing quantum-chemistry-based optimizations. The generated transition state structures achieve a median of 0.08 Å root mean square deviation compared to the true transition state. With a confidence scoring model for uncertainty quantification, we approach an accuracy required for reaction barrier estimation (2.6 kcal mol–1) by only performing quantum chemistry-based optimizations on 14\% of the most challenging reactions. We envision usefulness for our approach in constructing large reaction networks with unknown mechanisms. A diffusion model that generates chemical reactions in 3D with all desired symmetries preserved is established and shown to reduce transition state search from days to seconds and complement intuition-based reaction exploration with generative AI.},
  archive      = {J_NATCS},
  author       = {Duan, Chenru and Du, Yuanqi and Jia, Haojun and Kulik, Heather J.},
  doi          = {10.1038/s43588-023-00563-7},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1045-1055},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accurate transition state generation with an object-aware equivariant elementary reaction diffusion model},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Machine-learning-accelerated simulations to enable automatic
surface reconstruction. <em>NATCS</em>, <em>3</em>(12), 1034–1044. (<a
href="https://doi.org/10.1038/s43588-023-00571-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding material surfaces and interfaces is vital in applications such as catalysis or electronics. By combining energies from electronic structure with statistical mechanics, ab initio simulations can, in principle, predict the structure of material surfaces as a function of thermodynamic variables. However, accurate energy simulations are prohibitive when coupled to the vast phase space that must be statistically sampled. Here we present a bi-faceted computational loop to predict surface phase diagrams of multicomponent materials that accelerates both the energy scoring and statistical sampling methods. Fast, scalable and data-efficient machine learning interatomic potentials are trained on high-throughput density-functional-theory calculations through closed-loop active learning. Markov chain Monte Carlo sampling in the semigrand canonical ensemble is enabled by using virtual surface sites. The predicted surfaces for GaN(0001), Si(111) and SrTiO3(001) are in agreement with past work and indicate that the proposed strategy can model complex material surfaces and discover previously unreported surface terminations. VSSR-MC is a Markov chain method based on virtual adsorption sites that interfaces with a neural network force field to provide fast, accurate and comprehensive sampling of material surfaces.},
  archive      = {J_NATCS},
  author       = {Du, Xiaochen and Damewood, James K. and Lunger, Jaclyn R. and Millan, Reisel and Yildiz, Bilge and Li, Lin and Gómez-Bombarelli, Rafael},
  doi          = {10.1038/s43588-023-00571-7},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1034-1044},
  shortjournal = {Nat. Comput. Sci},
  title        = {Machine-learning-accelerated simulations to enable automatic surface reconstruction},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Emerging drug interaction prediction enabled by a flow-based
graph neural network with biomedical network. <em>NATCS</em>,
<em>3</em>(12), 1023–1033. (<a
href="https://doi.org/10.1038/s43588-023-00558-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug–drug interactions (DDIs) for emerging drugs offer possibilities for treating and alleviating diseases, and accurately predicting these with computational methods can improve patient care and contribute to efficient drug development. However, many existing computational methods require large amounts of known DDI information, which is scarce for emerging drugs. Here we propose EmerGNN, a graph neural network that can effectively predict interactions for emerging drugs by leveraging the rich information in biomedical networks. EmerGNN learns pairwise representations of drugs by extracting the paths between drug pairs, propagating information from one drug to the other, and incorporating the relevant biomedical concepts on the paths. The edges of the biomedical network are weighted to indicate the relevance for the target DDI prediction. Overall, EmerGNN has higher accuracy than existing approaches in predicting interactions for emerging drugs and can identify the most relevant information on the biomedical network. EmerGNN, a method to predict interactions for emerging drugs, may improve patient care and drug development by providing insight into the effects of using biomedical networks in interaction predictions.},
  archive      = {J_NATCS},
  author       = {Zhang, Yongqi and Yao, Quanming and Yue, Ling and Wu, Xian and Zhang, Ziheng and Lin, Zhenxi and Zheng, Yefeng},
  doi          = {10.1038/s43588-023-00558-4},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1023-1033},
  shortjournal = {Nat. Comput. Sci},
  title        = {Emerging drug interaction prediction enabled by a flow-based graph neural network with biomedical network},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Deep contrastive learning of molecular conformation for
efficient property prediction. <em>NATCS</em>, <em>3</em>(12),
1015–1022. (<a
href="https://doi.org/10.1038/s43588-023-00560-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data-driven deep learning algorithms provide accurate prediction of high-level quantum-chemical molecular properties. However, their inputs must be constrained to the same quantum-chemical level of geometric relaxation as the training dataset, limiting their flexibility. Adopting alternative cost-effective conformation generative methods introduces domain-shift problems, deteriorating prediction accuracy. Here we propose a deep contrastive learning-based domain-adaptation method called Local Atomic environment Contrastive Learning (LACL). LACL learns to alleviate the disparities in distribution between the two geometric conformations by comparing different conformation-generation methods. We found that LACL forms a domain-agnostic latent space that encapsulates the semantics of an atom’s local atomic environment. LACL achieves quantum-chemical accuracy while circumventing the geometric relaxation bottleneck and could enable future application scenarios such as inverse molecular engineering and large-scale screening. Our approach is also generalizable from small organic molecules to long chains of biological and pharmacological molecules. A graph-based contrastive learning framework, LACL, is proposed for geometric domain-agnostic prediction of molecular properties to alleviate the need for molecular geometry relaxation, enabling large-scale inference scenarios.},
  archive      = {J_NATCS},
  author       = {Park, Yang Jeong and Kim, HyunGi and Jo, Jeonghee and Yoon, Sungroh},
  doi          = {10.1038/s43588-023-00560-w},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1015-1022},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep contrastive learning of molecular conformation for efficient property prediction},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Moving towards a generalized denoising network for
microscopy. <em>NATCS</em>, <em>3</em>(12), 1013–1014. (<a
href="https://doi.org/10.1038/s43588-023-00574-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The visualization and analysis of biological events using fluorescence microscopy is limited by the noise inherent in the images obtained. Now, a self-supervised spatial redundancy denoising transformer is proposed to address this challenge.},
  archive      = {J_NATCS},
  author       = {Whitehead, Lachlan},
  doi          = {10.1038/s43588-023-00574-4},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1013-1014},
  shortjournal = {Nat. Comput. Sci},
  title        = {Moving towards a generalized denoising network for microscopy},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Interpretable model of CRISPR–Cas9 enzymatic reactions.
<em>NATCS</em>, <em>3</em>(12), 1011–1012. (<a
href="https://doi.org/10.1038/s43588-023-00570-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enzymatic pathways control a host of cellular processes, but the complexity of such pathways has made them difficult to predict. Elektrum combines neural architecture search, kinetic models and transfer learning to effectively discover CRISPR–Cas9 cleavage kinetics.},
  archive      = {J_NATCS},
  author       = {Wen, David J. and Theodoris, Christina V.},
  doi          = {10.1038/s43588-023-00570-8},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1011-1012},
  shortjournal = {Nat. Comput. Sci},
  title        = {Interpretable model of CRISPR–Cas9 enzymatic reactions},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Machine learning speeds up search for surface structure.
<em>NATCS</em>, <em>3</em>(12), 1009–1010. (<a
href="https://doi.org/10.1038/s43588-023-00575-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It is difficult to identify stable surface reconstructions of complex materials. Now a Monte Carlo sampling strategy is coupled with a machine learning interatomic potential that is iteratively improved via active learning during the search.},
  archive      = {J_NATCS},
  author       = {Andersen, Mie},
  doi          = {10.1038/s43588-023-00575-3},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1009-1010},
  shortjournal = {Nat. Comput. Sci},
  title        = {Machine learning speeds up search for surface structure},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Predicting emerging drug interactions using GNNs.
<em>NATCS</em>, <em>3</em>(12), 1007–1008. (<a
href="https://doi.org/10.1038/s43588-023-00555-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {EmerGNN is a flow-based graph neural network (GNN) approach that advances on conventional methodologies for predicting drug–drug interactions in emerging drugs by effectively leveraging biomedical networks.},
  archive      = {J_NATCS},
  author       = {Le, Nguyen Quoc Khanh},
  doi          = {10.1038/s43588-023-00555-7},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1007-1008},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting emerging drug interactions using GNNs},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Drawing statistically valid conclusions with ML.
<em>NATCS</em>, <em>3</em>(12), 1006. (<a
href="https://doi.org/10.1038/s43588-023-00577-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00577-1},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1006},
  shortjournal = {Nat. Comput. Sci},
  title        = {Drawing statistically valid conclusions with ML},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023k). One algorithm to play them all. <em>NATCS</em>,
<em>3</em>(12), 1005. (<a
href="https://doi.org/10.1038/s43588-023-00582-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00582-4},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1005},
  shortjournal = {Nat. Comput. Sci},
  title        = {One algorithm to play them all},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023k). Shedding light on GNN affinity predictions. <em>NATCS</em>,
<em>3</em>(12), 1004. (<a
href="https://doi.org/10.1038/s43588-023-00583-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00583-3},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1004},
  shortjournal = {Nat. Comput. Sci},
  title        = {Shedding light on GNN affinity predictions},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Towards programmed protein generation. <em>NATCS</em>,
<em>3</em>(12), 1003. (<a
href="https://doi.org/10.1038/s43588-023-00579-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-023-00579-z},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1003},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards programmed protein generation},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). From inception to current challenges in bioinformatics.
<em>NATCS</em>, <em>3</em>(12), 1001–1002. (<a
href="https://doi.org/10.1038/s43588-023-00567-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Paulien Hogeweg — professor of bioinformatics at Utrecht University, who in the 1970s, together with Ben Hesper, coined the term ‘bioinformatics’ — talks to Nature Computational Science about her work on the Cellular Potts model, the integration of spatial information in modeling approaches, and her ongoing research on multilevel evolution.},
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00567-3},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {1001-1002},
  shortjournal = {Nat. Comput. Sci},
  title        = {From inception to current challenges in bioinformatics},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Recent advances and outstanding challenges for machine
learning interatomic potentials. <em>NATCS</em>, <em>3</em>(12),
998–1000. (<a href="https://doi.org/10.1038/s43588-023-00561-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning interatomic potentials (MLIPs) enable materials simulations at extended length and time scales with near-ab initio accuracy. They have broad applications in the study and design of materials. Here, we discuss recent advances, challenges, and the outlook for MLIPs.},
  archive      = {J_NATCS},
  author       = {Ko, Tsz Wai and Ong, Shyue Ping},
  doi          = {10.1038/s43588-023-00561-9},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {998-1000},
  shortjournal = {Nat. Comput. Sci},
  title        = {Recent advances and outstanding challenges for machine learning interatomic potentials},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fewer false positives in electrocatalytic nitrogen reduction
by synergizing theory and experiment. <em>NATCS</em>, <em>3</em>(12),
994–997. (<a href="https://doi.org/10.1038/s43588-023-00556-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The electrocatalytic nitrogen reduction reaction is a promising alternative to the Haber–Bosch process. However, the reproducibility and reliability of this process suffer from the persistence of false positives. Computational tools have the potential to alleviate this issue but several challenges must be addressed.},
  archive      = {J_NATCS},
  author       = {Xu, Gaomou and Wang, Tao},
  doi          = {10.1038/s43588-023-00556-6},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {994-997},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fewer false positives in electrocatalytic nitrogen reduction by synergizing theory and experiment},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Formats for reporting primary research. <em>NATCS</em>,
<em>3</em>(12), 993. (<a
href="https://doi.org/10.1038/s43588-023-00588-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We describe the different formats for reporting primary research for which we currently provide support.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00588-y},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {993},
  shortjournal = {Nat. Comput. Sci},
  title        = {Formats for reporting primary research},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Identification and analysis of 3D pores in packed
particulate materials. <em>NATCS</em>, <em>3</em>(11), 975–992. (<a
href="https://doi.org/10.1038/s43588-023-00551-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We took the classic ‘guess the number of beans in a jar game’ and amplified the research question. Rather than estimate the quantity of particles in the jar, we sought to characterize the spaces between them. Here we present an approach for delineating the pockets of empty space (three-dimensional pores) between packed particles, which are hotspots for activity in applications and natural phenomena that deal with particulate materials. We utilize techniques from graph theory to exploit information about particle configuration that allows us to locate important spatial landmarks within the void space. These landmarks are the basis for our pore segmentation, where we consider both interior pores as well as entrance and exit pores into and out of the structure. Our method is robust for particles of varying size, form, stiffness and configuration, which allows us to study and compare three-dimensional pores across a range of packed particle types. We report striking relationships between particles and pores that are described mathematically, and we offer a visual library of pore types. With a meaningful discretization of void space, we demonstrate that packed particles can be understood not by their solid space, but by their empty space. LOVAMAP is an analysis software that accurately identifies 3D pores in packed particle systems by exploiting information about the particle configuration as a basis for segmentation. Using the software, the authors were able to uncover striking relationships between particle and pore properties.},
  archive      = {J_NATCS},
  author       = {Riley, Lindsay and Cheng, Peter and Segura, Tatiana},
  doi          = {10.1038/s43588-023-00551-x},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {975-992},
  shortjournal = {Nat. Comput. Sci},
  title        = {Identification and analysis of 3D pores in packed particulate materials},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Using machine learning to go beyond potential energy surface
benchmarking for chemical reactivity. <em>NATCS</em>, <em>3</em>(11),
965–974. (<a href="https://doi.org/10.1038/s43588-023-00549-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We train an equivariant machine learning (ML) model to predict energies and forces for hydrogen combustion under conditions of finite temperature and pressure. This challenging case for reactive chemistry illustrates that ML potential energy surfaces are difficult to make complete, due to overreliance on chemical intuition of what data are important for training. Instead, a ‘negative design’ data acquisition strategy using metadynamics as part of an active learning workflow helps to create a ML model that avoids unforeseen high-energy or unphysical energy configurations. This strategy more rapidly converges the potential energy surfaces such that it is now more efficient to make calls to the external ab initio source when query-by-committee models disagree to further molecular dynamics in time without need for ML retraining. With the hybrid ML–physics model we realize two orders of magnitude reduction in cost, allowing for prediction of the free-energy change in the transition-state mechanism for several hydrogen combustion reaction channels. A hybrid machine learning–physics model is developed that reduces simulation cost by two orders of magnitude while retaining high ab initio accuracy, to predict free-energy transition states for hydrogen combustion reactions.},
  archive      = {J_NATCS},
  author       = {Guan, Xingyi and Heindel, Joseph P. and Ko, Taehee and Yang, Chao and Head-Gordon, Teresa},
  doi          = {10.1038/s43588-023-00549-5},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {965-974},
  shortjournal = {Nat. Comput. Sci},
  title        = {Using machine learning to go beyond potential energy surface benchmarking for chemical reactivity},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A deep learning model for predicting selected organic
molecular spectra. <em>NATCS</em>, <em>3</em>(11), 957–964. (<a
href="https://doi.org/10.1038/s43588-023-00550-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate and efficient molecular spectra simulations are crucial for substance discovery and structure identification. However, the conventional approach of relying on the quantum chemistry is cost intensive, which hampers efficiency. Here we develop DetaNet, a deep-learning model combining E(3)-equivariance group and self-attention mechanism to predict molecular spectra with improved efficiency and accuracy. By passing high-order geometric tensorial messages, DetaNet is able to generate a wide variety of molecular properties, including scalars, vectors, and second- and third-order tensors—all at the accuracy of quantum chemistry calculations. Based on this we developed generalized modules to predict four important types of molecular spectra, namely infrared, Raman, ultraviolet–visible, and 1H and 13C nuclear magnetic resonance, taking the QM9S dataset containing 130,000 molecular species as an example. By speeding up the prediction of molecular spectra at quantum chemical accuracy, DetaNet could help progress toward real-time structural identification using spectroscopic measurements. A deep-learning model, DetaNet, is proposed to efficiently and precisely predict molecular scalars, vectorial and tensorial properties, as well as the infrared, Raman, ultraviolet–visible and nuclear magnetic resonance spectra.},
  archive      = {J_NATCS},
  author       = {Zou, Zihan and Zhang, Yujin and Liang, Lijun and Wei, Mingzhi and Leng, Jiancai and Jiang, Jun and Luo, Yi and Hu, Wei},
  doi          = {10.1038/s43588-023-00550-y},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {957-964},
  shortjournal = {Nat. Comput. Sci},
  title        = {A deep learning model for predicting selected organic molecular spectra},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Predictive analyses of regulatory sequences with EUGENe.
<em>NATCS</em>, <em>3</em>(11), 946–956. (<a
href="https://doi.org/10.1038/s43588-023-00544-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning has become a popular tool to study cis-regulatory function. Yet efforts to design software for deep-learning analyses in regulatory genomics that are findable, accessible, interoperable and reusable (FAIR) have fallen short of fully meeting these criteria. Here we present elucidating the utility of genomic elements with neural nets (EUGENe), a FAIR toolkit for the analysis of genomic sequences with deep learning. EUGENe consists of a set of modules and subpackages for executing the key functionality of a genomics deep learning workflow: (1) extracting, transforming and loading sequence data from many common file formats; (2) instantiating, initializing and training diverse model architectures; and (3) evaluating and interpreting model behavior. We designed EUGENe as a simple, flexible and extensible interface for streamlining and customizing end-to-end deep-learning sequence analyses, and illustrate these principles through application of the toolkit to three predictive modeling tasks. We hope that EUGENe represents a springboard towards a collaborative ecosystem for deep-learning applications in genomics research. Designing accessible, interoperable and reusable software for applying deep learning to the study of gene regulation has been a challenge in genomics research. EUGENe is a toolkit that addresses this gap and streamlines end-to-end analyses.},
  archive      = {J_NATCS},
  author       = {Klie, Adam and Laub, David and Talwar, James V. and Stites, Hayden and Jores, Tobias and Solvason, Joe J. and Farley, Emma K. and Carter, Hannah},
  doi          = {10.1038/s43588-023-00544-w},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {946-956},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predictive analyses of regulatory sequences with EUGENe},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Accelerating the prediction of stable materials with machine
learning. <em>NATCS</em>, <em>3</em>(11), 934–945. (<a
href="https://doi.org/10.1038/s43588-023-00536-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite the rise in computing power, the large space of possible combinations of elements and crystal structure types makes large-scale high-throughput surveys of stable materials prohibitively expensive, especially for complex materials and materials subject to environmental conditions such as finite temperature. When physics-based computational methods and labor-intensive experiments are not feasible, machine learning (ML) methods can be a rapid and powerful alternative. Owing to a wealth of experimental and first-principles data as well as improved ML frameworks designed for materials modeling, ML is shown to be effective in predicting stability parameters and accelerating the discovery of new stable materials. In this Review, we summarize the most recent advancements in applying ML methodologies in predicting materials stability, focusing particularly on predictions of zero- and finite-temperature stability. We also highlight the need for more ML development in predictions of other thermodynamic knobs, such as pressure and surface/interfacial energy, which practically impact materials stability. The capability of predicting stable materials is important to further accelerate the discovery of novel materials. In this Review, the authors discuss recent developments in machine learning techniques for assessing the stability of materials and highlight the opportunities in further advancing the field.},
  archive      = {J_NATCS},
  author       = {Griesemer, Sean D. and Xia, Yi and Wolverton, Chris},
  doi          = {10.1038/s43588-023-00536-w},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {934-945},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerating the prediction of stable materials with machine learning},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Designing molecules with autoencoder networks.
<em>NATCS</em>, <em>3</em>(11), 922–933. (<a
href="https://doi.org/10.1038/s43588-023-00548-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Autoencoders are versatile tools in molecular informatics. These unsupervised neural networks serve diverse tasks such as data-driven molecular representation and constructive molecular design. This Review explores their algorithmic foundations and applications in drug discovery, highlighting the most active areas of development and the contributions autoencoder networks have made in advancing this field. We also explore the challenges and prospects concerning the utilization of autoencoders and the various adaptations of this neural network architecture in molecular design. Autoencoders are versatile tools for molecular informatics with the opportunity for advancing molecule and drug design. In this Review, the authors highlight the active areas of development in the field and explore the challenges that need to be addressed moving forward.},
  archive      = {J_NATCS},
  author       = {Ilnicka, Agnieszka and Schneider, Gisbert},
  doi          = {10.1038/s43588-023-00548-6},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {922-933},
  shortjournal = {Nat. Comput. Sci},
  title        = {Designing molecules with autoencoder networks},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A software framework for end-to-end genomic sequence
analysis with deep learning. <em>NATCS</em>, <em>3</em>(11), 920–921.
(<a href="https://doi.org/10.1038/s43588-023-00557-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using deep learning methods to study gene regulation has become popular, but designing accessible and customizable software for this purpose remains a challenge. This work introduces a computational toolkit called EUGENe that facilitates the development of end-to-end deep learning workflows in regulatory genomics.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00557-5},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {920-921},
  shortjournal = {Nat. Comput. Sci},
  title        = {A software framework for end-to-end genomic sequence analysis with deep learning},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Accurately predicting molecular spectra with deep learning.
<em>NATCS</em>, <em>3</em>(11), 918–919. (<a
href="https://doi.org/10.1038/s43588-023-00553-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The accurate prediction of molecular spectra is essential for substance discovery and structure identification, but conventional quantum chemistry methods are computationally expensive. Now, DetaNet achieves the accuracy of quantum chemistry while improving the efficiency of prediction of organic molecular spectra.},
  archive      = {J_NATCS},
  author       = {Tetsassi Feugmo, Conrard Giresse},
  doi          = {10.1038/s43588-023-00553-9},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {918-919},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accurately predicting molecular spectra with deep learning},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Investigating the elegance of empty space. <em>NATCS</em>,
<em>3</em>(11), 916–917. (<a
href="https://doi.org/10.1038/s43588-023-00554-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study presents an approach for characterizing and quantifying the pore space in assemblies of particles, enabling research into pore-scale flow physics and insight into the interplay between the solid and void phases in granular materials.},
  archive      = {J_NATCS},
  author       = {Evans, T. Matthew},
  doi          = {10.1038/s43588-023-00554-8},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {916-917},
  shortjournal = {Nat. Comput. Sci},
  title        = {Investigating the elegance of empty space},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023j). Predicting electronic structure calculation results.
<em>NATCS</em>, <em>3</em>(11), 915. (<a
href="https://doi.org/10.1038/s43588-023-00562-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00562-8},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {915},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting electronic structure calculation results},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Impact of spillovers on technological changes.
<em>NATCS</em>, <em>3</em>(11), 914. (<a
href="https://doi.org/10.1038/s43588-023-00564-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00564-6},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {914},
  shortjournal = {Nat. Comput. Sci},
  title        = {Impact of spillovers on technological changes},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A full-stack platform for spiking deep learning.
<em>NATCS</em>, <em>3</em>(11), 913. (<a
href="https://doi.org/10.1038/s43588-023-00565-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-023-00565-5},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {913},
  shortjournal = {Nat. Comput. Sci},
  title        = {A full-stack platform for spiking deep learning},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). Memory and computation together at last. <em>NATCS</em>,
<em>3</em>(11), 912. (<a
href="https://doi.org/10.1038/s43588-023-00559-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00559-3},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {912},
  shortjournal = {Nat. Comput. Sci},
  title        = {Memory and computation together at last},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Building open-source AI. <em>NATCS</em>, <em>3</em>(11),
908–911. (<a href="https://doi.org/10.1038/s43588-023-00540-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence (AI) drives innovation across society, economies and science. We argue for the importance of building AI technology according to open-source principles to foster accessibility, collaboration, responsibility and interoperability.},
  archive      = {J_NATCS},
  author       = {Shrestha, Yash Raj and von Krogh, Georg and Feuerriegel, Stefan},
  doi          = {10.1038/s43588-023-00540-0},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {908-911},
  shortjournal = {Nat. Comput. Sci},
  title        = {Building open-source AI},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Code sharing in the spotlight. <em>NATCS</em>,
<em>3</em>(11), 907. (<a
href="https://doi.org/10.1038/s43588-023-00566-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Year of Open Science has highlighted the importance of sharing the code associated with peer-reviewed manuscripts. We at Nature Computational Science provide support — via policies and implementations within our submission system — to facilitate this task.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00566-4},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {907},
  shortjournal = {Nat. Comput. Sci},
  title        = {Code sharing in the spotlight},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Integrating spatial transcriptomics data across different
conditions, technologies and developmental stages. <em>NATCS</em>,
<em>3</em>(10), 894–906. (<a
href="https://doi.org/10.1038/s43588-023-00528-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the rapid generation of spatial transcriptomics (ST) data, integrative analysis of multiple ST datasets from different conditions, technologies and developmental stages is becoming increasingly important. Here we present a graph attention neural network called STAligner for integrating and aligning ST datasets, enabling spatially aware data integration, simultaneous spatial domain identification and downstream comparative analysis. We apply STAligner to ST datasets of the human cortex slices from different samples, the mouse olfactory bulb slices generated by two profiling technologies, the mouse hippocampus tissue slices under normal and Alzheimer’s disease conditions, and the spatiotemporal atlases of mouse organogenesis. STAligner efficiently captures the shared tissue structures across different slices, the disease-related substructures and the dynamical changes during mouse embryonic development. In addition, the shared spatial domain and nearest-neighbor pairs identified by STAligner can be further considered as corresponding pairs to guide the three-dimensional reconstruction of consecutive slices, achieving more accurate local structure-guided registration than the existing method. A graph attention neural network tool is introduced to integrate multiple spatial transcriptomics data from different individuals, technologies and developmental stages, enabling consensus spatial domain identification and three-dimensional tissue reconstruction.},
  archive      = {J_NATCS},
  author       = {Zhou, Xiang and Dong, Kangning and Zhang, Shihua},
  doi          = {10.1038/s43588-023-00528-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {894-906},
  shortjournal = {Nat. Comput. Sci},
  title        = {Integrating spatial transcriptomics data across different conditions, technologies and developmental stages},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Unifying pairwise interactions in complex dynamics.
<em>NATCS</em>, <em>3</em>(10), 883–893. (<a
href="https://doi.org/10.1038/s43588-023-00519-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists have developed hundreds of techniques to measure the interactions between pairs of processes in complex systems, but these computational methods—from contemporaneous correlation coefficients to causal inference methods—define and formulate interactions differently, using distinct quantitative theories that remain largely disconnected. Here we introduce a large assembled library of 237 statistics of pairwise interactions, and assess their behavior on 1,053 multivariate time series from a wide range of real-world and model-generated systems. Our analysis highlights commonalities between disparate mathematical formulations of interactions, providing a unified picture of a rich interdisciplinary literature. Using three real-world case studies, we then show that simultaneously leveraging diverse methods can uncover those most suitable for addressing a given problem, facilitating interpretable understanding of the quantitative formulation of pairwise dependencies that drive successful performance. Our results and accompanying software enable comprehensive analysis of time-series interactions by drawing on decades of diverse methodological contributions. This work unifies an interdisciplinary literature of over 230 computational methods for measuring interactions from complex systems, revealing previously unreported theoretical connections and demonstrating practical benefits of broad methodological comparison.},
  archive      = {J_NATCS},
  author       = {Cliff, Oliver M. and Bryant, Annie G. and Lizier, Joseph T. and Tsuchiya, Naotsugu and Fulcher, Ben D.},
  doi          = {10.1038/s43588-023-00519-x},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {883-893},
  shortjournal = {Nat. Comput. Sci},
  title        = {Unifying pairwise interactions in complex dynamics},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Guided diffusion for inverse molecular design.
<em>NATCS</em>, <em>3</em>(10), 873–882. (<a
href="https://doi.org/10.1038/s43588-023-00532-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The holy grail of materials science is de novo molecular design, meaning engineering molecules with desired characteristics. The introduction of generative deep learning has greatly advanced efforts in this direction, yet molecular discovery remains challenging and often inefficient. Herein we introduce GaUDI, a guided diffusion model for inverse molecular design that combines an equivariant graph neural net for property prediction and a generative diffusion model. We demonstrate GaUDI’s effectiveness in designing molecules for organic electronic applications by using single- and multiple-objective tasks applied to a generated dataset of 475,000 polycyclic aromatic systems. GaUDI shows improved conditional design, generating molecules with optimal properties and even going beyond the original distribution to suggest better molecules than those in the dataset. In addition to point-wise targets, GaUDI can also be guided toward open-ended targets (for example, a minimum or maximum) and in all cases achieves close to 100\% validity of generated molecules. GaUDI is a guided diffusion method for the design of molecular structures that features a flexible and scalable target function and that achieves high validity of generated molecules.},
  archive      = {J_NATCS},
  author       = {Weiss, Tomer and Mayo Yanes, Eduardo and Chakraborty, Sabyasachi and Cosmo, Luca and Bronstein, Alex M. and Gershoni-Poranne, Renana},
  doi          = {10.1038/s43588-023-00532-0},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {873-882},
  shortjournal = {Nat. Comput. Sci},
  title        = {Guided diffusion for inverse molecular design},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Computing the relative binding affinity of ligands based on
a pairwise binding comparison network. <em>NATCS</em>, <em>3</em>(10),
860–872. (<a href="https://doi.org/10.1038/s43588-023-00529-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structure-based lead optimization is an open challenge in drug discovery, which is still largely driven by hypotheses and depends on the experience of medicinal chemists. Here we propose a pairwise binding comparison network (PBCNet) based on a physics-informed graph attention mechanism, specifically tailored for ranking the relative binding affinity among congeneric ligands. Benchmarking on two held-out sets (provided by Schrödinger and Merck) containing over 460 ligands and 16 targets, PBCNet demonstrated substantial advantages in terms of both prediction accuracy and computational efficiency. Equipped with a fine-tuning operation, the performance of PBCNet reaches that of Schrödinger’s FEP+, which is much more computationally intensive and requires substantial expert intervention. A further simulation-based experiment showed that active learning-optimized PBCNet may accelerate lead optimization campaigns by 473\%. Finally, for the convenience of users, a web service for PBCNet is established to facilitate complex relative binding affinity prediction through an easy-to-operate graphical interface. A physics-informed deep learning model, PBCNet, is proposed for predicting the relative binding affinity of ligands in order to improve guiding structure-based drug lead optimization.},
  archive      = {J_NATCS},
  author       = {Yu, Jie and Li, Zhaojun and Chen, Geng and Kong, Xiangtai and Hu, Jie and Wang, Dingyan and Cao, Duanhua and Li, Yanbei and Huo, Ruifeng and Wang, Gang and Liu, Xiaohong and Jiang, Hualiang and Li, Xutong and Luo, Xiaomin and Zheng, Mingyue},
  doi          = {10.1038/s43588-023-00529-9},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {860-872},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computing the relative binding affinity of ligands based on a pairwise binding comparison network},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Learning on topological surface and geometric structure for
3D molecular generation. <em>NATCS</em>, <em>3</em>(10), 849–859. (<a
href="https://doi.org/10.1038/s43588-023-00530-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Highly effective de novo design is a grand challenge of computer-aided drug discovery. Practical structure-specific three-dimensional molecule generations have started to emerge in recent years, but most approaches treat the target structure as a conditional input to bias the molecule generation and do not fully learn the detailed atomic interactions that govern the molecular conformation and stability of the binding complexes. The omission of these fine details leads to many models having difficulty in outputting reasonable molecules for a variety of therapeutic targets. Here, to address this challenge, we formulate a model, called SurfGen, that designs molecules in a fashion closely resembling the figurative key-and-lock principle. SurfGen comprises two equivariant neural networks, Geodesic-GNN and Geoatom-GNN, which capture the topological interactions on the pocket surface and the spatial interaction between ligand atoms and surface nodes, respectively. SurfGen outperforms other methods in a number of benchmarks, and its high sensitivity on the pocket structures enables an effective generative-model-based solution to the thorny issue of mutation-induced drug resistance. SurfGen is a structure-based drug design approach that delves into topological and geometric deep learning techniques for interaction learning, echoing the classical lock-and-key model.},
  archive      = {J_NATCS},
  author       = {Zhang, Odin and Wang, Tianyue and Weng, Gaoqi and Jiang, Dejun and Wang, Ning and Wang, Xiaorui and Zhao, Huifeng and Wu, Jialu and Wang, Ercheng and Chen, Guangyong and Deng, Yafeng and Pan, Peichen and Kang, Yu and Hsieh, Chang-Yu and Hou, Tingjun},
  doi          = {10.1038/s43588-023-00530-2},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {849-859},
  shortjournal = {Nat. Comput. Sci},
  title        = {Learning on topological surface and geometric structure for 3D molecular generation},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A universal programmable gaussian boson sampler for drug
discovery. <em>NATCS</em>, <em>3</em>(10), 839–848. (<a
href="https://doi.org/10.1038/s43588-023-00526-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gaussian boson sampling (GBS) has the potential to solve complex graph problems, such as clique finding, which is relevant to drug discovery tasks. However, realizing the full benefits of quantum enhancements requires large-scale quantum hardware with universal programmability. Here we have developed a time-bin-encoded GBS photonic quantum processor that is universal, programmable and software-scalable. Our processor features freely adjustable squeezing parameters and can implement arbitrary unitary operations with a programmable interferometer. Leveraging our processor, we successfully executed clique finding on a 32-node graph, achieving approximately twice the success probability compared to classical sampling. As proof of concept, we implemented a versatile quantum drug discovery platform using this GBS processor, enabling molecular docking and RNA-folding prediction tasks. Our work achieves GBS circuitry with its universal and programmable architecture, advancing GBS toward use in real-world applications. This study develops a programmable quantum processor, named Abacus, and applies it to Gaussian boson sampling tasks targeting drug discovery challenges.},
  archive      = {J_NATCS},
  author       = {Yu, Shang and Zhong, Zhi-Peng and Fang, Yuhua and Patel, Raj B. and Li, Qing-Peng and Liu, Wei and Li, Zhenghao and Xu, Liang and Sagona-Stophel, Steven and Mer, Ewan and Thomas, Sarah E. and Meng, Yu and Li, Zhi-Peng and Yang, Yuan-Ze and Wang, Zhao-An and Guo, Nai-Jie and Zhang, Wen-Hao and Tranmer, Geoffrey K. and Dong, Ying and Wang, Yi-Tao and Tang, Jian-Shun and Li, Chuan-Feng and Walmsley, Ian A. and Guo, Guang-Can},
  doi          = {10.1038/s43588-023-00526-y},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {839-848},
  shortjournal = {Nat. Comput. Sci},
  title        = {A universal programmable gaussian boson sampler for drug discovery},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Human-like intuitive behavior and reasoning biases emerged
in large language models but disappeared in ChatGPT. <em>NATCS</em>,
<em>3</em>(10), 833–838. (<a
href="https://doi.org/10.1038/s43588-023-00527-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We design a battery of semantic illusions and cognitive reflection tests, aimed to elicit intuitive yet erroneous responses. We administer these tasks, traditionally used to study reasoning and decision-making in humans, to OpenAI’s generative pre-trained transformer model family. The results show that as the models expand in size and linguistic proficiency they increasingly display human-like intuitive system 1 thinking and associated cognitive errors. This pattern shifts notably with the introduction of ChatGPT models, which tend to respond correctly, avoiding the traps embedded in the tasks. Both ChatGPT-3.5 and 4 utilize the input–output context window to engage in chain-of-thought reasoning, reminiscent of how people use notepads to support their system 2 thinking. Yet, they remain accurate even when prevented from engaging in chain-of-thought reasoning, indicating that their system-1-like next-word generation processes are more accurate than those of older models. Our findings highlight the value of applying psychological methodologies to study large language models, as this can uncover previously undetected emergent characteristics. The reasoning capabilities of OpenAI’s generative pre-trained transformer family were tested using semantic illusions and cognitive reflection tests that are typically used in human studies. While early models were prone to human-like cognitive errors, ChatGPT decisively outperformed humans, avoiding the cognitive traps embedded in the tasks.},
  archive      = {J_NATCS},
  author       = {Hagendorff, Thilo and Fabi, Sarah and Kosinski, Michal},
  doi          = {10.1038/s43588-023-00527-x},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {833-838},
  shortjournal = {Nat. Comput. Sci},
  title        = {Human-like intuitive behavior and reasoning biases emerged in large language models but disappeared in ChatGPT},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). STAligner enables the integration and alignment of multiple
spatial transcriptomics datasets. <em>NATCS</em>, <em>3</em>(10),
831–832. (<a href="https://doi.org/10.1038/s43588-023-00543-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce STAligner — a graph neural network-based tool for the integration of multiple spatial transcriptomics datasets by generating batch effect-corrected embeddings, thereby enabling consensus spatial domain identification and accurate 3D tissue reconstruction.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00543-x},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {831-832},
  shortjournal = {Nat. Comput. Sci},
  title        = {STAligner enables the integration and alignment of multiple spatial transcriptomics datasets},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Efficient prediction of relative ligand binding affinity in
drug discovery. <em>NATCS</em>, <em>3</em>(10), 829–830. (<a
href="https://doi.org/10.1038/s43588-023-00531-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A pairwise binding comparison network (PBCNet) has been established for predicting the relative binding affinity among congeneric ligands, using a physics-informed graph attention mechanism with a pair of protein pocket-ligand complexes as input. PBCNet shows practical value in guiding structure-based drug lead optimization with speed, precision, and ease-of-use.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00531-1},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {829-830},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficient prediction of relative ligand binding affinity in drug discovery},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). AI-powered structure-based drug design inspired by the
lock-and-key model. <em>NATCS</em>, <em>3</em>(10), 827–828. (<a
href="https://doi.org/10.1038/s43588-023-00552-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inspired by the classic lock-and-key model and advances in equivariant deep network design, we present a structure-based drug design model, SurfGen, which uses two types of equivariant graph neural networks to learn on protein surfaces and geometric structures to directly design small-molecule drugs.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00552-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {827-828},
  shortjournal = {Nat. Comput. Sci},
  title        = {AI-powered structure-based drug design inspired by the lock-and-key model},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Measuring three-dimensional organization on cell surfaces
with cellular harmonics. <em>NATCS</em>, <em>3</em>(10), 825–826. (<a
href="https://doi.org/10.1038/s43588-023-00517-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Different cells can have very different three-dimensional morphologies. We present the computational framework u-signal3D that calculates the spatial scales at which molecules are organized on the surfaces of heterogeneously shaped cells, enabling high-throughput analyses and subsequent machine learning applications.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00517-z},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {825-826},
  shortjournal = {Nat. Comput. Sci},
  title        = {Measuring three-dimensional organization on cell surfaces with cellular harmonics},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A consolidated framework for quantifying interaction
dynamics. <em>NATCS</em>, <em>3</em>(10), 823–824. (<a
href="https://doi.org/10.1038/s43588-023-00520-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposes a unified framework that can compare different measures for quantifying the statistics of pairwise interactions in data from complex dynamical systems.},
  archive      = {J_NATCS},
  author       = {Klein, Brennan},
  doi          = {10.1038/s43588-023-00520-4},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {823-824},
  shortjournal = {Nat. Comput. Sci},
  title        = {A consolidated framework for quantifying interaction dynamics},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Crafting molecular architectures with guided diffusion.
<em>NATCS</em>, <em>3</em>(10), 821–822. (<a
href="https://doi.org/10.1038/s43588-023-00533-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A guided diffusion model pushes the boundaries of de novo molecular design, extensively exploring the chemical space and generating chemical compounds that satisfy custom target criteria.},
  archive      = {J_NATCS},
  author       = {Gryn’ova, Ganna},
  doi          = {10.1038/s43588-023-00533-z},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {821-822},
  shortjournal = {Nat. Comput. Sci},
  title        = {Crafting molecular architectures with guided diffusion},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Programmability empowering quantum boson sampling.
<em>NATCS</em>, <em>3</em>(10), 819–820. (<a
href="https://doi.org/10.1038/s43588-023-00534-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Programmability is crucial in noisy intermediate-scale quantum computing, facilitating various functionalities for practical applications. An arbitrary programmable time-bin-encoded quantum boson sampling device has been developed, specifically tailored for potential drug discovery.},
  archive      = {J_NATCS},
  author       = {Fu, Zhaorong and Bao, Jueming and Wang, Jianwei},
  doi          = {10.1038/s43588-023-00534-y},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {819-820},
  shortjournal = {Nat. Comput. Sci},
  title        = {Programmability empowering quantum boson sampling},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Alerting for imminent earthquakes. <em>NATCS</em>,
<em>3</em>(10), 818. (<a
href="https://doi.org/10.1038/s43588-023-00545-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00545-9},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {818},
  shortjournal = {Nat. Comput. Sci},
  title        = {Alerting for imminent earthquakes},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023k). The waiting game. <em>NATCS</em>, <em>3</em>(10), 817. (<a
href="https://doi.org/10.1038/s43588-023-00547-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00547-7},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {817},
  shortjournal = {Nat. Comput. Sci},
  title        = {The waiting game},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Adding charge to neural network potentials. <em>NATCS</em>,
<em>3</em>(10), 816. (<a
href="https://doi.org/10.1038/s43588-023-00542-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-023-00542-y},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {816},
  shortjournal = {Nat. Comput. Sci},
  title        = {Adding charge to neural network potentials},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Drug discovery with limited resources. <em>NATCS</em>,
<em>3</em>(10), 815. (<a
href="https://doi.org/10.1038/s43588-023-00546-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00546-8},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {815},
  shortjournal = {Nat. Comput. Sci},
  title        = {Drug discovery with limited resources},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Laying the foundations of programming and system design.
<em>NATCS</em>, <em>3</em>(10), 813–814. (<a
href="https://doi.org/10.1038/s43588-023-00539-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Barbara Liskov — a mostly retired Institute Professor at the Massachusetts Institute of Technology, a pioneer in object-oriented programming and distributed systems and the winner of the 2008 ACM A. M. Turing Award, which is the highest distinction in computer science — talks to Nature Computational Science about her work on data abstractions, her career trajectory and recognizing the contributions of women in computer science.},
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00539-7},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {813-814},
  shortjournal = {Nat. Comput. Sci},
  title        = {Laying the foundations of programming and system design},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023j). Tackling environmental challenges, one molecule at a time.
<em>NATCS</em>, <em>3</em>(10), 810–812. (<a
href="https://doi.org/10.1038/s43588-023-00537-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Angela K. Wilson, director of the Michigan State University Center for Quantum Computing, Science and Engineering and John A. Hannah Distinguished Professor at Michigan State University, talks to Nature Computational Science about protein-based carbon-capture, the use of machine learning in computational chemistry, and making the research field more equitable for female researchers.},
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00537-9},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {810-812},
  shortjournal = {Nat. Comput. Sci},
  title        = {Tackling environmental challenges, one molecule at a time},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Moving towards better communication. <em>NATCS</em>,
<em>3</em>(10), 808–809. (<a
href="https://doi.org/10.1038/s43588-023-00538-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Diyi Yang, Assistant Professor of computer science at Stanford University, talks to Nature Computational Science about understanding human communication in a social context, building natural language processing systems that are human-centered, and the challenges that female researchers face in the field.},
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00538-8},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {808-809},
  shortjournal = {Nat. Comput. Sci},
  title        = {Moving towards better communication},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ada lovelace, a role model for the ages. <em>NATCS</em>,
<em>3</em>(10), 807. (<a
href="https://doi.org/10.1038/s43588-023-00541-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ada Lovelace Day celebrates women in STEM careers, but also raises awareness of the challenges that women have faced in science, as well as the importance of female role models in STEM.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00541-z},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {807},
  shortjournal = {Nat. Comput. Sci},
  title        = {Ada lovelace, a role model for the ages},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Mapping disease regulatory circuits at
cell-type resolution from single-cell multiomics data. <em>NATCS</em>,
<em>3</em>(9), 805. (<a
href="https://doi.org/10.1038/s43588-023-00523-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chen, Xi and Wang, Yuan and Cappuccio, Antonio and Cheng, Wan-Sze and Zamojski, Frederique Ruf and Nair, Venugopalan D. and Miller, Clare M. and Rubenstein, Aliza B. and Nudelman, German and Tadych, Alicja and Theesfeld, Chandra L. and Vornholt, Alexandria and George, Mary-Catherine and Ruffin, Felicia and Dagher, Michael and Chawla, Daniel G. and Soares-Schanoski, Alessandra and Spurbeck, Rachel R. and Ndhlovu, Lishomwa C. and Sebra, Robert and Kleinstein, Steven H. and Letizia, Andrew G. and Ramos, Irene and Fowler, Vance G. and Woods, Christopher W. and Zaslavsky, Elena and Troyanskaya, Olga G. and Sealfon, Stuart C.},
  doi          = {10.1038/s43588-023-00523-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {805},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Mapping disease regulatory circuits at cell-type resolution from single-cell multiomics data},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Efficient and accurate large library ligand docking with
KarmaDock. <em>NATCS</em>, <em>3</em>(9), 789–804. (<a
href="https://doi.org/10.1038/s43588-023-00511-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ligand docking is one of the core technologies in structure-based virtual screening for drug discovery. However, conventional docking tools and existing deep learning tools may suffer from limited performance in terms of speed, pose quality and binding affinity accuracy. Here we propose KarmaDock, a deep learning approach for ligand docking that integrates the functions of docking acceleration, binding pose generation and correction, and binding strength estimation. The three-stage model consists of the following components: (1) encoders for the protein and ligand to learn the representations of intramolecular interactions; (2) E(n) equivariant graph neural networks with self-attention to update the ligand pose based on both protein–ligand and intramolecular interactions, followed by post-processing to ensure chemically plausible structures; (3) a mixture density network for scoring the binding strength. KarmaDock was validated on four benchmark datasets and tested in a real-world virtual screening project that successfully identified experiment-validated active inhibitors of leukocyte tyrosine kinase (LTK). KarmaDock, a deep learning approach, is proposed to improve the speed, accuracy and pose quality of molecular docking and is validated on multiple datasets and a real-world virtual screening.},
  archive      = {J_NATCS},
  author       = {Zhang, Xujun and Zhang, Odin and Shen, Chao and Qu, Wanglin and Chen, Shicheng and Cao, Hanqun and Kang, Yu and Wang, Zhe and Wang, Ercheng and Zhang, Jintu and Deng, Yafeng and Liu, Furui and Wang, Tianyue and Du, Hongyan and Wang, Langcheng and Pan, Peichen and Chen, Guangyong and Hsieh, Chang-Yu and Hou, Tingjun},
  doi          = {10.1038/s43588-023-00511-5},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {789-804},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficient and accurate large library ligand docking with KarmaDock},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cellular harmonics for the morphology-invariant analysis of
molecular organization at the cell surface. <em>NATCS</em>,
<em>3</em>(9), 777–788. (<a
href="https://doi.org/10.1038/s43588-023-00512-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The spatiotemporal organization of membrane-associated molecules is central to the regulation of cellular signals. Powerful new microscopy techniques enable the three-dimensional visualization of localization and activation of these molecules; however, the quantitative interpretation and comparison of molecular organization on the three-dimensional cell surface remains challenging because cells themselves vary greatly in morphology. Here we introduce u-signal3D, a framework to assess the spatial scales of molecular organization at the cell surface in a cell-morphology-invariant manner. We validated the framework by analyzing synthetic signaling patterns painted onto observed cell morphologies, as well as measured distributions of cytoskeletal and signaling molecules. To demonstrate the framework’s versatility, we further compared the spatial organization of cell surface signals both within, and between, cell populations, and powered an upstream machine-learning-based analysis of signaling motifs. The computational platform u-signal3D defines a shape-invariant representation of the spatial scales of molecular organization at the cell surface to enable comparison and machine learning of signaling across morphologically diverse cell populations.},
  archive      = {J_NATCS},
  author       = {Mazloom-Farsibaf, Hanieh and Zou, Qiongjing and Hsieh, Rebecca and Danuser, Gaudenz and Driscoll, Meghan K.},
  doi          = {10.1038/s43588-023-00512-4},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {777-788},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cellular harmonics for the morphology-invariant analysis of molecular organization at the cell surface},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Strategy evolution on dynamic networks. <em>NATCS</em>,
<em>3</em>(9), 763–776. (<a
href="https://doi.org/10.1038/s43588-023-00509-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Models of strategy evolution on static networks help us understand how population structure can promote the spread of traits like cooperation. One key mechanism is the formation of altruistic spatial clusters, where neighbors of a cooperative individual are likely to reciprocate, which protects prosocial traits from exploitation. However, most real-world interactions are ephemeral and subject to exogenous restructuring, so that social networks change over time. Strategic behavior on dynamic networks is difficult to study, and much less is known about the resulting evolutionary dynamics. Here we provide an analytical treatment of cooperation on dynamic networks, allowing for arbitrary spatial and temporal heterogeneity. We show that transitions among a large class of network structures can favor the spread of cooperation, even if each individual social network would inhibit cooperation when static. Furthermore, we show that spatial heterogeneity tends to inhibit cooperation, whereas temporal heterogeneity tends to promote it. Dynamic networks can have profound effects on the evolution of prosocial traits, even when individuals have no agency over network structures. Real-world social networks are often ephemeral and subject to exogenous restructuring. Q. Su et al. show that dynamic networks can foster cooperative behavior.},
  archive      = {J_NATCS},
  author       = {Su, Qi and McAvoy, Alex and Plotkin, Joshua B.},
  doi          = {10.1038/s43588-023-00509-z},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {763-776},
  shortjournal = {Nat. Comput. Sci},
  title        = {Strategy evolution on dynamic networks},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatial planning of urban communities via deep reinforcement
learning. <em>NATCS</em>, <em>3</em>(9), 748–762. (<a
href="https://doi.org/10.1038/s43588-023-00503-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Effective spatial planning of urban communities plays a critical role in the sustainable development of cities. Despite the convenience brought by geographic information systems and computer-aided design, determining the layout of land use and roads still heavily relies on human experts. Here we propose an artificial intelligence urban-planning model to generate spatial plans for urban communities. To overcome the difficulty of diverse and irregular urban geography, we construct a graph to describe the topology of cities in arbitrary forms and formulate urban planning as a sequential decision-making problem on the graph. To tackle the challenge of the vast solution space, we develop a reinforcement learning model based on graph neural networks. Experiments on both synthetic and real-world communities demonstrate that our computational model outperforms plans designed by human experts in objective metrics and that it can generate spatial plans responding to different circumstances and needs. We also propose a human–artificial intelligence collaborative workflow of urban planning, in which human designers can substantially benefit from our model to be more productive, generating more efficient spatial plans with much less time. Our method demonstrates the great potential of computational urban planning and paves the way for more explorations in leveraging computational methodologies to solve challenging real-world problems in urban science. A graph-based artificial intelligence model for urban planning outperforms human-designed plans in objective metrics, offering an efficient and adaptable collaborative workflow for future sustainable cities.},
  archive      = {J_NATCS},
  author       = {Zheng, Yu and Lin, Yuming and Zhao, Liang and Wu, Tinghai and Jin, Depeng and Li, Yong},
  doi          = {10.1038/s43588-023-00503-5},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {748-762},
  shortjournal = {Nat. Comput. Sci},
  title        = {Spatial planning of urban communities via deep reinforcement learning},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Denoising sparse microbial signals from single-cell
sequencing of mammalian host tissues. <em>NATCS</em>, <em>3</em>(9),
741–747. (<a href="https://doi.org/10.1038/s43588-023-00507-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing genomic sequencing data can be used to study host–microbiome ecosystems; however, distinguishing signals that originate from truly present microbes from contaminating species and artifacts is a substantial and often prohibitive challenge. Here we show that emerging sequencing technologies definitely capture reads from present microbes. We developed SAHMI, a computational resource to identify truly present microbial nucleic acids, as well as filter contaminants and spurious false-positive taxonomic assignments from standard transcriptomic sequencing of mammalian tissues. In benchmark studies, SAHMI correctly identifies known microbial infections present in diverse tissues, and we validate SAHMI’s enrichment for correctly classified, truly present species using multiple orthogonal computational experiments. The application of SAHMI to single-cell and spatial genomic data thus enables co-detection of somatic cells and microorganisms and joint analysis of host–microbiome ecosystems. SAHMI performs in silico denoising of microbial signals from existing host genomic sequencing data to select for present microbes, as well as filter out contaminants and false-positive misclassifications.},
  archive      = {J_NATCS},
  author       = {Ghaddar, Bassel and Blaser, Martin J. and De, Subhajyoti},
  doi          = {10.1038/s43588-023-00507-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {741-747},
  shortjournal = {Nat. Comput. Sci},
  title        = {Denoising sparse microbial signals from single-cell sequencing of mammalian host tissues},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Progress in using deep learning to treat cancer.
<em>NATCS</em>, <em>3</em>(9), 739–740. (<a
href="https://doi.org/10.1038/s43588-023-00514-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning approaches have potential to substantially reduce the astronomical costs and long timescales involved in drug discovery. KarmaDock proposes a deep learning workflow for ligand docking that shows improved performance against both benchmark cases and in a real-world virtual screening experiment.},
  archive      = {J_NATCS},
  author       = {Kamerlin, Shina Caroline Lynn},
  doi          = {10.1038/s43588-023-00514-2},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {739-740},
  shortjournal = {Nat. Comput. Sci},
  title        = {Progress in using deep learning to treat cancer},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Flipping the intuition for games on dynamic networks.
<em>NATCS</em>, <em>3</em>(9), 737–738. (<a
href="https://doi.org/10.1038/s43588-023-00513-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent work provides a mathematical framework for studying how changes in network structure over time can influence cooperative behavior.},
  archive      = {J_NATCS},
  author       = {Allen, Benjamin},
  doi          = {10.1038/s43588-023-00513-3},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {737-738},
  shortjournal = {Nat. Comput. Sci},
  title        = {Flipping the intuition for games on dynamic networks},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). AI improves the design of urban communities. <em>NATCS</em>,
<em>3</em>(9), 735–736. (<a
href="https://doi.org/10.1038/s43588-023-00515-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A reinforcement-learning-based framework is proposed for assisting urban planners in the complex task of optimizing the spatial design of urban communities.},
  archive      = {J_NATCS},
  author       = {Santi, Paolo},
  doi          = {10.1038/s43588-023-00515-1},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {735-736},
  shortjournal = {Nat. Comput. Sci},
  title        = {AI improves the design of urban communities},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Conversations that explain predictions. <em>NATCS</em>,
<em>3</em>(9), 734. (<a
href="https://doi.org/10.1038/s43588-023-00525-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00525-z},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {734},
  shortjournal = {Nat. Comput. Sci},
  title        = {Conversations that explain predictions},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). From data to conservation laws. <em>NATCS</em>,
<em>3</em>(9), 733. (<a
href="https://doi.org/10.1038/s43588-023-00524-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00524-0},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {733},
  shortjournal = {Nat. Comput. Sci},
  title        = {From data to conservation laws},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Greener urban communities. <em>NATCS</em>, <em>3</em>(9),
731–732. (<a href="https://doi.org/10.1038/s43588-023-00518-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Perrine Hamel — Assistant Professor at Nanyang Technological University’s Asian School of the Environment and Principal Investigator at the Earth Observatory of Singapore — talks to Nature Computational Science about making cities more sustainable and resilient by incorporating green infrastructure into urban environments, as well as about our current progress with the United Nations’ Sustainable Development Goals (SDGs) related to sustainable cities and climate action.},
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00518-y},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {731-732},
  shortjournal = {Nat. Comput. Sci},
  title        = {Greener urban communities},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023l). Towards equitable education. <em>NATCS</em>, <em>3</em>(9),
729–730. (<a href="https://doi.org/10.1038/s43588-023-00516-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Cristina Villalobos — Myles and Sylvia Aaronson endowed professor in the School of Mathematical and Statistical Sciences at The University of Texas Rio Grande Valley (UTRGV), Director of the Center of Excellence in STEM Education, and Fellow of the American Mathematical Society — talks to Nature Computational Science about her work on empowering underrepresented groups in STEM education and gives her insights into the United Nations Sustainable Development Goals (UN SDGs) related to equitable education and gender equality.},
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00516-0},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {729-730},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards equitable education},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Using computational tools to monitor and improve access to
quality food and water. <em>NATCS</em>, <em>3</em>(9), 726–728. (<a
href="https://doi.org/10.1038/s43588-023-00502-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Progress towards universal access to safe drinking water and nutritious food has been moving forward at a slower than desired rate. Computational tools can help accelerate progress towards these goals, but solutions need to be open source, and designed, developed and implemented in a participatory manner.},
  archive      = {J_NATCS},
  author       = {Omodei, Elisa},
  doi          = {10.1038/s43588-023-00502-6},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {726-728},
  shortjournal = {Nat. Comput. Sci},
  title        = {Using computational tools to monitor and improve access to quality food and water},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reducing disaster risk for the poor in tomorrow’s cities
with computational science. <em>NATCS</em>, <em>3</em>(9), 722–725. (<a
href="https://doi.org/10.1038/s43588-023-00521-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapid urban expansion presents a major challenge to delivering the United Nations Sustainable Development Goals. Urban populations are forecast to increase by 2.2 billion by 2050, and business as usual will condemn many of these new citizens to lives dominated by disaster risk. This need not be the case. Computational science can help urban planners and decision-makers to turn this threat into a time-limited opportunity to reduce disaster risk for hundreds of millions of people.},
  archive      = {J_NATCS},
  author       = {McCloskey, John and Pelling, Mark and Galasso, Carmine and Cremen, Gemma and Menteşe, Emin Yahya and Hope, Max and Comelli, Thaisa and Deshpande, Tanvi and Guragain, Ramesh and Barcena, Alejandro and Gentile, Roberto},
  doi          = {10.1038/s43588-023-00521-3},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {722-725},
  shortjournal = {Nat. Comput. Sci},
  title        = {Reducing disaster risk for the poor in tomorrow’s cities with computational science},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The next seven years. <em>NATCS</em>, <em>3</em>(9), 721.
(<a href="https://doi.org/10.1038/s43588-023-00522-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As we approach the half-way point in the implementation of the Sustainable Development Goals, we discuss how computational science could help in reaching some of these goals by 2030.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00522-2},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {721},
  shortjournal = {Nat. Comput. Sci},
  title        = {The next seven years},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Publisher correction: Reference-guided search for open
reading frames. <em>NATCS</em>, <em>3</em>(8), 720. (<a
href="https://doi.org/10.1038/s43588-023-00508-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Wang, Liguo},
  doi          = {10.1038/s43588-023-00508-0},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {720},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Reference-guided search for open reading frames},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Batch-effect correction with sample remeasurement in highly
confounded case-control studies. <em>NATCS</em>, <em>3</em>(8), 709–719.
(<a href="https://doi.org/10.1038/s43588-023-00500-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Batch effects are pervasive in biomedical studies. One approach to address the batch effects is repeatedly measuring a subset of samples in each batch. These remeasured samples are used to estimate and correct the batch effects. However, rigorous statistical methods for batch-effect correction with remeasured samples are severely underdeveloped. Here we developed a framework for batch-effect correction using remeasured samples in highly confounded case-control studies. We provided theoretical analyses of the proposed procedure, evaluated its power characteristics and provided a power calculation tool to aid in the study design. We found that the number of samples that need to be remeasured depends strongly on the between-batch correlation. When the correlation is high, remeasuring a small subset of samples is possible to rescue most of the power. Batch effects pose great statistical challenges to the analysis of biomedical data. One approach to address batch effects is through sample remeasurement in each batch. In this work, the researchers developed a rigorous batch-effect correction procedure based on remeasured samples.},
  archive      = {J_NATCS},
  author       = {Ye, Hanxuan and Zhang, Xianyang and Wang, Chen and Goode, Ellen L. and Chen, Jun},
  doi          = {10.1038/s43588-023-00500-8},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {709-719},
  shortjournal = {Nat. Comput. Sci},
  title        = {Batch-effect correction with sample remeasurement in highly confounded case-control studies},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Investigating open reading frames in known and novel
transcripts using ORFanage. <em>NATCS</em>, <em>3</em>(8), 700–708. (<a
href="https://doi.org/10.1038/s43588-023-00496-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {ORFanage is a system designed to assign open reading frames (ORFs) to known and novel gene transcripts while maximizing similarity to annotated proteins. The primary intended use of ORFanage is the identification of ORFs in the assembled results of RNA-sequencing experiments, a capability that most transcriptome assembly methods do not have. Our experiments demonstrate how ORFanage can be used to find novel protein variants in RNA-seq datasets, and to improve the annotations of ORFs in tens of thousands of transcript models in the human annotation databases. Through its implementation of a highly accurate and efficient pseudo-alignment algorithm, ORFanage is substantially faster than other ORF annotation methods, enabling its application to very large datasets. When used to analyze transcriptome assemblies, ORFanage can aid in the separation of signal from transcriptional noise and the identification of likely functional transcript variants, ultimately advancing our understanding of biology and medicine. A system called ORFanage can analyze RNA-seq data to find novel protein variants and improve gene annotations. In addition, the method is fast and scalable, being able to filter out noise and thus greatly improving the quality of transcriptome assemblies.},
  archive      = {J_NATCS},
  author       = {Varabyou, Ales and Erdogdu, Beril and Salzberg, Steven L. and Pertea, Mihaela},
  doi          = {10.1038/s43588-023-00496-1},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {700-708},
  shortjournal = {Nat. Comput. Sci},
  title        = {Investigating open reading frames in known and novel transcripts using ORFanage},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Revelation of hidden 2D atmospheric turbulence strength
fields from turbulence effects in infrared imaging. <em>NATCS</em>,
<em>3</em>(8), 687–699. (<a
href="https://doi.org/10.1038/s43588-023-00498-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Turbulence exists widely in the natural atmosphere and in industrial fluids. Strong randomness, anisotropy and mixing of multiple-scale eddies complicate the analysis and measurement of atmospheric turbulence. Although the spatially integrated strength of atmospheric turbulence can be roughly measured indirectly by Doppler radar or laser, direct measurement of two-dimensional (2D) strength fields of atmospheric turbulence is challenging. Here we attempt to solve this problem through infrared imaging. Specifically, we propose a physically boosted cooperative learning framework, termed the PBCL, to quantify 2D turbulence strength from infrared images. To demonstrate the capability of the PBCL, we constructed a dataset with 137,336 infrared images and corresponding 2D turbulence strength fields. The experimental results show that cooperative learning brings performance improvements, enabling the PBCL to simultaneously learn turbulence strength fields and inhibit adverse turbulence effects in images. Our work demonstrates the potential of imaging in measuring physical quantity fields. This study introduces a physically boosted cooperative learning framework (PBCL) to reveal 2D atmospheric turbulence strength fields from turbulence-distorted infrared images. The PBCL is further demonstrated to inhibit adverse turbulence effects in images.},
  archive      = {J_NATCS},
  author       = {Wang, Yadong and Jin, Darui and Chen, Junzhang and Bai, Xiangzhi},
  doi          = {10.1038/s43588-023-00498-z},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {687-699},
  shortjournal = {Nat. Comput. Sci},
  title        = {Revelation of hidden 2D atmospheric turbulence strength fields from turbulence effects in infrared imaging},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Defect graph neural networks for materials discovery in
high-temperature clean-energy applications. <em>NATCS</em>,
<em>3</em>(8), 675–686. (<a
href="https://doi.org/10.1038/s43588-023-00495-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a graph neural network approach that fully automates the prediction of defect formation enthalpies for any crystallographic site from the ideal crystal structure, without the need to create defected atomic structure models as input. Here we used density functional theory reference data for vacancy defects in oxides, to train a defect graph neural network (dGNN) model that replaces the density functional theory supercell relaxations otherwise required for each symmetrically unique crystal site. Interfaced with thermodynamic calculations of reduction entropies and associated free energies, the dGNN model is applied to the screening of oxides in the Materials Project database, connecting the zero-kelvin defect enthalpies to high-temperature process conditions relevant for solar thermochemical hydrogen production and other energy applications. The dGNN approach is applicable to arbitrary structures with an accuracy limited principally by the amount and diversity of the training data, and it is generalizable to other defect types and advanced graph convolution architectures. It will help to tackle future materials discovery problems in clean energy and beyond. Automating materials’ defect predictions with graph neural networks, when coupled to first principles thermodynamic calculations, accelerates materials discovery for a variety of high-temperature, clean-energy applications.},
  archive      = {J_NATCS},
  author       = {Witman, Matthew D. and Goyal, Anuj and Ogitsu, Tadashi and McDaniel, Anthony H. and Lany, Stephan},
  doi          = {10.1038/s43588-023-00495-2},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {675-686},
  shortjournal = {Nat. Comput. Sci},
  title        = {Defect graph neural networks for materials discovery in high-temperature clean-energy applications},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). An imaging-based approach to measure atmospheric turbulence.
<em>NATCS</em>, <em>3</em>(8), 673–674. (<a
href="https://doi.org/10.1038/s43588-023-00501-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The dissipation and bending of light waves by atmospheric turbulence adversely affects infrared imaging, leading to grayscale drift, distortion, and blurring. A deep learning method has been developed to both extract the two-dimensional atmospheric turbulence strength fields and obtain clear and stable images from turbulence-distorted infrared images.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00501-7},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {673-674},
  shortjournal = {Nat. Comput. Sci},
  title        = {An imaging-based approach to measure atmospheric turbulence},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Energy materials screening with defect graph neural
networks. <em>NATCS</em>, <em>3</em>(8), 671–672. (<a
href="https://doi.org/10.1038/s43588-023-00510-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Graph neural networks (GNNs) present a promising route for machine learning of solid-state materials’ properties, but methods capable of directly predicting defect properties from ideal, defect-free structures are needed. A GNN developed for direct defect property predictions enables high-throughput screening of redox-active oxides for energy applications and beyond.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00510-6},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {671-672},
  shortjournal = {Nat. Comput. Sci},
  title        = {Energy materials screening with defect graph neural networks},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Leveraging remeasured samples in biomedical studies.
<em>NATCS</em>, <em>3</em>(8), 669–670. (<a
href="https://doi.org/10.1038/s43588-023-00491-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study develops a statistical method of correcting batch effects by employing remeasured samples in highly confounded case–control studies.},
  archive      = {J_NATCS},
  author       = {Zhong, Luer and Bacher, Rhonda},
  doi          = {10.1038/s43588-023-00491-6},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {669-670},
  shortjournal = {Nat. Comput. Sci},
  title        = {Leveraging remeasured samples in biomedical studies},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Reference-guided search for open reading frames.
<em>NATCS</em>, <em>3</em>(8), 667–668. (<a
href="https://doi.org/10.1038/s43588-023-00497-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposes an approach for identifying open reading frames that uses reference annotation to improve the accuracy of the detection.},
  archive      = {J_NATCS},
  author       = {Wang, Liguo},
  doi          = {10.1038/s43588-023-00497-0},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {667-668},
  shortjournal = {Nat. Comput. Sci},
  title        = {Reference-guided search for open reading frames},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Estimating cell growth rate. <em>NATCS</em>, <em>3</em>(8),
666. (<a href="https://doi.org/10.1038/s43588-023-00505-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00505-3},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {666},
  shortjournal = {Nat. Comput. Sci},
  title        = {Estimating cell growth rate},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Optimal crystal structure solutions. <em>NATCS</em>,
<em>3</em>(8), 665. (<a
href="https://doi.org/10.1038/s43588-023-00504-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-023-00504-4},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {665},
  shortjournal = {Nat. Comput. Sci},
  title        = {Optimal crystal structure solutions},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Enhancing the ethics of user-sourced online data collection
and sharing. <em>NATCS</em>, <em>3</em>(8), 660–664. (<a
href="https://doi.org/10.1038/s43588-023-00490-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Social media and other internet platforms are making it even harder for researchers to investigate their effects on society. One way forward is user-sourced data collection of data to be shared among many researchers, using robust ethics tools to protect the interests of research participants and society.},
  archive      = {J_NATCS},
  author       = {Meyer, Michelle N. and Basl, John and Choffnes, David and Wilson, Christo and Lazer, David M. J.},
  doi          = {10.1038/s43588-023-00490-7},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {660-664},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enhancing the ethics of user-sourced online data collection and sharing},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The carbon footprint of computational research.
<em>NATCS</em>, <em>3</em>(8), 659. (<a
href="https://doi.org/10.1038/s43588-023-00506-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Monitoring and reporting on the environmental impact of computational science research is critical to ensure a more sustainable future.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00506-2},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {659},
  shortjournal = {Nat. Comput. Sci},
  title        = {The carbon footprint of computational research},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: The high-dimensional space of human
diseases built from diagnosis records and mapped to genetic loci.
<em>NATCS</em>, <em>3</em>(7), 658. (<a
href="https://doi.org/10.1038/s43588-023-00488-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Jia, Gengjie and Li, Yu and Zhong, Xue and Wang, Kanix and Pividori, Milton and Alomairy, Rabab and Esposito, Aniello and Ltaief, Hatem and Terao, Chikashi and Akiyama, Masato and Matsuda, Koichi and Keyes, David E. and Im, Hae Kyung and Gojobori, Takashi and Kamatani, Yoichiro and Kubo, Michiaki and Cox, Nancy J. and Evans, James and Gao, Xin and Rzhetsky, Andrey},
  doi          = {10.1038/s43588-023-00488-1},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {658},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: The high-dimensional space of human diseases built from diagnosis records and mapped to genetic loci},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Mapping disease regulatory circuits at cell-type resolution
from single-cell multiomics data. <em>NATCS</em>, <em>3</em>(7),
644–657. (<a href="https://doi.org/10.1038/s43588-023-00476-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Resolving chromatin-remodeling-linked gene expression changes at cell-type resolution is important for understanding disease states. Here we describe MAGICAL (Multiome Accessibility Gene Integration Calling and Looping), a hierarchical Bayesian approach that leverages paired single-cell RNA sequencing and single-cell transposase-accessible chromatin sequencing from different conditions to map disease-associated transcription factors, chromatin sites, and genes as regulatory circuits. By simultaneously modeling signal variation across cells and conditions in both omics data types, MAGICAL achieved high accuracy on circuit inference. We applied MAGICAL to study Staphylococcus aureus sepsis from peripheral blood mononuclear single-cell data that we generated from subjects with bloodstream infection and uninfected controls. MAGICAL identified sepsis-associated regulatory circuits predominantly in CD14 monocytes, known to be activated by bacterial sepsis. We addressed the challenging problem of distinguishing host regulatory circuit responses to methicillin-resistant and methicillin-susceptible S. aureus infections. Although differential expression analysis failed to show predictive value, MAGICAL identified epigenetic circuit biomarkers that distinguished methicillin-resistant from methicillin-susceptible S. aureus infections. A Bayesian method, called MAGICAL, that contrasts single cell multiomics data across conditions to accurately discover differences in gene regulatory circuits at cell type resolution is applied to specific host-based diagnosis of bacterial sepsis.},
  archive      = {J_NATCS},
  author       = {Chen, Xi and Wang, Yuan and Cappuccio, Antonio and Cheng, Wan-Sze and Zamojski, Frederique Ruf and Nair, Venugopalan D. and Miller, Clare M. and Rubenstein, Aliza B. and Nudelman, German and Tadych, Alicja and Theesfeld, Chandra L. and Vornholt, Alexandria and George, Mary-Catherine and Ruffin, Felicia and Dagher, Michael and Chawla, Daniel G. and Soares-Schanoski, Alessandra and Spurbeck, Rachel R. and Ndhlovu, Lishomwa C. and Sebra, Robert and Kleinstein, Steven H. and Letizia, Andrew G. and Ramos, Irene and Fowler, Vance G. and Woods, Christopher W. and Zaslavsky, Elena and Troyanskaya, Olga G. and Sealfon, Stuart C.},
  doi          = {10.1038/s43588-023-00476-5},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {644-657},
  shortjournal = {Nat. Comput. Sci},
  title        = {Mapping disease regulatory circuits at cell-type resolution from single-cell multiomics data},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mixed model-based deconvolution of cell-state abundances
(MeDuSA) along a one-dimensional trajectory. <em>NATCS</em>,
<em>3</em>(7), 630–643. (<a
href="https://doi.org/10.1038/s43588-023-00487-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deconvoluting cell-state abundances from bulk RNA-sequencing data can add considerable value to existing data, but achieving fine-resolution and high-accuracy deconvolution remains a challenge. Here we introduce MeDuSA, a mixed model-based method that leverages single-cell RNA-sequencing data as a reference to estimate cell-state abundances along a one-dimensional trajectory in bulk RNA-sequencing data. The advantage of MeDuSA lies primarily in estimating cell abundance in each state while fitting the remaining cells of the same type individually as random effects. Extensive simulations and real-data benchmark analyses demonstrate that MeDuSA greatly improves the estimation accuracy over existing methods for one-dimensional trajectories. Applying MeDuSA to cohort-level RNA-sequencing datasets reveals associations of cell-state abundances with disease or treatment conditions and cell-state-dependent genetic control of transcription. Our study provides a high-accuracy and fine-resolution method for cell-state deconvolution along a one-dimensional trajectory and demonstrates its utility in characterizing the dynamics of cell states in various biological processes. MeDuSA, a mixed model-based method, leverages single-cell RNA-sequencing data for high-accuracy, fine-resolution cellular deconvolution in bulk RNA-sequencing data. It improves deconvolution accuracy over existing methods, revealing cell-state dynamics in various biological processes.},
  archive      = {J_NATCS},
  author       = {Song, Liyang and Sun, Xiwei and Qi, Ting and Yang, Jian},
  doi          = {10.1038/s43588-023-00487-2},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {630-643},
  shortjournal = {Nat. Comput. Sci},
  title        = {Mixed model-based deconvolution of cell-state abundances (MeDuSA) along a one-dimensional trajectory},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Neural ADMIXTURE for rapid genomic clustering.
<em>NATCS</em>, <em>3</em>(7), 621–629. (<a
href="https://doi.org/10.1038/s43588-023-00482-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Characterizing the genetic structure of large cohorts has become increasingly important as genetic studies extend to massive, increasingly diverse biobanks. Popular methods decompose individual genomes into fractional cluster assignments with each cluster representing a vector of DNA variant frequencies. However, with rapidly increasing biobank sizes, these methods have become computationally intractable. Here we present Neural ADMIXTURE, a neural network autoencoder that follows the same modeling assumptions as the current standard algorithm, ADMIXTURE, while reducing the compute time by orders of magnitude surpassing even the fastest alternatives. One month of continuous compute using ADMIXTURE can be reduced to just hours with Neural ADMIXTURE. A multi-head approach allows Neural ADMIXTURE to offer even further acceleration by computing multiple cluster numbers in a single run. Furthermore, the models can be stored, allowing cluster assignment to be performed on new data in linear time without needing to share the training samples. Neural ADMIXTURE is a neural-network-based, interpretable autoencoder that performs rapid genomic clustering in biobank-scale databases.},
  archive      = {J_NATCS},
  author       = {Dominguez Mantes, Albert and Mas Montserrat, Daniel and Bustamante, Carlos D. and Giró-i-Nieto, Xavier and Ioannidis, Alexander G.},
  doi          = {10.1038/s43588-023-00482-7},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {621-629},
  shortjournal = {Nat. Comput. Sci},
  title        = {Neural ADMIXTURE for rapid genomic clustering},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Urban dynamics through the lens of human mobility.
<em>NATCS</em>, <em>3</em>(7), 611–620. (<a
href="https://doi.org/10.1038/s43588-023-00484-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The urban spatial structure represents the distribution of public and private spaces in cities and how people move within them. Although it usually evolves slowly, it can change quickly during large-scale emergency events, as well as due to urban renewal in rapidly developing countries. Here we present an approach to delineate such urban dynamics in quasi-real time through a human mobility metric, the mobility centrality index ΔKS. As a case study, we tracked the urban dynamics of eleven Spanish cities during the COVID-19 pandemic. The results revealed that their structures became more monocentric during the lockdown in the first wave, but kept their regular spatial structures during the second wave. To provide a more comprehensive understanding of mobility from home, we also introduce a dimensionless metric, KSHBT, which measures the extent of home-based travel and provides statistical insights into the transmission of COVID-19. By utilizing individual mobility data, our metrics enable the detection of changes in the urban spatial structure. The study presents a mobility centrality index to delineate urban dynamics in quasi-real time with mobile-phone data. The results indicate that urban structures were becoming more monocentric during the COVID-19 lockdown periods in major cities in Spain.},
  archive      = {J_NATCS},
  author       = {Xu, Yanyan and Olmos, Luis E. and Mateo, David and Hernando, Alberto and Yang, Xiaokang and González, Marta C.},
  doi          = {10.1038/s43588-023-00484-5},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {611-620},
  shortjournal = {Nat. Comput. Sci},
  title        = {Urban dynamics through the lens of human mobility},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Designing equitable algorithms. <em>NATCS</em>,
<em>3</em>(7), 601–610. (<a
href="https://doi.org/10.1038/s43588-023-00485-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predictive algorithms are now commonly used to distribute society’s resources and sanctions. But these algorithms can entrench and exacerbate inequities. To guard against this possibility, many have suggested that algorithms be subject to formal fairness constraints. Here we argue, however, that popular constraints—while intuitively appealing—often worsen outcomes for individuals in marginalized groups, and can even leave all groups worse off. We outline a more holistic path forward for improving the equity of algorithmically guided decisions. While the adherence to fairness constraints has become common practice in the design of algorithms across many contexts, a more holistic approach should be taken to avoid inflicting additional burdens on individuals in all groups, including those in marginalized communities.},
  archive      = {J_NATCS},
  author       = {Chohlas-Wood, Alex and Coots, Madison and Goel, Sharad and Nyarko, Julian},
  doi          = {10.1038/s43588-023-00485-4},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {601-610},
  shortjournal = {Nat. Comput. Sci},
  title        = {Designing equitable algorithms},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Future directions in human mobility science. <em>NATCS</em>,
<em>3</em>(7), 588–600. (<a
href="https://doi.org/10.1038/s43588-023-00469-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We provide a brief review of human mobility science and present three key areas where we expect to see substantial advancements. We start from the mind and discuss the need to better understand how spatial cognition shapes mobility patterns. We then move to societies and argue the importance of better understanding new forms of transportation. We conclude by discussing how algorithms shape mobility behavior and provide useful tools for modelers. Finally, we discuss how progress on these research directions may help us address some of the challenges our society faces today. The field of human mobility has evolved drastically in the past 20 years. In this Perspective, the authors discuss three key areas in human mobility, framed as minds, societies and algorithms, where they expect to see substantial improvements in the future.},
  archive      = {J_NATCS},
  author       = {Pappalardo, Luca and Manley, Ed and Sekara, Vedran and Alessandretti, Laura},
  doi          = {10.1038/s43588-023-00469-4},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {588-600},
  shortjournal = {Nat. Comput. Sci},
  title        = {Future directions in human mobility science},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A software resource for large graph processing and analysis.
<em>NATCS</em>, <em>3</em>(7), 586–587. (<a
href="https://doi.org/10.1038/s43588-023-00466-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {GRAPE is a software resource for graph processing, learning and embedding that is orders of magnitude faster than existing state-of-the-art libraries. GRAPE can quickly process real-world graphs with millions of nodes and billions of edges, enabling complex graph analyses and research in graph-based machine learning and in diverse disciplines.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00466-7},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {586-587},
  shortjournal = {Nat. Comput. Sci},
  title        = {A software resource for large graph processing and analysis},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Uncovering the genetic circuits that drive diseases.
<em>NATCS</em>, <em>3</em>(7), 584–585. (<a
href="https://doi.org/10.1038/s43588-023-00475-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A hierarchical Bayesian method identifies cell-type specific changes in gene regulatory circuits in disease by integrating single-cell and three-dimensional measurements of the genome.},
  archive      = {J_NATCS},
  author       = {Przytycki, Pawel F.},
  doi          = {10.1038/s43588-023-00475-6},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {584-585},
  shortjournal = {Nat. Comput. Sci},
  title        = {Uncovering the genetic circuits that drive diseases},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cellular deconvolution with continuous transitions.
<em>NATCS</em>, <em>3</em>(7), 582–583. (<a
href="https://doi.org/10.1038/s43588-023-00489-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent work introduces a cellular deconvolution method, MeDuSA, of estimating cell-state abundance along a one-dimensional trajectory from bulk RNA-seq data with fine resolution and high accuracy, enabling the characterization of cell-state transition in various biological processes.},
  archive      = {J_NATCS},
  author       = {Zhang, Zheyang and Huang, Jialiang},
  doi          = {10.1038/s43588-023-00489-0},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {582-583},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cellular deconvolution with continuous transitions},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Machine learning speeds up genetic structure analysis.
<em>NATCS</em>, <em>3</em>(7), 580–581. (<a
href="https://doi.org/10.1038/s43588-023-00481-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning is used to accelerate the inference of genetic clusters, allowing the analysis of hundreds of thousands of human genomic datasets in a computationally efficient way.},
  archive      = {J_NATCS},
  author       = {Smith, Chris C. R.},
  doi          = {10.1038/s43588-023-00481-8},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {580-581},
  shortjournal = {Nat. Comput. Sci},
  title        = {Machine learning speeds up genetic structure analysis},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Using fast and slow data to unfold city dynamics.
<em>NATCS</em>, <em>3</em>(7), 578–579. (<a
href="https://doi.org/10.1038/s43588-023-00486-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Real-time mobility data capturing city-wide human movement can be used to characterize cities, their segregation, and population responses to exogenous events such as pandemics.},
  archive      = {J_NATCS},
  author       = {Bagrow, James},
  doi          = {10.1038/s43588-023-00486-3},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {578-579},
  shortjournal = {Nat. Comput. Sci},
  title        = {Using fast and slow data to unfold city dynamics},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023l). Shedding light on microbiome–drug interactions.
<em>NATCS</em>, <em>3</em>(7), 577. (<a
href="https://doi.org/10.1038/s43588-023-00494-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00494-3},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {577},
  shortjournal = {Nat. Comput. Sci},
  title        = {Shedding light on microbiome–drug interactions},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A language model for medical predictive tasks.
<em>NATCS</em>, <em>3</em>(7), 576. (<a
href="https://doi.org/10.1038/s43588-023-00493-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00493-4},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {576},
  shortjournal = {Nat. Comput. Sci},
  title        = {A language model for medical predictive tasks},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Physics-inspired model for online content dynamics.
<em>NATCS</em>, <em>3</em>(7), 575. (<a
href="https://doi.org/10.1038/s43588-023-00492-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-023-00492-5},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {575},
  shortjournal = {Nat. Comput. Sci},
  title        = {Physics-inspired model for online content dynamics},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Structure prediction and materials design with generative
neural networks. <em>NATCS</em>, <em>3</em>(7), 572–574. (<a
href="https://doi.org/10.1038/s43588-023-00471-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The prediction of stable crystal structures is an important part of designing solid-state crystalline materials with desired properties. Recent advances in structural feature representations and generative neural networks promise the ability to efficiently create new stable structures to use for inverse design and to search for materials with tailored functionalities.},
  archive      = {J_NATCS},
  author       = {Yan, Da and Smith, Adam D. and Chen, Cheng-Chien},
  doi          = {10.1038/s43588-023-00471-w},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {572-574},
  shortjournal = {Nat. Comput. Sci},
  title        = {Structure prediction and materials design with generative neural networks},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Of data and transparency. <em>NATCS</em>, <em>3</em>(7),
571. (<a href="https://doi.org/10.1038/s43588-023-00499-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While the increasing availability of data creates unprecedented research opportunities, it is important to understand the provenance of these datasets to ensure reliable data-driven conclusions.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00499-y},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {571},
  shortjournal = {Nat. Comput. Sci},
  title        = {Of data and transparency},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Deep-learning electronic-structure
calculation of magnetic superstructures. <em>NATCS</em>, <em>3</em>(6),
569. (<a href="https://doi.org/10.1038/s43588-023-00472-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Li, He and Tang, Zechen and Gong, Xiaoxun and Zou, Nianlong and Duan, Wenhui and Xu, Yong},
  doi          = {10.1038/s43588-023-00472-9},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {569},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Deep-learning electronic-structure calculation of magnetic superstructures},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). GRAPE for fast and scalable graph processing and
random-walk-based embedding. <em>NATCS</em>, <em>3</em>(6), 552–568. (<a
href="https://doi.org/10.1038/s43588-023-00465-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Graph representation learning methods opened new avenues for addressing complex, real-world problems represented by graphs. However, many graphs used in these applications comprise millions of nodes and billions of edges and are beyond the capabilities of current methods and software implementations. We present GRAPE (Graph Representation Learning, Prediction and Evaluation), a software resource for graph processing and embedding that is able to scale with big graphs by using specialized and smart data structures, algorithms, and a fast parallel implementation of random-walk-based methods. Compared with state-of-the-art software resources, GRAPE shows an improvement of orders of magnitude in empirical space and time complexity, as well as competitive edge- and node-label prediction performance. GRAPE comprises approximately 1.7 million well-documented lines of Python and Rust code and provides 69 node-embedding methods, 25 inference models, a collection of efficient graph-processing utilities, and over 80,000 graphs from the literature and other sources. Standardized interfaces allow a seamless integration of third-party libraries, while ready-to-use and modular pipelines permit an easy-to-use evaluation of graph-representation-learning methods, therefore also positioning GRAPE as a software resource that performs a fair comparison between methods and libraries for graph processing and embedding. GRAPE is a software resource for graph learning and embedding that is orders of magnitude faster than existing state-of-the-art libraries, making large-graph analysis feasible in a wide range of real-world applications.},
  archive      = {J_NATCS},
  author       = {Cappelletti, Luca and Fontana, Tommaso and Casiraghi, Elena and Ravanmehr, Vida and Callahan, Tiffany J. and Cano, Carlos and Joachimiak, Marcin P. and Mungall, Christopher J. and Robinson, Peter N. and Reese, Justin and Valentini, Giorgio},
  doi          = {10.1038/s43588-023-00465-8},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {552-568},
  shortjournal = {Nat. Comput. Sci},
  title        = {GRAPE for fast and scalable graph processing and random-walk-based embedding},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Theory of overparametrization in quantum neural networks.
<em>NATCS</em>, <em>3</em>(6), 542–551. (<a
href="https://doi.org/10.1038/s43588-023-00467-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The prospect of achieving quantum advantage with quantum neural networks (QNNs) is exciting. Understanding how QNN properties (for example, the number of parameters M) affect the loss landscape is crucial to designing scalable QNN architectures. Here we rigorously analyze the overparametrization phenomenon in QNNs, defining overparametrization as the regime where the QNN has more than a critical number of parameters Mc allowing it to explore all relevant directions in state space. Our main results show that the dimension of the Lie algebra obtained from the generators of the QNN is an upper bound for Mc, and for the maximal rank that the quantum Fisher information and Hessian matrices can reach. Underparametrized QNNs have spurious local minima in the loss landscape that start disappearing when M ≥ Mc. Thus, the overparametrization onset corresponds to a computational phase transition where the QNN trainability is greatly improved. We then connect the notion of overparametrization to the QNN capacity, so that when a QNN is overparametrized, its capacity achieves its maximum possible value. A theoretical framework for quantum neural network (QNN) overparametrization, a phase transition in loss landscape complexity, is established. The precise characterization of the critical number of parameters offered is expected to impact QNN design.},
  archive      = {J_NATCS},
  author       = {Larocca, Martín and Ju, Nathan and García-Martín, Diego and Coles, Patrick J. and Cerezo, Marco},
  doi          = {10.1038/s43588-023-00467-6},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {542-551},
  shortjournal = {Nat. Comput. Sci},
  title        = {Theory of overparametrization in quantum neural networks},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ab initio real-time quantum dynamics of charge carriers in
momentum space. <em>NATCS</em>, <em>3</em>(6), 532–541. (<a
href="https://doi.org/10.1038/s43588-023-00456-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Application of the non-adiabatic molecular dynamics (NAMD) approach is limited to studying carrier dynamics in the momentum space, as a supercell is required to sample the phonon excitation and electron–phonon (e–ph) interaction at different momenta in a molecular dynamics simulation. Here we develop an ab initio approach for the real-time charge carrier quantum dynamics in the momentum space (NAMD_k) by directly introducing e–ph coupling into the Hamiltonian based on the harmonic approximation. The NAMD_k approach maintains the zero-point energy and includes memory effects of carrier dynamics. The application of NAMD_k to the hot carrier dynamics in graphene reveals the phonon-specific relaxation mechanism. An energy threshold of 0.2 eV—defined by two optical phonon modes—separates the hot electron relaxation into fast and slow regions with lifetimes of pico- and nanoseconds, respectively. The NAMD_k approach provides an effective tool to understand real-time carrier dynamics in the momentum space for different materials. This study presents an ab initio approach for the real-time charge carrier quantum dynamics in the momentum space, which is computationally more efficient than conventional real-space non-adiabatic molecular dynamics method. The method is applied to study hot carrier dynamics in graphene, which provides insights about the phonon-specific relaxation mechanism.},
  archive      = {J_NATCS},
  author       = {Zheng, Zhenfa and Shi, Yongliang and Zhou, Jin-Jian and Prezhdo, Oleg V. and Zheng, Qijing and Zhao, Jin},
  doi          = {10.1038/s43588-023-00456-9},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {532-541},
  shortjournal = {Nat. Comput. Sci},
  title        = {Ab initio real-time quantum dynamics of charge carriers in momentum space},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell meta-analysis of inflammatory bowel disease with
scIBD. <em>NATCS</em>, <em>3</em>(6), 522–531. (<a
href="https://doi.org/10.1038/s43588-023-00464-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the heterogeneous intestinal microenvironment is critical to uncover the pathogenesis of inflammatory bowel disease (IBD). Recent advances in single-cell RNA sequencing (scRNA-seq) have identified certain cell types and genes that could contribute to IBD; however, a comprehensively integrated analysis of these scRNA-seq datasets is not yet available. Here we introduce scIBD, a platform for single-cell meta-analysis of IBD with interactive and visualization features, which combines highly curated single-cell datasets in a uniform workflow, enabling identifying rare or less-characterized cell types in IBD and dissecting the commonalities, as well as the differences between ulcerative colitis and Crohn’s disease. scIBD also incorporates multifunctional information—including regulon activity, GWAS-implicated risk genes and genes targeted by therapeutics—to infer clinically relevant cell-type specificity. Collectively, scIBD is a user-friendly web-based platform for the community to analyze the transcriptome features and gene regulatory networks associated with the pathogenesis and treatment of IBD at single-cell resolution. A platform for single-cell meta-analysis of inflammatory bowel disease, named scIBD, enables identification of rare or less-characterized cell types and the dissection of the commonalities and differences between ulcerative colitis and Crohn’s disease.},
  archive      = {J_NATCS},
  author       = {Nie, Hu and Lin, Peilu and Zhang, Yu and Wan, Yihong and Li, Jiesheng and Yin, Chengqian and Zhang, Lei},
  doi          = {10.1038/s43588-023-00464-9},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {522-531},
  shortjournal = {Nat. Comput. Sci},
  title        = {Single-cell meta-analysis of inflammatory bowel disease with scIBD},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). GREENER principles for environmentally sustainable
computational science. <em>NATCS</em>, <em>3</em>(6), 514–521. (<a
href="https://doi.org/10.1038/s43588-023-00461-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The carbon footprint of scientific computing is substantial, but environmentally sustainable computational science (ESCS) is a nascent field with many opportunities to thrive. To realize the immense green opportunities and continued, yet sustainable, growth of computer science, we must take a coordinated approach to our current challenges, including greater awareness and transparency, improved estimation and wider reporting of environmental impacts. Here, we present a snapshot of where ESCS stands today and introduce the GREENER set of principles, as well as guidance for best practices moving forward. The carbon footprint of computational sciences is substantial, but there is an immense opportunity to lead the way towards sustainable research. In this Perspective the authors lay some fundamental principles to transform computational science into an exemplar of broad societal impact and sustainability.},
  archive      = {J_NATCS},
  author       = {Lannelongue, Loïc and Aronson, Hans-Erik G. and Bateman, Alex and Birney, Ewan and Caplan, Talia and Juckes, Martin and McEntyre, Johanna and Morris, Andrew D. and Reilly, Gerry and Inouye, Michael},
  doi          = {10.1038/s43588-023-00461-y},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {514-521},
  shortjournal = {Nat. Comput. Sci},
  title        = {GREENER principles for environmentally sustainable computational science},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The landscape of computational approaches for artificial
photosynthesis. <em>NATCS</em>, <em>3</em>(6), 504–513. (<a
href="https://doi.org/10.1038/s43588-023-00450-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial photosynthesis is an attractive strategy for converting solar energy into fuels, largely because the Earth receives enough solar energy in one hour to meet humanity’s energy needs for an entire year. However, developing devices for artificial photosynthesis remains difficult and requires computational approaches to guide and assist the interpretation of experiments. In this Perspective, we discuss current and future computational approaches, as well as the challenges of designing and characterizing molecular assemblies that absorb solar light, transfer electrons between interfaces, and catalyze water-splitting and fuel-forming reactions. Artificial photosynthesis has the potential to capture and store solar energy in the form of chemical bonds. Computational approaches provide useful guidelines for the experimental design of photosynthetic devices, but to make this possible, many challenges must be overcome.},
  archive      = {J_NATCS},
  author       = {Yang, Ke R. and Kyro, Gregory W. and Batista, Victor S.},
  doi          = {10.1038/s43588-023-00450-1},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {504-513},
  shortjournal = {Nat. Comput. Sci},
  title        = {The landscape of computational approaches for artificial photosynthesis},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Current and future machine learning approaches for modeling
atmospheric cluster formation. <em>NATCS</em>, <em>3</em>(6), 495–503.
(<a href="https://doi.org/10.1038/s43588-023-00435-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The formation of strongly bound atmospheric molecular clusters is the first step towards forming new aerosol particles. Recent advances in the application of machine learning models open an enormous opportunity for complementing expensive quantum chemical calculations with efficient machine learning predictions. In this Perspective, we present how data-driven approaches can be applied to accelerate cluster configurational sampling, thereby greatly increasing the number of chemically relevant systems that can be covered. Although the number of quantum chemical studies on atmospheric cluster formation continue to rise, data-driven approaches can greatly expand the number of chemically relevant systems that can be covered and increase our understanding of the aerosol particle formation process.},
  archive      = {J_NATCS},
  author       = {Kubečka, Jakub and Knattrup, Yosef and Engsvang, Morten and Jensen, Andreas Buchgraitz and Ayoubi, Daniel and Wu, Haide and Christiansen, Ove and Elm, Jonas},
  doi          = {10.1038/s43588-023-00435-0},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {495-503},
  shortjournal = {Nat. Comput. Sci},
  title        = {Current and future machine learning approaches for modeling atmospheric cluster formation},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Computational models to confront the complex pollution
footprint of plastic in the environment. <em>NATCS</em>, <em>3</em>(6),
486–494. (<a href="https://doi.org/10.1038/s43588-023-00445-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The threat posed by plastic in the environment is poorly characterized due to uncertainties and unknowns about sources, transport, transformation and removal processes, and the properties of the plastic pollution itself. Plastic creates a footprint of particulate pollution with a diversity of composition, size and shape, and a halo of chemicals. In this Perspective, we argue that process-based mass-balance models could provide a platform to synthesize knowledge about plastic pollution as a function of its measurable intrinsic properties. Addressing plastic pollution in the environment is difficult due to the high complexity and diversity of physical and chemical properties. This Perspective argues that process-based mass-balance models could provide a viable solution for evaluating environmental exposure to plastic pollution.},
  archive      = {J_NATCS},
  author       = {MacLeod, Matthew and Domercq, Prado and Harrison, Sam and Praetorius, Antonia},
  doi          = {10.1038/s43588-023-00445-y},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {486-494},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational models to confront the complex pollution footprint of plastic in the environment},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Why bigger quantum neural networks do better.
<em>NATCS</em>, <em>3</em>(6), 484–485. (<a
href="https://doi.org/10.1038/s43588-023-00468-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Increasing the number of parameters in a quantum neural network leads to a computational ‘phase transition’, beyond which training the network becomes significantly easier. An algebraic theory has been developed for this overparametrization phenomenon and predicts its onset above a certain parameter threshold.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00468-5},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {484-485},
  shortjournal = {Nat. Comput. Sci},
  title        = {Why bigger quantum neural networks do better},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A single-cell transcriptomic meta-analysis platform for
inflammatory bowel disease. <em>NATCS</em>, <em>3</em>(6), 482–483. (<a
href="https://doi.org/10.1038/s43588-023-00474-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By conducting single-cell meta-analyses of inflammatory bowel disease, we identify rare or less-characterized cell subtypes linked to GWAS risk genes and therapeutic targets and dissect the commonalities and differences between ulcerative colitis and Crohn’s disease. Consequently, we present an interactive and user-friendly platform for the research community.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00474-7},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {482-483},
  shortjournal = {Nat. Comput. Sci},
  title        = {A single-cell transcriptomic meta-analysis platform for inflammatory bowel disease},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Computing electron dynamics in momentum space.
<em>NATCS</em>, <em>3</em>(6), 480–481. (<a
href="https://doi.org/10.1038/s43588-023-00473-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A momentum-space algorithm is proposed to simulate electron dynamics with time-dependent density functional theory, which expands the scope of conventional real-space methods.},
  archive      = {J_NATCS},
  author       = {Bernardi, Marco},
  doi          = {10.1038/s43588-023-00473-8},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {480-481},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computing electron dynamics in momentum space},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023j). Treating quantum states as human faces. <em>NATCS</em>,
<em>3</em>(6), 479. (<a
href="https://doi.org/10.1038/s43588-023-00478-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-023-00478-3},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {479},
  shortjournal = {Nat. Comput. Sci},
  title        = {Treating quantum states as human faces},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Building molecules across the periodic table.
<em>NATCS</em>, <em>3</em>(6), 478. (<a
href="https://doi.org/10.1038/s43588-023-00479-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00479-2},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {478},
  shortjournal = {Nat. Comput. Sci},
  title        = {Building molecules across the periodic table},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Experts go deeper. <em>NATCS</em>, <em>3</em>(6), 477. (<a
href="https://doi.org/10.1038/s43588-023-00477-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00477-4},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {477},
  shortjournal = {Nat. Comput. Sci},
  title        = {Experts go deeper},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). Particle picking in tomograms. <em>NATCS</em>,
<em>3</em>(6), 476. (<a
href="https://doi.org/10.1038/s43588-023-00480-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00480-9},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {476},
  shortjournal = {Nat. Comput. Sci},
  title        = {Particle picking in tomograms},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Towards a welcoming climate for LGBTQ people.
<em>NATCS</em>, <em>3</em>(6), 473–475. (<a
href="https://doi.org/10.1038/s43588-023-00470-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Michael Falk, professor of Materials Science and Engineering, Mechanical Engineering, and Physics at Johns Hopkins University, talks to Nature Computational Science about his academic training at the intersection of physics and computer science, his research in condensed-matter physics, as well as his experience in improving diversity and inclusion in the physics research community.},
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-023-00470-x},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {473-475},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards a welcoming climate for LGBTQ people},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Little by little, a bird builds its nest. <em>NATCS</em>,
<em>3</em>(6), 470–472. (<a
href="https://doi.org/10.1038/s43588-023-00458-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Carla Gomes, Ronald C. and Antonia V. Nielsen Professor of Computing and Information Science, director of the Institute for Computational Sustainability, and co-director of the newly established AI for Science Institute at Cornell University, discusses with Nature Computational Science her research on sustainability and how we can address the world’s most pressing issues little by little.},
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00458-7},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {470-472},
  shortjournal = {Nat. Comput. Sci},
  title        = {Little by little, a bird builds its nest},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). People, places, and the planet. <em>NATCS</em>,
<em>3</em>(6), 467–469. (<a
href="https://doi.org/10.1038/s43588-023-00455-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Alexandre Caldas, a Director at the United Nations (UN) as Chief of Country Outreach, Technology and Innovation in the Science Division at the United Nations Environment Programme (UNEP) and Chair of the United Nations Geospatial Network across 40 agencies of the UN, talks to Nature Computational Science about the importance of data availability, the Sustainable Development Goals, and evolving policy.},
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00455-w},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {467-469},
  shortjournal = {Nat. Comput. Sci},
  title        = {People, places, and the planet},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Collaborations drive energy storage research.
<em>NATCS</em>, <em>3</em>(6), 464–466. (<a
href="https://doi.org/10.1038/s43588-023-00457-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Y. Shirley Meng, Professor of Molecular Engineering at the University of Chicago and Chief Scientist at the Argonne Collaborative Center for Energy Storage Science (ACCESS), discusses her research on energy storage materials and the importance of multidisciplinary collaborations.},
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00457-8},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {464-466},
  shortjournal = {Nat. Comput. Sci},
  title        = {Collaborations drive energy storage research},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Harnessing the power of decision-support tools to trigger
climate action. <em>NATCS</em>, <em>3</em>(6), 461–463. (<a
href="https://doi.org/10.1038/s43588-023-00460-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accelerating climate action requires harnessing the power of decision-support tools in new ways. This vision cannot be realized without interdisciplinary computational scientists that are capable of integrating knowledge from the environmental, social and cognitive sciences.},
  archive      = {J_NATCS},
  author       = {Molina-Perez, Edmundo},
  doi          = {10.1038/s43588-023-00460-z},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {461-463},
  shortjournal = {Nat. Comput. Sci},
  title        = {Harnessing the power of decision-support tools to trigger climate action},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Are quantum computers really energy efficient?
<em>NATCS</em>, <em>3</em>(6), 457–460. (<a
href="https://doi.org/10.1038/s43588-023-00459-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proponents often tout quantum computing as a more energy efficient alternative to classical computing methods. However, the extent to which it can reduce energy usage remains unclear, as experts have not yet agreed on metrics to determine its energy consumption.},
  archive      = {J_NATCS},
  author       = {Chen, Sophia},
  doi          = {10.1038/s43588-023-00459-6},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {457-460},
  shortjournal = {Nat. Comput. Sci},
  title        = {Are quantum computers really energy efficient?},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A computational quest for a sustainable world.
<em>NATCS</em>, <em>3</em>(6), 455–456. (<a
href="https://doi.org/10.1038/s43588-023-00483-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This month’s Focus issue highlights ongoing work by computational scientists to help address the Sustainable Development Goals, as well as discusses how the sustainability of computational science itself can be improved.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00483-6},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {455-456},
  shortjournal = {Nat. Comput. Sci},
  title        = {A computational quest for a sustainable world},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). An additive framework for kirigami design. <em>NATCS</em>,
<em>3</em>(5), 443–454. (<a
href="https://doi.org/10.1038/s43588-023-00448-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present an additive approach for the inverse design of kirigami-based mechanical metamaterials by focusing on the empty (negative) spaces instead of the solid tiles. By considering each negative space as a four-bar linkage, we identify a simple recursive relationship between adjacent linkages, yielding an efficient method for creating kirigami patterns. This allows us to solve the kirigami design problem using elementary linear algebra, with compatibility, reconfigurability and rigid-deployability encoded into an iterative procedure involving simple matrix multiplications. The resulting linear design strategy circumvents the solution of a non-convex global optimization problem and allows us to control the degrees of freedom in the deployment angle field, linkage offsets and boundary conditions. We demonstrate this by creating a large variety of rigid-deployable, compact, reconfigurable kirigami patterns. We then realize our kirigami designs physically using two simple but effective fabrication strategies with very different materials. Altogether, our additive approaches present routes for efficient mechanical metamaterial design and fabrication based on ori/kirigami art forms. Kirigami is an ancient art form that is now increasingly studied and applied in science and technology. This work presents an additive approach for the computational design of kirigami and two fabrication strategies for its physical instantiation.},
  archive      = {J_NATCS},
  author       = {Dudte, Levi H. and Choi, Gary P. T. and Becker, Kaitlyn P. and Mahadevan, L.},
  doi          = {10.1038/s43588-023-00448-9},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {443-454},
  shortjournal = {Nat. Comput. Sci},
  title        = {An additive framework for kirigami design},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fast evaluation of the adsorption energy of organic
molecules on metals via graph neural networks. <em>NATCS</em>,
<em>3</em>(5), 433–442. (<a
href="https://doi.org/10.1038/s43588-023-00437-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modeling in heterogeneous catalysis requires the extensive evaluation of the energy of molecules adsorbed on surfaces. This is done via density functional theory but for large organic molecules it requires enormous computational time, compromising the viability of the approach. Here we present GAME-Net, a graph neural network to quickly evaluate the adsorption energy. GAME-Net is trained on a well-balanced chemically diverse dataset with C1–4 molecules with functional groups including N, O, S and C6–10 aromatic rings. The model yields a mean absolute error of 0.18 eV on the test set and is 6 orders of magnitude faster than density functional theory. Applied to biomass and plastics (up to 30 heteroatoms), adsorption energies are predicted with a mean absolute error of 0.016 eV per atom. The framework represents a tool for the fast screening of catalytic materials, particularly for systems that cannot be simulated by traditional methods. GAME-Net is a graph deep learning model trained with small molecules containing a wide set of functional groups for predicting the adsorption energy of closed-shell organic molecules on metal surfaces, avoiding expensive density functional theory simulations.},
  archive      = {J_NATCS},
  author       = {Pablo-García, Sergio and Morandi, Santiago and Vargas-Hernández, Rodrigo A. and Jorner, Kjell and Ivković, Žarko and López, Núria and Aspuru-Guzik, Alán},
  doi          = {10.1038/s43588-023-00437-y},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {433-442},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fast evaluation of the adsorption energy of organic molecules on metals via graph neural networks},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Decoding reward–curiosity conflict in decision-making from
irrational behaviors. <em>NATCS</em>, <em>3</em>(5), 418–432. (<a
href="https://doi.org/10.1038/s43588-023-00439-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Humans and animals are not always rational. They not only rationally exploit rewards but also explore an environment owing to their curiosity. However, the mechanism of such curiosity-driven irrational behavior is largely unknown. Here, we developed a decision-making model for a two-choice task based on the free energy principle, which is a theory integrating recognition and action selection. The model describes irrational behaviors depending on the curiosity level. We also proposed a machine learning method to decode temporal curiosity from behavioral data. By applying it to rat behavioral data, we found that the rat had negative curiosity, reflecting conservative selection sticking to more certain options and that the level of curiosity was upregulated by the expected future information obtained from an uncertain environment. Our decoding approach can be a fundamental tool for identifying the neural basis for reward–curiosity conflicts. Furthermore, it could be effective in diagnosing mental disorders. Mental conflict has been regarded as subjective instead of quantitative. This study developed a data-driven method to decode temporal dynamics of conflict between reward and curiosity, which can elucidate mechanisms of irrational decision-making.},
  archive      = {J_NATCS},
  author       = {Konaka, Yuki and Naoki, Honda},
  doi          = {10.1038/s43588-023-00439-w},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {418-432},
  shortjournal = {Nat. Comput. Sci},
  title        = {Decoding reward–curiosity conflict in decision-making from irrational behaviors},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). The high-dimensional space of human diseases built from
diagnosis records and mapped to genetic loci. <em>NATCS</em>,
<em>3</em>(5), 403–417. (<a
href="https://doi.org/10.1038/s43588-023-00453-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human diseases are traditionally studied as singular, independent entities, limiting researchers’ capacity to view human illnesses as dependent states in a complex, homeostatic system. Here, using time-stamped clinical records of over 151 million unique Americans, we construct a disease representation as points in a continuous, high-dimensional space, where diseases with similar etiology and manifestations lie near one another. We use the UK Biobank cohort, with half a million participants, to perform a genome-wide association study of newly defined human quantitative traits reflecting individuals’ health states, corresponding to patient positions in our disease space. We discover 116 genetic associations involving 108 genetic loci and then use ten disease constellations resulting from clustering analysis of diseases in the embedding space, as well as 30 common diseases, to demonstrate that these genetic associations can be used to robustly predict various morbidities. A disease space is constructed from clinical records by embedding all diseases and considering a patient’s space coordinates as a measure of their health state. This measure was associated with 108 genetic loci, on which models were built to predict various morbidities.},
  archive      = {J_NATCS},
  author       = {Jia, Gengjie and Li, Yu and Zhong, Xue and Wang, Kanix and Pividori, Milton and Alomairy, Rabab and Esposito, Aniello and Ltaief, Hatem and Terao, Chikashi and Akiyama, Masato and Matsuda, Koichi and Keyes, David E. and Im, Hae Kyung and Gojobori, Takashi and Kamatani, Yoichiro and Kubo, Michiaki and Cox, Nancy J. and Evans, James and Gao, Xin and Rzhetsky, Andrey},
  doi          = {10.1038/s43588-023-00453-y},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {403-417},
  shortjournal = {Nat. Comput. Sci},
  title        = {The high-dimensional space of human diseases built from diagnosis records and mapped to genetic loci},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Domain-agnostic predictions of nanoscale interactions in
proteins and nanoparticles. <em>NATCS</em>, <em>3</em>(5), 393–402. (<a
href="https://doi.org/10.1038/s43588-023-00438-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although challenging, the accurate and rapid prediction of nanoscale interactions has broad applications for numerous biological processes and material properties. While several models have been developed to predict the interaction of specific biological components, they use system-specific information that hinders their application to more general materials. Here we present NeCLAS, a general and efficient machine learning pipeline that predicts the location of nanoscale interactions, providing human-intelligible predictions. NeCLAS outperforms current nanoscale prediction models for generic nanoparticles up to 10–20 nm, reproducing interactions for biological and non-biological systems. Two aspects contribute to these results: a low-dimensional representation of nanoparticles and molecules (to reduce the effect of data uncertainty), and environmental features (to encode the physicochemical neighborhood at multiple scales). This framework has several applications, from basic research to rapid prototyping and design in nanobiotechnology. NeCLAS is a machine learning pipeline that can accurately and efficiently predict nanoscale interactions, which has broad applications in biological processes and material properties.},
  archive      = {J_NATCS},
  author       = {Saldinger, Jacob Charles and Raymond, Matt and Elvati, Paolo and Violi, Angela},
  doi          = {10.1038/s43588-023-00438-x},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {393-402},
  shortjournal = {Nat. Comput. Sci},
  title        = {Domain-agnostic predictions of nanoscale interactions in proteins and nanoparticles},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Score-based generative modeling for de novo protein design.
<em>NATCS</em>, <em>3</em>(5), 382–392. (<a
href="https://doi.org/10.1038/s43588-023-00440-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The generation of de novo protein structures with predefined functions and properties remains a challenging problem in protein design. Diffusion models, also known as score-based generative models (SGMs), have recently exhibited astounding empirical performance in image synthesis. Here we use image-based representations of protein structure to develop ProteinSGM, a score-based generative model that produces realistic de novo proteins. Through unconditional generation, we show that ProteinSGM can generate native-like protein structures, surpassing the performance of previously reported generative models. We experimentally validate some de novo designs and observe secondary structure compositions consistent with generated backbones. Finally, we apply conditional generation to de novo protein design by formulating it as an image inpainting problem, allowing precise and modular design of protein structure. This study proposes a diffusion model, ProteinSGM, for the design of novel protein folds. The designed proteins are diverse, experimentally stable and structurally consistent with predicted models},
  archive      = {J_NATCS},
  author       = {Lee, Jin Sub and Kim, Jisun and Kim, Philip M.},
  doi          = {10.1038/s43588-023-00440-3},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {382-392},
  shortjournal = {Nat. Comput. Sci},
  title        = {Score-based generative modeling for de novo protein design},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The role of complexity for digital twins of cities.
<em>NATCS</em>, <em>3</em>(5), 374–381. (<a
href="https://doi.org/10.1038/s43588-023-00431-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We argue that theories and methods drawn from complexity science are urgently needed to guide the development and use of digital twins for cities. The theoretical framework from complexity science takes into account both the short-term and the long-term dynamics of cities and their interactions. This is the foundation for a new approach that treats cities not as large machines or logistic systems but as mutually interwoven self-organizing phenomena, which evolve, to an extent, like living systems. While digital twins have been recently used to represent cities and their physical structures, integrating complexity science into the digital twin approach will be key to deliver more explicable and trustworthy models and results.},
  archive      = {J_NATCS},
  author       = {Caldarelli, G. and Arcaute, E. and Barthelemy, M. and Batty, M. and Gershenson, C. and Helbing, D. and Mancuso, S. and Moreno, Y. and Ramasco, J. J. and Rozenblat, C. and Sánchez, A. and Fernández-Villacañas, J. L.},
  doi          = {10.1038/s43588-023-00431-4},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {374-381},
  shortjournal = {Nat. Comput. Sci},
  title        = {The role of complexity for digital twins of cities},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A graph neural network for predicting the adsorption energy
of molecules on metal surfaces. <em>NATCS</em>, <em>3</em>(5), 372–373.
(<a href="https://doi.org/10.1038/s43588-023-00449-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A graph neural network — GAME-Net — has been developed to predict the adsorption energy of organic molecules on metal surfaces, which is a key descriptor of heterogeneous catalytic activity. This method allows for the study of large molecules derived from raw materials such as plastic waste, avoiding the use of costly and time-intensive first-principles simulations.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00449-8},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {372-373},
  shortjournal = {Nat. Comput. Sci},
  title        = {A graph neural network for predicting the adsorption energy of molecules on metal surfaces},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A machine learning-based model for the quantification of
mental conflict. <em>NATCS</em>, <em>3</em>(5), 370–371. (<a
href="https://doi.org/10.1038/s43588-023-00444-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We often encounter mental conflict in our lives. Such mental conflict has long been regarded as subjective. However, a machine learning method can be used to quantify the temporal dynamics of conflict between reward and curiosity from behavioral time-series.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00444-z},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {370-371},
  shortjournal = {Nat. Comput. Sci},
  title        = {A machine learning-based model for the quantification of mental conflict},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Discovering kirigami patterns. <em>NATCS</em>,
<em>3</em>(5), 368–369. (<a
href="https://doi.org/10.1038/s43588-023-00452-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A computational tool based on an additive approach and linear algebra has been developed together with a fabrication strategy for the systematic exploration of rigid-deployable, compact and reconfigurable kirigami patterns.},
  archive      = {J_NATCS},
  author       = {Corigliano, Alberto},
  doi          = {10.1038/s43588-023-00452-z},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {368-369},
  shortjournal = {Nat. Comput. Sci},
  title        = {Discovering kirigami patterns},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). From noise to protein with image models. <em>NATCS</em>,
<em>3</em>(5), 366–367. (<a
href="https://doi.org/10.1038/s43588-023-00454-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An image-inspired deep-learning model is developed to generate realistic de novo protein structures and scaffolds around functional sites, which helps the search for new structures and functions in protein engineering.},
  archive      = {J_NATCS},
  author       = {Amini, Ava P. and Yang, Kevin K.},
  doi          = {10.1038/s43588-023-00454-x},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {366-367},
  shortjournal = {Nat. Comput. Sci},
  title        = {From noise to protein with image models},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Exploring robust pattern formation. <em>NATCS</em>,
<em>3</em>(5), 365. (<a
href="https://doi.org/10.1038/s43588-023-00463-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00463-w},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {365},
  shortjournal = {Nat. Comput. Sci},
  title        = {Exploring robust pattern formation},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The promise and pitfalls of AI for molecular and materials
synthesis. <em>NATCS</em>, <em>3</em>(5), 362–364. (<a
href="https://doi.org/10.1038/s43588-023-00446-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As artificial intelligence (AI) proliferates, synthetic chemistry stands to benefit from its progress. Despite hidden variables and ‘unknown unknowns’ in datasets that may impede the realization of a digital twin for the laboratory flask, there are many opportunities to leverage AI and large datasets to advance synthesis science.},
  archive      = {J_NATCS},
  author       = {David, Nicholas and Sun, Wenhao and Coley, Connor W.},
  doi          = {10.1038/s43588-023-00446-x},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {362-364},
  shortjournal = {Nat. Comput. Sci},
  title        = {The promise and pitfalls of AI for molecular and materials synthesis},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Experimental validation, anyone? <em>NATCS</em>,
<em>3</em>(5), 361. (<a
href="https://doi.org/10.1038/s43588-023-00462-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Even though Nature Computational Science is a computational-focused journal, some studies submitted to our journal might require experimental validation in order to verify the reported results and to demonstrate the usefulness of the proposed methods.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00462-x},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {361},
  shortjournal = {Nat. Comput. Sci},
  title        = {Experimental validation, anyone?},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Large-scale correlation network construction for unraveling
the coordination of complex biological systems. <em>NATCS</em>,
<em>3</em>(4), 346–359. (<a
href="https://doi.org/10.1038/s43588-023-00429-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advanced measurement and data storage technologies have enabled high-dimensional profiling of complex biological systems. For this, modern multiomics studies regularly produce datasets with hundreds of thousands of measurements per sample, enabling a new era of precision medicine. Correlation analysis is an important first step to gain deeper insights into the coordination and underlying processes of such complex systems. However, the construction of large correlation networks in modern high-dimensional datasets remains a major computational challenge owing to rapidly growing runtime and memory requirements. Here we address this challenge by introducing CorALS (Correlation Analysis of Large-scale (biological) Systems), an open-source framework for the construction and analysis of large-scale parametric as well as non-parametric correlation networks for high-dimensional biological data. It features off-the-shelf algorithms suitable for both personal and high-performance computers, enabling workflows and downstream analysis approaches. We illustrate the broad scope and potential of CorALS by exploring perspectives on complex biological processes in large-scale multiomics and single-cell studies. An approach, CorALS, is proposed to enable the construction and analysis of large-scale correlation networks for high-dimensional biological data as an open-source framework in Python.},
  archive      = {J_NATCS},
  author       = {Becker, Martin and Nassar, Huda and Espinosa, Camilo and Stelzer, Ina A. and Feyaerts, Dorien and Berson, Eloise and Bidoki, Neda H. and Chang, Alan L. and Saarunya, Geetha and Culos, Anthony and De Francesco, Davide and Fallahzadeh, Ramin and Liu, Qun and Kim, Yeasul and Marić, Ivana and Mataraso, Samson J. and Payrovnaziri, Seyedeh Neelufar and Phongpreecha, Thanaphong and Ravindra, Neal G. and Stanley, Natalie and Shome, Sayane and Tan, Yuqi and Thuraiappah, Melan and Xenochristou, Maria and Xue, Lei and Shaw, Gary and Stevenson, David and Angst, Martin S. and Gaudilliere, Brice and Aghaeepour, Nima},
  doi          = {10.1038/s43588-023-00429-y},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {346-359},
  shortjournal = {Nat. Comput. Sci},
  title        = {Large-scale correlation network construction for unraveling the coordination of complex biological systems},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Machine-guided path sampling to discover mechanisms of
molecular self-organization. <em>NATCS</em>, <em>3</em>(4), 334–345. (<a
href="https://doi.org/10.1038/s43588-023-00428-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular self-organization driven by concerted many-body interactions produces the ordered structures that define both inanimate and living matter. Here we present an autonomous path sampling algorithm that integrates deep learning and transition path theory to discover the mechanism of molecular self-organization phenomena. The algorithm uses the outcome of newly initiated trajectories to construct, validate and—if needed—update quantitative mechanistic models. Closing the learning cycle, the models guide the sampling to enhance the sampling of rare assembly events. Symbolic regression condenses the learned mechanism into a human-interpretable form in terms of relevant physical observables. Applied to ion association in solution, gas-hydrate crystal formation, polymer folding and membrane-protein assembly, we capture the many-body solvent motions governing the assembly process, identify the variables of classical nucleation theory, uncover the folding mechanism at different levels of resolution and reveal competing assembly pathways. The mechanistic descriptions are transferable across thermodynamic states and chemical space. A machine learning algorithm speeds up the sampling of rare assembly events, discovers their mechanisms, extrapolates them across chemical and thermodynamic space, and condenses the learned assembly mechanisms into a human-interpretable form.},
  archive      = {J_NATCS},
  author       = {Jung, Hendrik and Covino, Roberto and Arjun, A. and Leitold, Christian and Dellago, Christoph and Bolhuis, Peter G. and Hummer, Gerhard},
  doi          = {10.1038/s43588-023-00428-z},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {334-345},
  shortjournal = {Nat. Comput. Sci},
  title        = {Machine-guided path sampling to discover mechanisms of molecular self-organization},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Quantum harmonic free energies for biomolecules and
nanomaterials. <em>NATCS</em>, <em>3</em>(4), 328–333. (<a
href="https://doi.org/10.1038/s43588-023-00432-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Obtaining the free energy of large molecules from quantum mechanical energy functions is a long-standing challenge. We describe a method that allows us to estimate, at the quantum mechanical level, the harmonic contributions to the thermodynamics of molecular systems of large size, with modest cost. Using this approach, we compute the vibrational thermodynamics of a series of diamond nanocrystals, and show that the error per atom decreases with system size in the limit of large systems. We further show that we can obtain the vibrational contributions to the binding free energies of prototypical protein–ligand complexes where exact computation is too expensive to be practical. Our work raises the possibility of routine quantum mechanical estimates of thermodynamic quantities in complex systems. A method to compute the quantum harmonic free energy contributions in large materials and biomolecular simulations at a reasonable cost is proposed, making quantum mechanical estimates of thermodynamic quantities possible for complex systems.},
  archive      = {J_NATCS},
  author       = {White, Alec F. and Li, Chenghan and Zhang, Xing and Chan, Garnet Kin-Lic},
  doi          = {10.1038/s43588-023-00432-3},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {328-333},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantum harmonic free energies for biomolecules and nanomaterials},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Deep-learning electronic-structure calculation of magnetic
superstructures. <em>NATCS</em>, <em>3</em>(4), 321–327. (<a
href="https://doi.org/10.1038/s43588-023-00424-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ab initio studies of magnetic superstructures are indispensable to research on emergent quantum materials, but are currently bottlenecked by the formidable computational cost. Here, to break this bottleneck, we have developed a deep equivariant neural network framework to represent the density functional theory Hamiltonian of magnetic materials for efficient electronic-structure calculation. A neural network architecture incorporating a priori knowledge of fundamental physical principles, especially the nearsightedness principle and the equivariance requirements of Euclidean and time-reversal symmetries ( $$E(3)\times {I,{{{\mathcal{T}}}}}$$ ), is designed, which is critical to capture the subtle magnetic effects. Systematic experiments on spin-spiral, nanotube and moiré magnets were performed, making the challenging study of magnetic skyrmions feasible. A deep learning ab initio method for studying magnetic materials is developed, reducing the computational cost and opening opportunities to predict the electronic properties of magnetic superstructures, such as magnetic skyrmions.},
  archive      = {J_NATCS},
  author       = {Li, He and Tang, Zechen and Gong, Xiaoxun and Zou, Nianlong and Duan, Wenhui and Xu, Yong},
  doi          = {10.1038/s43588-023-00424-3},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {321-327},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep-learning electronic-structure calculation of magnetic superstructures},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Moiré magnetic exchange interactions in twisted magnets.
<em>NATCS</em>, <em>3</em>(4), 314–320. (<a
href="https://doi.org/10.1038/s43588-023-00430-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In addition to moiré superlattices, twisting can also generate moiré magnetic exchange interactions (MMEIs) in van der Waals magnets. However, owing to the extreme complexity and twist-angle-dependent sensitivity, all existing models fail to fully capture MMEIs and thus cannot provide an understanding of MMEI-induced physics. Here, we develop a microscopic moiré spin Hamiltonian that enables the effective description of MMEIs via a sliding-mapping approach in twisted magnets, as demonstrated in twisted bilayer CrI3. We show that the emergence of MMEIs can create a magnetic skyrmion bubble with non-conserved helicity, a ‘moiré-type skyrmion bubble’. This represents a unique spin texture solely generated by MMEIs and ready to be detected under the current experimental conditions. Importantly, the size and population of skyrmion bubbles can be finely controlled by twist angle, a key step for skyrmion-based information storage. Furthermore, we reveal that MMEIs can be effectively manipulated by substrate-induced interfacial Dzyaloshinskii–Moriya interactions, modulating the twist-angle-dependent magnetic phase diagram, which solves outstanding disagreements between theories and experiments. A microscopic moiré spin model that enables the description of moiré magnetic exchange interactions via a sliding-mapping method is proposed. The twist-angle and substrate-influenced magnetic phase diagram addresses disagreements between theories and experiments.},
  archive      = {J_NATCS},
  author       = {Yang, Baishun and Li, Yang and Xiang, Hongjun and Lin, Haiqing and Huang, Bing},
  doi          = {10.1038/s43588-023-00430-5},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {314-320},
  shortjournal = {Nat. Comput. Sci},
  title        = {Moiré magnetic exchange interactions in twisted magnets},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Unraveling non-genetic heterogeneity in cancer with
dynamical models and computational tools. <em>NATCS</em>, <em>3</em>(4),
301–313. (<a href="https://doi.org/10.1038/s43588-023-00427-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Individual cells within an otherwise genetically homogenous population constantly undergo fluctuations in their molecular state, giving rise to non-genetic heterogeneity. Such diversity is being increasingly implicated in cancer therapy resistance and metastasis. Identifying the origins of non-genetic heterogeneity is therefore crucial for making clinical breakthroughs. We discuss with examples how dynamical models and computational tools have provided critical multiscale insights into the nature and consequences of non-genetic heterogeneity in cancer. We demonstrate how mechanistic modeling has been pivotal in establishing key concepts underlying non-genetic diversity at various biological scales, from population dynamics to gene regulatory networks. We discuss advances in single-cell longitudinal profiling techniques to reveal patterns of non-genetic heterogeneity, highlighting the ongoing efforts and challenges in statistical frameworks to robustly interpret such multimodal datasets. Moving forward, we stress the need for data-driven statistical and mechanistically motivated dynamical frameworks to come together to develop predictive cancer models and inform therapeutic strategies. Identifying the origins and nature of non-genetic heterogeneity in cancer has widespread clinical ramifications. In this Review, the authors discuss how computational models and tools have been used to provide insights into this phenomenon and how they can help tackle the disease in the future.},
  archive      = {J_NATCS},
  author       = {Pillai, Maalavika and Hojel, Emilia and Jolly, Mohit Kumar and Goyal, Yogesh},
  doi          = {10.1038/s43588-023-00427-0},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {301-313},
  shortjournal = {Nat. Comput. Sci},
  title        = {Unraveling non-genetic heterogeneity in cancer with dynamical models and computational tools},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Exploring proton-coupled electron transfer at multiple
scales. <em>NATCS</em>, <em>3</em>(4), 291–300. (<a
href="https://doi.org/10.1038/s43588-023-00422-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The coupling of electron and proton transfer is critical for chemical and biological processes spanning a wide range of length and time scales and often occurring in complex environments. Thus, diverse modeling strategies, including analytical theories, quantum chemistry, molecular dynamics and kinetic modeling, are essential for a comprehensive understanding of such proton-coupled electron transfer reactions. Each of these computational methods provides one piece of the puzzle, and all these pieces must be viewed together to produce the full picture. Proton-coupled electron transfer occurs at a variety of length and time scales and often in complex environments. This Perspective summarizes a range of modeling strategies that can be used together to address remaining challenges and provide a better understanding of such reactions.},
  archive      = {J_NATCS},
  author       = {Hammes-Schiffer, Sharon},
  doi          = {10.1038/s43588-023-00422-5},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {291-300},
  shortjournal = {Nat. Comput. Sci},
  title        = {Exploring proton-coupled electron transfer at multiple scales},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A machine learning algorithm for studying how molecules
self-assemble and function. <em>NATCS</em>, <em>3</em>(4), 289–290. (<a
href="https://doi.org/10.1038/s43588-023-00441-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A machine learning algorithm has been developed to capture and analyze rare molecular processes, revealing how molecules self-organize and function. The algorithm is general and can be applied whenever a dynamic system has a notion of ‘likely fate’.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00441-2},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {289-290},
  shortjournal = {Nat. Comput. Sci},
  title        = {A machine learning algorithm for studying how molecules self-assemble and function},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A deep-learning method for studying magnetic
superstructures. <em>NATCS</em>, <em>3</em>(4), 287–288. (<a
href="https://doi.org/10.1038/s43588-023-00425-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A rotational and time-reversal equivariant neural network designed to represent the spin–orbital density functional theory Hamiltonian as a function of the atomic and magnetic structure enables ab initio electronic-structure calculations of magnetic superstructures. These calculations can efficiently and accurately predict subtle magnetic effects in various chemical environments.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00425-2},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {287-288},
  shortjournal = {Nat. Comput. Sci},
  title        = {A deep-learning method for studying magnetic superstructures},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Omics correlation for efficient network construction.
<em>NATCS</em>, <em>3</em>(4), 285–286. (<a
href="https://doi.org/10.1038/s43588-023-00436-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Discovering biological patterns from omics data is challenging due to the high dimensionality of biological data. A computational framework is presented to more efficiently calculate correlations among omics features and to build networks by estimating important connections.},
  archive      = {J_NATCS},
  author       = {Rahnavard, Ali},
  doi          = {10.1038/s43588-023-00436-z},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {285-286},
  shortjournal = {Nat. Comput. Sci},
  title        = {Omics correlation for efficient network construction},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Uncovering magnetic interactions in moiré magnets.
<em>NATCS</em>, <em>3</em>(4), 282–284. (<a
href="https://doi.org/10.1038/s43588-023-00434-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two computational methods — one physics-based, and the other one deep-learning based — are proposed to enable the systematic investigation of magnetic order in moiré magnets from first principles.},
  archive      = {J_NATCS},
  author       = {Soriano, David},
  doi          = {10.1038/s43588-023-00434-1},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {282-284},
  shortjournal = {Nat. Comput. Sci},
  title        = {Uncovering magnetic interactions in moiré magnets},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Learning properties of metal complexes. <em>NATCS</em>,
<em>3</em>(4), 281. (<a
href="https://doi.org/10.1038/s43588-023-00443-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00443-0},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {281},
  shortjournal = {Nat. Comput. Sci},
  title        = {Learning properties of metal complexes},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023i). Transfer learning for metal–organic frameworks.
<em>NATCS</em>, <em>3</em>(4), 280. (<a
href="https://doi.org/10.1038/s43588-023-00447-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-023-00447-w},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {280},
  shortjournal = {Nat. Comput. Sci},
  title        = {Transfer learning for metal–organic frameworks},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023j). Moving toward safer driverless vehicles. <em>NATCS</em>,
<em>3</em>(4), 279. (<a
href="https://doi.org/10.1038/s43588-023-00442-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00442-1},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {279},
  shortjournal = {Nat. Comput. Sci},
  title        = {Moving toward safer driverless vehicles},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Computationally probing moiré magnets. <em>NATCS</em>,
<em>3</em>(4), 277–278. (<a
href="https://doi.org/10.1038/s43588-023-00451-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We highlight two primary research papers, published in this issue of Nature Computational Science, on computational methods for moiré magnets.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00451-0},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {277-278},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computationally probing moiré magnets},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Full-scale scaffold model of the human hippocampus CA1 area.
<em>NATCS</em>, <em>3</em>(3), 264–276. (<a
href="https://doi.org/10.1038/s43588-023-00417-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The increasing availability of quantitative data on the human brain is opening new avenues to study neural function and dysfunction, thus bringing us closer and closer to the implementation of digital twin applications for personalized medicine. Here we provide a resource to the neuroscience community: a computational method to generate full-scale scaffold model of human brain regions starting from microscopy images. We have benchmarked the method to reconstruct the CA1 region of a right human hippocampus, which accounts for about half of the entire right hippocampal formation. Together with 3D soma positioning we provide a connectivity matrix generated using a morpho-anatomical connection strategy based on axonal and dendritic probability density functions accounting for morphological properties of hippocampal neurons. The data and algorithms are supplied in a ready-to-use format, suited to implement computational models at different scales and detail. A computational method is proposed to generate the full-scale dataset of the tridimensional position and connectivity of neurons in the CA1 region of the human hippocampus starting from high-resolution microscopy images and experimental data.},
  archive      = {J_NATCS},
  author       = {Gandolfi, Daniela and Mapelli, Jonathan and Solinas, Sergio M. G. and Triebkorn, Paul and D’Angelo, Egidio and Jirsa, Viktor and Migliore, Michele},
  doi          = {10.1038/s43588-023-00417-2},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {264-276},
  shortjournal = {Nat. Comput. Sci},
  title        = {Full-scale scaffold model of the human hippocampus CA1 area},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Statistical modeling of adaptive neural networks explains
co-existence of avalanches and oscillations in resting human brain.
<em>NATCS</em>, <em>3</em>(3), 254–263. (<a
href="https://doi.org/10.1038/s43588-023-00410-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neurons in the brain are wired into adaptive networks that exhibit collective dynamics as diverse as scale-specific oscillations and scale-free neuronal avalanches. Although existing models account for oscillations and avalanches separately, they typically do not explain both phenomena, are too complex to analyze analytically or intractable to infer from data rigorously. Here we propose a feedback-driven Ising-like class of neural networks that captures avalanches and oscillations simultaneously and quantitatively. In the simplest yet fully microscopic model version, we can analytically compute the phase diagram and make direct contact with human brain resting-state activity recordings via tractable inference of the model’s two essential parameters. The inferred model quantitatively captures the dynamics over a broad range of scales, from single sensor oscillations to collective behaviors of extreme events and neuronal avalanches. Importantly, the inferred parameters indicate that the co-existence of scale-specific (oscillations) and scale-free (avalanches) dynamics occurs close to a non-equilibrium critical point at the onset of self-sustained oscillations. The study shows that scale-specific oscillations and scale-free neuronal avalanches in resting brains co-exist in the simplest model of an adaptive neural network close to a non-equilibrium critical point at the onset of self-sustained oscillations.},
  archive      = {J_NATCS},
  author       = {Lombardi, Fabrizio and Pepić, Selver and Shriki, Oren and Tkačik, Gašper and De Martino, Daniele},
  doi          = {10.1038/s43588-023-00410-9},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {254-263},
  shortjournal = {Nat. Comput. Sci},
  title        = {Statistical modeling of adaptive neural networks explains co-existence of avalanches and oscillations in resting human brain},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multi-view manifold learning of human brain-state
trajectories. <em>NATCS</em>, <em>3</em>(3), 240–253. (<a
href="https://doi.org/10.1038/s43588-023-00419-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The complexity of the human brain gives the illusion that brain activity is intrinsically high-dimensional. Nonlinear dimensionality-reduction methods such as uniform manifold approximation and t-distributed stochastic neighbor embedding have been used for high-throughput biomedical data. However, they have not been used extensively for brain activity data such as those from functional magnetic resonance imaging (fMRI), primarily due to their inability to maintain dynamic structure. Here we introduce a nonlinear manifold learning method for time-series data—including those from fMRI—called temporal potential of heat-diffusion for affinity-based transition embedding (T-PHATE). In addition to recovering a low-dimensional intrinsic manifold geometry from time-series data, T-PHATE exploits the data’s autocorrelative structure to faithfully denoise and unveil dynamic trajectories. We empirically validate T-PHATE on three fMRI datasets, showing that it greatly improves data visualization, classification, and segmentation of the data relative to several other state-of-the-art dimensionality-reduction benchmarks. These improvements suggest many potential applications of T-PHATE to other high-dimensional datasets of temporally diffuse processes. A manifold learning method called T-PHATE is developed for high-dimensional time-series data. T-PHATE is applied to brain data (functional magnetic resonance imaging) where it faithfully denoises signals and unveils latent brain-state trajectories which correspond with cognitive processing.},
  archive      = {J_NATCS},
  author       = {Busch, Erica L. and Huang, Jessie and Benz, Andrew and Wallenstein, Tom and Lajoie, Guillaume and Wolf, Guy and Krishnaswamy, Smita and Turk-Browne, Nicholas B.},
  doi          = {10.1038/s43588-023-00419-0},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {240-253},
  shortjournal = {Nat. Comput. Sci},
  title        = {Multi-view manifold learning of human brain-state trajectories},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Uncertainty-driven dynamics for active learning of
interatomic potentials. <em>NATCS</em>, <em>3</em>(3), 230–239. (<a
href="https://doi.org/10.1038/s43588-023-00406-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning (ML) models, if trained to data sets of high-fidelity quantum simulations, produce accurate and efficient interatomic potentials. Active learning (AL) is a powerful tool to iteratively generate diverse data sets. In this approach, the ML model provides an uncertainty estimate along with its prediction for each new atomic configuration. If the uncertainty estimate passes a certain threshold, then the configuration is included in the data set. Here we develop a strategy to more rapidly discover configurations that meaningfully augment the training data set. The approach, uncertainty-driven dynamics for active learning (UDD-AL), modifies the potential energy surface used in molecular dynamics simulations to favor regions of configuration space for which there is large model uncertainty. The performance of UDD-AL is demonstrated for two AL tasks: sampling the conformational space of glycine and sampling the promotion of proton transfer in acetylacetone. The method is shown to efficiently explore the chemically relevant configuration space, which may be inaccessible using regular dynamical sampling at target temperature conditions. A biasing energy derived from the uncertainty of a neural network ensemble modifies the potential energy surface in molecular dynamics simulations to rapidly discover under-represented structural regions that meaningfully augment the training data set.},
  archive      = {J_NATCS},
  author       = {Kulichenko, Maksim and Barros, Kipton and Lubbers, Nicholas and Li, Ying Wai and Messerly, Richard and Tretiak, Sergei and Smith, Justin S. and Nebgen, Benjamin},
  doi          = {10.1038/s43588-023-00406-5},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {230-239},
  shortjournal = {Nat. Comput. Sci},
  title        = {Uncertainty-driven dynamics for active learning of interatomic potentials},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Simulating short-range order in compositionally complex
materials. <em>NATCS</em>, <em>3</em>(3), 221–229. (<a
href="https://doi.org/10.1038/s43588-023-00407-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In multicomponent materials, short-range order (SRO) is the development of correlated arrangements of atoms at the nanometer scale. Its impact in compositionally complex materials has stimulated an intense debate within the materials science community. Understanding SRO is critical to control the properties of technologically relevant materials, from metallic alloys to functional ceramics. In contrast to long-range order, quantitative characterization of the nature and spatial extent of SRO evades most of the experimentally available techniques. Simulations at the atomistic scale have full access to SRO but face the challenge of accurately sampling high-dimensional configuration spaces to identify the thermodynamic and kinetic conditions at which SRO is formed and what impact it has on material properties. Here we highlight recent progress in computational approaches, such as machine learning-based interatomic potentials, for quantifying and understanding SRO in compositionally complex materials. We briefly recap the key theoretical concepts and methods. The computational characterization of short-range order in compositionally complex materials relies on effective interatomic potentials. In this Review, challenges and opportunities in developing advanced potentials for such systems are discussed, with a focus on machine learning-based potentials.},
  archive      = {J_NATCS},
  author       = {Ferrari, Alberto and Körmann, Fritz and Asta, Mark and Neugebauer, Jörg},
  doi          = {10.1038/s43588-023-00407-4},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {221-229},
  shortjournal = {Nat. Comput. Sci},
  title        = {Simulating short-range order in compositionally complex materials},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Defect modeling and control in structurally and
compositionally complex materials. <em>NATCS</em>, <em>3</em>(3),
210–220. (<a href="https://doi.org/10.1038/s43588-023-00403-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Conventional computational approaches for modeling defects face difficulties when applied to complex materials, mainly due to the vast configurational space of defects. In this Perspective, we discuss the challenges in calculating defect properties in complex materials, review recent advances in computational techniques and showcase new mechanistic insights developed from these methods. We further discuss the remaining challenges in improving the accuracy and efficiency of defect modeling in complex materials, and provide an outlook on potential research directions. Complex materials offer promises for exotic materials properties that enable novel applications. Nevertheless, there are numerous computational challenges for a rational design of defects in such materials, thus inspiring opportunities for developing advanced defect models.},
  archive      = {J_NATCS},
  author       = {Zhang, Xie and Kang, Jun and Wei, Su-Huai},
  doi          = {10.1038/s43588-023-00403-8},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {210-220},
  shortjournal = {Nat. Comput. Sci},
  title        = {Defect modeling and control in structurally and compositionally complex materials},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Accelerating the design of compositionally complex materials
via physics-informed artificial intelligence. <em>NATCS</em>,
<em>3</em>(3), 198–209. (<a
href="https://doi.org/10.1038/s43588-023-00412-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The chemical space for designing materials is practically infinite. This makes disruptive progress by traditional physics-based modeling alone challenging. Yet, training data for identifying composition–structure–property relations by artificial intelligence are sparse. We discuss opportunities to discover new chemically complex materials by hybrid methods where physics laws are combined with artificial intelligence. Machine learning models have been widely applied to boost the computational efficiency of searching vast chemical space of compositionally complex materials. This Perspective summarizes the recent developments and proposes future opportunities, such as the physics-informed machine learning models.},
  archive      = {J_NATCS},
  author       = {Raabe, Dierk and Mianroodi, Jaber Rezaei and Neugebauer, Jörg},
  doi          = {10.1038/s43588-023-00412-7},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {198-209},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerating the design of compositionally complex materials via physics-informed artificial intelligence},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A single-cell-resolution mathematical model of the CA1 human
hippocampus. <em>NATCS</em>, <em>3</em>(3), 196–197. (<a
href="https://doi.org/10.1038/s43588-023-00421-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a computational method to generate a single-cell-resolution model of human brain regions starting from microscopy images. The developed method has been benchmarked to reconstruct the CA1 region of a right human hippocampus, including anatomical cell organization, connectivity, and network activity.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00421-6},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {196-197},
  shortjournal = {Nat. Comput. Sci},
  title        = {A single-cell-resolution mathematical model of the CA1 human hippocampus},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Oscillations and avalanches coexist in brain networks close
to criticality. <em>NATCS</em>, <em>3</em>(3), 194–195. (<a
href="https://doi.org/10.1038/s43588-023-00411-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We propose a minimal and analytically tractable class of neural networks, the adaptive Ising class. By inferring the model’s parameters from resting-state brain activity recordings, we show that scale-specific oscillations and scale-free avalanches can coexist in resting brains close to a non-equilibrium critical point at the onset of self-sustained oscillations.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00411-8},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {194-195},
  shortjournal = {Nat. Comput. Sci},
  title        = {Oscillations and avalanches coexist in brain networks close to criticality},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Revealing trajectories of the mind via non-linear manifolds
of brain activity. <em>NATCS</em>, <em>3</em>(3), 192–193. (<a
href="https://doi.org/10.1038/s43588-023-00423-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This work involved the design of a multi-view manifold learning algorithm that capitalizes on various types of structure in high-dimensional time-series data to model dynamic signals in low dimensions. The resulting embeddings of human functional brain imaging data unveil trajectories through brain states that predict cognitive processing during diverse experimental tasks.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00423-4},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {192-193},
  shortjournal = {Nat. Comput. Sci},
  title        = {Revealing trajectories of the mind via non-linear manifolds of brain activity},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Biasing energy surfaces towards the unknown. <em>NATCS</em>,
<em>3</em>(3), 190–191. (<a
href="https://doi.org/10.1038/s43588-023-00420-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A biasing potential is derived from the uncertainty of a neural network ensemble and used to modify the potential energy surface in molecular dynamics simulations and facilitate the determination of underrepresented structural regions.},
  archive      = {J_NATCS},
  author       = {Batzner, Simon},
  doi          = {10.1038/s43588-023-00420-7},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {190-191},
  shortjournal = {Nat. Comput. Sci},
  title        = {Biasing energy surfaces towards the unknown},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Accounting for quantum effects in catalysis.
<em>NATCS</em>, <em>3</em>(3), 189. (<a
href="https://doi.org/10.1038/s43588-023-00426-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00426-1},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {189},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accounting for quantum effects in catalysis},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023n). Understanding cancer drivers and passengers.
<em>NATCS</em>, <em>3</em>(3), 187–188. (<a
href="https://doi.org/10.1038/s43588-023-00409-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Núria López-Bigas, ICREA Research Professor and group leader in biomedical genomics at the Institute for Research in Biomedicine, discusses with Nature Computational Science about her research on cancer genomics.},
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00409-2},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {187-188},
  shortjournal = {Nat. Comput. Sci},
  title        = {Understanding cancer drivers and passengers},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Guiding element mixing. <em>NATCS</em>, <em>3</em>(3),
185–186. (<a href="https://doi.org/10.1038/s43588-023-00433-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Materials design has largely expanded to multiple compositions, which requires the mixing of an increasing number of elements. In this joint Focus issue with Nature Materials, we take a closer look at the role of computational methods for guiding exploration within such vast chemical spaces.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00433-2},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {185-186},
  shortjournal = {Nat. Comput. Sci},
  title        = {Guiding element mixing},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Gene expression model inference from snapshot RNA data using
bayesian non-parametrics. <em>NATCS</em>, <em>3</em>(2), 174–183. (<a
href="https://doi.org/10.1038/s43588-022-00392-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene expression models, which are key towards understanding cellular regulatory response, underlie observations of single-cell transcriptional dynamics. Although RNA expression data encode information on gene expression models, existing computational frameworks do not perform simultaneous Bayesian inference of gene expression models and parameters from such data. Rather, gene expression models—composed of gene states, their connectivities and associated parameters—are currently deduced by pre-specifying gene state numbers and connectivity before learning associated rate parameters. Here we propose a method to learn full distributions over gene states, state connectivities and associated rate parameters, simultaneously and self-consistently from single-molecule RNA counts. We propagate noise from fluctuating RNA counts over models by treating models themselves as random variables. We achieve this within a Bayesian non-parametric paradigm. We demonstrate our method on the Escherichia coli lacZ pathway and the Saccharomyces cerevisiae STL1 pathway, and verify its robustness on synthetic data. A method that infers gene networks and rate parameters directly from single-molecule fluorescence in situ hybridization RNA snapshot data is proposed and demonstrated on synthetic and real data, providing insights on data from S. cerevisiae and E. coli.},
  archive      = {J_NATCS},
  author       = {Kilic, Zeliha and Schweiger, Max and Moyer, Camille and Shepherd, Douglas and Pressé, Steve},
  doi          = {10.1038/s43588-022-00392-0},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {174-183},
  shortjournal = {Nat. Comput. Sci},
  title        = {Gene expression model inference from snapshot RNA data using bayesian non-parametrics},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Harnessing low dimensionality to visualize the
antibody–virus landscape for influenza. <em>NATCS</em>, <em>3</em>(2),
164–173. (<a href="https://doi.org/10.1038/s43588-022-00375-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Antibodies constitute a key line of defense against the diverse pathogens we encounter in our lives. Although the interactions between a single antibody and a single virus are routinely characterized in exquisite detail, the inherent tradeoffs between attributes such as potency and breadth remain unclear. Moreover, there is a wide gap between the discrete interactions of single antibodies and the collective behavior of antibody mixtures. Here we develop a form of antigenic cartography called a ‘neutralization landscape’ that visualizes and quantifies antibody–virus interactions for antibodies targeting the influenza hemagglutinin stem. This landscape transforms the potency–breadth tradeoff into a readily solvable geometry problem. With it, we decompose the collective neutralization from multiple antibodies to characterize the composition and functional properties of the stem antibodies within. Looking forward, this framework can leverage the serological assays routinely performed for influenza surveillance to analyze how an individual’s antibody repertoire evolves after vaccination or infection. A framework is presented to extrapolate the range of behaviors for influenza antibodies. Using this basis set of behaviors, the collective action of multiple antibodies can be teased apart to describe the individual antibodies within.},
  archive      = {J_NATCS},
  author       = {Einav, Tal and Creanga, Adrian and Andrews, Sarah F. and McDermott, Adrian B. and Kanekiyo, Masaru},
  doi          = {10.1038/s43588-022-00375-1},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {164-173},
  shortjournal = {Nat. Comput. Sci},
  title        = {Harnessing low dimensionality to visualize the antibody–virus landscape for influenza},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Persistent spectral theory-guided protein engineering.
<em>NATCS</em>, <em>3</em>(2), 149–163. (<a
href="https://doi.org/10.1038/s43588-022-00394-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although protein engineering, which iteratively optimizes protein fitness by screening the gigantic mutational space, is constrained by experimental capacity, various machine learning models have substantially expedited protein engineering. Three-dimensional protein structures promise further advantages, but their intricate geometric complexity hinders their applications in deep mutational screening. Persistent homology, an established algebraic topology tool for protein structural complexity reduction, fails to capture the homotopic shape evolution during filtration of given data. Here we introduce a Topology-offered Protein Fitness (TopFit) framework to complement protein sequence and structure embeddings. Equipped with an ensemble regression strategy, TopFit integrates the persistent spectral theory, which is a new topological Laplacian, and two auxiliary sequence embeddings to capture mutation-induced topological invariant, shape evolution and sequence disparity in the protein fitness landscape. The performance of TopFit is assessed by 34 benchmark datasets with 128,634 variants, involving a vast variety of protein structure acquisition modalities and training set size variations. A topological data analysis-driven machine learning model for guiding protein engineering is proposed, complementing protein sequence and structure embeddings when navigating the fitness landscape.},
  archive      = {J_NATCS},
  author       = {Qiu, Yuchi and Wei, Guo-Wei},
  doi          = {10.1038/s43588-022-00394-y},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {149-163},
  shortjournal = {Nat. Comput. Sci},
  title        = {Persistent spectral theory-guided protein engineering},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). High-throughput property-driven generative design of
functional organic molecules. <em>NATCS</em>, <em>3</em>(2), 139–148.
(<a href="https://doi.org/10.1038/s43588-022-00391-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The design of molecules and materials with tailored properties is challenging, as candidate molecules must satisfy multiple competing requirements that are often difficult to measure or compute. While molecular structures produced through generative deep learning will satisfy these patterns, they often only possess specific target properties by chance and not by design, which makes molecular discovery via this route inefficient. In this work, we predict molecules with (Pareto-)optimal properties by combining a generative deep learning model that predicts three-dimensional conformations of molecules with a supervised deep learning model that takes these as inputs and predicts their electronic structure. Optimization of (multiple) molecular properties is achieved by screening newly generated molecules for desirable electronic properties and reusing hit molecules to retrain the generative model with a bias. The approach is demonstrated to find optimal molecules for organic electronics applications. Our method is generally applicable and eliminates the need for quantum chemical calculations during predictions, making it suitable for high-throughput screening in materials and catalyst design. A generative deep learning model of molecular structure is combined with supervised deep learning models of molecular properties to achieve high-throughput (multi-)property-driven design of organic molecules.},
  archive      = {J_NATCS},
  author       = {Westermayr, Julia and Gilkes, Joe and Barrett, Rhyan and Maurer, Reinhard J.},
  doi          = {10.1038/s43588-022-00391-1},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {139-148},
  shortjournal = {Nat. Comput. Sci},
  title        = {High-throughput property-driven generative design of functional organic molecules},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evolving scattering networks for engineering disorder.
<em>NATCS</em>, <em>3</em>(2), 128–138. (<a
href="https://doi.org/10.1038/s43588-022-00395-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Network science provides a powerful tool for unraveling the complexities of social, technological and biological systems. Constructing networks using wave phenomena is also of great interest in devising advanced hardware for machine learning, as shown in optical neural networks. Although most wave-based networks have employed static network models, the impact of evolving models in network science provides strong motivation to apply dynamical network modeling to wave physics. Here the concept of evolving scattering networks for scattering phenomena is developed. The network is defined by links, node degrees and their evolution processes modeling multi-particle interferences, which directly determine scattering from disordered materials. I demonstrate the concept by examining network-based material classification, microstructure screening and preferential attachment in evolutions, which are applied to stealthy hyperuniformity. The results enable independent control of scattering from different length scales, revealing superdense material phases in short-range order. The proposed concept provides a bridge between wave physics and network science to resolve multiscale material complexities and open-system material design. The concept of evolving scattering networks is proposed for material design in wave physics. The concept has the potential to enable network-based material classification, microstructure screening and the design of stealthy hyperuniformity with superdense phases.},
  archive      = {J_NATCS},
  author       = {Yu, Sunkyu},
  doi          = {10.1038/s43588-022-00395-x},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {128-138},
  shortjournal = {Nat. Comput. Sci},
  title        = {Evolving scattering networks for engineering disorder},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Gene regulation meets bayesian non-parametrics.
<em>NATCS</em>, <em>3</em>(2), 126–127. (<a
href="https://doi.org/10.1038/s43588-023-00405-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inferring gene networks from discrete RNA counts across cells remains a complex problem. Following Bayesian non-parametrics, a computational framework is proposed to perform non-biased inference of transcription kinetics from single-cell RNA counting experiments.},
  archive      = {J_NATCS},
  author       = {Choubey, Sandeep},
  doi          = {10.1038/s43588-023-00405-6},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {126-127},
  shortjournal = {Nat. Comput. Sci},
  title        = {Gene regulation meets bayesian non-parametrics},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Sensing the shape of functional proteins with topology.
<em>NATCS</em>, <em>3</em>(2), 124–125. (<a
href="https://doi.org/10.1038/s43588-023-00404-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A topology-based approach is developed to better understand the relationship between protein structure and protein function.},
  archive      = {J_NATCS},
  author       = {Luo, Yunan},
  doi          = {10.1038/s43588-023-00404-7},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {124-125},
  shortjournal = {Nat. Comput. Sci},
  title        = {Sensing the shape of functional proteins with topology},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evolving wave networks for materials design. <em>NATCS</em>,
<em>3</em>(2), 122–123. (<a
href="https://doi.org/10.1038/s43588-023-00401-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A computational tool has been developed for the multiscale design of open disordered material systems, bridging network science, computational materials, and wave physics.},
  archive      = {J_NATCS},
  author       = {Jiao, Yang},
  doi          = {10.1038/s43588-023-00401-w},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {122-123},
  shortjournal = {Nat. Comput. Sci},
  title        = {Evolving wave networks for materials design},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023l). When and where the climate threshold will be reached.
<em>NATCS</em>, <em>3</em>(2), 121. (<a
href="https://doi.org/10.1038/s43588-023-00414-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00414-5},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {121},
  shortjournal = {Nat. Comput. Sci},
  title        = {When and where the climate threshold will be reached},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Effective phase analyzer learned from stock images.
<em>NATCS</em>, <em>3</em>(2), 120. (<a
href="https://doi.org/10.1038/s43588-023-00415-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-023-00415-4},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {120},
  shortjournal = {Nat. Comput. Sci},
  title        = {Effective phase analyzer learned from stock images},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Advancing organic reaction discovery. <em>NATCS</em>,
<em>3</em>(2), 119. (<a
href="https://doi.org/10.1038/s43588-023-00416-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00416-3},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {119},
  shortjournal = {Nat. Comput. Sci},
  title        = {Advancing organic reaction discovery},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). A successful few drive urban scaling. <em>NATCS</em>,
<em>3</em>(2), 118. (<a
href="https://doi.org/10.1038/s43588-023-00413-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00413-6},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {118},
  shortjournal = {Nat. Comput. Sci},
  title        = {A successful few drive urban scaling},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Advancing the science of synthesis. <em>NATCS</em>,
<em>3</em>(2), 116–117. (<a
href="https://doi.org/10.1038/s43588-023-00408-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Valentino Cooper, a Distinguished R&amp;amp;D Staff Member at Oak Ridge National Laboratory, talks to Nature Computational Science about his research on density functional theory and on designing high-entropy materials and piezoelectrics.},
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00408-3},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {116-117},
  shortjournal = {Nat. Comput. Sci},
  title        = {Advancing the science of synthesis},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Why can’t we predict earthquakes? <em>NATCS</em>,
<em>3</em>(2), 115. (<a
href="https://doi.org/10.1038/s43588-023-00418-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We discuss the challenging task of forecasting the occurrence and magnitude of earthquakes.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00418-1},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {115},
  shortjournal = {Nat. Comput. Sci},
  title        = {Why can’t we predict earthquakes?},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A machine learning route between band mapping and band
structure. <em>NATCS</em>, <em>3</em>(1), 101–114. (<a
href="https://doi.org/10.1038/s43588-022-00382-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The electronic band structure and crystal structure are the two complementary identifiers of solid-state materials. Although convenient instruments and reconstruction algorithms have made large, empirical, crystal structure databases possible, extracting the quasiparticle dispersion (closely related to band structure) from photoemission band mapping data is currently limited by the available computational methods. To cope with the growing size and scale of photoemission data, here we develop a pipeline including probabilistic machine learning and the associated data processing, optimization and evaluation methods for band-structure reconstruction, leveraging theoretical calculations. The pipeline reconstructs all 14 valence bands of a semiconductor and shows excellent performance on benchmarks and other materials datasets. The reconstruction uncovers previously inaccessible momentum-space structural information on both global and local scales, while realizing a path towards integration with materials science databases. Our approach illustrates the potential of combining machine learning and domain knowledge for scalable feature extraction in multidimensional data. A computational workflow centered on probabilistic machine learning is developed to reconstruct the energy dispersion from photoemission band-mapping data. The workflow uncovers previously inaccessible momentum-space structural information at scale.},
  archive      = {J_NATCS},
  author       = {Xian, R. Patrick and Stimper, Vincent and Zacharias, Marios and Dendzik, Maciej and Dong, Shuo and Beaulieu, Samuel and Schölkopf, Bernhard and Wolf, Martin and Rettig, Laurenz and Carbogno, Christian and Bauer, Stefan and Ernstorfer, Ralph},
  doi          = {10.1038/s43588-022-00382-2},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {101-114},
  shortjournal = {Nat. Comput. Sci},
  title        = {A machine learning route between band mapping and band structure},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dynamic visualization of high-dimensional data.
<em>NATCS</em>, <em>3</em>(1), 86–100. (<a
href="https://doi.org/10.1038/s43588-022-00380-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dimensionality reduction (DR) is commonly used to project high-dimensional data into lower dimensions for visualization, which could then generate new insights and hypotheses. However, DR algorithms introduce distortions in the visualization and cannot faithfully represent all relations in the data. Thus, there is a need for methods to assess the reliability of DR visualizations. Here we present DynamicViz, a framework for generating dynamic visualizations that capture the sensitivity of DR visualizations to perturbations in the data resulting from bootstrap sampling. DynamicViz can be applied to all commonly used DR methods. We show the utility of dynamic visualizations in diagnosing common interpretative pitfalls of static visualizations and extending existing single-cell analyses. We introduce the variance score to quantify the dynamic variability of observations in these visualizations. The variance score characterizes natural variability in the data and can be used to optimize DR algorithm implementations. Data visualization is widely used in science, but interpreting such visualizations is prone to error. Here a dynamic visualization is introduced for capturing more information and improving the reliability of visual interpretations.},
  archive      = {J_NATCS},
  author       = {Sun, Eric D. and Ma, Rong and Zou, James},
  doi          = {10.1038/s43588-022-00380-4},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {86-100},
  shortjournal = {Nat. Comput. Sci},
  title        = {Dynamic visualization of high-dimensional data},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dimensionality reduction of calcium-imaged neuronal
population activity. <em>NATCS</em>, <em>3</em>(1), 71–85. (<a
href="https://doi.org/10.1038/s43588-022-00390-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Calcium imaging has been widely adopted for its ability to record from large neuronal populations. To summarize the time course of neural activity, dimensionality-reduction methods, which have been applied extensively to population spiking activity, may be particularly useful. However, it is unclear whether the dimensionality-reduction methods applied to spiking activity are appropriate for calcium imaging. We thus carried out a systematic study of design choices based on standard dimensionality-reduction methods. We have also developed a method to perform deconvolution and dimensionality reduction simultaneously (calcium imaging linear dynamical system, CILDS). CILDS most accurately recovered the single-trial, low-dimensional time courses from simulated calcium imaging data. CILDS also outperformed the other methods on calcium imaging recordings from larval zebrafish and mice. More broadly, this study represents a foundation for summarizing calcium-imaging recordings of large neuronal populations using dimensionality reduction in diverse experimental settings. Design choices for dimensionality reduction on calcium imaging recordings are systematically evaluated, and a method called calcium imaging linear dynamical system (CILDS) is proposed for performing deconvolution and dimensionality reduction jointly.},
  archive      = {J_NATCS},
  author       = {Koh, Tze Hui and Bishop, William E. and Kawashima, Takashi and Jeon, Brian B. and Srinivasan, Ranjani and Mu, Yu and Wei, Ziqiang and Kuhlman, Sandra J. and Ahrens, Misha B. and Chase, Steven M. and Yu, Byron M.},
  doi          = {10.1038/s43588-022-00390-2},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {71-85},
  shortjournal = {Nat. Comput. Sci},
  title        = {Dimensionality reduction of calcium-imaged neuronal population activity},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Binding affinity estimation from restrained umbrella
sampling simulations. <em>NATCS</em>, <em>3</em>(1), 59–70. (<a
href="https://doi.org/10.1038/s43588-022-00389-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The protein–ligand binding affinity quantifies the binding strength between a protein and its ligand. Computer modeling and simulations can be used to estimate the binding affinity or binding free energy using data- or physics-driven methods or a combination thereof. Here we discuss a purely physics-based sampling approach based on biased molecular dynamics simulations. Our proposed method generalizes and simplifies previously suggested stratification strategies that use umbrella sampling or other enhanced sampling simulations with additional collective-variable-based restraints. The approach presented here uses a flexible scheme that can be easily tailored for any system of interest. We estimate the binding affinity of human fibroblast growth factor 1 to heparin hexasaccharide based on the available crystal structure of the complex as the initial model and four different variations of the proposed method to compare against the experimentally determined binding affinity obtained from isothermal titration calorimetry experiments. This work provides a physics-based theoretical framework for accurate protein–ligand binding affinity estimation based on molecular dynamics simulations, enhanced sampling, non-parametric reweighting and the orientation quaternion formalism.},
  archive      = {J_NATCS},
  author       = {Govind Kumar, Vivek and Polasa, Adithya and Agrawal, Shilpi and Kumar, Thallapuranam Krishnaswamy Suresh and Moradi, Mahmoud},
  doi          = {10.1038/s43588-022-00389-9},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {59-70},
  shortjournal = {Nat. Comput. Sci},
  title        = {Binding affinity estimation from restrained umbrella sampling simulations},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Supervised learning of a chemistry functional with damped
dispersion. <em>NATCS</em>, <em>3</em>(1), 48–58. (<a
href="https://doi.org/10.1038/s43588-022-00371-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Kohn–Sham density functional theory is widely used in chemistry, but no functional can accurately predict the whole range of chemical properties, although recent progress by some doubly hybrid functionals comes close. Here, we optimized a singly hybrid functional called CF22D with higher across-the-board accuracy for chemistry than most of the existing non-doubly hybrid functionals by using a flexible functional form that combines a global hybrid meta-nonseparable gradient approximation that depends on density and occupied orbitals with a damped dispersion term that depends on geometry. We optimized this energy functional by using a large database and performance-triggered iterative supervised training. We combined several databases to create a very large, combined database whose use demonstrated the good performance of CF22D on barrier heights, isomerization energies, thermochemistry, noncovalent interactions, radical and nonradical chemistry, small and large systems, simple and complex systems and transition-metal chemistry. A hybrid functional (CF22D) with higher across-the-board accuracy for chemistry than most existing non-doubly hybrid functionals is presented by using a large database and a performance-triggered iterative supervised training method.},
  archive      = {J_NATCS},
  author       = {Liu, Yiwei and Zhang, Cheng and Liu, Zhonghua and Truhlar, Donald G. and Wang, Ying and He, Xiao},
  doi          = {10.1038/s43588-022-00371-5},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {48-58},
  shortjournal = {Nat. Comput. Sci},
  title        = {Supervised learning of a chemistry functional with damped dispersion},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A transferable recommender approach for selecting the best
density functional approximations in chemical discovery. <em>NATCS</em>,
<em>3</em>(1), 38–47. (<a
href="https://doi.org/10.1038/s43588-022-00384-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Approximate density functional theory has become indispensable owing to its balanced cost–accuracy trade-off, including in large-scale screening. To date, however, no density functional approximation (DFA) with universal accuracy has been identified, leading to uncertainty in the quality of data generated from density functional theory. With electron density fitting and Δ-learning, we build a DFA recommender that selects the DFA with the lowest expected error with respect to the gold standard (but cost-prohibitive) coupled cluster theory in a system-specific manner. We demonstrate this recommender approach on the evaluation of vertical spin splitting energies of transition metal complexes. Our recommender predicts top-performing DFAs and yields excellent accuracy (about 2 kcal mol−1) for chemical discovery, outperforming both individual Δ-learning models and the best conventional single-functional approach from a set of 48 DFAs. By demonstrating transferability to diverse synthesized compounds, our recommender potentially addresses the accuracy versus scope dilemma broadly encountered in computational chemistry. A density functional recommender enables chemical space exploration by selecting the best exchange–correlation functional for each system, outperforming the use of a single functional for all systems or transfer learning models.},
  archive      = {J_NATCS},
  author       = {Duan, Chenru and Nandy, Aditya and Meyer, Ralf and Arunachalam, Naveen and Kulik, Heather J.},
  doi          = {10.1038/s43588-022-00384-0},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {38-47},
  shortjournal = {Nat. Comput. Sci},
  title        = {A transferable recommender approach for selecting the best density functional approximations in chemical discovery},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Quantum algorithms for quantum dynamics. <em>NATCS</em>,
<em>3</em>(1), 25–37. (<a
href="https://doi.org/10.1038/s43588-022-00374-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Among the many computational challenges faced across different disciplines, quantum-mechanical systems pose some of the hardest ones and offer a natural playground for the growing field of quantum technologies. In this Perspective, we discuss quantum algorithmic solutions for quantum dynamics, reporting on the latest developments and offering a viewpoint on their potential and current limitations. We present some of the most promising areas of application and identify possible research directions for the coming years. Quantum algorithms for simulating quantum dynamics have shown promising results to overcome the difficulties from the classical counterparts. This Perspective summarizes the recent developments in the field, and further discusses the limitations and research opportunities towards the goal of quantum advantage.},
  archive      = {J_NATCS},
  author       = {Miessen, Alexander and Ollitrault, Pauline J. and Tacchino, Francesco and Tavernelli, Ivano},
  doi          = {10.1038/s43588-022-00374-2},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {25-37},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantum algorithms for quantum dynamics},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chemical reaction networks and opportunities for machine
learning. <em>NATCS</em>, <em>3</em>(1), 12–24. (<a
href="https://doi.org/10.1038/s43588-022-00369-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chemical reaction networks (CRNs), defined by sets of species and possible reactions between them, are widely used to interrogate chemical systems. To capture increasingly complex phenomena, CRNs can be leveraged alongside data-driven methods and machine learning (ML). In this Perspective, we assess the diverse strategies available for CRN construction and analysis in pursuit of a wide range of scientific goals, discuss ML techniques currently being applied to CRNs and outline future CRN-ML approaches, presenting scientific and technical challenges to overcome. Chemical reaction networks are widely used to examine the behavior of chemical systems. While diverse strategies exist for chemical reaction network construction and analysis for a wide range of scientific goals, data-driven and machine learning methods must continue to capture increasingly complex phenomena to overcome existing unmet challenges.},
  archive      = {J_NATCS},
  author       = {Wen, Mingjian and Spotte-Smith, Evan Walter Clark and Blau, Samuel M. and McDermott, Matthew J. and Krishnapriyan, Aditi S. and Persson, Kristin A.},
  doi          = {10.1038/s43588-022-00369-z},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {12-24},
  shortjournal = {Nat. Comput. Sci},
  title        = {Chemical reaction networks and opportunities for machine learning},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Towards a purely physics-based computational binding
affinity estimation. <em>NATCS</em>, <em>3</em>(1), 10–11. (<a
href="https://doi.org/10.1038/s43588-023-00396-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determining whether a drug candidate has sufficient affinity to its target is a critical part of drug development. A purely physics-based computational method was developed that uses non-equilibrium statistical mechanics approaches alongside molecular dynamics simulations. This technique could enable researchers to accurately estimate the binding affinities of potential drug candidates.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00396-4},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {10-11},
  shortjournal = {Nat. Comput. Sci},
  title        = {Towards a purely physics-based computational binding affinity estimation},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dimensionality reduction under scrutiny. <em>NATCS</em>,
<em>3</em>(1), 8–9. (<a
href="https://doi.org/10.1038/s43588-022-00383-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A framework for generating and interpreting dynamic visualizations from traditional static dimensionality reduction visualization methods has been proposed in a recent study.},
  archive      = {J_NATCS},
  author       = {Yang, Yang and Tuong, Zewen K. and Yu, Di},
  doi          = {10.1038/s43588-022-00383-1},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {8-9},
  shortjournal = {Nat. Comput. Sci},
  title        = {Dimensionality reduction under scrutiny},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Using AI to navigate through the DFA zoo. <em>NATCS</em>,
<em>3</em>(1), 6–7. (<a
href="https://doi.org/10.1038/s43588-022-00393-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A proposed density functional approximation (DFA) recommender outperforms the use of a single functional by selecting the optimal exchange-correlation functional for a given system.},
  archive      = {J_NATCS},
  author       = {Vuckovic, Stefan},
  doi          = {10.1038/s43588-022-00393-z},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {6-7},
  shortjournal = {Nat. Comput. Sci},
  title        = {Using AI to navigate through the DFA zoo},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Large language model for molecular chemistry.
<em>NATCS</em>, <em>3</em>(1), 5. (<a
href="https://doi.org/10.1038/s43588-023-00399-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-023-00399-1},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {5},
  shortjournal = {Nat. Comput. Sci},
  title        = {Large language model for molecular chemistry},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023m). Uncovering contaminants via machine learning.
<em>NATCS</em>, <em>3</em>(1), 4. (<a
href="https://doi.org/10.1038/s43588-023-00400-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-023-00400-x},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {4},
  shortjournal = {Nat. Comput. Sci},
  title        = {Uncovering contaminants via machine learning},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Deep learning to estimate brain age. <em>NATCS</em>,
<em>3</em>(1), 3. (<a
href="https://doi.org/10.1038/s43588-023-00398-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-023-00398-2},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {3},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep learning to estimate brain age},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023h). Leveraging the power of crowds. <em>NATCS</em>,
<em>3</em>(1), 2. (<a
href="https://doi.org/10.1038/s43588-023-00397-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-023-00397-3},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {2},
  shortjournal = {Nat. Comput. Sci},
  title        = {Leveraging the power of crowds},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Cover runners-up of 2022. <em>NATCS</em>, <em>3</em>(1), 1.
(<a href="https://doi.org/10.1038/s43588-023-00402-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Find out more about some of the image submissions from 2022 that almost made it as Nature Computational Science covers.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00402-9},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cover runners-up of 2022},
  volume       = {3},
  year         = {2023},
}
</textarea>
</details></li>
</ul>

</body>
</html>
