<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NBT_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nbt---378">NBT - 378</h2>
<ul>
<li><details>
<summary>
(2021e). First rounders: Lita nelsen. <em>NBT</em>, <em>40</em>(1),
29. (<a href="https://doi.org/10.1038/s41587-021-01174-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01174-1},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {1},
  pages        = {29},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Lita nelsen},
  volume       = {40},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Forum: Drug delivery roundtable with langer and whitehead.
<em>NBT</em>, <em>40</em>(1), 29. (<a
href="https://doi.org/10.1038/s41587-021-01177-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01177-y},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {1},
  pages        = {29},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Drug delivery roundtable with langer and whitehead},
  volume       = {40},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). The PROTAC gold rush. <em>NBT</em>, <em>40</em>(1), 12–16.
(<a href="https://doi.org/10.1038/s41587-021-01173-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Investment in small-molecule protein degraders is surging, but their drawbacks, limitations and risks are becoming clear.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-021-01173-2},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {1},
  pages        = {12-16},
  shortjournal = {Nature Biotech.},
  title        = {The PROTAC gold rush},
  volume       = {40},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Japan embraces CRISPR-edited fish. <em>NBT</em>,
<em>40</em>(1), 10. (<a
href="https://doi.org/10.1038/s41587-021-01197-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01197-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {1},
  pages        = {10},
  shortjournal = {Nature Biotech.},
  title        = {Japan embraces CRISPR-edited fish},
  volume       = {40},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). GABA-enriched tomato is first CRISPR-edited food to enter
market. <em>NBT</em>, <em>40</em>(1), 9–11. (<a
href="https://doi.org/10.1038/d41587-021-00026-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sanatech Seed’s Sicilian Rouge CRISPR-edited ‘health-promoting’ tomatoes reach consumers and may open the market to more genome-edited fruit, vegetables and even fish. Sanatech Seed’s Sicilian Rouge CRISPR-edited ‘health-promoting’ tomatoes reach consumers and may open the market to more genome-edited fruit, vegetables and even fish.},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/d41587-021-00026-2},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {1},
  pages        = {9-11},
  shortjournal = {Nature Biotech.},
  title        = {GABA-enriched tomato is first CRISPR-edited food to enter market},
  volume       = {40},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). Off-the-shelf, gene-edited CAR-t cells forge ahead, despite
safety scare. <em>NBT</em>, <em>40</em>(1), 5–8. (<a
href="https://doi.org/10.1038/d41587-021-00027-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Race to the clinic reignites for an off-the-shelf alternative to autologous CAR-T cell therapy, even as concerns over chromosomal abnormalities linger. Race to the clinic reignites for an off-the-shelf alternative to autologous CAR-T cell therapy, even as concerns over chromosomal abnormalities linger.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/d41587-021-00027-1},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {1},
  pages        = {5-8},
  shortjournal = {Nature Biotech.},
  title        = {Off-the-shelf, gene-edited CAR-T cells forge ahead, despite safety scare},
  volume       = {40},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). T-cell vaccines could top up immunity to COVID, as variants
loom large. <em>NBT</em>, <em>40</em>(1), 3–4. (<a
href="https://doi.org/10.1038/d41587-021-00025-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peptide jabs targeting T cells could be especially useful for people with compromised immune systems, as backups for spike-based vaccines, or against Omicron and other emerging variants. Peptide jabs targeting T cells could be especially useful for people with compromised immune systems, as backups for spike-based vaccines, or against Omicron and other emerging variants.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/d41587-021-00025-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {1},
  pages        = {3-4},
  shortjournal = {Nature Biotech.},
  title        = {T-cell vaccines could top up immunity to COVID, as variants loom large},
  volume       = {40},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). People. <em>NBT</em>, <em>39</em>(12), 1602. (<a
href="https://doi.org/10.1038/s41587-021-01154-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01154-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1602},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). IGEM comes of age: Trends in its research output.
<em>NBT</em>, <em>39</em>(12), 1599–1601. (<a
href="https://doi.org/10.1038/s41587-021-01152-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Eighteen years after its inception, the international Genetically Engineered Machine competition has catalyzed the infusion of synthetic biology with interdisciplinary fundamental and translational research, as well as with inspired young scientists.},
  archive      = {J_NBT},
  author       = {Jainarayanan, Ashwin K. and Galanis, Anastasios and Sreejith, Athira and Suresh, Sourav and Nakara, Amatullah Mustafa and Kundlatsch, Guilherme E. and Rubio-Sánchez, Roger},
  doi          = {10.1038/s41587-021-01152-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1599-1601},
  shortjournal = {Nature Biotech.},
  title        = {IGEM comes of age: Trends in its research output},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Publisher correction: Biotech news from around the world.
<em>NBT</em>, <em>39</em>(12), 1598. (<a
href="https://doi.org/10.1038/s41587-021-01134-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01134-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1598},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Biotech news from around the world},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Roche deal with RNA-editing spinout.
<em>NBT</em>, <em>39</em>(12), 1597. (<a
href="https://doi.org/10.1038/s41587-021-01135-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01135-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1597},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Roche deal with RNA-editing spinout},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Drug hunters uncloak the non-coding
“hidden” genome. <em>NBT</em>, <em>39</em>(12), 1597. (<a
href="https://doi.org/10.1038/s41587-021-01141-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-021-01141-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1597},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Drug hunters uncloak the non-coding ‘hidden’ genome},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: Stepping to the exit. <em>NBT</em>,
<em>39</em>(12), 1597. (<a
href="https://doi.org/10.1038/s41587-021-01142-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {de Bloeme, Christiaan M. and Jansen, Robin W. and Krul, Mark R. L. and Geutjes, Ernst-Jan},
  doi          = {10.1038/s41587-021-01142-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1597},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Stepping to the exit},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A pan-cancer landscape of somatic mutations in non-unique
regions of the human genome. <em>NBT</em>, <em>39</em>(12), 1589–1596.
(<a href="https://doi.org/10.1038/s41587-021-00971-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A substantial fraction of the human genome displays high sequence similarity with at least one other genomic sequence, posing a challenge for the identification of somatic mutations from short-read sequencing data. Here we annotate genomic variants in 2,658 cancers from the Pan-Cancer Analysis of Whole Genomes (PCAWG) cohort with links to similar sites across the human genome. We train a machine learning model to use signals distributed over multiple genomic sites to call somatic events in non-unique regions and validate the data against linked-read sequencing in an independent dataset. Using this approach, we uncover previously hidden mutations in ~1,700 coding sequences and in thousands of regulatory elements, including in known cancer genes, immunoglobulins and highly mutated gene families. Mutations in non-unique regions are consistent with mutations in unique regions in terms of mutation burden and substitution profiles. The analysis provides a systematic summary of the mutation events in non-unique regions at a genome-wide scale across multiple human cancers. Cancer mutations in non-unique sequences are identified by machine learning on short-read data.},
  archive      = {J_NBT},
  author       = {Tarabichi, Maxime and Demeulemeester, Jonas and Verfaillie, Annelien and Flanagan, Adrienne M. and Van Loo, Peter and Konopka, Tomasz},
  doi          = {10.1038/s41587-021-00971-y},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1589-1596},
  shortjournal = {Nature Biotech.},
  title        = {A pan-cancer landscape of somatic mutations in non-unique regions of the human genome},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). RNA demethylation increases the yield and biomass of rice
and potato plants in field trials. <em>NBT</em>, <em>39</em>(12),
1581–1588. (<a
href="https://doi.org/10.1038/s41587-021-00982-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA N6-methyladenosine (m6A) modifications are essential in plants. Here, we show that transgenic expression of the human RNA demethylase FTO in rice caused a more than threefold increase in grain yield under greenhouse conditions. In field trials, transgenic expression of FTO in rice and potato caused ~50\% increases in yield and biomass. We demonstrate that the presence of FTO stimulates root meristem cell proliferation and tiller bud formation and promotes photosynthetic efficiency and drought tolerance but has no effect on mature cell size, shoot meristem cell proliferation, root diameter, plant height or ploidy. FTO mediates substantial m6A demethylation (around 7\% of demethylation in poly(A) RNA and around 35\% decrease of m6A in non-ribosomal nuclear RNA) in plant RNA, inducing chromatin openness and transcriptional activation. Therefore, modulation of plant RNA m6A methylation is a promising strategy to dramatically improve plant growth and crop yield. Rice and potato plants are more productive after epitranscriptome engineering.},
  archive      = {J_NBT},
  author       = {Yu, Qiong and Liu, Shun and Yu, Lu and Xiao, Yu and Zhang, Shasha and Wang, Xueping and Xu, Yingying and Yu, Hong and Li, Yulong and Yang, Junbo and Tang, Jun and Duan, Hong-Chao and Wei, Lian-Huan and Zhang, Haiyan and Wei, Jiangbo and Tang, Qian and Wang, Chunling and Zhang, Wutong and Wang, Ye and Song, Peizhe and Lu, Qiang and Zhang, Wei and Dong, Shunqing and Song, Baoan and He, Chuan and Jia, Guifang},
  doi          = {10.1038/s41587-021-00982-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1581-1588},
  shortjournal = {Nature Biotech.},
  title        = {RNA demethylation increases the yield and biomass of rice and potato plants in field trials},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). High-content single-cell combinatorial indexing.
<em>NBT</em>, <em>39</em>(12), 1574–1580. (<a
href="https://doi.org/10.1038/s41587-021-00962-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell combinatorial indexing (sci) with transposase-based library construction increases the throughput of single-cell genomics assays but produces sparse coverage in terms of usable reads per cell. We develop symmetrical strand sci (‘s3’), a uracil-based adapter switching approach that improves the rate of conversion of source DNA into viable sequencing library fragments following tagmentation. We apply this chemistry to assay chromatin accessibility (s3-assay for transposase-accessible chromatin, s3-ATAC) in human cortical and mouse whole-brain tissues, with mouse datasets demonstrating a six- to 13-fold improvement in usable reads per cell compared with other available methods. Application of s3 to single-cell whole-genome sequencing (s3-WGS) and to whole-genome plus chromatin conformation (s3-GCC) yields 148- and 14.8-fold improvements, respectively, in usable reads per cell compared with sci-DNA-sequencing and sci-HiC. We show that s3-WGS and s3-GCC resolve subclonal genomic alterations in patient-derived pancreatic cancer cell lines. We expect that the s3 platform will be compatible with other transposase-based techniques, including sci-MET or CUT&amp;amp;Tag. The number of usable reads in single-cell DNA and chromatin assays is increased by adapter switching.},
  archive      = {J_NBT},
  author       = {Mulqueen, Ryan M. and Pokholok, Dmitry and O’Connell, Brendan L. and Thornton, Casey A. and Zhang, Fan and O’Roak, Brian J. and Link, Jason and Yardımcı, Galip Gürkan and Sears, Rosalie C. and Steemers, Frank J. and Adey, Andrew C.},
  doi          = {10.1038/s41587-021-00962-z},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1574-1580},
  shortjournal = {Nature Biotech.},
  title        = {High-content single-cell combinatorial indexing},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MaxDIA enables library-based and library-free
data-independent acquisition proteomics. <em>NBT</em>, <em>39</em>(12),
1563–1573. (<a
href="https://doi.org/10.1038/s41587-021-00968-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {MaxDIA is a software platform for analyzing data-independent acquisition (DIA) proteomics data within the MaxQuant software environment. Using spectral libraries, MaxDIA achieves deep proteome coverage with substantially better coefficients of variation in protein quantification than other software. MaxDIA is equipped with accurate false discovery rate (FDR) estimates on both library-to-DIA match and protein levels, including when using whole-proteome predicted spectral libraries. This is the foundation of discovery DIA—hypothesis-free analysis of DIA samples without library and with reliable FDR control. MaxDIA performs three- or four-dimensional feature detection of fragment data, and scoring of matches is augmented by machine learning on the features of an identification. MaxDIA’s bootstrap DIA workflow performs multiple rounds of matching with increasing quality of recalibration and stringency of matching to the library. Combining MaxDIA with two new technologies—BoxCar acquisition and trapped ion mobility spectrometry—both lead to deep and accurate proteome quantification. The software platform MaxDIA streamlines analysis of data-independent acquisition proteomics.},
  archive      = {J_NBT},
  author       = {Sinitcyn, Pavel and Hamzeiy, Hamid and Salinas Soto, Favio and Itzhak, Daniel and McCarthy, Frank and Wichmann, Christoph and Steger, Martin and Ohmayer, Uli and Distler, Ute and Kaspar-Schoenefeld, Stephanie and Prianichnikov, Nikita and Yılmaz, Şule and Rudolph, Jan Daniel and Tenzer, Stefan and Perez-Riverol, Yasset and Nagaraj, Nagarjuna and Humphrey, Sean J. and Cox, Jürgen},
  doi          = {10.1038/s41587-021-00968-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1563-1573},
  shortjournal = {Nature Biotech.},
  title        = {MaxDIA enables library-based and library-free data-independent acquisition proteomics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). LAMP-seq enables sensitive, multiplexed COVID-19 diagnostics
using molecular barcoding. <em>NBT</em>, <em>39</em>(12), 1556–1562. (<a
href="https://doi.org/10.1038/s41587-021-00966-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Frequent testing of large population groups combined with contact tracing and isolation measures will be crucial for containing Coronavirus Disease 2019 outbreaks. Here we present LAMP-Seq, a modified, highly scalable reverse transcription loop-mediated isothermal amplification (RT–LAMP) method. Unpurified biosamples are barcoded and amplified in a single heat step, and pooled products are analyzed en masse by sequencing. Using commercial reagents, LAMP-Seq has a limit of detection of ~2.2 molecules per µl at 95\% confidence and near-perfect specificity for severe acute respiratory syndrome coronavirus 2 given its sequence readout. Clinical validation of an open-source protocol with 676 swab samples, 98 of which were deemed positive by standard RT–qPCR, demonstrated 100\% sensitivity in individuals with cycle threshold values of up to 33 and a specificity of 99.7\%, at a very low material cost. With a time-to-result of fewer than 24 h, low cost and little new infrastructure requirement, LAMP-Seq can be readily deployed for frequent testing as part of an integrated public health surveillance program. SARS-CoV-2 detection with LAMP-Seq combines isothermal amplification and barcoding with high-throughput sequencing.},
  archive      = {J_NBT},
  author       = {Ludwig, Kerstin U. and Schmithausen, Ricarda M. and Li, David and Jacobs, Max L. and Hollstein, Ronja and Blumenstock, Katja and Liebing, Jana and Słabicki, Mikołaj and Ben-Shmuel, Amir and Israeli, Ofir and Weiss, Shay and Ebert, Thomas S. and Paran, Nir and Rüdiger, Wibke and Wilbring, Gero and Feldman, David and Lippke, Bärbel and Ishorst, Nina and Hochfeld, Lara M. and Beins, Eva C. and Kaltheuner, Ines H. and Schmitz, Maximilian and Wöhler, Aliona and Döhla, Manuel and Sib, Esther and Jentzsch, Marius and Moench, Eva-Maria C. and Borrajo, Jacob D. and Strecker, Jonathan and Reinhardt, Julia and Cleary, Brian and Geyer, Matthias and Hölzel, Michael and Macrae, Rhiannon and Nöthen, Markus M. and Hoffmann, Per and Exner, Martin and Regev, Aviv and Zhang, Feng and Schmid-Burgk, Jonathan L.},
  doi          = {10.1038/s41587-021-00966-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1556-1562},
  shortjournal = {Nature Biotech.},
  title        = {LAMP-seq enables sensitive, multiplexed COVID-19 diagnostics using molecular barcoding},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Bioorthogonal information storage in l-DNA with a
high-fidelity mirror-image pfu DNA polymerase. <em>NBT</em>,
<em>39</em>(12), 1548–1555. (<a
href="https://doi.org/10.1038/s41587-021-00969-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Natural DNA is exquisitely evolved to store genetic information. The chirally inverted l-DNA, possessing the same informational capacity but resistant to biodegradation, may serve as a robust, bioorthogonal information repository. Here we chemically synthesize a 90-kDa high-fidelity mirror-image Pfu DNA polymerase that enables accurate assembly of a kilobase-sized mirror-image gene. We use the polymerase to encode in l-DNA an 1860 paragraph by Louis Pasteur that first proposed a mirror-image world of biology. We realize chiral steganography by embedding a chimeric d-DNA/l-DNA key molecule in a d-DNA storage library, which conveys a false or secret message depending on the chirality of reading. Furthermore, we show that a trace amount of an l-DNA barcode preserved in water from a local pond remains amplifiable and sequenceable for 1 year, whereas a d-DNA barcode under the same conditions could not be amplified after 1 day. These next-generation mirror-image molecular tools may transform the development of advanced mirror-image biology systems and pave the way for the realization of the mirror-image central dogma and exploration of their applications. A high-fidelity mirror-image polymerase that accurately synthesizes mirror-image DNA protects stored information from biodegradation.},
  archive      = {J_NBT},
  author       = {Fan, Chuyao and Deng, Qiang and Zhu, Ting F.},
  doi          = {10.1038/s41587-021-00969-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1548-1555},
  shortjournal = {Nature Biotech.},
  title        = {Bioorthogonal information storage in l-DNA with a high-fidelity mirror-image pfu DNA polymerase},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Enhanced detection of minimal residual disease by targeted
sequencing of phased variants in circulating tumor DNA. <em>NBT</em>,
<em>39</em>(12), 1537–1547. (<a
href="https://doi.org/10.1038/s41587-021-00981-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Circulating tumor-derived DNA (ctDNA) is an emerging biomarker for many cancers, but the limited sensitivity of current detection methods reduces its utility for diagnosing minimal residual disease. Here we describe phased variant enrichment and detection sequencing (PhasED-seq), a method that uses multiple somatic mutations in individual DNA fragments to improve the sensitivity of ctDNA detection. Leveraging whole-genome sequences from 2,538 tumors, we identify phased variants and their associations with mutational signatures. We show that even without molecular barcodes, the limits of detection of PhasED-seq outperform prior methods, including duplex barcoding, allowing ctDNA detection in the ppm range in participant samples. We profiled 678 specimens from 213 participants with B cell lymphomas, including serial cell-free DNA samples before and during therapy for diffuse large B cell lymphoma. In participants with undetectable ctDNA after two cycles of therapy using a next-generation sequencing-based approach termed cancer personalized profiling by deep sequencing, an additional 25\% have ctDNA detectable by PhasED-seq and have worse outcomes. Finally, we demonstrate the application of PhasED-seq to solid tumors. The sensitivity of circulating tumor DNA detection is improved by identifying sequences with two or more mutations.},
  archive      = {J_NBT},
  author       = {Kurtz, David M. and Soo, Joanne and Co Ting Keh, Lyron and Alig, Stefan and Chabon, Jacob J. and Sworder, Brian J. and Schultz, Andre and Jin, Michael C. and Scherer, Florian and Garofalo, Andrea and Macaulay, Charles W. and Hamilton, Emily G. and Chen, Binbin and Olsen, Mari and Schroers-Martin, Joseph G. and Craig, Alexander F. M. and Moding, Everett J. and Esfahani, Mohammad S. and Liu, Chih Long and Dührsen, Ulrich and Hüttmann, Andreas and Casasnovas, René-Olivier and Westin, Jason R. and Roschewski, Mark and Wilson, Wyndham H. and Gaidano, Gianluca and Rossi, Davide and Diehn, Maximilian and Alizadeh, Ash A.},
  doi          = {10.1038/s41587-021-00981-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1537-1547},
  shortjournal = {Nature Biotech.},
  title        = {Enhanced detection of minimal residual disease by targeted sequencing of phased variants in circulating tumor DNA},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cholesterol-functionalized DNA/RNA heteroduplexes cross the
blood–brain barrier and knock down genes in the rodent CNS.
<em>NBT</em>, <em>39</em>(12), 1529–1536. (<a
href="https://doi.org/10.1038/s41587-021-00972-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Achieving regulation of endogenous gene expression in the central nervous system (CNS) with antisense oligonucleotides (ASOs) administered systemically would facilitate the development of ASO-based therapies for neurological diseases. We demonstrate that DNA/RNA heteroduplex oligonucleotides (HDOs) conjugated to cholesterol or α-tocopherol at the 5′ end of the RNA strand reach the CNS after subcutaneous or intravenous administration in mice and rats. The HDOs distribute throughout the brain, spinal cord and peripheral tissues and suppress the expression of four target genes by up to 90\% in the CNS, whereas single-stranded ASOs conjugated to cholesterol have limited activity. Gene knockdown was observed in major CNS cell types and was greatest in neurons and microglial cells. Side effects, such as thrombocytopenia and focal brain necrosis, were limited by using subcutaneous delivery or by dividing intravenous injections. By crossing the blood–brain barrier more effectively, cholesterol-conjugated HDOs may overcome the limited efficacy of ASOs targeting the CNS without requiring intrathecal administration. Genes in the rodent brain are knocked down by DNA/RNA heteroduplexes injected intravenously.},
  archive      = {J_NBT},
  author       = {Nagata, Tetsuya and Dwyer, Chrissa A. and Yoshida-Tanaka, Kie and Ihara, Kensuke and Ohyagi, Masaki and Kaburagi, Hidetoshi and Miyata, Haruka and Ebihara, Satoe and Yoshioka, Kotaro and Ishii, Takashi and Miyata, Kanjiro and Miyata, Kenichi and Powers, Berit and Igari, Tomoko and Yamamoto, Syunsuke and Arimura, Naoto and Hirabayashi, Hideki and Uchihara, Toshiki and Hara, Rintaro Iwata and Wada, Takeshi and Bennett, C. Frank and Seth, Punit P. and Rigo, Frank and Yokota, Takanori},
  doi          = {10.1038/s41587-021-00972-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1529-1536},
  shortjournal = {Nature Biotech.},
  title        = {Cholesterol-functionalized DNA/RNA heteroduplexes cross the blood–brain barrier and knock down genes in the rodent CNS},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). High-throughput mapping of a whole rhesus monkey brain at
micrometer resolution. <em>NBT</em>, <em>39</em>(12), 1521–1528. (<a
href="https://doi.org/10.1038/s41587-021-00986-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whole-brain mesoscale mapping in primates has been hindered by large brain sizes and the relatively low throughput of available microscopy methods. Here, we present an approach that combines primate-optimized tissue sectioning and clearing with ultrahigh-speed fluorescence microscopy implementing improved volumetric imaging with synchronized on-the-fly-scan and readout technique, and is capable of completing whole-brain imaging of a rhesus monkey at 1 × 1 × 2.5 µm3 voxel resolution within 100 h. We also developed a highly efficient method for long-range tracing of sparse axonal fibers in datasets numbering hundreds of terabytes. This pipeline, which we call serial sectioning and clearing, three-dimensional microscopy with semiautomated reconstruction and tracing (SMART), enables effective connectome-scale mapping of large primate brains. With SMART, we were able to construct a cortical projection map of the mediodorsal nucleus of the thalamus and identify distinct turning and routing patterns of individual axons in the cortical folds while approaching their arborization destinations. A whole monkey brain is imaged at high resolution in 100 hours.},
  archive      = {J_NBT},
  author       = {Xu, Fang and Shen, Yan and Ding, Lufeng and Yang, Chao-Yu and Tan, Heng and Wang, Hao and Zhu, Qingyuan and Xu, Rui and Wu, Fengyi and Xiao, Yanyang and Xu, Cheng and Li, Qianwei and Su, Peng and Zhang, Li I. and Dong, Hong-Wei and Desimone, Robert and Xu, Fuqiang and Hu, Xintian and Lau, Pak-Ming and Bi, Guo-Qiang},
  doi          = {10.1038/s41587-021-00986-5},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1521-1528},
  shortjournal = {Nature Biotech.},
  title        = {High-throughput mapping of a whole rhesus monkey brain at micrometer resolution},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Nanopore sequencing of single-cell transcriptomes with
scCOLOR-seq. <em>NBT</em>, <em>39</em>(12), 1517–1520. (<a
href="https://doi.org/10.1038/s41587-021-00965-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we describe single-cell corrected long-read sequencing (scCOLOR-seq), which enables error correction of barcode and unique molecular identifier oligonucleotide sequences and permits standalone cDNA nanopore sequencing of single cells. Barcodes and unique molecular identifiers are synthesized using dimeric nucleotide building blocks that allow error detection. We illustrate the use of the method for evaluating barcode assignment accuracy, differential isoform usage in myeloma cell lines, and fusion transcript detection in a sarcoma cell line. Single-cell transcriptomes are accurately sequenced with error-tolerant barcodes.},
  archive      = {J_NBT},
  author       = {Philpott, Martin and Watson, Jonathan and Thakurta, Anjan and Brown, Tom and Oppermann, Udo and Cribbs, Adam P.},
  doi          = {10.1038/s41587-021-00965-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1517-1520},
  shortjournal = {Nature Biotech.},
  title        = {Nanopore sequencing of single-cell transcriptomes with scCOLOR-seq},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). RNA delivery with a human virus-like particle. <em>NBT</em>,
<em>39</em>(12), 1514–1515. (<a
href="https://doi.org/10.1038/s41587-021-01124-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA cargo is transferred into cultured cells using a fully human delivery system.},
  archive      = {J_NBT},
  author       = {Gutkin, Anna and Rosenblum, Daniel and Peer, Dan},
  doi          = {10.1038/s41587-021-01124-x},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1514-1515},
  shortjournal = {Nature Biotech.},
  title        = {RNA delivery with a human virus-like particle},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Improving CRISPR tools by elucidating DNA repair.
<em>NBT</em>, <em>39</em>(12), 1512–1514. (<a
href="https://doi.org/10.1038/s41587-021-01149-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome-editing tools are optimized by identifying genes that affect DNA repair.},
  archive      = {J_NBT},
  author       = {Lim, Jung Min and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-021-01149-2},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1512-1514},
  shortjournal = {Nature Biotech.},
  title        = {Improving CRISPR tools by elucidating DNA repair},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Wearable devices. <em>NBT</em>, <em>39</em>(12), 1511. (<a
href="https://doi.org/10.1038/s41587-021-01153-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to wearable technology and devices for patient monitoring and treatment.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01153-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1511},
  shortjournal = {Nature Biotech.},
  title        = {Wearable devices},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Combination therapy patents: A new front in evergreening.
<em>NBT</em>, <em>39</em>(12), 1504–1510. (<a
href="https://doi.org/10.1038/s41587-021-01137-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As pharmaceutical companies seek patent protection for combinations of cancer therapeutics, it is worthwhile to assess what constitutes an ‘unexpected result’ for the purpose of an appropriate patent and whether randomized, controlled trials of drug combinations have the ability to generate them.},
  archive      = {J_NBT},
  author       = {Strohbehn, Garth W. and Kacew, Alec J. and Goldstein, Daniel A. and Feldman, Robin C. and Ratain, Mark J.},
  doi          = {10.1038/s41587-021-01137-6},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1504-1510},
  shortjournal = {Nature Biotech.},
  title        = {Combination therapy patents: A new front in evergreening},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Reply to “clarifying US regulations on xenotransplantation”
and “international standards and guidelines for xenotransplantation.”
<em>NBT</em>, <em>39</em>(12), 1503. (<a
href="https://doi.org/10.1038/s41587-021-01151-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kwisda, Koko and Cantz, Tobias and Hoppe, Nils},
  doi          = {10.1038/s41587-021-01151-8},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1503},
  shortjournal = {Nature Biotech.},
  title        = {Reply to ‘Clarifying US regulations on xenotransplantation’ and ‘International standards and guidelines for xenotransplantation’},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). International standards and guidelines for
xenotransplantation. <em>NBT</em>, <em>39</em>(12), 1501–1502. (<a
href="https://doi.org/10.1038/s41587-021-01148-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hawthorne, Wayne J. and Cowan, Peter J. and Buhler, Leo and Wolf, Eckhard},
  doi          = {10.1038/s41587-021-01148-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1501-1502},
  shortjournal = {Nature Biotech.},
  title        = {International standards and guidelines for xenotransplantation},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Clarifying US regulations on xenotransplantation.
<em>NBT</em>, <em>39</em>(12), 1500–1501. (<a
href="https://doi.org/10.1038/s41587-021-01144-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Marks, Peter and Solomon, Steven},
  doi          = {10.1038/s41587-021-01144-7},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1500-1501},
  shortjournal = {Nature Biotech.},
  title        = {Clarifying US regulations on xenotransplantation},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The responsibility for ethical procurement of human embryos
and gametes. <em>NBT</em>, <em>39</em>(12), 1499–1500. (<a
href="https://doi.org/10.1038/s41587-021-01150-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Hyun, Insoo},
  doi          = {10.1038/s41587-021-01150-9},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1499-1500},
  shortjournal = {Nature Biotech.},
  title        = {The responsibility for ethical procurement of human embryos and gametes},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). NINDS launches network to develop treatments for ultra-rare
neurological diseases. <em>NBT</em>, <em>39</em>(12), 1497–1499. (<a
href="https://doi.org/10.1038/s41587-021-01125-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Schor, Nina F. and Tamiz, Amir P. and Koroshetz, Walter J. and Broome, Ann-Marie},
  doi          = {10.1038/s41587-021-01125-w},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1497-1499},
  shortjournal = {Nature Biotech.},
  title        = {NINDS launches network to develop treatments for ultra-rare neurological diseases},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Biotech news from around the world. <em>NBT</em>,
<em>39</em>(12), 1485. (<a
href="https://doi.org/10.1038/s41587-021-01157-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01157-2},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1485},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cellular meat gets first government investment.
<em>NBT</em>, <em>39</em>(12), 1484. (<a
href="https://doi.org/10.1038/s41587-021-01165-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01165-2},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1484},
  shortjournal = {Nature Biotech.},
  title        = {Cellular meat gets first government investment},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). 100,000 whole-genome sequences’ diagnostic bonus.
<em>NBT</em>, <em>39</em>(12), 1482. (<a
href="https://doi.org/10.1038/s41587-021-01164-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01164-3},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1482},
  shortjournal = {Nature Biotech.},
  title        = {100,000 whole-genome sequences’ diagnostic bonus},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Moderna feud with NIH over COVID vaccine. <em>NBT</em>,
<em>39</em>(12), 1481. (<a
href="https://doi.org/10.1038/s41587-021-01166-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01166-1},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1481},
  shortjournal = {Nature Biotech.},
  title        = {Moderna feud with NIH over COVID vaccine},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Technologies to advance COVID-19 vaccine equity.
<em>NBT</em>, <em>39</em>(12), 1477. (<a
href="https://doi.org/10.1038/s41587-021-01167-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Poor countries need vaccine formats with low barriers to manufacture, distribution and administration.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01167-0},
  journal      = {Nature Biotechnology},
  month        = {12},
  number       = {12},
  pages        = {1477},
  shortjournal = {Nature Biotech.},
  title        = {Technologies to advance COVID-19 vaccine equity},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Preparing for the next plague. <em>NBT</em>,
<em>39</em>(12), 1491–1496. (<a
href="https://doi.org/10.1038/s41587-021-01140-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whether pan-vaccines or antibodies, SARS-CoV-2 is adding impetus to the race for broad-spectrum countermeasures against the world’s next infectious scourges.},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-01140-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {12},
  pages        = {1491-1496},
  shortjournal = {Nature Biotech.},
  title        = {Preparing for the next plague},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The CRISPR children. <em>NBT</em>, <em>39</em>(12),
1486–1490. (<a
href="https://doi.org/10.1038/s41587-021-01138-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In China, the first children with germline-edited genomes are growing up. How might their edited genomes affect their lives?},
  archive      = {J_NBT},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41587-021-01138-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {12},
  pages        = {1486-1490},
  shortjournal = {Nature Biotech.},
  title        = {The CRISPR children},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). No bones, no scales, no eyeballs: Appetite grows for
cell-based seafood. <em>NBT</em>, <em>39</em>(12), 1483–1485. (<a
href="https://doi.org/10.1038/d41587-021-00022-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lab-grown fish taste the same as their wild and farmed counterparts, offering an alternative to traditional seafood. But will it ever make economic sense to produce this kind of protein? Lab-grown fish taste the same as their wild and farmed counterparts, offering an alternative to traditional seafood. But will it ever make economic sense to produce this kind of protein?},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/d41587-021-00022-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {12},
  pages        = {1483-1485},
  shortjournal = {Nature Biotech.},
  title        = {No bones, no scales, no eyeballs: Appetite grows for cell-based seafood},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). First COVID-19 DNA vaccine approved, others in hot pursuit.
<em>NBT</em>, <em>39</em>(12), 1479–1482. (<a
href="https://doi.org/10.1038/d41587-021-00023-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {India’s approval of a nucleic acid vaccine hints at a solution for low-income nations—if the limitations of current delivery technology can be overcome. India’s approval of a nucleic acid vaccine hints at a solution for low-income nations—if the limitations of current delivery technology can be overcome.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/d41587-021-00023-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {12},
  pages        = {1479-1482},
  shortjournal = {Nature Biotech.},
  title        = {First COVID-19 DNA vaccine approved, others in hot pursuit},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). People. <em>NBT</em>, <em>39</em>(11), 1476. (<a
href="https://doi.org/10.1038/s41587-021-01117-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01117-w},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1476},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Third-quarter biotech job picture. <em>NBT</em>,
<em>39</em>(11), 1475. (<a
href="https://doi.org/10.1038/s41587-021-01113-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-021-01113-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1475},
  shortjournal = {Nature Biotech.},
  title        = {Third-quarter biotech job picture},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Bridges to biotechnology and bioentrepreneurship: Improving
diversity in the biotechnology sector. <em>NBT</em>, <em>39</em>(11),
1468–1474. (<a
href="https://doi.org/10.1038/s41587-021-01110-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A summer pipeline program provides under-represented students from around the United States with exposure and first-hand training to advance their careers in the biotechnology industry.},
  archive      = {J_NBT},
  author       = {Howard, Angelita P. and Slaughter, Liane S. and Carey, Kaylin M. and Lillard, James W.},
  doi          = {10.1038/s41587-021-01110-3},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1468-1474},
  shortjournal = {Nature Biotech.},
  title        = {Bridges to biotechnology and bioentrepreneurship: Improving diversity in the biotechnology sector},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Publisher correction: The RNA atlas expands the catalog of
human non-coding RNAs. <em>NBT</em>, <em>39</em>(11), 1467. (<a
href="https://doi.org/10.1038/s41587-021-00996-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Lorenzi, Lucia and Chiu, Hua-Sheng and Avila Cobos, Francisco and Gross, Stephen and Volders, Pieter-Jan and Cannoodt, Robrecht and Nuytens, Justine and Vanderheyden, Katrien and Anckaert, Jasper and Lefever, Steve and Tay, Aidan P. and de Bony, Eric J. and Trypsteen, Wim and Gysens, Fien and Vromman, Marieke and Goovaerts, Tine and Hansen, Thomas Birkballe and Kuersten, Scott and Nijs, Nele and Taghon, Tom and Vermaelen, Karim and Bracke, Ken R. and Saeys, Yvan and De Meyer, Tim and Deshpande, Nandan P. and Anande, Govardhan and Chen, Ting-Wen and Wilkins, Marc R. and Unnikrishnan, Ashwin and De Preter, Katleen and Kjems, Jørgen and Koster, Jan and Schroth, Gary P. and Vandesompele, Jo and Sumazin, Pavel and Mestdagh, Pieter},
  doi          = {10.1038/s41587-021-00996-3},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1467},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: The RNA atlas expands the catalog of human non-coding RNAs},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Measurement of histone replacement
dynamics with genetically encoded exchange timers in yeast.
<em>NBT</em>, <em>39</em>(11), 1467. (<a
href="https://doi.org/10.1038/s41587-021-01103-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Yaakov, Gilad and Jonas, Felix and Barkai, Naama},
  doi          = {10.1038/s41587-021-01103-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1467},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Measurement of histone replacement dynamics with genetically encoded exchange timers in yeast},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Publisher correction: Oncologists greet lumakras: The
world’s first KRAS inhibitor. <em>NBT</em>, <em>39</em>(11), 1467. (<a
href="https://doi.org/10.1038/s41587-021-01120-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-021-01120-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1467},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: oncologists greet lumakras: the world’s first KRAS inhibitor},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: An injectable bone marrow–like scaffold
enhances t cell immunity after hematopoietic stem cell transplantation.
<em>NBT</em>, <em>39</em>(11), 1466. (<a
href="https://doi.org/10.1038/s41587-021-01081-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Shah, Nisarg J. and Mao, Angelo S. and Shih, Ting-Yu and Kerr, Matthew D. and Sharda, Azeem and Raimondo, Theresa M. and Weaver, James C. and Vrbanac, Vladimir D. and Deruaz, Maud and Tager, Andrew M. and Mooney, David J. and Scadden, David T.},
  doi          = {10.1038/s41587-021-01081-5},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1466},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: An injectable bone marrow–like scaffold enhances t cell immunity after hematopoietic stem cell transplantation},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: Performance assessment of DNA sequencing
platforms in the ABRF next-generation sequencing study. <em>NBT</em>,
<em>39</em>(11), 1466. (<a
href="https://doi.org/10.1038/s41587-021-01122-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Foox, Jonathan and Tighe, Scott W. and Nicolet, Charles M. and Zook, Justin M. and Byrska-Bishop, Marta and Clarke, Wayne E. and Khayat, Michael M. and Mahmoud, Medhat and Laaguiby, Phoebe K. and Herbert, Zachary T. and Warner, Derek and Grills, George S. and Jen, Jin and Levy, Shawn and Xiang, Jenny and Alonso, Alicia and Zhao, Xia and Zhang, Wenwei and Teng, Fei and Zhao, Yonggang and Lu, Haorong and Schroth, Gary P. and Narzisi, Giuseppe and Farmerie, William and Sedlazeck, Fritz J. and Baldwin, Don A. and Mason, Christopher E.},
  doi          = {10.1038/s41587-021-01122-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1466},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Performance assessment of DNA sequencing platforms in the ABRF next-generation sequencing study},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). The RNA atlas expands the catalog of human non-coding RNAs.
<em>NBT</em>, <em>39</em>(11), 1453–1465. (<a
href="https://doi.org/10.1038/s41587-021-00936-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing compendia of non-coding RNA (ncRNA) are incomplete, in part because they are derived almost exclusively from small and polyadenylated RNAs. Here we present a more comprehensive atlas of the human transcriptome, which includes small and polyA RNA as well as total RNA from 300 human tissues and cell lines. We report thousands of previously uncharacterized RNAs, increasing the number of documented ncRNAs by approximately 8\%. To infer functional regulation by known and newly characterized ncRNAs, we exploited pre-mRNA abundance estimates from total RNA sequencing, revealing 316 microRNAs and 3,310 long non-coding RNAs with multiple lines of evidence for roles in regulating protein-coding genes and pathways. Our study both refines and expands the current catalog of human ncRNAs and their regulatory interactions. All data, analyses and results are available for download and interrogation in the R2 web portal, serving as a basis for future exploration of RNA biology and function. The human transcriptome from 300 tissues and cell lines reveals novel non-coding RNAs.},
  archive      = {J_NBT},
  author       = {Lorenzi, Lucia and Chiu, Hua-Sheng and Avila Cobos, Francisco and Gross, Stephen and Volders, Pieter-Jan and Cannoodt, Robrecht and Nuytens, Justine and Vanderheyden, Katrien and Anckaert, Jasper and Lefever, Steve and Tay, Aidan P. and de Bony, Eric J. and Trypsteen, Wim and Gysens, Fien and Vromman, Marieke and Goovaerts, Tine and Hansen, Thomas Birkballe and Kuersten, Scott and Nijs, Nele and Taghon, Tom and Vermaelen, Karim and Bracke, Ken R. and Saeys, Yvan and De Meyer, Tim and Deshpande, Nandan P. and Anande, Govardhan and Chen, Ting-Wen and Wilkins, Marc R. and Unnikrishnan, Ashwin and De Preter, Katleen and Kjems, Jørgen and Koster, Jan and Schroth, Gary P. and Vandesompele, Jo and Sumazin, Pavel and Mestdagh, Pieter},
  doi          = {10.1038/s41587-021-00936-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1453-1465},
  shortjournal = {Nature Biotech.},
  title        = {The RNA atlas expands the catalog of human non-coding RNAs},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Prediction of drug efficacy from transcriptional profiles
with deep learning. <em>NBT</em>, <em>39</em>(11), 1444–1452. (<a
href="https://doi.org/10.1038/s41587-021-00946-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug discovery focused on target proteins has been a successful strategy, but many diseases and biological processes lack obvious targets to enable such approaches. Here, to overcome this challenge, we describe a deep learning–based efficacy prediction system (DLEPS) that identifies drug candidates using a change in the gene expression profile in the diseased state as input. DLEPS was trained using chemically induced changes in transcriptional profiles from the L1000 project. We found that the changes in transcriptional profiles for previously unexamined molecules were predicted with a Pearson correlation coefficient of 0.74. We examined three disorders and experimentally tested the top drug candidates in mouse disease models. Validation showed that perillen, chikusetsusaponin IV and trametinib confer disease-relevant impacts against obesity, hyperuricemia and nonalcoholic steatohepatitis, respectively. DLEPS can generate insights into pathogenic mechanisms, and we demonstrate that the MEK–ERK signaling pathway is a target for developing agents against nonalcoholic steatohepatitis. Our findings suggest that DLEPS is an effective tool for drug repurposing and discovery. Drug discovery based on transcriptional profiling does not require knowledge of protein targets.},
  archive      = {J_NBT},
  author       = {Zhu, Jie and Wang, Jingxiang and Wang, Xin and Gao, Mingjing and Guo, Bingbing and Gao, Miaomiao and Liu, Jiarui and Yu, Yanqiu and Wang, Liang and Kong, Weikaixin and An, Yongpan and Liu, Zurui and Sun, Xinpei and Huang, Zhuo and Zhou, Hong and Zhang, Ning and Zheng, Ruimao and Xie, Zhengwei},
  doi          = {10.1038/s41587-021-00946-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1444-1452},
  shortjournal = {Nature Biotech.},
  title        = {Prediction of drug efficacy from transcriptional profiles with deep learning},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Measurement of histone replacement dynamics with
genetically encoded exchange timers in yeast. <em>NBT</em>,
<em>39</em>(11), 1434–1443. (<a
href="https://doi.org/10.1038/s41587-021-00959-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Histone exchange between histones carrying position-specific marks and histones bearing general marks is important for gene regulation, but understanding of histone exchange remains incomplete. To overcome the poor time resolution of conventional pulse–chase histone labeling, we present a genetically encoded histone exchange timer sensitive to the duration that two tagged histone subunits co-reside at an individual genomic locus. We apply these sensors to map genome-wide patterns of histone exchange in yeast using single samples. Comparing H3 exchange in cycling and G1-arrested cells suggests that replication-independent H3 exchange occurs at several hundred nucleosomes (&amp;lt;1\% of all nucleosomes) per minute, with a maximal rate at histone promoters. We observed substantial differences between the two nucleosome core subcomplexes: H2A-H2B subcomplexes undergo rapid transcription-dependent replacement within coding regions, whereas H3-H4 replacement occurs predominantly within promoter nucleosomes, in association with gene activation or repression. Our timers allow the in vivo study of histone exchange dynamics with minute time scale resolution. The dynamics of histone exchange are measured with genetically encoded timers.},
  archive      = {J_NBT},
  author       = {Yaakov, Gilad and Jonas, Felix and Barkai, Naama},
  doi          = {10.1038/s41587-021-00959-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1434-1443},
  shortjournal = {Nature Biotech.},
  title        = {Measurement of histone replacement dynamics with genetically encoded exchange timers in yeast},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Adenine base editor engineering reduces editing of bystander
cytosines. <em>NBT</em>, <em>39</em>(11), 1426–1433. (<a
href="https://doi.org/10.1038/s41587-021-00943-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adenine base editors (ABEs) catalyze specific A-to-G conversions at genomic sites of interest. However, ABEs also induce cytosine deamination at the target site. To reduce the cytosine editing activity, we engineered a commonly used adenosine deaminase, TadA7.10, and found that ABE7.10 with a D108Q mutation in TadA7.10 exhibited tenfold reduced cytosine deamination activity. The D108Q mutation also reduces cytosine deamination activity in two recently developed high-activity versions of ABE, ABE8e and ABE8s, and is compatible with V106W, a mutation that reduces off-target RNA editing. ABE7.10 containing a P48R mutation displayed increased cytosine deamination activity and a substantially reduced adenine editing rate, yielding a TC-specific base editing tool for TC-to-TT or TC-to-TG conversions that broadens the utility of base editors. Engineered variants of adenine base editors have reduced cytosine base editing or a specific C-to-G base editing activity.},
  archive      = {J_NBT},
  author       = {Jeong, You Kyeong and Lee, SeokHoon and Hwang, Gue-Ho and Hong, Sung-Ah and Park, Se-eun and Kim, Jin-Soo and Woo, Jae-Sung and Bae, Sangsu},
  doi          = {10.1038/s41587-021-00943-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1426-1433},
  shortjournal = {Nature Biotech.},
  title        = {Adenine base editor engineering reduces editing of bystander cytosines},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Efficient c•g-to-g•c base editors developed using CRISPRi
screens, target-library analysis, and machine learning. <em>NBT</em>,
<em>39</em>(11), 1414–1425. (<a
href="https://doi.org/10.1038/s41587-021-00938-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Programmable C•G-to-G•C base editors (CGBEs) have broad scientific and therapeutic potential, but their editing outcomes have proved difficult to predict and their editing efficiency and product purity are often low. We describe a suite of engineered CGBEs paired with machine learning models to enable efficient, high-purity C•G-to-G•C base editing. We performed a CRISPR interference (CRISPRi) screen targeting DNA repair genes to identify factors that affect C•G-to-G•C editing outcomes and used these insights to develop CGBEs with diverse editing profiles. We characterized ten promising CGBEs on a library of 10,638 genomically integrated target sites in mammalian cells and trained machine learning models that accurately predict the purity and yield of editing outcomes (R = 0.90) using these data. These CGBEs enable correction to the wild-type coding sequence of 546 disease-related transversion single-nucleotide variants (SNVs) with &amp;gt;90\% precision (mean 96\%) and up to 70\% efficiency (mean 14\%). Computational prediction of optimal CGBE–single-guide RNA pairs enables high-purity transversion base editing at over fourfold more target sites than achieved using any single CGBE variant. Efficiency of transversion base editing is increased by matching a set of base editors to target sequences.},
  archive      = {J_NBT},
  author       = {Koblan, Luke W. and Arbab, Mandana and Shen, Max W. and Hussmann, Jeffrey A. and Anzalone, Andrew V. and Doman, Jordan L. and Newby, Gregory A. and Yang, Dian and Mok, Beverly and Replogle, Joseph M. and Xu, Albert and Sisley, Tyler A. and Weissman, Jonathan S. and Adamson, Britt and Liu, David R.},
  doi          = {10.1038/s41587-021-00938-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1414-1425},
  shortjournal = {Nature Biotech.},
  title        = {Efficient C•G-to-G•C base editors developed using CRISPRi screens, target-library analysis, and machine learning},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-wide interrogation of gene functions through base
editor screens empowered by barcoded sgRNAs. <em>NBT</em>,
<em>39</em>(11), 1403–1413. (<a
href="https://doi.org/10.1038/s41587-021-00944-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Canonical CRISPR–knockout (KO) screens rely on Cas9-induced DNA double-strand breaks (DSBs) to generate targeted gene KOs. These methodologies may yield distorted results because DSB-associated effects are often falsely assumed to be consequences of gene perturbation itself, especially when high copy-number sites are targeted. In the present study, we report a DSB-independent, genome-wide CRISPR screening method, termed iBARed cytosine base editing-mediated gene KO (BARBEKO). This method leverages CRISPR cytosine base editors for genome-scale KO screens by perturbing gene start codons or splice sites, or by introducing premature termination codons. Furthermore, it is integrated with iBAR, a strategy we devised for improving screening quality and efficiency. By constructing such a cell library through lentiviral infection at a high multiplicity of infection (up to 10), we achieved efficient and accurate screening results with substantially reduced starting cells. More importantly, in comparison with Cas9-mediated fitness screens, BARBEKO screens are no longer affected by DNA cleavage-induced cytotoxicity in HeLa-, K562- or DSB-sensitive retinal pigmented epithelial 1 cells. We anticipate that BARBEKO offers a valuable tool to complement the current CRISPR–KO screens in various settings. CRISPR genetic screens with base editors avoid the confounding effects of DNA breaks.},
  archive      = {J_NBT},
  author       = {Xu, Ping and Liu, Zhiheng and Liu, Ying and Ma, Huazheng and Xu, Yiyuan and Bao, Ying and Zhu, Shiyou and Cao, Zhongzheng and Wu, Zeguang and Zhou, Zhuo and Wei, Wensheng},
  doi          = {10.1038/s41587-021-00944-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1403-1413},
  shortjournal = {Nature Biotech.},
  title        = {Genome-wide interrogation of gene functions through base editor screens empowered by barcoded sgRNAs},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Identification of differential RNA modifications from
nanopore direct RNA sequencing with xPore. <em>NBT</em>,
<em>39</em>(11), 1394–1402. (<a
href="https://doi.org/10.1038/s41587-021-00949-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA modifications, such as N6-methyladenosine (m6A), modulate functions of cellular RNA species. However, quantifying differences in RNA modifications has been challenging. Here we develop a computational method, xPore, to identify differential RNA modifications from nanopore direct RNA sequencing (RNA-seq) data. We evaluate our method on transcriptome-wide m6A profiling data, demonstrating that xPore identifies positions of m6A sites at single-base resolution, estimates the fraction of modified RNA species in the cell and quantifies the differential modification rate across conditions. We apply xPore to direct RNA-seq data from six cell lines and multiple myeloma patient samples without a matched control sample and find that many m6A sites are preserved across cell types, whereas a subset exhibit significant differences in their modification rates. Our results show that RNA modifications can be identified from direct RNA-seq data with high accuracy, enabling analysis of differential modifications and expression from a single high-throughput experiment. m6A RNA modifications are quantified in cancer patient samples and cell lines using nanopore sequencing.},
  archive      = {J_NBT},
  author       = {Pratanwanich, Ploy N. and Yao, Fei and Chen, Ying and Koh, Casslynn W. Q. and Wan, Yuk Kei and Hendra, Christopher and Poon, Polly and Goh, Yeek Teck and Yap, Phoebe M. L. and Chooi, Jing Yuan and Chng, Wee Joo and Ng, Sarah B. and Thiery, Alexandre and Goh, W. S. Sho and Göke, Jonathan},
  doi          = {10.1038/s41587-021-00949-w},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1394-1402},
  shortjournal = {Nature Biotech.},
  title        = {Identification of differential RNA modifications from nanopore direct RNA sequencing with xPore},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Volumetric interferometric lattice light-sheet imaging.
<em>NBT</em>, <em>39</em>(11), 1385–1393. (<a
href="https://doi.org/10.1038/s41587-021-01042-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Live cell imaging with high spatiotemporal resolution and high detection sensitivity facilitates the study of the dynamics of cellular structure and function. However, extracting high-resolution 4D (3D space plus time) information from live cells remains challenging, because current methods are slow, require high peak excitation intensities or suffer from high out-of-focus background. Here we present 3D interferometric lattice light-sheet (3D-iLLS) imaging, a technique that requires low excitation light levels and provides high background suppression and substantially improved volumetric resolution by combining 4Pi interferometry with selective plane illumination. We demonstrate that 3D-iLLS has an axial resolution and single-particle localization precision of 100 nm (FWHM) and &amp;lt;10 nm (1σ), respectively. We illustrate the performance of 3D-iLLS in a range of systems: single messenger RNA molecules, nanoscale assemblies of transcription regulators in the nucleus, the microtubule cytoskeleton and mitochondria organelles. The enhanced 4D resolution and increased signal-to-noise ratio of 3D-iLLS will facilitate the analysis of biological processes at the sub-cellular level. New lattice light-sheet microscopy approach improves live cell imaging.},
  archive      = {J_NBT},
  author       = {Cao, Bin and Coelho, Simao and Li, Jieru and Wang, Guanshi and Pertsinidis, Alexandros},
  doi          = {10.1038/s41587-021-01042-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1385-1393},
  shortjournal = {Nature Biotech.},
  title        = {Volumetric interferometric lattice light-sheet imaging},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Spatial transcriptomics at subspot resolution with
BayesSpace. <em>NBT</em>, <em>39</em>(11), 1375–1384. (<a
href="https://doi.org/10.1038/s41587-021-00935-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent spatial gene expression technologies enable comprehensive measurement of transcriptomic profiles while retaining spatial context. However, existing analysis methods do not address the limited resolution of the technology or use the spatial information efficiently. Here, we introduce BayesSpace, a fully Bayesian statistical method that uses the information from spatial neighborhoods for resolution enhancement of spatial transcriptomic data and for clustering analysis. We benchmark BayesSpace against current methods for spatial and non-spatial clustering and show that it improves identification of distinct intra-tissue transcriptional profiles from samples of the brain, melanoma, invasive ductal carcinoma and ovarian adenocarcinoma. Using immunohistochemistry and an in silico dataset constructed from scRNA-seq data, we show that BayesSpace resolves tissue structure that is not detectable at the original resolution and identifies transcriptional heterogeneity inaccessible to histological analysis. Our results illustrate BayesSpace’s utility in facilitating the discovery of biological insights from spatial transcriptomic datasets. BayesSpace increases the resolution of spatial transcriptomics by using neighborhood information.},
  archive      = {J_NBT},
  author       = {Zhao, Edward and Stone, Matthew R. and Ren, Xing and Guenthoer, Jamie and Smythe, Kimberly S. and Pulliam, Thomas and Williams, Stephen R. and Uytingco, Cedric R. and Taylor, Sarah E. B. and Nghiem, Paul and Bielas, Jason H. and Gottardo, Raphael},
  doi          = {10.1038/s41587-021-00935-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1375-1384},
  shortjournal = {Nature Biotech.},
  title        = {Spatial transcriptomics at subspot resolution with BayesSpace},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Wearable materials with embedded synthetic biology sensors
for biomolecule detection. <em>NBT</em>, <em>39</em>(11), 1366–1374. (<a
href="https://doi.org/10.1038/s41587-021-00950-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integrating synthetic biology into wearables could expand opportunities for noninvasive monitoring of physiological status, disease states and exposure to pathogens or toxins. However, the operation of synthetic circuits generally requires the presence of living, engineered bacteria, which has limited their application in wearables. Here we report lightweight, flexible substrates and textiles functionalized with freeze-dried, cell-free synthetic circuits, including CRISPR-based tools, that detect metabolites, chemicals and pathogen nucleic acid signatures. The wearable devices are activated upon rehydration from aqueous exposure events and report the presence of specific molecular targets by colorimetric changes or via an optical fiber network that detects fluorescent and luminescent outputs. The detection limits for nucleic acids rival current laboratory methods such as quantitative PCR. We demonstrate the development of a face mask with a lyophilized CRISPR sensor for wearable, noninvasive detection of SARS-CoV-2 at room temperature within 90 min, requiring no user intervention other than the press of a button. Wearable materials are endowed with synthetic biology circuits to detect biomolecules, including SARS-CoV-2 RNA.},
  archive      = {J_NBT},
  author       = {Nguyen, Peter Q. and Soenksen, Luis R. and Donghia, Nina M. and Angenent-Mari, Nicolaas M. and de Puig, Helena and Huang, Ally and Lee, Rose and Slomovic, Shimyn and Galbersanini, Tommaso and Lansberry, Geoffrey and Sallum, Hani M. and Zhao, Evan M. and Niemi, James B. and Collins, James J.},
  doi          = {10.1038/s41587-021-00950-3},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1366-1374},
  shortjournal = {Nature Biotech.},
  title        = {Wearable materials with embedded synthetic biology sensors for biomolecule detection},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Nanopore sequencing technology, bioinformatics and
applications. <em>NBT</em>, <em>39</em>(11), 1348–1365. (<a
href="https://doi.org/10.1038/s41587-021-01108-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rapid advances in nanopore technologies for sequencing single long DNA and RNA molecules have led to substantial improvements in accuracy, read length and throughput. These breakthroughs have required extensive development of experimental and bioinformatics methods to fully exploit nanopore long reads for investigations of genomes, transcriptomes, epigenomes and epitranscriptomes. Nanopore sequencing is being applied in genome assembly, full-length transcript detection and base modification detection and in more specialized areas, such as rapid clinical diagnoses and outbreak surveillance. Many opportunities remain for improving data quality and analytical approaches through the development of new nanopores, base-calling methods and experimental protocols tailored to particular applications. Au and colleagues outline the field of nanopore sequencing.},
  archive      = {J_NBT},
  author       = {Wang, Yunhao and Zhao, Yue and Bollas, Audrey and Wang, Yuru and Au, Kin Fai},
  doi          = {10.1038/s41587-021-01108-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1348-1365},
  shortjournal = {Nature Biotech.},
  title        = {Nanopore sequencing technology, bioinformatics and applications},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Hope lies in dreams. <em>NBT</em>, <em>39</em>(11), 1347.
(<a href="https://doi.org/10.1038/s41587-021-01106-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01106-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1347},
  shortjournal = {Nature Biotech.},
  title        = {Hope lies in dreams},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Light-sheet microscopy at high resolution. <em>NBT</em>,
<em>39</em>(11), 1345–1346. (<a
href="https://doi.org/10.1038/s41587-021-01101-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An improved light-sheet microscope images live cells at sub-100-nm axial resolution.},
  archive      = {J_NBT},
  author       = {Fiolka, Reto},
  doi          = {10.1038/s41587-021-01101-4},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1345-1346},
  shortjournal = {Nature Biotech.},
  title        = {Light-sheet microscopy at high resolution},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Drug delivery. <em>NBT</em>, <em>39</em>(11), 1344. (<a
href="https://doi.org/10.1038/s41587-021-01118-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to methods of delivering RNA, protein and antibody therapeutics.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01118-9},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1344},
  shortjournal = {Nature Biotech.},
  title        = {Drug delivery},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mapping the european patent landscape for medical uses of
known products. <em>NBT</em>, <em>39</em>(11), 1336–1343. (<a
href="https://doi.org/10.1038/s41587-021-01104-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {European Patent Office data show an increasing number of patents for new medical uses of known products.},
  archive      = {J_NBT},
  author       = {Aboy, Mateo and Liddell, Kathleen and Liddicoat, Johnathon and Crespo, Cristina and Jordan, Matthew},
  doi          = {10.1038/s41587-021-01104-1},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1336-1343},
  shortjournal = {Nature Biotech.},
  title        = {Mapping the european patent landscape for medical uses of known products},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Will the biden administration “follow the science” on
regulatory reform? <em>NBT</em>, <em>39</em>(11), 1334–1335. (<a
href="https://doi.org/10.1038/s41587-021-01084-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Cohrssen, John J. and Miller, Henry I.},
  doi          = {10.1038/s41587-021-01084-2},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1334-1335},
  shortjournal = {Nature Biotech.},
  title        = {Will the biden administration ‘follow the science’ on regulatory reform?},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Top 20 translational researchers of 2020. <em>NBT</em>,
<em>39</em>(11), 1332–1333. (<a
href="https://doi.org/10.1038/s41587-021-01107-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Huggett, Brady and Paisner, Kathryn},
  doi          = {10.1038/s41587-021-01107-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1332-1333},
  shortjournal = {Nature Biotech.},
  title        = {Top 20 translational researchers of 2020},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). 3Q21—the market stalls. <em>NBT</em>, <em>39</em>(11),
1329–1331. (<a
href="https://doi.org/10.1038/s41587-021-01114-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-01114-z},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1329-1331},
  shortjournal = {Nature Biotech.},
  title        = {3Q21—the market stalls},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). Drug pipeline 3Q21—mixed news for genetic therapies.
<em>NBT</em>, <em>39</em>(11), 1326–1328. (<a
href="https://doi.org/10.1038/s41587-021-01119-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-01119-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1326-1328},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 3Q21—mixed news for genetic therapies},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Biotech news from around the world. <em>NBT</em>,
<em>39</em>(11), 1325. (<a
href="https://doi.org/10.1038/s41587-021-01123-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01123-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1325},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Quantum computers to explore precision oncology.
<em>NBT</em>, <em>39</em>(11), 1324–1325. (<a
href="https://doi.org/10.1038/s41587-021-01116-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Life scientists are preparing to test quantum computers for applications beyond computational chemistry, such as selecting responders to cancer therapies.},
  archive      = {J_NBT},
  author       = {Abbott, Alison},
  doi          = {10.1038/s41587-021-01116-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1324-1325},
  shortjournal = {Nature Biotech.},
  title        = {Quantum computers to explore precision oncology},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). ROCK2 agent stops GVHD. <em>NBT</em>, <em>39</em>(11), 1323.
(<a href="https://doi.org/10.1038/s41587-021-01128-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01128-7},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1323},
  shortjournal = {Nature Biotech.},
  title        = {ROCK2 agent stops GVHD},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Malaria’s WHO moment. <em>NBT</em>, <em>39</em>(11), 1322.
(<a href="https://doi.org/10.1038/s41587-021-01129-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01129-6},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1322},
  shortjournal = {Nature Biotech.},
  title        = {Malaria’s WHO moment},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021n). Sickle cell gene therapies approach watershed.
<em>NBT</em>, <em>39</em>(11), 1320–1323. (<a
href="https://doi.org/10.1038/s41587-021-01121-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Early results promise a one-off treatment, but gene-editing and gene therapies will have to show durable protection and be affordable to the many patients with sickle cell disease in the world.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-021-01121-0},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1320-1323},
  shortjournal = {Nature Biotech.},
  title        = {Sickle cell gene therapies approach watershed},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). First gene therapy for adrenoleukodystrophy. <em>NBT</em>,
<em>39</em>(11), 1319. (<a
href="https://doi.org/10.1038/s41587-021-01127-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01127-8},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1319},
  shortjournal = {Nature Biotech.},
  title        = {First gene therapy for adrenoleukodystrophy},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). Treg engineers take aim at autoimmunity. <em>NBT</em>,
<em>39</em>(11), 1317–1319. (<a
href="https://doi.org/10.1038/s41587-021-01115-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are deploying regulatory T cells, equipped with CAR and TCR constructs, to suppress immunity—locally and for the long term.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-021-01115-y},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1317-1319},
  shortjournal = {Nature Biotech.},
  title        = {Treg engineers take aim at autoimmunity},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). How biomanufacturing can save the world. <em>NBT</em>,
<em>39</em>(11), 1315. (<a
href="https://doi.org/10.1038/s41587-021-01132-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With supply-chain shortages and production problems exacerbating vaccine inequity, global manufacturing must expand to ensure the world’s poorest nations are better served.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01132-x},
  journal      = {Nature Biotechnology},
  month        = {11},
  number       = {11},
  pages        = {1315},
  shortjournal = {Nature Biotech.},
  title        = {How biomanufacturing can save the world},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). People. <em>NBT</em>, <em>39</em>(10), 1314. (<a
href="https://doi.org/10.1038/s41587-021-01086-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01086-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1314},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Improving scientific communication with service, education
and career development. <em>NBT</em>, <em>39</em>(10), 1309–1313. (<a
href="https://doi.org/10.1038/s41587-021-01077-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A trainee-run group that provides editing and schema design services not only enhances scientific communication for clients, but also trains its members in communication, writing and other transferrable skills.},
  archive      = {J_NBT},
  author       = {Dicks, Amanda and Bhatia, Himanshi and Clemens, Adam W. and Locke, Marissa C. and Mueller, Elizabeth A. and Murphy, Daniel and Pomper, Nathan and Robinson, Anne E. and Schoch, Kathleen M.},
  doi          = {10.1038/s41587-021-01077-1},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1309-1313},
  shortjournal = {Nature Biotech.},
  title        = {Improving scientific communication with service, education and career development},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Publisher correction: Send in the senolytics. <em>NBT</em>,
<em>39</em>(10), 1308. (<a
href="https://doi.org/10.1038/s41587-021-01073-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-021-01073-5},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1308},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Send in the senolytics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Learning on knowledge graph dynamics provides an early
warning of impactful research. <em>NBT</em>, <em>39</em>(10), 1300–1307.
(<a href="https://doi.org/10.1038/s41587-021-00907-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The scientific ecosystem relies on citation-based metrics that provide only imperfect, inconsistent and easily manipulated measures of research quality. Here we describe DELPHI (Dynamic Early-warning by Learning to Predict High Impact), a framework that provides an early-warning signal for ‘impactful’ research by autonomously learning high-dimensional relationships among features calculated across time from the scientific literature. We prototype this framework and deduce its performance and scaling properties on time-structured publication graphs from 1980 to 2019 drawn from 42 biotechnology-related journals, including over 7.8 million individual nodes, 201 million relationships and 3.8 billion calculated metrics. We demonstrate the framework’s performance by correctly identifying 19/20 seminal biotechnologies from 1980 to 2014 via a blinded retrospective study and provide 50 research papers from 2018 that DELPHI predicts will be in the top 5\% of time-rescaled node centrality in the future. We propose DELPHI as a tool to aid in the construction of diversified, impact-optimized funding portfolios. Biotechnology-related papers predicted to be of long-term impact are identified in a machine learning framework (DELPHI) that analyzes relationships among a range of features from the scientific literature over time.},
  archive      = {J_NBT},
  author       = {Weis, James W. and Jacobson, Joseph M.},
  doi          = {10.1038/s41587-021-00907-6},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1300-1307},
  shortjournal = {Nature Biotech.},
  title        = {Learning on knowledge graph dynamics provides an early warning of impactful research},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Genome-wide specificity of prime editors in plants.
<em>NBT</em>, <em>39</em>(10), 1292–1299. (<a
href="https://doi.org/10.1038/s41587-021-00891-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although prime editors (PEs) have the potential to facilitate precise genome editing in therapeutic, agricultural and research applications, their specificity has not been comprehensively evaluated. To provide a systematic assessment in plants, we first examined the mismatch tolerance of PEs in plant cells and found that the editing frequency was influenced by the number and location of mismatches in the primer binding site and spacer of the prime editing guide RNA (pegRNA). Assessing the activity of 12 pegRNAs at 179 predicted off-target sites, we detected only low frequencies of off-target edits (0.00~0.23\%). Whole-genome sequencing of 29 PE-treated rice plants confirmed that PEs do not induce genome-wide pegRNA-independent off-target single-nucleotide variants or small insertions/deletions. We also show that ectopic expression of the Moloney murine leukemia virus reverse transcriptase as part of the PE does not change retrotransposon copy number or telomere structure or cause insertion of pegRNA or messenger RNA sequences into the genome. A very high specificity of prime editors is demonstrated in a study of their off-target activity in plants.},
  archive      = {J_NBT},
  author       = {Jin, Shuai and Lin, Qiupeng and Luo, Yingfeng and Zhu, Zixu and Liu, Guanwen and Li, Yunjia and Chen, Kunling and Qiu, Jin-Long and Gao, Caixia},
  doi          = {10.1038/s41587-021-00891-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1292-1299},
  shortjournal = {Nature Biotech.},
  title        = {Genome-wide specificity of prime editors in plants},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Quantitative profiling of pseudouridylation dynamics in
native RNAs with nanopore sequencing. <em>NBT</em>, <em>39</em>(10),
1278–1291. (<a
href="https://doi.org/10.1038/s41587-021-00915-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nanopore RNA sequencing shows promise as a method for discriminating and identifying different RNA modifications in native RNA. Expanding on the ability of nanopore sequencing to detect N6-methyladenosine, we show that other modifications, in particular pseudouridine (Ψ) and 2′-O-methylation (Nm), also result in characteristic base-calling ‘error’ signatures in the nanopore data. Focusing on Ψ modification sites, we detected known and uncovered previously unreported Ψ sites in mRNAs, non-coding RNAs and rRNAs, including a Pus4-dependent Ψ modification in yeast mitochondrial rRNA. To explore the dynamics of pseudouridylation, we treated yeast cells with oxidative, cold and heat stresses and detected heat-sensitive Ψ-modified sites in small nuclear RNAs, small nucleolar RNAs and mRNAs. Finally, we developed a software, nanoRMS, that estimates per-site modification stoichiometries by identifying single-molecule reads with altered current intensity and trace profiles. This work demonstrates that Nm and Ψ RNA modifications can be detected in cellular RNAs and that their modification stoichiometry can be quantified by nanopore sequencing of native RNA. Nanopore sequencing detects pseudouridine and 2′-O-methylation modifications in cellular RNAs.},
  archive      = {J_NBT},
  author       = {Begik, Oguzhan and Lucas, Morghan C. and Pryszcz, Leszek P. and Ramirez, Jose Miguel and Medina, Rebeca and Milenkovic, Ivan and Cruciani, Sonia and Liu, Huanle and Vieira, Helaine Graziele Santos and Sas-Chen, Aldema and Mattick, John S. and Schwartz, Schraga and Novoa, Eva Maria},
  doi          = {10.1038/s41587-021-00915-6},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1278-1291},
  shortjournal = {Nature Biotech.},
  title        = {Quantitative profiling of pseudouridylation dynamics in native RNAs with nanopore sequencing},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Profiling the genetic determinants of chromatin
accessibility with scalable single-cell CRISPR screens. <em>NBT</em>,
<em>39</em>(10), 1270–1277. (<a
href="https://doi.org/10.1038/s41587-021-00902-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR screens have been used to connect genetic perturbations with changes in gene expression and phenotypes. Here we describe a CRISPR-based, single-cell combinatorial indexing assay for transposase-accessible chromatin (CRISPR–sciATAC) to link genetic perturbations to genome-wide chromatin accessibility in a large number of cells. In human myelogenous leukemia cells, we apply CRISPR–sciATAC to target 105 chromatin-related genes, generating chromatin accessibility data for ~30,000 single cells. We correlate the loss of specific chromatin remodelers with changes in accessibility globally and at the binding sites of individual transcription factors (TFs). For example, we show that loss of the H3K27 methyltransferase EZH2 increases accessibility at heterochromatic regions involved in embryonic development and triggers expression of genes in the HOXA and HOXD clusters. At a subset of regulatory sites, we also analyze changes in nucleosome spacing following the loss of chromatin remodelers. CRISPR–sciATAC is a high-throughput, single-cell method for studying the effect of genetic perturbations on chromatin in normal and disease states. The effects of gene knockouts on chromatin accessibility are measured with single-cell CRISPR screens.},
  archive      = {J_NBT},
  author       = {Liscovitch-Brauer, Noa and Montalbano, Antonino and Deng, Jiale and Méndez-Mancilla, Alejandro and Wessels, Hans-Hermann and Moss, Nicholas G. and Kung, Chia-Yu and Sookdeo, Akash and Guo, Xinyi and Geller, Evan and Jaini, Suma and Smibert, Peter and Sanjana, Neville E.},
  doi          = {10.1038/s41587-021-00902-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1270-1277},
  shortjournal = {Nature Biotech.},
  title        = {Profiling the genetic determinants of chromatin accessibility with scalable single-cell CRISPR screens},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Integrative single-cell analysis of allele-specific copy
number alterations and chromatin accessibility in cancer. <em>NBT</em>,
<em>39</em>(10), 1259–1269. (<a
href="https://doi.org/10.1038/s41587-021-00911-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cancer progression is driven by both somatic copy number aberrations (CNAs) and chromatin remodeling, yet little is known about the interplay between these two classes of events in shaping the clonal diversity of cancers. We present Alleloscope, a method for allele-specific copy number estimation that can be applied to single-cell DNA- and/or transposase-accessible chromatin-sequencing (scDNA-seq, ATAC-seq) data, enabling combined analysis of allele-specific copy number and chromatin accessibility. On scDNA-seq data from gastric, colorectal and breast cancer samples, with validation using matched linked-read sequencing, Alleloscope finds pervasive occurrence of highly complex, multiallelic CNAs, in which cells that carry varying allelic configurations adding to the same total copy number coevolve within a tumor. On scATAC-seq from two basal cell carcinoma samples and a gastric cancer cell line, Alleloscope detected multiallelic copy number events and copy-neutral loss-of-heterozygosity, enabling dissection of the contributions of chromosomal instability and chromatin remodeling to tumor evolution. Alleloscope reveals how tumor evolution depends on allele-specific copy number changes and chromatin remodeling.},
  archive      = {J_NBT},
  author       = {Wu, Chi-Yun and Lau, Billy T. and Kim, Heon Seok and Sathe, Anuja and Grimes, Susan M. and Ji, Hanlee P. and Zhang, Nancy R.},
  doi          = {10.1038/s41587-021-00911-w},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1259-1269},
  shortjournal = {Nature Biotech.},
  title        = {Integrative single-cell analysis of allele-specific copy number alterations and chromatin accessibility in cancer},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Scalable, multimodal profiling of chromatin accessibility,
gene expression and protein levels in single cells. <em>NBT</em>,
<em>39</em>(10), 1246–1258. (<a
href="https://doi.org/10.1038/s41587-021-00927-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent technological advances have enabled massively parallel chromatin profiling with scATAC-seq (single-cell assay for transposase accessible chromatin by sequencing). Here we present ATAC with select antigen profiling by sequencing (ASAP-seq), a tool to simultaneously profile accessible chromatin and protein levels. Our approach pairs sparse scATAC-seq data with robust detection of hundreds of cell surface and intracellular protein markers and optional capture of mitochondrial DNA for clonal tracking, capturing three distinct modalities in single cells. ASAP-seq uses a bridging approach that repurposes antibody:oligonucleotide conjugates designed for existing technologies that pair protein measurements with single-cell RNA sequencing. Together with DOGMA-seq, an adaptation of CITE-seq (cellular indexing of transcriptomes and epitopes by sequencing) for measuring gene activity across the central dogma of gene regulation, we demonstrate the utility of systematic multi-omic profiling by revealing coordinated and distinct changes in chromatin, RNA and surface proteins during native hematopoietic differentiation and peripheral blood mononuclear cell stimulation and as a combinatorial decoder and reporter of multiplexed perturbations in primary T cells. Chromatin accessibility, gene expression and protein levels are measured in the same single cell.},
  archive      = {J_NBT},
  author       = {Mimitou, Eleni P. and Lareau, Caleb A. and Chen, Kelvin Y. and Zorzetto-Fernandes, Andre L. and Hao, Yuhan and Takeshima, Yusuke and Luo, Wendy and Huang, Tse-Shun and Yeung, Bertrand Z. and Papalexi, Efthymia and Thakore, Pratiksha I. and Kibayashi, Tatsuya and Wing, James Badger and Hata, Mayu and Satija, Rahul and Nazor, Kristopher L. and Sakaguchi, Shimon and Ludwig, Leif S. and Sankaran, Vijay G. and Regev, Aviv and Smibert, Peter},
  doi          = {10.1038/s41587-021-00927-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1246-1258},
  shortjournal = {Nature Biotech.},
  title        = {Scalable, multimodal profiling of chromatin accessibility, gene expression and protein levels in single cells},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Revealing the spatio-phenotypic patterning of cells in
healthy and tumor tissues with mLSR-3D and STAPL-3D. <em>NBT</em>,
<em>39</em>(10), 1239–1245. (<a
href="https://doi.org/10.1038/s41587-021-00926-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite advances in three-dimensional (3D) imaging, it remains challenging to profile all the cells within a large 3D tissue, including the morphology and organization of the many cell types present. Here, we introduce eight-color, multispectral, large-scale single-cell resolution 3D (mLSR-3D) imaging and image analysis software for the parallelized, deep learning–based segmentation of large numbers of single cells in tissues, called segmentation analysis by parallelization of 3D datasets (STAPL-3D). Applying the method to pediatric Wilms tumor, we extract molecular, spatial and morphological features of millions of cells and reconstruct the tumor’s spatio-phenotypic patterning. In situ population profiling and pseudotime ordering reveals a highly disorganized spatial pattern in Wilms tumor compared to healthy fetal kidney, yet cellular profiles closely resembling human fetal kidney cells could be observed. In addition, we identify previously unreported tumor-specific populations, uniquely characterized by their spatial embedding or morphological attributes. Our results demonstrate the use of combining mLSR-3D and STAPL-3D to generate a comprehensive cellular map of human tumors. An imaging software for large-scale microscopy data shows how cells are organized in tissues.},
  archive      = {J_NBT},
  author       = {van Ineveld, Ravian L. and Kleinnijenhuis, Michiel and Alieva, Maria and de Blank, Sam and Barrera Roman, Mario and van Vliet, Esmée J. and Martínez Mir, Clara and Johnson, Hannah R. and Bos, Frank L. and Heukers, Raimond and Chuva de Sousa Lopes, Susana M. and Drost, Jarno and Dekkers, Johanna F. and Wehrens, Ellen J. and Rios, Anne C.},
  doi          = {10.1038/s41587-021-00926-3},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1239-1245},
  shortjournal = {Nature Biotech.},
  title        = {Revealing the spatio-phenotypic patterning of cells in healthy and tumor tissues with mLSR-3D and STAPL-3D},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fully implantable and bioresorbable cardiac pacemakers
without leads or batteries. <em>NBT</em>, <em>39</em>(10), 1228–1238.
(<a href="https://doi.org/10.1038/s41587-021-00948-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Temporary cardiac pacemakers used in periods of need during surgical recovery involve percutaneous leads and externalized hardware that carry risks of infection, constrain patient mobility and may damage the heart during lead removal. Here we report a leadless, battery-free, fully implantable cardiac pacemaker for postoperative control of cardiac rate and rhythm that undergoes complete dissolution and clearance by natural biological processes after a defined operating timeframe. We show that these devices provide effective pacing of hearts of various sizes in mouse, rat, rabbit, canine and human cardiac models, with tailored geometries and operation timescales, powered by wireless energy transfer. This approach overcomes key disadvantages of traditional temporary pacing devices and may serve as the basis for the next generation of postoperative temporary pacing technology. A biodegradable pacemaker without external leads improves the safety of temporary cardiac pacing.},
  archive      = {J_NBT},
  author       = {Choi, Yeon Sik and Yin, Rose T. and Pfenniger, Anna and Koo, Jahyun and Avila, Raudel and Benjamin Lee, K. and Chen, Sheena W. and Lee, Geumbee and Li, Gang and Qiao, Yun and Murillo-Berlioz, Alejandro and Kiss, Alexi and Han, Shuling and Lee, Seung Min and Li, Chenhang and Xie, Zhaoqian and Chen, Yu-Yu and Burrell, Amy and Geist, Beth and Jeong, Hyoyoung and Kim, Joohee and Yoon, Hong-Joon and Banks, Anthony and Kang, Seung-Kyun and Zhang, Zheng Jenny and Haney, Chad R. and Sahakian, Alan Varteres and Johnson, David and Efimova, Tatiana and Huang, Yonggang and Trachiotis, Gregory D. and Knight, Bradley P. and Arora, Rishi K. and Efimov, Igor R. and Rogers, John A.},
  doi          = {10.1038/s41587-021-00948-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1228-1238},
  shortjournal = {Nature Biotech.},
  title        = {Fully implantable and bioresorbable cardiac pacemakers without leads or batteries},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Detection of low-frequency DNA variants by targeted
sequencing of the watson and crick strands. <em>NBT</em>,
<em>39</em>(10), 1220–1227. (<a
href="https://doi.org/10.1038/s41587-021-00900-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identification and quantification of low-frequency mutations remain challenging despite improvements in the baseline error rate of next-generation sequencing technologies. Here, we describe a method, termed SaferSeqS, that addresses these challenges by (1) efficiently introducing identical molecular barcodes in the Watson and Crick strands of template molecules and (2) enriching target sequences with strand-specific PCR. The method achieves high sensitivity and specificity and detects variants at frequencies below 1 in 100,000 DNA template molecules with a background mutation rate of &amp;lt;5 × 10–7 mutants per base pair (bp). We demonstrate that it can evaluate mutations in a single amplicon or simultaneously in multiple amplicons, assess limited quantities of cell-free DNA with high recovery of both strands and reduce the error rate of existing PCR-based molecular barcoding approaches by &amp;gt;100-fold. Mutations present at a low frequency in a sample are detected with high sensitivity and a low error rate.},
  archive      = {J_NBT},
  author       = {Cohen, Joshua D. and Douville, Christopher and Dudley, Jonathan C. and Mog, Brian J. and Popoli, Maria and Ptak, Janine and Dobbyn, Lisa and Silliman, Natalie and Schaefer, Joy and Tie, Jeanne and Gibbs, Peter and Tomasetti, Cristian and Papadopoulos, Nickolas and Kinzler, Kenneth W. and Vogelstein, Bert},
  doi          = {10.1038/s41587-021-00900-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1220-1227},
  shortjournal = {Nature Biotech.},
  title        = {Detection of low-frequency DNA variants by targeted sequencing of the watson and crick strands},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Field trial demonstrating phytoremediation of the military
explosive RDX by XplA/XplB-expressing switchgrass. <em>NBT</em>,
<em>39</em>(10), 1216–1219. (<a
href="https://doi.org/10.1038/s41587-021-00909-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The explosive hexahydro-1,3,5-trinitro-1,3,5-triazine (RDX), a major component of munitions, is used extensively on military training ranges. As a result, widespread RDX pollution in groundwater and aquifers in the United States is now well documented. RDX is toxic, but its removal from training ranges is logistically challenging, lacking cost-effective and sustainable solutions. Previously, we have shown that thale cress (Arabidopsis thaliana) engineered to express two genes, xplA and xplB, encoding RDX-degrading enzymes from the soil bacterium Rhodococcus rhodochrous 11Y can break down this xenobiotic in laboratory studies. Here, we report the results of a 3-year field trial of XplA/XplB-expressing switchgrass (Panicum virgatum) conducted on three locations in a military site. Our data suggest that XplA/XplB switchgrass has in situ efficacy, with potential utility for detoxifying RDX on live-fire training ranges, munitions dumps and minefields. Switchgrass engineered to express two enzymes that degrade the explosive hexahydro-1,3,5-trinitro-1,3,5-triazine (RDX) detoxifies military sites contaminated with the munition when grown on these plots in a 3-year field trial.},
  archive      = {J_NBT},
  author       = {Cary, Timothy J. and Rylott, Elizabeth L. and Zhang, Long and Routsong, Ryan M. and Palazzo, Antonio J. and Strand, Stuart E. and Bruce, Neil C.},
  doi          = {10.1038/s41587-021-00909-4},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1216-1219},
  shortjournal = {Nature Biotech.},
  title        = {Field trial demonstrating phytoremediation of the military explosive RDX by XplA/XplB-expressing switchgrass},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Computational principles and challenges in single-cell data
integration. <em>NBT</em>, <em>39</em>(10), 1202–1215. (<a
href="https://doi.org/10.1038/s41587-021-00895-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The development of single-cell multimodal assays provides a powerful tool for investigating multiple dimensions of cellular heterogeneity, enabling new insights into development, tissue homeostasis and disease. A key challenge in the analysis of single-cell multimodal data is to devise appropriate strategies for tying together data across different modalities. The term ‘data integration’ has been used to describe this task, encompassing a broad collection of approaches ranging from batch correction of individual omics datasets to association of chromatin accessibility and genetic variation with transcription. Although existing integration strategies exploit similar mathematical ideas, they typically have distinct goals and rely on different principles and assumptions. Consequently, new definitions and concepts are needed to contextualize existing methods and to enable development of new methods. As the number of single-cell experiments with multiple data modalities increases, Argelaguet and colleagues review the concepts and challenges of data integration.},
  archive      = {J_NBT},
  author       = {Argelaguet, Ricard and Cuomo, Anna S. E. and Stegle, Oliver and Marioni, John C.},
  doi          = {10.1038/s41587-021-00895-7},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1202-1215},
  shortjournal = {Nature Biotech.},
  title        = {Computational principles and challenges in single-cell data integration},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Forum: Gene therapy roundtable with high and kohn.
<em>NBT</em>, <em>39</em>(10), 1201. (<a
href="https://doi.org/10.1038/s41587-021-01083-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01083-3},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1201},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Gene therapy roundtable with high and kohn},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Hope lies in dreams. <em>NBT</em>, <em>39</em>(10), 1201.
(<a href="https://doi.org/10.1038/s41587-021-01090-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01090-4},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1201},
  shortjournal = {Nature Biotech.},
  title        = {Hope lies in dreams},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Synthetic biology. <em>NBT</em>, <em>39</em>(10), 1200. (<a
href="https://doi.org/10.1038/s41587-021-01087-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to synthetic nucleotides, peptides and genomes.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01087-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1200},
  shortjournal = {Nature Biotech.},
  title        = {Synthetic biology},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Patent landscape of brain–machine interface technology.
<em>NBT</em>, <em>39</em>(10), 1194–1199. (<a
href="https://doi.org/10.1038/s41587-021-01071-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study of the brain–machine interface patent landscape suggests that the technology is in its early stages of development, but patent applications have been increasing exponentially in recent years.},
  archive      = {J_NBT},
  author       = {Greenberg, Anastasia and Cohen, Alexis and Grewal, Monica},
  doi          = {10.1038/s41587-021-01071-7},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1194-1199},
  shortjournal = {Nature Biotech.},
  title        = {Patent landscape of brain–machine interface technology},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021l). Synthetic virology: The experts speak. <em>NBT</em>,
<em>39</em>(10), 1185–1193. (<a
href="https://doi.org/10.1038/s41587-021-01078-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Even though only a few labs around the world have the means to engineer a purely synthetic virus, debate on the origins of SARS-CoV-2 has resurfaced concerns about the risks and benefits of synthetic virology.},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-01078-0},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1185-1193},
  shortjournal = {Nature Biotech.},
  title        = {Synthetic virology: The experts speak},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A public–private partnership model for COVID-19 diagnostics.
<em>NBT</em>, <em>39</em>(10), 1182–1184. (<a
href="https://doi.org/10.1038/s41587-021-01080-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Krijger, Peter H. L. and Hoek, Tim A. and Boersma, Sanne and Donders, Lieke I. P. M. and Broeders, Maaike M. C. and Pieterse, Mark and Toonen, Pim W. and Logister, Ive and Verhagen, Bram M. P. and Verstegen, Marjon J. A. M. and van Ravesteyn, Thomas W. and Roymans, Rene J. T. M. and Mattiroli, Francesca and Vandesompele, Jo and Nijhuis, Monique and Meijer, Stefan and van Weert, Anton and Dekker, Edwin and Dom, Fred J. and Ruijtenbeek, Rob and van der Velden, Lieven B. J. and van de Bovenkamp, Jeroen H. B. and Bosch, Martijn and de Laat, Wouter and Tanenbaum, Marvin E.},
  doi          = {10.1038/s41587-021-01080-6},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1182-1184},
  shortjournal = {Nature Biotech.},
  title        = {A public–private partnership model for COVID-19 diagnostics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Rapid delivery systems for future food security.
<em>NBT</em>, <em>39</em>(10), 1179–1181. (<a
href="https://doi.org/10.1038/s41587-021-01079-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Varshney, Rajeev K. and Bohra, Abhishek and Roorkiwal, Manish and Barmukh, Rutwik and Cowling, Wallace and Chitikineni, Annapurna and Lam, Hon-Ming and Hickey, Lee T. and Croser, Janine and Edwards, David and Farooq, Muhammad and Crossa, José and Weckwerth, Wolfram and Millar, A. Harvey and Kumar, Arvind and Bevan, Michael W. and Siddique, Kadambot H. M.},
  doi          = {10.1038/s41587-021-01079-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1179-1181},
  shortjournal = {Nature Biotech.},
  title        = {Rapid delivery systems for future food security},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ready-to-use public infrastructure for global SARS-CoV-2
monitoring. <em>NBT</em>, <em>39</em>(10), 1178–1179. (<a
href="https://doi.org/10.1038/s41587-021-01069-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Maier, Wolfgang and Bray, Simon and van den Beek, Marius and Bouvier, Dave and Coraor, Nathan and Miladi, Milad and Singh, Babita and De Argila, Jordi Rambla and Baker, Dannon and Roach, Nathan and Gladman, Simon and Coppens, Frederik and Martin, Darren P. and Lonie, Andrew and Grüning, Björn and Kosakovsky Pond, Sergei L. and Nekrutenko, Anton},
  doi          = {10.1038/s41587-021-01069-1},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1178-1179},
  shortjournal = {Nature Biotech.},
  title        = {Ready-to-use public infrastructure for global SARS-CoV-2 monitoring},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Biotech patenting 2020. <em>NBT</em>, <em>39</em>(10),
1176–1177. (<a
href="https://doi.org/10.1038/s41587-021-01076-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Huggett, Brady and Paisner, Kathryn},
  doi          = {10.1038/s41587-021-01076-2},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1176-1177},
  shortjournal = {Nature Biotech.},
  title        = {Biotech patenting 2020},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Biotech news from around the world. <em>NBT</em>,
<em>39</em>(10), 1175. (<a
href="https://doi.org/10.1038/s41587-021-01082-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01082-4},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1175},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). UK trials first CRISPR-edited wheat in europe. <em>NBT</em>,
<em>39</em>(10), 1174. (<a
href="https://doi.org/10.1038/s41587-021-01098-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01098-w},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1174},
  shortjournal = {Nature Biotech.},
  title        = {UK trials first CRISPR-edited wheat in europe},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Roche deal with RNA-editing spinout. <em>NBT</em>,
<em>39</em>(10), 1173. (<a
href="https://doi.org/10.1038/s41587-021-01097-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01097-x},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1173},
  shortjournal = {Nature Biotech.},
  title        = {Roche deal with RNA-editing spinout},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Nanotechnology offers alternative ways to fight COVID-19
pandemic with antivirals. <em>NBT</em>, <em>39</em>(10), 1172–1174. (<a
href="https://doi.org/10.1038/s41587-021-01085-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new wave of funding focuses on antiviral nanomaterials as pandemic countermeasures.},
  archive      = {J_NBT},
  author       = {Peplow, Mark},
  doi          = {10.1038/s41587-021-01085-1},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1172-1174},
  shortjournal = {Nature Biotech.},
  title        = {Nanotechnology offers alternative ways to fight COVID-19 pandemic with antivirals},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Church to de-extinct woolly mammoths. <em>NBT</em>,
<em>39</em>(10), 1171. (<a
href="https://doi.org/10.1038/s41587-021-01096-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-01096-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1171},
  shortjournal = {Nature Biotech.},
  title        = {Church to de-extinct woolly mammoths},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Drug hunters uncloak the non-coding “hidden” genome.
<em>NBT</em>, <em>39</em>(10), 1169–1171. (<a
href="https://doi.org/10.1038/s41587-021-01088-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientists are using new tools to mine the non-coding part of the genome, known as ‘dark matter’, to uncover disease-linked changes in promoters, silencers and enhancers for target discovery.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael},
  doi          = {10.1038/s41587-021-01088-y},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1169-1171},
  shortjournal = {Nature Biotech.},
  title        = {Drug hunters uncloak the non-coding ‘hidden’ genome},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The champions of drug development. <em>NBT</em>,
<em>39</em>(10), 1167. (<a
href="https://doi.org/10.1038/s41587-021-01095-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Our new podcast series ‘Hope Lies in Dreams’ highlights the importance of visionaries in shepherding drugs through the setbacks of drug development to commercial success.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01095-z},
  journal      = {Nature Biotechnology},
  month        = {10},
  number       = {10},
  pages        = {1167},
  shortjournal = {Nature Biotech.},
  title        = {The champions of drug development},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). People. <em>NBT</em>, <em>39</em>(9), 1166. (<a
href="https://doi.org/10.1038/s41587-021-01054-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01054-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1166},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). More than just content: Building community in the graduate
classroom. <em>NBT</em>, <em>39</em>(9), 1161–1165. (<a
href="https://doi.org/10.1038/s41587-021-01046-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Community building should be integrated into the graduate classroom to foster the development of a professional identity and build connections between students and faculty.},
  archive      = {J_NBT},
  author       = {Venkatesh, Madhvi J. and Elchert, Alexandra R. and Fakoya, Bolutife and Fernandez, Francisco and Kwong, Andrew C. and Liu, Yue J. and Lotfy, Peter and Lowe, David D. and Petty, Christopher A. and Rodríguez-delaRosa, Alejandra and Seguinot, Bryan O. and Shi, Yingxiao and Loparo, Joseph J.},
  doi          = {10.1038/s41587-021-01046-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1161-1165},
  shortjournal = {Nature Biotech.},
  title        = {More than just content: Building community in the graduate classroom},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Establishing community reference samples, data and call sets
for benchmarking cancer mutation detection using whole-genome
sequencing. <em>NBT</em>, <em>39</em>(9), 1151–1160. (<a
href="https://doi.org/10.1038/s41587-021-00993-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The lack of samples for generating standardized DNA datasets for setting up a sequencing pipeline or benchmarking the performance of different algorithms limits the implementation and uptake of cancer genomics. Here, we describe reference call sets obtained from paired tumor–normal genomic DNA (gDNA) samples derived from a breast cancer cell line—which is highly heterogeneous, with an aneuploid genome, and enriched in somatic alterations—and a matched lymphoblastoid cell line. We partially validated both somatic mutations and germline variants in these call sets via whole-exome sequencing (WES) with different sequencing platforms and targeted sequencing with &amp;gt;2,000-fold coverage, spanning 82\% of genomic regions with high confidence. Although the gDNA reference samples are not representative of primary cancer cells from a clinical sample, when setting up a sequencing pipeline, they not only minimize potential biases from technologies, assays and informatics but also provide a unique resource for benchmarking ‘tumor-only’ or ‘matched tumor–normal’ analyses. Tumor–normal paired DNA samples from a breast cancer cell line and a matched lymphoblastoid cell line enable calibration of clinical sequencing pipelines and benchmarking ‘tumor-only’ or ‘matched tumor–normal’ analyses.},
  archive      = {J_NBT},
  author       = {Fang, Li Tai and Zhu, Bin and Zhao, Yongmei and Chen, Wanqiu and Yang, Zhaowei and Kerrigan, Liz and Langenbach, Kurt and de Mars, Maryellen and Lu, Charles and Idler, Kenneth and Jacob, Howard and Zheng, Yuanting and Ren, Luyao and Yu, Ying and Jaeger, Erich and Schroth, Gary P. and Abaan, Ogan D. and Talsania, Keyur and Lack, Justin and Shen, Tsai-Wei and Chen, Zhong and Stanbouly, Seta and Tran, Bao and Shetty, Jyoti and Kriga, Yuliya and Meerzaman, Daoud and Nguyen, Cu and Petitjean, Virginie and Sultan, Marc and Cam, Margaret and Mehta, Monika and Hung, Tiffany and Peters, Eric and Kalamegham, Rasika and Sahraeian, Sayed Mohammad Ebrahim and Mohiyuddin, Marghoob and Guo, Yunfei and Yao, Lijing and Song, Lei and Lam, Hugo Y. K. and Drabek, Jiri and Vojta, Petr and Maestro, Roberta and Gasparotto, Daniela and Kõks, Sulev and Reimann, Ene and Scherer, Andreas and Nordlund, Jessica and Liljedahl, Ulrika and Jensen, Roderick V. and Pirooznia, Mehdi and Li, Zhipan and Xiao, Chunlin and Sherry, Stephen T. and Kusko, Rebecca and Moos, Malcolm and Donaldson, Eric and Tezak, Zivana and Ning, Baitang and Tong, Weida and Li, Jing and Duerken-Hughes, Penelope and Catalanotti, Claudia and Maheshwari, Shamoni and Shuga, Joe and Liang, Winnie S. and Keats, Jonathan and Adkins, Jonathan and Tassone, Erica and Zismann, Victoria and McDaniel, Timothy and Trent, Jeffrey and Foox, Jonathan and Butler, Daniel and Mason, Christopher E. and Hong, Huixiao and Shi, Leming and Wang, Charles and Xiao, Wenming},
  doi          = {10.1038/s41587-021-00993-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1151-1160},
  shortjournal = {Nature Biotech.},
  title        = {Establishing community reference samples, data and call sets for benchmarking cancer mutation detection using whole-genome sequencing},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Toward best practice in cancer mutation detection with
whole-genome and whole-exome sequencing. <em>NBT</em>, <em>39</em>(9),
1141–1150. (<a
href="https://doi.org/10.1038/s41587-021-00994-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clinical applications of precision oncology require accurate tests that can distinguish true cancer-specific mutations from errors introduced at each step of next-generation sequencing (NGS). To date, no bulk sequencing study has addressed the effects of cross-site reproducibility, nor the biological, technical and computational factors that influence variant identification. Here we report a systematic interrogation of somatic mutations in paired tumor–normal cell lines to identify factors affecting detection reproducibility and accuracy at six different centers. Using whole-genome sequencing (WGS) and whole-exome sequencing (WES), we evaluated the reproducibility of different sample types with varying input amount and tumor purity, and multiple library construction protocols, followed by processing with nine bioinformatics pipelines. We found that read coverage and callers affected both WGS and WES reproducibility, but WES performance was influenced by insert fragment size, genomic copy content and the global imbalance score (GIV; G &amp;gt; T/C &amp;gt; A). Finally, taking into account library preparation protocol, tumor content, read coverage and bioinformatics processes concomitantly, we recommend actionable practices to improve the reproducibility and accuracy of NGS experiments for cancer mutation detection. Recommendations are given on optimal read coverage and selection of calling algorithm to maximize the reproducibility of cancer mutation detection in whole-genome or whole-exome sequencing.},
  archive      = {J_NBT},
  author       = {Xiao, Wenming and Ren, Luyao and Chen, Zhong and Fang, Li Tai and Zhao, Yongmei and Lack, Justin and Guan, Meijian and Zhu, Bin and Jaeger, Erich and Kerrigan, Liz and Blomquist, Thomas M. and Hung, Tiffany and Sultan, Marc and Idler, Kenneth and Lu, Charles and Scherer, Andreas and Kusko, Rebecca and Moos, Malcolm and Xiao, Chunlin and Sherry, Stephen T. and Abaan, Ogan D. and Chen, Wanqiu and Chen, Xin and Nordlund, Jessica and Liljedahl, Ulrika and Maestro, Roberta and Polano, Maurizio and Drabek, Jiri and Vojta, Petr and Kõks, Sulev and Reimann, Ene and Madala, Bindu Swapna and Mercer, Timothy and Miller, Chris and Jacob, Howard and Truong, Tiffany and Moshrefi, Ali and Natarajan, Aparna and Granat, Ana and Schroth, Gary P. and Kalamegham, Rasika and Peters, Eric and Petitjean, Virginie and Walton, Ashley and Shen, Tsai-Wei and Talsania, Keyur and Vera, Cristobal Juan and Langenbach, Kurt and de Mars, Maryellen and Hipp, Jennifer A. and Willey, James C. and Wang, Jing and Shetty, Jyoti and Kriga, Yuliya and Raziuddin, Arati and Tran, Bao and Zheng, Yuanting and Yu, Ying and Cam, Margaret and Jailwala, Parthav and Nguyen, Cu and Meerzaman, Daoud and Chen, Qingrong and Yan, Chunhua and Ernest, Ben and Mehra, Urvashi and Jensen, Roderick V. and Jones, Wendell and Li, Jian-Liang and Papas, Brian N. and Pirooznia, Mehdi and Chen, Yun-Ching and Seifuddin, Fayaz and Li, Zhipan and Liu, Xuelu and Resch, Wolfgang and Wang, Jingya and Wu, Leihong and Yavas, Gokhan and Miles, Corey and Ning, Baitang and Tong, Weida and Mason, Christopher E. and Donaldson, Eric and Lababidi, Samir and Staudt, Louis M. and Tezak, Zivana and Hong, Huixiao and Wang, Charles and Shi, Leming},
  doi          = {10.1038/s41587-021-00994-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1141-1150},
  shortjournal = {Nature Biotech.},
  title        = {Toward best practice in cancer mutation detection with whole-genome and whole-exome sequencing},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Performance assessment of DNA sequencing platforms in the
ABRF next-generation sequencing study. <em>NBT</em>, <em>39</em>(9),
1129–1140. (<a
href="https://doi.org/10.1038/s41587-021-01049-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Assessing the reproducibility, accuracy and utility of massively parallel DNA sequencing platforms remains an ongoing challenge. Here the Association of Biomolecular Resource Facilities (ABRF) Next-Generation Sequencing Study benchmarks the performance of a set of sequencing instruments (HiSeq/NovaSeq/paired-end 2 × 250-bp chemistry, Ion S5/Proton, PacBio circular consensus sequencing (CCS), Oxford Nanopore Technologies PromethION/MinION, BGISEQ-500/MGISEQ-2000 and GS111) on human and bacterial reference DNA samples. Among short-read instruments, HiSeq 4000 and X10 provided the most consistent, highest genome coverage, while BGI/MGISEQ provided the lowest sequencing error rates. The long-read instrument PacBio CCS had the highest reference-based mapping rate and lowest non-mapping rate. The two long-read platforms PacBio CCS and PromethION/MinION showed the best sequence mapping in repeat-rich areas and across homopolymers. NovaSeq 6000 using 2 × 250-bp read chemistry was the most robust instrument for capturing known insertion/deletion events. This study serves as a benchmark for current genomics technologies, as well as a resource to inform experimental design and next-generation sequencing variant calling. Next-generation sequencing platforms are benchmarked using human, bacterial and metagenomics reference materials.},
  archive      = {J_NBT},
  author       = {Foox, Jonathan and Tighe, Scott W. and Nicolet, Charles M. and Zook, Justin M. and Byrska-Bishop, Marta and Clarke, Wayne E. and Khayat, Michael M. and Mahmoud, Medhat and Laaguiby, Phoebe K. and Herbert, Zachary T. and Warner, Derek and Grills, George S. and Jen, Jin and Levy, Shawn and Xiang, Jenny and Alonso, Alicia and Zhao, Xia and Zhang, Wenwei and Teng, Fei and Zhao, Yonggang and Lu, Haorong and Schroth, Gary P. and Narzisi, Giuseppe and Farmerie, William and Sedlazeck, Fritz J. and Baldwin, Don A. and Mason, Christopher E.},
  doi          = {10.1038/s41587-021-01049-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1129-1140},
  shortjournal = {Nature Biotech.},
  title        = {Performance assessment of DNA sequencing platforms in the ABRF next-generation sequencing study},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Evaluating the analytical validity of circulating tumor DNA
sequencing assays for precision oncology. <em>NBT</em>, <em>39</em>(9),
1115–1128. (<a
href="https://doi.org/10.1038/s41587-021-00857-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Circulating tumor DNA (ctDNA) sequencing is being rapidly adopted in precision oncology, but the accuracy, sensitivity and reproducibility of ctDNA assays is poorly understood. Here we report the findings of a multi-site, cross-platform evaluation of the analytical performance of five industry-leading ctDNA assays. We evaluated each stage of the ctDNA sequencing workflow with simulations, synthetic DNA spike-in experiments and proficiency testing on standardized, cell-line-derived reference samples. Above 0.5\% variant allele frequency, ctDNA mutations were detected with high sensitivity, precision and reproducibility by all five assays, whereas, below this limit, detection became unreliable and varied widely between assays, especially when input material was limited. Missed mutations (false negatives) were more common than erroneous candidates (false positives), indicating that the reliable sampling of rare ctDNA fragments is the key challenge for ctDNA assays. This comprehensive evaluation of the analytical performance of ctDNA assays serves to inform best practice guidelines and provides a resource for precision oncology. Reliable detection of mutations below 0.5\% variant allele frequency remains a key challenge for circulating tumor DNA sequencing assays.},
  archive      = {J_NBT},
  author       = {Deveson, Ira W. and Gong, Binsheng and Lai, Kevin and LoCoco, Jennifer S. and Richmond, Todd A. and Schageman, Jeoffrey and Zhang, Zhihong and Novoradovskaya, Natalia and Willey, James C. and Jones, Wendell and Kusko, Rebecca and Chen, Guangchun and Madala, Bindu Swapna and Blackburn, James and Stevanovski, Igor and Bhandari, Ambica and Close, Devin and Conroy, Jeffrey and Hubank, Michael and Marella, Narasimha and Mieczkowski, Piotr A. and Qiu, Fujun and Sebra, Robert and Stetson, Daniel and Sun, Lihyun and Szankasi, Philippe and Tan, Haowen and Tang, Lin-ya and Arib, Hanane and Best, Hunter and Burgher, Blake and Bushel, Pierre R. and Casey, Fergal and Cawley, Simon and Chang, Chia-Jung and Choi, Jonathan and Dinis, Jorge and Duncan, Daniel and Eterovic, Agda Karina and Feng, Liang and Ghosal, Abhisek and Giorda, Kristina and Glenn, Sean and Happe, Scott and Haseley, Nathan and Horvath, Kyle and Hung, Li-Yuan and Jarosz, Mirna and Kushwaha, Garima and Li, Dan and Li, Quan-Zhen and Li, Zhiguang and Liu, Liang-Chun and Liu, Zhichao and Ma, Charles and Mason, Christopher E. and Megherbi, Dalila B. and Morrison, Tom and Pabón-Peña, Carlos and Pirooznia, Mehdi and Proszek, Paula Z. and Raymond, Amelia and Rindler, Paul and Ringler, Rebecca and Scherer, Andreas and Shaknovich, Rita and Shi, Tieliu and Smith, Melissa and Song, Ping and Strahl, Maya and Thodima, Venkat J. and Tom, Nikola and Verma, Suman and Wang, Jiashi and Wu, Leihong and Xiao, Wenzhong and Xu, Chang and Yang, Mary and Zhang, Guangliang and Zhang, Sa and Zhang, Yilin and Shi, Leming and Tong, Weida and Johann, Donald J. and Mercer, Timothy R. and Xu, Joshua},
  doi          = {10.1038/s41587-021-00857-z},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1115-1128},
  shortjournal = {Nature Biotech.},
  title        = {Evaluating the analytical validity of circulating tumor DNA sequencing assays for precision oncology},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A multicenter study benchmarking single-cell RNA sequencing
technologies using reference samples. <em>NBT</em>, <em>39</em>(9),
1103–1114. (<a
href="https://doi.org/10.1038/s41587-020-00748-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comparing diverse single-cell RNA sequencing (scRNA-seq) datasets generated by different technologies and in different laboratories remains a major challenge. Here we address the need for guidance in choosing algorithms leading to accurate biological interpretations of varied data types acquired with different platforms. Using two well-characterized cellular reference samples (breast cancer cells and B cells), captured either separately or in mixtures, we compared different scRNA-seq platforms and several preprocessing, normalization and batch-effect correction methods at multiple centers. Although preprocessing and normalization contributed to variability in gene detection and cell classification, batch-effect correction was by far the most important factor in correctly classifying the cells. Moreover, scRNA-seq dataset characteristics (for example, sample and cellular heterogeneity and platform used) were critical in determining the optimal bioinformatic method. However, reproducibility across centers and platforms was high when appropriate bioinformatic methods were applied. Our findings offer practical guidance for optimizing platform and software selection when designing an scRNA-seq study. A comprehensive comparison of 20 single-cell RNA-seq datasets derived from the two cell lines analyzed using six preprocessing pipelines, eight normalization methods and seven batch-correction algorithms derived from four different sequencing platforms at different centers.},
  archive      = {J_NBT},
  author       = {Chen, Wanqiu and Zhao, Yongmei and Chen, Xin and Yang, Zhaowei and Xu, Xiaojiang and Bi, Yingtao and Chen, Vicky and Li, Jing and Choi, Hannah and Ernest, Ben and Tran, Bao and Mehta, Monika and Kumar, Parimal and Farmer, Andrew and Mir, Alain and Mehra, Urvashi Ann and Li, Jian-Liang and Moos, Malcolm and Xiao, Wenming and Wang, Charles},
  doi          = {10.1038/s41587-020-00748-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1103-1114},
  shortjournal = {Nature Biotech.},
  title        = {A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Gene signature extraction and cell identity recognition at
the single-cell level with cell-ID. <em>NBT</em>, <em>39</em>(9),
1095–1102. (<a
href="https://doi.org/10.1038/s41587-021-00896-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Because of the stochasticity associated with high-throughput single-cell sequencing, current methods for exploring cell-type diversity rely on clustering-based computational approaches in which heterogeneity is characterized at cell subpopulation rather than at full single-cell resolution. Here we present Cell-ID, a clustering-free multivariate statistical method for the robust extraction of per-cell gene signatures from single-cell sequencing data. We applied Cell-ID to data from multiple human and mouse samples, including blood cells, pancreatic islets and airway, intestinal and olfactory epithelium, as well as to comprehensive mouse cell atlas datasets. We demonstrate that Cell-ID signatures are reproducible across different donors, tissues of origin, species and single-cell omics technologies, and can be used for automatic cell-type annotation and cell matching across datasets. Cell-ID improves biological interpretation at individual cell level, enabling discovery of previously uncharacterized rare cell types or cell states. Cell-ID is distributed as an open-source R software package. Cell-ID facilitates the analysis of cell-type heterogeneity and cell identity across multiple samples at the single-cell level.},
  archive      = {J_NBT},
  author       = {Cortal, Akira and Martignetti, Loredana and Six, Emmanuelle and Rausell, Antonio},
  doi          = {10.1038/s41587-021-00896-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1095-1102},
  shortjournal = {Nature Biotech.},
  title        = {Gene signature extraction and cell identity recognition at the single-cell level with cell-ID},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Extended-representation bisulfite sequencing of gene
regulatory elements in multiplexed samples and single cells.
<em>NBT</em>, <em>39</em>(9), 1086–1094. (<a
href="https://doi.org/10.1038/s41587-021-00910-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The biological roles of DNA methylation have been elucidated by profiling methods based on whole-genome or reduced-representation bisulfite sequencing, but these approaches do not efficiently survey the vast numbers of non-coding regulatory elements in mammalian genomes. Here we present an extended-representation bisulfite sequencing (XRBS) method for targeted profiling of DNA methylation. Our design strikes a balance between expanding coverage of regulatory elements and reproducibly enriching informative CpG dinucleotides in promoters, enhancers and CTCF binding sites. Barcoded DNA fragments are pooled before bisulfite conversion, allowing multiplex processing and technical consistency in low-input samples. Application of XRBS to single leukemia cells enabled us to evaluate genetic copy number variations and methylation variability across individual cells. Our analysis highlights heterochromatic H3K9me3 regions as having the highest cell-to-cell variability in their methylation, likely reflecting inherent epigenetic instability of these late-replicating regions, compounded by differences in cell cycle stages among sampled cells. DNA methylation is efficiently profiled in non-coding regulatory sequences in single cells.},
  archive      = {J_NBT},
  author       = {Shareef, Sarah J. and Bevill, Samantha M. and Raman, Ayush T. and Aryee, Martin J. and van Galen, Peter and Hovestadt, Volker and Bernstein, Bradley E.},
  doi          = {10.1038/s41587-021-00910-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1086-1094},
  shortjournal = {Nature Biotech.},
  title        = {Extended-representation bisulfite sequencing of gene regulatory elements in multiplexed samples and single cells},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Long-term wireless streaming of neural recordings for
circuit discovery and adaptive stimulation in individuals with
parkinson’s disease. <em>NBT</em>, <em>39</em>(9), 1078–1085. (<a
href="https://doi.org/10.1038/s41587-021-00897-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neural recordings using invasive devices in humans can elucidate the circuits underlying brain disorders, but have so far been limited to short recordings from externalized brain leads in a hospital setting or from implanted sensing devices that provide only intermittent, brief streaming of time series data. Here, we report the use of an implantable two-way neural interface for wireless, multichannel streaming of field potentials in five individuals with Parkinson’s disease (PD) for up to 15 months after implantation. Bilateral four-channel motor cortex and basal ganglia field potentials streamed at home for over 2,600 h were paired with behavioral data from wearable monitors for the neural decoding of states of inadequate or excessive movement. We validated individual-specific neurophysiological biomarkers during normal daily activities and used those patterns for adaptive deep brain stimulation (DBS). This technological approach may be widely applicable to brain disorders treatable by invasive neuromodulation. An implanted device in the brain enables wireless neural monitoring and stimulation for up to 15 months following implantation.},
  archive      = {J_NBT},
  author       = {Gilron, Ro’ee and Little, Simon and Perrone, Randy and Wilt, Robert and de Hemptinne, Coralie and Yaroshinsky, Maria S. and Racine, Caroline A. and Wang, Sarah S. and Ostrem, Jill L. and Larson, Paul S. and Wang, Doris D. and Galifianakis, Nick B. and Bledsoe, Ian O. and San Luciano, Marta and Dawes, Heather E. and Worrell, Gregory A. and Kremen, Vaclav and Borton, David A. and Denison, Timothy and Starr, Philip A.},
  doi          = {10.1038/s41587-021-00897-5},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1078-1085},
  shortjournal = {Nature Biotech.},
  title        = {Long-term wireless streaming of neural recordings for circuit discovery and adaptive stimulation in individuals with parkinson’s disease},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). First rounders: Anthony atala. <em>NBT</em>,
<em>39</em>(9), 1077. (<a
href="https://doi.org/10.1038/s41587-021-01064-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01064-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1077},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Anthony atala},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Organoids. <em>NBT</em>, <em>39</em>(9), 1076. (<a
href="https://doi.org/10.1038/s41587-021-01055-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to methods of making and using organoids.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01055-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1076},
  shortjournal = {Nature Biotech.},
  title        = {Organoids},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Government targets, end-of-year patenting rush and
innovative performance in china. <em>NBT</em>, <em>39</em>(9),
1068–1075. (<a
href="https://doi.org/10.1038/s41587-021-01035-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study of the seasonality and overall quantity and quality of Chinese patenting, including international comparisons, suggests that government planning and annual targets encouraged gaming of the system, which increased patent counts but negatively affected patent quality.},
  archive      = {J_NBT},
  author       = {Sun, Zhen and Lei, Zhen and Wright, Brian D. and Cohen, Mark and Liu, Taoxiong},
  doi          = {10.1038/s41587-021-01035-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1068-1075},
  shortjournal = {Nature Biotech.},
  title        = {Government targets, end-of-year patenting rush and innovative performance in china},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MAQC and the era of genomic medicine. <em>NBT</em>,
<em>39</em>(9), 1066–1067. (<a
href="https://doi.org/10.1038/s41587-021-01050-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The MicroArray Quality Control consortium—a 16-year international effort led by the FDA and involving hundreds of scientists from academia, industry and government—helped make genomic medicine a reality.},
  archive      = {J_NBT},
  author       = {Salit, Marc and Woodcock, Janet},
  doi          = {10.1038/s41587-021-01050-y},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1066-1067},
  shortjournal = {Nature Biotech.},
  title        = {MAQC and the era of genomic medicine},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Utilize existing genetic diversity before genetic
modification in indigenous crops. <em>NBT</em>, <em>39</em>(9),
1064–1065. (<a
href="https://doi.org/10.1038/s41587-021-01048-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Borrell, James S. and Gebremariam, Zelalem and Abebe, Wendawek M.},
  doi          = {10.1038/s41587-021-01048-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1064-1065},
  shortjournal = {Nature Biotech.},
  title        = {Utilize existing genetic diversity before genetic modification in indigenous crops},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A python-based programming language for high-performance
computational genomics. <em>NBT</em>, <em>39</em>(9), 1062–1064. (<a
href="https://doi.org/10.1038/s41587-021-00985-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Shajii, Ariya and Numanagić, Ibrahim and Leighton, Alexander T. and Greenyer, Haley and Amarasinghe, Saman and Berger, Bonnie},
  doi          = {10.1038/s41587-021-00985-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1062-1064},
  shortjournal = {Nature Biotech.},
  title        = {A python-based programming language for high-performance computational genomics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The need for new test verification and regulatory support
for innovative diagnostics. <em>NBT</em>, <em>39</em>(9), 1060–1062. (<a
href="https://doi.org/10.1038/s41587-021-01047-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Roback, John D. and Tyburski, Erika A. and Alter, David and Asakrah, Saja and Chahroudi, Ann and Esper, Annette and Farmer, Sarah and Figueroa, Janet and K. Frediani, Jennifer and D. Gonzalez, Mark and S. Gottfried, David and Guarner, Jeannette and A. Gupta, Nitika and S. Heilman, Stacy and E. Hill, Charles and Jerris, Robert and R. Kempker, Russell and Ingersoll, Jessica and Levy, Joshua M. and Mavigner, Maud and S. Moreno, Carlos and R. Morris, Claudia and J. Nehl, Eric and S. Neish, Andrew and Peker, Deniz and Saakadze, Natia and Rebolledo, Paulina A. and A. Rostad, Christina and Schoof, Nils and Suessmith, Allie and Sullivan, Julie and Wang, Yun F. (Wayne) and Wood, Anna and Vos, Miriam B. and Brand, Oliver and Martin, Greg S. and Lam, Wilbur A.},
  doi          = {10.1038/s41587-021-01047-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1060-1062},
  shortjournal = {Nature Biotech.},
  title        = {The need for new test verification and regulatory support for innovative diagnostics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The lag in SARS-CoV-2 genome submissions to GISAID.
<em>NBT</em>, <em>39</em>(9), 1058–1060. (<a
href="https://doi.org/10.1038/s41587-021-01040-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kalia, Kishan and Saberwal, Gayatri and Sharma, Gaurav},
  doi          = {10.1038/s41587-021-01040-0},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1058-1060},
  shortjournal = {Nature Biotech.},
  title        = {The lag in SARS-CoV-2 genome submissions to GISAID},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Responsible governance of gene editing in agriculture and
the environment. <em>NBT</em>, <em>39</em>(9), 1055–1057. (<a
href="https://doi.org/10.1038/s41587-021-01023-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Gordon, Doria R. and Jaffe, Gregory and Doane, Michael and Glaser, Aviva and Gremillion, Thomas M. and Ho, Melissa D.},
  doi          = {10.1038/s41587-021-01023-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1055-1057},
  shortjournal = {Nature Biotech.},
  title        = {Responsible governance of gene editing in agriculture and the environment},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Stepping to the exit. <em>NBT</em>, <em>39</em>(9),
1048–1054. (<a
href="https://doi.org/10.1038/s41587-021-01007-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A historical analysis of oncology deals can help bioentrepreneurs navigate the right time for partnerships and exits and can inform on the type of investors likely to be interested.},
  archive      = {J_NBT},
  author       = {de Bloeme, Christiaan M. and Jansen, Robin W. and Krul, Mark R. L. and Geutjes, Ernst-Jan},
  doi          = {10.1038/s41587-021-01007-1},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1048-1054},
  shortjournal = {Nature Biotech.},
  title        = {Stepping to the exit},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Nature biotechnology’s academic spinouts 2020. <em>NBT</em>,
<em>39</em>(9), 1036–1047. (<a
href="https://doi.org/10.1038/s41587-021-01043-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Biotechnology’s annual survey highlights university startups that are, among other things, rethinking how to deliver gene-editing therapy and tackling various metabolic conditions, immune disorders and cancer with microbiome treatments or immunotherapy. Michael Eisenstein, Ken Garber, Esther Landhuis, Caroline Seydel and Laura DeFrancesco report.},
  archive      = {J_NBT},
  author       = {Eisenstein, Michael and Garber, Ken and Landhuis, Esther and Seydel, Caroline and DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-01043-x},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1036-1047},
  shortjournal = {Nature Biotech.},
  title        = {Nature biotechnology’s academic spinouts 2020},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Biotech news from around the world. <em>NBT</em>,
<em>39</em>(9), 1035. (<a
href="https://doi.org/10.1038/s41587-021-01052-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01052-w},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1035},
  shortjournal = {Nature Biotech.},
  title        = {Biotech news from around the world},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). First interchangeable insulin. <em>NBT</em>, <em>39</em>(9),
1034. (<a href="https://doi.org/10.1038/s41587-021-01062-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01062-8},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1034},
  shortjournal = {Nature Biotech.},
  title        = {First interchangeable insulin},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Oncologists greet lumakras: The world’s first KRAS
inhibitor. <em>NBT</em>, <em>39</em>(9), 1032–1034. (<a
href="https://doi.org/10.1038/s41587-021-01053-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The milestone approval for Amgen’s drug Lumakras will galvanize targeted therapy efforts against Ras oncoproteins and provides a powerful new last line of defense for patients with lung cancer.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-021-01053-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1032-1034},
  shortjournal = {Nature Biotech.},
  title        = {Oncologists greet lumakras: The world’s first KRAS inhibitor},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). First HIF-2 antagonist for VHL tumors. <em>NBT</em>,
<em>39</em>(9), 1032. (<a
href="https://doi.org/10.1038/s41587-021-01063-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01063-7},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1032},
  shortjournal = {Nature Biotech.},
  title        = {First HIF-2 antagonist for VHL tumors},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CRISPR-based portable COVID tests. <em>NBT</em>,
<em>39</em>(9), 1031. (<a
href="https://doi.org/10.1038/s41587-021-01061-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01061-9},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1031},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR-based portable COVID tests},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). Startups set off new wave of mRNA therapeutics.
<em>NBT</em>, <em>39</em>(9), 1029–1031. (<a
href="https://doi.org/10.1038/s41587-021-01056-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {After the vaccine triumphs of Pfizer/BioNTech and Moderna, a raft of startups is developing mRNA, circular RNA and self-amplifying RNA therapeutics.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-021-01056-6},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1029-1031},
  shortjournal = {Nature Biotech.},
  title        = {Startups set off new wave of mRNA therapeutics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Sequencing benchmarked. <em>NBT</em>, <em>39</em>(9), 1027.
(<a href="https://doi.org/10.1038/s41587-021-01067-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Sequencing Quality Control 2 (SEQC2/MAQC-IV) project provides resources to aid sequencing reproducibility and highlights factors that can guide platform and software choice.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01067-3},
  journal      = {Nature Biotechnology},
  month        = {9},
  number       = {9},
  pages        = {1027},
  shortjournal = {Nature Biotech.},
  title        = {Sequencing benchmarked},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). People. <em>NBT</em>, <em>39</em>(8), 1026. (<a
href="https://doi.org/10.1038/s41587-021-01014-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01014-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1026},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Second-quarter biotech job picture. <em>NBT</em>,
<em>39</em>(8), 1025. (<a
href="https://doi.org/10.1038/s41587-021-01011-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-021-01011-5},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1025},
  shortjournal = {Nature Biotech.},
  title        = {Second-quarter biotech job picture},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Trends in gender pay gaps of scientists and engineers in
academia and industry. <em>NBT</em>, <em>39</em>(8), 1019–1024. (<a
href="https://doi.org/10.1038/s41587-021-01008-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of sectoral differences in gender pay gap over career lifecycles reveals that academia has a wider gender pay gap relative to industry.},
  archive      = {J_NBT},
  author       = {Ding, Waverly W. and Ohyama, Atsushi and Agarwal, Rajshree},
  doi          = {10.1038/s41587-021-01008-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1019-1024},
  shortjournal = {Nature Biotech.},
  title        = {Trends in gender pay gaps of scientists and engineers in academia and industry},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Publisher correction: Voices of biotech leaders.
<em>NBT</em>, <em>39</em>(8), 1017. (<a
href="https://doi.org/10.1038/s41587-021-01000-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Bosley, Katrine and Casebourn, Charlotte and Chan, Priscilla and Chen, Janice and Chen, Michael and Church, George and Cumbers, John and de Wouters, Tomas and Dewey-Hagborg, Heather and Duportet, Xavier and Ene-Obong, Abasi and Elizondo, Arturo and Farrar, Jeremy and Gates, Bill and Gatto, Francesco and Giwa, Sebastian and Godec, Jernej and Gold, Silvia and LeProust, Emily and Lunshof, Jeantine and Martucci, Eddie and Heath, Michelle McMurray and Mellad, Jason and Oudova, Veronika and Oxman, Neri and Regev, Aviv and Richardson, Sarah and Scott, Christopher Thomas and Sherkow, Jake and Sibener, Leah and Tarragó, Teresa and Terry, Sharon and Venter, J. Craig and Wang, Spin and Wickramasekara, Sajith and Yadi, Hakim and Yang, Luhan and Zhao, Bowen},
  doi          = {10.1038/s41587-021-01000-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1017},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Voices of biotech leaders},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021k). Publisher correction: Questions swirl around failures of
disease-modifying huntington’s drugs. <em>NBT</em>, <em>39</em>(8),
1017. (<a href="https://doi.org/10.1038/s41587-021-01017-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-021-01017-z},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1017},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Questions swirl around failures of disease-modifying huntington’s drugs},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: COVID-19 long haulers. <em>NBT</em>,
<em>39</em>(8), 1017. (<a
href="https://doi.org/10.1038/s41587-021-01022-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Schmidt, Charles},
  doi          = {10.1038/s41587-021-01022-2},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1017},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: COVID-19 long haulers},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Bayesian inference of gene expression states from
single-cell RNA-seq data. <em>NBT</em>, <em>39</em>(8), 1008–1016. (<a
href="https://doi.org/10.1038/s41587-021-00875-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite substantial progress in single-cell RNA-seq (scRNA-seq) data analysis methods, there is still little agreement on how to best normalize such data. Starting from the basic requirements that inferred expression states should correct for both biological and measurement sampling noise and that changes in expression should be measured in terms of fold changes, we here derive a Bayesian normalization procedure called Sanity (SAmpling-Noise-corrected Inference of Transcription activitY) from first principles. Sanity estimates expression values and associated error bars directly from raw unique molecular identifier (UMI) counts without any tunable parameters. Using simulated and real scRNA-seq datasets, we show that Sanity outperforms other normalization methods on downstream tasks, such as finding nearest-neighbor cells and clustering cells into subtypes. Moreover, we show that by systematically overestimating the expression variability of genes with low expression and by introducing spurious correlations through mapping the data to a lower-dimensional representation, other methods yield severely distorted pictures of the data. A Bayesian procedure overcomes challenges in single-cell RNA-seq data normalization.},
  archive      = {J_NBT},
  author       = {Breda, Jérémie and Zavolan, Mihaela and van Nimwegen, Erik},
  doi          = {10.1038/s41587-021-00875-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1008-1016},
  shortjournal = {Nature Biotech.},
  title        = {Bayesian inference of gene expression states from single-cell RNA-seq data},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Iterative single-cell multi-omic integration using online
learning. <em>NBT</em>, <em>39</em>(8), 1000–1007. (<a
href="https://doi.org/10.1038/s41587-021-00867-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Integrating large single-cell gene expression, chromatin accessibility and DNA methylation datasets requires general and scalable computational approaches. Here we describe online integrative non-negative matrix factorization (iNMF), an algorithm for integrating large, diverse and continually arriving single-cell datasets. Our approach scales to arbitrarily large numbers of cells using fixed memory, iteratively incorporates new datasets as they are generated and allows many users to simultaneously analyze a single copy of a large dataset by streaming it over the internet. Iterative data addition can also be used to map new data to a reference dataset. Comparisons with previous methods indicate that the improvements in efficiency do not sacrifice dataset alignment and cluster preservation performance. We demonstrate the effectiveness of online iNMF by integrating more than 1 million cells on a standard laptop, integrating large single-cell RNA sequencing and spatial transcriptomic datasets, and iteratively constructing a single-cell multi-omic atlas of the mouse motor cortex. A new algorithm enables scalable and iterative integration of single-cell datasets.},
  archive      = {J_NBT},
  author       = {Gao, Chao and Liu, Jialin and Kriebel, April R. and Preissl, Sebastian and Luo, Chongyuan and Castanon, Rosa and Sandoval, Justin and Rivkin, Angeline and Nery, Joseph R. and Behrens, Margarita M. and Ecker, Joseph R. and Ren, Bing and Welch, Joshua D.},
  doi          = {10.1038/s41587-021-00867-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {1000-1007},
  shortjournal = {Nature Biotech.},
  title        = {Iterative single-cell multi-omic integration using online learning},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Generation of recombinant hyperimmune globulins from diverse
b-cell repertoires. <em>NBT</em>, <em>39</em>(8), 989–999. (<a
href="https://doi.org/10.1038/s41587-021-00894-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plasma-derived polyclonal antibody therapeutics, such as intravenous immunoglobulin, have multiple drawbacks, including low potency, impurities, insufficient supply and batch-to-batch variation. Here we describe a microfluidics and molecular genomics strategy for capturing diverse mammalian antibody repertoires to create recombinant multivalent hyperimmune globulins. Our method generates of diverse mixtures of thousands of recombinant antibodies, enriched for specificity and activity against therapeutic targets. Each hyperimmune globulin product comprised thousands to tens of thousands of antibodies derived from convalescent or vaccinated human donors or from immunized mice. Using this approach, we generated hyperimmune globulins with potent neutralizing activity against severe acute respiratory syndrome coronavirus-2 (SARS-CoV-2) in under 3 months, Fc-engineered hyperimmune globulins specific for Zika virus that lacked antibody-dependent enhancement of disease, and hyperimmune globulins specific for lung pathogens present in patients with primary immune deficiency. To address the limitations of rabbit-derived anti-thymocyte globulin, we generated a recombinant human version and demonstrated its efficacy in mice against graft-versus-host disease. Thousands of recombinant antibodies enriched for specificity against defined targets are assembled in multivalent mixtures with enhanced therapeutic activity.},
  archive      = {J_NBT},
  author       = {Keating, Sheila M. and Mizrahi, Rena A. and Adams, Matthew S. and Asensio, Michael A. and Benzie, Emily and Carter, Kyle P. and Chiang, Yao and Edgar, Robert C. and Gautam, Bishal K. and Gras, Ashley and Leong, Jackson and Leong, Renee and Lim, Yoong Wearn and Manickam, Vishal A. and Medina-Cucurella, Angelica V. and Niedecken, Ariel R. and Saini, Jasmeen and Simons, Jan Fredrik and Spindler, Matthew J. and Stadtmiller, Kacy and Tinsley, Brendan and Wagner, Ellen K. and Wayham, Nicholas and Tracy, LaRee and Lundberg, Carina Vingsbo and Büscher, Dirk and Terencio, Jose Vicente and Roalfe, Lucy and Pearce, Emma and Richardson, Hayley and Goldblatt, David and Ramjag, Anushka T. and Carrington, Christine V. F. and Simmons, Graham and Muench, Marcus O. and Chamow, Steven M. and Monroe, Bryan and Olson, Charles and Oguin, Thomas H. and Lynch, Heather and Jeanfreau, Robert and Mosher, Rachel A. and Walch, Matthew J. and Bartley, Christopher R. and Ross, Carl A. and Meyer, Everett H. and Adler, Adam S. and Johnson, David S.},
  doi          = {10.1038/s41587-021-00894-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {989-999},
  shortjournal = {Nature Biotech.},
  title        = {Generation of recombinant hyperimmune globulins from diverse B-cell repertoires},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Quantitative mapping of the cellular small RNA landscape
with AQRNA-seq. <em>NBT</em>, <em>39</em>(8), 978–988. (<a
href="https://doi.org/10.1038/s41587-021-00874-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current next-generation RNA-sequencing (RNA-seq) methods do not provide accurate quantification of small RNAs within a sample, due to sequence-dependent biases in capture, ligation and amplification during library preparation. We present a method, absolute quantification RNA-sequencing (AQRNA-seq), that minimizes biases and provides a direct, linear correlation between sequencing read count and copy number for all small RNAs in a sample. Library preparation and data processing were optimized and validated using a 963-member microRNA reference library, oligonucleotide standards of varying length, and RNA blots. Application of AQRNA-seq to a panel of human cancer cells revealed &amp;gt;800 detectable miRNAs that varied during cancer progression, while application to bacterial transfer RNA pools, with the challenges of secondary structure and abundant modifications, revealed 80-fold variation in tRNA isoacceptor levels, stress-induced site-specific tRNA fragmentation, quantitative modification maps, and evidence for stress-induced, tRNA-driven, codon-biased translation. AQRNA-seq thus provides a versatile means to quantitatively map the small RNA landscape in cells. AQRNA-seq allows accurate quantification of small RNAs.},
  archive      = {J_NBT},
  author       = {Hu, Jennifer F. and Yim, Daniel and Ma, Duanduan and Huber, Sabrina M. and Davis, Nick and Bacusmo, Jo Marie and Vermeulen, Sidney and Zhou, Jieliang and Begley, Thomas J. and DeMott, Michael S. and Levine, Stuart S. and de Crécy-Lagard, Valérie and Dedon, Peter C. and Cao, Bo},
  doi          = {10.1038/s41587-021-00874-y},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {978-988},
  shortjournal = {Nature Biotech.},
  title        = {Quantitative mapping of the cellular small RNA landscape with AQRNA-seq},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Spatial tissue profiling by imaging-free molecular
tomography. <em>NBT</em>, <em>39</em>(8), 968–977. (<a
href="https://doi.org/10.1038/s41587-021-00879-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Several techniques are currently being developed for spatially resolved omics profiling, but each new method requires the setup of specific detection strategies or specialized instrumentation. Here we describe an imaging-free framework to localize high-throughput readouts within a tissue by cutting the sample into thin strips in a way that allows subsequent image reconstruction. We implemented this framework to transform a low-input RNA sequencing protocol into an imaging-free spatial transcriptomics technique (called STRP-seq) and validated it by profiling the spatial transcriptome of the mouse brain. We applied the technique to the brain of the Australian bearded dragon, Pogona vitticeps. Our results reveal the molecular anatomy of the telencephalon of this lizard, providing evidence for a marked regionalization of the reptilian pallium and subpallium. We expect that STRP-seq can be used to derive spatially resolved data from a range of other omics techniques. A refined spatial sampling technique transforms standard RNA sequencing into a spatial transcriptomics method.},
  archive      = {J_NBT},
  author       = {Schede, Halima Hannah and Schneider, Christian G. and Stergiadou, Johanna and Borm, Lars E. and Ranjak, Anurag and Yamawaki, Tracy M. and David, Fabrice P. A. and Lönnerberg, Peter and Tosches, Maria Antonietta and Codeluppi, Simone and La Manno, Gioele},
  doi          = {10.1038/s41587-021-00879-7},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {968-977},
  shortjournal = {Nature Biotech.},
  title        = {Spatial tissue profiling by imaging-free molecular tomography},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Affinity-matured HLA class II dimers for robust staining of
antigen-specific CD4+ t cells. <em>NBT</em>, <em>39</em>(8), 958–967.
(<a href="https://doi.org/10.1038/s41587-021-00836-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peptide–major histocompatibility complex (pMHC) multimers enable the detection of antigen-specific T cells in studies ranging from vaccine efficacy to cancer immunotherapy. However, this technology is unreliable when applied to pMHC class II for the detection of CD4+ T cells. Here, using a combination of molecular biological and immunological techniques, we cloned sequences encoding human leukocyte antigen (HLA)-DP, HLA-DQ and HLA-DR molecules with enhanced CD4 binding affinity (with a Kd of 8.9 ± 1.1 µM between CD4 and affinity-matured HLA-DP4) and produced affinity-matured class II dimers that stain antigen-specific T cells better than conventional multimers in both in vitro and ex vivo analyses. Using a comprehensive library of dimers for HLA-DP4, which is the most frequent HLA allele in many ancestry groups, we mapped 103 HLA-DP4-restricted epitopes derived from diverse tumor-associated antigens and cloned the cognate T-cell antigen receptor (TCR) genes from in vitro-stimulated CD4+ T cells. The availability of affinity-matured class II dimers across HLA-DP, HLA-DQ and HLA-DR alleles will aid in the investigation of human CD4+ T-cell responses. CD4+ T cells are readily detected with class II MHC dimers.},
  archive      = {J_NBT},
  author       = {Sugata, Kenji and Matsunaga, Yukiko and Yamashita, Yuki and Nakatsugawa, Munehide and Guo, Tingxi and Halabelian, Levon and Ohashi, Yota and Saso, Kayoko and Rahman, Muhammed A. and Anczurowski, Mark and Wang, Chung-Hsi and Murata, Kenji and Saijo, Hiroshi and Kagoya, Yuki and Ly, Dalam and Burt, Brian D. and Butler, Marcus O. and Mak, Tak W. and Hirano, Naoto},
  doi          = {10.1038/s41587-021-00836-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {958-967},
  shortjournal = {Nature Biotech.},
  title        = {Affinity-matured HLA class II dimers for robust staining of antigen-specific CD4+ t cells},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). In vivo adenine base editing of PCSK9 in macaques reduces
LDL cholesterol levels. <em>NBT</em>, <em>39</em>(8), 949–957. (<a
href="https://doi.org/10.1038/s41587-021-00933-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most known pathogenic point mutations in humans are C•G to T•A substitutions, which can be directly repaired by adenine base editors (ABEs). In this study, we investigated the efficacy and safety of ABEs in the livers of mice and cynomolgus macaques for the reduction of blood low-density lipoprotein (LDL) levels. Lipid nanoparticle–based delivery of mRNA encoding an ABE and a single-guide RNA targeting PCSK9, a negative regulator of LDL, induced up to 67\% editing (on average, 61\%) in mice and up to 34\% editing (on average, 26\%) in macaques. Plasma PCSK9 and LDL levels were stably reduced by 95\% and 58\% in mice and by 32\% and 14\% in macaques, respectively. ABE mRNA was cleared rapidly, and no off-target mutations in genomic DNA were found. Re-dosing in macaques did not increase editing, possibly owing to the detected humoral immune response to ABE upon treatment. These findings support further investigation of ABEs to treat patients with monogenic liver diseases. Base editors are effective and safe for cholesterol reduction in non-human primates.},
  archive      = {J_NBT},
  author       = {Rothgangl, Tanja and Dennis, Melissa K. and Lin, Paulo J. C. and Oka, Rurika and Witzigmann, Dominik and Villiger, Lukas and Qi, Weihong and Hruzova, Martina and Kissling, Lucas and Lenggenhager, Daniela and Borrelli, Costanza and Egli, Sabina and Frey, Nina and Bakker, Noëlle and Walker, John A. and Kadina, Anastasia P. and Victorov, Denis V. and Pacesa, Martin and Kreutzer, Susanne and Kontarakis, Zacharias and Moor, Andreas and Jinek, Martin and Weissman, Drew and Stoffel, Markus and van Boxtel, Ruben and Holden, Kevin and Pardi, Norbert and Thöny, Beat and Häberle, Johannes and Tam, Ying K. and Semple, Sean C. and Schwank, Gerald},
  doi          = {10.1038/s41587-021-00933-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {949-957},
  shortjournal = {Nature Biotech.},
  title        = {In vivo adenine base editing of PCSK9 in macaques reduces LDL cholesterol levels},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MHC class II tetramers engineered for enhanced binding to
CD4 improve detection of antigen-specific t cells. <em>NBT</em>,
<em>39</em>(8), 943–948. (<a
href="https://doi.org/10.1038/s41587-021-00893-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ability to identify T cells that recognize specific peptide antigens bound to major histocompatibility complex (MHC) molecules has enabled enumeration and molecular characterization of the lymphocytes responsible for cell-mediated immunity. Fluorophore-labeled peptide:MHC class I (p:MHCI) tetramers are well-established reagents for identifying antigen-specific CD8+ T cells by flow cytometry, but efforts to extend the approach to CD4+ T cells have been less successful, perhaps owing to lower binding strength between CD4 and MHC class II (MHCII) molecules. Here we show that p:MHCII tetramers engineered by directed evolution for enhanced CD4 binding outperform conventional tetramers for the detection of cognate T cells. Using the engineered tetramers, we identified about twice as many antigen-specific CD4+ T cells in mice immunized against multiple peptides than when using traditional tetramers. CD4 affinity-enhanced p:MHCII tetramers, therefore, allow direct sampling of antigen-specific CD4+ T cells that cannot be accessed with conventional p:MHCII tetramer technology. These new reagents could provide a deeper understanding of the T cell repertoire. The detection of antigen-specific CD4+ T cells is improved with engineered peptide:MHCII tetramers.},
  archive      = {J_NBT},
  author       = {Dileepan, Thamotharampillai and Malhotra, Deepali and Kotov, Dmitri I. and Kolawole, Elizabeth M. and Krueger, Peter D. and Evavold, Brian D. and Jenkins, Marc K.},
  doi          = {10.1038/s41587-021-00893-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {943-948},
  shortjournal = {Nature Biotech.},
  title        = {MHC class II tetramers engineered for enhanced binding to CD4 improve detection of antigen-specific t cells},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Compressed sensing for highly efficient imaging
transcriptomics. <em>NBT</em>, <em>39</em>(8), 936–942. (<a
href="https://doi.org/10.1038/s41587-021-00883-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent methods for spatial imaging of tissue samples can identify up to ~100 individual proteins1–3 or RNAs4–10 at single-cell resolution. However, the number of proteins or genes that can be studied in these approaches is limited by long imaging times. Here we introduce Composite In Situ Imaging (CISI), a method that leverages structure in gene expression across both cells and tissues to limit the number of imaging cycles needed to obtain spatially resolved gene expression maps. CISI defines gene modules that can be detected using composite measurements from imaging probes for subsets of genes. The data are then decompressed to recover expression values for individual genes. CISI further reduces imaging time by not relying on spot-level resolution, enabling lower magnification acquisition, and is overall about 500-fold more efficient than current methods. Applying CISI to 12 mouse brain sections, we accurately recovered the spatial abundance of 37 individual genes from 11 composite measurements covering 180 mm2 and 476,276 cells. Spatial transcriptomics is enhanced by using imaging probes for subsets of genes.},
  archive      = {J_NBT},
  author       = {Cleary, Brian and Simonton, Brooke and Bezney, Jon and Murray, Evan and Alam, Shahul and Sinha, Anubhav and Habibi, Ehsan and Marshall, Jamie and Lander, Eric S. and Chen, Fei and Regev, Aviv},
  doi          = {10.1038/s41587-021-00883-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {936-942},
  shortjournal = {Nature Biotech.},
  title        = {Compressed sensing for highly efficient imaging transcriptomics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Engineering luminescent biosensors for point-of-care
SARS-CoV-2 antibody detection. <em>NBT</em>, <em>39</em>(8), 928–935.
(<a href="https://doi.org/10.1038/s41587-021-00878-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current serology tests for severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) antibodies mainly take the form of enzyme-linked immunosorbent assays, chemiluminescent microparticle immunoassays or lateral flow assays, which are either laborious, expensive or lacking sufficient sensitivity and scalability. Here we present the development and validation of a rapid, low-cost, solution-based assay to detect antibodies in serum, plasma, whole blood and to a lesser extent saliva, using rationally designed split luciferase antibody biosensors. This new assay, which generates quantitative results in 30 min, substantially reduces the complexity and improves the scalability of coronavirus disease 2019 (COVID-19) antibody tests. This assay is well-suited for point-of-care, broad population testing, and applications in low-resource settings, for monitoring host humoral responses to vaccination or viral infection. A rapid, split luciferase test for SARS-CoV-2 antibodies in blood and saliva compatible with lyophilization and use in low-resource settings.},
  archive      = {J_NBT},
  author       = {Elledge, Susanna K. and Zhou, Xin X. and Byrnes, James R. and Martinko, Alexander J. and Lui, Irene and Pance, Katarina and Lim, Shion A. and Glasgow, Jeff E. and Glasgow, Anum A. and Turcios, Keirstinne and Iyer, Nikita S. and Torres, Leonel and Peluso, Michael J. and Henrich, Timothy J. and Wang, Taia T. and Tato, Cristina M. and Leung, Kevin K. and Greenhouse, Bryan and Wells, James A.},
  doi          = {10.1038/s41587-021-00878-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {928-935},
  shortjournal = {Nature Biotech.},
  title        = {Engineering luminescent biosensors for point-of-care SARS-CoV-2 antibody detection},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). High-efficiency prime editing with optimized, paired pegRNAs
in plants. <em>NBT</em>, <em>39</em>(8), 923–927. (<a
href="https://doi.org/10.1038/s41587-021-00868-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editing (PE) applications are limited by low editing efficiency. Here we show that designing prime binding sites with a melting temperature of 30 °C leads to optimal performance in rice and that using two prime editing guide (peg) RNAs in trans encoding the same edits substantially enhances PE efficiency. Together, these approaches boost PE efficiency from 2.9-fold to 17.4-fold. Optimal pegRNAs or pegRNA pairs can be designed with our web application, PlantPegDesigner. Improved guide RNAs enhance the efficiency of prime editing.},
  archive      = {J_NBT},
  author       = {Lin, Qiupeng and Jin, Shuai and Zong, Yuan and Yu, Hong and Zhu, Zixu and Liu, Guanwen and Kou, Liquan and Wang, Yanpeng and Qiu, Jin-Long and Li, Jiayang and Gao, Caixia},
  doi          = {10.1038/s41587-021-00868-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {923-927},
  shortjournal = {Nature Biotech.},
  title        = {High-efficiency prime editing with optimized, paired pegRNAs in plants},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CRISPR base editing lowers cholesterol in monkeys.
<em>NBT</em>, <em>39</em>(8), 920–921. (<a
href="https://doi.org/10.1038/s41587-021-00975-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Targeting a gene linked to cardiovascular disease shows therapeutic promise.},
  archive      = {J_NBT},
  author       = {van Kampen, Sebastiaan Johannes and van Rooij, Eva},
  doi          = {10.1038/s41587-021-00975-8},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {920-921},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR base editing lowers cholesterol in monkeys},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Engineered models of the human embryo. <em>NBT</em>,
<em>39</em>(8), 918–920. (<a
href="https://doi.org/10.1038/s41587-021-01004-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Embryo models built from stem cells promise to shed light on human development.},
  archive      = {J_NBT},
  author       = {Popovic, Mina and Azpiroz, Felicitas and Chuva de Sousa Lopes, Susana M.},
  doi          = {10.1038/s41587-021-01004-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {918-920},
  shortjournal = {Nature Biotech.},
  title        = {Engineered models of the human embryo},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Base editors. <em>NBT</em>, <em>39</em>(8), 917. (<a
href="https://doi.org/10.1038/s41587-021-01015-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to the introduction of single-nucleotide variants into DNA or RNA in living cells.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01015-1},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {917},
  shortjournal = {Nature Biotech.},
  title        = {Base editors},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Negative innovation: When patents are bad for patients.
<em>NBT</em>, <em>39</em>(8), 914–916. (<a
href="https://doi.org/10.1038/s41587-021-00999-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Incentives in patent law have driven innovation into spaces that are affirmatively harmful to patients, and patentees are discouraged from taking steps to improve the product so as to prevent adverse health outcomes.},
  archive      = {J_NBT},
  author       = {Feldman, Robin C. and Hyman, David A. and Price, W. Nicholson and Ratain, Mark J.},
  doi          = {10.1038/s41587-021-00999-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {914-916},
  shortjournal = {Nature Biotech.},
  title        = {Negative innovation: When patents are bad for patients},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). 2Q21—a slowing of the pace. <em>NBT</em>, <em>39</em>(8),
906–907. (<a href="https://doi.org/10.1038/s41587-021-01010-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-01010-6},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {906-907},
  shortjournal = {Nature Biotech.},
  title        = {2Q21—a slowing of the pace},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). Drug pipeline 2Q21—accelerated approvals to the fore.
<em>NBT</em>, <em>39</em>(8), 904–905. (<a
href="https://doi.org/10.1038/s41587-021-01013-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-01013-3},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {904-905},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 2Q21—accelerated approvals to the fore},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Around the world in a month. <em>NBT</em>, <em>39</em>(8),
903. (<a href="https://doi.org/10.1038/s41587-021-01012-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01012-4},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {903},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CRISPR super-sizes corn. <em>NBT</em>, <em>39</em>(8), 902.
(<a href="https://doi.org/10.1038/s41587-021-01028-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01028-w},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {902},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR super-sizes corn},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). WHO takes up CRISPR babies. <em>NBT</em>, <em>39</em>(8),
901. (<a href="https://doi.org/10.1038/s41587-021-01027-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01027-x},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {901},
  shortjournal = {Nature Biotech.},
  title        = {WHO takes up CRISPR babies},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). AI drug discovery booms in china. <em>NBT</em>,
<em>39</em>(8), 900–902. (<a
href="https://doi.org/10.1038/s41587-021-01016-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chinese upstart companies and IT goliaths hope to turn country’s artificial intelligence prowess into world-leading drug innovation. Do they have an edge over Western players?},
  archive      = {J_NBT},
  author       = {Cyranoski, David},
  doi          = {10.1038/s41587-021-01016-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {900-902},
  shortjournal = {Nature Biotech.},
  title        = {AI drug discovery booms in china},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The 3D protein deluge. <em>NBT</em>, <em>39</em>(8), 899.
(<a href="https://doi.org/10.1038/s41587-021-01029-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01029-9},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {899},
  shortjournal = {Nature Biotech.},
  title        = {The 3D protein deluge},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The next debate on embryo science. <em>NBT</em>,
<em>39</em>(8), 895. (<a
href="https://doi.org/10.1038/s41587-021-01032-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The International Society for Stem Cell Research has called for broad public dialogue on the ethics of human embryo research beyond 14 days post-fertilization. National jurisdictions should seize the moment.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01032-0},
  journal      = {Nature Biotechnology},
  month        = {8},
  number       = {8},
  pages        = {895},
  shortjournal = {Nature Biotech.},
  title        = {The next debate on embryo science},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). First rounders: Nancy simonian. <em>NBT</em>,
<em>39</em>(8), 922. (<a
href="https://doi.org/10.1038/s41587-021-01003-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-01003-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {922},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Nancy simonian},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). COVID-19 long haulers. <em>NBT</em>, <em>39</em>(8),
908–913. (<a href="https://doi.org/10.1038/s41587-021-00984-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With the risks of drug development prohibitive, repurposed or repositioned medicines appear the best hope against long-COVID, a condition that still raises many unanswered questions.},
  archive      = {J_NBT},
  author       = {Schmidt, Charles},
  doi          = {10.1038/s41587-021-00984-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {908-913},
  shortjournal = {Nature Biotech.},
  title        = {COVID-19 long haulers},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). CRISPR therapies march into clinic, but genotoxicity
concerns linger. <em>NBT</em>, <em>39</em>(8), 897–899. (<a
href="https://doi.org/10.1038/d41587-021-00017-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Following reports of collateral damage caused by CRISPR genome editing, now chromothripsis, a phenomenon associated with cancer, enters the spotlight. Following reports of collateral damage caused by CRISPR genome editing, now chromothripsis, a phenomenon associated with cancer, enters the spotlight.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/d41587-021-00017-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {8},
  pages        = {897-899},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR therapies march into clinic, but genotoxicity concerns linger},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). People. <em>NBT</em>, <em>39</em>(7), 894. (<a
href="https://doi.org/10.1038/s41587-021-00977-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00977-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {894},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: Comprehensive profiling of circular RNAs
with nanopore sequencing and CIRI-long. <em>NBT</em>, <em>39</em>(7),
893. (<a href="https://doi.org/10.1038/s41587-021-00934-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41587-021-00934-3.},
  archive      = {J_NBT},
  author       = {Zhang, Jinyang and Hou, Lingling and Zuo, Zhenqiang and Ji, Peifeng and Zhang, Xiaorong and Xue, Yuanchao and Zhao, Fangqing},
  doi          = {10.1038/s41587-021-00934-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {893},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Comprehensive profiling of circular RNAs with nanopore sequencing and CIRI-long},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021l). Publisher correction: Questions swirl around failures of
disease-modifying huntington’s drugs. <em>NBT</em>, <em>39</em>(7), 893.
(<a href="https://doi.org/10.1038/s41587-021-00987-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-021-00987-4},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {893},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Questions swirl around failures of disease-modifying huntington’s drugs},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). Publisher correction: “Super-antibodies” could curb
COVID-19 and help avert future pandemics. <em>NBT</em>, <em>39</em>(7),
893. (<a href="https://doi.org/10.1038/s41587-021-00997-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-021-00997-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {893},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: ‘Super-antibodies’ could curb COVID-19 and help avert future pandemics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A unified haplotype-based method for accurate and
comprehensive variant calling. <em>NBT</em>, <em>39</em>(7), 885–892.
(<a href="https://doi.org/10.1038/s41587-021-00861-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Almost all haplotype-based variant callers were designed specifically for detecting common germline variation in diploid populations, and give suboptimal results in other scenarios. Here we present Octopus, a variant caller that uses a polymorphic Bayesian genotyping model capable of modeling sequencing data from a range of experimental designs within a unified haplotype-aware framework. Octopus combines sequencing reads and prior information to phase-called genotypes of arbitrary ploidy, including those with somatic mutations. We show that Octopus accurately calls germline variants in individuals, including single nucleotide variants, indels and small complex replacements such as microinversions. Using a synthetic tumor data set derived from clean sequencing data from a sample with known germline haplotypes and observed mutations in a large cohort of tumor samples, we show that Octopus is more sensitive to low-frequency somatic variation, yet calls considerably fewer false positives than other methods. Octopus also outputs realigned evidence BAM files to aid validation and interpretation. Octopus detects germline and somatic variants with high sensitivity and accuracy.},
  archive      = {J_NBT},
  author       = {Cooke, Daniel P. and Wedge, David C. and Lunter, Gerton},
  doi          = {10.1038/s41587-021-00861-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {885-892},
  shortjournal = {Nature Biotech.},
  title        = {A unified haplotype-based method for accurate and comprehensive variant calling},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Robust integration of multiple single-cell RNA sequencing
datasets using a single reference space. <em>NBT</em>, <em>39</em>(7),
877–884. (<a href="https://doi.org/10.1038/s41587-021-00859-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In many biological applications of single-cell RNA sequencing (scRNA-seq), an integrated analysis of data from multiple batches or studies is necessary. Current methods typically achieve integration using shared cell types or covariance correlation between datasets, which can distort biological signals. Here we introduce an algorithm that uses the gene eigenvectors from a reference dataset to establish a global frame for integration. Using simulated and real datasets, we demonstrate that this approach, called Reference Principal Component Integration (RPCI), consistently outperforms other methods by multiple metrics, with clear advantages in preserving genuine cross-sample gene expression differences in matching cell types, such as those present in cells at distinct developmental stages or in perturbated versus control studies. Moreover, RPCI maintains this robust performance when multiple datasets are integrated. Finally, we applied RPCI to scRNA-seq data for mouse gut endoderm development and revealed temporal emergence of genetic programs helping establish the anterior–posterior axis in visceral endoderm. Integration of multiple single-cell RNA sequencing datasets is improved by creating a common reference space using a new algorithm.},
  archive      = {J_NBT},
  author       = {Liu, Yang and Wang, Tao and Zhou, Bin and Zheng, Deyou},
  doi          = {10.1038/s41587-021-00859-x},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {877-884},
  shortjournal = {Nature Biotech.},
  title        = {Robust integration of multiple single-cell RNA sequencing datasets using a single reference space},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Variability within rare cell states enables multiple paths
toward drug resistance. <em>NBT</em>, <em>39</em>(7), 865–876. (<a
href="https://doi.org/10.1038/s41587-021-00837-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular differences between individual cells can lead to dramatic differences in cell fate, such as death versus survival of cancer cells upon drug treatment. These originating differences remain largely hidden due to difficulties in determining precisely what variable molecular features lead to which cellular fates. Thus, we developed Rewind, a methodology that combines genetic barcoding with RNA fluorescence in situ hybridization to directly capture rare cells that give rise to cellular behaviors of interest. Applying Rewind to BRAFV600E melanoma, we trace drug-resistant cell fates back to single-cell gene expression differences in their drug-naive precursors (initial frequency of ~1:1,000–1:10,000 cells) and relative persistence of MAP kinase signaling soon after drug treatment. Within this rare subpopulation, we uncover a rich substructure in which molecular differences among several distinct subpopulations predict future differences in phenotypic behavior, such as proliferative capacity of distinct resistant clones after drug treatment. Our results reveal hidden, rare-cell variability that underlies a range of latent phenotypic outcomes upon drug exposure. A new methodology, Rewind, traces vemurafenib-resistant melanoma back to its initial cell state before drug treatment, creating, effectively, a cellular time machine.},
  archive      = {J_NBT},
  author       = {Emert, Benjamin L. and Cote, Christopher J. and Torre, Eduardo A. and Dardani, Ian P. and Jiang, Connie L. and Jain, Naveen and Shaffer, Sydney M. and Raj, Arjun},
  doi          = {10.1038/s41587-021-00837-3},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {865-876},
  shortjournal = {Nature Biotech.},
  title        = {Variability within rare cell states enables multiple paths toward drug resistance},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Monitoring deep-tissue oxygenation with a millimeter-scale
ultrasonic implant. <em>NBT</em>, <em>39</em>(7), 855–864. (<a
href="https://doi.org/10.1038/s41587-021-00866-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Vascular complications following solid organ transplantation may lead to graft ischemia, dysfunction or loss. Imaging approaches can provide intermittent assessments of graft perfusion, but require highly skilled practitioners and do not directly assess graft oxygenation. Existing systems for monitoring tissue oxygenation are limited by the need for wired connections, the inability to provide real-time data or operation restricted to surface tissues. Here, we present a minimally invasive system to monitor deep-tissue O2 that reports continuous real-time data from centimeter-scale depths in sheep and up to a 10-cm depth in ex vivo porcine tissue. The system is composed of a millimeter-sized, wireless, ultrasound-powered implantable luminescence O2 sensor and an external transceiver for bidirectional data transfer, enabling deep-tissue oxygenation monitoring for surgical or critical care indications. The oxygenation of deep tissues is continuously measured using an ultrasound-powered wireless implant.},
  archive      = {J_NBT},
  author       = {Sonmezoglu, Soner and Fineman, Jeffrey R. and Maltepe, Emin and Maharbiz, Michel M.},
  doi          = {10.1038/s41587-021-00866-y},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {855-864},
  shortjournal = {Nature Biotech.},
  title        = {Monitoring deep-tissue oxygenation with a millimeter-scale ultrasonic implant},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ultra-fast proteomics with scanning SWATH. <em>NBT</em>,
<em>39</em>(7), 846–854. (<a
href="https://doi.org/10.1038/s41587-021-00860-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate quantification of the proteome remains challenging for large sample series and longitudinal experiments. We report a data-independent acquisition method, Scanning SWATH, that accelerates mass spectrometric (MS) duty cycles, yielding quantitative proteomes in combination with short gradients and high-flow (800 µl min–1) chromatography. Exploiting a continuous movement of the precursor isolation window to assign precursor masses to tandem mass spectrometry (MS/MS) fragment traces, Scanning SWATH increases precursor identifications by ~70\% compared to conventional data-independent acquisition (DIA) methods on 0.5–5-min chromatographic gradients. We demonstrate the application of ultra-fast proteomics in drug mode-of-action screening and plasma proteomics. Scanning SWATH proteomes capture the mode of action of fungistatic azoles and statins. Moreover, we confirm 43 and identify 11 new plasma proteome biomarkers of COVID-19 severity, advancing patient classification and biomarker discovery. Thus, our results demonstrate a substantial acceleration and increased depth in fast proteomic experiments that facilitate proteomic drug screens and clinical studies. Scanning SWATH increases the speed and selectivity of proteomics.},
  archive      = {J_NBT},
  author       = {Messner, Christoph B. and Demichev, Vadim and Bloomfield, Nic and Yu, Jason S. L. and White, Matthew and Kreidl, Marco and Egger, Anna-Sophia and Freiwald, Anja and Ivosev, Gordana and Wasim, Fras and Zelezniak, Aleksej and Jürgens, Linda and Suttorp, Norbert and Sander, Leif Erik and Kurth, Florian and Lilley, Kathryn S. and Mülleder, Michael and Tate, Stephen and Ralser, Markus},
  doi          = {10.1038/s41587-021-00860-4},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {846-854},
  shortjournal = {Nature Biotech.},
  title        = {Ultra-fast proteomics with scanning SWATH},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Comprehensive profiling of circular RNAs with nanopore
sequencing and CIRI-long. <em>NBT</em>, <em>39</em>(7), 836–845. (<a
href="https://doi.org/10.1038/s41587-021-00842-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reconstructing the sequence of circular RNAs (circRNAs) from short RNA sequencing reads has proved challenging given the similarity of circRNAs and their corresponding linear messenger RNAs. Previous sequencing methods were unable to achieve high-throughput detection of full-length circRNAs. Here we describe a protocol for enrichment and full-length sequencing of circRNA isoforms using nanopore technology. Circular reverse transcription and size selection achieves a 20-fold higher enrichment of circRNAs from total RNA compared to previous methods. We developed an algorithm, called circRNA identifier using long-read sequencing data (CIRI-long), to reconstruct the sequence of circRNAs. The workflow was validated with simulated data and by comparison to Illumina sequencing as well as quantitative real-time RT–PCR. We used CIRI-long to analyze adult mouse brain samples and systematically profile circRNAs, including mitochondria-derived and transcriptional read-through circRNAs. We identified a new type of intronic self-ligated circRNA that exhibits special splicing and expression patterns. Our method takes advantage of nanopore long reads and enables unbiased reconstruction of full-length circRNA sequences. The rich diversity of circular RNAs is sequenced using nanopore long reads.},
  archive      = {J_NBT},
  author       = {Zhang, Jinyang and Hou, Lingling and Zuo, Zhenqiang and Ji, Peifeng and Zhang, Xiaorong and Xue, Yuanchao and Zhao, Fangqing},
  doi          = {10.1038/s41587-021-00842-6},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {836-845},
  shortjournal = {Nature Biotech.},
  title        = {Comprehensive profiling of circular RNAs with nanopore sequencing and CIRI-long},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-cell CUT&amp;tag profiles histone modifications and
transcription factors in complex tissues. <em>NBT</em>, <em>39</em>(7),
825–835. (<a href="https://doi.org/10.1038/s41587-021-00869-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In contrast to single-cell approaches for measuring gene expression and DNA accessibility, single-cell methods for analyzing histone modifications are limited by low sensitivity and throughput. Here, we combine the CUT&amp;amp;Tag technology, developed to measure bulk histone modifications, with droplet-based single-cell library preparation to produce high-quality single-cell data on chromatin modifications. We apply single-cell CUT&amp;amp;Tag (scCUT&amp;amp;Tag) to tens of thousands of cells of the mouse central nervous system and probe histone modifications characteristic of active promoters, enhancers and gene bodies (H3K4me3, H3K27ac and H3K36me3) and inactive regions (H3K27me3). These scCUT&amp;amp;Tag profiles were sufficient to determine cell identity and deconvolute regulatory principles such as promoter bivalency, spreading of H3K4me3 and promoter–enhancer connectivity. We also used scCUT&amp;amp;Tag to investigate the single-cell chromatin occupancy of transcription factor OLIG2 and the cohesin complex component RAD21. Our results indicate that analysis of histone modifications and transcription factor occupancy at single-cell resolution provides unique insights into epigenomic landscapes in the central nervous system. An improved method for single-cell analysis of histone modifications is applied to the mouse brain.},
  archive      = {J_NBT},
  author       = {Bartosovic, Marek and Kabbe, Mukund and Castelo-Branco, Gonçalo},
  doi          = {10.1038/s41587-021-00869-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {825-835},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell CUT&amp;Tag profiles histone modifications and transcription factors in complex tissues},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-cell CUT&amp;tag analysis of chromatin modifications
in differentiation and tumor progression. <em>NBT</em>, <em>39</em>(7),
819–824. (<a href="https://doi.org/10.1038/s41587-021-00865-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods for quantifying gene expression1 and chromatin accessibility2 in single cells are well established, but single-cell analysis of chromatin regions with specific histone modifications has been technically challenging. In this study, we adapted the CUT&amp;amp;Tag method3 to scalable nanowell and droplet-based single-cell platforms to profile chromatin landscapes in single cells (scCUT&amp;amp;Tag) from complex tissues and during the differentiation of human embryonic stem cells. We focused on profiling polycomb group (PcG) silenced regions marked by histone H3 Lys27 trimethylation (H3K27me3) in single cells as an orthogonal approach to chromatin accessibility for identifying cell states. We show that scCUT&amp;amp;Tag profiling of H3K27me3 distinguishes cell types in human blood and allows the generation of cell-type-specific PcG landscapes from heterogeneous tissues. Furthermore, we used scCUT&amp;amp;Tag to profile H3K27me3 in a patient with a brain tumor before and after treatment, identifying cell types in the tumor microenvironment and heterogeneity in PcG activity in the primary sample and after treatment. An improved method for single-cell analysis of histone modifications is applied to stem cell differentiation and cancer.},
  archive      = {J_NBT},
  author       = {Wu, Steven J. and Furlan, Scott N. and Mihalas, Anca B. and Kaya-Okur, Hatice S. and Feroze, Abdullah H. and Emerson, Samuel N. and Zheng, Ye and Carson, Kalee and Cimino, Patrick J. and Keene, C. Dirk and Sarthy, Jay F. and Gottardo, Raphael and Ahmad, Kami and Henikoff, Steven and Patel, Anoop P.},
  doi          = {10.1038/s41587-021-00865-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {819-824},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell CUT&amp;Tag analysis of chromatin modifications in differentiation and tumor progression},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Modular, efficient and constant-memory single-cell RNA-seq
preprocessing. <em>NBT</em>, <em>39</em>(7), 813–818. (<a
href="https://doi.org/10.1038/s41587-021-00870-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We describe a workflow for preprocessing of single-cell RNA-sequencing data that balances efficiency and accuracy. Our workflow is based on the kallisto and bustools programs, and is near optimal in speed with a constant memory requirement providing scalability for arbitrarily large datasets. The workflow is modular, and we demonstrate its flexibility by showing how it can be used for RNA velocity analyses. A preprocessing workflow for single-cell RNA-seq data achieves near-optimal speed.},
  archive      = {J_NBT},
  author       = {Melsted, Páll and Booeshaghi, A. Sina and Liu, Lauren and Gao, Fan and Lu, Lambda and Min, Kyung Hoi (Joseph) and da Veiga Beltrame, Eduardo and Hjörleifsson, Kristján Eldjárn and Gehring, Jase and Pachter, Lior},
  doi          = {10.1038/s41587-021-00870-2},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {813-818},
  shortjournal = {Nature Biotech.},
  title        = {Modular, efficient and constant-memory single-cell RNA-seq preprocessing},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Circular RNAs sequenced at last. <em>NBT</em>,
<em>39</em>(7), 811–812. (<a
href="https://doi.org/10.1038/s41587-021-00916-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Full-length circular RNAs are sequenced at high throughput using nanopores.},
  archive      = {J_NBT},
  author       = {Zhang, Zhao and Han, Leng},
  doi          = {10.1038/s41587-021-00916-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {811-812},
  shortjournal = {Nature Biotech.},
  title        = {Circular RNAs sequenced at last},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Increasing proteomics throughput. <em>NBT</em>,
<em>39</em>(7), 809–810. (<a
href="https://doi.org/10.1038/s41587-021-00881-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new dimension for analyzing mass spectrometry data allows rapid quantification of up to 70\% more peptides.},
  archive      = {J_NBT},
  author       = {Slavov, Nikolai},
  doi          = {10.1038/s41587-021-00881-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {809-810},
  shortjournal = {Nature Biotech.},
  title        = {Increasing proteomics throughput},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Microbiomes. <em>NBT</em>, <em>39</em>(7), 808. (<a
href="https://doi.org/10.1038/s41587-021-00978-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to microbiome modulation and analysis, and the formulation of live biotherapeutics.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00978-5},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {808},
  shortjournal = {Nature Biotech.},
  title        = {Microbiomes},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ebola as a case study for the patent landscape of medical
countermeasures for emerging infectious diseases. <em>NBT</em>,
<em>39</em>(7), 799–807. (<a
href="https://doi.org/10.1038/s41587-021-00970-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A survey of patenting activity can facilitate coordination of R&amp;amp;D activities to fight emerging infectious diseases.},
  archive      = {J_NBT},
  author       = {Mohajel, Nasir and Arashkia, Arash},
  doi          = {10.1038/s41587-021-00970-z},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {799-807},
  shortjournal = {Nature Biotech.},
  title        = {Ebola as a case study for the patent landscape of medical countermeasures for emerging infectious diseases},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Regulatory and intellectual property conundrums surrounding
xenotransplantation. <em>NBT</em>, <em>39</em>(7), 796–798. (<a
href="https://doi.org/10.1038/s41587-021-00976-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kwisda, Koko and Cantz, Tobias and Hoppe, Nils},
  doi          = {10.1038/s41587-021-00976-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {796-798},
  shortjournal = {Nature Biotech.},
  title        = {Regulatory and intellectual property conundrums surrounding xenotransplantation},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). China at the threshold. <em>NBT</em>, <em>39</em>(7),
789–795. (<a href="https://doi.org/10.1038/s41587-021-00973-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {China’s growing influence in biotech was underscored in 2020 during a record-breaking year for sector financing.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-021-00973-w},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {789-795},
  shortjournal = {Nature Biotech.},
  title        = {China at the threshold},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Around the world in a month. <em>NBT</em>, <em>39</em>(7),
788. (<a href="https://doi.org/10.1038/s41587-021-00979-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00979-4},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {788},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Germany’s biotech behemoth. <em>NBT</em>, <em>39</em>(7),
787. (<a href="https://doi.org/10.1038/s41587-021-00990-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00990-9},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {787},
  shortjournal = {Nature Biotech.},
  title        = {Germany’s biotech behemoth},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Malaria vaccine wows and seeds COVID-19 vaccine effort.
<em>NBT</em>, <em>39</em>(7), 786–787. (<a
href="https://doi.org/10.1038/s41587-021-00983-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A malaria vaccine not only has achieved record-setting protection in a large trial, but also spurred the UK’s COVID-19 vaccine effort.},
  archive      = {J_NBT},
  author       = {Ledford, Heidi},
  doi          = {10.1038/s41587-021-00983-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {786-787},
  shortjournal = {Nature Biotech.},
  title        = {Malaria vaccine wows and seeds COVID-19 vaccine effort},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). First parkinson’s patients dosed with dopaminergic neurons.
<em>NBT</em>, <em>39</em>(7), 785. (<a
href="https://doi.org/10.1038/s41587-021-00992-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00992-7},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {785},
  shortjournal = {Nature Biotech.},
  title        = {First parkinson’s patients dosed with dopaminergic neurons},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Novavax’s fridge-friendly vaccine impresses. <em>NBT</em>,
<em>39</em>(7), 784. (<a
href="https://doi.org/10.1038/s41587-021-00991-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00991-8},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {784},
  shortjournal = {Nature Biotech.},
  title        = {Novavax’s fridge-friendly vaccine impresses},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A carte blanche approval in alzheimer’s. <em>NBT</em>,
<em>39</em>(7), 781. (<a
href="https://doi.org/10.1038/s41587-021-00995-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An Alzheimer’s drug approval is being touted as a triumph for patients. But US healthcare is ill prepared for Aduhelm’s questionable efficacy, unfettered market access and high price.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00995-4},
  journal      = {Nature Biotechnology},
  month        = {7},
  number       = {7},
  pages        = {781},
  shortjournal = {Nature Biotech.},
  title        = {A carte blanche approval in alzheimer’s},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). “Super-antibodies” could curb COVID-19 and help avert
future pandemics. <em>NBT</em>, <em>39</em>(7), 783–785. (<a
href="https://doi.org/10.1038/s41587-021-00980-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Companies are designing next-generation antibodies modeled on those taken from unique individuals whose immune systems can neutralize any COVID-19 variant—and related coronaviruses, too.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-021-00980-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {7},
  pages        = {783-785},
  shortjournal = {Nature Biotech.},
  title        = {‘Super-antibodies’ could curb COVID-19 and help avert future pandemics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). People. <em>NBT</em>, <em>39</em>(6), 780. (<a
href="https://doi.org/10.1038/s41587-021-00956-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00956-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {780},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). When mentoring matters: A french mentoring program for women
in science. <em>NBT</em>, <em>39</em>(6), 776–779. (<a
href="https://doi.org/10.1038/s41587-021-00951-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An innovative program addresses the need for support, encouragement and guidance on the part of women scientists in the early years of their career, during their PhD.},
  archive      = {J_NBT},
  author       = {Batut, Julie and Kvaskoff, Marina and Morris, May C.},
  doi          = {10.1038/s41587-021-00951-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {776-779},
  shortjournal = {Nature Biotech.},
  title        = {When mentoring matters: A french mentoring program for women in science},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Human heart-forming organoids
recapitulate early heart and foregut development. <em>NBT</em>,
<em>39</em>(6), 775. (<a
href="https://doi.org/10.1038/s41587-021-00960-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Drakhlis, Lika and Biswanath, Santoshi and Farr, Clara-Milena and Lupanow, Victoria and Teske, Jana and Ritzenhoff, Katharina and Franke, Annika and Manstein, Felix and Bolesani, Emiliano and Kempf, Henning and Liebscher, Simone and Schenke-Layland, Katja and Hegermann, Jan and Nolte, Lena and Meyer, Heiko and de la Roche, Jeanne and Thiemann, Stefan and Wahl-Schott, Christian and Martin, Ulrich and Zweigerdt, Robert},
  doi          = {10.1038/s41587-021-00960-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {775},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Human heart-forming organoids recapitulate early heart and foregut development},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Assessing single-cell transcriptomic variability through
density-preserving data visualization. <em>NBT</em>, <em>39</em>(6),
765–774. (<a href="https://doi.org/10.1038/s41587-020-00801-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nonlinear data visualization methods, such as t-distributed stochastic neighbor embedding (t-SNE) and uniform manifold approximation and projection (UMAP), summarize the complex transcriptomic landscape of single cells in two dimensions or three dimensions, but they neglect the local density of data points in the original space, often resulting in misleading visualizations where densely populated subsets of cells are given more visual space than warranted by their transcriptional diversity in the dataset. Here we present den-SNE and densMAP, which are density-preserving visualization tools based on t-SNE and UMAP, respectively, and demonstrate their ability to accurately incorporate information about transcriptomic variability into the visual interpretation of single-cell RNA sequencing data. Applied to recently published datasets, our methods reveal significant changes in transcriptomic variability in a range of biological processes, including heterogeneity in transcriptomic variability of immune cells in blood and tumor, human immune cell specialization and the developmental trajectory of Caenorhabditis elegans. Our methods are readily applicable to visualizing high-dimensional data in other scientific domains. den-SNE and densMAP enhance single-cell transcriptomic data visualization by incorporating density information.},
  archive      = {J_NBT},
  author       = {Narayan, Ashwin and Berger, Bonnie and Cho, Hyunghoon},
  doi          = {10.1038/s41587-020-00801-7},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {765-774},
  shortjournal = {Nature Biotech.},
  title        = {Assessing single-cell transcriptomic variability through density-preserving data visualization},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A bacterial protease depletes c-MYC and increases survival
in mouse models of bladder and colon cancer. <em>NBT</em>,
<em>39</em>(6), 754–764. (<a
href="https://doi.org/10.1038/s41587-020-00805-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Is the oncogene MYC upregulated or hyperactive? In the majority of human cancers, finding agents that target c-MYC has proved difficult. Here we report specific bacterial effector molecules that inhibit cellular MYC (c-MYC) in human cells. We show that uropathogenic Escherichia coli (UPEC) degrade the c-MYC protein and attenuate MYC expression in both human cells and animal tissues. c-MYC protein was rapidly degraded by both cell-free bacterial lysates and the purified bacterial protease Lon. In mice, intravesical or peroral delivery of Lon protease delayed tumor progression and increased survival in MYC-dependent bladder and colon cancer models, respectively. These results suggest that bacteria have evolved strategies to control c-MYC tissue levels in the host and that the Lon protease shows promise for therapeutic targeting of c-MYC in cancer. The oncoprotein c-Myc, long considered ‘undruggable’, is targeted by a bacterial protease in two tumor models.},
  archive      = {J_NBT},
  author       = {Butler, Daniel S. C. and Cafaro, Caterina and Putze, Johannes and Wan, Murphy Lam Yim and Tran, Thi Hien and Ambite, Ines and Ahmadi, Shahram and Kjellström, Sven and Welinder, Charlotte and Chao, Sing Ming and Dobrindt, Ulrich and Svanborg, Catharina},
  doi          = {10.1038/s41587-020-00805-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {754-764},
  shortjournal = {Nature Biotech.},
  title        = {A bacterial protease depletes c-MYC and increases survival in mouse models of bladder and colon cancer},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Biological activity-based modeling identifies antiviral
leads against SARS-CoV-2. <em>NBT</em>, <em>39</em>(6), 747–753. (<a
href="https://doi.org/10.1038/s41587-021-00839-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational approaches for drug discovery, such as quantitative structure–activity relationship, rely on structural similarities of small molecules to infer biological activity but are often limited to identifying new drug candidates in the chemical spaces close to known ligands. Here we report a biological activity-based modeling (BABM) approach, in which compound activity profiles established across multiple assays are used as signatures to predict compound activity in other assays or against a new target. This approach was validated by identifying candidate antivirals for Zika and Ebola viruses based on high-throughput screening data. BABM models were then applied to predict 311 compounds with potential activity against severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2). Of the predicted compounds, 32\% had antiviral activity in a cell culture live virus assay, the most potent compounds showing a half-maximal inhibitory concentration in the nanomolar range. Most of the confirmed anti-SARS-CoV-2 compounds were found to be viral entry inhibitors and/or autophagy modulators. The confirmed compounds have the potential to be further developed into anti-SARS-CoV-2 therapies. Activity profiles generated from quantitative high-throughput screening improve drug candidate prediction.},
  archive      = {J_NBT},
  author       = {Huang, Ruili and Xu, Miao and Zhu, Hu and Chen, Catherine Z. and Zhu, Wei and Lee, Emily M. and He, Shihua and Zhang, Li and Zhao, Jinghua and Shamim, Khalida and Bougie, Danielle and Huang, Wenwei and Xia, Menghang and Hall, Mathew D. and Lo, Donald and Simeonov, Anton and Austin, Christopher P. and Qiu, Xiangguo and Tang, Hengli and Zheng, Wei},
  doi          = {10.1038/s41587-021-00839-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {747-753},
  shortjournal = {Nature Biotech.},
  title        = {Biological activity-based modeling identifies antiviral leads against SARS-CoV-2},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Human heart-forming organoids recapitulate early heart and
foregut development. <em>NBT</em>, <em>39</em>(6), 737–746. (<a
href="https://doi.org/10.1038/s41587-021-00815-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Organoid models of early tissue development have been produced for the intestine, brain, kidney and other organs, but similar approaches for the heart have been lacking. Here we generate complex, highly structured, three-dimensional heart-forming organoids (HFOs) by embedding human pluripotent stem cell aggregates in Matrigel followed by directed cardiac differentiation via biphasic WNT pathway modulation with small molecules. HFOs are composed of a myocardial layer lined by endocardial-like cells and surrounded by septum-transversum-like anlagen; they further contain spatially and molecularly distinct anterior versus posterior foregut endoderm tissues and a vascular network. The architecture of HFOs closely resembles aspects of early native heart anlagen before heart tube formation, which is known to require an interplay with foregut endoderm development. We apply HFOs to study genetic defects in vitro by demonstrating that NKX2.5-knockout HFOs show a phenotype reminiscent of cardiac malformations previously observed in transgenic mice. Heart-forming organoids model early cardiac development.},
  archive      = {J_NBT},
  author       = {Drakhlis, Lika and Biswanath, Santoshi and Farr, Clara-Milena and Lupanow, Victoria and Teske, Jana and Ritzenhoff, Katharina and Franke, Annika and Manstein, Felix and Bolesani, Emiliano and Kempf, Henning and Liebscher, Simone and Schenke-Layland, Katja and Hegermann, Jan and Nolte, Lena and Meyer, Heiko and de la Roche, Jeanne and Thiemann, Stefan and Wahl-Schott, Christian and Martin, Ulrich and Zweigerdt, Robert},
  doi          = {10.1038/s41587-021-00815-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {737-746},
  shortjournal = {Nature Biotech.},
  title        = {Human heart-forming organoids recapitulate early heart and foregut development},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). InStrain profiles population microdiversity from metagenomic
data and sensitively detects shared microbial strains. <em>NBT</em>,
<em>39</em>(6), 727–736. (<a
href="https://doi.org/10.1038/s41587-020-00797-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coexisting microbial cells of the same species often exhibit genetic variation that can affect phenotypes ranging from nutrient preference to pathogenicity. Here we present inStrain, a program that uses metagenomic paired reads to profile intra-population genetic diversity (microdiversity) across whole genomes and compares microbial populations in a microdiversity-aware manner, greatly increasing the accuracy of genomic comparisons when benchmarked against existing methods. We use inStrain to profile &amp;gt;1,000 fecal metagenomes from newborn premature infants and find that siblings share significantly more strains than unrelated infants, although identical twins share no more strains than fraternal siblings. Infants born by cesarean section harbor Klebsiella with significantly higher nucleotide diversity than infants delivered vaginally, potentially reflecting acquisition from hospital rather than maternal microbiomes. Genomic loci that show diversity in individual infants include variants found between other infants, possibly reflecting inoculation from diverse hospital-associated sources. inStrain can be applied to any metagenomic dataset for microdiversity analysis and rigorous strain comparison. A metagenome analysis tool identifies microbial strains within and between populations.},
  archive      = {J_NBT},
  author       = {Olm, Matthew R. and Crits-Christoph, Alexander and Bouma-Gregson, Keith and Firek, Brian A. and Morowitz, Michael J. and Banfield, Jillian F.},
  doi          = {10.1038/s41587-020-00797-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {727-736},
  shortjournal = {Nature Biotech.},
  title        = {InStrain profiles population microdiversity from metagenomic data and sensitively detects shared microbial strains},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Treatment of influenza and SARS-CoV-2 infections via
mRNA-encoded cas13a in rodents. <em>NBT</em>, <em>39</em>(6), 717–726.
(<a href="https://doi.org/10.1038/s41587-021-00822-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cas13a has been used to target RNA viruses in cell culture, but efficacy has not been demonstrated in animal models. In this study, we used messenger RNA (mRNA)-encoded Cas13a for mitigating influenza virus A and severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) infection in mice and hamsters, respectively. We designed CRISPR RNAs (crRNAs) specific for PB1 and highly conserved regions of PB2 of influenza virus, and against the replicase and nucleocapsid genes of SARS-CoV-2, and selected the crRNAs that reduced viral RNA levels most efficiently in cell culture. We delivered polymer-formulated Cas13a mRNA and the validated guides to the respiratory tract using a nebulizer. In mice, Cas13a degraded influenza RNA in lung tissue efficiently when delivered after infection, whereas in hamsters, Cas13a delivery reduced SARS-CoV-2 replication and reduced symptoms. Our findings suggest that Cas13a-mediated targeting of pathogenic viruses can mitigate respiratory infections. Cas13a delivered to the lung of mice and hamsters inhibits replication of influenza virus and SARS-CoV-2.},
  archive      = {J_NBT},
  author       = {Blanchard, Emmeline L. and Vanover, Daryll and Bawage, Swapnil Subhash and Tiwari, Pooja Munnilal and Rotolo, Laura and Beyersdorf, Jared and Peck, Hannah E. and Bruno, Nicholas C. and Hincapie, Robert and Michel, Frank and Murray, Jackelyn and Sadhwani, Heena and Vanderheyden, Bob and Finn, M. G. and Brinton, Margo A. and Lafontaine, Eric R. and Hogan, Robert J. and Zurla, Chiara and Santangelo, Philip J.},
  doi          = {10.1038/s41587-021-00822-w},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {717-726},
  shortjournal = {Nature Biotech.},
  title        = {Treatment of influenza and SARS-CoV-2 infections via mRNA-encoded cas13a in rodents},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Hypertension delays viral clearance and exacerbates airway
hyperinflammation in patients with COVID-19. <em>NBT</em>,
<em>39</em>(6), 705–716. (<a
href="https://doi.org/10.1038/s41587-020-00796-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In coronavirus disease 2019 (COVID-19), hypertension and cardiovascular diseases are major risk factors for critical disease progression. However, the underlying causes and the effects of the main anti-hypertensive therapies—angiotensin-converting enzyme inhibitors (ACEIs) and angiotensin receptor blockers (ARBs)—remain unclear. Combining clinical data (n = 144) and single-cell sequencing data of airway samples (n = 48) with in vitro experiments, we observed a distinct inflammatory predisposition of immune cells in patients with hypertension that correlated with critical COVID-19 progression. ACEI treatment was associated with dampened COVID-19-related hyperinflammation and with increased cell intrinsic antiviral responses, whereas ARB treatment related to enhanced epithelial–immune cell interactions. Macrophages and neutrophils of patients with hypertension, in particular under ARB treatment, exhibited higher expression of the pro-inflammatory cytokines CCL3 and CCL4 and the chemokine receptor CCR1. Although the limited size of our cohort does not allow us to establish clinical efficacy, our data suggest that the clinical benefits of ACEI treatment in patients with COVID-19 who have hypertension warrant further investigation. Single-cell analysis reveals how anti-hypertensive drugs affect the risk of severe disease in patients with COVID-19 who have hypertension.},
  archive      = {J_NBT},
  author       = {Trump, Saskia and Lukassen, Soeren and Anker, Markus S. and Chua, Robert Lorenz and Liebig, Johannes and Thürmann, Loreen and Corman, Victor Max and Binder, Marco and Loske, Jennifer and Klasa, Christina and Krieger, Teresa and Hennig, Bianca P. and Messingschlager, Marey and Pott, Fabian and Kazmierski, Julia and Twardziok, Sven and Albrecht, Jan Philipp and Eils, Jürgen and Hadzibegovic, Sara and Lena, Alessia and Heidecker, Bettina and Bürgel, Thore and Steinfeldt, Jakob and Goffinet, Christine and Kurth, Florian and Witzenrath, Martin and Völker, Maria Theresa and Müller, Sarah Dorothea and Liebert, Uwe Gerd and Ishaque, Naveed and Kaderali, Lars and Sander, Leif-Erik and Drosten, Christian and Laudi, Sven and Eils, Roland and Conrad, Christian and Landmesser, Ulf and Lehmann, Irina},
  doi          = {10.1038/s41587-020-00796-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {705-716},
  shortjournal = {Nature Biotech.},
  title        = {Hypertension delays viral clearance and exacerbates airway hyperinflammation in patients with COVID-19},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Noncanonical open reading frames encode functional proteins
essential for cancer cell survival. <em>NBT</em>, <em>39</em>(6),
697–704. (<a href="https://doi.org/10.1038/s41587-020-00806-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although genomic analyses predict many noncanonical open reading frames (ORFs) in the human genome, it is unclear whether they encode biologically active proteins. Here we experimentally interrogated 553 candidates selected from noncanonical ORF datasets. Of these, 57 induced viability defects when knocked out in human cancer cell lines. Following ectopic expression, 257 showed evidence of protein expression and 401 induced gene expression changes. Clustered regularly interspaced short palindromic repeat (CRISPR) tiling and start codon mutagenesis indicated that their biological effects required translation as opposed to RNA-mediated effects. We found that one of these ORFs, G029442—renamed glycine-rich extracellular protein-1 (GREP1)—encodes a secreted protein highly expressed in breast cancer, and its knockout in 263 cancer cell lines showed preferential essentiality in breast cancer-derived lines. The secretome of GREP1-expressing cells has an increased abundance of the oncogenic cytokine GDF15, and GDF15 supplementation mitigated the growth-inhibitory effect of GREP1 knockout. Our experiments suggest that noncanonical ORFs can express biologically active proteins that are potential therapeutic targets. Noncanonical open reading frames are shown to be essential for cancer cell function.},
  archive      = {J_NBT},
  author       = {Prensner, John R. and Enache, Oana M. and Luria, Victor and Krug, Karsten and Clauser, Karl R. and Dempster, Joshua M. and Karger, Amir and Wang, Li and Stumbraite, Karolina and Wang, Vickie M. and Botta, Ginevra and Lyons, Nicholas J. and Goodale, Amy and Kalani, Zohra and Fritchman, Briana and Brown, Adam and Alan, Douglas and Green, Thomas and Yang, Xiaoping and Jaffe, Jacob D. and Roth, Jennifer A. and Piccioni, Federica and Kirschner, Marc W. and Ji, Zhe and Root, David E. and Golub, Todd R.},
  doi          = {10.1038/s41587-020-00806-2},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {697-704},
  shortjournal = {Nature Biotech.},
  title        = {Noncanonical open reading frames encode functional proteins essential for cancer cell survival},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep diversification of an AAV capsid protein by machine
learning. <em>NBT</em>, <em>39</em>(6), 691–696. (<a
href="https://doi.org/10.1038/s41587-020-00793-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern experimental technologies can assay large numbers of biological sequences, but engineered protein libraries rarely exceed the sequence diversity of natural protein families. Machine learning (ML) models trained directly on experimental data without biophysical modeling provide one route to accessing the full potential diversity of engineered proteins. Here we apply deep learning to design highly diverse adeno-associated virus 2 (AAV2) capsid protein variants that remain viable for packaging of a DNA payload. Focusing on a 28-amino acid segment, we generated 201,426 variants of the AAV2 wild-type (WT) sequence yielding 110,689 viable engineered capsids, 57,348 of which surpass the average diversity of natural AAV serotype sequences, with 12–29 mutations across this region. Even when trained on limited data, deep neural network models accurately predict capsid viability across diverse variants. This approach unlocks vast areas of functional but previously unreachable sequence space, with many potential applications for the generation of improved viral vectors and protein therapeutics. Viable AAV capsids are designed with a machine learning approach.},
  archive      = {J_NBT},
  author       = {Bryant, Drew H. and Bashir, Ali and Sinai, Sam and Jain, Nina K. and Ogden, Pierce J. and Riley, Patrick F. and Church, George M. and Colwell, Lucy J. and Kelsic, Eric D.},
  doi          = {10.1038/s41587-020-00793-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {691-696},
  shortjournal = {Nature Biotech.},
  title        = {Deep diversification of an AAV capsid protein by machine learning},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Super-resolution RNA imaging using a rhodamine-binding
aptamer with fast exchange kinetics. <em>NBT</em>, <em>39</em>(6),
686–690. (<a href="https://doi.org/10.1038/s41587-020-00794-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Overcoming limitations of previous fluorescent light-up RNA aptamers for super-resolution imaging, we present RhoBAST, an aptamer that binds a fluorogenic rhodamine dye with fast association and dissociation kinetics. Its intermittent fluorescence emission enables single-molecule localization microscopy with a resolution not limited by photobleaching. We use RhoBAST to image subcellular structures of RNA in live and fixed cells with about 10-nm localization precision and a high signal-to-noise ratio. A light-up RNA aptamer enables single-molecule localization microscopy with a resolution not limited by photobleaching.},
  archive      = {J_NBT},
  author       = {Sunbul, Murat and Lackner, Jens and Martin, Annabell and Englert, Daniel and Hacene, Benjamin and Grün, Franziska and Nienhaus, Karin and Nienhaus, G. Ulrich and Jäschke, Andres},
  doi          = {10.1038/s41587-020-00794-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {686-690},
  shortjournal = {Nature Biotech.},
  title        = {Super-resolution RNA imaging using a rhodamine-binding aptamer with fast exchange kinetics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). First rounders: Carl june. <em>NBT</em>, <em>39</em>(6),
685. (<a href="https://doi.org/10.1038/s41587-021-00952-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00952-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {685},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Carl june},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Forum: Optogenetics roundtable with deisseroth and
gradinaru. <em>NBT</em>, <em>39</em>(6), 685. (<a
href="https://doi.org/10.1038/s41587-021-00953-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00953-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {685},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Optogenetics roundtable with deisseroth and gradinaru},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Proteomics. <em>NBT</em>, <em>39</em>(6), 684. (<a
href="https://doi.org/10.1038/s41587-021-00958-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents for methods of proteomic screening, identification and analysis.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00958-9},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {684},
  shortjournal = {Nature Biotech.},
  title        = {Proteomics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Comprehensive analysis of 2.4 million patent-to-research
citations maps the biomedical innovation and translation landscape.
<em>NBT</em>, <em>39</em>(6), 678–683. (<a
href="https://doi.org/10.1038/s41587-021-00940-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A citation map connecting patents to biomedical publications provides insights that can be used to better evaluate productivity, diversity and translational impact.},
  archive      = {J_NBT},
  author       = {Manjunath, Anoop and Li, Hongyu and Song, Shuchen and Zhang, Zhixing and Liu, Shu and Kahrobai, Nathan and Gowda, Arya and Seffens, Angelina and Zou, James and Kumar, Ishan},
  doi          = {10.1038/s41587-021-00940-5},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {678-683},
  shortjournal = {Nature Biotech.},
  title        = {Comprehensive analysis of 2.4 million patent-to-research citations maps the biomedical innovation and translation landscape},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A call to arms against ultra-rare diseases. <em>NBT</em>,
<em>39</em>(6), 671–677. (<a
href="https://doi.org/10.1038/s41587-021-00945-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The not-for profit foundation n-Lorem seeks to provide experimental antisense oligonucleotide treatments to patients with ultra-rare disease free for their entire lifetime.},
  archive      = {J_NBT},
  author       = {Crooke, Stanley T.},
  doi          = {10.1038/s41587-021-00945-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {671-677},
  shortjournal = {Nature Biotech.},
  title        = {A call to arms against ultra-rare diseases},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Tracking cell lineages to improve research reproducibility.
<em>NBT</em>, <em>39</em>(6), 666–670. (<a
href="https://doi.org/10.1038/s41587-021-00928-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Zaaijer, Sophie and Groen, Simon C. and Sanjana, Neville E.},
  doi          = {10.1038/s41587-021-00928-1},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {666-670},
  shortjournal = {Nature Biotech.},
  title        = {Tracking cell lineages to improve research reproducibility},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The need for global access to biomedical innovations during
pandemics. <em>NBT</em>, <em>39</em>(6), 664–666. (<a
href="https://doi.org/10.1038/s41587-021-00942-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Gupta, Rajesh},
  doi          = {10.1038/s41587-021-00942-3},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {664-666},
  shortjournal = {Nature Biotech.},
  title        = {The need for global access to biomedical innovations during pandemics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mobilizing the private sector for responsible innovation in
neurotechnology. <em>NBT</em>, <em>39</em>(6), 661–664. (<a
href="https://doi.org/10.1038/s41587-021-00947-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Pfotenhauer, Sebastian M. and Frahm, Nina and Winickoff, David and Benrimoh, David and Illes, Judy and Marchant, Gary},
  doi          = {10.1038/s41587-021-00947-y},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {661-664},
  shortjournal = {Nature Biotech.},
  title        = {Mobilizing the private sector for responsible innovation in neurotechnology},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Voices of biotech leaders. <em>NBT</em>, <em>39</em>(6),
654–660. (<a href="https://doi.org/10.1038/s41587-021-00941-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Biotechnology asks a selection of leaders from across biotech to look at the future of the sector and make some predictions for the coming years.},
  archive      = {J_NBT},
  author       = {Bosley, Katrine and Casebourn, Charlotte and Chan, Priscilla and Chen, Janice and Chen, Michael and Church, George and Cumbers, John and de Wouters, Tomas and Dewey-Hagborg, Heather and Duportet, Xavier and Ene-Obong, Abasi and Elizondo, Arturo and Farrar, Jeremy and Gates, Bill and Gatto, Francesco and Giwa, Sebastian and Godec, Jernej and Gold, Silvia and LeProust, Emily and Lunshof, Jeantine and Martucci, Eddie and Heath, Michelle McMurray and Mellad, Jason and Oudova, Veronika and Oxman, Neri and Regev, Aviv and Richardson, Sarah and Scott, Christopher Thomas and Sherkow, Jake and Sibener, Leah and Tarragó, Teresa and Terry, Sharon and Venter, J. Craig and Wang, Spin and Wickramasekara, Sajith and Yadi, Hakim and Yang, Luhan and Zhao, Bowen},
  doi          = {10.1038/s41587-021-00941-4},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {654-660},
  shortjournal = {Nature Biotech.},
  title        = {Voices of biotech leaders},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Around the world in a month. <em>NBT</em>, <em>39</em>(6),
653. (<a href="https://doi.org/10.1038/s41587-021-00957-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00957-w},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {653},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Argentina first to market with drought-resistant GM wheat.
<em>NBT</em>, <em>39</em>(6), 652. (<a
href="https://doi.org/10.1038/s41587-021-00963-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00963-y},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {652},
  shortjournal = {Nature Biotech.},
  title        = {Argentina first to market with drought-resistant GM wheat},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021m). Questions swirl around failures of disease-modifying
huntington’s drugs. <em>NBT</em>, <em>39</em>(6), 650–652. (<a
href="https://doi.org/10.1038/s41587-021-00955-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Researchers hope that antisense oligo clinical trial failures in Huntington’s will shed light on mechanism and drug biodistribution and galvanize ongoing drug development efforts.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-021-00955-y},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {650-652},
  shortjournal = {Nature Biotech.},
  title        = {Questions swirl around failures of disease-modifying huntington’s drugs},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Microbiome test spots oral cancers. <em>NBT</em>,
<em>39</em>(6), 650. (<a
href="https://doi.org/10.1038/s41587-021-00964-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00964-x},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {650},
  shortjournal = {Nature Biotech.},
  title        = {Microbiome test spots oral cancers},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). COVID vaccines grow on leaves. <em>NBT</em>, <em>39</em>(6),
649. (<a href="https://doi.org/10.1038/s41587-021-00961-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00961-0},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {649},
  shortjournal = {Nature Biotech.},
  title        = {COVID vaccines grow on leaves},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Drugmakers go nuclear, continuing push into
radiopharmaceuticals. <em>NBT</em>, <em>39</em>(6), 647–649. (<a
href="https://doi.org/10.1038/s41587-021-00954-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With late-stage trial success, venture funding and next-generation alpha emitters on the way, targeted radiotherapy drugs are finally “getting the attention they deserve.”},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-021-00954-z},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {647-649},
  shortjournal = {Nature Biotech.},
  title        = {Drugmakers go nuclear, continuing push into radiopharmaceuticals},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ultra-precision medicine. <em>NBT</em>, <em>39</em>(6), 645.
(<a href="https://doi.org/10.1038/s41587-021-00967-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {New initiatives in N-of-1 drug development and clinical trial design offer the possibility of therapies for ultra-rare disease patients who have been long neglected by the drug industry.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00967-8},
  journal      = {Nature Biotechnology},
  month        = {6},
  number       = {6},
  pages        = {645},
  shortjournal = {Nature Biotech.},
  title        = {Ultra-precision medicine},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). People. <em>NBT</em>, <em>39</em>(5), 644. (<a
href="https://doi.org/10.1038/s41587-021-00923-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00923-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {644},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). First-quarter biotech job picture. <em>NBT</em>,
<em>39</em>(5), 643. (<a
href="https://doi.org/10.1038/s41587-021-00919-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-021-00919-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {643},
  shortjournal = {Nature Biotech.},
  title        = {First-quarter biotech job picture},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: ChIP-seq of plasma cell-free nucleosomes
identifies gene expression programs of the cells of origin.
<em>NBT</em>, <em>39</em>(5), 642. (<a
href="https://doi.org/10.1038/s41587-021-00831-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41587-021-00831-9.},
  archive      = {J_NBT},
  author       = {Sadeh, Ronen and Sharkia, Israa and Fialkoff, Gavriel and Rahat, Ayelet and Gutin, Jenia and Chappleboim, Alon and Nitzan, Mor and Fox-Fisher, Ilana and Neiman, Daniel and Meler, Guy and Kamari, Zahala and Yaish, Dayana and Peretz, Tamar and Hubert, Ayala and Cohen, Jonathan E. and Salah, Azzam and Temper, Mark and Grinshpun, Albert and Maoz, Myriam and Abu-Gazala, Samir and Ben Ya’acov, Ami and Shteyer, Eyal and Safadi, Rifaat and Kaplan, Tommy and Shemer, Ruth and Planer, David and Galun, Eithan and Glaser, Benjamin and Zick, Aviad and Dor, Yuval and Friedman, Nir},
  doi          = {10.1038/s41587-021-00831-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {642},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: ChIP-seq of plasma cell-free nucleosomes identifies gene expression programs of the cells of origin},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Author correction: ChIP-seq of plasma cell-free nucleosomes
identifies gene expression programs of the cells of origin.
<em>NBT</em>, <em>39</em>(5), 642. (<a
href="https://doi.org/10.1038/s41587-021-00853-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41587-021-00853-3.},
  archive      = {J_NBT},
  author       = {Sadeh, Ronen and Sharkia, Israa and Fialkoff, Gavriel and Rahat, Ayelet and Gutin, Jenia and Chappleboim, Alon and Nitzan, Mor and Fox-Fisher, Ilana and Neiman, Daniel and Meler, Guy and Kamari, Zahala and Yaish, Dayana and Peretz, Tamar and Hubert, Ayala and Cohen, Jonathan E. and Salah, Azzam and Temper, Mark and Grinshpun, Albert and Maoz, Myriam and Abu-Gazala, Samir and Ben Ya’acov, Ami and Shteyer, Eyal and Safadi, Rifaat and Kaplan, Tommy and Shemer, Ruth and Planer, David and Galun, Eithan and Glaser, Benjamin and Zick, Aviad and Dor, Yuval and Friedman, Nir},
  doi          = {10.1038/s41587-021-00853-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {642},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: ChIP-seq of plasma cell-free nucleosomes identifies gene expression programs of the cells of origin},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reimagining high-throughput profiling of reactive cysteines
for cell-based screening of large electrophile libraries. <em>NBT</em>,
<em>39</em>(5), 630–641. (<a
href="https://doi.org/10.1038/s41587-020-00778-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods used for measuring amino acid side-chain reactivity lack the throughput needed to screen large chemical libraries for interactions across the proteome. Here we redesigned the workflow for activity-based protein profiling of reactive cysteine residues by using a smaller desthiobiotin-based probe, sample multiplexing, reduced protein starting amounts and software to boost data acquisition in real time on the mass spectrometer. Our method, streamlined cysteine activity-based protein profiling (SLC-ABPP), achieved a 42-fold improvement in sample throughput, corresponding to profiling library members at a depth of &amp;gt;8,000 reactive cysteine sites at 18 min per compound. We applied it to identify proteome-wide targets of covalent inhibitors to mutant Kirsten rat sarcoma (KRAS)G12C and Bruton’s tyrosine kinase (BTK). In addition, we created a resource of cysteine reactivity to 285 electrophiles in three human cell lines, which includes &amp;gt;20,000 cysteines from &amp;gt;6,000 proteins per line. The goal of proteome-wide profiling of cysteine reactivity across thousand-member libraries under several cellular contexts is now within reach. An improved workflow enables a 42-fold higher throughput of activity-based protein profiling.},
  archive      = {J_NBT},
  author       = {Kuljanin, Miljan and Mitchell, Dylan C. and Schweppe, Devin K. and Gikandi, Ajami S. and Nusinow, David P. and Bulloch, Nathan J. and Vinogradova, Ekaterina V. and Wilson, David L. and Kool, Eric T. and Mancias, Joseph D. and Cravatt, Benjamin F. and Gygi, Steven P.},
  doi          = {10.1038/s41587-020-00778-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {630-641},
  shortjournal = {Nature Biotech.},
  title        = {Reimagining high-throughput profiling of reactive cysteines for cell-based screening of large electrophile libraries},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Quantifying the effect of experimental perturbations at
single-cell resolution. <em>NBT</em>, <em>39</em>(5), 619–629. (<a
href="https://doi.org/10.1038/s41587-020-00803-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods for comparing single-cell RNA sequencing datasets collected in multiple conditions focus on discrete regions of the transcriptional state space, such as clusters of cells. Here we quantify the effects of perturbations at the single-cell level using a continuous measure of the effect of a perturbation across the transcriptomic space. We describe this space as a manifold and develop a relative likelihood estimate of observing each cell in each of the experimental conditions using graph signal processing. This likelihood estimate can be used to identify cell populations specifically affected by a perturbation. We also develop vertex frequency clustering to extract populations of affected cells at the level of granularity that matches the perturbation response. The accuracy of our algorithm at identifying clusters of cells that are enriched or depleted in each condition is, on average, 57\% higher than the next-best-performing algorithm tested. Gene signatures derived from these clusters are more accurate than those of six alternative algorithms in ground truth comparisons. Matched treatment and control single-cell RNA sequencing samples are more accurately compared at the single-cell level.},
  archive      = {J_NBT},
  author       = {Burkhardt, Daniel B. and Stanley, Jay S. and Tong, Alexander and Perdigoto, Ana Luisa and Gigante, Scott A. and Herold, Kevan C. and Wolf, Guy and Giraldez, Antonio J. and van Dijk, David and Krishnaswamy, Smita},
  doi          = {10.1038/s41587-020-00803-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {619-629},
  shortjournal = {Nature Biotech.},
  title        = {Quantifying the effect of experimental perturbations at single-cell resolution},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Volumetric live cell imaging with three-dimensional
parallelized RESOLFT microscopy. <em>NBT</em>, <em>39</em>(5), 609–618.
(<a href="https://doi.org/10.1038/s41587-020-00779-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Elucidating the volumetric architecture of organelles and molecules inside cells requires microscopy methods with a sufficiently high spatial resolution in all three dimensions. Current methods are limited by insufficient resolving power along the optical axis, long recording times and photobleaching when applied to live cell imaging. Here, we present a 3D, parallelized, reversible, saturable/switchable optical fluorescence transition (3D pRESOLFT) microscope capable of delivering sub-80-nm 3D resolution in whole living cells. We achieved rapid (1–2 Hz) acquisition of large fields of view (~40 × 40 µm2) by highly parallelized image acquisition with an interference pattern that creates an array of 3D-confined and equally spaced intensity minima. This allowed us to reversibly turn switchable fluorescent proteins to dark states, leading to a targeted 3D confinement of fluorescence. We visualized the 3D organization and dynamics of organelles in living cells and volumetric structural alterations of synapses during plasticity in cultured hippocampal neurons. A super-resolution microscope images live cells in 3D with high speed, large fields of view and minimal photodamage.},
  archive      = {J_NBT},
  author       = {Bodén, Andreas and Pennacchietti, Francesca and Coceano, Giovanna and Damenti, Martina and Ratz, Michael and Testa, Ilaria},
  doi          = {10.1038/s41587-020-00779-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {609-618},
  shortjournal = {Nature Biotech.},
  title        = {Volumetric live cell imaging with three-dimensional parallelized RESOLFT microscopy},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Delineating copy number and clonal substructure in human
tumors from single-cell transcriptomes. <em>NBT</em>, <em>39</em>(5),
599–608. (<a href="https://doi.org/10.1038/s41587-020-00795-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell transcriptomic analysis is widely used to study human tumors. However, it remains challenging to distinguish normal cell types in the tumor microenvironment from malignant cells and to resolve clonal substructure within the tumor. To address these challenges, we developed an integrative Bayesian segmentation approach called copy number karyotyping of aneuploid tumors (CopyKAT) to estimate genomic copy number profiles at an average genomic resolution of 5 Mb from read depth in high-throughput single-cell RNA sequencing (scRNA-seq) data. We applied CopyKAT to analyze 46,501 single cells from 21 tumors, including triple-negative breast cancer, pancreatic ductal adenocarcinoma, anaplastic thyroid cancer, invasive ductal carcinoma and glioblastoma, to accurately (98\%) distinguish cancer cells from normal cell types. In three breast tumors, CopyKAT resolved clonal subpopulations that differed in the expression of cancer genes, such as KRAS, and signatures, including epithelial-to-mesenchymal transition, DNA repair, apoptosis and hypoxia. These data show that CopyKAT can aid in the analysis of scRNA-seq data in a variety of solid human tumors. Clonal subpopulations in human tumors are identified from single-cell RNA-seq data.},
  archive      = {J_NBT},
  author       = {Gao, Ruli and Bai, Shanshan and Henderson, Ying C. and Lin, Yiyun and Schalck, Aislyn and Yan, Yun and Kumar, Tapsi and Hu, Min and Sei, Emi and Davis, Alexander and Wang, Fang and Shaitelman, Simona F. and Wang, Jennifer Rui and Chen, Ken and Moulder, Stacy and Lai, Stephen Y. and Navin, Nicholas E.},
  doi          = {10.1038/s41587-020-00795-2},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {599-608},
  shortjournal = {Nature Biotech.},
  title        = {Delineating copy number and clonal substructure in human tumors from single-cell transcriptomes},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). ChIP-seq of plasma cell-free nucleosomes identifies gene
expression programs of the cells of origin. <em>NBT</em>,
<em>39</em>(5), 586–598. (<a
href="https://doi.org/10.1038/s41587-020-00775-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell-free DNA (cfDNA) in human plasma provides access to molecular information about the pathological processes in the organs or tumors from which it originates. These DNA fragments are derived from fragmented chromatin in dying cells and retain some of the cell-of-origin histone modifications. In this study, we applied chromatin immunoprecipitation of cell-free nucleosomes carrying active chromatin modifications followed by sequencing (cfChIP-seq) to 268 human samples. In healthy donors, we identified bone marrow megakaryocytes, but not erythroblasts, as major contributors to the cfDNA pool. In patients with a range of liver diseases, we showed that we can identify pathology-related changes in hepatocyte transcriptional programs. In patients with metastatic colorectal carcinoma, we detected clinically relevant and patient-specific information, including transcriptionally active human epidermal growth factor receptor 2 (HER2) amplifications. Altogether, cfChIP-seq, using low sequencing depth, provides systemic and genome-wide information and can inform diagnosis and facilitate interrogation of physiological and pathological processes using blood samples. Circulating cell-free DNA from patients is analyzed by ChIP-seq to reconstruct gene expression in human organs and tumors.},
  archive      = {J_NBT},
  author       = {Sadeh, Ronen and Sharkia, Israa and Fialkoff, Gavriel and Rahat, Ayelet and Gutin, Jenia and Chappleboim, Alon and Nitzan, Mor and Fox-Fisher, Ilana and Neiman, Daniel and Meler, Guy and Kamari, Zahala and Yaish, Dayana and Peretz, Tamar and Hubert, Ayala and Cohen, Jonathan E. and Salah, Azzam and Temper, Mark and Grinshpun, Albert and Maoz, Myriam and Abu-Gazala, Samir and Ben Ya’acov, Ami and Shteyer, Eyal and Safadi, Rifaat and Kaplan, Tommy and Shemer, Ruth and Planer, David and Galun, Eithan and Glaser, Benjamin and Zick, Aviad and Dor, Yuval and Friedman, Nir},
  doi          = {10.1038/s41587-020-00775-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {586-598},
  shortjournal = {Nature Biotech.},
  title        = {ChIP-seq of plasma cell-free nucleosomes identifies gene expression programs of the cells of origin},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CheckV assesses the quality and completeness of
metagenome-assembled viral genomes. <em>NBT</em>, <em>39</em>(5),
578–585. (<a href="https://doi.org/10.1038/s41587-020-00774-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Millions of new viral sequences have been identified from metagenomes, but the quality and completeness of these sequences vary considerably. Here we present CheckV, an automated pipeline for identifying closed viral genomes, estimating the completeness of genome fragments and removing flanking host regions from integrated proviruses. CheckV estimates completeness by comparing sequences with a large database of complete viral genomes, including 76,262 identified from a systematic search of publicly available metagenomes, metatranscriptomes and metaviromes. After validation on mock datasets and comparison to existing methods, we applied CheckV to large and diverse collections of metagenome-assembled viral sequences, including IMG/VR and the Global Ocean Virome. This revealed 44,652 high-quality viral genomes (that is, &amp;gt;90\% complete), although the vast majority of sequences were small fragments, which highlights the challenge of assembling viral genomes from short-read metagenomes. Additionally, we found that removal of host contamination substantially improved the accurate identification of auxiliary metabolic genes and interpretation of viral-encoded functions. The quality of viral genomes assembled from metagenome data is assessed by CheckV.},
  archive      = {J_NBT},
  author       = {Nayfach, Stephen and Camargo, Antonio Pedro and Schulz, Frederik and Eloe-Fadrosh, Emiley and Roux, Simon and Kyrpides, Nikos C.},
  doi          = {10.1038/s41587-020-00774-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {578-585},
  shortjournal = {Nature Biotech.},
  title        = {CheckV assesses the quality and completeness of metagenome-assembled viral genomes},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Targeting herpes simplex virus with CRISPR–Cas9 cures
herpetic stromal keratitis in mice. <em>NBT</em>, <em>39</em>(5),
567–577. (<a href="https://doi.org/10.1038/s41587-020-00781-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Herpes simplex virus type 1 (HSV-1) is a leading cause of infectious blindness. Current treatments for HSV-1 do not eliminate the virus from the site of infection or latent reservoirs in the trigeminal ganglia. Here, we target HSV-1 genomes directly using mRNA-carrying lentiviral particles that simultaneously deliver SpCas9 mRNA and viral-gene-targeting guide RNAs (designated HSV-1-erasing lentiviral particles, termed HELP). We show that HELP efficiently blocks HSV-1 replication and the occurrence of herpetic stromal keratitis (HSK) in three different infection models. HELP was capable of eliminating the viral reservoir via retrograde transport from corneas to trigeminal ganglia. Additionally, HELP inhibited viral replication in human-derived corneas without causing off-target effects, as determined by whole-genome sequencing. These results support the potential clinical utility of HELP for treating refractory HSK. CRISPR–Cas9 eliminates herpes simplex virus both in the corneal infection site and in the viral reservoir of the trigeminal ganglia.},
  archive      = {J_NBT},
  author       = {Yin, Di and Ling, Sikai and Wang, Dawei and Dai, Yao and Jiang, Hao and Zhou, Xujiao and Paludan, Soren R. and Hong, Jiaxu and Cai, Yujia},
  doi          = {10.1038/s41587-020-00781-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {567-577},
  shortjournal = {Nature Biotech.},
  title        = {Targeting herpes simplex virus with CRISPR–Cas9 cures herpetic stromal keratitis in mice},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A five-transgene cassette confers broad-spectrum resistance
to a fungal rust pathogen in wheat. <em>NBT</em>, <em>39</em>(5),
561–566. (<a href="https://doi.org/10.1038/s41587-020-00770-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Breeding wheat with durable resistance to the fungal pathogen Puccinia graminis f. sp. tritici (Pgt), a major threat to cereal production, is challenging due to the rapid evolution of pathogen virulence. Increased durability and broad-spectrum resistance can be achieved by introducing more than one resistance gene, but combining numerous unlinked genes by breeding is laborious. Here we generate polygenic Pgt resistance by introducing a transgene cassette of five resistance genes into bread wheat as a single locus and show that at least four of the five genes are functional. These wheat lines are resistant to aggressive and highly virulent Pgt isolates from around the world and show very high levels of resistance in the field. The simple monogenic inheritance of this multigene locus greatly simplifies its use in breeding. However, a new Pgt isolate with virulence to several genes at this locus suggests gene stacks will need strategic deployment to maintain their effectiveness. Combining fungal-resistance genes into a single cassette enables the generation of highly resistant wheat lines.},
  archive      = {J_NBT},
  author       = {Luo, Ming and Xie, Liqiong and Chakraborty, Soma and Wang, Aihua and Matny, Oadi and Jugovich, Michelle and Kolmer, James A. and Richardson, Terese and Bhatt, Dhara and Hoque, Mohammad and Patpour, Mehran and Sørensen, Chris and Ortiz, Diana and Dodds, Peter and Steuernagel, Burkhard and Wulff, Brande B. H. and Upadhyaya, Narayana M. and Mago, Rohit and Periyannan, Sambasivam and Lagudah, Evans and Freedman, Roger and Lynne Reuber, T. and Steffenson, Brian J. and Ayliffe, Michael},
  doi          = {10.1038/s41587-020-00770-x},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {561-566},
  shortjournal = {Nature Biotech.},
  title        = {A five-transgene cassette confers broad-spectrum resistance to a fungal rust pathogen in wheat},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Improved metagenome binning and assembly using deep
variational autoencoders. <em>NBT</em>, <em>39</em>(5), 555–560. (<a
href="https://doi.org/10.1038/s41587-020-00777-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite recent advances in metagenomic binning, reconstruction of microbial species from metagenomics data remains challenging. Here we develop variational autoencoders for metagenomic binning (VAMB), a program that uses deep variational autoencoders to encode sequence coabundance and k-mer distribution information before clustering. We show that a variational autoencoder is able to integrate these two distinct data types without any previous knowledge of the datasets. VAMB outperforms existing state-of-the-art binners, reconstructing 29–98\% and 45\% more near-complete (NC) genomes on simulated and real data, respectively. Furthermore, VAMB is able to separate closely related strains up to 99.5\% average nucleotide identity (ANI), and reconstructed 255 and 91 NC Bacteroides vulgatus and Bacteroides dorei sample-specific genomes as two distinct clusters from a dataset of 1,000 human gut microbiome samples. We use 2,606 NC bins from this dataset to show that species of the human gut microbiome have different geographical distribution patterns. VAMB can be run on standard hardware and is freely available at https://github.com/RasmussenLab/vamb . Metagenomics data are resolved into their constituent genomes using a new deep learning method.},
  archive      = {J_NBT},
  author       = {Nissen, Jakob Nybo and Johansen, Joachim and Allesøe, Rosa Lundbye and Sønderby, Casper Kaae and Armenteros, Jose Juan Almagro and Grønbech, Christopher Heje and Jensen, Lars Juhl and Nielsen, Henrik Bjørn and Petersen, Thomas Nordahl and Winther, Ole and Rasmussen, Simon},
  doi          = {10.1038/s41587-020-00777-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {555-560},
  shortjournal = {Nature Biotech.},
  title        = {Improved metagenome binning and assembly using deep variational autoencoders},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Left-handed DNA-PAINT for improved super-resolution imaging
in the nucleus. <em>NBT</em>, <em>39</em>(5), 551–554. (<a
href="https://doi.org/10.1038/s41587-020-00753-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA point accumulation in nanoscale topography (DNA-PAINT) increases the resolution and multiplexing capabilities of super-resolution imaging, but cellular DNA interferes with DNA–DNA hybridization between target and probe in the nucleus. Here, we introduce left-handed DNA (L-DNA) oligomers that do not hybridize to natural right-handed DNA (R-DNA) and demonstrate that L-DNA-PAINT has the same specificity and multiplexing capability as R-DNA-PAINT, but improves the imaging of nuclear targets by substantially reducing background signal. Left-handed DNA probes improve super-resolution DNA-PAINT imaging in the nucleus by reducing background.},
  archive      = {J_NBT},
  author       = {Geertsema, H. J. and Aimola, G. and Fabricius, V. and Fuerste, J. P. and Kaufer, B. B. and Ewers, H.},
  doi          = {10.1038/s41587-020-00753-y},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {551-554},
  shortjournal = {Nature Biotech.},
  title        = {Left-handed DNA-PAINT for improved super-resolution imaging in the nucleus},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). First rounders: Craig mello. <em>NBT</em>, <em>39</em>(5),
550. (<a href="https://doi.org/10.1038/s41587-021-00914-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00914-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {550},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Craig mello},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MRNA vaccines. <em>NBT</em>, <em>39</em>(5), 549. (<a
href="https://doi.org/10.1038/s41587-021-00924-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents for the composition, preparation and therapeutic use of mRNA vaccines.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00924-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {549},
  shortjournal = {Nature Biotech.},
  title        = {MRNA vaccines},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A network analysis of COVID-19 mRNA vaccine patents.
<em>NBT</em>, <em>39</em>(5), 546–548. (<a
href="https://doi.org/10.1038/s41587-021-00912-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A preliminary network analysis highlights the complex intellectual property landscape behind mRNA-based COVID-19 vaccines.},
  archive      = {J_NBT},
  author       = {Gaviria, Mario and Kilic, Burcu},
  doi          = {10.1038/s41587-021-00912-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {546-548},
  shortjournal = {Nature Biotech.},
  title        = {A network analysis of COVID-19 mRNA vaccine patents},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). 1Q21—thirst for preclinical biotechs unquenched.
<em>NBT</em>, <em>39</em>(5), 538–539. (<a
href="https://doi.org/10.1038/s41587-021-00918-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-00918-3},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {538-539},
  shortjournal = {Nature Biotech.},
  title        = {1Q21—thirst for preclinical biotechs unquenched},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Drug pipeline 1Q21—the old and the new. <em>NBT</em>,
<em>39</em>(5), 536–537. (<a
href="https://doi.org/10.1038/s41587-021-00917-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-00917-4},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {536-537},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 1Q21—the old and the new},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Around the world in a month. <em>NBT</em>, <em>39</em>(5),
535. (<a href="https://doi.org/10.1038/s41587-021-00921-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00921-8},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {535},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Barista milk: Climate-friendly and still dairy.
<em>NBT</em>, <em>39</em>(5), 534. (<a
href="https://doi.org/10.1038/s41587-021-00930-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00930-7},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {534},
  shortjournal = {Nature Biotech.},
  title        = {Barista milk: Climate-friendly and still dairy},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). COVID-19 testing turns to t cells. <em>NBT</em>,
<em>39</em>(5), 533–534. (<a
href="https://doi.org/10.1038/s41587-021-00920-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A diagnostic test based on sequencing long-lived SARS-CoV-2–specific memory T cells provides a complement to antibody testing for determining previous exposure to SARS-CoV-2.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-021-00920-9},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {533-534},
  shortjournal = {Nature Biotech.},
  title        = {COVID-19 testing turns to t cells},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). What two US counties will tell us about COVID-19.
<em>NBT</em>, <em>39</em>(5), 533. (<a
href="https://doi.org/10.1038/s41587-021-00931-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00931-6},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {533},
  shortjournal = {Nature Biotech.},
  title        = {What two US counties will tell us about COVID-19},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Armyworm meets friendly moth. <em>NBT</em>, <em>39</em>(5),
532. (<a href="https://doi.org/10.1038/s41587-021-00932-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-021-00932-5},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {532},
  shortjournal = {Nature Biotech.},
  title        = {Armyworm meets friendly moth},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). First CAR-t therapy to target BCMA gets FDA nod.
<em>NBT</em>, <em>39</em>(5), 531. (<a
href="https://doi.org/10.1038/s41587-021-00929-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00929-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {531},
  shortjournal = {Nature Biotech.},
  title        = {First CAR-T therapy to target BCMA gets FDA nod},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The viral arms race. <em>NBT</em>, <em>39</em>(5), 527. (<a
href="https://doi.org/10.1038/s41587-021-00937-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To keep ahead of an increasing number of SARS-CoV-2 variants, pathogen surveillance and testing must become a joined-up global, rather than local, endeavor.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00937-0},
  journal      = {Nature Biotechnology},
  month        = {5},
  number       = {5},
  pages        = {527},
  shortjournal = {Nature Biotech.},
  title        = {The viral arms race},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Slivers of the spectrum. <em>NBT</em>, <em>39</em>(5),
540–545. (<a href="https://doi.org/10.1038/s41587-021-00913-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale genomic studies are reinvigorating interest in a small group of molecularly defined autism-associated disorders and spurring renewed interest in genetic therapies.},
  archive      = {J_NBT},
  author       = {Branca, Malorye},
  doi          = {10.1038/s41587-021-00913-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {5},
  pages        = {540-545},
  shortjournal = {Nature Biotech.},
  title        = {Slivers of the spectrum},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fecal microbiota potentiate checkpoint inhibitors, unleash
microbiome startups. <em>NBT</em>, <em>39</em>(5), 529–532. (<a
href="https://doi.org/10.1038/d41587-021-00002-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbiome companies are scrambling for a piece of the action as checkpoint inhibitors combined with fecal microbiome transplants enhance patient responses to immunotherapy. Microbiome companies are scrambling for a piece of the action as checkpoint inhibitors combined with fecal microbiome transplants enhance patient responses to immunotherapy.},
  archive      = {J_NBT},
  author       = {Zipkin, Mark},
  doi          = {10.1038/d41587-021-00002-w},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {5},
  pages        = {529-532},
  shortjournal = {Nature Biotech.},
  title        = {Fecal microbiota potentiate checkpoint inhibitors, unleash microbiome startups},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). People. <em>NBT</em>, <em>39</em>(4), 526. (<a
href="https://doi.org/10.1038/s41587-021-00886-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00886-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {526},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Work–life integration survey addresses the needs of women
working in life sciences. <em>NBT</em>, <em>39</em>(4), 523–525. (<a
href="https://doi.org/10.1038/s41587-021-00882-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Addressing the needs of women in the life science workforce can improve work–life integration and contribute to a more diverse and inclusive world economy.},
  archive      = {J_NBT},
  author       = {Hockberger, Philip and Valdes, Kayla M. and Bosc, Jessica and Iadicicco, Lisa and Georganopoulou, Dimitra},
  doi          = {10.1038/s41587-021-00882-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {523-525},
  shortjournal = {Nature Biotech.},
  title        = {Work–life integration survey addresses the needs of women working in life sciences},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Author correction: A genomic catalog of earth’s
microbiomes. <em>NBT</em>, <em>39</em>(4), 521. (<a
href="https://doi.org/10.1038/s41587-021-00898-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41587-021-00898-4.},
  archive      = {J_NBT},
  author       = {Nayfach, Stephen and Roux, Simon and Seshadri, Rekha and Udwary, Daniel and Varghese, Neha and Schulz, Frederik and Wu, Dongying and Paez-Espino, David and Chen, I-Min and Huntemann, Marcel and Palaniappan, Krishna and Ladau, Joshua and Mukherjee, Supratim and Reddy, T. B. K. and Nielsen, Torben and Kirton, Edward and Faria, José P. and Edirisinghe, Janaka N. and Henry, Christopher S. and Jungbluth, Sean P. and Chivian, Dylan and Dehal, Paramvir and Wood-Charlson, Elisha M. and Arkin, Adam P. and Tringe, Susannah G. and Visel, Axel and Woyke, Tanja and Mouncey, Nigel J. and Ivanova, Natalia N. and Kyrpides, Nikos C. and Eloe-Fadrosh, Emiley A.},
  doi          = {10.1038/s41587-021-00898-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {521},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: A genomic catalog of earth’s microbiomes},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Publisher correction: A genomic catalog of earth’s
microbiomes. <em>NBT</em>, <em>39</em>(4), 520. (<a
href="https://doi.org/10.1038/s41587-020-00769-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Nayfach, Stephen and Roux, Simon and Seshadri, Rekha and Udwary, Daniel and Varghese, Neha and Schulz, Frederik and Wu, Dongying and Paez-Espino, David and Chen, I-Min and Huntemann, Marcel and Palaniappan, Krishna and Ladau, Joshua and Mukherjee, Supratim and Reddy, T. B. K. and Nielsen, Torben and Kirton, Edward and Faria, José P. and Edirisinghe, Janaka N. and Henry, Christopher S. and Jungbluth, Sean P. and Chivian, Dylan and Dehal, Paramvir and Wood-Charlson, Elisha M. and Arkin, Adam P. and Tringe, Susannah G. and Visel, Axel and Woyke, Tanja and Mouncey, Nigel J. and Ivanova, Natalia N. and Kyrpides, Nikos C. and Eloe-Fadrosh, Emiley A.},
  doi          = {10.1038/s41587-020-00769-4},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {520},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: A genomic catalog of earth’s microbiomes},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Publisher correction: Determination of isoform-specific RNA
structure with nanopore long reads. <em>NBT</em>, <em>39</em>(4), 520.
(<a href="https://doi.org/10.1038/s41587-021-00889-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41587-021-00889-5.},
  archive      = {J_NBT},
  author       = {Aw, Jong Ghut Ashley and Lim, Shaun W. and Wang, Jia Xu and Lambert, Finnlay R. P. and Tan, Wen Ting and Shen, Yang and Zhang, Yu and Kaewsapsak, Pornchai and Li, Chenhao and Ng, Sarah B. and Vardy, Leah A. and Tan, Meng How and Nagarajan, Niranjan and Wan, Yue},
  doi          = {10.1038/s41587-021-00889-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {520},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Determination of isoform-specific RNA structure with nanopore long reads},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A comprehensive library of human transcription factors for
cell fate engineering. <em>NBT</em>, <em>39</em>(4), 510–519. (<a
href="https://doi.org/10.1038/s41587-020-0742-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human pluripotent stem cells (hPSCs) offer an unprecedented opportunity to model diverse cell types and tissues. To enable systematic exploration of the programming landscape mediated by transcription factors (TFs), we present the Human TFome, a comprehensive library containing 1,564 TF genes and 1,732 TF splice isoforms. By screening the library in three hPSC lines, we discovered 290 TFs, including 241 that were previously unreported, that induce differentiation in 4 days without alteration of external soluble or biomechanical cues. We used four of the hits to program hPSCs into neurons, fibroblasts, oligodendrocytes and vascular endothelial-like cells that have molecular and functional similarity to primary cells. Our cell-autonomous approach enabled parallel programming of hPSCs into multiple cell types simultaneously. We also demonstrated orthogonal programming by including oligodendrocyte-inducible hPSCs with unmodified hPSCs to generate cerebral organoids, which expedited in situ myelination. Large-scale combinatorial screening of the Human TFome will complement other strategies for cell engineering based on developmental biology and computational systems biology. A library of human transcription factor genes is screened for differentiation of human pluripotent stem cells.},
  archive      = {J_NBT},
  author       = {Ng, Alex H. M. and Khoshakhlagh, Parastoo and Rojo Arias, Jesus Eduardo and Pasquini, Giovanni and Wang, Kai and Swiersy, Anka and Shipman, Seth L. and Appleton, Evan and Kiaee, Kiavash and Kohman, Richie E. and Vernet, Andyna and Dysart, Matthew and Leeper, Kathleen and Saylor, Wren and Huang, Jeremy Y. and Graveline, Amanda and Taipale, Jussi and Hill, David E. and Vidal, Marc and Melero-Martin, Juan M. and Busskamp, Volker and Church, George M.},
  doi          = {10.1038/s41587-020-0742-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {510-519},
  shortjournal = {Nature Biotech.},
  title        = {A comprehensive library of human transcription factors for cell fate engineering},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). A genomic catalog of earth’s microbiomes. <em>NBT</em>,
<em>39</em>(4), 499–509. (<a
href="https://doi.org/10.1038/s41587-020-0718-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The reconstruction of bacterial and archaeal genomes from shotgun metagenomes has enabled insights into the ecology and evolution of environmental and host-associated microbiomes. Here we applied this approach to &amp;gt;10,000 metagenomes collected from diverse habitats covering all of Earth’s continents and oceans, including metagenomes from human and animal hosts, engineered environments, and natural and agricultural soils, to capture extant microbial, metabolic and functional potential. This comprehensive catalog includes 52,515 metagenome-assembled genomes representing 12,556 novel candidate species-level operational taxonomic units spanning 135 phyla. The catalog expands the known phylogenetic diversity of bacteria and archaea by 44\% and is broadly available for streamlined comparative analyses, interactive exploration, metabolic modeling and bulk download. We demonstrate the utility of this collection for understanding secondary-metabolite biosynthetic potential and for resolving thousands of new host linkages to uncultivated viruses. This resource underscores the value of genome-centric approaches for revealing genomic properties of uncultivated microorganisms that affect ecosystem processes. Cataloging microbial genomes from Earth’s environments expands the known phylogenetic diversity of bacteria and archaea.},
  archive      = {J_NBT},
  author       = {Nayfach, Stephen and Roux, Simon and Seshadri, Rekha and Udwary, Daniel and Varghese, Neha and Schulz, Frederik and Wu, Dongying and Paez-Espino, David and Chen, I-Min and Huntemann, Marcel and Palaniappan, Krishna and Ladau, Joshua and Mukherjee, Supratim and Reddy, T. B. K. and Nielsen, Torben and Kirton, Edward and Faria, José P. and Edirisinghe, Janaka N. and Henry, Christopher S. and Jungbluth, Sean P. and Chivian, Dylan and Dehal, Paramvir and Wood-Charlson, Elisha M. and Arkin, Adam P. and Tringe, Susannah G. and Visel, Axel and Woyke, Tanja and Mouncey, Nigel J. and Ivanova, Natalia N. and Kyrpides, Nikos C. and Eloe-Fadrosh, Emiley A.},
  doi          = {10.1038/s41587-020-0718-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {499-509},
  shortjournal = {Nature Biotech.},
  title        = {A genomic catalog of earth’s microbiomes},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Identification of highly selective covalent inhibitors by
phage display. <em>NBT</em>, <em>39</em>(4), 490–498. (<a
href="https://doi.org/10.1038/s41587-020-0733-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecules that covalently bind macromolecular targets have found widespread applications as activity-based probes and as irreversibly binding drugs. However, the general reactivity of the electrophiles needed for covalent bond formation makes control of selectivity difficult. There is currently no rapid, unbiased screening method to identify new classes of covalent inhibitors from highly diverse pools of candidate molecules. Here we describe a phage display method to directly screen for ligands that bind to protein targets through covalent bond formation. This approach makes use of a reactive linker to form cyclic peptides on the phage surface while simultaneously introducing an electrophilic ‘warhead’ to covalently react with a nucleophile on the target. Using this approach, we identified cyclic peptides that irreversibly inhibited a cysteine protease and a serine hydrolase with nanomolar potency and exceptional specificity. This approach should enable rapid, unbiased screening to identify new classes of highly selective covalent inhibitors for diverse molecular targets. Covalent inhibitors with high selectivity are rapidly identified by phage display.},
  archive      = {J_NBT},
  author       = {Chen, Shiyu and Lovell, Scott and Lee, Sumin and Fellner, Matthias and Mace, Peter D. and Bogyo, Matthew},
  doi          = {10.1038/s41587-020-0733-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {490-498},
  shortjournal = {Nature Biotech.},
  title        = {Identification of highly selective covalent inhibitors by phage display},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CRISPR RNA-guided integrases for high-efficiency,
multiplexed bacterial genome engineering. <em>NBT</em>, <em>39</em>(4),
480–489. (<a href="https://doi.org/10.1038/s41587-020-00745-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Existing technologies for site-specific integration of kilobase-sized DNA sequences in bacteria are limited by low efficiency, a reliance on recombination, the need for multiple vectors, and challenges in multiplexing. To address these shortcomings, we introduce a substantially improved version of our previously reported Tn7-like transposon from Vibrio cholerae, which uses a Type I-F CRISPR–Cas system for programmable, RNA-guided transposition. The optimized insertion of transposable elements by guide RNA–assisted targeting (INTEGRATE) system achieves highly accurate and marker-free DNA integration of up to 10 kilobases at ~100\% efficiency in bacteria. Using multi-spacer CRISPR arrays, we achieved simultaneous multiplexed insertions in three genomic loci and facile, multi-loci deletions by combining orthogonal integrases and recombinases. Finally, we demonstrated robust function in biomedically and industrially relevant bacteria and achieved target- and species-specific integration in a complex bacterial community. This work establishes INTEGRATE as a versatile tool for multiplexed, kilobase-scale genome engineering. Optimized RNA-guided transposons efficiently integrate large DNA sequences in multiple bacterial species.},
  archive      = {J_NBT},
  author       = {Vo, Phuc Leo H. and Ronda, Carlotta and Klompe, Sanne E. and Chen, Ethan E. and Acree, Christopher and Wang, Harris H. and Sternberg, Samuel H.},
  doi          = {10.1038/s41587-020-00745-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {480-489},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR RNA-guided integrases for high-efficiency, multiplexed bacterial genome engineering},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MPL resolves genetic linkage in fitness inference from
complex evolutionary histories. <em>NBT</em>, <em>39</em>(4), 472–479.
(<a href="https://doi.org/10.1038/s41587-020-0737-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic linkage causes the fate of new mutations in a population to be contingent on the genetic background on which they appear. This makes it challenging to identify how individual mutations affect fitness. To overcome this challenge, we developed marginal path likelihood (MPL), a method to infer selection from evolutionary histories that resolves genetic linkage. Validation on real and simulated data sets shows that MPL is fast and accurate, outperforming existing inference approaches. We found that resolving linkage is crucial for accurately quantifying selection in complex evolving populations, which we demonstrate through a quantitative analysis of intrahost HIV-1 evolution using multiple patient data sets. Linkage effects generated by variants that sweep rapidly through the population are particularly strong, extending far across the genome. Taken together, our results argue for the importance of resolving linkage in studies of natural selection. A new method models the influence of genetic background on the fitness effects of mutations.},
  archive      = {J_NBT},
  author       = {Sohail, Muhammad Saqib and Louie, Raymond H. Y. and McKay, Matthew R. and Barton, John P.},
  doi          = {10.1038/s41587-020-0737-3},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {472-479},
  shortjournal = {Nature Biotech.},
  title        = {MPL resolves genetic linkage in fitness inference from complex evolutionary histories},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Systematic classification of unknown metabolites using
high-resolution fragmentation mass spectra. <em>NBT</em>,
<em>39</em>(4), 462–471. (<a
href="https://doi.org/10.1038/s41587-020-0740-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metabolomics using nontargeted tandem mass spectrometry can detect thousands of molecules in a biological sample. However, structural molecule annotation is limited to structures present in libraries or databases, restricting analysis and interpretation of experimental data. Here we describe CANOPUS (class assignment and ontology prediction using mass spectrometry), a computational tool for systematic compound class annotation. CANOPUS uses a deep neural network to predict 2,497 compound classes from fragmentation spectra, including all biologically relevant classes. CANOPUS explicitly targets compounds for which neither spectral nor structural reference data are available and predicts classes lacking tandem mass spectrometry training data. In evaluation using reference data, CANOPUS reached very high prediction performance (average accuracy of 99.7\% in cross-validation) and outperformed four baseline methods. We demonstrate the broad utility of CANOPUS by investigating the effect of microbial colonization in the mouse digestive system, through analysis of the chemodiversity of different Euphorbia plants and regarding the discovery of a marine natural product, revealing biological insights at the compound class level. Unknown metabolites are classified from mass spectrometry data.},
  archive      = {J_NBT},
  author       = {Dührkop, Kai and Nothias, Louis-Félix and Fleischauer, Markus and Reher, Raphael and Ludwig, Marcus and Hoffmann, Martin A. and Petras, Daniel and Gerwick, William H. and Rousu, Juho and Dorrestein, Pieter C. and Böcker, Sebastian},
  doi          = {10.1038/s41587-020-0740-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {462-471},
  shortjournal = {Nature Biotech.},
  title        = {Systematic classification of unknown metabolites using high-resolution fragmentation mass spectra},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Massively parallel single-cell mitochondrial DNA genotyping
and chromatin profiling. <em>NBT</em>, <em>39</em>(4), 451–461. (<a
href="https://doi.org/10.1038/s41587-020-0645-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Natural mitochondrial DNA (mtDNA) mutations enable the inference of clonal relationships among cells. mtDNA can be profiled along with measures of cell state, but has not yet been combined with the massively parallel approaches needed to tackle the complexity of human tissue. Here, we introduce a high-throughput, droplet-based mitochondrial single-cell assay for transposase-accessible chromatin with sequencing (scATAC-seq), a method that combines high-confidence mtDNA mutation calling in thousands of single cells with their concomitant high-quality accessible chromatin profile. This enables the inference of mtDNA heteroplasmy, clonal relationships, cell state and accessible chromatin variation in individual cells. We reveal single-cell variation in heteroplasmy of a pathologic mtDNA variant, which we associate with intra-individual chromatin variability and clonal evolution. We clonally trace thousands of cells from cancers, linking epigenomic variability to subclonal evolution, and infer cellular dynamics of differentiating hematopoietic cells in vitro and in vivo. Taken together, our approach enables the study of cellular population dynamics and clonal properties in vivo. Combining droplet-based ATAC-seq and mitochondrial DNA sequencing reveals clonal variation in human cells and tissues.},
  archive      = {J_NBT},
  author       = {Lareau, Caleb A. and Ludwig, Leif S. and Muus, Christoph and Gohil, Satyen H. and Zhao, Tongtong and Chiang, Zachary and Pelka, Karin and Verboon, Jeffrey M. and Luo, Wendy and Christian, Elena and Rosebrock, Daniel and Getz, Gad and Boland, Genevieve M. and Chen, Fei and Buenrostro, Jason D. and Hacohen, Nir and Wu, Catherine J. and Aryee, Martin J. and Regev, Aviv and Sankaran, Vijay G.},
  doi          = {10.1038/s41587-020-0645-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {451-461},
  shortjournal = {Nature Biotech.},
  title        = {Massively parallel single-cell mitochondrial DNA genotyping and chromatin profiling},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Readfish enables targeted nanopore sequencing of
gigabase-sized genomes. <em>NBT</em>, <em>39</em>(4), 442–450. (<a
href="https://doi.org/10.1038/s41587-020-00746-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nanopore sequencers can be used to selectively sequence certain DNA molecules in a pool by reversing the voltage across individual nanopores to reject specific sequences, enabling enrichment and depletion to address biological questions. Previously, we achieved this using dynamic time warping to map the signal to a reference genome, but the method required substantial computational resources and did not scale to gigabase-sized references. Here we overcome this limitation by using graphical processing unit (GPU) base-calling. We show enrichment of specific chromosomes from the human genome and of low-abundance organisms in mixed populations without a priori knowledge of sample composition. Finally, we enrich targeted panels comprising 25,600 exons from 10,000 human genes and 717 genes implicated in cancer, identifying PML–RARA fusions in the NB4 cell line in &amp;lt;15 h sequencing. These methods can be used to efficiently screen any target panel of genes without specialized sample preparation using any computer and a suitable GPU. Our toolkit, readfish, is available at https://www.github.com/looselab/readfish . A nanopore sequencer achieves selective sequencing of any region in the human genome.},
  archive      = {J_NBT},
  author       = {Payne, Alexander and Holmes, Nadine and Clarke, Thomas and Munro, Rory and Debebe, Bisrat J. and Loose, Matthew},
  doi          = {10.1038/s41587-020-00746-x},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {442-450},
  shortjournal = {Nature Biotech.},
  title        = {Readfish enables targeted nanopore sequencing of gigabase-sized genomes},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Targeted nanopore sequencing by real-time mapping of raw
electrical signal with UNCALLED. <em>NBT</em>, <em>39</em>(4), 431–441.
(<a href="https://doi.org/10.1038/s41587-020-0731-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Conventional targeted sequencing methods eliminate many of the benefits of nanopore sequencing, such as the ability to accurately detect structural variants or epigenetic modifications. The ReadUntil method allows nanopore devices to selectively eject reads from pores in real time, which could enable purely computational targeted sequencing. However, this requires rapid identification of on-target reads while most mapping methods require computationally intensive basecalling. We present UNCALLED ( https://github.com/skovaka/UNCALLED ), an open source mapper that rapidly matches streaming of nanopore current signals to a reference sequence. UNCALLED probabilistically considers k-mers that could be represented by the signal and then prunes the candidates based on the reference encoded within a Ferragina–Manzini index. We used UNCALLED to deplete sequencing of known bacterial genomes within a metagenomics community, enriching the remaining species 4.46-fold. UNCALLED also enriched 148 human genes associated with hereditary cancers to 29.6× coverage using one MinION flowcell, enabling accurate detection of single-nucleotide polymorphisms, insertions and deletions, structural variants and methylation in these genes. UNCALLED enables targeted sequencing in nanopores by mapping the signal to large reference genomes.},
  archive      = {J_NBT},
  author       = {Kovaka, Sam and Fan, Yunfan and Ni, Bohan and Timp, Winston and Schatz, Michael C.},
  doi          = {10.1038/s41587-020-0731-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {431-441},
  shortjournal = {Nature Biotech.},
  title        = {Targeted nanopore sequencing by real-time mapping of raw electrical signal with UNCALLED},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Efficient hybrid de novo assembly of human genomes with
WENGAN. <em>NBT</em>, <em>39</em>(4), 422–430. (<a
href="https://doi.org/10.1038/s41587-020-00747-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generating accurate genome assemblies of large, repeat-rich human genomes has proved difficult using only long, error-prone reads, and most human genomes assembled from long reads add accurate short reads to polish the consensus sequence. Here we report an algorithm for hybrid assembly, WENGAN, that provides very high quality at low computational cost. We demonstrate de novo assembly of four human genomes using a combination of sequencing data generated on ONT PromethION, PacBio Sequel, Illumina and MGI technology. WENGAN implements efficient algorithms to improve assembly contiguity as well as consensus quality. The resulting genome assemblies have high contiguity (contig NG50: 17.24–80.64 Mb), few assembly errors (contig NGA50: 11.8–59.59 Mb), good consensus quality (QV: 27.84–42.88) and high gene completeness (BUSCO complete: 94.6–95.2\%), while consuming low computational resources (CPU hours: 187–1,200). In particular, the WENGAN assembly of the haploid CHM13 sample achieved a contig NG50 of 80.64 Mb (NGA50: 59.59 Mb), which surpasses the contiguity of the current human reference genome (GRCh38 contig NG50: 57.88 Mb). The genome assembler WENGAN produces high-quality human genome sequences at low computational cost.},
  archive      = {J_NBT},
  author       = {Di Genova, Alex and Buena-Atienza, Elena and Ossowski, Stephan and Sagot, Marie-France},
  doi          = {10.1038/s41587-020-00747-w},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {422-430},
  shortjournal = {Nature Biotech.},
  title        = {Efficient hybrid de novo assembly of human genomes with WENGAN},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). MRNA vaccines take on immune tolerance. <em>NBT</em>,
<em>39</em>(4), 419–421. (<a
href="https://doi.org/10.1038/s41587-021-00880-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A tolerizing mRNA vaccine prevents autoimmune disease in mice.},
  archive      = {J_NBT},
  author       = {Wardell, Christine M. and Levings, Megan K.},
  doi          = {10.1038/s41587-021-00880-0},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {419-421},
  shortjournal = {Nature Biotech.},
  title        = {MRNA vaccines take on immune tolerance},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Xenotransplantation. <em>NBT</em>, <em>39</em>(4), 418. (<a
href="https://doi.org/10.1038/s41587-021-00888-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to the production of cells, tissues and animals for use in the field of xenotransplantation.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00888-6},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {418},
  shortjournal = {Nature Biotech.},
  title        = {Xenotransplantation},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Identifying potential prescription drug product hopping.
<em>NBT</em>, <em>39</em>(4), 414–417. (<a
href="https://doi.org/10.1038/s41587-021-00877-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modified versions of existing drugs can substantially increase costs for patients and the health care system if the new version does not offer meaningful clinical improvement.},
  archive      = {J_NBT},
  author       = {Gowda, Vrushab and Beall, Reed F. and Kesselheim, Aaron S. and Sarpatwari, Ameet},
  doi          = {10.1038/s41587-021-00877-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {414-417},
  shortjournal = {Nature Biotech.},
  title        = {Identifying potential prescription drug product hopping},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Biotech bubbles during the global recession. <em>NBT</em>,
<em>39</em>(4), 408–413. (<a
href="https://doi.org/10.1038/s41587-021-00876-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Private biotech firms are awash with funding, raising record-breaking sums in 2020. Melanie Senior reports.},
  archive      = {J_NBT},
  author       = {Senior, Melanie},
  doi          = {10.1038/s41587-021-00876-w},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {408-413},
  shortjournal = {Nature Biotech.},
  title        = {Biotech bubbles during the global recession},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Alternative hosts as the missing link for equitable
therapeutic protein production. <em>NBT</em>, <em>39</em>(4), 404–407.
(<a href="https://doi.org/10.1038/s41587-021-00884-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Brady, Joseph R. and Love, J. Christopher},
  doi          = {10.1038/s41587-021-00884-w},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {404-407},
  shortjournal = {Nature Biotech.},
  title        = {Alternative hosts as the missing link for equitable therapeutic protein production},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Democratizing biotechnology requires more than availability.
<em>NBT</em>, <em>39</em>(4), 403. (<a
href="https://doi.org/10.1038/s41587-021-00890-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Beumer, Koen},
  doi          = {10.1038/s41587-021-00890-y},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {403},
  shortjournal = {Nature Biotech.},
  title        = {Democratizing biotechnology requires more than availability},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reply to: Democratizing biotechnology requires more than
availability. <em>NBT</em>, <em>39</em>(4), 403–404. (<a
href="https://doi.org/10.1038/s41587-021-00892-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Jackson, Shawn S. and Sumner, Louise E. and Casagrande, Rocco J.},
  doi          = {10.1038/s41587-021-00892-w},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {403-404},
  shortjournal = {Nature Biotech.},
  title        = {Reply to: Democratizing biotechnology requires more than availability},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mapping innovation trajectories on SARS-CoV-2 and its
variants. <em>NBT</em>, <em>39</em>(4), 401–403. (<a
href="https://doi.org/10.1038/s41587-021-00849-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Jefferson, Osmat Azzam and Koellhofer, Deniz and Warren, Ben and Ehrich, Thomas and Lang, Simon and Williams, Kenny and Ballagh, Aaron and Schellberg, Bernard and Sharma, Roshan and Jefferson, Richard A.},
  doi          = {10.1038/s41587-021-00849-z},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {401-403},
  shortjournal = {Nature Biotech.},
  title        = {Mapping innovation trajectories on SARS-CoV-2 and its variants},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). GM chestnut, sierra club darling. <em>NBT</em>,
<em>39</em>(4), 400. (<a
href="https://doi.org/10.1038/s41587-021-00903-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Melton, Lisa},
  doi          = {10.1038/s41587-021-00903-w},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {400},
  shortjournal = {Nature Biotech.},
  title        = {GM chestnut, sierra club darling},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). First-in-class and discovered in india. <em>NBT</em>,
<em>39</em>(4), 399. (<a
href="https://doi.org/10.1038/s41587-021-00906-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00906-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {399},
  shortjournal = {Nature Biotech.},
  title        = {First-in-class and discovered in india},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Lentiviral vector cleared of causing blood cancer.
<em>NBT</em>, <em>39</em>(4), 398. (<a
href="https://doi.org/10.1038/s41587-021-00904-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00904-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {398},
  shortjournal = {Nature Biotech.},
  title        = {Lentiviral vector cleared of causing blood cancer},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). First GM pigs for allergies. Could xenotransplants be next?
<em>NBT</em>, <em>39</em>(4), 397–400. (<a
href="https://doi.org/10.1038/s41587-021-00885-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The FDA greenlights α-Gal allergy-safe meat, but its makers have organs for transplants in their sights.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-021-00885-9},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {397-400},
  shortjournal = {Nature Biotech.},
  title        = {First GM pigs for allergies. could xenotransplants be next?},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). Around the world in a month. <em>NBT</em>, <em>39</em>(4),
396. (<a href="https://doi.org/10.1038/s41587-021-00887-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00887-7},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {396},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Waive COVID-19 IP rights, poor nations urge. <em>NBT</em>,
<em>39</em>(4), 395. (<a
href="https://doi.org/10.1038/s41587-021-00905-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00905-8},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {395},
  shortjournal = {Nature Biotech.},
  title        = {Waive COVID-19 IP rights, poor nations urge},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Radical solutions. <em>NBT</em>, <em>39</em>(4), 391. (<a
href="https://doi.org/10.1038/s41587-021-00908-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The US RADx program has spawned a phalanx of diagnostic products to market in just 12 months. Its long-term impact on point-of-care, at-home and population testing may be even more profound.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00908-5},
  journal      = {Nature Biotechnology},
  month        = {4},
  number       = {4},
  pages        = {391},
  shortjournal = {Nature Biotech.},
  title        = {Radical solutions},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). Innovators target vaccines for variants and shortages in
global south. <em>NBT</em>, <em>39</em>(4), 393–396. (<a
href="https://doi.org/10.1038/d41587-021-00001-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The next generation of COVID-19 vaccines will not only tackle viral variants but also provide solutions across the globe at a fraction of the cost. The next generation of COVID-19 vaccines will not only tackle viral variants but also provide solutions across the globe at a fraction of the cost.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/d41587-021-00001-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {4},
  pages        = {393-396},
  shortjournal = {Nature Biotech.},
  title        = {Innovators target vaccines for variants and shortages in global south},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021j). People. <em>NBT</em>, <em>39</em>(3), 390. (<a
href="https://doi.org/10.1038/s41587-021-00852-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00852-4},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {390},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Medical students need artificial intelligence and machine
learning training. <em>NBT</em>, <em>39</em>(3), 388–389. (<a
href="https://doi.org/10.1038/s41587-021-00846-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As the implementation of artificial intelligence and machine learning technologies progresses toward the clinic, the traditional roles of physicians will change.},
  archive      = {J_NBT},
  author       = {Pucchio, Aidan and Eisenhauer, Elizabeth A. and Moraes, Fabio Ynoe},
  doi          = {10.1038/s41587-021-00846-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {388-389},
  shortjournal = {Nature Biotech.},
  title        = {Medical students need artificial intelligence and machine learning training},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Author correction: Determination of isoform-specific RNA
structure with nanopore long reads. <em>NBT</em>, <em>39</em>(3), 387.
(<a href="https://doi.org/10.1038/s41587-020-00755-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Aw, Jong Ghut Ashley and Lim, Shaun W. and Wang, Jia Xu and Lambert, Finnlay R. P. and Tan, Wen Ting and Shen, Yang and Zhang, Yu and Kaewsapsak, Pornchai and Li, Chenhao and Ng, Sarah B. and Vardy, Leah A. and Tan, Meng How and Nagarajan, Niranjan and Wan, Yue},
  doi          = {10.1038/s41587-020-00755-w},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {387},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Determination of isoform-specific RNA structure with nanopore long reads},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). Publisher correction: First small-molecule drug targeting
RNA gains momentum. <em>NBT</em>, <em>39</em>(3), 387. (<a
href="https://doi.org/10.1038/s41587-021-00841-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41587-021-00841-7.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-021-00841-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {387},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: First small-molecule drug targeting RNA gains momentum},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Light-activated genetic therapy to
treat blindness enters clinic. <em>NBT</em>, <em>39</em>(3), 387. (<a
href="https://doi.org/10.1038/s41587-021-00871-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41587-021-00871-1.},
  archive      = {J_NBT},
  author       = {Ratner, Mark},
  doi          = {10.1038/s41587-021-00871-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {387},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Light-activated genetic therapy to treat blindness enters clinic},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Molecular goniometers for single-particle cryo-electron
microscopy of DNA-binding proteins. <em>NBT</em>, <em>39</em>(3),
378–386. (<a href="https://doi.org/10.1038/s41587-020-0716-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Correct reconstruction of macromolecular structure by cryo-electron microscopy (cryo-EM) relies on accurate determination of the orientation of single-particle images. For small (&amp;lt;100 kDa) DNA-binding proteins, obtaining particle images with sufficiently asymmetric features to correctly guide alignment is challenging. We apply DNA origami to construct molecular goniometers—instruments that precisely orient objects—and use them to dock a DNA-binding protein on a double-helix stage that has user-programmable tilt and rotation angles. We construct goniometers with 14 different stage configurations to orient and visualize the protein just above the cryo-EM grid surface. Each goniometer has a distinct barcode pattern that we use during particle classification to assign angle priors to the bound protein. We use goniometers to obtain a 6.5-Å structure of BurrH, an 82-kDa DNA-binding protein whose helical pseudosymmetry prevents accurate image orientation using traditional cryo-EM. Our approach should be adaptable to other DNA-binding proteins as well as small proteins fused to DNA-binding domains. DNA origami orients single proteins on a surface to aid structure determination by cryo-electron microscopy.},
  archive      = {J_NBT},
  author       = {Aksel, Tural and Yu, Zanlin and Cheng, Yifan and Douglas, Shawn M.},
  doi          = {10.1038/s41587-020-0716-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {378-386},
  shortjournal = {Nature Biotech.},
  title        = {Molecular goniometers for single-particle cryo-electron microscopy of DNA-binding proteins},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A near-infrared genetically encoded calcium indicator for in
vivo imaging. <em>NBT</em>, <em>39</em>(3), 368–377. (<a
href="https://doi.org/10.1038/s41587-020-0710-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While calcium imaging has become a mainstay of modern neuroscience, the spectral properties of current fluorescent calcium indicators limit deep-tissue imaging as well as simultaneous use with other probes. Using two monomeric near-infrared (NIR) fluorescent proteins (FPs), we engineered an NIR Förster resonance energy transfer (FRET)-based genetically encoded calcium indicator (iGECI). iGECI exhibits high levels of brightness and photostability and an increase up to 600\% in the fluorescence response to calcium. In dissociated neurons, iGECI detects spontaneous neuronal activity and electrically and optogenetically induced firing. We validated the performance of iGECI up to a depth of almost 400 µm in acute brain slices using one-photon light-sheet imaging. Applying hybrid photoacoustic and fluorescence microscopy, we simultaneously monitored neuronal and hemodynamic activities in the mouse brain through an intact skull, with resolutions of ~3 μm (lateral) and ~25–50 μm (axial). Using two-photon imaging, we detected evoked and spontaneous neuronal activity in the mouse visual cortex, with fluorescence changes of up to 25\%. iGECI allows biosensors and optogenetic actuators to be multiplexed without spectral crosstalk. A near-infrared fluorescent calcium indicator can be combined with other optogenetic tools in vivo without spectral crosstalk.},
  archive      = {J_NBT},
  author       = {Shemetov, Anton A. and Monakhov, Mikhail V. and Zhang, Qinrong and Canton-Josh, Jose Ernesto and Kumar, Manish and Chen, Maomao and Matlashov, Mikhail E. and Li, Xuan and Yang, Wei and Nie, Liming and Shcherbakova, Daria M. and Kozorovitskiy, Yevgenia and Yao, Junjie and Ji, Na and Verkhusha, Vladislav V.},
  doi          = {10.1038/s41587-020-0710-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {368-377},
  shortjournal = {Nature Biotech.},
  title        = {A near-infrared genetically encoded calcium indicator for in vivo imaging},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Mitochondrial copper depletion suppresses triple-negative
breast cancer in mice. <em>NBT</em>, <em>39</em>(3), 357–367. (<a
href="https://doi.org/10.1038/s41587-020-0707-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Depletion of mitochondrial copper, which shifts metabolism from respiration to glycolysis and reduces energy production, is known to be effective against cancer types that depend on oxidative phosphorylation. However, existing copper chelators are too toxic or ineffective for cancer treatment. Here we develop a safe, mitochondria-targeted, copper-depleting nanoparticle (CDN) and test it against triple-negative breast cancer (TNBC). We show that CDNs decrease oxygen consumption and oxidative phosphorylation, cause a metabolic switch to glycolysis and reduce ATP production in TNBC cells. This energy deficiency, together with compromised mitochondrial membrane potential and elevated oxidative stress, results in apoptosis. CDNs should be less toxic than existing copper chelators because they favorably deprive copper in the mitochondria in cancer cells instead of systemic depletion. Indeed, we demonstrate low toxicity of CDNs in healthy mice. In three mouse models of TNBC, CDN administration inhibits tumor growth and substantially improves survival. The efficacy and safety of CDNs suggest the potential clinical relevance of this approach. Triple-negative breast cancer is inhibited by depleting mitochondrial copper in mice.},
  archive      = {J_NBT},
  author       = {Cui, Liyang and Gouw, Arvin M. and LaGory, Edward L. and Guo, Shenghao and Attarwala, Nabeel and Tang, Yao and Qi, Ji and Chen, Yun-Sheng and Gao, Zhou and Casey, Kerriann M. and Bazhin, Arkadiy A. and Chen, Min and Hu, Leeann and Xie, Jinghang and Fang, Mingxi and Zhang, Cissy and Zhu, Qihua and Wang, Zhiyuan and Giaccia, Amato J. and Gambhir, Sanjiv Sam and Zhu, Weiping and Felsher, Dean W. and Pegram, Mark D. and Goun, Elena A. and Le, Anne and Rao, Jianghong},
  doi          = {10.1038/s41587-020-0707-9},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {357-367},
  shortjournal = {Nature Biotech.},
  title        = {Mitochondrial copper depletion suppresses triple-negative breast cancer in mice},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Analysis of RNA–protein networks with RNP-MaP defines
functional hubs on RNA. <em>NBT</em>, <em>39</em>(3), 347–356. (<a
href="https://doi.org/10.1038/s41587-020-0709-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA–protein interaction networks govern many biological processes but are difficult to examine comprehensively. We devised ribonucleoprotein networks analyzed by mutational profiling (RNP-MaP), a live-cell chemical probing strategy that maps cooperative interactions among multiple proteins bound to single RNA molecules at nucleotide resolution. RNP-MaP uses a hetero-bifunctional crosslinker to freeze interacting proteins in place on RNA and then maps multiple bound proteins on single RNA strands by read-through reverse transcription and DNA sequencing. RNP-MaP revealed that RNase P and RMRP, two sequence-divergent but structurally related non-coding RNAs, share RNP networks and that network hubs define functional sites in these RNAs. RNP-MaP also identified protein interaction networks conserved between mouse and human XIST long non-coding RNAs and defined protein communities whose binding sites colocalize and form networks in functional regions of XIST. RNP-MaP enables discovery and efficient validation of functional protein interaction networks on long RNAs in living cells. Networks of proteins bound to single RNAs are identified by correlated chemical crosslinking.},
  archive      = {J_NBT},
  author       = {Weidmann, Chase A. and Mustoe, Anthony M. and Jariwala, Parth B. and Calabrese, J. Mauro and Weeks, Kevin M.},
  doi          = {10.1038/s41587-020-0709-7},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {347-356},
  shortjournal = {Nature Biotech.},
  title        = {Analysis of RNA–protein networks with RNP-MaP defines functional hubs on RNA},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Determination of isoform-specific RNA structure with
nanopore long reads. <em>NBT</em>, <em>39</em>(3), 336–346. (<a
href="https://doi.org/10.1038/s41587-020-0712-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current methods for determining RNA structure with short-read sequencing cannot capture most differences between distinct transcript isoforms. Here we present RNA structure analysis using nanopore sequencing (PORE-cupine), which combines structure probing using chemical modifications with direct long-read RNA sequencing and machine learning to detect secondary structures in cellular RNAs. PORE-cupine also captures global structural features, such as RNA-binding-protein binding sites and reactivity differences at single-nucleotide variants. We show that shared sequences in different transcript isoforms of the same gene can fold into different structures, highlighting the importance of long-read sequencing for obtaining phase information. We also demonstrate that structural differences between transcript isoforms of the same gene lead to differences in translation efficiency. By revealing isoform-specific RNA structure, PORE-cupine will deepen understanding of the role of structures in controlling gene regulation. Using long reads to probe RNA structure reveals differences between transcript isoforms.},
  archive      = {J_NBT},
  author       = {Aw, Jong Ghut Ashley and Lim, Shaun W. and Wang, Jia Xu and Lambert, Finnlay R. P. and Tan, Wen Ting and Shen, Yang and Zhang, Yu and Kaewsapsak, Pornchai and Li, Chenhao and Ng, Sarah B. and Vardy, Leah A. and Tan, Meng How and Nagarajan, Niranjan and Wan, Yue},
  doi          = {10.1038/s41587-020-0712-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {336-346},
  shortjournal = {Nature Biotech.},
  title        = {Determination of isoform-specific RNA structure with nanopore long reads},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Plug-and-play control of a brain–computer interface through
neural map stabilization. <em>NBT</em>, <em>39</em>(3), 326–335. (<a
href="https://doi.org/10.1038/s41587-020-0662-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brain–computer interfaces (BCIs) enable control of assistive devices in individuals with severe motor impairments. A limitation of BCIs that has hindered real-world adoption is poor long-term reliability and lengthy daily recalibration times. To develop methods that allow stable performance without recalibration, we used a 128-channel chronic electrocorticography (ECoG) implant in a paralyzed individual, which allowed stable monitoring of signals. We show that long-term closed-loop decoder adaptation, in which decoder weights are carried across sessions over multiple days, results in consolidation of a neural map and ‘plug-and-play’ control. In contrast, daily reinitialization led to degradation of performance with variable relearning. Consolidation also allowed the addition of control features over days, that is, long-term stacking of dimensions. Our results offer an approach for reliable, stable BCI control by leveraging the stability of ECoG interfaces and neural plasticity. A paralyzed individual controls a neuroprosthetic without daily recalibration.},
  archive      = {J_NBT},
  author       = {Silversmith, Daniel B. and Abiri, Reza and Hardy, Nicholas F. and Natraj, Nikhilesh and Tu-Chan, Adelyn and Chang, Edward F. and Ganguly, Karunesh},
  doi          = {10.1038/s41587-020-0662-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {326-335},
  shortjournal = {Nature Biotech.},
  title        = {Plug-and-play control of a brain–computer interface through neural map stabilization},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). RNA timestamps identify the age of single molecules in RNA
sequencing. <em>NBT</em>, <em>39</em>(3), 320–325. (<a
href="https://doi.org/10.1038/s41587-020-0704-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current approaches to single-cell RNA sequencing (RNA-seq) provide only limited information about the dynamics of gene expression. Here we present RNA timestamps, a method for inferring the age of individual RNAs in RNA-seq data by exploiting RNA editing. To introduce timestamps, we tag RNA with a reporter motif consisting of multiple MS2 binding sites that recruit the adenosine deaminase ADAR2 fused to an MS2 capsid protein. ADAR2 binding to tagged RNA causes A-to-I edits to accumulate over time, allowing the age of the RNA to be inferred with hour-scale accuracy. By combining observations of multiple timestamped RNAs driven by the same promoter, we can determine when the promoter was active. We demonstrate that the system can infer the presence and timing of multiple past transcriptional events. Finally, we apply the method to cluster single cells according to the timing of past transcriptional activity. RNA timestamps will allow the incorporation of temporal information into RNA-seq workflows. The age of individual RNA molecules at 1-h resolution is inferred by measuring A-to-I editing.},
  archive      = {J_NBT},
  author       = {Rodriques, Samuel G. and Chen, Linlin M. and Liu, Sophia and Zhong, Ellen D. and Scherrer, Joseph R. and Boyden, Edward S. and Chen, Fei},
  doi          = {10.1038/s41587-020-0704-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {320-325},
  shortjournal = {Nature Biotech.},
  title        = {RNA timestamps identify the age of single molecules in RNA sequencing},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Highly sensitive spatial transcriptomics at near-cellular
resolution with slide-seqV2. <em>NBT</em>, <em>39</em>(3), 313–319. (<a
href="https://doi.org/10.1038/s41587-020-0739-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Measurement of the location of molecules in tissues is essential for understanding tissue formation and function. Previously, we developed Slide-seq, a technology that enables transcriptome-wide detection of RNAs with a spatial resolution of 10 μm. Here we report Slide-seqV2, which combines improvements in library generation, bead synthesis and array indexing to reach an RNA capture efficiency ~50\% that of single-cell RNA-seq data (~10-fold greater than Slide-seq), approaching the detection efficiency of droplet-based single-cell RNA-seq techniques. First, we leverage the detection efficiency of Slide-seqV2 to identify dendritically localized mRNAs in neurons of the mouse hippocampus. Second, we integrate the spatial information of Slide-seqV2 data with single-cell trajectory analysis tools to characterize the spatiotemporal development of the mouse neocortex, identifying underlying genetic programs that were poorly sampled with Slide-seq. The combination of near-cellular resolution and high transcript detection efficiency makes Slide-seqV2 useful across many experimental contexts. An improved method for spatial transcriptomics with detection efficiency approaching that of droplet-based single-cell RNA-seq techniques.},
  archive      = {J_NBT},
  author       = {Stickels, Robert R. and Murray, Evan and Kumar, Pawan and Li, Jilong and Marshall, Jamie L. and Di Bella, Daniela J. and Arlotta, Paola and Macosko, Evan Z. and Chen, Fei},
  doi          = {10.1038/s41587-020-0739-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {313-319},
  shortjournal = {Nature Biotech.},
  title        = {Highly sensitive spatial transcriptomics at near-cellular resolution with slide-seqV2},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Chromosome-scale, haplotype-resolved assembly of human
genomes. <em>NBT</em>, <em>39</em>(3), 309–312. (<a
href="https://doi.org/10.1038/s41587-020-0711-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Haplotype-resolved or phased genome assembly provides a complete picture of genomes and their complex genetic variations. However, current algorithms for phased assembly either do not generate chromosome-scale phasing or require pedigree information, which limits their application. We present a method named diploid assembly (DipAsm) that uses long, accurate reads and long-range conformation data for single individuals to generate a chromosome-scale phased assembly within 1 day. Applied to four public human genomes, PGP1, HG002, NA12878 and HG00733, DipAsm produced haplotype-resolved assemblies with minimum contig length needed to cover 50\% of the known genome (NG50) up to 25 Mb and phased ~99.5\% of heterozygous sites at 98–99\% accuracy, outperforming other approaches in terms of both contiguity and phasing completeness. We demonstrate the importance of chromosome-scale phased assemblies for the discovery of structural variants (SVs), including thousands of new transposon insertions, and of highly polymorphic and medically important regions such as the human leukocyte antigen (HLA) and killer cell immunoglobulin-like receptor (KIR) regions. DipAsm will facilitate high-quality precision medicine and studies of individual haplotype variation and population diversity. Assembly of phased human genomes is achieved by combining long reads and long-range conformational data.},
  archive      = {J_NBT},
  author       = {Garg, Shilpa and Fungtammasan, Arkarachai and Carroll, Andrew and Chou, Mike and Schmitt, Anthony and Zhou, Xiang and Mac, Stephen and Peluso, Paul and Hatas, Emily and Ghurye, Jay and Maguire, Jared and Mahmoud, Medhat and Cheng, Haoyu and Heller, David and Zook, Justin M. and Moemke, Tobias and Marschall, Tobias and Sedlazeck, Fritz J. and Aach, John and Chin, Chen-Shan and Church, George M. and Li, Heng},
  doi          = {10.1038/s41587-020-0711-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {309-312},
  shortjournal = {Nature Biotech.},
  title        = {Chromosome-scale, haplotype-resolved assembly of human genomes},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Fully phased human genome assembly without parental data
using single-cell strand sequencing and long reads. <em>NBT</em>,
<em>39</em>(3), 302–308. (<a
href="https://doi.org/10.1038/s41587-020-0719-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human genomes are typically assembled as consensus sequences that lack information on parental haplotypes. Here we describe a reference-free workflow for diploid de novo genome assembly that combines the chromosome-wide phasing and scaffolding capabilities of single-cell strand sequencing1,2 with continuous long-read or high-fidelity3 sequencing data. Employing this strategy, we produced a completely phased de novo genome assembly for each haplotype of an individual of Puerto Rican descent (HG00733) in the absence of parental data. The assemblies are accurate (quality value &amp;gt; 40) and highly contiguous (contig N50 &amp;gt; 23 Mbp) with low switch error rates (0.17\%), providing fully phased single-nucleotide variants, indels and structural variants. A comparison of Oxford Nanopore Technologies and Pacific Biosciences phased assemblies identified 154 regions that are preferential sites of contig breaks, irrespective of sequencing technology or phasing algorithms. Assembly of haplotype-resolved human genomes is achieved by combining short and long reads.},
  archive      = {J_NBT},
  author       = {Porubsky, David and Ebert, Peter and Audano, Peter A. and Vollger, Mitchell R. and Harvey, William T. and Marijon, Pierre and Ebler, Jana and Munson, Katherine M. and Sorensen, Melanie and Sulovari, Arvis and Haukness, Marina and Ghareghani, Maryam and Lansdorp, Peter M. and Paten, Benedict and Devine, Scott E. and Sanders, Ashley D. and Lee, Charles and Chaisson, Mark J. P. and Korbel, Jan O. and Eichler, Evan E. and Marschall, Tobias},
  doi          = {10.1038/s41587-020-0719-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {302-308},
  shortjournal = {Nature Biotech.},
  title        = {Fully phased human genome assembly without parental data using single-cell strand sequencing and long reads},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Life sciences intellectual property licensing at the
massachusetts institute of technology. <em>NBT</em>, <em>39</em>(3),
293–301. (<a href="https://doi.org/10.1038/s41587-021-00843-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Academic institutions play a central role in the biotech industry through technology licensing and the creation of startups, but few data are available on their performance and the magnitude of their impact. Here we present a systematic study of technology licensing by one such institution, the Massachusetts Institute of Technology (MIT). Using data on the 76 therapeutics-focused life sciences companies formed through MIT’s Technology Licensing Office from 1983 to 2017, we construct several measures of impact, including MIT patents cited in the Orange Book, capital raised, outcomes from mergers and acquisitions, patents granted to MIT intellectual property licensees, drug candidates discovered and US drug approvals—a key benchmark of innovation in the biopharmaceutical industry. As of December 2017, Orange Book listings for four approved small-molecule drugs cite MIT patents, but another 31 FDA-approved drugs (excluding candidates acquired after phase 3) had some involvement of MIT licensees. Fifty-five percent of the latter were either a new molecular entity or a new biological entity, and 55\% were granted priority review, an indication that they address an unmet medical need. The methodology described here may be a useful framework for other academic institutions to track outcomes of intellectual property in the therapeutics domain. An analysis of Massachusetts Institute of Technology intellectual property licenses in the life sciences not only provides a window on commercialization success but also a case study for transparency in understanding technology transfer performance.},
  archive      = {J_NBT},
  author       = {Huang, Samuel and Siah, Kien Wei and Vasileva, Detelina and Chen, Shirley and Nelsen, Lita and Lo, Andrew W.},
  doi          = {10.1038/s41587-021-00843-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {293-301},
  shortjournal = {Nature Biotech.},
  title        = {Life sciences intellectual property licensing at the massachusetts institute of technology},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). Forum: Precision oncology roundtable with mardis and siu.
<em>NBT</em>, <em>39</em>(3), 292. (<a
href="https://doi.org/10.1038/s41587-021-00858-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00858-y},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {292},
  shortjournal = {Nature Biotech.},
  title        = {Forum: Precision oncology roundtable with mardis and siu},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). First rounders: Neil kumar. <em>NBT</em>, <em>39</em>(3),
292. (<a href="https://doi.org/10.1038/s41587-021-00873-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00873-z},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {292},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Neil kumar},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Metagenomics. <em>NBT</em>, <em>39</em>(3), 291. (<a
href="https://doi.org/10.1038/s41587-021-00851-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to methods of preparing DNA sequencing libraries for metagenomic analysis and systems for processing metagenomic data.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00851-5},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {291},
  shortjournal = {Nature Biotech.},
  title        = {Metagenomics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). European patent office grants controversial patent
protecting virus: Lessons from the middle east respiratory syndrome
coronavirus outbreak. <em>NBT</em>, <em>39</em>(3), 287–290. (<a
href="https://doi.org/10.1038/s41587-021-00840-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The scope of patents covering SARS-CoV-2 and diagnostics, vaccines and treatments related to the virus remains unclear, but the pandemic may have created a permanent demand for SARS-CoV-2-related products and spark long-term interest in relevant intellectual property rights.},
  archive      = {J_NBT},
  author       = {Richter, Carsten D.},
  doi          = {10.1038/s41587-021-00840-8},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {287-290},
  shortjournal = {Nature Biotech.},
  title        = {European patent office grants controversial patent protecting virus: Lessons from the middle east respiratory syndrome coronavirus outbreak},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Voices of biotech research. <em>NBT</em>, <em>39</em>(3),
281–286. (<a href="https://doi.org/10.1038/s41587-021-00847-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Biotechnology asks a selection of faculty about the most exciting frontier in their field and the most needed technologies for advancing knowledge and applications.},
  archive      = {J_NBT},
  author       = {Annabi, Nasim and Baker, Matthew and Boettiger, Alistair and Chakraborty, Debojyoti and Chen, Yvonne and Corbett, Kizzmekia S. and Correia, Bruno and Dahlman, James and de Oliveira, Tulio and Ertuerk, Ali and Yanik, Mehmet Fatih and Henaff, Elizabeth and Huch, Meritxell and Iliev, Iliyan D. and Jacobs, Thomas and Junca, Howard and Keung, Albert and Kolodkin-Gal, Ilana and Krishnaswamy, Smita and Lancaster, Madeline and Macosko, Evan and Martínez-Núñez, Mario Alberto and Miura, Kyoko and Molloy, Jenny and Cruz, Andrés Ochoa and Platt, Randall J. and Posey, Avery D. and Shao, Huilin and Simunovic, Mijo and Slavov, Nikolai and Takebe, Takanori and Vandenberghe, Luk H. and Varshney, Rajeev K. and Wang, Jianbin},
  doi          = {10.1038/s41587-021-00847-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {281-286},
  shortjournal = {Nature Biotech.},
  title        = {Voices of biotech research},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Maintaining “standards” for biosimilar monoclonal
antibodies. <em>NBT</em>, <em>39</em>(3), 276–280. (<a
href="https://doi.org/10.1038/s41587-021-00848-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Prior, Sandra and Metcalfe, Clive and Hufton, Simon E. and Wadhwa, Meenu and Schneider, Christian K. and Burns, Chris},
  doi          = {10.1038/s41587-021-00848-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {276-280},
  shortjournal = {Nature Biotech.},
  title        = {Maintaining ‘standards’ for biosimilar monoclonal antibodies},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Could mutations of SARS-CoV-2 suppress diagnostic detection?
<em>NBT</em>, <em>39</em>(3), 274–275. (<a
href="https://doi.org/10.1038/s41587-021-00845-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Ascoli, Carl A.},
  doi          = {10.1038/s41587-021-00845-3},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {274-275},
  shortjournal = {Nature Biotech.},
  title        = {Could mutations of SARS-CoV-2 suppress diagnostic detection?},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Balancing clinical evidence in the context of a pandemic.
<em>NBT</em>, <em>39</em>(3), 270–274. (<a
href="https://doi.org/10.1038/s41587-021-00834-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Adashek, Jacob J. and Kurzrock, Razelle},
  doi          = {10.1038/s41587-021-00834-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {270-274},
  shortjournal = {Nature Biotech.},
  title        = {Balancing clinical evidence in the context of a pandemic},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). How large pharma impacts biotechnology startup success.
<em>NBT</em>, <em>39</em>(3), 266–269. (<a
href="https://doi.org/10.1038/s41587-021-00821-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Implications for bioentrepreneurs.},
  archive      = {J_NBT},
  author       = {Melchner von Dydiowa, Gina and van Deventer, Sander and Couto, Daniela S.},
  doi          = {10.1038/s41587-021-00821-x},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {266-269},
  shortjournal = {Nature Biotech.},
  title        = {How large pharma impacts biotechnology startup success},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021f). Around the world in a month. <em>NBT</em>, <em>39</em>(3),
259. (<a href="https://doi.org/10.1038/s41587-021-00854-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00854-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {259},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Soccer rivals ally over biotech. <em>NBT</em>,
<em>39</em>(3), 258. (<a
href="https://doi.org/10.1038/s41587-021-00863-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00863-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {258},
  shortjournal = {Nature Biotech.},
  title        = {Soccer rivals ally over biotech},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Club-goers take first bites of lab-made chicken.
<em>NBT</em>, <em>39</em>(3), 257–258. (<a
href="https://doi.org/10.1038/s41587-021-00855-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Slaughter-free meat grown in bioreactors reaches select diners in Singapore in the form of chicken nuggets.},
  archive      = {J_NBT},
  author       = {Waltz, Emily},
  doi          = {10.1038/s41587-021-00855-1},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {257-258},
  shortjournal = {Nature Biotech.},
  title        = {Club-goers take first bites of lab-made chicken},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Green funds for lab-made breast milk. <em>NBT</em>,
<em>39</em>(3), 256. (<a
href="https://doi.org/10.1038/s41587-021-00862-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00862-2},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {256},
  shortjournal = {Nature Biotech.},
  title        = {Green funds for lab-made breast milk},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Neural dust swept up in latest leap for bioelectronic
medicine. <em>NBT</em>, <em>39</em>(3), 255–256. (<a
href="https://doi.org/10.1038/s41587-021-00856-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As big pharma interest grows in bioelectronics medicine, Astellas swoops in for a startup developing wireless, grain-sized devices powered by ultrasound for CNS disease monitoring and intervention.},
  archive      = {J_NBT},
  author       = {Patch, Kimberly},
  doi          = {10.1038/s41587-021-00856-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {255-256},
  shortjournal = {Nature Biotech.},
  title        = {Neural dust swept up in latest leap for bioelectronic medicine},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Agents shuttle across blood–brain barrier to treat hunter
syndrome. <em>NBT</em>, <em>39</em>(3), 253. (<a
href="https://doi.org/10.1038/s41587-021-00864-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00864-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {253},
  shortjournal = {Nature Biotech.},
  title        = {Agents shuttle across blood–brain barrier to treat hunter syndrome},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Bispecific antibodies poised to deliver wave of cancer
therapies. <em>NBT</em>, <em>39</em>(3), 251–254. (<a
href="https://doi.org/10.1038/s41587-021-00850-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An explosion of new formats for bispecific antibodies promises to propel the field to better clinical outcomes in oncology.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-021-00850-6},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {251-254},
  shortjournal = {Nature Biotech.},
  title        = {Bispecific antibodies poised to deliver wave of cancer therapies},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The next 25 years. <em>NBT</em>, <em>39</em>(3), 249. (<a
href="https://doi.org/10.1038/s41587-021-00872-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Since this journal launched in March 1996, biotech has become a US economic powerhouse. To reach its full potential over the next 25 years, touching all corners of the globe, it must become more inclusive.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00872-0},
  journal      = {Nature Biotechnology},
  month        = {3},
  number       = {3},
  pages        = {249},
  shortjournal = {Nature Biotech.},
  title        = {The next 25 years},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The missing sex. <em>NBT</em>, <em>39</em>(3), 260–265. (<a
href="https://doi.org/10.1038/s41587-021-00844-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For most of its history, biomedical research and clinical testing has neglected over half of the world’s population. Finally, researchers and funders are starting to recognize the importance of sex differences.},
  archive      = {J_NBT},
  author       = {Seydel, Caroline},
  doi          = {10.1038/s41587-021-00844-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {3},
  pages        = {260-265},
  shortjournal = {Nature Biotech.},
  title        = {The missing sex},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021k). People. <em>NBT</em>, <em>39</em>(2), 248. (<a
href="https://doi.org/10.1038/s41587-021-00826-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00826-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {248},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021c). Fourth-quarter biotech job picture. <em>NBT</em>,
<em>39</em>(2), 247. (<a
href="https://doi.org/10.1038/s41587-021-00820-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quarterly snapshot of job expansions, reductions and availability in the biotech and pharma sectors.},
  archive      = {J_NBT},
  author       = {Francisco, Michael},
  doi          = {10.1038/s41587-021-00820-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {247},
  shortjournal = {Nature Biotech.},
  title        = {Fourth-quarter biotech job picture},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021k). Publisher correction: COVID-19 antibodies on trial.
<em>NBT</em>, <em>39</em>(2), 246. (<a
href="https://doi.org/10.1038/s41587-021-00813-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41587-021-00813-x.},
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-00813-x},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {246},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: COVID-19 antibodies on trial},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Benchmarking of t cell receptor repertoire profiling methods
reveals large systematic biases. <em>NBT</em>, <em>39</em>(2), 236–245.
(<a href="https://doi.org/10.1038/s41587-020-0656-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Monitoring the T cell receptor (TCR) repertoire in health and disease can provide key insights into adaptive immune responses, but the accuracy of current TCR sequencing (TCRseq) methods is unclear. In this study, we systematically compared the results of nine commercial and academic TCRseq methods, including six rapid amplification of complementary DNA ends (RACE)-polymerase chain reaction (PCR) and three multiplex-PCR approaches, when applied to the same T cell sample. We found marked differences in accuracy and intra- and inter-method reproducibility for T cell receptor α (TRA) and T cell receptor β (TRB) TCR chains. Most methods showed a lower ability to capture TRA than TRB diversity. Low RNA input generated non-representative repertoires. Results from the 5′ RACE-PCR methods were consistent among themselves but differed from the RNA-based multiplex-PCR results. Using an in silico meta-repertoire generated from 108 replicates, we found that one genomic DNA-based method and two non-unique molecular identifier (UMI) RNA-based methods were more sensitive than UMI methods in detecting rare clonotypes, despite the better clonotype quantification accuracy of the latter. A comparison of T cell receptor repertoire profiling methods shows substantial differences in their outputs.},
  archive      = {J_NBT},
  author       = {Barennes, Pierre and Quiniou, Valentin and Shugay, Mikhail and Egorov, Evgeniy S. and Davydov, Alexey N. and Chudakov, Dmitriy M. and Uddin, Imran and Ismail, Mazlina and Oakes, Theres and Chain, Benny and Eugster, Anne and Kashofer, Karl and Rainer, Peter P. and Darko, Samuel and Ransier, Amy and Douek, Daniel C. and Klatzmann, David and Mariotti-Ferrandiz, Encarnita},
  doi          = {10.1038/s41587-020-0656-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {236-245},
  shortjournal = {Nature Biotech.},
  title        = {Benchmarking of t cell receptor repertoire profiling methods reveals large systematic biases},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Direct DNA crosslinking with CAP-c uncovers
transcription-dependent chromatin organization at high resolution.
<em>NBT</em>, <em>39</em>(2), 225–235. (<a
href="https://doi.org/10.1038/s41587-020-0643-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determining the spatial organization of chromatin in cells mainly relies on crosslinking-based chromosome conformation capture techniques, but resolution and signal-to-noise ratio of these approaches is limited by interference from DNA-bound proteins. Here we introduce chemical-crosslinking assisted proximity capture (CAP-C), a method that uses multifunctional chemical crosslinkers with defined sizes to capture chromatin contacts. CAP-C generates chromatin contact maps at subkilobase (sub-kb) resolution with low background noise. We applied CAP-C to formaldehyde prefixed mouse embryonic stem cells (mESCs) and investigated loop domains (median size of 200 kb) and nonloop domains (median size of 9 kb). Transcription inhibition caused a greater loss of contacts in nonloop domains than loop domains. We uncovered conserved, transcription-state-dependent chromatin compartmentalization at high resolution that is shared from Drosophila to human, and a transcription-initiation-dependent nuclear subcompartment that brings multiple nonloop domains in close proximity. We also showed that CAP-C could be used to detect native chromatin conformation without formaldehyde prefixing. CAP-C uses multifunctional DNA crosslinkers of defined sizes to increase resolution and signal-to-noise in chromosome conformation capture experiments.},
  archive      = {J_NBT},
  author       = {You, Qiancheng and Cheng, Anthony Youzhi and Gu, Xi and Harada, Bryan T. and Yu, Miao and Wu, Tong and Ren, Bing and Ouyang, Zhengqing and He, Chuan},
  doi          = {10.1038/s41587-020-0643-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {225-235},
  shortjournal = {Nature Biotech.},
  title        = {Direct DNA crosslinking with CAP-C uncovers transcription-dependent chromatin organization at high resolution},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Oncoprotein-specific molecular interaction maps (SigMaps)
for cancer network analyses. <em>NBT</em>, <em>39</em>(2), 215–224. (<a
href="https://doi.org/10.1038/s41587-020-0652-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Tumor-specific elucidation of physical and functional oncoprotein interactions could improve tumorigenic mechanism characterization and therapeutic response prediction. Current interaction models and pathways, however, lack context specificity and are not oncoprotein specific. We introduce SigMaps as context-specific networks, comprising modulators, effectors and cognate binding-partners of a specific oncoprotein. SigMaps are reconstructed de novo by integrating diverse evidence sources—including protein structure, gene expression and mutational profiles—via the OncoSig machine learning framework. We first generated a KRAS-specific SigMap for lung adenocarcinoma, which recapitulated published KRAS biology, identified novel synthetic lethal proteins that were experimentally validated in three-dimensional spheroid models and established uncharacterized crosstalk with RAB/RHO. To show that OncoSig is generalizable, we first inferred SigMaps for the ten most mutated human oncoproteins and then for the full repertoire of 715 proteins in the COSMIC Cancer Gene Census. Taken together, these SigMaps show that the cell’s regulatory and signaling architecture is highly tissue specific. Interaction networks for any protein can be generated using a machine learning algorithm.},
  archive      = {J_NBT},
  author       = {Broyde, Joshua and Simpson, David R. and Murray, Diana and Paull, Evan O. and Chu, Brennan W. and Tagore, Somnath and Jones, Sunny J. and Griffin, Aaron T. and Giorgi, Federico M. and Lachmann, Alexander and Jackson, Peter and Sweet-Cordero, E. Alejandro and Honig, Barry and Califano, Andrea},
  doi          = {10.1038/s41587-020-0652-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {215-224},
  shortjournal = {Nature Biotech.},
  title        = {Oncoprotein-specific molecular interaction maps (SigMaps) for cancer network analyses},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Characterizing allele- and haplotype-specific copy numbers
in single cells with CHISEL. <em>NBT</em>, <em>39</em>(2), 207–214. (<a
href="https://doi.org/10.1038/s41587-020-0661-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell barcoding technologies enable genome sequencing of thousands of individual cells in parallel, but with extremely low sequencing coverage (&amp;lt;0.05×) per cell. While the total copy number of large multi-megabase segments can be derived from such data, important allele-specific mutations—such as copy-neutral loss of heterozygosity (LOH) in cancer—are missed. We introduce copy-number haplotype inference in single cells using evolutionary links (CHISEL), a method to infer allele- and haplotype-specific copy numbers in single cells and subpopulations of cells by aggregating sparse signal across hundreds or thousands of individual cells. We applied CHISEL to ten single-cell sequencing datasets of ~2,000 cells from two patients with breast cancer. We identified extensive allele-specific copy-number aberrations (CNAs) in these samples, including copy-neutral LOHs, whole-genome duplications (WGDs) and mirrored-subclonal CNAs. These allele-specific CNAs affect genomic regions containing well-known breast-cancer genes. We also refined the reconstruction of tumor evolution, timing allele-specific CNAs before and after WGDs, identifying low-frequency subpopulations distinguished by unique CNAs and uncovering evidence of convergent evolution. Genomic copy-number aberrations in single cells are identified with allele specificity.},
  archive      = {J_NBT},
  author       = {Zaccaria, Simone and Raphael, Benjamin J.},
  doi          = {10.1038/s41587-020-0661-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {207-214},
  shortjournal = {Nature Biotech.},
  title        = {Characterizing allele- and haplotype-specific copy numbers in single cells with CHISEL},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Predicting the efficiency of prime editing guide RNAs in
human cells. <em>NBT</em>, <em>39</em>(2), 198–206. (<a
href="https://doi.org/10.1038/s41587-020-0677-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prime editing enables the introduction of virtually any small-sized genetic change without requiring donor DNA or double-strand breaks. However, evaluation of prime editing efficiency requires time-consuming experiments, and the factors that affect efficiency have not been extensively investigated. In this study, we performed high-throughput evaluation of prime editor 2 (PE2) activities in human cells using 54,836 pairs of prime editing guide RNAs (pegRNAs) and their target sequences. The resulting data sets allowed us to identify factors affecting PE2 efficiency and to develop three computational models to predict pegRNA efficiency. For a given target sequence, the computational models predict efficiencies of pegRNAs with different lengths of primer binding sites and reverse transcriptase templates for edits of various types and positions. Testing the accuracy of the predictions using test data sets that were not used for training, we found Spearman’s correlations between 0.47 and 0.81. Our computational models and information about factors affecting PE2 efficiency will facilitate practical application of prime editing. Prime editing is optimized by a method to choose the most efficient guide RNA.},
  archive      = {J_NBT},
  author       = {Kim, Hui Kwon and Yu, Goosang and Park, Jinman and Min, Seonwoo and Lee, Sungtae and Yoon, Sungroh and Kim, Hyongbum Henry},
  doi          = {10.1038/s41587-020-0677-y},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {198-206},
  shortjournal = {Nature Biotech.},
  title        = {Predicting the efficiency of prime editing guide RNAs in human cells},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Single-cell metabolic profiling of human cytotoxic t cells.
<em>NBT</em>, <em>39</em>(2), 186–197. (<a
href="https://doi.org/10.1038/s41587-020-0651-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular metabolism regulates immune cell activation, differentiation and effector functions, but current metabolic approaches lack single-cell resolution and simultaneous characterization of cellular phenotype. In this study, we developed an approach to characterize the metabolic regulome of single cells together with their phenotypic identity. The method, termed single-cell metabolic regulome profiling (scMEP), quantifies proteins that regulate metabolic pathway activity using high-dimensional antibody-based technologies. We employed mass cytometry (cytometry by time of flight, CyTOF) to benchmark scMEP against bulk metabolic assays by reconstructing the metabolic remodeling of in vitro-activated naive and memory CD8+ T cells. We applied the approach to clinical samples and identified tissue-restricted, metabolically repressed cytotoxic T cells in human colorectal carcinoma. Combining our method with multiplexed ion beam imaging by time of flight (MIBI-TOF), we uncovered the spatial organization of metabolic programs in human tissues, which indicated exclusion of metabolically repressed immune cells from the tumor–immune boundary. Overall, our approach enables robust approximation of metabolic and functional states in individual cells. An antibody-based method enables profiling of metabolic protein expression and regulation in single cells using mass spectrometry.},
  archive      = {J_NBT},
  author       = {Hartmann, Felix J. and Mrdjen, Dunja and McCaffrey, Erin and Glass, David R. and Greenwald, Noah F. and Bharadwaj, Anusha and Khair, Zumana and Verberk, Sanne G. S. and Baranski, Alex and Baskar, Reema and Graf, William and Van Valen, David and Van den Bossche, Jan and Angelo, Michael and Bendall, Sean C.},
  doi          = {10.1038/s41587-020-0651-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {186-197},
  shortjournal = {Nature Biotech.},
  title        = {Single-cell metabolic profiling of human cytotoxic t cells},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Epineural optogenetic activation of nociceptors initiates
and amplifies inflammation. <em>NBT</em>, <em>39</em>(2), 179–185. (<a
href="https://doi.org/10.1038/s41587-020-0673-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Activation of nociceptor sensory neurons by noxious stimuli both triggers pain and increases capillary permeability and blood flow to produce neurogenic inflammation1,2, but whether nociceptors also interact with the immune system remains poorly understood. Here we report a neurotechnology for selective epineural optogenetic neuromodulation of nociceptors and demonstrate that nociceptor activation drives both protective pain behavior and inflammation. The wireless optoelectronic system consists of sub-millimeter-scale light-emitting diodes embedded in a soft, circumneural sciatic nerve implant, powered and driven by a miniaturized head-mounted control unit. Photostimulation of axons in freely moving mice that express channelrhodopsin only in nociceptors resulted in behaviors characteristic of pain, reflecting orthodromic input to the spinal cord. It also led to immune reactions in the skin in the absence of inflammation and potentiation of established inflammation, a consequence of the antidromic activation of nociceptor peripheral terminals. These results reveal a link between nociceptors and immune cells, which might have implications for the treatment of inflammation. An optogenetic study shows that activation of pain-triggering sensory neurons induces an immune response.},
  archive      = {J_NBT},
  author       = {Michoud, Frédéric and Seehus, Corey and Schönle, Philipp and Brun, Noé and Taub, Daniel and Zhang, Zihe and Jain, Aakanksha and Furfaro, Ivan and Akouissi, Outman and Moon, Rachel and Meier, Pascale and Galan, Katia and Doyle, Benjamin and Tetreault, Michael and Talbot, Sébastien and Browne, Liam E. and Huang, Qiuting and Woolf, Clifford J. and Lacour, Stéphanie P.},
  doi          = {10.1038/s41587-020-0673-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {179-185},
  shortjournal = {Nature Biotech.},
  title        = {Epineural optogenetic activation of nociceptors initiates and amplifies inflammation},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Isolating live cell clones from barcoded populations using
CRISPRa-inducible reporters. <em>NBT</em>, <em>39</em>(2), 174–178. (<a
href="https://doi.org/10.1038/s41587-020-0614-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed a functional lineage tracing tool termed CaTCH (CRISPRa tracing of clones in heterogeneous cell populations). CaTCH combines precise clonal tracing of millions of cells with the ability to retrospectively isolate founding clones alive before and during selection, allowing functional experiments. Using CaTCH, we captured rare clones representing as little as 0.001\% of a population and investigated the emergence of resistance to targeted melanoma therapy in vivo. Cell clones in lineage-tracing studies can be isolated for functional experiments.},
  archive      = {J_NBT},
  author       = {Umkehrer, Christian and Holstein, Felix and Formenti, Laura and Jude, Julian and Froussios, Kimon and Neumann, Tobias and Cronin, Shona M. and Haas, Lisa and Lipp, Jesse J. and Burkard, Thomas R. and Fellner, Michaela and Wiesner, Thomas and Zuber, Johannes and Obenauf, Anna C.},
  doi          = {10.1038/s41587-020-0614-0},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {174-178},
  shortjournal = {Nature Biotech.},
  title        = {Isolating live cell clones from barcoded populations using CRISPRa-inducible reporters},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Auto-deconvolution and molecular networking of gas
chromatography–mass spectrometry data. <em>NBT</em>, <em>39</em>(2),
169–173. (<a href="https://doi.org/10.1038/s41587-020-0700-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We engineered a machine learning approach, MSHub, to enable auto-deconvolution of gas chromatography–mass spectrometry (GC–MS) data. We then designed workflows to enable the community to store, process, share, annotate, compare and perform molecular networking of GC–MS data within the Global Natural Product Social (GNPS) Molecular Networking analysis platform. MSHub/GNPS performs auto-deconvolution of compound fragmentation patterns via unsupervised non-negative matrix factorization and quantifies the reproducibility of fragmentation patterns across samples. A machine learning workflow enables auto-deconvolution of gas chromatography–mass spectrometry data.},
  archive      = {J_NBT},
  author       = {Aksenov, Alexander A. and Laponogov, Ivan and Zhang, Zheng and Doran, Sophie L. F. and Belluomo, Ilaria and Veselkov, Dennis and Bittremieux, Wout and Nothias, Louis Felix and Nothias-Esposito, Mélissa and Maloney, Katherine N. and Misra, Biswapriya B. and Melnik, Alexey V. and Smirnov, Aleksandr and Du, Xiuxia and Jones, Kenneth L. and Dorrestein, Kathleen and Panitchpakdi, Morgan and Ernst, Madeleine and van der Hooft, Justin J. J. and Gonzalez, Mabel and Carazzone, Chiara and Amézquita, Adolfo and Callewaert, Chris and Morton, James T. and Quinn, Robert A. and Bouslimani, Amina and Orio, Andrea Albarracín and Petras, Daniel and Smania, Andrea M. and Couvillion, Sneha P. and Burnet, Meagan C. and Nicora, Carrie D. and Zink, Erika and Metz, Thomas O. and Artaev, Viatcheslav and Humston-Fulmer, Elizabeth and Gregor, Rachel and Meijler, Michael M. and Mizrahi, Itzhak and Eyal, Stav and Anderson, Brooke and Dutton, Rachel and Lugan, Raphaël and Boulch, Pauline Le and Guitton, Yann and Prevost, Stephanie and Poirier, Audrey and Dervilly, Gaud and Le Bizec, Bruno and Fait, Aaron and Persi, Noga Sikron and Song, Chao and Gashu, Kelem and Coras, Roxana and Guma, Monica and Manasson, Julia and Scher, Jose U. and Barupal, Dinesh Kumar and Alseekh, Saleh and Fernie, Alisdair R. and Mirnezami, Reza and Vasiliou, Vasilis and Schmid, Robin and Borisov, Roman S. and Kulikova, Larisa N. and Knight, Rob and Wang, Mingxun and Hanna, George B. and Dorrestein, Pieter C. and Veselkov, Kirill},
  doi          = {10.1038/s41587-020-0700-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {169-173},
  shortjournal = {Nature Biotech.},
  title        = {Auto-deconvolution and molecular networking of gas chromatography–mass spectrometry data},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Context-aware dimensionality reduction deconvolutes gut
microbial community dynamics. <em>NBT</em>, <em>39</em>(2), 165–168. (<a
href="https://doi.org/10.1038/s41587-020-0660-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The translational power of human microbiome studies is limited by high interindividual variation. We describe a dimensionality reduction tool, compositional tensor factorization (CTF), that incorporates information from the same host across multiple samples to reveal patterns driving differences in microbial composition across phenotypes. CTF identifies robust patterns in sparse compositional datasets, allowing for the detection of microbial changes associated with specific phenotypes that are reproducible across datasets. Gut microbiome composition is associated with phenotypes as revealed by a dimensionality reduction tool.},
  archive      = {J_NBT},
  author       = {Martino, Cameron and Shenhav, Liat and Marotz, Clarisse A. and Armstrong, George and McDonald, Daniel and Vázquez-Baeza, Yoshiki and Morton, James T. and Jiang, Lingjing and Dominguez-Bello, Maria Gloria and Swafford, Austin D. and Halperin, Eran and Knight, Rob},
  doi          = {10.1038/s41587-020-0660-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {165-168},
  shortjournal = {Nature Biotech.},
  title        = {Context-aware dimensionality reduction deconvolutes gut microbial community dynamics},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Deep brain optogenetics without intracranial surgery.
<em>NBT</em>, <em>39</em>(2), 161–164. (<a
href="https://doi.org/10.1038/s41587-020-0679-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Achieving temporally precise, noninvasive control over specific neural cell types in the deep brain would advance the study of nervous system function. Here we use the potent channelrhodopsin ChRmine to achieve transcranial photoactivation of defined neural circuits, including midbrain and brainstem structures, at unprecedented depths of up to 7 mm with millisecond precision. Using systemic viral delivery of ChRmine, we demonstrate behavioral modulation without surgery, enabling implant-free deep brain optogenetics. Optogenetic control of neural activity in the deep brain is achieved without intracranial surgery using ChRmine.},
  archive      = {J_NBT},
  author       = {Chen, Ritchie and Gore, Felicity and Nguyen, Quynh-Anh and Ramakrishnan, Charu and Patel, Sneha and Kim, Soo Hyun and Raffiee, Misha and Kim, Yoon Seok and Hsueh, Brian and Krook-Magnusson, Esther and Soltesz, Ivan and Deisseroth, Karl},
  doi          = {10.1038/s41587-020-0679-9},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {161-164},
  shortjournal = {Nature Biotech.},
  title        = {Deep brain optogenetics without intracranial surgery},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reply to: UMI or not UMI, that is the question for scRNA-seq
zero-inflation. <em>NBT</em>, <em>39</em>(2), 160. (<a
href="https://doi.org/10.1038/s41587-020-00811-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Svensson, Valentine},
  doi          = {10.1038/s41587-020-00811-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {160},
  shortjournal = {Nature Biotech.},
  title        = {Reply to: UMI or not UMI, that is the question for scRNA-seq zero-inflation},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). UMI or not UMI, that is the question for scRNA-seq
zero-inflation. <em>NBT</em>, <em>39</em>(2), 158–159. (<a
href="https://doi.org/10.1038/s41587-020-00810-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Cao, Yingying and Kitanovski, Simo and Küppers, Ralf and Hoffmann, Daniel},
  doi          = {10.1038/s41587-020-00810-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {158-159},
  shortjournal = {Nature Biotech.},
  title        = {UMI or not UMI, that is the question for scRNA-seq zero-inflation},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Initialization is critical for preserving global data
structure in both t-SNE and UMAP. <em>NBT</em>, <em>39</em>(2), 156–157.
(<a href="https://doi.org/10.1038/s41587-020-00809-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Kobak, Dmitry and Linderman, George C.},
  doi          = {10.1038/s41587-020-00809-z},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {156-157},
  shortjournal = {Nature Biotech.},
  title        = {Initialization is critical for preserving global data structure in both t-SNE and UMAP},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Liquid biopsy. <em>NBT</em>, <em>39</em>(2), 154. (<a
href="https://doi.org/10.1038/s41587-021-00829-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents relating to devices, kits and detection methods for liquid biopsy.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00829-3},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {154},
  shortjournal = {Nature Biotech.},
  title        = {Liquid biopsy},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Building a high-quality human cell atlas. <em>NBT</em>,
<em>39</em>(2), 149–153. (<a
href="https://doi.org/10.1038/s41587-020-00812-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Rozenblatt-Rosen, Orit and Shin, Jay W. and Rood, Jennifer E. and Hupalowska, Anna and Regev, Aviv and Heyn, Holger},
  doi          = {10.1038/s41587-020-00812-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {149-153},
  shortjournal = {Nature Biotech.},
  title        = {Building a high-quality human cell atlas},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Will ethiopia be a springboard or a stonewall for GM crops
in africa? <em>NBT</em>, <em>39</em>(2), 147–148. (<a
href="https://doi.org/10.1038/s41587-021-00827-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Sida, Tesfaye (Shiferaw)},
  doi          = {10.1038/s41587-021-00827-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {147-148},
  shortjournal = {Nature Biotech.},
  title        = {Will ethiopia be a springboard or a stonewall for GM crops in africa?},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Research priorities for COVID-19 sensor technology.
<em>NBT</em>, <em>39</em>(2), 144–147. (<a
href="https://doi.org/10.1038/s41587-021-00816-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Tong, Allison and Sorrell, Tania C. and Black, Andrew J. and Caillaud, Corinne and Chrzanowski, Wojciech and Li, Eugena and Martinez-Martin, David and McEwan, Alistair and Wang, Rex and Motion, Alice and Bedoya, Alvaro Casas and Huang, Jun and Azizi, Lamiae and Eggleton, Benjamin J.},
  doi          = {10.1038/s41587-021-00816-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {144-147},
  shortjournal = {Nature Biotech.},
  title        = {Research priorities for COVID-19 sensor technology},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021i). Financing breaks all records in 2020. <em>NBT</em>,
<em>39</em>(2), 133–134. (<a
href="https://doi.org/10.1038/s41587-021-00817-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-00817-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {133-134},
  shortjournal = {Nature Biotech.},
  title        = {Financing breaks all records in 2020},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Drug pipeline 4Q20. <em>NBT</em>, <em>39</em>(2), 131–132.
(<a href="https://doi.org/10.1038/s41587-021-00818-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {DeFrancesco, Laura},
  doi          = {10.1038/s41587-021-00818-6},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {131-132},
  shortjournal = {Nature Biotech.},
  title        = {Drug pipeline 4Q20},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021g). Around the world in a month. <em>NBT</em>, <em>39</em>(2),
130. (<a href="https://doi.org/10.1038/s41587-021-00825-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00825-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {130},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Trop2 deal heats up antibody–drug conjugate space in cancer.
<em>NBT</em>, <em>39</em>(2), 128–130. (<a
href="https://doi.org/10.1038/s41587-021-00824-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gilead’s $21 billion gamble is a dramatic shot to secure a promising first-in-class oncology drug.},
  archive      = {J_NBT},
  author       = {Shaffer, Catherine},
  doi          = {10.1038/s41587-021-00824-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {128-130},
  shortjournal = {Nature Biotech.},
  title        = {Trop2 deal heats up antibody–drug conjugate space in cancer},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Diabetes drug on trial for alzheimer’s. <em>NBT</em>,
<em>39</em>(2), 127. (<a
href="https://doi.org/10.1038/s41587-021-00833-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00833-7},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {127},
  shortjournal = {Nature Biotech.},
  title        = {Diabetes drug on trial for alzheimer’s},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Light-activated genetic therapy to treat blindness enters
clinic. <em>NBT</em>, <em>39</em>(2), 126–127. (<a
href="https://doi.org/10.1038/s41587-021-00823-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Novartis is betting that optogenetics, a technology used by researchers to control cell signaling, is ready for the clinic.},
  archive      = {J_NBT},
  author       = {Ratner, Mark},
  doi          = {10.1038/s41587-021-00823-9},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {126-127},
  shortjournal = {Nature Biotech.},
  title        = {Light-activated genetic therapy to treat blindness enters clinic},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Atalanta pioneers siRNA for neurodegenerative diseases.
<em>NBT</em>, <em>39</em>(2), 125. (<a
href="https://doi.org/10.1038/s41587-021-00838-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00838-2},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {125},
  shortjournal = {Nature Biotech.},
  title        = {Atalanta pioneers siRNA for neurodegenerative diseases},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Dendritic cell revival. <em>NBT</em>, <em>39</em>(2), 124.
(<a href="https://doi.org/10.1038/s41587-021-00835-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00835-5},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {124},
  shortjournal = {Nature Biotech.},
  title        = {Dendritic cell revival},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Drug startups coalesce around condensates. <em>NBT</em>,
<em>39</em>(2), 123–125. (<a
href="https://doi.org/10.1038/s41587-021-00828-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Once a neglected biological phenomenon, condensates are attracting interest from small companies and large pharmas.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-021-00828-4},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {123-125},
  shortjournal = {Nature Biotech.},
  title        = {Drug startups coalesce around condensates},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cleaning house. <em>NBT</em>, <em>39</em>(2), 121. (<a
href="https://doi.org/10.1038/s41587-021-00832-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With drug pricing a priority for the US Biden administration, the biotech industry must clean up its act: stop profiteering from old products and stamp out anticompetitive practices.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-021-00832-8},
  journal      = {Nature Biotechnology},
  month        = {2},
  number       = {2},
  pages        = {121},
  shortjournal = {Nature Biotech.},
  title        = {Cleaning house},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021d). First rounders: Grace colón. <em>NBT</em>, <em>39</em>(2),
155. (<a href="https://doi.org/10.1038/s41587-020-00808-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00808-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {2},
  pages        = {155},
  shortjournal = {Nature Biotech.},
  title        = {First rounders: Grace colón},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Refreshing the biologic pipeline 2020. <em>NBT</em>,
<em>39</em>(2), 135–143. (<a
href="https://doi.org/10.1038/s41587-021-00814-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the absence of face-to-face meetings, FDA and industry implemented regulatory workarounds to maintain drug and biologics approvals. These could be here to stay.},
  archive      = {J_NBT},
  author       = {Hodgson, John},
  doi          = {10.1038/s41587-021-00814-w},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {2},
  pages        = {135-143},
  shortjournal = {Nature Biotech.},
  title        = {Refreshing the biologic pipeline 2020},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021l). People. <em>NBT</em>, <em>39</em>(1), 120. (<a
href="https://doi.org/10.1038/s41587-020-00786-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent moves of note in and around the biotech and pharma industries.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00786-3},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {120},
  shortjournal = {Nature Biotech.},
  title        = {People},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Immersive virtual reality as a competitive training strategy
for the biopharma industry. <em>NBT</em>, <em>39</em>(1), 116–119. (<a
href="https://doi.org/10.1038/s41587-020-00784-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immersive virtual reality simulations tailored to aspiring industry operators in biopharma manufacturing could become a cost-effective alternative to real-life training for teaching practical skills.},
  archive      = {J_NBT},
  author       = {Wismer, Philip and Lopez Cordoba, Ainara and Baceviciute, Sarune and Clauson-Kaas, Frederik and Sommer, Morten Otto Alexander},
  doi          = {10.1038/s41587-020-00784-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {116-119},
  shortjournal = {Nature Biotech.},
  title        = {Immersive virtual reality as a competitive training strategy for the biopharma industry},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021b). Publisher correction: Glycosylase base editors enable
c-to-a and c-to-g base changes. <em>NBT</em>, <em>39</em>(1), 115. (<a
href="https://doi.org/10.1038/s41587-020-0648-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An amendment to this paper has been published and can be accessed via a link at the top of the paper.},
  archive      = {J_NBT},
  author       = {Zhao, Dongdong and Li, Ju and Li, Siwei and Xin, Xiuqing and Hu, Muzi and Price, Marcus A. and Rosser, Susan J. and Bi, Changhao and Zhang, Xueli},
  doi          = {10.1038/s41587-020-0648-3},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {115},
  shortjournal = {Nature Biotech.},
  title        = {Publisher correction: Glycosylase base editors enable C-to-A and C-to-G base changes},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: Supporting US public health experts.
<em>NBT</em>, <em>39</em>(1), 115. (<a
href="https://doi.org/10.1038/s41587-020-00792-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A Correction to this paper has been published: https://doi.org/10.1038/s41587-020-00792-5.},
  archive      = {J_NBT},
  author       = {Cohen, Ron and Francois, Cedric and Crowley, John and Hastings, Paul and King, Rachel and Love, Ted W. and Maraganore, John and McMurry-Heath, Michelle and Levin, Jeremy},
  doi          = {10.1038/s41587-020-00792-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {115},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Supporting US public health experts},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Author correction: Why the european union needs a national
GMO opt-in mechanism. <em>NBT</em>, <em>39</em>(1), 115. (<a
href="https://doi.org/10.1038/s41587-020-00800-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A correction to this paper has been published: https://doi.org/10.1038/s41587-020-00800-8},
  archive      = {J_NBT},
  author       = {Eriksson, Dennis and de Andrade, Eugénia and Bohanec, Borut and Chatzopoulou, Sevasti and Defez, Roberto and Eriksson, Nélida Leiva and van der Meer, Piet and van der Meulen, Bernd and Ritala, Anneli and Sági, László and Schiemann, Joachim and Twardowski, Tomasz and Vaněk, Tomáš},
  doi          = {10.1038/s41587-020-00800-8},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {115},
  shortjournal = {Nature Biotech.},
  title        = {Author correction: Why the european union needs a national GMO opt-in mechanism},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A unified catalog of 204,938 reference genomes from the
human gut microbiome. <em>NBT</em>, <em>39</em>(1), 105–114. (<a
href="https://doi.org/10.1038/s41587-020-0603-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comprehensive, high-quality reference genomes are required for functional characterization and taxonomic assignment of the human gut microbiota. We present the Unified Human Gastrointestinal Genome (UHGG) collection, comprising 204,938 nonredundant genomes from 4,644 gut prokaryotes. These genomes encode &amp;gt;170 million protein sequences, which we collated in the Unified Human Gastrointestinal Protein (UHGP) catalog. The UHGP more than doubles the number of gut proteins in comparison to those present in the Integrated Gene Catalog. More than 70\% of the UHGG species lack cultured representatives, and 40\% of the UHGP lack functional annotations. Intraspecies genomic variation analyses revealed a large reservoir of accessory genes and single-nucleotide variants, many of which are specific to individual human populations. The UHGG and UHGP collections will enable studies linking genotypes to phenotypes in the human gut microbiome. More than 200,000 gut prokaryotic reference genomes and the proteins they encode are collated, providing comprehensive resources for microbiome researchers.},
  archive      = {J_NBT},
  author       = {Almeida, Alexandre and Nayfach, Stephen and Boland, Miguel and Strozzi, Francesco and Beracochea, Martin and Shi, Zhou Jason and Pollard, Katherine S. and Sakharova, Ekaterina and Parks, Donovan H. and Hugenholtz, Philip and Segata, Nicola and Kyrpides, Nikos C. and Finn, Robert D.},
  doi          = {10.1038/s41587-020-0603-3},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {105-114},
  shortjournal = {Nature Biotech.},
  title        = {A unified catalog of 204,938 reference genomes from the human gut microbiome},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Optimization of AsCas12a for combinatorial genetic screens
in human cells. <em>NBT</em>, <em>39</em>(1), 94–104. (<a
href="https://doi.org/10.1038/s41587-020-0600-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cas12a RNA-guided endonucleases are promising tools for multiplexed genetic perturbations because they can process multiple guide RNAs expressed as a single transcript, and subsequently cleave target DNA. However, their widespread adoption has lagged behind Cas9-based strategies due to low activity and the lack of a well-validated pooled screening toolkit. In the present study, we describe the optimization of enhanced Cas12a from Acidaminococcus (enAsCas12a) for pooled, combinatorial genetic screens in human cells. By assaying the activity of thousands of guides, we refine on-target design rules and develop a comprehensive set of off-target rules to predict and exclude promiscuous guides. We also identify 38 direct repeat variants that can substitute for the wild-type sequence. We validate our optimized AsCas12a toolkit by screening for synthetic lethalities in OVCAR8 and A375 cancer cells, discovering an interaction between MARCH5 and WSB2. Finally, we show that enAsCas12a delivers similar performance to Cas9 in genome-wide dropout screens but at greatly reduced library size, which will facilitate screens in challenging models. Improved Cas12a variants and sgRNA design rules enhance genome-wide screens.},
  archive      = {J_NBT},
  author       = {DeWeirdt, Peter C. and Sanson, Kendall R. and Sangree, Annabel K. and Hegde, Mudra and Hanna, Ruth E. and Feeley, Marissa N. and Griffith, Audrey L. and Teng, Teng and Borys, Samantha M. and Strand, Christine and Joung, J. Keith and Kleinstiver, Benjamin P. and Pan, Xuewen and Huang, Alan and Doench, John G.},
  doi          = {10.1038/s41587-020-0600-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {94-104},
  shortjournal = {Nature Biotech.},
  title        = {Optimization of AsCas12a for combinatorial genetic screens in human cells},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Massively parallel kinetic profiling of natural and
engineered CRISPR nucleases. <em>NBT</em>, <em>39</em>(1), 84–93. (<a
href="https://doi.org/10.1038/s41587-020-0646-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Engineered SpCas9s and AsCas12a cleave fewer off-target genomic sites than wild-type (wt) Cas9. However, understanding their fidelity, mechanisms and cleavage outcomes requires systematic profiling across mispaired target DNAs. Here we describe NucleaSeq—nuclease digestion and deep sequencing—a massively parallel platform that measures the cleavage kinetics and time-resolved cleavage products for over 10,000 targets containing mismatches, insertions and deletions relative to the guide RNA. Combining cleavage rates and binding specificities on the same target libraries, we benchmarked five SpCas9 variants and AsCas12a. A biophysical model built from these data sets revealed mechanistic insights into off-target cleavage. Engineered Cas9s, especially Cas9-HF1, dramatically increased cleavage specificity but not binding specificity compared to wtCas9. Surprisingly, AsCas12a cleavage specificity differed little from that of wtCas9. Initial DNA cleavage sites and end trimming varied by nuclease, guide RNA and the positions of mispaired nucleotides. More broadly, NucleaSeq enables rapid, quantitative and systematic comparisons of specificity and cleavage outcomes across engineered and natural nucleases. The enzymatic properties of RNA-guided nucleases are revealed through massively parallel analysis.},
  archive      = {J_NBT},
  author       = {Jones, Stephen K. and Hawkins, John A. and Johnson, Nicole V. and Jung, Cheulhee and Hu, Kuang and Rybarski, James R. and Chen, Janice S. and Doudna, Jennifer A. and Press, William H. and Finkelstein, Ilya J.},
  doi          = {10.1038/s41587-020-0646-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {84-93},
  shortjournal = {Nature Biotech.},
  title        = {Massively parallel kinetic profiling of natural and engineered CRISPR nucleases},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Imaging volumetric dynamics at high speed in mouse and
zebrafish brain with confocal light field microscopy. <em>NBT</em>,
<em>39</em>(1), 74–83. (<a
href="https://doi.org/10.1038/s41587-020-0628-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A detailed understanding of the function of neural networks and how they are supported by a dynamic vascular system requires fast three-dimensional imaging in thick tissues. Here we present confocal light field microscopy, a method that enables fast volumetric imaging in the brain at depths of hundreds of micrometers. It uses a generalized confocal detection scheme that selectively collects fluorescent signals from the in-focus volume and provides optical sectioning capability to improve imaging resolution and sensitivity in thick tissues. We demonstrate recording of whole-brain calcium transients in freely swimming zebrafish larvae and observe behaviorally correlated activities in single neurons during prey capture. Furthermore, in the mouse brain, we detect neural activities at depths of up to 370 μm and track blood cells at 70 Hz over a volume of diameter 800 μm × thickness 150 μm and depth of up to 600 μm. A new imaging method enables deep imaging of neural networks.},
  archive      = {J_NBT},
  author       = {Zhang, Zhenkun and Bai, Lu and Cong, Lin and Yu, Peng and Zhang, Tianlei and Shi, Wanzhuo and Li, Funing and Du, Jiulin and Wang, Kai},
  doi          = {10.1038/s41587-020-0628-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {74-83},
  shortjournal = {Nature Biotech.},
  title        = {Imaging volumetric dynamics at high speed in mouse and zebrafish brain with confocal light field microscopy},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Inducible de novo expression of neoantigens in tumor cells
and mice. <em>NBT</em>, <em>39</em>(1), 64–73. (<a
href="https://doi.org/10.1038/s41587-020-0613-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inducible expression of neoantigens in mice would enable the study of endogenous antigen-specific naïve T cell responses in disease and infection, but has been difficult to generate because leaky antigen expression in the thymus results in central T cell tolerance. Here we develop inversion-induced joined neoantigen (NINJA), using RNA splicing, DNA recombination and three levels of regulation to prevent leakiness and allow tight control over neoantigen expression. We apply NINJA to create tumor cell lines with inducible neoantigen expression, which could be used to study antitumor immunity. We also show that the genetic regulation in NINJA mice bypasses central and peripheral tolerance mechanisms and allows for robust endogenous CD8 and CD4 T cell responses on neoantigen induction in peripheral tissues. NINJA will enable studies of how T cells respond to defined neoantigens in the context of peripheral tolerance, transplantation, autoimmune diseases and cancer. A system to express neoantigens in tumor cells and mice circumvents central T cell tolerance.},
  archive      = {J_NBT},
  author       = {Damo, Martina and Fitzgerald, Brittany and Lu, Yisi and Nader, Mursal and William, Ivana and Cheung, Julie F. and Connolly, Kelli A. and Foster, Gena G. and Akama-Garren, Elliot and Lee, Da-Yae and Chang, Greg P. and Gocheva, Vasilena and Schmidt, Leah M. and Boileve, Alice and Wilson, Josephine H. and Cui, Can and Monroy, Isabel and Gokare, Prashanth and Cabeceiras, Peter and Jacks, Tyler and Joshi, Nikhil S.},
  doi          = {10.1038/s41587-020-0613-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {64-73},
  shortjournal = {Nature Biotech.},
  title        = {Inducible de novo expression of neoantigens in tumor cells and mice},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Engineered off-the-shelf therapeutic t cells resist host
immune rejection. <em>NBT</em>, <em>39</em>(1), 56–63. (<a
href="https://doi.org/10.1038/s41587-020-0601-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Engineered T cells are effective therapies against a range of malignancies, but current approaches rely on autologous T cells, which are difficult and expensive to manufacture. Efforts to develop potent allogeneic T cells that are not rejected by the recipient’s immune system require abrogating both T- and natural killer (NK)-cell responses, which eliminate foreign cells through various mechanisms. In the present study, we engineered a receptor that mediates deletion of activated host T and NK cells, preventing rejection of allogeneic T cells. Our alloimmune defense receptor (ADR) selectively recognizes 4-1BB, a cell surface receptor temporarily upregulated by activated lymphocytes. ADR-expressing T cells resist cellular rejection by targeting alloreactive lymphocytes in vitro and in vivo, while sparing resting lymphocytes. Cells co-expressing chimeric antigen receptors and ADRs persisted in mice and produced sustained tumor eradication in two mouse models of allogeneic T-cell therapy of hematopoietic and solid cancers. This approach enables generation of rejection-resistant, ‘off-the-shelf’, allogeneic T-cell products to produce long-term therapeutic benefit in immunocompetent recipients. Off-the-shelf CAR T cells avoid immune rejection in mouse models of allogeneic cancer cell therapy.},
  archive      = {J_NBT},
  author       = {Mo, Feiyan and Watanabe, Norihiro and McKenna, Mary K. and Hicks, M. John and Srinivasan, Madhuwanti and Gomes-Silva, Diogo and Atilla, Erden and Smith, Tyler and Ataca Atilla, Pinar and Ma, Royce and Quach, David and Heslop, Helen E. and Brenner, Malcolm K. and Mamonkin, Maksim},
  doi          = {10.1038/s41587-020-0601-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {56-63},
  shortjournal = {Nature Biotech.},
  title        = {Engineered off-the-shelf therapeutic t cells resist host immune rejection},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). A long-term study of AAV gene therapy in dogs with
hemophilia a identifies clonal expansions of transduced liver cells.
<em>NBT</em>, <em>39</em>(1), 47–55. (<a
href="https://doi.org/10.1038/s41587-020-0741-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nine dogs with hemophilia A were treated with adeno-associated viral (AAV) gene therapy and followed for up to 10 years. Administration of AAV8 or AAV9 vectors expressing canine factor VIII (AAV-cFVIII) corrected the FVIII deficiency to 1.9–11.3\% of normal FVIII levels. In two of nine dogs, levels of FVIII activity increased gradually starting about 4 years after treatment. None of the dogs showed evidence of tumors or altered liver function. Analysis of integration sites in liver samples from six treated dogs identified 1,741 unique AAV integration events in genomic DNA and expanded cell clones in five dogs, with 44\% of the integrations near genes involved in cell growth. All recovered integrated vectors were partially deleted and/or rearranged. Our data suggest that the increase in FVIII protein expression in two dogs may have been due to clonal expansion of cells harboring integrated vectors. These results support the clinical development of liver-directed AAV gene therapy for hemophilia A, while emphasizing the importance of long-term monitoring for potential genotoxicity. AAV therapy in dogs leads to clonal expansions of transduced cells.},
  archive      = {J_NBT},
  author       = {Nguyen, Giang N. and Everett, John K. and Kafle, Samita and Roche, Aoife M. and Raymond, Hayley E. and Leiby, Jacob and Wood, Christian and Assenmacher, Charles-Antoine and Merricks, Elizabeth P. and Long, C. Tyler and Kazazian, Haig H. and Nichols, Timothy C. and Bushman, Frederic D. and Sabatino, Denise E.},
  doi          = {10.1038/s41587-020-0741-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {47-55},
  shortjournal = {Nature Biotech.},
  title        = {A long-term study of AAV gene therapy in dogs with hemophilia a identifies clonal expansions of transduced liver cells},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). CRISPR c-to-g base editors for inducing targeted DNA
transversions in human cells. <em>NBT</em>, <em>39</em>(1), 41–46. (<a
href="https://doi.org/10.1038/s41587-020-0609-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CRISPR-guided DNA cytosine and adenine base editors are widely used for many applications1–4 but primarily create DNA base transitions (that is, pyrimidine-to-pyrimidine or purine-to-purine). Here we describe the engineering of two base editor architectures that can efficiently induce targeted C-to-G base transversions, with reduced levels of unwanted C-to-W (W = A or T) and indel mutations. One of these C-to-G base editors (CGBE1), consists of an RNA-guided Cas9 nickase, an Escherichia coli–derived uracil DNA N-glycosylase (eUNG) and a rat APOBEC1 cytidine deaminase variant (R33A) previously shown to have reduced off-target RNA and DNA editing activities5,6. We show that CGBE1 can efficiently induce C-to-G edits, particularly in AT-rich sequence contexts in human cells. We also removed the eUNG domain to yield miniCGBE1, which reduced indel frequencies but only modestly decreased editing efficiency. CGBE1 and miniCGBE1 enable C-to-G edits and will serve as a basis for optimizing C-to-G base editors for research and therapeutic applications. A new base editor enables the creation of C-to-G base changes in human cells.},
  archive      = {J_NBT},
  author       = {Kurt, Ibrahim C. and Zhou, Ronghao and Iyer, Sowmya and Garcia, Sara P. and Miller, Bret R. and Langner, Lukas M. and Grünewald, Julian and Joung, J. Keith},
  doi          = {10.1038/s41587-020-0609-x},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {41-46},
  shortjournal = {Nature Biotech.},
  title        = {CRISPR C-to-G base editors for inducing targeted DNA transversions in human cells},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Glycosylase base editors enable c-to-a and c-to-g base
changes. <em>NBT</em>, <em>39</em>(1), 35–40. (<a
href="https://doi.org/10.1038/s41587-020-0592-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Current base editors (BEs) catalyze only base transitions (C to T and A to G) and cannot produce base transversions. Here we present BEs that cause C-to-A transversions in Escherichia coli and C-to-G transversions in mammalian cells. These glycosylase base editors (GBEs) consist of a Cas9 nickase, a cytidine deaminase and a uracil-DNA glycosylase (Ung). Ung excises the U base created by the deaminase, forming an apurinic/apyrimidinic (AP) site that initiates the DNA repair process. In E. coli, we used activation-induced cytidine deaminase (AID) to construct AID-nCas9-Ung and found that it converts C to A with an average editing specificity of 93.8\% ± 4.8\% and editing efficiency of 87.2\% ± 6.9\%. For use in mammalian cells, we replaced AID with rat APOBEC1 (APOBEC-nCas9-Ung). We tested APOBEC-nCas9-Ung at 30 endogenous sites, and we observed C-to-G conversions with a high editing specificity at the sixth position of the protospacer between 29.7\% and 92.2\% and an editing efficiency between 5.3\% and 53.0\%. APOBEC-nCas9-Ung supplements the current adenine and cytidine BEs (ABE and CBE, respectively) and could be used to target G/C disease-causing mutations. New base editors change C to A in bacteria and C to G in mammalian cells.},
  archive      = {J_NBT},
  author       = {Zhao, Dongdong and Li, Ju and Li, Siwei and Xin, Xiuqing and Hu, Muzi and Price, Marcus A. and Rosser, Susan J. and Bi, Changhao and Zhang, Xueli},
  doi          = {10.1038/s41587-020-0592-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {35-40},
  shortjournal = {Nature Biotech.},
  title        = {Glycosylase base editors enable C-to-A and C-to-G base changes},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Cell type prioritization in single-cell data. <em>NBT</em>,
<em>39</em>(1), 30–34. (<a
href="https://doi.org/10.1038/s41587-020-0605-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present Augur, a method to prioritize the cell types most responsive to biological perturbations in single-cell data. Augur employs a machine-learning framework to quantify the separability of perturbed and unperturbed cells within a high-dimensional space. We validate our method on single-cell RNA sequencing, chromatin accessibility and imaging transcriptomics datasets, and show that Augur outperforms existing methods based on differential gene expression. Augur identified the neural circuits restoring locomotion in mice following spinal cord neurostimulation. The cell types affected by biological perturbations in complex tissues are uncovered by single-cell analysis.},
  archive      = {J_NBT},
  author       = {Skinnider, Michael A. and Squair, Jordan W. and Kathe, Claudia and Anderson, Mark A. and Gautier, Matthieu and Matson, Kaya J. E. and Milano, Marco and Hutson, Thomas H. and Barraud, Quentin and Phillips, Aaron A. and Foster, Leonard J. and La Manno, Gioele and Levine, Ariel J. and Courtine, Grégoire},
  doi          = {10.1038/s41587-020-0605-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {30-34},
  shortjournal = {Nature Biotech.},
  title        = {Cell type prioritization in single-cell data},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Reply to: EEG-based model and antidepressant response.
<em>NBT</em>, <em>39</em>(1), 28–29. (<a
href="https://doi.org/10.1038/s41587-020-0738-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Wu, Wei and Pizzagall, Diego A. and Trivedi, Madhukar H. and Etkin, Amit},
  doi          = {10.1038/s41587-020-0738-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {28-29},
  shortjournal = {Nature Biotech.},
  title        = {Reply to: EEG-based model and antidepressant response},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). EEG-based model and antidepressant response. <em>NBT</em>,
<em>39</em>(1), 27. (<a
href="https://doi.org/10.1038/s41587-020-00768-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Nilsonne, Gustav and Harrell, Frank E.},
  doi          = {10.1038/s41587-020-00768-5},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {27},
  shortjournal = {Nature Biotech.},
  title        = {EEG-based model and antidepressant response},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Safety questions for AAV gene therapy. <em>NBT</em>,
<em>39</em>(1), 24–26. (<a
href="https://doi.org/10.1038/s41587-020-00756-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A long-term study of AAV therapy in dogs finds clonal expansions and vector integrations.},
  archive      = {J_NBT},
  author       = {Venditti, Charles P.},
  doi          = {10.1038/s41587-020-00756-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {24-26},
  shortjournal = {Nature Biotech.},
  title        = {Safety questions for AAV gene therapy},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Circular RNA. <em>NBT</em>, <em>39</em>(1), 23. (<a
href="https://doi.org/10.1038/s41587-020-00787-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent patents regarding the preparation, amplification and therapeutic use of circular RNA.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00787-2},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {23},
  shortjournal = {Nature Biotech.},
  title        = {Circular RNA},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). An administrative fix for manufacturing process patent
thickets. <em>NBT</em>, <em>39</em>(1), 20–22. (<a
href="https://doi.org/10.1038/s41587-020-00780-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biosimilar manufacturing process patent applications first filed more than a year after the biologic is approved, which represent the largest source of patent assertions in biosimilar litigation, could be addressed without requiring intervention by either the courts or Congress.},
  archive      = {J_NBT},
  author       = {Rai, Arti K. and Price, W. Nicholson},
  doi          = {10.1038/s41587-020-00780-9},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {20-22},
  shortjournal = {Nature Biotech.},
  title        = {An administrative fix for manufacturing process patent thickets},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Ananda mohan “al” chakrabarty 1938–2020. <em>NBT</em>,
<em>39</em>(1), 18–19. (<a
href="https://doi.org/10.1038/s41587-020-00785-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Davey, Neil and Rader, Randall Ray and Chakravarti, Debabrata},
  doi          = {10.1038/s41587-020-00785-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {18-19},
  shortjournal = {Nature Biotech.},
  title        = {Ananda mohan ‘Al’ chakrabarty 1938–2020},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). The organoid cell atlas. <em>NBT</em>, <em>39</em>(1),
13–17. (<a href="https://doi.org/10.1038/s41587-020-00762-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  author       = {Bock, Christoph and Boutros, Michael and Camp, J. Gray and Clarke, Laura and Clevers, Hans and Knoblich, Juergen A. and Liberali, Prisca and Regev, Aviv and Rios, Anne C. and Stegle, Oliver and Stunnenberg, Hendrik G. and Teichmann, Sarah A. and Treutlein, Barbara and Vries, Robert G. J.},
  doi          = {10.1038/s41587-020-00762-x},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {13-17},
  shortjournal = {Nature Biotech.},
  title        = {The organoid cell atlas},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021h). Around the world in a month. <em>NBT</em>, <em>39</em>(1),
12. (<a href="https://doi.org/10.1038/s41587-020-00789-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00789-0},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {12},
  shortjournal = {Nature Biotech.},
  title        = {Around the world in a month},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). AstraZeneca joins russia to boost coronavirus vaccine.
<em>NBT</em>, <em>39</em>(1), 11. (<a
href="https://doi.org/10.1038/s41587-020-00804-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00804-4},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {11},
  shortjournal = {Nature Biotech.},
  title        = {AstraZeneca joins russia to boost coronavirus vaccine},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Antibody–drug conjugates for cancer score with ROR1.
<em>NBT</em>, <em>39</em>(1), 10. (<a
href="https://doi.org/10.1038/s41587-020-00798-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00798-z},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {10},
  shortjournal = {Nature Biotech.},
  title        = {Antibody–drug conjugates for cancer score with ROR1},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). Hunt for improved monoclonals against coronavirus gathers
pace. <em>NBT</em>, <em>39</em>(1), 9–12. (<a
href="https://doi.org/10.1038/s41587-020-00791-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Following mixed results in the clinic against COVID-19, antibody engineers are further optimizing SARS-CoV-2 antibodies with the aim of improving outcomes.},
  archive      = {J_NBT},
  author       = {Garber, Ken},
  doi          = {10.1038/s41587-020-00791-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {9-12},
  shortjournal = {Nature Biotech.},
  title        = {Hunt for improved monoclonals against coronavirus gathers pace},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Thumbs up for cholesterol-slashing siRNA blockbuster.
<em>NBT</em>, <em>39</em>(1), 9. (<a
href="https://doi.org/10.1038/s41587-020-00799-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00799-y},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {9},
  shortjournal = {Nature Biotech.},
  title        = {Thumbs up for cholesterol-slashing siRNA blockbuster},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). FDA clears first drug for premature aging disease.
<em>NBT</em>, <em>39</em>(1), 7. (<a
href="https://doi.org/10.1038/s41587-020-00802-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00802-6},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {7},
  shortjournal = {Nature Biotech.},
  title        = {FDA clears first drug for premature aging disease},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021e). First small-molecule drug targeting RNA gains momentum.
<em>NBT</em>, <em>39</em>(1), 6–8. (<a
href="https://doi.org/10.1038/s41587-020-00788-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The new drug not only is a boon for patients with spinal muscular atrophy, but also may spearhead the development of further chemical RNA modulators for a broad range of conditions.},
  archive      = {J_NBT},
  author       = {Sheridan, Cormac},
  doi          = {10.1038/s41587-020-00788-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {6-8},
  shortjournal = {Nature Biotech.},
  title        = {First small-molecule drug targeting RNA gains momentum},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021a). BTK blockers make headway in multiple sclerosis.
<em>NBT</em>, <em>39</em>(1), 3–5. (<a
href="https://doi.org/10.1038/s41587-020-00790-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By targeting both arms of the immune system, not just B cells, brain-penetrant inhibitors of Bruton’s tyrosine kinase could improve on anti-CD20 therapy for patients with multiple sclerosis.},
  archive      = {J_NBT},
  author       = {Dolgin, Elie},
  doi          = {10.1038/s41587-020-00790-7},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {3-5},
  shortjournal = {Nature Biotech.},
  title        = {BTK blockers make headway in multiple sclerosis},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
<li><details>
<summary>
(2021). Messengers of hope. <em>NBT</em>, <em>39</em>(1), 1. (<a
href="https://doi.org/10.1038/s41587-020-00807-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Emergency Use Authorizations for two mRNA COVID-19 vaccines represent a turning point in the pandemic. They also herald a new era for vaccinology.},
  archive      = {J_NBT},
  doi          = {10.1038/s41587-020-00807-1},
  journal      = {Nature Biotechnology},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nature Biotech.},
  title        = {Messengers of hope},
  volume       = {39},
  year         = {2021},
}
</textarea>
</details></li>
</ul>

</body>
</html>
