<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATEE_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natee---316">NATEE - 316</h2>
<ul>
<li><details>
<summary>
(2023b). Author correction: A global biodiversity observing system
to unite monitoring and guide action. <em>NATEE</em>, <em>7</em>(12),
2173. (<a href="https://doi.org/10.1038/s41559-023-02263-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Gonzalez, Andrew and Vihervaara, Petteri and Balvanera, Patricia and Bates, Amanda E. and Bayraktarov, Elisa and Bellingham, Peter J. and Bruder, Andreas and Campbell, Jillian and Catchen, Michael D. and Cavender-Bares, Jeannine and Chase, Jonathan and Coops, Nicholas and Costello, Mark J. and Czúcz, Bálint and Delavaud, Aurélie and Dornelas, Maria and Dubois, Grégoire and Duffy, Emmett J. and Eggermont, Hilde and Fernandez, Miguel and Fernandez, Nestor and Ferrier, Simon and Geller, Gary N. and Gill, Michael and Gravel, Dominique and Guerra, Carlos A. and Guralnick, Robert and Harfoot, Michael and Hirsch, Tim and Hoban, Sean and Hughes, Alice C. and Hugo, Wim and Hunter, Margaret E. and Isbell, Forest and Jetz, Walter and Juergens, Norbert and Kissling, W. Daniel and Krug, Cornelia B. and Kullberg, Peter and Le Bras, Yvan and Leung, Brian and Londoño-Murcia, Maria Cecilia and Lord, Jean-Michel and Loreau, Michel and Luers, Amy and Ma, Keping and MacDonald, Anna J. and Maes, Joachim and McGeoch, Melodie and Mihoub, Jean Baptiste and Millette, Katie L. and Molnar, Zsolt and Montes, Enrique and Mori, Akira S. and Muller-Karger, Frank E. and Muraoka, Hiroyuki and Nakaoka, Masahiro and Navarro, Laetitia and Newbold, Tim and Niamir, Aidin and Obura, David and O’Connor, Mary and Paganini, Marc and Pelletier, Dominique and Pereira, Henrique and Poisot, Timothée and Pollock, Laura J. and Purvis, Andy and Radulovici, Adriana and Rocchini, Duccio and Roeoesli, Claudia and Schaepman, Michael and Schaepman-Strub, Gabriela and Schmeller, Dirk S. and Schmiedel, Ute and Schneider, Fabian D. and Shakya, Mangal Man and Skidmore, Andrew and Skowno, Andrew L. and Takeuchi, Yayioi and Tuanmu, Mao-Ning and Turak, Eren and Turner, Woody and Urban, Mark C. and Urbina-Cardona, Nicolás and Valbuena, Ruben and Van de Putte, Anton and van Havre, Basile and Wingate, Vladimir Ruslan and Wright, Elaine and Torrelio, Carlos Zambrana},
  doi          = {10.1038/s41559-023-02263-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2173},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: A global biodiversity observing system to unite monitoring and guide action},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genome sequences of 36,000- to 37,000-year-old modern humans
at buran-kaya III in crimea. <em>NATEE</em>, <em>7</em>(12), 2160–2172.
(<a href="https://doi.org/10.1038/s41559-023-02211-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Populations genetically related to present-day Europeans first appeared in Europe at some point after 38,000–40,000 years ago, following a cold period of severe climatic disruption. These new migrants would eventually replace the pre-existing modern human ancestries in Europe, but initial interactions between these groups are unclear due to the lack of genomic evidence from the earliest periods of the migration. Here we describe the genomes of two 36,000–37,000-year-old individuals from Buran-Kaya III in Crimea as belonging to this newer migration. Both genomes share the highest similarity to Gravettian-associated individuals found several thousand years later in southwestern Europe. These genomes also revealed that the population turnover in Europe after 40,000 years ago was accompanied by admixture with pre-existing modern human populations. European ancestry before 40,000 years ago persisted not only at Buran-Kaya III but is also found in later Gravettian-associated populations of western Europe and Mesolithic Caucasus populations. Two Early Upper Palaeolithic genomes of humans that lived 36,000–37,000 years ago in Crimea reveal migration and admixture dynamics against a backdrop of climate change.},
  archive      = {J_NATEE},
  author       = {Bennett, E. Andrew and Parasayan, Oğuzhan and Prat, Sandrine and Péan, Stéphane and Crépin, Laurent and Yanevich, Alexandr and Grange, Thierry and Geigl, Eva-Maria},
  doi          = {10.1038/s41559-023-02211-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2160-2172},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Genome sequences of 36,000- to 37,000-year-old modern humans at buran-kaya III in crimea},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A multifunctional wnt regulator underlies the evolution of
rodent stripe patterns. <em>NATEE</em>, <em>7</em>(12), 2143–2159. (<a
href="https://doi.org/10.1038/s41559-023-02213-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Animal pigment patterns are excellent models to elucidate mechanisms of biological organization. Although theoretical simulations, such as Turing reaction–diffusion systems, recapitulate many animal patterns, they are insufficient to account for those showing a high degree of spatial organization and reproducibility. Here, we study the coat of the African striped mouse (Rhabdomys pumilio) to uncover how periodic stripes form. Combining transcriptomics, mathematical modelling and mouse transgenics, we show that the Wnt modulator Sfrp2 regulates the distribution of hair follicles and establishes an embryonic prepattern that foreshadows pigment stripes. Moreover, by developing in vivo gene editing in striped mice, we find that Sfrp2 knockout is sufficient to alter the stripe pattern. Strikingly, mutants exhibited changes in pigmentation, revealing that Sfrp2 also regulates hair colour. Lastly, through evolutionary analyses, we find that striped mice have evolved lineage-specific changes in regulatory elements surrounding Sfrp2, many of which may be implicated in modulating the expression of this gene. Altogether, our results show that a single factor controls coat pattern formation by acting both as an orienting signalling mechanism and a modulator of pigmentation. More broadly, our work provides insights into how spatial patterns are established in developing embryos and the mechanisms by which phenotypic novelty originates. Combining transcriptomics, mathematical modelling and in vivo gene editing, this study shows that Sfrp2 contributes to stripe formation and hair colour in the African striped mouse.},
  archive      = {J_NATEE},
  author       = {Johnson, Matthew R. and Li, Sha and Guerrero-Juarez, Christian F. and Miller, Pearson and Brack, Benjamin J. and Mereby, Sarah A. and Moreno, Jorge A. and Feigin, Charles Y. and Gaska, Jenna and Rivera-Perez, Jaime A. and Nie, Qing and Ploss, Alexander and Shvartsman, Stanislav Y. and Mallarino, Ricardo},
  doi          = {10.1038/s41559-023-02213-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2143-2159},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A multifunctional wnt regulator underlies the evolution of rodent stripe patterns},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell transcriptomics reveals the brain evolution of
web-building spiders. <em>NATEE</em>, <em>7</em>(12), 2125–2142. (<a
href="https://doi.org/10.1038/s41559-023-02238-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spiders are renowned for their efficient capture of flying insects using intricate aerial webs. How the spider nervous systems evolved to cope with this specialized hunting strategy and various environmental clues in an aerial space remains unknown. Here we report a brain-cell atlas of &amp;gt;30,000 single-cell transcriptomes from a web-building spider (Hylyphantes graminicola). Our analysis revealed the preservation of ancestral neuron types in spiders, including the potential coexistence of noradrenergic and octopaminergic neurons, and many peptidergic neuronal types that are lost in insects. By comparing the genome of two newly sequenced plesiomorphic burrowing spiders with three aerial web-building spiders, we found that the positively selected genes in the ancestral branch of web-building spiders were preferentially expressed (42%) in the brain, especially in the three mushroom body-like neuronal types. By gene enrichment analysis and RNAi experiments, these genes were suggested to be involved in the learning and memory pathway and may influence the spiders’ web-building and hunting behaviour. Our results provide key sources for understanding the evolution of behaviour in spiders and reveal how molecular evolution drives neuron innovation and the diversification of associated complex behaviours. A brain-cell atlas of a web-building spider and genomic comparisons between web-building and burrowing species identifies preserved ancestral neuron types and candidate neuronal genes that may be involved in the learning and memory pathways underlying web-building and hunting behaviours.},
  archive      = {J_NATEE},
  author       = {Jin, Pengyu and Zhu, Bingyue and Jia, Yinjun and Zhang, Yiming and Wang, Wei and Shen, Yunxiao and Zhong, Yu and Zheng, Yami and Wang, Yang and Tong, Yan and Zhang, Wei and Li, Shuqiang},
  doi          = {10.1038/s41559-023-02238-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2125-2142},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Single-cell transcriptomics reveals the brain evolution of web-building spiders},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evolutionary dynamics of whole-body regeneration across
planarian flatworms. <em>NATEE</em>, <em>7</em>(12), 2108–2124. (<a
href="https://doi.org/10.1038/s41559-023-02221-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Regenerative abilities vary dramatically across animals. Even amongst planarian flatworms, well-known for complete regeneration from tiny body fragments, some species have restricted regeneration abilities while others are almost entirely regeneration incompetent. Here, we assemble a diverse live collection of 40 planarian species to probe the evolution of head regeneration in the group. Combining quantification of species-specific head-regeneration abilities with a comprehensive transcriptome-based phylogeny reconstruction, we show multiple independent transitions between robust whole-body regeneration and restricted regeneration in freshwater species. RNA-mediated genetic interference inhibition of canonical Wnt signalling in RNA-mediated genetic interference-sensitive species bypassed all head-regeneration defects, suggesting that the Wnt pathway is linked to the emergence of planarian regeneration defects. Our finding that Wnt signalling has multiple roles in the reproductive system of the model species Schmidtea mediterranea raises the possibility that a trade-off between egg-laying, asexual reproduction by fission/regeneration and Wnt signalling drives regenerative trait evolution. Although quantitative comparisons of Wnt signalling levels, yolk content and reproductive strategy across our species collection remained inconclusive, they revealed divergent Wnt signalling roles in the reproductive system of planarians. Altogether, our study establishes planarians as a model taxon for comparative regeneration research and presents a framework for the mechanistic evolution of regenerative abilities. A comparative analysis of head-regeneration capacity across planarian species in a phylogenetic context reveals multiple Wnt-dependent transitions in head-regeneration ability and proposes Wnt functions in the reproductive system as possible evolutionary drivers.},
  archive      = {J_NATEE},
  author       = {Vila-Farré, Miquel and Rozanski, Andrei and Ivanković, Mario and Cleland, James and Brand, Jeremias N. and Thalen, Felix and Grohme, Markus A. and von Kannen, Stephanie and Grosbusch, Alexandra L. and Vu, Hanh T.-K. and Prieto, Carlos E. and Carbayo, Fernando and Egger, Bernhard and Bleidorn, Christoph and Rasko, John E. J. and Rink, Jochen C.},
  doi          = {10.1038/s41559-023-02221-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2108-2124},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolutionary dynamics of whole-body regeneration across planarian flatworms},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Community composition and the environment modulate the
population dynamics of type VI secretion in human gut bacteria.
<em>NATEE</em>, <em>7</em>(12), 2092–2107. (<a
href="https://doi.org/10.1038/s41559-023-02230-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the relationship between the composition of the human gut microbiota and the ecological forces shaping it is of great importance; however, knowledge of the biogeographical and ecological relationships between physically interacting taxa is limited. Interbacterial antagonism may play an important role in gut community dynamics, yet the conditions under which antagonistic behaviour is favoured or disfavoured by selection in the gut are not well understood. Here, using genomics, we show that a species-specific type VI secretion system (T6SS) repeatedly acquires inactivating mutations in Bacteroides fragilis in the human gut. This result implies a fitness cost to the T6SS, but we could not identify laboratory conditions under which such a cost manifests. Strikingly, experiments in mice illustrate that the T6SS can be favoured or disfavoured in the gut depending on the strains and species in the surrounding community and their susceptibility to T6SS antagonism. We use ecological modelling to explore the conditions that could underlie these results and find that community spatial structure modulates interaction patterns among bacteria, thereby modulating the costs and benefits of T6SS activity. Our findings point towards new integrative models for interrogating the evolutionary dynamics of type VI secretion and other modes of antagonistic interaction in microbiomes. A combination of phylogenomics, mouse gut experimental assays and ecological modelling shows how community structure controls the fitness costs and benefits of the type VI secretion system in the human gut symbiont Bacteroides fragilis.},
  archive      = {J_NATEE},
  author       = {Robitaille, Sophie and Simmons, Emilia L. and Verster, Adrian J. and McClure, Emily Ann and Royce, Darlene B. and Trus, Evan and Swartz, Kerry and Schultz, Daniel and Nadell, Carey D. and Ross, Benjamin D.},
  doi          = {10.1038/s41559-023-02230-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2092-2107},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Community composition and the environment modulate the population dynamics of type VI secretion in human gut bacteria},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The evolution of short- and long-range weapons for bacterial
competition. <em>NATEE</em>, <em>7</em>(12), 2080–2091. (<a
href="https://doi.org/10.1038/s41559-023-02234-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bacteria possess a diverse range of mechanisms for inhibiting competitors, including bacteriocins, tailocins, type VI secretion systems and contact-dependent inhibition (CDI). Why bacteria have evolved such a wide array of weapon systems remains a mystery. Here we develop an agent-based model to compare short-range weapons that require cell–cell contact, with long-range weapons that rely on diffusion. Our model predicts that contact weapons are useful when an attacking strain is outnumbered, facilitating invasion and establishment. By contrast, ranged weapons tend to be effective only when attackers are abundant. We test our predictions with the opportunistic pathogen Pseudomonas aeruginosa, which naturally carries multiple weapons, including CDI and diffusing tailocins. As predicted, short-range CDI can function at low and high frequencies, while long-range tailocins require high frequency and cell density to function effectively. Head-to-head competition experiments with the two weapon types further support our predictions: a tailocin attacker defeats CDI only when it is numerically dominant, but then we find it can be devastating. Finally, we show that the two weapons work well together when one strain employs both. We conclude that short- and long-range weapons serve different functions and allow bacteria to fight both as individuals and as a group. An agent-based model suggests that bacteria use direct-contact systems for inhibiting competitors when the attacking strain is outnumbered, and long-range diffusion systems when the attacker is common. These predictions are supported by competition experiments with Pseudomonas aeruginosa, which also show that strains can use both types of system in conjunction.},
  archive      = {J_NATEE},
  author       = {Booth, Sean C. and Smith, William P. J. and Foster, Kevin R.},
  doi          = {10.1038/s41559-023-02234-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2080-2091},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The evolution of short- and long-range weapons for bacterial competition},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Selection of a de novo gene that can promote survival of
escherichia coli by modulating protein homeostasis pathways.
<em>NATEE</em>, <em>7</em>(12), 2067–2079. (<a
href="https://doi.org/10.1038/s41559-023-02224-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular novelty can emerge when non-functional loci become functional genes in a process termed de novo gene birth. But how proteins with random amino acid sequences beneficially integrate into existing cellular pathways remains poorly understood. We screened ~108 genes, generated from random nucleotide sequences and devoid of homology to natural genes, for their ability to rescue growth arrest of Escherichia coli cells producing the ribonuclease toxin MazF. We identified ~2,000 genes that could promote growth, probably by reducing transcription from the promoter driving toxin expression. Additionally, one random protein, named Random antitoxin of MazF (RamF), modulated protein homeostasis by interacting with chaperones, leading to MazF proteolysis and a consequent loss of its toxicity. Finally, we demonstrate that random proteins can improve during evolution by identifying beneficial mutations that turned RamF into a more efficient inhibitor. Our work provides a mechanistic basis for how de novo gene birth can produce functional proteins that effectively benefit cells evolving under stress. Screening 100 million random genes, which lack homology to natural sequences, for their ability to rescue growth arrest of Escherichia coli cells caused by the toxin MazF, the authors find ~2,000 hits and demonstrate that one (RamF) confers resistance through interaction with chaperones and induction of toxin proteolysis.},
  archive      = {J_NATEE},
  author       = {Frumkin, Idan and Laub, Michael T.},
  doi          = {10.1038/s41559-023-02224-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2067-2079},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Selection of a de novo gene that can promote survival of escherichia coli by modulating protein homeostasis pathways},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Recent co-evolution of two pandemic plant diseases in a
multi-hybrid swarm. <em>NATEE</em>, <em>7</em>(12), 2055–2066. (<a
href="https://doi.org/10.1038/s41559-023-02237-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most plant pathogens exhibit host specificity but when former barriers to infection break down, new diseases can rapidly emerge. For a number of fungal diseases, there is increasing evidence that hybridization plays a major role in driving host jumps. However, the relative contributions of existing variation versus new mutations in adapting to new host(s) is unclear. Here we reconstruct the evolutionary history of two recently emerged populations of the fungus Pyricularia oryzae that are responsible for two new plant diseases: wheat blast and grey leaf spot of ryegrasses. We provide evidence that wheat blast/grey leaf spot evolved through two distinct mating episodes: the first occurred ~60 years ago, when a fungal individual adapted to Eleusine mated with another individual from Urochloa. Then, about 10 years later, a single progeny from this cross underwent a series of matings with a small number of individuals from three additional host-specialized populations. These matings introduced non-functional alleles of two key host-specificity factors, whose recombination in a multi-hybrid swarm probably facilitated the host jump. We show that very few mutations have arisen since the founding event and a majority are private to individual isolates. Thus, adaptation to the wheat or Lolium hosts appears to have been instantaneous, and driven entirely by selection on repartitioned standing variation, with no obvious role for newly formed mutations. Analysing the evolutionary history of two recent populations of a pathogenic fungus that infects wheat and ryegrasses, the authors show how recombination and selection on standing genetic variation contributed to adaptation to the new hosts.},
  archive      = {J_NATEE},
  author       = {Rahnama, Mostafa and Condon, Bradford and Ascari, João P. and Dupuis, Julian R. and Del Ponte, Emerson M. and Pedley, Kerry F. and Martinez, Sebastián and Valent, Barbara and Farman, Mark L.},
  doi          = {10.1038/s41559-023-02237-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2055-2066},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Recent co-evolution of two pandemic plant diseases in a multi-hybrid swarm},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetics re-establish the utility of 2-methylhopanes as
cyanobacterial biomarkers before 750 million years ago. <em>NATEE</em>,
<em>7</em>(12), 2045–2054. (<a
href="https://doi.org/10.1038/s41559-023-02223-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fossilized lipids offer a rare glimpse into ancient ecosystems. 2-Methylhopanes in sedimentary rocks were once used to infer the importance of cyanobacteria as primary producers throughout geological history. However, the discovery of hopanoid C-2 methyltransferase (HpnP) in Alphaproteobacteria led to the downfall of this molecular proxy. In the present study, we re-examined the distribution of HpnP in a new phylogenetic framework including recently proposed candidate phyla and re-interpreted a revised geological record of 2-methylhopanes based on contamination-free samples. We show that HpnP was probably present in the last common ancestor of cyanobacteria, while the gene appeared in Alphaproteobacteria only around 750 million years ago (Ma). A subsequent rise of sedimentary 2-methylhopanes around 600 Ma probably reflects the expansion of Alphaproteobacteria that coincided with the rise of eukaryotic algae—possibly connected by algal dependency on microbially produced vitamin B12. Our findings re-establish 2-methylhopanes as cyanobacterial biomarkers before 750 Ma and thus as a potential tool to measure the importance of oxygenic cyanobacteria as primary producers on early Earth. Our study illustrates how genetics can improve the diagnostic value of biomarkers and refine the reconstruction of early ecosystems. Using phylogenetic analysis of a hopanoid-producing gene in bacteria, the authors establish fossil 2-methylhopanes as lipid biomarkers for aerobic cyanobacteria and reveal the role of cyanobacteria in primary production before 750 million years ago.},
  archive      = {J_NATEE},
  author       = {Hoshino, Yosuke and Nettersheim, Benjamin J. and Gold, David A. and Hallmann, Christian and Vinnichenko, Galina and van Maldegem, Lennart M. and Bishop, Caleb and Brocks, Jochen J. and Gaucher, Eric A.},
  doi          = {10.1038/s41559-023-02223-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2045-2054},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Genetics re-establish the utility of 2-methylhopanes as cyanobacterial biomarkers before 750 million years ago},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Freshwater fish diversity in the western amazon basin shaped
by andean uplift since the late cretaceous. <em>NATEE</em>,
<em>7</em>(12), 2037–2044. (<a
href="https://doi.org/10.1038/s41559-023-02220-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {South America is home to the highest freshwater fish biodiversity on Earth, and the hotspot of species richness is located in the western Amazon basin. The location of this hotspot is enigmatic, as it is inconsistent with the pattern observed in river systems across the world of increasing species richness towards a river’s mouth. Here we investigate the role of river capture events caused by Andean mountain building and repeated episodes of flooding in western Amazonia in shaping the modern-day richness pattern of freshwater fishes in South America, and in Amazonia in particular. To this end, we combine a reconstruction of river networks since 80 Ma with a mechanistic model simulating dispersal, allopatric speciation and extinction over the dynamic landscape of rivers and lakes. We show that Andean mountain building and consequent numerous small river capture events in western Amazonia caused freshwater habitats to be highly dynamic, leading to high diversification rates and exceptional richness. The history of marine incursions and lakes, including the Miocene Pebas mega-wetland system in western Amazonia, played a secondary role. Reconstructing river networks over the past 80 Ma reveals the role of Andean uplift in creating a dynamic habitat leading to increased fish species diversity in western Amazonia.},
  archive      = {J_NATEE},
  author       = {Boschman, Lydian M. and Carraro, Luca and Cassemiro, Fernanda A. S. and de Vries, Jorad and Altermatt, Florian and Hagen, Oskar and Hoorn, Carina and Pellissier, Loïc},
  doi          = {10.1038/s41559-023-02220-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2037-2044},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Freshwater fish diversity in the western amazon basin shaped by andean uplift since the late cretaceous},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fire activity and deforestation in remote oceanian islands
caused by anthropogenic and climate interactions. <em>NATEE</em>,
<em>7</em>(12), 2028–2036. (<a
href="https://doi.org/10.1038/s41559-023-02212-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Remote islands in the Pacific Ocean (Oceania) experienced dramatic environmental transformations after initial human settlement in the past 3,000 yr. Here, human causality of this environmental degradation has been unquestioned and viewed as evidence of the inherent destructive tendencies of human societies in both archaeological and popular discourse. We use charcoal and stable carbon isotopes from deep soil cores to reconstruct the dynamics of fire activity and deforestation across the Sigatoka River valley on the leeward (dry) side of Viti Levu, Fiji. Fires and pyrogenic patches of grassland predated human settlement by millennia, but the magnitude of fire activity and landscape transformation accelerated with the establishment and expansion of swidden agriculture. Regional comparisons with previous studies in Fiji and elsewhere in Remote Oceania settled between 3,200 and 2,900 yr bp reveal a similar pattern of pre- and post-settlement fire activity and landscape change. Pre-settlement fires generally corresponded to droughts, probably driven by El Niño, often correlating with drought-driven wildfires elsewhere in the region. Post-settlement, charcoal and C4 grasses increased dramatically, but nearly all major peaks in charcoal and grasses corresponded to increased El Niño activity. This indicates that fire activity and deforestation were a product of the interaction between swidden agriculture and climate rather than land use alone. Palaeoclimatic and palaeoecological reconstructions from Fiji suggest that the dramatic environmental transformations that followed human settlement of Remote Oceanian islands are the product of interactions between human activity and climate rather than human activity alone.},
  archive      = {J_NATEE},
  author       = {Roos, Christopher I. and Field, Julie S. and Dudgeon, John V.},
  doi          = {10.1038/s41559-023-02212-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2028-2036},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Fire activity and deforestation in remote oceanian islands caused by anthropogenic and climate interactions},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Meta-analysis reveals less sensitivity of non-native animals
than natives to extreme weather worldwide. <em>NATEE</em>,
<em>7</em>(12), 2004–2027. (<a
href="https://doi.org/10.1038/s41559-023-02235-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Extreme weather events (EWEs; for example, heatwaves, cold spells, storms, floods and droughts) and non-native species invasions are two major threats to global biodiversity and are increasing in both frequency and consequences. Here we synthesize 443 studies and apply multilevel mixed-effects metaregression analyses to compare the responses of 187 non-native and 1,852 native animal species across terrestrial, freshwater and marine ecosystems to different types of EWE. Our results show that marine animals, regardless of whether they are non-native or native, are overall insensitive to EWEs, except for negative effects of heatwaves on native mollusks, corals and anemone. By contrast, terrestrial and freshwater non-native animals are only adversely affected by heatwaves and storms, respectively, whereas native animals negatively respond to heatwaves, cold spells and droughts in terrestrial ecosystems and are vulnerable to most EWEs except cold spells in freshwater ecosystems. On average, non-native animals displayed low abundance in terrestrial ecosystems, and decreased body condition and life history traits in freshwater ecosystems, whereas native animals displayed declines in body condition, life history traits, abundance, distribution and recovery in terrestrial ecosystems, and community structure in freshwater ecosystems. By identifying areas with high overlap between EWEs and EWE-tolerant non-native species, we also provide locations where native biodiversity might be adversely affected by their joint effects and where EWEs might facilitate the establishment and/or spread of non-native species under continuing global change. A synthesis of 443 studies across terrestrial, freshwater and marine ecosystems reveals differences in the responses of non-native and native animal species to heatwaves, cold spells, storms, floods and droughts, and prompts discussion of how such extreme events may facilitate success of non-native species.},
  archive      = {J_NATEE},
  author       = {Gu, Shimin and Qi, Tianyi and Rohr, Jason R. and Liu, Xuan},
  doi          = {10.1038/s41559-023-02235-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {2004-2027},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Meta-analysis reveals less sensitivity of non-native animals than natives to extreme weather worldwide},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Temperate species underfill their tropical thermal
potentials on land. <em>NATEE</em>, <em>7</em>(12), 1993–2003. (<a
href="https://doi.org/10.1038/s41559-023-02239-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding how temperature determines the distribution of life is necessary to assess species’ sensitivities to contemporary climate change. Here, we test the importance of temperature in limiting the geographic ranges of ectotherms by comparing the temperatures and areas that species occupy to the temperatures and areas species could potentially occupy on the basis of their physiological thermal tolerances. We find that marine species across all latitudes and terrestrial species from the tropics occupy temperatures that closely match their thermal tolerances. However, terrestrial species from temperate and polar latitudes are absent from warm, thermally tolerable areas that they could potentially occupy beyond their equatorward range limits, indicating that extreme temperature is often not the factor limiting their distributions at lower latitudes. This matches predictions from the hypothesis that adaptation to cold environments that facilitates survival in temperate and polar regions is associated with a performance trade-off that reduces species’ abilities to contend in the tropics, possibly due to biotic exclusion. Our findings predict more direct responses to climate warming of marine ranges and cool range edges of terrestrial species. The authors test whether a wide array of marine and terrestrial animal species occupy the full extent of their potential geographic range based on thermal tolerances. They find that many species are underfilling the warm part of their potential range, suggesting that biotic interactions can limit occupancy in climatically suitable areas adjacent to their ranges.},
  archive      = {J_NATEE},
  author       = {Moore, Nikki A. and Morales-Castilla, Ignacio and Hargreaves, Anna L. and Olalla-Tárraga, Miguel Ángel and Villalobos, Fabricio and Calosi, Piero and Clusella-Trullas, Susana and Rubalcaba, Juan G. and Algar, Adam C. and Martínez, Brezo and Rodríguez, Laura and Gravel, Sarah and Bennett, Joanne M. and Vega, Greta C. and Rahbek, Carsten and Araújo, Miguel B. and Bernhardt, Joey R. and Sunday, Jennifer M.},
  doi          = {10.1038/s41559-023-02239-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1993-2003},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Temperate species underfill their tropical thermal potentials on land},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Warming indirectly simplifies food webs through effects on
apex predators. <em>NATEE</em>, <em>7</em>(12), 1983–1992. (<a
href="https://doi.org/10.1038/s41559-023-02216-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Warming alters ecosystems through direct physiological effects on organisms and indirect effects via biotic interactions, but their relative impacts in the wild are unknown due to the difficulty in warming natural environments. Here we bridge this gap by embedding manipulative field experiments within a natural stream temperature gradient to test whether warming and apex fish predators have interactive effects on freshwater ecosystems. Fish exerted cascading effects on algal production and microbial decomposition via both green and brown pathways in the food web, but only under warming. Neither temperature nor the presence of fish altered food web structure alone, but connectance and mean trophic level declined as consumer species were lost when both drivers acted together. A mechanistic model indicates that this temperature-induced trophic cascade is determined primarily by altered interactions, which cautions against extrapolating the impacts of warming from reductionist approaches that do not consider the wider food web. Using a bioenergetic model and manipulative field experiment along a natural stream temperature gradient, the authors identify a temperature-induced trophic cascade where the presence of fish increases algal biomass and reduces decomposition, but only under warming.},
  archive      = {J_NATEE},
  author       = {O’Gorman, Eoin J. and Zhao, Lei and Kordas, Rebecca L. and Dudgeon, Steve and Woodward, Guy},
  doi          = {10.1038/s41559-023-02216-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1983-1992},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Warming indirectly simplifies food webs through effects on apex predators},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chase-away evolution maintains imperfect mimicry in a brood
parasite–host system despite rapid evolution of mimics. <em>NATEE</em>,
<em>7</em>(12), 1978–1982. (<a
href="https://doi.org/10.1038/s41559-023-02232-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We studied a brood parasite–host system (the cuckoo finch Anomalospiza imberbis and its host, the tawny-flanked prinia Prinia subflava) to test (1) the fundamental hypothesis that deceptive mimics evolve to resemble models, selecting in turn for models to evolve away from mimics (‘chase-away evolution’) and (2) whether such reciprocal evolution maintains imperfect mimicry over time. Over only 50 years, parasites evolved towards hosts and hosts evolved away from parasites, resulting in no detectible increase in mimetic fidelity. Our results reflect rapid adaptive evolution in wild populations of models and mimics and show that chase-away evolution in models can counteract even rapid evolution of mimics, resulting in the persistence of imperfect mimicry. Using a 50 year time series of photos of cuckoo finch eggs and those of its host, prinia, the authors document that cuckoo eggs evolve towards prinia eggs, but progressive evolution of prinia eggs away from cuckoo eggs results in no detectible increase in mimetic fidelity.},
  archive      = {J_NATEE},
  author       = {Dixit, Tanmay and Lund, Jess and Fulford, Anthony J. C. and Apostol, Andrei L. and Chen, Kuan-Chi and Tong, Wenfei and Feeney, William E. and Hamusikili, Lazaro and Colebrook-Robjent, John F. R. and Town, Christopher P. and Spottiswoode, Claire N.},
  doi          = {10.1038/s41559-023-02232-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1978-1982},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Chase-away evolution maintains imperfect mimicry in a brood parasite–host system despite rapid evolution of mimics},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Why the geosciences are becoming increasingly vital to the
interpretation of the human evolutionary record. <em>NATEE</em>,
<em>7</em>(12), 1971–1977. (<a
href="https://doi.org/10.1038/s41559-023-02215-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Advanced geoscience techniques are essential to contextualize fossils, artefacts and other archaeologically important material accurately and effectively. Their appropriate use will increase confidence in new interpretations of the fossil and archaeological record, providing important information about the life and depositional history of these materials and so should form an integral component of all human evolutionary studies. Many of the most remarkable recent finds that have transformed the field of human evolution are small and scarce, ranging in size from teeth to strands of DNA, recovered from complex sedimentary environments. Nevertheless, if properly analysed, they hold immense potential to rewrite what we know about the evolution of our species and our closest hominin ancestors. The authors review methodological developments arising from the geosciences that are becoming increasingly key to interpreting the human evolutionary record.},
  archive      = {J_NATEE},
  author       = {Morley, Mike W. and Moffat, Ian and Kotarba-Morley, Anna M. and Hernandez, Vito C. and Zerboni, Andrea and Herries, Andy I. R. and Joannes-Boyau, Renaud and Westaway, Kira},
  doi          = {10.1038/s41559-023-02215-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1971-1977},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Why the geosciences are becoming increasingly vital to the interpretation of the human evolutionary record},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Land use, fire and climate contributed to deforestation on
the pacific islands. <em>NATEE</em>, <em>7</em>(12), 1969–1970. (<a
href="https://doi.org/10.1038/s41559-023-02229-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fire activity and deforestation accelerated in Remote Oceania following human settlement. However, geoarchaeology and palaeoecology indicate that peak fire activity and grassland expansion primarily coincided with high frequencies of El Niño droughts, which suggests that there are complex relationships among human land use, fire and climate in the western Pacific.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02229-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1969-1970},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Land use, fire and climate contributed to deforestation on the pacific islands},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Warming induces a trophic cascade in freshwater streams.
<em>NATEE</em>, <em>7</em>(12), 1967–1968. (<a
href="https://doi.org/10.1038/s41559-023-02222-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We examined the interactive effects of temperature and the presence of apex fish predators on food web structure in Icelandic geothermal streams. Fish suppressed the biomass of invertebrates and thus released algae from grazing pressure, but only at higher temperatures, which illustrates how the combination of warming and apex predators triggers this trophic cascade.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02222-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1967-1968},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Warming induces a trophic cascade in freshwater streams},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Transitions and trade-offs in regeneration. <em>NATEE</em>,
<em>7</em>(12), 1965–1966. (<a
href="https://doi.org/10.1038/s41559-023-02179-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Exploring regenerative abilities in planarian flatworms across a diverse living collection reveals independent gains and losses of head regeneration and the rescue of regeneration defects by inhibiting a single cell–cell interaction signal.},
  archive      = {J_NATEE},
  author       = {Lewin, Thomas D. and Luo, Yi-Jyun},
  doi          = {10.1038/s41559-023-02179-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1965-1966},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Transitions and trade-offs in regeneration},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Toxin rescue by a random sequence. <em>NATEE</em>,
<em>7</em>(12), 1963–1964. (<a
href="https://doi.org/10.1038/s41559-023-02252-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A random sequence variant in an experimental screen can rescue Escherichi coli from the deleterious effects of a RNase toxin by interacting with chaperones.},
  archive      = {J_NATEE},
  author       = {Hlouchova, Klara},
  doi          = {10.1038/s41559-023-02252-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1963-1964},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Toxin rescue by a random sequence},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Recombination triggers fungal crop disease. <em>NATEE</em>,
<em>7</em>(12), 1961–1962. (<a
href="https://doi.org/10.1038/s41559-023-02132-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Population genomics of the fungal plant pathogen Pyricularia oryzae shows that recombination of existing genetic variation during a multi-hybrid swarm probably facilitated an instantaneous host jump onto wheat.},
  archive      = {J_NATEE},
  author       = {Hill, Rowena and McMullan, Mark},
  doi          = {10.1038/s41559-023-02132-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1961-1962},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Recombination triggers fungal crop disease},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Non-native species resist extreme events. <em>NATEE</em>,
<em>7</em>(12), 1959–1960. (<a
href="https://doi.org/10.1038/s41559-023-02218-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A meta-analysis demonstrates that non-native species are often more resilient to extreme weather events than their native counterparts.},
  archive      = {J_NATEE},
  author       = {Doherty, Tim S. and Macdonald, Kristina J.},
  doi          = {10.1038/s41559-023-02218-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1959-1960},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Non-native species resist extreme events},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Robert l. Pressey (1953–2023). <em>NATEE</em>,
<em>7</em>(12), 1957–1958. (<a
href="https://doi.org/10.1038/s41559-023-02240-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Leading conservation planner and champion for biodiversity.},
  archive      = {J_NATEE},
  author       = {Adams, Vanessa M. and Álvarez-Romero, Jorge G. and Ban, Natalie C. and Mills, Morena and Visconti, Piero},
  doi          = {10.1038/s41559-023-02240-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1957-1958},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Robert l. pressey (1953–2023)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Make global biodiversity information useful to national
decision-makers. <em>NATEE</em>, <em>7</em>(12), 1953–1956. (<a
href="https://doi.org/10.1038/s41559-023-02226-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Participants in the Convention on Biological Diversity’s processes for implementing the Kunming-Montréal Global Biodiversity Framework need clarity on what makes biodiversity information useful to national decision-makers. Here we present seven preconditions of useful biodiversity information and describe how these can be supported through regional support centres and south–south cooperation.},
  archive      = {J_NATEE},
  author       = {Buschke, Falko T. and Capitani, Claudia and Sow, El Hadji and Khaemba, Yvonne and Kaplin, Beth A. and Skowno, Andrew and Chiawo, David and Hirsch, Tim and Ellwood, Elizabeth R. and Clements, Hayley and Child, Matthew F. and Huber, Patrick R. and von Staden, Lize and Hagenimana, Thacien and Killion, Alexander K. and Mindje, Mapendo and Mpakairi, Kudzai Shaun and Raymond, Mélianie and Matlombe, Domingas and Mbeya, Dickson and von Hase, Amrei},
  doi          = {10.1038/s41559-023-02226-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1953-1956},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Make global biodiversity information useful to national decision-makers},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A global biodiversity observing system to unite monitoring
and guide action. <em>NATEE</em>, <em>7</em>(12), 1947–1952. (<a
href="https://doi.org/10.1038/s41559-023-02171-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The rate and extent of global biodiversity change is surpassing our ability to measure, monitor and forecast trends. We propose an interconnected worldwide system of observation networks — a global biodiversity observing system (GBiOS) — to coordinate monitoring worldwide and inform action to reach international biodiversity targets.},
  archive      = {J_NATEE},
  author       = {Gonzalez, Andrew and Vihervaara, Petteri and Balvanera, Patricia and Bates, Amanda E. and Bayraktarov, Elisa and Bellingham, Peter J. and Bruder, Andreas and Campbell, Jillian and Catchen, Michael D. and Cavender-Bares, Jeannine and Chase, Jonathan and Coops, Nicholas and Costello, Mark J. and Czúcz, Bálint and Delavaud, Aurélie and Dornelas, Maria and Dubois, Grégoire and Duffy, Emmett J. and Eggermont, Hilde and Fernandez, Miguel and Fernandez, Nestor and Ferrier, Simon and Geller, Gary N. and Gill, Michael and Gravel, Dominique and Guerra, Carlos A. and Guralnick, Robert and Harfoot, Michael and Hirsch, Tim and Hoban, Sean and Hughes, Alice C. and Hugo, Wim and Hunter, Margaret E. and Isbell, Forest and Jetz, Walter and Juergens, Norbert and Kissling, W. Daniel and Krug, Cornelia B. and Kullberg, Peter and Le Bras, Yvan and Leung, Brian and Londoño-Murcia, Maria Cecilia and Lord, Jean-Michel and Loreau, Michel and Luers, Amy and Ma, Keping and MacDonald, Anna J. and Maes, Joachim and McGeoch, Melodie and Mihoub, Jean Baptiste and Millette, Katie L. and Molnar, Zsolt and Montes, Enrique and Mori, Akira S. and Muller-Karger, Frank E. and Muraoka, Hiroyuki and Nakaoka, Masahiro and Navarro, Laetitia and Newbold, Tim and Niamir, Aidin and Obura, David and O’Connor, Mary and Paganini, Marc and Pelletier, Dominique and Pereira, Henrique and Poisot, Timothée and Pollock, Laura J. and Purvis, Andy and Radulovici, Adriana and Rocchini, Duccio and Roeoesli, Claudia and Schaepman, Michael and Schaepman-Strub, Gabriela and Schmeller, Dirk S. and Schmiedel, Ute and Schneider, Fabian D. and Shakya, Mangal Man and Skidmore, Andrew and Skowno, Andrew L. and Takeuchi, Yayioi and Tuanmu, Mao-Ning and Turak, Eren and Turner, Woody and Urban, Mark C. and Urbina-Cardona, Nicolás and Valbuena, Ruben and Van de Putte, Anton and van Havre, Basile and Wingate, Vladimir Ruslan and Wright, Elaine and Torrelio, Carlos Zambrana},
  doi          = {10.1038/s41559-023-02171-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1947-1952},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A global biodiversity observing system to unite monitoring and guide action},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Increasing wildfires threaten progress on halting
deforestation in brazilian amazonia. <em>NATEE</em>, <em>7</em>(12),
1945–1946. (<a
href="https://doi.org/10.1038/s41559-023-02233-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {de Oliveira, Gabriel and Mataveli, Guilherme and Stark, Scott C. and Jones, Matthew W. and Carmenta, Rachel and Brunsell, Nathaniel A. and Santos, Celso A. G. and da Silva Junior, Carlos A. and Cunha, Helenilza F. A. and da Cunha, Alan C. and dos Santos, Carlos A. C. and Stewart, Hannah and Boanada Fuchs, Vanessa and Hellenkamp, Skye and Artaxo, Paulo and Alencar, Ane A. C. and Moutinho, Paulo and Shimabukuro, Yosio E.},
  doi          = {10.1038/s41559-023-02233-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {12},
  number       = {12},
  pages        = {1945-1946},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Increasing wildfires threaten progress on halting deforestation in brazilian amazonia},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Identifying species likely threatened by
international trade on the IUCN red list can inform CITES trade
measures. <em>NATEE</em>, <em>7</em>(11), 1944. (<a
href="https://doi.org/10.1038/s41559-023-02228-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Challender, Daniel W. S. and Cremona, Patricia J. and Malsch, Kelly and Robinson, Janine E. and Pavitt, Alyson T. and Scott, Janet and Hoffmann, Rachel and Joolia, Ackbar and Oldfield, Thomasina E. E. and Jenkins, Richard K. B. and Conde, Dalia A. and Hilton-Taylor, Craig and Hoffmann, Michael},
  doi          = {10.1038/s41559-023-02228-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1944},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Identifying species likely threatened by international trade on the IUCN red list can inform CITES trade measures},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Comparative single-cell transcriptomic analysis of primate
brains highlights human-specific regulatory evolution. <em>NATEE</em>,
<em>7</em>(11), 1930–1943. (<a
href="https://doi.org/10.1038/s41559-023-02186-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Enhanced cognitive function in humans is hypothesized to result from cortical expansion and increased cellular diversity. However, the mechanisms that drive these phenotypic innovations remain poorly understood, in part because of the lack of high-quality cellular resolution data in human and non-human primates. Here, we take advantage of single-cell expression data from the middle temporal gyrus of five primates (human, chimp, gorilla, macaque and marmoset) to identify 57 homologous cell types and generate cell type-specific gene co-expression networks for comparative analysis. Although orthologue expression patterns are generally well conserved, we find 24% of genes with extensive differences between human and non-human primates (3,383 out of 14,131), which are also associated with multiple brain disorders. To assess the functional significance of gene expression differences in an evolutionary context, we evaluate changes in network connectivity across meta-analytic co-expression networks from 19 animals. We find that a subset of these genes has deeply conserved co-expression across all non-human animals, and strongly divergent co-expression relationships in humans (139 out of 3,383, &amp;lt;1% of primate orthologues). Genes with human-specific cellular expression and co-expression profiles (such as NHEJ1, GTF2H2, C2 and BBS5) typically evolve under relaxed selective constraints and may drive rapid evolutionary change in brain function. Analysis of single-cell expression data from the middle temporal gyrus of five primates together with analysis of network connectivity across 19 animals identifies genes with human-specific cellular expression and co-expression profiles that might contribute to evolutionary change in brain function.},
  archive      = {J_NATEE},
  author       = {Suresh, Hamsini and Crow, Megan and Jorstad, Nikolas and Hodge, Rebecca and Lein, Ed and Dobin, Alexander and Bakken, Trygve and Gillis, Jesse},
  doi          = {10.1038/s41559-023-02186-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1930-1943},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Comparative single-cell transcriptomic analysis of primate brains highlights human-specific regulatory evolution},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ancient DNA reveals genetic admixture in china during tiger
evolution. <em>NATEE</em>, <em>7</em>(11), 1914–1929. (<a
href="https://doi.org/10.1038/s41559-023-02185-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The tiger (Panthera tigris) is a charismatic megafauna species that originated and diversified in Asia and probably experienced population contraction and expansion during the Pleistocene, resulting in low genetic diversity of modern tigers. However, little is known about patterns of genomic diversity in ancient populations. Here we generated whole-genome sequences from ancient or historical (100–10,000 yr old) specimens collected across mainland Asia, including a 10,600-yr-old Russian Far East specimen (RUSA21, 8× coverage) plus six ancient mitogenomes, 14 South China tigers (0.1–12×) and three Caspian tigers (4–8×). Admixture analysis showed that RUSA21 clustered within modern Northeast Asian phylogroups and partially derived from an extinct Late Pleistocene lineage. While some of the 8,000–10,000-yr-old Russian Far East mitogenomes are basal to all tigers, one 2,000-yr-old specimen resembles present Amur tigers. Phylogenomic analyses suggested that the Caspian tiger probably dispersed from an ancestral Northeast Asian population and experienced gene flow from southern Bengal tigers. Lastly, genome-wide monophyly supported the South China tiger as a distinct subspecies, albeit with mitochondrial paraphyly, hence resolving its longstanding taxonomic controversy. The distribution of mitochondrial haplogroups corroborated by biogeographical modelling suggested that Southwest China was a Late Pleistocene refugium for a relic basal lineage. As suitable habitat returned, admixture between divergent lineages of South China tigers took place in Eastern China, promoting the evolution of other northern subspecies. Altogether, our analysis of ancient genomes sheds light on the evolutionary history of tigers and supports the existence of nine modern subspecies. Phylogenetics and population genetics analysis of ancient tiger genomes inform tiger evolution and suggest post-glaciation admixture of divergent tiger lineages in China.},
  archive      = {J_NATEE},
  author       = {Sun, Xin and Liu, Yue-Chen and Tiunov, Mikhail P. and Gimranov, Dmitry O. and Zhuang, Yan and Han, Yu and Driscoll, Carlos A. and Pang, Yuhong and Li, Chunmei and Pan, Yan and Velasco, Marcela Sandoval and Gopalakrishnan, Shyam and Yang, Rui-Zheng and Li, Bao-Guo and Jin, Kun and Xu, Xiao and Uphyrkina, Olga and Huang, Yanyi and Wu, Xiao-Hong and Gilbert, M. Thomas P. and O’Brien, Stephen J. and Yamaguchi, Nobuyuki and Luo, Shu-Jin},
  doi          = {10.1038/s41559-023-02185-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1914-1929},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ancient DNA reveals genetic admixture in china during tiger evolution},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Derived faunivores are the forerunners of major synapsid
radiations. <em>NATEE</em>, <em>7</em>(11), 1903–1913. (<a
href="https://doi.org/10.1038/s41559-023-02200-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evolutionary radiations generate most of Earth’s biodiversity, but are there common ecomorphological traits among the progenitors of radiations? In Synapsida (the mammalian total group), ‘small-bodied faunivore’ has been hypothesized as the ancestral state of most major radiating clades, but this has not been quantitatively assessed across multiple radiations. To examine macroevolutionary patterns in a phylogenetic context, we generated a time-calibrated metaphylogeny (‘metatree’) comprising 1,888 synapsid species from the Carboniferous through the Eocene (305–34 Ma) based on 269 published character matrices. We used comparative methods to investigate body size and dietary evolution during successive synapsid radiations. Faunivory is the ancestral dietary regime of each major synapsid radiation, but relatively small body size is only established as the common ancestral state of radiations near the origin of Mammaliaformes in the Late Triassic. The faunivorous ancestors of synapsid radiations typically have numerous novel characters compared with their contemporaries, and these derived traits may have helped them to survive faunal turnover events and subsequently radiate. Small-bodied faunivory has been proposed as the ancestral condition of most major synapsid clades, but here using a time-calibrated metatree of 1,888 fossil synapsids, the authors show that while faunivory is commonly ancestral, small body size in radiation forerunners is a relatively late innovation, arising in the Late Triassic.},
  archive      = {J_NATEE},
  author       = {Hellert, Spencer M. and Grossnickle, David M. and Lloyd, Graeme T. and Kammerer, Christian F. and Angielczyk, Kenneth D.},
  doi          = {10.1038/s41559-023-02200-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1903-1913},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Derived faunivores are the forerunners of major synapsid radiations},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Aqueous microdroplets promote c–c bond formation and
sequences in the reverse tricarboxylic acid cycle. <em>NATEE</em>,
<em>7</em>(11), 1892–1902. (<a
href="https://doi.org/10.1038/s41559-023-02193-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The reverse tricarboxylic acid cycle (rTCA) is a central anabolic network that uses carbon dioxide (CO2) and may have provided complex carbon substrates for life before the advent of RNA or enzymes. However, non-enzymatic promotion of the rTCA cycle, in particular carbon fixation, remains challenging, even with primordial metal catalysis. Here, we report that the fixation of CO2 by reductive carboxylation of succinate and α-ketoglutarate was achieved in aqueous microdroplets under ambient conditions without the use of catalysts. Under identical conditions, the aqueous microdroplets also facilitated the sequences in the rTCA cycle, including reduction, hydration, dehydration and retro-aldol cleavage and linked with the glyoxylate cycle. These reactions of the rTCA cycle were compatible with the aqueous microdroplets, as demonstrated with two-reaction and four-reaction sequences. A higher selectivity giving higher product yields was also observed. Our results suggest that the microdroplets provide an energetically favourable microenvironment and facilitate a non-enzymatic version of the rTCA cycle in prebiotic carbon anabolism. Using aqueous microdroplets to study reactions in the reverse tricarboxylic acid cycle (rTCA), the authors show rapid carbon fixation through reductive carboxylation reactions at ambient temperature, without requiring enzymes or metal catalysts, suggesting that microdroplets might have facilitated a non-enzymatic version of the rTCA cycle in prebiotic carbon anabolism.},
  archive      = {J_NATEE},
  author       = {Ju, Yun and Zhang, Hong and Jiang, Yanxiao and Wang, Wenxin and Kan, Guangfeng and Yu, Kai and Wang, Xiaofei and Liu, Jilin and Jiang, Jie},
  doi          = {10.1038/s41559-023-02193-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1892-1902},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Aqueous microdroplets promote C–C bond formation and sequences in the reverse tricarboxylic acid cycle},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Indirect genetic effects are shaped by demographic history
and ecology in arabidopsis thaliana. <em>NATEE</em>, <em>7</em>(11),
1878–1891. (<a
href="https://doi.org/10.1038/s41559-023-02189-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The phenotype of an individual can be affected by the genes of its conspecifics through indirect genetic effects (IGEs). IGEs have been studied across different organisms including wild and domesticated animals and plants, but little is known about their genetic architecture. Here, in a large-scale intraspecific interaction experiment, we show that the contribution of IGEs to the biomass variation of Arabidopsis thaliana is comparable to values classically reported in animals. Moreover, we identify 11 loci explaining 85.1% of the variability in IGEs. We find that positive IGE alleles (that is, those with positive effects on neighbour biomass) occur both in relict accessions from southern Eurasia and in post-glacial colonizers from northern Scandinavia, and that they are likely to have two divergent origins: for nine loci, they evolved in the post-glacial colonizers independently from the relicts, while the two others were introgressed in the post-glacial colonizer from the relicts. Finally, we find that variation in IGEs probably reflects divergent adaptations to the contrasting environments of the edges and the centre of the native range of the species. These findings reveal a surprisingly tractable genetic basis of IGEs in A. thaliana that is shaped by the ecology and the demographic history of the species. This study reveals the genomic architecture of indirect genetic effects (IGEs) in Arabidopsis thaliana and reconstructs the evolutionary history of alleles responsible for IGEs in Eurasian populations.},
  archive      = {J_NATEE},
  author       = {Montazeaud, Germain and Helleu, Quentin and Wuest, Samuel E. and Keller, Laurent},
  doi          = {10.1038/s41559-023-02189-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1878-1891},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Indirect genetic effects are shaped by demographic history and ecology in arabidopsis thaliana},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Adaptation to climate change through dispersal and inherited
timing in an avian migrant. <em>NATEE</em>, <em>7</em>(11), 1869–1877.
(<a href="https://doi.org/10.1038/s41559-023-02191-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many organisms fail to adjust their phenology sufficiently to climate change. Studies have concentrated on adaptive responses within localities, but little is known about how latitudinal dispersal enhances evolutionary potential. Rapid adaptation is expected if dispersers from lower latitudes have improved synchrony to northern conditions, thereby gain fitness and introduce genotypes on which selection acts. Here we provide experimental evidence that dispersal in an avian migrant enables rapid evolutionary adaptation. We translocated Dutch female pied flycatchers (Ficedula hypoleuca) and eggs to Sweden, where breeding phenology is ~15 days later. Translocated females bred earlier, and their fitness was 2.5 times higher than local Swedish flycatchers. We show that between-population variation in timing traits is highly heritable, and hence immigration of southern genotypes promotes the necessary evolutionary response. We conclude that studies on adaptation to large-scale environmental change should not just focus on plasticity and evolution based on standing genetic variation but should also include phenotype–habitat matching through dispersal as a viable route to adjust. Translocation of pied flycatcher (Ficedula hypoleuca) females and eggs from the Netherlands to Sweden facilitates adaptation to changing phenological conditions, highlighting the importance of dispersal as well as plasticity and standing genetic variation in evolutionary responses to environmental change.},
  archive      = {J_NATEE},
  author       = {Lamers, Koosje P. and Nilsson, Jan-Åke and Nicolaus, Marion and Both, Christiaan},
  doi          = {10.1038/s41559-023-02191-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1869-1877},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Adaptation to climate change through dispersal and inherited timing in an avian migrant},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatial sorting promotes rapid (mal)adaptation in the
red-shouldered soapberry bug after hurricane-driven local extinctions.
<em>NATEE</em>, <em>7</em>(11), 1856–1868. (<a
href="https://doi.org/10.1038/s41559-023-02205-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting future evolutionary change is a critical challenge in the Anthropocene as geographic range shifts and local extinction emerge as hallmarks of planetary change. Hence, spatial sorting—a driver of rapid evolution in which dispersal-associated traits accumulate along expanding range edges and within recolonized habitats—might be of growing importance in ecology and conservation. We report on the results of a natural experiment that monitored recolonization of host plants by the seed-feeding, red-shouldered soapberry bug, Jadera haematoloma, after local extinctions from catastrophic flooding in an extreme hurricane. We tested the contribution of spatial sorting to generate rapid and persistent evolution in dispersal traits, as well as in feeding traits unrelated to dispersal. Long-winged dispersal forms accumulated in recolonized habitats and due to genetic correlation, mouthparts also became longer and this shift persisted across generations. Those longer mouthparts were probably adaptive on one host plant species but maladaptive on two others based on matching the optimum depth of seeds within their host fruits. Moreover, spatial sorting eroded recently evolved adaptive divergence in mouthpart length among all host-associated biotypes, an outcome pointing to profound practical consequences of the extreme weather event for local adaptation, population resilience and evolutionary futures. Catastrophic flooding caused by an extreme hurricane offered a rare natural experiment monitoring recolonization of host plants by a herbivorous predator, in which the authors found that spatial sorting is responsible for the rapid and persistent evolution of dispersal and feeding traits in the red-shouldered soapberry bug.},
  archive      = {J_NATEE},
  author       = {Comerford, Mattheau S. and La, Tatum M. and Carroll, Scott and Egan, Scott P.},
  doi          = {10.1038/s41559-023-02205-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1856-1868},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Spatial sorting promotes rapid (mal)adaptation in the red-shouldered soapberry bug after hurricane-driven local extinctions},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Local human impacts disrupt depth-dependent zonation of
tropical reef fish communities. <em>NATEE</em>, <em>7</em>(11),
1844–1855. (<a
href="https://doi.org/10.1038/s41559-023-02201-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The influence of depth and associated gradients in light, nutrients and plankton on the ecological organization of tropical reef communities was first described over six decades ago but remains untested across broad geographies. During this time humans have become the dominant driver of planetary change, requiring that we revisit historic ecological paradigms to ensure they capture the dynamics of contemporary ecological systems. Analysing &amp;gt;5,500 in-water reef fish surveys between 0 and 30 m depth on reef slopes of 35 islands across the Pacific, we assess whether a depth gradient consistently predicts variation in reef fish biomass. We reveal predictable ecological organization at unpopulated locations, with increased biomass of planktivores and piscivores and decreased primary consumer biomass with increasing depth. Bathymetric steepness also had a striking influence on biomass patterns, primarily for planktivores, emphasizing potential links between local hydrodynamics and the upslope propagation of pelagic subsidies to the shallows. However, signals of resource-driven change in fish biomass with depth were altered or lost for populated islands, probably due to depleted fish biomass baselines. While principles of depth zonation broadly held, our findings expose limitations of the paradigm for predicting ecological dynamics where human impacts confound connections between ecological communities and their surrounding environment. Analysis of tropical reef fish communities across 35 Pacific islands identifies predictable energetic resource-driven relationships between depth and biomass of different trophic groups of fish on reefs without local human impacts, but changes in these relationships for human-populated islands.},
  archive      = {J_NATEE},
  author       = {Richardson, Laura E. and Heenan, Adel and Delargy, Adam J. and Neubauer, Philipp and Lecky, Joey and Gove, Jamison M. and Green, J. A. Mattias and Kindinger, Tye L. and Ingeman, Kurt E. and Williams, Gareth J.},
  doi          = {10.1038/s41559-023-02201-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1844-1855},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Local human impacts disrupt depth-dependent zonation of tropical reef fish communities},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Host phylogeny shapes viral transmission networks in an
island ecosystem. <em>NATEE</em>, <em>7</em>(11), 1834–1843. (<a
href="https://doi.org/10.1038/s41559-023-02192-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Virus transmission between host species underpins disease emergence. Both host phylogenetic relatedness and aspects of their ecology, such as species interactions and predator–prey relationships, may govern rates and patterns of cross-species virus transmission and hence zoonotic risk. To address the impact of host phylogeny and ecology on virus diversity and evolution, we characterized the virome structure of a relatively isolated island ecological community in Fiordland, New Zealand, that are linked through a food web. We show that phylogenetic barriers that inhibited cross-species virus transmission occurred at the level of host phyla (between the Chordata, Arthropoda and Streptophyta) as well as at lower taxonomic levels. By contrast, host ecology, manifest as predator–prey interactions and diet, had a smaller influence on virome composition, especially at higher taxonomic levels. The virus–host community comprised a ‘small world’ network, in which hosts with a high diversity of viruses were more likely to acquire new viruses, and generalist viruses that infect multiple hosts were more likely to infect additional species compared to host specialist viruses. Such a highly connected ecological community increases the likelihood of cross-species virus transmission, particularly among closely related species, and suggests that host generalist viruses present the greatest risk of disease emergence. Sampling the viromes of vertebrates, arthropods and plants on an island ecosystem shows that viral transmission between species is strongly affected by phylogeny but less affected by predator–prey relationships and that generalist viruses pose the greatest zoonotic risk.},
  archive      = {J_NATEE},
  author       = {French, Rebecca K. and Anderson, Sandra H. and Cain, Kristal E. and Greene, Terry C. and Minor, Maria and Miskelly, Colin M. and Montoya, Jose M. and Wille, Michelle and Muller, Chris G. and Taylor, Michael W. and Digby, Andrew and Holmes, Edward C.},
  doi          = {10.1038/s41559-023-02192-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1834-1843},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Host phylogeny shapes viral transmission networks in an island ecosystem},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Statistically learning the functional landscape of microbial
communities. <em>NATEE</em>, <em>7</em>(11), 1823–1833. (<a
href="https://doi.org/10.1038/s41559-023-02197-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microbial consortia exhibit complex functional properties in contexts ranging from soils to bioreactors to human hosts. Understanding how community composition determines function is a major goal of microbial ecology. Here we address this challenge using the concept of community-function landscapes—analogues to fitness landscapes—that capture how changes in community composition alter collective function. Using datasets that represent a broad set of community functions, from production/degradation of specific compounds to biomass generation, we show that statistically inferred landscapes quantitatively predict community functions from knowledge of species presence or absence. Crucially, community-function landscapes allow prediction without explicit knowledge of abundance dynamics or interactions between species and can be accurately trained using measurements from a small subset of all possible community compositions. The success of our approach arises from the fact that empirical community-function landscapes appear to be not rugged, meaning that they largely lack high-order epistatic contributions that would be difficult to fit with limited data. Finally, we show that this observation holds across a wide class of ecological models, suggesting community-function landscapes can be efficiently inferred across a broad range of ecological regimes. Our results open the door to the rational design of consortia without detailed knowledge of abundance dynamics or interactions. Landscapes of microbial community function inferred statistically from a broad range of datasets can predict community function on the basis on presence and absence data, without the need for abundance dynamics or interaction data.},
  archive      = {J_NATEE},
  author       = {Skwara, Abigail and Gowda, Karna and Yousef, Mahmoud and Diaz-Colunga, Juan and Raman, Arjun S. and Sanchez, Alvaro and Tikhonov, Mikhail and Kuehn, Seppe},
  doi          = {10.1038/s41559-023-02197-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1823-1833},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Statistically learning the functional landscape of microbial communities},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Viral but not bacterial community successional patterns
reflect extreme turnover shortly after rewetting dry soils.
<em>NATEE</em>, <em>7</em>(11), 1809–1822. (<a
href="https://doi.org/10.1038/s41559-023-02207-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As central members of soil trophic networks, viruses have the potential to drive substantial microbial mortality and nutrient turnover. Pinpointing viral contributions to terrestrial ecosystem processes remains a challenge, as temporal dynamics are difficult to unravel in the spatially and physicochemically heterogeneous soil environment. In Mediterranean grasslands, the first rainfall after seasonal drought provides an ecosystem reset, triggering microbial activity during a tractable window for capturing short-term dynamics. Here, we simulated precipitation in microcosms from four distinct dry grassland soils and generated 144 viromes, 84 metagenomes and 84 16S ribosomal RNA gene amplicon datasets to characterize viral, prokaryotic and relic DNA dynamics over 10 days. Vastly different viral communities in each soil followed remarkably similar successional trajectories. Wet-up triggered a significant increase in viral richness, followed by extensive compositional turnover. Temporal succession in prokaryotic communities was much less pronounced, perhaps suggesting differences in the scales of activity captured by viromes (representing recently produced, ephemeral viral particles) and total DNA. Still, differences in the relative abundances of Actinobacteria (enriched in dry soils) and Proteobacteria (enriched in wetted soils) matched those of their predicted phages, indicating viral predation of dominant bacterial taxa. Rewetting also rapidly depleted relic DNA, which subsequently reaccumulated, indicating substantial new microbial mortality in the days after wet-up, particularly of the taxa putatively under phage predation. Production of abundant, diverse viral particles via microbial host cell lysis appears to be a conserved feature of the early response to soil rewetting, and results suggest the potential for ‘Cull-the-Winner’ dynamics, whereby viruses infect and cull but do not decimate dominant host populations. Characterizing the responses of viral, prokaryotic and relic DNA dynamics to simulated precipitation on dry grassland soils, the authors show that soil viral communities follow remarkably similar successional patterns, despite large differences in composition.},
  archive      = {J_NATEE},
  author       = {Santos-Medellín, Christian and Blazewicz, Steven J. and Pett-Ridge, Jennifer and Firestone, Mary K. and Emerson, Joanne B.},
  doi          = {10.1038/s41559-023-02207-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1809-1822},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Viral but not bacterial community successional patterns reflect extreme turnover shortly after rewetting dry soils},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reliability of vegetation resilience estimates depends on
biomass density. <em>NATEE</em>, <em>7</em>(11), 1799–1808. (<a
href="https://doi.org/10.1038/s41559-023-02194-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Concerns have been raised that the resilience of vegetated ecosystems may be negatively impacted by ongoing anthropogenic climate and land-use change at the global scale. Several recent studies present global vegetation resilience trends based on satellite data using diverse methodological set-ups. Here, upon a systematic comparison of data sets, spatial and temporal pre-processing, and resilience estimation methods, we propose a methodology that avoids different biases present in previous results. Nevertheless, we find that resilience estimation using optical satellite vegetation data is broadly problematic in dense tropical and high-latitude boreal forests, regardless of the vegetation index chosen. However, for wide parts of the mid-latitudes—especially with low biomass density—resilience can be reliably estimated using several optical vegetation indices. We infer a spatially consistent global pattern of resilience gain and loss across vegetation indices, with more regions facing declining resilience, especially in Africa, Australia and central Asia. A systematic comparison of optical satellite data shows that vegetation resilience is broadly declining in many areas, despite the unreliability of resilience estimates in high biomass regions.},
  archive      = {J_NATEE},
  author       = {Smith, Taylor and Boers, Niklas},
  doi          = {10.1038/s41559-023-02194-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1799-1808},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reliability of vegetation resilience estimates depends on biomass density},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Structural complexity biases vegetation greenness measures.
<em>NATEE</em>, <em>7</em>(11), 1790–1798. (<a
href="https://doi.org/10.1038/s41559-023-02187-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Vegetation ‘greenness’ characterized by spectral vegetation indices (VIs) is an integrative measure of vegetation leaf abundance, biochemical properties and pigment composition. Surprisingly, satellite observations reveal that several major VIs over the US Corn Belt are higher than those over the Amazon rainforest, despite the forests having a greater leaf area. This contradicting pattern underscores the pressing need to understand the underlying drivers and their impacts to prevent misinterpretations. Here we show that macroscale shadows cast by complex forest structures result in lower greenness measures compared with those cast by structurally simple and homogeneous crops. The shadow-induced contradictory pattern of VIs is inevitable because most Earth-observing satellites do not view the Earth in the solar direction and thus view shadows due to the sun–sensor geometry. The shadow impacts have important implications for the interpretation of VIs and solar-induced chlorophyll fluorescence as measures of global vegetation changes. For instance, a land-conversion process from forests to crops over the Amazon shows notable increases in VIs despite a decrease in leaf area. Our findings highlight the importance of considering shadow impacts to accurately interpret remotely sensed VIs and solar-induced chlorophyll fluorescence for assessing global vegetation and its changes. Remote sensing often detects higher vegetation greenness for croplands than for forests, despite forests having a greater leaf area. This study shows that this is an artefact of shadows caused by forest structures and explores how to correct for this when interpreting global vegetation change data.},
  archive      = {J_NATEE},
  author       = {Zeng, Yelu and Hao, Dalei and Park, Taejin and Zhu, Peng and Huete, Alfredo and Myneni, Ranga and Knyazikhin, Yuri and Qi, Jianbo and Nemani, Ramakrishna R. and Li, Fa and Huang, Jianxi and Gao, Yongyuan and Li, Baoguo and Ji, Fujiang and Köhler, Philipp and Frankenberg, Christian and Berry, Joseph A. and Chen, Min},
  doi          = {10.1038/s41559-023-02187-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1790-1798},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Structural complexity biases vegetation greenness measures},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A high-resolution canopy height model of the earth.
<em>NATEE</em>, <em>7</em>(11), 1778–1789. (<a
href="https://doi.org/10.1038/s41559-023-02206-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The worldwide variation in vegetation height is fundamental to the global carbon cycle and central to the functioning of ecosystems and their biodiversity. Geospatially explicit and, ideally, highly resolved information is required to manage terrestrial ecosystems, mitigate climate change and prevent biodiversity loss. Here we present a comprehensive global canopy height map at 10 m ground sampling distance for the year 2020. We have developed a probabilistic deep learning model that fuses sparse height data from the Global Ecosystem Dynamics Investigation (GEDI) space-borne LiDAR mission with dense optical satellite images from Sentinel-2. This model retrieves canopy-top height from Sentinel-2 images anywhere on Earth and quantifies the uncertainty in these estimates. Our approach improves the retrieval of tall canopies with typically high carbon stocks. According to our map, only 5% of the global landmass is covered by trees taller than 30 m. Further, we find that only 34% of these tall canopies are located within protected areas. Thus, the approach can serve ongoing efforts in forest conservation and has the potential to foster advances in climate, carbon and biodiversity modelling. Using a deep learning approach, the authors outline a global canopy height map at 10-m resolution combining publicly available optical satellite images and space-borne LiDAR and show that only 5% of the global landmass is covered by trees taller than 30 m.},
  archive      = {J_NATEE},
  author       = {Lang, Nico and Jetz, Walter and Schindler, Konrad and Wegner, Jan Dirk},
  doi          = {10.1038/s41559-023-02206-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1778-1789},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A high-resolution canopy height model of the earth},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Natural capital investments in china undermined by
reclamation for cropland. <em>NATEE</em>, <em>7</em>(11), 1771–1777. (<a
href="https://doi.org/10.1038/s41559-023-02198-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Globally, rising food demand has caused widespread biodiversity and ecosystem services loss, prompting growing efforts in ecological protection and restoration. However, these efforts have been significantly undercut by further reclamation for cropland. Focusing on China, the world’s largest grain producer, we found that at the national level from 2000 to 2015, reclamation for cropland undermined gains in wildlife habitat and the ecosystem services of water retention, sandstorm prevention, carbon sequestration and soil retention by 113.8%, 63.4%, 52.5%, 29.0% and 10.2%, respectively. To achieve global sustainability goals, conflicts between inefficient reclamation for cropland and natural capital investment need to be alleviated. Between 2000 and 2015, reclamation for cropland in China undermined gains in wildlife habitat and the ecosystem services of water retention, sandstorm prevention, carbon sequestration and soil retention by 113.8%, 63.4%, 52.5%, 29.0% and 10.2%, respectively.},
  archive      = {J_NATEE},
  author       = {Kong, Lingqiao and Wu, Tong and Xiao, Yi and Xu, Weihua and Zhang, Xiaobiao and Daily, Gretchen C. and Ouyang, Zhiyun},
  doi          = {10.1038/s41559-023-02198-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1771-1777},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Natural capital investments in china undermined by reclamation for cropland},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evolutionary determinants of curability in cancer.
<em>NATEE</em>, <em>7</em>(11), 1761–1770. (<a
href="https://doi.org/10.1038/s41559-023-02159-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The emergence of drug-resistant cells, most of which have a mutated TP53 gene, prevents curative treatment in most advanced and common metastatic cancers of adults. Yet, a few, rarer malignancies, all of which are TP53 wild type, have high cure rates. In this Perspective, we discuss how common features of curable cancers offer insights into the evolutionary and developmental determinants of drug resistance. Acquired loss of TP53 protein function is the most common genetic change in cancer. This probably reflects positive selection in the context of strong ecosystem pressures including microenvironmental hypoxia. Loss of TP53’s functions results in multiple fitness benefits and enhanced evolvability of cancer cells. TP53-null cells survive apoptosis, and tolerate potent oncogenic signalling, DNA damage and genetic instability. In addition, critically, they provide an expanded pool of self-renewing, or stem, cells, the primary units of evolutionary selection in cancer, making subsequent adaptation to therapeutic challenge by drug resistance highly probable. The exceptional malignancies that are curable, including the common genetic subtype of childhood acute lymphoblastic leukaemia and testicular seminoma, differ from the common adult cancers in originating prenatally from embryonic or fetal cells that are developmentally primed for TP53-dependent apoptosis. Plus, they have other genetic and phenotypic features that enable dissemination without exposure to selective pressures for TP53 loss, retaining their intrinsic drug hypersensitivity. This Perspective discusses common features of curable cancers to gain insights into the evolutionary and developmental determinants of drug resistance.},
  archive      = {J_NATEE},
  author       = {Mansur, Marcela Braga and deSouza, Nandita M. and Natrajan, Rachael and Abegglen, Lisa M. and Schiffman, Joshua D. and Greaves, Mel},
  doi          = {10.1038/s41559-023-02159-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1761-1770},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolutionary determinants of curability in cancer},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Depth zonation of reef fish is predictable but disrupted on
contemporary coral reefs. <em>NATEE</em>, <em>7</em>(11), 1759–1760. (<a
href="https://doi.org/10.1038/s41559-023-02202-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data from 5,525 in-water reef fish surveys conducted between 1- and 30-m depth reveal predictable depth-dependent zonation across the Pacific Ocean, particularly in the absence of a local human population. By contrast, relationships between depth and biomass were reduced or absent at populated islands, which suggests a human impact on depth-dependent ecological organization.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02202-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1759-1760},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Depth zonation of reef fish is predictable but disrupted on contemporary coral reefs},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatial sorting creates winners and losers. <em>NATEE</em>,
<em>7</em>(11), 1756–1758. (<a
href="https://doi.org/10.1038/s41559-023-02217-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A catastrophic flooding event offered an unusual chance to demonstrate that spatial sorting — the differential dispersal of phenotypes — occurs in soapberry bugs as they recolonize after a major disturbance. But this process does not always prove to be adaptive.},
  archive      = {J_NATEE},
  author       = {Gordon, Swanne P. and Axelrod, Caleb J.},
  doi          = {10.1038/s41559-023-02217-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1756-1758},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Spatial sorting creates winners and losers},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Smooth functional landscapes in microcosms. <em>NATEE</em>,
<em>7</em>(11), 1754–1755. (<a
href="https://doi.org/10.1038/s41559-023-02214-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inspired by systems biology, a statistical model now shows that low-order ecological interactions — which are inferable from relatively limited species-presence datasets — can successfully predict functional performance across synthetic microcosms.},
  archive      = {J_NATEE},
  author       = {Amor, Daniel R.},
  doi          = {10.1038/s41559-023-02214-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1754-1755},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Smooth functional landscapes in microcosms},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). To cull or kill. <em>NATEE</em>, <em>7</em>(11), 1752–1753.
(<a href="https://doi.org/10.1038/s41559-023-02210-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An experiment that simulates rainfall events in dry soils reveals that virus members of the soil microbiome maintain the turnover of prokaryotic host communities through a ‘cull-the-winner’ model.},
  archive      = {J_NATEE},
  author       = {Kosmopoulos, James C. and Anantharaman, Karthik},
  doi          = {10.1038/s41559-023-02210-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1752-1753},
  shortjournal = {Nat. Ecol. Evol},
  title        = {To cull or kill},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A life spent at the roots. <em>NATEE</em>, <em>7</em>(11),
1750–1751. (<a
href="https://doi.org/10.1038/s41559-023-02203-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Hortal, Joaquín},
  doi          = {10.1038/s41559-023-02203-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1750-1751},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A life spent at the roots},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ecovoltaic principles for a more sustainable, ecologically
informed solar energy future. <em>NATEE</em>, <em>7</em>(11), 1746–1749.
(<a href="https://doi.org/10.1038/s41559-023-02174-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The majority of power generated by photovoltaic energy infrastructure is derived from ground-mounted solar arrays that prioritize energy production, minimize operating costs and, at best, accommodate limited ecosystem services. We argue that co-prioritizing ecosystem services and energy generation using an ecologically informed, ‘ecovoltaics’ approach to solar array design and operation will have multiple benefits for climate, biodiversity and the restoration of degraded lands.},
  archive      = {J_NATEE},
  author       = {Sturchio, Matthew A. and Knapp, Alan K.},
  doi          = {10.1038/s41559-023-02174-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1746-1749},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ecovoltaic principles for a more sustainable, ecologically informed solar energy future},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Data leakage jeopardizes ecological applications of machine
learning. <em>NATEE</em>, <em>7</em>(11), 1743–1745. (<a
href="https://doi.org/10.1038/s41559-023-02162-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning is a popular tool in ecology but many scientific applications suffer from data leakage, causing misleading results. We highlight common pitfalls in ecological machine-learning methods and argue that discipline-specific model info sheets must be developed to aid in model evaluations.},
  archive      = {J_NATEE},
  author       = {Stock, Andy and Gregr, Edward J. and Chan, Kai M. A.},
  doi          = {10.1038/s41559-023-02162-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {11},
  number       = {11},
  pages        = {1743-1745},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Data leakage jeopardizes ecological applications of machine learning},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Publisher correction: A lamprey neural cell type atlas
illuminates the origins of the vertebrate brain. <em>NATEE</em>,
<em>7</em>(10), 1741. (<a
href="https://doi.org/10.1038/s41559-023-02227-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Lamanna, Francesco and Hervas-Sotomayor, Francisca and Oel, A. Phillip and Jandzik, David and Sobrido-Cameán, Daniel and Santos-Durán, Gabriel N. and Martik, Megan L. and Stundl, Jan and Green, Stephen A. and Brüning, Thoomke and Mößinger, Katharina and Schmidt, Julia and Schneider, Celine and Sepp, Mari and Murat, Florent and Smith, Jeramiah J. and Bronner, Marianne E. and Rodicio, María Celina and Barreiro-Iglesias, Antón and Medeiros, Daniel M. and Arendt, Detlev and Kaessmann, Henrik},
  doi          = {10.1038/s41559-023-02227-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1741},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: A lamprey neural cell type atlas illuminates the origins of the vertebrate brain},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Fluctuating climate and dietary
innovation drove ratcheted evolution of proboscidean dental traits.
<em>NATEE</em>, <em>7</em>(10), 1740. (<a
href="https://doi.org/10.1038/s41559-023-02204-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Saarinen, Juha and Lister, Adrian M.},
  doi          = {10.1038/s41559-023-02204-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1740},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Fluctuating climate and dietary innovation drove ratcheted evolution of proboscidean dental traits},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rules of teeth development align microevolution with
macroevolution in extant and extinct primates. <em>NATEE</em>,
<em>7</em>(10), 1729–1739. (<a
href="https://doi.org/10.1038/s41559-023-02167-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Macroevolutionary biologists have classically rejected the notion that higher-level patterns of divergence arise through microevolutionary processes acting within populations. For morphology, this consensus partly derives from the inability of quantitative genetics models to correctly predict the behaviour of evolutionary processes at the scale of millions of years. Developmental studies (evo-devo) have been proposed to reconcile micro- and macroevolution. However, there has been little progress in establishing a formal framework to apply evo-devo models of phenotypic diversification. Here we reframe this issue by asking whether using evo-devo models to quantify biological variation can improve the explanatory power of comparative models, thus helping us bridge the gap between micro- and macroevolution. We test this prediction by evaluating the evolution of primate lower molars in a comprehensive dataset densely sampled across living and extinct taxa. Our results suggest that biologically informed morphospaces alongside quantitative genetics models allow a seamless transition between the micro- and macroscales, whereas biologically uninformed spaces do not. We show that the adaptive landscape for primate teeth is corridor like, with changes in morphology within the corridor being nearly neutral. Overall, our framework provides a basis for integrating evo-devo into the modern synthesis, allowing an operational way to evaluate the ultimate causes of macroevolution. Here the authors show with extant and extinct primate molar data that morphospaces specifically designed to quantify developmental phenomena maximize the ability to bridge the gap between micro- and macroevolution.},
  archive      = {J_NATEE},
  author       = {Machado, Fabio A. and Mongle, Carrie S. and Slater, Graham and Penna, Anna and Wisniewski, Anna and Soffin, Anna and Dutra, Vitor and Uyeda, Josef C.},
  doi          = {10.1038/s41559-023-02167-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1729-1739},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Rules of teeth development align microevolution with macroevolution in extant and extinct primates},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). A lamprey neural cell type atlas illuminates the origins of
the vertebrate brain. <em>NATEE</em>, <em>7</em>(10), 1714–1728. (<a
href="https://doi.org/10.1038/s41559-023-02170-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The vertebrate brain emerged more than ~500 million years ago in common evolutionary ancestors. To systematically trace its cellular and molecular origins, we established a spatially resolved cell type atlas of the entire brain of the sea lamprey—a jawless species whose phylogenetic position affords the reconstruction of ancestral vertebrate traits—based on extensive single-cell RNA-seq and in situ sequencing data. Comparisons of this atlas to neural data from the mouse and other jawed vertebrates unveiled various shared features that enabled the reconstruction of cell types, tissue structures and gene expression programs of the ancestral vertebrate brain. However, our analyses also revealed key tissues and cell types that arose later in evolution. For example, the ancestral brain was probably devoid of cerebellar cell types and oligodendrocytes (myelinating cells); our data suggest that the latter emerged from astrocyte-like evolutionary precursors in the jawed vertebrate lineage. Altogether, our work illuminates the cellular and molecular architecture of the ancestral vertebrate brain and provides a foundation for exploring its diversification during evolution. This study provides a cell atlas of the sea lamprey brain based on single-cell RNA-seq and in situ sequencing data and includes comparative analyses with other vertebrates that reveal key features of the ancestral vertebrate brain as well as traits that arose later in evolution.},
  archive      = {J_NATEE},
  author       = {Lamanna, Francesco and Hervas-Sotomayor, Francisca and Oel, A. Phillip and Jandzik, David and Sobrido-Cameán, Daniel and Santos-Durán, Gabriel N. and Martik, Megan L. and Stundl, Jan and Green, Stephen A. and Brüning, Thoomke and Mößinger, Katharina and Schmidt, Julia and Schneider, Celine and Sepp, Mari and Murat, Florent and Smith, Jeramiah J. and Bronner, Marianne E. and Rodicio, María Celina and Barreiro-Iglesias, Antón and Medeiros, Daniel M. and Arendt, Detlev and Kaessmann, Henrik},
  doi          = {10.1038/s41559-023-02170-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1714-1728},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A lamprey neural cell type atlas illuminates the origins of the vertebrate brain},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Preservation of corneous β-proteins in mesozoic feathers.
<em>NATEE</em>, <em>7</em>(10), 1706–1713. (<a
href="https://doi.org/10.1038/s41559-023-02177-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fossil proteins are valuable tools in evolutionary biology. Recent technological advances and better integration of experimental methods have confirmed the feasibility of biomolecular preservation in deep time, yielding new insights into the timing of key evolutionary transitions. Keratins (formerly α-keratins) and corneous β-proteins (CBPs, formerly β-keratins) are of particular interest as they define tissue structures that underpin fundamental physiological and ecological strategies and have the potential to inform on the molecular evolution of the vertebrate integument. Reports of CBPs in Mesozoic fossils, however, appear to conflict with experimental evidence for CBP degradation during fossilization. Further, the recent model for molecular modification of feather chemistry during the dinosaur–bird transition does not consider the relative preservation potential of different feather proteins. Here we use controlled taphonomic experiments coupled with infrared and sulfur X-ray spectroscopy to show that the dominant β-sheet structure of CBPs is progressively altered to α-helices with increasing temperature, suggesting that (α-)keratins and α-helices in fossil feathers are most likely artefacts of fossilization. Our analyses of fossil feathers shows that this process is independent of geological age, as even Cenozoic feathers can comprise primarily α-helices and disordered structures. Critically, our experiments show that feather CBPs can survive moderate thermal maturation. As predicted by our experiments, analyses of Mesozoic feathers confirm that evidence of feather CBPs can persist through deep time. Taphonomic experiments and analyses of fossil feathers reveal that while fossil keratins (formerly known as α-keratins) found in fossil feathers are likely artefacts of fossilization, corneous β-proteins (formerly β-keratins) can survive moderate thermal damage and persist over hundreds of millions of years to inform understanding of feather evolution.},
  archive      = {J_NATEE},
  author       = {Slater, Tiffany S. and Edwards, Nicholas P. and Webb, Samuel M. and Zhang, Fucheng and McNamara, Maria E.},
  doi          = {10.1038/s41559-023-02177-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1706-1713},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Preservation of corneous β-proteins in mesozoic feathers},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Species-wide genomics of kākāpō provides tools to accelerate
recovery. <em>NATEE</em>, <em>7</em>(10), 1693–1705. (<a
href="https://doi.org/10.1038/s41559-023-02165-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The kākāpō is a critically endangered, intensively managed, long-lived nocturnal parrot endemic to Aotearoa New Zealand. We generated and analysed whole-genome sequence data for nearly all individuals living in early 2018 (169 individuals) to generate a high-quality species-wide genetic variant callset. We leverage extensive long-term metadata to quantify genome-wide diversity of the species over time and present new approaches using probabilistic programming, combined with a phenotype dataset spanning five decades, to disentangle phenotypic variance into environmental and genetic effects while quantifying uncertainty in small populations. We find associations for growth, disease susceptibility, clutch size and egg fertility within genic regions previously shown to influence these traits in other species. Finally, we generate breeding values to predict phenotype and illustrate that active management over the past 45 years has maintained both genome-wide diversity and diversity in breeding values and, hence, evolutionary potential. We provide new pathways for informing future conservation management decisions for kākāpō, including prioritizing individuals for translocation and monitoring individuals with poor growth or high disease risk. Overall, by explicitly addressing the challenge of the small sample size, we provide a template for the inclusion of genomic data that will be transformational for species recovery efforts around the globe. The kākāpō is an intensively managed parrot endemic to New Zealand. Using genome sequencing data for all living kākāpō together with long-term phenotypic data, the authors devise an approach to identify genetic associations with fitness traits, which is informing species recovery plans.},
  archive      = {J_NATEE},
  author       = {Guhlin, Joseph and Le Lec, Marissa F. and Wold, Jana and Koot, Emily and Winter, David and Biggs, Patrick J. and Galla, Stephanie J. and Urban, Lara and Foster, Yasmin and Cox, Murray P. and Digby, Andrew and Uddstrom, Lydia R. and Eason, Daryl and Vercoe, Deidre and Davis, Tāne and Howard, Jason T. and Jarvis, Erich D. and Robertson, Fiona E. and Robertson, Bruce C. and Gemmell, Neil J. and Steeves, Tammy E. and Santure, Anna W. and Dearden, Peter K.},
  doi          = {10.1038/s41559-023-02165-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1693-1705},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Species-wide genomics of kākāpō provides tools to accelerate recovery},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). An invasive pathogen drives directional niche contractions
in amphibians. <em>NATEE</em>, <em>7</em>(10), 1682–1692. (<a
href="https://doi.org/10.1038/s41559-023-02155-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global change is causing an unprecedented restructuring of ecosystems, with the spread of invasive species being a key driver. While population declines of native species due to invasives are well documented, much less is known about whether new biotic interactions reshape niches of native species. Here we quantify geographic range and realized-niche contractions in Australian frog species following the introduction of amphibian chytrid fungus Batrachochytrium dendrobatidis, a pathogen responsible for catastrophic amphibian declines worldwide. We show that chytrid-impacted species experienced proportionately greater contractions in niche breadth than geographic distribution following chytrid emergence. Furthermore, niche contractions were directional, with contemporary distributions of chytrid-impacted species characterized by higher temperatures, lower diurnal temperature range, higher precipitation and lower elevations. Areas with these conditions may enable host persistence with chytrid through lower pathogenicity of the fungus and/or greater demographic resilience. Nevertheless, contraction to a narrower subset of environmental conditions could increase host vulnerability to other threatening processes and should be considered in assessments of extinction risk and during conservation planning. More broadly, our results emphasize that biotic interactions can strongly shape species realized niches and that large-scale niche contractions due to new species interactions—particularly emerging pathogens—could be widespread. Analysing the extent of occurrence and niche hypervolumes of 55 species of frogs in eastern Australia, the authors show that species impacted by the introduction of the pathogenic amphibian chytrid fungus underwent niche contractions, and that these were in a direction that could inhibit chytrid fungus and/or promote host demographic resilience.},
  archive      = {J_NATEE},
  author       = {Scheele, Ben C. and Heard, Geoffrey W. and Cardillo, Marcel and Duncan, Richard P. and Gillespie, Graeme R. and Hoskin, Conrad J. and Mahony, Michael and Newell, David and Rowley, Jodi J. L. and Sopniewski, Jarrod},
  doi          = {10.1038/s41559-023-02155-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1682-1692},
  shortjournal = {Nat. Ecol. Evol},
  title        = {An invasive pathogen drives directional niche contractions in amphibians},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pathogens are an important driving force for the rapid
spread of symbionts in an insect host. <em>NATEE</em>, <em>7</em>(10),
1667–1681. (<a
href="https://doi.org/10.1038/s41559-023-02160-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {One of the biggest challenges for pathogens invading hosts is microbial symbionts but the role of pathogens in symbionts in nature is unknown. By tracking the dynamics of the entomopathogenic fungal Cordyceps javanica and symbionts in natural populations of the whitefly Bemisia tabaci from 2016 to 2021 across China, we reveal that Rickettsia, a newly invaded symbiont, is positively correlated with the pathogen in both frequency and density. We confirm that applying pathogen pressure can selectively drive Rickettsia to sudden fixation in whiteflies both in the laboratory and in the field. Furthermore, the driving force is elucidated by the Rickettsia-conferred suppression of pathogen infection quantity, proliferation and sporulation, acting as a potential barrier of onward transmission of the pathogen. These results show that pathogens are an important driving force for rapid shifts in host symbionts in the natural niche. A combination of field and experimental work suggests that fungal pathogen pressure can selectively drive the symbiont Ricksettia to fixation in its whitefly host, through Ricksettia-conferred suppression of the pathogen.},
  archive      = {J_NATEE},
  author       = {Zhao, Dongxiao and Zhang, Zhichun and Niu, Hongtao and Guo, Huifang},
  doi          = {10.1038/s41559-023-02160-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1667-1681},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pathogens are an important driving force for the rapid spread of symbionts in an insect host},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Urbanization, climate and species traits shape mammal
communities from local to continental scales. <em>NATEE</em>,
<em>7</em>(10), 1654–1666. (<a
href="https://doi.org/10.1038/s41559-023-02166-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human-driven environmental changes shape ecological communities from local to global scales. Within cities, landscape-scale patterns and processes and species characteristics generally drive local-scale wildlife diversity. However, cities differ in their structure, species pools, geographies and histories, calling into question the extent to which these drivers of wildlife diversity are predictive at continental scales. In partnership with the Urban Wildlife Information Network, we used occurrence data from 725 sites located across 20 North American cities and a multi-city, multi-species occupancy modelling approach to evaluate the effects of ecoregional characteristics and mammal species traits on the urbanization–diversity relationship. Among 37 native terrestrial mammal species, regional environmental characteristics and species traits influenced within-city effects of urbanization on species occupancy and community composition. Species occupancy and diversity were most negatively related to urbanization in the warmer, less vegetated cities. Additionally, larger-bodied species were most negatively impacted by urbanization across North America. Our results suggest that shifting climate conditions could worsen the effects of urbanization on native wildlife communities, such that conservation strategies should seek to mitigate the combined effects of a warming and urbanizing world. Multi-species occupancy modelling using camera-trap data from 725 sites across 20 North American cities shows how environmental and climate characteristics interact with species traits to influence the effects of urbanization on wildlife communities.},
  archive      = {J_NATEE},
  author       = {Haight, Jeffrey D. and Hall, Sharon J. and Fidino, Mason and Adalsteinsson, Solny A. and Ahlers, Adam A. and Angstmann, Julia and Anthonysamy, Whitney J. B. and Biro, Elizabeth and Collins, Merri K. and Dugelby, Barbara and Gallo, Travis and Green, Austin M. and Hartley, Laura and Jordan, Mark J. and Kay, Cria A. M. and Lehrer, Elizabeth W. and Long, Robert A. and MacDougall, Brandon and Magle, Seth B. and Minier, Darren E. and Mowry, Chris and Murray, Maureen and Nininger, Kristina and Pendergast, Mary E. and Remine, Katie R. and Ryan, Travis and Salsbury, Carmen and Sander, Heather A. and Schell, Christopher J. and Șekercioğlu, Çagan H. and Shier, Catherine J. and Simon, Kelly C. and St. Clair, Colleen C. and Stankowich, Theodore and Stevenson, Cassondra J. and Wayne, Lisa and Will, Dave and Williamson, Jacque and Wilson, Larry and Zellmer, Amanda J. and Lewis, Jesse S.},
  doi          = {10.1038/s41559-023-02166-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1654-1666},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Urbanization, climate and species traits shape mammal communities from local to continental scales},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Megaherbivores provide biotic resistance against alien plant
dominance. <em>NATEE</em>, <em>7</em>(10), 1645–1653. (<a
href="https://doi.org/10.1038/s41559-023-02181-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While human-driven biological invasions are rapidly spreading, finding scalable and effective control methods poses an unresolved challenge. Here, we assess whether megaherbivores—herbivores reaching ≥1,000 kg of body mass—offer a nature-based solution to plant invasions. Invasive plants are generally adapted to maximize vegetative growth. Megaherbivores, with broad dietary tolerances, could remove large biomass of established plants, facilitating new plant growth. We used a massive dataset obtained from 26,838 camera stations and 158,979 vegetation plots to assess the relationships between megaherbivores, native plants and alien plants across India (~121,330 km2). We found a positive relationship between megaherbivore abundance and native plant richness and abundance, and a concomitant reduction in alien plant abundance. This relationship was strongest in protected areas with midproductive ecosystem and high megaherbivore density but it was lost in areas where thicket-forming alien plants predominated (&amp;gt;40% cover). By incorporating the role of ecosystem productivity, plants traits and densities of megaherbivores on megaherbivore–vegetation relationships, our study highlights a function of megaherbivores in controlling alien plant proliferation and facilitating diverse native plants in invaded ecosystems. The study shows great potential for megafauna-based trophic rewilding as a nature-based solution to counteract dominance of plant invasions. Analysing camera trap data from &amp;gt;26,000 stations and &amp;gt;158,000 vegetation plots across 12 ecoregions in India, the authors find a positive relationship between megaherbivore abundance and native plant species richness and abundance, but reductions in alien plant species abundance.},
  archive      = {J_NATEE},
  author       = {Mungi, Ninad Avinash and Jhala, Yadvendradev V. and Qureshi, Qamar and le Roux, Elizabeth and Svenning, Jens-Christian},
  doi          = {10.1038/s41559-023-02181-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1645-1653},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Megaherbivores provide biotic resistance against alien plant dominance},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evolutionary imbalance, climate and human history jointly
shape the global biogeography of alien plants. <em>NATEE</em>,
<em>7</em>(10), 1633–1644. (<a
href="https://doi.org/10.1038/s41559-023-02172-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human activities are causing global biotic redistribution, translocating species and providing them with opportunities to establish populations beyond their native ranges. Species originating from certain global regions, however, are disproportionately represented among naturalized aliens. The evolutionary imbalance hypothesis posits that differences in absolute fitness among biogeographic divisions determine outcomes when biotas mix. Here, we compile data from native and alien distributions for nearly the entire global seed plant flora and find that biogeographic conditions predicted to drive evolutionary imbalance act alongside climate and anthropogenic factors to shape flows of successful aliens among regional biotas. Successful aliens tend to originate from large, biodiverse regions that support abundant populations and where species evolve against a diverse backdrop of competitors and enemies. We also reveal that these same native distribution characteristics are shared among the plants that humans select for cultivation and economic use. In addition to influencing species’ innate potentials as invaders, we therefore suggest that evolutionary imbalance shapes plants’ relationships with humans, impacting which species are translocated beyond their native distributions. Analysing the distributions of &amp;gt;99% of the world’s seed plant flora, the authors find evidence supporting the evolutionary imbalance hypothesis, with species originating from large, biodiverse regions being more successful at naturalizing beyond their native ranges.},
  archive      = {J_NATEE},
  author       = {Fristoe, Trevor S. and Bleilevens, Jonas and Kinlock, Nicole L. and Yang, Qiang and Zhang, Zhijie and Dawson, Wayne and Essl, Franz and Kreft, Holger and Pergl, Jan and Pyšek, Petr and Weigelt, Patrick and Dufour-Dror, Jean-Marc and Sennikov, Alexander N. and Wasowicz, Pawel and Westergaard, Kristine B. and van Kleunen, Mark},
  doi          = {10.1038/s41559-023-02172-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1633-1644},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolutionary imbalance, climate and human history jointly shape the global biogeography of alien plants},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Increased hydraulic risk in assemblages of woody plant
species predicts spatial patterns of drought-induced mortality.
<em>NATEE</em>, <em>7</em>(10), 1620–1632. (<a
href="https://doi.org/10.1038/s41559-023-02180-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Predicting drought-induced mortality (DIM) of woody plants remains a key research challenge under climate change. Here, we integrate information on the edaphoclimatic niches, phylogeny and hydraulic traits of species to model the hydraulic risk of woody plants globally. We combine these models with species distribution records to estimate the hydraulic risk faced by local woody plant species assemblages. Thus, we produce global maps of hydraulic risk and test for its relationship with observed DIM. Our results show that local assemblages modelled as having higher hydraulic risk present a higher probability of DIM. Metrics characterizing this hydraulic risk improve DIM predictions globally, relative to models accounting only for edaphoclimatic predictors or broad functional groupings. The methodology we present here allows mapping of functional trait distributions and elucidation of global macro-evolutionary and biogeographical patterns, improving our ability to predict potential global change impacts on vegetation. Combining global-scale data on species’ edaphoclimatic niches, phylogeny and hydraulic traits for &amp;gt;44,000 woody plant species, the authors map areas of hydraulic risk and show that local assemblages at greater hydraulic risk have a higher probability of drought-induced mortality.},
  archive      = {J_NATEE},
  author       = {Sanchez-Martinez, Pablo and Mencuccini, Maurizio and García-Valdés, Raúl and Hammond, William M. and Serra-Diaz, Josep M. and Guo, Wen-Yong and Segovia, Ricardo A. and Dexter, Kyle G. and Svenning, Jens-Christian and Allen, Craig and Martínez-Vilalta, Jordi},
  doi          = {10.1038/s41559-023-02180-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1620-1632},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Increased hydraulic risk in assemblages of woody plant species predicts spatial patterns of drought-induced mortality},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Time delays modulate the stability of complex ecosystems.
<em>NATEE</em>, <em>7</em>(10), 1610–1619. (<a
href="https://doi.org/10.1038/s41559-023-02158-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {What drives the stability, or instability, of complex ecosystems? This question sits at the heart of community ecology and has motivated a large body of theoretical work exploring how community properties shape ecosystem dynamics. However, the overwhelming majority of current theory assumes that species interactions are instantaneous, meaning that changes in the abundance of one species will lead to immediate changes in the abundances of its partners. In practice, time delays in how species respond to one another are widespread across ecological contexts, yet the impact of these delays on ecosystems remains unclear. Here we derive a new body of theory to comprehensively study the impact of time delays on ecological stability. We find that time delays are important for ecosystem stability. Large delays are typically destabilizing but, surprisingly, short delays can substantially increase community stability. Moreover, in stark contrast to delay-free systems, delays dictate that communities with more abundant species can be less stable than ones with less abundant species. Finally, we show that delays fundamentally shift how species interactions impact ecosystem stability, with communities of mixed interaction types becoming the most stable class of ecosystem. Our work demonstrates that time delays can be critical for the stability of complex ecosystems. Time delays in the responses of species to one another are expected to occur widely in nature. Using a new theoretical framework, the authors show that these delays can fundamentally shift how different communities respond to perturbations.},
  archive      = {J_NATEE},
  author       = {Yang, Yuguang and Foster, Kevin R. and Coyte, Katharine Z. and Li, Aming},
  doi          = {10.1038/s41559-023-02158-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1610-1619},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Time delays modulate the stability of complex ecosystems},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Unveiling global species abundance distributions.
<em>NATEE</em>, <em>7</em>(10), 1600–1609. (<a
href="https://doi.org/10.1038/s41559-023-02173-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Whether most species are rare or have some intermediate abundance is a long-standing question in ecology. Here, we use more than one billion observations from the Global Biodiversity Information Facility to assess global species abundance distributions (gSADs) of 39 taxonomic classes of eukaryotic organisms from 1900 to 2019. We show that, as sampling effort increases through time, the shape of the gSAD is unveiled; that is, the shape of the sampled gSAD changes, revealing the underlying gSAD. The fraction of species unveiled for each class decreases with the total number of species in that class and increases with the number of individuals sampled, with some groups, such as birds, being fully unveiled. The best statistical fit for almost all classes was the Poisson log-normal distribution. This strong evidence for a universal pattern of gSADs across classes suggests that there may be general ecological or evolutionary mechanisms governing the commonness and rarity of life on Earth. Using occurrence records from the Global Biodiversity Information Facility, the authors show that the global species abundance distribution of 39 taxonomic classes of eukaryotes is best fit by a Poisson log-normal distribution.},
  archive      = {J_NATEE},
  author       = {Callaghan, Corey T. and Borda-de-Água, Luís and van Klink, Roel and Rozzi, Roberto and Pereira, Henrique M.},
  doi          = {10.1038/s41559-023-02173-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1600-1609},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Unveiling global species abundance distributions},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Recognizing and marshalling the pre-publication error
correction potential of open data for more reproducible science.
<em>NATEE</em>, <em>7</em>(10), 1597–1599. (<a
href="https://doi.org/10.1038/s41559-023-02152-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Chen, Rebecca Shuhua and Berthelsen, Ane Liv and Lamartinière, Etienne Brejon and Spangenberg, Matthias Christian and Schmoll, Tim},
  doi          = {10.1038/s41559-023-02152-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1597-1599},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Recognizing and marshalling the pre-publication error correction potential of open data for more reproducible science},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reply to: Recognizing and marshalling the pre-publication
error correction potential of open data for more reproducible science.
<em>NATEE</em>, <em>7</em>(10), 1595–1596. (<a
href="https://doi.org/10.1038/s41559-023-02142-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Berberi, Ilias and Roche, Dominique G.},
  doi          = {10.1038/s41559-023-02142-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1595-1596},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reply to: Recognizing and marshalling the pre-publication error correction potential of open data for more reproducible science},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A fungal pathogen drives the spread of a defensive symbiont
in an insect host. <em>NATEE</em>, <em>7</em>(10), 1593–1594. (<a
href="https://doi.org/10.1038/s41559-023-02140-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A defensive bacterial symbiont, spreading rapidly through populations of whitefly in nature, suppresses the proliferation, sporulation and transmission of a fungal pathogen in the whitefly. The pathogen is shown to be an important driving force for rapid shifts of the symbiont in the natural niche.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02140-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1593-1594},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A fungal pathogen drives the spread of a defensive symbiont in an insect host},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A cell type atlas of the lamprey brain. <em>NATEE</em>,
<em>7</em>(10), 1591–1592. (<a
href="https://doi.org/10.1038/s41559-023-02195-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A cell atlas of the lamprey brain reveals the evolutionary origin of specialized vertebrate glia and neuron types},
  archive      = {J_NATEE},
  author       = {Gumnit, Elias and Tosches, Maria Antonietta},
  doi          = {10.1038/s41559-023-02195-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1591-1592},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A cell type atlas of the lamprey brain},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). New tools for the recovery of the kākāpō. <em>NATEE</em>,
<em>7</em>(10), 1589–1590. (<a
href="https://doi.org/10.1038/s41559-023-02112-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genome sequences of all living kākāpō provide new approaches for evidence-based conservation management, including the identification of genomic regions that are associated with fitness traits, at a time of increased need for breeding programmes for species recovery.},
  archive      = {J_NATEE},
  author       = {Taylor, Rebecca S.},
  doi          = {10.1038/s41559-023-02112-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1589-1590},
  shortjournal = {Nat. Ecol. Evol},
  title        = {New tools for the recovery of the kākāpō},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chytrid invasion drives frog redistributions.
<em>NATEE</em>, <em>7</em>(10), 1587–1588. (<a
href="https://doi.org/10.1038/s41559-023-02184-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of the geographical range and climatic niche dynamics of Australian frogs highlights the role of an emerging chytrid fungal disease in reshaping the distributions of native species through novel host–pathogen interactions.},
  archive      = {J_NATEE},
  author       = {García-Rodríguez, Adrián and Zumbado-Ulate, Héctor},
  doi          = {10.1038/s41559-023-02184-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1587-1588},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Chytrid invasion drives frog redistributions},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Warming amplifies urbanization effects on mammals.
<em>NATEE</em>, <em>7</em>(10), 1585–1586. (<a
href="https://doi.org/10.1038/s41559-023-02164-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Camera-trap data from across 20 North American cities show how urbanization affects multiple mammal species, and that these effects are stronger in warmer and less-vegetated cities.},
  archive      = {J_NATEE},
  author       = {Liu, Jiajia},
  doi          = {10.1038/s41559-023-02164-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1585-1586},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Warming amplifies urbanization effects on mammals},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Origins of successful invasions. <em>NATEE</em>,
<em>7</em>(10), 1583–1584. (<a
href="https://doi.org/10.1038/s41559-023-02190-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of distributions of over 99% of the world’s seed plant flora shows that species that originate from large and biodiverse regions are more successful at establishing naturalized populations outside of their native range. They are also more likely to be used by humans for economic purposes.},
  archive      = {J_NATEE},
  author       = {Moyano, Jaime},
  doi          = {10.1038/s41559-023-02190-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1583-1584},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Origins of successful invasions},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Strong selection in male haplodiploids. <em>NATEE</em>,
<em>7</em>(10), 1582. (<a
href="https://doi.org/10.1038/s41559-023-02183-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Gilligan, Conor},
  doi          = {10.1038/s41559-023-02183-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1582},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Strong selection in male haplodiploids},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Local networks inferred from regional knowledge.
<em>NATEE</em>, <em>7</em>(10), 1581. (<a
href="https://doi.org/10.1038/s41559-023-02188-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bhaumik, Vaishali},
  doi          = {10.1038/s41559-023-02188-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1581},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Local networks inferred from regional knowledge},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Cultural lens on cooperation. <em>NATEE</em>,
<em>7</em>(10), 1580. (<a
href="https://doi.org/10.1038/s41559-023-02176-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Grinham, Luke R.},
  doi          = {10.1038/s41559-023-02176-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1580},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Cultural lens on cooperation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Lower lignin for wood efficiency. <em>NATEE</em>,
<em>7</em>(10), 1579. (<a
href="https://doi.org/10.1038/s41559-023-02175-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-023-02175-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1579},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Lower lignin for wood efficiency},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Sloth ornaments trace the first americans. <em>NATEE</em>,
<em>7</em>(10), 1578. (<a
href="https://doi.org/10.1038/s41559-023-02168-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-023-02168-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1578},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sloth ornaments trace the first americans},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rekindling our love for nature. <em>NATEE</em>,
<em>7</em>(10), 1576–1577. (<a
href="https://doi.org/10.1038/s41559-023-02169-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Soga, Masashi},
  doi          = {10.1038/s41559-023-02169-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1576-1577},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Rekindling our love for nature},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Priorities for progress towards sustainable development goal
15 “life on land.” <em>NATEE</em>, <em>7</em>(10), 1570–1575. (<a
href="https://doi.org/10.1038/s41559-023-02209-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This year marks the mid-point for the implementation of the United Nations 2030 Agenda for Sustainable Development and its 17 Sustainable Development Goals, including Sustainable Development Goal 15 ‘Life on land’. We asked a range of researchers, working across biodiversity science, conservation, policy and implementation, to reflect on priorities for action on conserving terrestrial and freshwater ecosystems over the next seven years.},
  archive      = {J_NATEE},
  author       = {Cooper, H. David and Mumba, Musonda and Dhyani, Shalini and Shen, Xiaoli and Ma, Keping and Wynberg, Rachel and Laird, Sarah and Rodríguez, Jon Paul},
  doi          = {10.1038/s41559-023-02209-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1570-1575},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Priorities for progress towards sustainable development goal 15 ‘Life on land’},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Priorities for progress towards sustainable development goal
14 “life below water.” <em>NATEE</em>, <em>7</em>(10), 1564–1569. (<a
href="https://doi.org/10.1038/s41559-023-02208-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This year marks the mid-point for the implementation of the United Nations 2030 Agenda for Sustainable Development and its 17 Sustainable Development Goals, including Sustainable Development Goal 14 (‘Life below water’). We asked a range of researchers working across marine science, conservation, policy and implementation to reflect on priorities for action on ocean health and biodiversity over the next seven years.},
  archive      = {J_NATEE},
  author       = {Lubchenco, Jane and Camp, Emma F. and Vargas, Cristian A. and Belhabib, Dyhia and Anna, Zuzy and Amon, Diva J. and Metaxas, Anna and Harden-Davies, Harriet},
  doi          = {10.1038/s41559-023-02208-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1564-1569},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Priorities for progress towards sustainable development goal 14 ‘Life below water’},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Richard s. Gomulkiewicz (1962–2023). <em>NATEE</em>,
<em>7</em>(10), 1562–1563. (<a
href="https://doi.org/10.1038/s41559-023-02136-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Incisive theoretical population geneticist.},
  archive      = {J_NATEE},
  author       = {Holt, Robert D. and Hastings, Alan and Nuismer, Scott L.},
  doi          = {10.1038/s41559-023-02136-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1562-1563},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Richard s. gomulkiewicz (1962–2023)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Charles kimberlin (bob) brain (1931–2023). <em>NATEE</em>,
<em>7</em>(10), 1560–1561. (<a
href="https://doi.org/10.1038/s41559-023-02129-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {African naturalist and trailblazer in the taphonomy of human origins},
  archive      = {J_NATEE},
  author       = {Pickering, Travis Rayne and Kuman, Kathleen and Clarke, Ronald J. and Heaton, Jason L.},
  doi          = {10.1038/s41559-023-02129-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1560-1561},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Charles kimberlin (Bob) brain (1931–2023)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Successful implementation of global targets to reduce
nutrient and pesticide pollution requires suitable indicators.
<em>NATEE</em>, <em>7</em>(10), 1556–1559. (<a
href="https://doi.org/10.1038/s41559-023-02120-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Indicators proposed for nutrient and pesticide pollution in the current text of the Convention on Biological Diversity’s post-2020 Global Biodiversity Framework (GBF) are inadequate for tracking progress and informing policy. We highlight a set of more relevant pollution indicators that would strengthen the monitoring framework of the GBF and discuss conditions for their successful implementation.},
  archive      = {J_NATEE},
  author       = {Möhring, Niklas and Kanter, David and Aziz, Tariq and Castro, Italo B. and Maggi, Federico and Schulte-Uebbing, Lena and Seufert, Verena and Tang, Fiona H. M. and Zhang, Xin and Leadley, Paul},
  doi          = {10.1038/s41559-023-02120-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1556-1559},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Successful implementation of global targets to reduce nutrient and pesticide pollution requires suitable indicators},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Better integration of chemical pollution research will
further our understanding of biodiversity loss. <em>NATEE</em>,
<em>7</em>(10), 1552–1555. (<a
href="https://doi.org/10.1038/s41559-023-02117-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chemical pollution research should be better integrated with other drivers of biodiversity loss and the assessment of human impacts on ecosystems, to more effectively guide management strategies for biodiversity loss mitigation.},
  archive      = {J_NATEE},
  author       = {Sylvester, Francisco and Weichert, Fabian G. and Lozano, Verónica L. and Groh, Ksenia J. and Bálint, Miklós and Baumann, Lisa and Bässler, Claus and Brack, Werner and Brandl, Barbara and Curtius, Joachim and Dierkes, Paul and Döll, Petra and Ebersberger, Ingo and Fragkostefanakis, Sotirios and Helfrich, Eric J. N. and Hickler, Thomas and Johann, Sarah and Jourdan, Jonas and Klimpel, Sven and Kminek, Helge and Liquin, Florencia and Möllendorf, Darrel and Mueller, Thomas and Oehlmann, Jörg and Ottermanns, Richard and Pauls, Steffen U. and Piepenbring, Meike and Pfefferle, Jakob and Schenk, Gerrit Jasper and Scheepens, J. F. and Scheringer, Martin and Schiwy, Sabrina and Schlottmann, Antje and Schneider, Flurina and Schulte, Lisa M. and Schulze-Sylvester, Maria and Stelzer, Ernst and Strobl, Frederic and Sundermann, Andrea and Tockner, Klement and Tröger, Tobias and Vilcinskas, Andreas and Völker, Carolin and Winkelmann, Ricarda and Hollert, Henner},
  doi          = {10.1038/s41559-023-02117-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1552-1555},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Better integration of chemical pollution research will further our understanding of biodiversity loss},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Applying the “CARE principles for indigenous data
governance” to ecology and biodiversity research. <em>NATEE</em>,
<em>7</em>(10), 1547–1551. (<a
href="https://doi.org/10.1038/s41559-023-02161-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Indigenous Peoples are increasingly being sought out for research partnerships that incorporate Indigenous Knowledges into ecology research. In such research partnerships, it is essential that Indigenous data are cared for ethically and responsibly. Here we outline how the ‘CARE Principles for Indigenous Data Governance’ can sow community ethics into disciplines that are inundated with extractive helicopter research practices, and we provide standardized practices for evolving data and research landscapes.},
  archive      = {J_NATEE},
  author       = {Jennings, Lydia and Anderson, Talia and Martinez, Andrew and Sterling, Rogena and Chavez, Dominique David and Garba, Ibrahim and Hudson, Maui and Garrison, Nanibaa’ A. and Carroll, Stephanie Russo},
  doi          = {10.1038/s41559-023-02161-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1547-1551},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Applying the ‘CARE principles for indigenous data governance’ to ecology and biodiversity research},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Research codes and contracts do not guarantee equitable
research with indigenous communities. <em>NATEE</em>, <em>7</em>(10),
1543–1546. (<a
href="https://doi.org/10.1038/s41559-023-02101-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Research codes and contracts have been developed to protect Indigenous and marginalized peoples from exploitation and to promote inclusion, so that research will become more beneficial to them. We highlight three important but often overlooked challenges for such instruments, drawing on examples from the San of southern Africa.},
  archive      = {J_NATEE},
  author       = {Koot, Stasja and Grant, Julie and //Khumûb, Moses and Fernando, Kileni and Mushavanga, Tienie and Dommerholt, Taliah and Gressier, Catie and Pienaar, Dirk and /Ui Kunta, Steve and Puckett, Robert Fleming and Paksi, Attila and Moeti, Smith and ≠Oma Tsamkgao, Leon and Steenkamp, Luce and Hitchcock, Robert and Maruyama, Junko and Gordon, Robert and Mushavanga, David},
  doi          = {10.1038/s41559-023-02101-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1543-1546},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Research codes and contracts do not guarantee equitable research with indigenous communities},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The plight and role of wild birds in the current bird flu
panzootic. <em>NATEE</em>, <em>7</em>(10), 1541–1542. (<a
href="https://doi.org/10.1038/s41559-023-02182-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Klaassen, Marcel and Wille, Michelle},
  doi          = {10.1038/s41559-023-02182-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1541-1542},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The plight and role of wild birds in the current bird flu panzootic},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). SDGs at the halfway mark. <em>NATEE</em>, <em>7</em>(10),
1539–1540. (<a
href="https://doi.org/10.1038/s41559-023-02219-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biodiversity directly and indirectly contributes to all 17 of the UN’s Sustainable Development Goals. Making meaningful progress towards achieving these goals in the next seven years will require embracing their interdependencies.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02219-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {10},
  number       = {10},
  pages        = {1539-1540},
  shortjournal = {Nat. Ecol. Evol},
  title        = {SDGs at the halfway mark},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Publisher correction: Heavy reliance on private finance
alone will not deliver conservation goals. <em>NATEE</em>,
<em>7</em>(9), 1538. (<a
href="https://doi.org/10.1038/s41559-023-02178-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Kedward, Katie and zu Ermgassen, Sophus and Ryan-Collins, Josh and Wunder, Sven},
  doi          = {10.1038/s41559-023-02178-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1538},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Publisher correction: Heavy reliance on private finance alone will not deliver conservation goals},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: Dispersal governs the reorganization of
ecological networks under environmental change. <em>NATEE</em>,
<em>7</em>(9), 1537. (<a
href="https://doi.org/10.1038/s41559-023-02163-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Thompson, Patrick L. and Gonzalez, Andrew},
  doi          = {10.1038/s41559-023-02163-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1537},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Dispersal governs the reorganization of ecological networks under environmental change},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Empirical evidence of widespread exaggeration bias and
selective reporting in ecology. <em>NATEE</em>, <em>7</em>(9),
1525–1536. (<a
href="https://doi.org/10.1038/s41559-023-02144-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In many scientific disciplines, common research practices have led to unreliable and exaggerated evidence about scientific phenomena. Here we describe some of these practices and quantify their pervasiveness in recent ecology publications in five popular journals. In an analysis of over 350 studies published between 2018 and 2020, we detect empirical evidence of exaggeration bias and selective reporting of statistically significant results. This evidence implies that the published effect sizes in ecology journals exaggerate the importance of the ecological relationships that they aim to quantify. An exaggerated evidence base hinders the ability of empirical ecology to reliably contribute to science, policy, and management. To increase the credibility of ecology research, we describe a set of actions that ecologists should take, including changes to scientific norms about what high-quality ecology looks like and expectations about what high-quality studies can deliver. Analysing &amp;gt;18,000 effect sizes from recent ecology studies published in five leading journals, the authors identify widespread under-powered designs, exaggeration bias, selective reporting and few corrections for multiple hypotheses among statistically significant results.},
  archive      = {J_NATEE},
  author       = {Kimmel, Kaitlin and Avolio, Meghan L. and Ferraro, Paul J.},
  doi          = {10.1038/s41559-023-02144-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1525-1536},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Empirical evidence of widespread exaggeration bias and selective reporting in ecology},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A genetic history of continuity and mobility in the iron age
central mediterranean. <em>NATEE</em>, <em>7</em>(9), 1515–1524. (<a
href="https://doi.org/10.1038/s41559-023-02143-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Iron Age was a dynamic period in central Mediterranean history, with the expansion of Greek and Phoenician colonies and the growth of Carthage into the dominant maritime power of the Mediterranean. These events were facilitated by the ease of long-distance travel following major advances in seafaring. We know from the archaeological record that trade goods and materials were moving across great distances in unprecedented quantities, but it is unclear how these patterns correlate with human mobility. Here, to investigate population mobility and interactions directly, we sequenced the genomes of 30 ancient individuals from coastal cities around the central Mediterranean, in Tunisia, Sardinia and central Italy. We observe a meaningful contribution of autochthonous populations, as well as highly heterogeneous ancestry including many individuals with non-local ancestries from other parts of the Mediterranean region. These results highlight both the role of local populations and the extreme interconnectedness of populations in the Iron Age Mediterranean. By studying these trans-Mediterranean neighbours together, we explore the complex interplay between local continuity and mobility that shaped the Iron Age societies of the central Mediterranean. The authors generate genomic data from 30 ancient human individuals, spanning the Bronze Age and the Iron Age from four archaeological sites in the Mediterranean (located in Tunisia, Sardinia and central Italy). Comparing with additional published ancient genomes, they generate insights into mobility and admixture in this interconnected region},
  archive      = {J_NATEE},
  author       = {Moots, Hannah M. and Antonio, Margaret and Sawyer, Susanna and Spence, Jeffrey P. and Oberreiter, Victoria and Weiß, Clemens L. and Lucci, Michaela and Cherifi, Yahia Mehdi Seddik and La Pastina, Francesco and Genchi, Francesco and Praxmeier, Elisa and Zagorc, Brina and Cheronet, Olivia and Özdoğan, Kadir T. and Demetz, Lea and Amrani, Selma and Candilio, Francesca and De Angelis, Daniela and Gasperetti, Gabriella and Fernandes, Daniel and Gao, Ziyue and Fantar, Mounir and Coppa, Alfredo and Pritchard, Jonathan K. and Pinhasi, Ron},
  doi          = {10.1038/s41559-023-02143-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1515-1524},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A genetic history of continuity and mobility in the iron age central mediterranean},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ghost admixture in eastern gorillas. <em>NATEE</em>,
<em>7</em>(9), 1503–1514. (<a
href="https://doi.org/10.1038/s41559-023-02145-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Archaic admixture has had a substantial impact on human evolution with multiple events across different clades, including from extinct hominins such as Neanderthals and Denisovans into modern humans. In great apes, archaic admixture has been identified in chimpanzees and bonobos but the possibility of such events has not been explored in other species. Here, we address this question using high-coverage whole-genome sequences from all four extant gorilla subspecies, including six newly sequenced eastern gorillas from previously unsampled geographic regions. Using approximate Bayesian computation with neural networks to model the demographic history of gorillas, we find a signature of admixture from an archaic ‘ghost’ lineage into the common ancestor of eastern gorillas but not western gorillas. We infer that up to 3% of the genome of these individuals is introgressed from an archaic lineage that diverged more than 3 million years ago from the common ancestor of all extant gorillas. This introgression event took place before the split of mountain and eastern lowland gorillas, probably more than 40 thousand years ago and may have influenced perception of bitter taste in eastern gorillas. When comparing the introgression landscapes of gorillas, humans and bonobos, we find a consistent depletion of introgressed fragments on the X chromosome across these species. However, depletion in protein-coding content is not detectable in eastern gorillas, possibly as a consequence of stronger genetic drift in this species. Using a Bayesian approach with neural networks the authors model the demographic history of gorillas, finding admixture from an archaic ‘ghost’ lineage into the common ancestor of eastern gorillas, but not western gorillas.},
  archive      = {J_NATEE},
  author       = {Pawar, Harvinder and Rymbekova, Aigerim and Cuadros-Espinoza, Sebastian and Huang, Xin and de Manuel, Marc and van der Valk, Tom and Lobon, Irene and Alvarez-Estape, Marina and Haber, Marc and Dolgova, Olga and Han, Sojung and Esteller-Cucala, Paula and Juan, David and Ayub, Qasim and Bautista, Ruben and Kelley, Joanna L. and Cornejo, Omar E. and Lao, Oscar and Andrés, Aida M. and Guschanski, Katerina and Ssebide, Benard and Cranfield, Mike and Tyler-Smith, Chris and Xue, Yali and Prado-Martinez, Javier and Marques-Bonet, Tomas and Kuhlwilm, Martin},
  doi          = {10.1038/s41559-023-02145-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1503-1514},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ghost admixture in eastern gorillas},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Fluctuating climate and dietary innovation drove ratcheted
evolution of proboscidean dental traits. <em>NATEE</em>, <em>7</em>(9),
1490–1502. (<a
href="https://doi.org/10.1038/s41559-023-02151-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identification of the selective forces that shaped adaptive phenotypes generally relies on current habitat and function, but these may differ from the context in which adaptations arose. Moreover, the fixation of adaptive change in a fluctuating environment and the mechanisms of long-term trends are still poorly understood, as is the role of behaviour in triggering these processes. Time series of fossils can provide evidence on these questions, but examples of individual lineages with adequate fossil and proxy data over extended periods are rare. Here, we present new data on proboscidean dental evolution in East Africa over the past 26 million years, tracking temporal patterns of morphological change in relation to proxy evidence of diet, vegetation and climate (aridity). We show that behavioural experimentation in diet is correlated with environmental context, and that major adaptive change in dental traits followed the changes in diet and environment but only after acquisition of functional innovations in the masticatory system. We partition traits by selective agent, showing that the acquisition of high, multiridged molars was primarily a response to an increase in open, arid environments with high dust accumulation, whereas enamel folding was more associated with the amount of grass in the diet. We further show that long-term trends in these features proceeded in a ratchet-like mode, alternating between directional change at times of high selective pressure and stasis when the selective regime reversed. This provides an explanation for morphology adapted to more extreme conditions than current usage (Liem’s Paradox). Our study illustrates how, in fossil series with adequate stratigraphic control and proxy data, environmental and behavioural factors can be mapped on to time series of morphological change, illuminating the mode of acquisition of an adaptive complex. Tracking temporal patterns of proboscidean dental evolution in relation to diet and environment, the authors find that the effects of diet and environment are decoupled, and evolution of dentition alternated between directional change at times of high selective pressure and stasis when selective regimes reversed.},
  archive      = {J_NATEE},
  author       = {Saarinen, Juha and Lister, Adrian M.},
  doi          = {10.1038/s41559-023-02151-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1490-1502},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Fluctuating climate and dietary innovation drove ratcheted evolution of proboscidean dental traits},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mechanistic neutral models show that sampling biases drive
the apparent explosion of early tetrapod diversity. <em>NATEE</em>,
<em>7</em>(9), 1480–1489. (<a
href="https://doi.org/10.1038/s41559-023-02128-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Estimates of deep-time biodiversity typically rely on statistical methods to mitigate the impacts of sampling biases in the fossil record. However, these methods are limited by the spatial and temporal scale of the underlying data. Here we use a spatially explicit mechanistic model, based on neutral theory, to test hypotheses of early tetrapod diversity change during the late Carboniferous and early Permian, critical intervals for the diversification of vertebrate life on land. Our simulations suggest that apparent increases in early tetrapod diversity were not driven by local endemism following the ‘Carboniferous rainforest collapse’. Instead, changes in face-value diversity can be explained by variation in sampling intensity through time. Our results further demonstrate the importance of accounting for sampling biases in analyses of the fossil record and highlight the vast potential of mechanistic models, including neutral models, for testing hypotheses in palaeobiology. A spatially explicit mechanistic model, based on neutral theory, is used to show that changes in apparent diversity in the fossil record of the late Carboniferous and early Permian are explained by variation in sampling intensity through time.},
  archive      = {J_NATEE},
  author       = {Dunne, Emma M. and Thompson, Samuel E. D. and Butler, Richard J. and Rosindell, James and Close, Roger A.},
  doi          = {10.1038/s41559-023-02128-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1480-1489},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mechanistic neutral models show that sampling biases drive the apparent explosion of early tetrapod diversity},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Body size and life history shape the historical biogeography
of tetrapods. <em>NATEE</em>, <em>7</em>(9), 1467–1479. (<a
href="https://doi.org/10.1038/s41559-023-02150-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dispersal across biogeographic barriers is a key process determining global patterns of biodiversity as it allows lineages to colonize and diversify in new realms. Here we demonstrate that past biogeographic dispersal events often depended on species’ traits, by analysing 7,009 tetrapod species in 56 clades. Biogeographic models incorporating body size or life history accrued more statistical support than trait-independent models in 91% of clades. In these clades, dispersal rates increased by 28–32% for lineages with traits favouring successful biogeographic dispersal. Differences between clades in the effect magnitude of life history on dispersal rates are linked to the strength and type of biogeographic barriers and intra-clade trait variability. In many cases, large body sizes and fast life histories facilitate dispersal success. However, species with small bodies and/or slow life histories, or those with average traits, have an advantage in a minority of clades. Body size–dispersal relationships were related to a clade’s average body size and life history strategy. These results provide important new insight into how traits have shaped the historical biogeography of tetrapod lineages and may impact present-day and future biogeographic dispersal. Using trait-dependent biogeographic models to analyse data for &amp;gt;7,000 tetrapod species, the authors show that large body size and a fast life history strategy facilitate dispersal success, although this was also true for small body size and a slow life history strategy in a minority of clades.},
  archive      = {J_NATEE},
  author       = {Weil, Sarah-Sophie and Gallien, Laure and Nicolaï, Michaël P. J. and Lavergne, Sébastien and Börger, Luca and Allen, William L.},
  doi          = {10.1038/s41559-023-02150-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1467-1479},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Body size and life history shape the historical biogeography of tetrapods},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Immune evasion and ACE2 binding affinity contribute to
SARS-CoV-2 evolution. <em>NATEE</em>, <em>7</em>(9), 1457–1466. (<a
href="https://doi.org/10.1038/s41559-023-02123-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mutations in the SARS-CoV-2 genome could confer resistance to pre-existing antibodies and/or increased transmissibility. The recently emerged Omicron subvariants exhibit a strong tendency for immune evasion, suggesting adaptive evolution. However, because previous studies have been limited to specific lineages or subsets of mutations, the overall evolutionary trajectory of SARS-CoV-2 and the underlying driving forces are still not fully understood. Here we analysed all open-access SARS-CoV-2 genomes (up to November 2022) and correlated the mutation incidence and fitness changes with the impacts of mutations on immune evasion and ACE2 binding affinity. Our results show that the Omicron lineage had an accelerated mutation rate in the RBD region, while the mutation incidence in other genomic regions did not change dramatically over time. Mutations in the RBD region exhibited a lineage-specific pattern and tended to become more aggregated over time, and the mutation incidence was positively correlated with the strength of antibody pressure. Additionally, mutation incidence was positively correlated with changes in ACE2 binding affinity, but with a lower correlation coefficient than with immune evasion. In contrast, the effect of mutations on fitness was more closely correlated with changes in ACE2 binding affinity than with immune evasion. Our findings suggest that immune evasion and ACE2 binding affinity play significant and diverse roles in the evolution of SARS-CoV-2. Correlations of mutation incidence and fitness changes with the impacts of mutations on immune evasion and ACE2 binding affinity across available SARS-CoV-2 genomes show that immune evasion and ACE2 binding affinity contributed to SARS-CoV-2 evolution.},
  archive      = {J_NATEE},
  author       = {Ma, Wentai and Fu, Haoyi and Jian, Fanchong and Cao, Yunlong and Li, Mingkun},
  doi          = {10.1038/s41559-023-02123-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1457-1466},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Immune evasion and ACE2 binding affinity contribute to SARS-CoV-2 evolution},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Convergent resistance to GABA receptor neurotoxins through
plant–insect coevolution. <em>NATEE</em>, <em>7</em>(9), 1444–1456. (<a
href="https://doi.org/10.1038/s41559-023-02127-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The molecular mechanisms of coevolution between plants and insects remain elusive. GABA receptors are targets of many neurotoxic terpenoids, which represent the most diverse array of natural products known. Over deep evolutionary time, as plant terpene synthases diversified in plants, so did plant terpenoid defence repertoires. Here we show that herbivorous insects and their predators evolved convergent amino acid changing substitutions in duplicated copies of the Resistance to dieldrin (Rdl) gene that encodes the GABA receptor, and that the evolution of duplicated Rdl and terpenoid-resistant GABA receptors is associated with the diversification of moths and butterflies. These same substitutions also evolved in pests exposed to synthetic insecticides that target the GABA receptor. We used in vivo genome editing in Drosophila melanogaster to evaluate the fitness effects of each putative resistance mutation and found that pleiotropy both facilitates and constrains the evolution of GABA receptor resistance. The same genetic changes that confer resistance to terpenoids across 300 Myr of insect evolution have re-evolved in response to synthetic analogues over one human lifespan. Antagonistic coevolution of plants and insects is hypothesized to promote biological diversity. This study shows convergent evolution of a gene that encodes the GABA receptor, a target of plant toxins, in herbivorous insects and their predators. It further demonstrates that the evolution of resistant GABA receptors is associated with insect diversification.},
  archive      = {J_NATEE},
  author       = {Guo, Lei and Qiao, Xiaomu and Haji, Diler and Zhou, Tianhao and Liu, Zhihan and Whiteman, Noah K. and Huang, Jia},
  doi          = {10.1038/s41559-023-02127-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1444-1456},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Convergent resistance to GABA receptor neurotoxins through plant–insect coevolution},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dynamics of species-rich predator–prey networks and seasonal
alternations of core species. <em>NATEE</em>, <em>7</em>(9), 1432–1443.
(<a href="https://doi.org/10.1038/s41559-023-02130-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In nature, entangled webs of predator–prey interactions constitute the backbones of ecosystems. Uncovering the network architecture of such trophic interactions has been recognized as the essential step for exploring species with great impacts on ecosystem-level phenomena and functions. However, it has remained a major challenge to reveal how species-rich networks of predator–prey interactions are continually reshaped through time in the wild. Here, we show that dynamics of species-rich predator–prey interactions can be characterized by remarkable network structural changes and alternations of species with greatest impacts on community processes. On the basis of high-throughput detection of prey DNA from 1,556 spider individuals collected in a grassland ecosystem, we reconstructed dynamics of interaction networks involving, in total, 50 spider species and 974 prey species and strains through 8 months. The networks were compartmentalized into modules (groups) of closely interacting predators and prey in each month. Those modules differed in detritus/grazing food chain properties, forming complex fission–fusion dynamics of belowground and aboveground energy channels across the seasons. The substantial shifts of network structure entailed alternations of spider species located at the core positions within the entangled webs of interactions. These results indicate that knowledge of dynamically shifting food webs is crucial for understanding temporally varying roles of ‘core species’ in ecosystem processes. Using high-throughput detection of prey DNA from spiders in a grassland ecosystem, the authors reconstruct interaction networks between 50 spider species and 974 prey species and strains, documenting shifting seasonal dynamics.},
  archive      = {J_NATEE},
  author       = {Suzuki, Sayaka S. and Baba, Yuki G. and Toju, Hirokazu},
  doi          = {10.1038/s41559-023-02130-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1432-1443},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Dynamics of species-rich predator–prey networks and seasonal alternations of core species},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Experimentally determined traits shape bacterial community
composition one and five years following wildfire. <em>NATEE</em>,
<em>7</em>(9), 1419–1431. (<a
href="https://doi.org/10.1038/s41559-023-02135-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Wildfires represent major ecological disturbances, burning 2–3% of Earth’s terrestrial area each year with sometimes drastic effects above- and belowground. Soil bacteria offer an ideal, yet understudied system within which to explore fundamental principles of fire ecology. To understand how wildfires restructure soil bacterial communities and alter their functioning, we sought to translate aboveground fire ecology to belowground systems by determining which microbial traits are important post-fire and whether changes in bacterial communities affect carbon cycling. We employed an uncommon approach to assigning bacterial traits, by first running three laboratory experiments to directly determine which microbes survive fires, grow quickly post-fire and/or thrive in the post-fire environment, while tracking CO2 emissions. We then quantified the abundance of taxa assigned to each trait in a large field dataset of soils one and five years after wildfires in the boreal forest of northern Canada. We found that fast-growing bacteria rapidly dominate post-fire soils but return to pre-burn relative abundances by five years post-fire. Although both fire survival and affinity for the post-fire environment were statistically significant predictors of post-fire community composition, neither are particularly influential. Our results from the incubation trials indicate that soil carbon fluxes post-wildfire are not likely limited by microbial communities, suggesting strong functional resilience. From these findings, we offer a traits-based framework of bacterial responses to wildfire. To link aboveground fire ecology with belowground microbial trait ecology, the authors conduct laboratory experiments followed by analysis of field data from northern Canada. Fast-growing species initially dominate but decline between 1 and 5 yr post-fire, while neither fire survival nor adaptation to a post-fire environment are strong determinants of success.},
  archive      = {J_NATEE},
  author       = {Johnson, Dana B. and Woolet, Jamie and Yedinak, Kara M. and Whitman, Thea},
  doi          = {10.1038/s41559-023-02135-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1419-1431},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Experimentally determined traits shape bacterial community composition one and five years following wildfire},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multidimensional specialization and generalization are
pervasive in soil prokaryotes. <em>NATEE</em>, <em>7</em>(9), 1408–1418.
(<a href="https://doi.org/10.1038/s41559-023-02149-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Habitat specialization underpins biological processes from species distributions to speciation. However, organisms are often described as specialists or generalists based on a single niche axis, despite facing complex, multidimensional environments. Here, we analysed 236 environmental soil microbiomes across the United States and demonstrate that 90% of &amp;gt;1,200 prokaryotes followed one of two trajectories: specialization on all niche axes (multidimensional specialization) or generalization on all axes (multidimensional generalization). We then documented that this pervasive multidimensional specialization/generalization had many ecological and evolutionary consequences. First, multidimensional specialization and generalization are highly conserved with very few transitions between these two trajectories. Second, multidimensional generalists dominated communities because they were 73 times more abundant than specialists. Lastly, multidimensional specialists played important roles in community structure with ~220% more connections in microbiome networks. These results indicate that multidimensional generalization and specialization are evolutionarily stable with multidimensional generalists supporting larger populations and multidimensional specialists playing important roles within communities, probably stemming from their overrepresentation among pollutant detoxifiers and nutrient cyclers. Taken together, we demonstrate that the vast majority of soil prokaryotes are restricted to one of two multidimensional niche trajectories, multidimensional specialization or multidimensional generalization, which then has far-reaching consequences for evolutionary transitions, microbial dominance and community roles. Analyses of soil microbiomes across the United States reveal pervasive multidimensional specialization and generalization across prokaryotic communities impacting evolutionary trajectories, microbial dominance and community roles.},
  archive      = {J_NATEE},
  author       = {Hernandez, Damian J. and Kiesewetter, Kasey N. and Almeida, Brianna K. and Revillini, Daniel and Afkhami, Michelle E.},
  doi          = {10.1038/s41559-023-02149-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1408-1418},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Multidimensional specialization and generalization are pervasive in soil prokaryotes},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Late acquisition of the rTCA carbon fixation pathway by
chlorobi. <em>NATEE</em>, <em>7</em>(9), 1398–1407. (<a
href="https://doi.org/10.1038/s41559-023-02147-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The reverse tricarboxylic acid (rTCA) cycle is touted as a primordial mode of carbon fixation due to its autocatalytic propensity and oxygen intolerance. Despite this inferred antiquity, however, the earliest rock record affords scant supporting evidence. In fact, based on the chimeric inheritance of rTCA cycle steps within the Chlorobiaceae, even the use of the chemical fossil record of this group is now subject to question. While the 1.64-billion-year-old Barney Creek Formation contains chemical fossils of the earliest known putative Chlorobiaceae-derived carotenoids, interferences from the accompanying hydrocarbon matrix have hitherto precluded the carbon isotope measurements necessary to establish the physiology of the organisms that produced them. Overcoming this obstacle, here we report a suite of compound-specific carbon isotope measurements identifying a cyanobacterially dominated ecosystem featuring heterotrophic bacteria. We demonstrate chlorobactane is 13C-depleted when compared to contemporary equivalents, showing only slight 13C-enrichment over co-existing cyanobacterial carotenoids. The absence of this diagnostic isotopic fingerprint, in turn, confirms phylogenomic hypotheses that call for the late assembly of the rTCA cycle and, thus, the delayed acquisition of autotrophy within the Chlorobiaceae. We suggest that progressive oxygenation of the Earth System caused an increase in the marine sulfate inventory thereby providing the selective pressure to fuel the Neoproterozoic shift towards energy-efficient photoautotrophy within the Chlorobiaceae. Compound-specific carbon isotope measurements of the 1.64-billion-year-old Barney Creek Formation show an ecosystem dominated by cyanobacteria and heterotrophic bacteria. Isotope data for carotenoids suggest that the assembly of the reverse tricarboxylic acid cycle in Chlorobi occurred later.},
  archive      = {J_NATEE},
  author       = {Zhang, Xiaowen and Paoletti, Madeline M. and Izon, Gareth and Fournier, Gregory P. and Summons, Roger E.},
  doi          = {10.1038/s41559-023-02147-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1398-1407},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Late acquisition of the rTCA carbon fixation pathway by chlorobi},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Carbonate compensation depth drives abyssal biogeography in
the northeast pacific. <em>NATEE</em>, <em>7</em>(9), 1388–1397. (<a
href="https://doi.org/10.1038/s41559-023-02122-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Abyssal seafloor communities cover more than 60% of Earth’s surface. Despite their great size, abyssal plains extend across modest environmental gradients compared to other marine ecosystems. However, little is known about the patterns and processes regulating biodiversity or potentially delimiting biogeographical boundaries at regional scales in the abyss. Improved macroecological understanding of remote abyssal environments is urgent as threats of widespread anthropogenic disturbance grow in the deep ocean. Here, we use a new, basin-scale dataset to show the existence of clear regional zonation in abyssal communities across the 5,000 km span of the Clarion–Clipperton Zone (northeast Pacific), an area targeted for deep-sea mining. We found two pronounced biogeographic provinces, deep and shallow-abyssal, separated by a transition zone between 4,300 and 4,800 m depth. Surprisingly, species richness was maintained across this boundary by phylum-level taxonomic replacements. These regional transitions are probably related to calcium carbonate saturation boundaries as taxa dependent on calcium carbonate structures, such as shelled molluscs, appear restricted to the shallower province. Our results suggest geochemical and climatic forcing on distributions of abyssal populations over large spatial scales and provide a potential paradigm for deep-sea macroecology, opening a new basis for regional-scale biodiversity research and conservation strategies in Earth’s largest biome. Analysing seabed imagery data spanning 5,000 km and &amp;gt;50,000 megafaunal specimens across the Clarion–Clipperton Zone abyssal plain in the northeast Pacific, the authors identify two distinct biogeographical provinces in which taxa that are dependent on calcium carbonate structures are restricted to depths above 4,300–4,880 m.},
  archive      = {J_NATEE},
  author       = {Simon-Lledó, Erik and Amon, Diva J. and Bribiesca‐Contreras, Guadalupe and Cuvelier, Daphne and Durden, Jennifer M. and Ramalho, Sofia P. and Uhlenkott, Katja and Arbizu, Pedro Martinez and Benoist, Noëlie and Copley, Jonathan and Dahlgren, Thomas G. and Glover, Adrian G. and Fleming, Bethany and Horton, Tammy and Ju, Se-Jong and Mejía-Saenz, Alejandra and McQuaid, Kirsty and Pape, Ellen and Park, Chailinn and Smith, Craig R. and Jones, Daniel O. B.},
  doi          = {10.1038/s41559-023-02122-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1388-1397},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Carbonate compensation depth drives abyssal biogeography in the northeast pacific},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evidence for widespread thermal optimality of ecosystem
respiration. <em>NATEE</em>, <em>7</em>(9), 1379–1387. (<a
href="https://doi.org/10.1038/s41559-023-02121-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ecosystem respiration (ER) is among the largest carbon fluxes between the biosphere and the atmosphere. Understanding the temperature response of ER is crucial for predicting the climate change–carbon cycle feedback. However, whether there is an apparent optimum temperature of ER ( $${{T}}_{{\rm{opt}}}^{\,{\rm{ER}}}$$ ) and how it changes with temperature remain poorly understood. Here we analyse the temperature response curves of ER at 212 sites from global FLUXNET. We find that ER at 183 sites shows parabolic temperature response curves and $${{T}}_{{\rm{opt}}}^{\,{\rm{ER}}}$$ at which ER reaches the maximum exists widely across biomes around the globe. Among the 15 biotic and abiotic variables examined, $${{T}}_{{\rm{opt}}}^{\,{\rm{ER}}}$$ is mostly related to the optimum temperature of gross primary production (GPP, $${{T}}_{{\rm{opt}}}^{\,{\rm{GPP}}}$$ ) and annual maximum daily temperature (Tmax). In addition, $${{T}}_{{\rm{opt}}}^{\,{\rm{ER}}}$$ linearly increases with Tmax across sites and over vegetation types, suggesting its thermal adaptation. The adaptation magnitude of $${{T}}_{{\rm{opt}}}^{\,{\rm{ER}}}$$ , which is measured by the change in $${{T}}_{{\rm{opt}}}^{\,{\rm{ER}}}$$ per unit change in Tmax, is positively correlated with the adaptation magnitude of $${{T}}_{{\rm{opt}}}^{\,{\rm{GPP}}}$$ . This study provides evidence of the widespread existence of $${{T}}_{{\rm{opt}}}^{\,{\rm{ER}}}$$ and its thermal adaptation with Tmax across different biomes around the globe. Our findings suggest that carbon cycle models that consider the existence of $${{T}}_{{\rm{opt}}}^{\,{\rm{ER}}}$$ and its adaptation have the potential to more realistically predict terrestrial carbon sequestration in a world with changing climate. Temperature response curves of ecosystem respiration from 212 globally distributed sites show widespread evidence for an optimum temperature that is the result of thermal adaptation.},
  archive      = {J_NATEE},
  author       = {Chen, Weinan and Wang, Song and Wang, Jinsong and Xia, Jianyang and Luo, Yiqi and Yu, Guirui and Niu, Shuli},
  doi          = {10.1038/s41559-023-02121-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1379-1387},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evidence for widespread thermal optimality of ecosystem respiration},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Limits to the accurate and generalizable use of soundscapes
to monitor biodiversity. <em>NATEE</em>, <em>7</em>(9), 1373–1378. (<a
href="https://doi.org/10.1038/s41559-023-02148-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Although eco-acoustic monitoring has the potential to deliver biodiversity insight on vast scales, existing analytical approaches behave unpredictably across studies. We collated 8,023 audio recordings with paired manual avifaunal point counts to investigate whether soundscapes could be used to monitor biodiversity across diverse ecosystems. We found that neither univariate indices nor machine learning models were predictive of species richness across datasets but soundscape change was consistently indicative of community change. Our findings indicate that there are no common features of biodiverse soundscapes and that soundscape monitoring should be used cautiously and in conjunction with more reliable in-person ecological surveys. Analysis of global eco-acoustic data shows that soundscape-based predictions of avian biodiversity do not generalize across datasets, but changes in soundscapes do consistently indicate changes in avian communities.},
  archive      = {J_NATEE},
  author       = {Sethi, Sarab S. and Bick, Avery and Ewers, Robert M. and Klinck, Holger and Ramesh, Vijay and Tuanmu, Mao-Ning and Coomes, David A.},
  doi          = {10.1038/s41559-023-02148-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1373-1378},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Limits to the accurate and generalizable use of soundscapes to monitor biodiversity},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A vision for incorporating human mobility in the study of
human–wildlife interactions. <em>NATEE</em>, <em>7</em>(9), 1362–1372.
(<a href="https://doi.org/10.1038/s41559-023-02125-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As human activities increasingly shape land- and seascapes, understanding human–wildlife interactions is imperative for preserving biodiversity. Habitats are impacted not only by static modifications, such as roads, buildings and other infrastructure, but also by the dynamic movement of people and their vehicles occurring over shorter time scales. Although there is increasing realization that both components of human activity substantially affect wildlife, capturing more dynamic processes in ecological studies has proved challenging. Here we propose a conceptual framework for developing a ‘dynamic human footprint’ that explicitly incorporates human mobility, providing a key link between anthropogenic stressors and ecological impacts across spatiotemporal scales. Specifically, the dynamic human footprint integrates a range of metrics to fully acknowledge the time-varying nature of human activities and to enable scale-appropriate assessments of their impacts on wildlife behaviour, demography and distributions. We review existing terrestrial and marine human-mobility data products and provide a roadmap for how these could be integrated and extended to enable more comprehensive analyses of human impacts on biodiversity in the Anthropocene. Wildlife are affected by human movement and static human infrastructure. In this Perspective, the authors propose a ‘dynamic human footprint’ that incorporates metrics accounting for time-varying human activities.},
  archive      = {J_NATEE},
  author       = {Ellis-Soto, Diego and Oliver, Ruth Y. and Brum-Bastos, Vanessa and Demšar, Urška and Jesmer, Brett and Long, Jed A. and Cagnacci, Francesca and Ossi, Federico and Queiroz, Nuno and Hindell, Mark and Kays, Roland and Loretto, Matthias-Claudio and Mueller, Thomas and Patchett, Robert and Sims, David W. and Tucker, Marlee A. and Ropert-Coudert, Yan and Rutz, Christian and Jetz, Walter},
  doi          = {10.1038/s41559-023-02125-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1362-1372},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A vision for incorporating human mobility in the study of human–wildlife interactions},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ecological neutral theory reveals the effect of biases in
the fossil record. <em>NATEE</em>, <em>7</em>(9), 1360–1361. (<a
href="https://doi.org/10.1038/s41559-023-02138-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using a mechanistic model based on neutral theory, we examined the effects of the ‘Carboniferous rainforest collapse’ on early tetrapod diversity. Our findings highlight the power of mechanistic models for decoding the fossil record and underscore the criticality of adjusting for sampling biases.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02138-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1360-1361},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ecological neutral theory reveals the effect of biases in the fossil record},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Biogeographic boundaries and high diversity in abyssal
seafloor communities. <em>NATEE</em>, <em>7</em>(9), 1358–1359. (<a
href="https://doi.org/10.1038/s41559-023-02139-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of an ocean basin-scale dataset revealed the existence of clear biogeographic provinces (deep and shallow abyssal zones) delimited by the carbonate compensation depth in Pacific Ocean seabed communities. Species diversity is maintained or increases with depth owing to phylum-level taxonomic replacements.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02139-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1358-1359},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Biogeographic boundaries and high diversity in abyssal seafloor communities},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Exaggerated effects in ecology. <em>NATEE</em>,
<em>7</em>(9), 1356–1357. (<a
href="https://doi.org/10.1038/s41559-023-02156-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study of over 18,000 effect sizes from more than 350 published studies in ecology finds clear evidence of selective reporting and exaggeration of effect sizes.},
  archive      = {J_NATEE},
  author       = {Parker, Timothy H. and Yang, Yefeng},
  doi          = {10.1038/s41559-023-02156-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1356-1357},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Exaggerated effects in ecology},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Mixed picture for global forests. <em>NATEE</em>,
<em>7</em>(9), 1355. (<a
href="https://doi.org/10.1038/s41559-023-02154-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Harold, Simon},
  doi          = {10.1038/s41559-023-02154-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1355},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mixed picture for global forests},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023g). Worn down by the weather. <em>NATEE</em>, <em>7</em>(9),
1354. (<a href="https://doi.org/10.1038/s41559-023-02153-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-023-02153-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1354},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Worn down by the weather},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Common patterns of macroevolution. <em>NATEE</em>,
<em>7</em>(9), 1353. (<a
href="https://doi.org/10.1038/s41559-023-02146-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-023-02146-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1353},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Common patterns of macroevolution},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Awakening innovations in biology and culture.
<em>NATEE</em>, <em>7</em>(9), 1351–1352. (<a
href="https://doi.org/10.1038/s41559-023-02131-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Ogbunugafor, C. Brandon},
  doi          = {10.1038/s41559-023-02131-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1351-1352},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Awakening innovations in biology and culture},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Michael w. Bruford (1963–2023). <em>NATEE</em>,
<em>7</em>(9), 1349–1350. (<a
href="https://doi.org/10.1038/s41559-023-02110-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Conservation and livestock geneticist who made a significant and lasting contribution to biodiversity policy and the management of endangered species.},
  archive      = {J_NATEE},
  author       = {Hoban, Sean and Segelbacher, Gernot and Vernesi, Cristiano and Russo, Isa-Rita M.},
  doi          = {10.1038/s41559-023-02110-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1349-1350},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Michael w. bruford (1963–2023)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Masatoshi nei (1931–2023). <em>NATEE</em>, <em>7</em>(9),
1347–1348. (<a
href="https://doi.org/10.1038/s41559-023-02119-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evolutionary geneticist whose powerful statistical methods facilitated the molecular revolution of evolutionary biology.},
  archive      = {J_NATEE},
  author       = {Zhang, Jianzhi and Kumar, Sudhir},
  doi          = {10.1038/s41559-023-02119-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1347-1348},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Masatoshi nei (1931–2023)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The taskforce on nature-related financial disclosures must
engage widely and justify its market-led approach. <em>NATEE</em>,
<em>7</em>(9), 1343–1346. (<a
href="https://doi.org/10.1038/s41559-023-02113-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Taskforce on Nature-related Financial Disclosures is a key initiative that seeks to convince companies and financial institutions to manage nature-related risks. Its inherent structures — market-led, voluntary and corporate-governed — present challenges that should be addressed during the final development and initial implementation of the framework.},
  archive      = {J_NATEE},
  author       = {Linsley, Philip and Abdelbadie, Roba and Abdelbadie, Rasha},
  doi          = {10.1038/s41559-023-02113-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1343-1346},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The taskforce on nature-related financial disclosures must engage widely and justify its market-led approach},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Heavy reliance on private finance alone will not deliver
conservation goals. <em>NATEE</em>, <em>7</em>(9), 1339–1342. (<a
href="https://doi.org/10.1038/s41559-023-02098-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Kunming-Montreal Global Biodiversity Framework envisages an increasing reliance on large-scale private finance to fund biodiversity targets. We warn that this may pose contradictions in delivering conservation outcomes and propose a critical ongoing role for direct public funding of conservation and public oversight of private nature-related financial mechanisms.},
  archive      = {J_NATEE},
  author       = {Kedward, Katie and zu Ermgassen, Sophus and Ryan-Collins, Josh and Wunder, Sven},
  doi          = {10.1038/s41559-023-02098-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1339-1342},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Heavy reliance on private finance alone will not deliver conservation goals},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Loss and damage finance should apply to biodiversity loss.
<em>NATEE</em>, <em>7</em>(9), 1336–1338. (<a
href="https://doi.org/10.1038/s41559-023-02088-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global biodiversity loss has been disproportionately driven by consumption of people in rich nations. The concept of ‘loss and damage’ — familiar from international agreements on climate change — should be considered for the effects of biodiversity loss in countries of the Global South.},
  archive      = {J_NATEE},
  author       = {Roe, Dilys and Holland, Ebony and Nisi, Nora and Mitchell, Tom and Tasnim, Tasfia},
  doi          = {10.1038/s41559-023-02088-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1336-1338},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Loss and damage finance should apply to biodiversity loss},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ethical considerations for human sequences in environmental
DNA. <em>NATEE</em>, <em>7</em>(9), 1334–1335. (<a
href="https://doi.org/10.1038/s41559-023-02124-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Doi, Hideyuki and Kelly, Ryan P.},
  doi          = {10.1038/s41559-023-02124-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1334-1335},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ethical considerations for human sequences in environmental DNA},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Biodiversity on the balance sheet. <em>NATEE</em>,
<em>7</em>(9), 1333. (<a
href="https://doi.org/10.1038/s41559-023-02196-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The September 2023 release of the Taskforce for Nature-Related Financial Disclosures is just one event in a groundswell of discussion around who must pay to protect and restore nature.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02196-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {9},
  number       = {9},
  pages        = {1333},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Biodiversity on the balance sheet},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Reply to: Capturing stochasticity
properly is key to understanding the nuances of the living planet index.
<em>NATEE</em>, <em>7</em>(8), 1331. (<a
href="https://doi.org/10.1038/s41559-023-02157-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Buschke, Falko T. and Hagan, James G. and Santini, Luca and Coetzee, Bernard W. T.},
  doi          = {10.1038/s41559-023-02157-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1331},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: reply to: capturing stochasticity properly is key to understanding the nuances of the living planet index},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genomic history of coastal societies from eastern south
america. <em>NATEE</em>, <em>7</em>(8), 1315–1330. (<a
href="https://doi.org/10.1038/s41559-023-02114-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sambaqui (shellmound) societies are among the most intriguing archaeological phenomena in pre-colonial South America, extending from approximately 8,000 to 1,000 years before present (yr bp) across 3,000 km on the Atlantic coast. However, little is known about their connection to early Holocene hunter-gatherers, how this may have contributed to different historical pathways and the processes through which late Holocene ceramists came to rule the coast shortly before European contact. To contribute to our understanding of the population history of indigenous societies on the eastern coast of South America, we produced genome-wide data from 34 ancient individuals as early as 10,000 yr bp from four different regions in Brazil. Early Holocene hunter-gatherers were found to lack shared genetic drift among themselves and with later populations from eastern South America, suggesting that they derived from a common radiation and did not contribute substantially to later coastal groups. Our analyses show genetic heterogeneity among contemporaneous Sambaqui groups from the southeastern and southern Brazilian coast, contrary to the similarity expressed in the archaeological record. The complex history of intercultural contact between inland horticulturists and coastal populations becomes genetically evident during the final horizon of Sambaqui societies, from around 2,200 yr bp, corroborating evidence of cultural change. The authors report genome-wide data from 34 ancient individuals spanning the Holocene, from what is now Brazil, illuminating connections and disjoints between Sambaqui shellmound societies and preceding and later peoples.},
  archive      = {J_NATEE},
  author       = {Ferraz, Tiago and Suarez Villagran, Ximena and Nägele, Kathrin and Radzevičiūtė, Rita and Barbosa Lemes, Renan and Salazar-García, Domingo C. and Wesolowski, Verônica and Lopes Alves, Marcony and Bastos, Murilo and Rapp Py-Daniel, Anne and Pinto Lima, Helena and Mendes Cardoso, Jéssica and Estevam, Renata and Liryo, Andersen and Guimarães, Geovan M. and Figuti, Levy and Eggers, Sabine and Plens, Cláudia R. and Azevedo Erler, Dionne Miranda and Valadares Costa, Henrique Antônio and da Silva Erler, Igor and Koole, Edward and Henriques, Gilmar and Solari, Ana and Martin, Gabriela and Serafim Monteiro da Silva, Sérgio Francisco and Kipnis, Renato and Müller, Letícia Morgana and Ferreira, Mariane and Carvalho Resende, Janine and Chim, Eliane and da Silva, Carlos Augusto and Borella, Ana Claudia and Tomé, Tiago and Müller Plumm Gomes, Lisiane and Barros Fonseca, Diego and Santos da Rosa, Cassia and de Moura Saldanha, João Darcy and Costa Leite, Lúcio and Cunha, Claudia M. S. and Viana, Sibeli Aparecida and Ozorio Almeida, Fernando and Klokler, Daniela and Fernandes, Henry Luydy Abraham and Talamo, Sahra and DeBlasis, Paulo and Mendonça de Souza, Sheila and de Paula Moraes, Claide and Elias Oliveira, Rodrigo and Hünemeier, Tábita and Strauss, André and Posth, Cosimo},
  doi          = {10.1038/s41559-023-02114-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1315-1330},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Genomic history of coastal societies from eastern south america},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evidence for hunter-gatherer impacts on raven diet and
ecology in the gravettian of southern moravia. <em>NATEE</em>,
<em>7</em>(8), 1302–1314. (<a
href="https://doi.org/10.1038/s41559-023-02107-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The earlier Gravettian of Southern Moravia—the Pavlovian—is notable for the many raven bones (Corvus corax) documented in its faunal assemblages. On the basis of the rich zooarchaeological and settlement data from the Pavlovian, previous work suggested that common ravens were attracted by human domestic activities and subsequently captured by Pavlovian people, presumably for feathers and perhaps food. Here, we report independent δ15N, δ13C and δ34S stable isotope data obtained from 12 adult ravens from the Pavlovian key sites of Předmostí I, Pavlov I and Dolní Věstonice I to test this idea. We show that Pavlovian ravens regularly fed on larger herbivores and especially mammoths, aligning in feeding preferences with contemporaneous Gravettian foragers. We argue that opportunistic-generalist ravens were encouraged by human settlement and carcass provisioning. Our data may thus provide surprisingly early evidence for incipient synanthropism among Palaeolithic ravens. We suggest that anthropogenic manipulation of carrion supply dynamics furnished unique contexts for the emergence of human-oriented animal behaviours, in turn promoting novel human foraging opportunities—dynamics which are therefore important for understanding early hunter-gatherer ecosystem impacts. The authors report isotopic data from ravens from Pavlovian (29,000–25,000 yr bp) archaeological sites that indicate ravens were consuming the same range of foods as commensal humans at the sites, which the authors refer to as ‘incipient synanthropism’.},
  archive      = {J_NATEE},
  author       = {Baumann, Chris and Hussain, Shumon T. and Roblíčková, Martina and Riede, Felix and Mannino, Marcello A. and Bocherens, Hervé},
  doi          = {10.1038/s41559-023-02107-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1302-1314},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evidence for hunter-gatherer impacts on raven diet and ecology in the gravettian of southern moravia},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Same-sex sociosexual behaviour is widespread and heritable
in male rhesus macaques. <em>NATEE</em>, <em>7</em>(8), 1287–1301. (<a
href="https://doi.org/10.1038/s41559-023-02111-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Numerous reports have documented the occurrence of same-sex sociosexual behaviour (SSB) across animal species. However, the distribution of the behaviour within a species needs to be studied to test hypotheses describing its evolution and maintenance, in particular whether the behaviour is heritable and can therefore evolve by natural selection. Here we collected detailed observations across 3 yr of social and mounting behaviour of 236 male semi-wild rhesus macaques, which we combined with a pedigree dating back to 1938, to show that SSB is both repeatable (19.35%) and heritable (6.4%). Demographic factors (age and group structure) explained SSB variation only marginally. Furthermore, we found a positive genetic correlation between same-sex mounter and mountee activities, indicating a common basis to different forms of SSB. Finally, we found no evidence of fitness costs to SSB, but show instead that the behaviour mediated coalitionary partnerships that have been linked to improved reproductive success. Together, our results demonstrate that SSB is frequent in rhesus macaques, can evolve, and is not costly, indicating that SSB may be a common feature of primate reproductive ecology. The authors study social and mounting behaviours in male rhesus macaques across 3 yr, finding that same-sex sociosexual behaviour is both repeatable and heritable within this population.},
  archive      = {J_NATEE},
  author       = {Clive, Jackson and Flintham, Ewan and Savolainen, Vincent},
  doi          = {10.1038/s41559-023-02111-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1287-1301},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Same-sex sociosexual behaviour is widespread and heritable in male rhesus macaques},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). North african fox genomes show signatures of repeated
introgression and adaptation to life in deserts. <em>NATEE</em>,
<em>7</em>(8), 1267–1286. (<a
href="https://doi.org/10.1038/s41559-023-02094-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Elucidating the evolutionary process of animal adaptation to deserts is key to understanding adaptive responses to climate change. Here we generated 82 individual whole genomes of four fox species (genus Vulpes) inhabiting the Sahara Desert at different evolutionary times. We show that adaptation of new colonizing species to a hot arid environment has probably been facilitated by introgression and trans-species polymorphisms shared with older desert resident species, including a putatively adaptive 25 Mb genomic region. Scans for signatures of selection implicated genes affecting temperature perception, non-renal water loss and heat production in the recent adaptation of North African red foxes (Vulpes vulpes), after divergence from Eurasian populations approximately 78 thousand years ago. In the extreme desert specialists, Rueppell’s fox (V. rueppellii) and fennec (V. zerda), we identified repeated signatures of selection in genes affecting renal water homeostasis supported by gene expression and physiological differences. Our study provides insights into the mechanisms and genetic underpinnings of a natural experiment of repeated adaptation to extreme conditions. Analysis of whole genomes of four fox species shows that introgression facilitated adaptation to the hot arid environment of the Sahara Desert and suggests renal water homeostasis as a mechanism of adaptation in the extreme desert specialist species.},
  archive      = {J_NATEE},
  author       = {L. Rocha, Joana and Silva, Pedro and Santos, Nuno and Nakamura, Mónia and Afonso, Sandra and Qninba, Abdeljebbar and Boratynski, Zbyszek and Sudmant, Peter H. and Brito, José C. and Nielsen, Rasmus and Godinho, Raquel},
  doi          = {10.1038/s41559-023-02094-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1267-1286},
  shortjournal = {Nat. Ecol. Evol},
  title        = {North african fox genomes show signatures of repeated introgression and adaptation to life in deserts},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Low morphological disparity and decelerated rate of limb
size evolution close to the origin of birds. <em>NATEE</em>,
<em>7</em>(8), 1257–1266. (<a
href="https://doi.org/10.1038/s41559-023-02091-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The origin of birds from theropod dinosaurs involves many changes in musculoskeletal anatomy and epidermal structures, including multiple instances of convergence and homology-related traits that contribute to the refinement of flight capability. Changes in limb sizes and proportions are important for locomotion (for example, the forelimb for bird flight); thus, understanding these patterns is central to investigating the transition from terrestrial to volant theropods. Here we analyse the patterns of morphological disparity and the evolutionary rate of appendicular limbs along avialan stem lineages using phylogenetic comparative approaches. Contrary to the traditional wisdom that an evolutionary innovation like flight would promote and accelerate evolvability, our results show a shift to low disparity and decelerated rate near the origin of avialans that is largely ascribed to the evolutionarily constrained forelimb. These results suggest that natural selection shaped patterns of limb evolution close to the origin of avialans in a way that may reflect the winged forelimb ‘blueprint’ associated with powered flight. Analysing patterns of morphological disparity and the evolutionary rate of appendicular limbs across theropods, the authors show lower disparity and decelerated rates of forelimb size evolution, which may have been caused by restrictions on morphospace given the requirements for powered flight.},
  archive      = {J_NATEE},
  author       = {Wang, Min and Zhou, Zhonghe},
  doi          = {10.1038/s41559-023-02091-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1257-1266},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Low morphological disparity and decelerated rate of limb size evolution close to the origin of birds},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Y chromosome toxicity does not contribute to sex-specific
differences in longevity. <em>NATEE</em>, <em>7</em>(8), 1245–1256. (<a
href="https://doi.org/10.1038/s41559-023-02089-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While sex chromosomes carry sex-determining genes, they also often differ from autosomes in size and composition, consisting mainly of silenced heterochromatic repetitive DNA. Even though Y chromosomes show structural heteromorphism, the functional significance of such differences remains elusive. Correlative studies suggest that the amount of Y chromosome heterochromatin might be responsible for several male-specific traits, including sex-specific differences in longevity observed across a wide spectrum of species, including humans. However, experimental models to test this hypothesis have been lacking. Here we use the Drosophila melanogaster Y chromosome to investigate the relevance of sex chromosome heterochromatin in somatic organs in vivo. Using CRISPR–Cas9, we generated a library of Y chromosomes with variable levels of heterochromatin. We show that these different Y chromosomes can disrupt gene silencing in trans, on other chromosomes, by sequestering core components of the heterochromatin machinery. This effect is positively correlated to the level of Y heterochromatin. However, we also find that the ability of the Y chromosome to affect genome-wide heterochromatin does not generate physiological sex differences, including sexual dimorphism in longevity. Instead, we discovered that it is the phenotypic sex, female or male, that controls sex-specific differences in lifespan, rather than the presence of a Y chromosome. Altogether, our findings dismiss the ‘toxic Y’ hypothesis that postulates that the Y chromosome leads to reduced lifespan in XY individuals. Studying Drosophila melanogaster Y chromosomes with variable levels of heterochromatin generated using genome editing, the authors show that while different Y chromosomes could disrupt gene silencing on other chromosomes, they did not affect sexual dimorphism in longevity.},
  archive      = {J_NATEE},
  author       = {Delanoue, Rénald and Clot, Charlène and Leray, Chloé and Pihl, Thomas and Hudry, Bruno},
  doi          = {10.1038/s41559-023-02089-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1245-1256},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Y chromosome toxicity does not contribute to sex-specific differences in longevity},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell dissection of aggression in honeybee colonies.
<em>NATEE</em>, <em>7</em>(8), 1232–1244. (<a
href="https://doi.org/10.1038/s41559-023-02090-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding how genotypic variation results in phenotypic variation is especially difficult for collective behaviour because group phenotypes arise from complex interactions among group members. A genome-wide association study identified hundreds of genes associated with colony-level variation in honeybee aggression, many of which also showed strong signals of positive selection, but the influence of these ‘colony aggression genes’ on brain function was unknown. Here we use single-cell (sc) transcriptomics and gene regulatory network (GRN) analyses to test the hypothesis that genetic variation for colony aggression influences individual differences in brain gene expression and/or gene regulation. We compared soldiers, which respond to territorial intrusion with stinging attacks, and foragers, which do not. Colony environment showed stronger influences on soldier-forager differences in brain gene regulation compared with brain gene expression. GRN plasticity was strongly associated with colony aggression, with larger differences in GRN dynamics detected between soldiers and foragers from more aggressive relative to less aggressive colonies. The regulatory dynamics of subnetworks composed of genes associated with colony aggression genes were more strongly correlated with each other across different cell types and brain regions relative to other genes, especially in brain regions involved with olfaction and vision and multimodal sensory integration, which are known to mediate bee aggression. These results show how group genetics can shape a collective phenotype by modulating individual brain gene regulatory network architecture. The genetic basis of collective behaviour is complex. Single-cell transcriptomics of honeybee brains and gene regulatory network analysis showed differences in brain gene regulation and gene regulatory network plasticity between aggressive soldiers and non-aggressive foragers.},
  archive      = {J_NATEE},
  author       = {Traniello, Ian M. and Bukhari, Syed Abbas and Dibaeinia, Payam and Serrano, Guillermo and Avalos, Arian and Ahmed, Amy Cash and Sankey, Alison L. and Hernaez, Mikel and Sinha, Saurabh and Zhao, Sihai Dave and Catchen, Julian and Robinson, Gene E.},
  doi          = {10.1038/s41559-023-02090-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1232-1244},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Single-cell dissection of aggression in honeybee colonies},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genomes of fungi and relatives reveal delayed loss of
ancestral gene families and evolution of key fungal traits.
<em>NATEE</em>, <em>7</em>(8), 1221–1231. (<a
href="https://doi.org/10.1038/s41559-023-02095-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fungi are ecologically important heterotrophs that have radiated into most niches on Earth and fulfil key ecological services. Despite intense interest in their origins, major genomic trends of their evolutionary route from a unicellular opisthokont ancestor to derived multicellular fungi remain poorly known. Here we provide a highly resolved genome-wide catalogue of gene family changes across fungal evolution inferred from the genomes of 123 fungi and relatives. We show that a dominant trend in early fungal evolution has been the gradual shedding of protist genes and the punctuated emergence of innovation by two main gene duplication events. We find that the gene content of non-Dikarya fungi resembles that of unicellular opisthokonts in many respects, owing to the conservation of protist genes in their genomes. The most rapidly duplicating gene groups included extracellular proteins and transcription factors, as well as ones linked to the coordination of nutrient uptake with growth, highlighting the transition to a sessile osmotrophic feeding strategy and subsequent lifestyle evolution as important elements of early fungal history. These results suggest that the genomes of pre-fungal ancestors evolved into the typical filamentous fungal genome by a combination of gradual gene loss, turnover and several large duplication events rather than by abrupt changes. Consequently, the taxonomically defined Fungi represents a genomically non-uniform assemblage of species. Fungi exhibit remarkable morphological and ecological diversity. An analysis of the genomes of 123 fungi and relatives shows gradual loss of protist genes, major gene turnover and duplication leading to the evolution of modern traits of filamentous fungi.},
  archive      = {J_NATEE},
  author       = {Merényi, Zsolt and Krizsán, Krisztina and Sahu, Neha and Liu, Xiao-Bin and Bálint, Balázs and Stajich, Jason E. and Spatafora, Joseph W. and Nagy, László G.},
  doi          = {10.1038/s41559-023-02095-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1221-1231},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Genomes of fungi and relatives reveal delayed loss of ancestral gene families and evolution of key fungal traits},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Identifying species likely threatened by international
trade on the IUCN red list can inform CITES trade measures.
<em>NATEE</em>, <em>7</em>(8), 1211–1220. (<a
href="https://doi.org/10.1038/s41559-023-02115-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Overexploitation is a major threat to biodiversity and international trade in many species is regulated through the Convention on International Trade in Endangered Species of Wild Fauna and Flora (CITES). However, there is no established method to systematically determine which species are most at risk from international trade to inform potential trade measures under CITES. Here, we develop a mechanism using the International Union for Conservation of Nature’s Red List of Threatened Species to identify species that are likely to be threatened by international trade. Of 2,211 such species, CITES includes 59% (1,307 species), leaving two-fifths overlooked and in potential need of international trade regulation. Our results can inform deliberations on potential proposals to revise trade measures for species at CITES Conference of the Parties meetings. We also show that, for taxa with biological resource use documented as a threat, the number of species threatened by local and national use is four times greater than species likely threatened by international trade. To effectively address the overexploitation of species, interventions focused on achieving sustainability in international trade need to be complemented by commensurate measures to ensure that local and national use and trade of wildlife is well-regulated and sustainable. A systematic assessment finds that at least 5.8% of threatened and near threatened species on the IUCN Red List are likely to be threatened by international trade but that only 59% of these species are currently listed for protection from such trade by CITES.},
  archive      = {J_NATEE},
  author       = {Challender, Daniel W. S. and Cremona, Patricia J. and Malsch, Kelly and Robinson, Janine E. and Pavitt, Alyson T. and Scott, Janet and Hoffmann, Rachel and Joolia, Ackbar and Oldfield, Thomasina E. E. and Jenkins, Richard K. B. and Conde, Dalia A. and Hilton-Taylor, Craig and Hoffmann, Michael},
  doi          = {10.1038/s41559-023-02115-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1211-1220},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Identifying species likely threatened by international trade on the IUCN red list can inform CITES trade measures},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Biome-scale temperature sensitivity of ecosystem respiration
revealed by atmospheric CO2 observations. <em>NATEE</em>, <em>7</em>(8),
1199–1210. (<a
href="https://doi.org/10.1038/s41559-023-02093-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The temperature sensitivity of ecosystem respiration regulates how the terrestrial carbon sink responds to a warming climate but has been difficult to constrain observationally beyond the plot scale. Here we use observations of atmospheric CO2 concentrations from a network of towers together with carbon flux estimates from state-of-the-art terrestrial biosphere models to characterize the temperature sensitivity of ecosystem respiration, as represented by the Arrhenius activation energy, over various North American biomes. We infer activation energies of 0.43 eV for North America and 0.38 eV to 0.53 eV for major biomes therein, which are substantially below those reported for plot-scale studies (approximately 0.65 eV). This discrepancy suggests that sparse plot-scale observations do not capture the spatial-scale dependence and biome specificity of the temperature sensitivity. We further show that adjusting the apparent temperature sensitivity in model estimates markedly improves their ability to represent observed atmospheric CO2 variability. This study provides observationally constrained estimates of the temperature sensitivity of ecosystem respiration directly at the biome scale and reveals that temperature sensitivities at this scale are lower than those based on earlier plot-scale studies. These findings call for additional work to assess the resilience of large-scale carbon sinks to warming. Atmospheric CO2 concentration measured across a network of towers in North America shows that continent- and biome-scale measurements of the temperature sensitivity of ecosystem respiration are lower than have previously been estimated from plot-level studies.},
  archive      = {J_NATEE},
  author       = {Sun, Wu and Luo, Xiangzhong and Fang, Yuanyuan and Shiga, Yoichi P. and Zhang, Yao and Fisher, Joshua B. and Keenan, Trevor F. and Michalak, Anna M.},
  doi          = {10.1038/s41559-023-02093-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1199-1210},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Biome-scale temperature sensitivity of ecosystem respiration revealed by atmospheric CO2 observations},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Reply to: Capturing stochasticity properly is key to
understanding the nuances of the living planet index. <em>NATEE</em>,
<em>7</em>(8), 1196–1198. (<a
href="https://doi.org/10.1038/s41559-023-02087-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Buschke, Falko T. and Hagan, James G. and Santini, Luca and Coetzee, Bernard W. T.},
  doi          = {10.1038/s41559-023-02087-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1196-1198},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reply to: Capturing stochasticity properly is key to understanding the nuances of the living planet index},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Capturing stochasticity properly is key to understanding the
nuances of the living planet index. <em>NATEE</em>, <em>7</em>(8),
1194–1195. (<a
href="https://doi.org/10.1038/s41559-023-02086-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Talis, Emma J. and Lynch, Heather J.},
  doi          = {10.1038/s41559-023-02086-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1194-1195},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Capturing stochasticity properly is key to understanding the nuances of the living planet index},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Conceptual and empirical bridges between micro- and
macroevolution. <em>NATEE</em>, <em>7</em>(8), 1181–1193. (<a
href="https://doi.org/10.1038/s41559-023-02116-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Explaining broad molecular, phenotypic and species biodiversity patterns necessitates a unifying framework spanning multiple evolutionary scales. Here we argue that although substantial effort has been made to reconcile microevolution and macroevolution, much work remains to identify the links between biological processes at play. We highlight four major questions of evolutionary biology whose solutions require conceptual bridges between micro and macroevolution. We review potential avenues for future research to establish how mechanisms at one scale (drift, mutation, migration, selection) translate to processes at the other scale (speciation, extinction, biogeographic dispersal) and vice versa. We propose ways in which current comparative methods to infer molecular evolution, phenotypic evolution and species diversification could be improved to specifically address these questions. We conclude that researchers are in a better position than ever before to build a synthesis to understand how microevolutionary dynamics unfold over millions of years. This Perspective discusses four questions of evolutionary biology that bridge macro and microevolution perspectives and proposes future research avenues to link evolutionary mechanisms and processes.},
  archive      = {J_NATEE},
  author       = {Rolland, Jonathan and Henao-Diaz, L. Francisco and Doebeli, Michael and Germain, Rachel and Harmon, Luke J. and Knowles, L. Lacey and Liow, Lee Hsiang and Mank, Judith E. and Machac, Antonin and Otto, Sarah P. and Pennell, Matt and Salamin, Nicolas and Silvestro, Daniele and Sugawara, Mauro and Uyeda, Josef and Wagner, Catherine E. and Schluter, Dolph},
  doi          = {10.1038/s41559-023-02116-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1181-1193},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Conceptual and empirical bridges between micro- and macroevolution},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetic insights into brazil’s ancient shell mound builders.
<em>NATEE</em>, <em>7</em>(8), 1179–1180. (<a
href="https://doi.org/10.1038/s41559-023-02134-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An ancient genomics study of Holocene human individuals in Brazil provides hard-won data that illuminate the early history of population settlement in South America},
  archive      = {J_NATEE},
  author       = {Carlo Colonese, André and McGrath, Krista},
  doi          = {10.1038/s41559-023-02134-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1179-1180},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Genetic insights into brazil’s ancient shell mound builders},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Why ys are not necessarily toxic. <em>NATEE</em>,
<em>7</em>(8), 1177–1178. (<a
href="https://doi.org/10.1038/s41559-023-02079-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sex differences in physiology and longevity are widely observed. A study that manipulates heterochromatin content in Drosophila Y chromosomes shows no association between the length of the Y chromosome and longevity, thus challenging the hypothesis that Y chromosome-derived heterochromatin causes Y chromosome-bearing animals to live shorter lives.},
  archive      = {J_NATEE},
  author       = {Yamashita, Yukiko M.},
  doi          = {10.1038/s41559-023-02079-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1177-1178},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Why ys are not necessarily toxic},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Species risk assessment informs trade regulation.
<em>NATEE</em>, <em>7</em>(8), 1175–1176. (<a
href="https://doi.org/10.1038/s41559-023-02097-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {International trade poses a risk to many species, especially those threatened with extinction. A new assessment tool based on the IUCN Red List may help to improve transparency, oversight and regulation of the international trade in wildlife.},
  archive      = {J_NATEE},
  author       = {Scheffers, Brett R.},
  doi          = {10.1038/s41559-023-02097-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1175-1176},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Species risk assessment informs trade regulation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Convergent evolution in distant woodpeckers.
<em>NATEE</em>, <em>7</em>(8), 1174. (<a
href="https://doi.org/10.1038/s41559-023-02109-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bhaumik, Vaishali},
  doi          = {10.1038/s41559-023-02109-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1174},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Convergent evolution in distant woodpeckers},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Cenozoic megaherbivore emergence. <em>NATEE</em>,
<em>7</em>(8), 1173. (<a
href="https://doi.org/10.1038/s41559-023-02100-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Grinham, Luke R.},
  doi          = {10.1038/s41559-023-02100-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1173},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Cenozoic megaherbivore emergence},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Kathy MacKinnon (1948–2023). <em>NATEE</em>, <em>7</em>(8),
1171–1172. (<a
href="https://doi.org/10.1038/s41559-023-02092-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determined and generous champion of protected-areas science and policy},
  archive      = {J_NATEE},
  author       = {Rao, Madhu and Woodley, Stephen and Dudley, Nigel},
  doi          = {10.1038/s41559-023-02092-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1171-1172},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Kathy MacKinnon (1948–2023)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Some, but not all, eponyms should be disallowed.
<em>NATEE</em>, <em>7</em>(8), 1170. (<a
href="https://doi.org/10.1038/s41559-023-02106-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Thiele, Kevin},
  doi          = {10.1038/s41559-023-02106-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1170},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Some, but not all, eponyms should be disallowed},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Inclusive and productive ways forward needed for
species-naming conventions. <em>NATEE</em>, <em>7</em>(8), 1168–1169.
(<a href="https://doi.org/10.1038/s41559-023-02103-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Orr, Michael C. and Hughes, Alice C. and Carvajal, Omar Torres and Ferrari, Rafael R. and Luo, Arong and Rajaei, Hossein and Ron, Santiago R. and Warrit, Natapot and Zamani, Alireza and Zhang, Yuanmeng Miles and Zhu, Chao-Dong},
  doi          = {10.1038/s41559-023-02103-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1168-1169},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Inclusive and productive ways forward needed for species-naming conventions},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Change in biological nomenclature is overdue and possible.
<em>NATEE</em>, <em>7</em>(8), 1166–1167. (<a
href="https://doi.org/10.1038/s41559-023-02104-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Roksandic, Mirjana and Musiba, Charles and Radović, Predrag and Lindal, Joshua and Wu, Xiu-Jie and Figueiredo, Estrela and Smith, Gideon F. and Roksandic, Ivan and Bae, Christopher J.},
  doi          = {10.1038/s41559-023-02104-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1166-1167},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Change in biological nomenclature is overdue and possible},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Eponyms are important tools for biologists in the global
south. <em>NATEE</em>, <em>7</em>(8), 1164–1165. (<a
href="https://doi.org/10.1038/s41559-023-02102-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Jost, Lou and Yanez-Muñoz, Mario Humberto and Brito, Jorge and Reyes-Puig, Carolina and Reyes-Puig, Juan Pablo and Guayasamín, Juan M. and Ron, Santiago R. and Quintana, Catalina and Iturralde, Gabriel and Baquero, Luis and Monteros, Marco and Freire-Fierro, Alina and Fernández, Diana and Mendieta-Leiva, Glenda and Morales, J. Francisco and Karremans, Adam P. and Vázquez-García, J. Antonio and Salazar, Gerardo A. and Hágsater, Eric and Solano, Rodolfo and Fernández-Concha, Germán Carnevali and Arana, Marcelo},
  doi          = {10.1038/s41559-023-02102-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1164-1165},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Eponyms are important tools for biologists in the global south},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Disrupting the epistemic empire is necessary for a
decolonial ecology. <em>NATEE</em>, <em>7</em>(8), 1163. (<a
href="https://doi.org/10.1038/s41559-023-02105-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Mabele, Mathew Bukhi and Kiwango, Wilhelm Andrew and Mwanyoka, Iddi},
  doi          = {10.1038/s41559-023-02105-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1163},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Disrupting the epistemic empire is necessary for a decolonial ecology},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). People-inspired names remain valuable. <em>NATEE</em>,
<em>7</em>(8), 1161–1162. (<a
href="https://doi.org/10.1038/s41559-023-02108-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Antonelli, Alexandre and Farooq, Harith and Colli-Silva, Matheus and Araújo, João P. M. and Freitas, André V. L. and Gardner, Elliot M. and Grace, Olwen and Gu, Shiran and Marline, Lovanomenjanahary and Nesbitt, Mark and Niskanen, Tuula and Onana, Jean Michel and Pérez-Escobar, Oscar A. and Taylor, Charlotte and Knapp, Sandra},
  doi          = {10.1038/s41559-023-02108-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1161-1162},
  shortjournal = {Nat. Ecol. Evol},
  title        = {People-inspired names remain valuable},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Eponyms have no place in 21st-century biological
nomenclature. <em>NATEE</em>, <em>7</em>(8), 1157–1160. (<a
href="https://doi.org/10.1038/s41559-023-02022-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We argue that naming species in honour of a specific person is unjustifiable and out of step with equality and representation. Reforming taxonomy to remove eponyms will not be easy but could bring multiple benefits for both conservation and society.},
  archive      = {J_NATEE},
  author       = {Guedes, Patrícia and Alves-Martins, Fernanda and Arribas, Javier Martínez and Chatterjee, Sumita and Santos, Ana M. C. and Lewin, Amir and Bako, Longji and Webala, Paul W. and Correia, Ricardo A. and Rocha, Ricardo and Ladle, Richard J.},
  doi          = {10.1038/s41559-023-02022-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1157-1160},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Eponyms have no place in 21st-century biological nomenclature},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evolution at the cellular level. <em>NATEE</em>,
<em>7</em>(8), 1155–1156. (<a
href="https://doi.org/10.1038/s41559-023-02133-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Capitalizing on developments in genome sequencing technology, the Biodiversity Cell Atlas is a multinational project that uses single-cell transcriptomics to map cell types of whole organisms across the tree of life.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02133-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {8},
  number       = {8},
  pages        = {1155-1156},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolution at the cellular level},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Symbioses shape feeding niches and
diversification across insects. <em>NATEE</em>, <em>7</em>(7), 1153. (<a
href="https://doi.org/10.1038/s41559-023-02137-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Cornwallis, Charlie K. and van ’t Padje, Anouk and Ellers, Jacintha and Klein, Malin and Jackson, Raphaella and Kiers, E. Toby and West, Stuart A. and Henry, Lee M.},
  doi          = {10.1038/s41559-023-02137-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1153},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Symbioses shape feeding niches and diversification across insects},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Coordinated surveillance is essential to
monitor and mitigate the evolutionary impacts of SARS-CoV-2 spillover
and circulation in animal hosts. <em>NATEE</em>, <em>7</em>(7), 1152.
(<a href="https://doi.org/10.1038/s41559-023-02118-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Kuchipudi, Suresh V. and Tan, Cedric and van Dorp, Lucy and Lichtveld, Maureen and Pickering, Bradley and Bowman, Jeff and Mubareka, Samira and Balloux, Francois},
  doi          = {10.1038/s41559-023-02118-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1152},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Coordinated surveillance is essential to monitor and mitigate the evolutionary impacts of SARS-CoV-2 spillover and circulation in animal hosts},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Longstanding behavioural stability in west africa extends to
the middle pleistocene at bargny, coastal senegal. <em>NATEE</em>,
<em>7</em>(7), 1141–1151. (<a
href="https://doi.org/10.1038/s41559-023-02046-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Middle Stone Age (MSA) technologies first appear in the archaeological records of northern, eastern and southern Africa during the Middle Pleistocene epoch. The absence of MSA sites from West Africa limits evaluation of shared behaviours across the continent during the late Middle Pleistocene and the diversity of subsequent regionalized trajectories. Here we present evidence for the late Middle Pleistocene MSA occupation of the West African littoral at Bargny, Senegal, dating to 150 thousand years ago. Palaeoecological evidence suggests that Bargny was a hydrological refugium during the MSA occupation, supporting estuarine conditions during Middle Pleistocene arid phases. The stone tool technology at Bargny presents characteristics widely shared across Africa in the late Middle Pleistocene but which remain uniquely stable in West Africa to the onset of the Holocene. We explore how the persistent habitability of West African environments, including mangroves, contributes to distinctly West African trajectories of behavioural stability. The late Middle Pleistocene site of Bargny, Senegal, documents stone tool trends seen across contemporary sites in Africa but which, in West Africa, remain uniquely stable into the Holocene. Palaeoenvironmental data suggest that persistently stable environments in West Africa through the Late Pleistocene, including estuarine refugia, may have supported consistent behavioural responses.},
  archive      = {J_NATEE},
  author       = {Niang, Khady and Blinkhorn, James and Bateman, Mark D. and Kiahtipes, Christopher A.},
  doi          = {10.1038/s41559-023-02046-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1141-1151},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Longstanding behavioural stability in west africa extends to the middle pleistocene at bargny, coastal senegal},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Extended embryo retention and viviparity in the first
amniotes. <em>NATEE</em>, <em>7</em>(7), 1131–1140. (<a
href="https://doi.org/10.1038/s41559-023-02074-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The amniotic egg with its complex fetal membranes was a key innovation in vertebrate evolution that enabled the great diversification of reptiles, birds and mammals. It is debated whether these fetal membranes evolved in eggs on land as an adaptation to the terrestrial environment or to control antagonistic fetal–maternal interaction in association with extended embryo retention (EER). Here we report an oviparous choristodere from the Lower Cretaceous period of northeast China. The ossification sequence of the embryo confirms that choristoderes are basal archosauromorphs. The discovery of oviparity in this assumed viviparous extinct clade, together with existing evidence, suggests that EER was the primitive reproductive mode in basal archosauromorphs. Phylogenetic comparative analyses on extant and extinct amniotes suggest that the first amniote displayed EER (including viviparity). The authors report an oviparous new specimen of the choristodere Ikechosaurus sp. from the Lower Cretaceous of China, confirming the basal archosauromorph affiliation of choristoderes. Phylogenetic analyses of this specimen along with other extinct and extant amniotes suggest that the ancestral amniote displayed extended embryo retention, including viviparity.},
  archive      = {J_NATEE},
  author       = {Jiang, Baoyu and He, Yiming and Elsler, Armin and Wang, Shengyu and Keating, Joseph N. and Song, Junyi and Kearns, Stuart L. and Benton, Michael J.},
  doi          = {10.1038/s41559-023-02074-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1131-1140},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Extended embryo retention and viviparity in the first amniotes},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Eighty million years of rapid evolution of the primate y
chromosome. <em>NATEE</em>, <em>7</em>(7), 1114–1130. (<a
href="https://doi.org/10.1038/s41559-022-01974-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Y chromosome usually plays a critical role in determining male sex and comprises sequence classes that have experienced unique evolutionary trajectories. Here we generated 19 new primate sex chromosome assemblies, analysed them with 10 existing assemblies and report rapid evolution of the Y chromosome across primates. The pseudoautosomal boundary has shifted at least six times during primate evolution, leading to the formation of a Simiiformes-specific evolutionary stratum and to the independent start of young strata in Catarrhini and Platyrrhini. Different primate lineages experienced different rates of gene loss and structural and chromatin change on their Y chromosomes. Selection on several Y-linked genes has contributed to the evolution of male developmental traits across the primates. Additionally, lineage-specific expansions of ampliconic regions have further increased the diversification of the structure and gene composition of the Y chromosome. Overall, our comprehensive analysis has broadened our knowledge of the evolution of the primate Y chromosome. Comparative analysis of Y chromosomes across 29 primate species reveals rapid evolution and shows different patterns of evolution among regions of the Y chromosome during primate diversification.},
  archive      = {J_NATEE},
  author       = {Zhou, Yang and Zhan, Xiaoyu and Jin, Jiazheng and Zhou, Long and Bergman, Juraj and Li, Xuemei and Rousselle, Marjolaine Marie C. and Belles, Meritxell Riera and Zhao, Lan and Fang, Miaoquan and Chen, Jiawei and Fang, Qi and Kuderna, Lukas and Marques-Bonet, Tomas and Kitayama, Haruka and Hayakawa, Takashi and Yao, Yong-Gang and Yang, Huanming and Cooper, David N. and Qi, Xiaoguang and Wu, Dong-Dong and Schierup, Mikkel Heide and Zhang, Guojie},
  doi          = {10.1038/s41559-022-01974-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1114-1130},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Eighty million years of rapid evolution of the primate y chromosome},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Climate co-benefits of tiger conservation. <em>NATEE</em>,
<em>7</em>(7), 1104–1113. (<a
href="https://doi.org/10.1038/s41559-023-02069-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biodiversity conservation is increasingly being recognized as an important co-benefit in climate change mitigation programmes that use nature-based climate solutions. However, the climate co-benefits of biodiversity conservation interventions, such as habitat protection and restoration, remain understudied. Here we estimate the forest carbon storage co-benefits of a national policy intervention for tiger (Panthera tigris) conservation in India. We used a synthetic control approach to model avoided forest loss and associated carbon emissions reductions in protected areas that underwent enhanced protection for tiger conservation. Over a third of the analysed reserves showed significant but mixed effects, where 24% of all reserves successfully reduced the rate of deforestation and the remaining 9% reported higher-than-expected forest loss. The policy had a net positive benefit with over 5,802 hectares of averted forest loss, corresponding to avoided emissions of 1.08 ± 0.51 MtCO2 equivalent between 2007 and 2020. This translated to US$92.55 ± 43.56 million in ecosystem services from the avoided social cost of emissions and potential revenue of US$6.24 ± 2.94 million in carbon offsets. Our findings offer an approach to quantitatively track the carbon sequestration co-benefits of a species conservation strategy and thus help align the objectives of climate action and biodiversity conservation. A synthetic control approach to model avoided forest loss shows that a protected-areas programme in India aimed at tiger conservation is associated with significant reductions in carbon emissions.},
  archive      = {J_NATEE},
  author       = {Lamba, Aakash and Teo, Hoong Chen and Sreekar, Rachakonda and Zeng, Yiwen and Carrasco, Luis Roman and Koh, Lian Pin},
  doi          = {10.1038/s41559-023-02069-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1104-1113},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Climate co-benefits of tiger conservation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Occurrence dynamics of mammals in protected tropical forests
respond to human presence and activities. <em>NATEE</em>, <em>7</em>(7),
1092–1103. (<a
href="https://doi.org/10.1038/s41559-023-02060-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Protected areas (PAs) play a vital role in wildlife conservation. Nonetheless there is concern and uncertainty regarding how and at what spatial scales anthropogenic stressors influence the occurrence dynamics of wildlife populations inside PAs. Here we assessed how anthropogenic stressors influence occurrence dynamics of 159 mammal species in 16 tropical PAs from three biogeographic regions. We quantified these relationships for species groups (habitat specialists and generalists) and individual species. We used long-term camera-trap data (1,002 sites) and fitted Bayesian dynamic multispecies occupancy models to estimate local colonization (the probability that a previously empty site is colonized) and local survival (the probability that an occupied site remains occupied). Multiple covariates at both the local scale and landscape scale influenced mammal occurrence dynamics, although responses differed among species groups. Colonization by specialists increased with local-scale forest cover when landscape-scale fragmentation was low. Survival probability of generalists was higher near the edge than in the core of the PA when landscape-scale human population density was low but the opposite occurred when population density was high. We conclude that mammal occurrence dynamics are impacted by anthropogenic stressors acting at multiple scales including outside the PA itself. Camera-trap data for 159 mammalian species at 1,002 sites across 16 tropical-forest protected areas show how local survival and colonization probabilities of specialist and generalist species are differently affected by human-induced stressors at different spatial scales, such as human population density and forest fragmentation.},
  archive      = {J_NATEE},
  author       = {Semper-Pascual, Asunción and Sheil, Douglas and Beaudrot, Lydia and Dupont, Pierre and Dey, Soumen and Ahumada, Jorge and Akampurira, Emmanuel and Bitariho, Robert and Espinosa, Santiago and Jansen, Patrick A. and Lima, Marcela Guimarães Moreira and Martin, Emanuel H. and Mugerwa, Badru and Rovero, Francesco and Santos, Fernanda and Uzabaho, Eustrate and Bischof, Richard},
  doi          = {10.1038/s41559-023-02060-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1092-1103},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Occurrence dynamics of mammals in protected tropical forests respond to human presence and activities},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Climate-driven variation in dispersal ability predicts
responses to forest fragmentation in birds. <em>NATEE</em>,
<em>7</em>(7), 1079–1091. (<a
href="https://doi.org/10.1038/s41559-023-02077-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Species sensitivity to forest fragmentation varies latitudinally, peaking in the tropics. A prominent explanation for this pattern is that historical landscape disturbance at higher latitudes has removed fragmentation-sensitive species or promoted the evolution of more resilient survivors. However, it is unclear whether this so-called extinction filter is the dominant driver of geographic variation in fragmentation sensitivity, particularly because climatic factors may also cause latitudinal gradients in dispersal ability, a key trait mediating sensitivity to habitat fragmentation. Here we combine field survey data with a morphological proxy for avian dispersal ability (hand-wing index) to assess responses to forest fragmentation in 1,034 bird species worldwide. We find that fragmentation sensitivity is strongly predicted by dispersal limitation and that other factors—latitude, body mass and historical disturbance events—have relatively limited explanatory power after accounting for species differences in dispersal. We also show that variation in dispersal ability is only weakly predicted by historical disturbance and more strongly associated with intra-annual temperature fluctuations (seasonality). Our results suggest that climatic factors play a dominant role in driving global variation in the impacts of forest fragmentation, emphasizing the need for more nuanced environmental policies that take into account local context and associated species traits. Field survey data from 1,034 bird species worldwide are used to show that species’ sensitivities to habitat fragmentation are more strongly related to their dispersal ability than to latitude and past habitat disturbance, and that variation in dispersal ability is in turn strongly associated with climate.},
  archive      = {J_NATEE},
  author       = {Weeks, Thomas L. and Betts, Matthew G. and Pfeifer, Marion and Wolf, Christopher and Banks-Leite, Cristina and Barbaro, Luc and Barlow, Jos and Cerezo, Alexis and Kennedy, Christina M. and Kormann, Urs G. and Marsh, Charles J. and Olivier, Pieter I. and Phalan, Benjamin T. and Possingham, Hugh P. and Wood, Eric M. and Tobias, Joseph A.},
  doi          = {10.1038/s41559-023-02077-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1079-1091},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Climate-driven variation in dispersal ability predicts responses to forest fragmentation in birds},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Human pressures modulate climate-warming-induced changes in
size spectra of stream fish communities. <em>NATEE</em>, <em>7</em>(7),
1072–1078. (<a
href="https://doi.org/10.1038/s41559-023-02083-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Climate warming can negatively affect the body size of ectothermic organisms and, based on known temperature–size rules, tends to benefit small-bodied organisms. Our understanding of the interactive effects of climate warming and other environmental factors on the temporal changes of body size structure is limited. We quantified the annual trends in size spectra of 583 stream fish communities sampled for more than 20 years across France. The results show that climate warming steepened the slope of the community size spectrum in streams with limited impacts from other human pressures. These changes were caused by increasing abundance of small‐bodied individuals and decreasing abundance of large‐bodied individuals. However, opposite effects of climate warming on the size spectrum slopes were observed in streams facing high levels of other human pressures. This demonstrates that the effects of temperature on body size structure can depend on other human pressures, disrupting the natural patterns of size spectra in wild communities with potentially strong implications for the fluxes of energy and nutrients in ecosystems. Analysing more than 20 years of data on stream fish communities in France, the authors show that streams facing high levels of human pressures have reversed effects of climate warming on fish body size spectrum slopes.},
  archive      = {J_NATEE},
  author       = {Arranz, Ignasi and Grenouillet, Gaël and Cucherousset, Julien},
  doi          = {10.1038/s41559-023-02083-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1072-1078},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Human pressures modulate climate-warming-induced changes in size spectra of stream fish communities},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Abrupt expansion of climate change risks for species
globally. <em>NATEE</em>, <em>7</em>(7), 1060–1071. (<a
href="https://doi.org/10.1038/s41559-023-02070-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Climate change is already exposing species to dangerous temperatures driving widespread population and geographical contractions. However, little is known about how these risks of thermal exposure will expand across species’ existing geographical ranges over time as climate change continues. Here, using geographical data for approximately 36,000 marine and terrestrial species and climate projections to 2100, we show that the area of each species’ geographical range at risk of thermal exposure will expand abruptly. On average, more than 50% of the increase in exposure projected for a species will occur in a single decade. This abruptness is partly due to the rapid pace of future projected warming but also because the greater area available at the warm end of thermal gradients constrains species to disproportionately occupy sites close to their upper thermal limit. These geographical constraints on the structure of species ranges operate both on land and in the ocean and mean that, even in the absence of amplifying ecological feedbacks, thermally sensitive species may be inherently vulnerable to sudden warming-driven collapse. With higher levels of warming, the number of species passing these thermal thresholds, and at risk of abrupt and widespread thermal exposure, increases, doubling from less than 15% to more than 30% between 1.5 °C and 2.5 °C of global warming. These results indicate that climate threats to thousands of species are expected to expand abruptly in the coming decades, thereby highlighting the urgency of mitigation and adaptation actions. Using geographical data for approximately 36,000 marine and terrestrial species and climate projections to 2100, the authors show that the area of each species’ geographical range at risk of thermal exposure will expand abruptly, highlighting the urgency of mitigation and adaptation actions.},
  archive      = {J_NATEE},
  author       = {Pigot, Alex L. and Merow, Cory and Wilson, Adam and Trisos, Christopher H.},
  doi          = {10.1038/s41559-023-02070-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1060-1071},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Abrupt expansion of climate change risks for species globally},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evidence-based target setting informs blue carbon strategies
for nationally determined contributions. <em>NATEE</em>, <em>7</em>(7),
1045–1059. (<a
href="https://doi.org/10.1038/s41559-023-02081-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The magnitude and pace of global climate change demand ambitious and effective implementation of nationally determined contributions (NDCs). Nature-based solutions present an efficient approach to achieving mitigation, adaptation and resilience goals. Yet few nations have quantified the diverse benefits of nature-based solutions to evaluate and select ecosystem targets for their NDCs. Here we report on Belize’s pursuit of innovative, evidence-based target setting by accounting for multiple benefits of blue carbon strategies. Through quantification of carbon storage and sequestration and optimization of co-benefits, we explore time-bound targets and prioritize locations for mangrove protection and restoration. We find increases in carbon benefits with larger mangrove investments, while fisheries, tourism and coastal risk-reduction co-benefits grow initially and then plateau. We identify locations, currently lacking protected status, where prioritizing blue carbon strategies would provide the greatest delivery of co-benefits to communities. These findings informed Belize’s updated NDCs to include an additional 12,000 ha of mangrove protection and 4,000 ha of mangrove restoration, respectively, by 2030. Our study serves as an example for the more than 150 other countries that have the opportunity to enhance greenhouse gas sequestration and climate adaptation by incorporating blue carbon strategies that provide multiple societal benefits into their NDCs. An assessment of blue carbon strategies in Belize shows how quantifying fisheries, tourism and coastal risk co-benefits alongside carbon benefits can inform spatial and temporal target setting for nationally determined climate contributions that simultaneously provide societal benefits.},
  archive      = {J_NATEE},
  author       = {Arkema, Katie K. and Delevaux, Jade M. S. and Silver, Jessica M. and Winder, Samantha G. and Schile-Beers, Lisa M. and Bood, Nadia and Crooks, Stephen and Douthwaite, Karen and Durham, Courtney and Hawthorne, Peter L. and Hickey, Thomas and Mattis, Colin and Rosado, Andria and Ruckelshaus, Mary and von Unger, Moritz and Young, Arlene},
  doi          = {10.1038/s41559-023-02081-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1045-1059},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evidence-based target setting informs blue carbon strategies for nationally determined contributions},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Symbioses shape feeding niches and diversification across
insects. <em>NATEE</em>, <em>7</em>(7), 1022–1044. (<a
href="https://doi.org/10.1038/s41559-023-02058-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For over 300 million years, insects have relied on symbiotic microbes for nutrition and defence. However, it is unclear whether specific ecological conditions have repeatedly favoured the evolution of symbioses, and how this has influenced insect diversification. Here, using data on 1,850 microbe–insect symbioses across 402 insect families, we found that symbionts have allowed insects to specialize on a range of nutrient-imbalanced diets, including phloem, blood and wood. Across diets, the only limiting nutrient consistently associated with the evolution of obligate symbiosis was B vitamins. The shift to new diets, facilitated by symbionts, had mixed consequences for insect diversification. In some cases, such as herbivory, it resulted in spectacular species proliferation. In other niches, such as strict blood feeding, diversification has been severely constrained. Symbioses therefore appear to solve widespread nutrient deficiencies for insects, but the consequences for insect diversification depend on the feeding niche that is invaded. Insects rely on symbiotic microbes for nutrition and defence. Analysing a large dataset of microbe–insect symbioses, the authors show that symbiosis evolved in response to nutrient deficiencies but its impacts on insect diversification depend on their feeding niche.},
  archive      = {J_NATEE},
  author       = {Cornwallis, Charlie K. and van ’t Padje, Anouk and Ellers, Jacintha and Klein, Malin and Jackson, Raphaella and Kiers, E. Toby and West, Stuart A. and Henry, Lee M.},
  doi          = {10.1038/s41559-023-02058-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1022-1044},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Symbioses shape feeding niches and diversification across insects},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Convergence of dominance and neglect in flying insect
diversity. <em>NATEE</em>, <em>7</em>(7), 1012–1021. (<a
href="https://doi.org/10.1038/s41559-023-02066-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most of arthropod biodiversity is unknown to science. Consequently, it has been unclear whether insect communities around the world are dominated by the same or different taxa. This question can be answered through standardized sampling of biodiversity followed by estimation of species diversity and community composition with DNA barcodes. Here this approach is applied to flying insects sampled by 39 Malaise traps placed in five biogeographic regions, eight countries and numerous habitats (&amp;gt;225,000 specimens belonging to &amp;gt;25,000 species in 458 families). We find that 20 insect families (10 belonging to Diptera) account for &amp;gt;50% of local species diversity regardless of clade age, continent, climatic region and habitat type. Consistent differences in family-level dominance explain two-thirds of variation in community composition despite massive levels of species turnover, with most species (&amp;gt;97%) in the top 20 families encountered at a single site only. Alarmingly, the same families that dominate insect diversity are ‘dark taxa’ in that they suffer from extreme taxonomic neglect, with little signs of increasing activities in recent years. Taxonomic neglect tends to increase with diversity and decrease with body size. Identifying and tackling the diversity of ‘dark taxa’ with scalable techniques emerge as urgent priorities in biodiversity science. Using DNA barcoding to analyse flying insect diversity of &amp;gt;225,000 specimens from five biogeographic regions, the authors show that more than half of local species diversity is represented by only 20 insect families, most of which suffer from taxonomic neglect.},
  archive      = {J_NATEE},
  author       = {Srivathsan, Amrita and Ang, Yuchen and Heraty, John M. and Hwang, Wei Song and Jusoh, Wan F. A. and Kutty, Sujatha Narayanan and Puniamoorthy, Jayanthi and Yeo, Darren and Roslin, Tomas and Meier, Rudolf},
  doi          = {10.1038/s41559-023-02066-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1012-1021},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Convergence of dominance and neglect in flying insect diversity},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Water availability creates global thresholds in
multidimensional soil biodiversity and functions. <em>NATEE</em>,
<em>7</em>(7), 1002–1011. (<a
href="https://doi.org/10.1038/s41559-023-02071-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Soils support an immense portion of Earth’s biodiversity and maintain multiple ecosystem functions which are essential for human well-being. Environmental thresholds are known to govern global vegetation patterns, but it is still unknown whether they can be used to predict the distribution of soil organisms and functions across global biomes. Using a global field survey of 383 sites across contrasting climatic and vegetation conditions, here we showed that soil biodiversity and functions exhibited pervasive nonlinear patterns worldwide and are mainly governed by water availability (precipitation and potential evapotranspiration). Changes in water availability resulted in drastic shifts in soil biodiversity (bacteria, fungi, protists and invertebrates) and soil functions including plant–microbe interactions, plant productivity, soil biogeochemical cycles and soil carbon sequestration. Our findings highlight that crossing specific water availability thresholds can have critical consequences for the provision of essential ecosystem services needed to sustain our planet. A global field survey of 383 sites with different vegetation types spanning an environmental gradient reveals that soil biodiversity and functions exhibit pervasive nonlinear behaviours worldwide and are mainly governed by water availability.},
  archive      = {J_NATEE},
  author       = {Zhang, Jianwei and Feng, Youzhi and Maestre, Fernando T. and Berdugo, Miguel and Wang, Juntao and Coleine, Claudia and Sáez-Sandino, Tadeo and García-Velázquez, Laura and Singh, Brajesh K. and Delgado-Baquerizo, Manuel},
  doi          = {10.1038/s41559-023-02071-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {1002-1011},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Water availability creates global thresholds in multidimensional soil biodiversity and functions},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Biodiversity time series are biased towards increasing
species richness in changing environments. <em>NATEE</em>,
<em>7</em>(7), 994–1001. (<a
href="https://doi.org/10.1038/s41559-023-02078-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The discrepancy between global loss and local constant species richness has led to debates over data quality, systematic biases in monitoring programmes and the adequacy of species richness to capture changes in biodiversity. We show that, more fundamentally, null expectations of stable richness can be wrong, despite independent yet equal colonization and extinction. We analysed fish and bird time series and found an overall richness increase. This increase reflects a systematic bias towards an earlier detection of colonizations than extinctions. To understand how much this bias influences richness trends, we simulated time series using a neutral model controlling for equilibrium richness and temporal autocorrelation (that is, no trend expected). These simulated time series showed significant changes in richness, highlighting the effect of temporal autocorrelation on the expected baseline for species richness changes. The finite nature of time series, the long persistence of declining populations and the potential strong dispersal limitation probably lead to richness changes when changing conditions promote compositional turnover. Temporal analyses of richness should incorporate this bias by considering appropriate neutral baselines for richness changes. Absence of richness trends over time, as previously reported, can actually reflect a negative deviation from the positive biodiversity trend expected by default. There is great interest in describing biodiversity change through time, but such analyses present various technical challenges. Here, using datasets for fish and birds, the authors show that a bias towards colonization over extinction can result in an increasing species richness over time, especially in short time series, and argue that studies should account for this bias.},
  archive      = {J_NATEE},
  author       = {Kuczynski, Lucie and Ontiveros, Vicente J. and Hillebrand, Helmut},
  doi          = {10.1038/s41559-023-02078-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {994-1001},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Biodiversity time series are biased towards increasing species richness in changing environments},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The roles of sexual selection and sexual conflict in shaping
patterns of genome and transcriptome variation. <em>NATEE</em>,
<em>7</em>(7), 981–993. (<a
href="https://doi.org/10.1038/s41559-023-02019-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sexual dimorphism is one of the most prevalent, and often the most extreme, examples of phenotypic variation within species, and arises primarily from genomic variation that is shared between females and males. Many sexual dimorphisms arise through sex differences in gene expression, and sex-biased expression is one way that a single, shared genome can generate multiple, distinct phenotypes. Although many sexual dimorphisms are expected to result from sexual selection, and many studies have invoked the possible role of sexual selection to explain sex-specific traits, the role of sexual selection in the evolution of sexually dimorphic gene expression remains difficult to differentiate from other forms of sex-specific selection. In this Review, we propose a holistic framework for the study of sex-specific selection and transcriptome evolution. We advocate for a comparative approach, across tissues, developmental stages and species, which incorporates an understanding of the molecular mechanisms, including genomic variation and structure, governing gene expression. Such an approach is expected to yield substantial insights into the evolution of genetic variation and have important applications in a variety of fields, including ecology, evolution and behaviour. This Review discusses how sexual selection and sexual conflict shape genomes and transcriptomes and proposes an integrative approach combining behavioural ecology with developmental and genomic biology to answer open questions.},
  archive      = {J_NATEE},
  author       = {Tosto, Nicole M. and Beasley, Emily R. and Wong, Bob B. M. and Mank, Judith E. and Flanagan, Sarah P.},
  doi          = {10.1038/s41559-023-02019-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {981-993},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The roles of sexual selection and sexual conflict in shaping patterns of genome and transcriptome variation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Humans influence mammal populations even inside protected
areas. <em>NATEE</em>, <em>7</em>(7), 979–980. (<a
href="https://doi.org/10.1038/s41559-023-02068-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of millions of wildlife photographs has revealed that survival and colonization probabilities of mammals in protected areas are associated with people and what they do both inside and outside these areas.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02068-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {979-980},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Humans influence mammal populations even inside protected areas},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Climate warming effects on stream-fish size spectra are
modulated by other perturbations. <em>NATEE</em>, <em>7</em>(7),
977–978. (<a href="https://doi.org/10.1038/s41559-023-02085-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Across taxa, increased temperature has been reported to generally reduce individual body size and change the body size structure of wild communities. Our research reveals that the effects of climate warming on the body size structure of stream fishes are also dependent on the intensity of other human pressures.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02085-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {977-978},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Climate warming effects on stream-fish size spectra are modulated by other perturbations},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). One species, many roots? <em>NATEE</em>, <em>7</em>(7),
975–976. (<a href="https://doi.org/10.1038/s41559-023-02080-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new genetic study provides strong support for the view that our species evolved from exchanges between several ancestral populations in different African regions.},
  archive      = {J_NATEE},
  author       = {Scerri, Eleanor M. L.},
  doi          = {10.1038/s41559-023-02080-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {975-976},
  shortjournal = {Nat. Ecol. Evol},
  title        = {One species, many roots?},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Egg-bound in deep time. <em>NATEE</em>, <em>7</em>(7),
973–974. (<a href="https://doi.org/10.1038/s41559-023-02084-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A fossilized egg anchors an analysis of early egg evolution, suggesting that ancestral amniotes retained eggs for an extended period of development.},
  archive      = {J_NATEE},
  author       = {Evans, Susan E.},
  doi          = {10.1038/s41559-023-02084-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {973-974},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Egg-bound in deep time},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Structural shifts in primate y. <em>NATEE</em>,
<em>7</em>(7), 971–972. (<a
href="https://doi.org/10.1038/s41559-023-01984-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of Y chromosomes from 29 primate species shows lineage-specific evolutionary strata as well as changes in the 3D structure, rearrangements and positive selection that have shaped the primate Y chromosome over the past 80 million years.},
  archive      = {J_NATEE},
  author       = {Cortez, Diego},
  doi          = {10.1038/s41559-023-01984-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {971-972},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Structural shifts in primate y},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Tiger protection brings carbon benefits. <em>NATEE</em>,
<em>7</em>(7), 969–970. (<a
href="https://doi.org/10.1038/s41559-023-02062-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A quasi-experimental impact evaluation quantifies reduced forest loss, avoided social cost of emissions and potential carbon-offset revenue associated with India’s designation of protected areas as tiger-conservation reserves with enhanced protection.},
  archive      = {J_NATEE},
  author       = {Sills, Erin O. and Kramer, Randall A.},
  doi          = {10.1038/s41559-023-02062-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {969-970},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Tiger protection brings carbon benefits},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Blue carbon for climate and co-benefits. <em>NATEE</em>,
<em>7</em>(7), 967–968. (<a
href="https://doi.org/10.1038/s41559-023-02034-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study from Belize demonstrates how to set targets for coastal ecosystem conservation and restoration, and to quantify the resulting suite of benefits for achieving climate change mitigation and adaptation goals under the Paris Agreement.},
  archive      = {J_NATEE},
  author       = {Lester, Sarah E.},
  doi          = {10.1038/s41559-023-02034-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {967-968},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Blue carbon for climate and co-benefits},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Domestication impacts restoration. <em>NATEE</em>,
<em>7</em>(7), 966. (<a
href="https://doi.org/10.1038/s41559-023-02096-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Harold, Simon},
  doi          = {10.1038/s41559-023-02096-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {966},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Domestication impacts restoration},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Sick again! <em>NATEE</em>, <em>7</em>(7), 965. (<a
href="https://doi.org/10.1038/s41559-023-02075-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Dobson, Andy},
  doi          = {10.1038/s41559-023-02075-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {965},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sick again!},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). John alcock (1943–2023). <em>NATEE</em>, <em>7</em>(7),
963–964. (<a href="https://doi.org/10.1038/s41559-023-02073-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An inspirational advocate for adaptation, animal behaviour and natural history.},
  archive      = {J_NATEE},
  author       = {Kemp, Darrell J. and Rutowski, Ronald L. and Simmons, Leigh W.},
  doi          = {10.1038/s41559-023-02073-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {963-964},
  shortjournal = {Nat. Ecol. Evol},
  title        = {John alcock (1943–2023)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). AI chatbots can boost scientific coding. <em>NATEE</em>,
<em>7</em>(7), 960–962. (<a
href="https://doi.org/10.1038/s41559-023-02063-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chatbots powered by artificial intelligence, such as ChatGPT, are ready to speed up monotonous coding tasks and teach you new skills. We highlight, with worked examples, some advantages and limitations of using generative artificial intelligence for scientific coding and argue that if you are willing to debug, you can get a head start on more challenging tasks.},
  archive      = {J_NATEE},
  author       = {Merow, Cory and Serra-Diaz, Josep M. and Enquist, Brian J. and Wilson, Adam M.},
  doi          = {10.1038/s41559-023-02063-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {960-962},
  shortjournal = {Nat. Ecol. Evol},
  title        = {AI chatbots can boost scientific coding},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Coordinated surveillance is essential to monitor and
mitigate the evolutionary impacts of SARS-CoV-2 spillover and
circulation in animal hosts. <em>NATEE</em>, <em>7</em>(7), 956–959. (<a
href="https://doi.org/10.1038/s41559-023-02082-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SARS-CoV-2 lineages circulating in animal reservoirs may broaden the evolutionary potential of the virus and increase the risk of novel variants emerging. There is an urgent need for more-comprehensive surveillance of SARS-CoV-2 circulating in nonhuman hosts.},
  archive      = {J_NATEE},
  author       = {Kuchipudi, Suresh V. and Tan, Cedric and van Dorp, Lucy and Lichtveld, Maureen and Pickering, Bradley and Bowman, Jeff and Mubareka, Samira and Balloux, Francois},
  doi          = {10.1038/s41559-023-02082-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {956-959},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Coordinated surveillance is essential to monitor and mitigate the evolutionary impacts of SARS-CoV-2 spillover and circulation in animal hosts},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Co-benefits towards commitments. <em>NATEE</em>,
<em>7</em>(7), 955. (<a
href="https://doi.org/10.1038/s41559-023-02126-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Successful examples of climate and social benefits that derive from national conservation actions may help to catalyse the development of other policies that are designed to maximize these synergies.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02126-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {7},
  number       = {7},
  pages        = {955},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Co-benefits towards commitments},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Resurrecting the alternative splicing landscape of archaic
hominins using machine learning. <em>NATEE</em>, <em>7</em>(6), 939–953.
(<a href="https://doi.org/10.1038/s41559-023-02053-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Alternative splicing contributes to adaptation and divergence in many species. However, it has not been possible to directly compare splicing between modern and archaic hominins. Here, we unmask the recent evolution of this previously unobservable regulatory mechanism by applying SpliceAI, a machine-learning algorithm that identifies splice-altering variants (SAVs), to high-coverage genomes from three Neanderthals and a Denisovan. We discover 5,950 putative archaic SAVs, of which 2,186 are archaic-specific and 3,607 also occur in modern humans via introgression (244) or shared ancestry (3,520). Archaic-specific SAVs are enriched in genes that contribute to traits potentially relevant to hominin phenotypic divergence, such as the epidermis, respiration and spinal rigidity. Compared to shared SAVs, archaic-specific SAVs occur in sites under weaker selection and are more common in genes with tissue-specific expression. Further underscoring the importance of negative selection on SAVs, Neanderthal lineages with low effective population sizes are enriched for SAVs compared to Denisovan and shared SAVs. Finally, we find that nearly all introgressed SAVs in humans were shared across the three Neanderthals, suggesting that older SAVs were more tolerated in human genomes. Our results reveal the splicing landscape of archaic hominins and identify potential contributions of splicing to phenotypic differences among hominins. The authors use machine learning to characterize the splicing landscape of archaic hominins. Archaic-specific splice-altering variants might have contributed to phenotypic differences among hominins and were under negative selection.},
  archive      = {J_NATEE},
  author       = {Brand, Colin M. and Colbran, Laura L. and Capra, John A.},
  doi          = {10.1038/s41559-023-02053-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {939-953},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Resurrecting the alternative splicing landscape of archaic hominins using machine learning},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Great ape cognition is structured by stable cognitive
abilities and predicted by developmental conditions. <em>NATEE</em>,
<em>7</em>(6), 927–938. (<a
href="https://doi.org/10.1038/s41559-023-02050-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Great ape cognition is used as a reference point to specify the evolutionary origins of complex cognitive abilities, including in humans. This research often assumes that great ape cognition consists of cognitive abilities (traits) that account for stable differences between individuals, which change and develop in response to experience. Here, we test the validity of these assumptions by assessing repeatability of cognitive performance among captive great apes (Gorilla gorilla, Pongo abelii, Pan paniscus, Pan troglodytes) in five tasks covering a range of cognitive domains. We examine whether individual characteristics (age, group, test experience) or transient situational factors (life events, testing arrangements or sociality) influence cognitive performance. Our results show that task-level performance is generally stable over time; four of the five tasks were reliable measurement tools. Performance in the tasks was best explained by stable differences in cognitive abilities (traits) between individuals. Cognitive abilities were further correlated, suggesting shared cognitive processes. Finally, when predicting cognitive performance, we found stable individual characteristics to be more important than variables capturing transient experience. Taken together, this study shows that great ape cognition is structured by stable cognitive abilities that respond to different developmental conditions. Most comparative animal cognition studies assume that results are stable in individuals and groups, but this is not often tested. Here the authors assess repeatability of cognitive tasks in several species of captive great apes, finding that individual performance over time is stable and predicted by fixed differences among individuals rather than transient experimental conditions.},
  archive      = {J_NATEE},
  author       = {Bohn, Manuel and Eckert, Johanna and Hanus, Daniel and Lugauer, Benedikt and Holtmann, Jana and Haun, Daniel B. M.},
  doi          = {10.1038/s41559-023-02050-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {927-938},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Great ape cognition is structured by stable cognitive abilities and predicted by developmental conditions},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). PLIN2-induced ectopic lipid accumulation promotes muscle
ageing in gregarious locusts. <em>NATEE</em>, <em>7</em>(6), 914–926.
(<a href="https://doi.org/10.1038/s41559-023-02059-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ageing plasticity represents the flexibility of the ageing process in response to non-genetic factors, occurring commonly in animals. However, the regulatory mechanisms underlying ageing plasticity are largely unclear. The density-dependent polyphenism of locusts, Locusta migratoria, displays dramatic lifespan divergence between solitary and gregarious phases, providing a useful system for studying ageing plasticity. Here, we found that gregarious locusts displayed faster locomotor deficits and increased muscle degeneration on ageing than solitary locusts. Comparative transcriptome analysis in flight muscles revealed significant differences in transcriptional patterns on ageing between two phases. RNA interference screening showed that the knockdown of the upregulated PLIN2 gene significantly relieved the ageing-related flight impairments in gregarious locusts. Mechanistically, the gradual upregulation of PLIN2 could induce the accumulation of ectopic lipid droplets and triacylglycerols in flight muscles during the ageing process. Further experiments suggested that ectopic lipid accumulation led to an ageing-related β-oxidation decline through limiting fatty acid transport and content. These findings reveal the key roles of lipid metabolism in the differences of muscle ageing between solitary and gregarious locusts and provide a potential mechanism underlying environment-induced muscle ageing plasticity. Comparative transcriptomic analysis of flight muscles in migratory locusts reveals that plasticity in expression of the lipid metabolism gene PLIN2 regulates differences in ageing-related muscle degeneration between gregarious and solitary phases.},
  archive      = {J_NATEE},
  author       = {Guo, Siyuan and Hou, Li and Dong, Liushu and Nie, Xin and Kang, Le and Wang, Xianhui},
  doi          = {10.1038/s41559-023-02059-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {914-926},
  shortjournal = {Nat. Ecol. Evol},
  title        = {PLIN2-induced ectopic lipid accumulation promotes muscle ageing in gregarious locusts},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A global phylogeny of butterflies reveals their evolutionary
history, ancestral hosts and biogeographic origins. <em>NATEE</em>,
<em>7</em>(6), 903–913. (<a
href="https://doi.org/10.1038/s41559-023-02041-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Butterflies are a diverse and charismatic insect group that are thought to have evolved with plants and dispersed throughout the world in response to key geological events. However, these hypotheses have not been extensively tested because a comprehensive phylogenetic framework and datasets for butterfly larval hosts and global distributions are lacking. We sequenced 391 genes from nearly 2,300 butterfly species, sampled from 90 countries and 28 specimen collections, to reconstruct a new phylogenomic tree of butterflies representing 92% of all genera. Our phylogeny has strong support for nearly all nodes and demonstrates that at least 36 butterfly tribes require reclassification. Divergence time analyses imply an origin ~100 million years ago for butterflies and indicate that all but one family were present before the K/Pg extinction event. We aggregated larval host datasets and global distribution records and found that butterflies are likely to have first fed on Fabaceae and originated in what is now the Americas. Soon after the Cretaceous Thermal Maximum, butterflies crossed Beringia and diversified in the Palaeotropics. Our results also reveal that most butterfly species are specialists that feed on only one larval host plant family. However, generalist butterflies that consume two or more plant families usually feed on closely related plants. A new phylogenomic tree for butterflies is constructed using 391 genes from 2,300 species, representing 92% of genera. It suggests that butterflies originated around 100 million years ago in what is now the Americas and originally fed on Fabaceae.},
  archive      = {J_NATEE},
  author       = {Kawahara, Akito Y. and Storer, Caroline and Carvalho, Ana Paula S. and Plotkin, David M. and Condamine, Fabien L. and Braga, Mariana P. and Ellis, Emily A. and St Laurent, Ryan A. and Li, Xuankun and Barve, Vijay and Cai, Liming and Earl, Chandra and Frandsen, Paul B. and Owens, Hannah L. and Valencia-Montoya, Wendy A. and Aduse-Poku, Kwaku and Toussaint, Emmanuel F. A. and Dexter, Kelly M. and Doleck, Tenzing and Markee, Amanda and Messcher, Rebeccah and Nguyen, Y-Lan and Badon, Jade Aster T. and Benítez, Hugo A. and Braby, Michael F. and Buenavente, Perry A. C. and Chan, Wei-Ping and Collins, Steve C. and Rabideau Childers, Richard A. and Dankowicz, Even and Eastwood, Rod and Fric, Zdenek F. and Gott, Riley J. and Hall, Jason P. W. and Hallwachs, Winnie and Hardy, Nate B. and Sipe, Rachel L. Hawkins and Heath, Alan and Hinolan, Jomar D. and Homziak, Nicholas T. and Hsu, Yu-Feng and Inayoshi, Yutaka and Itliong, Micael G. A. and Janzen, Daniel H. and Kitching, Ian J. and Kunte, Krushnamegh and Lamas, Gerardo and Landis, Michael J. and Larsen, Elise A. and Larsen, Torben B. and Leong, Jing V. and Lukhtanov, Vladimir and Maier, Crystal A. and Martinez, Jose I. and Martins, Dino J. and Maruyama, Kiyoshi and Maunsell, Sarah C. and Mega, Nicolás Oliveira and Monastyrskii, Alexander and Morais, Ana B. B. and Müller, Chris J. and Naive, Mark Arcebal K. and Nielsen, Gregory and Padrón, Pablo Sebastián and Peggie, Djunijanti and Romanowski, Helena Piccoli and Sáfián, Szabolcs and Saito, Motoki and Schröder, Stefan and Shirey, Vaughn and Soltis, Doug and Soltis, Pamela and Sourakov, Andrei and Talavera, Gerard and Vila, Roger and Vlasanek, Petr and Wang, Houshuai and Warren, Andrew D. and Willmott, Keith R. and Yago, Masaya and Jetz, Walter and Jarzyna, Marta A. and Breinholt, Jesse W. and Espeland, Marianne and Ries, Leslie and Guralnick, Robert P. and Pierce, Naomi E. and Lohman, David J.},
  doi          = {10.1038/s41559-023-02041-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {903-913},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A global phylogeny of butterflies reveals their evolutionary history, ancestral hosts and biogeographic origins},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Single-cell adaptations shape evolutionary transitions to
multicellularity in green algae. <em>NATEE</em>, <em>7</em>(6), 889–902.
(<a href="https://doi.org/10.1038/s41559-023-02044-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The evolution of multicellular life has played a pivotal role in shaping biological diversity. However, we know surprisingly little about the natural environmental conditions that favour the formation of multicellular groups. Here we experimentally examine how key environmental factors (predation, nitrogen and water turbulence) combine to influence multicellular group formation in 35 wild unicellular green algae strains (19 Chlorophyta species). All environmental factors induced the formation of multicellular groups (more than four cells), but there was no evidence this was adaptive, as multicellularity (% cells in groups) was not related to population growth rate under any condition. Instead, population growth was related to extracellular matrix (ECM) around single cells and palmelloid formation, a unicellular life-cycle stage where two to four cells are retained within a mother-cell wall after mitosis. ECM production increased with nitrogen levels resulting in more cells being in palmelloids and higher rates of multicellular group formation. Examining the distribution of 332 algae species across 478 lakes monitored over 55 years, showed that ECM and nitrogen availability also predicted patterns of obligate multicellularity in nature. Our results highlight that adaptations of unicellular organisms to cope with environmental challenges may be key to understanding evolutionary routes to multicellular life. This study shows that environmental conditions promote multicellular group formation in green algae and that retention of daughter cells reliant on nitrogen availability promotes fitness in the lab and in natural lake systems in Sweden.},
  archive      = {J_NATEE},
  author       = {Cornwallis, Charlie K. and Svensson-Coelho, Maria and Lindh, Markus and Li, Qinyang and Stábile, Franca and Hansson, Lars-Anders and Rengefors, Karin},
  doi          = {10.1038/s41559-023-02044-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {889-902},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Single-cell adaptations shape evolutionary transitions to multicellularity in green algae},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Inadvertent human genomic bycatch and intentional capture
raise beneficial applications and ethical concerns with environmental
DNA. <em>NATEE</em>, <em>7</em>(6), 873–888. (<a
href="https://doi.org/10.1038/s41559-023-02056-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The field of environmental DNA (eDNA) is advancing rapidly, yet human eDNA applications remain underutilized and underconsidered. Broader adoption of eDNA analysis will produce many well-recognized benefits for pathogen surveillance, biodiversity monitoring, endangered and invasive species detection, and population genetics. Here we show that deep-sequencing-based eDNA approaches capture genomic information from humans (Homo sapiens) just as readily as that from the intended target species. We term this phenomenon human genetic bycatch (HGB). Additionally, high-quality human eDNA could be intentionally recovered from environmental substrates (water, sand and air), holding promise for beneficial medical, forensic and environmental applications. However, this also raises ethical dilemmas, from consent, privacy and surveillance to data ownership, requiring further consideration and potentially novel regulation. We present evidence that human eDNA is readily detectable from ‘wildlife’ environmental samples as human genetic bycatch, demonstrate that identifiable human DNA can be intentionally recovered from human-focused environmental sampling and discuss the translational and ethical implications of such findings. The recovery of human genomic data from environmental DNA samples raises ethical questions regarding consent, privacy, surveillance and data ownership, which will need to be grappled with as the environmental DNA field moves forward.},
  archive      = {J_NATEE},
  author       = {Whitmore, Liam and McCauley, Mark and Farrell, Jessica A. and Stammnitz, Maximilian R. and Koda, Samantha A. and Mashkour, Narges and Summers, Victoria and Osborne, Todd and Whilde, Jenny and Duffy, David J.},
  doi          = {10.1038/s41559-023-02056-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {873-888},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Inadvertent human genomic bycatch and intentional capture raise beneficial applications and ethical concerns with environmental DNA},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Species-specific traits mediate avian demographic responses
under past climate change. <em>NATEE</em>, <em>7</em>(6), 862–872. (<a
href="https://doi.org/10.1038/s41559-023-02055-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Anticipating species’ responses to environmental change is a pressing mission in biodiversity conservation. Despite decades of research investigating how climate change may affect population sizes, historical context is lacking, and the traits that mediate demographic sensitivity to changing climate remain elusive. We use whole-genome sequence data to reconstruct the demographic histories of 263 bird species over the past million years and identify networks of interacting morphological and life history traits associated with changes in effective population size (Ne) in response to climate warming and cooling. Our results identify direct and indirect effects of key traits representing dispersal, reproduction and survival on long-term demographic responses to climate change, thereby highlighting traits most likely to influence population responses to ongoing climate warming. Using avian trait data and genomic data, the authors infer whether changes in net effective population size over time in response to climate change are correlated with multiple morphological and life history traits; they find that larger-bodied, slower-reproducing species with limited dispersal capacity are most sensitive to changes in warming and cooling climates.},
  archive      = {J_NATEE},
  author       = {Germain, Ryan R. and Feng, Shaohong and Chen, Guangji and Graves, Gary R. and Tobias, Joseph A. and Rahbek, Carsten and Lei, Fumin and Fjeldså, Jon and Hosner, Peter A. and Gilbert, M. Thomas P. and Zhang, Guojie and Nogués-Bravo, David},
  doi          = {10.1038/s41559-023-02055-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {862-872},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Species-specific traits mediate avian demographic responses under past climate change},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Phenological shifts and mismatch with marine productivity
vary among pacific salmon species and populations. <em>NATEE</em>,
<em>7</em>(6), 852–861. (<a
href="https://doi.org/10.1038/s41559-023-02057-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global climate change is shifting the timing of life-cycle events, sometimes resulting in phenological mismatches between predators and prey. Phenological shifts and subsequent mismatches may be consistent across populations, or they could vary unpredictably across populations within the same species. For anadromous Pacific salmon (Oncorhynchus spp.), juveniles from thousands of locally adapted populations migrate from diverse freshwater habitats to the Pacific Ocean every year. Both the timing of freshwater migration and ocean arrival, relative to nearshore prey (phenological match/mismatch), can control marine survival and population dynamics. Here we examined phenological change of 66 populations across six anadromous Pacific salmon species throughout their range in western North America with the longest time series spanning 1951–2019. We show that different salmon species have different rates of phenological change but that there was substantial within-species variation that was not correlated with changing environmental conditions or geographic patterns. Moreover, outmigration phenologies have not tracked shifts in the timing of marine primary productivity, potentially increasing the frequency of future phenological mismatches. Understanding population responses to mismatches with prey are an important part of characterizing overall population-specific climate vulnerability. Shifts in species’ migration timing as a result of climate change can result in mismatched temporal overlap with their critical resources. Here the authors show that the magnitude and direction of shifts in juvenile Pacific salmon migration timing vary among species and populations, resulting in variable mismatch with marine productivity, which has implications for climate change vulnerability.},
  archive      = {J_NATEE},
  author       = {Wilson, Samantha M. and Moore, Jonathan W. and Ward, Eric J. and Kinsel, Clayton W. and Anderson, Joseph H. and Buehrens, Thomas W. and Carr-Harris, Charmaine N. and Cochran, Patrick C. and Davies, Trevor D. and Downen, Mark R. and Godbout, Lyse and Lisi, Peter J. and Litz, Marisa N. C. and Patterson, David A. and Selbie, Daniel T. and Sloat, Matthew R. and Suring, Erik J. and Tattam, Ian A. and Wyatt, Garth J.},
  doi          = {10.1038/s41559-023-02057-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {852-861},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Phenological shifts and mismatch with marine productivity vary among pacific salmon species and populations},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Glacier retreat reorganizes river habitats leaving refugia
for alpine invertebrate biodiversity poorly protected. <em>NATEE</em>,
<em>7</em>(6), 841–851. (<a
href="https://doi.org/10.1038/s41559-023-02061-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Alpine river biodiversity around the world is under threat from glacier retreat driven by rapid warming, yet our ability to predict the future distributions of specialist cold-water species is currently limited. Here we link future glacier projections, hydrological routing methods and species distribution models to quantify the changing influence of glaciers on population distributions of 15 alpine river invertebrate species across the entire European Alps, from 2020 to 2100. Glacial influence on rivers is projected to decrease steadily, with river networks expanding into higher elevations at a rate of 1% per decade. Species are projected to undergo upstream distribution shifts where glaciers persist but become functionally extinct where glaciers disappear completely. Several alpine catchments are predicted to offer climate refugia for cold-water specialists. However, present-day protected area networks provide relatively poor coverage of these future refugia, suggesting that alpine conservation strategies must change to accommodate the future effects of global warming. Organisms living at high elevations are particularly vulnerable to climate warming. Here the authors combine hydrological and glacier modelling with species distribution models to show how glacier retreat in the European Alps could impact the biodiversity and distributions of invertebrates in alpine rivers.},
  archive      = {J_NATEE},
  author       = {Wilkes, M. A. and Carrivick, J. L. and Castella, E. and Ilg, C. and Cauvy-Fraunié, S. and Fell, S. C. and Füreder, L. and Huss, M. and James, W. and Lencioni, V. and Robinson, C. and Brown, L. E.},
  doi          = {10.1038/s41559-023-02061-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {841-851},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Glacier retreat reorganizes river habitats leaving refugia for alpine invertebrate biodiversity poorly protected},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multitrophic arthropod diversity mediates tree diversity
effects on primary productivity. <em>NATEE</em>, <em>7</em>(6), 832–840.
(<a href="https://doi.org/10.1038/s41559-023-02049-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Forests sustain 80% of terrestrial biodiversity and provide essential ecosystem services. Biodiversity experiments have demonstrated that plant diversity correlates with both primary productivity and higher trophic diversity. However, whether higher trophic diversity can mediate the effects of plant diversity on productivity remains unclear. Here, using 5 years of data on aboveground herbivorous, predatory and parasitoid arthropods along with tree growth data within a large-scale forest biodiversity experiment in southeast China, we provide evidence of multidirectional enhancement among the diversity of trees and higher trophic groups and tree productivity. We show that the effects of experimentally increased tree species richness were consistently positive for species richness and abundance of herbivores, predators and parasitoids. Richness effects decreased as trophic levels increased for species richness and abundance of all trophic groups. Multitrophic species richness and abundance of arthropods were important mediators of plant diversity effects on tree productivity, suggesting that optimizing forest management for increased carbon capture can be more effective when the diversity of higher trophic groups is promoted in concert with that of trees. Using data from the BEF-China tree diversity experiment, the authors demonstrate that the diversity of arthropods is higher in plots with higher tree diversity and that the suppression of herbivores by enemy arthropods could be a potential mechanism through which higher tree diversity promotes productivity.},
  archive      = {J_NATEE},
  author       = {Li, Yi and Schmid, Bernhard and Schuldt, Andreas and Li, Shan and Wang, Ming-Qiang and Fornoff, Felix and Staab, Michael and Guo, Peng-Fei and Anttonen, Perttu and Chesters, Douglas and Bruelheide, Helge and Zhu, Chao-Dong and Ma, Keping and Liu, Xiaojuan},
  doi          = {10.1038/s41559-023-02049-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {832-840},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Multitrophic arthropod diversity mediates tree diversity effects on primary productivity},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mass production of unvouchered records fails to represent
global biodiversity patterns. <em>NATEE</em>, <em>7</em>(6), 816–831.
(<a href="https://doi.org/10.1038/s41559-023-02047-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ever-increasing human footprint even in very remote places on Earth has inspired efforts to document biodiversity vigorously in case organisms go extinct. However, the data commonly gathered come from either primary voucher specimens in a natural history collection or from direct field observations that are not traceable to tangible material in a museum or herbarium. Although both datasets are crucial for assessing how anthropogenic drivers affect biodiversity, they have widespread coverage gaps and biases that may render them inefficient in representing patterns of biodiversity. Using a large global dataset of around 1.9 billion occurrence records of terrestrial plants, butterflies, amphibians, birds, reptiles and mammals, we quantify coverage and biases of expected biodiversity patterns by voucher and observation records. We show that the mass production of observation records does not lead to higher coverage of expected biodiversity patterns but is disproportionately biased toward certain regions, clades, functional traits and time periods. Such coverage patterns are driven by the ease of accessibility to air and ground transportation, level of security and extent of human modification at each sampling site. Conversely, voucher records are vastly infrequent in occurrence data but in the few places where they are sampled, showed relative congruence with expected biodiversity patterns for all dimensions. The differences in coverage and bias by voucher and observation records have important implications on the utility of these records for research in ecology, evolution and conservation research. An interrogation of almost 2 billion occurrence records for terrestrial plants and animals derived from either primary voucher specimens or direct observations, including citizen-science data, reveals differences in their coverage of global biodiversity patterns.},
  archive      = {J_NATEE},
  author       = {Daru, Barnabas H. and Rodriguez, Jordan},
  doi          = {10.1038/s41559-023-02047-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {816-831},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mass production of unvouchered records fails to represent global biodiversity patterns},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evolution and implications of de novo genes in humans.
<em>NATEE</em>, <em>7</em>(6), 804–815. (<a
href="https://doi.org/10.1038/s41559-023-02014-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genes and translated open reading frames (ORFs) that emerged de novo from previously non-coding sequences provide species with opportunities for adaptation. When aberrantly activated, some human-specific de novo genes and ORFs have disease-promoting properties—for instance, driving tumour growth. Thousands of putative de novo coding sequences have been described in humans, but we still do not know what fraction of those ORFs has readily acquired a function. Here, we discuss the challenges and controversies surrounding the detection, mechanisms of origin, annotation, validation and characterization of de novo genes and ORFs. Through manual curation of literature and databases, we provide a thorough table with most de novo genes reported for humans to date. We re-evaluate each locus by tracing the enabling mutations and list proposed disease associations, protein characteristics and supporting evidence for translation and protein detection. This work will support future explorations of de novo genes and ORFs in humans. This Review discusses challenges with detection and characterization of de novo genes and their mechanism of origin, and includes a curated list of de novo genes reported for humans.},
  archive      = {J_NATEE},
  author       = {Broeils, Luuk A. and Ruiz-Orera, Jorge and Snel, Berend and Hubner, Norbert and van Heesch, Sebastiaan},
  doi          = {10.1038/s41559-023-02014-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {804-815},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolution and implications of de novo genes in humans},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Specimens trump field observations in capturing biodiversity
trends. <em>NATEE</em>, <em>7</em>(6), 802–803. (<a
href="https://doi.org/10.1038/s41559-023-02051-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Efforts to document biodiversity have created large species datasets, but new research shows that field observations are biased towards particular regions, clades, traits and time periods, and do not accurately represent global biodiversity patterns. Although specimens are only infrequently preserved in natural history collections, they show relative congruence with expected biodiversity patterns and are vital for ecological research.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02051-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {802-803},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Specimens trump field observations in capturing biodiversity trends},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Archaic hominin traits through the splicing lens.
<em>NATEE</em>, <em>7</em>(6), 800–801. (<a
href="https://doi.org/10.1038/s41559-023-02045-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine-learning-based prediction of splicing in extinct hominin species highlights the effect of natural selection on splice-altering variants and reveals phenotypic differences with modern humans.},
  archive      = {J_NATEE},
  author       = {Rotival, Maxime},
  doi          = {10.1038/s41559-023-02045-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {800-801},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Archaic hominin traits through the splicing lens},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Adaptive physiology drives ageing plasticity in locusts.
<em>NATEE</em>, <em>7</em>(6), 798–799. (<a
href="https://doi.org/10.1038/s41559-023-02052-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Gene–environment interactions have been found to shape ageing plasticity in the muscle tissue of migratory locusts through adaptive changes in lipid metabolic processes.},
  archive      = {J_NATEE},
  author       = {Li, Xiaotong and Karpac, Jason},
  doi          = {10.1038/s41559-023-02052-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {798-799},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Adaptive physiology drives ageing plasticity in locusts},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The ethics of human sequences in environmental samples.
<em>NATEE</em>, <em>7</em>(6), 796–797. (<a
href="https://doi.org/10.1038/s41559-023-02072-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human genetic material can be inadvertently captured and enriched from environmental DNA samples. This has both legal and ethical implications for future research.},
  archive      = {J_NATEE},
  author       = {Ram, Natalie},
  doi          = {10.1038/s41559-023-02072-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {796-797},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The ethics of human sequences in environmental samples},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Foregone uptake. <em>NATEE</em>, <em>7</em>(6), 795. (<a
href="https://doi.org/10.1038/s41559-023-02067-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-023-02067-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {795},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Foregone uptake},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Deforestation linked to respiratory health. <em>NATEE</em>,
<em>7</em>(6), 794. (<a
href="https://doi.org/10.1038/s41559-023-02065-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {McKay, Alexa},
  doi          = {10.1038/s41559-023-02065-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {794},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Deforestation linked to respiratory health},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Global inequities in local science. <em>NATEE</em>,
<em>7</em>(6), 793. (<a
href="https://doi.org/10.1038/s41559-023-02064-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Bhaumik, Vaishali},
  doi          = {10.1038/s41559-023-02064-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {793},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Global inequities in local science},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Phenology meets extreme events. <em>NATEE</em>,
<em>7</em>(6), 792. (<a
href="https://doi.org/10.1038/s41559-023-02043-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-023-02043-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {792},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Phenology meets extreme events},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Eleanor jane sterling (1960–2023). <em>NATEE</em>,
<em>7</em>(6), 790–791. (<a
href="https://doi.org/10.1038/s41559-023-02054-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A visionary conservationist, deep thinker and versatile scholar who challenged the field of conservation to reflect on itself and its purpose in inclusive ways.},
  archive      = {J_NATEE},
  author       = {Betley, Erin and Blair, Mary E. and Cullman, Georgina},
  doi          = {10.1038/s41559-023-02054-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {790-791},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Eleanor jane sterling (1960–2023)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Biological anthropology must reassess museum collections for
a more ethical future. <em>NATEE</em>, <em>7</em>(6), 786–789. (<a
href="https://doi.org/10.1038/s41559-023-02036-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With growing attention on the remains of people held by museums and universities around the world, we outline ethical considerations that researchers working with human remains in anthropology must bear in mind.},
  archive      = {J_NATEE},
  author       = {Stantis, Chris and de la Cova, Carlina and Lippert, Dorothy and Sholts, Sabrina B.},
  doi          = {10.1038/s41559-023-02036-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {786-789},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Biological anthropology must reassess museum collections for a more ethical future},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Social considerations are crucial to success in implementing
the 30×30 global conservation target. <em>NATEE</em>, <em>7</em>(6),
784–785. (<a href="https://doi.org/10.1038/s41559-023-02048-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Sandbrook, Chris and Albury-Smith, Shenique and Allan, James R. and Bhola, Nina and Bingham, Heather C. and Brockington, Dan and Byaruhanga, Achilles B. and Fajardo, Javier and Fitzsimons, James and Franks, Phil and Fleischman, Forrest and Frechette, Alain and Kakuyo, Kagumaho and Kaptoyo, Edna and Kuemmerle, Tobias and Kalunda, Pauline Nantongo and Nuvunga, Milagre and O’Donnell, Brian and Onyai, Fred and Pfeifer, Marion and Pritchard, Rose and Ramos, Ameyali and Rao, Madhu and Ryan, Casey M. and Shyamsundar, Priya and Tauli, Josefa and Tumusiime, David Mwesigye and Vilaça, Mônica and Watmough, Gary R. and Worsdell, Thomas and Zaehringer, Julie G.},
  doi          = {10.1038/s41559-023-02048-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {784-785},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Social considerations are crucial to success in implementing the 30×30 global conservation target},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multiple ocean threats. <em>NATEE</em>, <em>7</em>(6), 783.
(<a href="https://doi.org/10.1038/s41559-023-02099-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plastic pollution has rapidly risen to the top of public and policy discourse on the environment. For World Environment Day on 5 June and World Oceans Day on 8 June, we reflect on its intersection with other ocean threats from biodiversity loss and climate change.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02099-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {6},
  number       = {6},
  pages        = {783},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Multiple ocean threats},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A social niche breadth score reveals niche range strategies
of generalists and specialists. <em>NATEE</em>, <em>7</em>(5), 768–781.
(<a href="https://doi.org/10.1038/s41559-023-02027-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generalists can survive in many environments, whereas specialists are restricted to a single environment. Although a classical concept in ecology, niche breadth has remained challenging to quantify for microorganisms because it depends on an objective definition of the environment. Here, by defining the environment of a microorganism as the community it resides in, we integrated information from over 22,000 environmental sequencing samples to derive a quantitative measure of the niche, which we call social niche breadth. At the level of genera, we explored niche range strategies throughout the prokaryotic tree of life. We found that social generalists include opportunists that stochastically dominate local communities, whereas social specialists are stable but low in abundance. Social generalists have a more diverse and open pan-genome than social specialists, but we found no global correlation between social niche breadth and genome size. Instead, we observed two distinct evolutionary strategies, whereby specialists have relatively small genomes in habitats with low local diversity, but relatively large genomes in habitats with high local diversity. Together, our analysis shines data-driven light on microbial niche range strategies. Defining the niche of a microorganism is more difficult than doing so for a macroorganism. Here the authors define a microorganism’s niche based on the communities of other microorganisms it is found with; they apply this social niche breadth metric to reveal the ecological and genomic correlates of microbial specialism versus generalism.},
  archive      = {J_NATEE},
  author       = {von Meijenfeldt, F. A. Bastiaan and Hogeweg, Paulien and Dutilh, Bas E.},
  doi          = {10.1038/s41559-023-02027-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {768-781},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A social niche breadth score reveals niche range strategies of generalists and specialists},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fortuitously compatible protein surfaces primed allosteric
control in cyanobacterial photoprotection. <em>NATEE</em>,
<em>7</em>(5), 756–767. (<a
href="https://doi.org/10.1038/s41559-023-02018-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Highly specific interactions between proteins are a fundamental prerequisite for life, but how they evolve remains an unsolved problem. In particular, interactions between initially unrelated proteins require that they evolve matching surfaces. It is unclear whether such surface compatibilities can only be built by selection in small incremental steps, or whether they can also emerge fortuitously. Here, we used molecular phylogenetics, ancestral sequence reconstruction and biophysical characterization of resurrected proteins to retrace the evolution of an allosteric interaction between two proteins that act in the cyanobacterial photoprotection system. We show that this interaction between the orange carotenoid protein (OCP) and its unrelated regulator, the fluorescence recovery protein (FRP), evolved when a precursor of FRP was horizontally acquired by cyanobacteria. FRP’s precursors could already interact with and regulate OCP even before these proteins first encountered each other in an ancestral cyanobacterium. The OCP–FRP interaction exploits an ancient dimer interface in OCP, which also predates the recruitment of FRP into the photoprotection system. Together, our work shows how evolution can fashion complex regulatory systems easily out of pre-existing components. Molecular phylogenetics, ancestral sequence reconstruction and biophysical protein characterization are used to investigate the interaction between the orange carotenoid protein and its unrelated regulator, the fluorescence recovery protein (FRP). This interaction evolved when a precursor of FRP was horizontally acquired by cyanobacteria.},
  archive      = {J_NATEE},
  author       = {Steube, Niklas and Moldenhauer, Marcus and Weiland, Paul and Saman, Dominik and Kilb, Alexandra and Ramírez Rojas, Adán A. and Garg, Sriram G. and Schindler, Daniel and Graumann, Peter L. and Benesch, Justin L. P. and Bange, Gert and Friedrich, Thomas and Hochberg, Georg K. A.},
  doi          = {10.1038/s41559-023-02018-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {756-767},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Fortuitously compatible protein surfaces primed allosteric control in cyanobacterial photoprotection},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Multiple intermolecular interactions facilitate rapid
evolution of essential genes. <em>NATEE</em>, <em>7</em>(5), 745–755.
(<a href="https://doi.org/10.1038/s41559-023-02029-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Essential genes are commonly assumed to function in basic cellular processes and to change slowly. However, it remains unclear whether all essential genes are similarly conserved or if their evolutionary rates can be accelerated by specific factors. To address these questions, we replaced 86 essential genes of Saccharomyces cerevisiae with orthologues from four other species that diverged from S. cerevisiae about 50, 100, 270 and 420 Myr ago. We identify a group of fast-evolving genes that often encode subunits of large protein complexes, including anaphase-promoting complex/cyclosome (APC/C). Incompatibility of fast-evolving genes is rescued by simultaneously replacing interacting components, suggesting it is caused by protein co-evolution. Detailed investigation of APC/C further revealed that co-evolution involves not only primary interacting proteins but also secondary ones, suggesting the evolutionary impact of epistasis. Multiple intermolecular interactions in protein complexes may provide a microenvironment facilitating rapid evolution of their subunits. Replacing 86 essential genes in Saccharomyces cerevisiae with orthologues from species with different diverging times, the authors show that intermolecular epistasis plays a key role in their evolution and that conserved physiological functions are maintained by co-evolution of interacting components.},
  archive      = {J_NATEE},
  author       = {Lai, Huei-Yi and Yu, Yen-Hsin and Jhou, Yu-Ting and Liao, Chia-Wei and Leu, Jun-Yi},
  doi          = {10.1038/s41559-023-02029-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {745-755},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Multiple intermolecular interactions facilitate rapid evolution of essential genes},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). An ancient metalloenzyme evolves through metal preference
modulation. <em>NATEE</em>, <em>7</em>(5), 732–744. (<a
href="https://doi.org/10.1038/s41559-023-02012-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evolution creates functional diversity of proteins, the essential building blocks of all biological systems. However, studies of natural proteins sampled across the tree of life and evaluated in a single experimental system are lacking. Almost half of enzymes require metals, and metalloproteins tend to optimally utilize the physicochemical properties of a specific metal co-factor. Life must adapt to changes in metal bioavailability, including those during the transition from anoxic to oxic Earth or pathogens’ exposure to nutritional immunity. These changes can challenge the ability of metalloenzymes to maintain activity, presumptively driving their evolution. Here we studied metal-preference evolution within the natural diversity of the iron/manganese superoxide dismutase (SodFM) family of reactive oxygen species scavengers. We identified and experimentally verified residues with conserved roles in determining metal preference that, when combined with an understanding of the protein’s evolutionary history, improved prediction of metal utilization across the five SodFM subfamilies defined herein. By combining phylogenetics, biochemistry and structural biology, we demonstrate that SodFM metal utilization can be evolutionarily fine tuned by sliding along a scale between perfect manganese and iron specificities. Over the history of life, SodFM metal preference has been modulated multiple independent times within different evolutionary and ecological contexts, and can be changed within short evolutionary timeframes. The iron/manganese superoxide dismutases constitute a family of metalloenzymes that function as scavengers of reactive oxygen species. Here the authors use phylogenetics, biochemistry and structural biology to show how differential metal preference for Fe and Mn has been modulated throughout iron/manganese superoxide dismutases evolution.},
  archive      = {J_NATEE},
  author       = {Sendra, K. M. and Barwinska-Sendra, A. and Mackenzie, E. S. and Baslé, A. and Kehl-Fie, T. E. and Waldron, K. J.},
  doi          = {10.1038/s41559-023-02012-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {732-744},
  shortjournal = {Nat. Ecol. Evol},
  title        = {An ancient metalloenzyme evolves through metal preference modulation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Within-host adaptation alters priority effects within the
tomato phyllosphere microbiome. <em>NATEE</em>, <em>7</em>(5), 725–731.
(<a href="https://doi.org/10.1038/s41559-023-02040-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To predict the composition and function of ecological communities over time, it is essential to understand how in situ evolution alters priority effects between resident and invading species. Phyllosphere microbial communities are a useful model system to explore priority effects because the system is clearly spatially delineated and can be manipulated experimentally. We conducted an experimental evolution study with tomato plants and the early-colonizing bacterium species Pantoea dispersa, exploring priority effects when P. dispersa was introduced before, simultaneously with or after competitor species. P. dispersa rapidly evolved to invade a new niche within the plant tissue and altered its ecological interactions with other members of the plant microbiome and its effect on the host. Prevailing models have assumed that adaptation primarily improves the efficiency of resident species within their existing niches, yet in our study system, the resident species expanded its niche instead. This finding suggests potential limitations to the application of existing ecological theory to microbial communities. In microbial community assembly, species that establish earlier often have an advantage. Here, the authors explore these priority effects in the tomato plant-associated microbiome, showing that experimental evolution selecting for host colonization alters priority effects among competitors.},
  archive      = {J_NATEE},
  author       = {Debray, Reena and Conover, Asa and Zhang, Xuening and Dewald-Wang, Emily A. and Koskella, Britt},
  doi          = {10.1038/s41559-023-02040-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {725-731},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Within-host adaptation alters priority effects within the tomato phyllosphere microbiome},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Annotation-free discovery of functional groups in microbial
communities. <em>NATEE</em>, <em>7</em>(5), 716–724. (<a
href="https://doi.org/10.1038/s41559-023-02021-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Recent studies have shown that microbial communities are composed of groups of functionally cohesive taxa whose abundance is more stable and better-associated with metabolic fluxes than that of any individual taxon. However, identifying these functional groups in a manner that is independent of error-prone functional gene annotations remains a major open problem. Here we tackle this structure–function problem by developing a novel unsupervised approach that coarse-grains taxa into functional groups, solely on the basis of the patterns of statistical variation in species abundances and functional read-outs. We demonstrate the power of this approach on three distinct datasets. On data of replicate microcosms with heterotrophic soil bacteria, our unsupervised algorithm recovered experimentally validated functional groups that divide metabolic labour and remain stable despite large variation in species composition. When leveraged against the ocean microbiome data, our approach discovered a functional group that combines aerobic and anaerobic ammonia oxidizers whose summed abundance tracks closely with nitrate concentrations in the water column. Finally, we show that our framework can enable the detection of species groups that are probably responsible for the production or consumption of metabolites abundant in animal gut microbiomes, serving as a hypothesis-generating tool for mechanistic studies. Overall, this work advances our understanding of structure–function relationships in complex microbiomes and provides a powerful approach to discover functional groups in an objective and systematic manner. An unsupervised, annotation-free method is developed that can identify microbial functional groups on the basis of variation in microbiome data and environmental variables. Here, the authors demonstrate its application in several different datasets including the Tara oceans microbiome and animal gut microbiomes.},
  archive      = {J_NATEE},
  author       = {Shan, Xiaoyu and Goyal, Akshit and Gregor, Rachel and Cordero, Otto X.},
  doi          = {10.1038/s41559-023-02021-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {716-724},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Annotation-free discovery of functional groups in microbial communities},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Incentives and barriers to private finance for forest and
landscape restoration. <em>NATEE</em>, <em>7</em>(5), 707–715. (<a
href="https://doi.org/10.1038/s41559-023-02037-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Increased private finance can accelerate forest and landscape restoration globally. Here we conduct semi-structured interviews with asset managers, corporations and restoration finance experts to examine incentives and barriers to private restoration finance. Next, we assess what type of restoration projects and regions appeal to different private funders and how current financial barriers can be overcome. We show that market incentives for corporations include meeting net-emission-reduction commitments, impact and sustainable branding opportunities, and promotion of sustainability in supply chains. Conversely, asset managers face stronger barriers to investing in restoration as it is deemed a high-risk, unknown investment with low profitability. We find that investment finance biases towards restoration projects in low-risk areas and corporate finance towards areas with business presence. Both private finance types tend to omit projects focusing on natural regeneration. Through expanded and diversified markets for restoration benefits, strong public policy support and new financial instruments, private finance for restoration can be scaled for a wider variety of restoration projects in more diverse geographical contexts. Interest in private financing of restoration is growing, but funding remains low. Semi-structured interviews with financial actors and restoration finance experts show that there are some market incentives for private actors to finance restoration, but policy mandates are needed to scale private finance and ensure it is steered towards ecologically sound and equitable objectives},
  archive      = {J_NATEE},
  author       = {Löfqvist, Sara and Garrett, Rachael D. and Ghazoul, Jaboury},
  doi          = {10.1038/s41559-023-02037-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {707-715},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Incentives and barriers to private finance for forest and landscape restoration},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Current levels of microplastic pollution impact wild seabird
gut microbiomes. <em>NATEE</em>, <em>7</em>(5), 698–706. (<a
href="https://doi.org/10.1038/s41559-023-02013-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Microplastics contaminate environments worldwide and are ingested by numerous species, whose health is affected in multiple ways. A key dimension of health that may be affected is the gut microbiome, but these effects are relatively unexplored. Here, we investigated if microplastics are associated with changes in proventricular and cloacal microbiomes in two seabird species that chronically ingest microplastics: northern fulmars and Cory’s shearwaters. The amount of microplastics in the gut was significantly correlated with gut microbial diversity and composition: microplastics were associated with decreases in commensal microbiota and increases in (zoonotic) pathogens and antibiotic-resistant and plastic-degrading microbes. These results illustrate that environmentally relevant microplastic concentrations and mixtures are associated with changes in gut microbiomes in wild seabirds. Consuming microplastics is known to harm marine wildlife in several ways, but effects on the microbiome are understudied. Here the authors demonstrate that two species of wild seabirds with larger amounts of microplastic in their guts had fewer commensal gut microbial species but more pathogens.},
  archive      = {J_NATEE},
  author       = {Fackelmann, Gloria and Pham, Christopher K. and Rodríguez, Yasmina and Mallory, Mark L. and Provencher, Jennifer F. and Baak, Julia E. and Sommer, Simone},
  doi          = {10.1038/s41559-023-02013-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {698-706},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Current levels of microplastic pollution impact wild seabird gut microbiomes},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Extent and reproduction of coastal species on plastic debris
in the north pacific subtropical gyre. <em>NATEE</em>, <em>7</em>(5),
687–697. (<a href="https://doi.org/10.1038/s41559-023-01997-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We show that the high seas are colonized by a diverse array of coastal species, which survive and reproduce in the open ocean, contributing strongly to its floating community composition. Analysis of rafting plastic debris in the eastern North Pacific Subtropical Gyre revealed 37 coastal invertebrate taxa, largely of Western Pacific origin, exceeding pelagic taxa richness by threefold. Coastal taxa, including diverse taxonomic groups and life history traits, occurred on 70.5% of debris items. Most coastal taxa possessed either direct development or asexual reproduction, possibly facilitating long-term persistence on rafts. Our results suggest that the historical lack of available substrate limited the colonization of the open ocean by coastal species, rather than physiological or ecological constraints as previously assumed. It appears that coastal species persist now in the open ocean as a substantial component of a neopelagic community sustained by the vast and expanding sea of plastic debris. Analysing plastic debris in the Great Pacific Garbage Patch, the authors identify three times more coastal invertebrate taxa than pelagic taxa, along with evidence that coastal taxa are reproducing while rafting on plastic.},
  archive      = {J_NATEE},
  author       = {Haram, Linsey E. and Carlton, James T. and Centurioni, Luca and Choong, Henry and Cornwell, Brendan and Crowley, Mary and Egger, Matthias and Hafner, Jan and Hormann, Verena and Lebreton, Laurent and Maximenko, Nikolai and McCuller, Megan and Murray, Cathryn and Par, Jenny and Shcherbina, Andrey and Wright, Cynthia and Ruiz, Gregory M.},
  doi          = {10.1038/s41559-023-01997-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {687-697},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Extent and reproduction of coastal species on plastic debris in the north pacific subtropical gyre},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Inbreeding depression explains killer whale population
dynamics. <em>NATEE</em>, <em>7</em>(5), 675–686. (<a
href="https://doi.org/10.1038/s41559-023-01995-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the factors that cause endangered populations to either grow or decline is crucial for preserving biodiversity. Conservation efforts often address extrinsic threats, such as environmental degradation and overexploitation, that can limit the recovery of endangered populations. Genetic factors such as inbreeding depression can also affect population dynamics but these effects are rarely measured in the wild and thus often neglected in conservation efforts. Here we show that inbreeding depression strongly influences the population dynamics of an endangered killer whale population, despite genomic signatures of purging of deleterious alleles via natural selection. We find that the ‘Southern Residents’, which are currently endangered despite nearly 50 years of conservation efforts, exhibit strong inbreeding depression for survival. Our population models suggest that this inbreeding depression limits population growth and predict further decline if the population remains genetically isolated and typical environmental conditions continue. The Southern Residents also had more inferred homozygous deleterious alleles than three other, growing, populations, further suggesting that inbreeding depression affects population fitness. These results demonstrate that inbreeding depression can substantially limit the recovery of endangered populations. Conservation actions focused only on extrinsic threats may therefore fail to account for key intrinsic genetic factors that also limit population growth. Genomic and demographic analyses of the ‘Southern Resident’ killer whales in the North Pacific find that strong inbreeding depression is inhibiting growth of this small and isolated population. The findings help to explain why this group of whales is still declining despite 50 years of conservation efforts.},
  archive      = {J_NATEE},
  author       = {Kardos, Marty and Zhang, Yaolei and Parsons, Kim M. and A, Yunga and Kang, Hui and Xu, Xun and Liu, Xin and Matkin, Craig O. and Zhang, Peijun and Ward, Eric J. and Hanson, M. Bradley and Emmons, Candice and Ford, Michael J. and Fan, Guangyi and Li, Songhai},
  doi          = {10.1038/s41559-023-01995-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {675-686},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Inbreeding depression explains killer whale population dynamics},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A middle ordovician burgess shale-type fauna from castle
bank, wales (UK). <em>NATEE</em>, <em>7</em>(5), 666–674. (<a
href="https://doi.org/10.1038/s41559-023-02038-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Burgess Shale-type faunas are critical to our understanding of animal evolution during the Cambrian, giving an unrivalled view of the morphology of ancient organisms and the ecology of the earliest animal-dominated communities. Rare examples in Lower Ordovician strata such as the Fezouata Biota illustrate the subsequent evolution of ecosystems but only from before the main phase of the Great Ordovician Biodiversification Event. Later Ordovician Konservat-Lagerstätten are not directly comparable with the Burgess Shale-type faunas as they do not represent diverse, open-shelf communities, limiting our ability to track ecological development through the critical Ordovician biodiversification interval. Here we present the Castle Bank fauna: a highly diverse Middle Ordovician Burgess Shale-type fauna from Wales (UK) that is directly comparable with the Burgess Shale and Chengjiang biotas in palaeoenvironment and preservational style. The deposit includes animals with morphologies similar to the iconic Cambrian taxa Opabinia, Yohoia and Wiwaxia, combined with early examples of more derived groups such as barnacles. Many taxa such as kinorhynchs show the small sizes typical of modern faunas, illustrating post-Cambrian miniaturization. Castle Bank provides a new perspective on early animal evolution, revealing the next chapter in ecosystem development following the Chengjiang, Burgess Shale and Fezouata biotas. The authors report a highly diverse Middle Ordovician Burgess Shale-type fauna from Wales (UK) that compares with the Burgess Shale and Chengjiang biotas in palaeoenvironment and preservational style.},
  archive      = {J_NATEE},
  author       = {Botting, Joseph P. and Muir, Lucy A. and Pates, Stephen and McCobb, Lucy M. E. and Wallet, Elise and Willman, Sebastian and Zhang, Yuandong and Ma, Junye},
  doi          = {10.1038/s41559-023-02038-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {666-674},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A middle ordovician burgess shale-type fauna from castle bank, wales (UK)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Little transparency and equity in scientific awards for
early- and mid-career researchers in ecology and evolution.
<em>NATEE</em>, <em>7</em>(5), 655–665. (<a
href="https://doi.org/10.1038/s41559-023-02028-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Scientific awards can shape scientific careers, helping to secure jobs and grants, but can also contribute to the lack of diversity at senior levels and in the elite networks of scientists. To assess the status quo and historical trends, we evaluated ‘best researcher’ awards and ‘best paper’ early- and mid-career awards from broad-scope international journals and societies in ecology and evolution. Specifically, we collated information on eligibility rules, assessment criteria and potential gender bias. Our results reveal that, overall, few awards foster equitable access and assessment. Although many awards now explicitly allow extensions of the eligibility period for substantial career interruptions, there is a general lack of transparency in terms of assessment and consideration of other differences in access to opportunities and resources among junior researchers. Strikingly, open science practices were mentioned and valued in only one award. By highlighting instances of desirable award characteristics, we hope this work will nudge award committees to shift from simple but non-equitable award policies and practices towards strategies enhancing inclusivity and diversity. Such a shift would benefit not only those at the early- and mid-career stages but the whole research community. It is also an untapped opportunity to reward open science practices, promoting transparent and robust science. An analysis of the eligibility rules, assessment criteria and potential gender bias in best researcher and best paper awards from broad-scope ecology and evolution journals and societies shows a lack of assessment transparency, few attempts to foster equitable access and minimal consideration of open science.},
  archive      = {J_NATEE},
  author       = {Lagisz, Malgorzata and Aich, Upama and Amin, Bawan and Rutkowska, Joanna and Sánchez-Mercado, Ada and Lara, Carlos Esteban and Nakagawa, Shinichi},
  doi          = {10.1038/s41559-023-02028-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {655-665},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Little transparency and equity in scientific awards for early- and mid-career researchers in ecology and evolution},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Accidental surface compatibility paved the way for a new
molecular interaction. <em>NATEE</em>, <em>7</em>(5), 653–654. (<a
href="https://doi.org/10.1038/s41559-023-02031-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In cyanobacteria, the interaction between an orange carotenoid protein and its allosteric regulator evolved when a horizontal gene transfer event first brought the two proteins together. However, the surface compatibility between the proteins had already emerged. This finding implies that specific protein–protein interactions can evolve without the action of direct natural selection.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02031-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {653-654},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Accidental surface compatibility paved the way for a new molecular interaction},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Identifying microbial guilds on the basis of ecological
patterns. <em>NATEE</em>, <em>7</em>(5), 651–652. (<a
href="https://doi.org/10.1038/s41559-023-02030-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ensemble quotient optimization (EQO) uses patterns of variation in species abundance and ecosystem functions across microbial communities to identify microbial guilds in a systematic, objective manner. EQO is robust in recovering functional groups in soil, ocean and animal-gut microbial communities.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02030-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {651-652},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Identifying microbial guilds on the basis of ecological patterns},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The social dimension of microbial niches. <em>NATEE</em>,
<em>7</em>(5), 649–650. (<a
href="https://doi.org/10.1038/s41559-023-02020-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For a long time, the ecological niche concept was less popular for microbes than for other organisms. A new proxy for the ecological niche breadth of a microorganism, based on the variability of the communities with which it associates, enables investigation of the correlates of being a social generalist or social specialist.},
  archive      = {J_NATEE},
  author       = {Muller, Emilie E. L.},
  doi          = {10.1038/s41559-023-02020-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {649-650},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The social dimension of microbial niches},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Inbreeding threatens iconic killer whales. <em>NATEE</em>,
<em>7</em>(5), 647–648. (<a
href="https://doi.org/10.1038/s41559-023-02024-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unlike other North Pacific killer whales, Southern Resident killer whales have failed to thrive despite decades of conservation. Genomics combined with long-term observational records reveal inbreeding depression as a compelling explanation.},
  archive      = {J_NATEE},
  author       = {Robinson, Jacqueline A.},
  doi          = {10.1038/s41559-023-02024-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {647-648},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Inbreeding threatens iconic killer whales},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Bridged knowledge. <em>NATEE</em>, <em>7</em>(5), 646. (<a
href="https://doi.org/10.1038/s41559-023-02039-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Goymer, Patrick},
  doi          = {10.1038/s41559-023-02039-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {646},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Bridged knowledge},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Between ice and ocean. <em>NATEE</em>, <em>7</em>(5), 645.
(<a href="https://doi.org/10.1038/s41559-023-02033-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-023-02033-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {645},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Between ice and ocean},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Old fish debate settled. <em>NATEE</em>, <em>7</em>(5),
644. (<a href="https://doi.org/10.1038/s41559-023-02016-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-023-02016-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {644},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Old fish debate settled},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Jianwu (jim) tang (1970–2023). <em>NATEE</em>,
<em>7</em>(5), 642–643. (<a
href="https://doi.org/10.1038/s41559-023-02032-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ecosystem ecologist who made fundamental contributions to carbon cycle science and advocated for the next generation of scientists},
  archive      = {J_NATEE},
  author       = {Yang, Xi and Baldocchi, Dennis and Yang, Hualei},
  doi          = {10.1038/s41559-023-02032-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {642-643},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Jianwu (Jim) tang (1970–2023)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A national assessment of nature. <em>NATEE</em>,
<em>7</em>(5), 640–641. (<a
href="https://doi.org/10.1038/s41559-023-02015-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The US government has launched a national nature assessment, to be completed by 2026. This assessment is designed to take stock of changes in the country’s lands, waters, biodiversity, ecosystems and the benefits they provide. We spoke to Heather Tallis about the aims of the project, and how individuals can contribute to its development. Heather is the Acting Director of the assessment for the US Global Change Research Program, as well as the Assistant Director for Biodiversity and Conservation Sciences in the White House Office of Science and Technology Policy.},
  archive      = {J_NATEE},
  author       = {Turner, Marian and Tallis, Heather},
  doi          = {10.1038/s41559-023-02015-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {640-641},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A national assessment of nature},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). How to diversify your department’s seminar series.
<em>NATEE</em>, <em>7</em>(5), 637–639. (<a
href="https://doi.org/10.1038/s41559-023-02026-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Seminar series are a key part of academic culture. We present practices that are aimed at increasing the diversity of seminar speakers, and thus broadening associated opportunities to more members of the ecology and evolutionary biology research community.},
  archive      = {J_NATEE},
  author       = {Del Carpio, Christina A. and Ford, Ashlyn T. and Lowell, Eva S. Horna and Ochoa, Marissa E. and Speck, Hayden P.},
  doi          = {10.1038/s41559-023-02026-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {637-639},
  shortjournal = {Nat. Ecol. Evol},
  title        = {How to diversify your department’s seminar series},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mapping our reliance on the tropics can reveal the roots of
the anthropocene. <em>NATEE</em>, <em>7</em>(5), 632–636. (<a
href="https://doi.org/10.1038/s41559-023-01998-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Projecting and managing the feedback between tropical deforestation and global Earth system dynamics, and identifying potential critical thresholds or tipping points, will be key to our species’ future on this planet. By understanding the major historical processes that underpin the origins of this interaction, and bringing natural and social systems together in interdisciplinary models, we can evaluate the degree to which past human impacts on tropical forests resulted in observable planetary ramifications that have left legacies for the twenty-first century and beyond.},
  archive      = {J_NATEE},
  author       = {Roberts, Patrick and Kaplan, Jed O. and Findley, David Max and Hamilton, Rebecca and Caetano-Andrade, Victor Lery and Amano, Noel and Kay, Andrea U. and Renn, Jürgen and Winkelmann, Ricarda},
  doi          = {10.1038/s41559-023-01998-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {632-636},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mapping our reliance on the tropics can reveal the roots of the anthropocene},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Five essentials for area-based biodiversity protection.
<em>NATEE</em>, <em>7</em>(5), 630–631. (<a
href="https://doi.org/10.1038/s41559-023-02023-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The international commitment to protect 30% of the world’s surface by 2030 is laudable and necessary, but scientists must now work with governments and other groups to ensure success in its implementation and evaluation, by using inclusive and evidence-led approaches, argues Alexandre Antonelli, Director of Science at the Royal Botanic Gardens, Kew and author of The Hidden Universe: Adventures in Biodiversity.},
  archive      = {J_NATEE},
  author       = {Antonelli, Alexandre},
  doi          = {10.1038/s41559-023-02023-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {630-631},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Five essentials for area-based biodiversity protection},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The last afromontane forests in angola are threatened by
fires. <em>NATEE</em>, <em>7</em>(5), 628–629. (<a
href="https://doi.org/10.1038/s41559-023-02025-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Powell, Luke L. and Vaz Pinto, Pedro and Mills, Michael S. L. and Baptista, Ninda L. and Costa, Kerllen and Dijkstra, Klaas-Douwe B. and Gomes, Amândio Luís and Guedes, Patricia and Júlio, Timóteo and Monadjem, Ara and Palmeirim, Ana Filipa and Russo, Vladimir and Melo, Martim},
  doi          = {10.1038/s41559-023-02025-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {628-629},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The last afromontane forests in angola are threatened by fires},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Better incentives are needed to reward academic software
development. <em>NATEE</em>, <em>7</em>(5), 626–627. (<a
href="https://doi.org/10.1038/s41559-023-02008-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Merow, Cory and Boyle, Brad and Enquist, Brian J. and Feng, Xiao and Kass, Jamie M. and Maitner, Brian S. and McGill, Brian and Owens, Hannah and Park, Daniel S. and Paz, Andrea and Pinilla-Buitrago, Gonzalo E. and Urban, Mark C. and Varela, Sara and Wilson, Adam M.},
  doi          = {10.1038/s41559-023-02008-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {626-627},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Better incentives are needed to reward academic software development},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rolling out registered reports. <em>NATEE</em>,
<em>7</em>(5), 625. (<a
href="https://doi.org/10.1038/s41559-023-02076-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Ecology &amp;amp; Evolution is now open to submissions of Registered Reports, a format that aims to reduce publication bias by reviewing study design and results in two separate stages.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02076-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {5},
  number       = {5},
  pages        = {625},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Rolling out registered reports},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Author correction: Positive citation bias and
overinterpreted results lead to misinformation on common mycorrhizal
networks in forests. <em>NATEE</em>, <em>7</em>(4), 623. (<a
href="https://doi.org/10.1038/s41559-023-02035-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Karst, Justine and Jones, Melanie D. and Hoeksema, Jason D.},
  doi          = {10.1038/s41559-023-02035-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {623},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Positive citation bias and overinterpreted results lead to misinformation on common mycorrhizal networks in forests},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). First peoples’ knowledge leads scientists to reveal “fairy
circles” and termite linyji are linked in australia. <em>NATEE</em>,
<em>7</em>(4), 610–622. (<a
href="https://doi.org/10.1038/s41559-023-01994-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the past, when scientists encountered and studied ‘new’ environmental phenomena, they rarely considered the existing knowledge of First Peoples (also known as Indigenous or Aboriginal people). The scientific debate over the regularly spaced bare patches (so-called fairy circles) in arid grasslands of Australian deserts is a case in point. Previous researchers used remote sensing, numerical modelling, aerial images and field observations to propose that fairy circles arise from plant self-organization. Here we present Australian Aboriginal art and narratives, and soil excavation data, that suggest these regularly spaced, bare and hard circles in grasslands are pavement nests occupied by Drepanotermes harvester termites. These circles, called linyji (Manyjilyjarra language) or mingkirri (Warlpiri language), have been used by Aboriginal people in their food economies and for other domestic and sacred purposes across generations. Knowledge of the linyji has been encoded in demonstration and oral transmission, ritual art and ceremony and other media. While the exact origins of the bare circles are unclear, being buried in deep time and Jukurrpa, termites need to be incorporated as key players in a larger system of interactions between soil, water and grass. Ecologically transformative feedbacks across millennia of land use and manipulation by Aboriginal people must be accounted for. We argue that the co-production of knowledge can both improve the care and management of those systems and support intergenerational learning within and across diverse cultures. Integrating Australian Aboriginal art and narratives with soil excavation data suggests that the regularly spaced bare circles in Australian arid grasslands (sometimes known as ‘fairy circles’) are in fact linyji or mingkirri, termite pavement nests used by Aboriginal people for domestic and sacred purposes over generations.},
  archive      = {J_NATEE},
  author       = {Walsh, Fiona and Bidu, Gladys Karimarra and Bidu, Ngamaru Karimarra and Evans, Theodore A. and Judson, Thelma Milangka and Kendrick, Peter and Michaels, Alice Nampijinpa and Moore, Danae and Nelson, Matilda and Oldham, Carolyn and Schofield, Josef and Sparrow, Ashley and Taylor, Muuki Karimarra and Taylor, Desmond Purungu and Wayne, Lee Nangala and Williams, Carol Milangka},
  doi          = {10.1038/s41559-023-01994-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {610-622},
  shortjournal = {Nat. Ecol. Evol},
  title        = {First peoples’ knowledge leads scientists to reveal ‘fairy circles’ and termite linyji are linked in australia},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A 23,000-year-old southern iberian individual links human
groups that lived in western europe before and after the last glacial
maximum. <em>NATEE</em>, <em>7</em>(4), 597–609. (<a
href="https://doi.org/10.1038/s41559-023-01987-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human populations underwent range contractions during the Last Glacial Maximum (LGM) which had lasting and dramatic effects on their genetic variation. The genetic ancestry of individuals associated with the post-LGM Magdalenian technocomplex has been interpreted as being derived from groups associated with the pre-LGM Aurignacian. However, both these ancestries differ from that of central European individuals associated with the chronologically intermediate Gravettian. Thus, the genomic transition from pre- to post-LGM remains unclear also in western Europe, where we lack genomic data associated with the intermediate Solutrean, which spans the height of the LGM. Here we present genome-wide data from sites in Andalusia in southern Spain, including from a Solutrean-associated individual from Cueva del Malalmuerzo, directly dated to ~23,000 cal yr bp. The Malalmuerzo individual carried genetic ancestry that directly connects earlier Aurignacian-associated individuals with post-LGM Magdalenian-associated ancestry in western Europe. This scenario differs from Italy, where individuals associated with the transition from pre- and post-LGM carry different genetic ancestries. This suggests different dynamics in the proposed southern refugia of Ice Age Europe and posits Iberia as a potential refugium for western European pre-LGM ancestry. More, individuals from Cueva Ardales, which were thought to be of Palaeolithic origin, date younger than expected and, together with individuals from the Andalusian sites Caserones and Aguilillas, fall within the genetic variation of the Neolithic, Chalcolithic and Bronze Age individuals from southern Iberia. The authors sequence genome-wide data from multiple human individuals in southern Spain and find long-lasting genetic continuity. Here, in contrast to regions elsewhere in Europe, a 23,000-year-old individual from Malalmuerzo carries genetic ancestry that connects earlier Aurignacian-associated individuals with western European hunter-gatherers long after the Last Glacial Maximum.},
  archive      = {J_NATEE},
  author       = {Villalba-Mouco, Vanessa and van de Loosdrecht, Marieke S. and Rohrlach, Adam B. and Fewlass, Helen and Talamo, Sahra and Yu, He and Aron, Franziska and Lalueza-Fox, Carles and Cabello, Lidia and Cantalejo Duarte, Pedro and Ramos-Muñoz, José and Posth, Cosimo and Krause, Johannes and Weniger, Gerd-Christian and Haak, Wolfgang},
  doi          = {10.1038/s41559-023-01987-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {597-609},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A 23,000-year-old southern iberian individual links human groups that lived in western europe before and after the last glacial maximum},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). State-dependent evolutionary models reveal modes of solid
tumour growth. <em>NATEE</em>, <em>7</em>(4), 581–596. (<a
href="https://doi.org/10.1038/s41559-023-02000-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatial properties of tumour growth have profound implications for cancer progression, therapeutic resistance and metastasis. Yet, how spatial position governs tumour cell division remains difficult to evaluate in clinical tumours. Here, we demonstrate that faster division on the tumour periphery leaves characteristic genetic patterns, which become evident when a phylogenetic tree is reconstructed from spatially sampled cells. Namely, rapidly dividing peripheral lineages branch more extensively and acquire more mutations than slower-dividing centre lineages. We develop a Bayesian state-dependent evolutionary phylodynamic model (SDevo) that quantifies these patterns to infer the differential division rates between peripheral and central cells. We demonstrate that this approach accurately infers spatially varying birth rates of simulated tumours across a range of growth conditions and sampling strategies. We then show that SDevo outperforms state-of-the-art, non-cancer multi-state phylodynamic methods that ignore differential sequence evolution. Finally, we apply SDevo to single-time-point, multi-region sequencing data from clinical hepatocellular carcinomas and find evidence of a three- to six-times-higher division rate on the tumour edge. With the increasing availability of high-resolution, multi-region sequencing, we anticipate that SDevo will be useful in interrogating spatial growth restrictions and could be extended to model non-spatial factors that influence tumour progression. A Bayesian state-dependent evolutionary phylodynamic model (SDevo) quantifies the difference in division rates between cells at the periphery and interior of a tumour. In simulations and using clinical hepatocellular carcinoma data, the authors use SDevo to interrogate spatial patterns in tumour growth.},
  archive      = {J_NATEE},
  author       = {Lewinsohn, Maya A. and Bedford, Trevor and Müller, Nicola F. and Feder, Alison F.},
  doi          = {10.1038/s41559-023-02000-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {581-596},
  shortjournal = {Nat. Ecol. Evol},
  title        = {State-dependent evolutionary models reveal modes of solid tumour growth},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Experimental characterization of de novo proteins and their
unevolved random-sequence counterparts. <em>NATEE</em>, <em>7</em>(4),
570–580. (<a href="https://doi.org/10.1038/s41559-023-02010-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {De novo gene emergence provides a route for new proteins to be formed from previously non-coding DNA. Proteins born in this way are considered random sequences and typically assumed to lack defined structure. While it remains unclear how likely a de novo protein is to assume a soluble and stable tertiary structure, intersecting evidence from random sequence and de novo-designed proteins suggests that native-like biophysical properties are abundant in sequence space. Taking putative de novo proteins identified in human and fly, we experimentally characterize a library of these sequences to assess their solubility and structure propensity. We compare this library to a set of synthetic random proteins with no evolutionary history. Bioinformatic prediction suggests that de novo proteins may have remarkably similar distributions of biophysical properties to unevolved random sequences of a given length and amino acid composition. However, upon expression in vitro, de novo proteins exhibit moderately higher solubility which is further induced by the DnaK chaperone system. We suggest that while synthetic random sequences are a useful proxy for de novo proteins in terms of structure propensity, de novo proteins may be better integrated in the cellular system than random expectation, given their higher solubility. While a bioinformatic comparison of de novo genes and synthetic random sequence does not identify a difference in biophysical properties, in vitro expression shows higher solubility for the de novo proteins, suggesting better integration into the cellular system than random sequences.},
  archive      = {J_NATEE},
  author       = {Heames, Brennen and Buchel, Filip and Aubel, Margaux and Tretyachenko, Vyacheslav and Loginov, Dmitry and Novák, Petr and Lange, Andreas and Bornberg-Bauer, Erich and Hlouchová, Klára},
  doi          = {10.1038/s41559-023-02010-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {570-580},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Experimental characterization of de novo proteins and their unevolved random-sequence counterparts},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Convergent and complementary selection shaped gains and
losses of eusociality in sweat bees. <em>NATEE</em>, <em>7</em>(4),
557–569. (<a href="https://doi.org/10.1038/s41559-023-02001-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sweat bees have repeatedly gained and lost eusociality, a transition from individual to group reproduction. Here we generate chromosome-length genome assemblies for 17 species and identify genomic signatures of evolutionary trade-offs associated with transitions between social and solitary living. Both young genes and regulatory regions show enrichment for these molecular patterns. We also identify loci that show evidence of complementary signals of positive and relaxed selection linked specifically to the convergent gains and losses of eusociality in sweat bees. This includes two pleiotropic proteins that bind and transport juvenile hormone (JH)—a key regulator of insect development and reproduction. We find that one of these proteins is primarily expressed in subperineurial glial cells that form the insect blood–brain barrier and that brain levels of JH vary by sociality. Our findings are consistent with a role of JH in modulating social behaviour and suggest that eusocial evolution was facilitated by alteration of the proteins that bind and transport JH, revealing how an ancestral developmental hormone may have been co-opted during one of life’s major transitions. More broadly, our results highlight how evolutionary trade-offs have structured the molecular basis of eusociality in these bees and demonstrate how both directional selection and release from constraint can shape trait evolution. Leveraging independent gains and losses of eusociality, the authors identify complementary signatures of selection in sweat bee genomes, including two proteins that bind and transport juvenile hormone.},
  archive      = {J_NATEE},
  author       = {Jones, Beryl M. and Rubin, Benjamin E. R. and Dudchenko, Olga and Kingwell, Callum J. and Traniello, Ian M. and Wang, Z. Yan and Kapheim, Karen M. and Wyman, Eli S. and Adastra, Per A. and Liu, Weijie and Parsons, Lance R. and Jackson, S. RaElle and Goodwin, Katharine and Davidson, Shawn M. and McBride, Matthew J. and Webb, Andrew E. and Omufwoko, Kennedy S. and Van Dorp, Nikki and Otárola, Mauricio Fernández and Pham, Melanie and Omer, Arina D. and Weisz, David and Schraiber, Joshua and Villanea, Fernando and Wcislo, William T. and Paxton, Robert J. and Hunt, Brendan G. and Aiden, Erez Lieberman and Kocher, Sarah D.},
  doi          = {10.1038/s41559-023-02001-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {557-569},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Convergent and complementary selection shaped gains and losses of eusociality in sweat bees},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ecological traits interact with landscape context to
determine bees’ pesticide risk. <em>NATEE</em>, <em>7</em>(4), 547–556.
(<a href="https://doi.org/10.1038/s41559-023-01990-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Widespread contamination of ecosystems with pesticides threatens non-target organisms. However, the extent to which life-history traits affect pesticide exposure and resulting risk in different landscape contexts remains poorly understood. We address this for bees across an agricultural land-use gradient based on pesticide assays of pollen and nectar collected by Apis mellifera, Bombus terrestris and Osmia bicornis, representing extensive, intermediate and limited foraging traits. We found that extensive foragers (A. mellifera) experienced the highest pesticide risk—additive toxicity-weighted concentrations. However, only intermediate (B. terrestris) and limited foragers (O. bicornis) responded to landscape context—experiencing lower pesticide risk with less agricultural land. Pesticide risk correlated among bee species and between food sources and was greatest in A. mellifera-collected pollen—useful information for future postapproval pesticide monitoring. We provide foraging trait- and landscape-dependent information on the occurrence, concentration and identity of pesticides that bees encounter to estimate pesticide risk, which is necessary for more realistic risk assessment and essential information for tracking policy goals to reduce pesticide risk. Analysing pesticide residues in pollen and nectar collected by three bee species along a land-use gradient, the authors show that extensive foragers like Apis mellifera have higher pesticide risk than species that forage at intermediate or limited ranges, irrespective of landscape context.},
  archive      = {J_NATEE},
  author       = {Knapp, Jessica L. and Nicholson, Charlie C. and Jonsson, Ove and de Miranda, Joachim R. and Rundlöf, Maj},
  doi          = {10.1038/s41559-023-01990-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {547-556},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ecological traits interact with landscape context to determine bees’ pesticide risk},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Minke whale feeding rate limitations suggest constraints on
the minimum body size for engulfment filtration feeding. <em>NATEE</em>,
<em>7</em>(4), 535–546. (<a
href="https://doi.org/10.1038/s41559-023-01993-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Bulk filter feeding has enabled gigantism throughout evolutionary history. The largest animals, extant rorqual whales, utilize intermittent engulfment filtration feeding (lunge feeding), which increases in efficiency with body size, enabling their gigantism. The smallest extant rorquals (7–10 m minke whales), however, still exhibit short-term foraging efficiencies several times greater than smaller non-filter-feeding cetaceans, raising the question of why smaller animals do not utilize this foraging modality. We collected 437 h of bio-logging data from 23 Antarctic minke whales (Balaenoptera bonaerensis) to test the relationship of feeding rates (λf) to body size. Here, we show that while ultra-high nighttime λf (mean ± s.d.: 165 ± 40 lunges h−1; max: 236 lunges h−1; mean depth: 28 ± 46 m) were indistinguishable from predictions from observations of larger species, daytime λf (mean depth: 72 ± 72 m) were only 25–40% of predicted rates. Both λf were near the maxima allowed by calculated biomechanical, physiological and environmental constraints, but these temporal constraints meant that maximum λf was below the expected λf for animals smaller than ~5 m—the length of weaned minke whales. Our findings suggest that minimum size for specific filter-feeding body plans may relate broadly to temporal restrictions on filtration rate and have implications for the evolution of filter feeding. Theoretical models of foraging efficiency suggest that lunge-filter-feeding marine vertebrates could be as small as 10 kg. However, here the authors show with bio-logged data from filter-feeding minke whales that in practice there are minimum body-size constraints on filter feeders, below which this becomes an unviable feeding strategy.},
  archive      = {J_NATEE},
  author       = {Cade, David E. and Kahane-Rapport, Shirel R. and Gough, William T. and Bierlich, K. C. and Linsky, Jacob M. J. and Calambokidis, John and Johnston, David W. and Goldbogen, Jeremy A. and Friedlaender, Ari S.},
  doi          = {10.1038/s41559-023-01993-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {535-546},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Minke whale feeding rate limitations suggest constraints on the minimum body size for engulfment filtration feeding},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Climate drives global functional trait variation in lizards.
<em>NATEE</em>, <em>7</em>(4), 524–534. (<a
href="https://doi.org/10.1038/s41559-023-02007-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A major challenge in ecology and evolution is to disentangle the mechanisms driving broad-scale variation in biological traits such as body size, colour, thermal physiology traits and behaviour. Climate has long been thought to drive trait evolution and abiotic filtering of trait variation in ectotherms because their thermal performance and fitness are closely related to environmental conditions. However, previous studies investigating climatic variables associated with trait variation have lacked a mechanistic description of the underpinning processes. Here, we use a mechanistic model to predict how climate affects thermal performance of ectotherms and thereby the direction and strength of the effect of selection on different functional traits. We show that climate drives macro-evolutionary patterns in body size, cold tolerance and preferred body temperatures among lizards, and that trait variation is more constrained in regions where selection is predicted to be stronger. These findings provide a mechanistic explanation for observations on how climate drives trait variation in ectotherms through its effect on thermal performance. By connecting physical, physiological and macro-evolutionary principles, the model and results provide an integrative, mechanistic framework for predicting organismal responses to present climates and climate change. The authors incorporate mechanistic information about lizard physiological responses to heat into predictions of trait variation across time and space, finding that the range of functional traits is more constrained in locations where the local climate strongly selects for thermal performance.},
  archive      = {J_NATEE},
  author       = {Rubalcaba, Juan G. and Gouveia, Sidney F. and Villalobos, Fabricio and Olalla-Tárraga, Miguel Á. and Sunday, Jennifer},
  doi          = {10.1038/s41559-023-02007-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {524-534},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Climate drives global functional trait variation in lizards},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Peer review perpetuates barriers for historically excluded
groups. <em>NATEE</em>, <em>7</em>(4), 512–523. (<a
href="https://doi.org/10.1038/s41559-023-01999-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peer review is central to the scientific process and scientists’ career advancement, but bias at various stages of the review process disadvantages some authors. Here we use peer review data from 312,740 biological sciences manuscripts across 31 studies to (1) examine evidence for differential peer review outcomes based on author demographics, (2) evaluate the efficacy of solutions to reduce bias and (3) describe the current landscape of peer review policies for 541 ecology and evolution journals. We found notably worse review outcomes (for example, lower overall acceptance rates) for authors whose institutional affiliations were in Asia, for authors whose country’s primary language is not English and in countries with relatively low Human Development Indices. We found few data evaluating efficacy of interventions outside of reducing gender bias through double-blind review or diversifying reviewer/editorial boards. Despite evidence for review outcome gaps based on author demographics, few journals currently implement policies intended to mitigate bias (for example, 15.9% of journals practised double-blind review and 2.03% had reviewer guidelines that mentioned social justice issues). The lack of demographic equity signals an urgent need to better understand and implement evidence-based bias mitigation strategies. A meta-analysis of peer-review data from over 300,000 biological sciences manuscripts reveals worse review outcomes for authors from historically excluded groups, and limited data evaluating the effectiveness of interventions to address bias in peer review.},
  archive      = {J_NATEE},
  author       = {Smith, Olivia M. and Davis, Kayla L. and Pizza, Riley B. and Waterman, Robin and Dobson, Kara C. and Foster, Brianna and Jarvey, Julie C. and Jones, Leonard N. and Leuenberger, Wendy and Nourn, Nan and Conway, Emily E. and Fiser, Cynthia M. and Hansen, Zoe A. and Hristova, Ani and Mack, Caitlin and Saunders, Alyssa N. and Utley, Olivia J. and Young, Moriah L. and Davis, Courtney L.},
  doi          = {10.1038/s41559-023-01999-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {512-523},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Peer review perpetuates barriers for historically excluded groups},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Positive citation bias and overinterpreted results lead to
misinformation on common mycorrhizal networks in forests.
<em>NATEE</em>, <em>7</em>(4), 501–511. (<a
href="https://doi.org/10.1038/s41559-023-01986-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A common mycorrhizal network (CMN) is formed when mycorrhizal fungal hyphae connect the roots of multiple plants of the same or different species belowground. Recently, CMNs have captured the interest of broad audiences, especially with respect to forest function and management. We are concerned, however, that recent claims in the popular media about CMNs in forests are disconnected from evidence, and that bias towards citing positive effects of CMNs has developed in the scientific literature. We first evaluated the evidence supporting three common claims. The claims that CMNs are widespread in forests and that resources are transferred through CMNs to increase seedling performance are insufficiently supported because results from field studies vary too widely, have alternative explanations or are too limited to support generalizations. The claim that mature trees preferentially send resources and defence signals to offspring through CMNs has no peer-reviewed, published evidence. We next examined how the results from CMN research are cited and found that unsupported claims have doubled in the past 25 years; a bias towards citing positive effects may obscure our understanding of the structure and function of CMNs in forests. We conclude that knowledge on CMNs is presently too sparse and unsettled to inform forest management. In this Perspective, Karst et al. discuss how both the popular media and scientific literature have inflated the extent of evidence for various roles of mycorrhizal fungal networks in forests.},
  archive      = {J_NATEE},
  author       = {Karst, Justine and Jones, Melanie D. and Hoeksema, Jason D.},
  doi          = {10.1038/s41559-023-01986-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {501-511},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Positive citation bias and overinterpreted results lead to misinformation on common mycorrhizal networks in forests},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Convergent selection and evolutionary trade-offs shape
social transitions in sweat bees. <em>NATEE</em>, <em>7</em>(4),
499–500. (<a href="https://doi.org/10.1038/s41559-023-02006-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using genome assemblies and comparative analyses, we identified evolutionary signatures of selection associated with repeated gains and losses of social behaviour in sweat bees. These signatures include changes in regulatory regions and young genes, as well as complementary patterns of positive and relaxed selection on proteins involved in juvenile hormone signalling.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02006-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {499-500},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Convergent selection and evolutionary trade-offs shape social transitions in sweat bees},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Modelling evolution at the boundaries of solid tumours.
<em>NATEE</em>, <em>7</em>(4), 497–498. (<a
href="https://doi.org/10.1038/s41559-023-01996-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A mathematical model of eco-evolutionary dynamics estimates different birth rates of cells at the periphery of a tumour versus its centre, giving insight into locally stable evolutionary mechanisms that arise as a result of boundary-driven growth.},
  archive      = {J_NATEE},
  author       = {Chattopadhyay, Subhayan and Gisselsson, David},
  doi          = {10.1038/s41559-023-01996-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {497-498},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Modelling evolution at the boundaries of solid tumours},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Landscapes of pesticide risk. <em>NATEE</em>, <em>7</em>(4),
495–496. (<a href="https://doi.org/10.1038/s41559-023-02003-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A large-scale field study finds that different bee species experience different levels of risk from pesticides, depending on how much land is farmed within their foraging range. For bumblebees and solitary bees, more seminatural habitat means less risk from pesticides, but this is not true for honeybees.},
  archive      = {J_NATEE},
  author       = {Straw, Edward A.},
  doi          = {10.1038/s41559-023-02003-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {495-496},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Landscapes of pesticide risk},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Photovoltaics alter plant productivity. <em>NATEE</em>,
<em>7</em>(4), 494. (<a
href="https://doi.org/10.1038/s41559-023-02011-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Harold, Simon},
  doi          = {10.1038/s41559-023-02011-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {494},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Photovoltaics alter plant productivity},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Cetacean cooperation. <em>NATEE</em>, <em>7</em>(4), 493.
(<a href="https://doi.org/10.1038/s41559-023-02005-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {McKay, Alexa},
  doi          = {10.1038/s41559-023-02005-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {493},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Cetacean cooperation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Heritable trait variation for climate resilience.
<em>NATEE</em>, <em>7</em>(4), 492. (<a
href="https://doi.org/10.1038/s41559-023-02004-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-023-02004-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {492},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Heritable trait variation for climate resilience},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023e). Sparrows adapt to pollution. <em>NATEE</em>, <em>7</em>(4),
491. (<a href="https://doi.org/10.1038/s41559-023-01992-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-023-01992-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {491},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sparrows adapt to pollution},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). On our (children’s) bookshelf. <em>NATEE</em>,
<em>7</em>(4), 486–490. (<a
href="https://doi.org/10.1038/s41559-023-01988-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera and Goymer, Patrick and Harold, Simon and McKay, Alexa and Nic Eoin, Luíseach and Turner, Marian},
  doi          = {10.1038/s41559-023-01988-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {486-490},
  shortjournal = {Nat. Ecol. Evol},
  title        = {On our (children’s) bookshelf},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Making marine biotechnology work for people and nature.
<em>NATEE</em>, <em>7</em>(4), 482–485. (<a
href="https://doi.org/10.1038/s41559-022-01976-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transforming the rapidly growing ocean economy into a ‘blue economy’ based on principles of sustainability, equity and inclusivity is crucial. We contend that marine biotechnology is not on this trajectory and that a more holistic approach for people and nature is needed to bring marine biotechnology into the blue economy.},
  archive      = {J_NATEE},
  author       = {Blasiak, Robert and Jouffray, Jean-Baptiste and Amon, Diva J. and Claudet, Joachim and Dunshirn, Paul and Søgaard Jørgensen, Peter and Pranindita, Agnes and Wabnitz, Colette C. C. and Zhivkoplias, Erik and Österblom, Henrik},
  doi          = {10.1038/s41559-022-01976-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {482-485},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Making marine biotechnology work for people and nature},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The case for the reintroduction of cheetahs to india.
<em>NATEE</em>, <em>7</em>(4), 480–481. (<a
href="https://doi.org/10.1038/s41559-023-02002-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Tordiffe, Adrian S. W. and Jhala, Yadvendradev V. and Boitani, Luigi and Cristescu, Bogdan and Kock, Richard A. and Meyer, Leith R. C. and Naylor, Simon and O’Brien, Stephen J. and Schmidt-Küntzel, Anne and Stanley Price, Mark R. and van der Merwe, Vincent and Marker, Laurie},
  doi          = {10.1038/s41559-023-02002-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {480-481},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The case for the reintroduction of cheetahs to india},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Public interest in individual study animals can bolster
wildlife conservation. <em>NATEE</em>, <em>7</em>(4), 478–479. (<a
href="https://doi.org/10.1038/s41559-023-02009-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Wilkinson, Christine E.},
  doi          = {10.1038/s41559-023-02009-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {478-479},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Public interest in individual study animals can bolster wildlife conservation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Sea change. <em>NATEE</em>, <em>7</em>(4), 477. (<a
href="https://doi.org/10.1038/s41559-023-02042-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The UN agreement to protect the biodiversity of the high seas is an important first step towards meeting the 30×30 target in the marine realm.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02042-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {4},
  number       = {4},
  pages        = {477},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sea change},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Author correction: Functional diversity and community
assembly of river invertebrates show globally consistent responses to
decreasing glacier cover. <em>NATEE</em>, <em>7</em>(3), 472–476. (<a
href="https://doi.org/10.1038/s41559-023-01989-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Brown, Lee E. and Khamis, Kieran and Wilkes, Martin and Blaen, Phillip and Brittain, John E. and Carrivick, Jonathan L. and Fell, Sarah and Friberg, Nikolai and Füreder, Leopold and Gislason, Gisli M. and Hainie, Sarah and Hannah, David M. and James, William H. M. and Lencioni, Valeria and Olafsson, Jon S. and Robinson, Christopher T. and Saltveit, Svein J. and Thompson, Craig and Milner, Alexander M.},
  doi          = {10.1038/s41559-023-01989-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {472-476},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Author correction: Functional diversity and community assembly of river invertebrates show globally consistent responses to decreasing glacier cover},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Spatial memory predicts home range size and predation risk
in pheasants. <em>NATEE</em>, <em>7</em>(3), 461–471. (<a
href="https://doi.org/10.1038/s41559-022-01950-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most animals confine their activities to a discrete home range, long assumed to reflect the fitness benefits of obtaining spatial knowledge about the landscape. However, few empirical studies have linked spatial memory to home range development or determined how selection operates on spatial memory via the latter’s role in mediating space use. We assayed the cognitive ability of juvenile pheasants (Phasianus colchicus) reared under identical conditions before releasing them into the wild. Then, we used high-throughput tracking to record their movements as they developed their home ranges, and determined the location, timing and cause of mortality events. Individuals with greater spatial reference memory developed larger home ranges. Mortality risk from predators was highest at the periphery of an individual’s home range in areas where they had less experience and opportunity to obtain spatial information. Predation risk was lower in individuals with greater spatial memory and larger core home ranges, suggesting selection may operate on spatial memory by increasing the ability to learn about predation risk across the landscape. Our results reveal that spatial memory, determined from abstract cognitive assays, shapes home range development and variation, and suggests predation risk selects for spatial memory via experience-dependent spatial variation in mortality. After assaying the cognitive ability of juvenile pheasants and releasing them into a new landscape, the authors show that pheasants with better memory developed larger home range sizes and more successfully avoided predation at the edges of the home range.},
  archive      = {J_NATEE},
  author       = {Heathcote, Robert J. P. and Whiteside, Mark A. and Beardsworth, Christine E. and Van Horik, Jayden O. and Laker, Philippa R. and Toledo, Sivan and Orchan, Yotam and Nathan, Ran and Madden, Joah R.},
  doi          = {10.1038/s41559-022-01950-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {461-471},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Spatial memory predicts home range size and predation risk in pheasants},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pathogen evasion of social immunity. <em>NATEE</em>,
<em>7</em>(3), 450–460. (<a
href="https://doi.org/10.1038/s41559-023-01981-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Treating sick group members is a hallmark of collective disease defence in vertebrates and invertebrates alike. Despite substantial effects on pathogen fitness and epidemiology, it is still largely unknown how pathogens react to the selection pressure imposed by care intervention. Using social insects and pathogenic fungi, we here performed a serial passage experiment in the presence or absence of colony members, which provide social immunity by grooming off infectious spores from exposed individuals. We found specific effects on pathogen diversity, virulence and transmission. Under selection of social immunity, pathogens invested into higher spore production, but spores were less virulent. Notably, they also elicited a lower grooming response in colony members, compared with spores from the individual host selection lines. Chemical spore analysis suggested that the spores from social selection lines escaped the caregivers’ detection by containing lower levels of ergosterol, a key fungal membrane component. Experimental application of chemically pure ergosterol indeed induced sanitary grooming, supporting its role as a microbe-associated cue triggering host social immunity against fungal pathogens. By reducing this detection cue, pathogens were able to evade the otherwise very effective collective disease defences of their social hosts. Social insects often groom nestmates as an anti-pathogen behaviour, providing social immunity. Here the authors show that anti-fungal grooming behaviour by ants selects for the fungus to produce more but less-virulent and less-detectable spores, suggesting pathogen adaptation to social immunity.},
  archive      = {J_NATEE},
  author       = {Stock, Miriam and Milutinović, Barbara and Hoenigsberger, Michaela and Grasse, Anna V. and Wiesenhofer, Florian and Kampleitner, Niklas and Narasimhan, Madhumitha and Schmitt, Thomas and Cremer, Sylvia},
  doi          = {10.1038/s41559-023-01981-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {450-460},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pathogen evasion of social immunity},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Transcriptional and mutational signatures of the drosophila
ageing germline. <em>NATEE</em>, <em>7</em>(3), 440–449. (<a
href="https://doi.org/10.1038/s41559-022-01958-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ageing is a complex biological process that is accompanied by changes in gene expression and mutational load. In many species, including humans, older fathers pass on more paternally derived de novo mutations; however, the cellular basis and cell types driving this pattern are still unclear. To explore the root causes of this phenomenon, we performed single-cell RNA sequencing on testes from young and old male Drosophila and genomic sequencing (DNA sequencing) on somatic tissues from the same flies. We found that early germ cells from old and young flies enter spermatogenesis with similar mutational loads but older flies are less able to remove mutations during spermatogenesis. Mutations in old cells may also increase during spermatogenesis. Our data reveal that old and young flies have distinct mutational biases. Many classes of genes show increased postmeiotic expression in the germlines of older flies. Late spermatogenesis-biased genes have higher dN/dS (ratio of non-synonymous to synonymous substitutions) than early spermatogenesis-biased genes, supporting the hypothesis that late spermatogenesis is a source of evolutionary innovation. Surprisingly, genes biased in young germ cells show higher dN/dS than genes biased in old germ cells. Our results provide new insights into the role of the germline in de novo mutation. Single-cell RNA sequencing of testes from young and old male Drosophila, together with genomic sequencing of somatic tissues of the same flies, shows distinct mutational biases in old and young flies and suggests late spermatogenesis as a source of evolutionary innovation.},
  archive      = {J_NATEE},
  author       = {Witt, Evan and Langer, Christopher B. and Svetec, Nicolas and Zhao, Li},
  doi          = {10.1038/s41559-022-01958-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {440-449},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Transcriptional and mutational signatures of the drosophila ageing germline},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Chromosome fusions repatterned recombination rate and
facilitated reproductive isolation during pristionchus nematode
speciation. <em>NATEE</em>, <em>7</em>(3), 424–439. (<a
href="https://doi.org/10.1038/s41559-022-01980-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale genome-structural evolution is common in various organisms. Recent developments in speciation genomics revealed the importance of inversions, whereas the role of other genome-structural rearrangements, including chromosome fusions, have not been well characterized. We study genomic divergence and reproductive isolation of closely related nematodes: the androdioecious (hermaphroditic) model Pristionchus pacificus and its dioecious sister species Pristionchus exspectatus. A chromosome-level genome assembly of P. exspectatus using single-molecule and Hi-C sequencing revealed a chromosome-wide rearrangement relative to P. pacificus. Strikingly, genomic characterization and cytogenetic studies including outgroup species Pristionchus occultus indicated two independent fusions involving the same chromosome, ChrIR, between these related species. Genetic linkage analysis indicated that these fusions altered the chromosome-wide pattern of recombination, resulting in large low-recombination regions that probably facilitated the coevolution between some of the ~14.8% of genes across the entire genomes. Quantitative trait locus analyses for hybrid sterility in all three sexes revealed that major quantitative trait loci mapped to the fused chromosome ChrIR. While abnormal chromosome segregations of the fused chromosome partially explain hybrid female sterility, hybrid-specific recombination that breaks linkage of genes in the low-recombination region was associated with hybrid male sterility. Thus, recent chromosome fusions repatterned recombination rate and drove reproductive isolation during Pristionchus speciation. Genomic analyses show that two independent fusions involving the same chromosome altered recombination patterns and contributed to reproductive isolation between two Pristionchus nematodes.},
  archive      = {J_NATEE},
  author       = {Yoshida, Kohta and Rödelsperger, Christian and Röseler, Waltraud and Riebesell, Metta and Sun, Simo and Kikuchi, Taisei and Sommer, Ralf J.},
  doi          = {10.1038/s41559-022-01980-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {424-439},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Chromosome fusions repatterned recombination rate and facilitated reproductive isolation during pristionchus nematode speciation},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Genetic assimilation of ancestral plasticity during parallel
adaptation to zinc contamination in silene uniflora. <em>NATEE</em>,
<em>7</em>(3), 414–423. (<a
href="https://doi.org/10.1038/s41559-022-01975-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phenotypic plasticity in ancestral populations is hypothesized to facilitate adaptation, but evidence is piecemeal and often contradictory. Further, whether ancestral plasticity increases the probability of parallel adaptive changes has not been explored. The most general finding is that ancestral responses to a new environment are reversed following adaptation (known as reversion). We investigated the contribution of ancestral plasticity to adaptive evolution of gene expression in two independently evolved lineages of zinc-tolerant Silene uniflora. We found that the general pattern of reversion is driven by the absence of a widespread stress response in zinc-adapted plants compared with zinc-sensitive plants. We show that ancestral plasticity that moves expression closer to the optimum value in the new environment influences the evolution of gene expression among genes that are likely to be involved in adaptation and increases the chance that genes are recruited repeatedly during adaptation. However, despite convergence in gene expression levels between independently adapted lineages, ancestral plasticity does not influence how similar expression values of adaptive genes become. Surprisingly, we also observed that ancestral plasticity that increases fitness often becomes genetically determined and fixed, that is, genetically assimilated. These results emphasize the important role of ancestral plasticity in parallel adaptation. Adaptation to new environments often involves changes in gene expression. This study shows a role of ancestral gene expression plasticity in heavy metal adaptation of two independent lineages of Silene uniflora.},
  archive      = {J_NATEE},
  author       = {Wood, Daniel P. and Holmberg, Jon A. and Osborne, Owen G. and Helmstetter, Andrew J. and Dunning, Luke T. and Ellison, Amy R. and Smith, Rhian J. and Lighten, Jackie and Papadopulos, Alexander S. T.},
  doi          = {10.1038/s41559-022-01975-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {414-423},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Genetic assimilation of ancestral plasticity during parallel adaptation to zinc contamination in silene uniflora},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rapid upwards spread of non-native plants in mountains
across continents. <em>NATEE</em>, <em>7</em>(3), 405–413. (<a
href="https://doi.org/10.1038/s41559-022-01979-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-elevation ecosystems are among the few ecosystems worldwide that are not yet heavily invaded by non-native plants. This is expected to change as species expand their range limits upwards to fill their climatic niches and respond to ongoing anthropogenic disturbances. Yet, whether and how quickly these changes are happening has only been assessed in a few isolated cases. Starting in 2007, we conducted repeated surveys of non-native plant distributions along mountain roads in 11 regions from 5 continents. We show that over a 5- to 10-year period, the number of non-native species increased on average by approximately 16% per decade across regions. The direction and magnitude of upper range limit shifts depended on elevation across all regions. Supported by a null-model approach accounting for range changes expected by chance alone, we found greater than expected upward shifts at lower/mid elevations in at least seven regions. After accounting for elevation dependence, significant average upward shifts were detected in a further three regions (revealing evidence for upward shifts in 10 of 11 regions). Together, our results show that mountain environments are becoming increasingly exposed to biological invasions, emphasizing the need to monitor and prevent potential biosecurity issues emerging in high-elevation ecosystems. The authors resurveyed a previously sampled set of mountain transects on five continents, showing that the ranges of non-native plant species have shifted upslope in most locations in just 5–10 years.},
  archive      = {J_NATEE},
  author       = {Iseli, Evelin and Chisholm, Chelsea and Lenoir, Jonathan and Haider, Sylvia and Seipel, Tim and Barros, Agustina and Hargreaves, Anna L. and Kardol, Paul and Lembrechts, Jonas J. and McDougall, Keith and Rashid, Irfan and Rumpf, Sabine B. and Arévalo, José Ramón and Cavieres, Lohengrin and Daehler, Curtis and Dar, Pervaiz A. and Endress, Bryan and Jakobs, Gabi and Jiménez, Alejandra and Küffer, Christoph and Mihoc, Maritza and Milbau, Ann and Morgan, John W. and Naylor, Bridgett J. and Pauchard, Aníbal and Ratier Backes, Amanda and Reshi, Zafar A. and Rew, Lisa J. and Righetti, Damiano and Shannon, James M. and Valencia, Graciela and Walsh, Neville and Wright, Genevieve T. and Alexander, Jake M.},
  doi          = {10.1038/s41559-022-01979-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {405-413},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Rapid upwards spread of non-native plants in mountains across continents},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Forest edges increase pollinator network robustness to
extinction with declining area. <em>NATEE</em>, <em>7</em>(3), 393–404.
(<a href="https://doi.org/10.1038/s41559-022-01973-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Edge effects often exacerbate the negative effects of habitat loss on biodiversity. In forested ecosystems, however, many pollinators actually prefer open sunny conditions created by edge disturbances. We tested the hypothesis that forest edges have a positive buffering effect on plant-pollinator interaction networks in the face of declining forest area. In a fragmented land-bridge island system, we recorded ~20,000 plant-pollinator interactions on 41 islands over 3 yr. We show that plant richness and floral resources decline with decreasing forest area at both interior and edge sites, but edges maintain 10-fold higher pollinator abundance and richness regardless of area loss. Edge networks contain highly specialized species, with higher nestedness and lower modularity than interior networks, maintaining high robustness to extinction following area loss while forest interior networks collapse. Anthropogenic forest edges benefit community diversity and network robustness to extinction in the absence of natural gap-phase dynamics in small degraded forest remnants. Analysing 20,000 plant-pollinator interactions over 3 yr in a fragmented island ecosystem, the authors show that forest edges benefit community diversity and network robustness to extinction in the face of declining forest area.},
  archive      = {J_NATEE},
  author       = {Ren, Peng and Didham, Raphael K. and Murphy, Mark V. and Zeng, Di and Si, Xingfeng and Ding, Ping},
  doi          = {10.1038/s41559-022-01973-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {393-404},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Forest edges increase pollinator network robustness to extinction with declining area},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Colombian biodiversity is governed by a rich and diverse
policy mix. <em>NATEE</em>, <em>7</em>(3), 382–392. (<a
href="https://doi.org/10.1038/s41559-023-01983-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We lack an understanding of how diverse policymakers interact to govern biodiversity. Taking Colombia as a focal case, we examined six decades of biodiversity governance (1959–2018). Here we analysed the composition of the policy mix, and how it has evolved over time, how policies differ among lead actors and ecosystems, and whether the policy mix addresses the primary threats to biodiversity. We identified 186 biodiversity-related policies that govern multiple ecosystems, use different instruments and address the main threats to biodiversity (that is, agriculture and aquaculture, and biological resource use). We found policy gaps in the governance of invasive species and wildlife trade. Biodiversity policy integration into some sectoral policies, such as climate change, poverty and pollution, has become more common in the past decade. Our results point to an increased need for effective coordination across sectors and actors, as new ones influence and implement the policy mix. A qualitative and quantitative analysis of 186 biodiversity-related policies in Colombia is used to describe how biodiversity has been integrated into policy domains, which policy instruments are most prevalent and how the policy mix has changed over six decades.},
  archive      = {J_NATEE},
  author       = {Echeverri, Alejandra and Furumo, Paul R. and Moss, Sydney and Figot Kuthy, Alan G. and García Aguirre, Daniela and Mandle, Lisa and Valencia, Ivan Darío and Ruckelshaus, Mary and Daily, Gretchen C. and Lambin, Eric F.},
  doi          = {10.1038/s41559-023-01983-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {382-392},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Colombian biodiversity is governed by a rich and diverse policy mix},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Divergent vertebral formulae shape the evolution of axial
complexity in mammals. <em>NATEE</em>, <em>7</em>(3), 367–381. (<a
href="https://doi.org/10.1038/s41559-023-01982-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Complexity, defined as the number of parts and their degree of differentiation, is a poorly explored aspect of macroevolutionary dynamics. The maximum anatomical complexity of organisms has undoubtedly increased through evolutionary time. However, it is unclear whether this increase is a purely diffusive process or whether it is at least partly driven, occurring in parallel in most or many lineages and with increases in the minima as well as the means. Highly differentiated and serially repeated structures, such as vertebrae, are useful systems with which to investigate these patterns. We focus on the serial differentiation of the vertebral column in 1,136 extant mammal species, using two indices that quantify complexity as the numerical richness and proportional distribution of vertebrae across presacral regions and a third expressing the ratio between thoracic and lumbar vertebrae. We address three questions. First, we ask whether the distribution of complexity values in major mammal groups is similar or whether clades have specific signatures associated with their ecology. Second, we ask whether changes in complexity throughout the phylogeny are biased towards increases and whether there is evidence of driven trends. Third, we ask whether evolutionary shifts in complexity depart from a uniform Brownian motion model. Vertebral counts, but not complexity indices, differ significantly between major groups and exhibit greater within-group variation than recognized hitherto. We find strong evidence of a trend towards increasing complexity, where higher values propagate further increases in descendant lineages. Several increases are inferred to have coincided with major ecological or environmental shifts. We find support for multiple-rate models of evolution for all complexity metrics, suggesting that increases in complexity occurred in stepwise shifts, with evidence for widespread episodes of recent rapid divergence. Different subclades evolve more complex vertebral columns in different configurations and probably under different selective pressures and constraints, with widespread convergence on the same formulae. Further work should therefore focus on the ecological relevance of differences in complexity and a more detailed understanding of historical patterns. Focusing on the serial differentiation of the presacral column across 1,136 extant mammal species, the authors find evidence of high within-group variation and an evolutionary trend towards increasing complexity.},
  archive      = {J_NATEE},
  author       = {Li, Yimeng and Brinkworth, Andrew and Green, Emily and Oyston, Jack and Wills, Matthew and Ruta, Marcello},
  doi          = {10.1038/s41559-023-01982-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {367-381},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Divergent vertebral formulae shape the evolution of axial complexity in mammals},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reconstruction of hundreds of reference ancestral genomes
across the eukaryotic kingdom. <em>NATEE</em>, <em>7</em>(3), 355–366.
(<a href="https://doi.org/10.1038/s41559-022-01956-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ancestral sequence reconstruction is a fundamental aspect of molecular evolution studies and can trace small-scale sequence modifications through the evolution of genomes and species. In contrast, fine-grained reconstructions of ancestral genome organizations are still in their infancy, limiting our ability to draw comprehensive views of genome and karyotype evolution. Here we reconstruct the detailed gene contents and organizations of 624 ancestral vertebrate, plant, fungi, metazoan and protist genomes, 183 of which are near-complete chromosomal gene order reconstructions. Reconstructed ancestral genomes are similar to their descendants in terms of gene content as expected and agree precisely with reference cytogenetic and in silico reconstructions when available. By comparing successive ancestral genomes along the phylogenetic tree, we estimate the intra- and interchromosomal rearrangement history of all major vertebrate clades at high resolution. This freely available resource introduces the possibility to follow evolutionary processes at genomic scales in chronological order, across multiple clades and without relying on a single extant species as reference. Using an algorithm for ancestral genome reconstruction, the authors present 624 ancestral genomes for vertebrates, plants, fungi, metazoans and protists and reconstruct the chromosomal rearrangement history of all major vertebrate clades.},
  archive      = {J_NATEE},
  author       = {Muffato, Matthieu and Louis, Alexandra and Nguyen, Nga Thi Thuy and Lucas, Joseph and Berthelot, Camille and Roest Crollius, Hugues},
  doi          = {10.1038/s41559-022-01956-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {355-366},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reconstruction of hundreds of reference ancestral genomes across the eukaryotic kingdom},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Increases in intraspecific body size variation are common
among north american mammals and birds between 1880 and 2020.
<em>NATEE</em>, <em>7</em>(3), 347–354. (<a
href="https://doi.org/10.1038/s41559-022-01967-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many studies have documented the average body size of animals declining over time. Compared to mean body size, less is known about long-term changes in intraspecific trait variation (ITV), which is also important to understanding species’ ability to cope with environmental challenges. On the basis of 393,499 specimen records from 380 species collected in North America between 1880 and 2020, we found that body size ITV increased by 9.59% for mammals (n = 302) and 30.67% for birds (n = 78); human-harvested species had higher probability of ITV increase. The observed increasing ITV in many species suggests possible niche expansion and potential buffering effects against downsizing but it risks increased maladaptation to rapidly changing environments. The results demonstrate that trait mean and variance do not necessarily respond in similar ways to anthropogenic pressures and both should be considered. The authors compiled body size data from mammal and bird museum collections in North America to show that intraspecific variation in body size, but not mean body size, has increased over time.},
  archive      = {J_NATEE},
  author       = {Zheng, Shilu and Hu, Juntao and Ma, Zhijun and Lindenmayer, David and Liu, Jiajia},
  doi          = {10.1038/s41559-022-01967-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {347-354},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Increases in intraspecific body size variation are common among north american mammals and birds between 1880 and 2020},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A surge in obsidian exploitation more than 1.2 million years
ago at simbiro III (melka kunture, upper awash, ethiopia).
<em>NATEE</em>, <em>7</em>(3), 337–346. (<a
href="https://doi.org/10.1038/s41559-022-01970-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pleistocene archaeology records the changing behaviour and capacities of early hominins. These behavioural changes, for example, to stone tools, are commonly linked to environmental constraints. It has been argued that, in earlier times, multiple activities of everyday life were all uniformly conducted at the same spot. The separation of focused activities across different localities, which indicates a degree of planning, according to this mindset characterizes later hominins since only 500,000 years ago. Simbiro III level C, in the upper Awash valley of Ethiopia, allows us to test this assumption in its assemblage of stone tools made only with obsidian, dated to more than 1.2 million years (Myr) old. Here we first reconstruct the palaeoenvironment, showing that the landscape was seasonally flooded. Following the deposition of an accumulation of obsidian cobbles by a meandering river, hominins began to exploit these in new ways, producing large tools with sharp cutting edges. We show through statistical analysis that this was a focused activity, that very standardized handaxes were produced and that this was a stone-tool workshop. We argue that at Simbiro III, hominins were doing much more than simply reacting to environmental changes; they were taking advantage of new opportunities, and developing new techniques and new skills according to them. The authors report a specialized obsidian handaxe workshop at the site of Simbiro III in Ethiopia, suggesting that hominins more than 1.2 million years ago took advantage of opportunities provided by changing environmental conditions.},
  archive      = {J_NATEE},
  author       = {Mussi, Margherita and Mendez-Quintas, Eduardo and Barboni, Doris and Bocherens, Hervé and Bonnefille, Raymonde and Briatico, Giuseppe and Geraads, Denis and Melis, Rita T. and Panera, Joaquin and Pioli, Laura and Domínguez, Andrea Serodio and Jara, Susana Rubio},
  doi          = {10.1038/s41559-022-01970-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {337-346},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A surge in obsidian exploitation more than 1.2 million years ago at simbiro III (Melka kunture, upper awash, ethiopia)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reply to: Plant traits alone are good predictors of
ecosystem properties when used carefully. <em>NATEE</em>, <em>7</em>(3),
335–336. (<a href="https://doi.org/10.1038/s41559-022-01957-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {van der Plas, Fons and Schröder-Georgi, Thomas and Weigelt, Alexandra and Barry, Kathryn and Meyer, Sebastian and Alzate, Adriana and Barnard, Romain L. and Buchmann, Nina and de Kroon, Hans and Ebeling, Anne and Eisenhauer, Nico and Engels, Christof and Fischer, Markus and Gleixner, Gerd and Hildebrandt, Anke and Koller-France, Eva and Leimer, Sophia and Milcu, Alexandru and Mommer, Liesje and Niklaus, Pascal A. and Oelmann, Yvonne and Roscher, Christiane and Scherber, Christoph and Scherer-Lorenzen, Michael and Scheu, Stefan and Schmid, Bernhard and Schulze, Ernst-Detlef and Temperton, Vicky and Tscharntke, Teja and Voigt, Winfried and Weisser, Wolfgang and Wilcke, Wolfgang and Wirth, Christian},
  doi          = {10.1038/s41559-022-01957-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {335-336},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reply to: Plant traits alone are good predictors of ecosystem properties when used carefully},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Plant traits alone are good predictors of ecosystem
properties when used carefully. <em>NATEE</em>, <em>7</em>(3), 332–334.
(<a href="https://doi.org/10.1038/s41559-022-01920-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Hagan, James G. and Henn, Jonathan J. and Osterman, Wilhelm H. A.},
  doi          = {10.1038/s41559-022-01920-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {332-334},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Plant traits alone are good predictors of ecosystem properties when used carefully},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A predictive timeline of wildlife population collapse.
<em>NATEE</em>, <em>7</em>(3), 320–331. (<a
href="https://doi.org/10.1038/s41559-023-01985-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Contemporary rates of biodiversity decline emphasize the need for reliable ecological forecasting, but current methods vary in their ability to predict the declines of real-world populations. Acknowledging that stressor effects start at the individual level, and that it is the sum of these individual-level effects that drives populations to collapse, shifts the focus of predictive ecology away from using predominantly abundance data. Doing so opens new opportunities to develop predictive frameworks that utilize increasingly available multi-dimensional data, which have previously been overlooked for ecological forecasting. Here, we propose that stressed populations will exhibit a predictable sequence of observable changes through time: changes in individuals’ behaviour will occur as the first sign of increasing stress, followed by changes in fitness-related morphological traits, shifts in the dynamics (for example, birth rates) of populations and finally abundance declines. We discuss how monitoring the sequential appearance of these signals may allow us to discern whether a population is increasingly at risk of collapse, or is adapting in the face of environmental change, providing a conceptual framework to develop new forecasting methods that combine multi-dimensional (for example, behaviour, morphology, life history and abundance) data. The authors outline a framework for predicting animal population collapse under external stressors, based on a predictable sequence of observable changes through time.},
  archive      = {J_NATEE},
  author       = {Cerini, Francesco and Childs, Dylan Z. and Clements, Christopher F.},
  doi          = {10.1038/s41559-023-01985-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {320-331},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A predictive timeline of wildlife population collapse},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Increasing body-size variation in many downsizing north
american mammals and birds. <em>NATEE</em>, <em>7</em>(3), 318–319. (<a
href="https://doi.org/10.1038/s41559-022-01969-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the period 1880 to 2020, intraspecific body-size variation increased in many mammal and bird species in North America, along with declines in average body size. These results suggest potential buffering effects against species downsizing and species capacity to cope with environmental change, but warn of an increasing possibility of maladaptation.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01969-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {318-319},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Increasing body-size variation in many downsizing north american mammals and birds},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Memory pays off. <em>NATEE</em>, <em>7</em>(3), 316–317. (<a
href="https://doi.org/10.1038/s41559-022-01968-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Laboratory-quantified spatial memory and subsequent free-ranging movements show how learning about space and establishing familiar areas increase fitness in pheasants.},
  archive      = {J_NATEE},
  author       = {Cagnacci, Francesca},
  doi          = {10.1038/s41559-022-01968-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {316-317},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Memory pays off},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Pollinators and the habitat fragmentation puzzle.
<em>NATEE</em>, <em>7</em>(3), 314–315. (<a
href="https://doi.org/10.1038/s41559-022-01977-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A study of plant–pollinator networks in a fragmented island landscape in China finds that habitat edges might not be bad for biodiversity.},
  archive      = {J_NATEE},
  author       = {Dodonov, Pavel and Cazetta, Eliana},
  doi          = {10.1038/s41559-022-01977-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {314-315},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pollinators and the habitat fragmentation puzzle},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Evolutionary justice. <em>NATEE</em>, <em>7</em>(3),
312–313. (<a href="https://doi.org/10.1038/s41559-022-01955-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Stuart, Yoel E.},
  doi          = {10.1038/s41559-022-01955-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {312-313},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Evolutionary justice},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Alma dal co (1989–2022). <em>NATEE</em>, <em>7</em>(3),
310–311. (<a href="https://doi.org/10.1038/s41559-022-01978-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A visionary and interdisciplinary scientist who brought a fearless passion to everything she did, inspiring all those around her.},
  archive      = {J_NATEE},
  author       = {van Vliet, Simon and Ackermann, Martin},
  doi          = {10.1038/s41559-022-01978-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {310-311},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Alma dal co (1989–2022)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Avoid the reproduction of coloniality in decolonial studies
in ecology. <em>NATEE</em>, <em>7</em>(3), 306–309. (<a
href="https://doi.org/10.1038/s41559-022-01971-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Calls to decolonize disciplines and institutions circulate in the scientific community. In ecology, only the surface of the colonial structure has been scratched. We propose that two gaps must be filled to decolonize the ‘decolonial turn’ itself: recognition of decolonial theories produced in the Global South and a deeper historical and socioeconomic analysis of forms of production and validation of knowledge in ecology.},
  archive      = {J_NATEE},
  author       = {Hirschfeld, María N. Clerici and Faria, Luiz R. R. and Fonseca, Carlos Roberto},
  doi          = {10.1038/s41559-022-01971-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {306-309},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Avoid the reproduction of coloniality in decolonial studies in ecology},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Fifty years of CITES. <em>NATEE</em>, <em>7</em>(3), 305.
(<a href="https://doi.org/10.1038/s41559-023-02017-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The text of the Convention on International Trade in Endangered Species of Wild Fauna and Flora (CITES) was agreed 50 years ago, and it continues to be a valuable tool for species protection and an important early example of an international environmental agreement.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-02017-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {3},
  number       = {3},
  pages        = {305},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Fifty years of CITES},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Ancient DNA reveals admixture history and endogamy in the
prehistoric aegean. <em>NATEE</em>, <em>7</em>(2), 290–303. (<a
href="https://doi.org/10.1038/s41559-022-01952-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Neolithic and Bronze Ages were highly transformative periods for the genetic history of Europe but for the Aegean—a region fundamental to Europe’s prehistory—the biological dimensions of cultural transitions have been elucidated only to a limited extent so far. We have analysed newly generated genome-wide data from 102 ancient individuals from Crete, the Greek mainland and the Aegean Islands, spanning from the Neolithic to the Iron Age. We found that the early farmers from Crete shared the same ancestry as other contemporaneous Neolithic Aegeans. In contrast, the end of the Neolithic period and the following Early Bronze Age were marked by ‘eastern’ gene flow, which was predominantly of Anatolian origin in Crete. Confirming previous findings for additional Central/Eastern European ancestry in the Greek mainland by the Middle Bronze Age, we additionally show that such genetic signatures appeared in Crete gradually from the seventeenth to twelfth centuries bc, a period when the influence of the mainland over the island intensified. Biological and cultural connectedness within the Aegean is also supported by the finding of consanguineous endogamy practiced at high frequencies, unprecedented in the global ancient DNA record. Our results highlight the potential of archaeogenomic approaches in the Aegean for unravelling the interplay of genetic admixture, marital and other cultural practices. The authors generate a genome-wide dataset of 102 individuals who lived in Crete, the Greek mainland and the Aegean islands between the Neolithic and the Iron Age, identifying high levels of biological and cultural connectedness within the ancient Aegean.},
  archive      = {J_NATEE},
  author       = {Skourtanioti, Eirini and Ringbauer, Harald and Gnecchi Ruscone, Guido Alberto and Bianco, Raffaela Angelina and Burri, Marta and Freund, Cäcilia and Furtwängler, Anja and Gomes Martins, Nuno Filipe and Knolle, Florian and Neumann, Gunnar U. and Tiliakou, Anthi and Agelarakis, Anagnostis and Andreadaki-Vlazaki, Maria and Betancourt, Philip and Hallager, Birgitta P. and Jones, Olivia A. and Kakavogianni, Olga and Kanta, Athanasia and Karkanas, Panagiotis and Kataki, Efthymia and Kissas, Konstantinos and Koehl, Robert and Kvapil, Lynne and Maran, Joseph and McGeorge, Photini J. P. and Papadimitriou, Alkestis and Papathanasiou, Anastasia and Papazoglou-Manioudaki, Lena and Paschalidis, Kostas and Polychronakou-Sgouritsa, Naya and Preve, Sofia and Prevedorou, Eleni-Anna and Price, Gypsy and Protopapadaki, Eftychia and Schmidt-Schultz, Tyede and Schultz, Michael and Shelton, Kim and Wiener, Malcolm H. and Krause, Johannes and Jeong, Choongwon and Stockhammer, Philipp W.},
  doi          = {10.1038/s41559-022-01952-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {290-303},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Ancient DNA reveals admixture history and endogamy in the prehistoric aegean},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Dietary strategies of pleistocene pongo sp. And homo erectus
on java (indonesia). <em>NATEE</em>, <em>7</em>(2), 279–289. (<a
href="https://doi.org/10.1038/s41559-022-01947-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {During the Early to Middle Pleistocene, Java was inhabited by hominid taxa of great diversity. However, their seasonal dietary strategies have never been explored. We undertook geochemical analyses of orangutan (Pongo sp.), Homo erectus and other mammalian Pleistocene teeth from Sangiran. We reconstructed past dietary strategies at subweekly resolution and inferred seasonal ecological patterns. Histologically controlled spatially resolved elemental analyses by laser-based plasma mass spectrometry confirmed the preservation of authentic biogenic signals despite the effect of spatially restricted diagenetic overprint. The Sr/Ca record of faunal remains is in line with expected trophic positions, contextualizing fossil hominid diet. Pongo sp. displays marked seasonal cycles with ~3 month-long strongly elevated Sr/Ca peaks, reflecting contrasting plant food consumption presumably during the monsoon season, while lower Sr/Ca ratios suggest different food availability during the dry season. In contrast, omnivorous H. erectus shows low and less accentuated intra-annual Sr/Ca variability compared to Pongo sp., with δ13C data of one individual indicating a dietary shift from C4 to a mix of C3 and C4 plants. Our data suggest that H. erectus on Java was maximizing the resources available in more open mosaic habitats and was less dependent on variations in seasonal resource availability. While still influenced by seasonal food availability, we infer that H. erectus was affected to a lesser degree than Pongo sp., which inhabited monsoonal rain forests on Java. We suggest that H. erectus maintained a greater degree of nutritional independence by exploiting the regional diversity of food resources across the seasons. Trace element ratios (strontium/calcium) in teeth of Pleistocene Homo erectus and fossil orangutans (Pongo sp.) reveal different dietary strategies and contrasting adaptations to seasonal food resources. H. erectus but not Pongo sp. was able to buffer against seasonal food oscillations by exploiting more varied food sources.},
  archive      = {J_NATEE},
  author       = {Kubat, Jülide and Nava, Alessia and Bondioli, Luca and Dean, M. Christopher and Zanolli, Clément and Bourgon, Nicolas and Bacon, Anne-Marie and Demeter, Fabrice and Peripoli, Beatrice and Albert, Richard and Lüdecke, Tina and Hertler, Christine and Mahoney, Patrick and Kullmer, Ottmar and Schrenk, Friedemann and Müller, Wolfgang},
  doi          = {10.1038/s41559-022-01947-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {279-289},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Dietary strategies of pleistocene pongo sp. and homo erectus on java (Indonesia)},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). De novo genes with an lncRNA origin encode unique human
brain developmental functionality. <em>NATEE</em>, <em>7</em>(2),
264–278. (<a href="https://doi.org/10.1038/s41559-022-01925-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human de novo genes can originate from neutral long non-coding RNA (lncRNA) loci and are evolutionarily significant in general, yet how and why this all-or-nothing transition to functionality happens remains unclear. Here, in 74 human/hominoid-specific de novo genes, we identified distinctive U1 elements and RNA splice-related sequences accounting for RNA nuclear export, differentiating mRNAs from lncRNAs, and driving the origin of de novo genes from lncRNA loci. The polymorphic sites facilitating the lncRNA–mRNA conversion through regulating nuclear export are selectively constrained, maintaining a boundary that differentiates mRNAs from lncRNAs. The functional new genes actively passing through it thus showed a mode of pre-adaptive origin, in that they acquire functions along with the achievement of their coding potential. As a proof of concept, we verified the regulations of splicing and U1 recognition on the nuclear export efficiency of one of these genes, the ENSG00000205704, in human neural progenitor cells. Notably, knock-out or over-expression of this gene in human embryonic stem cells accelerates or delays the neuronal maturation of cortical organoids, respectively. The transgenic mice with ectopically expressed ENSG00000205704 showed enlarged brains with cortical expansion. We thus demonstrate the key roles of nuclear export in de novo gene origin. These newly originated genes should reflect the novel uniqueness of human brain development. Studying human-specific de novo genes originated from long non-coding RNA, the authors reveal molecular mechanisms that facilitate nuclear export of these young genes, and show experimental evidence for the role of one such gene in brain development.},
  archive      = {J_NATEE},
  author       = {An, Ni A. and Zhang, Jie and Mo, Fan and Luan, Xuke and Tian, Lu and Shen, Qing Sunny and Li, Xiangshang and Li, Chunqiong and Zhou, Fanqi and Zhang, Boya and Ji, Mingjun and Qi, Jianhuan and Zhou, Wei-Zhen and Ding, Wanqiu and Chen, Jia-Yu and Yu, Jia and Zhang, Li and Shu, Shaokun and Hu, Baoyang and Li, Chuan-Yun},
  doi          = {10.1038/s41559-022-01925-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {264-278},
  shortjournal = {Nat. Ecol. Evol},
  title        = {De novo genes with an lncRNA origin encode unique human brain developmental functionality},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Implications of zero-deforestation palm oil for tropical
grassy and dry forest biodiversity. <em>NATEE</em>, <em>7</em>(2),
250–263. (<a href="https://doi.org/10.1038/s41559-022-01941-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many companies have made zero-deforestation commitments (ZDCs) to reduce carbon emissions and biodiversity losses linked to tropical commodities. However, ZDCs conserve areas primarily based on tree cover and aboveground carbon, potentially leading to the unintended consequence that agricultural expansion could be encouraged in biomes outside tropical rainforest, which also support important biodiversity. We examine locations suitable for zero-deforestation expansion of commercial oil palm, which is increasingly expanding outside the tropical rainforest biome, by generating empirical models of global suitability for rainfed and irrigated oil palm. We find that tropical grassy and dry forest biomes contain &amp;gt;50% of the total area of land climatically suitable for rainfed oil palm expansion in compliance with ZDCs (following the High Carbon Stock Approach; in locations outside urban areas and cropland), and that irrigation could double the area suitable for expansion in these biomes. Within these biomes, ZDCs fail to protect areas of high vertebrate richness from oil palm expansion. To prevent unintended consequences of ZDCs and minimize the environmental impacts of oil palm expansion, policies and governance for sustainable development and conservation must expand focus from rainforests to all tropical biomes. Modelling reveals large swathes of land in tropical grassy and dry forest biomes that are climatically suitable for commercial plantations of oil palm and would comply with current zero-deforestation commitments, but where conversion to oil palm would, in many locations, cause loss of habitat and biodiversity.},
  archive      = {J_NATEE},
  author       = {Fleiss, Susannah and Parr, Catherine L. and Platts, Philip J. and McClean, Colin J. and Beyer, Robert M. and King, Henry and Lucey, Jennifer M. and Hill, Jane K.},
  doi          = {10.1038/s41559-022-01941-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {250-263},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Implications of zero-deforestation palm oil for tropical grassy and dry forest biodiversity},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The supply of multiple ecosystem services requires
biodiversity across spatial scales. <em>NATEE</em>, <em>7</em>(2),
236–249. (<a href="https://doi.org/10.1038/s41559-022-01918-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The impact of local biodiversity loss on ecosystem functioning is well established, but the role of larger-scale biodiversity dynamics in the delivery of ecosystem services remains poorly understood. Here we address this gap using a comprehensive dataset describing the supply of 16 cultural, regulating and provisioning ecosystem services in 150 European agricultural grassland plots, and detailed multi-scale data on land use and plant diversity. After controlling for land-use and abiotic factors, we show that both plot-level and surrounding plant diversity play an important role in the supply of cultural and aboveground regulating ecosystem services. In contrast, provisioning and belowground regulating ecosystem services are more strongly driven by field-level management and abiotic factors. Structural equation models revealed that surrounding plant diversity promotes ecosystem services both directly, probably by fostering the spill-over of ecosystem service providers from surrounding areas, and indirectly, by maintaining plot-level diversity. By influencing the ecosystem services that local stakeholders prioritized, biodiversity at different scales was also shown to positively influence a wide range of stakeholder groups. These results provide a comprehensive picture of which ecosystem services rely most strongly on biodiversity, and the respective scales of biodiversity that drive these services. This key information is required for the upscaling of biodiversity–ecosystem service relationships, and the informed management of biodiversity within agricultural landscapes. The authors test whether spatial scale (plot, local and landscape) affects the supply of various ecosystem services in grasslands, finding that some services are predicted by plot-level properties while others depend more on landscape-level management.},
  archive      = {J_NATEE},
  author       = {Le Provost, Gaëtane and Schenk, Noëlle V. and Penone, Caterina and Thiele, Jan and Westphal, Catrin and Allan, Eric and Ayasse, Manfred and Blüthgen, Nico and Boeddinghaus, Runa S. and Boesing, Andrea Larissa and Bolliger, Ralph and Busch, Verena and Fischer, Markus and Gossner, Martin M. and Hölzel, Norbert and Jung, Kirsten and Kandeler, Ellen and Klaus, Valentin H. and Kleinebecker, Till and Leimer, Sophia and Marhan, Sven and Morris, Kathryn and Müller, Sandra and Neff, Felix and Neyret, Margot and Oelmann, Yvonne and Perović, David J. and Peter, Sophie and Prati, Daniel and Rillig, Matthias C. and Saiz, Hugo and Schäfer, Deborah and Scherer-Lorenzen, Michael and Schloter, Michael and Schöning, Ingo and Schrumpf, Marion and Steckel, Juliane and Steffan-Dewenter, Ingolf and Tschapka, Marco and Vogt, Juliane and Weiner, Christiane and Weisser, Wolfgang and Wells, Konstans and Werner, Michael and Wilcke, Wolfgang and Manning, Peter},
  doi          = {10.1038/s41559-022-01918-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {236-249},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The supply of multiple ecosystem services requires biodiversity across spatial scales},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Changing plant functional diversity over the last 12,000
years provides perspectives for tracking future changes in vegetation
communities. <em>NATEE</em>, <em>7</em>(2), 224–235. (<a
href="https://doi.org/10.1038/s41559-022-01943-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Plant communities are largely reshaped by climate and the environment over millennia, providing a powerful tool for understanding their response to future climates. Using a globally applicable functional palaeocological approach, we provide a deeper understanding of fossil pollen-inferred long-term response of vegetation to past climatic disturbances based on changes in functional trait composition. Specifically, we show how and why the ecological strategies exhibited by vegetation have changed through time by linking observations of plant traits to multiple pollen records from southeast Australia to reconstruct past functional diversity (FD, the value and the range of species traits that influence ecosystem functioning). The drivers of FD changes were assessed quantitatively by comparing FD reconstructions to independent records of past climates. During the last 12,000 years, peaks in FD were associated with both dry and wet climates in southeast Australia, with shifts in leaf traits particularly pronounced under wet conditions. Continentality determined the degree of stability maintained by high FD, with the greatest seen on the mainland. We expect projected frequent drier conditions in southeast Australia over coming decades to drive changes in vegetation community functioning and productivity mirroring the functional palaeocological record, particularly in western Tasmania and western southeast mainland. Linking plant traits to long-term pollen records over the Holocene, the authors track changes in functional diversity over the last 12,000 years in southeast Australia and use this to predict vegetation changes under future climate change.},
  archive      = {J_NATEE},
  author       = {Adeleye, Matthew Adesanya and Haberle, Simon Graeme and Gallagher, Rachael and Andrew, Samuel Charles and Herbert, Annika},
  doi          = {10.1038/s41559-022-01943-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {224-235},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Changing plant functional diversity over the last 12,000 years provides perspectives for tracking future changes in vegetation communities},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Resolving the intricate role of climate in litter
decomposition. <em>NATEE</em>, <em>7</em>(2), 214–223. (<a
href="https://doi.org/10.1038/s41559-022-01948-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {With approximately 60 Pg of carbon (C) released as CO2 annually, the decomposition of dead organic matter feeds the major terrestrial global CO2 flux to the atmosphere. Macroclimate control over this critical C flux facilitates the parametrization of the C cycle in Earth system models and the understanding of climate change effects on the global C balance. Yet, the long-standing paradigm of climate control was recently challenged by the so far underestimated environmental heterogeneity at local scales, questioning the conceptual framework of thousands of decomposition studies and accuracy of current predictive models. Using three complementary decomposition experiments at a European scale, we showed that macroclimate and litter characteristics largely control plant litter decomposition, reaffirming the role of macroclimate as an integrative decomposition driver through direct environmental control and by influencing co-evolving local plant and decomposer communities. Neglecting this latter indirect effect, commonly used standard litter types overrated micro-environmental control and failed to predict local decomposition of plot-specific litter. Our data help clarify a key question on the regulation of the global C cycle by identifying the relative role of control factors over decomposition and the scales at which they matter and by highlighting sources of confusion in the literature. Three complementary decomposition experiments across a climatic gradient in Europe, representing 110 different tree species mixtures in 194 forest plots, reveals that macroclimate is a dominant control on plant litter decomposition through both direct and indirect effects.},
  archive      = {J_NATEE},
  author       = {Joly, François-Xavier and Scherer-Lorenzen, Michael and Hättenschwiler, Stephan},
  doi          = {10.1038/s41559-022-01948-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {214-223},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Resolving the intricate role of climate in litter decomposition},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Temperature fluctuation promotes the thermal adaptation of
soil microbial respiration. <em>NATEE</em>, <em>7</em>(2), 205–213. (<a
href="https://doi.org/10.1038/s41559-022-01944-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The magnitude of the feedback between soil microbial respiration and increased mean temperature may decrease (a process called thermal adaptation) or increase over time, and accurately representing this feedback in models improves predictions of soil carbon loss rates. However, climate change entails changes not only in mean temperature but also in temperature fluctuation, and how this fluctuation regulates the thermal response of microbial respiration has never been systematically evaluated. By analysing subtropical forest soils from a 2,000 km transect across China, we showed that although a positive relationship between soil microbial biomass-specific respiration and temperature was observed under increased constant incubation temperature, an increasing temperature fluctuation had a stronger negative effect. Our results further indicated that changes in bacterial community composition and reduced activities of carbon degradation enzymes promoted the effect of temperature fluctuation. This adaptive response of soil microbial respiration suggests that climate warming may have a lesser exacerbating effect on atmospheric CO2 concentrations than predicted. Analysing subtropical forest soils from a 2,000 km transect across China, the authors show that temperature fluctuations can induce the thermal adaptation of microbial respiration, in contrast to findings derived from mean temperature alone.},
  archive      = {J_NATEE},
  author       = {Zhang, Yan and Li, Jin-Tao and Xu, Xiao and Chen, Hong-Yang and Zhu, Ting and Xu, Jian-Jun and Xu, Xiao-Ni and Li, Jin-Quan and Liang, Chao and Li, Bo and Fang, Chang-Ming and Nie, Ming},
  doi          = {10.1038/s41559-022-01944-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {205-213},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Temperature fluctuation promotes the thermal adaptation of soil microbial respiration},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Acclimation of phenology relieves leaf longevity constraints
in deciduous forests. <em>NATEE</em>, <em>7</em>(2), 198–204. (<a
href="https://doi.org/10.1038/s41559-022-01946-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Leaf phenology is key for regulating total growing-season mass and energy fluxes. Long-term temporal trends towards earlier leaf unfolding are observed across Northern Hemisphere forests. Phenological dates also vary between years, whereby end-of-season (EOS) dates correlate positively with start-of-season (SOS) dates and negatively with growing-season total net CO2 assimilation (Anet). These associations have been interpreted as the effect of a constrained leaf longevity or of premature carbon (C) sink saturation—with far-reaching consequences for long-term phenology projections under climate change and rising CO2. Here, we use multidecadal ground and remote-sensing observations to show that the relationships between Anet and EOS are opposite at the interannual and the decadal time scales. A decadal trend towards later EOS persists in parallel with a trend towards increasing Anet—in spite of the negative Anet–EOS relationship at the interannual scale. This finding is robust against the use of diverse observations and models. Results indicate that acclimation of phenology has enabled plants to transcend a constrained leaf longevity or premature C sink saturation over the course of several decades, leading to a more effective use of available light and a sustained extension of the vegetation CO2 uptake season over time. Multidecadal ground and remote-sensing observations of Northern Hemisphere forests show that, at the decadal scale, autumn senescence dates and total net carbon assimilation are positively related, despite a negative relationship at the annual scale. This suggests that acclimation relieves the leaf longevity constraints.},
  archive      = {J_NATEE},
  author       = {Marqués, Laura and Hufkens, Koen and Bigler, Christof and Crowther, Thomas W. and Zohner, Constantin M. and Stocker, Benjamin D.},
  doi          = {10.1038/s41559-022-01946-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {198-204},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Acclimation of phenology relieves leaf longevity constraints in deciduous forests},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Reply to: Erroneous predictions of auxotrophies by CarveMe.
<em>NATEE</em>, <em>7</em>(2), 196–197. (<a
href="https://doi.org/10.1038/s41559-022-01939-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Machado, Daniel and Patil, Kiran R.},
  doi          = {10.1038/s41559-022-01939-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {196-197},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Reply to: Erroneous predictions of auxotrophies by CarveMe},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Erroneous predictions of auxotrophies by CarveMe.
<em>NATEE</em>, <em>7</em>(2), 194–195. (<a
href="https://doi.org/10.1038/s41559-022-01936-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Price, Morgan},
  doi          = {10.1038/s41559-022-01936-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {194-195},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Erroneous predictions of auxotrophies by CarveMe},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Origins and diversification of animal innate immune
responses against viral infections. <em>NATEE</em>, <em>7</em>(2),
182–193. (<a href="https://doi.org/10.1038/s41559-022-01951-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Immune systems are of pivotal importance to any living organism on Earth, as they protect the organism against deleterious effects of viral infections. Though the current knowledge about these systems is still biased towards the immune response in vertebrates, some studies have focused on the identification and characterization of components of invertebrate antiviral immune systems. Two classic model organisms, the insect Drosophila melanogaster and the nematode Caenorhabditis elegans, were instrumental in the discovery of several important components of the innate immune system, such as the Toll-like receptors and the RNA interference pathway. However, these two model organisms provide only a limited view of the evolutionary history of the immune system, as they both are ecdysozoan protostomes. Recent functional studies in non-classic models such as unicellular holozoans (for example, choanoflagellates), lophotrochozoans (for example, oysters) and cnidarians (for example, sea anemones) have added crucial information for understanding the evolution of antiviral systems, as they revealed unexpected ancestral complexity. This Review aims to summarize this information and present the ancestral nature of the antiviral immune response in animals. We also discuss lineage-specific adaptations and future perspectives for the comparative study of the innate immune system that are essential for understanding its evolution. This Review collates the current understanding of the innate immune systems in classic model and non-model organisms, highlighting ancestral complexity and lineage-specific adaptations and identifying targets for future comparative studies.},
  archive      = {J_NATEE},
  author       = {Iwama, Rafael E. and Moran, Yehu},
  doi          = {10.1038/s41559-022-01951-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {182-193},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Origins and diversification of animal innate immune responses against viral infections},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). De novo gene increases brain size. <em>NATEE</em>,
<em>7</em>(2), 180–181. (<a
href="https://doi.org/10.1038/s41559-022-01942-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Comparative analysis of human and macaque brain transcripts together with experiments in mice and in a cortical organoid model show the de novo emergence of a hominoid-specific protein-coding gene implicated in brain development. The evolution of RNA nuclear export signals enabled a new protein to become translated from an ancestral long-noncoding RNA locus.},
  archive      = {J_NATEE},
  author       = {Rich, April and Carvunis, Anne-Ruxandra},
  doi          = {10.1038/s41559-022-01942-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {180-181},
  shortjournal = {Nat. Ecol. Evol},
  title        = {De novo gene increases brain size},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Sustainable palm oil puts grasslands at risk.
<em>NATEE</em>, <em>7</em>(2), 178–179. (<a
href="https://doi.org/10.1038/s41559-022-01913-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Zero-deforestation policies are reducing the loss of tropical rainforest to oil palm expansion, but spatial analyses indicate that this may cause unintended large-scale loss of biodiverse grasslands and dry forests unless protections are extended under certification agreements.},
  archive      = {J_NATEE},
  author       = {Gilroy, James J.},
  doi          = {10.1038/s41559-022-01913-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {178-179},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Sustainable palm oil puts grasslands at risk},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Scale matters in service supply. <em>NATEE</em>,
<em>7</em>(2), 176–177. (<a
href="https://doi.org/10.1038/s41559-022-01924-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An analysis of 16 ecosystem services measured across sites in Europe shows that the supply of some services is predicted by plot-scale diversity, whereas others rely on intact habitats at the landscape scale, highlighting the importance of cross-scale management efforts to maintain ecosystem services.},
  archive      = {J_NATEE},
  author       = {Chaplin-Kramer, Rebecca},
  doi          = {10.1038/s41559-022-01924-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {176-177},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Scale matters in service supply},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Functional vegetation change over millennia. <em>NATEE</em>,
<em>7</em>(2), 174–175. (<a
href="https://doi.org/10.1038/s41559-022-01949-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Analysis of regional-scale pollen data from southeast Australia that span the entire Holocene epoch reveals that plant functional diversity has been highly variable in time and space. A functional perspective on palaeoecological data helps us to better understand the current climate–biodiversity crisis and to predict future changes.},
  archive      = {J_NATEE},
  author       = {Reitalu, Triin and Nogué, Sandra},
  doi          = {10.1038/s41559-022-01949-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {174-175},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Functional vegetation change over millennia},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). The global biodiversity framework needs a robust action
agenda. <em>NATEE</em>, <em>7</em>(2), 172–173. (<a
href="https://doi.org/10.1038/s41559-022-01953-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Chan, Sander and Bauer, Steffen and Betsill, Michele M. and Biermann, Frank and Boran, Idil and Bridgewater, Peter and Bulkeley, Harriet and Bustamente, Mercedes M. C. and Deprez, Alexandra and Dodds, Felix and Hoffmann, Michael and Hornidge, Anna-Katharina and Hughes, Alice and Imbach, Pablo and Ivanova, Maria and Köberle, Alexandre and Kok, Marcel T. J. and Lwasa, Shuaib and Morrison, Tiffany and Pörtner, Hans-O. and Sari, Agus P. and VanDeveer, Stacy D. and Vollmer, Derek and Widerberg, Oscar and Pettorelli, Nathalie},
  doi          = {10.1038/s41559-022-01953-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {172-173},
  shortjournal = {Nat. Ecol. Evol},
  title        = {The global biodiversity framework needs a robust action agenda},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Climate science needs biology. <em>NATEE</em>,
<em>7</em>(2), 171. (<a
href="https://doi.org/10.1038/s41559-023-01991-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Incorporating biological feedbacks into climate and Earth system models is challenging owing to multiple sources of uncertainty. Filling these knowledge gaps and learning lessons from climate forecasts will help to improve our understanding of biodiversity under future environmental change.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-023-01991-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {2},
  number       = {2},
  pages        = {171},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Climate science needs biology},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Detecting macroevolutionary genotype–phenotype associations
using error-corrected rates of protein convergence. <em>NATEE</em>,
<em>7</em>(1), 155–170. (<a
href="https://doi.org/10.1038/s41559-022-01932-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {On macroevolutionary timescales, extensive mutations and phylogenetic uncertainty mask the signals of genotype–phenotype associations underlying convergent evolution. To overcome this problem, we extended the widely used framework of non-synonymous to synonymous substitution rate ratios and developed the novel metric ωC, which measures the error-corrected convergence rate of protein evolution. While ωC distinguishes natural selection from genetic noise and phylogenetic errors in simulation and real examples, its accuracy allows an exploratory genome-wide search of adaptive molecular convergence without phenotypic hypothesis or candidate genes. Using gene expression data, we explored over 20 million branch combinations in vertebrate genes and identified the joint convergence of expression patterns and protein sequences with amino acid substitutions in functionally important sites, providing hypotheses on undiscovered phenotypes. We further extended our method with a heuristic algorithm to detect highly repetitive convergence among computationally non-trivial higher-order phylogenetic combinations. Our approach allows bidirectional searches for genotype–phenotype associations, even in lineages that diverged for hundreds of millions of years. The authors develop a new metric to measure the error-corrected convergence rate of protein evolution, together with a heuristic algorithm to detect signals of adaptive protein convergence.},
  archive      = {J_NATEE},
  author       = {Fukushima, Kenji and Pollock, David D.},
  doi          = {10.1038/s41559-022-01932-7},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {155-170},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Detecting macroevolutionary genotype–phenotype associations using error-corrected rates of protein convergence},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mutualism-enhancing mutations dominate early adaptation in a
two-species microbial community. <em>NATEE</em>, <em>7</em>(1), 143–154.
(<a href="https://doi.org/10.1038/s41559-022-01923-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Species interactions drive evolution while evolution shapes these interactions. The resulting eco-evolutionary dynamics and their repeatability depend on how adaptive mutations available to community members affect fitness and ecologically relevant traits. However, the diversity of adaptive mutations is not well characterized, and we do not know how this diversity is affected by the ecological milieu. Here we use barcode lineage tracking to address this question in a community of yeast Saccharomyces cerevisiae and alga Chlamydomonas reinhardtii that have a net commensal relationship that results from a balance between competitive and mutualistic interactions. We find that yeast has access to many adaptive mutations with diverse ecological consequences, in particular those that increase and reduce the yields of both species. The presence of the alga does not change which mutations are adaptive in yeast (that is, there is no fitness trade-off for yeast between growing alone or with alga), but rather shifts selection to favour yeast mutants that increase the yields of both species and make the mutualism stronger. Thus, in the presence of the alga, adaptative mutations contending for fixation in yeast are more likely to enhance the mutualism, even though cooperativity is not directly favoured by natural selection in our system. Our results demonstrate that ecological interactions not only alter the trajectory of evolution but also dictate its repeatability; in particular, weak mutualisms can repeatably evolve to become stronger. Barcode lineage tracking of a competitive mutualism between Saccharomyces cerevisiae and Chlamydomonas reinhardtii shows that selection favours yeast mutants that increase the yields of both species and strengthen the mutualism.},
  archive      = {J_NATEE},
  author       = {Venkataram, Sandeep and Kuo, Huan-Yu and Hom, Erik F. Y. and Kryazhimskiy, Sergey},
  doi          = {10.1038/s41559-022-01923-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {143-154},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mutualism-enhancing mutations dominate early adaptation in a two-species microbial community},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Carbohydrate complexity limits microbial growth and reduces
the sensitivity of human gut communities to perturbations.
<em>NATEE</em>, <em>7</em>(1), 127–142. (<a
href="https://doi.org/10.1038/s41559-022-01930-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dietary fibre impacts the growth dynamics of human gut microbiota, yet we lack a detailed and quantitative understanding of how these nutrients shape microbial interaction networks and responses to perturbations. By building human gut communities coupled with computational modelling, we dissect the effects of fibres that vary in chemical complexity and each of their constituent sugars on community assembly and response to perturbations. We demonstrate that the degree of chemical complexity across different fibres limits microbial growth and the number of species that can utilize these nutrients. The prevalence of negative interspecies interactions is reduced in the presence of fibres compared with their constituent sugars. Carbohydrate chemical complexity enhances the reproducibility of community assembly and resistance of the community to invasion. We demonstrate that maximizing or minimizing carbohydrate competition between resident and invader species enhances resistance to invasion. In sum, the quantitative effects of carbohydrate chemical complexity on microbial interaction networks could be exploited to inform dietary and bacterial interventions to modulate community resistance to perturbations. Using synthetic human gut communities and computational modelling, the authors show that increasing the complexity of dietary carbohydrates reduces microbial growth, balances positive and negative interspecies interactions, and reduces community sensitivity to perturbations.},
  archive      = {J_NATEE},
  author       = {Ostrem Loss, Erin and Thompson, Jaron and Cheung, Pak Lun Kevin and Qian, Yili and Venturelli, Ophelia S.},
  doi          = {10.1038/s41559-022-01930-9},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {127-142},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Carbohydrate complexity limits microbial growth and reduces the sensitivity of human gut communities to perturbations},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Soil biodiversity supports the delivery of multiple
ecosystem functions in urban greenspaces. <em>NATEE</em>, <em>7</em>(1),
113–126. (<a href="https://doi.org/10.1038/s41559-022-01935-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {While the contribution of biodiversity to supporting multiple ecosystem functions is well established in natural ecosystems, the relationship of the above- and below-ground diversity with ecosystem multifunctionality remains virtually unknown in urban greenspaces. Here we conducted a standardized survey of urban greenspaces from 56 municipalities across six continents, aiming to investigate the relationships of plant and soil biodiversity (diversity of bacteria, fungi, protists and invertebrates, and metagenomics-based functional diversity) with 18 surrogates of ecosystem functions from nine ecosystem services. We found that soil biodiversity across biomes was significantly and positively correlated with multiple dimensions of ecosystem functions, and contributed to key ecosystem services such as microbially driven carbon pools, organic matter decomposition, plant productivity, nutrient cycling, water regulation, plant–soil mutualism, plant pathogen control and antibiotic resistance regulation. Plant diversity only indirectly influenced multifunctionality in urban greenspaces via changes in soil conditions that were associated with soil biodiversity. These findings were maintained after controlling for climate, spatial context, soil properties, vegetation and management practices. This study provides solid evidence that conserving soil biodiversity in urban greenspaces is key to supporting multiple dimensions of ecosystem functioning, which is critical for the sustainability of urban ecosystems and human wellbeing. In a widespread sampling campaign across urban soils, the authors find that soil biodiversity, but not plant diversity, is positively related to multiple ecosystem functions in urban environments.},
  archive      = {J_NATEE},
  author       = {Fan, Kunkun and Chu, Haiyan and Eldridge, David J. and Gaitan, Juan J. and Liu, Yu-Rong and Sokoya, Blessing and Wang, Jun-Tao and Hu, Hang-Wei and He, Ji-Zheng and Sun, Wei and Cui, Haiying and Alfaro, Fernando D. and Abades, Sebastian and Bastida, Felipe and Díaz-López, Marta and Bamigboye, Adebola R. and Berdugo, Miguel and Blanco-Pastor, José L. and Grebenc, Tine and Duran, Jorge and Illán, Javier G. and Makhalanyane, Thulani P. and Mukherjee, Arpan and Nahberger, Tina U. and Peñaloza-Bojacá, Gabriel F. and Plaza, César and Verma, Jay Prakash and Rey, Ana and Rodríguez, Alexandra and Siebe, Christina and Teixido, Alberto L. and Trivedi, Pankaj and Wang, Ling and Wang, Jianyong and Yang, Tianxue and Zhou, Xin-Quan and Zhou, Xiaobing and Zaady, Eli and Tedersoo, Leho and Delgado-Baquerizo, Manuel},
  doi          = {10.1038/s41559-022-01935-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {113-126},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Soil biodiversity supports the delivery of multiple ecosystem functions in urban greenspaces},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Plant–pollinator network change across a century in the
subarctic. <em>NATEE</em>, <em>7</em>(1), 102–112. (<a
href="https://doi.org/10.1038/s41559-022-01928-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Animal-mediated pollination is a vital ecosystem service to crops and wild plants, and long-term stability of plant–pollinator interactions is therefore crucial for maintaining plant biodiversity and food security. However, it is unknown how the composition of pollinators and the structure of pollinator interactions have changed across longer time spans relevant to examining responses to human activities such as climate change. We resampled an historical dataset of plant–pollinator interactions across several orders of pollinating insects in a subarctic location in Finland that has already experienced substantial climate warming but little land use change. Our results reveal a dramatic turnover in pollinator species and rewiring of plant–pollinator interactions, with only 7% of the interactions shared across time points. The relative abundance of moth and hoverfly pollinators declined between time points, whereas muscoid flies, a group for which little is known regarding conservation status and responses to climate, became more common. Specialist pollinators disproportionately declined, leading to a decrease in network-level specialization, which could have harmful consequences for pollination services. Our results exemplify the changes in plant–pollinator networks that might be expected in other regions as climate change progresses. The authors resample a plant–pollinator network that was initially characterized by a naturalist in the late nineteenth century in Finland. They find that only 7% of the original interactions persisted; generally, specialist pollinators disproportionally declined while generalist muscoid flies increased as the abundant pollinators.},
  archive      = {J_NATEE},
  author       = {Zoller, Leana and Bennett, Joanne and Knight, Tiffany M.},
  doi          = {10.1038/s41559-022-01928-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {102-112},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Plant–pollinator network change across a century in the subarctic},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Rare and declining bird species benefit most from
designating protected areas for conservation in the UK. <em>NATEE</em>,
<em>7</em>(1), 92–101. (<a
href="https://doi.org/10.1038/s41559-022-01927-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There have been recent renewed commitments to increase the extent of protected areas to combat the growing biodiversity crisis but the underpinning evidence for their effectiveness is mixed and causal connections are rarely evaluated. We used data gathered by three large-scale citizen science programmes in the UK to provide the most comprehensive assessment to date of whether national (Sites of Special Scientific Interest) and European (Special Protection Areas/Special Areas of Conservation) designated areas are associated with improved state (occurrence, abundance), change (rates of colonization, persistence and trend in abundance), community structure and, uniquely, demography (productivity) on a national avifauna, while controlling for differences in land cover, elevation and climate. We found positive associations with state that suggest these areas are well targeted and that the greatest benefit accrued to the most conservation-dependent species since positive associations with change were largely restricted to rare and declining species and habitat specialists. We suggest that increased productivity provides a plausible demographic mechanism for positive effects of designation. Data from three citizen science programmes on the occurrence, abundance, community structure and demography of birds across the UK show that protected areas are associated with improved state for most species and provide the most benefit to specialist, rare and declining species.},
  archive      = {J_NATEE},
  author       = {Barnes, A. E. and Davies, J. G. and Martay, B. and Boersch-Supan, P. H. and Harris, S. J. and Noble, D. G. and Pearce-Higgins, J. W. and Robinson, R. A.},
  doi          = {10.1038/s41559-022-01927-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {92-101},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Rare and declining bird species benefit most from designating protected areas for conservation in the UK},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Terrestrial invasive species alter marine vertebrate
behaviour. <em>NATEE</em>, <em>7</em>(1), 82–91. (<a
href="https://doi.org/10.1038/s41559-022-01931-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human-induced environmental changes, such as the introduction of invasive species, are driving declines in the movement of nutrients across ecosystems with negative consequences for ecosystem function. Declines in nutrient inputs could thus have knock-on effects at higher trophic levels and broader ecological scales, yet these interconnections remain relatively unknown. Here we show that a terrestrial invasive species (black rats, Rattus rattus) disrupts a nutrient pathway provided by seabirds, ultimately altering the territorial behaviour of coral reef fish. In a replicated ecosystem-scale natural experiment, we found that reef fish territories were larger and the time invested in aggression lower on reefs adjacent to rat-infested islands compared with rat-free islands. This response reflected changes in the economic defendability of lower-quality resources, with reef fish obtaining less nutritional gain per unit foraging effort adjacent to rat-infested islands with low seabird populations. These results provide a novel insight into how the disruption of nutrient flows by invasive species can affect variation in territorial behaviour. Rat eradication as a conservation strategy therefore has the potential to restore species interactions via territoriality, which can scale up to influence populations and communities at higher ecological levels. In a replicated ecosystem-scale natural experiment across ten islands in the Indian Ocean, invasive black rats disrupted nutrients provided by seabirds, leading to a coral reef fish having larger territories and investing less time in aggression than on rat-free islands.},
  archive      = {J_NATEE},
  author       = {Gunn, Rachel L. and Benkwitt, Cassandra E. and Graham, Nicholas A. J. and Hartley, Ian R. and Algar, Adam C. and Keith, Sally A.},
  doi          = {10.1038/s41559-022-01931-8},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {82-91},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Terrestrial invasive species alter marine vertebrate behaviour},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Benthic composition changes on coral reefs at global scales.
<em>NATEE</em>, <em>7</em>(1), 71–81. (<a
href="https://doi.org/10.1038/s41559-022-01937-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Globally, ecosystems are being reconfigured by a range of intensifying human-induced stressors. Coral reefs are at the forefront of this environmental transformation, and if we are to secure their key ecosystem functions and services, it is important to understand the likely configuration of future reefs. However, the composition and trajectory of global coral reef benthic communities is currently unclear. Here our global dataset of 24,468 observations spanning 22 years (1997–2018) revealed that particularly marked declines in coral cover occurred in the Western Atlantic and Central Pacific. The data also suggest that high macroalgal cover, widely regarded as the major degraded state on coral reefs, is a phenomenon largely restricted to the Western Atlantic. At a global scale, the raw data suggest decreased average (± standard error of the mean) hard coral cover from 36 ± 1.4% to 19 ± 0.4% (during a period delineated by the first global coral bleaching event (1998) until the end of the most recent event (2017)) was largely associated with increased low-lying algal cover such as algal turfs and crustose coralline algae. Enhanced understanding of reef change, typified by decreased hard coral cover and increased cover of low-lying algal communities, will be key to managing Anthropocene coral reefs. Analysing a global dataset of &amp;gt;24,000 observations of coral reef benthic cover, the authors show that high macroalgal cover is largely restricted to the Western Atlantic, where alongside the Central Pacific there have also been marked declines in coral cover since the late 1990s.},
  archive      = {J_NATEE},
  author       = {Tebbett, Sterling B. and Connolly, Sean R. and Bellwood, David R.},
  doi          = {10.1038/s41559-022-01937-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {71-81},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Benthic composition changes on coral reefs at global scales},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Challenges and opportunities for achieving sustainable
development goals through restoration of indonesia’s mangroves.
<em>NATEE</em>, <em>7</em>(1), 62–70. (<a
href="https://doi.org/10.1038/s41559-022-01926-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Indonesia, the most mangrove-rich nation in the world, has proposed the most globally ambitious mangrove rehabilitation target (600,000 ha) of any nation, to be achieved by 2024 to support multiple Sustainable Development Goals (SDG 1–3, 6, 13 and 14). Yet, mangrove restoration and rehabilitation across the world have often suffered low success rates and been applied at small scales. Here, we identify 193,367 ha (estimated costs at US$0.29–1.74 billion) that have the potential to align with the national mangrove rehabilitation programme. Despite being only 30% of the national target, our robust assessment considered biogeomorphology, 20 years of land-use and land-cover change and state forest land status, all key factors moderating mangrove restoration success which have often been neglected in Indonesia. Increasing subnational government representation in mangrove governance as well as improving monitoring and evaluation will increase the likelihood of achieving the mangrove rehabilitation targets and reduce risks of failure. Rehabilitating and conserving mangroves in Indonesia could benefit 74 million coastal people and can potentially contribute to the national land-sector emissions reduction of up to 16%. An Indonesia-wide analysis identifies locations for potential mangrove restoration, ranked by scenarios of success likelihood according to biogeomorphology, current and past land use and land tenure, and estimates the restoration costs.},
  archive      = {J_NATEE},
  author       = {Sasmito, Sigit D. and Basyuni, Mohammad and Kridalaksana, Age and Saragi-Sasmito, Meli F. and Lovelock, Catherine E. and Murdiyarso, Daniel},
  doi          = {10.1038/s41559-022-01926-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {62-70},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Challenges and opportunities for achieving sustainable development goals through restoration of indonesia’s mangroves},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Mapping the planet’s critical natural assets.
<em>NATEE</em>, <em>7</em>(1), 51–61. (<a
href="https://doi.org/10.1038/s41559-022-01934-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sustaining the organisms, ecosystems and processes that underpin human wellbeing is necessary to achieve sustainable development. Here we define critical natural assets as the natural and semi-natural ecosystems that provide 90% of the total current magnitude of 14 types of nature’s contributions to people (NCP), and we map the global locations of these critical natural assets at 2 km resolution. Critical natural assets for maintaining local-scale NCP (12 of the 14 NCP) account for 30% of total global land area and 24% of national territorial waters, while 44% of land area is required to also maintain two global-scale NCP (carbon storage and moisture recycling). These areas overlap substantially with cultural diversity (areas containing 96% of global languages) and biodiversity (covering area requirements for 73% of birds and 66% of mammals). At least 87% of the world’s population live in the areas benefitting from critical natural assets for local-scale NCP, while only 16% live on the lands containing these assets. Many of the NCP mapped here are left out of international agreements focused on conserving species or mitigating climate change, yet this analysis shows that explicitly prioritizing critical natural assets and the NCP they provide could simultaneously advance development, climate and conservation goals. Bringing together multiple models and databases on nature’s contributions to people, the authors map these contributions globally and determine the critical areas where their magnitude is the highest and where they provide the highest potential human benefit.},
  archive      = {J_NATEE},
  author       = {Chaplin-Kramer, Rebecca and Neugarten, Rachel A. and Sharp, Richard P. and Collins, Pamela M. and Polasky, Stephen and Hole, David and Schuster, Richard and Strimas-Mackey, Matthew and Mulligan, Mark and Brandon, Carter and Diaz, Sandra and Fluet-Chouinard, Etienne and Gorenflo, L. J. and Johnson, Justin A. and Kennedy, Christina M. and Keys, Patrick W. and Longley-Wood, Kate and McIntyre, Peter B. and Noon, Monica and Pascual, Unai and Reidy Liermann, Catherine and Roehrdanz, Patrick R. and Schmidt-Traub, Guido and Shaw, M. Rebecca and Spalding, Mark and Turner, Will R. and van Soesbergen, Arnout and Watson, Reg A.},
  doi          = {10.1038/s41559-022-01934-5},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {51-61},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mapping the planet’s critical natural assets},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Homo sapiens and neanderthals share high cerebral cortex
integration into adulthood. <em>NATEE</em>, <em>7</em>(1), 42–50. (<a
href="https://doi.org/10.1038/s41559-022-01933-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There is controversy around the mechanisms that guided the change in brain shape during the evolution of modern humans. It has long been held that different cortical areas evolved independently from each other to develop their unique functional specializations. However, some recent studies suggest that high integration between different cortical areas could facilitate the emergence of equally extreme, highly specialized brain functions. Here, we analyse the evolution of brain shape in primates using three-dimensional geometric morphometrics of endocasts. We aim to determine, firstly, whether modern humans present unique developmental patterns of covariation between brain cortical areas; and secondly, whether hominins experienced unusually high rates of evolution in brain covariation as compared to other primates. On the basis of analyses including modern humans and other extant great apes at different developmental stages, we first demonstrate that, unlike our closest living relatives, Homo sapiens retain high levels of covariation between cortical areas into adulthood. Among the other great apes, high levels of covariation are only found in immature individuals. Secondly, at the macro-evolutionary level, our analysis of 400 endocasts, representing 148 extant primate species and 6 fossil hominins, shows that strong covariation between different areas of the brain in H. sapiens and Homo neanderthalensis evolved under distinctly higher evolutionary rates than in any other primate, suggesting that natural selection favoured a greatly integrated brain in both species. These results hold when extinct species are excluded and allometric effects are accounted for. Our findings demonstrate that high covariation in the brain may have played a critical role in the evolution of unique cognitive capacities and complex behaviours in both modern humans and Neanderthals. Comparing the brain anatomy of fossil hominins and extant primates, the authors determine that strong covariation between different areas of the brain in Homo sapiens and Homo neanderthalensis evolved under higher evolutionary rates than in any other primate. Strong covariation is present in juvenile and adult H. sapiens, and in juvenile but not adult great apes.},
  archive      = {J_NATEE},
  author       = {Sansalone, Gabriele and Profico, Antonio and Wroe, Stephen and Allen, Kari and Ledogar, Justin and Ledogar, Sarah and Mitchell, Dave Rex and Mondanaro, Alessandro and Melchionna, Marina and Castiglione, Silvia and Serio, Carmela and Raia, Pasquale},
  doi          = {10.1038/s41559-022-01933-6},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {42-50},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Homo sapiens and neanderthals share high cerebral cortex integration into adulthood},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Arched footprints preserve the motions of fossil hominin
feet. <em>NATEE</em>, <em>7</em>(1), 32–41. (<a
href="https://doi.org/10.1038/s41559-022-01929-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The longitudinal arch of the human foot is viewed as a pivotal adaptation for bipedal walking and running. Fossil footprints from Laetoli, Tanzania, and Ileret, Kenya, are believed to provide direct evidence of longitudinally arched feet in hominins from the Pliocene and Pleistocene, respectively. We studied the dynamics of track formation using biplanar X-ray, three-dimensional animation and discrete element particle simulation. Here, we demonstrate that longitudinally arched footprints are false indicators of foot anatomy; instead they are generated through a specific pattern of foot kinematics that is characteristic of human walking. Analyses of fossil hominin tracks from Laetoli show only partial evidence of this walking style, with a similar heel strike but a different pattern of propulsion. The earliest known evidence for fully modern human-like bipedal kinematics comes from the early Pleistocene Ileret tracks, which were presumably made by members of the genus Homo. This result signals important differences in the foot kinematics recorded at Laetoli and Ileret and underscores an emerging picture of locomotor diversity within the hominin clade. Combining modelling of living human participants and chimpanzees with analysis of fossil hominin trackways, the authors distinguish between the earliest evidence of modern human-like bipedal kinematics and earlier hominin precursors.},
  archive      = {J_NATEE},
  author       = {Hatala, Kevin G. and Gatesy, Stephen M. and Falkingham, Peter L.},
  doi          = {10.1038/s41559-022-01929-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {32-41},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Arched footprints preserve the motions of fossil hominin feet},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Decoupling the skull and skeleton in a cretaceous bird with
unique appendicular morphologies. <em>NATEE</em>, <em>7</em>(1), 20–31.
(<a href="https://doi.org/10.1038/s41559-022-01921-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Cretaceous is a critical time interval that encompasses explosive diversifications of terrestrial vertebrates, particularly the period when the earliest-branching birds, after divergence from their theropod ancestors, evolved the characteristic avian Bauplan that led eventually to their global radiation. This early phylogenetic diversity is overwhelmed by the Ornithothoraces, consisting of the Enantiornithes and Ornithuromorpha, whose members evolved key derived features of crown birds. This disparity consequently circumscribes a large morphological gap between these derived clades and the oldest bird Archaeopteryx. The non-ornithothoracine pygostylians, with an intermediate phylogenetic position, are key to deciphering those evolutionary transformations, but progress in their study has been hampered by the limited diversity of known fossils. Here we report an Early Cetaceous non-ornithothoracine pygostylian, Cratonavis zhui gen. et sp. nov., that exhibits a unique combination of a non-avialan dinosaurian akinetic skull with an avialan post-cranial skeleton, revealing the key role of evolutionary mosaicism in early bird diversification. The unusually elongated scapular and metatarsal one preserved in Cratonavis highlights a breadth of skeletal plasticity, stemming from their distinct developmental modules and selection for possibly raptorial behaviour. Mapped changes in these two elements across theropod phylogeny demonstrate clade-specific evolutionary lability. Cratonavis zhui, a bird from the Early Cretaceous of China, preserves a combination of non-avialan theropod skull features and a bird-like post-cranial skeleton.},
  archive      = {J_NATEE},
  author       = {Li, Zhiheng and Wang, Min and Stidham, Thomas A. and Zhou, Zhonghe},
  doi          = {10.1038/s41559-022-01921-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {20-31},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Decoupling the skull and skeleton in a cretaceous bird with unique appendicular morphologies},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). A late devonian actinopterygian suggests high lineage
survivorship across the end-devonian mass extinction. <em>NATEE</em>,
<em>7</em>(1), 10–19. (<a
href="https://doi.org/10.1038/s41559-022-01919-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many accounts of the early history of actinopterygians (ray-finned fishes) posit that the end-Devonian mass extinction had a major influence on their evolution. Existing phylogenies suggest this episode could have acted as a bottleneck, paring the early diversity of the group to a handful of survivors. This picture, coupled with increases in taxonomic and morphological diversity in the Carboniferous, contributes to a model of explosive post-extinction radiation. However, most actinopterygians from within a roughly 20-million year (Myr) window surrounding the extinction are poorly known, contributing to uncertainty about the meaning of these patterns. Here, we report an exceptionally preserved fossil from 7 Myr before the extinction that reveals unexpected anatomical features. Palaeoneiros clackorum gen. et sp. nov. nests within a clade of post-Devonian species and, in an expanded phylogenetic analysis, draws multiple lineages of Carboniferous actinopterygians into the Devonian. This suggests cryptic but extensive lineage diversification in the latest Devonian, followed by more conspicuous feeding and locomotor structure diversification in the Carboniferous. Our revised model matches more complex patterns of divergence, survival and diversification around the Devonian/Carboniferous boundary in other vertebrate clades. It also fundamentally recalibrates the onset of diversification early in the history of this major radiation. An actinopterygian fossil with unexpected anatomical features from 7 million years before the end-Devonian mass extinction suggests complex patterns of divergence and diversification around the Devonian/Carboniferous boundary for ray-finned fishes.},
  archive      = {J_NATEE},
  author       = {Giles, Sam and Feilich, Kara and Warnock, Rachel C. M. and Pierce, Stephanie E. and Friedman, Matt},
  doi          = {10.1038/s41559-022-01919-4},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {10-19},
  shortjournal = {Nat. Ecol. Evol},
  title        = {A late devonian actinopterygian suggests high lineage survivorship across the end-devonian mass extinction},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023d). Mitigating a mine. <em>NATEE</em>, <em>7</em>(1), 9. (<a
href="https://doi.org/10.1038/s41559-022-01965-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-022-01965-y},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {9},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mitigating a mine},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Indigenous-led recovery. <em>NATEE</em>, <em>7</em>(1), 8.
(<a href="https://doi.org/10.1038/s41559-022-01961-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Harold, Simon},
  doi          = {10.1038/s41559-022-01961-2},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {8},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Indigenous-led recovery},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023f). Protection for migratory species. <em>NATEE</em>,
<em>7</em>(1), 7. (<a
href="https://doi.org/10.1038/s41559-022-01966-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Turner, Marian},
  doi          = {10.1038/s41559-022-01966-x},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {7},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Protection for migratory species},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023b). Leaf-like butterflies. <em>NATEE</em>, <em>7</em>(1), 6.
(<a href="https://doi.org/10.1038/s41559-022-01960-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-022-01960-3},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {6},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Leaf-like butterflies},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Mutations are not random. <em>NATEE</em>, <em>7</em>(1), 5.
(<a href="https://doi.org/10.1038/s41559-022-01959-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Domingues, Vera},
  doi          = {10.1038/s41559-022-01959-w},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {5},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Mutations are not random},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Tracking amber trends. <em>NATEE</em>, <em>7</em>(1), 4.
(<a href="https://doi.org/10.1038/s41559-022-01963-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Nic Eoin, Luíseach},
  doi          = {10.1038/s41559-022-01963-0},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {4},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Tracking amber trends},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023a). Big bird data. <em>NATEE</em>, <em>7</em>(1), 3. (<a
href="https://doi.org/10.1038/s41559-022-01962-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {Goymer, Patrick},
  doi          = {10.1038/s41559-022-01962-1},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {3},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Big bird data},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023c). Pathogens in a warming world. <em>NATEE</em>,
<em>7</em>(1), 2. (<a
href="https://doi.org/10.1038/s41559-022-01964-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATEE},
  author       = {McKay, Alexa},
  doi          = {10.1038/s41559-022-01964-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {2},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Pathogens in a warming world},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
<li><details>
<summary>
(2023). Remembering wallace. <em>NATEE</em>, <em>7</em>(1), 1. (<a
href="https://doi.org/10.1038/s41559-022-01972-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We celebrate Alfred Russel Wallace on the bicentenary of his birth.},
  archive      = {J_NATEE},
  doi          = {10.1038/s41559-022-01972-z},
  journal      = {Nature Ecology &amp; Evolution},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Ecol. Evol},
  title        = {Remembering wallace},
  volume       = {7},
  year         = {2023},
}
</textarea>
</details></li>
</ul>

</body>
</html>
