<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATCS_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natcs---164">NATCS - 164</h2>
<ul>
<li><details>
<summary>
(2024a). Author correction: Joint inference of discrete cell types
and continuous type-specific variability in single-cell datasets with
MMIDAS. <em>NATCS</em>, <em>4</em>(12), 991. (<a
href="https://doi.org/10.1038/s43588-024-00759-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Marghi, Yeganeh and Gala, Rohan and Baftizadeh, Fahimeh and Sümbül, Uygar},
  doi          = {10.1038/s43588-024-00759-5},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {991},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Joint inference of discrete cell types and continuous type-specific variability in single-cell datasets with MMIDAS},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An integrative data-driven model simulating c. Elegans
brain, body and environment interactions. <em>NATCS</em>,
<em>4</em>(12), 978–990. (<a
href="https://doi.org/10.1038/s43588-024-00738-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The behavior of an organism is influenced by the complex interplay between its brain, body and environment. Existing data-driven models focus on either the brain or the body–environment. Here we present BAAIWorm, an integrative data-driven model of Caenorhabditis elegans, which consists of two submodels: the brain model and the body–environment model. The brain model was built by multicompartment models with realistic morphology, connectome and neural population dynamics based on experimental data. Simultaneously, the body–environment model used a lifelike body and a three-dimensional physical environment. Through the closed-loop interaction between the two submodels, BAAIWorm reproduced the realistic zigzag movement toward attractors observed in C. elegans. Leveraging this model, we investigated the impact of neural system structure on both neural activities and behaviors. Consequently, BAAIWorm can enhance our understanding of how the brain controls the body to interact with its surrounding environment. BAAIWorm is an integrative data-driven model of C. elegans that simulates interactions between the brain, body and environment. The biophysically detailed neuronal model is capable of replicating the zigzag movement observed in this species.},
  archive      = {J_NATCS},
  author       = {Zhao, Mengdi and Wang, Ning and Jiang, Xinrui and Ma, Xiaoyang and Ma, Haixin and He, Gan and Du, Kai and Ma, Lei and Huang, Tiejun},
  doi          = {10.1038/s43588-024-00738-w},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {978-990},
  shortjournal = {Nat. Comput. Sci},
  title        = {An integrative data-driven model simulating c. elegans brain, body and environment interactions},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Mapping the gene space at single-cell resolution with gene
signal pattern analysis. <em>NATCS</em>, <em>4</em>(12), 955–977. (<a
href="https://doi.org/10.1038/s43588-024-00734-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In single-cell sequencing analysis, several computational methods have been developed to map the cellular state space, but little has been done to map or create embeddings of the gene space. Here we formulate the gene embedding problem, design tasks with simulated single-cell data to evaluate representations, and establish ten relevant baselines. We then present a graph signal processing approach, called gene signal pattern analysis (GSPA), that learns rich gene representations from single-cell data using a dictionary of diffusion wavelets on the cell–cell graph. GSPA enables characterization of genes based on their patterning and localization on the cellular manifold. We motivate and demonstrate the efficacy of GSPA as a framework for diverse biological tasks, such as capturing gene co-expression modules, condition-specific enrichment and perturbation-specific gene–gene interactions. Then we showcase the broad utility of gene representations derived from GSPA, including for cell–cell communication (GSPA-LR), spatial transcriptomics (GSPA-multimodal) and patient response (GSPA-Pt) analysis. This work presents a graph signal processing method, gene signal pattern analysis, to embed gene signals from single-cell sequencing data. In diverse experimental set-ups and case studies, GSPA establishes a gene-based framework for single-cell analysis.},
  archive      = {J_NATCS},
  author       = {Venkat, Aarthi and Leone, Sam and Youlten, Scott E. and Fagerberg, Eric and Attanasio, John and Joshi, Nikhil S. and Perlmutter, Michael and Krishnaswamy, Smita},
  doi          = {10.1038/s43588-024-00734-0},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {955-977},
  shortjournal = {Nat. Comput. Sci},
  title        = {Mapping the gene space at single-cell resolution with gene signal pattern analysis},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Interpreting single-cell and spatial omics data using deep
neural network training dynamics. <em>NATCS</em>, <em>4</em>(12),
941–954. (<a href="https://doi.org/10.1038/s43588-024-00721-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell and spatial omics datasets can be organized and interpreted by annotating single cells to distinct types, states, locations or phenotypes. However, cell annotations are inherently ambiguous, as discrete labels with subjective interpretations are assigned to heterogeneous cell populations on the basis of noisy, sparse and high-dimensional data. Here we developed Annotatability, a framework for identifying annotation mismatches and characterizing biological data structure by monitoring the dynamics and difficulty of training a deep neural network over such annotated data. Following this, we developed a signal-aware graph embedding method that enables downstream analysis of biological signals. This embedding captures cellular communities associated with target signals. Using Annotatability, we address key challenges in the interpretation of genomic data, demonstrated over eight single-cell RNA sequencing and spatial omics datasets, including identifying erroneous annotations and intermediate cell states, delineating developmental or disease trajectories, and capturing cellular heterogeneity. These results underscore the broad applicability of annotation-trainability analysis via Annotatability for unraveling cellular diversity and interpreting collective cell behaviors in health and disease. The Annotatability framework analyzes neural network training dynamics to interpret single-cell and spatial omics data. It identifies erroneous annotations and ambiguous cell states, infers trajectories from binary labels and enables signal-aware analysis.},
  archive      = {J_NATCS},
  author       = {Karin, Jonathan and Mintz, Reshef and Raveh, Barak and Nitzan, Mor},
  doi          = {10.1038/s43588-024-00721-5},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {941-954},
  shortjournal = {Nat. Comput. Sci},
  title        = {Interpreting single-cell and spatial omics data using deep neural network training dynamics},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A scalable framework for learning the geometry-dependent
solution operators of partial differential equations. <em>NATCS</em>,
<em>4</em>(12), 928–940. (<a
href="https://doi.org/10.1038/s43588-024-00732-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Solving partial differential equations (PDEs) using numerical methods is a ubiquitous task in engineering and medicine. However, the computational costs can be prohibitively high when many-query evaluations of PDE solutions on multiple geometries are needed. Here we aim to address this challenge by introducing Diffeomorphic Mapping Operator Learning (DIMON), a generic artificial intelligence framework that learns geometry-dependent solution operators of different types of PDE on a variety of geometries. We present several examples to demonstrate the performance, efficiency and scalability of the framework in learning both static and time-dependent PDEs on parameterized and non-parameterized domains; these include solving the Laplace equations, reaction–diffusion equations and a system of multiscale PDEs that characterize the electrical propagation on thousands of personalized heart digital twins. DIMON can reduce the computational costs of solution approximations on multiple geometries from hours to seconds with substantially less computational resources. This work presents an artificial intelligence framework to learn geometry-dependent solution operators of partial differential equations (PDEs). The framework enables scalable and fast approximations of PDE solutions on a variety of 3D geometries.},
  archive      = {J_NATCS},
  author       = {Yin, Minglang and Charon, Nicolas and Brody, Ryan and Lu, Lu and Trayanova, Natalia and Maggioni, Mauro},
  doi          = {10.1038/s43588-024-00732-2},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {928-940},
  shortjournal = {Nat. Comput. Sci},
  title        = {A scalable framework for learning the geometry-dependent solution operators of partial differential equations},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Real-time non-line-of-sight computational imaging using
spectrum filtering and motion compensation. <em>NATCS</em>,
<em>4</em>(12), 920–927. (<a
href="https://doi.org/10.1038/s43588-024-00722-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Non-line-of-sight (NLOS) imaging aims at recovering the shape and albedo of hidden objects. Despite recent advances, real-time video of complex and dynamic scenes remains a major challenge owing to the weak signal of multiply scattered light. Here we propose and demonstrate a framework of spectrum filtering and motion compensation to realize high-quality NLOS video for room-sized scenes. Spectrum filtering leverages a wave-based model for denoising and deblurring in the frequency domain, enabling computational image reconstruction with a small number of sampling points. Motion compensation tailored with an interleaved scanning scheme can compute high-resolution live video during the acquisition of low-quality image sequences. Together, we demonstrate live NLOS videos at 4 fps for a variety of dynamic real-life scenes. The results mark a substantial stride toward real-time, large-scale and low-power NLOS imaging and sensing applications. The authors propose a framework incorporating spectrum filtering and motion compensation, which enables non-line-of-sight live videos at 4 fps for a variety of dynamic real-life scenes.},
  archive      = {J_NATCS},
  author       = {Ye, Jun-Tian and Sun, Yi and Li, Wenwen and Zeng, Jian-Wei and Hong, Yu and Li, Zheng-Ping and Huang, Xin and Xue, Xianghui and Yuan, Xin and Xu, Feihu and Dou, Xiankang and Pan, Jian-Wei},
  doi          = {10.1038/s43588-024-00722-4},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {920-927},
  shortjournal = {Nat. Comput. Sci},
  title        = {Real-time non-line-of-sight computational imaging using spectrum filtering and motion compensation},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Spin-symmetry-enforced solution of the many-body schrödinger
equation with a deep neural network. <em>NATCS</em>, <em>4</em>(12),
910–919. (<a href="https://doi.org/10.1038/s43588-024-00730-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The integration of deep neural networks with the variational Monte Carlo (VMC) method has marked a substantial advancement in solving the Schrödinger equation. In this work we enforce spin symmetry in the neural-network-based VMC calculation using a modified optimization target. Our method is designed to solve for the ground state and multiple excited states with target spin symmetry at a low computational cost. It predicts accurate energies while maintaining the correct symmetry in strongly correlated systems, even in cases in which different spin states are nearly degenerate. Our approach also excels at spin–gap calculations, including the singlet–triplet gap in biradical systems, which is of high interest in photochemistry. Overall, this work establishes a robust framework for efficiently calculating various quantum states with specific spin symmetry in correlated systems. An efficient approach is developed to enforce spin symmetry for neural network wavefunctions when solving the many-body Schrödinger equation. This enables accurate and spin-pure simulations of both ground and excited states.},
  archive      = {J_NATCS},
  author       = {Li, Zhe and Lu, Zixiang and Li, Ruichen and Wen, Xuelan and Li, Xiang and Wang, Liwei and Chen, Ji and Ren, Weiluo},
  doi          = {10.1038/s43588-024-00730-4},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {910-919},
  shortjournal = {Nat. Comput. Sci},
  title        = {Spin-symmetry-enforced solution of the many-body schrödinger equation with a deep neural network},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Structure-based drug design with equivariant diffusion
models. <em>NATCS</em>, <em>4</em>(12), 899–909. (<a
href="https://doi.org/10.1038/s43588-024-00737-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structure-based drug design (SBDD) aims to design small-molecule ligands that bind with high affinity and specificity to pre-determined protein targets. Generative SBDD methods leverage structural data of drugs with their protein targets to propose new drug candidates. However, most existing methods focus exclusively on bottom-up de novo design of compounds or tackle other drug development challenges with task-specific models. The latter requires curation of suitable datasets, careful engineering of the models and retraining from scratch for each task. Here we show how a single pretrained diffusion model can be applied to a broader range of problems, such as off-the-shelf property optimization, explicit negative design and partial molecular design with inpainting. We formulate SBDD as a three-dimensional conditional generation problem and present DiffSBDD, an SE(3)-equivariant diffusion model that generates novel ligands conditioned on protein pockets. Furthermore, we show how additional constraints can be used to improve the generated drug candidates according to a variety of computational metrics. This work applies diffusion models to conditional molecule generation and shows how they can be used to tackle various structure-based drug design problems},
  archive      = {J_NATCS},
  author       = {Schneuing, Arne and Harris, Charles and Du, Yuanqi and Didi, Kieran and Jamasb, Arian and Igashov, Ilia and Du, Weitao and Gomes, Carla and Blundell, Tom L. and Lio, Pietro and Welling, Max and Bronstein, Michael and Correia, Bruno},
  doi          = {10.1038/s43588-024-00737-x},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {899-909},
  shortjournal = {Nat. Comput. Sci},
  title        = {Structure-based drug design with equivariant diffusion models},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Simulation and assimilation of the digital human brain.
<em>NATCS</em>, <em>4</em>(12), 890–898. (<a
href="https://doi.org/10.1038/s43588-024-00731-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we present the Digital Brain (DB)—a platform for simulating spiking neuronal networks at the large neuron scale of the human brain on the basis of personalized magnetic resonance imaging data and biological constraints. The DB aims to reproduce both the resting state and certain aspects of the action of the human brain. An architecture with up to 86 billion neurons and 14,012 GPUs—including a two-level routing scheme between GPUs to accelerate spike transmission in up to 47.8 trillion neuronal synapses—was implemented as part of the simulations. We show that the DB can reproduce blood-oxygen-level-dependent signals of the resting state of the human brain with a high correlation coefficient, as well as interact with its perceptual input, as demonstrated in a visual task. These results indicate the feasibility of implementing a digital representation of the human brain, which can open the door to a broad range of potential applications. The Digital Brain platform is capable of simulating spiking neuronal networks at the neuronal scale of the human brain. The platform is used to reproduce blood-oxygen-level-dependent signals in both the resting state and action, thereby predicting the visual evaluation scores.},
  archive      = {J_NATCS},
  author       = {Lu, Wenlian and Du, Xin and Wang, Jiexiang and Zeng, Longbin and Ye, Leijun and Xiang, Shitong and Zheng, Qibao and Zhang, Jie and Xu, Ningsheng and Feng, Jianfeng},
  doi          = {10.1038/s43588-024-00731-3},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {890-898},
  shortjournal = {Nat. Comput. Sci},
  title        = {Simulation and assimilation of the digital human brain},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A simulated c. Elegans with biophysically detailed neurons
and muscle dynamics. <em>NATCS</em>, <em>4</em>(12), 888–889. (<a
href="https://doi.org/10.1038/s43588-024-00740-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We created an open-source model that simulates Caenorhabditis elegans in a closed-loop system, by integrating simulations of its brain, its physical body, and its environment. BAAIWorm replicated C. elegans locomotive behaviors, and synthetic perturbations of synaptic connections impacted neural control of movement and affected the embodied motor behavior.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00740-2},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {888-889},
  shortjournal = {Nat. Comput. Sci},
  title        = {A simulated c. elegans with biophysically detailed neurons and muscle dynamics},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deep learning training dynamics analysis for single-cell
data. <em>NATCS</em>, <em>4</em>(12), 886–887. (<a
href="https://doi.org/10.1038/s43588-024-00728-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Inspired by recent approaches for natural language processing and computer vision, we developed Annotatability, a framework that analyzes deep neural network training dynamics to interpret pre-annotated single-cell and spatial omics data. Annotatability identified erroneous annotations and ambiguous cell states, inferred trajectories from binary labels, and revealed underlying biological signals.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00728-y},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {886-887},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep learning training dynamics analysis for single-cell data},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Teaching spin symmetry while learning neural network wave
functions. <em>NATCS</em>, <em>4</em>(12), 884–885. (<a
href="https://doi.org/10.1038/s43588-024-00727-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By developing an efficient spin symmetry penalty, a recent study has substantially accelerated the calculation of accurate energies with correct spin states in variational Monte Carlo for both ground and excited states of quantum many-particle systems.},
  archive      = {J_NATCS},
  author       = {Li, Yongle and Chen, Yuhao and He, Xiao},
  doi          = {10.1038/s43588-024-00727-z},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {884-885},
  shortjournal = {Nat. Comput. Sci},
  title        = {Teaching spin symmetry while learning neural network wave functions},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). On the path toward brain-scale simulations. <em>NATCS</em>,
<em>4</em>(12), 882–883. (<a
href="https://doi.org/10.1038/s43588-024-00743-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Today’s high-performance computing systems are nearing an ability to simulate the human brain at scale. This presents a new challenge: going forward, will the bigger challenge be the brain’s size or its complexity?},
  archive      = {J_NATCS},
  author       = {Wang, Felix and Aimone, James B.},
  doi          = {10.1038/s43588-024-00743-z},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {882-883},
  shortjournal = {Nat. Comput. Sci},
  title        = {On the path toward brain-scale simulations},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Effective quantum error correction by AI. <em>NATCS</em>,
<em>4</em>(12), 881. (<a
href="https://doi.org/10.1038/s43588-024-00755-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-024-00755-9},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {881},
  shortjournal = {Nat. Comput. Sci},
  title        = {Effective quantum error correction by AI},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cover runners-up of 2024. <em>NATCS</em>, <em>4</em>(12),
879–880. (<a href="https://doi.org/10.1038/s43588-024-00758-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It is time to bring our favorite cover suggestions from 2024 to light.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00758-6},
  journal      = {Nature Computational Science},
  month        = {12},
  number       = {12},
  pages        = {879-880},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cover runners-up of 2024},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Publisher correction: Reliable deep learning in anomalous
diffusion against out-of-distribution dynamics. <em>NATCS</em>,
<em>4</em>(11), 877. (<a
href="https://doi.org/10.1038/s43588-024-00729-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Feng, Xiaochen and Sha, Hao and Zhang, Yongbing and Su, Yaoquan and Liu, Shuai and Jiang, Yuan and Hou, Shangguo and Han, Sanyang and Ji, Xiangyang},
  doi          = {10.1038/s43588-024-00729-x},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {877},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Reliable deep learning in anomalous diffusion against out-of-distribution dynamics},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Author correction: Deep-learning density functional theory
hamiltonian for efficient ab initio electronic-structure calculation.
<em>NATCS</em>, <em>4</em>(11), 876. (<a
href="https://doi.org/10.1038/s43588-024-00723-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Li, He and Wang, Zun and Zou, Nianlong and Ye, Meng and Xu, Runzhang and Gong, Xiaoxun and Duan, Wenhui and Xu, Yong},
  doi          = {10.1038/s43588-024-00723-3},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {876},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Deep-learning density functional theory hamiltonian for efficient ab initio electronic-structure calculation},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Provable bounds for noise-free expectation values computed
from noisy samples. <em>NATCS</em>, <em>4</em>(11), 865–875. (<a
href="https://doi.org/10.1038/s43588-024-00709-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum computing has emerged as a powerful computational paradigm capable of solving problems beyond the reach of classical computers. However, today’s quantum computers are noisy, posing challenges to obtaining accurate results. Here, we explore the impact of noise on quantum computing, focusing on the challenges in sampling bit strings from noisy quantum computers and the implications for optimization and machine learning. We formally quantify the sampling overhead to extract good samples from noisy quantum computers and relate it to the layer fidelity, a metric to determine the performance of noisy quantum processors. Further, we show how this allows us to use the conditional value at risk of noisy samples to determine provable bounds on noise-free expectation values. We discuss how to leverage these bounds for different algorithms and demonstrate our findings through experiments on real quantum computers involving up to 127 qubits. The results show strong alignment with theoretical predictions. In this study, the authors investigate the impact of noise on quantum computing with a focus on the challenges in sampling bit strings from noisy quantum computers, which has implications for optimization and machine learning.},
  archive      = {J_NATCS},
  author       = {Barron, Samantha V. and Egger, Daniel J. and Pelofske, Elijah and Bärtschi, Andreas and Eidenbenz, Stephan and Lehmkuehler, Matthis and Woerner, Stefan},
  doi          = {10.1038/s43588-024-00709-1},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {865-875},
  shortjournal = {Nat. Comput. Sci},
  title        = {Provable bounds for noise-free expectation values computed from noisy samples},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A deep learning approach for rational ligand generation with
toxicity control via reactive building blocks. <em>NATCS</em>,
<em>4</em>(11), 851–864. (<a
href="https://doi.org/10.1038/s43588-024-00718-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep generative models are gaining attention in the field of de novo drug design. However, the rational design of ligand molecules for novel targets remains challenging, particularly in controlling the properties of the generated molecules. Here, inspired by the DNA-encoded compound library technique, we introduce DeepBlock, a deep learning approach for block-based ligand generation tailored to target protein sequences while enabling precise property control. DeepBlock neatly divides the generation process into two steps: building blocks generation and molecule reconstruction, accomplished by a neural network and a rule-based reconstruction algorithm we proposed, respectively. Furthermore, DeepBlock synergizes the optimization algorithm and deep learning to regulate the properties of the generated molecules. Experiments show that DeepBlock outperforms existing methods in generating ligands with affinity, synthetic accessibility and drug likeness. Moreover, when integrated with simulated annealing or Bayesian optimization using toxicity as the optimization objective, DeepBlock successfully generates ligands with low toxicity while preserving affinity with the target. DeepBlock is a deep learning framework for ligand generation, inspired by the DNA-encoded compound library technique, that enhances ligand design with building blocks and a rule-based reconstruction algorithm, achieving better drug properties.},
  archive      = {J_NATCS},
  author       = {Li, Pengyong and Zhang, Kaihao and Liu, Tianxiao and Lu, Ruiqiang and Chen, Yangyang and Yao, Xiaojun and Gao, Lin and Zeng, Xiangxiang},
  doi          = {10.1038/s43588-024-00718-0},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {851-864},
  shortjournal = {Nat. Comput. Sci},
  title        = {A deep learning approach for rational ligand generation with toxicity control via reactive building blocks},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Improving the prediction of protein stability changes upon
mutations by geometric learning and a pre-training strategy.
<em>NATCS</em>, <em>4</em>(11), 840–850. (<a
href="https://doi.org/10.1038/s43588-024-00716-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate prediction of protein mutation effects is of great importance in protein engineering and design. Here we propose GeoStab-suite, a suite of three geometric learning-based models—GeoFitness, GeoDDG and GeoDTm—for the prediction of fitness score, ΔΔG and ΔTm of a protein upon mutations, respectively. GeoFitness engages a specialized loss function to allow supervised training of a unified model using the large amount of multi-labeled fitness data in the deep mutational scanning database. To further improve the downstream tasks of ΔΔG and ΔTm prediction, the encoder of GeoFitness is reutilized as a pre-trained module in GeoDDG and GeoDTm to overcome the challenge of lacking sufficient labeled data. This pre-training strategy, in combination with data expansion, markedly improves model performance and generalizability. In the benchmark test, GeoDDG and GeoDTm outperform the other state-of-the-art methods by at least 30% and 70%, respectively, in terms of the Spearman correlation coefficient. In this study, the authors propose a strategy to train a unified model to learn the general mutational effects based on multi-labeled deep mutational scanning (DMS) data, and then reutilize this pre-trained model to improve the downstream protein stability prediction tasks.},
  archive      = {J_NATCS},
  author       = {Xu, Yunxin and Liu, Di and Gong, Haipeng},
  doi          = {10.1038/s43588-024-00716-2},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {840-850},
  shortjournal = {Nat. Comput. Sci},
  title        = {Improving the prediction of protein stability changes upon mutations by geometric learning and a pre-training strategy},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deep generative design of RNA aptamers using structural
predictions. <em>NATCS</em>, <em>4</em>(11), 829–839. (<a
href="https://doi.org/10.1038/s43588-024-00720-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNAs represent a class of programmable biomolecules capable of performing diverse biological functions. Recent studies have developed accurate RNA three-dimensional structure prediction methods, which may enable new RNAs to be designed in a structure-guided manner. Here, we develop a structure-to-sequence deep learning platform for the de novo generative design of RNA aptamers. We show that our approach can design RNA aptamers that are predicted to be structurally similar, yet sequence dissimilar, to known light-up aptamers that fluoresce in the presence of small molecules. We experimentally validate several generated RNA aptamers to have fluorescent activity, show that these aptamers can be optimized for activity in silico, and find that they exhibit a mechanism of fluorescence similar to that of known light-up aptamers. Our results demonstrate how structural predictions can guide the targeted and resource-efficient design of new RNA sequences. A deep learning platform for structure-guided, generative design of RNA sequences is developed and used to discover fluorescent RNA aptamers.},
  archive      = {J_NATCS},
  author       = {Wong, Felix and He, Dongchen and Krishnan, Aarti and Hong, Liang and Wang, Alexander Z. and Wang, Jiuming and Hu, Zhihang and Omori, Satotaka and Li, Alicia and Rao, Jiahua and Yu, Qinze and Jin, Wengong and Zhang, Tianqing and Ilia, Katherine and Chen, Jack X. and Zheng, Shuangjia and King, Irwin and Li, Yu and Collins, James J.},
  doi          = {10.1038/s43588-024-00720-6},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {829-839},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep generative design of RNA aptamers using structural predictions},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). MassiveFold: Unveiling AlphaFold’s hidden potential with
optimized and parallelized massive sampling. <em>NATCS</em>,
<em>4</em>(11), 824–828. (<a
href="https://doi.org/10.1038/s43588-024-00714-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Massive sampling in AlphaFold enables access to increased structural diversity. In combination with its efficient confidence ranking, this unlocks elevated modeling capabilities for monomeric structures and foremost for protein assemblies. However, the approach struggles with GPU cost and data storage. Here we introduce MassiveFold, an optimized and customizable version of AlphaFold that runs predictions in parallel, reducing the computing time from several months to hours. MassiveFold is scalable and able to run on anything from a single computer to a large GPU infrastructure, where it can fully benefit from all the computing nodes. Although AlphaFold is very efficient for protein structure prediction, massive sampling is a very GPU demanding task. MassiveFold overcomes this limitation, being capable of parallelizing structure prediction computation.},
  archive      = {J_NATCS},
  author       = {Raouraoua, Nessim and Mirabello, Claudio and Véry, Thibaut and Blanchet, Christophe and Wallner, Björn and Lensink, Marc F. and Brysbaert, Guillaume},
  doi          = {10.1038/s43588-024-00714-4},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {824-828},
  shortjournal = {Nat. Comput. Sci},
  title        = {MassiveFold: Unveiling AlphaFold’s hidden potential with optimized and parallelized massive sampling},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). E-waste challenges of generative artificial intelligence.
<em>NATCS</em>, <em>4</em>(11), 818–823. (<a
href="https://doi.org/10.1038/s43588-024-00712-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Generative artificial intelligence (GAI) requires substantial computational resources for model training and inference, but the electronic-waste (e-waste) implications of GAI and its management strategies remain underexplored. Here we introduce a computational power-driven material flow analysis framework to quantify and explore ways of managing the e-waste generated by GAI, with a particular focus on large language models. Our findings indicate that this e-waste stream could increase, potentially reaching a total accumulation of 1.2–5.0 million tons during 2020–2030, under different future GAI development settings. This may be intensified in the context of geopolitical restrictions on semiconductor imports and the rapid server turnover for operational cost savings. Meanwhile, we show that the implementation of circular economy strategies along the GAI value chain could reduce e-waste generation by 16–86%. This underscores the importance of proactive e-waste management in the face of advancing GAI technologies. Generative artificial intelligence (GAI) is driving a surge in e-waste due to intensive computational infrastructure needs. This study emphasizes the necessity for proactive implementation of circular economy practices throughout GAI value chains.},
  archive      = {J_NATCS},
  author       = {Wang, Peng and Zhang, Ling-Yu and Tzachor, Asaf and Chen, Wei-Qiang},
  doi          = {10.1038/s43588-024-00712-6},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {818-823},
  shortjournal = {Nat. Comput. Sci},
  title        = {E-waste challenges of generative artificial intelligence},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The zettabyte era is in our DNA. <em>NATCS</em>,
<em>4</em>(11), 813–817. (<a
href="https://doi.org/10.1038/s43588-024-00717-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This Perspective surveys the critical computational challenges associated with in vitro DNA-based data storage. As digital data expand exponentially, traditional storage media are becoming less viable, making DNA a promising solution due to its density and durability. However, numerous obstacles remain, including error correction, data retrieval from large volumes of noisy reads, and scalability. The Perspective also highlights challenges for DNA-based data centers, such as fault tolerance, random access, and data removal, which must be addressed to make DNA-based storage practical. As digital data expand exponentially, traditional storage media are becoming less viable, making DNA a promising solution due to its density and durability. In this Perspective, the authors discuss the critical computational challenges associated with in vitro DNA-based data storage.},
  archive      = {J_NATCS},
  author       = {Bar-Lev, Daniella and Sabary, Omer and Yaakobi, Eitan},
  doi          = {10.1038/s43588-024-00717-1},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {813-817},
  shortjournal = {Nat. Comput. Sci},
  title        = {The zettabyte era is in our DNA},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Extracting reliable quantum outputs for noisy devices.
<em>NATCS</em>, <em>4</em>(11), 811–812. (<a
href="https://doi.org/10.1038/s43588-024-00713-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A method is introduced to compute provable bounds on noise-free quantum expectation values from noisy samples, promising potential applications in quantum optimization and machine learning.},
  archive      = {J_NATCS},
  author       = {Li, Weikang and Deng, Dong-Ling},
  doi          = {10.1038/s43588-024-00713-5},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {811-812},
  shortjournal = {Nat. Comput. Sci},
  title        = {Extracting reliable quantum outputs for noisy devices},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Harnessing deep learning to build optimized ligands.
<em>NATCS</em>, <em>4</em>(11), 809–810. (<a
href="https://doi.org/10.1038/s43588-024-00725-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposes DeepBlock, a deep learning-based approach for generating ligands with targeted properties, such as low toxicity and high affinity with the given target. This approach outperforms existing methods in the field while maintaining synthetic accessibility and drug-likeness.},
  archive      = {J_NATCS},
  author       = {Ntintas, Orestis A. and Daglis, Theodoros and Gorgoulis, Vassilis G.},
  doi          = {10.1038/s43588-024-00725-1},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {809-810},
  shortjournal = {Nat. Comput. Sci},
  title        = {Harnessing deep learning to build optimized ligands},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Enhancing protein stability prediction with geometric
learning and pre-training strategies. <em>NATCS</em>, <em>4</em>(11),
807–808. (<a href="https://doi.org/10.1038/s43588-024-00724-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study introduces a series of approaches that predict protein fitness and stability after the introduction of mutations. The work focuses on combining different data and pre-training to overcome data scarcity.},
  archive      = {J_NATCS},
  author       = {Li, Minghui},
  doi          = {10.1038/s43588-024-00724-2},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {807-808},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enhancing protein stability prediction with geometric learning and pre-training strategies},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Modeling the increase of electronic waste due to generative
AI. <em>NATCS</em>, <em>4</em>(11), 805–806. (<a
href="https://doi.org/10.1038/s43588-024-00726-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study has modeled and quantified the expected rise in electronic waste due to the increasing deployment of generative artificial intelligence.},
  archive      = {J_NATCS},
  author       = {Lannelongue, Loïc},
  doi          = {10.1038/s43588-024-00726-0},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {805-806},
  shortjournal = {Nat. Comput. Sci},
  title        = {Modeling the increase of electronic waste due to generative AI},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An interdisciplinary effort to integrate coding into science
courses. <em>NATCS</em>, <em>4</em>(11), 803–804. (<a
href="https://doi.org/10.1038/s43588-024-00708-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Vizcarra, Christina L. and Trainor, Ryan F. and Ringer McDonald, Ashley and Richardson, Chris T. and Potoyan, Davit and Nash, Jessica A. and Lundgren, Britt and Luchko, Tyler and Hocky, Glen M. and Foley, Jonathan J. and Atherton, Timothy J. and Stokes, Grace Y.},
  doi          = {10.1038/s43588-024-00708-2},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {803-804},
  shortjournal = {Nat. Comput. Sci},
  title        = {An interdisciplinary effort to integrate coding into science courses},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Collective deliberation driven by AI. <em>NATCS</em>,
<em>4</em>(11), 802. (<a
href="https://doi.org/10.1038/s43588-024-00736-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-024-00736-y},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {802},
  shortjournal = {Nat. Comput. Sci},
  title        = {Collective deliberation driven by AI},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Harnessing the power of DNA for computing. <em>NATCS</em>,
<em>4</em>(11), 801. (<a
href="https://doi.org/10.1038/s43588-024-00742-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We discuss the thirty-year anniversary of the seminal work on DNA computing and its implications for the field of biotechnology.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00742-0},
  journal      = {Nature Computational Science},
  month        = {11},
  number       = {11},
  pages        = {801},
  shortjournal = {Nat. Comput. Sci},
  title        = {Harnessing the power of DNA for computing},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Publisher correction: Joint inference of discrete cell
types and continuous type-specific variability in single-cell datasets
with MMIDAS. <em>NATCS</em>, <em>4</em>(10), 799. (<a
href="https://doi.org/10.1038/s43588-024-00711-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Marghi, Yeganeh and Gala, Rohan and Baftizadeh, Fahimeh and Sümbül, Uygar},
  doi          = {10.1038/s43588-024-00711-7},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {799},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Joint inference of discrete cell types and continuous type-specific variability in single-cell datasets with MMIDAS},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Publisher correction: Increasing the presence of BIPOC
researchers in computational science. <em>NATCS</em>, <em>4</em>(10),
798. (<a href="https://doi.org/10.1038/s43588-024-00710-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chen, Christine Yifeng and Christoffels, Alan and Dube, Roger and Enos, Kamuela and Gilbert, Juan E. and Koyejo, Sanmi and Leigh, Jason and Liquido, Carlo and McKee, Amy and Noe, Kari and Peng, Tai-Quan and Taiuru, Karaitiana},
  doi          = {10.1038/s43588-024-00710-8},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {798},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: Increasing the presence of BIPOC researchers in computational science},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Publisher correction: A perspective on brain-age estimation
and its clinical promise. <em>NATCS</em>, <em>4</em>(10), 797. (<a
href="https://doi.org/10.1038/s43588-024-00681-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Gaser, Christian and Kalc, Polona and Cole, James H.},
  doi          = {10.1038/s43588-024-00681-w},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {797},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: A perspective on brain-age estimation and its clinical promise},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Traversing chemical space with active deep learning for
low-data drug discovery. <em>NATCS</em>, <em>4</em>(10), 786–796. (<a
href="https://doi.org/10.1038/s43588-024-00697-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep learning is accelerating drug discovery. However, current approaches are often affected by limitations in the available data, in terms of either size or molecular diversity. Active deep learning has high potential for low-data drug discovery, as it allows iterative model improvement during the screening process. However, there are several ‘known unknowns’ that limit the wider adoption of active deep learning in drug discovery: (1) what the best computational strategies are for chemical space exploration, (2) how active learning holds up to traditional, non-iterative, approaches and (3) how it should be used in the low-data scenarios typical of drug discovery. To provide answers, this study simulates a low-data drug discovery scenario, and systematically analyzes six active learning strategies combined with two deep learning architectures, on three large-scale molecular libraries. We identify the most important determinants of success in low-data regimes and show that active learning can achieve up to a sixfold improvement in hit discovery when compared with traditional screening methods. Active deep learning is a promising approach to learn from low-data scenarios in drug discovery. This study illuminates key success factors of active learning and shows that it can boost hit discovery by up to sixfold over traditional methods.},
  archive      = {J_NATCS},
  author       = {van Tilborg, Derek and Grisoni, Francesca},
  doi          = {10.1038/s43588-024-00697-2},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {786-796},
  shortjournal = {Nat. Comput. Sci},
  title        = {Traversing chemical space with active deep learning for low-data drug discovery},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Toward subtask-decomposition-based learning and benchmarking
for predicting genetic perturbation outcomes and beyond. <em>NATCS</em>,
<em>4</em>(10), 773–785. (<a
href="https://doi.org/10.1038/s43588-024-00698-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deciphering cellular responses to genetic perturbations is fundamental for a wide array of biomedical applications. However, there are three main challenges: predicting single-genetic-perturbation outcomes, predicting multiple-genetic-perturbation outcomes and predicting genetic outcomes across cell lines. Here we introduce Subtask Decomposition Modeling for Genetic Perturbation Prediction (STAMP), a flexible artificial intelligence strategy for genetic perturbation outcome prediction and downstream applications. STAMP formulates genetic perturbation prediction as a subtask decomposition problem by resolving three progressive subtasks in a problem decomposition manner, that is, identifying postperturbation differentially expressed genes, determining the expression change directions of differentially expressed genes and finally estimating the magnitudes of gene expression changes. STAMP exhibits a substantial improvement over the existing approaches on three subtasks and beyond, including the ability to identify key regulatory genes and pathways on small samples and to reveal precise genetic interactions of diverse types. By employing the subtask decomposition strategy, STAMP outperforms existing models in single, multiple and cross-cell-line scenarios for genetic perturbation prediction, showing potential to uncover gene regulations and interactions.},
  archive      = {J_NATCS},
  author       = {Gao, Yicheng and Wei, Zhiting and Dong, Kejing and Chen, Ke and Yang, Jingya and Chuai, Guohui and Liu, Qi},
  doi          = {10.1038/s43588-024-00698-1},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {773-785},
  shortjournal = {Nat. Comput. Sci},
  title        = {Toward subtask-decomposition-based learning and benchmarking for predicting genetic perturbation outcomes and beyond},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Reliable deep learning in anomalous diffusion against
out-of-distribution dynamics. <em>NATCS</em>, <em>4</em>(10), 761–772.
(<a href="https://doi.org/10.1038/s43588-024-00703-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Anomalous diffusion plays a crucial rule in understanding molecular-level dynamics by offering valuable insights into molecular interactions, mobility states and the physical properties of systems across both biological and materials sciences. Deep-learning techniques have recently outperformed conventional statistical methods in anomalous diffusion recognition. However, deep-learning networks are typically trained by data with limited distribution, which inevitably fail to recognize unknown diffusion models and misinterpret dynamics when confronted with out-of-distribution (OOD) scenarios. In this work, we present a general framework for evaluating deep-learning-based OOD dynamics-detection methods. We further develop a baseline approach that achieves robust OOD dynamics detection as well as accurate recognition of in-distribution anomalous diffusion. We demonstrate that this method enables a reliable characterization of complex behaviors across a wide range of experimentally diverse systems, including nicotinic acetylcholine receptors in membranes, fluorescent beads in dextran solutions and silver nanoparticles undergoing active endocytosis. This work introduces a framework that enhances deep learning for anomalous diffusion, enabling reliable detection of out-of-distribution dynamics and characterization of complex behaviors across diverse systems.},
  archive      = {J_NATCS},
  author       = {Feng, Xiaochen and Sha, Hao and Zhang, Yongbing and Su, Yaoquan and Liu, Shuai and Jiang, Yuan and Hou, Shangguo and Han, Sanyang and Ji, Xiangyang},
  doi          = {10.1038/s43588-024-00703-7},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {761-772},
  shortjournal = {Nat. Comput. Sci},
  title        = {Reliable deep learning in anomalous diffusion against out-of-distribution dynamics},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Generalizing deep learning electronic structure calculation
to the plane-wave basis. <em>NATCS</em>, <em>4</em>(10), 752–760. (<a
href="https://doi.org/10.1038/s43588-024-00701-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Deep neural networks capable of representing the density functional theory (DFT) Hamiltonian as a function of material structure hold great promise for revolutionizing future electronic structure calculations. However, a notable limitation of previous neural networks is their compatibility solely with the atomic-orbital (AO) basis, excluding the widely used plane-wave (PW) basis. Here we overcome this critical limitation by proposing an accurate and efficient real-space reconstruction method for directly computing AO Hamiltonian matrices from PW DFT results. The reconstruction method is orders of magnitude faster than traditional projection-based methods to convert PW results to the AO basis, and the reconstructed Hamiltonian matrices can faithfully reproduce the PW electronic structure, thus bridging the longstanding gap between the AO basis deep learning electronic structure approach and PW DFT. Advantages of the PW methods, such as high accuracy, high flexibility and wide applicability, thus can be all integrated into deep learning electronic structure methods without sacrificing these methods’ inherent benefits. This allows for the construction of large-scale and high-fidelity training datasets with the help of PW DFT results towards the development of precise and broadly applicable deep learning electronic structure models. Deep learning electronic structure calculations are generalized from the atomic-orbital basis to the plane-wave basis, resulting in higher accuracy, improved transferability and the capability to utilize existing electronic structure big data.},
  archive      = {J_NATCS},
  author       = {Gong, Xiaoxun and Louie, Steven G. and Duan, Wenhui and Xu, Yong},
  doi          = {10.1038/s43588-024-00701-9},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {752-760},
  shortjournal = {Nat. Comput. Sci},
  title        = {Generalizing deep learning electronic structure calculation to the plane-wave basis},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). A perspective on brain-age estimation and its clinical
promise. <em>NATCS</em>, <em>4</em>(10), 744–751. (<a
href="https://doi.org/10.1038/s43588-024-00659-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Brain-age estimation has gained increased attention in the neuroscientific community owing to its potential use as a biomarker of brain health. The difference between estimated and chronological age based on neuroimaging data enables a unique perspective on brain development and aging, with multiple open questions still remaining in the brain-age research field. This Perspective presents an overview of current advancements in the field and envisions the future evolution of the brain-age framework before its potential deployment in hospital settings. Brain-age estimation is gaining attention as a biomarker for brain health as it provides a unique perspective on aging. This Perspective reviews current advancements and future directions to ensure deployment in hospital settings.},
  archive      = {J_NATCS},
  author       = {Gaser, Christian and Kalc, Polona and Cole, James H.},
  doi          = {10.1038/s43588-024-00659-8},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {744-751},
  shortjournal = {Nat. Comput. Sci},
  title        = {A perspective on brain-age estimation and its clinical promise},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The future of machine learning for small-molecule drug
discovery will be driven by data. <em>NATCS</em>, <em>4</em>(10),
735–743. (<a href="https://doi.org/10.1038/s43588-024-00699-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many studies have prophesied that the integration of machine learning techniques into small-molecule therapeutics development will help to deliver a true leap forward in drug discovery. However, increasingly advanced algorithms and novel architectures have not always yielded substantial improvements in results. In this Perspective, we propose that a greater focus on the data for training and benchmarking these models is more likely to drive future improvement, and explore avenues for future research and strategies to address these data challenges. The application of machine learning techniques to small-molecule drug discovery has not yet yielded a true leap forward in the field. This Perspective discusses how a renewed focus on data and validation could help unlock machine learning’s potential.},
  archive      = {J_NATCS},
  author       = {Durant, Guy and Boyles, Fergus and Birchall, Kristian and Deane, Charlotte M.},
  doi          = {10.1038/s43588-024-00699-0},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {735-743},
  shortjournal = {Nat. Comput. Sci},
  title        = {The future of machine learning for small-molecule drug discovery will be driven by data},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Joint inference of discrete and continuous factors captures
variability across and within cell types. <em>NATCS</em>,
<em>4</em>(10), 733–734. (<a
href="https://doi.org/10.1038/s43588-024-00696-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed mixture model inference with discrete-coupled autoencoders (MMIDAS), an unsupervised variational framework that jointly learns discrete clusters and continuous cluster-specific variability. When applied to unimodal or multimodal single-cell omic data, MMIDAS learned single-cell representations with robust cell type definitions and interpretable, continuous within-cell type variability.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00696-3},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {733-734},
  shortjournal = {Nat. Comput. Sci},
  title        = {Joint inference of discrete and continuous factors captures variability across and within cell types},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Effectively detecting anomalous diffusion via deep learning.
<em>NATCS</em>, <em>4</em>(10), 731–732. (<a
href="https://doi.org/10.1038/s43588-024-00705-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A deep learning algorithm is presented to classify single-particle tracking trajectories into theoretical models of anomalous diffusion and detect if the trajectory is related to a model not originally found within the training dataset.},
  archive      = {J_NATCS},
  author       = {Pacheco-Pozo, Adrian and Krapf, Diego},
  doi          = {10.1038/s43588-024-00705-5},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {731-732},
  shortjournal = {Nat. Comput. Sci},
  title        = {Effectively detecting anomalous diffusion via deep learning},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Bridging the gap in electronic structure calculations via
machine learning. <em>NATCS</em>, <em>4</em>(10), 729–730. (<a
href="https://doi.org/10.1038/s43588-024-00707-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A highly efficient reconstruction method has been developed for the direct computation of Hamiltonian matrices in the atomic orbital basis from density functional theory calculations originally performed in the plane wave basis. This enables machine learning calculations of electronic structures on a large scale, which are otherwise not feasible with standard methods, and thus fills a methodological gap in terms of accessible length scales.},
  archive      = {J_NATCS},
  author       = {Cangi, Attila},
  doi          = {10.1038/s43588-024-00707-3},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {729-730},
  shortjournal = {Nat. Comput. Sci},
  title        = {Bridging the gap in electronic structure calculations via machine learning},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Taking a deep dive with active learning for drug discovery.
<em>NATCS</em>, <em>4</em>(10), 727–728. (<a
href="https://doi.org/10.1038/s43588-024-00704-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Active machine learning is employed in academia and industry to support drug discovery. A recent study unravels the factors that influence a deep learning models’ ability to guide iterative discovery.},
  archive      = {J_NATCS},
  author       = {Fralish, Zachary and Reker, Daniel},
  doi          = {10.1038/s43588-024-00704-6},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {727-728},
  shortjournal = {Nat. Comput. Sci},
  title        = {Taking a deep dive with active learning for drug discovery},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The decomposition of perturbation modeling. <em>NATCS</em>,
<em>4</em>(10), 725–726. (<a
href="https://doi.org/10.1038/s43588-024-00706-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposes a strategy for the prediction of genetic perturbation outcomes by breaking it down into three subtasks: identifying differentially expressed genes, determining expression change directions, and estimating gene expression magnitudes.},
  archive      = {J_NATCS},
  author       = {Peidli, Stefan},
  doi          = {10.1038/s43588-024-00706-4},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {725-726},
  shortjournal = {Nat. Comput. Sci},
  title        = {The decomposition of perturbation modeling},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Efficient simulations of electronic spectra. <em>NATCS</em>,
<em>4</em>(10), 724. (<a
href="https://doi.org/10.1038/s43588-024-00715-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {McCardle, Kaitlin},
  doi          = {10.1038/s43588-024-00715-3},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {724},
  shortjournal = {Nat. Comput. Sci},
  title        = {Efficient simulations of electronic spectra},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Fostering discussions on topical issues. <em>NATCS</em>,
<em>4</em>(10), 723. (<a
href="https://doi.org/10.1038/s43588-024-00719-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Computational Science invites researchers to submit Correspondence pieces.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00719-z},
  journal      = {Nature Computational Science},
  month        = {10},
  number       = {10},
  pages        = {723},
  shortjournal = {Nat. Comput. Sci},
  title        = {Fostering discussions on topical issues},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Joint inference of discrete cell types and continuous
type-specific variability in single-cell datasets with MMIDAS.
<em>NATCS</em>, <em>4</em>(9), 706–722. (<a
href="https://doi.org/10.1038/s43588-024-00683-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reproducible definition and identification of cell types is essential to enable investigations into their biological function and to understand their relevance in the context of development, disease and evolution. Current approaches model variability in data as continuous latent factors, followed by clustering as a separate step, or immediately apply clustering on the data. We show that such approaches can suffer from qualitative mistakes in identifying cell types robustly, particularly when the number of such cell types is in the hundreds or even thousands. Here we propose an unsupervised method, Mixture Model Inference with Discrete-coupled AutoencoderS (MMIDAS), which combines a generalized mixture model with a multi-armed deep neural network to jointly infer the discrete type and continuous type-specific variability. Using four recent datasets of brain cells spanning different technologies, species and conditions, we demonstrate that MMIDAS can identify reproducible cell types and infer cell type-dependent continuous variability in both unimodal and multimodal datasets. Clustering in high-dimensional spaces with a large number of clusters and identifying common aspects of within-cluster variability remain challenging. Here the authors develop an unsupervised method for this purpose and demonstrate it on brain single-cell datasets.},
  archive      = {J_NATCS},
  author       = {Marghi, Yeganeh and Gala, Rohan and Baftizadeh, Fahimeh and Sümbül, Uygar},
  doi          = {10.1038/s43588-024-00683-8},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {706-722},
  shortjournal = {Nat. Comput. Sci},
  title        = {Joint inference of discrete cell types and continuous type-specific variability in single-cell datasets with MMIDAS},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Automated customization of large-scale spiking network
models to neuronal population activity. <em>NATCS</em>, <em>4</em>(9),
690–705. (<a href="https://doi.org/10.1038/s43588-024-00688-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding brain function is facilitated by constructing computational models that accurately reproduce aspects of brain activity. Networks of spiking neurons capture the underlying biophysics of neuronal circuits, yet their activity’s dependence on model parameters is notoriously complex. As a result, heuristic methods have been used to configure spiking network models, which can lead to an inability to discover activity regimes complex enough to match large-scale neuronal recordings. Here we propose an automatic procedure, Spiking Network Optimization using Population Statistics (SNOPS), to customize spiking network models that reproduce the population-wide covariability of large-scale neuronal recordings. We first confirmed that SNOPS accurately recovers simulated neural activity statistics. Then, we applied SNOPS to recordings in macaque visual and prefrontal cortices and discovered previously unknown limitations of spiking network models. Taken together, SNOPS can guide the development of network models, thereby enabling deeper insight into how networks of neurons give rise to brain function. An automatic framework, SNOPS, is developed for configuring a spiking network model to reproduce neuronal recordings. It is used to discover previously unknown limitations of spiking network models, thereby guiding model development.},
  archive      = {J_NATCS},
  author       = {Wu, Shenghao and Huang, Chengcheng and Snyder, Adam C. and Smith, Matthew A. and Doiron, Brent and Yu, Byron M.},
  doi          = {10.1038/s43588-024-00688-3},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {690-705},
  shortjournal = {Nat. Comput. Sci},
  title        = {Automated customization of large-scale spiking network models to neuronal population activity},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Biophysically interpretable inference of cell types from
multimodal sequencing data. <em>NATCS</em>, <em>4</em>(9), 677–689. (<a
href="https://doi.org/10.1038/s43588-024-00689-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multimodal, single-cell genomics technologies enable simultaneous measurement of multiple facets of DNA and RNA processing in the cell. This creates opportunities for transcriptome-wide, mechanistic studies of cellular processing in heterogeneous cell populations, such as regulation of cell fate by transcriptional stochasticity or tumor proliferation through aberrant splicing dynamics. However, current methods for determining cell types or ‘clusters’ in multimodal data often rely on ad hoc approaches to balance or integrate measurements, and assumptions ignoring inherent properties of the data. To enable interpretable and consistent cell cluster determination, we present meK-means (mechanistic K-means) which integrates modalities through a unifying model of transcription to learn underlying, shared biophysical states. With meK-means we can cluster cells with nascent and mature mRNA measurements, utilizing the causal, physical relationships between these modalities. This identifies shared transcription dynamics across cells, which induce the observed molecule counts, and provides an alternative definition for ‘clusters’ through the governing parameters of cellular processes. MeK-means clusters single-cell multimodal data by linking modalities through their biophysical relationships. We redefine clusters through transcription kinetics to reveal how RNA production and processing drive cellular diversity and disease.},
  archive      = {J_NATCS},
  author       = {Chari, Tara and Gorin, Gennady and Pachter, Lior},
  doi          = {10.1038/s43588-024-00689-2},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {677-689},
  shortjournal = {Nat. Comput. Sci},
  title        = {Biophysically interpretable inference of cell types from multimodal sequencing data},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The motive cocktail in altruistic behaviors. <em>NATCS</em>,
<em>4</em>(9), 659–676. (<a
href="https://doi.org/10.1038/s43588-024-00685-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prosocial motives such as social equality and efficiency are key to altruistic behaviors. However, predicting the range of altruistic behaviors in varying contexts and individuals proves challenging if we limit ourselves to one or two motives. Here we demonstrate the numerous, interdependent motives in altruistic behaviors and the possibility to disentangle them through behavioral experimental data and computational modeling. In one laboratory experiment (N = 157) and one preregistered online replication (N = 1,258), across 100 different situations, we found that both third-party punishment and third-party helping behaviors (that is, an unaffected individual punishes the transgressor or helps the victim) aligned best with a model of seven socioeconomic motives, referred to as a motive cocktail. For instance, the inequality discounting motives imply that individuals, when confronted with costly interventions, behave as if the inequality between others barely exists. The motive cocktail model also provides a unified explanation for the differences in intervention willingness between second parties (victims) and third parties, and between punishment and helping. The authors find, through experimental data and computational modeling, that altruistic acts stem from a motive cocktail of up to seven social and economic motives, whose strengths explain distinct behavior patterns across individuals and situations.},
  archive      = {J_NATCS},
  author       = {Wu, Xiaoyan and Ren, Xiangjuan and Liu, Chao and Zhang, Hang},
  doi          = {10.1038/s43588-024-00685-6},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {659-676},
  shortjournal = {Nat. Comput. Sci},
  title        = {The motive cocktail in altruistic behaviors},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Delineating cell types with transcriptional kinetics.
<em>NATCS</em>, <em>4</em>(9), 657–658. (<a
href="https://doi.org/10.1038/s43588-024-00691-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposes an approach that integrates unspliced and spliced mRNA count data by leveraging shared biophysical states across cells, offering a more interpretable and consistent framework for determining cell clusters based on transcriptional kinetics.},
  archive      = {J_NATCS},
  author       = {Gao, Yicheng and Liu, Qi},
  doi          = {10.1038/s43588-024-00691-8},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {657-658},
  shortjournal = {Nat. Comput. Sci},
  title        = {Delineating cell types with transcriptional kinetics},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deconstructing the compounds of altruism. <em>NATCS</em>,
<em>4</em>(9), 655–656. (<a
href="https://doi.org/10.1038/s43588-024-00690-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A computational model is proposed to provide a better understanding of human altruism, highlighting the role of multiple motives that influence altruistic behaviors.},
  archive      = {J_NATCS},
  author       = {Hu, Jie},
  doi          = {10.1038/s43588-024-00690-9},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {655-656},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deconstructing the compounds of altruism},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Heat wave attribution assessment using deep learning.
<em>NATCS</em>, <em>4</em>(9), 654. (<a
href="https://doi.org/10.1038/s43588-024-00700-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-024-00700-w},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {654},
  shortjournal = {Nat. Comput. Sci},
  title        = {Heat wave attribution assessment using deep learning},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Increasing the presence of BIPOC researchers in
computational science. <em>NATCS</em>, <em>4</em>(9), 646–653. (<a
href="https://doi.org/10.1038/s43588-024-00693-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Computational Science asked a group of scientists to discuss strategies for increasing the presence of Black, Indigenous, People of Color (BIPOC) researchers in computational science, as well as the various considerations to be made for improving education and methods design.},
  archive      = {J_NATCS},
  author       = {Chen, Christine Yifeng and Christoffels, Alan and Dube, Roger and Enos, Kamuela and Gilbert, Juan E. and Koyeji, Sanmi and Leigh, Jason and Liquido, Carlo and McKee, Amy and Noe, Kari and Peng, Tai-Quan and Taiuru, Karaitiana},
  doi          = {10.1038/s43588-024-00693-6},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {646-653},
  shortjournal = {Nat. Comput. Sci},
  title        = {Increasing the presence of BIPOC researchers in computational science},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accelerating economic development in latin america through
overcoming access challenges to supercomputing infrastructure.
<em>NATCS</em>, <em>4</em>(9), 644–645. (<a
href="https://doi.org/10.1038/s43588-024-00686-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The current global economy heavily relies on digital and data-based technologies, which have the use of supercomputing at their core. Latin America is a vast source of human talent in computer science, but the lag in infrastructure investment due to economic and political struggles may cause the economic development of the region to fall behind.},
  archive      = {J_NATCS},
  author       = {Barroso-Flores, Joaquín},
  doi          = {10.1038/s43588-024-00686-5},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {644-645},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerating economic development in latin america through overcoming access challenges to supercomputing infrastructure},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Defining our future with generative AI. <em>NATCS</em>,
<em>4</em>(9), 641–643. (<a
href="https://doi.org/10.1038/s43588-024-00694-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We can design, build and use AI systems with intentionality, to make them an equalizing force within society, or we can use AI without intentionality, in which case AI could become a force that exacerbates inequality, or both. Society has the power to decide which.},
  archive      = {J_NATCS},
  author       = {Suri, Siddharth},
  doi          = {10.1038/s43588-024-00694-5},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {641-643},
  shortjournal = {Nat. Comput. Sci},
  title        = {Defining our future with generative AI},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Using labels to limit AI misuse in health. <em>NATCS</em>,
<em>4</em>(9), 638–640. (<a
href="https://doi.org/10.1038/s43588-024-00676-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The proliferation of artificial intelligence (AI) algorithms for public use has led to many creative healthcare applications, some with the potential to create or worsen health inequities. Here, we argue that similar to prescription medicine labels, AI algorithms should be accompanied by a responsible use label.},
  archive      = {J_NATCS},
  author       = {Nsoesie, Elaine O. and Ghassemi, Marzyeh},
  doi          = {10.1038/s43588-024-00676-7},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {638-640},
  shortjournal = {Nat. Comput. Sci},
  title        = {Using labels to limit AI misuse in health},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Harnessing the power of emerging computational capabilities
for independent mobility for persons with disabilities. <em>NATCS</em>,
<em>4</em>(9), 636–637. (<a
href="https://doi.org/10.1038/s43588-024-00692-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Navigating built environments can be a challenge for persons with disabilities. Emerging computational capabilities are promising to help by providing the right information at the right time in accessible formats.},
  archive      = {J_NATCS},
  author       = {Namboodiri, Vinod},
  doi          = {10.1038/s43588-024-00692-7},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {636-637},
  shortjournal = {Nat. Comput. Sci},
  title        = {Harnessing the power of emerging computational capabilities for independent mobility for persons with disabilities},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Gaps in gender and socioeconomic mobility disparity studies.
<em>NATCS</em>, <em>4</em>(9), 633–635. (<a
href="https://doi.org/10.1038/s43588-024-00667-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The widespread availability of digital traces capturing individuals’ daily mobility has the potential to enrich the understanding of the relationship between mobility, gender and socioeconomic factors. In fact, it has led to a heightened interest in deriving policy insights from these data. However, it is also essential to put the focus on methodological aspects to address the data gaps and biases.},
  archive      = {J_NATCS},
  author       = {Gauvin, Laetitia},
  doi          = {10.1038/s43588-024-00667-8},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {633-635},
  shortjournal = {Nat. Comput. Sci},
  title        = {Gaps in gender and socioeconomic mobility disparity studies},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The lost data: How AI systems censor LGBTQ+ content in the
name of safety. <em>NATCS</em>, <em>4</em>(9), 629–632. (<a
href="https://doi.org/10.1038/s43588-024-00695-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many AI companies implement safety systems to protect users from offensive or inaccurate content. Though well intentioned, these filters can exacerbate existing inequalities, and data shows that they have disproportionately removed LGBTQ+ content.},
  archive      = {J_NATCS},
  author       = {Chen, Sophia},
  doi          = {10.1038/s43588-024-00695-4},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {629-632},
  shortjournal = {Nat. Comput. Sci},
  title        = {The lost data: How AI systems censor LGBTQ+ content in the name of safety},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Putting a spotlight on diversity, equity, and inclusion.
<em>NATCS</em>, <em>4</em>(9), 627–628. (<a
href="https://doi.org/10.1038/s43588-024-00702-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a Focus that calls attention to the current state of diversity, equity, and inclusion in computational science, including discussions on the challenges of improving equitable access and representation, as well as on strategies for improving computational tools to avoid contributing to inequalities.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00702-8},
  journal      = {Nature Computational Science},
  month        = {9},
  number       = {9},
  pages        = {627-628},
  shortjournal = {Nat. Comput. Sci},
  title        = {Putting a spotlight on diversity, equity, and inclusion},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accelerating the calculation of electron–phonon coupling
strength with machine learning. <em>NATCS</em>, <em>4</em>(8), 615–625.
(<a href="https://doi.org/10.1038/s43588-024-00668-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The calculation of electron–phonon couplings (EPCs) is essential for understanding various fundamental physical properties, including electrical transport, optical and superconducting behaviors in materials. However, obtaining EPCs through fully first-principles methods is notably challenging, particularly for large systems or when employing advanced functionals. Here we introduce a machine learning framework to accelerate EPC calculations by utilizing atomic orbital-based Hamiltonian matrices and gradients predicted by an equivariant graph neural network. We demonstrate that our method not only yields EPC values in close agreement with first-principles results but also enhances calculation efficiency by several orders of magnitude. Application to GaAs using the Heyd–Scuseria–Ernzerhof functional reveals the necessity of advanced functionals for accurate carrier mobility predictions, while for the large Kagome crystal CsV3Sb5, our framework reproduces the experimentally observed double domes in pressure-induced superconducting phase diagrams. This machine learning framework offers a powerful and efficient tool for the investigation of diverse EPC-related phenomena in complex materials. A machine learning framework is proposed to accurately predict electron–phonon coupling (EPC) strengths while reducing computational costs compared with first-principles methods. This approach facilitates EPC calculations with advanced functionals, allowing the accurate determination of real-world material properties such as carrier mobility and superconductivity.},
  archive      = {J_NATCS},
  author       = {Zhong, Yang and Liu, Shixu and Zhang, Binhua and Tao, Zhiguo and Sun, Yuting and Chu, Weibin and Gong, Xin-Gao and Yang, Ji-Hui and Xiang, Hongjun},
  doi          = {10.1038/s43588-024-00668-7},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {615-625},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerating the calculation of electron–phonon coupling strength with machine learning},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deep learning large-scale drug discovery and repurposing.
<em>NATCS</em>, <em>4</em>(8), 600–614. (<a
href="https://doi.org/10.1038/s43588-024-00679-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale drug discovery and repurposing is challenging. Identifying the mechanism of action (MOA) is crucial, yet current approaches are costly and low-throughput. Here we present an approach for MOA identification by profiling changes in mitochondrial phenotypes. By temporally imaging mitochondrial morphology and membrane potential, we established a pipeline for monitoring time-resolved mitochondrial images, resulting in a dataset comprising 570,096 single-cell images of cells exposed to 1,068 United States Food and Drug Administration-approved drugs. A deep learning model named MitoReID, using a re-identification (ReID) framework and an Inflated 3D ResNet backbone, was developed. It achieved 76.32% Rank-1 and 65.92% mean average precision on the testing set and successfully identified the MOAs for six untrained drugs on the basis of mitochondrial phenotype. Furthermore, MitoReID identified cyclooxygenase-2 inhibition as the MOA of the natural compound epicatechin in tea, which was successfully validated in vitro. Our approach thus provides an automated and cost-effective alternative for target identification that could accelerate large-scale drug discovery and repurposing. A deep learning-based model, MitoReID, is presented for profiling changes in mitochondrial phenotypes, allowing for the identification of various drugs’ mechanism of action.},
  archive      = {J_NATCS},
  author       = {Yu, Min and Li, Weiming and Yu, Yunru and Zhao, Yu and Xiao, Lizhi and Lauschke, Volker M. and Cheng, Yiyu and Zhang, Xingcai and Wang, Yi},
  doi          = {10.1038/s43588-024-00679-4},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {600-614},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deep learning large-scale drug discovery and repurposing},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Network model with internal complexity bridges artificial
intelligence and neuroscience. <em>NATCS</em>, <em>4</em>(8), 584–599.
(<a href="https://doi.org/10.1038/s43588-024-00674-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Artificial intelligence (AI) researchers currently believe that the main approach to building more general model problems is the big AI model, where existing neural networks are becoming deeper, larger and wider. We term this the big model with external complexity approach. In this work we argue that there is another approach called small model with internal complexity, which can be used to find a suitable path of incorporating rich properties into neurons to construct larger and more efficient AI models. We uncover that one has to increase the scale of the network externally to stimulate the same dynamical properties. To illustrate this, we build a Hodgkin–Huxley (HH) network with rich internal complexity, where each neuron is an HH model, and prove that the dynamical properties and performance of the HH network can be equivalent to a bigger leaky integrate-and-fire (LIF) network, where each neuron is a LIF neuron with simple internal complexity. This study shows that by enhancing internal complexity of neurons in a Hodgkin–Huxley network, similar performance to larger, simpler networks can be achieved, suggesting an alternative path for powerful AI systems by focusing on neuron complexity.},
  archive      = {J_NATCS},
  author       = {He, Linxuan and Xu, Yunhui and He, Weihua and Lin, Yihan and Tian, Yang and Wu, Yujie and Wang, Wenhui and Zhang, Ziyang and Han, Junwei and Tian, Yonghong and Xu, Bo and Li, Guoqi},
  doi          = {10.1038/s43588-024-00674-9},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {584-599},
  shortjournal = {Nat. Comput. Sci},
  title        = {Network model with internal complexity bridges artificial intelligence and neuroscience},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Computational design of mechanical metamaterials.
<em>NATCS</em>, <em>4</em>(8), 574–583. (<a
href="https://doi.org/10.1038/s43588-024-00672-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the past few years, design of mechanical metamaterials has been empowered by computational tools that have allowed the community to overcome limitations of human intuition. By leveraging efficient optimization algorithms and computational physics models, it is now possible to explore vast design spaces, achieving new material functionalities with unprecedented performance. Here, we present our viewpoint on the state of the art of computational metamaterials design, discussing recent advances in topology optimization and machine learning design with respect to challenges in additive manufacturing. Computational tools have recently empowered mechanical metamaterials design. In this Perspective, advances to these approaches are discussed, notably mechanism-based design, topology optimization, the use of machine learning and the challenges for additive-manufactured metamaterial structures.},
  archive      = {J_NATCS},
  author       = {Bonfanti, Silvia and Hiemer, Stefan and Zulkarnain, Raja and Guerra, Roberto and Zaiser, Michael and Zapperi, Stefano},
  doi          = {10.1038/s43588-024-00672-x},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {574-583},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational design of mechanical metamaterials},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Programmable responsive metamaterials for mechanical
computing and robotics. <em>NATCS</em>, <em>4</em>(8), 567–573. (<a
href="https://doi.org/10.1038/s43588-024-00673-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Unconventional computing based on mechanical metamaterials has been of growing interest, including how such metamaterials might process information via autonomous interactions with their environment. Here we describe recent efforts to combine responsive materials with nonlinear mechanical metamaterials to achieve stimuli-responsive mechanical logic and computation. We also describe some key challenges and opportunities in the design and construction of these devices, including the lack of comprehensive computational tools, and the challenges associated with patterning multi-material mechanisms. Mechanical metamaterials have shown potential for processing information via autonomous environmental interactions. This Perspective summarizes recent efforts and challenges on integrating stimuli-responsive materials with mechanical metamaterials for mechanical computing, and explores the remaining challenges in the field.},
  archive      = {J_NATCS},
  author       = {He, Qiguang and Ferracin, Samuele and Raney, Jordan R.},
  doi          = {10.1038/s43588-024-00673-w},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {567-573},
  shortjournal = {Nat. Comput. Sci},
  title        = {Programmable responsive metamaterials for mechanical computing and robotics},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A machine learning tool to efficiently calculate
electron–phonon coupling. <em>NATCS</em>, <em>4</em>(8), 565–566. (<a
href="https://doi.org/10.1038/s43588-024-00680-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A machine learning framework that uses atomic orbital-based Hamiltonian matrices and gradients predicted by an equivariant graph neural network is established to calculate electron–phonon coupling (EPC). This approach accelerates the calculations by several orders of magnitude, enabling EPC-related properties to be predicted for complex systems using highly accurate functionals.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00680-x},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {565-566},
  shortjournal = {Nat. Comput. Sci},
  title        = {A machine learning tool to efficiently calculate electron–phonon coupling},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). AI-recognized mitochondrial phenotype enables identification
of drug targets. <em>NATCS</em>, <em>4</em>(8), 563–564. (<a
href="https://doi.org/10.1038/s43588-024-00682-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Revealing a drug’s mechanism of action (MOA) is costly and time-consuming. In this study, we used deep learning to extract temporal mitochondrial phenotypic features after exposure to drugs with known MOAs using re-identification algorithms. The trained model could then predict the MOAs of unidentified substances, facilitating phenotypic screening-based drug discovery and repurposing.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00682-9},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {563-564},
  shortjournal = {Nat. Comput. Sci},
  title        = {AI-recognized mitochondrial phenotype enables identification of drug targets},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accelerating predictions of electronic transport and
superconductivity. <em>NATCS</em>, <em>4</em>(8), 561–562. (<a
href="https://doi.org/10.1038/s43588-024-00678-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By developing a machine learning framework, a recent study substantially accelerates the calculation of electron–phonon coupling, making it computationally feasible to predict and understand a range of important physical phenomena, including electronic transport, hot-carrier relaxation, and superconductivity in complex materials.},
  archive      = {J_NATCS},
  author       = {Cao, Ting},
  doi          = {10.1038/s43588-024-00678-5},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {561-562},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerating predictions of electronic transport and superconductivity},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Bridging the gap between artificial intelligence and natural
intelligence. <em>NATCS</em>, <em>4</em>(8), 559–560. (<a
href="https://doi.org/10.1038/s43588-024-00677-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {According to a recent study, a small network consisting of four leaky integrate-and-fire neurons can reproduce the behavior of a single Hodgkin–Huxley neuron, thereby bridging the gap between endogenous and exogenous complexity.},
  archive      = {J_NATCS},
  author       = {Zhu, Rui-Jie and Gunasekaran, Skye and Eshraghian, Jason},
  doi          = {10.1038/s43588-024-00677-6},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {559-560},
  shortjournal = {Nat. Comput. Sci},
  title        = {Bridging the gap between artificial intelligence and natural intelligence},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Enhancing structured light with optimized metasurfaces.
<em>NATCS</em>, <em>4</em>(8), 558. (<a
href="https://doi.org/10.1038/s43588-024-00684-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-024-00684-7},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {558},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enhancing structured light with optimized metasurfaces},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024d). Synergy between photonic metamaterials and AI.
<em>NATCS</em>, <em>4</em>(8), 556–557. (<a
href="https://doi.org/10.1038/s43588-024-00675-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Yongmin Liu — professor of mechanical and industrial engineering and professor of electrical and computer engineering at Northeastern University — talks to Nature Computational Science about his career trajectory, his research on photonic metamaterials, and the synergistic effects between photonic metamaterials research and artificial intelligence (AI).},
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-024-00675-8},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {556-557},
  shortjournal = {Nat. Comput. Sci},
  title        = {Synergy between photonic metamaterials and AI},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Computational challenges in additive manufacturing for
metamaterials design. <em>NATCS</em>, <em>4</em>(8), 553–555. (<a
href="https://doi.org/10.1038/s43588-024-00669-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Additive manufacturing plays an essential role in producing metamaterials by precisely controlling geometries and multiscale structures to achieve the desired properties. In this Comment, we highlight the challenges and opportunities from additive manufacturing for computational metamaterials design.},
  archive      = {J_NATCS},
  author       = {Brown, Keith A. and Gu, Grace X.},
  doi          = {10.1038/s43588-024-00669-6},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {553-555},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational challenges in additive manufacturing for metamaterials design},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Computational design of art-inspired metamaterials.
<em>NATCS</em>, <em>4</em>(8), 549–552. (<a
href="https://doi.org/10.1038/s43588-024-00671-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In recent years, there has been a surge of interest in the design of mechanical metamaterials for different science and engineering applications. In particular, various computational approaches have been developed to facilitate the systematic design of art-inspired metamaterials including origami and kirigami metamaterials. In this Comment, we highlight the recent advances and discuss the outlook for the computational design of art-inspired metamaterials.},
  archive      = {J_NATCS},
  author       = {Choi, Gary P. T.},
  doi          = {10.1038/s43588-024-00671-y},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {549-552},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational design of art-inspired metamaterials},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Exploring the role of metamaterials in achieving advantage
in optical computing. <em>NATCS</em>, <em>4</em>(8), 545–548. (<a
href="https://doi.org/10.1038/s43588-024-00657-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical and wave-based computing is attracting renewed interest, motivated by the need for new platforms for resource-intensive special-purpose processing tasks. Here, we discuss whether, why, and how metamaterials and metasurfaces could contribute to achieving an ‘optical advantage’ in computing.},
  archive      = {J_NATCS},
  author       = {Li, Yandong and Monticone, Francesco},
  doi          = {10.1038/s43588-024-00657-w},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {545-548},
  shortjournal = {Nat. Comput. Sci},
  title        = {Exploring the role of metamaterials in achieving advantage in optical computing},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Metamaterials design via and for computation.
<em>NATCS</em>, <em>4</em>(8), 543–544. (<a
href="https://doi.org/10.1038/s43588-024-00687-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This issue of Nature Computational Science features a Focus that highlights recent advancements, challenges, and opportunities in computational models for metamaterials design and manufacturing, as well as explores their potential promises in emerging information processors and computing technologies.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00687-4},
  journal      = {Nature Computational Science},
  month        = {8},
  number       = {8},
  pages        = {543-544},
  shortjournal = {Nat. Comput. Sci},
  title        = {Metamaterials design via and for computation},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Free-form and multi-physical metamaterials with forward
conformality-assisted tracing. <em>NATCS</em>, <em>4</em>(7), 532–541.
(<a href="https://doi.org/10.1038/s43588-024-00660-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transformation theory, active control and inverse design have been mainstream in creating free-form metamaterials. However, existing frameworks cannot simultaneously satisfy the requirements of isotropic, passive and forward design. Here we propose a forward conformality-assisted tracing method to address the geometric and single-physical-field constraints of conformal transformation. Using a conformal mesh composed of orthogonal streamlines and isotherms (or isothermal surfaces), this method quasi-analytically produces free-form metamaterials using only isotropic media. The geometric nature of this approach allows for universal regulation of both dissipative thermal fields and non-dissipative electromagnetic fields. We experimentally demonstrate free-form thermal cloaking in both two and three dimensions. Additionally, the multi-physical functionalities of our method, including optical cloaking, bending and thermo-electric transparency, confirm its broad applicability. Our method features improvements in efficiency, accuracy and adaptability over previous approaches. This study provides an effective method for designing complex metamaterials with arbitrary shapes across various physical domains. Here a conformality-assisted tracing method is proposed to devise free-form and three-dimensional conformal metamaterials, featuring accuracy and efficiency in handling complex geometry and adaptability to various diffusion and wave fields.},
  archive      = {J_NATCS},
  author       = {Xu, Liujun and Dai, Gaole and Yang, Fubao and Liu, Jinrong and Zhou, Yuhong and Wang, Jun and Xu, Guoqiang and Huang, Jiping and Qiu, Cheng-Wei},
  doi          = {10.1038/s43588-024-00660-1},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {532-541},
  shortjournal = {Nat. Comput. Sci},
  title        = {Free-form and multi-physical metamaterials with forward conformality-assisted tracing},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Virtual node graph neural network for full phonon
prediction. <em>NATCS</em>, <em>4</em>(7), 522–531. (<a
href="https://doi.org/10.1038/s43588-024-00661-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding the structure–property relationship is crucial for designing materials with desired properties. The past few years have witnessed remarkable progress in machine-learning methods for this connection. However, substantial challenges remain, including the generalizability of models and prediction of properties with materials-dependent output dimensions. Here we present the virtual node graph neural network to address the challenges. By developing three virtual node approaches, we achieve Γ-phonon spectra and full phonon dispersion prediction from atomic coordinates. We show that, compared with the machine-learning interatomic potentials, our approach achieves orders-of-magnitude-higher efficiency with comparable to better accuracy. This allows us to generate databases for Γ-phonon containing over 146,000 materials and phonon band structures of zeolites. Our work provides an avenue for rapid and high-quality prediction of phonon band structures enabling materials design with desired phonon properties. The virtual node method also provides a generic method for machine-learning design with a high level of flexibility. In this study, the authors present a virtual node graph neural network to enable the prediction of material properties with variable output dimensions. This method offers fast and accurate predictions of phonon band structures in complex solids.},
  archive      = {J_NATCS},
  author       = {Okabe, Ryotaro and Chotrattanapituk, Abhijatmedhi and Boonkird, Artittaya and Andrejevic, Nina and Fu, Xiang and Jaakkola, Tommi S. and Song, Qichen and Nguyen, Thanh and Drucker, Nathan and Mu, Sai and Wang, Yao and Liao, Bolin and Cheng, Yongqiang and Li, Mingda},
  doi          = {10.1038/s43588-024-00661-0},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {522-531},
  shortjournal = {Nat. Comput. Sci},
  title        = {Virtual node graph neural network for full phonon prediction},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Structure-based prediction of t cell receptor recognition of
unseen epitopes using TCRen. <em>NATCS</em>, <em>4</em>(7), 510–521. (<a
href="https://doi.org/10.1038/s43588-024-00653-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {T cell receptor (TCR) recognition of foreign peptides presented by major histocompatibility complex protein is a major event in triggering the adaptive immune response to pathogens or cancer. The prediction of TCR–peptide interactions has great importance for therapy of cancer as well as infectious and autoimmune diseases but remains a major challenge, particularly for novel (unseen) peptide epitopes. Here we present TCRen, a structure-based method for ranking candidate unseen epitopes for a given TCR. The first stage of the TCRen pipeline is modeling of the TCR–peptide–major histocompatibility complex structure. Then a TCR–peptide residue contact map is extracted from this structure and used to rank all candidate epitopes on the basis of an interaction score with the target TCR. Scoring is performed using an energy potential derived from the statistics of TCR–peptide contact preferences in existing crystal structures. We show that TCRen has high performance in discriminating cognate versus unrelated peptides and can facilitate the identification of cancer neoepitopes recognized by tumor-infiltrating lymphocytes. TCRen predicts TCR specificity by modeling the TCR–peptide–MHC structure and estimating the TCR–peptide interaction energy using a statistical potential. The use of structural information allows TCRen to generalize to unseen epitopes, such as cancer neoepitopes.},
  archive      = {J_NATCS},
  author       = {Karnaukhov, Vadim K. and Shcherbinin, Dmitrii S. and Chugunov, Anton O. and Chudakov, Dmitriy M. and Efremov, Roman G. and Zvyagin, Ivan V. and Shugay, Mikhail},
  doi          = {10.1038/s43588-024-00653-0},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {510-521},
  shortjournal = {Nat. Comput. Sci},
  title        = {Structure-based prediction of t cell receptor recognition of unseen epitopes using TCRen},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Overcoming data scarcity in biomedical imaging with a
foundational multi-task model. <em>NATCS</em>, <em>4</em>(7), 495–509.
(<a href="https://doi.org/10.1038/s43588-024-00662-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Foundational models, pretrained on a large scale, have demonstrated substantial success across non-medical domains. However, training these models typically requires large, comprehensive datasets, which contrasts with the smaller and more specialized datasets common in biomedical imaging. Here we propose a multi-task learning strategy that decouples the number of training tasks from memory requirements. We trained a universal biomedical pretrained model (UMedPT) on a multi-task database including tomographic, microscopic and X-ray images, with various labeling strategies such as classification, segmentation and object detection. The UMedPT foundational model outperformed ImageNet pretraining and previous state-of-the-art models. For classification tasks related to the pretraining database, it maintained its performance with only 1% of the original training data and without fine-tuning. For out-of-domain tasks it required only 50% of the original training data. In an external independent validation, imaging features extracted using UMedPT proved to set a new standard for cross-center transferability. UMedPT, a foundational model for biomedical imaging, has been trained on a variety of medical tasks with different types of label. It has achieved high performance with less training data in various clinical applications.},
  archive      = {J_NATCS},
  author       = {Schäfer, Raphael and Nicke, Till and Höfener, Henning and Lange, Annkristin and Merhof, Dorit and Feuerhake, Friedrich and Schulz, Volkmar and Lotz, Johannes and Kiessling, Fabian},
  doi          = {10.1038/s43588-024-00662-z},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {495-509},
  shortjournal = {Nat. Comput. Sci},
  title        = {Overcoming data scarcity in biomedical imaging with a foundational multi-task model},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Promising directions of machine learning for partial
differential equations. <em>NATCS</em>, <em>4</em>(7), 483–494. (<a
href="https://doi.org/10.1038/s43588-024-00643-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Partial differential equations (PDEs) are among the most universal and parsimonious descriptions of natural physical laws, capturing a rich variety of phenomenology and multiscale physics in a compact and symbolic representation. Here, we examine several promising avenues of PDE research that are being advanced by machine learning, including (1) discovering new governing PDEs and coarse-grained approximations for complex natural and engineered systems, (2) learning effective coordinate systems and reduced-order models to make PDEs more amenable to analysis, and (3) representing solution operators and improving traditional numerical algorithms. In each of these fields, we summarize key advances, ongoing challenges, and opportunities for further development. Machine learning has enabled major advances in the field of partial differential equations. This Review discusses some of these efforts and other ongoing challenges and opportunities for development.},
  archive      = {J_NATCS},
  author       = {Brunton, Steven L. and Kutz, J. Nathan},
  doi          = {10.1038/s43588-024-00643-2},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {483-494},
  shortjournal = {Nat. Comput. Sci},
  title        = {Promising directions of machine learning for partial differential equations},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Boosting graph neural networks with virtual nodes to predict
phonon properties. <em>NATCS</em>, <em>4</em>(7), 481–482. (<a
href="https://doi.org/10.1038/s43588-024-00665-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A graph neural network using virtual nodes is proposed to predict the properties of complex materials with variable dimensions or dimensions that depend on the input. The method is used to accurately and quickly predict phonon dispersion relations in complex solids and alloys.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00665-w},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {481-482},
  shortjournal = {Nat. Comput. Sci},
  title        = {Boosting graph neural networks with virtual nodes to predict phonon properties},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A multi-task learning strategy to pretrain models for
medical image analysis. <em>NATCS</em>, <em>4</em>(7), 479–480. (<a
href="https://doi.org/10.1038/s43588-024-00666-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pretraining powerful deep learning models requires large, comprehensive training datasets, which are often unavailable for medical imaging. In response, the universal biomedical pretrained (UMedPT) foundational model was developed based on multiple small and medium-sized datasets. This model reduced the amount of data required to learn new target tasks by at least 50%.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00666-9},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {479-480},
  shortjournal = {Nat. Comput. Sci},
  title        = {A multi-task learning strategy to pretrain models for medical image analysis},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Free-form metamaterials design with isotropic materials.
<em>NATCS</em>, <em>4</em>(7), 477–478. (<a
href="https://doi.org/10.1038/s43588-024-00663-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposes a computational method for the design of free-form metamaterials systems. The method simplifies the design process by avoiding the use of anisotropic materials that are usually required for the conventional methods. The method can be applied in designing both two-dimensional and three-dimensional metamaterials that are subject to multiple physical fields.},
  archive      = {J_NATCS},
  author       = {Restrepo-Flórez, Juan Manuel},
  doi          = {10.1038/s43588-024-00663-y},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {477-478},
  shortjournal = {Nat. Comput. Sci},
  title        = {Free-form metamaterials design with isotropic materials},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unlocking t-cell receptor–epitope insights with structural
analysis. <em>NATCS</em>, <em>4</em>(7), 475–476. (<a
href="https://doi.org/10.1038/s43588-024-00654-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A method leverages protein structural data to predict T-cell receptor–peptide interactions for unseen peptide epitopes, which can be particularly useful for applications in cancer immunotherapy, autoimmunity studies, and vaccine design.},
  archive      = {J_NATCS},
  author       = {Huo, Miaozhe and Jiang, Yuepeng and Li, Shuai Cheng},
  doi          = {10.1038/s43588-024-00654-z},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {475-476},
  shortjournal = {Nat. Comput. Sci},
  title        = {Unlocking T-cell receptor–epitope insights with structural analysis},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Multi-task learning for medical foundation models.
<em>NATCS</em>, <em>4</em>(7), 473–474. (<a
href="https://doi.org/10.1038/s43588-024-00658-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To address the challenge of pretraining foundational models with large datasets, a multi-task approach is proposed, thus helping to overcome the data scarcity problem in biomedical imaging.},
  archive      = {J_NATCS},
  author       = {Yang, Jiancheng},
  doi          = {10.1038/s43588-024-00658-9},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {473-474},
  shortjournal = {Nat. Comput. Sci},
  title        = {Multi-task learning for medical foundation models},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Enhancing human mobility research with open and standardized
datasets. <em>NATCS</em>, <em>4</em>(7), 469–472. (<a
href="https://doi.org/10.1038/s43588-024-00650-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Human mobility research intersects with various disciplines, with profound implications for urban planning, transportation engineering, public health, disaster management, and economic analysis. Here, we discuss the urgent need for open and standardized datasets in the field, including current challenges and lessons from other computational science domains, and propose collaborative efforts to enhance the validity and reproducibility of human mobility research.},
  archive      = {J_NATCS},
  author       = {Yabe, Takahiro and Luca, Massimiliano and Tsubouchi, Kota and Lepri, Bruno and Gonzalez, Marta C. and Moro, Esteban},
  doi          = {10.1038/s43588-024-00650-3},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {469-472},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enhancing human mobility research with open and standardized datasets},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Software in science is ubiquitous yet overlooked.
<em>NATCS</em>, <em>4</em>(7), 465–468. (<a
href="https://doi.org/10.1038/s43588-024-00651-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Software is much more than just code. It is time to confront the complexity of licenses, uses, governance, infrastructure and other facets of software in science. Their influence is ubiquitous yet overlooked.},
  archive      = {J_NATCS},
  author       = {Hocquet, Alexandre and Wieber, Frédéric and Gramelsberger, Gabriele and Hinsen, Konrad and Diesmann, Markus and Pasquini Santos, Fernando and Landström, Catharina and Peters, Benjamin and Kasprowicz, Dawid and Borrelli, Arianna and Roth, Phillip and Lee, Clarissa Ai Ling and Olteanu, Alin and Böschen, Stefan},
  doi          = {10.1038/s43588-024-00651-2},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {465-468},
  shortjournal = {Nat. Comput. Sci},
  title        = {Software in science is ubiquitous yet overlooked},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Computational electrochemistry of oxygen 150 years after
priestley. <em>NATCS</em>, <em>4</em>(7), 462–464. (<a
href="https://doi.org/10.1038/s43588-024-00664-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Since the first isolation of oxygen, chemists have explored oxygen reduction and evolution reactions. Now, computational chemists are trying to understand and predict the best catalysts for them. Here, the importance of various considerations for such calculations, as well as their challenges and opportunities, are discussed.},
  archive      = {J_NATCS},
  author       = {Jiang, De-en},
  doi          = {10.1038/s43588-024-00664-x},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {462-464},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational electrochemistry of oxygen 150 years after priestley},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). 150 years of oxygen chemistry. <em>NATCS</em>,
<em>4</em>(7), 461. (<a
href="https://doi.org/10.1038/s43588-024-00670-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We look back on the discovery of oxygen in light of its upcoming milestone anniversary and highlight the computational contributions to oxygen reduction and evolution in chemistry.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00670-z},
  journal      = {Nature Computational Science},
  month        = {7},
  number       = {7},
  pages        = {461},
  shortjournal = {Nat. Comput. Sci},
  title        = {150 years of oxygen chemistry},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Computing the committor with the committor to study the
transition state ensemble. <em>NATCS</em>, <em>4</em>(6), 451–460. (<a
href="https://doi.org/10.1038/s43588-024-00645-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The study of the kinetic bottlenecks that hinder the rare transitions between long-lived metastable states is a major challenge in atomistic simulations. Here we propose a method to explore the transition state ensemble, which is the distribution of configurations that the system passes through as it translocates from one metastable basin to another. We base our method on the committor function and the variational principle that it obeys. We find its minimum through a self-consistent procedure that starts from information limited to the initial and final states. Right from the start, our procedure allows the sampling of very many transition state configurations. With the help of the variational principle, we perform a detailed analysis of the transition state ensemble, ranking quantitatively the degrees of freedom mostly involved in the transition and enabling a systematic approach for the interpretation of simulation results and the construction of efficient physics-informed collective variables. A self-consistent iterative procedure is proposed to compute the committor function for rare events, via a variational principle, and extensively sample the transition state ensemble, allowing for the identification of the relevant variables in the process.},
  archive      = {J_NATCS},
  author       = {Kang, Peilin and Trizio, Enrico and Parrinello, Michele},
  doi          = {10.1038/s43588-024-00645-0},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {451-460},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computing the committor with the committor to study the transition state ensemble},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An algorithmic framework for synthetic cost-aware decision
making in molecular design. <em>NATCS</em>, <em>4</em>(6), 440–450. (<a
href="https://doi.org/10.1038/s43588-024-00639-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Small molecules exhibiting desirable property profiles are often discovered through an iterative process of designing, synthesizing and testing sets of molecules. The selection of molecules to synthesize from all possible candidates is a complex decision-making process that typically relies on expert chemist intuition. Here we propose a quantitative decision-making framework, SPARROW, that prioritizes molecules for evaluation by balancing expected information gain and synthetic cost. SPARROW integrates molecular design, property prediction and retrosynthetic planning to balance the utility of testing a molecule with the cost of batch synthesis. We demonstrate, through three case studies, that the developed algorithm captures the non-additive costs inherent to batch synthesis, leverages common reaction steps and intermediates, and scales to hundreds of molecules. The downselection of compounds for synthesis is a key challenge in molecular design cycles that typically relies on expert chemist intuition. Fromer and Coley propose a cost-aware method to automatically select compounds and synthetic routes.},
  archive      = {J_NATCS},
  author       = {Fromer, Jenna C. and Coley, Connor W.},
  doi          = {10.1038/s43588-024-00639-y},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {440-450},
  shortjournal = {Nat. Comput. Sci},
  title        = {An algorithmic framework for synthetic cost-aware decision making in molecular design},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Large-scale photonic computing with nonlinear disordered
media. <em>NATCS</em>, <em>4</em>(6), 429–439. (<a
href="https://doi.org/10.1038/s43588-024-00644-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Neural networks find widespread use in scientific and technological applications, yet their implementations in conventional computers have encountered bottlenecks due to ever-expanding computational needs. Photonic computing is a promising neuromorphic platform with potential advantages of massive parallelism, ultralow latency and reduced energy consumption but mostly for computing linear operations. Here we demonstrate a large-scale, high-performance nonlinear photonic neural system based on a disordered polycrystalline slab composed of lithium niobate nanocrystals. Mediated by random quasi-phase-matching and multiple scattering, linear and nonlinear optical speckle features are generated as the interplay between the simultaneous linear random scattering and the second-harmonic generation, defining a complex neural network in which the second-order nonlinearity acts as internal nonlinear activation functions. Benchmarked against linear random projection, such nonlinear mapping embedded with rich physical computational operations shows improved performance across a large collection of machine learning tasks in image classification, regression and graph classification. Demonstrating up to 27,648 input and 3,500 nonlinear output nodes, the combination of optical nonlinearity and random scattering serves as a scalable computing engine for diverse applications. Nonlinear optical computations have been essential yet challenging for developing optical neural networks with appreciable expressivity. In this paper, light scattering is combined with optical nonlinearity to empower a high-performance, large-scale nonlinear photonic neural system.},
  archive      = {J_NATCS},
  author       = {Wang, Hao and Hu, Jianqi and Morandi, Andrea and Nardi, Alfonso and Xia, Fei and Li, Xuanchen and Savo, Romolo and Liu, Qiang and Grange, Rachel and Gigan, Sylvain},
  doi          = {10.1038/s43588-024-00644-1},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {429-439},
  shortjournal = {Nat. Comput. Sci},
  title        = {Large-scale photonic computing with nonlinear disordered media},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Scalable design of orthogonal DNA barcode libraries.
<em>NATCS</em>, <em>4</em>(6), 423–428. (<a
href="https://doi.org/10.1038/s43588-024-00646-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Orthogonal DNA barcode library design is an essential task in bioengineering. Here we present seqwalk, an efficient method for designing barcode libraries that satisfy a sequence symmetry minimization (SSM) heuristic for orthogonality, with theoretical guarantees of maximal or near-maximal library size under certain design constraints. Seqwalk encodes SSM constraints in a de Bruijn graph representation of sequence space, enabling the application of recent advances in discrete mathematics1 to the problem of orthogonal sequence design. We demonstrate the scalability of seqwalk by designing a library of &amp;gt;106 SSM-satisfying barcode sequences in less than 20 s on a standard laptop. Seqwalk is a scalable method for designing orthogonal DNA barcode libraries, producing one million barcodes in 20 s on a standard laptop.},
  archive      = {J_NATCS},
  author       = {Gowri, Gokul and Sheng, Kuanwei and Yin, Peng},
  doi          = {10.1038/s43588-024-00646-z},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {423-428},
  shortjournal = {Nat. Comput. Sci},
  title        = {Scalable design of orthogonal DNA barcode libraries},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Linguistics-based formalization of the antibody language as
a basis for antibody language models. <em>NATCS</em>, <em>4</em>(6),
412–422. (<a href="https://doi.org/10.1038/s43588-024-00642-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Apparent parallels between natural language and antibody sequences have led to a surge in deep language models applied to antibody sequences for predicting cognate antigen recognition. However, a linguistic formal definition of antibody language does not exist, and insight into how antibody language models capture antibody-specific binding features remains largely uninterpretable. Here we describe how a linguistic formalization of the antibody language, by characterizing its tokens and grammar, could address current challenges in antibody language model rule mining. The parallels between natural language and antibody sequences could serve as a stepping stone to using deep language models for analyzing antibody sequences. This Perspective discusses how issues in antibody language model rule mining could be addressed by linguistically formalizing the antibody language.},
  archive      = {J_NATCS},
  author       = {Vu, Mai Ha and Robert, Philippe A. and Akbar, Rahmad and Swiatczak, Bartlomiej and Sandve, Geir Kjetil and Haug, Dag Trygve Truslew and Greiff, Victor},
  doi          = {10.1038/s43588-024-00642-3},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {412-422},
  shortjournal = {Nat. Comput. Sci},
  title        = {Linguistics-based formalization of the antibody language as a basis for antibody language models},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The exciting potential and daunting challenge of using GPS
human-mobility data for epidemic modeling. <em>NATCS</em>,
<em>4</em>(6), 398–411. (<a
href="https://doi.org/10.1038/s43588-024-00637-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale GPS location datasets hold immense potential for measuring human mobility and interpersonal contact, both of which are essential for data-driven epidemiology. However, despite their potential and widespread adoption during the COVID-19 pandemic, there are several challenges with these data that raise concerns regarding the validity and robustness of its applications. Here we outline two types of challenges—some related to accessing and processing these data, and some related to data quality—and propose several research directions to address them moving forward. While large-scale GPS location datasets have been instrumental to applications in epidemiology, there are still several challenges with these data that should be considered and addressed to make data-driven epidemiology more reliable.},
  archive      = {J_NATCS},
  author       = {Barreras, Francisco and Watts, Duncan J.},
  doi          = {10.1038/s43588-024-00637-0},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {398-411},
  shortjournal = {Nat. Comput. Sci},
  title        = {The exciting potential and daunting challenge of using GPS human-mobility data for epidemic modeling},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Systematic simulations and analysis of transition states
using committor functions. <em>NATCS</em>, <em>4</em>(6), 396–397. (<a
href="https://doi.org/10.1038/s43588-024-00652-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Data about the transition states of rare transitions between long-lived states are needed to simulate physical and chemical processes; however, existing computational approaches often gather little information about these states. A machine-learning technique resolves this challenge by exploiting the century-old theory of committor functions.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00652-1},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {396-397},
  shortjournal = {Nat. Comput. Sci},
  title        = {Systematic simulations and analysis of transition states using committor functions},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A nonlinear dimension for machine learning in optical
disordered media. <em>NATCS</em>, <em>4</em>(6), 394–395. (<a
href="https://doi.org/10.1038/s43588-024-00648-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study shows that, by leveraging nonlinear optical processes in disordered media, photonic processors can transform high-dimensional machine-learning data, using nonlinear functions that are otherwise challenging for digital electronic processors to compute.},
  archive      = {J_NATCS},
  author       = {Wang, Tianyu},
  doi          = {10.1038/s43588-024-00648-x},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {394-395},
  shortjournal = {Nat. Comput. Sci},
  title        = {A nonlinear dimension for machine learning in optical disordered media},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). The whole picture in digital pathology. <em>NATCS</em>,
<em>4</em>(6), 393. (<a
href="https://doi.org/10.1038/s43588-024-00655-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-024-00655-y},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {393},
  shortjournal = {Nat. Comput. Sci},
  title        = {The whole picture in digital pathology},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Integrating computational and experimental worlds.
<em>NATCS</em>, <em>4</em>(6), 391–392. (<a
href="https://doi.org/10.1038/s43588-024-00649-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Kelly Ruggles, associate professor at New York University Langone Health, discusses with Nature Computational Science how she uses computational approaches to gain insights into cancer, inflammation and cardiovascular disease, as well as the importance of mentorship.},
  archive      = {J_NATCS},
  author       = {Rastogi, Ananya},
  doi          = {10.1038/s43588-024-00649-w},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {391-392},
  shortjournal = {Nat. Comput. Sci},
  title        = {Integrating computational and experimental worlds},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Computational morphology and morphogenesis for empowering
soft-matter engineering. <em>NATCS</em>, <em>4</em>(6), 388–390. (<a
href="https://doi.org/10.1038/s43588-024-00647-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Morphing soft matter, which is capable of changing its shape and function in response to stimuli, has wide-ranging applications in robotics, medicine and biology. Recently, computational models have accelerated its development. Here, we highlight advances and challenges in developing computational techniques, and explore the potential applications enabled by such models.},
  archive      = {J_NATCS},
  author       = {Yang, Yifan and Xu, Fan},
  doi          = {10.1038/s43588-024-00647-y},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {388-390},
  shortjournal = {Nat. Comput. Sci},
  title        = {Computational morphology and morphogenesis for empowering soft-matter engineering},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Data as the next challenge in atomistic machine learning.
<em>NATCS</em>, <em>4</em>(6), 384–387. (<a
href="https://doi.org/10.1038/s43588-024-00636-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As machine learning models are becoming mainstream tools for molecular and materials research, there is an urgent need to improve the nature, quality, and accessibility of atomistic data. In turn, there are opportunities for a new generation of generally applicable datasets and distillable models.},
  archive      = {J_NATCS},
  author       = {Ben Mahmoud, Chiheb and Gardner, John L. A. and Deringer, Volker L.},
  doi          = {10.1038/s43588-024-00636-1},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {384-387},
  shortjournal = {Nat. Comput. Sci},
  title        = {Data as the next challenge in atomistic machine learning},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Advancing computational sustainability in higher education.
<em>NATCS</em>, <em>4</em>(6), 382–383. (<a
href="https://doi.org/10.1038/s43588-024-00638-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Kejriwal, Mayank and Petryshyn, Victoria},
  doi          = {10.1038/s43588-024-00638-z},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {382-383},
  shortjournal = {Nat. Comput. Sci},
  title        = {Advancing computational sustainability in higher education},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An invitation to social scientists. <em>NATCS</em>,
<em>4</em>(6), 381. (<a
href="https://doi.org/10.1038/s43588-024-00656-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nature Computational Science wants to publish your computational social science research.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00656-x},
  journal      = {Nature Computational Science},
  month        = {6},
  number       = {6},
  pages        = {381},
  shortjournal = {Nat. Comput. Sci},
  title        = {An invitation to social scientists},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Publisher correction: SimuCell3D: Three-dimensional
simulation of tissue mechanics with cell polarization. <em>NATCS</em>,
<em>4</em>(5), 379. (<a
href="https://doi.org/10.1038/s43588-024-00635-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Runser, Steve and Vetter, Roman and Iber, Dagmar},
  doi          = {10.1038/s43588-024-00635-2},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {379},
  shortjournal = {Nat. Comput. Sci},
  title        = {Publisher correction: SimuCell3D: three-dimensional simulation of tissue mechanics with cell polarization},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). MISATO: Machine learning dataset of protein–ligand complexes
for structure-based drug discovery. <em>NATCS</em>, <em>4</em>(5),
367–378. (<a href="https://doi.org/10.1038/s43588-024-00627-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large language models have greatly enhanced our ability to understand biology and chemistry, yet robust methods for structure-based drug discovery, quantum chemistry and structural biology are still sparse. Precise biomolecule–ligand interaction datasets are urgently needed for large language models. To address this, we present MISATO, a dataset that combines quantum mechanical properties of small molecules and associated molecular dynamics simulations of ~20,000 experimental protein–ligand complexes with extensive validation of experimental data. Starting from the existing experimental structures, semi-empirical quantum mechanics was used to systematically refine these structures. A large collection of molecular dynamics traces of protein–ligand complexes in explicit water is included, accumulating over 170 μs. We give examples of machine learning (ML) baseline models proving an improvement of accuracy by employing our data. An easy entry point for ML experts is provided to enable the next generation of drug discovery artificial intelligence models. MISATO is a database for structure-based drug discovery that combines quantum mechanics data with molecular dynamics simulations on ~20,000 protein–ligand structures. The artificial intelligence models included provide an easy entry point for the machine learning and drug discovery communities.},
  archive      = {J_NATCS},
  author       = {Siebenmorgen, Till and Menezes, Filipe and Benassou, Sabrina and Merdivan, Erinc and Didi, Kieran and Mourão, André Santos Dias and Kitel, Radosław and Liò, Pietro and Kesselheim, Stefan and Piraud, Marie and Theis, Fabian J. and Sattler, Michael and Popowicz, Grzegorz M.},
  doi          = {10.1038/s43588-024-00627-2},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {367-378},
  shortjournal = {Nat. Comput. Sci},
  title        = {MISATO: Machine learning dataset of protein–ligand complexes for structure-based drug discovery},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A resampling-based approach to share reference panels.
<em>NATCS</em>, <em>4</em>(5), 360–366. (<a
href="https://doi.org/10.1038/s43588-024-00630-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For many genome-wide association studies, imputing genotypes from a haplotype reference panel is a necessary step. Over the past 15 years, reference panels have become larger and more diverse, leading to improvements in imputation accuracy. However, the latest generation of reference panels is subject to restrictions on data sharing due to concerns about privacy, limiting their usefulness for genotype imputation. In this context, here we propose RESHAPE, a method that employs a recombination Poisson process on a reference panel to simulate the genomes of hypothetical descendants after multiple generations. This data transformation helps to protect against re-identification threats and preserves data attributes, such as linkage disequilibrium patterns and, to some degree, identity-by-descent sharing, allowing for genotype imputation. Our experiments on gold-standard datasets show that simulated descendants up to eight generations can serve as reference panels without substantially reducing genotype imputation accuracy. The authors develop the tool RESHAPE to share reference panels in a safer way. The genome–phenome links in reference panels can generate re-identification threats and RESHAPE breaks these links by shuffling haplotypes while preserving imputation accuracy.},
  archive      = {J_NATCS},
  author       = {Cavinato, Théo and Rubinacci, Simone and Malaspinas, Anna-Sapfo and Delaneau, Olivier},
  doi          = {10.1038/s43588-024-00630-7},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {360-366},
  shortjournal = {Nat. Comput. Sci},
  title        = {A resampling-based approach to share reference panels},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Discrete latent embedding of single-cell chromatin
accessibility sequencing data for uncovering cell heterogeneity.
<em>NATCS</em>, <em>4</em>(5), 346–359. (<a
href="https://doi.org/10.1038/s43588-024-00625-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell epigenomic data has been growing continuously at an unprecedented pace, but their characteristics such as high dimensionality and sparsity pose substantial challenges to downstream analysis. Although deep learning models—especially variational autoencoders—have been widely used to capture low-dimensional feature embeddings, the prevalent Gaussian assumption somewhat disagrees with real data, and these models tend to struggle to incorporate reference information from abundant cell atlases. Here we propose CASTLE, a deep generative model based on the vector-quantized variational autoencoder framework to extract discrete latent embeddings that interpretably characterize single-cell chromatin accessibility sequencing data. We validate the performance and robustness of CASTLE for accurate cell-type identification and reasonable visualization compared with state-of-the-art methods. We demonstrate the advantages of CASTLE for effective incorporation of existing massive reference datasets in a weakly supervised or supervised manner. We further demonstrate CASTLE’s capacity for intuitively distilling cell-type-specific feature spectra that unveil cell heterogeneity and biological implications quantitatively. A method based on a vector-quantized variational autoencoder, called CASTLE, can interpretably extract discrete latent embeddings and quantitatively generate the cell-type-specific feature spectrum for single-cell chromatin accessibility sequencing data.},
  archive      = {J_NATCS},
  author       = {Cui, Xuejian and Chen, Xiaoyang and Li, Zhen and Gao, Zijing and Chen, Shengquan and Jiang, Rui},
  doi          = {10.1038/s43588-024-00625-4},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {346-359},
  shortjournal = {Nat. Comput. Sci},
  title        = {Discrete latent embedding of single-cell chromatin accessibility sequencing data for uncovering cell heterogeneity},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Automated discovery of symbolic laws governing skill
acquisition from naturally occurring data. <em>NATCS</em>,
<em>4</em>(5), 334–345. (<a
href="https://doi.org/10.1038/s43588-024-00629-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Skill acquisition is a key area of research in cognitive psychology as it encompasses multiple psychological processes. The laws discovered under experimental paradigms are controversial and lack generalizability. This paper aims to unearth the laws of skill learning from large-scale training log data. A two-stage algorithm was developed to tackle the issues of unobservable cognitive states and an algorithmic explosion in searching. A deep learning model is initially employed to determine the learner’s cognitive state and assess the feature importance. Symbolic regression algorithms are then used to parse the neural network model into algebraic equations. Experimental results show that the algorithm can accurately restore preset laws within a noise range in continuous feedback settings. When applied to Lumosity training data, the method outperforms traditional and recent models in fitness terms. The study reveals two new forms of skill acquisition laws and reaffirms some previous findings. This paper introduces an algorithm to uncover laws of skill acquisition from naturally occurring data. By combining deep learning and symbolic regression, it accurately identifies cognitive states and extracts algebraic equations.},
  archive      = {J_NATCS},
  author       = {Liu, Sannyuya and Li, Qing and Shen, Xiaoxuan and Sun, Jianwen and Yang, Zongkai},
  doi          = {10.1038/s43588-024-00629-0},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {334-345},
  shortjournal = {Nat. Comput. Sci},
  title        = {Automated discovery of symbolic laws governing skill acquisition from naturally occurring data},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Designing semiconductor materials and devices in the
post-moore era by tackling computational challenges with data-driven
strategies. <em>NATCS</em>, <em>4</em>(5), 322–333. (<a
href="https://doi.org/10.1038/s43588-024-00632-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the post-Moore’s law era, the progress of electronics relies on discovering superior semiconductor materials and optimizing device fabrication. Computational methods, augmented by emerging data-driven strategies, offer a promising alternative to the traditional trial-and-error approach. In this Perspective, we highlight data-driven computational frameworks for enhancing semiconductor discovery and device development by elaborating on their advances in exploring the materials design space, predicting semiconductor properties and optimizing device fabrication, with a concluding discussion on the challenges and opportunities in these areas. Discovering improved semiconductor materials is essential for optimal device fabrication. In this Perspective, data-driven computational frameworks for semiconductor discovery and device development are discussed, including the challenges and opportunities moving forward.},
  archive      = {J_NATCS},
  author       = {Xie, Jiahao and Zhou, Yansong and Faizan, Muhammad and Li, Zewei and Li, Tianshu and Fu, Yuhao and Wang, Xinjiang and Zhang, Lijun},
  doi          = {10.1038/s43588-024-00632-5},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {322-333},
  shortjournal = {Nat. Comput. Sci},
  title        = {Designing semiconductor materials and devices in the post-moore era by tackling computational challenges with data-driven strategies},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Shuffling haplotypes to share reference panels for
imputation. <em>NATCS</em>, <em>4</em>(5), 320–321. (<a
href="https://doi.org/10.1038/s43588-024-00640-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a method to alleviate re-identification risks behind sharing haplotype reference panels for imputation. In an anonymized reference panel, one might try to infer the genomes’ phenotypes to re-identify their owner. Our method protects against such attack by shuffling the reference panels genomes while maintaining imputation accuracy.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00640-5},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {320-321},
  shortjournal = {Nat. Comput. Sci},
  title        = {Shuffling haplotypes to share reference panels for imputation},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A multidimensional dataset for structure-based machine
learning. <em>NATCS</em>, <em>4</em>(5), 318–319. (<a
href="https://doi.org/10.1038/s43588-024-00631-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {MISATO, a dataset for structure-based drug discovery combines quantum mechanics property data and molecular dynamics simulations on ~20,000 protein–ligand structures, substantially extends the amount of data available to the community and holds potential for advancing work in drug discovery.},
  archive      = {J_NATCS},
  author       = {Holcomb, Matthew and Forli, Stefano},
  doi          = {10.1038/s43588-024-00631-6},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {318-319},
  shortjournal = {Nat. Comput. Sci},
  title        = {A multidimensional dataset for structure-based machine learning},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Discrete latent embeddings illuminate cellular diversity in
single-cell epigenomics. <em>NATCS</em>, <em>4</em>(5), 316–317. (<a
href="https://doi.org/10.1038/s43588-024-00634-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CASTLE, a deep learning approach, extracts interpretable discrete representations from single-cell chromatin accessibility data, enabling accurate cell type identification, effective data integration, and quantitative insights into gene regulatory mechanisms.},
  archive      = {J_NATCS},
  author       = {Wei, Zhi},
  doi          = {10.1038/s43588-024-00634-3},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {316-317},
  shortjournal = {Nat. Comput. Sci},
  title        = {Discrete latent embeddings illuminate cellular diversity in single-cell epigenomics},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Outsourcing eureka moments to artificial intelligence.
<em>NATCS</em>, <em>4</em>(5), 314–315. (<a
href="https://doi.org/10.1038/s43588-024-00633-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A two-stage learning algorithm is proposed to directly uncover the symbolic representation of rules for skill acquisition from large-scale training log data.},
  archive      = {J_NATCS},
  author       = {Meeter, Martijn},
  doi          = {10.1038/s43588-024-00633-4},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {314-315},
  shortjournal = {Nat. Comput. Sci},
  title        = {Outsourcing eureka moments to artificial intelligence},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Advancements in multicellular simulations. <em>NATCS</em>,
<em>4</em>(5), 312–313. (<a
href="https://doi.org/10.1038/s43588-024-00624-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multicellular modeling is increasingly being used to understand biological systems. SimuCell3D is a tool that allows mechanically realistic simulations, using the deformable cell model, to be developed and run.},
  archive      = {J_NATCS},
  author       = {Germano, Domenic P. J. and Osborne, James M.},
  doi          = {10.1038/s43588-024-00624-5},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {312-313},
  shortjournal = {Nat. Comput. Sci},
  title        = {Advancements in multicellular simulations},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accelerating scientific progress with preprints.
<em>NATCS</em>, <em>4</em>(5), 311. (<a
href="https://doi.org/10.1038/s43588-024-00641-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We recognize the importance of preprint posting in communicating research findings and encourage our authors to make use of this service.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00641-4},
  journal      = {Nature Computational Science},
  month        = {5},
  number       = {5},
  pages        = {311},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerating scientific progress with preprints},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). SimuCell3D: Three-dimensional simulation of tissue
mechanics with cell polarization. <em>NATCS</em>, <em>4</em>(4),
299–309. (<a href="https://doi.org/10.1038/s43588-024-00620-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The three-dimensional (3D) organization of cells determines tissue function and integrity, and changes markedly in development and disease. Cell-based simulations have long been used to define the underlying mechanical principles. However, high computational costs have so far limited simulations to either simplified cell geometries or small tissue patches. Here, we present SimuCell3D, an efficient open-source program to simulate large tissues in three dimensions with subcellular resolution, growth, proliferation, extracellular matrix, fluid cavities, nuclei and non-uniform mechanical properties, as found in polarized epithelia. Spheroids, vesicles, sheets, tubes and other tissue geometries can readily be imported from microscopy images and simulated to infer biomechanical parameters. Doing so, we show that 3D cell shapes in layered and pseudostratified epithelia are largely governed by a competition between surface tension and intercellular adhesion. SimuCell3D enables the large-scale in silico study of 3D tissue organization in development and disease at a great level of detail. A fast and versatile three-dimensional cell-based model, called SimuCell3D, is developed for high-resolution simulations of large and complex biological tissues. SimuCell3D natively integrates intra- and extracellular entities, including extracellular matrix, nuclei and polarized cell surfaces.},
  archive      = {J_NATCS},
  author       = {Runser, Steve and Vetter, Roman and Iber, Dagmar},
  doi          = {10.1038/s43588-024-00620-9},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {299-309},
  shortjournal = {Nat. Comput. Sci},
  title        = {SimuCell3D: Three-dimensional simulation of tissue mechanics with cell polarization},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Deciphering cell types by integrating scATAC-seq data with
genome sequences. <em>NATCS</em>, <em>4</em>(4), 285–298. (<a
href="https://doi.org/10.1038/s43588-024-00622-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The single-cell assay for transposase-accessible chromatin using sequencing (scATAC-seq) technology provides insight into gene regulation and epigenetic heterogeneity at single-cell resolution, but cell annotation from scATAC-seq remains challenging due to high dimensionality and extreme sparsity within the data. Existing cell annotation methods mostly focus on the cell peak matrix without fully utilizing the underlying genomic sequence. Here we propose a method, SANGO, for accurate single-cell annotation by integrating genome sequences around the accessibility peaks within scATAC data. The genome sequences of peaks are encoded into low-dimensional embeddings, and then iteratively used to reconstruct the peak statistics of cells through a fully connected network. The learned weights are considered as regulatory modes to represent cells, and utilized to align the query cells and the annotated cells in the reference data through a graph transformer network for cell annotations. SANGO was demonstrated to consistently outperform competing methods on 55 paired scATAC-seq datasets across samples, platforms and tissues. SANGO was also shown to be able to detect unknown tumor cells through attention edge weights learned by the graph transformer. Moreover, from the annotated cells, we found cell-type-specific peaks that provide functional insights/biological signals through expression enrichment analysis, cis-regulatory chromatin interaction analysis and motif enrichment analysis. This study introduces SANGO, a method for accurate single-cell annotation leveraging genomic sequences around accessibility peaks within single-cell ATAC sequencing data. SANGO consistently outperforms existing methods across diverse datasets for identification of cell type and detection of unknown tumor cells. SANGO enables the discovery of cell-type-specific functional insights through expression enrichment, cis-regulatory chromatin interactions and motif enrichment analyses.},
  archive      = {J_NATCS},
  author       = {Zeng, Yuansong and Luo, Mai and Shangguan, Ningyuan and Shi, Peiyu and Feng, Junxi and Xu, Jin and Chen, Ken and Lu, Yutong and Yu, Weijiang and Yang, Yuedong},
  doi          = {10.1038/s43588-024-00622-7},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {285-298},
  shortjournal = {Nat. Comput. Sci},
  title        = {Deciphering cell types by integrating scATAC-seq data with genome sequences},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Strategy evolution on higher-order networks. <em>NATCS</em>,
<em>4</em>(4), 274–284. (<a
href="https://doi.org/10.1038/s43588-024-00621-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cooperation is key to prosperity in human societies. Population structure is well understood as a catalyst for cooperation, where research has focused on pairwise interactions. But cooperative behaviors are not simply dyadic, and they often involve coordinated behavior in larger groups. Here we develop a framework to study the evolution of behavioral strategies in higher-order population structures, which include pairwise and multi-way interactions. We provide an analytical treatment of when cooperation will be favored by higher-order interactions, accounting for arbitrary spatial heterogeneity and nonlinear rewards for cooperation in larger groups. Our results indicate that higher-order interactions can act to promote the evolution of cooperation across a broad range of networks, in public goods games. Higher-order interactions consistently provide an advantage for cooperation when interaction hyper-networks feature multiple conjoined communities. Our analysis provides a systematic account of how higher-order interactions modulate the evolution of prosocial traits. Cooperation is not merely a dyadic phenomenon, it also includes multi-way social interactions. A mathematical framework is developed to study how the structure of higher-order interactions influences cooperative behavior.},
  archive      = {J_NATCS},
  author       = {Sheng, Anzhi and Su, Qi and Wang, Long and Plotkin, Joshua B.},
  doi          = {10.1038/s43588-024-00621-8},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {274-284},
  shortjournal = {Nat. Comput. Sci},
  title        = {Strategy evolution on higher-order networks},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Directional multiobjective optimization of metal complexes
at the billion-system scale. <em>NATCS</em>, <em>4</em>(4), 263–273. (<a
href="https://doi.org/10.1038/s43588-024-00616-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The discovery of transition metal complexes (TMCs) with optimal properties requires large ligand libraries and efficient multiobjective optimization algorithms. Here we provide the tmQMg-L library, containing 30k diverse and synthesizable ligands with robustly assigned charges and metal coordination modes. tmQMg-L enabled the generation of 1.37 million palladium TMCs, which were used to develop and benchmark the Pareto-Lighthouse multiobjective genetic algorithm (PL-MOGA). With fine control over aim and scope, this algorithm maximized both the polarizability and highest occupied molecular orbital–lowest unoccupied molecular orbital gap of the TMCs within selected regions of the Pareto front, without requiring prior knowledge on the objective limits. Instead of genetic operations on small ligand fragments, the PL-MOGA did whole-ligand mutation and crossover operations, which in chemical spaces containing billions of systems, yielded thousands of highly diverse TMCs in an interpretable manner. A method is developed for the directional optimization of multiple properties without prior knowledge on their nature. Using a large ligand dataset, diverse metal complexes are found along the Pareto front of vast chemical spaces.},
  archive      = {J_NATCS},
  author       = {Kneiding, Hannes and Nova, Ainara and Balcells, David},
  doi          = {10.1038/s43588-024-00616-5},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {263-273},
  shortjournal = {Nat. Comput. Sci},
  title        = {Directional multiobjective optimization of metal complexes at the billion-system scale},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Annotating cell types in single-cell ATAC data via the
guidance of the underlying DNA sequences. <em>NATCS</em>, <em>4</em>(4),
261–262. (<a href="https://doi.org/10.1038/s43588-024-00626-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {SANGO efficiently removed batch effects between the query and reference single-cell ATAC signals through the underlying genome sequences, to enable cell type assignment according to the reference data. The method achieved superior performance on diverse datasets and could detect unknown tumor cells, providing valuable functional biological signals.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00626-3},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {261-262},
  shortjournal = {Nat. Comput. Sci},
  title        = {Annotating cell types in single-cell ATAC data via the guidance of the underlying DNA sequences},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Discovering metal complexes in vast chemical spaces.
<em>NATCS</em>, <em>4</em>(4), 259–260. (<a
href="https://doi.org/10.1038/s43588-024-00618-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Approaches are needed to accelerate the discovery of transition metal complexes (TMCs), which is challenging owing to their vast chemical space. A large dataset of diverse ligands is now introduced and leveraged in a multiobjective genetic algorithm that enables the efficient optimization of TMCs in chemical spaces containing billions of them.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00618-3},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {259-260},
  shortjournal = {Nat. Comput. Sci},
  title        = {Discovering metal complexes in vast chemical spaces},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). In search of the most cooperative network. <em>NATCS</em>,
<em>4</em>(4), 257–258. (<a
href="https://doi.org/10.1038/s43588-024-00623-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cooperation is crucial for human prosperity, and population structure fosters it through pairwise interactions and coordinated behavior in larger groups. A recent study explores the evolution of behavioral strategies in higher-order population structures, including pairwise and multi-way interactions to reveal that higher-order interactions promote cooperation across networks, especially when they are formed by conjoined communities.},
  archive      = {J_NATCS},
  author       = {Capraro, Valerio and Perc, Matjaž},
  doi          = {10.1038/s43588-024-00623-6},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {257-258},
  shortjournal = {Nat. Comput. Sci},
  title        = {In search of the most cooperative network},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Leveraging epidemic network models towards wildfire
resilience. <em>NATCS</em>, <em>4</em>(4), 253–256. (<a
href="https://doi.org/10.1038/s43588-024-00619-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Wildfires have increased in frequency and intensity due to climate change and have had severe impacts on the built environment worldwide. Moving forward, models should take inspiration from epidemic network modeling to predict damage to individual buildings and understand the impact of different mitigations on the community vulnerability in a network setting.},
  archive      = {J_NATCS},
  author       = {Mahmoud, Hussam},
  doi          = {10.1038/s43588-024-00619-2},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {253-256},
  shortjournal = {Nat. Comput. Sci},
  title        = {Leveraging epidemic network models towards wildfire resilience},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Harnessing quantum information to advance computing.
<em>NATCS</em>, <em>4</em>(4), 251–252. (<a
href="https://doi.org/10.1038/s43588-024-00628-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We highlight the vibrant discussions on quantum computing and quantum algorithms that took place at the 2024 American Physical Society March Meeting and invite submissions that notably drive the field of quantum information science forward.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00628-1},
  journal      = {Nature Computational Science},
  month        = {4},
  number       = {4},
  pages        = {251-252},
  shortjournal = {Nat. Comput. Sci},
  title        = {Harnessing quantum information to advance computing},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Population-level comparisons of gene regulatory networks
modeled on high-throughput single-cell transcriptomics data.
<em>NATCS</em>, <em>4</em>(3), 237–250. (<a
href="https://doi.org/10.1038/s43588-024-00597-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Single-cell technologies enable high-resolution studies of phenotype-defining molecular mechanisms. However, data sparsity and cellular heterogeneity make modeling biological variability across single-cell samples difficult. Here we present SCORPION, a tool that uses a message-passing algorithm to reconstruct comparable gene regulatory networks from single-cell/nuclei RNA-sequencing data that are suitable for population-level comparisons by leveraging the same baseline priors. Using synthetic data, we found that SCORPION outperformed 12 existing gene regulatory network reconstruction techniques. Using supervised experiments, we show that SCORPION can accurately identify differences in regulatory networks between wild-type and transcription factor-perturbed cells. We demonstrate SCORPION’s scalability to population-level analyses using a single-cell RNA-sequencing atlas containing 200,436 cells from colorectal cancer and adjacent healthy tissues. The differences between tumor regions detected by SCORPION are consistent across multiple cohorts as well as with our understanding of disease progression, and elucidate phenotypic regulators that may impact patient survival. SCORPION is an algorithm to model gene regulatory networks based on single-cell data. The authors show that SCORPION outperforms other methods, accurately detects transcription factor activity and can potentially help with the discovery of disease markers.},
  archive      = {J_NATCS},
  author       = {Osorio, Daniel and Capasso, Anna and Eckhardt, S. Gail and Giri, Uma and Somma, Alexander and Pitts, Todd M. and Lieu, Christopher H. and Messersmith, Wells A. and Bagby, Stacey M. and Singh, Harinder and Das, Jishnu and Sahni, Nidhi and Yi, S. Stephen and Kuijjer, Marieke L.},
  doi          = {10.1038/s43588-024-00597-5},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {237-250},
  shortjournal = {Nat. Comput. Sci},
  title        = {Population-level comparisons of gene regulatory networks modeled on high-throughput single-cell transcriptomics data},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Machine learning-guided engineering of genetically encoded
fluorescent calcium indicators. <em>NATCS</em>, <em>4</em>(3), 224–236.
(<a href="https://doi.org/10.1038/s43588-024-00611-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we used machine learning to engineer genetically encoded fluorescent indicators, protein-based sensors critical for real-time monitoring of biological activity. We used machine learning to predict the outcomes of sensor mutagenesis by analyzing established libraries that link sensor sequences to functions. Using the GCaMP calcium indicator as a scaffold, we developed an ensemble of three regression models trained on experimentally derived GCaMP mutation libraries. The trained ensemble performed an in silico functional screen on 1,423 novel, uncharacterized GCaMP variants. As a result, we identified the ensemble-derived GCaMP (eGCaMP) variants, eGCaMP and eGCaMP+, which achieve both faster kinetics and larger ∆F/F0 responses upon stimulation than previously published fast variants. Furthermore, we identified a combinatorial mutation with extraordinary dynamic range, eGCaMP2+, which outperforms the tested sixth-, seventh- and eighth-generation GCaMPs. These findings demonstrate the value of machine learning as a tool to facilitate the efficient engineering of proteins for desired biophysical characteristics. Andre Berndt and colleagues introduce a machine learning approach to enhance the biophysical characteristics of genetically encoded fluorescent indicators, deriving and testing in vitro new GCaMP mutations that surpass the performance of existing fast GCaMP indicators.},
  archive      = {J_NATCS},
  author       = {Wait, Sarah J. and Expòsit, Marc and Lin, Sophia and Rappleye, Michael and Lee, Justin Daho and Colby, Samuel A. and Torp, Lily and Asencio, Anthony and Smith, Annette and Regnier, Michael and Moussavi-Harami, Farid and Baker, David and Kim, Christina K. and Berndt, Andre},
  doi          = {10.1038/s43588-024-00611-w},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {224-236},
  shortjournal = {Nat. Comput. Sci},
  title        = {Machine learning-guided engineering of genetically encoded fluorescent calcium indicators},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Overcoming the barrier of orbital-free density functional
theory for molecular systems using deep learning. <em>NATCS</em>,
<em>4</em>(3), 210–223. (<a
href="https://doi.org/10.1038/s43588-024-00605-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Orbital-free density functional theory (OFDFT) is a quantum chemistry formulation that has a lower cost scaling than the prevailing Kohn–Sham DFT, which is increasingly desired for contemporary molecular research. However, its accuracy is limited by the kinetic energy density functional, which is notoriously hard to approximate for non-periodic molecular systems. Here we propose M-OFDFT, an OFDFT approach capable of solving molecular systems using a deep learning functional model. We build the essential non-locality into the model, which is made affordable by the concise density representation as expansion coefficients under an atomic basis. With techniques to address unconventional learning challenges therein, M-OFDFT achieves a comparable accuracy to Kohn–Sham DFT on a wide range of molecules untouched by OFDFT before. More attractively, M-OFDFT extrapolates well to molecules much larger than those seen in training, which unleashes the appealing scaling of OFDFT for studying large molecules including proteins, representing an advancement of the accuracy–efficiency trade-off frontier in quantum chemistry. M-OFDFT is a deep learning implementation of orbital-free density functional theory (OFDFT) that achieves DFT-level accuracy on molecular systems with lower cost complexity, and can extrapolate to much larger molecules than those seen during training.},
  archive      = {J_NATCS},
  author       = {Zhang, He and Liu, Siyuan and You, Jiacheng and Liu, Chang and Zheng, Shuxin and Lu, Ziheng and Wang, Tong and Zheng, Nanning and Shao, Bin},
  doi          = {10.1038/s43588-024-00605-8},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {210-223},
  shortjournal = {Nat. Comput. Sci},
  title        = {Overcoming the barrier of orbital-free density functional theory for molecular systems using deep learning},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Electron density-based GPT for optimization and suggestion
of host–guest binders. <em>NATCS</em>, <em>4</em>(3), 200–209. (<a
href="https://doi.org/10.1038/s43588-024-00602-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we present a machine learning model trained on electron density for the production of host–guest binders. These are read out as simplified molecular-input line-entry system (SMILES) format with &amp;gt;98% accuracy, enabling a complete characterization of the molecules in two dimensions. Our model generates three-dimensional representations of the electron density and electrostatic potentials of host–guest systems using a variational autoencoder, and then utilizes these representations to optimize the generation of guests via gradient descent. Finally the guests are converted to SMILES using a transformer. The successful practical application of our model to established molecular host systems, cucurbit[n]uril and metal–organic cages, resulted in the discovery of 9 previously validated guests for CB[6] and 7 unreported guests (with association constant Ka ranging from 13.5 M−1 to 5,470 M−1) and the discovery of 4 unreported guests for [Pd214]4+ (with Ka ranging from 44 M−1 to 529 M−1). An optimization algorithm is used to discover guest molecules based on knowing only the structure of the host. The molecules are represented as 3D volumes, optimized to improve host–guest interaction and converted into SMILES using a transformer model.},
  archive      = {J_NATCS},
  author       = {Parrilla-Gutiérrez, Juan M. and Granda, Jarosław M. and Ayme, Jean-François and Bajczyk, Michał D. and Wilbraham, Liam and Cronin, Leroy},
  doi          = {10.1038/s43588-024-00602-x},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {200-209},
  shortjournal = {Nat. Comput. Sci},
  title        = {Electron density-based GPT for optimization and suggestion of host–guest binders},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Digital twins in city planning. <em>NATCS</em>,
<em>4</em>(3), 192–199. (<a
href="https://doi.org/10.1038/s43588-024-00606-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here, I provide a perspective on digital twins of cities that cover a wide array of different types, ranging from aggregate economic and behavioral processes to more disaggregate agent-based, cellular and micro-simulations. A key element in these applications is the way that we as scientists, policymakers and planners interact with real cities with respect to their understanding, prediction and design. I note a range of spatial models, from analytical simulations of local neighborhoods to large-scale systems of cities and city systems, and briefly describe computational challenges that geospatial applications in cities pose. Although digital twins first originated as models of physical systems, they are rapidly being applied to social systems, such as cities. This Perspective discusses the development and use of digital twins for urban planning.},
  archive      = {J_NATCS},
  author       = {Batty, Michael},
  doi          = {10.1038/s43588-024-00606-7},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {192-199},
  shortjournal = {Nat. Comput. Sci},
  title        = {Digital twins in city planning},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Digital twins in medicine. <em>NATCS</em>, <em>4</em>(3),
184–191. (<a href="https://doi.org/10.1038/s43588-024-00607-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Medical digital twins, which are potentially vital for personalized medicine, have become a recent focus in medical research. Here we present an overview of the state of the art in medical digital twin development, especially in oncology and cardiology, where it is most advanced. We discuss major challenges, such as data integration and privacy, and provide an outlook on future advancements. Emphasizing the importance of this technology in healthcare, we highlight the potential for substantial improvements in patient-specific treatments and diagnostics. The digital twin concept, while initially formulated and developed in industry and engineering, has compelling potential applications in medicine. There are, however, major challenges that need to be overcome to fully embrace digital twin technology in the medical context.},
  archive      = {J_NATCS},
  author       = {Laubenbacher, R. and Mehrad, B. and Shmulevich, I. and Trayanova, N.},
  doi          = {10.1038/s43588-024-00607-6},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {184-191},
  shortjournal = {Nat. Comput. Sci},
  title        = {Digital twins in medicine},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Digital twins in mechanical and aerospace engineering.
<em>NATCS</em>, <em>4</em>(3), 178–183. (<a
href="https://doi.org/10.1038/s43588-024-00613-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Digital twins bring value to mechanical and aerospace systems by speeding up development, reducing risk, predicting issues and reducing sustainment costs. Realizing these benefits at scale requires a structured and intentional approach to digital twin conception, design, development, operation and sustainment. To bring maximal value, a digital twin does not need to be an exquisite virtual replica but instead must be envisioned to be fit for purpose, where the determination of fitness depends on the capability needs and the cost–benefit trade-offs. While there is a clear opportunity for digital twins to bring value in mechanical and aerospace engineering, they must be considered as an asset in their own right so that their full potential can be realized.},
  archive      = {J_NATCS},
  author       = {Ferrari, Alberto and Willcox, Karen},
  doi          = {10.1038/s43588-024-00613-8},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {178-183},
  shortjournal = {Nat. Comput. Sci},
  title        = {Digital twins in mechanical and aerospace engineering},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Advancements and challenges of digital twins in industry.
<em>NATCS</em>, <em>4</em>(3), 169–177. (<a
href="https://doi.org/10.1038/s43588-024-00603-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Digital twins, which are considered an effective approach to realize the fusion between virtual and physical spaces, have attracted a substantial amount of attention in the past decade. With their rapid development in recent years, digital twins have been applied in various fields, particularly in industry. However, there are still some gaps to be filled and some limitations to be addressed. Here we provide a brief overview of digital twin advancements in industry and highlight the main pitfalls to avoid and challenges to overcome, to improve the maturity of digital twins and facilitate large-scale industrial applications in the future. The application of digital twins in industry has become increasingly common, but not without important challenges to be addressed by the research community.},
  archive      = {J_NATCS},
  author       = {Tao, Fei and Zhang, He and Zhang, Chenyuan},
  doi          = {10.1038/s43588-024-00603-w},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {169-177},
  shortjournal = {Nat. Comput. Sci},
  title        = {Advancements and challenges of digital twins in industry},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Enabling comparative gene regulatory network analysis on
single-cell data with SCORPION. <em>NATCS</em>, <em>4</em>(3), 167–168.
(<a href="https://doi.org/10.1038/s43588-024-00615-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present SCORPION, a computational tool to model gene regulatory networks based on single-cell transcriptomic data and prior knowledge of gene regulation. SCORPION networks can be modeled for specific cell types in individual samples, and are therefore suitable for conducting comparisons between experimental groups.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00615-6},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {167-168},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enabling comparative gene regulatory network analysis on single-cell data with SCORPION},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accelerating protein sensor optimization with machine
learning. <em>NATCS</em>, <em>4</em>(3), 165–166. (<a
href="https://doi.org/10.1038/s43588-024-00612-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study introduces a machine learning approach to investigate the effects of mutations on protein sensors commonly employed in fluorescence microscopy, thus enabling the discovery of high-performance sensors.},
  archive      = {J_NATCS},
  author       = {Dana, Hod},
  doi          = {10.1038/s43588-024-00612-9},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {165-166},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerating protein sensor optimization with machine learning},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pushing the limits of OFDFT with neural networks.
<em>NATCS</em>, <em>4</em>(3), 163–164. (<a
href="https://doi.org/10.1038/s43588-024-00610-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A neural network-based method for advancing orbital-free density functional theory (OFDFT) is developed, which reaches DFT accuracy and maintains lower cost complexity.},
  archive      = {J_NATCS},
  author       = {Hauser, Andreas W.},
  doi          = {10.1038/s43588-024-00610-x},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {163-164},
  shortjournal = {Nat. Comput. Sci},
  title        = {Pushing the limits of OFDFT with neural networks},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Enhancing discovery of host–guest binders. <em>NATCS</em>,
<em>4</em>(3), 161–162. (<a
href="https://doi.org/10.1038/s43588-023-00584-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Determining what guest can effectively bind in a host, or the reverse, is a central challenge in chemistry. To address this, an electron-density-based transformer method of generating and optimizing host–guest binders is proposed, applied to two different host systems and validated by experiment.},
  archive      = {J_NATCS},
  author       = {Avci, Gokay and Jelfs, Kim E.},
  doi          = {10.1038/s43588-023-00584-2},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {161-162},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enhancing discovery of host–guest binders},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Materials informatics heralds a collaborative future.
<em>NATCS</em>, <em>4</em>(3), 158–160. (<a
href="https://doi.org/10.1038/s43588-024-00614-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dr Zhimei Sun – professor of Materials Science and Engineering at Beihang University – talks to Nature Computational Science about her career trajectory, her research on computational materials science and materials informatics, as well as her advice to young women scientists in these fields.},
  archive      = {J_NATCS},
  author       = {Pan, Jie},
  doi          = {10.1038/s43588-024-00614-7},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {158-160},
  shortjournal = {Nat. Comput. Sci},
  title        = {Materials informatics heralds a collaborative future},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Digital twins of earth and the computing challenge of human
interaction. <em>NATCS</em>, <em>4</em>(3), 154–157. (<a
href="https://doi.org/10.1038/s43588-024-00599-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Digital twins of Earth have the capability to offer versatile access to detailed information on our changing world, helping societies to adapt to climate change and to manage the effects of local impacts, globally. Nevertheless, human interaction with digital twins requires advances in computational science, particularly where complex geophysical data is turned into information to support decision making.},
  archive      = {J_NATCS},
  author       = {Bauer, Peter and Hoefler, Torsten and Stevens, Bjorn and Hazeleger, Wilco},
  doi          = {10.1038/s43588-024-00599-3},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {154-157},
  shortjournal = {Nat. Comput. Sci},
  title        = {Digital twins of earth and the computing challenge of human interaction},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Recent achievements and conceptual challenges for urban
digital twins. <em>NATCS</em>, <em>4</em>(3), 150–153. (<a
href="https://doi.org/10.1038/s43588-024-00604-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Urban digital twins hold immense promise as live computational models of cities, synthesizing diverse knowledge, streaming data, and supporting decisions towards more inclusive planning and policy. The size, heterogeneity, and open-ended character of cities, however, pose many difficult questions, at the frontiers of what is currently possible in computational science. Overcoming these challenges provides pathways for fundamental progress in the field and a proving ground for its economic value and social relevance.},
  archive      = {J_NATCS},
  author       = {Bettencourt, Luís M. A.},
  doi          = {10.1038/s43588-024-00604-9},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {150-153},
  shortjournal = {Nat. Comput. Sci},
  title        = {Recent achievements and conceptual challenges for urban digital twins},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The role of computational science in digital twins.
<em>NATCS</em>, <em>4</em>(3), 147–149. (<a
href="https://doi.org/10.1038/s43588-024-00609-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Digital twins hold immense promise in accelerating scientific discovery, but the publicity currently outweighs the evidence base of success. We summarize key research opportunities in the computational sciences to enable digital twin technologies, as identified by a recent National Academies of Sciences, Engineering, and Medicine consensus study report.},
  archive      = {J_NATCS},
  author       = {Willcox, Karen and Segundo, Brittany},
  doi          = {10.1038/s43588-024-00609-4},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {147-149},
  shortjournal = {Nat. Comput. Sci},
  title        = {The role of computational science in digital twins},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The increasing potential and challenges of digital twins.
<em>NATCS</em>, <em>4</em>(3), 145–146. (<a
href="https://doi.org/10.1038/s43588-024-00617-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This issue of Nature Computational Science includes a Focus that highlights recent advancements, challenges, and opportunities in the development and use of digital twins across different domains.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00617-4},
  journal      = {Nature Computational Science},
  month        = {3},
  number       = {3},
  pages        = {145-146},
  shortjournal = {Nat. Comput. Sci},
  title        = {The increasing potential and challenges of digital twins},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Extracting, filtering and simulating cellular barcodes using
CellBarcode tools. <em>NATCS</em>, <em>4</em>(2), 128–143. (<a
href="https://doi.org/10.1038/s43588-024-00595-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying true DNA cellular barcodes among polymerase chain reaction and sequencing errors is challenging. Current tools are restricted in the diversity of barcode types supported or the analysis strategies implemented. As such, there is a need for more versatile and efficient tools for barcode extraction, as well as for tools to investigate which factors impact barcode detection and which filtering strategies to best apply. Here we introduce the package CellBarcode and its barcode simulation kit, CellBarcodeSim, that allows efficient and versatile barcode extraction and filtering for a range of barcode types from bulk or single-cell sequencing data using a variety of filtering strategies. Using the barcode simulation kit and biological data, we explore the technical and biological factors influencing barcode identification and provide a decision tree on how to optimize barcode identification for different barcode settings. We believe that CellBarcode and CellBarcodeSim have the capability to enhance the reproducibility and interpretation of barcode results across studies. The authors introduce two cellular barcoding tools: CellBarcode, for extracting and filtering diverse DNA barcodes from bulk and single-cell sequencing data; and CellBarcodeSim, for simulating barcoding experiments, thus enabling the investigation of the impact of biological and technical factors on barcode detection.},
  archive      = {J_NATCS},
  author       = {Sun, Wenjie and Perkins, Meghan and Huyghe, Mathilde and Faraldo, Marisa M. and Fre, Silvia and Perié, Leïla and Lyne, Anne-Marie},
  doi          = {10.1038/s43588-024-00595-7},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {128-143},
  shortjournal = {Nat. Comput. Sci},
  title        = {Extracting, filtering and simulating cellular barcodes using CellBarcode tools},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). An error correction strategy for image reconstruction by DNA
sequencing microscopy. <em>NATCS</em>, <em>4</em>(2), 119–127. (<a
href="https://doi.org/10.1038/s43588-023-00589-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By pairing adjacent molecules in situ and then mapping these pairs, DNA microscopy could substantially reduce the workload in spatial omics methods by directly inferring geometry from sequencing data alone. However, experimental artifacts can lead to errors in the adjacency data, which distort the spatial reconstruction. Here we describe a method to correct two such errors: spurious crosslinks formed between any two nodes, and fused nodes that are formed out of multiple molecules. We build on the principle that spatially close molecules should be connected and show that these errors violate this principle, allowing for their detection and correction. Our method corrects errors in simulated data, even in the presence of up to 20% errors, and proves to be more efficient at removing errors from experimental data than a read count filter. Integrating this method in DNA microscopy will substantially improve the accuracy of spatial reconstructions with lower data loss. DNA microscopy reconstructs the spatial organization of a sample from a neighborhood graph. In this work, MinIPath efficiently corrects errors from these graphs that distort the reconstruction, both in simulated and experimental data.},
  archive      = {J_NATCS},
  author       = {Kloosterman, Alexander and Baars, Igor and Högberg, Björn},
  doi          = {10.1038/s43588-023-00589-x},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {119-127},
  shortjournal = {Nat. Comput. Sci},
  title        = {An error correction strategy for image reconstruction by DNA sequencing microscopy},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Automated discovery of algorithms from data. <em>NATCS</em>,
<em>4</em>(2), 110–118. (<a
href="https://doi.org/10.1038/s43588-024-00593-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To automate the discovery of new scientific and engineering principles, artificial intelligence must distill explicit rules from experimental data. This has proven difficult because existing methods typically search through the enormous space of possible functions. Here we introduce deep distilling, a machine learning method that does not perform searches but instead learns from data using symbolic essence neural networks and then losslessly condenses the network parameters into a concise algorithm written in computer code. This distilled code, which can contain loops and nested logic, is equivalent to the neural network but is human-comprehensible and orders-of-magnitude more compact. On arithmetic, vision and optimization tasks, the distilled code is capable of out-of-distribution systematic generalization to solve cases orders-of-magnitude larger and more complex than the training data. The distilled algorithms can sometimes outperform human-designed algorithms, demonstrating that deep distilling is able to discover generalizable principles complementary to human expertise. Automated algorithm discovery has been difficult for artificial intelligence given the immense search space of possible functions. Here explainable neural networks are used to discover algorithms that outperform those designed by humans.},
  archive      = {J_NATCS},
  author       = {Blazek, Paul J. and Venkatesh, Kesavan and Lin, Milo M.},
  doi          = {10.1038/s43588-024-00593-9},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {110-118},
  shortjournal = {Nat. Comput. Sci},
  title        = {Automated discovery of algorithms from data},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Indexing and real-time user-friendly queries in
terabyte-sized complex genomic datasets with kmindex and ORA.
<em>NATCS</em>, <em>4</em>(2), 104–109. (<a
href="https://doi.org/10.1038/s43588-024-00596-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Public sequencing databases contain vast amounts of biological information, yet they are largely underutilized as it is challenging to efficiently search them for any sequence(s) of interest. We present kmindex, an approach that can index thousands of metagenomes and perform sequence searches in a fraction of a second. The index construction is an order of magnitude faster than previous methods, while search times are two orders of magnitude faster. With negligible false positive rates below 0.01%, kmindex outperforms the precision of existing approaches by four orders of magnitude. Here we demonstrate the scalability of kmindex by successfully indexing 1,393 marine seawater metagenome samples from the Tara Oceans project. Additionally, we introduce the publicly accessible web server Ocean Read Atlas, which enables real-time queries on the Tara Oceans dataset. kmindex is a tool able to index thousands of environmental metagenomes and perform sequence searches in a fraction of a second, thus enabling real-time queries on complex genomic datasets.},
  archive      = {J_NATCS},
  author       = {Lemane, Téo and Lezzoche, Nolan and Lecubin, Julien and Pelletier, Eric and Lescot, Magali and Chikhi, Rayan and Peterlongo, Pierre},
  doi          = {10.1038/s43588-024-00596-6},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {104-109},
  shortjournal = {Nat. Comput. Sci},
  title        = {Indexing and real-time user-friendly queries in terabyte-sized complex genomic datasets with kmindex and ORA},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Characterizing emerging companies in computational drug
development. <em>NATCS</em>, <em>4</em>(2), 96–103. (<a
href="https://doi.org/10.1038/s43588-024-00594-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computation promises to accelerate, de-risk and optimize drug research and development. An increasing number of companies have entered this space, specializing in the design of new algorithms, computing on proprietary data, and/or development of hardware to improve distinct drug pipeline stages. The large number of such companies and their unique strategies and deals have created a highly complex and competitive industry. We comprehensively analyze the companies in this space to highlight trends and opportunities, identifying highly occupied areas of risk and currently underrepresented niches of high value. As computation is increasingly integrated into drug research and development, this Perspective analyzes company business models, funding and deals to provide unique insights into risks and opportunities in this quickly maturing industry, which aims to expedite the creation of life-saving therapeutics.},
  archive      = {J_NATCS},
  author       = {Markey, Chloe and Croset, Samuel and Woolley, Olivia Ruth and Buldun, Can Martin and Koch, Christian and Koller, Daniel and Reker, Daniel},
  doi          = {10.1038/s43588-024-00594-8},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {96-103},
  shortjournal = {Nat. Comput. Sci},
  title        = {Characterizing emerging companies in computational drug development},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Refining the lens of DNA microscopy. <em>NATCS</em>,
<em>4</em>(2), 94–95. (<a
href="https://doi.org/10.1038/s43588-024-00601-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A method for correcting errors in the spatial-genetic reconstruction of DNA microscopy is proposed, leading to more accurate results and potential to resolve new biology.},
  archive      = {J_NATCS},
  author       = {Weinstein, Joshua},
  doi          = {10.1038/s43588-024-00601-y},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {94-95},
  shortjournal = {Nat. Comput. Sci},
  title        = {Refining the lens of DNA microscopy},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Distilling data into code. <em>NATCS</em>, <em>4</em>(2),
92–93. (<a href="https://doi.org/10.1038/s43588-024-00598-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {One of the greatest limitations of deep neural networks is the difficulty of interpreting what they learn from the data. Deep distilling addresses this problem by extracting human-comprehensible and executable code from observations.},
  archive      = {J_NATCS},
  author       = {Bakarji, Joseph},
  doi          = {10.1038/s43588-024-00598-4},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {92-93},
  shortjournal = {Nat. Comput. Sci},
  title        = {Distilling data into code},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Accelerating discovery in organic redox flow batteries.
<em>NATCS</em>, <em>4</em>(2), 89–91. (<a
href="https://doi.org/10.1038/s43588-024-00600-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We highlight the challenges and opportunities in organic redox flow battery research, underscoring the need for collaborative research efforts. The synergy between computation and experimentation holds the potential to expedite progress in this field and can have far-reaching impacts beyond energy storage applications.},
  archive      = {J_NATCS},
  author       = {Cao, Yang and Aspuru-Guzik, Alán},
  doi          = {10.1038/s43588-024-00600-z},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {89-91},
  shortjournal = {Nat. Comput. Sci},
  title        = {Accelerating discovery in organic redox flow batteries},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A look into our non-primary formats. <em>NATCS</em>,
<em>4</em>(2), 87–88. (<a
href="https://doi.org/10.1038/s43588-024-00608-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We discuss the different formats for publishing non-primary research that are currently supported by Nature Computational Science.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00608-5},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {87-88},
  shortjournal = {Nat. Comput. Sci},
  title        = {A look into our non-primary formats},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Constructing custom thermodynamics using deep learning.
<em>NATCS</em>, <em>4</em>(1), 66–85. (<a
href="https://doi.org/10.1038/s43588-023-00581-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {One of the most exciting applications of artificial intelligence is automated scientific discovery based on previously amassed data, coupled with restrictions provided by known physical principles, including symmetries and conservation laws. Such automated hypothesis creation and verification can assist scientists in studying complex phenomena, where traditional physical intuition may fail. Here we develop a platform based on a generalized Onsager principle to learn macroscopic dynamical descriptions of arbitrary stochastic dissipative systems directly from observations of their microscopic trajectories. Our method simultaneously constructs reduced thermodynamic coordinates and interprets the dynamics on these coordinates. We demonstrate its effectiveness by studying theoretically and validating experimentally the stretching of long polymer chains in an externally applied field. Specifically, we learn three interpretable thermodynamic coordinates and build a dynamical landscape of polymer stretching, including the identification of stable and transition states and the control of the stretching rate. Our general methodology can be used to address a wide range of scientific and technological applications. The authors develop a general method that combines machine learning and physics to construct macroscopic dynamics directly from microscopic observations, leading to an intuitive understanding of polymer stretching in elongational flow.},
  archive      = {J_NATCS},
  author       = {Chen, Xiaoli and Soh, Beatrice W. and Ooi, Zi-En and Vissol-Gaudin, Eleonore and Yu, Haijun and Novoselov, Kostya S. and Hippalgaonkar, Kedar and Li, Qianxiao},
  doi          = {10.1038/s43588-023-00581-5},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {66-85},
  shortjournal = {Nat. Comput. Sci},
  title        = {Constructing custom thermodynamics using deep learning},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Quantifying spatial under-reporting disparities in resident
crowdsourcing. <em>NATCS</em>, <em>4</em>(1), 57–65. (<a
href="https://doi.org/10.1038/s43588-023-00572-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern city governance relies heavily on crowdsourcing to identify problems such as downed trees and power lines. A major concern is that residents do not report problems at the same rates, with heterogeneous reporting delays directly translating to downstream disparities in how quickly incidents can be addressed. Here we develop a method to identify reporting delays without using external ground-truth data. Our insight is that the rates at which duplicate reports are made about the same incident can be leveraged to disambiguate whether an incident has occurred by investigating its reporting rate once it has occurred. We apply our method to over 100,000 resident reports made in New York City and to over 900,000 reports made in Chicago, finding that there are substantial spatial and socioeconomic disparities in how quickly incidents are reported. We further validate our methods using external data and demonstrate how estimating reporting delays leads to practical insights and interventions for a more equitable, efficient government service. Zhi Liu et al. develop a method to measure disparities in reporting delays in urban crowdsourcing systems, uncovering socioeconomic disparities and providing actionable insights for interventions that enhance the efficiency and equity of city services.},
  archive      = {J_NATCS},
  author       = {Liu, Zhi and Bhandaram, Uma and Garg, Nikhil},
  doi          = {10.1038/s43588-023-00572-6},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {57-65},
  shortjournal = {Nat. Comput. Sci},
  title        = {Quantifying spatial under-reporting disparities in resident crowdsourcing},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Using sequences of life-events to predict human lives.
<em>NATCS</em>, <em>4</em>(1), 43–56. (<a
href="https://doi.org/10.1038/s43588-023-00573-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Here we represent human lives in a way that shares structural similarity to language, and we exploit this similarity to adapt natural language processing techniques to examine the evolution and predictability of human lives based on detailed event sequences. We do this by drawing on a comprehensive registry dataset, which is available for Denmark across several years, and that includes information about life-events related to health, education, occupation, income, address and working hours, recorded with day-to-day resolution. We create embeddings of life-events in a single vector space, showing that this embedding space is robust and highly structured. Our models allow us to predict diverse outcomes ranging from early mortality to personality nuances, outperforming state-of-the-art models by a wide margin. Using methods for interpreting deep learning models, we probe the algorithm to understand the factors that enable our predictions. Our framework allows researchers to discover potential mechanisms that impact life outcomes as well as the associated possibilities for personalized interventions. Using registry data from Denmark, Lehmann et al. create individual-level trajectories of events related to health, education, occupation, income and address, and also apply transformer models to build rich embeddings of life-events and to predict outcomes ranging from time of death to personality.},
  archive      = {J_NATCS},
  author       = {Savcisens, Germans and Eliassi-Rad, Tina and Hansen, Lars Kai and Mortensen, Laust Hvas and Lilleholt, Lau and Rogers, Anna and Zettler, Ingo and Lehmann, Sune},
  doi          = {10.1038/s43588-023-00573-5},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {43-56},
  shortjournal = {Nat. Comput. Sci},
  title        = {Using sequences of life-events to predict human lives},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Unbiased organism-agnostic and highly sensitive signal
peptide predictor with deep protein language model. <em>NATCS</em>,
<em>4</em>(1), 29–42. (<a
href="https://doi.org/10.1038/s43588-023-00576-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Signal peptides (SPs) are essential to target and transfer transmembrane and secreted proteins to the correct positions. Many existing computational tools for predicting SPs disregard the extreme data imbalance problem and rely on additional group information of proteins. Here we introduce Unbiased Organism-agnostic Signal Peptide Network (USPNet), an SP classification and cleavage-site prediction deep learning method. Extensive experimental results show that USPNet substantially outperforms previous methods on classification performance by 10%. An SP-discovering pipeline with USPNet is designed to explore unprecedented SPs from metagenomic data. It reveals 347 SP candidates, with the lowest sequence identity between our candidates and the closest SP in the training dataset at only 13%. In addition, the template modeling scores between candidates and SPs in the training set are mostly above 0.8. The results showcase that USPNet has learnt the SP structure with raw amino acid sequences and the large protein language model, thereby enabling the discovery of unknown SPs. Signal peptides (SPs) are vital for protein–transmembrane communication. In this work, the authors introduce USPNet, a deep learning method based on a protein language model for SP prediction that shows both high sensitivity and efficiency, thereby contributing to the identification of novel SPs.},
  archive      = {J_NATCS},
  author       = {Shen, Junbo and Yu, Qinze and Chen, Shenyang and Tan, Qingxiong and Li, Jingchen and Li, Yu},
  doi          = {10.1038/s43588-023-00576-2},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {29-42},
  shortjournal = {Nat. Comput. Sci},
  title        = {Unbiased organism-agnostic and highly sensitive signal peptide predictor with deep protein language model},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Dendritic excitability controls overdispersion.
<em>NATCS</em>, <em>4</em>(1), 19–28. (<a
href="https://doi.org/10.1038/s43588-023-00580-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The brain is an intricate assembly of intercommunicating neurons whose input–output function is only partially understood. The role of active dendrites in shaping spiking responses, in particular, is unclear. Although existing models account for active dendrites and spiking responses, they are too complex to analyze analytically and demand long stochastic simulations. Here we combine cable and renewal theory to describe how input fluctuations shape the response of neuronal ensembles with active dendrites. We found that dendritic input readily and potently controls interspike interval dispersion. This phenomenon can be understood by considering that neurons display three fundamental operating regimes: one mean-driven regime and two fluctuation-driven regimes. We show that these results are expected to appear for a wide range of dendritic properties and verify predictions of the model in experimental data. These findings have implications for the role of interspike interval dispersion in learning and for theories of attractor states. A mathematical framework that allows computing the input–output function of neurons with active dendrites reveals how dendrites readily and potently control the response variability, a result that is experimentally confirmed.},
  archive      = {J_NATCS},
  author       = {Friedenberger, Zachary and Naud, Richard},
  doi          = {10.1038/s43588-023-00580-6},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {19-28},
  shortjournal = {Nat. Comput. Sci},
  title        = {Dendritic excitability controls overdispersion},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Language models for quantum simulation. <em>NATCS</em>,
<em>4</em>(1), 11–18. (<a
href="https://doi.org/10.1038/s43588-023-00578-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A key challenge in the effort to simulate today’s quantum computing devices is the ability to learn and encode the complex correlations that occur between qubits. Emerging technologies based on language models adopted from machine learning have shown unique abilities to learn quantum states. We highlight the contributions that language models are making in the effort to build quantum computers and discuss their future role in the race to quantum advantage. Language models offer promises in encoding quantum correlations and learning complex quantum states. This Perspective discusses the advantages of employing language models in quantum simulation, explores recent model developments, and offers insights into opportunities for realizing scalable and accurate quantum simulation.},
  archive      = {J_NATCS},
  author       = {Melko, Roger G. and Carrasquilla, Juan},
  doi          = {10.1038/s43588-023-00578-0},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {11-18},
  shortjournal = {Nat. Comput. Sci},
  title        = {Language models for quantum simulation},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Learning physical laws from observations of complex
dynamics. <em>NATCS</em>, <em>4</em>(1), 9–10. (<a
href="https://doi.org/10.1038/s43588-023-00590-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The laws of physics, formulated in a compact form, are elusive for complex dynamic phenomena. However, it is now shown that, using artificial intelligence constrained by the physical Onsager principle, a custom thermodynamic description of a complex system can be constructed from the observation of its dynamical behavior.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00590-4},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {9-10},
  shortjournal = {Nat. Comput. Sci},
  title        = {Learning physical laws from observations of complex dynamics},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A transformer method that predicts human lives from
sequences of life events. <em>NATCS</em>, <em>4</em>(1), 7–8. (<a
href="https://doi.org/10.1038/s43588-023-00586-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transformer methods are revolutionizing how computers process human language. Exploiting the structural similarity between human lives, seen as sequences of events, and natural-language sentences, a transformer method — dubbed life2vec — has been used to create rich vector representations of human lives, from which accurate predictions can be made.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-023-00586-0},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {7-8},
  shortjournal = {Nat. Comput. Sci},
  title        = {A transformer method that predicts human lives from sequences of life events},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Disentangling truth from bias in naturally occurring data.
<em>NATCS</em>, <em>4</em>(1), 5–6. (<a
href="https://doi.org/10.1038/s43588-023-00587-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A technique that leverages duplicate records in crowdsourcing data could help to mitigate the effects of biases in research and services that are dependent on government records.},
  archive      = {J_NATCS},
  author       = {O’Brien, Daniel T.},
  doi          = {10.1038/s43588-023-00587-z},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {5-6},
  shortjournal = {Nat. Comput. Sci},
  title        = {Disentangling truth from bias in naturally occurring data},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024c). Mining the chemical space for new antibiotics.
<em>NATCS</em>, <em>4</em>(1), 4. (<a
href="https://doi.org/10.1038/s43588-024-00591-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Chirigati, Fernando},
  doi          = {10.1038/s43588-024-00591-x},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {4},
  shortjournal = {Nat. Comput. Sci},
  title        = {Mining the chemical space for new antibiotics},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Using proprietary language models in academic research
requires explicit justification. <em>NATCS</em>, <em>4</em>(1), 2–3. (<a
href="https://doi.org/10.1038/s43588-023-00585-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Palmer, Alexis and Smith, Noah A. and Spirling, Arthur},
  doi          = {10.1038/s43588-023-00585-1},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {2-3},
  shortjournal = {Nat. Comput. Sci},
  title        = {Using proprietary language models in academic research requires explicit justification},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Cover runners-up of 2023. <em>NATCS</em>, <em>4</em>(1), 1.
(<a href="https://doi.org/10.1038/s43588-024-00592-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We continue our yearly tradition of highlighting our favorite cover suggestions that did not make it as covers at Nature Computational Science.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00592-w},
  journal      = {Nature Computational Science},
  month        = {1},
  number       = {1},
  pages        = {1},
  shortjournal = {Nat. Comput. Sci},
  title        = {Cover runners-up of 2023},
  volume       = {4},
  year         = {2024},
}
</textarea>
</details></li>
</ul>

</body>
</html>
