<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>JCC_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="jcc---178">JCC - 178</h2>
<ul>
<li><details>
<summary>
(2022). Development of QM/MM (ABEEM polarizable force field) method
to simulate the excision reaction mechanism of damaged cytosine.
<em>JCC</em>, <em>43</em>(32), 2139–2153. (<a
href="https://doi.org/10.1002/jcc.27008">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {DNA damages are regarded as having harmful effects on cell. The base excision repair mechanism combats these effects by removing damaged bases. The deglycosylation mechanism of excising damaged bases by DNA glycosylase and the state of the leaving base have been controversial. The enzymatic reaction of DNA glycosylase to remove the damaged bases involves not only the formation and breaking of chemical bonds, but also complex polarization effect and charge transfer, which cannot be accurately simulated by the QM/MM method combined with the fixed charge force field. This work has developed the ABEEM fluctuating polarizable force field combining with the QM method, that is (QM/MM[ABEEM]), to accurately simulate the proton transfer, charge transfer and the charge distribution. The piecewise function is used as the valence-state electronegativity in the QM/MM (ABEEM) to realize the accurate fitting of the charge distribution in reaction. And the charge transfer is accurately simulated by the local charge conservation conditions. Four deglycosylation mechanisms including the monofunctional and difunctional mechanisms of four neutral and protonated cytosine derivatives are explored. It is confirmed that the monofunctional mechanism of Asp-activated nucleophile water is a better deglycosylation mechanism and the base is protonated before the reaction occurs. Protonization of the base reduced the activation energy by 10.00–17.00 kcal/mol. Asp provides the necessary charge for the reaction, and DNA glycosylase preferentially cleaves ɛC. This work provides a theoretical basis for the research of excising damaged bases by DNA glycosylase.},
  archive      = {J_JCC},
  author       = {Cui Liu and Yang Ren and Xiao-qin Gao and Xue Du and Zhong-zhi Yang},
  doi          = {10.1002/jcc.27008},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2139-2153},
  shortjournal = {J. Comput. Chem.},
  title        = {Development of QM/MM (ABEEM polarizable force field) method to simulate the excision reaction mechanism of damaged cytosine},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Benchmark of density functional theory methods for the study
of organic polysulfides. <em>JCC</em>, <em>43</em>(32), 2131–2138. (<a
href="https://doi.org/10.1002/jcc.27007">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Elemental sulfur is often used in organic synthesis as its low cost and high abundance make it a highly desirable source of sulfur atoms. However, sulfur&#39;s unpredictable catenation behavior poses challenges to its widespread usage due to difficulties in designing new reactions that can account for its multifaceted reactivity. In order to accurately model sulfur&#39;s mechanisms using computational approaches, it is necessary to identify density functional theory (DFT) methods that are accurate on these systems. This study benchmarks 12 well-known DFT functionals that include local, non-local, and hybrid methods against DLPNO-CCSD(T)/aug-cc-pV(Q+d)Z//MP2/aug-cc-pV(T+d)Z/SMD(MeCN) for the accurate treatment of organic polysulfides, taking cyanide as a nucleophile. Our benchmarking results indicate that the M06-2X and B3LYP-D3(BJ) density functionals are the most accurate for calculating reaction energies, while local functionals performed the worst. For activation energies, MN15, MN15-L, M06-2X, and ωB97X-D are the most accurate. Our analysis of structural parameters shows that all functionals perform well for ground state optimizations except B97D3, while MN15-L and M06-2X performed best for transition structure optimizations. Overall, the four hybrid functionals MN15, M06-2X, ωB97X-D, and B3LYP-D3(BJ) appear adequate for studying the reaction mechanisms of polysulfides.},
  archive      = {J_JCC},
  author       = {Jyoti Sharma and Pier Alexandre Champagne},
  doi          = {10.1002/jcc.27007},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2131-2138},
  shortjournal = {J. Comput. Chem.},
  title        = {Benchmark of density functional theory methods for the study of organic polysulfides},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Toward more potent imidazopyridine inhibitors of candida
albicans bdf1: Modeling the role of structural waters in selective
ligand binding. <em>JCC</em>, <em>43</em>(32), 2121–2130. (<a
href="https://doi.org/10.1002/jcc.26997">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Novel agents to treat invasive fungal infections are urgently needed because the small number of established targets in pathogenic fungi makes the existing drug repertoire particularly vulnerable to the emergence of resistant strains. Recently, we reported thatCandida albicansBdf1, a bromodomain and extra-terminal domain (BET) bromodomain with paired acetyl-lysine (AcK) binding sites (BD1 and BD2) is essential for fungal cell growth and that an imidazopyridine (1) binds to BD2 with selectivity versus both BD1 and human BET bromodomains. Bromodomain binding pockets contain a conserved array of structural waters. Molecular dynamics simulations now reveal that one water molecule is less tightly bound to BD2 than to BD1, explaining the site selectivity of1. This insight is useful in the performance of ligand docking studies to guide design of more effective Bdf1 inhibitors, as illustrated by the design of 10 new imidazopyridine BD2 ligands1a–j, for which experimental binding and site selectivity data are presented.},
  archive      = {J_JCC},
  author       = {Yingsheng Zhou and Justin M. Overhulse and Nathan J. Dupper and Yanchun Guo and Boris A. Kashemirov and Kaiyao Wei and Jérôme Govin and Carlo Petosa and Charles E. McKenna},
  doi          = {10.1002/jcc.26997},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2121-2130},
  shortjournal = {J. Comput. Chem.},
  title        = {Toward more potent imidazopyridine inhibitors of candida albicans bdf1: Modeling the role of structural waters in selective ligand binding},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sensitivity of coupled cluster electronic properties on the
reference determinant: Can kohn–sham orbitals be more beneficial than
hartree–fock orbitals? <em>JCC</em>, <em>43</em>(32), 2103–2120. (<a
href="https://doi.org/10.1002/jcc.26996">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Coupled cluster calculations are traditionally performed over Hartree–Fock reference orbitals (HF-CC methodology). However, in the literature it has been repeatedly raised whether the use of a Kohn–Sham reference (KS-CC methodology) might result in improved performance relative to HF-CC. In the present study, we re-examine the relation of HF-CC and KS-CC methods by comparing the results of widely applied truncated CC calculations (CCSD, CCSD(T), CCSDT) to the limit of full configuration interaction (FCI), which serves as an undebatable reference point of accuracy. Based on a series of CC calculations on diatoms and transition metal complexes, we demonstrate that no systematic improvement of coupled cluster electronic energies, densities and chemical reaction energies is expected when changing from HF to a KS reference. Nevertheless, fortuitous error cancellations might occasionally result in illusory improvement compared to HF-CC. Altogether, the application of KS-CC is not advantageous over HF-CC, but it is also not unreasonable as the choice of reference has negligible influence on the results at sufficiently high CC levels. In addition, KS-CC can be a particularly useful alternative if difficulties are encountered in HF or HF-CC convergence. It is also notable that KS-CC results are found to be practically independent of the chosen density functional, which implies that almost any KS-CC method can be used in place of HF-CC.},
  archive      = {J_JCC},
  author       = {Zsolt Benedek and Paula Timár and Tibor Szilvási and Gergely Barcza},
  doi          = {10.1002/jcc.26996},
  journal      = {Journal of Computational Chemistry},
  number       = {32},
  pages        = {2103-2120},
  shortjournal = {J. Comput. Chem.},
  title        = {Sensitivity of coupled cluster electronic properties on the reference determinant: Can Kohn–Sham orbitals be more beneficial than Hartree–Fock orbitals?},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Producing chemically accurate atomic gaussian process
regression models by active learning for molecular simulation.
<em>JCC</em>, <em>43</em>(31), 2084–2098. (<a
href="https://doi.org/10.1002/jcc.27006">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning is becoming increasingly more important in the field of force field development. Never has it been more vital to have chemically accurate machine learning potentials because force fields become more sophisticated and their applications expand. In this study a method for developing chemically accurate Gaussian process regression models is demonstrated for an increasingly complex set of molecules. This work is an extension to previous work showing the progression of the active learning technique in producing more accurate models in much less CPU time than ever before. The per-atom active learning approach has unlocked the potential to generate chemically accurate models for molecules such as peptide-capped glycine.},
  archive      = {J_JCC},
  author       = {Matthew J. Burn and Paul L. A. Popelier},
  doi          = {10.1002/jcc.27006},
  journal      = {Journal of Computational Chemistry},
  number       = {31},
  pages        = {2084-2098},
  shortjournal = {J. Comput. Chem.},
  title        = {Producing chemically accurate atomic gaussian process regression models by active learning for molecular simulation},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Branched polyethyleneimine: CHARMM force field and molecular
dynamics simulations. <em>JCC</em>, <em>43</em>(31), 2072–2083. (<a
href="https://doi.org/10.1002/jcc.27005">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polyethyleneimine (PEI), one of the non-viral vectors of great interest for gene delivery, was investigated at all-atom level, with particular emphasis on its branched form. We report the extension of our previously published CHARMM force field (FF) for linear PEI, with parameters optimized specifically for branched configurations. A new residue type for the branch connector is introduced and the charges and bonded parameters are derived from ab initio calculations based on a model polymer. The new FF is validated by extensive molecular dynamics simulations of solvated branched PEIs of various protonation fractions and branch lengths. The gyration radii, end-to-end distances, and diffusion coefficients are compared with results for linear PEIs of similar molecular weights and protonation patterns. Solvated complexes of DNA with (linear/branched) PEI were also investigated to determine favorable attachment conformations. The parametrized atomistic force field is suitable for simulations of PEI with arbitrary branching pattern, protonation, and size, and is expected to provide relevant insights regarding optimal conditions for DNA-PEI complex formation.},
  archive      = {J_JCC},
  author       = {Andrada-Elena Terteci-Popescu and Titus Adrian Beu},
  doi          = {10.1002/jcc.27005},
  journal      = {Journal of Computational Chemistry},
  number       = {31},
  pages        = {2072-2083},
  shortjournal = {J. Comput. Chem.},
  title        = {Branched polyethyleneimine: CHARMM force field and molecular dynamics simulations},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Density functional theory studies on molecular geometry,
spectroscopy, HOMO–LUMO and reactivity descriptors of titanium(IV) and
oxidozirconium(IV) complexes of phenylacetohydroxamic acid.
<em>JCC</em>, <em>43</em>(31), 2060–2071. (<a
href="https://doi.org/10.1002/jcc.27004">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The molecular geometry of new titanium(IV) and oxidozirconium(IV) phenylacetohydroxamate complexes [TiCl2(L1)2] (I) and [ZrO(L1)2] (II) (where L1 = Potassium phenylacetohydroxamate = C6H5CH2CONHOK) computed by B3LYP/6-311++G(d,p) method has shown these to be distorted octahedral and square pyramidal, respectively. A comparison of computed characteristic bond lengths (CO, CN, and NO) of complexes with that of free ligand has shown chelation through carbonyl and hydroxamic oxygen atoms (O, O coordination). The TiO/ZrO bond lengths in complexes are suggestive of weak coordination through (carbonyl CO) and strong covalent (hydroxamic NO) bonding of the ligand. The magnitude of ClTiCl bond angle involving two chloride atoms is suggestive of cis-conformation at titanium metal in (I). The thermodynamic parameters Gibbs free energy, enthalpy, entropy, nuclear internal energy, constant volume heat capacity, and internal energy of ligand and complexes have been computed. From the energies of highest occupied molecular orbital (HOMO) and lowest unoccupied molecular orbital (LUMO), the global reactivity descriptors such as ionization potential (IP), electron affinity (EA), chemical potential (μ), hardness (η), softness (S), electronegativity (χ), electrophilicity index (ω), and dipole moment have been calculated. The computed vibrational frequencies,1H and13C NMR spectra have substantiated the molecular structure of complexes. The thermal behavior of complexes has been studied by thermogravimetric techniques (TGA, DTG, and DTA) in N2atmosphere has shown complexes are thermally stable.},
  archive      = {J_JCC},
  author       = {Vineet Kumar Choudhary and Kanika Mandhan and Dibyajit Dash and Sachin Bhardwaj and Meena Kumari and Neeraj Sharma},
  doi          = {10.1002/jcc.27004},
  journal      = {Journal of Computational Chemistry},
  number       = {31},
  pages        = {2060-2071},
  shortjournal = {J. Comput. Chem.},
  title        = {Density functional theory studies on molecular geometry, spectroscopy, HOMO–LUMO and reactivity descriptors of titanium(IV) and oxidozirconium(IV) complexes of phenylacetohydroxamic acid},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A coarse-grained approach to NMR-data-assisted modeling of
protein structures. <em>JCC</em>, <em>43</em>(31), 2047–2059. (<a
href="https://doi.org/10.1002/jcc.27003">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ESCASA algorithm for analytical estimation of proton positions from coarse-grained geometry developed in our recent work has been implemented in modeling protein structures with the highly coarse-grained UNRES model of polypeptide chains (two sites per residue) and nuclear magnetic resonance (NMR) data. A penalty function with the shape of intersecting gorges was applied to treat ambiguous distance restraints, which automatically selects consistent restraints. Hamiltonian replica exchange molecular dynamics was used to carry out the conformational search. The method was tested with both unambiguous and ambiguous restraints producing good-quality models with GDT_TS from 7.4 units higher to 14.4 units lower than those obtained with the CYANA or MELD software for protein-structure determination from NMR data at the all-atom resolution. The method can thus be applied in modeling the structures of flexible proteins, for which extensive conformational search enabled by coarse-graining is more important than high modeling accuracy.},
  archive      = {J_JCC},
  author       = {Emilia A. Lubecka and Adam Liwo},
  doi          = {10.1002/jcc.27003},
  journal      = {Journal of Computational Chemistry},
  number       = {31},
  pages        = {2047-2059},
  shortjournal = {J. Comput. Chem.},
  title        = {A coarse-grained approach to NMR-data-assisted modeling of protein structures},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). On the origin of photodynamic activity of hypericin and its
iodine-containing derivatives. <em>JCC</em>, <em>43</em>(30), 2037–2042.
(<a href="https://doi.org/10.1002/jcc.27002">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The main photophysical properties, useful for establishing whether hypericin in anionic form and some of its derivatives containing heavy atoms such as iodine, can be proposed for their use in photodynamic therapy, were determined using density functional based computations. The results showed that in the anionic form and in the iodinated derivatives, the absorption wavelength undergoes a bathochromic shift, the singlet-triplet energy gap assumes values ​that allow to excite the oxygen molecule from its ground to the excited singlet state, and that the spin–orbit couplings between singlet and triplet states significantly increase.},
  archive      = {J_JCC},
  author       = {Bruna Clara De Simone and Gloria Mazzone and Marirosa Toscano and Nino Russo},
  doi          = {10.1002/jcc.27002},
  journal      = {Journal of Computational Chemistry},
  number       = {30},
  pages        = {2037-2042},
  shortjournal = {J. Comput. Chem.},
  title        = {On the origin of photodynamic activity of hypericin and its iodine-containing derivatives},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A computational approach for modeling electronic circular
dichroism of solvated chromophores. <em>JCC</em>, <em>43</em>(30),
2023–2036. (<a href="https://doi.org/10.1002/jcc.27001">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The present study consists in a novel computational protocol to model the UV-circular dichroism spectra of solvated species. It makes use of quantum-chemical calculations on a series of conformations of a flexible chromophore or on a series of chromophore/solvent clusters extracted from molecular dynamic simulations. The protocol is described and applied to the aqueous cationic tripeptide GAG+and to the aqueous neutral decapeptide (GVGVP)2. The protocol has proven able to: (i) properly consider the conformational motion of solute in the given environment; (ii) give the actual statistical weight of each conformational state; (iii) provide a reliable quantum mechanical method able to reproduce the spectral features. Temperature effects on conformations and spectral properties are properly taken into account. The role of explicit solvent on the conformational analysis and the spectra calculation is discussed. The comparison of the calculated circular dichroism spectra with experimental ones recorded at different temperatures represents a strict validation test of the method.},
  archive      = {J_JCC},
  author       = {Marta Monti and Mauro Stener and Massimiliano Aschi},
  doi          = {10.1002/jcc.27001},
  journal      = {Journal of Computational Chemistry},
  number       = {30},
  pages        = {2023-2036},
  shortjournal = {J. Comput. Chem.},
  title        = {A computational approach for modeling electronic circular dichroism of solvated chromophores},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Modeling pyranose ring pucker in carbohydrates using machine
learning and semi-empirical quantum chemical methods. <em>JCC</em>,
<em>43</em>(30), 2009–2022. (<a
href="https://doi.org/10.1002/jcc.27000">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pyranose ring pucker is a key coordinate governing the structure, interactions and reactivity of carbohydrates. We assess the ability of the machine learning potentials, ANI-1ccx and ANI-2x, and the GFN2-xTB semiempirical quantum chemical method, to model ring pucker conformers of five monosaccharides and oxane in the gas phase. Relative to coupled-cluster quantum mechanical calculations, we find that ANI-1ccx most accurately reproduces the ring pucker energy landscape for these molecules, with a correlation coefficientr2of 0.83. This correlation in relative energies lowers to values of 0.70 for ANI-2x and 0.60 for GFN2-xTB. The ANI-1ccx also provides the most accurate estimate of the energetics of the4C1-to-1C4minimum energy pathway for the six molecules. All three models reproduce chair more accurately than non-chair geometries. Analysis of small model molecules suggests that the ANI-1ccx model favors puckers with equatorial hydrogen bonding substituents; that ANI-2x and GFN2-xTB models overstabilize conformers with axially oriented groups; and that theendo-anomeric effect is overestimated by the machine learning models and underestimated via the GFN2-xTB method. While the pucker conformers considered in this study correspond to a gas phase environment, the accuracy and computational efficiency of the ANI-1ccx approach in modeling ring pucker in vacuo provides a promising basis for future evaluation and application to condensed phase environments.},
  archive      = {J_JCC},
  author       = {Linghan Kong and Richard A. Bryce},
  doi          = {10.1002/jcc.27000},
  journal      = {Journal of Computational Chemistry},
  number       = {30},
  pages        = {2009-2022},
  shortjournal = {J. Comput. Chem.},
  title        = {Modeling pyranose ring pucker in carbohydrates using machine learning and semi-empirical quantum chemical methods},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Modeling the spectral properties of poly(x-phenylenediamine)
conducting polymers using a combined TD-DFT and electrostatic embedding
approach. <em>JCC</em>, <em>43</em>(30), 2001–2008. (<a
href="https://doi.org/10.1002/jcc.26955">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The absorption spectra of polymers derived fromortho,metaandparaphenylenediamines (o-PDA,m-PDA andp-PDA) have been simulated combining periodic density functional theory (DFT) calculations with time-dependent DFT simulations. These latter have been carried out on finite clusters embedded in a set of point charges devised to exactly reproduce the electrostatic potential of the periodic chains. The results are compared with those obtained for solvatedo-PDA,m-PDA andp-PDA oligomers of increasing sizes extracted from the periodic structures. The electronic transitions involved have been investigated by a qualitative analysis based on isodensity maps completed by a quantitative analysis based on the density-based index (DCT). For poly-(o)- and poly-(p)- phenylenediamines the agreement with the experimental data is achieved already by modeling solvated dimers whereas the inclusion of long-range electrostatic effects is mandatory for poly-(m)-phenylenediamine highlighting the importance of an accurate treatment of the electrostatic environment when a finite cluster approach is considered.},
  archive      = {J_JCC},
  author       = {Davide Luise and Massimo Christian D&#39;Alterio and Giovanni Talarico and Ilaria Ciofini and Frédéric Labat},
  doi          = {10.1002/jcc.26955},
  journal      = {Journal of Computational Chemistry},
  number       = {30},
  pages        = {2001-2008},
  shortjournal = {J. Comput. Chem.},
  title        = {Modeling the spectral properties of poly(x-phenylenediamine) conducting polymers using a combined TD-DFT and electrostatic embedding approach},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). On closed-shell interactions between heavy main-group
elements. <em>JCC</em>, <em>43</em>(29), 1985–1996. (<a
href="https://doi.org/10.1002/jcc.26999">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A series of di- and polymetal complexes involving closed-shell, heavy main-group atoms and ions shows a selection of special physical properties. These involve short metal–metal contacts, low entropies of formation and, most interestingly, strong Raman bands at low wavenumbers. These results together with the constitution of the coordination compounds, where the majority of electrons are assembled on the highly polarizable metal atoms and ions, experimental results have been interpreted in terms of direct, partial covalent metal–metal bonding. Previous theoretical studies have challenged this view and instead attributed the obvious attractive forces involved to secondary-type of interactions, such as dispersion. This study utilizes a multitude of theoretical tools, such as natural bond order (NBO) and natural energy decomposition analysis (NEDA), non-covalent interaction (NCI) analysis, electron localization functions (ELFs), and atoms-in-molecules (AIM) to characterize the interactions in models comprising closed-shell dimers, as well as experimentally studied ring and cage systems constituting the main reason for the hypotheses on metal–metal interactions. The results show that all experimental results can be attributed to the covalent interactions between the electron-rich, metal centers and the bridging anions in ring and cage coordination compounds of high symmetry, where the experimentally observed effects can be traced to the combination of covalent interactions between the metal centers and the anions along the edges of the ring and cage systems in combination with the cooperative effects generated by the high symmetry of these ring and cage systems.},
  archive      = {J_JCC},
  author       = {Lars Kloo},
  doi          = {10.1002/jcc.26999},
  journal      = {Journal of Computational Chemistry},
  number       = {29},
  pages        = {1985-1996},
  shortjournal = {J. Comput. Chem.},
  title        = {On closed-shell interactions between heavy main-group elements},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Structural evolution and electronic properties of pure and
semiconductor atom doped in clusters: Inn−, InnSi−, and InnGe− (n =
3–16). <em>JCC</em>, <em>43</em>(29), 1978–1984. (<a
href="https://doi.org/10.1002/jcc.26998">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The bonding and electronic properties of Inn−, InnSi−, and InnGe−(n= 3–16) clusters have been computationally investigated. An intensive global search for the ground-state structures of these clusters were conducted using the genetic algorithm coupled with density functional theory (DFT). The ground-state structures of these clusters have been identified through the comparison between simulated photoelectron spectra (PES) of the found lowest-energy isomers and the experimentally measured ones. Doping semiconductor atom (Si or Ge) can significantly change the structures of the In clusters in most sizes, and the dopant prefers to be surrounded by In atoms. There are three structural motifs for InnX−(X = Si, Ge,n= 3–16), and the transition occurs at sizesn= 5 and 13. All InnSi−and InnGe−share the same configurations and similar electronic properties except forn= 8. Among all above studied clusters, In13−stands out with the largest vertical detachment energy (VDE), HOMO–LUMO gap, (Eb) and second order energy difference Δ2Edue to its closed electronic shell of (1S)2(1P)6(1D)10(2S)2(1F)14(2P)6. Similarly, the neutral In12X (X = Si, Ge) clusters are also identified as superatoms but with electronic configuration of (1S)2(1P)6(2S)2(1D)10(1F)14(2P)6.},
  archive      = {J_JCC},
  author       = {Kai Wang and Lin Miao and Zezhao Jia and Runyu Wang and Guangjia Yin and Xiaodong Zhu and Ramiro Moro and Lei Ma},
  doi          = {10.1002/jcc.26998},
  journal      = {Journal of Computational Chemistry},
  number       = {29},
  pages        = {1978-1984},
  shortjournal = {J. Comput. Chem.},
  title        = {Structural evolution and electronic properties of pure and semiconductor atom doped in clusters: Inn−, InnSi−, and InnGe− (n = 3–16)},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ligand-stabilized heteronuclear diatomics of group 13 and
15. <em>JCC</em>, <em>43</em>(29), 1964–1977. (<a
href="https://doi.org/10.1002/jcc.26995">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A theoretical investigation of ligand-stabilized MX diatomics (M = group 13, X = group 15 element) withN-heterocyclic carbene (NHC) ligands has been carried out to assess bonding and electronic structure. Binding of two ligands in the form L-MX-L is generally preferred over binding of a single ligand as L-MX or MX-L. Binding of carbene donor ligands is predicted to be thermodynamically favorable for all the systems, and is very favorable for the lighter group 15 systems (nitrogen and phosphorus). Detailed analysis of the bonding in these complexes has been carried out with energy decomposition analysis (EDA). In all cases, the carbene to boron and carbene to nitrogen bonding is described as an electron-sharing double bond with both σ and π bonding interactions. For the heavier elements, bonding to C (except for PC interactions) is best described as a donor-acceptor σ single bond.},
  archive      = {J_JCC},
  author       = {Aishvaryadeep Kaur and David J. D. Wilson},
  doi          = {10.1002/jcc.26995},
  journal      = {Journal of Computational Chemistry},
  number       = {29},
  pages        = {1964-1977},
  shortjournal = {J. Comput. Chem.},
  title        = {Ligand-stabilized heteronuclear diatomics of group 13 and 15},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A molecular evolution algorithm for ligand design in DOCK.
<em>JCC</em>, <em>43</em>(29), 1942–1963. (<a
href="https://doi.org/10.1002/jcc.26993">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As a complement to virtual screening, de novo design of small molecules is an alternative approach for identifying potential drug candidates. Here, we present a new 3D genetic algorithm to evolve molecules through breeding, mutation, fitness pressure, and selection. The method, termed DOCK_GA, builds upon and leverages powerful sampling, scoring, and searching routines previously implemented into DOCK6. Three primary experiments were used during development:Single-molecule evolutionevaluated three selection methods (elitism, tournament, and roulette), in four clinically relevant systems, in terms of mutation type and crossover success, chemical properties, ensemble diversity, and fitness convergence, among others.Large scale benchmarkingassessed performance across 651 different protein-ligand systems.Ensemble-based evolutiondemonstrated using multiple inhibitors simultaneously to seed growth in a SARS-CoV-2 target. Key takeaways include: (1) The algorithm is robust as demonstrated by the successful evolution of molecules across a large diverse dataset. (2) Users have flexibility with regards to parent input, selection method, fitness function, and molecular descriptors. (3) The program is straightforward to run and only requires a single executable and input file at run-time. (4) The elitism selection method yields more tightly clustered molecules in terms of 2D/3D similarity, with more favorable fitness, followed by tournament and roulette.},
  archive      = {J_JCC},
  author       = {Lauren E. Prentis and Courtney D. Singleton and John D. Bickel and William J. Allen and Robert C. Rizzo},
  doi          = {10.1002/jcc.26993},
  journal      = {Journal of Computational Chemistry},
  number       = {29},
  pages        = {1942-1963},
  shortjournal = {J. Comput. Chem.},
  title        = {A molecular evolution algorithm for ligand design in DOCK},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Cover image. <em>JCC</em>, <em>43</em>(28), i. (<a
href="https://doi.org/10.1002/jcc.27014">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The cover image is based on the Research ArticlePrediction of self-diffusion coefficients of chemically diverse pure liquids by all-atom molecular dynamics simulationsby Hiromi Baba et al.,https://doi.org/10.1002/jcc.26975},
  archive      = {J_JCC},
  author       = {Hiromi Baba and Ryo Urano and Tetsuro Nagai and Susumu Okazaki},
  doi          = {10.1002/jcc.27014},
  journal      = {Journal of Computational Chemistry},
  number       = {28},
  pages        = {i},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Optimization of density fitting auxiliary slater-type basis
functions for time-dependent density functional theory. <em>JCC</em>,
<em>43</em>(28), 1923–1935. (<a
href="https://doi.org/10.1002/jcc.26992">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A new set of auxiliary basis function suitable to fit the induced electron density is presented. Such set has been optimized in order to furnish accurate absorption spectra using the complex polarizability algorithm of time-dependent density functional theory (TDDFT). An automatic procedure has been set up, able, thanks to the definition of suitable descriptors, to evaluate the resemblance of the auxiliary basis-dependent calculated spectra with respect to a reference. In this way, it has been possible to reduce the size of the basis set maximizing the basis set accuracy. Thanks to the choice to employ a collection of molecules for each element, such basis has proven transferable to molecules outside the collection. The final sets are therefore much more accurate and smaller than the previously optimized ones and have been already included in the database of the last release of the AMS suite of programs. The availability of the present new set will allow to improve drastically the applicability range of the polTDDFT method with higher accuracy and less computational effort.},
  archive      = {J_JCC},
  author       = {Marco Medves and Giovanna Fronzoni and Mauro Stener},
  doi          = {10.1002/jcc.26992},
  journal      = {Journal of Computational Chemistry},
  number       = {28},
  pages        = {1923-1935},
  shortjournal = {J. Comput. Chem.},
  title        = {Optimization of density fitting auxiliary slater-type basis functions for time-dependent density functional theory},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Simulation of ligand dissociation kinetics from the protein
kinase PYK2. <em>JCC</em>, <em>43</em>(28), 1911–1922. (<a
href="https://doi.org/10.1002/jcc.26991">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Early-stage drug discovery projects often focus on equilibrium binding affinity to the target alongside selectivity and other pharmaceutical properties. The kinetics of drug binding are ignored but can have significant influence on drug efficacy. Therefore, increasing attention has been paid on evaluating drug-binding kinetics early in a drug discovery process. Simulating drug-binding kinetics at the atomic level is challenging for the long time scale involved. Here, we used the transition-based reweighting analysis method (TRAM) with the Markov state model to study the dissociation of a ligand from the protein kinase PYK2. TRAM combines biased and unbiased simulations to reduce computational costs. This work used the umbrella sampling technique for the biased simulations. Although using the potential of mean force from umbrella sampling simulations with the transition-state theory over-estimated the dissociation rate by three orders of magnitude, TRAM gave a dissociation rate within an order of magnitude of the experimental value.},
  archive      = {J_JCC},
  author       = {Justin Spiriti and Frank Noé and Chung F. Wong},
  doi          = {10.1002/jcc.26991},
  journal      = {Journal of Computational Chemistry},
  number       = {28},
  pages        = {1911-1922},
  shortjournal = {J. Comput. Chem.},
  title        = {Simulation of ligand dissociation kinetics from the protein kinase PYK2},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Relativistic adapted gaussian basis sets free of variational
prolapse of small and medium size for hydrogen through xenon.
<em>JCC</em>, <em>43</em>(28), 1901–1910. (<a
href="https://doi.org/10.1002/jcc.26990">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Two new relativistic adapted Gaussian basis sets of small and medium size are presented for all elements from Hydrogen through Xenon. These sets are free of variational prolapse and were developed with the polynomial generator coordinate Dirac-Fock method to be used with two finite nuclear models, uniform sphere and Gaussian. The largest basis set errors for electronic configurations from the Aufbau principle are around 10.0 and 4.7 mHartree for the small– and medium–size sets, respectively, which is in accordance with the accuracy level expected in each case. Hence, to our knowledge, these are the smallest prolapse free basis sets developed for these elements.},
  archive      = {J_JCC},
  author       = {Eriosvaldo Florentino Gusmão and Roberto Luiz Andrade Haiduke},
  doi          = {10.1002/jcc.26990},
  journal      = {Journal of Computational Chemistry},
  number       = {28},
  pages        = {1901-1910},
  shortjournal = {J. Comput. Chem.},
  title        = {Relativistic adapted gaussian basis sets free of variational prolapse of small and medium size for hydrogen through xenon},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Prediction of self-diffusion coefficients of chemically
diverse pure liquids by all-atom molecular dynamics simulations.
<em>JCC</em>, <em>43</em>(28), 1892–1900. (<a
href="https://doi.org/10.1002/jcc.26975">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular self-diffusion coefficients underlie various kinetic properties of the liquids involved in chemistry, physics, and pharmaceutics. In this study, 547 self-diffusion coefficients are calculated based on all-atom molecular dynamics (MD) simulations of 152 diverse pure liquids at various temperatures employing the OPLS4 force field. The calculated coefficients are compared with experimental data (424 extracted from the literature and 123 newly measured by pulsed-field gradient nuclear magnetic resonance). The calculations well agree with the experimental values. The determination coefficient and root mean square error between the observed and calculated logarithmic self-diffusion coefficients of the 547 entries are 0.931 and 0.213, respectively, demonstrating that the MD calculation can be an excellent industrial tool for predicting, for example, molecular transportation in liquids such as the diffusion of active ingredients in biological and pharmaceutical liquids. The self-diffusion coefficients collected in this study are compiled into a database for broad researches including artificial intelligence calculations.},
  archive      = {J_JCC},
  author       = {Hiromi Baba and Ryo Urano and Tetsuro Nagai and Susumu Okazaki},
  doi          = {10.1002/jcc.26975},
  journal      = {Journal of Computational Chemistry},
  number       = {28},
  pages        = {1892-1900},
  shortjournal = {J. Comput. Chem.},
  title        = {Prediction of self-diffusion coefficients of chemically diverse pure liquids by all-atom molecular dynamics simulations},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Conformer-RL: A deep reinforcement learning library for
conformer generation. <em>JCC</em>, <em>43</em>(27), 1880–1886. (<a
href="https://doi.org/10.1002/jcc.26984">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Conformer-RLis an open-source Python package for applying deep reinforcement learning (RL) to the task of generating a diverse set of low-energy conformations for a single molecule. The library features a simple interface to train a deep RL conformer generation model on any covalently bonded molecule or polymer, including most drug-like molecules. Under the hood, it implements state-of-the-art RL algorithms and graph neural network architectures tuned specifically for molecular structures.Conformer-RLis also a platform for researching new algorithms and neural network architectures for conformer generation, as the library contains modular class interfaces for RL environments and agents, allowing users to easily swap components with their own implementations. Additionally, it comes with tools to visualize and save generated conformers for further analysis.Conformer-RLis well-tested and thoroughly documented with tutorials for each of the functionalities mentioned above, and is available on PyPi and Github:https://github.com/ZimmermanGroup/conformer-rl.},
  archive      = {J_JCC},
  author       = {Runxuan Jiang and Tarun Gogineni and Joshua Kammeraad and Yifei He and Ambuj Tewari and Paul M. Zimmerman},
  doi          = {10.1002/jcc.26984},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1880-1886},
  shortjournal = {J. Comput. Chem.},
  title        = {Conformer-RL: A deep reinforcement learning library for conformer generation},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Path integral brownian chain molecular dynamics: A simple
approximation of quantum vibrational dynamics. <em>JCC</em>,
<em>43</em>(27), 1864–1879. (<a
href="https://doi.org/10.1002/jcc.26989">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An approximate approach to quantum vibrational dynamics, “Brownian chain molecular dynamics (BCMD),” is proposed to alleviate the chain resonance and curvature problems in the imaginary time-based path integral (PI) simulation. Here the non-centroid velocity is randomized at each step when solving the equation of motion of path integral molecular dynamics. This leads to a combination of the Newton equation and the overdamped Langevin equation for the centroid and non-centroid variables, respectively. BCMD shares the basic properties of other PI approaches such as centroid and ring polymer molecular dynamics: It gives the correct Kubo-transformed correlation function at short times, conserves the time symmetry, has the correct high-temperature/classical limits, gives exactly the position and velocity autocorrelations of harmonic oscillator systems, and does not have the zero-point leakage problem. Numerical tests were done on simple molecular models and liquid water. On-the-fly ab initio BCMD simulations were performed for the protonated water cluster,H5O2+, and its isotopologue,D5O2+.},
  archive      = {J_JCC},
  author       = {Motoyuki Shiga},
  doi          = {10.1002/jcc.26989},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1864-1879},
  shortjournal = {J. Comput. Chem.},
  title        = {Path integral brownian chain molecular dynamics: A simple approximation of quantum vibrational dynamics},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Conformational energies of microsolvated na+ clusters with
protic and aprotic solvents from GFNn-xTB methods. <em>JCC</em>,
<em>43</em>(27), 1856–1863. (<a
href="https://doi.org/10.1002/jcc.26988">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Performance of contemporary tight-binding semiempirical GFNn-xTB methods for the conformational energies of singly charged sodium clusters Na+(S)n(n= 4–8) with 3 protic and 8 aprotic solvents is examined against the reference RI-MP2/CBS method. The median Pearson correlation coefficients ofρ= 0.84 (GFN2-xTB) andρ= 0.82 (GFN1-xTB) do not give the clear preference to any tested approach. GFN1-xTB method demonstrates more stable performance than its GFN2-xTB successor with the average mean absolute errors (MAEs)/mean signed errors (MSEs) of 1.2/0.2 and 2.3/1.6 kcal mol−1, respectively. Conformational energies produced by the computationally efficient DFT functional PBE and double-ζ basis set complemented with –D3(BJ) dispersion correction are suitable for the preliminary sampling (medianρ= 0.93), but should be used with a caution for the calculations of the average ensemble properties (MAE/MSE = 1.7/1.1 kcal mol−1). Higher-ranking PBE0-D3(BJ) and ωB97M-V with triple-ζ basis sets yield significantly lower MAEs/MSEs of 0.55/0.20 and 0.51/0.23 kcal mol−1, respectively.},
  archive      = {J_JCC},
  author       = {Arseniy A. Otlyotov and Yury Minenkov},
  doi          = {10.1002/jcc.26988},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1856-1863},
  shortjournal = {J. Comput. Chem.},
  title        = {Conformational energies of microsolvated na+ clusters with protic and aprotic solvents from GFNn-xTB methods},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bonding and 13C-NMR properties of coinage metal
tris(ethylene) and tris(norbornene) complexes: Evaluation of the role of
relativistic effects from DFT calculations. <em>JCC</em>,
<em>43</em>(27), 1848–1855. (<a
href="https://doi.org/10.1002/jcc.26987">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The π-complexes of cationic coinage metal ions (Cu(I), Ag(I), Au(I)) provide useful experimental support for understanding fundamental characteristics of bonding and13C-NMR patterns of the group 11 triad. Here, we account for the role of relativistic effects on olefin-coinage metal ion interaction for cationic, homoleptic tris-ethylene, and tris-norbornene complexes, [M(η2-C2H4)3]+and [M(η2-C7H10)3]+(M = Cu, Ag, Au), as representative case of studies. The M-(CC) bond strength in the cationic, tris-ethylene complexes is affected sizably for Au and to a lesser extent for Ag and Cu (48.6\%, 16.7\%, and 4.3\%, respectively), owing to the influence on the different stabilizing terms accounting for the interaction energy in the formation of coinage metal cation-π complexes. The bonding elements provided by olefin → M σ-donation and olefin ← M π-backbonding are consequently affected, leading to a lesser covalent interaction going down in the triad if the relativistic effects are ignored. Analysis of the13C-NMR tensors provides further understanding of the observed experimental values, where the degree of backbonding charge donation to π2*-olefin orbital is the main influence on the observed high-field shifts in comparison to the free olefin. This donation is larger for ethylene complexes and lower for norbornene counterparts. However, the bonding energy in the later complexes is slightly stabilized given by the enhancement in the electrostatic character of the interaction. Thus, the theoretical evaluation of metal-alkene bonds, and other metal-bonding situations, benefits from the incorporation of relativistic effects even in lighter counterparts, which have an increasing role going down in the group.},
  archive      = {J_JCC},
  author       = {Alvaro Muñoz-Castro and H. V. Rasika Dias},
  doi          = {10.1002/jcc.26987},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1848-1855},
  shortjournal = {J. Comput. Chem.},
  title        = {Bonding and 13C-NMR properties of coinage metal tris(ethylene) and tris(norbornene) complexes: Evaluation of the role of relativistic effects from DFT calculations},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Zinc oxide nanoclusters and their potential application as
CH4 and CO2 gas sensors: Insight from DFT and TD-DFT. <em>JCC</em>,
<em>43</em>(27), 1839–1847. (<a
href="https://doi.org/10.1002/jcc.26986">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We have investigated the adsorption of CH4and CO2gases on zinc oxide nanoclusters (ZnO NCs) using density functional theory (DFT). It was found that the CH4tends to be physically adsorbed on the surface of all the ZnO NCs with adsorption energy in the range −11 to −14 kcal/mol. Even though, the CO2is favorably chemisorbed on the Zn12O12and Zn15O15NCs, with adsorption energy about −38 kcal/mol at B3LYP/6-311G(d,p) level of theory. When the CH4and CO2gases are adsorbed on the ZnO NCs, their electrical conductivities are decreased, and thus the studied ZnO NCs do not generate an electrical signal in the presence of CH4and CO2gases. Interestingly, both pure and gas adsorbed Zn22O22NC exhibited more favorable electronic and reactive properties than other NCs. Comparison of the structural, electronic, and optical data predicted by DFT/B3LYP and TD-DFT/CAM-B3LYP calculations with those experimentally obtained show good agreement.},
  archive      = {J_JCC},
  author       = {İskender Muz and Mustafa Kurban},
  doi          = {10.1002/jcc.26986},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1839-1847},
  shortjournal = {J. Comput. Chem.},
  title        = {Zinc oxide nanoclusters and their potential application as CH4 and CO2 gas sensors: Insight from DFT and TD-DFT},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The performance of exchange—correlation functionals in
describing electron density parameters of saddle point structures along
chemical reactions. <em>JCC</em>, <em>43</em>(27), 1830–1838. (<a
href="https://doi.org/10.1002/jcc.26985">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This work is focused on evaluating the performance of exchange-correlation functionals from density functional theory in providing descriptor values derived from the electron density of saddle point structures (transition states) in chemical reactions. The properties investigated were obtained from the quantum theory of atoms in molecules, including atomic charges and electron density topological data at the bond critical points. In addition, parameters from the Interacting quantum atom energy partition were used as well in this comparative study. The reference values are attained in coupled cluster calculations with iterative single and double excitations (CCSD). Six elementary reactions are considered here: CO + H2↔ H2CO, CO + H2O ↔ HCOOH, HCN ↔ HNC, H + F2↔ HF + F, H + N2↔ HN2, and H + CO ↔ HCO. In general, the BB1K functional (hybrid-meta-generalized gradient approximation) provides the best description of these properties. Our study indicates that an intermediate percentage of nonlocal exact exchange, around 40\%–55\% (perhaps even larger), is probably required for attaining more accurate values with actual functionals, although this condition is not able of explaining all the trends observed.},
  archive      = {J_JCC},
  author       = {Gabriel Kossaka Macedo and Roberto Luiz Andrade Haiduke},
  doi          = {10.1002/jcc.26985},
  journal      = {Journal of Computational Chemistry},
  number       = {27},
  pages        = {1830-1838},
  shortjournal = {J. Comput. Chem.},
  title        = {The performance of exchange—correlation functionals in describing electron density parameters of saddle point structures along chemical reactions},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). On the reliability of atoms in molecules, noncovalent
index, and natural bond orbital to identify and quantify noncovalent
bonds. <em>JCC</em>, <em>43</em>(26), 1814–1824. (<a
href="https://doi.org/10.1002/jcc.26983">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Atoms in molecules, noncovalent index, and natural bond orbital methods are commonly invoked to identify the presence of various noncovalent bonds and to measure their strength. However, there are numerous instances in the literature where these methods provide contradictory or apparently erroneous interpretations of the bonding. The range of reliability of these methods is assessed by calculations of a variety of systems, which include an H-bond, halogen bond, π-tetrel bond, CH··HC interaction, and a pairing of two anions. While the results appear to be meaningful for the equilibrium geometries, and those where the two subunits are progressively pulled apart, these techniques erroneously predict a progressively stronger bonding interaction as the two units are compressed and the interaction becomes clearly repulsive. The methods falsely indicate a bonding interaction in the CH··HC arrangement, and incorrectly mimic the behavior of the energy when two anions approach. These approaches are also unreliable for understanding angular deformations.},
  archive      = {J_JCC},
  author       = {Steve Scheiner},
  doi          = {10.1002/jcc.26983},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1814-1824},
  shortjournal = {J. Comput. Chem.},
  title        = {On the reliability of atoms in molecules, noncovalent index, and natural bond orbital to identify and quantify noncovalent bonds},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Molecular mechanisms from reaction coordinate graph enabled
multidimensional free energies illustrated on water dimer hydrogen
bonding. <em>JCC</em>, <em>43</em>(26), 1802–1813. (<a
href="https://doi.org/10.1002/jcc.26982">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computing the free energies of molecular mechanisms in multidimensional space relies on combinations of geometrically complex reaction coordinates. We show how a graph theory implementation reduces complexity, and illustrate this on the arrangements of hydrogen bonding of a water dimer. The reaction coordinates and forces are computed using graphs that define the dependencies on the atoms in the Free Energy from Adaptive Reaction Coordinate Forces (FEARCF) library. The library can be interfaced with classical molecular dynamics as well as quantum molecular dynamics packages. Multidimensional interdependent reaction coordinates are constructed to produce complex free energy hypersurfaces. The reaction coordinates are graphed from atomic and molecular components to define points, distances, vectors, angles, planes and combinations thereof. The resultant free energy surfaces that are a function of the distance, angles, planes, and so on, can represent molecular mechanisms in reduced dimensions from the component atomic Cartesian coordinate degrees of freedom. The FEARCF library can be interfaced with any molecular package. Here, we demonstrate the link to NWChem to compute a hyperdimensional DFT (aug-cc-pVDZ basis set and X3LYP exchange correlation functionals) free energy space of a water dimer. Analysis of the water dimer free energy hypervolume reveals that while the chain and cyclic hydrogen bonding configurations are located in stable minimum energy wells, the bifurcated hydrogen bond configuration is a gateway to instability and dimer dissociation.},
  archive      = {J_JCC},
  author       = {Tomás Bruce-Chwatt and Kevin J. Naidoo},
  doi          = {10.1002/jcc.26982},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1802-1813},
  shortjournal = {J. Comput. Chem.},
  title        = {Molecular mechanisms from reaction coordinate graph enabled multidimensional free energies illustrated on water dimer hydrogen bonding},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Investigation of glucose electrooxidation mechanism over
n-modified metal-doped graphene electrode by density functional theory
approach. <em>JCC</em>, <em>43</em>(26), 1793–1801. (<a
href="https://doi.org/10.1002/jcc.26981">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this work, various precious and non-precious metals reported in the literature as the most effective catalysts for glucose electrooxidation reaction were investigated by the density functional theory (DFT) approach in order to reveal the mechanisms taking place over the catalysts in the fuel cell. The use of a single-atom catalyst model was adopted by insertion of one Au, Cu, Ni, Pd, Pt, and Zn metal atom on the pyridinic N atoms doped graphene surface (NG). β form ofd-glucose in alkaline solution was used to determine the reaction mechanism and intermediates that formed during the reaction. DFT results showed that the desired glucono-lactone was formed on the Cu-3NG electrode in a single-step reaction pathway whereas it was produced via different two-step pathways on the Au and Pt-3NG electrodes. Although the interaction of glucose with Ni, Pd, and Zn-doped surfaces resulted in the deprotonation of the molecule, lactone product formation did not occur on these electrode surfaces. When the calculation results are evaluated in terms of energy content and product formation, it can be concluded that Cu, Pt, and especially Au doped graphene catalysts are effective for direct glucose oxidation in fuel cells reactor.},
  archive      = {J_JCC},
  author       = {Derya Düzenli and Isik Onal and Ilker Tezsevin},
  doi          = {10.1002/jcc.26981},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1793-1801},
  shortjournal = {J. Comput. Chem.},
  title        = {Investigation of glucose electrooxidation mechanism over N-modified metal-doped graphene electrode by density functional theory approach},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The reactivity of the trapping reaction of the
benzene-bridged boron/phosphorus-based frustrated lewis pair with
difluorocarbene and its group 14 analogs: A theoretical investigation.
<em>JCC</em>, <em>43</em>(26), 1783–1792. (<a
href="https://doi.org/10.1002/jcc.26980">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The trapping reactions of carbene analogsG14F2(G14 = group 14 element) by the benzene-bridgedB/P-Reafrustrated Lewis pair (FLPs) molecule are studied using density functional theory (B3LYP-D3(BJ)/def2-TZVP). Our theoretical investigations predict that only theCF2intermediate rather than other heavy carbene analogs can be trapped by theB/P-ReaFLP-type molecule. Energy decomposition analysis-natural orbitals for chemical valence (EDA-NOCV) analyses indicate that the bonding nature of theG14F2catching reactions by theB/P-ReaFLP-type molecule is a donor–acceptor (singlet–singlet) interaction rather than an electron-sharing (triplet–triplet) interaction. Moreover, EDA-NOCV and frontier molecular orbital (FMO) theory findings strongly suggest that the lone pair (LP) (P) → vacant p–π-orbital (G14F2) interaction rather than the empty σ-orbital (B) ← sp2-σ-orbital (G14F2) interaction plays a predominant role in establishing its bonding condition during theG14F2trapping reaction with theB/P-ReaFLP-associated molecule. Our activation strain model findings reveal that the atomic radius of the G14 element ofG14F2plays a key role in determining the activation barrier of theG14F2trapping reactions by the benzene-bridgedB/P-ReaFLP. The valence bond state correlation diagram (VBSCD) model developed by Shaik is used to rationalize the calculated results. The VBSCD findings demonstrate that in the present trapping reactions, the singlet triplet splitting ofG14F2plays a significant role in influencing its reaction barrier and reaction enthalpy. Our theoretical results demonstrate that the relationship between the geometrical parameters of the transition states and the corresponding reaction free energy barriers agrees well with the findings based on the Hammond postulate.},
  archive      = {J_JCC},
  author       = {Zheng-Feng Zhang and Ming-Der Su},
  doi          = {10.1002/jcc.26980},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1783-1792},
  shortjournal = {J. Comput. Chem.},
  title        = {The reactivity of the trapping reaction of the benzene-bridged boron/phosphorus-based frustrated lewis pair with difluorocarbene and its group 14 analogs: A theoretical investigation},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Predicting antibiotic resistance in complex protein targets
using alchemical free energy methods. <em>JCC</em>, <em>43</em>(26),
1771–1782. (<a href="https://doi.org/10.1002/jcc.26979">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug resistantMycobacterium tuberculosis, which mostly results from single nucleotide polymorphisms in antibiotic target genes, poses a major threat to tuberculosis treatment outcomes. Relative binding free energy (RBFE) calculations can rapidly predict the effects of mutations, but this approach has not been tested on large, complex proteins. We use RBFE calculations to predict the effects ofM. tuberculosisRNA polymerase and DNA gyrase mutations on rifampicin and moxifloxacin susceptibility respectively. These mutations encompass a range of amino acid substitutions with known effects and include large steric perturbations and charged moieties. We find that moderate numbers (n =3–15) of short RBFE calculations can predict resistance in cases where the mutation results in a large change in the binding free energy. We show that the method lacks discrimination in cases with either a small change in energy or that involve charged amino acids, and we investigate how these calculation errors may be decreased.},
  archive      = {J_JCC},
  author       = {Alice E. Brankin and Philip W. Fowler},
  doi          = {10.1002/jcc.26979},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1771-1782},
  shortjournal = {J. Comput. Chem.},
  title        = {Predicting antibiotic resistance in complex protein targets using alchemical free energy methods},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Hydrogen storage capacity of Be2(NLi)2 cluster with
ultra-short beryllium–beryllium distance. <em>JCC</em>, <em>43</em>(26),
1764–1770. (<a href="https://doi.org/10.1002/jcc.26976">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum chemical calculations have been carried out to investigate the hydrogen storage capacity of Be2(NLi)2cluster. Calculations reveal that the cluster can take up to eight H2molecules reaching a maximum gravimetric density of 21.04 wt\%. Six H2molecules bind at the Li atoms and two H2bind at the Be atoms with moderate binding energy which is required for reversible storage of H2. Symmetry-adapted perturbation analysis reveals the significant contribution of electrostatic and induction and very minor contribution of dispersion toward the total interaction energy. Molecular dynamics simulations reveal that the H2molecules are strongly bound at 77 K and get slowly released at elevated temperatures.},
  archive      = {J_JCC},
  author       = {Kangkan Sarmah and Amlan J. Kalita and Dimpul Konwar and Ankur K. Guha},
  doi          = {10.1002/jcc.26976},
  journal      = {Journal of Computational Chemistry},
  number       = {26},
  pages        = {1764-1770},
  shortjournal = {J. Comput. Chem.},
  title        = {Hydrogen storage capacity of Be2(NLi)2 cluster with ultra-short beryllium–beryllium distance},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Ppdx: Automated modeling of protein–protein interaction
descriptors for use with machine learning. <em>JCC</em>,
<em>43</em>(25), 1747–1757. (<a
href="https://doi.org/10.1002/jcc.26974">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This paper describes ppdx, a python workflow tool that combines protein sequence alignment, homology modeling, and structural refinement, to compute a broad array of descriptors for characterizing protein–protein interactions. The descriptors can be used to predict various properties of interest, such as protein–protein binding affinities, or inhibitory concentrations (IC50), using approaches that range from simple regression to more complex machine learning models. The software is highly modular. It supports different protocols for generating structures, and 95 descriptors can be currently computed. More protocols and descriptors can be easily added. The implementation is highly parallel and can fully exploit the available cores in a single workstation, or multiple nodes on a supercomputer, allowing many systems to be analyzed simultaneously. As an illustrative application, ppdx is used to parametrize a model that predicts the IC50of a set of antigens and a class of antibodies directed to the influenza hemagglutinin stalk.},
  archive      = {J_JCC},
  author       = {Simone Conti and Victor Ovchinnikov and Martin Karplus},
  doi          = {10.1002/jcc.26974},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1747-1757},
  shortjournal = {J. Comput. Chem.},
  title        = {Ppdx: Automated modeling of protein–protein interaction descriptors for use with machine learning},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CO bonding in hexa- and pentacoordinate carboxy-neuroglobin:
A quantum mechanics/molecular mechanics and local vibrational mode
study. <em>JCC</em>, <em>43</em>(25), 1725–1746. (<a
href="https://doi.org/10.1002/jcc.26973">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a comprehensive investigation on the different role of CO in carboxyneuroglobin (1) as ligand of the heme group in the active site forming a bond with the heme iron and (2) dissociated from the heme group but still trapped inside the active site, focusing on two specific orientations, one with CO perpendicular to the plane defined by the distal histidine of the enzyme (form A) and one with CO located parallel to that plane (form B). Our study includes wild type carboxy-neuroglobin and nine known protein mutations. Considering that the distal histidine interacting with the heme group can adapt two different tautomeric forms and the two possible orientations of the dissociated CO, a total of 36 protein systems were analyzed in this study. Fully optimized geometries and vibrational frequencies were calculated at the QM/MM level, followed by the local mode analysis, to decode CO bond properties. The intrinsic bond strengths derived from the local mode analysis, complemented with NBO and QTAIM data, reveal that the strength of the CO bond, in the hexacoordinate (where CO is a ligand of the heme group) and pentacoordinate (where CO is dissociated from the heme group) scenarios, is dominated by through bond and through space charge transfer between CO and Fe, fine-tuned by electrostatic and dispersion interactions with the side chain amino acids in the distal heme pocket. Suggestions are made as to advise on how protein modifications can influence the molecular properties of the coordinated or dissociated CO, which could serve the fine-tuning of existing and the design of new neuroglobin models with specific FeC and CO bond strengths.},
  archive      = {J_JCC},
  author       = {Marek Freindorf and Alexis Antoinette Ann Delgado and Elfi Kraka},
  doi          = {10.1002/jcc.26973},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1725-1746},
  shortjournal = {J. Comput. Chem.},
  title        = {CO bonding in hexa- and pentacoordinate carboxy-neuroglobin: A quantum mechanics/molecular mechanics and local vibrational mode study},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Stabilization of zn group dimers: A theoretical study.
<em>JCC</em>, <em>43</em>(25), 1719–1724. (<a
href="https://doi.org/10.1002/jcc.26972">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantum chemical calculations have been carried out to investigate the possibility of Group 12 (Zn, Cd, Hg) dimer formation supported by the B3ring. All these complexes have significantly shorter M-M (M = Zn, Cd, Hg) distances compared to their bare dimers, M2. The staggered and eclipsed forms of these complexes were found to be nearly degenerate. They have significant bond strengths as revealed from the significantly higher values of Wiberg bond indices as well as bond dissociation energies. The topological feature of electron density reveals significant covalency in the M-M interaction which has also been confirmed from natural orbital for chemical valence analysis coupled with extended transition state (ETS-NOCV) calculations.},
  archive      = {J_JCC},
  author       = {Ritam R. Borah and Rinu P. Deka},
  doi          = {10.1002/jcc.26972},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1719-1724},
  shortjournal = {J. Comput. Chem.},
  title        = {Stabilization of zn group dimers: A theoretical study},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Nonlinear optical properties DFT calculations of
polyacethylene and copolymers models substituted with aldimines
chromophores as side chains. <em>JCC</em>, <em>43</em>(25), 1701–1718.
(<a href="https://doi.org/10.1002/jcc.26971">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Aldimine derivatives chromophores grafted on polyacetylene oligomers were first designed to investigate the nonlinear optical (NLO) response of the resulting materials using CAMB3LYP method. The effects of different factors such as the chain length separating the substitutions as well as the configurations and the orientations of these latter, were examined and discussed. In a second part of this paper, NLO responses, in particular the static hyperpolarizabilities of polyvinyl oligomers substituted by aldimine chromophores via carboxylic groups and by pyrrolidone groups were calculated for different configurations. The stability and the hydrogen bonding in each oligomer were also discussed.},
  archive      = {J_JCC},
  author       = {Fatima-Zohra Benchehaima and Michael Springborg and Majda Sekkal Rahal},
  doi          = {10.1002/jcc.26971},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1701-1718},
  shortjournal = {J. Comput. Chem.},
  title        = {Nonlinear optical properties DFT calculations of polyacethylene and copolymers models substituted with aldimines chromophores as side chains},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Automatic generation of complementary auxiliary basis sets
for explicitly correlated methods. <em>JCC</em>, <em>43</em>(25),
1690–1700. (<a href="https://doi.org/10.1002/jcc.26970">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Explicitly correlated calculations, aside from the orbital basis set, typically require three auxiliary basis sets: Coulomb-exchange fitting (JK), resolution of the identity MP2 (RI-MP2), and complementary auxiliary basis set (CABS). If unavailable for the orbital basis set and chemical elements of interest, the first two can be auto-generated on the fly using existing algorithms, but not the third. In this paper, we present a quite simple algorithm named autoCABS; a Python implementation under a free software license is offered at Github. For the cc-pVnZ-F12 (n= D,T,Q,5), the W4-08 thermochemical benchmark, and the HFREQ2014 set of harmonic frequencies, we demonstrate that autoCABS-generated CABS basis sets are comparable in quality to purpose-optimized OptRI basis sets from the literature, and that the quality difference becomes entirely negligible asnincreases.},
  archive      = {J_JCC},
  author       = {Emmanouil Semidalas and Jan M. L. Martin},
  doi          = {10.1002/jcc.26970},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1690-1700},
  shortjournal = {J. Comput. Chem.},
  title        = {Automatic generation of complementary auxiliary basis sets for explicitly correlated methods},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The numerical evaluation of slater integrals on graphics
processing units. <em>JCC</em>, <em>43</em>(25), 1680–1689. (<a
href="https://doi.org/10.1002/jcc.26968">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This article presents SlaterGPU, a graphics processing unit (GPU) accelerated library that uses OpenACC to numerically compute Slater-type orbital (STO) integrals. The electron repulsion integrals (ERI) are computed under the RI approximation using the Coulomb potential of the Slater basis function. To fully realize the performance capabilities of modern GPUs, the Slater integrals are evaluated in mixed-precision, resulting in speedups for the ERIs of over 80×. Parallelization on multiple GPUs allows for integral throughput of over 3 million integrals per second. This places STO integral throughput within reach of single-threaded, conventional Gaussian integration schemes. To test the quality of the integrals, the fluorine exchange reaction barrier in fluoromethane was computed using heat-bath configuration interaction (HBCI). In addition, the singlet-triplet gap of cyclobutadiene was examined using HBCI in a triple-ζ, polarized basis set. These benchmarks demonstrate the library&#39;s ability to generate the full set of integrals necessary for configuration interaction with up to6hfunctions in the auxiliary basis.},
  archive      = {J_JCC},
  author       = {Duy-Khoi Dang and Leighton W. Wilson and Paul M. Zimmerman},
  doi          = {10.1002/jcc.26968},
  journal      = {Journal of Computational Chemistry},
  number       = {25},
  pages        = {1680-1689},
  shortjournal = {J. Comput. Chem.},
  title        = {The numerical evaluation of slater integrals on graphics processing units},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Automated input structure generation for single-ended
reaction path optimizations. <em>JCC</em>, <em>43</em>(24), 1662–1674.
(<a href="https://doi.org/10.1002/jcc.26969">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The exploration of a reaction network requires highly automated workflows to avoid error-prone and time-consuming manual steps. In this respect, a major bottleneck is the search for transition-state (TS) structures, which frequently fails and, therefore, makes (manual) revision necessary. In this work, we present a technique for obtaining suitable input structures for automated TS searches based on single-ended reaction path optimization algorithms, which makes subsequent TS searches via this method significantly more robust. First, possible input structures are generated based on the spatial alignment of the reactants. The appropriate orientation of reacting groups is achieved via stepwise rotations along selected torsional degrees of freedom. Second, a ranking of the obtained structures is performed according to selected geometric criteria. The main goals are to properly align the reactive atoms, to avoid hindrance within the reaction channel and to resolve steric clashes between the reactants. The developed procedure has been carefully tested on a variety of examples and provides suitable input structures for TS searches within seconds. The method is in daily use in an industrial setting.},
  archive      = {J_JCC},
  author       = {Julian Geiger and Volker Settels and Peter Deglmann and Ansgar Schäfer and Maike Bergeler},
  doi          = {10.1002/jcc.26969},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1662-1674},
  shortjournal = {J. Comput. Chem.},
  title        = {Automated input structure generation for single-ended reaction path optimizations},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Conformational distributions of helical perfluoroalkyl
substances and impacts on stability. <em>JCC</em>, <em>43</em>(24),
1656–1661. (<a href="https://doi.org/10.1002/jcc.26967">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Per- and polyfluoroalkyl substances (PFAS) have been widely used the past 70 years in numerous applications due to their chemical and thermal stability. Due to their robust stability, they are environmentally recalcitrant which made them one of the most persistent environmental contaminants. In addition to strong CF bond strength, oleophobicity, hydrophobicity, and high reduction–oxidation (redox) potential of PFAS has led to their inefficient degradation by traditional means. A characteristic of their structure is also their preference to adopt helical conformations along the carbon backbone, contrary to their hydrocarbon analogues. This work investigates the helical nature of perfluoroalkanes through their conformational distributions, especially as a benchmark for determining the impact of polar head groups, heteroatoms, and radical center on helical conformations. Since structure governs reactivity and molecular properties, it is important to assess if minor chemical perturbations in the structure will lead to changes in the conformations. Based on density functional theory calculations and comprehensive conformational distributions, it was concluded that the helicity is a local structural property which changes significantly with the presence of heteroatoms in the perfluoroalkyl chain as well as with the presence of radical centers.},
  archive      = {J_JCC},
  author       = {Maleigh Mifkovic and Daniel J. Van Hoomissen and Shubham Vyas},
  doi          = {10.1002/jcc.26967},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1656-1661},
  shortjournal = {J. Comput. Chem.},
  title        = {Conformational distributions of helical perfluoroalkyl substances and impacts on stability},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Automatized protocol and interface to simulate QM/MM
time-resolved transient absorption at TD-DFT level with COBRAMM.
<em>JCC</em>, <em>43</em>(24), 1641–1655. (<a
href="https://doi.org/10.1002/jcc.26966">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present a series of new implementations that we recently introduced in COBRAMM, the open-source academic software developed in our group. The goal of these implementations is to offer an automatized workflow and interface to simulate time-resolved transient absorption (TA) spectra of medium-to-big chromophore embedded in a complex environment. Therefore, the excited states absorption and the stimulated emission are simulated along nonadiabatic dynamics performed with trajectory surface hopping. The possibility of treating systems from medium to big size is given by the use of time-dependent density functional theory (TD-DFT) and the presence of the environment is taken into account employing a hybrid quantum mechanics/molecular mechanics (QM/MM) scheme. The full implementation includes a series of auxiliary scripts to properly setup the QM/MM system, the calculation of the wavefunction overlap along the dynamics for the propagation, the evaluation of the transition dipole moment at linear response TD-DFT level, and scripts to setup, run and analyze the TA from an ensemble of trajectories. Altogether, we believe that our implementation will open the door to the easily simulate the time-resolved TA of systems so far computationally inaccessible.},
  archive      = {J_JCC},
  author       = {Davide Avagliano and Matteo Bonfanti and Artur Nenov and Marco Garavelli},
  doi          = {10.1002/jcc.26966},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1641-1655},
  shortjournal = {J. Comput. Chem.},
  title        = {Automatized protocol and interface to simulate QM/MM time-resolved transient absorption at TD-DFT level with COBRAMM},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Explicit solvent repulsive scaling replica exchange
molecular dynamics (RS-REMD) in molecular modeling of
protein-glycosaminoglycan complexes. <em>JCC</em>, <em>43</em>(24),
1633–1640. (<a href="https://doi.org/10.1002/jcc.26965">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Glycosaminoglcyans (GAGs), linear anionic periodic polysaccharides, are crucial for many biologically relevant functions in the extracellular matrix. By interacting with proteins GAGs mediate processes such as cancer development, cell proliferation and the onset of neurodegenerative diseases. Despite this eminent importance of GAGs, they still represent a limited focus for the computational community in comparison to other classes of biomolecules. Therefore, there is a lack of modeling tools designed specifically for docking GAGs. One has to rely on existing docking software developed mostly for small drug molecules substantially differing from GAGs in their basic physico-chemical properties. In this study, we present an updated protocol for docking GAGs based on the Repulsive Scaling Replica Exchange Molecular Dynamics (RS-REMD) that includes explicit solvent description. The use of this water model improved docking performance both in terms of its accuracy and speed. This method represents a significant computational progress in GAG-related research.},
  archive      = {J_JCC},
  author       = {Mateusz Marcisz and Margrethe Gaardløs and Krzysztof K. Bojarski and Till Siebenmorgen and Martin Zacharias and Sergey A. Samsonov},
  doi          = {10.1002/jcc.26965},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1633-1640},
  shortjournal = {J. Comput. Chem.},
  title        = {Explicit solvent repulsive scaling replica exchange molecular dynamics (RS-REMD) in molecular modeling of protein-glycosaminoglycan complexes},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Physical insight into the entropy-driven ion association.
<em>JCC</em>, <em>43</em>(24), 1621–1632. (<a
href="https://doi.org/10.1002/jcc.26963">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ion association is widely believed to be dominated by the favorable entropy change arising from the release of water molecules from ion hydration shells. However, no direct thermodynamic evidence exists to validate the reliability and suitability of this view. Herein, we employ complicated free energy calculations to rigorously split the free energy including its entropic and enthalpic components into the water-induced contributions and ion–ion interaction terms for several ion pairs from monatomic to polyatomic ions, spanning the size range from small kosmotropes to large chaotropes (Na+, Cs+, Ca2+, F−, I−, CO32−, and HPO42−). Our results successfully reveal that though ion associations are indeed determined by a delicate balance between the favorable entropy variation and the repulsive enthalpy change, the entropy gain dominated by the solvent occurs only for the monatomic ion pairing. The water-induced entropic contribution significantly goes against the ion pairing between polyatomic anion and cation, which is, alternatively, dominated by the favorable entropy from the ion–ion interaction term, due to the configurational arrangement of polyatomic anions involved in ion association. The structural and dynamic analysis demonstrates that the entropy penalty from the water phase is primarily ascribed to the enhanced stability of water molecules around the cation imposed by the incoming anion. Our study successfully provides a fundamental understanding of water-mediated ion associations and highlights disparate lengthscale dependencies of the dehydration thermodynamics on the specific types of ions.},
  archive      = {J_JCC},
  author       = {Xiao Yang and Mingyu Ji and Cong Zhang and Xiaoning Yang and Zhijun Xu},
  doi          = {10.1002/jcc.26963},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1621-1632},
  shortjournal = {J. Comput. Chem.},
  title        = {Physical insight into the entropy-driven ion association},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Conformational preferences of endohedral metallofullerenes
on ag, au, and MgO surfaces: Theoretical studies. <em>JCC</em>,
<em>43</em>(24), 1614–1620. (<a
href="https://doi.org/10.1002/jcc.26962">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this report, we study the ordering of C60, Sc3N@C80, and Dy2ScN@C80molecules on different metallic and dielectric surfaces such as Ag(100), Au(111), and MgO(100). By using DFT techniques, we can classify different types of cage-to-surface arrangements and their relative energies. Using a proposed homogenous sampling of the conformational space for the M3N cluster, we determine a potential energy map that is capable of providing a structural distribution for a given energy window. We find that Coulomb interaction is a dominant force that governs the system&#39;s stability and order. However, a deep analysis of the charge density rearrangements reveals that even though the integral charges may be considered as a qualitative control parameter, it fails to provide quantitative data due to the importance of spatial characteristics of charge densities.},
  archive      = {J_JCC},
  author       = {Vasilii Dubrovin and Stanislav M. Avdoshenko},
  doi          = {10.1002/jcc.26962},
  journal      = {Journal of Computational Chemistry},
  number       = {24},
  pages        = {1614-1620},
  shortjournal = {J. Comput. Chem.},
  title        = {Conformational preferences of endohedral metallofullerenes on ag, au, and MgO surfaces: Theoretical studies},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Molecular tailoring approach as tool for revealing
resonance-assisted hydrogen bond: Case study of z-pyrrolylenones with
the n?h⋯oс intramolecular hydrogen bond. <em>JCC</em>, <em>43</em>(23),
1596–1607. (<a href="https://doi.org/10.1002/jcc.26960">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Both the experimental and calculated data reveal that a strong NH⋯OС intramolecular hydrogen bond closing the seven-membered quasi-cycle is formed in theZ-isomers of pyrrolylenones. Comparison of the NH⋯OС intramolecular hydrogen bonds energies in the pyrrolylenones, estimated via the molecular tailoring approach, with the similar data for reference malonaldehydes shows that the resonance-assisted hydrogen bonding occurs in both cases, the hydrogen bond energy being varied mainly within 10–20 kcal/mol. The combined application of function-based and molecular tailoring approaches makes it possible to decompose the NH⋯OС total hydrogen bond energy in the pyrrolylenones into the π- and σ-components. It is established that the contribution of the π-component to the total N(O)H⋯OС hydrogen bond energy in the pyrrolylenones and malonaldehydes is almost the same (6–7 kcal/mol). Comparison of the π-contribution to the total energy of the resonance-assisted hydrogen bonding in theZ-isomer of pyrrolylenones with the energy of the push-pull effect in theE-isomer of pyrrolylenones reveals that the resonance contribution to the total energy of the resonance-assisted hydrogen bond in the former significantly enhances with reference to the net resonance energy in the latter. The appearance of the resonance-assisted hydrogen bond in the pyrrolylenones is possible due to the participation in the interaction of 10 or 14 π-electrons satisfying the Hückel aromaticity rule.},
  archive      = {J_JCC},
  author       = {Andrei V. Afonin and Danuta Rusinska-Roszak},
  doi          = {10.1002/jcc.26960},
  journal      = {Journal of Computational Chemistry},
  number       = {23},
  pages        = {1596-1607},
  shortjournal = {J. Comput. Chem.},
  title        = {Molecular tailoring approach as tool for revealing resonance-assisted hydrogen bond: Case study of Z-pyrrolylenones with the N?H⋯OС intramolecular hydrogen bond},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Imidazole and nitroimidazole derivatives as NADH-fumarate
reductase inhibitors: Density functional theory studies, homology
modeling, and molecular docking. <em>JCC</em>, <em>43</em>(23),
1573–1595. (<a href="https://doi.org/10.1002/jcc.26959">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chagas disease is caused byTrypanosoma cruzi. Benznidazole and nifurtimox are drugs used for its therapy; nevertheless, they have collateral effects. NADH-fumarate (FUM) reductase is a potential pharmacological target since it is essential for survival of parasite and is not found in humans. The objectives are to design and characterize the electronic structure of imidazole and nitroimidazole derivatives at DFT-M06-2X level in aqueous solution; also, to model the NADH-FUM reductase and analyze its intermolecular interactions by molecular docking. Quantum-chemical descriptors allowed to select the molecules with the best physicochemical properties and lowest toxicity. A high-quality three-dimensional structure of NADH-FUM reductase was obtained by homology modeling. Water molecules do not have influence in the interaction between FUM and NADH-FUM reductase. The main hydrogen-binding interactions for FUM were identified in NADH, Lys172, and Arg89; while hydrophobic interactions in Phe479, Thr174, Met63. The molecules S3-8, S2-8, and S1-8 could be inhibitors of NADH-FUM reductase.},
  archive      = {J_JCC},
  author       = {Linda Campos-Fernández and Rocío Ortiz-Muñiz and Edith Cortés-Barberena and Sergio Mares-Sámano and Ramón Garduño-Juárez and Catalina Soriano-Correa},
  doi          = {10.1002/jcc.26959},
  journal      = {Journal of Computational Chemistry},
  number       = {23},
  pages        = {1573-1595},
  shortjournal = {J. Comput. Chem.},
  title        = {Imidazole and nitroimidazole derivatives as NADH-fumarate reductase inhibitors: Density functional theory studies, homology modeling, and molecular docking},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Molecular properties and tautomeric equilibria of isolated
flavins. <em>JCC</em>, <em>43</em>(23), 1561–1572. (<a
href="https://doi.org/10.1002/jcc.26957">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Flavins are employed as redox cofactors and chromophores in a plethora of flavoenzymes. Their versatility is an outcome of intrinsic molecular properties of the isoalloxazine ring modulated by the protein scaffold and surrounding solvent. Thus, an investigation of isolated flavins with high-level electronic-structure methods and with error assessment of the calculated properties will contribute to building better models of flavin reactivity. Here, we benchmarked ground-state properties such as electron affinity, gas-phase basicity, dipole moment, torsion energy, and tautomer stability for lumiflavins in all biologically relevant oxidation and charge states. Overall, multiconfigurational effects are small and chemical accuracy is achieved by coupled-cluster treatments of energetic properties. Augmented basis sets and extrapolations to the complete basis-set limit are necessary for consistent agreement with experimental energetics. Among DFT functionals tested, M06-2X shows the best performance for most properties, except gas-phase basicity, in which M06 and CAM-B3LYP perform better. Moreover, dipole moments of radical flavins show large deviations for all functionals studied. Tautomers with noncanonical protonation states are significantly populated at normal temperatures, adding to the complexity of modeling flavins. These results will guide future computational studies of flavoproteins and flavin chemistry by indicating the limitations of electronic-structure methodologies and the contributions of multiple tautomeric states.},
  archive      = {J_JCC},
  author       = {Felipe Curtolo and Guilherme M. Arantes},
  doi          = {10.1002/jcc.26957},
  journal      = {Journal of Computational Chemistry},
  number       = {23},
  pages        = {1561-1572},
  shortjournal = {J. Comput. Chem.},
  title        = {Molecular properties and tautomeric equilibria of isolated flavins},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Uncovering the hidden reactivity of benzyne/aryne precursors
utilized under milder condition: Bonding and stability studies by
EDA-NOCV analyses. <em>JCC</em>, <em>43</em>(23), 1543–1560. (<a
href="https://doi.org/10.1002/jcc.26956">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Arenes [C6H3R(TMS)(OTf); also called benzyne/aryne precursors] containing inter-related leaving groups Me3Si (TMS) and CF3SO3(OTf) on the adjacent positions (1,2-position) are generally converted to their corresponding aryne-intermediates via the addition of fluoride anion (F−) and subsequent elimination of TMS and OTf groups. This reaction is believed to proceed via the formation of an anionic intermediate [C6H4(TMS-F)(OTf)]−. The EDA-NOCV analysis (EDA-NOCV = energy decomposition analysis-natural orbital for chemical valence) of over 35 such precursors of varied types have been reported to reveal bonding and stability of CArSi and COTf bonds. EDA-NOCV showed that the nature of the CArSi bond of C6H3R(TMS)(OTf) can be expressed as both dative and electron sharing [CArSi, CAr→Si]. The CArOTf bond, on the other hand, can be described explicitly as dative [CAr←OTf]. The nature of CArSi bond of [C6H4(TMS-F)(OTf)]−exclusively changes to covalent dative σ-bond CAr→S(Me)3F on the attachment of F−to the TMS group of C6H4(TMS)(OTf). Introduction of σ-electron withdrawing group (like OMe, NMe2, and NO2) to the ortho-position of the TMS group of functionalized arynes C6H3R(TMS)(OTf) prefer to have a covalent dative σ-bond (CAr→Si) over an electron-sharing covalent σ-bond (CArSi). If this σ-electron withdrawing group is shifted from ortho-position to meta- and para-positions, then the preference for a dative bond decreases significantly, implying that the electronic effect on the nature of chemical bonds affects through bond paths. This effect dies with distance, similar to the well-known inductive effect.},
  archive      = {J_JCC},
  author       = {Sai Manoj N. V. T. Gorantla and Kartik Chandra Mondal},
  doi          = {10.1002/jcc.26956},
  journal      = {Journal of Computational Chemistry},
  number       = {23},
  pages        = {1543-1560},
  shortjournal = {J. Comput. Chem.},
  title        = {Uncovering the hidden reactivity of benzyne/aryne precursors utilized under milder condition: Bonding and stability studies by EDA-NOCV analyses},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Automation of AMOEBA polarizable force field for small
molecules: Poltype 2. <em>JCC</em>, <em>43</em>(23), 1530–1542. (<a
href="https://doi.org/10.1002/jcc.26954">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A next-generation protocol (Poltype 2) has been developed which automatically generates AMOEBA polarizable force field parameters for small molecules. Both features and computational efficiency have been drastically improved. Notable advances include improved database transferability using SMILES, robust torsion fitting, non-aromatic ring torsion parameterization, coupled torsion-torsion parameterization, Van der Waals parameter refinement using ab initio dimer data and an intelligent fragmentation scheme that produces parameters with dramatically reduced ab initio computational cost. Additional improvements include better local frame assignment for atomic multipoles, automated formal charge assignment, Zwitterion detection, smart memory resource defaults, parallelized fragment job submission, incorporation of Psi4 quantum package, ab initio error handling, ionization state enumeration, hydration free energy prediction and binding free energy prediction. For validation, we have applied Poltype 2 to ~1000 FDA approved drug molecules from DrugBank. The ab initio molecular dipole moments and electrostatic potential values were compared with Poltype 2 derived AMOEBA counterparts. Parameters were further substantiated by calculating hydration free energy (HFE) on 40 small organic molecules and were compared with experimental data, resulting in an RMSE error of 0.59 kcal/mol. The torsion database has expanded to include 3543 fragments derived from FDA approved drugs. Poltype 2 provides a convenient utility for applications including binding free energy prediction for computational drug discovery. Further improvement will focus on automated parameter refinement by experimental liquid properties, expansion of the Van der Waals parameter database and automated parametrization of modified bio-fragments such as amino and nucleic acids.},
  archive      = {J_JCC},
  author       = {Brandon Walker and Chengwen Liu and Elizabeth Wait and Pengyu Ren},
  doi          = {10.1002/jcc.26954},
  journal      = {Journal of Computational Chemistry},
  number       = {23},
  pages        = {1530-1542},
  shortjournal = {J. Comput. Chem.},
  title        = {Automation of AMOEBA polarizable force field for small molecules: Poltype 2},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Automated reaction mechanisms and kinetics with the nudged
elastic band method-based AMK_mountain and its description of the
preliminary alkaline hydrolysis of nitrocellulose monomer. <em>JCC</em>,
<em>43</em>(22), 1513–1523. (<a
href="https://doi.org/10.1002/jcc.26891">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To improve the transition state (TS) search capability in complex chemical environments, AMK_Mountain is constructed based on the automated reaction mechanisms and kinetics (AutoMeKin) package. AMK_Mountain does not distinguish the reaction type of the TSs, which is beneficial to obtaining a more comprehensive reaction mechanism. In this study, the first step of the alkaline hydrolysis process of nitrocellulose monomer was adopted as the research object, and 730 possible initial configurations are constructed and 22 TSs pass high-level calculations. Energy difference and interaction region indicator reveal that the first step of alkaline hydrolysis is mainly the combination of nitrogen-containing functional groups at the positions α and β with hydroxide anions, followed by the formation of nitric acid and the further loss of protons to form nitrate. Overall, in combination with GFN2-xTB and ORCA, the AMK_Mountain technique provides a reliable method for the location of the TSs in complex environments.},
  archive      = {J_JCC},
  author       = {Guan Zhang and Jin Li and Xinxin Liang and Zongkuan Liu},
  doi          = {10.1002/jcc.26891},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1513-1523},
  shortjournal = {J. Comput. Chem.},
  title        = {Automated reaction mechanisms and kinetics with the nudged elastic band method-based AMK_Mountain and its description of the preliminary alkaline hydrolysis of nitrocellulose monomer},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Factor analysis of error in oxidation potential calculation:
A machine learning study. <em>JCC</em>, <em>43</em>(22), 1504–1512. (<a
href="https://doi.org/10.1002/jcc.26953">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The conductor-like polarizable continuum model (C-PCM), which is a low-cost solvation model, cannot treat characteristic interactions between the solvent and substructure(s) of the solute. Moreover, the error in a charged system is significant. Using machine learning, we clarified that the systematic error of the oxidation potential calculated by the G3B3/C-PCM was correlated with the molecular size of a solute. The G3B3/C-PCM overestimated the Gibbs oxidation energy by averaging 6.94 kcal/mol. According to the performance of related methods reported in previous studies, this error is mainly due to the solvation energy of the charged solute. Additionally, we succeeded in reducing the error to 2.27 kcal/mol (32\%)–3.2 kcal/mol (40\%) by correction based on the substructure information of the solute. To modify the C-PCM, effects that correlate with the molecular size of the solute in the charged system should be incorporated.},
  archive      = {J_JCC},
  author       = {Yuki Kanamaru and Toru Matsui},
  doi          = {10.1002/jcc.26953},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1504-1512},
  shortjournal = {J. Comput. Chem.},
  title        = {Factor analysis of error in oxidation potential calculation: A machine learning study},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Permutationally invariant polynomial representation of
polarizability tensor surfaces for linear regression analysis.
<em>JCC</em>, <em>43</em>(22), 1495–1503. (<a
href="https://doi.org/10.1002/jcc.26952">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A linearly parameterized functional form for a Cartesian representation of molecular dipole polarizability tensor surfaces (PTS) is described. The proposed expression for the PTS is a linearization of the recently reported power series ansatz of the original Applequist model, which by construction is non-linear in parameter space. This new approach possesses (i) a unique solution to the least-squares fitting problem; (ii) a low level of the computational complexity of the resulting linear regression procedure, comparable to those of the potential energy and dipole moment surfaces; and (iii) a competitive level of accuracy compared to the non-linear PTS model. Calculations of CH4PTS, with polarizabilities fitted to 9000 training set points with the energies up to 14,000 cm−1show an impressive level of accuracy of the linear PTS model obtained with ~1600 parameters: ~1\% versus 0.3\% RMSE for the non-linear vs. linear model on a test set of 1000 configurations.},
  archive      = {J_JCC},
  author       = {Oluwaseun Omodemi and Martina Kaledin and Alexey L. Kaledin},
  doi          = {10.1002/jcc.26952},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1495-1503},
  shortjournal = {J. Comput. Chem.},
  title        = {Permutationally invariant polynomial representation of polarizability tensor surfaces for linear regression analysis},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Calculation of absolute raman scattering cross-sections
using vibrational self-consistent field/vibrational configuration
interaction wave functions. <em>JCC</em>, <em>43</em>(22), 1484–1494.
(<a href="https://doi.org/10.1002/jcc.26951">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the present study, the differential scattering cross-sections, depolarization ratios and Raman shifts of small molecular systems are obtained from configuration iteration wave functions of vibrational self-consistent field (VSCF) states. The transition polarizabilities were modeled using the Placzek approximation, neglecting those contributions not arising from the electric dipole mechanism. This theoretical approach is considered a good approximation for samples that absorb in the UV range if the excitation radiation falls in the visible region, as is the case of the molecules selected for the present study, namely: water, methane, and acetylene. Potential energy and electronic polarizability surfaces are calculated by the CCSD(T) and CC3 methods with aug-cc-p(C)V(T,Q,5)Z basis sets. The vibrational Hamiltonian includes the vibrational angular momentum contribution of the Watson kinetic energy operator. As expected, due to the variational nature of the VSCF and vibrational configuration interaction (VCI) methods, the Raman transition wavenumbers are substantially improved over the harmonic predictions. Surprisingly, the scattering cross-sections obtained using the harmonic approximation or the VSCF method better agrees with the experimental values than those cross-sections predicted using VCI wave functions. The more significant deviations of the VCI results from the experimental reference may be related to the significant uncertainties of the measured cross-sections. Still, it may also indicate that the VCI Raman transition moments may require a more accurate description of the electronic polarizability surface. Finally, the depolarization ratios calculated for H2O and C2D2using harmonic and VCI wave functions have similar accuracy, whereas, for C2H2and C2HD, the VCI results are more accurate.},
  archive      = {J_JCC},
  author       = {Jhonatas R. Carvalho and Luciano N. Vidal},
  doi          = {10.1002/jcc.26951},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1484-1494},
  shortjournal = {J. Comput. Chem.},
  title        = {Calculation of absolute raman scattering cross-sections using vibrational self-consistent field/vibrational configuration interaction wave functions},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Boron based intramolecular heterocyclic frustrated lewis
pairs as organocatalysts for CO2 adsorption and activation.
<em>JCC</em>, <em>43</em>(22), 1474–1483. (<a
href="https://doi.org/10.1002/jcc.26949">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The massive increase in the amount of carbon dioxide (CO2) in the atmosphere has led to serious environmental problems. One of the best ways to tackle this problem is the CO2capture and its utilization as a C1 carbon source for the production of industrially valuable chemicals. But the thermodynamic stability of the CO2molecule poses a great challenge in its transformation. Since the last two decades, various metal-based and organic catalysts have been developed for the adsorption and activation of CO2. Among all the catalysts the Frustrated Lewis pairs (FLPs) have been shown great potential in CO2capture and conversion. Thus, in the present work, Intramolecular Frustrated Lewis pairs (IFLP) based on N-Heterocycles with boron group functionalization at the α-position to N has been theoretically investigated for CO2activation. Thorough orbital analysis has been carried out to investigate the reactivity of the proposed catalytic systems. The result shows that the considered IFLPs are capable of activating CO2with minimum energy requirements. The CO2activation energy range between 8 and 14 kcal/mol. The non-polar solvent was found to be the suitable medium for the reaction. Also, the reversibility of the adducts formed with the IFLPs can be controlled by appropriate substitution at B atom in the IFLPs.},
  archive      = {J_JCC},
  author       = {Mohmmad Faizan and Ravinder Pawar},
  doi          = {10.1002/jcc.26949},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1474-1483},
  shortjournal = {J. Comput. Chem.},
  title        = {Boron based intramolecular heterocyclic frustrated lewis pairs as organocatalysts for CO2 adsorption and activation},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Following the density evolution using real time density
functional theory and density based indexes: Application to model
push–pull molecules. <em>JCC</em>, <em>43</em>(22), 1464–1473. (<a
href="https://doi.org/10.1002/jcc.26932">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Considering as test case a family of organic rod like push–pull molecules, we derived and applied density based index enabling the description and diagnostic of the electronic density evolution in real time-time dependent density functional theory (RT-TDDFT) simulations. In particular, both the charge transfer (CT) distance and a diagnostic index, the DCTand MACRTrespectively, were computed on the fly from the density distribution obtained at a given time and the reference ground state density and their mean values were compared with what obtained at Linear Response-TDDFT level. Besides giving a way of analyzing the density redistribution occurring in time, these tools allowed to show how RT-TDDFT, which is definitely a powerful method to model the evolution of the density in CT or charge separation processes, can be affected by the same artifacts known for LR-TDDFT approaches and, particularly, to those related to the use of approximate exchange correlation functionals. The analysis here performed allowed to identify and discard on fly the electronic configurations corresponding to spurious situations.},
  archive      = {J_JCC},
  author       = {Feven Alemu Korsaye and Aurélien de la Lande and Ilaria Ciofini},
  doi          = {10.1002/jcc.26932},
  journal      = {Journal of Computational Chemistry},
  number       = {22},
  pages        = {1464-1473},
  shortjournal = {J. Comput. Chem.},
  title        = {Following the density evolution using real time density functional theory and density based indexes: Application to model push–pull molecules},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Cover image. <em>JCC</em>, <em>43</em>(21), i. (<a
href="https://doi.org/10.1002/jcc.26961">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Ilia A. Solov&#39;yov and Gennady Sushko and Ida Friis and Andrey V. Solov&#39;yov},
  doi          = {10.1002/jcc.26961},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {i},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Multiscale modeling of stochastic dynamics processes with
MBN explorer. <em>JCC</em>, <em>43</em>(21), 1442–1458. (<a
href="https://doi.org/10.1002/jcc.26948">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stochastic dynamics describes processes in complex systems having the probabilistic nature. They can involve very different dynamical systems and occur on very different temporal and spatial scale. This paper discusses the concept of stochastic dynamics and its implementation in the popular programMBN Explorer. Stochastic dynamics inMBN Explorerrelies on the Monte Carlo approach and permits simulations of physical, chemical, and biological processes. The paper presents the basic theoretical concepts underlying stochastic dynamics implementation and provides several examples highlighting its applicability to different systems, such as diffusing proteins seeking an anchor point on a cell membrane, deposition of nanoparticles on a surface leading to structures with fractal morphologies, and oscillations of compounds in an autocatalytic reaction. The chosen examples illustrate the diversity of applications that can be modeled by means of stochastic dynamics withMBN Explorer.},
  archive      = {J_JCC},
  author       = {Ilia A. Solov&#39;yov and Gennady Sushko and Ida Friis and Andrey V. Solov&#39;yov},
  doi          = {10.1002/jcc.26948},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {1442-1458},
  shortjournal = {J. Comput. Chem.},
  title        = {Multiscale modeling of stochastic dynamics processes with MBN explorer},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Tree parzen estimator for global geometry optimization: A
benchmark and database of experimental gas-phase structures of organic
molecules. <em>JCC</em>, <em>43</em>(21), 1434–1441. (<a
href="https://doi.org/10.1002/jcc.26947">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Finding global and local minima on the potential energy surface is a key task for most studies in computational chemistry. Having a set of possible conformations for chemical structures and their corresponding energies, one can judge their chemical activity, understand the mechanisms of reactions, describe the formation of metal-ligand and ligand-protein complexes, and so forth. Despite the fact that the interest in various minima search algorithms in computational chemistry arose a while ago (during the formation of this science), new methods are still emerging. These methods allow to perform conformational analysis and geometry optimization faster, more accurately, or for more specific tasks. This article presents the application of a novel global geometry optimization approach based on the Tree Parzen Estimator method. For benchmarking, a database of small organic molecule geometries in the global minimum conformation was created, as well as a software package to perform the tests.},
  archive      = {J_JCC},
  author       = {Nikolai Andreadi and Dmitry Zankov and Kirill Karpov and Artem Mitrofanov},
  doi          = {10.1002/jcc.26947},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {1434-1441},
  shortjournal = {J. Comput. Chem.},
  title        = {Tree parzen estimator for global geometry optimization: A benchmark and database of experimental gas-phase structures of organic molecules},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Dipolar 1,3-cycloaddition of thioformaldehyde s-methylide
(CH2SCH2) to ethylene and acetylene. A comparison with (valence)
isoelectronic o3, SO2, CH2OO and CH2SO. <em>JCC</em>, <em>43</em>(21),
1420–1433. (<a href="https://doi.org/10.1002/jcc.26946">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Methods rooted in the density functional theory and in the coupled cluster ansatz were employed to investigate the cycloaddition reactions to ethylene and acetylene of 1,3-dipolar species including ozone and the derivatives issued from replacement of the central oxygen atom by the valence-isoelectronic sulfur atom, and/or of one or both terminal oxygen atoms by the isoelectronic CH2group. This gives rise to five different 1,3-dipolar compounds, namely ozone itself (O3), sulfur dioxide (SO2), the simplest Criegee intermediate (CH2OO), sulfine (CH2SO), and thioformaldehydeS-methylide (CH2SCH2, TSM). The experimental and accurate theoretical data available for some of those molecules were employed to assess the accuracy of two last-generation composite methods employing conventional or explicitly correlated post-Hartree-Fock contributions (jun-Cheap and SVECV-f12, respectively), which were then applied to investigate the reactivity of TSM. The energy barriers provided by both composite methods are very close (the average values for the two composite methods are 7.1 and 8.3 kcal mol−1for the addition to ethylene and acetylene, respectively) and comparable to those ruling the corresponding additions of ozone (4.0 and 7.7 kcal mol−1, respectively). These and other evidences strongly suggest that, at least in the case of cycloadditions, the reactivity of TSM is similar to that of O3and very different from that of SO2.},
  archive      = {J_JCC},
  author       = {Zoi Salta and Mauricio Vega-Teijido and Aline Katz and Nicola Tasinato and Vincenzo Barone and Oscar N. Ventura},
  doi          = {10.1002/jcc.26946},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {1420-1433},
  shortjournal = {J. Comput. Chem.},
  title        = {Dipolar 1,3-cycloaddition of thioformaldehyde S-methylide (CH2SCH2) to ethylene and acetylene. a comparison with (valence) isoelectronic o3, SO2, CH2OO and CH2SO},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Assessment of density functional approximations for n2 and
CO2 physisorption on benzene and graphene. <em>JCC</em>,
<em>43</em>(21), 1403–1419. (<a
href="https://doi.org/10.1002/jcc.26945">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Experimental isotherms of N2and CO2on carbon-based porous materials and models of the physisorption of gases on surfaces are used to obtain the pore size distribution (PSD). An accurate modelization of the physisorption of N2and CO2on the surface of carbon-based porous materials is important to obtain accurate N2and CO2storage capacities and reliable PSDs. Physisorption depends on the dispersion interactions. High precision ab initio methods, such as CCSD(T), consider accurately the dispersion interactions, but they are computationally expensive. Double hybrid, hybrid and DFT-based methods are much less expensive. In the case of graphene, there are experimental data of the adsorption of N2and CO2on graphite that can be used to build the Steele interaction potential of these gases on graphene. The goal is to find out hybrid and/or DFT methods that are as accurate as the CCSD(T) on benzene and as accurate as the experimental results on graphene. Calculations of the interaction energy curves of N2and CO2on benzene and graphene have been carried out using the CCSD(T) method and several double hybrid, hybrid, and DFT methods that consider the dispersion interactions. The energy curves on benzene have been compared to the CCSD(T) and the energy curves on graphene have been compared with the Steele energy curves. The comparisons indicate that double hybrids with dispersion corrections andωB97 based DFT methods are accurate enough for benzene. For graphene, only the PBE-XDM functional has a good agreement with the Steele energy curves.},
  archive      = {J_JCC},
  author       = {Víctor M. Rayón and Iván Cabria},
  doi          = {10.1002/jcc.26945},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {1403-1419},
  shortjournal = {J. Comput. Chem.},
  title        = {Assessment of density functional approximations for n2 and CO2 physisorption on benzene and graphene},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Assessment of DLPNO-CCSD(t)-f12 and its use for the
formulation of the low-cost and reliable l-W1X composite method.
<em>JCC</em>, <em>43</em>(21), 1394–1402. (<a
href="https://doi.org/10.1002/jcc.26892">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the present study, we have investigated the performance of RIJCOSX DLPNO-CCSD(T)-F12 methods for a wide range of systems. Calculations with a high-accuracy option [“DefGrid3 RIJCOSX DLPNO-CCSD(T1)-F12”] extrapolated to the complete-basis-set limit using the maug-cc-pV[D+d,T+d]Z basis sets provides fairly good agreements with the canonical CCSD(T)/CBS reference for a diverse set of thermochemical and kinetic properties [with mean absolute deviations (MADs) of ~1–2 kJ mol−1except for atomization energies]. On the other hand, the low-cost “RIJCOSX DLPNO-CCSD(T)-F12D” option leads to substantial deviations for certain properties, notably atomization energies (MADs of up to tens of kJ mol−1). With the high-accuracy CBS approach, we have formulated the L-W1X method, which further includes a low-cost core–valence plus scalar-relativistic term. It shows generally good accuracy. For improved accuracies in specific cases, we advise replacing maug-cc-pV(n+d)Z with jun-cc-pV(n+d)Z for the calculation of electron affinities, and using well-constructed isodesmic-type reactions to obtain atomization energies. For medium-sized systems, DefGrid3 RIJCOSX DLPNO-CCSD(T1)-F12 calculations are several times faster than the corresponding canonical computation; the use of the local approximations (RIJCOSX and DLPNO) leads to a better scaling than that for the canonical calculation (from ~6–7 down to ~2–4 for our test systems). Thus, the DefGrid3 RIJCOSX DLPNO-CCSD(T1)-F12 method, and the L-W1X protocol that based on it, represent a useful means for obtaining accurate thermochemical quantities for larger systems.},
  archive      = {J_JCC},
  author       = {Bun Chan and Amir Karton},
  doi          = {10.1002/jcc.26892},
  journal      = {Journal of Computational Chemistry},
  number       = {21},
  pages        = {1394-1402},
  shortjournal = {J. Comput. Chem.},
  title        = {Assessment of DLPNO-CCSD(T)-f12 and its use for the formulation of the low-cost and reliable L-W1X composite method},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Towards a transferable nonelectrostatic model for continuum
solvation: The electrostatic and nonelectrostatic energy correction
model. <em>JCC</em>, <em>43</em>(20), 1372–1387. (<a
href="https://doi.org/10.1002/jcc.26944">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this work, we introduce an electrostatic and non-electrostatic (ENE) correction to the solvation energy based on the Solvent-Accessible Surface Area (SASA) of the solute and the solvent static dielectric constant. The proposed correction was developed for neutral solutes in non-aqueous solvents, considering three different implicit solvation models based on a Self-Consistent Reaction Field treatment of solute-solvent mutual polarization using an Apparent Surface Charge formalism, namely the Integral Equation Formalism of the Polarizable Continuum Model using a continuous surface charge scheme (PCM), the Solvation Model based on solute electron density (SMD), and the generalized Finite-Difference Poisson-Boltzmann (FDPB) model. The proposed correction was parametrized on a diverse training set of 4980 solvation data from the Solv@tum database of experimental solvation energies, and validated on the non-aqueous subset of the MNSOL database comprising 2140 solvation energies. The performances of the proposed ENE models with minimal and extended parameters formulations have been analyzed and the latter variant has been further compared to the widely used Cavity, Dispersion, and Solvent structural effects (CDS) non-electrostatic model originally developed for the SMx family of implicit solvation models. Overall, a very good agreement between the computed solvation energies with the ENE correction and the reference experimental data has been found on both the training and test sets for all continuum solvation models considered. Furthermore, results for the ENE correction are on par with the reference CDS non-electrostatic model for both SMD and FDPB electrostatics, but with the advantage of using a lower number of parameters and thus an improved transferability between different electrostatics treatments.},
  archive      = {J_JCC},
  author       = {Dario Vassetti and Frédéric Labat},
  doi          = {10.1002/jcc.26944},
  journal      = {Journal of Computational Chemistry},
  number       = {20},
  pages        = {1372-1387},
  shortjournal = {J. Comput. Chem.},
  title        = {Towards a transferable nonelectrostatic model for continuum solvation: The electrostatic and nonelectrostatic energy correction model},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Protein–ligand binding affinity prediction of
cyclin-dependent kinase-2 inhibitors by dynamically averaged fragment
molecular orbital-based interaction energy. <em>JCC</em>,
<em>43</em>(20), 1362–1371. (<a
href="https://doi.org/10.1002/jcc.26940">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Fragment molecular orbital (FMO) method is a powerful computational tool for structure-based drug design, in which protein–ligand interactions can be described by the inter-fragment interaction energy (IFIE) and its pair interaction energy decomposition analysis (PIEDA). Here, we introduced a dynamically averaged (DA) FMO-based approach in which molecular dynamics simulations were used to generate multiple protein–ligand complex structures for FMO calculations. To assess this approach, we examined the correlation between the experimental binding free energies and DA-IFIEs of six CDK2 inhibitors whose net charges are zero. The correlation between the experimental binding free energies and snapshot IFIEs for X-ray crystal structures wasR2= 0.75. Using the DA-IFIEs, the correlation significantly improved to 0.99. When an additional CDK2 inhibitor with net charge of −1 was added, the DA FMO-based scheme with the dispersion energies still achievedR2= 0.99, whereasR2decreased to 0.32 employing all the energy terms of PIEDA.},
  archive      = {J_JCC},
  author       = {Kenichiro Takaba and Chiduru Watanabe and Atsushi Tokuhisa and Yoshinobu Akinaga and Biao Ma and Ryo Kanada and Mitsugu Araki and Yasushi Okuno and Yusuke Kawashima and Hirotomo Moriwaki and Norihito Kawashita and Teruki Honma and Kaori Fukuzawa and Shigenori Tanaka},
  doi          = {10.1002/jcc.26940},
  journal      = {Journal of Computational Chemistry},
  number       = {20},
  pages        = {1362-1371},
  shortjournal = {J. Comput. Chem.},
  title        = {Protein–ligand binding affinity prediction of cyclin-dependent kinase-2 inhibitors by dynamically averaged fragment molecular orbital-based interaction energy},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Computational protocol for predicting 19F NMR chemical
shifts for PFAS and connection to PFAS structure. <em>JCC</em>,
<em>43</em>(20), 1355–1361. (<a
href="https://doi.org/10.1002/jcc.26939">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Per- and polyfluoroalkyl substances (PFAS) are robust “forever” chemicals that have become global environmental contaminants due to their inability to degrade using traditional techniques. In addition to the persistent nature of PFAS, the structural and functional diversity in PFAS creates a unique challenge in identification and remediation. Their identification is further complicated by the absence of standards for many PFAS. This work is aimed at developing a protocol for computing and establishing accurate19F NMR chemical shifts for PFAS using density functional theory (DFT), which can aid in the identification of PFAS. The impact of solvation and basis sets was evaluated by comparing the computed data with the experimental measurements. Results showed the addition of dispersion corrections in the methodology improve the accuracy of calculated NMR parameters within 4 ppm of the experimental values. Adding a second diffuse function and additional polarization did not improve the accuracy, likely because of the electronegativity of fluorine which does not allow the electron density of fluorine atoms to be polarized. The inclusion of various implicit solvation (DMSO, chloroform, and water) yielded negligible differences in accuracy, and were overall less accurate than the gas phase calculations. The most accurate methodology was then applied to more environmentally relevant PFAS, and the impact of helical nature on the NMR signatures was evaluated. The implication of this work is to be able to improve the identification of structurally diverse PFAS using the19F NMR.},
  archive      = {J_JCC},
  author       = {Maleigh Mifkovic and Jessica Pauling and Shubham Vyas},
  doi          = {10.1002/jcc.26939},
  journal      = {Journal of Computational Chemistry},
  number       = {20},
  pages        = {1355-1361},
  shortjournal = {J. Comput. Chem.},
  title        = {Computational protocol for predicting 19F NMR chemical shifts for PFAS and connection to PFAS structure},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Sparse group selection and analysis of function-related
residue for protein-state recognition. <em>JCC</em>, <em>43</em>(20),
1342–1354. (<a href="https://doi.org/10.1002/jcc.26937">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning methods have helped to advance wide range of scientific and technological field in recent years, including computational chemistry. As the chemical systems could become complex with high dimension, feature selection could be critical but challenging to develop reliable machine learning based prediction models, especially for proteins as bio-macromolecules. In this study, we applied sparse group lasso (SGL) method as a general feature selection method to develop classification model for an allosteric protein in different functional states. This results into a much improved model with comparable accuracy (Acc) and only 28 selected features comparing to 289 selected features from a previous study. The Acc achieves 91.50\% with 1936 selected feature, which is far higher than that of baseline methods. In addition, grouping protein amino acids into secondary structures provides additional interpretability of the selected features. The selected features are verified as associated with key allosteric residues through comparison with both experimental and computational works about the model protein, and demonstrate the effectiveness and necessity of applying rigorous feature selection and evaluation methods on complex chemical systems.},
  archive      = {J_JCC},
  author       = {Fangyun Bai and Kin Ming Puk and Jin Liu and Hongyu Zhou and Peng Tao and Wenyong Zhou and Shouyi Wang},
  doi          = {10.1002/jcc.26937},
  journal      = {Journal of Computational Chemistry},
  number       = {20},
  pages        = {1342-1354},
  shortjournal = {J. Comput. Chem.},
  title        = {Sparse group selection and analysis of function-related residue for protein-state recognition},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Theoretical exploration of the reactivity of cellulose
models under non-thermal plasma conditions—mechanistic and NBO studies.
<em>JCC</em>, <em>43</em>(20), 1334–1341. (<a
href="https://doi.org/10.1002/jcc.26934">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Mechanistic details of cellulose depolymerization by non-thermal (atmospheric) plasma (NTAP) remains under-explored given the complexity of the medium. In this study, we have investigated the reaction mechanism of glycosidic-bond degradation triggered by reaction with hydroxyl radicals, considered as the principal reactive species in NTAP medium. In the first step of reaction sequence, H-abstraction reactions by HO‧. radical on different CH sites of the pyranose ring were found to be non-selective and markedly exergonic giving rise to a set of cellobiosyl carboradicals likely to undergo further reactions. We then showed that cellobiosyl carboradicals are protected against direct hydrolysis, no activation of the (1–4)-β-glycosidic bond being characterized. Interestingly, a simple homolytic bond cleavage allowed to obtain desired monomer. Among the 18 possible fragmentations, involving CC and CO bond breaking from cellobiosyl carboradicals, 14 transition states were successfully identified, and only three reaction pathways proved kinetically and thermodynamically feasible. Natural bond orbital (NBO) analysis was performed to shed light on electronic structures of different compounds.},
  archive      = {J_JCC},
  author       = {Walid Lamine and Frédéric Guégan and François Jérôme and Gilles Frapper},
  doi          = {10.1002/jcc.26934},
  journal      = {Journal of Computational Chemistry},
  number       = {20},
  pages        = {1334-1341},
  shortjournal = {J. Comput. Chem.},
  title        = {Theoretical exploration of the reactivity of cellulose models under non-thermal plasma conditions—mechanistic and NBO studies},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). PyFREC 2.0: Software for excitation energy transfer
modeling. <em>JCC</em>, <em>43</em>(19), 1320–1328. (<a
href="https://doi.org/10.1002/jcc.26930">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Excitation energy transfer is a ubiquitous process of fundamental importance for understanding natural phenomena, such as photosynthesis, as well as advancing technologies ranging from photovoltaics to development of photosensitizers and fluorescent probes used to explore molecular interactions inside living cells. The current version ofPyFREC2.0 is an advancement of the previously reported software (D. Kosenkov,J. Comput. Chem.2016, 37, 1847–1854). The current update is primarily focused on providing a computational tool based on Förster theory for bridging a gap between theoretically calculated molecular properties (e.g., electronic couplings, orientation factors, etc.) and experimentally measured emission and absorption spectra of molecules. The software is aimed to facilitate deeper understanding of photochemical mechanisms of fluorescence resonance energy transfer (FRET) in donor–acceptor pairs. Specific updates of the software include implementations of overlap integrals between donor emission and acceptor absorption spectra of FRET pairs, estimation of Strickler–Berg fluorescence lifetimes, calculation of Förster radii, energy transfer efficiency, and radiation zones that, in particular, determine applicability of the Förster theory.},
  archive      = {J_JCC},
  author       = {Dmitri Kosenkov},
  doi          = {10.1002/jcc.26930},
  journal      = {Journal of Computational Chemistry},
  number       = {19},
  pages        = {1320-1328},
  shortjournal = {J. Comput. Chem.},
  title        = {PyFREC 2.0: Software for excitation energy transfer modeling},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Computational mechanistic studies on persulfate assisted
p-phenylenediamine polymerization. <em>JCC</em>, <em>43</em>(19),
1313–1319. (<a href="https://doi.org/10.1002/jcc.26943">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {p-Phenylenediamine (p-PDA) is a monomer of many important polymers such as kevlar, twaron, poly-p-PDA. Most of the noticed polymers formation is initiated by a free-radical, but their polymerization mechanism is not elucidated computationally. The proposed study helps to fully understand the frequently utilized initiator/oxidant, potassium persulfate (K2S2O8) role in the aromatic diamines polymerization, which support experimental protocols, and a polymer scope. The formation of the poly-p-PDA is studied with the density functional theory (DFT) B3LYP-D3 functional using experimental polymerization parameters (0°C and aqueous media). K2S2O8initiated free-radical polymerization of p-PDA is studied in detail, taking into account sulfate free-radical (SO4−)·, SFR, persulfate anion (S2O8)2−, PA and K2S2O8cluster, PP. The reaction mechanism is calculated as the conversion of p-PDA to free-radical, the p-PDA free-radical attack to the next p-PDA (dimerization), ammonia extrusion from the dimer adduct, the dimer adduct conversion to the free-radical (completion of p-PDA polymerization cycle) for the polymer chain elongation. Calculations show that the dimerization step is the rate-limiting step with a 29.2 kcal/mol energy barrier when SFR initiates polymerization. In contrast, the PA-assisted dimerization energy barrier is only 12.7 kcal/mol. PP supported polymerization is calculated to have very shallow energy barriers completing the polymerization cycle, i.e., dimerization (TS2K, ∆G‡= 11.6 kcal/mol) and ammonia extrusion (TS3K, ∆G‡= 6.7 kcal/mol).},
  archive      = {J_JCC},
  author       = {Yusif Abdullayev and Ramil Rzayev and Jochen Autschbach},
  doi          = {10.1002/jcc.26943},
  journal      = {Journal of Computational Chemistry},
  number       = {19},
  pages        = {1313-1319},
  shortjournal = {J. Comput. Chem.},
  title        = {Computational mechanistic studies on persulfate assisted p-phenylenediamine polymerization},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Evaluating the conformational space of the active site of d2
dopamine receptor. Scope and limitations of the standard docking
methods. <em>JCC</em>, <em>43</em>(19), 1298–1312. (<a
href="https://doi.org/10.1002/jcc.26938">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We report here for the first time the potential energy surfaces (PES) of phenyletilamine (PEA) and meta-tyramine (m-OH-PEA) at the D2dopamine receptor (D2DR) binding site. PESs not only allow us to observe all the critical points of the surface (minimums, maximums, and transition states), but also to note the ease or difficulty that each local minima have for their conformational inter-conversions and therefore know the conformational flexibility that these ligands have in their active sites. Taking advantage of possessing this valuable information, we analyze how accurate a standard docking study is in these cases. Our results indicate that although we have to be careful in how to carry out this type of study and to consider performing some extra-simulations, docking calculations can be satisfactory. In order to analyze in detail the different molecular interactions that are stabilizing the different ligand-receptor (L-R) complexes, we carried out quantum theory of atoms in molecules (QTAIM) computations and NMR shielding calculations. Although some of these techniques are a bit tedious and require more computational time, our results demonstrate the importance of performing computational simulations using different types of combined techniques (docking/MD/hybrid QM-MM/QTAIM and NMR shielding calculations) in order to obtain more accurate results. Our results allow us to understand in details the molecular interactions stabilizing and destabilizing the different L-R complexes reported here. Thus, the different activities observed for dopamine (DA), m-OH-PEA, and PEA can be clearly explained at molecular level.},
  archive      = {J_JCC},
  author       = {Rodrigo D. Tosso and M. Natalia C. Zarycz and Ayelén Schiel and Luisa Goicoechea Moro and Héctor A. Baldoni and Emilio Angelina and Ricardo D. Enriz},
  doi          = {10.1002/jcc.26938},
  journal      = {Journal of Computational Chemistry},
  number       = {19},
  pages        = {1298-1312},
  shortjournal = {J. Comput. Chem.},
  title        = {Evaluating the conformational space of the active site of d2 dopamine receptor. scope and limitations of the standard docking methods},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Hydration of divalent lanthanides, sm2+ and eu2+: A
molecular dynamics study with polarizable AMOEBA force field.
<em>JCC</em>, <em>43</em>(19), 1286–1297. (<a
href="https://doi.org/10.1002/jcc.26933">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The chemistry of divalent lanthanides, Ln2+, is a growing sub-field of heavy element chemistry owing to new synthetic approaches. However, some theoretical aspects of these unusual cations are currently underdeveloped, especially as they relate to their dynamic properties in solution. In this work, we address the hydration of two of the classical Ln2+cations, Sm2+and Eu2+, using atomic multipole optimized energetic for biomolecular applications (AMOEBA) force fields. These cations have not been parameterized to date with AMOEBA, and few studies are available because of their instability with respect to oxidation in aqueous media. Coordination numbers (CN&#39;s) of 8.2 and 8.1 respectively for Sm2+and Eu2+, and 8.8 for both Sm3+and Eu3+have been obtained and are in good agreement with the few available AIMD and X-ray absorption fine structures studies. The decreased CN of Ln2+compared with Ln3+arises from progressive water exchange events that indicates the gradual stabilization of 8-coordinate structures with respect to 9-coordinate geometries. Moreover, the effects of the chloride counter anions on the coordination of Ln2+cations have been studied at different chloride concentrations in this work. Lastly, water exchange times of Ln2+cations have been calculated to provide a comprehensive understanding of the behavior of Eu2+and Sm2+in aqueous chloride media.},
  archive      = {J_JCC},
  author       = {Hesam Arabzadeh and Chengwen Liu and Orlando Acevedo and Pengyu Ren and Wei Yang and Thomas Albrecht-Schönzart},
  doi          = {10.1002/jcc.26933},
  journal      = {Journal of Computational Chemistry},
  number       = {19},
  pages        = {1286-1297},
  shortjournal = {J. Comput. Chem.},
  title        = {Hydration of divalent lanthanides, sm2+ and eu2+: A molecular dynamics study with polarizable AMOEBA force field},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Composition-selective full inclusion host–guest interaction
of azobenzene-containing photoresponsive nanoring with fullerene c60.
<em>JCC</em>, <em>43</em>(19), 1276–1285. (<a
href="https://doi.org/10.1002/jcc.26931">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By means of density functional theory calculations, the encapsulation capabilities of a series of azobenzen-containing photoresponsive nanoring hosts (labeled as host1,2,3,4,5and6according to the number of the azo unit, respectively) for fullerene C60were surveyed. Interestingly and abnormally, it is found that the host5, of which the diameter is only 1.218 nm, can form stable full inclusion complex with C60. However, irrespective of their cavity sizes (11.98 ~ 12.94 Å) of the hosts, the structures1~ 4 and6were all disable to form inclusion complex with C60. In this paper, the group-number-composition-selective full inclusion host-guest interaction of the azobenzene-containing nanorings with fullerene C60is firstly presented. The calculated interaction energies, together with the detection and visualization of the weak interaction regions, provided evidences for the host-guest binding based on relative strong repulsion interaction in the full inclusion complex. Analysis on the frontier orbital feature of the host-guest systems suggests that under the electron excited condition, the chemical activity may be transferred from host5to guest C60by formation of the floating host-guest complex, and the chemical reactivity of the host5can be passivated via formation of the full inclusion host-guest complex. Additionally, UV–vis–NIR and1H NMR spectra of the hosts before and after the formations of the complexes have been simulated and discussed qualitatively, which may be helpful for further experimental investigations in future.},
  archive      = {J_JCC},
  author       = {Kun Yuan and Mengyang Li and Yaoxiao Zhao and Yanzhi Liu and Xiang Zhao},
  doi          = {10.1002/jcc.26931},
  journal      = {Journal of Computational Chemistry},
  number       = {19},
  pages        = {1276-1285},
  shortjournal = {J. Comput. Chem.},
  title        = {Composition-selective full inclusion host–guest interaction of azobenzene-containing photoresponsive nanoring with fullerene c60},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Accelerating the 3D reference interaction site model theory
of molecular solvation with treecode summation and cut-offs.
<em>JCC</em>, <em>43</em>(18), 1251–1270. (<a
href="https://doi.org/10.1002/jcc.26889">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The 3D reference interaction site model (3D-RISM) of molecular solvation is a powerful tool for computing the equilibrium thermodynamics and density distributions of solvents, such as water and co-ions, around solute molecules. However, 3D-RISM solutions can be expensive to calculate, especially for proteins and other large molecules where calculating the potential energy between solute and solvent requires more than half the computation time. To address this problem, we have developed and implemented treecode summation for long-range interactions and analytically corrected cut-offs for short-range interactions to accelerate the potential energy and long-range asymptotics calculations in non-periodic 3D-RISM in the AmberTools molecular modeling suite. For the largest single protein considered in this work, tubulin, the total computation time was reduced by a factor of 4. In addition, parallel calculations with these new methods scale almost linearly and the iterative solver remains the largest impediment to parallel scaling. To demonstrate the utility of our approach for large systems, we used 3D-RISM to calculate the solvation thermodynamics and density distribution of 7-ring microtubule, consisting of 910 tubulin dimers, over 1.2 million atoms.},
  archive      = {J_JCC},
  author       = {Leighton Wilson and Robert Krasny and Tyler Luchko},
  doi          = {10.1002/jcc.26889},
  journal      = {Journal of Computational Chemistry},
  number       = {18},
  pages        = {1251-1270},
  shortjournal = {J. Comput. Chem.},
  title        = {Accelerating the 3D reference interaction site model theory of molecular solvation with treecode summation and cut-offs},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mechanistic insights of key host proteins and potential
repurposed inhibitors regulating SARS-CoV-2 pathway. <em>JCC</em>,
<em>43</em>(18), 1237–1250. (<a
href="https://doi.org/10.1002/jcc.26888">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The emergence of pandemic situations originated from severe acute respiratory syndrome (SARS)-CoV-2 and its new variants created worldwide medical emergencies. Due to the non-availability of efficient drugs and vaccines at these emergency hours, repurposing existing drugs can effectively treat patients critically infected by SARS-CoV-2. Finding a suitable repurposing drug with inhibitory efficacy to a host-protein is challenging. A detailed mechanistic understanding of the kinetics, (dis)association pathways, key protein residues facilitating the entry–exit of the drugs with targets are fundamental in selecting these repurposed drugs. Keeping this target as the goal of the paper, the potential repurposing drugs, Nafamostat, Camostat, Silmitasertib, Valproic acid, and Zotatifin with host-proteins HDAC2, CSK22, eIF4E2 are studied to elucidate energetics, kinetics, and dissociation pathways. From an ensemble of independent simulations, we observed the presence of single or multiple dissociation pathways with varying host-proteins-drug systems and quantitatively estimated the probability of unbinding through these specific pathways. We also explored the crucial gateway residues facilitating these dissociation mechanisms. Interestingly, the residues we obtained for HDAC2 and CSK22 are also involved in the catalytic activity. Our results demonstrate how these potential drugs interact with the host machinery and the specific target residues, showing involvement in the mechanism. Most of these drugs are in the preclinical phase, and some are already being used to treat severe COVID-19 patients. Hence, the mechanistic insight presented in this study is envisaged to support further findings of clinical studies and eventually develop efficient inhibitors to treat SARS-CoV-2.},
  archive      = {J_JCC},
  author       = {Debabrata Pramanik and Aiswarya B. Pawar and Sudip Roy and Jayant Kumar Singh},
  doi          = {10.1002/jcc.26888},
  journal      = {Journal of Computational Chemistry},
  number       = {18},
  pages        = {1237-1250},
  shortjournal = {J. Comput. Chem.},
  title        = {Mechanistic insights of key host proteins and potential repurposed inhibitors regulating SARS-CoV-2 pathway},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Simulation studies of polypeptoids using replica exchange
with dynamical scaling and dihedral biasing. <em>JCC</em>,
<em>43</em>(18), 1229–1236. (<a
href="https://doi.org/10.1002/jcc.26887">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Polypeptoids differ from polypeptides in that the amide bond can more frequently adopt bothcisandtransconformations. The transition between the two conformations requires overcoming a large energy barrier, making it difficult for conventional molecular simulations to adequately visit thecisandtransstructures. A replica-exchange method is presented that allows for easy rotations of the amide bond and also an efficient linking to a high temperature replica. The method allows for just three replicas (one at the temperature and Hamiltonian of interest, a second high temperature replica with a biased dihedral potential, and a third connecting them) to overcome the amide bond sampling problem and also enhance sampling for other coordinates. The results indicate that for short peptoid oligomers, the conformations can range from allcisto alltranswith an averagecis/transratio that depends on side chain and potential model.},
  archive      = {J_JCC},
  author       = {Bryan A. Raubenolt and Steven W. Rick},
  doi          = {10.1002/jcc.26887},
  journal      = {Journal of Computational Chemistry},
  number       = {18},
  pages        = {1229-1236},
  shortjournal = {J. Comput. Chem.},
  title        = {Simulation studies of polypeptoids using replica exchange with dynamical scaling and dihedral biasing},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mechanistic aspects of the synthesis of seven-membered
internal nitronates via stepwise [4 + 3] cycloaddition involving
conjugated nitroalkenes: Molecular electron density theory computational
study. <em>JCC</em>, <em>43</em>(18), 1221–1228. (<a
href="https://doi.org/10.1002/jcc.26885">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The density functional theory computational study indicates the possibility of the synthesis of seven-membered internal nitronates via cycloaddition reactions involving Z-C-aryl-N-methylnitrones and E-2-aryl-1-cyano-1-nitroethenes. The detailed exploration of the reaction paths indicates a polar, stepwise reaction mechanism through the zwitterionic intermediate. Using bonding evolution theory (BET), we have deciphered the molecular mechanism of the [4 + 3] cycloaddition reaction between E-2-phenyl-1-cyano-1-nitroethene and Z-C-phenyl-N-methylnitrone. The BET study has revealed that the formation of two CO single bonds takes place in the same way, through the depopulation of NC and CC bonding regions and monosynaptic basins, respectively. The first O1C7 single bond was formed in the sixth phase, while the second C3O4 bond was formed in the last ninth phase.},
  archive      = {J_JCC},
  author       = {Agnieszka Kącka-Zych and Radomir Jasiński},
  doi          = {10.1002/jcc.26885},
  journal      = {Journal of Computational Chemistry},
  number       = {18},
  pages        = {1221-1228},
  shortjournal = {J. Comput. Chem.},
  title        = {Mechanistic aspects of the synthesis of seven-membered internal nitronates via stepwise [4 + 3] cycloaddition involving conjugated nitroalkenes: Molecular electron density theory computational study},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). The physical nature of the ultrashort spike–ring
interaction in iron maiden molecules. <em>JCC</em>, <em>43</em>(18),
1206–1220. (<a href="https://doi.org/10.1002/jcc.26879">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The so-called ‘iron maiden’ molecules belong to one of the most interesting subgroups of cyclophanes due to the presence of the ultrashort interaction between the CX apical bond and the benzene ring. This article presents an in-depth theoretical study of 16 ‘iron maiden’ molecules, in which X = H, F, Cl or Br and the side chains are of various lengths and types: CSC, CSCC, CCC, and CCCC. It is shown that the H → F → Cl → Br substitution leads to a significant expansion of the ‘iron maiden’ molecule. Shorter chains lead to more pronounced effects, while insertion of sulfur atoms into the side chains lowers them. Structural changes are associated with an increase in energetic destabilization of X. Moreover, unlike for H, in the case of X = halogen, theout→inisomerization is energetically disadvantageous. The ‘iron maiden’ molecules are characterized by the presence of only three X⋯CArbond paths. Particularly noteworthy are unusually large (even up to 32) values of the X⋯CArbond ellipticity, which results from flat electron density distribution. The X⋯πinteraction in each of the investigated ‘iron maiden’ molecule turned out to be multi-center, stabilizing and almost purely covalent in nature as indicated by the definitely dominant percentage (94.8\%–101.6\%) of the exchange-correlation energy. The spatial hindrance within the ‘iron maiden’ molecules appears to be not so much due to the X⋯πrepulsion, but due to unfavorable steric interactions between X and the CC side bonds. It is also confirmed that some CH⋯HC interactions in aliphatic chains can be very weakly stabilizing.},
  archive      = {J_JCC},
  author       = {Mirosław Jabłoński},
  doi          = {10.1002/jcc.26879},
  journal      = {Journal of Computational Chemistry},
  number       = {18},
  pages        = {1206-1220},
  shortjournal = {J. Comput. Chem.},
  title        = {The physical nature of the ultrashort spike–ring interaction in iron maiden molecules},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Exploration of high dimensional free energy landscapes by a
combination of temperature-accelerated sliced sampling and parallel
biasing. <em>JCC</em>, <em>43</em>(17), 1186–1200. (<a
href="https://doi.org/10.1002/jcc.26882">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Temperature-accelerated sliced sampling (TASS) is an enhanced sampling method for achieving accelerated and controlled exploration of high-dimensional free energy landscapes in molecular dynamics simulations. With the aid of umbrella bias potentials, the TASS method realizes a controlled exploration and divide-and-conquer strategy for computing high-dimensional free energy surfaces. In TASS, diffusion of the system in the collective variable (CV) space is enhanced with the help of metadynamics bias and elevated-temperature of the auxiliary degrees of freedom (DOF) that are coupled to the CVs. Usually, a low-dimensional metadynamics bias is applied in TASS. In order to further improve the performance of TASS, we propose here to use a highdimensional metadynamics bias, in the same form as in a parallel bias metadynamics scheme. Here, a modified reweighting scheme, in combination with artificial neural network is used for computing unbiased probability distribution of CVs and projections of high-dimensional free energy surfaces. We first validate the accuracy and efficiency of our method in computing the four-dimensional free energy landscape for alanine tripeptide in vacuo. Subsequently, we employ the approach to calculate the eight-dimensional free energy landscape of alanine pentapeptide in vacuo. Finally, the method is applied to a more realistic problem wherein we compute the broad four-dimensional free energy surface corresponding to the deacylation of a drug molecule which is covalently complexed with aβ-lactamase enzyme. We demonstrate that using parallel bias in TASS improves the efficiency of exploration of high-dimensional free energy landscapes.},
  archive      = {J_JCC},
  author       = {Abhinav Gupta and Shivani Verma and Ramsha Javed and Suraj Sudhakar and Saurabh Srivastava and Nisanth N. Nair},
  doi          = {10.1002/jcc.26882},
  journal      = {Journal of Computational Chemistry},
  number       = {17},
  pages        = {1186-1200},
  shortjournal = {J. Comput. Chem.},
  title        = {Exploration of high dimensional free energy landscapes by a combination of temperature-accelerated sliced sampling and parallel biasing},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The second derivative of the electronic energy with respect
to the compression scaling factor in the XP-PCM model: Theory and
applications to compression response functions of atoms. <em>JCC</em>,
<em>43</em>(17), 1176–1185. (<a
href="https://doi.org/10.1002/jcc.26883">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present the analytical theory for the second derivative of the electronic energy with respect to the scaling factor of the compression cavity within the eXtreme pressure polarizable continuum model (XP-PCM) for the study of compressed atomic and molecular systems. The theory has been exploited to study compression response functions describing how the atomic/molecular properties are effected by an external pressure. The response functions considered include the atomic compressibility and the pressure coefficients of the ionization energy (IE) and electron affinity (EA). The theory has been validated by numerical application to compressed neon, argon, and krypton atoms.},
  archive      = {J_JCC},
  author       = {Roberto Cammi and Bo Chen},
  doi          = {10.1002/jcc.26883},
  journal      = {Journal of Computational Chemistry},
  number       = {17},
  pages        = {1176-1185},
  shortjournal = {J. Comput. Chem.},
  title        = {The second derivative of the electronic energy with respect to the compression scaling factor in the XP-PCM model: Theory and applications to compression response functions of atoms},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Investigation of water substitution at RuII complexes by
conceptual density function theory approach. <em>JCC</em>,
<em>43</em>(17), 1161–1175. (<a
href="https://doi.org/10.1002/jcc.26878">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this paper, we investigated water exchange reactions and substitution of aqua RuIIcomplexes of general formula [Ru(terpy)(N^N)(H2O)]2+(whereN^N= ethylenediamine (en), 1,2-(aminomethyl)pyridine (ampy) and 2,2′-bipyridine (bipy)) by ammonia and thioformaldehyde. These reactions were studied in detail by applying conceptual density functional theory. This approach enabled us to gain further insight into the underlying reaction mechanism at the microscopic level (involving only direct participants of the reaction, without the influence of the solvent) and to put the concept of reaction mechanism on a quantitative basis. The course of the chemical reaction along the reaction coordinateξ, is rationalized in terms of reaction energy, force, dipole moment, and reaction electronic flux (REF). The results yield and characterize the significant influence of an intermolecular hydrogen bond formed between the entering and the spectator ligand to the overall energy barrier of the reactions.},
  archive      = {J_JCC},
  author       = {Dušan Ćoćić and Biljana Petrović and Ralph Puchta and Marta Chrzanowska and Anna Katafias and Rudi van Eldik},
  doi          = {10.1002/jcc.26878},
  journal      = {Journal of Computational Chemistry},
  number       = {17},
  pages        = {1161-1175},
  shortjournal = {J. Comput. Chem.},
  title        = {Investigation of water substitution at RuII complexes by conceptual density function theory approach},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Alchemical free energy simulations without speed limits. A
generic framework to calculate free energy differences independent of
the underlying molecular dynamics program. <em>JCC</em>,
<em>43</em>(17), 1151–1160. (<a
href="https://doi.org/10.1002/jcc.26877">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We describe the theory of the so-called common-core/serial-atom-insertion (CC/SAI) approach to compute alchemical free energy differences and its practical implementation in a Python package called Transformato. CC/SAI is not tied to a specific biomolecular simulation program and does not rely on special purpose code for alchemical transformations. To calculate the alchemical free energy difference between several small molecules, the physical end-states are mutated into a suitable common core. Since this only requires turning off interactions, the setup of intermediate states is straightforward to automate. Transformato currently supports CHARMM and OpenMM as back ends to carry out the necessary molecular dynamics simulations, as well as post-processing calculations. We validate the method by computing a series of relative solvation free energy differences.},
  archive      = {J_JCC},
  author       = {Marcus Wieder and Markus Fleck and Benedict Braunsfeld and Stefan Boresch},
  doi          = {10.1002/jcc.26877},
  journal      = {Journal of Computational Chemistry},
  number       = {17},
  pages        = {1151-1160},
  shortjournal = {J. Comput. Chem.},
  title        = {Alchemical free energy simulations without speed limits. a generic framework to calculate free energy differences independent of the underlying molecular dynamics program},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Using steered molecular dynamic tension for assessing
quality of computational protein structure models. <em>JCC</em>,
<em>43</em>(17), 1140–1150. (<a
href="https://doi.org/10.1002/jcc.26876">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The native structures of proteins, except for notable exceptions of intrinsically disordered proteins, in general take their most stable conformation in the physiological condition to maintain their structural framework so that their biological function can be properly carried out. Experimentally, the stability of a protein can be measured by several means, among which the pulling experiment using the atomic force microscope (AFM) stands as a unique method. AFM directly measures the resistance from unfolding, which can be quantified from the observed force-extension profile. It has been shown that key features observed in an AFM pulling experiment can be well reproduced by computational molecular dynamics simulations. Here, we applied computational pulling for estimating the accuracy of computational protein structure models under the hypothesis that the structural stability would positively correlated with the accuracy, i.e. the closeness to the native, of a model. We used in total 4929 structure models for 24 target proteins from the Critical Assessment of Techniques of Structure Prediction (CASP) and investigated if the magnitude of the break force, that is, the force required to rearrange the model&#39;s structure, from the force profile was sufficient information for selecting near-native models. We found that near-native models can be successfully selected by examining their break forces suggesting that high break force indeed indicates high stability of models. On the other hand, there were also near-native models that had relatively low peak forces. The mechanisms of the stability exhibited by the break forces were explored and discussed.},
  archive      = {J_JCC},
  author       = {Lyman Monroe and Daisuke Kihara},
  doi          = {10.1002/jcc.26876},
  journal      = {Journal of Computational Chemistry},
  number       = {17},
  pages        = {1140-1150},
  shortjournal = {J. Comput. Chem.},
  title        = {Using steered molecular dynamic tension for assessing quality of computational protein structure models},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Endo- and exohedral complexes of superphane with cations.
<em>JCC</em>, <em>43</em>(16), 1120–1133. (<a
href="https://doi.org/10.1002/jcc.26874">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quite recently it has been shown in a previous study that superphane, that is, [2.2.2.2.2.2](1,2,3,4,5,6)cyclophane, is a very convenient molecule in the study of endohedral complexes and especially in the study of the influence of the caged entity (i.e., guest) on the structure of the host molecule. This advantage results from the presence of two parallel benzene rings joined together by six quite flexible ethylene bridges (spacers). This article examines the energetic and structural properties of endo- and exohedral complexes of superphane with the following cations: H+, Li+, Na+, K+, Be2+, Mg2+, Ca2+, B3+, Al3+, Ga3+. The stability of endohedral complexes has been shown to be strongly dependent on the charge and radius of the caged cation. The inclusion of the cation inside the superphane molecule causes its ‘swelling’, which is manifested by an increase in the distance between the benzene rings and elongations of the ring and spacer C–C bonds. In the case of exohedral complexes, three forms are investigated: with the cation above the benzene ring, with the cation interacting with the superphane window in the equatorial position, and with the cation interacting with the center of the C–C spacer bond. The first of these forms has been shown to be preferred. The cation⋯acceptor distance depends on the cation radius. Among the cations investigated, H+and Be2+are particularly reactive and predisposed to induce significant structural changes in the superphane molecule, forming C–H bond or C–Be–C bridges, respectively.},
  archive      = {J_JCC},
  author       = {Mirosław Jabłoński},
  doi          = {10.1002/jcc.26874},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1120-1133},
  shortjournal = {J. Comput. Chem.},
  title        = {Endo- and exohedral complexes of superphane with cations},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). TUPÃ: Electric field analyses for molecular simulations.
<em>JCC</em>, <em>43</em>(16), 1113–1119. (<a
href="https://doi.org/10.1002/jcc.26873">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We introduce TUPÃ, a Python-based algorithm to calculate and analyze electric fields in molecular simulations. To demonstrate the features in TUPÃ, we present three test cases in which the orientation and magnitude of the electric field exerted by biomolecules help explain biological phenomena or observed kinetics. As part of TUPÃ, we also provide a PyMOL plugin to help researchers visualize how electric fields are organized within the simulation system. The code is freely available and can be obtained athttps://mdpoleto.github.io/tupa/.},
  archive      = {J_JCC},
  author       = {Marcelo D. Polêto and Justin A. Lemkul},
  doi          = {10.1002/jcc.26873},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1113-1119},
  shortjournal = {J. Comput. Chem.},
  title        = {TUPÃ: Electric field analyses for molecular simulations},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Data-driven and constrained optimization of semi-local
exchange and nonlocal correlation functionals for materials and surface
chemistry. <em>JCC</em>, <em>43</em>(16), 1104–1112. (<a
href="https://doi.org/10.1002/jcc.26872">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reliable predictions of surface chemical reaction energetics require an accurate description of both chemisorption and physisorption. Here, we present an empirical approach to simultaneously optimize semi-local exchange and nonlocal correlation of a density functional approximation to improve these energetics. A combination of reference data for solid bulk, surface, and gas-phase chemistry and physical exchange-correlation model constraints leads to the VCML-rVV10 exchange-correlation functional. Owing to the variety of training data, the applicability of VCML-rVV10 extends beyond surface chemistry simulations. It provides optimized gas phase reaction energetics and an accurate description of bulk lattice constants and elastic properties.},
  archive      = {J_JCC},
  author       = {Kai Trepte and Johannes Voss},
  doi          = {10.1002/jcc.26872},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1104-1112},
  shortjournal = {J. Comput. Chem.},
  title        = {Data-driven and constrained optimization of semi-local exchange and nonlocal correlation functionals for materials and surface chemistry},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Polarization energies in the fragment molecular orbital
method. <em>JCC</em>, <em>43</em>(16), 1094–1103. (<a
href="https://doi.org/10.1002/jcc.26869">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using isolated and polarized states of fragments, a method for computing the polarization energies in density functional theory (DFT) and density-functional tight-binding (DFTB) is developed in the framework of the fragment molecular orbital method. For DFTB, the method is extended into the use of periodic boundary conditions (PBC), for which a new component, a periodic self-polarization energy, is derived. The couplings of the polarization to other components in the pair interaction energy analysis (PIEDA) are derived for DFT and DFTB, and compared to Hartree–Fock and second-order Møller-Plesset perturbation theory (MP2). The effect of the self-consistent (DFT) and perturbative (MP2) treatment of the electron correlation on the polarization is discussed. The difference in the polarization in the bulk (PBC) and micro (cluster) solvation is elucidated.},
  archive      = {J_JCC},
  author       = {Dmitri G. Fedorov},
  doi          = {10.1002/jcc.26869},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1094-1103},
  shortjournal = {J. Comput. Chem.},
  title        = {Polarization energies in the fragment molecular orbital method},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Comparison of approximate intermolecular potentials for ab
initio fragment calculations on medium sized n-heterocycles.
<em>JCC</em>, <em>43</em>(16), 1079–1093. (<a
href="https://doi.org/10.1002/jcc.26866">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ground state intermolecular potential of bimolecular complexes of N-heterocycles is analyzed for the impact of individual terms in the interaction energy as provided by various, conceptually different theories. Novel combinations with several formulations of the electrostatic, Pauli repulsion, and dispersion contributions are tested at both short- and long-distance sides of the potential energy surface, for various alignments of the pyrrole dimer as well as the cytosine–uracil complex. The integration of a DFT/CCSD density embedding scheme, with dispersion terms from the effective fragment potential (EFP) method is found to provide good agreement with a reference CCSD(T) potential overall; simultaneously, a quantum mechanics/molecular mechanics approach using CHELPG atomic point charges for the electrostatic interaction, augmented by EFP dispersion and Pauli repulsion, comes also close to the reference result. Both schemes have the advantage of not relying on predefined force fields; rather, the interaction parameters can be determined for the system under study, thus being excellent candidates for ab initio modeling.},
  archive      = {J_JCC},
  author       = {Bónis Barcza and Ádám B. Szirmai and Katalin J. Szántó and Attila Tajti and Péter G. Szalay},
  doi          = {10.1002/jcc.26866},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1079-1093},
  shortjournal = {J. Comput. Chem.},
  title        = {Comparison of approximate intermolecular potentials for ab initio fragment calculations on medium sized N-heterocycles},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A fast approximate extension of the interacting quantum
atoms energy decomposition to excited states. <em>JCC</em>,
<em>43</em>(16), 1068–1078. (<a
href="https://doi.org/10.1002/jcc.26863">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An approach is developed for the fast calculation of the interacting quantum atoms energy decomposition (IQA) from the information contained in the first order reduced density matrix only. The proposed methodology utilizes an approximate exchange-correlation density from Density Matrix Functional Theory without the need to evaluate the correlation-exchange contribution directly. Instead, weight factors are estimated to decompose the exactVxcinto atomic and pairwise contributions. In this way, the sum of the IQA contributions recovers the energy obtained from the electronic structure calculation. This method can, hence, be applied to obtain atomic contributions in excited states on the same footing as in their ground states using any method that delivers the reduced first-order density matrix. In this way, one can locate chromophores from first principles quantum chemical calculations. Test calculations on the ground and excited states of a set of small molecules indicate that the scaled atomic contributions reproduce vertical electronic transition energies calculated exactly. This approach may be useful to extend the applicability of the IQA approach in the study of large photochemical systems especially when the calculations of the second order reduced density matrices is prohibitive or not possible.},
  archive      = {J_JCC},
  author       = {Jesús Jara-Cortés and Chérif F. Matta and Jesús Hernández-Trujillo},
  doi          = {10.1002/jcc.26863},
  journal      = {Journal of Computational Chemistry},
  number       = {16},
  pages        = {1068-1078},
  shortjournal = {J. Comput. Chem.},
  title        = {A fast approximate extension of the interacting quantum atoms energy decomposition to excited states},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The pfizer crystal structure database: An essential tool for
structure-based design at pfizer. <em>JCC</em>, <em>43</em>(15),
1053–1062. (<a href="https://doi.org/10.1002/jcc.26862">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Pfizer&#39;s Crystal Structure Database (CSDB) is a key enabling technology that allows scientists on structure-based projects rapid access to Pfizer&#39;s vast library of in-house crystal structures, as well as a significant number of structures imported from the Protein Data Bank. In addition to capturing basic information such as the asymmetric unit coordinates, reflection data, and the like, CSDB employs a variety of automated methods to first ensure a standard level of annotations and error checking, and then to add significant value for design teams by processing the structures through a sequence of algorithms that prepares the structures for use in modeling. The structures are made available, both as the original asymmetric unit as submitted, as well as the final prepared structures, through REST-based web services that are consumed by several client desktop applications. The structures can be searched by keyword, sequence, submission date, ligand substructure and similarity search, and other common queries.},
  archive      = {J_JCC},
  author       = {Daniel K. Gehlhaar and Brock A. Luty and Philip P. Cheung and Andy H. Litman and Robert M. Owen and Peter W. Rose},
  doi          = {10.1002/jcc.26862},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1053-1062},
  shortjournal = {J. Comput. Chem.},
  title        = {The pfizer crystal structure database: An essential tool for structure-based design at pfizer},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ZZS similarity tool: The online tool for similarity
screening to identify chemicals of potential concern. <em>JCC</em>,
<em>43</em>(15), 1042–1052. (<a
href="https://doi.org/10.1002/jcc.26859">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Screening and prioritization of chemicals is essential to ensure that available evaluation capacity is invested in those substances that are of highest concern. We, therefore, recently developed structural similarity models that evaluate the structural similarity of substances with unknown properties to known Substances of Very High Concern (SVHC), which could be an indication of comparable effects. In the current study the performance of these models is improved by (1) separating known SVHCs in more specific subgroups, (2) (re-)optimizing similarity models for the various SVHC-subgroups, and (3) improving interpretability of the predicted outcomes by providing a confidence score. The improvements are directly incorporated in a freely accessible web-based tool, named the ZZS similarity tool:https://rvszoeksysteem.rivm.nl/ZzsSimilarityTool. Accordingly, this tool can be used by risk assessors, academia and industrial partners to screen and prioritize chemicals for further action and evaluation within varying frameworks, and could support the identification of tomorrow&#39;s substances of concern.},
  archive      = {J_JCC},
  author       = {Pim N. H. Wassenaar and Emiel Rorije and Martina G. Vijver and Willie J. G. M. Peijnenburg},
  doi          = {10.1002/jcc.26859},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1042-1052},
  shortjournal = {J. Comput. Chem.},
  title        = {ZZS similarity tool: The online tool for similarity screening to identify chemicals of potential concern},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Time-dependent density functional theory study of the
optical properties of tetrahedral aluminum nanoparticles. <em>JCC</em>,
<em>43</em>(15), 1033–1041. (<a
href="https://doi.org/10.1002/jcc.26868">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The plasmonic properties of tetrahedral aluminum nanoparticles have been investigated using time-dependent functional theory (TDDFT) calculations. The excitation energies are calculated for tetrahedral aluminum nanoparticles (Aln,n =10–120) with different charge states. The BP86/DZ model is used to perform geometric optimization calculations for these clusters. The SAOP/DZ and LB94/DZ levels of theory have been used for the excitation energy calculations of these tetrahedral aluminum clusters. The absorption peaks are red-shifted upon increasing the size of the aluminum nanoparticles.},
  archive      = {J_JCC},
  author       = {Gyun-Tack Bae and Christine M. Aikens},
  doi          = {10.1002/jcc.26868},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1033-1041},
  shortjournal = {J. Comput. Chem.},
  title        = {Time-dependent density functional theory study of the optical properties of tetrahedral aluminum nanoparticles},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A theoretical investigation into gallic acid pyrolysis.
<em>JCC</em>, <em>43</em>(15), 1023–1032. (<a
href="https://doi.org/10.1002/jcc.26865">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Thermodynamic and kinetic information on the first two steps of gallic acid pyrolysis, a decarboxylation followed by a dehydrogenation, is obtained based on density functional theory and quantum chemistry. For the kinetics, transition states are identified with the help of the climbing image nudged elastic band method. Both reactions exhibit two transition states. One of them is related to the rotation of OH groups, and the other one is related to the breaking and forming of bonds. The gallic acid pyrolysis as a whole is judged to be endothermal, and it changes from endergonic to exergonic between 500 and 750 K. The second reaction, the dehydrogenation of pyrogallol, is identified as the rate-determining step of gallic acid pyrolysis, with reaction rate constants below 1 s−1for temperatures below 1250 K.},
  archive      = {J_JCC},
  author       = {Jakob Kraus and Jens Kortus},
  doi          = {10.1002/jcc.26865},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1023-1032},
  shortjournal = {J. Comput. Chem.},
  title        = {A theoretical investigation into gallic acid pyrolysis},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Prediction of acid pKa values in the solvent acetone based
on COSMO-RS. <em>JCC</em>, <em>43</em>(15), 1011–1022. (<a
href="https://doi.org/10.1002/jcc.26864">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this contribution we extent the use of the conductor-like screening model for realistic solvation (COSMO-RS) to the prediction of pKavalues in acetone, a commonly used dipolar aprotic solvent. For this, we calculated the Gibbs free energy of dissociation of 120 organic acids (nine acrylic acids, 87 benzoic acids, nine phenols, and 15 benzenesulfonamides) using COSMO-RS at the two levels BP-TZVP and BP-TZVPD-FINE and determined the parameters for a linear free energy relation for thepKaprediction by performing linear fits to experimental values. Our results suggest that the data set dissects into two groups, with the phenols being different from the other three subsets. The acrylic and benzoic acids and the sulfonamides can be treated together and yield an excellent linear correlation (r2&gt;0.95) with an RMSD of only ~0.3. The slope is found to be significantly smaller than the theoretical value (1/RTln10), only 45\% of it, which is in accordance with findings in the literature. The phenols, however, while similarly well correlated in their own subset with an RMSD of 1.7–1.9, exhibit a slope larger than one. We discuss both a higher uncertainty in the reference values as well as physical origins as possible reasons.},
  archive      = {J_JCC},
  author       = {Niklas Sülzner and Julia Haberhauer and Christof Hättig and Arnim Hellweg},
  doi          = {10.1002/jcc.26864},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1011-1022},
  shortjournal = {J. Comput. Chem.},
  title        = {Prediction of acid pKa values in the solvent acetone based on COSMO-RS},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Keto-enol tautomerism from the electron delocalization
perspective. <em>JCC</em>, <em>43</em>(15), 1000–1010. (<a
href="https://doi.org/10.1002/jcc.26858">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The equilibrium between keto and enol forms in acetylacetone and its derivatives is studied using electron delocalization indices and delocalization tensor density. We demonstrate how electron delocalization governs the equilibrium between keto and enol forms. The less stable enols have more distinct double and single bond character in the CCC fragment, while electron delocalization in this fragment is more pronounced in more stable enols. Looking for the origin of such behavior, we considered the one-electron potentials entering the Euler equation for the electron density. We found that electron delocalization is mainly governed by the static exchange potential, which depends on the three-dimensional atomic structure. It, however, does not distinguish differences in electron delocalization in more and less stable enols, the effect arising from the kinetic exchange contribution, which reflects spin-dependent effects in the electron motion. The local depletion of kinetic exchange in the conjugated fragment yields the enhanced electron delocalization along the CCC bonds in more stable enols. Thus, a combination of considered descriptors allowed us to reveal the influence of electron delocalization on the equilibrium between keto and enol forms and showed the significant features of this phenomenon.},
  archive      = {J_JCC},
  author       = {Elena O. Levina and Maria G. Khrenova and Andrey A. Astakhov and Vladimir G. Tsirelson},
  doi          = {10.1002/jcc.26858},
  journal      = {Journal of Computational Chemistry},
  number       = {15},
  pages        = {1000-1010},
  shortjournal = {J. Comput. Chem.},
  title        = {Keto-enol tautomerism from the electron delocalization perspective},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Ab initio characterization of the aluminum dimer in its x
3Πu and a 3∑g− electronic states. <em>JCC</em>, <em>43</em>(14),
986–993. (<a href="https://doi.org/10.1002/jcc.26857">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The potential energy functions of the aluminum dimer, Al2, in its lowest-energy electronic states,X3ΠuandA3∑g−,have been determined from ab initio calculations using the multi-reference averaged coupled-pair functional method in conjunction with the correlation-consistent basis sets up to septuple-zeta quality. The core-electron correlation, scalar relativistic, and spin-orbit effects were taken into account. The vibration-rotation energy levels for both the states of Al2were calculated to near the “spectroscopic” accuracy. The3Πustate was unequivocally confirmed to be the electronic ground state of Al2, and the electronic term valueT0of the3∑g−state was predicted to be 247 cm−1. The energies and intensities of direct electronic-vibration transitionsX3Πu↔A3∑g−were predicted.},
  archive      = {J_JCC},
  author       = {Jacek Koput},
  doi          = {10.1002/jcc.26857},
  journal      = {Journal of Computational Chemistry},
  number       = {14},
  pages        = {986-993},
  shortjournal = {J. Comput. Chem.},
  title        = {Ab initio characterization of the aluminum dimer in its x 3Πu and a 3∑g− electronic states},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Decoding the reaction mechanism of the cyclocondensation of
ethyl acetate2-oxo-2-(4-oxo-4H-pyrido [1.2-a] pyrimidin-3-yl)
polyazaheterocycle and ethylenediamine using bond evolution theory.
<em>JCC</em>, <em>43</em>(14), 972–985. (<a
href="https://doi.org/10.1002/jcc.26853">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We investigated the flow of electron density along the cyclocondensation reaction between ethyl acetate 2-oxo-2-(4-oxo-4H-pyrido[1.2-a]pyrimidin-3-yl) polyazaheterocycle (1) and ethylenediamine (2) at the ωB97XD/6–311++G(d,p)computational method within of bond evolution theory (BET). The exploration of potential energy surface shows that this reaction has three channels (1–3) with the formation of product3viachannel-2(the most favorable one) as the main product and this is in good agreement with experimental observations. The BET analysis allows identifying unambiguously the main chemical events happening along channel-2. The mechanism along first step (TS2-a) is described by a series of four structural stability domains (SSDs), while five SSDs for the last two steps (TS2-b and TS2-c). The first and third steps can be summarized as follows, the formation of N1-C6 bond (SSD-II), then, the restoration of the nitrogen N1 lone pair (SSD-III), and finally, the formation of the last O1-H1 bond (SSD-IV). For the second step, the formation of hydroxide ion is noted, as a result of the disappearance of V(C6,O7) basin and the transformation of C6-N1 single bond into double one (SSD-IV). Finally, the appearance of V(O7,H2) basin lead to the elimination of water molecule within the last domain is observed.},
  archive      = {J_JCC},
  author       = {Mbah Bake Maraf and Adjieufack Abel Idrice and Manwal A. Mekoung Pélagie and Auguste Abouem A. Zintchem and Gouet Bebga and Lydia Rhyman and Mbouombouo Ndassa Ibrahim and Ponnadurai Ramasami},
  doi          = {10.1002/jcc.26853},
  journal      = {Journal of Computational Chemistry},
  number       = {14},
  pages        = {972-985},
  shortjournal = {J. Comput. Chem.},
  title        = {Decoding the reaction mechanism of the cyclocondensation of ethyl acetate2-oxo-2-(4-oxo-4H-pyrido [1.2-a] pyrimidin-3-yl) polyazaheterocycle and ethylenediamine using bond evolution theory},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mechanistic study on the coupling reaction of CO2 with
propylene oxide catalyzed by (CH3)4PI·MgCl2. <em>JCC</em>,
<em>43</em>(14), 961–971. (<a
href="https://doi.org/10.1002/jcc.26852">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The mechanistic study of CO2coupling with propylene oxide (PO) into cyclic carbonate catalyzed by (CH3)4PI has been investigated using the B3LYP/6-311++G (d, p)/B3LYP/6-31G (d) level of theory for non-iodine atoms and LANL2DZ was used, together with its associated basis set for the iodine atom. Two hypothetical reaction mechanisms were proposed for the studied reaction and thermodynamic and kinetic parameters were computed for each step to determine the more favorable route. The density functional theory (DFT) study reveals that the reaction prefers to proceed through a three-step mechanism (pathway II) than a tri-molecular intermediate (pathway I) where the CO2and the catalyst act simultaneously on the PO ring. The rate-determining step of the catalytic reaction is found to be the ring-opening step with an energy barrier of 27.1 kcal/mol (pathway II) in the gas phase, which is kinetically more favorable than that of non-catalytic CO2fixation with a relatively higher barrier of 63.7 kcal/mol. The synergetic effect of MgCl2is tested as a cocatalyst for the (CH3)4PI/MgCl2catalyzed reaction and it gave a better result and minimized the activation energy for the reaction and the rate-determining step was the ring closure with the free energy of activation 18.8 kcal/mol in the gas phase. The polarizable continuum model was used to account for the solvent effect, obtaining the best results of 23.1 kcal/mol in water for pathway I and 16.5 kcal/mol and 14.9 kcal/mol in dimethyl sulfoxide for pathway II and binary system, respectively.},
  archive      = {J_JCC},
  author       = {Jemal Mohamed Ali and Ahmed Mustefa Mohammed and Yedilfana Setarge Mekonnen},
  doi          = {10.1002/jcc.26852},
  journal      = {Journal of Computational Chemistry},
  number       = {14},
  pages        = {961-971},
  shortjournal = {J. Comput. Chem.},
  title        = {Mechanistic study on the coupling reaction of CO2 with propylene oxide catalyzed by (CH3)4PI·MgCl2},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Electronic structure, sodium diffusion and redox potentials
of low-symmetry NaMFe(MoO4)3 (m = mg, ni). <em>JCC</em>,
<em>43</em>(14), 953–960. (<a
href="https://doi.org/10.1002/jcc.26851">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sodium polyanionic compounds with transition metals are of considerable research interest for the search of new cathode materials for sodium-ion batteries. In this work, we employedab initiocalculations to evaluate three key battery properties: phase stability, diffusion barriers and redox voltage of low-symmetry molybdates NaMFe(MoO4)3(M= Mg, Ni) with α- and β-NaFe2(MoO4)3structures. First, we showed that the HubbardUcorrection within the GGA +Uapproach is necessary to correctly refer these molybdates to semiconductors. The sodium diffusion along various pathways was thoroughly examined, which allowed us to establish the most probable pathways with the lowest migration barriers. These compounds have different directions of sodium diffusion with twice different barriers that was associated to the peculiarities of their crystal structure. The high potentials and structural stability during sodium extraction, along with low-diffusion barriers predicted for NaMFe(MoO4)3(M= Mg, Ni) indicate that these molybdates may be promising high-voltage cathode materials for sodium-ion batteries.},
  archive      = {J_JCC},
  author       = {Alexander V. Serdtsev and Nadezhda I. Medvedeva},
  doi          = {10.1002/jcc.26851},
  journal      = {Journal of Computational Chemistry},
  number       = {14},
  pages        = {953-960},
  shortjournal = {J. Comput. Chem.},
  title        = {Electronic structure, sodium diffusion and redox potentials of low-symmetry NaMFe(MoO4)3 (M = mg, ni)},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Folding molecular dynamics simulation of t-peptide, a HIV
viral entry inhibitor: Structure, dynamics, and comparison with the
experimental data. <em>JCC</em>, <em>43</em>(14), 942–952. (<a
href="https://doi.org/10.1002/jcc.26850">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Peptide T is a synthetic octapeptide fragment, which corresponds to the region 185–192 of the gp120 HIV coat protein and functions as a viral entry inhibitor. In this work, a folding molecular dynamics simulation of peptide T in a membrane-mimicking (DMSO) solution was performed with the aim of characterizing the peptide&#39;s structural and dynamical properties. We show that peptide T is highly flexible and dynamic. The main structural characteristics observed were rapidly interconverting short helical stretches and turns, with a notable preference for the formation ofβ-turns. The simulation also indicated that the C-terminal part appears to be more stable than the rest of the peptide, with the most preferred conformation for residues 5–8 being aβ-turn. In order to validate the accuracy of the simulations, we compared our results with the experimental NMR data obtained for the T-peptide in the same solvent. In agreement with the simulation, the NMR data indicated the presence of a preferred structure in solution that was consistent with aβ-turn comprising the four C-terminal residues. An additional comparison between the experimental and simulation-derived chemical shifts also showed a reasonable agreement between experiment and simulation, further validating the simulation-derived structural characterization of the T-peptide. We conclude that peptide folding simulations produce physically relevant results even when performed with organic solvents that were not part of the force field parameterization procedure.},
  archive      = {J_JCC},
  author       = {Ioanna Gkogka and Nicholas M. Glykos},
  doi          = {10.1002/jcc.26850},
  journal      = {Journal of Computational Chemistry},
  number       = {14},
  pages        = {942-952},
  shortjournal = {J. Comput. Chem.},
  title        = {Folding molecular dynamics simulation of T-peptide, a HIV viral entry inhibitor: Structure, dynamics, and comparison with the experimental data},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Prepareforleap: An automated tool for fast PDB-to-parameter
generation. <em>JCC</em>, <em>43</em>(13), 930–935. (<a
href="https://doi.org/10.1002/jcc.26847">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Setting up molecular dynamics simulations from experimentally determined structures is often complicated by a variety of factors, particularly the inclusion of carbohydrates, since these have several anomer types which can be linked in a variety of ways. Here we present a stand-alone tool implemented in the widely-used software CPPTRAJ that can be used to automate building structures and generating a “ready to run” parameter and coordinate file pair. This tool automatically identifies carbohydrate anomer type, configuration, linkage, and functional groups, and performs topology modifications (e.g., renaming residue/atom names) required to build the final system using state of the art GLYCAM force field parameters. It will also generate the necessary commands for bonding carbohydrates and creating any disulfide bonds.},
  archive      = {J_JCC},
  author       = {Daniel R. Roe and Christina Bergonzo},
  doi          = {10.1002/jcc.26847},
  journal      = {Journal of Computational Chemistry},
  number       = {13},
  pages        = {930-935},
  shortjournal = {J. Comput. Chem.},
  title        = {Prepareforleap: An automated tool for fast PDB-to-parameter generation},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Effects of oxidative adsorbates and cluster formation on the
electronic structure of nanodiamonds. <em>JCC</em>, <em>43</em>(13),
923–929. (<a href="https://doi.org/10.1002/jcc.26849">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Nanodiamonds (NDs) are modern high-potential materials relevant for applications in biomedicine, photocatalysis, and various other fields. Their electronic surface properties, especially in the liquid phase, are key to their function in the applications, but we show that they are sensitively modified by their interactions with the environment. Two important interaction modes are those with oxidative aqueous adsorbates as well as ND self-aggregation towards the formation of ND clusters. For planar diamond surfaces it is known that the electron density migrates from the diamond towards oxidative adsorbates, which is known as transfer doping. Here, we quantify this effect for highly curved NDs of varying sizes (35–147 C atoms) and surface terminations (H, OH, F), focusing on their interactions with the most abundant aqueous oxidative adsorbates (H3O+, O2, O3). We prove that the concept of transfer doping stays valid for the case of the high-curvature NDs and can be tuned via the ND&#39;s specific properties. Secondly, we investigate the electronic structures of clusters of NDs which are known to form in particular in aqueous dispersions. Upon cluster formation, we find that the optical gaps of the structures are significantly reduced, which explains why different experimental values were obtained for the optical gap of the same structures, and the cluster&#39;s LUMO shapes resemble atom-type orbitals, as in the case of isolated spherical NDs. Our findings have implications for ND applications as photocatalysts or electronic devices, where the specific electronic properties are key to the functionality of the ND material.},
  archive      = {J_JCC},
  author       = {Thorren Kirschbaum and Tristan Petit and Joachim Dzubiella and Annika Bande},
  doi          = {10.1002/jcc.26849},
  journal      = {Journal of Computational Chemistry},
  number       = {13},
  pages        = {923-929},
  shortjournal = {J. Comput. Chem.},
  title        = {Effects of oxidative adsorbates and cluster formation on the electronic structure of nanodiamonds},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Is conformation relevant for QSAR purposes? 2D chemical
representation in a 3D-QSAR perspective. <em>JCC</em>, <em>43</em>(13),
917–922. (<a href="https://doi.org/10.1002/jcc.26848">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Conformation has a key role in the mechanism of interaction between small molecules and biological receptors. However, encoding this type of information in molecular descriptors for the construction of robust quantitative structure–activity relationships (QSAR) models is not an easy task and, so far, the dependence of these models on such feature has not been thoroughly investigated. In the present study, the authors explore the effects of conformational information on a 3D-QSAR technique by comparing models built with descriptors that encode fully described tridimensional aspects (structures docked inside a biological target), with descriptors in which this information is suppressed (flat structures) or not fully described (structures with quantum-chemically optimized geometries). As a result, the validation parameters indicate that the robustness of the models seems to be more related to the alignment aspect of the structures than to how well their tridimensional features are described.},
  archive      = {J_JCC},
  author       = {Joyce K. Daré and Matheus P. Freitas},
  doi          = {10.1002/jcc.26848},
  journal      = {Journal of Computational Chemistry},
  number       = {13},
  pages        = {917-922},
  shortjournal = {J. Comput. Chem.},
  title        = {Is conformation relevant for QSAR purposes? 2D chemical representation in a 3D-QSAR perspective},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Conformations and binding pockets of HRas and its guanine
nucleotide exchange factors complexes in the guanosine triphosphate
exchange process. <em>JCC</em>, <em>43</em>(13), 906–916. (<a
href="https://doi.org/10.1002/jcc.26846">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The human Son of Sevenless (SOS) activates the signal-transduction protein Ras by forming the complex SOS·Ras and accelerating the guanosine triphosphate (GTP) exchange in Ras. Inhibition of SOS·Ras could regulate the function of Ras in cells and has emerged as an effective strategy for battling Ras related cancers. A key factor to the success of this approach is to understand the conformational change of Ras during the GTP exchange process. In this study, we perform an extensive molecular dynamics simulation to characterize the specific conformations of Ras without and with guanine nucleotide exchange factors (GEFs) of SOS, especially for the substates of State 1 of HRasGTP∙Mg2+. The potent binding pockets on the surfaces of the RasGDP∙Mg2+, the S1.1 and S1.2 substates in State 1 of RasGTP∙Mg2+and the ternary complexes with SOS are predicted, including the binding sites of other domains of SOS. These findings help to obtain a more thorough understanding of Ras functions in the GTP cycling process and provide a structural foundation for future drug design.},
  archive      = {J_JCC},
  author       = {Yuqing Xiong and Juan Zeng and Fei Xia and Qiang Cui and Xianming Deng and Xin Xu},
  doi          = {10.1002/jcc.26846},
  journal      = {Journal of Computational Chemistry},
  number       = {13},
  pages        = {906-916},
  shortjournal = {J. Comput. Chem.},
  title        = {Conformations and binding pockets of HRas and its guanine nucleotide exchange factors complexes in the guanosine triphosphate exchange process},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CSiGaAl2−/0 and CGeGaAl2−/0 having planar tetracoordinate
carbon atoms in their global minimum energy structures. <em>JCC</em>,
<em>43</em>(13), 894–905. (<a
href="https://doi.org/10.1002/jcc.26845">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Density functional theory (DFT) is used to explore the structure, stability, and bonding in CSiGaAl2−/0and CGeGaAl2−/0systems having planar tetracoordinate carbon (ptC). The neutral systems have 17 valence electrons and the mono-anionic systems have 18 valence electrons. The ab initio molecular dynamics simulations for 2000 fs time at two different temperatures (300 and 500 K) supported the kinetic stability of the systems. From the natural bond orbital (NBO) analysis it is shown that there is a strong electron donation from the ligand atoms to the ptC atom. We have used Li+ion for the neutralization of the mono-anionic systems and more interestingly it does not disrupt the planar structure. The most preferable site for binding of Li+ion is along the AlAl bond in both of the mono-anionic systems. All the systems in this work have both σ and π aromaticity which is predicted from the computations of nucleus independent chemical shift (NICS). Although the anionic species obey the 18 valence electronic rule, the neutral systems break the rule with 17 valence electrons. However, both sets of systems are stable in the planar form. The bonding analysis of the systems includes molecular orbital, adaptive natural density partitioning (AdNDP), quantum theory of atoms in molecules (QTAIM), electron localization function (ELF) basin, and aromaticity analyses. The energy decomposition analysis (EDA) determines the interaction of Li+ion with CSiGaAl2−and CGeGaAl2−in Li@SiGaAl2and Li@GeGaAl2, respectively.},
  archive      = {J_JCC},
  author       = {Prasenjit Das and Pratim Kumar Chattaraj},
  doi          = {10.1002/jcc.26845},
  journal      = {Journal of Computational Chemistry},
  number       = {13},
  pages        = {894-905},
  shortjournal = {J. Comput. Chem.},
  title        = {CSiGaAl2−/0 and CGeGaAl2−/0 having planar tetracoordinate carbon atoms in their global minimum energy structures},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Towards improved FAIRness of the ThermoML archive.
<em>JCC</em>, <em>43</em>(12), 879–887. (<a
href="https://doi.org/10.1002/jcc.26842">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The ThermoML Archive is a subset of Thermodynamics Research Center (TRC) data holdings corresponding to cooperation between NIST TRC and five journals:Journal of Chemical Engineering and Data(ISSN: 1520-5134),The Journal of Chemical Thermodynamics(ISSN: 1096-3626),Fluid Phase Equilibria(ISSN: 0378-3812),Thermochimica Acta(ISSN: 0040-6031), andInternational Journal of Thermophysics(ISSN: 1572-9567). Data from initial cooperation (around 2003) through the 2019 calendar year are included. The archive has undergone a major update with the goal of improving the FAIRness and user experience of the data provided by the service. The web application provides comprehensive property browsing and searching capabilities; searching relies on a RESTful API provided by the Cordra software for managing digital objects. JSON files with a schema derived from ThermoML are provided as an additional serialization to lower the barrier to programmatic consumption of the information, for stakeholders who may have a preference of JSON over XML. The ThermoML and JSON files for all available entries can be downloaded fromdata.nist.gov(https://data.nist.gov/od/id/mds2-2422).},
  archive      = {J_JCC},
  author       = {Demian Riccardi and Zachary Trautt and Ala Bazyleva and Eugene Paulechka and Vladimir Diky and Joseph W. Magee and Andrei F. Kazakov and Scott A. Townsend and Chris D. Muzny},
  doi          = {10.1002/jcc.26842},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {879-887},
  shortjournal = {J. Comput. Chem.},
  title        = {Towards improved FAIRness of the ThermoML archive},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Valence energy correction for electron reactive force field.
<em>JCC</em>, <em>43</em>(12), 870–878. (<a
href="https://doi.org/10.1002/jcc.26844">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reactive force fields (ReaxFF) are a classical method to describe material properties based on a bond-order formalism, that allows bond dissociation and consequently investigations of reactive systems. Semiclassical treatment of electrons was introduced within ReaxFF simulations, better known as electron reactive force fields (eReaxFF), to explicitly treat electrons as spherical Gaussian waves. In the original version of eReaxFF, the electrons and electron–holes can lead to changes in both the bond energy and the Coulomb energy of the system. In the present study, the method was modified to allow an electron to modify the valence energy, therefore, permitting that the electron&#39;s presence modifies the three-body interactions, affecting the angle among three atoms. When a reaction path involving electron transfer is more sensitive to the geometric configuration of the molecules, corrections in the angular structure in the presence of electrons become more relevant; in this case, bond dissociation may not be enough to describe a reaction path. Consequently, the application of the extended eReaxFF method developed in this work should provide an improved description of a reaction path. As a first demonstration this semiclassical force field was parametrized for hydrogen and oxygen interactions, including water and water&#39;s ions. With the modified methodology both the overall accuracy of the force field but also the description of the angles within the molecules in presence of electrons could be improved.},
  archive      = {J_JCC},
  author       = {Samuel Bertolini and Timo Jacob},
  doi          = {10.1002/jcc.26844},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {870-878},
  shortjournal = {J. Comput. Chem.},
  title        = {Valence energy correction for electron reactive force field},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The effect of off-center σ-hole on the atom-centered partial
charges in halogenated molecules. <em>JCC</em>, <em>43</em>(12),
864–869. (<a href="https://doi.org/10.1002/jcc.26841">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Partial atomic charges belong to key concepts of computational chemistry. In some cases, however, they fail in describing the electrostatics of molecules. One such example is theσ-hole, a region of positive electrostatic potential located on halogens and other atoms. In molecular mechanics, theσ-hole is often modeled as a pseudo-atom with a positive partial charge located off the halogen nucleus. Here we address a question, to what extent the pseudo-atom affects partial charges of other atoms in the molecule. To this aim, we have thoroughly analyzed partial charges of over 2300 halogenated molecules from the ZINC database calculated by the restricted electrostatic potential (RESP) method and compared them with the charges fitted by RESP including the pseudo-atom. We show that the pseudo-atom improves charge fitting for a vast majority of molecules. Theσ-hole, modeled as the off-center charge, affects the atoms within three covalent bonds from the halogen.},
  archive      = {J_JCC},
  author       = {Aneta Leskourová and Michal H. Kolář},
  doi          = {10.1002/jcc.26841},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {864-869},
  shortjournal = {J. Comput. Chem.},
  title        = {The effect of off-center σ-hole on the atom-centered partial charges in halogenated molecules},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). PharmRF: A machine-learning scoring function to identify the
best protein-ligand complexes for structure-based pharmacophore
screening with high enrichments. <em>JCC</em>, <em>43</em>(12), 847–863.
(<a href="https://doi.org/10.1002/jcc.26840">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Structure-based pharmacophore models are often developed by selecting a single protein-ligand complex with good resolution and better binding affinity data which prevents the analysis of other structures having a similar potential to act as better templates. PharmRF is a pharmacophore-based scoring function for selecting the best crystal structures with the potential to attain high enrichment rates in pharmacophore-based virtual screening prospectively. The PharmRF scoring function is trained and tested on the PDBbind v2018 protein-ligand complex dataset and employs a random forest regressor to correlate protein pocket descriptors and ligand pharmacophoric elements with binding affinity. PharmRF score represents the calculated binding affinity which identifies high-affinity ligands by thorough pruning of all the PDB entries available for a particular protein of interest with a high PharmRF score. Ligands with high PharmRF scores can provide a better basis for structure-based pharmacophore enumerations with a better enrichment rate. Evaluated on 10 protein-ligand systems of the DUD-E dataset, PharmRF achieved superior performance (average success rate: 77.61\%, median success rate: 87.16\%) than Vina docking score (75.47\%, 79.39\%). PharmRF was further evaluated using the CASF-2016 benchmark set yielding a moderate correlation of 0.591 with experimental binding affinity, similar in performance to 25 scoring functions tested on this dataset. Independent assessment of PharmRF on 8 protein-ligand systems of LIT-PCBA dataset exhibited average and median success rates of 57.55\% and 74.72\% with 4 targets attaining success rate &gt; 90\%. The PharmRF scoring model, scripts, and related resources can be accessed athttps://github.com/Prasanth-Kumar87/PharmRF.},
  archive      = {J_JCC},
  author       = {Sivakumar Prasanth Kumar and Nandan Y. Dixit and Chirag N. Patel and Rakesh M. Rawal and Himanshu A. Pandya},
  doi          = {10.1002/jcc.26840},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {847-863},
  shortjournal = {J. Comput. Chem.},
  title        = {PharmRF: A machine-learning scoring function to identify the best protein-ligand complexes for structure-based pharmacophore screening with high enrichments},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). BSSE-corrected consistent gaussian basis sets of triple-zeta
valence with polarization quality of the fifth period for solid-state
calculations. <em>JCC</em>, <em>43</em>(12), 839–846. (<a
href="https://doi.org/10.1002/jcc.26839">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Revised versions of our published pob-TZVP basis sets [Laun, J.; Vilela Oliveira, D. and Bredow, T., J. Comput. Chem., 2018, 39 (19), 1285–1290] have been derived for periodic quantum-chemical solid-state calculations. They complete our pob-TZVP-rev2 series [Vilela Oliveira, D.; Laun, J.; Peintinger, M. F. and Bredow, T., J. Comput. Chem., 2019, 40 (27), 2364–2376 and Laun, J. and Bredow, J. Comput. Chem. 2021; 42 (15), 1064–1072] for the elements of the fifth period and are based on the fully relativistic effective core potentials (ECPs) of the Stuttgart/Cologne group and the def2-TZVP valence basis of the Ahlrichs group. The pob-TZVP-rev2 basis sets are developed to minimize the basis set superposition error (BSSE) in crystalline systems. For the applied PW1PW hybrid functional, the overall performance, transferability, and SCF stability of the resulting pob-TZVP-rev2 basis sets are significantly improved compared to the original pob-TZVP basis sets. After augmentation with single diffuses- andp-functions, reference plane-wave band structures of metals can be accurately reproduced.},
  archive      = {J_JCC},
  author       = {Joachim Laun and Thomas Bredow},
  doi          = {10.1002/jcc.26839},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {839-846},
  shortjournal = {J. Comput. Chem.},
  title        = {BSSE-corrected consistent gaussian basis sets of triple-zeta valence with polarization quality of the fifth period for solid-state calculations},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Geometries, electronic structures, and bonding properties of
endohedral group-14 zintl clusters TM@E10 (TM = fe, co, ni; e = ge, sn,
pb). <em>JCC</em>, <em>43</em>(12), 828–838. (<a
href="https://doi.org/10.1002/jcc.26838">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The geometries, electronic structures, and bonding properties of the title endohedral Zintl clusters have been studied by using ab initio calculations. [Fe@Ge10]4−and [Co@Ge10]3−have D5h-symmetric pentagonal prismatic structure and [Fe@Sn10]4−adopts the C2v-symmetric structure as their ground-state structures, whereas all the other clusters possess D4dbicapped square antiprismatic structures, in consistent with the experimental values when available. Natural bonding orbital and electron localization function disclosed that the negative charges are localized on the central atoms rather than the cages while the TME ionic bonding interactions increase in the order of Ge &lt; Sn &lt; Pb. The energy decomposition analysis revealed that the total bonding energy ∆Eintbetween central TM and E10cage is above 150 kcal/mol. The ionic bonding interaction termed as electrostatic interaction ∆Eelstatincreases in the order of Ge &lt; Sn &lt; Pb and becomes higher than the covalent bonding interactions termed as total orbital interactions ∆Eorb. Among the total orbital interactions, the π back donations from the TM–d orbitals to the empty cage orbitals consisting of E-p orbitals, the magnitude of which is importantly affected by the cage symmetry, are dominant contributions.},
  archive      = {J_JCC},
  author       = {Dong Liu and Song Xu and Gerui Pei and Jianzhi Xu and Xintian Zhao and Chuncai Kong and Zhimao Yang and Tao Yang},
  doi          = {10.1002/jcc.26838},
  journal      = {Journal of Computational Chemistry},
  number       = {12},
  pages        = {828-838},
  shortjournal = {J. Comput. Chem.},
  title        = {Geometries, electronic structures, and bonding properties of endohedral group-14 zintl clusters TM@E10 (TM = fe, co, ni; e = ge, sn, pb)},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Influence of conventional hydrogen bonds in the
intercalation of phenanthroline derivatives with DNA: The important role
of the sugar and phosphate backbone. <em>JCC</em>, <em>43</em>(11),
804–821. (<a href="https://doi.org/10.1002/jcc.26836">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The influence of hydrogen bonds in model intercalated systems between guanine-cytosine and adenine-thymine DNA base pairs (bps) was analyzed with the popular intercalator 1,10-phenanthroline (phen) and derivatives obtained by substitution withOH andNH2groups in positions 4 and 7. Semiempirical and Density Functional Theory (DFT) methods were used both including dispersion effects: PM6-DH2, M06-2X and B3LYP-D3 along with the recently developed near linear-scaling coupled cluster method DLPNO-CCSD(T) for benchmark calculations. Our results given by QTAIM and non-covalent interaction analysis confirmed the existence of hydrogen bonds created byOH andNH2. The trends in the energy decomposition analysis for the interaction energy, ΔEint, showed that the ΔEelstatcontributions are equal or even a little bit higher than the values for ΔEdisp. Such important ΔEelstatattractive contribution comes mainly from the conventional hydrogen bonds formed byOH andNH2functional groups with DNA not only with bps but specially with the sugar and phosphate backbone. This behavior is very different from that of phen and other classical intercalators that cannot form conventional hydrogen bonds, where the ΔEdispis the most important attractive contribution to the ΔEint. The inclusion of explicit water molecules in molecular dynamics simulations showed, as a general trend, that the hydrogen bonds with the bps disappear during the simulations but those with the sugar and phosphate backbone remain in time, which highlights the important role of the sugar and phosphate backbone in the stabilization of these systems.},
  archive      = {J_JCC},
  author       = {Ángel Sánchez-González and Pierre Grenut and Adrià Gil},
  doi          = {10.1002/jcc.26836},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {804-821},
  shortjournal = {J. Comput. Chem.},
  title        = {Influence of conventional hydrogen bonds in the intercalation of phenanthroline derivatives with DNA: The important role of the sugar and phosphate backbone},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Bonding situation in isolable silver(i) carbonyl complexes
of the scorpionates. <em>JCC</em>, <em>43</em>(11), 796–803. (<a
href="https://doi.org/10.1002/jcc.26835">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The bonding situation of Ag(I)CO complexes having a Scorpionate ligand directly attached to the transition metal has been analyzed in detail by means of relativistic density functional theory calculations. To this end, different experimentally characterized complexes together with other representative species have been considered to rationalize the observed shift of the correspondingν(CO) stretching frequencies and the influence of the substituents in the Scorpionate ligand. With the help of the energy decomposition analysis method combined with the natural orbital for chemical valence it is found that the main contribution to the bonding comes from the electrostatic attractions between the LAg(I) and CO fragments. Despite that, the LAg → CO π-backdonation is also significant in these species as well as in related LCu(I)CO complexes.},
  archive      = {J_JCC},
  author       = {Israel Fernández and Anurag Noonikara-Poyil and H. V. Rasika Dias},
  doi          = {10.1002/jcc.26835},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {796-803},
  shortjournal = {J. Comput. Chem.},
  title        = {Bonding situation in isolable silver(I) carbonyl complexes of the scorpionates},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A real space picture of the role of steric effects in SN2
reactions. <em>JCC</em>, <em>43</em>(11), 785–795. (<a
href="https://doi.org/10.1002/jcc.26834">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Within substitution reactions, the Bimolecular Nucleophilic Substitution (SN2) reaction mechanism is one of the most frequently found and studied ones. Among other factors, the easiness of the SN2 pathway is classically considered to be determined by steric hindrance. However, the diffuse nature of the latter inevitably darkens these and other arguments holding the pillars of chemical intuition. In this work, we employ the steric energy (EST) descriptor, formulated within the Interacting Quantum Atoms approach, to offer insights regarding this problem. The steric demands of the substrate, nucleophile and leaving group were studied using the gas-phase SN2 reaction with different organic skeletons (CH3, CH3CH2, (CH3)2CH, (CH3)3C, (CH3)3CCH2) and halogens (F, Cl, and Br) as test-bed systems. Our results show that, according to EST, the SH experienced along these simple reactions fits, in the general case, the trends predicted by a meticulous and rigorous application of chemical intuition. However, steric clash alone should not be considered as the only argument used to explain the easiness of the SN2 reaction over different electrophiles.},
  archive      = {J_JCC},
  author       = {Miguel Gallegos and Aurora Costales and Ángel Martín Pendás},
  doi          = {10.1002/jcc.26834},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {785-795},
  shortjournal = {J. Comput. Chem.},
  title        = {A real space picture of the role of steric effects in SN2 reactions},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Di- and trioxides of triazolotetrazine: Computational
prediction of crystal structures and estimation of physicochemical
characteristics. <em>JCC</em>, <em>43</em>(11), 778–784. (<a
href="https://doi.org/10.1002/jcc.26833">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Simulation of crystal structures of series 1(2)-R-1(2)H-[1,2,3]triazolo[4,5-e][1,2,3,4]tetrazine 5,7-dioxides, 1,5,7-trioxides, 4,6-dioxides and 3,4,6-trioxides was carried out using an original technique based on the method of atom-atom potentials and quantum chemistry. The effect of the position of the substituent in the triazole ring on the change in the crystal structures of these compounds and their thermochemical characteristics was studied for the first time. For some of synthesized compounds, thermochemical characteristics were investigated and differential scanning calorimetry curves were obtained. Detonation parameters were calculated, on the basis of which the prospects for the use of the considered compounds were assessed.},
  archive      = {J_JCC},
  author       = {Dmitry V. Khakimov and Victor P. Zelenov and Tatyana S. Pivina},
  doi          = {10.1002/jcc.26833},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {778-784},
  shortjournal = {J. Comput. Chem.},
  title        = {Di- and trioxides of triazolotetrazine: Computational prediction of crystal structures and estimation of physicochemical characteristics},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). EDA-NOCV analysis of carbene-borylene bonded dinitrogen
complexes for deeper bonding insight: A fair comparison with a
metal-dinitrogen system. <em>JCC</em>, <em>43</em>(11), 757–777. (<a
href="https://doi.org/10.1002/jcc.26832">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Binding of dinitrogen (N2) to a transition metal center (M) and followed by its activation under milder conditions is no longer impossible; rather, it is routinely studied in laboratories by transition metal complexes. In contrast, binding of N2by main group elements has been a challenge for decades, until very recently, an exotic cAAC-borylene (cAAC = cyclic alkyl(amino) carbene) species showed similar binding affinity to kinetically inert and non-polar dinitrogen (N2) gas under ambient conditions. Since then, N2binding by short lived borylene species has made a captivating news in different journals for its unusual features and future prospects. Herein, we carried out different types of DFT calculations, including EDA-NOCV analysis of the relevant cAAC-boron-dinitrogen complexes and their precursors, to shed light on the deeper insight of the bonding secret (EDA-NOCV = energy decomposition analysis coupled with natural orbital for chemical valence). The hidden bonding aspects have been uncovered and are presented in details. Additionally, similar calculations have been carried out in comparison with a selected stable dinitrogen bridged-diiron(I) complex. Singlet cAAC ligand is known to be an exotic stable species which, combined with the BAr group, produces an intermediate singlet electron-deficient (cAAC)(BAr) species possessing a high lying HOMO suitable for overlapping with the high lying π*-orbital of N2via effective π-backdonation. The BN2interaction energy has been compared with that of the FeN2bond. Our thorough bonding analysis might answer the unasked questions of experimental chemists about how boron compounds could mimic the transition metal of dinitrogen binding and activation, uncovering hidden bonding aspects. Importantly, Pauling repulsion energy also plays a crucial role and decides the binding efficiency in terms of intrinsic interaction energy between the boron-center and the N2ligand.},
  archive      = {J_JCC},
  author       = {Kavita Devi and Sai Manoj N. V. T. Gorantla and Kartik Chandra Mondal},
  doi          = {10.1002/jcc.26832},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {757-777},
  shortjournal = {J. Comput. Chem.},
  title        = {EDA-NOCV analysis of carbene-borylene bonded dinitrogen complexes for deeper bonding insight: A fair comparison with a metal-dinitrogen system},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Probing the range of applicability of structure- and
energy-adjusted QM/MM link bonds II: Optimized link bond parameters for
density functional tight binding approaches. <em>JCC</em>,
<em>43</em>(11), 746–756. (<a
href="https://doi.org/10.1002/jcc.26830">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optimized link bond parameters for the CαCβbond of 22 different capped amino acid model systems have been determined at SCC DFTB/mio (self-consistent charge density functional tight-binding), SCC DFTB/3ob and GFNn-xTB (n= 0, 1, and 2) level in conjunction with the AMBER 99SB, 14SB, and 19B force fields. The resulting parameter sets have been compared to newly calculated reference data obtained via resolution-of-identity 2nd order Møller–Plesset perturbation theory. The data collected in this work suggests that the optimized values in this study provide a more suitable setup of the QM/MM link bonds compared to the use of a single global setting applied to every amino acid fragmented by the QM/MM interface. The results also imply that a transfer of the ideal link bond settings between different levels of theory is not advised. In contrast, virtually identical parameters were obtained in calculations employing different variants of the AMBER force field. Considering the increasing success of tight binding based approaches being inter alia a results of their exceptional accuracy/effort ratio the provided collection of link atoms parameters provides a valuable resource for QM/MM studies of biomacromolecular systems as demonstrated in an exemplary QM/MM MD simulation of theβ-amyloid/Zn2+complex.},
  archive      = {J_JCC},
  author       = {Hans Georg Gallmetzer and Thomas S. Hofer},
  doi          = {10.1002/jcc.26830},
  journal      = {Journal of Computational Chemistry},
  number       = {11},
  pages        = {746-756},
  shortjournal = {J. Comput. Chem.},
  title        = {Probing the range of applicability of structure- and energy-adjusted QM/MM link bonds II: Optimized link bond parameters for density functional tight binding approaches},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Complex machine learning model needs complex testing:
Examining predictability of molecular binding affinity by a graph neural
network. <em>JCC</em>, <em>43</em>(10), 728–739. (<a
href="https://doi.org/10.1002/jcc.26831">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Drug discovery pipelines typically involve high-throughput screening of large amounts of compounds in a search of potential drugs candidates. As a chemical space of small organic molecules is huge, a “navigation” over it urges for fast and lightweight computational methods, thus promoting machine-learning approaches for processing huge pools of candidates. In this contribution, we present a graph-based deep neural network for prediction of protein-drug binding affinity and assess its predictive power under thorough testing conditions. Within the suggested approach, both protein and drug molecules are represented as graphs and passed to separate graph sub-networks, then concatenated and regressed towards a binding affinity. The neural network is trained on two binding affinity datasets—PDBbind and data imported from RCSB Protein Data Bank. In order to explore the generalization capabilities of the model we go beyond traditional random or leave-cluster-out techniques and demonstrate the need for more elaborate model performance assessment – six different strategies for test/train data partitioning (random, time- and property-arranged, protein- and ligand-clustered) with a k-fold cross-validation are engaged. Finally, we discuss the model performance in terms of a set of metrics for different split strategies and fold arrangement. Our code is available athttps://github.com/SoftServeInc/affinity-by-GNN.},
  archive      = {J_JCC},
  author       = {Tymofii Nikolaienko and Oleksandr Gurbych and Maksym Druchok},
  doi          = {10.1002/jcc.26831},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {728-739},
  shortjournal = {J. Comput. Chem.},
  title        = {Complex machine learning model needs complex testing: Examining predictability of molecular binding affinity by a graph neural network},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An ETS-NOCV-based computational strategies for the
characterization of concerted transition states involving CO2.
<em>JCC</em>, <em>43</em>(10), 717–727. (<a
href="https://doi.org/10.1002/jcc.26829">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Due to the presence of both a slightly acidic carbon and a slightly basic oxygen, carbon dioxide is often involved in concerted transition states (TSs) with two (or more) different molecular events interlaced in the same step. The possibility of isolating and quantitatively evaluating each molecular event would be important to characterize and understand the reaction mechanism in depth. This could be done, in principle, by measuring the relevant distances in the optimized TS, but often distances are not accurate enough, especially in the presence of many simultaneous processes. Here, we have applied the Extended Transition State-Natural Orbital for Chemical Valence-method (ETS-NOCV), also in combination with the Activation Strain Model (ASM) and Energy Decomposition Analysis (EDA), to separate and quantify these molecular events at the TS of both organometallic and organic reactions. For the former, we chose the decomposition of formic acid to CO2by an iridium catalyst, and for the latter, a CO2-mediated transamidation and its chemical variations (hydro- and aminolysis of an ester) as case studies. We demonstrate that the one-to-one mapping between the “molecular events” and the ETS-NOCV components is maintained along the entire lowest energy path connecting reactants and products around the TS, thus enabling a detailed picture on the relative importance of each interacting component. The methodology proposed here provides valuable insights into the effect of different chemical substituents on the reaction mechanism and promises to be generally applicable for any concerted TSs.},
  archive      = {J_JCC},
  author       = {Diego Sorbelli and Paola Belanzoni and Leonardo Belpassi and Ji-Woong Lee and Gianluca Ciancaleoni},
  doi          = {10.1002/jcc.26829},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {717-727},
  shortjournal = {J. Comput. Chem.},
  title        = {An ETS-NOCV-based computational strategies for the characterization of concerted transition states involving CO2},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Structures and properties of ionic crystals and condensed
phase ionic liquids predicted with the generalized energy-based
fragmentation method. <em>JCC</em>, <em>43</em>(10), 704–716. (<a
href="https://doi.org/10.1002/jcc.26828">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The generalized energy-based fragmentation (GEBF) approach is extended to facilitate ab initio investigations of structures, lattice energies, vibrational spectra and1H NMR chemical shifts of ionic crystals and condensed-phase ionic liquids (ILs) with the periodic boundary conditions (PBC). For selected periodic systems, our results demonstrate that the so-called PBC-GEBF approach can provide satisfactory descriptions on ground-state energies, structures, and vibrational spectra of ionic crystals and IL crystals. The PBC-GEBF approach is then applied to three realistic condensed phase systems. For three ionic crystals (LiCl, NaCl, and KCl), we apply the PBC-GEBF approach with MP2 theory as well as some popular DFT methods to investigate their crystal structures and lattice energies. Our calculations indicate that the crystal structures obtained with PBC-GEBF-MP2/6–311 + G** are very close to the corresponding X-ray structures, while PBC-GEBF-ωB97X-D/6–311 + G** provides satisfactory prediction for crystal structures and lattice energies. For two polymorphs of [n-C4mim][Cl] crystals, we find that the PBC-GEBF approach at the M06-2X/6–311 + G** level can give a satisfactory descriptions on structures and Raman spectra of these two crystals. Furthermore, for [C2mim][BF4] ILs, we demonstrate that their1H NMR chemical shifts can be estimated from averaging over 5 typical snapshots (extracted from MD simulations) with the PBC-GEBF approach at the B97-2/pcSseg-2 level. The calculated results account for the observed experimental data quite well. Therefore, we expect that the PBC-GEBF approach, combined with various quantum chemistry methods, will become an effective tool in predicting structures and properties of ionic crystals and condensed-phase ILs.},
  archive      = {J_JCC},
  author       = {Yunzhi Li and Dong Wang and Fangjia Fu and Qiying Xia and Wei Li and Shuhua Li},
  doi          = {10.1002/jcc.26828},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {704-716},
  shortjournal = {J. Comput. Chem.},
  title        = {Structures and properties of ionic crystals and condensed phase ionic liquids predicted with the generalized energy-based fragmentation method},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Deep generative models for ligand-based de novo design
applied to multi-parametric optimization. <em>JCC</em>, <em>43</em>(10),
692–703. (<a href="https://doi.org/10.1002/jcc.26826">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Multi-parameter optimization (MPO) is a major challenge in new chemical entity (NCE) drug discovery. Recently, promising results were reported for deep learning generative models applied to de novo molecular design, but, to our knowledge, until now no report was made of the value of this new technology for addressing MPO in an actual drug discovery project. In this study, we demonstrate the benefit of applying AI technology in a real drug discovery project. We evaluate the potential of a ligand-based de novo design technology using deep learning generative models to accelerate the obtention of lead compounds meeting 11 different biological activity objectives simultaneously. Using the initial dataset of the project, we built QSAR models for all the 11 objectives, with moderate to high performance (precision between 0.67 and 1.0 on an independent test set). Our DL-based AI de novo design algorithm, combined with the QSAR models, generated 150 virtual compounds predicted as active on all objectives. Eleven were synthetized and tested. The AI-designed compounds met 9.5 objectives on average (i.e., 86\% success rate) versus 6.4 (i.e., 58\% success rate) for the initial molecules measured on all objectives. One of the AI-designed molecules was active on all 11 measured objectives, and two were active on 10 objectives while being in the error margin of the assay for the last one. The AI algorithm designed compounds with functional groups, which, although being rare or absent in the initial dataset, turned out to be highly beneficial for the MPO.},
  archive      = {J_JCC},
  author       = {Quentin Perron and Olivier Mirguet and Hamza Tajmouati and Adam Skiredj and Anne Rojas and Arnaud Gohier and Pierre Ducrot and Marie-Pierre Bourguignon and Patricia Sansilvestri-Morel and Nicolas Do Huu and Françoise Gellibert and Yann Gaston-Mathé},
  doi          = {10.1002/jcc.26826},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {692-703},
  shortjournal = {J. Comput. Chem.},
  title        = {Deep generative models for ligand-based de novo design applied to multi-parametric optimization},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Towards optimal boundary integral formulations of the
poisson–boltzmann equation for molecular electrostatics. <em>JCC</em>,
<em>43</em>(10), 674–691. (<a
href="https://doi.org/10.1002/jcc.26825">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Poisson-Boltzmann equation offers an efficient way to study electrostatics in molecular settings. Its numerical solution with the boundary element method is widely used, as the complicated molecular surface is accurately represented by the mesh, and the point charges are accounted for explicitly. In fact, there are several well-known boundary integral formulations available in the literature. This work presents a generalized expression of the boundary integral representation of the implicit solvent model, giving rise to new forms to compute the electrostatic potential. Moreover, it proposes a strategy to build efficient preconditioners for any of the resulting systems, improving the convergence of the linear solver. We perform systematic benchmarking of a set of formulations and preconditioners, focusing on the time to solution, matrix conditioning, and eigenvalue spectrum. We see that the eigenvalue clustering is a good indicator of the matrix conditioning, and show that they can be easily manipulated by scaling the preconditioner. Our results suggest that the optimal choice is problem-size dependent, where a simpler direct formulation is the fastest for small molecules, but more involved second-kind equations are better for larger problems. We also present a fast Calderón preconditioner for first-kind formulations, which shows promising behavior for future analysis. This work sets the basis towards choosing the most convenient boundary integral formulation of the Poisson–Boltzmann equation for a given problem.},
  archive      = {J_JCC},
  author       = {Stefan D. Search and Christopher D. Cooper and Elwin van&#39;t Wout},
  doi          = {10.1002/jcc.26825},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {674-691},
  shortjournal = {J. Comput. Chem.},
  title        = {Towards optimal boundary integral formulations of the Poisson–Boltzmann equation for molecular electrostatics},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Separation of hydrocarbon mixture of neopentane and n-hexane
using NaY zeolite: Large distinct diffusivity. <em>JCC</em>,
<em>43</em>(10), 660–673. (<a
href="https://doi.org/10.1002/jcc.26824">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recently proposed method based on Levitation and Blow torch effects, is employed here to see if it can separate a mixture of neopentane andn-hexane. The results show that the mixture can be separated with a hot zone temperature of just 40 K above the ambient temperature, 300 K. The two components are found to accumulate at the two extreme ends of the zeolite column. The computed separation factor is in the range of 1015–1020(as compared to 104for existing separation methods). The energy expense for the separation is significantly smaller than for existing separation methods by several orders of magnitude. Transport (D11), self (Ds), and distinct diffusivities (Dd) of the mixture were computed. The contribution of distinct diffusivity to the transport diffusivity is 70\% as compared to 10\%–30\% seen in other separation methods and is larger by 2.3 times as compared to the self-diffusivity.},
  archive      = {J_JCC},
  author       = {Shubhadeep Nag and Yashonath Subramanian},
  doi          = {10.1002/jcc.26824},
  journal      = {Journal of Computational Chemistry},
  number       = {10},
  pages        = {660-673},
  shortjournal = {J. Comput. Chem.},
  title        = {Separation of hydrocarbon mixture of neopentane and n-hexane using NaY zeolite: Large distinct diffusivity},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Simultaneous parametrization of torsional and third-neighbor
interaction terms in force-field development: The LLS-SC algorithm.
<em>JCC</em>, <em>43</em>(9), 644–653. (<a
href="https://doi.org/10.1002/jcc.26819">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The calibration of torsional interaction terms by fitting relative gas-phase conformational energies against their quantum-mechanical values is a common procedure in force-field development. However, much less attention has been paid to the optimization of third-neighbor nonbonded interaction parameters, despite their strong coupling with the torsions. This article introduces an algorithm termed LLS-SC, aimed at simultaneously parametrizing torsional and third-neighbor interaction terms based on relative conformational energies. It relies on a self-consistent (SC) procedure where each iteration involves a linear least-squares (LLS) regression followed by a geometry optimization of the reference structures. As a proof-of-principle, this method is applied to obtain torsional and third-neighbor interaction parameters for aliphatic chains in the context of the GROMOS 53A6 united-atom force field. The optimized parameter set is compared to the original one, which has been fitted manually against thermodynamic properties for small linear alkanes. The LLS-SC implementation is freely available underhttp://github.com/mssm-labmmol/profiler.},
  archive      = {J_JCC},
  author       = {Yan M. H. Gonçalves and Sadra Kashefolgheta and Marina P. Oliveira and Philippe H. Hünenberger and Bruno A. C. Horta},
  doi          = {10.1002/jcc.26819},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {644-653},
  shortjournal = {J. Comput. Chem.},
  title        = {Simultaneous parametrization of torsional and third-neighbor interaction terms in force-field development: The LLS-SC algorithm},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Guide to tuning the chalcone molecular properties based on
the push-pull effect energy scale created via the molecular tailoring
approach. <em>JCC</em>, <em>43</em>(9), 631–643. (<a
href="https://doi.org/10.1002/jcc.26827">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Using the molecular tailoring approach, a total energy scale for the push-pull effect in the range from −40 to 100 kcal/mol is created for the wide series of neutral, charged and doubly charged compounds on the chalcone platform. Taking into account similar energy scale for hydrogen bonds, the strength of the push-pull effect is ranked in the seven categories, ranging from negative (anti-push-pull) to very weak and very strong push-pull effect. It is demonstrated that the molecular properties of chalcone can be tuned prior synthesis due to the created energy scale for the push-pull effect. The single bonds of the π-spacer in the chalcones are shortened, the double ones are lengthened, and the C=O bond vibrations are red shifted when the push-pull effect is enhanced along the energy scale. The HOMO and LUMO energies change systematically while the HOMO–LUMO energy gap narrows as the strength of the push–pull effect increases.},
  archive      = {J_JCC},
  author       = {Andrei V. Afonin and Danuta Rusinska-Roszak},
  doi          = {10.1002/jcc.26827},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {631-643},
  shortjournal = {J. Comput. Chem.},
  title        = {Guide to tuning the chalcone molecular properties based on the push-pull effect energy scale created via the molecular tailoring approach},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Identification of novel inhibitors for prp protein of
mycobacterium tuberculosis by structure based drug design, and molecular
dynamics simulations. <em>JCC</em>, <em>43</em>(9), 619–630. (<a
href="https://doi.org/10.1002/jcc.26823">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this study, we assess the effective inhibition of a series of thiazolidine derivatives (1a–1q) were adopting structure-based drug design. Thiazolidine is a five-membered ring structure with thioether and amino groups at positions 1 and 3. Although, thiazolidine may bind to a wide range of protein targets, it is a major heterocyclic core in medicinal chemistry. Different scoring utilities including AutoDock Vina, Glide, and MM/GBSA analysis were performed to commensurate the improvement of screening progress. The evaluated binding affinities were validated by molecular dynamics simulations over a period of 20 ns for the interactions between theMycobacterium tuberculosisprotein PrpR with three novel scaffolds (1b,1j, and1k). All the scaffolds exhibited distinct stable interactions with the significant residues like Arg169, Arg197, Tyr248, Arg308, and Gly311 respectively. Further, the inhibitory activities of scaffolds were predicted and evaluated by machine learning based algorithm to rank the above proposed compounds. This study reveals the potential of1kand1jas effective inhibitor candidates for the treatment of tuberculosis.},
  archive      = {J_JCC},
  author       = {Sreerama Rajasekhar and Soumyadip Das and Ramanathan Karuppasamy and Balamurali Musuvathi Motilal and Kaushik Chanda},
  doi          = {10.1002/jcc.26823},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {619-630},
  shortjournal = {J. Comput. Chem.},
  title        = {Identification of novel inhibitors for prp protein of mycobacterium tuberculosis by structure based drug design, and molecular dynamics simulations},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The influence of tetrel bonds on the acidities of group 14
tetrafluoride - inorganic acid complexes. <em>JCC</em>, <em>43</em>(9),
611–618. (<a href="https://doi.org/10.1002/jcc.26822">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Ab initio methods were used to determine the influence of tetrel bond formation on the acidity. The systems composed of inorganic acids and tetrafluorides of 14 group elements have been tested - HA/EF4, where HA = H2O, NH3, HF, HCN, HNC, HCNO, HOCN and E = C, Si, Ge, Sn or Pb. It turns out that the electron density flow involved with formation of tetrel bond to carbon-based systems leads to negligible increase in acidity. In the case of the acceptor compounds based on the remaining 14 group elements however, the effect is much more apparent, as most of those compounds may be considered a Brønsted superacids. The electronic stability of anions formed after the deprotonation of aforementioned complexes has been investigated. Vast majority of the anions were found to exhibit significant electron binding energies.},
  archive      = {J_JCC},
  author       = {Jakub Brzeski},
  doi          = {10.1002/jcc.26822},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {611-618},
  shortjournal = {J. Comput. Chem.},
  title        = {The influence of tetrel bonds on the acidities of group 14 tetrafluoride - inorganic acid complexes},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Search for transition states with external forces.
<em>JCC</em>, <em>43</em>(9), 598–610. (<a
href="https://doi.org/10.1002/jcc.26821">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {It is much more difficult to find on the potential energy surface (PES) a transition state (TS) than a local minimum (LM). We propose a new methodology which makes this task much easier. Applying external forces to nuclei in a molecule we can locate on PES not just one particular TS but a number of consecutive transition states. With our approach it is possible to move over PES from one transition state to another without involving any local minima. The latter can be located in a separate step through the reaction path calculations performed for every transition state found before. Preliminary results for the 2-fluorofuran molecule illustrate the usefulness of the proposed method.},
  archive      = {J_JCC},
  author       = {Agnieszka Brzyska and Krzysztof Wolinski},
  doi          = {10.1002/jcc.26821},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {598-610},
  shortjournal = {J. Comput. Chem.},
  title        = {Search for transition states with external forces},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Improving the scaling and performance of multiple time
stepping-based molecular dynamics with hybrid density functionals.
<em>JCC</em>, <em>43</em>(9), 588–597. (<a
href="https://doi.org/10.1002/jcc.26816">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Density functionals at the level of the generalized gradient approximation (GGA) and a plane-wave basis set are widely used today to performab initiomolecular dynamics (AIMD) simulations. Going up in the ladder of accuracy of density functionals from GGA (second rung) to hybrid density functionals (fourth rung) is much desired pertaining to the accuracy of the latter in describing structure, dynamics, and energetics of molecular and condensed matter systems. On the other hand, hybrid density functional based AIMD simulations are about two orders of magnitude slower than GGA based AIMD for systems containing ~100 atoms using ~100 compute cores. Two methods, namely MTACE and s-MTACE, based on a multiple time step integrator and adaptively compressed exchange operator formalism are able to provide a speed-up of about 7–9 in performing hybrid density functional based AIMD. In this work, we report an implementation of these methods using a task-group based parallelization within the CPMD program package, with the intention to take advantage of the large number of compute cores available on modern high-performance computing platforms. We present here the boost in performance achieved through this algorithm. This work also identifies the computational bottleneck in the s-MTACE method and proposes a way to overcome it.},
  archive      = {J_JCC},
  author       = {Sagarmoy Mandal and Ritama Kar and Tobias Klöffel and Bernd Meyer and Nisanth N. Nair},
  doi          = {10.1002/jcc.26816},
  journal      = {Journal of Computational Chemistry},
  number       = {9},
  pages        = {588-597},
  shortjournal = {J. Comput. Chem.},
  title        = {Improving the scaling and performance of multiple time stepping-based molecular dynamics with hybrid density functionals},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A theoretical perspective of the ultrafast transient
absorption dynamics of CsPbBr3. <em>JCC</em>, <em>43</em>(8), 577–582.
(<a href="https://doi.org/10.1002/jcc.26815">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Transient absorption spectra (TAS) of lead halide perovskites can provide important insights into the nature of the photoexcited state dynamics of this prototypical class of materials. Here, we perform ground and excited state molecular dynamics (MD) simulations within a restricted open shell Kohn-Sham (ROKS) approach in order to interpret the characteristic features of the TAS of CsPbBr3. Our results reveal that properties such as the finite temperature band gap, the Stokes shift, and therefore, also the TAS are strongly size-dependent. Our TAS simulations show an early positive red-shifted feature on the fs scale that can be explained by geometric relaxation in the excited state. As excited-state processes can crucially affect the electronic properties of this class of photoactive materials, our observations are an important ingredient for further optimization of lead halide based optoelectronic devices.},
  archive      = {J_JCC},
  author       = {Ariadni Boziki and Pablo Baudin and Elisa Liberatore and Negar Ashari Astani and Ursula Rothlisberger},
  doi          = {10.1002/jcc.26815},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {577-582},
  shortjournal = {J. Comput. Chem.},
  title        = {A theoretical perspective of the ultrafast transient absorption dynamics of CsPbBr3},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Behavior of counterpoise correction in many-body molecular
clusters of organic compounds: Hartree–fock interaction energy
perspective. <em>JCC</em>, <em>43</em>(8), 568–576. (<a
href="https://doi.org/10.1002/jcc.26814">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The counterpoise (CP) correction by Boys and Bernardi has been well accepted as a reliable strategy to account for basis set superposition error (BSSE) in intermolecular complexes. The behavior of the CP correction was thoroughly studied in individual molecules of molecular complexes. This work studies the performance of the CP correction in many-body clusters including three-body clusters of organic compounds in the 3B-69 dataset. Additionally, we used crystal structures of polymorphs of benzene, aspirin, and oxalyl dihydrazide (ODH) to construct a many-body cluster dataset, abbreviated as the MBC-36 dataset, consisting of two, four and eight molecules, and 16 molecules in the case of benzene. A series of Dunning&#39;s basis sets—cc-pXZ and aug-cc-pXZ (X=DandT)—were used to predict CP-corrected Hartree–Fock (HF) interaction energies of the 3B-69 and MBC-36 datasets. The CP-corrected interaction energies were found to be basis-set independent, whereas the non-CP corrected interaction energies were found not to a follow a smooth exponential fitting as previously found for electronic energies of individual molecules. This observation was attributed to the presence of non-additive induction forces in some clusters. Two 2 × 2 × 2 supercells of benzene polymorphs were constructed to explore the local nature of BSSE effects. A cut-off radius of 10 Å was demonstrated to be sufficient to fully recover these effects. Although the behavior of CP correction was found to be non-conventional in many-body clusters of organic compounds, the use of a small basis set such as cc-pVDZ showed excellent performance in the prediction of HF interaction energies.},
  archive      = {J_JCC},
  author       = {Anh L. P. Nguyen and Ekaterina I. Izgorodina},
  doi          = {10.1002/jcc.26814},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {568-576},
  shortjournal = {J. Comput. Chem.},
  title        = {Behavior of counterpoise correction in many-body molecular clusters of organic compounds: Hartree–Fock interaction energy perspective},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Chemical repair of damaged leucine and tryptophane by
thiophenols at close to diffusion-controlled rates: Mechanisms and
kinetics. <em>JCC</em>, <em>43</em>(8), 556–567. (<a
href="https://doi.org/10.1002/jcc.26813">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Thiophenols are chemical species with multiple desirable biological properties, including their primary and secondary antioxidant capacity. In this work, the repairing antioxidant activity of eight different thiophenols has been investigated for damaged leucine and tryptophane. The investigation was carried out employing quantum mechanical and transition state methods to calculate the thermodynamic and kinetic data of the reactions involved, while simulating the biological conditions at physiological pH and aqueous and lipidic medium. The analysis of the atomic charges and the spin densities at each of the points on the potential energy surface was the tool that allowed the elucidation of the reaction mechanisms through which thiophenols repair the oxidative damage caused to the amino acids leucine and tryptophan. It was found that thiophenols can repair leucine via a hydrogen atom transfer mechanism in a manner which is similar to the one used by glutathione to repair the carbon-centered radicals of guanosine. In addition, thiophenols can also restore tryptophane, a nitrogen-centered radical, via proton-coupled electron transfer and single electron transfer mechanisms. Moreover, both processes occur at close to diffusion-controlled rates.},
  archive      = {J_JCC},
  author       = {Mirzam Carreon-Gonzalez and Leonardo Muñoz-Rugeles and Annik Vivier-Bunge and Juan Raul Alvarez-Idaboy},
  doi          = {10.1002/jcc.26813},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {556-567},
  shortjournal = {J. Comput. Chem.},
  title        = {Chemical repair of damaged leucine and tryptophane by thiophenols at close to diffusion-controlled rates: Mechanisms and kinetics},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Independent gradient model based on hirshfeld partition: A
new method for visual study of interactions in chemical systems.
<em>JCC</em>, <em>43</em>(8), 539–555. (<a
href="https://doi.org/10.1002/jcc.26812">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The powerful independent gradient model (IGM) method has been increasingly popular in visual analysis of intramolecular and intermolecular interactions in recent years. However, we frequently observed that there is an evident shortcoming of IGM map in graphically studying weak interactions, that is its isosurfaces are usually too bulgy; in these cases, not only the graphical effect is poor, but also the color on some areas on the isosurfaces is inappropriate and may lead to erroneous analysis conclusions. In addition, the IGM method was originally proposed based on promolecular density, which is quite crude and does not take actual electronic structure into account. In this article, we propose an improvement version of IGM, namely IGM based on Hirshfeld partition of molecular density (IGMH), which replaces the free-state atomic densities involved in the IGM method with the atomic densities derived by Hirshfeld partition of actual molecular electron density. This change makes IGM have more rigorous physical background. A large number of application examples in this article, including molecular and periodic systems, weak and chemical bond interactions, fully demonstrate the important value of IGMH in intuitively understanding interactions in chemical systems. Comparisons also showed that the IGMH usually has markedly better graphical effect than IGM and overcomes known problems in IGM. Currently IGMH analysis has been supported in our wavefunction analysis code Multiwfn (http://sobereva.com/multiwfn). We hope that IGMH will become a new useful method among chemists for exploring interactions in wide variety of chemical systems.},
  archive      = {J_JCC},
  author       = {Tian Lu and Qinxue Chen},
  doi          = {10.1002/jcc.26812},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {539-555},
  shortjournal = {J. Comput. Chem.},
  title        = {Independent gradient model based on hirshfeld partition: A new method for visual study of interactions in chemical systems},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A quantum-chemical perspective on the laser-induced
alignment and orientation dynamics of the CH3X (x = f, cl, br, i)
molecules. <em>JCC</em>, <em>43</em>(8), 519–538. (<a
href="https://doi.org/10.1002/jcc.26811">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Motivated by recent experiments, the laser-induced alignment-and-orientation (A&amp;O) dynamics of the prolate symmetric top CH3X (X = F, Cl, Br, I) molecules is investigated, with particular emphasis on the effect of halogen substitution on the rotational constants, dipole moments, and polarizabilities of these species, as these quantities determine the A&amp;O dynamics. Insight into possible control schemes for preferred A&amp;O dynamics of halogenated molecules and best practices for A&amp;O simulations are provided, as well. It is shown that for accurate A&amp;O -dynamics simulations it is necessary to employ large basis sets and high levels of electron correlation when computing the rotational constants, dipole moments, and polarizabilities. The benchmark-quality values of these molecular parameters, corresponding to the equilibrium, as well as the vibrationally averaged structures are obtained with the help of the focal-point analysis (FPA) technique and explicit electronic-structure computations utilizing the gold-standard CCSD(T) approach, basis sets up to quintuple-zeta quality, core-correlation contributions and, in particular, relativistic effects for CH3Br and CH3I. It is shown that the different A&amp;O behavior of the CH3X molecules in the optical regime is mostly caused by the differences in their polarizability anisotropy, in other terms, the size of the halogen atom. In contrast, the A&amp;O dynamics of the CH3X series induced by an intense few-cycle THz pulse is mostly governed by changes in the rotational constants, due to the similar dipole moments of the CH3X molecules. The A&amp;O dynamics is most sensitive to theBrotational constant: even the difference between its equilibrium and vibrationally-averaged values results in noticeably different A&amp;O dynamics. The contribution of rotational states having different symmetry, weighted by nuclear-spin statistics, to the A&amp;O dynamics is also studied.},
  archive      = {J_JCC},
  author       = {Irén Simkó and Kalyani Chordiya and Attila G. Császár and Mousumi Upadhyay Kahaly and Tamás Szidarovszky},
  doi          = {10.1002/jcc.26811},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {519-538},
  shortjournal = {J. Comput. Chem.},
  title        = {A quantum-chemical perspective on the laser-induced alignment and orientation dynamics of the CH3X (X = f, cl, br, i) molecules},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Data set and fitting dependencies when estimating protein
mutant stability: Toward simple, balanced, and interpretable models.
<em>JCC</em>, <em>43</em>(8), 504–518. (<a
href="https://doi.org/10.1002/jcc.26810">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurate prediction of protein stability changes upon mutation (ΔΔG) is increasingly important to evolution studies, protein engineering, and screening of disease-causing gene variants but is challenged by biases in training data. We investigated 45 linear regression models trained on data sets that account systematically for destabilization bias and mutation-type biasBM. The models were externally validated on three test data sets probing different pathologies and for internal consistency (symmetry and neutrality). Model structure and performance substantially depended on training data and even fitting method. We developed two final models: SimBa-IB for typical natural mutations and SimBa-SYM for situations where stabilizing and destabilizing mutations occur to a similar extent. SimBa-SYM, despite is simplicity, is essentially non-biased (vs. the Ssymdata set) while still performing well for all data sets (R~ 0.46–0.54, MAE = 1.16–1.24 kcal/mol). The simple models provide advantage in terms of interpretability, use and future improvement, and are freely available on GitHub.},
  archive      = {J_JCC},
  author       = {Kristoffer T. Bæk and Kasper P. Kepp},
  doi          = {10.1002/jcc.26810},
  journal      = {Journal of Computational Chemistry},
  number       = {8},
  pages        = {504-518},
  shortjournal = {J. Comput. Chem.},
  title        = {Data set and fitting dependencies when estimating protein mutant stability: Toward simple, balanced, and interpretable models},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Ground-state potential energy functions and
vibration-rotation energy levels of beryllium lithium and its cation.
<em>JCC</em>, <em>43</em>(7), 491–498. (<a
href="https://doi.org/10.1002/jcc.26809">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The accurate ground-state potential energy functions of the beryllium lithium dimer, BeLi, and its cation, BeLi+, have been determined from ab initio calculations using the multi-reference averaged coupled-pair functional (MR-ACPF) method in conjunction with the correlation-consistent core-valence basis sets up to septuple-zeta quality. The importance of the electron correlation beyond the MR-ACPF level of approximation, scalar relativistic, and adiabatic effects was discussed. For BeLi+, the similar calculation was performed using the single-reference coupled-cluster method, up to the CCSDTQ level of approximation. The vibration-rotation energy levels of the two species were predicted to near the “spectroscopic” accuracy. Strangely enough, the predicted vibrational term values do not compare well with the recent experimental data for either BeLi or BeLi+, with discrepancies reaching as much as 20–60 cm-1for highly excited vibrational levels.},
  archive      = {J_JCC},
  author       = {Jacek Koput},
  doi          = {10.1002/jcc.26809},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {491-498},
  shortjournal = {J. Comput. Chem.},
  title        = {Ground-state potential energy functions and vibration-rotation energy levels of beryllium lithium and its cation},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Hydration patterns of rings in drugs and relationship to
lipophilicity: A DFT study. <em>JCC</em>, <em>43</em>(7), 477–490. (<a
href="https://doi.org/10.1002/jcc.26808">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Rings are one of the major scaffold components of drugs in medicinal chemistry, due to their unique electronic distribution, scaffold rigidity, and three-dimensionality while lipophilicity is considered as a vital parameter of rings that can influence the reactivity, metabolic stability, and toxicity. We have analyzed the electronic features, hydration patterns, solvation effect and lipophilicity data for 51 most widely used ring systems in drugs. Molecular electrostatic potential (MESP) topology analysis has been used to assess the electronic distribution in rings which provided an easy interpretation of the most suitable hydration patterns of the ring with H2O molecule. Further, the global minimum of ring…H2O complex has been utilized to predict lipophilicity (logP) with the incorporation of implicit solvation effect. Classification of ring systems based on their molecular weight into four categories, viz. small ring ‘sr’, medium ring ‘mr’, large ring ‘lr’ and extra large ring ‘xlr’ systems has led to the finding of strong correlations between logP and hydration energy withR= 0.942, 0.933, 0.968 and 0.933, respectively. The micro solvation model is found to be useful for locating the hydrophobic-hydrophilic border for each category of rings in terms of hydration energy whereas the implicit solvation model used for two solvents, n-octanol and water on the most stable hydrated structure led to a global correlation between logP and solvation energy ratio. This correlation predicts a limiting logP value −7.03 for the most hydrophilic ring system and also suggests a clear partitioning of the ring molecules into hydrophobic and hydrophilic classes. The MESP topology-guided approach to understand the electronic features and hydration patterns of rings in drugs lead to powerful predictions on their lipophilicity behavior.},
  archive      = {J_JCC},
  author       = {Mambatta Haritha and Cherumuttathu H. Suresh},
  doi          = {10.1002/jcc.26808},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {477-490},
  shortjournal = {J. Comput. Chem.},
  title        = {Hydration patterns of rings in drugs and relationship to lipophilicity: A DFT study},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Reliability of computed molecular structures. <em>JCC</em>,
<em>43</em>(7), 465–476. (<a
href="https://doi.org/10.1002/jcc.26807">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When the structures of 1342 molecules are optimized by 30 methods and 7 basis sets, there appear 289 (21.54\%) problematic molecules and 112 (8.35\%) failed ones. When 278 problematic molecules are compared, the best methods are BHandH and LC-wPBE, while B97D, BP86, HFS, VSXC, and HCTH are very unreliable. When 179 problematic molecules are computed with larger basis sets, the smallest mean absolute deviation (MAD) of bond angle (2.3°) is shown by QCISD(T)/cc-pVTZ, while the smallest MAD of bond length (0.021 Å), the best SUM1 (4.9 unit), and the best SUM2 (2.4 unit) are shown by DSDPBEP86(Full), DSDPBEP86, PBE1PBE-D3, MP2, and MP2(Full) in combination with aug-cc-pVQZ, cc-pVQZ, Def2QZVP, Def2TZVPP, and/or 6–311++G(3df,3pd). Very large basis sets, for example, larger than cc-pVTZ usually have to be used to obtain very good structures and the performances of many density-functional theory methods are encouraging. The best results may be the limit of modern computational chemistry.},
  archive      = {J_JCC},
  author       = {Yi-Liang Zhang and Fu-Li Wang and Ai-Min Ren},
  doi          = {10.1002/jcc.26807},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {465-476},
  shortjournal = {J. Comput. Chem.},
  title        = {Reliability of computed molecular structures},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Uncovering clar’s aromatic -sextet rule in the hubbard model
using maximum probability domain partitions. <em>JCC</em>,
<em>43</em>(7), 457–464. (<a
href="https://doi.org/10.1002/jcc.26806">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Clar&#39;s aromatic-sextet rule is a widely used qualitative method for assessing the electronic structure of polycyclic benzenoid hydrocarbons. Unfortunately, many of the quantum chemical concordances for this rule have a limited range of applicability. Here, we show that the fundamental probabilities associated with a distribution of electrons over domain partitions support Clar&#39;s rule in both mean-field and static correlation regimes. In particular, domain partitions that maximize those probabilities reflect the dominance of Clar structures in the electronic structure of these molecules. These findings suggest that extending methods that aim to maximize probabilities by deforming domain partitions could lead to novel quantum chemical underpinnings for many chemical concepts.},
  archive      = {J_JCC},
  author       = {Daria Van Hende and Laurent Lemmens and Stijn De Baerdemacker and Dimitri Van Neck and Patrick Bultinck and Guillaume Acke},
  doi          = {10.1002/jcc.26806},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {457-464},
  shortjournal = {J. Comput. Chem.},
  title        = {Uncovering clar&#39;s aromatic -sextet rule in the hubbard model using maximum probability domain partitions},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Tuning the quasi-harmonic treatment of crystalline ionic
liquids within the density functional theory. <em>JCC</em>,
<em>43</em>(7), 448–456. (<a
href="https://doi.org/10.1002/jcc.26804">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Five ionic liquids are selected for benchmarking the performance of quasi-harmonic density functional theory (DFT) calculations of structural, phonon, and thermodynamic properties of their crystals. Data predicted by individual computational setups are sorted, establishing a distinct hierarchy among the first-principles approaches. PBE-D3 and B3LYP-D3 functionals are coupled with various plane wave and Gaussian-type orbital (GTO) basis sets. Propagation of the basis set superposition error and of the imperfections of both functionals into finite-temperature properties is discussed in detail. PBE-D3 together with a triple-zeta GTO basis set often yields the most accurate predictions of predicted molar volume and heat capacity with errors at 1\% and 8\%, respectively, representing the state-of-the-art for quasi-harmonic DFT calculations for crystalline ionic liquids. Fortuitous error cancellation between the basis-set superposition (overbinding) and PBE imperfection (overexpanding) strongly affects the overall accuracy, unlike the case of B3LYP/GTO calculations, impeding systematic convergence of the methodology towards higher accuracy.},
  archive      = {J_JCC},
  author       = {Ctirad Červinka},
  doi          = {10.1002/jcc.26804},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {448-456},
  shortjournal = {J. Comput. Chem.},
  title        = {Tuning the quasi-harmonic treatment of crystalline ionic liquids within the density functional theory},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Mechanistic study of cobalt(i)-catalyzed asymmetric coupling
of ethylene and enynes to functionalized cyclobutanes. <em>JCC</em>,
<em>43</em>(7), 440–447. (<a
href="https://doi.org/10.1002/jcc.26803">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Density functional theory (DFT) calculations have been performed to gain insight into the reaction mechanism of the Co(I)-catalyzed asymmetric [2 + 2] cycloaddition reaction of enyne1awith ethylene2to give the functionalized cyclobuteneE-4apossessing a chiral, all-carbon quaternary center in the ring framework (Science, 361, 68–72). This study reveals that the whole catalysis can be characterized via three stages: (i) oxidative dimerization followed by reductive elimination gives the intermediateIM3, (ii) the alkenyl-Co(III) metallacyclohepteneIM6formation with the addition of another equivalent ethylene via an oxidative dimerization process, (iii)β-Hydrogen elimination and reductive elimination fromIM6to result in the final productE-4aand regenerate the active speicesIM1for the next catalytic cycle. Each stage is kinetically and thermodynamically feasible for experimental realization under mild conditions, and the formation of the alkenyl-Co(III) metallacyclohepteneIM6, with a barrier of 27.2 kcal mol−1(i.e.,IM2 → TS4), should be the rate-determining step (RDS) during the whole catalysis. In addition, the origins of enantioselectivity and regioselectivity of the product are discussed.},
  archive      = {J_JCC},
  author       = {Runfeng Huang and Jiali Cai and Lei Yao and Yuna Bai and Kai Guo and Lili Zhao},
  doi          = {10.1002/jcc.26803},
  journal      = {Journal of Computational Chemistry},
  number       = {7},
  pages        = {440-447},
  shortjournal = {J. Comput. Chem.},
  title        = {Mechanistic study of cobalt(I)-catalyzed asymmetric coupling of ethylene and enynes to functionalized cyclobutanes},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Taking the monte-carlo gamble: How not to buckle under the
pressure! <em>JCC</em>, <em>43</em>(6), 431–434. (<a
href="https://doi.org/10.1002/jcc.26798">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Consistent buckling distortions of a large membrane patch (200 × 200 Å) are observed during molecular dynamics (MD) simulations using the Monte-Carlo (MC) barostat in combination with a hard Lennard–Jones (LJ) cutoff. The buckling behavior is independent of both the simulation engine and the force field but requires the MC barostat-hard LJ cutoff combination. Similar simulations of a smaller patch (90 × 90 Å) do not show buckling, but do show a small, systematic reduction in the surface area accompanied by ~1 Å thickening suggestive of compression. We show that a mismatch in the way potentials and forces are handled in the dynamical equations versus the MC barostat results in a compressive load on the membrane. Moreover, a straightforward application of elasticity theory reveals that a minimal compression of the linear dimensions of the membrane, inversely proportional to the edge length, is required for buckling, explaining this differential behavior. We recommend always using LJ force or potential-switching when the MC barostat is employed to avoid undesirable membrane deformations.},
  archive      = {J_JCC},
  author       = {Yessica K. Gomez and Andrew M. Natale and James Lincoff and Charles W. Wolgemuth and John M. Rosenberg and Michael Grabe},
  doi          = {10.1002/jcc.26798},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {431-434},
  shortjournal = {J. Comput. Chem.},
  title        = {Taking the monte-carlo gamble: How not to buckle under the pressure!},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A tiny charge-scaling in the OPLS-AA + l-OPLS force field
delivers the realistic dynamics and structure of liquid primary
alcohols. <em>JCC</em>, <em>43</em>(6), 421–430. (<a
href="https://doi.org/10.1002/jcc.26802">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We carry out molecular dynamics simulations for pure liquid primary alcohols ranging from methanol to 1-decanol under ambient conditions. Based on the OPLS-AA force field with the L-OPLS correction, we demonstrate that a few\% increases in the partial charges deliver the realistic dynamics (self-diffusion coefficient and shear viscosity) and structure (density and X-ray scattering intensity) as well as enthalpy of vaporization and isothermal compressibility. The validity against thermal expansion coefficient, isobaric heat capacity, and static dielectric constant are also discussed.},
  archive      = {J_JCC},
  author       = {Xuan Zhang and Yongtao Wang and Jia Yao and Haoran Li and Kenji Mochizuki},
  doi          = {10.1002/jcc.26802},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {421-430},
  shortjournal = {J. Comput. Chem.},
  title        = {A tiny charge-scaling in the OPLS-AA + L-OPLS force field delivers the realistic dynamics and structure of liquid primary alcohols},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Adatom mediated adsorption of n-heterocyclic carbenes on
cu(111) and au(111). <em>JCC</em>, <em>43</em>(6), 413–420. (<a
href="https://doi.org/10.1002/jcc.26801">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The adsorption of N-heterocyclic carbenes (NHCs) on Cu(111) and Au(111) surfaces is studied with density-functional theory. The role of the molecular side groups as well as the surface morphology in determining the adsorption geometry are explored in detail. Flat-laying NHCs, as observed experimentally for NHC with relatively small side groups, result from the adsorption at adatoms and give rise to the so-called ballbot configurations, which are more stable than adsorption on flat surfaces and provide an efficient precursor for the formation of bis(NHC) dimers. On Au(111), the resulting (NHC)2Au complexes are purely physisorbed and thus mobile. On the more reactive Cu(111), in contrast, the central Cu atom in the (NHC)2Cu dimer is still covalently bound to the surface, resulting in a mobility, which has to be thermally activated.},
  archive      = {J_JCC},
  author       = {Mitisha Jain and Uwe Gerstmann and Wolf Gero Schmidt and Hazem Aldahhak},
  doi          = {10.1002/jcc.26801},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {413-420},
  shortjournal = {J. Comput. Chem.},
  title        = {Adatom mediated adsorption of N-heterocyclic carbenes on cu(111) and au(111)},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). BF3-catalyzed mukaiyama aldol reaction of acetaldehyde with
2-siloxy-1-propene. <em>JCC</em>, <em>43</em>(6), 402–412. (<a
href="https://doi.org/10.1002/jcc.26800">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Mukaiyama aldol reaction is a powerful tool for the construction of the carbon–carbon bond and the formation of β-hydroxycarbonyl compounds. In this work, the mechanism of acetaldehyde and 2-siloxy-1-propene both in the absence and presence of the catalyst BF3was investigated based on density functional theory. The mechanism includes two major steps: the formation of the carbon–carbon bond and the removal of SiH3/BF2by water. The energy barrier of the carbon–carbon bond formation process in the presence of BF3is obviously lower, indicating that BF3is a good catalyst for this reaction. In terms of molecular configuration, the different tensions between the five-membered-ring and six-membered-ring can be considered as the possible reason for the catalytic effect of BF3. In terms of charge transfer, the charges of natural population analysis in the carbon atom of the carbonyl group in acetaldehyde becomes more positive, which is easier to attack by nucleophiles and promote the nucleophilic process.},
  archive      = {J_JCC},
  author       = {Xuening Li and Ying Li and Yuanyuan Sun and Lingpeng Meng and Yanli Zeng},
  doi          = {10.1002/jcc.26800},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {402-412},
  shortjournal = {J. Comput. Chem.},
  title        = {BF3-catalyzed mukaiyama aldol reaction of acetaldehyde with 2-siloxy-1-propene},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Analysis of changes of cavity volumes in predefined
directions of protein motions and cavity flexibility. <em>JCC</em>,
<em>43</em>(6), 391–401. (<a
href="https://doi.org/10.1002/jcc.26799">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dynamics of protein cavities associated with protein fluctuations and conformational plasticity is essential for their biological function. NMR ensembles, molecular dynamics (MD) simulations, and normal mode analysis (NMA) provide appropriate frameworks to explore functionally relevant protein dynamics and cavity changes relationships. Within this context, we have recently developed analysis of null areas (ANA), an efficient method to calculate cavity volumes. ANA is based on a combination of algorithms that guarantees its robustness against numerical differentiations. This is a unique feature with respect to other methods. Herein, we present an updated and improved version that expands it use to quantify changes in cavity features, like volume and flexibility, due to protein structural distortions performed on predefined biologically relevant directions, for example, directions of largest contribution to protein fluctuations (principal component analysis [PCA modes]) obtained by MD simulations or ensembles of NMR structures, collective NMA modes or any other direction of motion associated with specific conformational changes. A web page has been developed where its facilities are explained in detail. First, we show that ANA can be useful to explore gradual changes of cavity volume and flexibility associated with protein ligand binding. Secondly, we perform a comparison study of the extent of variability between protein backbone structural distortions, and changes in cavity volumes and flexibilities evaluated for an ensemble of NMR active and inactive conformers of the epidermal growth factor receptor structures. Finally, we compare changes in size and flexibility between sets of NMR structures for different homologous chains of dynein.},
  archive      = {J_JCC},
  author       = {German P. Barletta and Matias Barletta and Tadeo E. Saldaño and Sebastian Fernandez-Alberti},
  doi          = {10.1002/jcc.26799},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {391-401},
  shortjournal = {J. Comput. Chem.},
  title        = {Analysis of changes of cavity volumes in predefined directions of protein motions and cavity flexibility},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Using projection operators with maximum overlap methods to
simplify challenging self-consistent field optimization. <em>JCC</em>,
<em>43</em>(6), 382–390. (<a
href="https://doi.org/10.1002/jcc.26797">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Maximum overlap methods are effective tools for optimizing challenging ground- and excited-state wave functions using self-consistent field models such as Hartree-Fock and Kohn-Sham density functional theory. Nevertheless, such models have shown significant sensitivity to the user-defined initial guess of the target wave function. In this work, a projection operator framework is defined and used to provide a metric for non-aufbau orbital selection in maximum-overlap-methods. The resulting algorithms, termed the Projection-based Maximum Overlap Method (PMOM) and Projection-based Initial Maximum Overlap Method (PIMOM), are shown to perform exceptionally well when using simple user-defined target solutions based on occupied/virtual molecular orbital permutations. This work also presents a new metric that provides a simple and conceptually convenient measure of agreement between the desired target and the current or final SCF results during a calculation employing a maximum-overlap method.},
  archive      = {J_JCC},
  author       = {Hector H. Corzo and Ali Abou Taka and Aurora Pribram-Jones and Hrant P. Hratchian},
  doi          = {10.1002/jcc.26797},
  journal      = {Journal of Computational Chemistry},
  number       = {6},
  pages        = {382-390},
  shortjournal = {J. Comput. Chem.},
  title        = {Using projection operators with maximum overlap methods to simplify challenging self-consistent field optimization},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CHARMM-GUI drude prepper for molecular dynamics simulation
using the classical drude polarizable force field. <em>JCC</em>,
<em>43</em>(5), 359–375. (<a
href="https://doi.org/10.1002/jcc.26795">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Explicit treatment of electronic polarizability in empirical force fields (FFs) represents an extension over a traditional additive or pairwise FF and provides a more realistic model of the variations in electronic structure in condensed phase, macromolecular simulations. To facilitate utilization of the polarizable FF based on the classical Drude oscillator model,Drude Prepperhas been developed in CHARMM-GUI.Drude Prepperingests additive CHARMM protein structures file (PSF) and pre-equilibrated coordinates in CHARMM, PDB, or NAMD format, from which the molecular components of the system are identified. These include all residues and patches connecting those residues along with water, ions, and other solute molecules. This information is then used to construct the Drude FF-based PSF using molecular generation capabilities in CHARMM, followed by minimization and equilibration. In addition, inputs are generated for molecular dynamics (MD) simulations using CHARMM, GROMACS, NAMD, and OpenMM. Validation of theDrude Prepperprotocol and inputs is performed through conversion and MD simulations of various heterogeneous systems that include proteins, nucleic acids, lipids, polysaccharides, and atomic ions using the aforementioned simulation packages. Stable simulations are obtained in all studied systems, including 5 μs simulation of ubiquitin, verifying the integrity of the generated Drude PSFs. In addition, the ability of the Drude FF to model variations in electronic structure is shown through dipole moment analysis in selected systems. The capabilities and availability ofDrude Prepperin CHARMM-GUI is anticipated to greatly facilitate the application of the Drude FF to a range of condensed phase, macromolecular systems.},
  archive      = {J_JCC},
  author       = {Abhishek A. Kognole and Jumin Lee and Sang-Jun Park and Sunhwan Jo and Payal Chatterjee and Justin A. Lemkul and Jing Huang and Alexander D. MacKerell and Wonpil Im},
  doi          = {10.1002/jcc.26795},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {359-375},
  shortjournal = {J. Comput. Chem.},
  title        = {CHARMM-GUI drude prepper for molecular dynamics simulation using the classical drude polarizable force field},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Distance geometry and protein loop modeling. <em>JCC</em>,
<em>43</em>(5), 349–358. (<a
href="https://doi.org/10.1002/jcc.26796">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Due to the role of loops in protein function, loop modeling is an important problem in computational biology. We present a new approach to loop modeling based on a combinatorial version of distance geometry, where the search space of the associated problem is represented by a binary tree and a branch-and-prune method is defined to explore it, following an atomic ordering previously given. This ordering is used to calculate the coordinates of atoms from the positions of its predecessors. In addition to the theoretical development, computational results are presented to illustrate the advantage of the proposed method, compared with another approach of the literature. Our algorithm is freely available athttps://github.com/michaelsouza/bpl.},
  archive      = {J_JCC},
  author       = {Rodrigo Labiak and Carlile Lavor and Michael Souza},
  doi          = {10.1002/jcc.26796},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {349-358},
  shortjournal = {J. Comput. Chem.},
  title        = {Distance geometry and protein loop modeling},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Computational study on noncovalent interactions between (n,
n) single-walled carbon nanotubes and simple lignin model-compounds.
<em>JCC</em>, <em>43</em>(5), 340–348. (<a
href="https://doi.org/10.1002/jcc.26794">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Composites of carbon nanotubes (CNTs) and lignin are promising and potentially cheap precursors of—to this day—expensive carbon fibers. Since the control of the CNT-lignin interface is crucial to maximize fiber performance, it is imperative to understand the fundamental noncovalent interactions between lignin and CNT. In the present study a density functional theory study is conducted to investigate the fundamental noncovalent interaction strength between metallic (n, n) single-walled CNT (SWCNT) and simple lignin model molecules. In particular, the respective adsorption energies are used to gauge the strength of interaction classes (ππ interaction, CHπ hydrogen bonding and OH-related hydrogen bonding. From the data, substituent-dependent interaction trends as well as class- and curvature-dependent interaction trends are derived. Overall, we find that most of the interaction strength trends appear to be strongly influenced by geometry: flat orientation of the test molecules relative to the (n, n) SWCNT surface and small (n, n) SWCNT curvature—that is, large diameter enhances the CHπ and ππ interactions.},
  archive      = {J_JCC},
  author       = {Jan Badorrek and Michael Walter},
  doi          = {10.1002/jcc.26794},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {340-348},
  shortjournal = {J. Comput. Chem.},
  title        = {Computational study on noncovalent interactions between (n, n) single-walled carbon nanotubes and simple lignin model-compounds},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CellSys: An open-source tool for building initial structures
for bio-membranes and drug-delivery systems. <em>JCC</em>,
<em>43</em>(5), 331–339. (<a
href="https://doi.org/10.1002/jcc.26793">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Since phospholipids are the most important components in the structure of biomembranes, they deserve to be considered with a lot of attention in both experimental and computational theoretical studies using molecular simulation methods related to the research in the fields of drug design and drug delivery where they involve knowledge about the interactions of drug molecules with cell membranes. To employ the molecular simulation approach for this purpose the essential requirement is having information about the initial structure of phospholipids and how they interact with the drugs. Therefore in this article, we introduce an open-source software package in Python programming language for utilizing data manipulation for generation and developing the initial structure of biomolecular cells to provide the needed information for investigation in drug delivery systems. In addition, the proposed software package can be used for the efficient storage of membrane structural data to be exploited in designing new drug delivery systems. To verify the performance of the code and the results of the simulations, several analyses have been done, such as the calculation of area per lipid and self-diffusion coefficient, in addition to lipid order parameter. The results were in complete agreement with the references.},
  archive      = {J_JCC},
  author       = {Ali Abbasi and Sepideh Amjad-Iranagh and Bahram Dabir},
  doi          = {10.1002/jcc.26793},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {331-339},
  shortjournal = {J. Comput. Chem.},
  title        = {CellSys: An open-source tool for building initial structures for bio-membranes and drug-delivery systems},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Maximal occupation by bases of π-hole bands surrounding
linear molecules. <em>JCC</em>, <em>43</em>(5), 319–330. (<a
href="https://doi.org/10.1002/jcc.26792">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Linear molecules such as CO2contain a positive π-hole ring that surrounds C on the molecule&#39;s equator. Quantum calculations examine the question as to how many bases can simultaneously bind to this ring. Linear molecules examined are TO2, where T = C, Si, Ge, Sn; bases are NCH and NH3. CO2engages in the weakest of the tetrel bonds, and can bind up to three NCH and two NH3. Unlikeσ-hole tetrel bonds, Si forms the strongest tetrel bonds, with interaction energies as high as 43 kcal/mol with NH3. But like GeO2, SiO2can sustain only two bases in its equatorial ring. The π-hole ring of SnO2can engage in up to four tetrel bonds with either NCH or NH3, even though these bonds are weaker than those with GeO2or SiO2. As all of these complexes cast TO2in the role of multiple electron acceptor, the resulting negative cooperativity makes each successive bond weaker than its predecessor as bases are added, as well as reducing the magnitude of the central molecule&#39;s π-hole.},
  archive      = {J_JCC},
  author       = {Steve Scheiner},
  doi          = {10.1002/jcc.26792},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {319-330},
  shortjournal = {J. Comput. Chem.},
  title        = {Maximal occupation by bases of π-hole bands surrounding linear molecules},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Benchmark study on deep neural network potentials for small
organic molecules. <em>JCC</em>, <em>43</em>(5), 308–318. (<a
href="https://doi.org/10.1002/jcc.26790">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {There has been tremendous advancement in machine learning (ML) applications in computational chemistry, particularly in neural network potentials (NNP). NNPs can approximate potential energy surface (PES) as a high dimensional function by learning from existing reference data, thereby circumventing the need to solve the electronic Schrödinger equation explicitly. As a result, ML accelerates chemical space exploration and property prediction compared to quantum mechanical methods. Novel ML methods have the potential to provide efficient means for predicting the properties of molecules. However, this potential has been limited by the lack of standard comparative evaluations. In this work, we compare four selected models, that is, ANI, PhysNet, SchNet, and BAND-NN, developed to represent the PES of small organic molecules. We evaluate these models for their accuracy and transferability on two different test sets (i) Small organic molecules of up to eight-heavy atoms on which ANI and SchNet achieve root mean square error (RMSE) of 0.55 and 0.60 kcal/mol, respectively. (ii) On random selection of molecules from the GDB-11 database with 10-heavy atoms, ANI achieves RMSE of 1.17 kcal/mol and SchNet achieves RMSE of 1.89 kcal/mol. We examine their ability to produce smooth meaningful surface by performing PES scans for bond stretch, angle bend, and dihedral rotations on relatively large molecules to assess their possible application in molecular dynamics simulations. We also evaluate their performance for yielding minimum energy structures via geometry optimization using various minimization algorithms. All these models were also able to accurately differentiate different isomers of the same empirical formula. ANI and PhysNet achieve an RMSE of 0.29 and 0.52 kcal/mol, respectively, onisomers.},
  archive      = {J_JCC},
  author       = {Rohit Modee and Siddhartha Laghuvarapu and U. Deva Priyakumar},
  doi          = {10.1002/jcc.26790},
  journal      = {Journal of Computational Chemistry},
  number       = {5},
  pages        = {308-318},
  shortjournal = {J. Comput. Chem.},
  title        = {Benchmark study on deep neural network potentials for small organic molecules},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Cover image. <em>JCC</em>, <em>43</em>(4), i. (<a
href="https://doi.org/10.1002/jcc.26805">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Ying Qian and Xuelian Li and Jian Wu and Aimin Zhou and Zhijian Xu and Qian Zhang},
  doi          = {10.1002/jcc.26805},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {i},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). XGBoost-based intelligence yield prediction and reaction
factors analysis of amination reaction. <em>JCC</em>, <em>43</em>(4),
289–302. (<a href="https://doi.org/10.1002/jcc.26791">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Buchwald-Hartwig amination reaction catalyzed by palladium plays an important role in drug synthesis. In the last few years, machine learning-assisted strategies emerged and quickly gained attention. In this article, an importance and relevance-based integrated feature screening method is proposed to effectively filter high-dimensional feature descriptor data. Then, a regularized machine learning boosting tree model, eXtreme Gradient Boosting, is introduced to intelligently predict reaction performance in multidimensional chemistry space. Furthermore, convergence, interpretability, generalization, and the internal association between reaction conditions and yields are excavated, which provides intelligent assistance for the optimal design of coupling reaction system and evaluating the reaction conditions. Compared with recently published results, the proposed method requires fewer feature descriptors, takes less time, and achieves more accurate prediction accuracy.},
  archive      = {J_JCC},
  author       = {Jing Dong and Lichao Peng and Xiaohui Yang and Zelin Zhang and Puyu Zhang},
  doi          = {10.1002/jcc.26791},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {289-302},
  shortjournal = {J. Comput. Chem.},
  title        = {XGBoost-based intelligence yield prediction and reaction factors analysis of amination reaction},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Computer-aided simulation of infrared spectra of ethanol
conformations in gas, liquid and in CCl4 solution. <em>JCC</em>,
<em>43</em>(4), 279–288. (<a
href="https://doi.org/10.1002/jcc.26788">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The recently developed efficient protocol combining implicit and explicit, accurate quantum-mechanical modeling of the condensed state (Katsyuba et al., J. Chem. Phys. 155, 024507 [2021]) is used to describe the IR spectra of liquid ethanol and its solutions in CCl4. The relative abundance of theantiandgaucheconformers of ethanol is shown to increase from ~40:60 in the gas phase to ~55:45 in the liquid phase. In spite of a moderate impact of media effects on the conformational composition of the liquid, the solvent strongly influences vibrational frequencies, IR intensities, and normal modes of each conformer, producing qualitatively different spectra compared to the gas phase and CCl4solution. Further, these solvent effects affecting IR frequencies and intensities depend not only on the conformation of the solvated molecule but also on the solvating species. Nevertheless, vibrational frequencies ofantiandgaucheconformers of liquid ethanol and its several isotopomers practically coincide with each other. Convenient liquid-state conformational markers in the fingerprint region of IR spectra are revealed for the hydroxyl-deuterated species: CH3CH2OD, CH3CHDOD, CH3CD2OD, and CD3CD2OD.},
  archive      = {J_JCC},
  author       = {Sergey A. Katsyuba and Tatiana P. Gerasimova and Sebastian Spicher and Fabian Bohle and Stefan Grimme},
  doi          = {10.1002/jcc.26788},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {279-288},
  shortjournal = {J. Comput. Chem.},
  title        = {Computer-aided simulation of infrared spectra of ethanol conformations in gas, liquid and in CCl4 solution},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Toward universal substituent constants: Transferability of
group descriptors from the quantum theory of atoms in molecules.
<em>JCC</em>, <em>43</em>(4), 265–278. (<a
href="https://doi.org/10.1002/jcc.26787">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Traditionally, substituents are described not by their intrinsic properties, but by their effect elsewhere in a molecule. However, the quantum theory of atoms in molecules (QTAIM) provides a route to intrinsic substituent descriptors. Ideally, these descriptors would exhibit minimal change as the local environment changes, and hence be considered transferable. Whether this is true is an open question. Here, we evaluated the transferability of QTAIM functional group descriptors for 117 functional groups in a series of 17 substrates to determine whether descriptors obtained using hydrogen as substrate are transferable. The functional group volume has a strong, consistent, linear relationship throughout. All other hydrogen-based group descriptors exhibit a relatively strong linear relationship with those in carbon-based substrates and a reasonable linear relationship with those in non-carbon-based substrates. Outliers are readily interpretable in terms of substrate induced functional group geometry changes. As expected, directional descriptors lying along the substituent-substrate axis are the least conserved.},
  archive      = {J_JCC},
  author       = {Kevin M. Lefrancois-Gagnon and Robert C. Mawhinney},
  doi          = {10.1002/jcc.26787},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {265-278},
  shortjournal = {J. Comput. Chem.},
  title        = {Toward universal substituent constants: Transferability of group descriptors from the quantum theory of atoms in molecules},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Picture-word order compound protein interaction: Predicting
compound-protein interaction using structural images of compounds.
<em>JCC</em>, <em>43</em>(4), 255–264. (<a
href="https://doi.org/10.1002/jcc.26786">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying potential associations between proteins and compounds is significant and challenging in the drug discovery process. Existing deep-learning-based methods tend to treat compounds and proteins as sequences or graphs. Inspired by the rapid development of computer vision technology, we argue that more abundant characterizations can be extracted from the images of compounds than from their sequences or graphs. Therefore, we propose an interaction model named picture-word order compound protein interaction (PWO-CPI) which learns the representation from structural images of compounds and protein sequences by using convolutional neural network (CNN). The experiments show that PWO-CPI outperforms state-of-the-art CPI prediction models. We also perform drug–drug interaction (DDI) experiments to validate the strong potential of structural formula images of molecular structures as molecular features. In addition, with the aid of generative adversarial networks, the visualization of image features demonstrates PWO-CPI can learn compound structural features implicitly and automatically.},
  archive      = {J_JCC},
  author       = {Ying Qian and Xuelian Li and Jian Wu and Aimin Zhou and Zhijian Xu and Qian Zhang},
  doi          = {10.1002/jcc.26786},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {255-264},
  shortjournal = {J. Comput. Chem.},
  title        = {Picture-word order compound protein interaction: Predicting compound-protein interaction using structural images of compounds},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Efficient prediction for high precision CO-n2 potential
energy surface by stacking ensemble DNN. <em>JCC</em>, <em>43</em>(4),
244–254. (<a href="https://doi.org/10.1002/jcc.26785">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {High-dimensional potential energy surface (PES) for van der Waals systems with spectroscopic accuracy, is of great importance for quantum dynamics and an extremely challenge job. CO-N2is a typical van der Waals system and its high-precision PES may help elucidate weak interaction mechanisms. Taking CO-N2potential energies calculated by CCSD(T)-F12b/aug-cc-pVQZ as the benchmark, we establish an accurate, robust, and efficient machine learning model by using only four molecular structure descriptors based on 7966 benchmark potential energies. The highest accuracy is obtained by a stacking ensemble DNN (SeDNN). Its evaluation parameters MAE, RMSE, andR2reach 0.096, 0.163, 0.9999 cm−1, respectively, and the spectroscopic accuracy for vibration spectrum is achieved with predicted PES, which shows SeDNN superior goodness-of-fit and prediction performance. An elaborated PES with the reported global minimum has been predicted with the model, which perfectly reproduces CCSD(T) potential energies and the analytical MLR PES [PCCP,2018,20,2036]. The critical points (global minimum, TSI, TSII, and their barriers), potential curve, and entire PES profile are remarkably consistent with CCSD(T) calculations. To further improve the usability of constructing PESs in practice, the size of the training set (energy points) for the model is reduced to 50\%, 30\%, and 20\% of the database, respectively. The results show that even training with the smallest training set (1593 points), the PES only differs 2.555 cm−1with the analytic MLR PES. Therefore, the proposed SeDNN is promisingly an alternative efficient tool to construct subtle PES for van der Waals systems.},
  archive      = {J_JCC},
  author       = {Shanshan Xu and You Li and Donghan Wang and Chao Fang and Chengwei Luo and Jiankun Deng and LiHong Hu and Hui Li and Hongzhi Li},
  doi          = {10.1002/jcc.26785},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {244-254},
  shortjournal = {J. Comput. Chem.},
  title        = {Efficient prediction for high precision CO-n2 potential energy surface by stacking ensemble DNN},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Assessment of the van der waals, hubbard u parameter and
spin-orbit coupling corrections on the 2D/3D structures from metal gold
congeners clusters. <em>JCC</em>, <em>43</em>(4), 230–243. (<a
href="https://doi.org/10.1002/jcc.26784">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The coinage-metal clusters possess a natural complexity in their theoretical treatment that may be accompanied by inherent shortcomings in the methodological approach. Herein, we performed a scalar-relativistic density functional theory study, considering Perdew, Burke, and Ernzerhof (PBE) with (empirical and semi empirical) van der Waals (vdW), spin-orbit coupling (SOC), +U(Hubbard term), and their combinations, to treat the, andclusters in different structural motifs. The energetic scenario is given by the confirmation of the 3D lowest energy configurations forandwithin all approaches, while forthere is a 2D/3D competition, depending on the applied correction. The 2D geometry is 0.43 eV more stable with plain PBE than the 3D one, the SOC, +U, and/or vdW inclusion decreases the overestimated stability of the planar configurations, where the most surprising result is found by the D3 and D3BJ vdW corrections, for which the 3D configuration is 0.29 and 0.11 eV, respectively, more stable than the 2D geometry (with even higher values when SOC and/or +Uare added). The D3 dispersion correction represents 7.9\% (4.4\%) of the total binding energy for the 3D (2D) configuration, (not) being enough to change thehybridization and the position of the occupied-states. Our predictions are in agreement with experimental results and in line with the best results obtained for bulk systems, as well as with hybrid functionals within D3 corrections. The properties description undergoes small corrections with the different approaches, where general trends are maintained, that is, the average bond length is smaller (larger) for lower (higher)-coordinated structures, since a same number of electrons are shared by a smaller (larger) number of bonds, consequently, the bonds are stronger (weaker) and shorter (longer) and thehybridization index is larger (smaller). Thus, Au has a distinct behavior in relation to its lighter congeners, with a complex potential energy surface, where in addition to the relevant relativistic effects, correlation and dispersion effects must also be considered.},
  archive      = {J_JCC},
  author       = {Maurício J. Piotrowski and Renato P. Orenha and Renato L. T. Parreira and Diego Guedes-Sobrinho},
  doi          = {10.1002/jcc.26784},
  journal      = {Journal of Computational Chemistry},
  number       = {4},
  pages        = {230-243},
  shortjournal = {J. Comput. Chem.},
  title        = {Assessment of the van der waals, hubbard u parameter and spin-orbit coupling corrections on the 2D/3D structures from metal gold congeners clusters},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). FSATOOL 2.0: An integrated molecular dynamics simulation and
trajectory data analysis program. <em>JCC</em>, <em>43</em>(3), 215–224.
(<a href="https://doi.org/10.1002/jcc.26772">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular dynamics simulation is important in the computational study of the biomolecules. In this paper, we upgrade our previous FSATOOL to version 2.0. It is no longer a plugin as before. Besides the existed enhanced sampling and Markov state model analysis module, FSATOOL 2.0 has three new features now. First, it contains a molecular dynamics simulation engine on both CPU and GPU device. The engine works with an embedded enhanced sampling module. Second, it can do the free energy calculation by various practical methods, including the weighted histogram analysis method and Gaussian mixture model. Third, it has many subroutines to process the trajectory data, such as principal component analysis, time-structure based independent component analysis, contact analysis, and Φ-value analysis. Most importantly, all these calculations are integrated into one package. The trajectory data format is compatible with all the modules. With a proper input parameter file, users can do the molecular dynamics simulation and data analysis work by only a few simplified commands. The capabilities and theoretical backgrounds of FSATOOL 2.0 are introduced in the paper.},
  archive      = {J_JCC},
  author       = {Zirui Shu and Mincong Wu and Jun Liao and Changjun Chen},
  doi          = {10.1002/jcc.26772},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {215-224},
  shortjournal = {J. Comput. Chem.},
  title        = {FSATOOL 2.0: An integrated molecular dynamics simulation and trajectory data analysis program},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Next generation quantum theory of atoms in molecules for the
design of emitters exhibiting thermally activated delayed fluorescence
with laser irradiation. <em>JCC</em>, <em>43</em>(3), 206–214. (<a
href="https://doi.org/10.1002/jcc.26783">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The effect of a static electric (E)-field and an unchirped and chirped laser pulse field on the cycl[3.3.3]azine molecule was investigated using next-generation quantum theory of atoms in molecules (NG-QTAIM). Despite the magnitude of the E-field of the laser pulses being an order of magnitude lower than for the static E-field, the variation of the energy gap between the lowest lying singlet (S1) and triplet (T1) excited states was orders of magnitude greater for the laser pulse than for the static E-field. Insights into the response of the electronic structure were captured by NG-QTAIM, where differences in the inverted singlet-triplet gap due to the laser pulses were significant larger compared to those induced by the static E-field. The response of theS1andT1excited states, as determined by NG-QTAIM, switched discontinuously between weak and strong chemical character for the static E-field. In contrast, the response to the laser pulses, determined by NG-QTAIM, is to induce a continuous range of chemical character, indicating the unique ability of the laser pulses to induce polarization effects in the form of “mixed” bond types. Our analysis demonstrates that NG-QTAIM is a useful tool for understanding the response to laser irradiation of the lowest-lying singletS1and tripletT1excited states of emitters exhibiting thermally activated delayed fluorescence. The chirped laser pulse led to more frequent instances of the desired outcome of an inverted singlet-triplet gap than the unchirped pulse, indicating its usefulness as a tool to design more efficient organic light-emitting diode devices.},
  archive      = {J_JCC},
  author       = {Zi Li and Yong Yang and Tianlv Xu and Herbert Früchtl and Tanja van Mourik and Martin J. Paterson and Yasuteru Shigeta and Steven R. Kirk and Samantha Jenkins},
  doi          = {10.1002/jcc.26783},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {206-214},
  shortjournal = {J. Comput. Chem.},
  title        = {Next generation quantum theory of atoms in molecules for the design of emitters exhibiting thermally activated delayed fluorescence with laser irradiation},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Thermostabilization mechanisms in thermophilic versus
mesophilic three-helix bundle proteins. <em>JCC</em>, <em>43</em>(3),
197–205. (<a href="https://doi.org/10.1002/jcc.26782">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The engineered three-helix bundle, UVF, is thermostabilized entropically due to heightened, native-state dynamics. However, it is unclear whether this thermostabilization strategy is observed in natural proteins from thermophiles. We performed all-atom, explicit solvent molecular dynamics simulations of two three-helix bundles from thermophilicH. butylicus(2lvsN and 2lvsC) and compared their dynamics to a mesophilic three-helix bundle, the Engrailed homeodomain (EnHD). Like UVF, 2lvsC had heightened native dynamics, which it maintained without unfolding at 100°C. Shortening and rigidification of loops in 2lvsN and 2lvsC and increased surface hydrogen bonds in 2lvsN were observed, as is common in thermophilic proteins. A buried disulfide and salt bridge in 2lvsN and 2lvsC, respectively, provided some stabilization, and addition of a homologous disulfide bond in EnHD slowed unfolding. The transferability and commonality of stabilization strategies among members of the three-helix bundle fold suggest that these strategies may be general and deployable in designing thermostable proteins.},
  archive      = {J_JCC},
  author       = {Catrina Nguyen and Lauren M. Yearwood and Michelle E. McCully},
  doi          = {10.1002/jcc.26782},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {197-205},
  shortjournal = {J. Comput. Chem.},
  title        = {Thermostabilization mechanisms in thermophilic versus mesophilic three-helix bundle proteins},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Strategies for the optimization of the structure of
crystalline compounds. <em>JCC</em>, <em>43</em>(3), 184–196. (<a
href="https://doi.org/10.1002/jcc.26781">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {When different proposals exist (or can reasonably be formulated) for the size of the unit cell (in terms of number of atoms) and space group of crystalline compounds, a strategy for exploring with simulation methods the various cases and for investigating their relative stability must be defined. The optimization schemes of periodic quantum mechanical codes work in fact at fixed space group and number of atoms per unit cell, so that only the fractional coordinates of the atoms and the lattice parameters are optimized. A strategy is here presented, based on four standard tools, used synergistically and in sequence: (1) the optimization of inner coordinates and unit cell parameters; (2) the calculation of the vibrational frequencies not only at, but also at a set ofpoints (in the example presented here they are eight, generated by a shrinking factor 2), looking for possible negative wavenumbers. The latter correspond to maxima, rather than minima, along the coordinate described by the corresponding normal mode; (3) the exploration of the total energy along the mode with negative wavenumber, looking for the minimum of the curve; (4) the identification of the new space group corresponding to the reduced symmetry resulting from the previous step. The strategy is illustrated with reference to the KMnF3perovskite compound, for which many space groups are proposed in the literature, ranging from cubicto tetragonaland orthorhombic (Pnma and Cmcm) down to monoclinic (P21/m). The corresponding primitive cells contain 5, 10, and 20 atoms in the various cases, and the point symmetry reduces from 48 to 4 operators. In nature, the KMnF3phase transitions also include the magnetic phases. For simplicity, here we limit the analysis to the ones that take place between ferromagnetic phases, as they are sufficiently rich for illustrating the proposed strategy. As the total energy differences involved can be as small as, say, 10–50μHartree, a high numerical accuracy at each one of the steps mentioned above is required. The present calculations, performed with the CRYSTAL code, by using anall electronbasis set and the Hartree-Fock and B3LYP functionals, document such an accuracy. The energy difference between the tetragonaland cubicphases is 225μHartree, with a volume reduction of 0.58 Å3; the differences between the orthorhombic and tetragonal phases are an order of magnitude smaller, being 23μHartree and 0.06 Å3for total energy and cell volume, respectively.},
  archive      = {J_JCC},
  author       = {Fabien Pascale and Philippe D&#39;Arco and Francesco Silvio Gentile and Roberto Dovesi},
  doi          = {10.1002/jcc.26781},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {184-196},
  shortjournal = {J. Comput. Chem.},
  title        = {Strategies for the optimization of the structure of crystalline compounds},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). An inexpensive density functional theory-based protocol to
predict accurate 19F-NMR chemical shifts. <em>JCC</em>, <em>43</em>(3),
170–183. (<a href="https://doi.org/10.1002/jcc.26780">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Thanks to its advantages,19F-NMR is an increasingly popular technique for the structural characterization of F-containing molecules, among which polymers, materials, fluorophores, pharmaceuticals, and so forth. However, the computational calculation of the19F-NMR chemical shifts, both for prediction and interpretation of experimental spectra, remains a challenge. In this work a density functional theory (DFT) based protocol for the calculation of the chemical shifts is established within the framework of the gauge-independent atomic orbital method, upon verifying the performance of Hartree–Fock and 60 DFT functionals coupled with seven different basis sets. The benchmark is conducted using two sets of molecules, namely one used for testing methods and another used for probing; the former set consists of 134 molecules, the latter 50, yet both of them with F in different chemical environments. Following Bally–Rablen–Tantillo strategy, the scaling parameters and other statistical quantities were computed for each method upon least squares linear regression between experimental and computed chemical shifts. The designed computational workflow is computationally inexpensive and represents a significant improvement with respect to the current state of the art.},
  archive      = {J_JCC},
  author       = {Enrico Benassi},
  doi          = {10.1002/jcc.26780},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {170-183},
  shortjournal = {J. Comput. Chem.},
  title        = {An inexpensive density functional theory-based protocol to predict accurate 19F-NMR chemical shifts},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Improving ligand-ranking of AutoDock vina by changing the
empirical parameters. <em>JCC</em>, <em>43</em>(3), 160–169. (<a
href="https://doi.org/10.1002/jcc.26779">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {AutoDock Vina (Vina) achieved a very high docking-success rate, but give a rather low correlation coefficient, for binding affinity with respect to experiments. This low correlation can be an obstacle for ranking of ligand-binding affinity, which is the main objective of docking simulations. In this context, we evaluated the dependence of VinaRcoefficient upon its empirical parameters.is affected more by changing the gauss2 and rotation than other terms. The docking-success rateis sensitive to the alterations of the gauss1, gauss2, repulsion, and hydrogen bond parameters. Based on our benchmarks, the parameter set1 has been suggested to be the most optimal. The testing study over 800 complexes indicated that the modified Vina provided higher correlation with experimentcompared withobtained by the original Vina andby Vina version 1.2. Besides, the modified Vina can be also applied more widely, givingfor 32/48 targets, compared with the default package, givingfor 31/48 targets. In addition, validation calculations for 1036 complexes obtained from version 2019 of PDBbind refined structures showed that the set1 of parameters gave higher correlation coefficient () than the default package () and Vina version 1.2 (). The version of Vina with set1 of parameters can be downloaded athttps://github.com/sontungngo/mvina. The outcomes would enhance the ranking of ligand-binding affinity using Autodock Vina.},
  archive      = {J_JCC},
  author       = {T. Ngoc Han Pham and Trung Hai Nguyen and Nguyen Minh Tam and Thien Y. Vu and Nhat Truong Pham and Nguyen Truong Huy and Binh Khanh Mai and Nguyen Thanh Tung and Minh Quan Pham and Van V. Vu and Son Tung Ngo},
  doi          = {10.1002/jcc.26779},
  journal      = {Journal of Computational Chemistry},
  number       = {3},
  pages        = {160-169},
  shortjournal = {J. Comput. Chem.},
  title        = {Improving ligand-ranking of AutoDock vina by changing the empirical parameters},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022a). Cover image. <em>JCC</em>, <em>43</em>(2), i. (<a
href="https://doi.org/10.1002/jcc.26789">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_JCC},
  author       = {Asja A. Kroeger and Amir Karton},
  doi          = {10.1002/jcc.26789},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {i},
  shortjournal = {J. Comput. Chem.},
  title        = {Cover image},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). ParaCopasi: A package for parallel biochemical simulation
and analysis. <em>JCC</em>, <em>43</em>(2), 144–154. (<a
href="https://doi.org/10.1002/jcc.26775">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Biochemical simuflation and analysis play a significant role in systems biology research. Numerous software tools have been developed to serve this area. Using these tools for completing tasks, for example, stochastic simulation, parameter fitting and optimization, usually requires sufficient computational power to make the duration of completion acceptable. COPASI is one of the most powerful tools for quantitative simulation and analysis targeted at biological systems. It supports systems biology markup language and covers multiple categories of tasks. This work develops an open source package ParaCopasi for parallel COPASI tasks and investigates its performance regarding accelerations. ParaCopasi can be installed on platforms equipped with multicore CPU to exploit the cores, scaling from desktop computers to large scale high-performance computing clusters. More cores bring more performance. The results show that the parallel efficiency has a positive correlation with the total workload. The parallel efficiency reaches a level of at least 95\% for both homogeneous and heterogenous tasks when computational workload is adequate. An example is illustrated by applicating this package in parameter estimation to calibrate a biochemical kinetics model.},
  archive      = {J_JCC},
  author       = {Lihuan Yuan and Chengkun Wu and Xiaowei Guo and Canqun Yang},
  doi          = {10.1002/jcc.26775},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {144-154},
  shortjournal = {J. Comput. Chem.},
  title        = {ParaCopasi: A package for parallel biochemical simulation and analysis},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The ad-MD method to calculate NMR shift including effects
due to conformational dynamics: The 31P NMR shift in DNA. <em>JCC</em>,
<em>43</em>(2), 132–143. (<a
href="https://doi.org/10.1002/jcc.26778">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A method for averaging of NMR parameters by molecular dynamics (MD) has been derived from the method of statistical averaging in MD snapshots, benchmarked and applied to structurally dynamic interpretation of the31P NMR shift (δ31P) in DNA phosphates. The method employs adiabatic dependence of an NMR parameter on selected geometric parameter(s) that is weighted by MD-calculated probability distribution(s) for the geometric parameter(s) (Ad-MD method). The usage of Ad-MD for polymers is computationally convenient when one pre-calculated structural dependence of an NMR parameter is employed for all chemically equivalent units differing only in dynamic behavior. The Ad-MD method is benchmarked against the statistical averaging method for δ31Pin the model phosphates featuring distinctively different structures and dynamic behavior. The applicability of Ad-MD is illustrated by calculating31P NMR spectra in the Dickerson-Drew DNA dodecamer. δ31Pwas calculated with the B3LYP/IGLO-III/PCM(water) and the probability distributions for the torsion angles adjacent to the phosphorus atoms in the DNA phosphates were calculated using the OL15 force field.},
  archive      = {J_JCC},
  author       = {Jiří Fukal and Miloš Buděšínský and Ondřej Páv and Petr Jurečka and Marie Zgarbová and Jakub Šebera and Vladimír Sychrovský},
  doi          = {10.1002/jcc.26778},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {132-143},
  shortjournal = {J. Comput. Chem.},
  title        = {The ad-MD method to calculate NMR shift including effects due to conformational dynamics: The 31P NMR shift in DNA},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). The effect of midbond functions on interaction energies
computed using MP2 and CCSD(t). <em>JCC</em>, <em>43</em>(2), 121–131.
(<a href="https://doi.org/10.1002/jcc.26777">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this article we use MP2 and CCSD(T) calculations for the A24 and S66 data sets to explore how midbond functions can be used to generate cost effective counterpoise corrected supramolecular interaction energies of noncovalent complexes. We use the A24 data set to show that the primary role of midbond functions is not to approach the complete basis set limit, but rather to ensure a balanced description of the molecules and the interaction region (unrelated to the basis set superposition error). The need for balance is a consequence of using atom centered basis sets. In the complete basis set limit, the error will disappear, but reaching the complete basis set limit is not feasible beyond small systems. For S66 we investigate the need for increasing the number of midbond centers. Results show that adding a second midbond center increases the accuracy, but the effect is secondary to changing the atom centered basis set. Further, by comparing calculations using the 3s3p2d1f1g midbond set with using aug-cc-pVDZ and aug-cc-pVTZ as midbond sets, we see that the requirements for the midbond set to be effective, is not just that it contains diffuse functions, but also that high angular momentum functions are included. By comparing two approaches for placing midbond centers we show that results are not particularly sensitive to placement as long as the placement is reasonable.},
  archive      = {J_JCC},
  author       = {Regina Matveeva and Merete Falck Erichsen and Henrik Koch and Ida-Marie Høyvik},
  doi          = {10.1002/jcc.26777},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {121-131},
  shortjournal = {J. Comput. Chem.},
  title        = {The effect of midbond functions on interaction energies computed using MP2 and CCSD(T)},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). A DLPNO-CCSD(t) benchmarking study of intermolecular
interactions of ionic liquids. <em>JCC</em>, <em>43</em>(2), 106–120.
(<a href="https://doi.org/10.1002/jcc.26776">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The accuracy of correlation energy recovered by coupled cluster single-, double-, and perturbative triple-excitations, CCSD(T), has led to the method being considered the gold standard of computational chemistry. The application of CCSD(T) has been limited to medium-sized molecular systems due to its steep scaling with molecular size. The recent development of alternative domain-based local pair natural orbital coupled-cluster method, DLPNO-CCSD(T), has significantly broadened the range of chemical systems to which CCSD(T) level calculations can be applied. Condensed systems such as ionic liquids (ILs) have a large contribution from London dispersion forces of up to 150 kJ mol−1in large-scale clusters. Ionic liquids show appreciable charge transfer effects that result in the increased valence orbital delocalization over the entire ionic network, raising the question whether the application of methods based on localized orbitals is reliable for these semi-Coulombic materials. Here the performance of DLPNO-CCSD(T) is validated for the prediction ofcorrelation interaction energiesof two data sets incorporating single-ion pairs of protic and aprotic ILs. DLPNO-CCSD(T) produced results within chemical accuracy with tight parameter settings and a non-iterative treatment of triple excitations. To achieve spectroscopic accuracy of 1 kJ mol−1, especially for hydrogen-bonded ILs and those containing halides, the DLPNO settings had to be increased by two orders of magnitude and include the iterative treatment of triple excitations, resulting in a 2.5-fold increase in computational cost. Two new sets of parameters are put forward to produce the performance of DLPNO-CCSD(T) within chemical and spectroscopic accuracy.},
  archive      = {J_JCC},
  author       = {Zoe L. Seeger and Ekaterina I. Izgorodina},
  doi          = {10.1002/jcc.26776},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {106-120},
  shortjournal = {J. Comput. Chem.},
  title        = {A DLPNO-CCSD(T) benchmarking study of intermolecular interactions of ionic liquids},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022b). Graphene-induced planarization of cyclooctatetraene
derivatives. <em>JCC</em>, <em>43</em>(2), 96–105. (<a
href="https://doi.org/10.1002/jcc.26774">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Stable equilibrium compounds containing a planar antiaromatic cyclooctatetraene (COT) ring are promising candidates for organic electronic devices such as organic semiconductor transistors. The planarization of COT by incorporation into rigid planar π-systems, as well as by oxidation or reduction has attracted considerable attention in recent years. Using dispersion-corrected density functional theory calculations, we explore an alternative approach of planarizing COT derivatives by adsorption onto graphene. We show that strong π–π stacking interactions between graphene and COT derivatives induce a planar structure with an antiaromatic central COT ring. In addition to being reversible, this strategy provides a novel approach for planarizing COT without the need for incorporation into a rigid structure, atomic substitution, oxidation, or reduction.},
  archive      = {J_JCC},
  author       = {Asja A. Kroeger and Amir Karton},
  doi          = {10.1002/jcc.26774},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {96-105},
  shortjournal = {J. Comput. Chem.},
  title        = {Graphene-induced planarization of cyclooctatetraene derivatives},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). CIFDock: A novel CHARMM-based flexible receptor–flexible
ligand docking protocol. <em>JCC</em>, <em>43</em>(2), 84–95. (<a
href="https://doi.org/10.1002/jcc.26759">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Docking studies play a critical role in the current workflow of drug discovery. However, limitations may often arise through factors including inadequate ligand sampling, a lack of protein flexibility, scoring function inadequacies (e.g., due to metals, co-factors, etc.), and difficulty in retaining explicit water molecules. Herein, we present a novel CHARMM-based induced fit docking (CIFDock) workflow that can circumvent these limitations by employing all-atom force fields coupled to enhanced sampling molecular dynamics procedures. Self-guided Langevin dynamics simulations are used to effectively sample relevant ligand conformations, side chain orientations, crystal water positions, and active site residue motion. Protein flexibility is further enhanced by dynamic sampling of side chain orientations using an expandable rotamer library. Steps in the procedure consisting of fixing individual components (e.g., the ligand) while sampling the other components (e.g., the residues in the active site of the protein) allow for the complex to adapt to conformational changes. Ultimately, all components of the complex—the protein, ligand, and waters—are sampled simultaneously and unrestrained with SGLD to capture any induced fit effects. This modular flexible docking procedure is automated using CHARMM scripting, interfaced with SLURM array processing, and parallelized to use the desired number of processors. We validated the CIFDock procedure by performing cross-docking studies using a data set comprised of 21 pharmaceutically relevant proteins. Five variants of the CHARMM-based SWISSDOCK scoring functions were created to quantify the results of the final generated poses. Results obtained were comparable to, or in some cases improved upon, commercial docking program data.},
  archive      = {J_JCC},
  author       = {Sai L. Vankayala and Luke C. Warrensford and Amanda R. Pittman and Benjamin C. Pollard and Fiona L. Kearns and Joseph D. Larkin and H. Lee Woodcock},
  doi          = {10.1002/jcc.26759},
  journal      = {Journal of Computational Chemistry},
  number       = {2},
  pages        = {84-95},
  shortjournal = {J. Comput. Chem.},
  title        = {CIFDock: A novel CHARMM-based flexible receptor–flexible ligand docking protocol},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). MP-NeRF: A massively parallel method for accelerating
protein structure reconstruction from internal coordinates.
<em>JCC</em>, <em>43</em>(1), 74–78. (<a
href="https://doi.org/10.1002/jcc.26768">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The conversion of proteins between internal and cartesian coordinates is a limiting step in many pipelines, such as molecular dynamics simulations and machine learning models. This conversion is typically carried out by sequential or parallel applications of the Natural extension of Reference Frame (NeRF) algorithm. This work proposes a massively parallel NeRF implementation which, depending on the polymer length, achieves speedups between 400 and 1200× over the previous state-of-the-art. It accomplishes this by dividing the conversion into three main phases: parallel composition of the monomer backbone, assembly of backbone subunits, and parallel elongation of sidechains; and by batching these computations into a minimal number of efficient matrix operations. Special emphasis is placed on reusability and ease of use. We open source the code (available athttps://github.com/EleutherAI/mp_nerf) and provide a corresponding python package.},
  archive      = {J_JCC},
  author       = {Eric Alcaide and Stella Biderman and Amalio Telenti and M. Cyrus Maher},
  doi          = {10.1002/jcc.26768},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {74-78},
  shortjournal = {J. Comput. Chem.},
  title        = {MP-NeRF: A massively parallel method for accelerating protein structure reconstruction from internal coordinates},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Effects of mussel-inspired co-deposition of 2-hydroxymethyl
methacrylate and poly (2-methoxyethyl acrylate) on the hydrophilicity
and binding tendency of common hemodialysis membranes: Molecular
dynamics simulations and molecular docking studies. <em>JCC</em>,
<em>43</em>(1), 57–73. (<a
href="https://doi.org/10.1002/jcc.26773">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite advances in the field, hemoincompatibility remains a critical issue for hemodialysis (HD) as interactions between various human blood constituents and the polymeric structure of HD membranes results in complications such as activation of immune system cascades. Adding hydrophilic polymer structures to the membranes is one modification approach that can decrease the extent of protein adsorption. This study conducted molecular dynamics (MD) simulations to understand the interactions between three human serum proteins (fibrinogen [FB], human serum albumin, and transferrin) and common HD membranes in untreated and modified forms. Poly(aryl ether sulfone) (PAES) and cellulose triacetate were used as the common dialyzer polymers, and membrane modifications were performed with 2-hydroxymethyl methacrylate (HEMA) and poly (2-methoxyethyl acrylate) (PMEA), using polydopamine-assisted co-deposition. The MD simulations were used as the framework for binding energy simulations, and molecular docking simulations were also performed to conduct molecular-level investigations between the two modifying polymers (HEMA and PMEA) and FB. Each of the three proteins acted differently with the membranes due to their unique nature and surface chemistry. The simulations show PMEA binds less intensively to FB with a higher number of hydrogen bonds, which reflects PMEA&#39;s superior performance compared to HEMA. The simulations suggest PAES membranes could be used in modified forms for blood-contact applications as they reflect the lowest binding energy to blood proteins.},
  archive      = {J_JCC},
  author       = {Arash Mollahosseini and Shaghayegh Saadati and Amira Abdelrasoul},
  doi          = {10.1002/jcc.26773},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {57-73},
  shortjournal = {J. Comput. Chem.},
  title        = {Effects of mussel-inspired co-deposition of 2-hydroxymethyl methacrylate and poly (2-methoxyethyl acrylate) on the hydrophilicity and binding tendency of common hemodialysis membranes: Molecular dynamics simulations and molecular docking studies},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Estimating reaction parameters in mechanism-enabled
population balance models of nanoparticle size distributions: A bayesian
inverse problem approach. <em>JCC</em>, <em>43</em>(1), 43–56. (<a
href="https://doi.org/10.1002/jcc.26770">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In order to quantitatively predict nano- as well as other particle-size distributions, one needs to have both a mathematical model and estimates of the parameters that appear in these models. Here, we show how one can use Bayesian inversion to obtain statistical estimates for the parameters that appear in recently derived mechanism-enabled population balance models (ME-PBM) of nanoparticle growth. The Bayesian approach addresses the question of “how well do we know our parameters, along with their uncertainties?.” The results reveal that Bayesian inversion statistical analysis on an example, prototypenanoparticle formation system allows one to estimate not just the most likely rate constants and other parameter values, but also their SDs, confidence intervals, and other statistical information. Moreover, knowing the reliability of the mechanistic model&#39;s parameters in turn helps inform one about the reliability of the proposed mechanism, as well as the reliability of its predictions. The paper can also be seen as a tutorial with the additional goal of achieving a “Gold Standard” Bayesian inversion ME-PBM benchmark that others can use as a control to check their own use of this methodology for other systems of interest throughout nature. Overall, the results provide strong support for the hypothesis that there is substantial value in using a Bayesian inversion methodology for parameter estimation in particle formation systems.},
  archive      = {J_JCC},
  author       = {Danny K. Long and Wolfgang Bangerth and Derek R. Handwerk and Christopher B. Whitehead and Patrick D. Shipman and Richard G. Finke},
  doi          = {10.1002/jcc.26770},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {43-56},
  shortjournal = {J. Comput. Chem.},
  title        = {Estimating reaction parameters in mechanism-enabled population balance models of nanoparticle size distributions: A bayesian inverse problem approach},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Analysis in silico of chemical reactivity employing the
local hyper-softness in some classic aromatic compounds, boron aromatic
clusters and all-metal aromatic clusters. <em>JCC</em>, <em>43</em>(1),
29–42. (<a href="https://doi.org/10.1002/jcc.26769">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the current work, the authors analyzed and compared the chemical behavior of some (anti)aromatic compounds. The species selected are benzene and cyclobutadiene as the aromatic and antiaromatic classical examples, respectively. Next, the anion, which is the first all-metal molecule catalogued as aromatic and its non-metallic isoelectronic analog,were also analyzed. The antiaromatic clustersandwere studied in form of lithium salts. And the end, the non-planarboron cluster and its isoelectronic analogs (and) were considered for being analyzed under the same criterium. The analysis was realized employing the following descriptors: molecular electrostatic potential and local hyper-softness to get insights concerning local reactivity when facing reagents leading to ionic or covalent interactions. The results show that all the molecules analyzed presented some specific variations in their respective local reactivity despite being labeled as aromatic compounds. This analysis provides a notion that the local reactivity is more based on the nature of the atoms and the molecular geometry than the aromatic character by itself.},
  archive      = {J_JCC},
  author       = {Jorge I. Martínez-Araya and Rafael Islas},
  doi          = {10.1002/jcc.26769},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {29-42},
  shortjournal = {J. Comput. Chem.},
  title        = {Analysis in silico of chemical reactivity employing the local hyper-softness in some classic aromatic compounds, boron aromatic clusters and all-metal aromatic clusters},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Benchmark ab initio proton affinity and gas-phase basicity
of α-alanine based on coupled-cluster theory and statistical mechanics.
<em>JCC</em>, <em>43</em>(1), 19–28. (<a
href="https://doi.org/10.1002/jcc.26767">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We determine the proton affinity (PA) and gas-phase basicity (GB) of amino acid α-alanine at a chemically accurate level by performing explicitly-correlated CCSD(T)-F12b/aug-cc-pVDZ geometry optimizations and normal mode vibrational frequency calculations as well as CCSD(T)-F12b/aug-cc-pVTZ energy computations at the possible neutral and protonated geometries. Temperature effects at 298.15 K considering translational, rotational, and vibrational enthalpy and entropy corrections are obtained via standard statistical mechanics utilizing the molecular geometries and the harmonic vibrational energy levels. Both the amino nitrogen (N) and the carbonyl oxygen (O) atoms are proven to be potential protonation sites and a systematic conformational search reveals 3 N- and 9 O-protonated conformers in the 0.00–7.88 and 25.43–30.43 kcal/mol energy ranges at 0 K, respectively. The final computed PA and GB values at (0)298.15 K in case of N-protonation are (214.47)216.80 and 207.07 kcal/mol, respectively, whereas the corresponding values for O-protonation are (189.04)190.63 and 182.31 kcal/mol. The results of the benchmark high-level coupled-cluster computations are utilized to assess the accuracy of several lower-level cost-effective methods such as MP2 and density functional theory with various functionals (SOGGA11-X, M06-2X, PBE0, B3LYP, M06, TPSS).},
  archive      = {J_JCC},
  author       = {Attila Á. Dékány and Gábor Czakó},
  doi          = {10.1002/jcc.26767},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {19-28},
  shortjournal = {J. Comput. Chem.},
  title        = {Benchmark ab initio proton affinity and gas-phase basicity of α-alanine based on coupled-cluster theory and statistical mechanics},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
<li><details>
<summary>
(2022). Advances in vibrational configuration interaction theory -
part 2: Fast screening of the correlation space. <em>JCC</em>,
<em>43</em>(1), 6–18. (<a
href="https://doi.org/10.1002/jcc.26764">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {For larger molecules, the computational demands of configuration selective vibrational configuration interaction theory (cs-VCI) are usually dominated by the configuration selection process, which commonly is based on second order vibrational Møller-Plesset perturbation (VMP2) theory. Here we present two techniques, which lead to substantial accelerations of such calculations while retaining the desired high accuracy of the final results. The first one introduces the concept of configuration classes, which allows for a highly efficient exploitation of the analogs of the Slater-Condon rules in vibrational structure calculations with large correlation spaces. The second approach uses a VMP2 like vector for augmenting the targeted vibrational wavefunction within the selection of configurations and thus avoids any intermediate diagonalization steps. The underlying theory is outlined and benchmark calculations are provided for highly correlated vibrational states of several molecules.},
  archive      = {J_JCC},
  author       = {Tina Mathea and Taras Petrenko and Guntram Rauhut},
  doi          = {10.1002/jcc.26764},
  journal      = {Journal of Computational Chemistry},
  number       = {1},
  pages        = {6-18},
  shortjournal = {J. Comput. Chem.},
  title        = {Advances in vibrational configuration interaction theory - part 2: Fast screening of the correlation space},
  volume       = {43},
  year         = {2022},
}
</textarea>
</details></li>
</ul>

</body>
</html>
