<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NMETH_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="nmeth---37">NMETH - 37</h2>
<ul>
<li><details>
<summary>
(2025b). The value of lab values. <em>NMETH</em>, <em>22</em>(3),
451. (<a href="https://doi.org/10.1038/s41592-025-02602-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {These researchers put their labs’ philosophies into practice and find it empowers science and collaboration.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02602-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {3},
  pages        = {451},
  shortjournal = {Nat. Meth.},
  title        = {The value of lab values},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Author correction: Large-scale benchmarking of circRNA
detection tools reveals large differences in sensitivity but not in
precision. <em>NMETH</em>, <em>22</em>(2), 448. (<a
href="https://doi.org/10.1038/s41592-024-02569-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vromman, Marieke and Anckaert, Jasper and Bortoluzzi, Stefania and Buratin, Alessia and Chen, Chia-Ying and Chu, Qinjie and Chuang, Trees-Juen and Dehghannasiri, Roozbeh and Dieterich, Christoph and Dong, Xin and Flicek, Paul and Gaffo, Enrico and Gu, Wanjun and He, Chunjiang and Hoffmann, Steve and Izuogu, Osagie and Jackson, Michael S. and Jakobi, Tobias and Lai, Eric C. and Nuytens, Justine and Salzman, Julia and Santibanez-Koref, Mauro and Stadler, Peter and Thas, Olivier and Vanden Eynde, Eveline and Verniers, Kimberly and Wen, Guoxia and Westholm, Jakub and Yang, Li and Ye, Chu-Yu and Yigit, Nurten and Yuan, Guo-Hua and Zhang, Jinyang and Zhao, Fangqing and Vandesompele, Jo and Volders, Pieter-Jan},
  doi          = {10.1038/s41592-024-02569-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {448},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: Large-scale benchmarking of circRNA detection tools reveals large differences in sensitivity but not in precision},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Author correction: Cell painting: A decade of discovery and
innovation in cellular imaging. <em>NMETH</em>, <em>22</em>(2), 447. (<a
href="https://doi.org/10.1038/s41592-024-02578-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Seal, Srijit and Trapotsi, Maria-Anna and Spjuth, Ola and Singh, Shantanu and Carreras-Puigvert, Jordi and Greene, Nigel and Bender, Andreas and Carpenter, Anne E.},
  doi          = {10.1038/s41592-024-02578-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {447},
  shortjournal = {Nat. Meth.},
  title        = {Author correction: cell painting: a decade of discovery and innovation in cellular imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Publisher correction: Massively parallel single-cell
sequencing of diverse microbial populations. <em>NMETH</em>,
<em>22</em>(2), 446. (<a
href="https://doi.org/10.1038/s41592-024-02570-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Lan, Freeman and Saba, Jason and Ross, Tyler D. and Zhou, Zhichao and Krauska, Katie and Anantharaman, Karthik and Landick, Robert and Venturelli, Ophelia S.},
  doi          = {10.1038/s41592-024-02570-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {446},
  shortjournal = {Nat. Meth.},
  title        = {Publisher correction: Massively parallel single-cell sequencing of diverse microbial populations},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Integrative mapping of human CD8+ t cells in inflammation
and cancer. <em>NMETH</em>, <em>22</em>(2), 435–445. (<a
href="https://doi.org/10.1038/s41592-024-02530-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {CD8+ T cells exhibit remarkable phenotypic diversity in inflammation and cancer. However, a comprehensive understanding of their clonal landscape and dynamics remains elusive. Here we introduce scAtlasVAE, a deep-learning-based model for the integration of large-scale single-cell RNA sequencing data and cross-atlas comparisons. scAtlasVAE has enabled us to construct an extensive human CD8+ T cell atlas, comprising 1,151,678 cells from 961 samples across 68 studies and 42 disease conditions, with paired T cell receptor information. Through incorporating information in T cell receptor clonal expansion and sharing, we have successfully established connections between distinct cell subtypes and shed light on their phenotypic and functional transitions. Notably, our approach characterizes three distinct exhausted T cell subtypes and reveals diverse transcriptome and clonal sharing patterns in autoimmune and immune-related adverse event inflammation. Furthermore, scAtlasVAE facilitates the automatic annotation of CD8+ T cell subtypes in query single-cell RNA sequencing datasets, enabling unbiased and scalable analyses. In conclusion, our work presents a comprehensive single-cell reference and computational framework for CD8+ T cell research. scAtlasVAE is a deep learning-based model for cross-atlas integration. Here it enables the development of a large-scale human CD8+ T cell atlas with integrated T cell receptor data.},
  archive      = {J_NMETH},
  author       = {Xue, Ziwei and Wu, Lize and Tian, Ruonan and Gao, Bing and Zhao, Yu and He, Bing and Sun, Di and Zhao, Bingkang and Li, Yicheng and Zhu, Kaixiang and Wang, Lie and Yao, Jianhua and Liu, Wanlu and Lu, Linrong},
  doi          = {10.1038/s41592-024-02530-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {435-445},
  shortjournal = {Nat. Meth.},
  title        = {Integrative mapping of human CD8+ t cells in inflammation and cancer},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). The dynamics of hematopoiesis over the human lifespan.
<em>NMETH</em>, <em>22</em>(2), 422–434. (<a
href="https://doi.org/10.1038/s41592-024-02495-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Over a lifetime, hematopoietic stem cells (HSCs) adjust their lineage output to support age-aligned physiology. In model organisms, stereotypic waves of hematopoiesis have been observed corresponding to defined age-biased HSC hallmarks. However, how the properties of hematopoietic stem and progenitor cells change over the human lifespan remains unclear. To address this gap, we profiled individual transcriptome states of human hematopoietic stem and progenitor cells spanning gestation, maturation and aging. Here we define the gene expression networks dictating age-specific differentiation of HSCs and the dynamics of fate decisions and lineage priming throughout life. We additionally identifiy and functionally validate a fetal-specific HSC state with robust engraftment and multilineage capacity. Furthermore, we observe that classification of acute myeloid leukemia against defined transcriptional age states demonstrates that utilization of early life transcriptional programs associates with poor prognosis. Overall, we provide a disease-relevant framework for heterochronic orientation of stem cell ontogeny along the real time axis of the human lifespan. This Resource presents an atlas classifying the age-related changes in human hematopoietic stem and progenitor cells from gestation through aging.},
  archive      = {J_NMETH},
  author       = {Li, Hojun and Côté, Parker and Kuoch, Michael and Ezike, Jideofor and Frenis, Katie and Afanassiev, Anton and Greenstreet, Laura and Tanaka-Yano, Mayuri and Tarantino, Giuseppe and Zhang, Stephen and Whangbo, Jennifer and Butty, Vincent L. and Moiso, Enrico and Falchetti, Marcelo and Lu, Kate and Connelly, Guinevere G. and Morris, Vivian and Wang, Dahai and Chen, Antonia F. and Bianchi, Giada and Daley, George Q. and Garg, Salil and Liu, David and Chou, Stella T. and Regev, Aviv and Lummertz da Rocha, Edroaldo and Schiebinger, Geoffrey and Rowe, R. Grant},
  doi          = {10.1038/s41592-024-02495-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {422-434},
  shortjournal = {Nat. Meth.},
  title        = {The dynamics of hematopoiesis over the human lifespan},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A proteome-wide quantitative platform for nanoscale
spatially resolved extraction of membrane proteins into native
nanodiscs. <em>NMETH</em>, <em>22</em>(2), 412–421. (<a
href="https://doi.org/10.1038/s41592-024-02517-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The native membrane environment profoundly influences every aspect of membrane protein (MP) biology. Despite this, the most prevalent method of studying MPs uses detergents to disrupt and remove this vital membrane context, impeding our ability to decipher the local molecular context and its effect. Here we develop a membrane proteome-wide platform that enables rapid spatially resolved extraction of target MPs directly from cellular membranes into native nanodiscs that maintain the local membrane context, using a library of membrane-active polymers. We accompany this with an open-access database that quantifies the polymer-specific extraction efficiency for 2,065 unique mammalian MPs and provides the most optimized extraction condition for each. To validate, we demonstrate how this resource can enable rapid extraction and purification of target MPs from different organellar membranes with high efficiency and purity. Further, we show how the database can be extended to capture overexpressed multiprotein complexes by taking two synaptic vesicle MPs. We expect these publicly available resources to empower researchers across disciplines to efficiently capture membrane ‘nano-scoops’ containing a target MP and interface with structural, functional and bioanalytical approaches. This manuscript reports a high-throughput platform for nanoscale spatially resolved extraction of membrane proteins into native nanodiscs by using a library of membrane-active polymers while maintaining their local membrane context.},
  archive      = {J_NMETH},
  author       = {Brown, Caroline and Ghosh, Snehasish and McAllister, Rachel and Kumar, Mukesh and Walker, Gerard and Sun, Eric and Aman, Talat and Panda, Aniruddha and Kumar, Shailesh and Li, Wenxue and Coleman, Jeff and Liu, Yansheng and Rothman, James E. and Bhattacharyya, Moitrayee and Gupta, Kallol},
  doi          = {10.1038/s41592-024-02517-x},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {412-421},
  shortjournal = {Nat. Meth.},
  title        = {A proteome-wide quantitative platform for nanoscale spatially resolved extraction of membrane proteins into native nanodiscs},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). RNA-puzzles round v: Blind predictions of 23 RNA structures.
<em>NMETH</em>, <em>22</em>(2), 399–411. (<a
href="https://doi.org/10.1038/s41592-024-02543-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {RNA-Puzzles is a collective endeavor dedicated to the advancement and improvement of RNA three-dimensional structure prediction. With agreement from structural biologists, RNA structures are predicted by modeling groups before publication of the experimental structures. We report a large-scale set of predictions by 18 groups for 23 RNA-Puzzles: 4 RNA elements, 2 Aptamers, 4 Viral elements, 5 Ribozymes and 8 Riboswitches. We describe automatic assessment protocols for comparisons between prediction and experiment. Our analyses reveal some critical steps to be overcome to achieve good accuracy in modeling RNA structures: identification of helix-forming pairs and of non-Watson–Crick modules, correct coaxial stacking between helices and avoidance of entanglements. Three of the top four modeling groups in this round also ranked among the top four in the CASP15 contest. The results of the Fifth RNA-Puzzles contest highlights advances in RNA three-dimensional structure prediction and uncovers new insights into RNA folding and structure.},
  archive      = {J_NMETH},
  author       = {Bu, Fan and Adam, Yagoub and Adamiak, Ryszard W. and Antczak, Maciej and de Aquino, Belisa Rebeca H. and Badepally, Nagendar Goud and Batey, Robert T. and Baulin, Eugene F. and Boinski, Pawel and Boniecki, Michal J. and Bujnicki, Janusz M. and Carpenter, Kristy A. and Chacon, Jose and Chen, Shi-Jie and Chiu, Wah and Cordero, Pablo and Das, Naba Krishna and Das, Rhiju and Dawson, Wayne K. and DiMaio, Frank and Ding, Feng and Dock-Bregeon, Anne-Catherine and Dokholyan, Nikolay V. and Dror, Ron O. and Dunin-Horkawicz, Stanisław and Eismann, Stephan and Ennifar, Eric and Esmaeeli, Reza and Farsani, Masoud Amiri and Ferré-D’Amaré, Adrian R. and Geniesse, Caleb and Ghanim, George E. and Guzman, Horacio V. and Hood, Iris V. and Huang, Lin and Jain, Dharm Skandh and Jaryani, Farhang and Jin, Lei and Joshi, Astha and Karelina, Masha and Kieft, Jeffrey S. and Kladwang, Wipapat and Kmiecik, Sebastian and Koirala, Deepak and Kollmann, Markus and Kretsch, Rachael C. and Kurciński, Mateusz and Li, Jun and Li, Shuang and Magnus, Marcin and Masquida, BenoÎt and Moafinejad, S. Naeim and Mondal, Arup and Mukherjee, Sunandan and Nguyen, Thi Hoang Duong and Nikolaev, Grigory and Nithin, Chandran and Nye, Grace and Pandaranadar Jeyeram, Iswarya P. N. and Perez, Alberto and Pham, Phillip and Piccirilli, Joseph A. and Pilla, Smita Priyadarshini and Pluta, Radosław and Poblete, Simón and Ponce-Salvatierra, Almudena and Popenda, Mariusz and Popenda, Lukasz and Pucci, Fabrizio and Rangan, Ramya and Ray, Angana and Ren, Aiming and Sarzynska, Joanna and Sha, Congzhou Mike and Stefaniak, Filip and Su, Zhaoming and Suddala, Krishna C. and Szachniuk, Marta and Townshend, Raphael and Trachman, Robert J. and Wang, Jian and Wang, Wenkai and Watkins, Andrew and Wirecki, Tomasz K. and Xiao, Yi and Xiong, Peng and Xiong, Yiduo and Yang, Jianyi and Yesselman, Joseph David and Zhang, Jinwei and Zhang, Yi and Zhang, Zhenzhen and Zhou, Yuanzhe and Zok, Tomasz and Zhang, Dong and Zhang, Sicheng and Żyła, Adriana and Westhof, Eric and Miao, Zhichao},
  doi          = {10.1038/s41592-024-02543-9},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {399-411},
  shortjournal = {Nat. Meth.},
  title        = {RNA-puzzles round v: Blind predictions of 23 RNA structures},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Moculus: An immersive virtual reality system for mice
incorporating stereo vision. <em>NMETH</em>, <em>22</em>(2), 386–398.
(<a href="https://doi.org/10.1038/s41592-024-02554-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Due to technical roadblocks, it is unclear how visual circuits represent multiple features or how behaviorally relevant representations are selected for long-term memory. Here we developed Moculus, a head-mounted virtual reality platform for mice that covers the entire visual field, and allows binocular depth perception and full visual immersion. This controllable environment, with three-dimensional (3D) corridors and 3D objects, in combination with 3D acousto-optical imaging, affords rapid visual learning and the uncovering of circuit substrates in one measurement session. Both the control and reinforcement-associated visual cue coding neuronal assemblies are transiently expanded by reinforcement feedback to near-saturation levels. This increases computational capability and allows competition among assemblies that encode behaviorally relevant information. The coding assemblies form partially orthogonal and overlapping clusters centered around hub cells with higher and earlier ramp-like responses, as well as locally increased functional connectivity. Moculus is a virtual reality headset for mice that allows realistic display of three-dimensional environments. The system improves performance in visual-learning tasks and can be combined with various microscopes to study neural circuitry.},
  archive      = {J_NMETH},
  author       = {Judák, Linda and Dobos, Gergely and Ócsai, Katalin and Báthory, Eszter and Szebik, Huba and Tarján, Balázs and Maák, Pál and Szadai, Zoltán and Takács, István and Chiovini, Balázs and Lőrincz, Tibor and Szepesi, Áron and Roska, Botond and Szalay, Gergely and Rózsa, Balázs},
  doi          = {10.1038/s41592-024-02554-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {386-398},
  shortjournal = {Nat. Meth.},
  title        = {Moculus: An immersive virtual reality system for mice incorporating stereo vision},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MouseGoggles: An immersive virtual reality headset for mouse
neuroscience and behavior. <em>NMETH</em>, <em>22</em>(2), 380–385. (<a
href="https://doi.org/10.1038/s41592-024-02540-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Small-animal virtual reality (VR) systems have become invaluable tools in neuroscience for studying complex behavior during head-fixed neural recording, but they lag behind commercial human VR systems in terms of miniaturization, immersivity and advanced features such as eye tracking. Here we present MouseGoggles, a miniature VR headset for head-fixed mice that delivers independent, binocular visual stimulation over a wide field of view while enabling eye tracking and pupillometry in VR. Neural recordings in the visual cortex validate the quality of image presentation, while hippocampal recordings, associative reward learning and innate fear responses to virtual looming stimuli demonstrate an immersive VR experience. Our open-source system’s simplicity and compact size will enable the broader adoption of VR methods in neuroscience. MouseGoggles is a miniaturized virtual reality (VR) headset for mice that provides an improved immersive experience compared with existing VR equipment. MouseGoggles can also be combined with pupil tracking and has been applied in combination with calcium imaging or electrophysiological recordings.},
  archive      = {J_NMETH},
  author       = {Isaacson, Matthew and Chang, Hongyu and Berkowitz, Laura and Zirkel, Rick and Park, Yusol and Hu, Danyu and Ellwood, Ian and Schaffer, Chris B.},
  doi          = {10.1038/s41592-024-02540-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {380-385},
  shortjournal = {Nat. Meth.},
  title        = {MouseGoggles: An immersive virtual reality headset for mouse neuroscience and behavior},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Vibrational fiber photometry: Label-free and reporter-free
minimally invasive raman spectroscopy deep in the mouse brain.
<em>NMETH</em>, <em>22</em>(2), 371–379. (<a
href="https://doi.org/10.1038/s41592-024-02557-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Optical approaches to monitor neural activity are transforming neuroscience, owing to a fast-evolving palette of genetically encoded molecular reporters. However, the field still requires robust and label-free technologies to monitor the multifaceted biomolecular changes accompanying brain development, aging or disease. Here, we have developed vibrational fiber photometry as a low-invasive method for label-free monitoring of the biomolecular content of arbitrarily deep regions of the mouse brain in vivo through spontaneous Raman spectroscopy. Using a tapered fiber probe as thin as 1 µm at its tip, we elucidate the cytoarchitecture of the mouse brain, monitor molecular alterations caused by traumatic brain injury, as well as detect markers of brain metastasis with high accuracy. We view our approach, which introduces a deep learning algorithm to suppress probe background, as a promising complement to the existing palette of tools for the optical interrogation of neural function, with application to fundamental and preclinical investigations of the brain and other organs. Vibrational fiber photometry enables Raman spectroscopy in the mouse brain with low impact, owing to the use of tapered fibers.},
  archive      = {J_NMETH},
  author       = {Pisano, Filippo and Masmudi-Martín, Mariam and Andriani, Maria Samuela and Cid, Elena and Kazemzadeh, Mohammadrahim and Pisanello, Marco and Balena, Antonio and Collard, Liam and Parras, Teresa Jurado and Bianco, Marco and Baena, Patricia and Tantussi, Francesco and Grande, Marco and Sileo, Leonardo and Gentile, Francesco and De Angelis, Francesco and De Vittorio, Massimo and Menendez de la Prida, Liset and Valiente, Manuel and Pisanello, Ferruccio},
  doi          = {10.1038/s41592-024-02557-3},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {371-379},
  shortjournal = {Nat. Meth.},
  title        = {Vibrational fiber photometry: Label-free and reporter-free minimally invasive raman spectroscopy deep in the mouse brain},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiplexing cortical brain organoids for the longitudinal
dissection of developmental traits at single-cell resolution.
<em>NMETH</em>, <em>22</em>(2), 358–370. (<a
href="https://doi.org/10.1038/s41592-024-02555-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dissecting human neurobiology at high resolution and with mechanistic precision requires a major leap in scalability, given the need for experimental designs that include multiple individuals and, prospectively, population cohorts. To lay the foundation for this, we have developed and benchmarked complementary strategies to multiplex brain organoids by pooling cells from different pluripotent stem cell (PSC) lines either during organoid generation (mosaic models) or before single-cell RNA sequencing (scRNA-seq) library preparation (downstream multiplexing). We have also developed a new computational method, SCanSNP, and a consensus call to deconvolve cell identities, overcoming current criticalities in doublets and low-quality cell identification. We validated both multiplexing methods for charting neurodevelopmental trajectories at high resolution, thus linking specific individuals’ trajectories to genetic variation. Finally, we modeled their scalability across different multiplexing combinations and showed that mosaic organoids represent an enabling method for high-throughput settings. Together, this multiplexing suite of experimental and computational methods provides a highly scalable resource for brain disease and neurodiversity modeling. This paper develops two approaches for multiplexing cortical organoids and SCanSNP, a method for deconvolving cell identities, to trace neurodevelopmental trajectories at scale.},
  archive      = {J_NMETH},
  author       = {Caporale, Nicolò and Castaldi, Davide and Rigoli, Marco Tullio and Cheroni, Cristina and Valenti, Alessia and Stucchi, Sarah and Lessi, Manuel and Bulgheresi, Davide and Trattaro, Sebastiano and Pezzali, Martina and Vitriolo, Alessandro and Lopez-Tobon, Alejandro and Bonfanti, Matteo and Ricca, Dario and Schmid, Katharina T. and Heinig, Matthias and Theis, Fabian J. and Villa, Carlo Emanuele and Testa, Giuseppe},
  doi          = {10.1038/s41592-024-02555-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {358-370},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexing cortical brain organoids for the longitudinal dissection of developmental traits at single-cell resolution},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). CelloType: A unified model for segmentation and
classification of tissue images. <em>NMETH</em>, <em>22</em>(2),
348–357. (<a href="https://doi.org/10.1038/s41592-024-02513-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cell segmentation and classification are critical tasks in spatial omics data analysis. Here we introduce CelloType, an end-to-end model designed for cell segmentation and classification for image-based spatial omics data. Unlike the traditional two-stage approach of segmentation followed by classification, CelloType adopts a multitask learning strategy that integrates these tasks, simultaneously enhancing the performance of both. CelloType leverages transformer-based deep learning techniques for improved accuracy in object detection, segmentation and classification. It outperforms existing segmentation methods on a variety of multiplexed fluorescence and spatial transcriptomic images. In terms of cell type classification, CelloType surpasses a model composed of state-of-the-art methods for individual tasks and a high-performance instance segmentation model. Using multiplexed tissue images, we further demonstrate the utility of CelloType for multiscale segmentation and classification of both cellular and noncellular elements in a tissue. The enhanced accuracy and multitask learning ability of CelloType facilitate automated annotation of rapidly growing spatial omics data. CelloType is an end-to-end method for spatial omics data analysis that uses a transformer-based deep neural network for concurrent object detection, segmentation and classification and performs with high accuracy on diverse datasets.},
  archive      = {J_NMETH},
  author       = {Pang, Minxing and Roy, Tarun Kanti and Wu, Xiaodong and Tan, Kai},
  doi          = {10.1038/s41592-024-02513-1},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {348-357},
  shortjournal = {Nat. Meth.},
  title        = {CelloType: A unified model for segmentation and classification of tissue images},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Elucidating subcellular architecture and dynamics at
isotropic 100-nm resolution with 4Pi-SIM. <em>NMETH</em>,
<em>22</em>(2), 335–347. (<a
href="https://doi.org/10.1038/s41592-024-02515-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Three-dimensional structured illumination microscopy (3D-SIM) provides excellent optical sectioning and doubles the resolution in all dimensions compared with wide-field microscopy. However, its much lower axial resolution results in blurred fine details in that direction and overall image distortion. Here we present 4Pi-SIM, a substantial revamp of I5S that synergizes 3D-SIM with interferometric microscopy to achieve isotropic optical resolution through interference in both the illumination and detection wavefronts. We evaluate the performance of 4Pi-SIM by imaging various subcellular structures across different cell types with high fidelity. Furthermore, we demonstrate its capability by conducting time-lapse volumetric imaging over hundreds of time points, achieving a 3D resolution of approximately 100 nm. Additionally, we illustrate its ability to simultaneously image in two colors and capture the rapid interactions between closely positioned organelles in three dimensions. These results underscore the great potential of 4Pi-SIM for elucidating subcellular architecture and revealing dynamic behaviors at the nanoscale. The 4Pi-SIM microscope combines the benefits of structured illumination microscopy with 4Pi geometry for improved axial resolution, offering isotropic, multicolor super-resolution imaging of dynamic processes in living cells.},
  archive      = {J_NMETH},
  author       = {Ouyang, Zijing and Wang, Qian and Li, Xiaoyu and Dai, Qiuyang and Tang, Muyuan and Shao, Lin and Gou, Wen and Yu, Zijing and Chen, Yanqin and Zheng, Bei and Chen, Linlin and Ping, Conghui and Bi, Xiuli and Xiao, Bin and Yu, Xiaochun and Liu, Changliang and Chen, Liangyi and Fan, Junchao and Huang, Xiaoshuai and Zhang, Yongdeng},
  doi          = {10.1038/s41592-024-02515-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {335-347},
  shortjournal = {Nat. Meth.},
  title        = {Elucidating subcellular architecture and dynamics at isotropic 100-nm resolution with 4Pi-SIM},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). IFlpMosaics enable the multispectral barcoding and
high-throughput comparative analysis of mutant and wild-type cells.
<em>NMETH</em>, <em>22</em>(2), 323–334. (<a
href="https://doi.org/10.1038/s41592-024-02534-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {To understand gene function, it is necessary to compare cells carrying the mutated target gene with normal cells. In most biomedical studies, the cells being compared are in different mutant and control animals and, therefore, do not experience the same epigenetic changes and tissue microenvironment. The experimental induction of genetic mosaics is essential to determine a gene cell-autonomous function and to model the etiology of diseases caused by somatic mutations. Current technologies used to induce genetic mosaics in mice lack either accuracy, throughput or barcoding diversity. Here we present the iFlpMosaics toolkit comprising a large set of new genetic tools and mouse lines that enable recombinase-dependent ratiometric induction and single-cell clonal tracking of multiple fluorescently labeled wild-type and Cre-mutant cells within the same time window and tissue microenvironment. The labeled cells can be profiled by multispectral imaging or by fluorescence-activated flow cytometry and single-cell RNA sequencing. iFlpMosaics facilitate the induction and analysis of genetic mosaics in any quiescent or progenitor cell, and for any given single or combination of floxed genes, thus enabling a more accurate understanding of how induced genetic mutations affect the biology of single cells during tissue development, homeostasis and disease. iFlpMosaics are a compendium of mouse lines and genetic tools for tissue-agnostic generation and analysis of genetic mosaics.},
  archive      = {J_NMETH},
  author       = {Garcia-Gonzalez, Irene and Gambera, Stefano and Rocha, Susana F. and Regano, Alvaro and Garcia-Ortega, Lourdes and Lytvyn, Mariya and Diago-Domingo, Luis and Sanchez-Muñoz, Maria S. and Garcia-Cabero, Aroa and Zagorac, Ivana and Luo, Wen and De Andrés-Laguillo, Macarena and Fernández-Chacón, Macarena and Casquero-Garcia, Verónica and Lunella, Federica Francesca and Torroja, Carlos and Sánchez-Cabo, Fátima and Benedito, Rui},
  doi          = {10.1038/s41592-024-02534-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {323-334},
  shortjournal = {Nat. Meth.},
  title        = {IFlpMosaics enable the multispectral barcoding and high-throughput comparative analysis of mutant and wild-type cells},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SurfDock is a surface-informed diffusion generative model
for reliable and accurate protein–ligand complex prediction.
<em>NMETH</em>, <em>22</em>(2), 310–322. (<a
href="https://doi.org/10.1038/s41592-024-02516-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Accurately predicting protein–ligand interactions is crucial for understanding cellular processes. We introduce SurfDock, a deep-learning method that addresses this challenge by integrating protein sequence, three-dimensional structural graphs and surface-level features into an equivariant architecture. SurfDock employs a generative diffusion model on a non-Euclidean manifold, optimizing molecular translations, rotations and torsions to generate reliable binding poses. Our extensive evaluations across various benchmarks demonstrate SurfDock’s superiority over existing methods in docking success rates and adherence to physical constraints. It also exhibits remarkable generalizability to unseen proteins and predicted apo structures, while achieving state-of-the-art performance in virtual screening tasks. In a real-world application, SurfDock identified seven novel hit molecules in a virtual screening project targeting aldehyde dehydrogenase 1B1, a key enzyme in cellular metabolism. This showcases SurfDock’s ability to elucidate molecular mechanisms underlying cellular processes. These results highlight SurfDock’s potential as a transformative tool in structural biology, offering enhanced accuracy, physical plausibility and practical applicability in understanding protein–ligand interactions. SurfDock is a method for predicting protein–ligand complex structures by leveraging multimodal protein information and generative diffusion frameworks. Its results can be generalized to unseen proteins and real-world settings.},
  archive      = {J_NMETH},
  author       = {Cao, Duanhua and Chen, Mingan and Zhang, Runze and Wang, Zhaokun and Huang, Manlin and Yu, Jie and Jiang, Xinyu and Fan, Zhehuan and Zhang, Wei and Zhou, Hao and Li, Xutong and Fu, Zunyun and Zhang, Sulin and Zheng, Mingyue},
  doi          = {10.1038/s41592-024-02516-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {310-322},
  shortjournal = {Nat. Meth.},
  title        = {SurfDock is a surface-informed diffusion generative model for reliable and accurate protein–ligand complex prediction},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Mapping the topography of spatial gene expression with
interpretable deep learning. <em>NMETH</em>, <em>22</em>(2), 298–309.
(<a href="https://doi.org/10.1038/s41592-024-02503-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spatially resolved transcriptomics technologies provide high-throughput measurements of gene expression in a tissue slice, but the sparsity of these data complicates analysis of spatial gene expression patterns. We address this issue by deriving a topographic map of a tissue slice—analogous to a map of elevation in a landscape—using a quantity called the isodepth. Contours of constant isodepths enclose domains with distinct cell type composition, while gradients of the isodepth indicate spatial directions of maximum change in expression. We develop GASTON (gradient analysis of spatial transcriptomics organization with neural networks), an unsupervised and interpretable deep learning algorithm that simultaneously learns the isodepth, spatial gradients and piecewise linear expression functions that model both continuous gradients and discontinuous variation in gene expression. We show that GASTON accurately identifies spatial domains and marker genes across several tissues, gradients of neuronal differentiation and firing in the brain, and gradients of metabolism and immune activity in the tumor microenvironment. Gene expression topography analysis by GASTON portrays domain organization and spatial gradients of gene expression and cell type composition using spatially resolved transcriptomics data.},
  archive      = {J_NMETH},
  author       = {Chitra, Uthsav and Arnold, Brian J. and Sarkar, Hirak and Sanno, Kohei and Ma, Cong and Lopez-Darwin, Sereno and Raphael, Benjamin J.},
  doi          = {10.1038/s41592-024-02503-3},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {298-309},
  shortjournal = {Nat. Meth.},
  title        = {Mapping the topography of spatial gene expression with interpretable deep learning},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nucleotide transformer: Building and evaluating robust
foundation models for human genomics. <em>NMETH</em>, <em>22</em>(2),
287–297. (<a href="https://doi.org/10.1038/s41592-024-02523-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The prediction of molecular phenotypes from DNA sequences remains a longstanding challenge in genomics, often driven by limited annotated data and the inability to transfer learnings between tasks. Here, we present an extensive study of foundation models pre-trained on DNA sequences, named Nucleotide Transformer, ranging from 50 million up to 2.5 billion parameters and integrating information from 3,202 human genomes and 850 genomes from diverse species. These transformer models yield context-specific representations of nucleotide sequences, which allow for accurate predictions even in low-data settings. We show that the developed models can be fine-tuned at low cost to solve a variety of genomics applications. Despite no supervision, the models learned to focus attention on key genomic elements and can be used to improve the prioritization of genetic variants. The training and application of foundational models in genomics provides a widely applicable approach for accurate molecular phenotype prediction from DNA sequence. Nucleotide Transformer is a series of genomics foundation models of different parameter sizes and training datasets that can be applied to various downstream tasks by fine-tuning.},
  archive      = {J_NMETH},
  author       = {Dalla-Torre, Hugo and Gonzalez, Liam and Mendoza-Revilla, Javier and Lopez Carranza, Nicolas and Grzywaczewski, Adam Henryk and Oteri, Francesco and Dallago, Christian and Trop, Evan and de Almeida, Bernardo P. and Sirelkhatim, Hassan and Richard, Guillaume and Skwark, Marcin and Beguir, Karim and Lopez, Marie and Pierrot, Thomas},
  doi          = {10.1038/s41592-024-02523-z},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {287-297},
  shortjournal = {Nat. Meth.},
  title        = {Nucleotide transformer: Building and evaluating robust foundation models for human genomics},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Efficiently accelerated bioimage analysis with NanoPyx, a
liquid engine-powered python framework. <em>NMETH</em>, <em>22</em>(2),
283–286. (<a href="https://doi.org/10.1038/s41592-024-02562-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The expanding scale and complexity of microscopy image datasets require accelerated analytical workflows. NanoPyx meets this need through an adaptive framework enhanced for high-speed analysis. At the core of NanoPyx, the Liquid Engine dynamically generates optimized central processing unit and graphics processing unit code variations, learning and predicting the fastest based on input data and hardware. This data-driven optimization achieves considerably faster processing, becoming broadly relevant to reactive microscopy and computing fields requiring efficiency. NanoPyx optimizes bioimage analysis workflows by dynamically selecting code variations based on input data and hardware. This ensures maximum computational efficiency, enabling fast analysis of large datasets and real-time reactive microscopy.},
  archive      = {J_NMETH},
  author       = {Saraiva, Bruno M. and Cunha, Inês and Brito, António D. and Follain, Gautier and Portela, Raquel and Haase, Robert and Pereira, Pedro M. and Jacquemet, Guillaume and Henriques, Ricardo},
  doi          = {10.1038/s41592-024-02562-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {283-286},
  shortjournal = {Nat. Meth.},
  title        = {Efficiently accelerated bioimage analysis with NanoPyx, a liquid engine-powered python framework},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A peptide-centric local stability assay enables
proteome-scale identification of the protein targets and binding regions
of diverse ligands. <em>NMETH</em>, <em>22</em>(2), 278–282. (<a
href="https://doi.org/10.1038/s41592-024-02553-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {By using a limited-proteolysis strategy that employs a large amount of trypsin to generate peptides directly from native proteins, we found that ligand-induced protein local stability shifts can be sensitively detected on a proteome-wide scale. This enabled us to develop the peptide-centric local stability assay, a modification-free approach that achieves unprecedented sensitivity in proteome-wide target identification and binding-region determination. We demonstrate the broad applications of the peptide-centric local stability assay by investigating interactions across various biological contexts. The peptide-centric local stability assay (PELSA) can be used to determine ligand-binding targets and binding regions on a proteome-wide scale with high sensitivity.},
  archive      = {J_NMETH},
  author       = {Li, Kejia and Chen, Shijie and Wang, Keyun and Wang, Yan and Xue, Lianji and Ye, Yuying and Fang, Zheng and Lyu, Jiawen and Zhu, Haiyang and Li, Yanan and Yu, Ting and Yang, Feng and Zhang, Xiaolei and Guo, Siqi and Ruan, Chengfei and Zhou, Jiahua and Wang, Qi and Dong, Mingming and Luo, Cheng and Ye, Mingliang},
  doi          = {10.1038/s41592-024-02553-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {278-282},
  shortjournal = {Nat. Meth.},
  title        = {A peptide-centric local stability assay enables proteome-scale identification of the protein targets and binding regions of diverse ligands},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). ProHap enables human proteomic database generation
accounting for population diversity. <em>NMETH</em>, <em>22</em>(2),
273–277. (<a href="https://doi.org/10.1038/s41592-024-02506-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Amid the advances in genomics, the availability of large reference panels of human haplotypes is key to account for human diversity within and across populations. However, mass spectrometry-based proteomics does not benefit from this information. To address this gap, we introduce ProHap, a Python-based tool that constructs protein sequence databases from phased genotypes of reference panels. ProHap enables researchers to account for haplotype diversity in proteomic searches. ProHap is a Python-based tool that constructs protein sequence databases from phased genotypes of reference panels to enable researchers to account for haplotype diversity in proteomic searches.},
  archive      = {J_NMETH},
  author       = {Vašíček, Jakub and Kuznetsova, Ksenia G. and Skiadopoulou, Dafni and Unger, Lucas and Chera, Simona and Ghila, Luiza M. and Bandeira, Nuno and Njølstad, Pål R. and Johansson, Stefan and Bruckner, Stefan and Käll, Lukas and Vaudel, Marc},
  doi          = {10.1038/s41592-024-02506-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {273-277},
  shortjournal = {Nat. Meth.},
  title        = {ProHap enables human proteomic database generation accounting for population diversity},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Orthology inference at scale with FastOMA. <em>NMETH</em>,
<em>22</em>(2), 269–272. (<a
href="https://doi.org/10.1038/s41592-024-02552-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The surge in genome data, with ongoing efforts aiming to sequence 1.5 M eukaryotes in a decade, could revolutionize genomics, revealing the origins, evolution and genetic innovations of biological processes. Yet, traditional genomics methods scale poorly with such large datasets. Here, addressing this, ‘FastOMA’ provides linear scalability for orthology inference, enabling the processing of thousands of eukaryotic genomes within a day. FastOMA maintains the high accuracy and resolution of the well-established Orthologous Matrix (OMA) approach in benchmarks. FastOMA is available via GitHub at https://github.com/DessimozLab/FastOMA/ . FastOMA achieves fast and accurate orthology inference, with linear scalability.},
  archive      = {J_NMETH},
  author       = {Majidian, Sina and Nevers, Yannis and Yazdizadeh Kharrazi, Ali and Warwick Vesztrocy, Alex and Pascarelli, Stefano and Moi, David and Glover, Natasha and Altenhoff, Adrian M. and Dessimoz, Christophe},
  doi          = {10.1038/s41592-024-02552-8},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {269-272},
  shortjournal = {Nat. Meth.},
  title        = {Orthology inference at scale with FastOMA},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Cell painting: A decade of discovery and innovation in
cellular imaging. <em>NMETH</em>, <em>22</em>(2), 254–268. (<a
href="https://doi.org/10.1038/s41592-024-02528-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Modern quantitative image analysis techniques have enabled high-throughput, high-content imaging experiments. Image-based profiling leverages the rich information in images to identify similarities or differences among biological samples, rather than measuring a few features, as in high-content screening. Here, we review a decade of advancements and applications of Cell Painting, a microscopy-based cell-labeling assay aiming to capture a cell’s state, introduced in 2013 to optimize and standardize image-based profiling. Cell Painting’s ability to capture cellular responses to various perturbations has expanded owing to improvements in the protocol, adaptations for different perturbations, and enhanced methodologies for feature extraction, quality control, and batch-effect correction. Cell Painting is a versatile tool that has been used in various applications, alone or with other -omics data, to decipher the mechanism of action of a compound, its toxicity profile, and other biological effects. Future advances will likely involve computational and experimental techniques, new publicly available datasets, and integration with other high-content data types. This Review synthesizes the literature from over 10 years of Cell Painting for image-based profiling and highlights how advances in this technology enable new biological discovery of cellular phenotypes and cell responses to perturbations.},
  archive      = {J_NMETH},
  author       = {Seal, Srijit and Trapotsi, Maria-Anna and Spjuth, Ola and Singh, Shantanu and Carreras-Puigvert, Jordi and Greene, Nigel and Bender, Andreas and Carpenter, Anne E.},
  doi          = {10.1038/s41592-024-02528-8},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {254-268},
  shortjournal = {Nat. Meth.},
  title        = {Cell painting: A decade of discovery and innovation in cellular imaging},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Nanopore approaches for single-molecule temporal omics:
Promises and challenges. <em>NMETH</em>, <em>22</em>(2), 241–253. (<a
href="https://doi.org/10.1038/s41592-024-02492-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The great molecular heterogeneity within single cells demands omics analysis from a single-molecule perspective. Moreover, considering the perpetual metabolism and communication within cells, it is essential to determine the time-series changes of the molecular library, rather than obtaining data at only one time point. Thus, there is an urgent need to develop a single-molecule strategy for this omics analysis to elucidate the biosystem heterogeneity and temporal dynamics. In this Perspective, we explore the potential application of nanopores for single-molecule temporal omics to characterize individual molecules beyond mass, in both a single-molecule and high-throughput manner. Accordingly, recent advances in nanopores available for single-molecule temporal omics are reviewed from the view of single-molecule mass identification, revealing single-molecule heterogeneity and illustrating temporal evolution. Furthermore, we discuss the primary challenges associated with using nanopores for single-molecule temporal omics in complex biological samples, and present the potential strategies and notes to respond to these challenges. This Perspective highlights the need to develop methods for single-molecule temporal omics studies and discusses nanopores as a potential solution, as well as the challenges associated with using nanopores for the analysis of complex biological samples.},
  archive      = {J_NMETH},
  author       = {Li, Meng-Yin and Jiang, Jie and Li, Jun-Ge and Niu, Hongyan and Ying, Yi-Lun and Tian, Ruijun and Long, Yi-Tao},
  doi          = {10.1038/s41592-024-02492-3},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {241-253},
  shortjournal = {Nat. Meth.},
  title        = {Nanopore approaches for single-molecule temporal omics: Promises and challenges},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Human CD8+ t cell map with single-cell transcriptome and TCR
information. <em>NMETH</em>, <em>22</em>(2), 239–240. (<a
href="https://doi.org/10.1038/s41592-024-02529-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed scAtlasVAE, a deep-learning-based method to integrate large-scale single-cell RNA sequencing data. Using this tool, we established a comprehensive, million-scale, pan-disease human CD8+ T cell reference atlas, which incorporates information on transcriptomes and the diversity of the αβ T cell receptor repertoire.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02529-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {239-240},
  shortjournal = {Nat. Meth.},
  title        = {Human CD8+ t cell map with single-cell transcriptome and TCR information},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Capture of membrane proteins in their native membrane
milieu. <em>NMETH</em>, <em>22</em>(2), 237–238. (<a
href="https://doi.org/10.1038/s41592-024-02518-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This work presents optimized experimental protocols for the extraction of &amp;gt;2,000 unique membrane proteins directly from their native membranes into native nanodiscs. Combined with orthogonal structural, biochemical or biophysical approaches, these native nanodiscs enable membrane proteins to be studied directly in their native membrane context.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02518-w},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {237-238},
  shortjournal = {Nat. Meth.},
  title        = {Capture of membrane proteins in their native membrane milieu},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Multiplexing strategies to scale up brain organoid modeling.
<em>NMETH</em>, <em>22</em>(2), 235–236. (<a
href="https://doi.org/10.1038/s41592-024-02556-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We provide scalable new tools, which include mosaic models and downstream multiplexing, for longitudinal single-cell profiling of brain organoids derived from multiple individuals. In addition, we benchmark computational methods for cell identity deconvolution and provide guidelines on how to use each multiplexing method for charting neurodevelopmental trajectories.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02556-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {235-236},
  shortjournal = {Nat. Meth.},
  title        = {Multiplexing strategies to scale up brain organoid modeling},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Isotropic 100-nm resolution live-cell imaging with 4Pi-SIM.
<em>NMETH</em>, <em>22</em>(2), 233–234. (<a
href="https://doi.org/10.1038/s41592-024-02514-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We developed 4Pi-SIM, which integrates 4Pi microscopy with structured illumination microscopy to achieve isotropic optical resolution through interference in both illumination and detection. We demonstrate its capabilities by time-lapse volumetric imaging of various subcellular structures at 100-nm resolution.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02514-0},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {233-234},
  shortjournal = {Nat. Meth.},
  title        = {Isotropic 100-nm resolution live-cell imaging with 4Pi-SIM},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Generalized AI models for genomics applications.
<em>NMETH</em>, <em>22</em>(2), 231–232. (<a
href="https://doi.org/10.1038/s41592-024-02524-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Nucleotide Transformer is a series of foundation models pre-trained on DNA sequences through self-supervised learning that extracts context-specific representations of nucleotide sequences. These representations can then be used to accurately predict molecular phenotypes.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-024-02524-y},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {231-232},
  shortjournal = {Nat. Meth.},
  title        = {Generalized AI models for genomics applications},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Taking control with RNA. <em>NMETH</em>, <em>22</em>(2),
226–230. (<a href="https://doi.org/10.1038/s41592-025-02596-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {As they study the emerging roles of RNA in disease and homeostasis, some scientists use RNA editing to direct precise RNA changes that shape cellular events.},
  archive      = {J_NMETH},
  author       = {Marx, Vivien},
  doi          = {10.1038/s41592-025-02596-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {226-230},
  shortjournal = {Nat. Meth.},
  title        = {Taking control with RNA},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Structural biology of heterogeneous RNAs. <em>NMETH</em>,
<em>22</em>(2), 225. (<a
href="https://doi.org/10.1038/s41592-025-02606-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Singh, Arunima},
  doi          = {10.1038/s41592-025-02606-5},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {225},
  shortjournal = {Nat. Meth.},
  title        = {Structural biology of heterogeneous RNAs},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). RNA-seq coverage prediction. <em>NMETH</em>,
<em>22</em>(2), 225. (<a
href="https://doi.org/10.1038/s41592-025-02607-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lin},
  doi          = {10.1038/s41592-025-02607-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {225},
  shortjournal = {Nat. Meth.},
  title        = {RNA-seq coverage prediction},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A DREADD for the periphery. <em>NMETH</em>, <em>22</em>(2),
224. (<a href="https://doi.org/10.1038/s41592-025-02604-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Vogt, Nina},
  doi          = {10.1038/s41592-025-02604-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {224},
  shortjournal = {Nat. Meth.},
  title        = {A DREADD for the periphery},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Tackling in vivo screening complexity. <em>NMETH</em>,
<em>22</em>(2), 224. (<a
href="https://doi.org/10.1038/s41592-025-02605-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Tang, Lei},
  doi          = {10.1038/s41592-025-02605-6},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {224},
  shortjournal = {Nat. Meth.},
  title        = {Tackling in vivo screening complexity},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Instructing embryonic development. <em>NMETH</em>,
<em>22</em>(2), 223. (<a
href="https://doi.org/10.1038/s41592-025-02603-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Engineered synthetic organizer cells enable spatial and biochemical control of developmental processes.},
  archive      = {J_NMETH},
  author       = {Mukhopadhyay, Madhura},
  doi          = {10.1038/s41592-025-02603-8},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {223},
  shortjournal = {Nat. Meth.},
  title        = {Instructing embryonic development},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). SMR-portal: An online platform for integrative analysis of
GWAS and xQTL data to identify complex trait genes. <em>NMETH</em>,
<em>22</em>(2), 220–222. (<a
href="https://doi.org/10.1038/s41592-024-02561-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NMETH},
  author       = {Guo, Yazhou and Xu, Tao and Luo, Jie and Jiang, Zhuqing and Chen, Wenhao and Chen, Hui and Qi, Ting and Yang, Jian},
  doi          = {10.1038/s41592-024-02561-7},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {220-222},
  shortjournal = {Nat. Meth.},
  title        = {SMR-portal: An online platform for integrative analysis of GWAS and xQTL data to identify complex trait genes},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Reporting methods for reusability. <em>NMETH</em>,
<em>22</em>(2), 217. (<a
href="https://doi.org/10.1038/s41592-025-02615-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Proper methods reporting is crucial for transparency, but ensuring method reusability by other labs takes a bit of extra effort. Here we discuss best practices for reporting methods so that they can be reused.},
  archive      = {J_NMETH},
  doi          = {10.1038/s41592-025-02615-4},
  journal      = {Nature Methods},
  month        = {2},
  number       = {2},
  pages        = {217},
  shortjournal = {Nat. Meth.},
  title        = {Reporting methods for reusability},
  volume       = {22},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
