<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NACO_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="naco---42">NACO - 42</h2>
<ul>
<li><details>
<summary>
(2024). A self-adaptive evolutionary algorithm using monte carlo
fragment insertion and conformation clustering for the protein structure
prediction problem. <em>NACO</em>, <em>23</em>(4), 687–705. (<a
href="https://doi.org/10.1007/s11047-022-09916-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The Protein Structure Prediction Problem is one of the most important and challenging open problems in Computer Science and Structural Bioinformatics. Accurately predicting protein conformations would significantly impact several fields, such as understanding proteinopathies and developing smart protein-based drugs. As such, this work has as its primary goal to improve the prediction power of ab initio methods by utilizing a self-adaptive evolutionary algorithm using Monte Carlo based fragment insertion and conformational clustering. A meta-heuristic is used as the core of the conformation sampling process with fragment insertion, feeding domain-specific information into the process. The online parameter control routines allow the method to adapt to a protein’s structure specificity and behave dynamically in different stages of the optimization process. The results obtained by the proposed method were compared to results obtained from several other algorithms found in the literature. It is possible to conclude that the proposed method is highly competitive in terms of free-energy and RMSD for the protein set used in the experiments.},
  archive      = {J_NACO},
  author       = {Parpinelli, Rafael Stubs and Will, Nilcimar Neitzel and da Silva, Renan Samuel},
  doi          = {10.1007/s11047-022-09916-z},
  journal      = {Natural Computing},
  month        = {12},
  number       = {4},
  pages        = {687-705},
  shortjournal = {Nat. Comput.},
  title        = {A self-adaptive evolutionary algorithm using monte carlo fragment insertion and conformation clustering for the protein structure prediction problem},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Physical reservoir computing: A tutorial. <em>NACO</em>,
<em>23</em>(4), 665–685. (<a
href="https://doi.org/10.1007/s11047-024-09997-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This tutorial covers physical reservoir computing from a computer science perspective. It first defines what it means for a physical system to compute, rather than merely evolve under the laws of physics. It describes the underlying computational model, the Echo State Network (ESN), and also some variants designed to make physical implementation easier. It explains why the ESN model is particularly suitable for direct physical implementation. It then discusses the issues around choosing a suitable material substrate, and interfacing the inputs and outputs. It describes how to characterise a physical reservoir in terms of benchmark tasks, and task-independent measures. It covers optimising configuration parameters, exploring the space of potential configurations, and simulating the physical reservoir. It ends with a look at the future of physical reservoir computing as devices get more powerful, and are integrated into larger systems.},
  archive      = {J_NACO},
  author       = {Stepney, Susan},
  doi          = {10.1007/s11047-024-09997-y},
  journal      = {Natural Computing},
  month        = {12},
  number       = {4},
  pages        = {665-685},
  shortjournal = {Nat. Comput.},
  title        = {Physical reservoir computing: A tutorial},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Universal shape replication via self-assembly with
signal-passing tiles. <em>NACO</em>, <em>23</em>(4), 627–664. (<a
href="https://doi.org/10.1007/s11047-024-09987-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this paper, we investigate shape-assembling power of a tile-based model of self-assembly called the Signal-Passing Tile Assembly Model (STAM). In this model, the glues that bind tiles together can be turned on and off by the binding actions of other glues via “signals”. Specifically, the problem we investigate is “shape replication” wherein, given a set of input assemblies of arbitrary shape, a system must construct an arbitrary number of assemblies with the same shapes and, with the exception of size-bounded junk assemblies that result from the process, no others. We provide the first fully universal shape replication result, namely a single tile set capable of performing shape replication on arbitrary sets of any 3-dimensional shapes without requiring any scaling or pre-encoded information in the input assemblies. Our result requires the input assemblies to be composed of signal-passing tiles whose glues can be deactivated to allow deconstruction of those assemblies, which we also prove is necessary by showing that there are shapes whose geometry cannot be replicated without deconstruction. Additionally, we modularize our construction to create systems capable of creating binary encodings of arbitrary shapes, and building arbitrary shapes from their encodings. Because the STAM is capable of universal computation, this then allows for arbitrary programs to be run within an STAM system, using the shape encodings as input, so that any computable transformation can be performed on the shapes. This is the full version, containing all construction and proof details, of a previously published extended abstract version that had most details omitted.},
  archive      = {J_NACO},
  author       = {Alseth, Andrew and Hader, Daniel and Patitz, Matthew J.},
  doi          = {10.1007/s11047-024-09987-0},
  journal      = {Natural Computing},
  month        = {12},
  number       = {4},
  pages        = {627-664},
  shortjournal = {Nat. Comput.},
  title        = {Universal shape replication via self-assembly with signal-passing tiles},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). The structural power of reconfigurable circuits in the
amoebot model. <em>NACO</em>, <em>23</em>(4), 603–625. (<a
href="https://doi.org/10.1007/s11047-024-09981-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The amoebot model (Derakhshandeh et al. in: SPAA ACM, pp 220–222. https://doi.org/10.1145/2612669.2612712 , 2014) has been proposed as a model for programmable matter consisting of tiny, robotic elements called amoebots. We consider the reconfigurable circuit extension (Feldmann et al. in J Comput Biol 29(4):317–343. https://doi.org/10.1089/cmb.2021.0363 , 2022) of the geometric amoebot model that allows the amoebot structure to interconnect amoebots by so-called circuits. A circuit permits the instantaneous transmission of signals between the connected amoebots. In this paper, we examine the structural power of the reconfigurable circuits. We start with fundamental problems like the stripe computation problem where, given any connected amoebot structure S, an amoebot u in S, and some axis X, all amoebots belonging to axis X through u have to be identified. Second, we consider the global maximum problem, which identifies an amoebot at the highest possible position with respect to some direction in some given amoebot (sub)structure. A solution to this problem can be used to solve the skeleton problem, where a cycle of amoebots has to be found in the given amoebot structure which contains all boundary amoebots. A canonical solution to that problem can be used to come up with a canonical path, which provides a unique characterization of the shape of the given amoebot structure. Constructing canonical paths for different directions allows the amoebots to set up a spanning tree and to check symmetry properties of the given amoebot structure. The problems are important for a number of applications like rapid shape transformation, energy dissemination, and structural monitoring. Interestingly, the reconfigurable circuit extension allows polylogarithmic-time solutions to all of these problems.},
  archive      = {J_NACO},
  author       = {Padalkin, Andreas and Scheideler, Christian and Warner, Daniel},
  doi          = {10.1007/s11047-024-09981-6},
  journal      = {Natural Computing},
  month        = {12},
  number       = {4},
  pages        = {603-625},
  shortjournal = {Nat. Comput.},
  title        = {The structural power of reconfigurable circuits in the amoebot model},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Distinguishing genelet circuit input pulses via a pulse
detector. <em>NACO</em>, <em>23</em>(4), 591–601. (<a
href="https://doi.org/10.1007/s11047-024-09992-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chemical systems have the potential to direct the next generation of dynamic materials if they can be integrated with a material while acting as the material’s own regulatory network. Chemical networks that use DNA and RNA strand displacement coupled with RNA synthesis and degradation, such as genelets, are promising chemical systems for this role. Genelets can produce a range of dynamic behaviors that respond to unique sets of environmental inputs. While a number of networks that generate specific types of outputs which vary in both time and amplitude have been developed, there are fewer examples of networks that recognize specific types of inputs in time and amplitude. Advanced chemical circuits in biology are capable of reading a given substrate concentration with relatively high accuracy to direct downstream function, demonstrating that such a chemical circuit is possible. Taking inspiration from this, we designed a genelet circuit which responds to a range of inputs by delivering a binary output based on the input concentration, and tested the network’s performance using an in silico model of circuit behavior. By modifying the concentrations of two circuit elements, we demonstrated that such a network topography could yield various target input concentration profiles to which a given circuit is sensitive. The number of unique elements in the final network topography as well as the individual circuit element concentrations are commensurate with properties of circuits that have been demonstrated experimentally. These factors suggest that such a network could be built and characterized in the laboratory.},
  archive      = {J_NACO},
  author       = {Yancey, Colin and Schulman, Rebecca},
  doi          = {10.1007/s11047-024-09992-3},
  journal      = {Natural Computing},
  month        = {12},
  number       = {4},
  pages        = {591-601},
  shortjournal = {Nat. Comput.},
  title        = {Distinguishing genelet circuit input pulses via a pulse detector},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Editorial for the special issue on DNA computing and
molecular programming. <em>NACO</em>, <em>23</em>(4), 589. (<a
href="https://doi.org/10.1007/s11047-024-09993-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NACO},
  author       = {Ouldridge, Thomas E. and Wickham, Shelley F. J.},
  doi          = {10.1007/s11047-024-09993-2},
  journal      = {Natural Computing},
  month        = {12},
  number       = {4},
  pages        = {589},
  shortjournal = {Nat. Comput.},
  title        = {Editorial for the special issue on DNA computing and molecular programming},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Is integration of mechanisms a way to enhance a
nature-inspired algorithm? <em>NACO</em>, <em>23</em>(3), 567–587. (<a
href="https://doi.org/10.1007/s11047-022-09920-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A lot of discussion is done these days regarding the actual novelty of newcomer nature-inspired approaches. Crucial role on that matter is played by the mechanisms included in these approaches, where many of these mechanisms have been previously introduced as part of another algorithm. On the other hand, a good practice would be to use the mechanisms of a nature-inspired algorithm to enhance the performance or to overcome the drawbacks of another one. This paper investigates this issue, where four mechanisms have been isolated and studied. Furthermore, the well-known Particle Swarm Optimization and Firefly Algorithm were used to test the effect of the studied mechanisms on the exploration and exploitation of established approaches that suffer from premature convergence or mostly explore the search space, respectively.},
  archive      = {J_NACO},
  author       = {Thymianis, Marios and Tzanetos, Alexandros},
  doi          = {10.1007/s11047-022-09920-3},
  journal      = {Natural Computing},
  month        = {9},
  number       = {3},
  pages        = {567-587},
  shortjournal = {Nat. Comput.},
  title        = {Is integration of mechanisms a way to enhance a nature-inspired algorithm?},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Population diversity and inheritance in genetic programming
for symbolic regression. <em>NACO</em>, <em>23</em>(3), 531–566. (<a
href="https://doi.org/10.1007/s11047-022-09934-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this work we aim to empirically characterize two important dynamical aspects of GP search: the evolution of diversity and the propagation of inheritance patterns. Diversity is calculated at the genotypic and phenotypic levels using efficient similarity metrics. Inheritance information is obtained via a full genealogical record of evolution as a directed acyclic graph and a set of methods for extracting relevant patterns. Advances in processing power enable our approach to handle previously infeasible graph sizes of millions of arcs and vertices. To enable a more comprehensive analysis we employ three closely-related but different evolutionary models: canonical GP, offspring selection and age-layered population structure. Our analysis reveals that a relatively small number of ancestors are responsible for producing the majority of descendants in later generations, leading to diversity loss. We show empirically across a selection of five benchmark problems that each configuration is characterized by different rates of diversity loss and different inheritance patterns, in support of the idea that each new problem may require a unique approach to solve optimally.},
  archive      = {J_NACO},
  author       = {Burlacu, Bogdan and Yang, Kaifeng and Affenzeller, Michael},
  doi          = {10.1007/s11047-022-09934-x},
  journal      = {Natural Computing},
  month        = {9},
  number       = {3},
  pages        = {531-566},
  shortjournal = {Nat. Comput.},
  title        = {Population diversity and inheritance in genetic programming for symbolic regression},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Self-replication via tile self-assembly. <em>NACO</em>,
<em>23</em>(3), 497–530. (<a
href="https://doi.org/10.1007/s11047-023-09971-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this paper we present a model containing modifications to the Signal-passing Tile Assembly Model (STAM), a tile-based self-assembly model whose tiles are capable of activating and deactivating glues based on the binding of other glues. These modifications consist of an extension to 3D, the ability of tiles to form “flexible” bonds that allow bound tiles to rotate relative to each other, and allowing tiles of multiple shapes within the same system. We call this new model the STAM*, and we present a series of constructions within it that are capable of self-replicating behavior. Namely, the input seed assemblies to our STAM* systems can encode either “genomes” specifying the instructions for building a target shape, or can be copies of the target shape with instructions built in. A universal tile set exists for any target shape (at scale factor 2), and from a genome assembly creates infinite copies of the genome as well as the target shape. An input target structure, on the other hand, can be “deconstructed” by the universal tile set to form a genome encoding it, which will then replicate and also initiate the growth of copies of assemblies of the target shape. Since the lengths of the genomes for these constructions are proportional to the number of points in the target shape, we also present a replicator which utilizes hierarchical self-assembly to greatly reduce the size of the genomes required. The main goals of this work are to examine minimal requirements of self-assembling systems capable of self-replicating behavior, with the aim of better understanding self-replication in nature as well as understanding the complexity of mimicking it.},
  archive      = {J_NACO},
  author       = {Alseth, Andrew and Hader, Daniel and Patitz, Matthew J.},
  doi          = {10.1007/s11047-023-09971-0},
  journal      = {Natural Computing},
  month        = {9},
  number       = {3},
  pages        = {497-530},
  shortjournal = {Nat. Comput.},
  title        = {Self-replication via tile self-assembly},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Reactamole: Functional reactive molecular programming.
<em>NACO</em>, <em>23</em>(3), 477–495. (<a
href="https://doi.org/10.1007/s11047-024-09982-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Chemical reaction networks (CRNs) are an important tool for molecular programming. This field is rapidly expanding our ability to deploy computer programs into biological systems for various applications. However, CRNs are also difficult to work with due to their massively parallel nature, leading to the need for higher-level languages that allow for more straightforward computation with CRNs. Recently, research has been conducted into various higher-level languages for deterministic CRNs but modeling CRN parallelism, managing error accumulation, and finding natural CRN representations are ongoing challenges. We introduce Reactamole, a higher-level language for deterministic CRNs that utilizes the functional reactive programming (FRP) paradigm to represent CRNs as a reactive dataflow network. Reactamole equates a CRN with a functional reactive program, implementing the key primitives of the FRP paradigm directly as CRNs. The functional nature of Reactamole makes reasoning about molecular programs easier, and its strong static typing allows us to ensure that a CRN is well-formed by virtue of being well-typed. In this paper, we describe the design of Reactamole and how we use CRNs to represent the common datatypes and operations found in FRP. We demonstrate the potential of this functional reactive approach to molecular programming by giving an extended example where a CRN is constructed using FRP to modulate and demodulate an amplitude-modulated signal. We also show how Reactamole can be used to specify abstract CRNs whose structure depends on the reactions and species of its input, allowing users to specify more general CRN behaviors.},
  archive      = {J_NACO},
  author       = {Klinge, Titus H. and Lathrop, James I. and Osera, Peter-Michael and Rogers, Allison},
  doi          = {10.1007/s11047-024-09982-5},
  journal      = {Natural Computing},
  month        = {9},
  number       = {3},
  pages        = {477-495},
  shortjournal = {Nat. Comput.},
  title        = {Reactamole: Functional reactive molecular programming},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Parallel pairwise operations on data stored in DNA: Sorting,
XOR, shifting, and searching. <em>NACO</em>, <em>23</em>(3), 455–476.
(<a href="https://doi.org/10.1007/s11047-023-09964-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Prior research has introduced the Single-Instruction-Multiple-Data paradigm for DNA computing (SIMD DNA). It offers the potential for storing information and performing in-memory computations on DNA, with massive parallelism. This paper introduces three new SIMD DNA operations: sorting, shifting, and searching. Each is a fundamental operation in computer science. Our implementations demonstrate the effectiveness of parallel pairwise operations with this new paradigm.},
  archive      = {J_NACO},
  author       = {Solanki, Arnav and Chen, Tonglin and Riedel, Marc},
  doi          = {10.1007/s11047-023-09964-z},
  journal      = {Natural Computing},
  month        = {9},
  number       = {3},
  pages        = {455-476},
  shortjournal = {Nat. Comput.},
  title        = {Parallel pairwise operations on data stored in DNA: Sorting, XOR, shifting, and searching},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Preface. <em>NACO</em>, <em>23</em>(3), 453. (<a
href="https://doi.org/10.1007/s11047-024-09988-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NACO},
  author       = {Lakin, Matthew R. and Sulc, Petr},
  doi          = {10.1007/s11047-024-09988-z},
  journal      = {Natural Computing},
  month        = {9},
  number       = {3},
  pages        = {453},
  shortjournal = {Nat. Comput.},
  title        = {Preface},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Models to classify the difficulty of genetic algorithms to
solve continuous optimization problems. <em>NACO</em>, <em>23</em>(2),
431–451. (<a href="https://doi.org/10.1007/s11047-022-09936-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {What constitutes a hard optimization problem to an Evolutionary Algorithm (EA)? To answer the question, the study of Fitness Landscape (FL) has emerged as one of the most successful techniques. FL measures the landscape depicted by the problem’s cost function. Fitness Landscape Analysis (FLA) uses a set of metrics to try to determine the hardness of problems; FL metrics can be divided in descriptive and dynamic. Descriptive metrics measure the intrinsic problem features, examples of these measures are ruggedness, neutrality, basins of attraction, and epistasis. Dynamic metrics measure the evolvability of EA, examples of these measures are fitness distance correlation, and negative slope coefficient. This contribution presents a procedure called Performance Classification Models (PCM) which creates learnings models to predict the performance exhibited by Genetic Algorithms (GA) in the solution of optimization problems in the continuous domain. PCM classifies the performance in two classes (easy or difficult). The dataset has as predictor variables, FL features, and as target variable the performance exhibited by the GA. The problems used in experiments are benchmark optimization functions. A product of this approach, is a procedure to Recommend Population Size (RPS): given an optimization problem, RPS recommends the minimal population size to get an efficient level of performance. This work can be easily extended to use other metrics, or a different set of problems, or the use of other EA. Developing performance models for other EA, we can solve an instance of the algorithm selection problem.},
  archive      = {J_NACO},
  author       = {Rodríguez-Maya, Noel E. and Flores, Juan J. and Verel, Sébastien and Graff, Mario},
  doi          = {10.1007/s11047-022-09936-9},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {431-451},
  shortjournal = {Nat. Comput.},
  title        = {Models to classify the difficulty of genetic algorithms to solve continuous optimization problems},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Turning machines: A simple algorithmic model for molecular
robotics. <em>NACO</em>, <em>23</em>(2), 407–430. (<a
href="https://doi.org/10.1007/s11047-022-09880-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Molecular robotics is challenging, so it seems best to keep it simple. We consider an abstract molecular robotics model based on simple folding instructions that execute asynchronously. Turning Machines are a simple 1D to 2D folding model, also easily generalisable to 2D to 3D folding. A Turning Machine starts out as a line of connected monomers in the discrete plane, each with an associated turning number. A monomer turns relative to its neighbours, executing a unit-distance translation that drags other monomers along with it, and through collective motion the initial set of monomers eventually folds into a programmed shape. We provide a suite of tools for reasoning about Turning Machines by fully characterising their ability to execute line rotations: executing an almost-full line rotation of $$5\pi /3$$ radians is possible, yet a full $$2\pi$$ rotation is impossible. Furthermore, line rotations up to $$5\pi /3$$ are executed efficiently, in $$O(\log n)$$ expected time in our continuous time Markov chain time model. We then show that such line-rotations represent a fundamental primitive in the model, by using them to efficiently and asynchronously fold shapes. In particular, arbitrarily large zig-zag-rastered squares and zig-zag paths are foldable, as are y-monotone shapes albeit with error (bounded by perimeter length). Finally, we give shapes that despite having paths that traverse all their points, are in fact impossible to fold, as well as techniques for folding certain classes of (scaled) shapes without error. Our approach relies on careful geometric-based analyses of the feats possible and impossible by a very simple robotic system, and pushes conceptional hardness towards mathematical analysis and away from molecular implementation.},
  archive      = {J_NACO},
  author       = {Kostitsyna, Irina and Wood, Cai and Woods, Damien},
  doi          = {10.1007/s11047-022-09880-8},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {407-430},
  shortjournal = {Nat. Comput.},
  title        = {Turning machines: A simple algorithmic model for molecular robotics},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Verification and computation in restricted tile automata.
<em>NACO</em>, <em>23</em>(2), 387–405. (<a
href="https://doi.org/10.1007/s11047-021-09875-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many models of self-assembly have been shown to be capable of performing computation. Tile Automata was recently introduced combining features of both Cellular Automata and the 2-Handed Model of self-assembly both capable of universal computation. In this work we study the complexity of Tile Automata utilizing features inherited from the two models mentioned above. We first present a construction for simulating Turing machines that performs both covert and fuel efficient computation. We then explore the capabilities of limited Tile Automata systems such as 1-dimensional systems (all assemblies are of height 1) and freezing systems (tiles may not repeat states). Using these results we provide a connection between the problem of finding the largest uniquely producible assembly using n states and the busy beaver problem for non-freezing systems and provide a freezing system capable of uniquely assembling an assembly whose length is exponential in the number of states of the system. We finish by exploring the complexity of the Unique Assembly Verification problem in Tile Automata with different limitations such as freezing and systems without the power of detachment.},
  archive      = {J_NACO},
  author       = {Caballero, David and Gomez, Timothy and Schweller, Robert and Wylie, Tim},
  doi          = {10.1007/s11047-021-09875-x},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {387-405},
  shortjournal = {Nat. Comput.},
  title        = {Verification and computation in restricted tile automata},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). ALCH: An imperative language for chemical reaction
network-controlled tile assembly. <em>NACO</em>, <em>23</em>(2),
365–385. (<a href="https://doi.org/10.1007/s11047-021-09878-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Schiefer and Winfree recently introduced the chemical reaction network-controlled tile assembly model (CRN-TAM), a variant of the abstract tile assembly model (aTAM). In the CRN-TAM, tile reactions are mediated via non-local chemical signals controlled by a chemical reaction network. This paper introduces ALCH, an imperative programming language for specifying CRN-TAM programs that can be compiled and simulated. ALCH includes standard language features such as Boolean variables, conditionals, loops, and CRN-TAM-specific constructs such as adding and removing tiles. ALCH also includes the branch and parallel structures which harness the nondeterministic and parallel nature of the CRN-TAM. ALCH also supports functional tileset specification. Using ALCH, we show that the discrete Sierpinski triangle and the discrete Sierpinski carpet can be strictly self-assembled in the CRN-TAM, which shows the CRN-TAM can self-assemble infinite shapes at scale 1 that the aTAM cannot. ALCH allows us to present these constructions at a high level, abstracting species and reactions into C-like code that is simpler to understand. We employ two new CRN-TAM techniques in our constructions. First, we use ALCH’s nondeterministic branching feature to probe previously placed tiles of the assembly and detect the presence and absence of tiles. Second, we use scaffolding tiles to precisely control tile placement by occluding any undesired binding sites. This paper is an extension of our previous work, updated to include a Sierpinski carpet construction and the parallel command.},
  archive      = {J_NACO},
  author       = {Klinge, Titus H. and Lathrop, James I. and Moreno, Sonia and Potter, Hugh D. and Raman, Narun K. and Riley, Matthew R.},
  doi          = {10.1007/s11047-021-09878-8},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {365-385},
  shortjournal = {Nat. Comput.},
  title        = {ALCH: An imperative language for chemical reaction network-controlled tile assembly},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Population-induced phase transitions and the verification of
chemical reaction networks. <em>NACO</em>, <em>23</em>(2), 347–363. (<a
href="https://doi.org/10.1007/s11047-021-09877-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We show that very simple molecular systems, modeled as chemical reaction networks, can have behaviors that exhibit dramatic phase transitions at certain population thresholds. Moreover, the magnitudes of these thresholds can thwart attempts to use simulation, model checking, or approximation by differential equations to formally verify the behaviors of such systems at realistic populations. We show how formal theorem provers can successfully verify some such systems at populations where other verification methods fail.},
  archive      = {J_NACO},
  author       = {Lathrop, James I. and Lutz, Jack H. and Lutz, Robyn R. and Potter, Hugh D. and Riley, Matthew R.},
  doi          = {10.1007/s11047-021-09877-9},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {347-363},
  shortjournal = {Nat. Comput.},
  title        = {Population-induced phase transitions and the verification of chemical reaction networks},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Preface. <em>NACO</em>, <em>23</em>(2), 345–346. (<a
href="https://doi.org/10.1007/s11047-024-09985-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NACO},
  author       = {Patitz, Matt and Geary, Cody},
  doi          = {10.1007/s11047-024-09985-2},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {345-346},
  shortjournal = {Nat. Comput.},
  title        = {Preface},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Reaction mining for reaction systems. <em>NACO</em>,
<em>23</em>(2), 323–343. (<a
href="https://doi.org/10.1007/s11047-024-09989-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reaction systems are a formal model for computational processing in which reactions operate on sets of entities (molecules) providing a framework for dealing with qualitative aspects of biochemical systems. This paper is concerned with reaction systems in which entities can have discrete concentrations, and so reactions operate on multisets rather than sets of entities. The resulting framework allows one to deal with quantitative aspects of reaction systems, and a bespoke linear-time temporal logic allows one to express and verify a wide range of key behavioural system properties. In practical applications, a reaction system with discrete concentrations may only be partially specified, and the possibility of an effective automated calculation of the missing details provides an attractive design approach. With this idea in mind, the current paper discusses parametric reaction systems with parameters representing unknown parts of hypothetical reactions. The main result is a method aimed at replacing the parameters in such a way that the resulting reaction system operating in a specified external environment satisfies a given temporal logic formula.This paper provides an encoding of parametric reaction systems in smt, and outlines a synthesis procedure based on bounded model checking for solving the synthesis problem. It also reports on the initial experimental results demonstrating the feasibility of the novel synthesis method.},
  archive      = {J_NACO},
  author       = {Męski, Artur and Koutny, Maciej and Mikulski, Łukasz and Penczek, Wojciech},
  doi          = {10.1007/s11047-024-09989-y},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {323-343},
  shortjournal = {Nat. Comput.},
  title        = {Reaction mining for reaction systems},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Modeling NP-problems with families of extended graph-based
reaction systems. <em>NACO</em>, <em>23</em>(2), 309–322. (<a
href="https://doi.org/10.1007/s11047-024-09984-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this paper, we continue the investigation of graph-based reaction systems. We extend the notion by input and output states as well as admitted context sequences to model explicitly input–output relations and decision problems on the inputs. Moreover, we combine extended graph-based reaction systems into families to cover infinite input–output relations and decision problems on infinite sets of graphs. This is used to model NP-problems on graphs and reductions between them as well as to prove their correctness.},
  archive      = {J_NACO},
  author       = {Kreowski, Hans-Jörg and Lye, Aaron},
  doi          = {10.1007/s11047-024-09984-3},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {309-322},
  shortjournal = {Nat. Comput.},
  title        = {Modeling NP-problems with families of extended graph-based reaction systems},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). On categorical approach to reaction systems. <em>NACO</em>,
<em>23</em>(2), 295–307. (<a
href="https://doi.org/10.1007/s11047-024-09978-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In every matured theory, there is a need to investigate possible relationships between considered objects. To address this issue, it is natural to relate a category with given model of computing. Thanks to such approach, many properties are unified and simplified. In this paper, we investigate how category theory can be used to give a faithful semantics for reaction systems. In particular, we propose and discuss possible approaches to the problem of defining morphisms between reaction systems. We provide the definition of morphism that keeps the behaviour of the original reaction system. Especially, some equivalences of reaction systems are reflected in terms of morphisms. For this purpose we expressed isomorphisms and sections in term of transition systems. Moreover, the accelerating morphism defined in the last section gives a new approach for including time in reaction systems.},
  archive      = {J_NACO},
  author       = {Kaniecki, Mariusz and Mikulski, Łukasz},
  doi          = {10.1007/s11047-024-09978-1},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {295-307},
  shortjournal = {Nat. Comput.},
  title        = {On categorical approach to reaction systems},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Ranks of functions specified by minimal reaction systems and
induced by images of singletons. <em>NACO</em>, <em>23</em>(2), 285–293.
(<a href="https://doi.org/10.1007/s11047-024-09973-6">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This paper studies mathematical properties of reaction systems, which is a formal model introduced by Ehrenfeucht and Rozenberg and inspired by biochemical reactions that occur in living cells. Numerous studies have focused on reaction system ranks of functions specified by minimal reaction systems, where the rank refers to the smallest size among the specifying reaction systems. We particularly study the reaction system ranks for a class of union-additive functions specified by minimal reaction systems introduced by Salomaa, which is closed under taking composition. More precisely, when the signature size is two, we obtain a general formula for the reaction system ranks that shows the reaction system rank of each function from this subclass depends on the characteristic of the function. Then, we study the reaction system ranks of such functions with signature size three, as well as establishing a general upper bound for reaction system ranks of such functions with one-to-one signatures for any background set.},
  archive      = {J_NACO},
  author       = {Intekhab, Husain and Teh, Wen Chean},
  doi          = {10.1007/s11047-024-09973-6},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {285-293},
  shortjournal = {Nat. Comput.},
  title        = {Ranks of functions specified by minimal reaction systems and induced by images of singletons},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Variants of distributed reaction systems. <em>NACO</em>,
<em>23</em>(2), 269–284. (<a
href="https://doi.org/10.1007/s11047-024-09974-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A distributed reaction system consists of a finite set of reaction systems that either interact with a common environment or interact with each other by communicating products or reactions. A reaction system is a well-known qualitative formal model of interactions between biochemical reactions. A reaction is a triplet of nonempty sets representing chemicals, called the set of reactants, the set of inhibitors, and the set of products. A reaction corresponds to a chemical reaction performed on a set of chemicals, and a reaction system is a finite nonempty set of reactions. In this paper, we examine two variants of distributed reaction systems. We introduce the notion of a distributed reaction system with communication by request (a qDRS for short), where sets of products are communicated between the component reaction systems by queries. First, we show that every qDRS can be represented by a reaction system. After that we compare distributed reaction systems with communication by request to extended distributed reaction systems (EDRSs), models that were introduced in a previous paper. We prove that extended distributed reaction systems, where a context automaton provides input for the component reaction systems, simulate distributed reaction systems with communication by request and distributed reaction systems with communication by request simulate special variants of extended distributed reaction systems. Furthermore, we assign languages to these two variants of distributed reaction systems. We prove that the class of agreement languages of extended distributed reaction systems is equal to the class of languages of nondeterministic multihead finite automata and the agreement language of every distributed reaction system with communication by request is an element of a certain subregular language class.},
  archive      = {J_NACO},
  author       = {Csuhaj-Varjú, Erzsébet and Vaszil, György},
  doi          = {10.1007/s11047-024-09974-5},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {269-284},
  shortjournal = {Nat. Comput.},
  title        = {Variants of distributed reaction systems},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Melding boolean networks and reaction systems under
synchronous, asynchronous and most permissive semantics. <em>NACO</em>,
<em>23</em>(2), 235–267. (<a
href="https://doi.org/10.1007/s11047-024-09990-5">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This paper forges a strong connection between two well known computational frameworks for representing biological systems, in order to facilitate the seamless transfer of techniques between them. Boolean networks are a well established formalism employed from biologists. They have been studied under different (synchronous and asynchronous) update semantics, enabling the observation and characterisation of distinct facets of system behaviour. Recently, a new semantics for Boolean networks has been proposed, called most permissive semantics, that enables a more faithful representation of biological phenomena. Reaction systems offer a streamlined formalism inspired by biochemical reactions in living cells. Reaction systems support a full range of analysis techniques that can help for gaining deeper insights into the underlying biological phenomena. Our goal is to leverage the available toolkit for predicting and comprehending the behaviour of reaction systems within the realm of Boolean networks. In this paper, we first extend the behaviour of reaction systems to several asynchronous semantics, including the most permissive one, and then we demonstrate that Boolean networks and reaction systems exhibit isomorphic behaviours under the synchronous, general/fully asynchronous and most permissive semantics.},
  archive      = {J_NACO},
  author       = {Bruni, Roberto and Gori, Roberta and Milazzo, Paolo and Siboulet, Hélène},
  doi          = {10.1007/s11047-024-09990-5},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {235-267},
  shortjournal = {Nat. Comput.},
  title        = {Melding boolean networks and reaction systems under synchronous, asynchronous and most permissive semantics},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A framework for monitored dynamic slicing of reaction
systems. <em>NACO</em>, <em>23</em>(2), 217–234. (<a
href="https://doi.org/10.1007/s11047-024-09976-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reaction systems (RSs) are a computational framework inspired by biochemical mechanisms. A RS defines a finite set of reactions over a finite set of entities. Typically each reaction has a local scope, because it is concerned with a small set of entities, but complex models can involve a large number of reactions and entities, and their computation can manifest unforeseen emerging behaviours. When a deviation is detected, like the unexpected production of some entities, it is often difficult to establish its causes, e.g., which entities were directly responsible or if some reaction was misconceived. Slicing is a well-known technique for debugging, which can point out the program lines containing the faulty code. In this paper, we define the first dynamic slicer for RSs and show that it can help to detect the causes of erroneous behaviour and highlight the involved reactions for a closer inspection. To fully automate the debugging process, we propose to distil monitors for starting the slicing whenever a violation from a safety specification is detected. We have integrated our slicer in BioResolve, written in Prolog which provides many useful features for the formal analysis of RSs. We define the slicing algorithm for basic RSs and then enhance it for dealing with quantitative extensions of RSs, where timed processes and linear processes can be represented. Our framework is shown at work on suitable biologically inspired RS models.},
  archive      = {J_NACO},
  author       = {Brodo, Linda and Bruni, Roberto and Falaschi, Moreno},
  doi          = {10.1007/s11047-024-09976-3},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {217-234},
  shortjournal = {Nat. Comput.},
  title        = {A framework for monitored dynamic slicing of reaction systems},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Fixed points and attractors of additive reaction systems.
<em>NACO</em>, <em>23</em>(2), 205–215. (<a
href="https://doi.org/10.1007/s11047-024-09977-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reaction systems are discrete dynamical systems that simulate biological processes within living cells through finite sets of reactants, inhibitors, and products. In this paper, we study the computational complexity of deciding on the existence of fixed points and attractors in the restricted class of additive reaction systems, in which each reaction involves at most one reactant and no inhibitors. We prove that all the considered problems, that are known to be hard for other classes of reaction systems, are polynomially solvable in additive systems. To arrive at these results, we provide several non-trivial reductions to problems on a polynomially computable graph representation of reaction systems that might prove useful for addressing other related problems in the future.},
  archive      = {J_NACO},
  author       = {Ascone, Rocco and Bernardini, Giulia and Manzoni, Luca},
  doi          = {10.1007/s11047-024-09977-2},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {205-215},
  shortjournal = {Nat. Comput.},
  title        = {Fixed points and attractors of additive reaction systems},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Pure reaction automata. <em>NACO</em>, <em>23</em>(2),
189–204. (<a href="https://doi.org/10.1007/s11047-024-09980-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {This work introduces the new class of pure reaction automata, as well as a new update manner, called maximal reactive manner, that can also be applied to standard reaction automata. Pure reaction automata differ from the standard model in that they don’t have permanence: the entities that are not consumed by the reactions happening at a certain state are not conserved in the result states. We prove that the set of languages accepted by the new class under the maximal reactive manner contains the set of languages accepted by standard reaction automata under the same manner or under the maximal parallel manner. We also prove that a strict subclass of pure reaction automata can compute any partial recursive function.},
  archive      = {J_NACO},
  author       = {Ascone, Rocco and Bernardini, Giulia and Formenti, Enrico and Leiter, Francesco and Manzoni, Luca},
  doi          = {10.1007/s11047-024-09980-7},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {189-204},
  shortjournal = {Nat. Comput.},
  title        = {Pure reaction automata},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Solving subset sum and SAT problems by reaction systems.
<em>NACO</em>, <em>23</em>(2), 177–187. (<a
href="https://doi.org/10.1007/s11047-024-09972-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We study the efficiency of the reaction systems in solving NP-complete problems. Due to the fact that standard reaction systems are qualitative, in order to accomplish our aim, in this paper we consider communicating reaction systems with direct communication extended with duration for resources and a mutual exclusion relation between reactions forbidding two reactions to be used in the same step, in parallel. We show that these systems, working in a non-deterministic way, are powerful enough to provide polynomial-time solutions to the subset sum and SAT problems. We consider a semi-uniform approach by constructing a system for each instance of the subset sum and SAT problems and embedding the parameters into the constructed systems.},
  archive      = {J_NACO},
  author       = {Aman, Bogdan and Ciobanu, Gabriel},
  doi          = {10.1007/s11047-024-09972-7},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {177-187},
  shortjournal = {Nat. Comput.},
  title        = {Solving subset sum and SAT problems by reaction systems},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). On the spectrum between reaction systems and string
rewriting. <em>NACO</em>, <em>23</em>(2), 159–175. (<a
href="https://doi.org/10.1007/s11047-024-09986-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reaction systems are a model of computing aiming to formalize biochemistry by capturing the qualitative relations between the species, and explicitly discarding any accounts of multiplicity. From the point of view of the formal language theory, this situates them in the realm of set rewriting. In this work, we propose a series of extensions of reaction systems to use strings. These extensions form a spectrum in the sense that all of them honor the hallmark features of the original model: the threshold principle and the non-permanency principle. We thoroughly discuss the details of the structure and the behavior of these variants, and commence studying their expressive power by comparing them to some classic models of computing.},
  archive      = {J_NACO},
  author       = {Alhazov, Artiom and Freund, Rudolf and Ivanov, Sergiu},
  doi          = {10.1007/s11047-024-09986-1},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {159-175},
  shortjournal = {Nat. Comput.},
  title        = {On the spectrum between reaction systems and string rewriting},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Preface. <em>NACO</em>, <em>23</em>(2), 157–158. (<a
href="https://doi.org/10.1007/s11047-024-09991-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NACO},
  author       = {Genova, Daniela and Petre, Ion},
  doi          = {10.1007/s11047-024-09991-4},
  journal      = {Natural Computing},
  month        = {6},
  number       = {2},
  pages        = {157-158},
  shortjournal = {Nat. Comput.},
  title        = {Preface},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Variants of string assembling systems. <em>NACO</em>,
<em>23</em>(1), 131–156. (<a
href="https://doi.org/10.1007/s11047-022-09918-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {String assembling systems are biologically inspired mechanisms that generate strings from copies out of finite sets of assembly units. The underlying mechanism is based on piecewise assembly of a double-stranded sequence of symbols, where the upper and lower strand have to match. The generation is additionally controlled by the requirement that the first symbol of a unit has to be the same as the last symbol of the strand generated so far, as well as by the distinction of assembly units that may appear at the beginning, during, or at the end of the assembling process and by a length restriction on the units. We investigate the power of these model-inherent control mechanisms by considering variants where one or more of these mechanisms are relaxed. The generative capacities and the relative power of the variants are our main interest. In particular, we prove that the power gained in the different control mechanisms may yield strictly more powerful systems and incomparable capacities. Additionally, we generalize these systems to multi-stranded systems. We obtain a strong connection to one-way multi-head finite automata and show an infinite, dense, and strict strand hierarchy. Finally, we examine the closure properties of the different variants of string assembling systems.},
  archive      = {J_NACO},
  author       = {Kutrib, Martin and Wendlandt, Matthias},
  doi          = {10.1007/s11047-022-09918-x},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {131-156},
  shortjournal = {Nat. Comput.},
  title        = {Variants of string assembling systems},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Large population sizes and crossover help in dynamic
environments. <em>NACO</em>, <em>23</em>(1), 115–129. (<a
href="https://doi.org/10.1007/s11047-022-09915-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Dynamic linear functions on the boolean hypercube are functions which assign to each bit a positive weight, but the weights change over time. Throughout optimization, these functions maintain the same global optimum, and never have defecting local optima. Nevertheless, it was recently shown [Lengler, Schaller, FOCI 2019] that the $$(1+1)$$ -Evolutionary Algorithm needs exponential time to find or approximate the optimum for some algorithm configurations. In this experimental paper, we study the effect of larger population sizes for dynamic binval, the extreme form of dynamic linear functions. We find that moderately increased population sizes extend the range of efficient algorithm configurations, and that crossover boosts this positive effect substantially. Remarkably, similar to the static setting of monotone functions in [Lengler, Zou, FOGA 2019], the hardest region of optimization for $$(\mu +1)$$ -EA is not close the optimum, but far away from it. In contrast, for the $$(\mu +1)$$ -GA, the region around the optimum is the hardest region in all studied cases.Kindly check and confirm the inserted city name is correctly identified.Correct.},
  archive      = {J_NACO},
  author       = {Lengler, Johannes and Meier, Jonas},
  doi          = {10.1007/s11047-022-09915-0},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {115-129},
  shortjournal = {Nat. Comput.},
  title        = {Large population sizes and crossover help in dynamic environments},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A roadmap for solving optimization problems with estimation
of distribution algorithms. <em>NACO</em>, <em>23</em>(1), 99–113. (<a
href="https://doi.org/10.1007/s11047-022-09913-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In recent decades, Estimation of Distribution Algorithms (EDAs) have gained much popularity in the evolutionary computation community for solving optimization problems. Characterized by the use of probabilistic models to represent the solutions and the interactions between the variables of the problem, EDAs can be applied to either discrete, continuous or mixed domain problems. Due to this robustness, these algorithms have been used to solve a diverse set of real-world and academic optimization problems. However, a straightforward application is only limited to a few cases, and for the general case, an efficient application requires intuition from the problem as well as notable understanding in probabilistic modeling. In this paper, we provide a roadmap for solving optimization problems via EDAs. It is not the aim of the paper to provide a thorough review of EDAs, but to present a guide for those practitioners interested in using the potential of EDAs when solving optimization problems. In order to present a roadmap which is as useful as possible, we address the key aspects involved in the design and application of EDAs, in a sequence of stages: (1) the choice of the codification, (2) the choice of the probability model, (3) strategies to incorporate knowledge about the problem to the model, and (4) balancing the diversification-intensification behavior of the EDA. At each stage, first, the contents are presented together with common practices and advice to follow. Then, an illustration is given with an example which shows different alternatives. In addition to the roadmap, the paper presents current open challenges when developing EDAs, and revises paths for future research advances in the context of EDAs.},
  archive      = {J_NACO},
  author       = {Ceberio, Josu and Mendiburu, Alexander and Lozano, Jose A.},
  doi          = {10.1007/s11047-022-09913-2},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {99-113},
  shortjournal = {Nat. Comput.},
  title        = {A roadmap for solving optimization problems with estimation of distribution algorithms},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Optimal safe driving dynamics for autonomous interacting
vehicles. <em>NACO</em>, <em>23</em>(1), 85–97. (<a
href="https://doi.org/10.1007/s11047-023-09960-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We consider the car following problem for a set of autonomous vehicles following each other on either an infinite or circular road. The behavior of each car is specified by its &quot;speed regulator&quot;, a device that decides to increase or decrease the speed of the car as a function of the head-tail distance to its predecessor and the speed of both cars. A collective behavior emerges that corresponds to previously proposed cellular automata traffic models. We further analyze the traffic patterns of the system in the long term, as governed by the speed regulator and we study under which conditions traffic patterns of maximum flow can or cannot be reach. We show the existence of suboptimal flow conditions that require external coordination mechanisms (that we do not consider in this paper) in order to reach the optimal flow achievable with the given density. In contrast with other approaches, we do not try to reproduce observed or measured traffic patterns. We analyze a deterministic speed regulator in order to decipher the emergent dynamics, and to ponder what maneuvers can be safely performed. Here, we restrict our attention to the car following problem. By comparing our speed regulator with classical models, auch as the Nagel–Schreckenberg and KKW models, we observe that although our regulator is formulated in simple terms, its dynamics share similarities with these models. In particular, the KKW model is designed to reproduce the observed behavior that a trailing car in the synchronization range of the leading car tends to regulate its speed to maintain a constant distance. this same behavior is adopted by our speed regulator, showing that this is a safe way of driving.},
  archive      = {J_NACO},
  author       = {Cohen, Nathan and Chopard, Bastien and Leone, Pierre},
  doi          = {10.1007/s11047-023-09960-3},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {85-97},
  shortjournal = {Nat. Comput.},
  title        = {Optimal safe driving dynamics for autonomous interacting vehicles},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Distance-based affective states in cellular automata
pedestrian simulation. <em>NACO</em>, <em>23</em>(1), 71–83. (<a
href="https://doi.org/10.1007/s11047-023-09957-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cellular Automata have successfully been successfully applied to the modeling and simulation of pedestrian and crowd dynamics. In particular, the investigated scenarios have often been focused on the evaluation of medium–high population density situations, in which the motivation of pedestrians to reach a certain location overcomes their tendency to naturally respect proxemic distances. The global COVID-19 outbreak, though, has shown that sometimes it is crucial to contemplate how proxemic tendencies are emphasized and amplified by the affective state of the individuals involved in the scenario, representing an important factor to take into consideration when investigating the behaviour of a crowd. In this paper we present a research effort aimed at integrating results of quantitative analyses regarding the effects of affective states on the perception of distances maintained by different types of pedestrians with the modeling of pedestrian movement choices in a cellular automata framework.},
  archive      = {J_NACO},
  author       = {Bandini, Stefania and Briola, Daniela and Dennunzio, Alberto and Gasparini, Francesca and Giltri, Marta and Vizzari, Giuseppe},
  doi          = {10.1007/s11047-023-09957-y},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {71-83},
  shortjournal = {Nat. Comput.},
  title        = {Distance-based affective states in cellular automata pedestrian simulation},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024b). Correction: Cellular automata simulations of the sintering
behavior of ceramics driven by surface energy reduction. <em>NACO</em>,
<em>23</em>(1), 69–70. (<a
href="https://doi.org/10.1007/s11047-023-09970-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NACO},
  author       = {Laura, Garrido-Regife and Francisco, Jiménez-Morales and Manuela, González-Sánchez and Pedro, Rivero-Antúnez and Víctor, Morales-Flórez},
  doi          = {10.1007/s11047-023-09970-1},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {69-70},
  shortjournal = {Nat. Comput.},
  title        = {Correction: Cellular automata simulations of the sintering behavior of ceramics driven by surface energy reduction},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024a). Cellular automata simulations of the sintering behavior of
ceramics driven by surface energy reduction. <em>NACO</em>,
<em>23</em>(1), 61–68. (<a
href="https://doi.org/10.1007/s11047-023-09958-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A cellular automata model has been developed to study the sintering behavior of ceramic particles. In this model, the only physical rule that drives the evolution of the system is to reduce the energy at the interface between the mass cells and the void cells. The meaning of several computational parameters, such as particle size or computational temperature, were investigated. Experiments of partial sintering of spherical particles of silica were performed and it was verified that this model successfully mimics the neck formation. Moreover, other experimental evidences of the stages of the densification, such as the formation of the intermediate vermicular microstructure or the dependence of porosity with temperature, were also qualitatively simulated.},
  archive      = {J_NACO},
  author       = {Laura, Garrido-Regife and Francisco, Jiménez-Morales and Manuela, González-Sánchez and Pedro, Rivero-Antúnez and Víctor, Morales-Flórez},
  doi          = {10.1007/s11047-023-09958-x},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {61-68},
  shortjournal = {Nat. Comput.},
  title        = {Cellular automata simulations of the sintering behavior of ceramics driven by surface energy reduction},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Handling sudoku puzzles with irregular learning cellular
automata. <em>NACO</em>, <em>23</em>(1), 41–60. (<a
href="https://doi.org/10.1007/s11047-024-09975-4">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The use of Cellular Automata (CA) in combination with Learning Automata (LA) has demonstrated effectiveness in handling hard-to-be-solved problems. Due to their capacity to learn and adapt, as well as their inherent parallelism, they can expedite the problem-solving process for a range of problems, such as challenging logic puzzles. One such puzzle is Sudoku, which poses a combinatorial optimization challenge of great difficulty and complexity. In this study, a Sudoku puzzle was represented as an Irregular Learning Cellular Automaton (ILCA), using a reward and penalty algorithm to resolve it. Simulations for an amount of 400 puzzles were performed, while the results demonstrate that the proposed algorithm operates effectively, highlighting the concurrent and learning capabilities of the ILCA structure. Furthermore, two different performance enhancement methods are investigated, namely learning rates method and selective probability reset rule, which are able to increase the initial performance by $$26.8\%$$ and to achieve an overall $$99.3\%$$ resolution rate.},
  archive      = {J_NACO},
  author       = {Chatzinikolaou, Theodoros Panagiotis and Karamani, Rafailia-Eleni and Fyrigos, Iosif-Angelos and Sirakoulis, Georgios Ch.},
  doi          = {10.1007/s11047-024-09975-4},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {41-60},
  shortjournal = {Nat. Comput.},
  title        = {Handling sudoku puzzles with irregular learning cellular automata},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Synchronization of elementary cellular automata.
<em>NACO</em>, <em>23</em>(1), 31–40. (<a
href="https://doi.org/10.1007/s11047-023-09962-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In this paper, we study how synchronization and state estimation are related in the context of elementary cellular automata. We first characterize the geometric properties of the synchronization error between two replicas of a 1D elementary cellular automata following Wolfram’s rule 18. We propose a simple approach to statistically model the transient phase of the spreading of the synchronization error. We finally present a way to utilize our model of the error spreading to place mobile sensors in order to improve the overall replica synchronization in the case in which the initial error is small.},
  archive      = {J_NACO},
  author       = {Plénet, Théo and Bagnoli, Franco and El Yacoubi, Samira and Raïevsky, Clément and Lefèvre, Laurent},
  doi          = {10.1007/s11047-023-09962-1},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {31-40},
  shortjournal = {Nat. Comput.},
  title        = {Synchronization of elementary cellular automata},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). System reduction: An approach based on probabilistic
cellular automata. <em>NACO</em>, <em>23</em>(1), 17–29. (<a
href="https://doi.org/10.1007/s11047-023-09959-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The goal of this paper it to explore the possibility to simplify a stochastic complex dynamical system by reducing the number of its degrees of freedom, through a coarse graining procedure. Our objective is to create reduced systems which requires less computational burden to control. In other words, the question is whether one can act on the simplified system to control its evolution towards a given target, so that the same strategy also applies to the full system. We analyze this problem in three stages. First we consider the general case of coarse-graining 1D cellular automata. We show that the procedure is expected to produce extra stochasticity in the system, while still retaining some characteristics of the original system. In a second step, we study a 1D voter model for which the coarse-graining procedure requires component weighting, with weights based on causal paths. Finally, in a third part, we consider the same voter model on a scale-free graph, and perform a reduction according to a partition of the graph in communities. The equivalence between the reduced and full systems, whether controlled or not, is investigated through a delayed mutual information analysis, a concept from information theory.},
  archive      = {J_NACO},
  author       = {Toupance, Pierre-Alain and Chopard, Bastien and Lefèvre, Laurent},
  doi          = {10.1007/s11047-023-09959-w},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {17-29},
  shortjournal = {Nat. Comput.},
  title        = {System reduction: An approach based on probabilistic cellular automata},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). A classification of s-boxes generated by orthogonal cellular
automata. <em>NACO</em>, <em>23</em>(1), 5–16. (<a
href="https://doi.org/10.1007/s11047-023-09956-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most of the approaches published in the literature to construct S-boxes via Cellular Automata (CA) work by either iterating a finite CA for several time steps, or by a one-shot application of the global rule. The main characteristic that brings together these works is that they employ a single CA rule to define the vectorial Boolean function of the S-box. In this work, we explore a different direction for the design of S-boxes that leverages on Orthogonal CA (OCA), i.e. pairs of CA rules giving rise to orthogonal Latin squares. The motivation stands on the facts that an OCA pair already defines a bijective transformation, and moreover the orthogonality property of the resulting Latin squares ensures a minimum amount of diffusion. We exhaustively enumerate all S-boxes generated by OCA pairs of diameter $$4 \le d \le 6$$ , and measure their nonlinearity. Interestingly, we observe that for $$d=4$$ and $$d=5$$ all S-boxes are linear, despite the underlying CA local rules being nonlinear. The smallest nonlinear S-boxes emerges for $$d=6$$ , but their nonlinearity is still too low to be used in practice. Nonetheless, we unearth an interesting structure of linear OCA S-boxes, proving that their Linear Components Space is itself the image of a linear CA, or equivalently a polynomial code. We finally classify all linear OCA S-boxes in terms of their generator polynomials.},
  archive      = {J_NACO},
  author       = {Mariot, Luca and Manzoni, Luca},
  doi          = {10.1007/s11047-023-09956-z},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {5-16},
  shortjournal = {Nat. Comput.},
  title        = {A classification of S-boxes generated by orthogonal cellular automata},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
<li><details>
<summary>
(2024). Preface. <em>NACO</em>, <em>23</em>(1), 1–3. (<a
href="https://doi.org/10.1007/s11047-023-09966-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NACO},
  author       = {Bandini, Stefania and Chopard, Bastien and Mauri, Giancarlo},
  doi          = {10.1007/s11047-023-09966-x},
  journal      = {Natural Computing},
  month        = {3},
  number       = {1},
  pages        = {1-3},
  shortjournal = {Nat. Comput.},
  title        = {Preface},
  volume       = {23},
  year         = {2024},
}
</textarea>
</details></li>
</ul>

</body>
</html>
