<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>NATCS_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="natcs---15">NATCS - 15</h2>
<ul>
<li><details>
<summary>
(2025). Author correction: Spatial modeling algorithms for reactions
and transport in biological cells. <em>NATCS</em>, <em>5</em>(2), 185.
(<a href="https://doi.org/10.1038/s43588-025-00773-1">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Francis, Emmet A. and Laughlin, Justin G. and Dokken, Jørgen S. and Finsberg, Henrik N. T. and Lee, Christopher T. and Rognes, Marie E. and Rangamani, Padmini},
  doi          = {10.1038/s43588-025-00773-1},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {185},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Spatial modeling algorithms for reactions and transport in biological cells},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025b). Author correction: Approaching coupled-cluster accuracy for
molecular electronic structures with multi-task learning.
<em>NATCS</em>, <em>5</em>(2), 184. (<a
href="https://doi.org/10.1038/s43588-025-00767-z">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_NATCS},
  author       = {Tang, Hao and Xiao, Brian and He, Wenhao and Subasic, Pero and Harutyunyan, Avetik R. and Wang, Yao and Liu, Fang and Xu, Haowei and Li, Ju},
  doi          = {10.1038/s43588-025-00767-z},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {184},
  shortjournal = {Nat. Comput. Sci},
  title        = {Author correction: Approaching coupled-cluster accuracy for molecular electronic structures with multi-task learning},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A programmable environment for shape optimization and
shapeshifting problems. <em>NATCS</em>, <em>5</em>(2), 170–183. (<a
href="https://doi.org/10.1038/s43588-024-00749-7">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Soft materials underpin many domains of science and engineering, including soft robotics, structured fluids, and biological and particulate media. In response to applied mechanical, electromagnetic or chemical stimuli, such materials typically change shape, often dramatically. Predicting their structure is of great interest to facilitate design and mechanistic understanding, and can be cast as an optimization problem where a given energy function describing the physics of the material is minimized with respect to the shape of the domain and additional fields. However, shape-optimization problems are very challenging to solve, and there is a lack of suitable simulation tools that are both readily accessible and general in purpose. Here we present an open-source programmable environment, Morpho, and demonstrate its versatility by showcasing a range of applications from different areas of soft-matter physics: swelling hydrogels, complex fluids that form aspherical droplets, soap films and membranes, and filaments. This study introduces an extensible framework—Morpho—for shape optimization, enabling researchers to predict the structure of soft materials, such as complex fluids, gels, particulate and biological materials.},
  archive      = {J_NATCS},
  author       = {Joshi, Chaitanya and Hellstein, Daniel and Wennerholm, Cole and Downey, Eoghan and Hamilton, Emmett and Hocking, Samuel and Andrei, Anca S. and Adler, James H. and Atherton, Timothy J.},
  doi          = {10.1038/s43588-024-00749-7},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {170-183},
  shortjournal = {Nat. Comput. Sci},
  title        = {A programmable environment for shape optimization and shapeshifting problems},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A spatiotemporal style transfer algorithm for dynamic visual
stimulus generation. <em>NATCS</em>, <em>5</em>(2), 155–169. (<a
href="https://doi.org/10.1038/s43588-024-00746-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding how visual information is encoded in biological and artificial systems often requires the generation of appropriate stimuli to test specific hypotheses, but available methods for video generation are scarce. Here we introduce the spatiotemporal style transfer (STST) algorithm, a dynamic visual stimulus generation framework that allows the manipulation and synthesis of video stimuli for vision research. We show how stimuli can be generated that match the low-level spatiotemporal features of their natural counterparts, but lack their high-level semantic features, providing a useful tool to study object recognition. We used these stimuli to probe PredNet, a predictive coding deep network, and found that its next-frame predictions were not disrupted by the omission of high-level information, with human observers also confirming the preservation of low-level features and lack of high-level information in the generated stimuli. We also introduce a procedure for the independent spatiotemporal factorization of dynamic stimuli. Testing such factorized stimuli on humans and deep vision models suggests a spatial bias in how humans and deep vision models encode dynamic visual information. These results showcase potential applications of the STST algorithm as a versatile tool for dynamic stimulus generation in vision science. The spatiotemporal style transfer (STST) algorithm enables video generation by selectively manipulating the spatial and temporal features of natural videos, fostering vision science research in both biological and artificial systems.},
  archive      = {J_NATCS},
  author       = {Greco, Antonino and Siegel, Markus},
  doi          = {10.1038/s43588-024-00746-w},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {155-169},
  shortjournal = {Nat. Comput. Sci},
  title        = {A spatiotemporal style transfer algorithm for dynamic visual stimulus generation},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025a). Approaching coupled-cluster accuracy for molecular
electronic structures with multi-task learning. <em>NATCS</em>,
<em>5</em>(2), 144–154. (<a
href="https://doi.org/10.1038/s43588-024-00747-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Machine learning plays an important role in quantum chemistry, providing fast-to-evaluate predictive models for various properties of molecules; however, most existing machine learning models for molecular electronic properties use density functional theory (DFT) databases as ground truth in training, and their prediction accuracy cannot surpass that of DFT. In this work we developed a unified machine learning method for electronic structures of organic molecules using the gold-standard CCSD(T) calculations as training data. Tested on hydrocarbon molecules, our model outperforms DFT with several widely used hybrid and double-hybrid functionals in terms of both computational cost and prediction accuracy of various quantum chemical properties. We apply the model to aromatic compounds and semiconducting polymers, evaluating both ground- and excited-state properties. The results demonstrate the model’s accuracy and generalization capability to complex systems that cannot be calculated using CCSD(T)-level methods due to scaling. A multi-task deep learning method for molecular electronic structures, called MEHnet, is developed to predict various molecular properties in a unified framework, approaching chemical accuracy while exhibiting local DFT-level computational costs.},
  archive      = {J_NATCS},
  author       = {Tang, Hao and Xiao, Brian and He, Wenhao and Subasic, Pero and Harutyunyan, Avetik R. and Wang, Yao and Liu, Fang and Xu, Haowei and Li, Ju},
  doi          = {10.1038/s43588-024-00747-9},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {144-154},
  shortjournal = {Nat. Comput. Sci},
  title        = {Approaching coupled-cluster accuracy for molecular electronic structures with multi-task learning},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A statistical framework for multi-trait rare variant
analysis in large-scale whole-genome sequencing studies. <em>NATCS</em>,
<em>5</em>(2), 125–143. (<a
href="https://doi.org/10.1038/s43588-024-00764-8">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Large-scale whole-genome sequencing (WGS) studies have improved our understanding of the contributions of coding and noncoding rare variants to complex human traits. Leveraging association effect sizes across multiple traits in WGS rare variant association analysis can improve statistical power over single-trait analysis, and also detect pleiotropic genes and regions. Existing multi-trait methods have limited ability to perform rare variant analysis of large-scale WGS data. We propose MultiSTAAR, a statistical framework and computationally scalable analytical pipeline for functionally informed multi-trait rare variant analysis in large-scale WGS studies. MultiSTAAR accounts for relatedness, population structure and correlation among phenotypes by jointly analyzing multiple traits, and further empowers rare variant association analysis by incorporating multiple functional annotations. We applied MultiSTAAR to jointly analyze three lipid traits in 61,838 multi-ethnic samples from the Trans-Omics for Precision Medicine (TOPMed) Program. We discovered and replicated new associations with lipid traits missed by single-trait analysis. MultiSTAAR provides a general and flexible statistical framework for functionally informed multi-trait rare variant analysis of biobank-scale sequencing studies by jointly analyzing multiple traits and incorporating annotation information.},
  archive      = {J_NATCS},
  author       = {Li, Xihao and Chen, Han and Selvaraj, Margaret Sunitha and Van Buren, Eric and Zhou, Hufeng and Wang, Yuxuan and Sun, Ryan and McCaw, Zachary R. and Yu, Zhi and Jiang, Min-Zhi and DiCorpo, Daniel and Gaynor, Sheila M. and Dey, Rounak and Arnett, Donna K. and Benjamin, Emelia J. and Bis, Joshua C. and Blangero, John and Boerwinkle, Eric and Bowden, Donald W. and Brody, Jennifer A. and Cade, Brian E. and Carson, April P. and Carlson, Jenna C. and Chami, Nathalie and Chen, Yii-Der Ida and Curran, Joanne E. and de Vries, Paul S. and Fornage, Myriam and Franceschini, Nora and Freedman, Barry I. and Gu, Charles and Heard-Costa, Nancy L. and He, Jiang and Hou, Lifang and Hung, Yi-Jen and Irvin, Marguerite R. and Kaplan, Robert C. and Kardia, Sharon L. R. and Kelly, Tanika N. and Konigsberg, Iain and Kooperberg, Charles and Kral, Brian G. and Li, Changwei and Li, Yun and Lin, Honghuang and Liu, Ching-Ti and Loos, Ruth J. F. and Mahaney, Michael C. and Martin, Lisa W. and Mathias, Rasika A. and Mitchell, Braxton D. and Montasser, May E. and Morrison, Alanna C. and Naseri, Take and North, Kari E. and Palmer, Nicholette D. and Peyser, Patricia A. and Psaty, Bruce M. and Redline, Susan and Reiner, Alexander P. and Rich, Stephen S. and Sitlani, Colleen M. and Smith, Jennifer A. and Taylor, Kent D. and Tiwari, Hemant K. and Vasan, Ramachandran S. and Viali, Satupa’itea and Wang, Zhe and Wessel, Jennifer and Yanek, Lisa R. and Yu, Bing and Dupuis, Josée and Meigs, James B. and Auer, Paul L. and Raffield, Laura M. and Manning, Alisa K. and Rice, Kenneth M. and Rotter, Jerome I. and Peloso, Gina M. and Natarajan, Pradeep and Li, Zilin and Liu, Zhonghua and Lin, Xihong},
  doi          = {10.1038/s43588-024-00764-8},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {125-143},
  shortjournal = {Nat. Comput. Sci},
  title        = {A statistical framework for multi-trait rare variant analysis in large-scale whole-genome sequencing studies},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enabling efficient analysis of biobank-scale data with
genotype representation graphs. <em>NATCS</em>, <em>5</em>(2), 112–124.
(<a href="https://doi.org/10.1038/s43588-024-00739-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Computational analysis of a large number of genomes requires a data structure that can represent the dataset compactly while also enabling efficient operations on variants and samples. However, encoding genetic data in existing tabular data structures and file formats has become costly and unsustainable. Here we introduce the genotype representation graph (GRG), a fully connected hierarchical data structure that losslessly encodes phased whole-genome polymorphisms. Exploiting variant-sharing across samples enables GRG to compress 200,000 UK Biobank phased human genomes to 5–26 gigabytes per chromosome, also enabling graph-traversal algorithms to reuse computed values in random access memory. Constructing and processing GRG files scales to a million whole genomes. Using allele frequencies and association effects as examples, we show that computation on GRG via graph traversal runs the fastest among all tested alternatives. GRG-based algorithms have the potential to increase the scalability and reduce the cost of analyzing large genomic datasets. The genotype representation graph (GRG) is a compact data structure that encodes 200,000 human genomes in just 5–26 gigabytes per chromosome. Computation on GRG via graph traversal greatly accelerates genome-wide analysis.},
  archive      = {J_NATCS},
  author       = {DeHaas, Drew and Pan, Ziqing and Wei, Xinzhu},
  doi          = {10.1038/s43588-024-00739-9},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {112-124},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enabling efficient analysis of biobank-scale data with genotype representation graphs},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Predicting emergence of crystals from amorphous precursors
with deep learning potentials. <em>NATCS</em>, <em>5</em>(2), 105–111.
(<a href="https://doi.org/10.1038/s43588-024-00752-y">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Crystallization of amorphous precursors into metastable crystals plays a fundamental role in the formation of new matter, from geological to biological processes in nature to the synthesis and development of new materials in the laboratory. Reliably predicting the outcome of such a process would enable new research directions in these areas, but has remained beyond the reach of molecular modeling or ab initio methods. Here we show that candidates for the crystallization products of amorphous precursors can be predicted in many inorganic systems by sampling the local structural motifs at the atomistic level using universal deep learning interatomic potentials. We show that this approach identifies, with high accuracy, the most likely crystal structures of the polymorphs that initially nucleate from amorphous precursors, across a diverse set of material systems, including polymorphic oxides, nitrides, carbides, fluorides, chlorides, chalcogenides and metal alloys. This study introduces a2c, a computational method that leverages machine learning and atomistic simulations to predict the most likely crystallization products upon annealing of amorphous precursors. The a2c tool was demonstrated on a variety of materials, including oxides, nitrides and metallic glasses, and can assist researchers in discovering synthesis pathways for materials design.},
  archive      = {J_NATCS},
  author       = {Aykol, Muratahan and Merchant, Amil and Batzner, Simon and Wei, Jennifer N. and Cubuk, Ekin Dogus},
  doi          = {10.1038/s43588-024-00752-y},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {105-111},
  shortjournal = {Nat. Comput. Sci},
  title        = {Predicting emergence of crystals from amorphous precursors with deep learning potentials},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). A new tool for shape and structure optimization of soft
materials. <em>NATCS</em>, <em>5</em>(2), 103–104. (<a
href="https://doi.org/10.1038/s43588-024-00754-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present Morpho, an extensible programmable environment that uses finite elements for shape optimization in soft matter. Given an energy functional that incorporates physical boundaries and effects such as elasticity and electromagnetism, together with additional constraints to be satisfied, Morpho predicts the optimized shape and structure adopted by the material.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-024-00754-w},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {103-104},
  shortjournal = {Nat. Comput. Sci},
  title        = {A new tool for shape and structure optimization of soft materials},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). MultiSTAAR delivers multi-trait rare variant analysis of
biobank-scale sequencing data. <em>NATCS</em>, <em>5</em>(2), 101–102.
(<a href="https://doi.org/10.1038/s43588-025-00766-0">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying pleiotropic associations for rare variants in multi-ethnic biobank-scale whole-genome sequencing data poses considerable challenges. This study introduced MultiSTAAR as a scalable and robust multi-trait rare variant analysis framework designed for both coding and noncoding regions by integrating multiple variant functional annotations and leveraging multivariate modeling across diverse phenotypes.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00766-0},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {101-102},
  shortjournal = {Nat. Comput. Sci},
  title        = {MultiSTAAR delivers multi-trait rare variant analysis of biobank-scale sequencing data},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Shedding light on spatial signal transduction in cells using
computational simulations. <em>NATCS</em>, <em>5</em>(2), 99–100. (<a
href="https://doi.org/10.1038/s43588-025-00772-2">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We present Spatial Modeling Algorithms for Reactions and Transport (SMART), a software package that simulates spatiotemporally detailed biochemical reaction networks within realistic cellular and subcellular geometries. This paper highlights the use of SMART in several biological test cases including cellular mechanotransduction, calcium signaling in neurons and cardiomyocytes, and adenosine triphosphate synthesis.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00772-2},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {99-100},
  shortjournal = {Nat. Comput. Sci},
  title        = {Shedding light on spatial signal transduction in cells using computational simulations},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Biologically inspired graphs to explore massive genetic
datasets. <em>NATCS</em>, <em>5</em>(2), 97–98. (<a
href="https://doi.org/10.1038/s43588-024-00763-9">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {A recent study proposes a data structure that addresses crucial challenges related to storage and computation of large genome databases.},
  archive      = {J_NATCS},
  author       = {Layer, Ryan M.},
  doi          = {10.1038/s43588-024-00763-9},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {97-98},
  shortjournal = {Nat. Comput. Sci},
  title        = {Biologically inspired graphs to explore massive genetic datasets},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Enhancing synthesis prediction via machine learning.
<em>NATCS</em>, <em>5</em>(2), 95–96. (<a
href="https://doi.org/10.1038/s43588-025-00771-3">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Identifying promising synthesis targets and designing routes to their synthesis is a grand challenge in chemistry and materials science. Recent work employing machine learning in combination with traditional approaches is opening new ways to address this truly Herculean task.},
  archive      = {J_NATCS},
  author       = {Schön, J. C.},
  doi          = {10.1038/s43588-025-00771-3},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {95-96},
  shortjournal = {Nat. Comput. Sci},
  title        = {Enhancing synthesis prediction via machine learning},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Balancing autonomy and expertise in autonomous synthesis
laboratories. <em>NATCS</em>, <em>5</em>(2), 92–94. (<a
href="https://doi.org/10.1038/s43588-025-00769-x">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Autonomous synthesis laboratories promise to streamline the plan–make–measure–analyze iteration loop. Here, we comment on the barriers in the field, the promise of a human on-the-loop approach, and strategies for optimizing accessibility, accuracy, and efficiency of autonomous laboratories.},
  archive      = {J_NATCS},
  author       = {Liu, Xiaozhao and Ouyang, Bin and Zeng, Yan},
  doi          = {10.1038/s43588-025-00769-x},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {92-94},
  shortjournal = {Nat. Comput. Sci},
  title        = {Balancing autonomy and expertise in autonomous synthesis laboratories},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
<li><details>
<summary>
(2025). Based on the science, diversity matters. <em>NATCS</em>,
<em>5</em>(2), 91. (<a
href="https://doi.org/10.1038/s43588-025-00778-w">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {We reflect on what science tells us about the importance of diversity.},
  archive      = {J_NATCS},
  doi          = {10.1038/s43588-025-00778-w},
  journal      = {Nature Computational Science},
  month        = {2},
  number       = {2},
  pages        = {91},
  shortjournal = {Nat. Comput. Sci},
  title        = {Based on the science, diversity matters},
  volume       = {5},
  year         = {2025},
}
</textarea>
</details></li>
</ul>

</body>
</html>
