<!DOCTYPE html>
<html>
<head>
  <meta charset="utf-8">
  <title>EVLETT_complex_beauty</title>
  <style>
    html {font-size: 22px;}
    body {margin: 0 auto; max-width: 76em;}
    #copyID {font-size: 18px;}
  </style>
  <script>
    function copy(element) {
      if (element.type == "button"){
      element.type="text";
      }
      element.style.color="black";
      element.style.backgroundColor="#C7EDCC";
      element.select();
      element.setSelectionRange(0, 99999);
      navigator.clipboard.writeText(element.value);
      window.getSelection().removeAllRanges();
      element.type="button";
    }
  </script>
</head>
<body>

<h2 id="evlett---50">EVLETT - 50</h2>
<ul>
<li><details>
<summary>
(2020). Plasticity via feedback reduces the cost of developmental
instability. <em>EVLETT</em>, <em>4</em>(6), 570–580. (<a
href="https://doi.org/10.1002/evl3.202">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Costs of plasticity are thought to have important physiological and evolutionary consequences. A commonly predicted cost to plasticity is that plastic genotypes are likely to suffer from developmental instability. Adaptive plasticity requires that the developing organism can in some way sense what environment it is in or how well it is performing in that environment. These two information pathways—an “environmental signal” or a “performance signal” that indicates how well a developing phenotype matches the optimum in the current environment—can differ in their consequences for the organism and its evolution. Here, we consider how developmental instability might emerge as a side-effect of these two distinct mechanisms. Because a performance cue allows a regulatory feedback loop connecting a trait to a feedback signal, we hypothesized that plastic genotypes using a performance signal would be more developmentally robust compared to those using a purely environmental signal. Using a numerical model of a network of gene interactions, we show that plasticity comes at a cost of developmental instability when the plastic response is mediated via an environmental signal, but not when it is mediated via a performance signal. We also show that a performance signal mechanism can evolve even in a constant environment, leading to genotypes preadapted for plasticity to novel environments even in populations without a history of environmental heterogeneity.},
  archive      = {J_EVLETT},
  author       = {Matthey-Doret, Remi and Draghi, Jeremy A. and Whitlock, Michael C.},
  doi          = {10.1002/evl3.202},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {570-580},
  shortjournal = {Evol. Lett.},
  title        = {Plasticity via feedback reduces the cost of developmental instability},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Herbivory and pollination impact on the evolution of
herbivore-induced plasticity in defense and floral traits.
<em>EVLETT</em>, <em>4</em>(6), 556–569. (<a
href="https://doi.org/10.1002/evl3.200">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Theory predicts that herbivory should primarily determine the evolution of herbivore-induced plasticity in plant defenses, but little is known about the influence of other interactions such as pollination. Pollinators may exert negative selection on the herbivore-induced plasticity of chemical defenses when floral signals and rewards are indirectly affected, provoking deterrent effects on these mutualists. We investigated the influence of constant herbivory and pollination on the evolved patterns and degree of herbivore-induced plasticity in chemical plant defenses and floral morphometry and volatiles in fast-cycling ; plants. To do this, we used plants from an evolution experiment that had evolved under bee/hand pollination and herbivory manipulated in a 2 × 2 factorial design during six generations, producing four selection treatments. We grew sibling plant pairs from each of the four selection treatments of the last generation and infested one group with herbivores and left the other uninfested. Herbivore-induced plasticity was analyzed within- and between-selection treatments. We found support for the hypothesis that constant herbivory favors the evolution of higher constitutive yet lower herbivore-induced plasticity in defenses. However, this only occurred in plants that evolved under hand pollination and constant herbivory. Bee pollination had a strong influence on the evolution of herbivore-induced plasticity of all traits studied. Plants that evolved under bee pollination, with and without constant herbivory, showed remarkably similar patterns of herbivore-induced plasticity in their defense- and floral traits and had a higher number of plastic responses compared to plants with hand pollination. Such patterns support the hypothesis that bee pollination influenced the evolution of herbivore-induced plasticity, most likely via indirect effects, such as links between defense- and floral traits. We conclude that interactions other than herbivory, such as pollination, may impact herbivore-induced plasticity, through indirect effects and metabolic trade-offs, when it contributes to trait evolution in plants.},
  archive      = {J_EVLETT},
  author       = {Ramos, Sergio E. and Schiestl, Florian P.},
  doi          = {10.1002/evl3.200},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {556-569},
  shortjournal = {Evol. Lett.},
  title        = {Herbivory and pollination impact on the evolution of herbivore-induced plasticity in defense and floral traits},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Butterfly dichromatism primarily evolved via darwin’s, not
wallace’s, model. <em>EVLETT</em>, <em>4</em>(6), 545–555. (<a
href="https://doi.org/10.1002/evl3.199">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sexual dimorphism is typically thought to result from sexual selection for elaborated male traits, as proposed by Darwin. However, natural selection could reduce expression of elaborated traits in females, as proposed by Wallace. Darwin and Wallace debated the origins of dichromatism in birds and butterflies, and although evidence in birds is roughly equal, if not in favor of Wallace&#39;s model, butterflies lack a similar scale of study. Here, we present a large-scale comparative phylogenetic analysis of the evolution of butterfly coloration, using all European non-hesperiid butterfly species (; = 369). We modeled evolutionary changes in coloration for each species and sex along their phylogeny, thereby estimating the rate and direction of evolution in three-dimensional color space using a novel implementation of phylogenetic ridge regression. We show that male coloration evolved faster than female coloration, especially in strongly dichromatic clades, with male contribution to changes in dichromatism roughly twice that of females. These patterns are consistent with a classic Darwinian model of dichromatism via sexual selection on male coloration, suggesting this model was the dominant driver of dichromatism in European butterflies.},
  archive      = {J_EVLETT},
  author       = {van der Bijl, Wouter and Zeuss, Dirk and Chazot, Nicolas and Tunström, Kalle and Wahlberg, Niklas and Wiklund, Christer and Fitzpatrick, John L. and Wheat, Christopher W.},
  doi          = {10.1002/evl3.199},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {545-555},
  shortjournal = {Evol. Lett.},
  title        = {Butterfly dichromatism primarily evolved via darwin&#39;s, not wallace&#39;s, model},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Surprising spatiotemporal stability of a multi-peak fitness
landscape revealed by independent field experiments measuring hybrid
fitness. <em>EVLETT</em>, <em>4</em>(6), 530–544. (<a
href="https://doi.org/10.1002/evl3.195">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The effect of the environment on fitness in natural populations is a fundamental question in evolutionary biology. However, experimental manipulations of both environment and phenotype at the same time are rare. Thus, the relative importance of the competitive environment versus intrinsic organismal performance in shaping the location, height, and fluidity of fitness peaks and valleys remains largely unknown. Here, we experimentally tested the effect of competitor frequency on the complex fitness landscape driving adaptive radiation of a generalist and two trophic specialist pupfishes, a scale-eater and molluscivore, endemic to hypersaline lakes on San Salvador Island (SSI), Bahamas. We manipulated phenotypes, by generating 3407 F4/F5 lab-reared hybrids, and competitive environment, by altering the frequency of rare transgressive hybrids between field enclosures in two independent lake populations. We then tracked hybrid survival and growth rates across these four field enclosures for 3–11 months. In contrast to competitive speciation theory, we found no evidence that the frequency of hybrid phenotypes affected their survival. Instead, we observed a strikingly similar fitness landscape to a previous independent field experiment, each supporting multiple fitness peaks for generalist and molluscivore phenotypes and a large fitness valley isolating the divergent scale-eater phenotype. These features of the fitness landscape were stable across manipulated competitive environments, multivariate trait axes, and spatiotemporal heterogeneity. We suggest that absolute performance constraints and divergent gene regulatory networks shape macroevolutionary (interspecific) fitness landscapes in addition to microevolutionary (intraspecific) competitive dynamics. This interplay between organism and environment underlies static and dynamic features of the adaptive landscape.},
  archive      = {J_EVLETT},
  author       = {Martin, Christopher H. and Gould, Katelyn J.},
  doi          = {10.1002/evl3.195},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {530-544},
  shortjournal = {Evol. Lett.},
  title        = {Surprising spatiotemporal stability of a multi-peak fitness landscape revealed by independent field experiments measuring hybrid fitness},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Identifying the fitness consequences of sex in complex
natural environments. <em>EVLETT</em>, <em>4</em>(6), 516–529. (<a
href="https://doi.org/10.1002/evl3.194">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {In the natural world, sex prevails, despite its costs. Although much effort has been dedicated to identifying the intrinsic costs of sex (e.g., the cost of males), few studies have identified the ecological fitness consequences of sex. Furthermore, correlated biological traits that differ between sexuals and asexuals may alter these costs, or even render the typical costs of sex irrelevant. We conducted a large-scale, multisite, reciprocal transplant using multiple sexual and asexual genotypes of a native North American wildflower to show that sexual genotypes have reduced lifetime fitness, despite lower herbivory. We separated the effects of sex from those of hybridity, finding that overwinter survival is elevated in asexuals regardless of hybridity, but herbivores target hybrid asexuals more than nonhybrid asexual or sexual genotypes. Survival is lowest in homozygous sexual lineages, implicating inbreeding depression as a cost of sex. Our results show that the consequences of sex are shaped not just by sex itself, but by complex natural environments, correlated traits, and the identity and availability of mates.},
  archive      = {J_EVLETT},
  author       = {Rushworth, Catherine A. and Brandvain, Yaniv and Mitchell-Olds, Tom},
  doi          = {10.1002/evl3.194},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {516-529},
  shortjournal = {Evol. Lett.},
  title        = {Identifying the fitness consequences of sex in complex natural environments},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Selection on a small genomic region underpins
differentiation in multiple color traits between two warbler species.
<em>EVLETT</em>, <em>4</em>(6), 502–515. (<a
href="https://doi.org/10.1002/evl3.198">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Speciation is one of the most important processes in biology, yet the study of the genomic changes underlying this process is in its infancy. North American warbler species ; and ; hybridize in a stable hybrid zone, following a period of geographic separation. Genomic differentiation accumulated during geographic isolation can be homogenized by introgression at secondary contact, whereas genetic regions that cause low hybrid fitness can be shielded from such introgression. Here, we examined the genomic underpinning of speciation by investigating (1) the genetic basis of divergent pigmentation traits between species, (2) variation in differentiation across the genome, and (3) the evidence for selection maintaining differentiation in the pigmentation genes. Using tens of thousands of single nucleotide polymorphisms (SNPs) genotyped in hundreds of individuals within and near the hybrid zone, genome-wide association mapping revealed a single SNP associated with cheek, crown, breast coloration, and flank streaking, reflecting pleiotropy (one gene affecting multiple traits) or close physical linkage of different genes affecting different traits. This SNP is within an intron of the RALY gene, hence we refer to it as the RALY SNP. We then examined between-species genomic differentiation, using both genotyping-by-sequencing and whole genome sequencing. We found that the RALY SNP is within one of the highest peaks of differentiation, which contains three genes known to influence pigmentation: ASIP, EIF2S2, and RALY (the ASIP-RALY gene block). Heterozygotes at this gene block are likely of reduced fitness, as the geographic cline of the RALY SNP has been narrow over two decades. Together, these results reflect at least one barrier to gene flow within this narrow (∼200 kb) genomic region that modulates plumage difference between species. Despite extensive gene flow between species across the genome, this study provides evidence that selection on a phenotype-associated genomic region maintains a stable species boundary.},
  archive      = {J_EVLETT},
  author       = {Wang, Silu and Rohwer, Sievert and de Zwaan, Devin R. and Toews, David P. L. and Lovette, Irby J. and Mackenzie, Jacqueline and Irwin, Darren},
  doi          = {10.1002/evl3.198},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {502-515},
  shortjournal = {Evol. Lett.},
  title        = {Selection on a small genomic region underpins differentiation in multiple color traits between two warbler species},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Experimental evidence for stabilizing selection on virulence
in a bacterial pathogen. <em>EVLETT</em>, <em>4</em>(6), 491–501. (<a
href="https://doi.org/10.1002/evl3.203">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The virulence-transmission trade-off hypothesis has provided a dominant theoretical basis for predicting pathogen virulence evolution, but empirical tests are rare, particularly at pathogen emergence. The central prediction of this hypothesis is that pathogen fitness is maximized at intermediate virulence due to a trade-off between infection duration and transmission rate. However, obtaining sufficient numbers of pathogen isolates of contrasting virulence to test the shape of relationships between key pathogen traits, and doing so without the confounds of evolved host protective immunity (as expected at emergence), is challenging. Here, we inoculated 55 isolates of the bacterial pathogen, ; , into non-resistant house finches (; ) from populations that have never been exposed to the disease. Isolates were collected over a 20-year period from outbreak in disease-exposed populations of house finches and vary markedly in virulence. We found a positive linear relationship between pathogen virulence and transmission rate to an uninfected sentinel, supporting the core assumption of the trade-off hypothesis. Further, in support of the key prediction, there was no evidence for directional selection on a quantitative proxy of pathogen virulence and, instead, isolates of intermediate virulence were fittest. Surprisingly, however, the positive relationship between virulence and transmission rate was not underpinned by variation in pathogen load or replication rate as is commonly assumed. Our results indicate that selection favors pathogens of intermediate virulence at disease emergence in a novel host species, even when virulence and transmission are not linked to pathogen load.},
  archive      = {J_EVLETT},
  author       = {Bonneaud, Camille and Tardy, Luc and Hill, Geoffrey E. and McGraw, Kevin J. and Wilson, Alastair J. and Giraudeau, Mathieu},
  doi          = {10.1002/evl3.203},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {491-501},
  shortjournal = {Evol. Lett.},
  title        = {Experimental evidence for stabilizing selection on virulence in a bacterial pathogen},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Parent of origin gene expression in the bumblebee, bombus
terrestris, supports haig’s kinship theory for the evolution of genomic
imprinting. <em>EVLETT</em>, <em>4</em>(6), 479–490. (<a
href="https://doi.org/10.1002/evl3.197">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic imprinting is the differential expression alleles in diploid individuals, with the expression being dependent on the sex of the parent from which it was inherited. Haig&#39;s kinship theory hypothesizes that genomic imprinting is due to an evolutionary conflict of interest between alleles from the mother and father. In social insects, it has been suggested that genomic imprinting should be widespread. One recent study identified parent-of-origin expression in honey bees and found evidence supporting the kinship theory. However, little is known about genomic imprinting in insects and multiple theoretical predictions must be tested to avoid single-study confirmation bias. We, therefore, tested for parent-of-origin expression in a primitively eusocial bee. We found equal numbers of maternally and paternally biased expressed genes. The most highly biased genes were maternally expressed, offering support for the kinship theory. We also found low conservation of potentially imprinted genes with the honey bee, suggesting rapid evolution of genomic imprinting in Hymenoptera.},
  archive      = {J_EVLETT},
  author       = {Marshall, Hollie and van Zweden, Jelle S. and Van Geystelen, Anneleen and Benaets, Kristof and Wäckers, Felix and Mallon, Eamonn B. and Wenseleers, Tom},
  doi          = {10.1002/evl3.197},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {479-490},
  shortjournal = {Evol. Lett.},
  title        = {Parent of origin gene expression in the bumblebee, bombus terrestris, supports haig&#39;s kinship theory for the evolution of genomic imprinting},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). How does the strength of selection influence genetic
correlations? <em>EVLETT</em>, <em>4</em>(6), 468–478. (<a
href="https://doi.org/10.1002/evl3.201">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genetic correlations between traits can strongly impact evolutionary responses to selection, and may thus impose constraints on adaptation. Theoretical and empirical work has made it clear that without strong linkage and with random mating, genetic correlations at evolutionary equilibrium result from an interplay of correlated pleiotropic effects of mutations, and correlational selection favoring combinations of trait values. However, it is not entirely clear how change in the overall strength of stabilizing selection across traits (breadth of the fitness peak, given its shape) influences this compromise between mutation and selection effects on genetic correlation. Here, we show that the answer to this question crucially depends on the intensity of genetic drift. In large, effectively infinite populations, genetic correlations are unaffected by the strength of selection, regardless of whether the genetic architecture involves common small-effect mutations (Gaussian regime), or rare large-effect mutations (House-of-Cards regime). In contrast in finite populations, the strength of selection does affect genetic correlations, by shifting the balance from drift-dominated to selection-dominated evolutionary dynamics. The transition between these domains depends on mutation parameters to some extent, but with a similar dependence of genetic correlation on the strength of selection. Our results are particularly relevant for understanding how senescence shapes patterns of genetic correlations across ages, and genetic constraints on adaptation during colonization of novel habitats.},
  archive      = {J_EVLETT},
  author       = {Chantepie, Stéphane and Chevin, Luis-Miguel},
  doi          = {10.1002/evl3.201},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {468-478},
  shortjournal = {Evol. Lett.},
  title        = {How does the strength of selection influence genetic correlations?},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020a). Issue information. <em>EVLETT</em>, <em>4</em>(6), 465–467.
(<a href="https://doi.org/10.1002/evl3.211">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_EVLETT},
  doi          = {10.1002/evl3.211},
  journal      = {Evolution Letters},
  number       = {6},
  pages        = {465-467},
  shortjournal = {Evol. Lett.},
  title        = {Issue information},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). A novel seed dispersal mode of apostasia nipponica could
provide some clues to the early evolution of the seed dispersal system
in orchidaceae. <em>EVLETT</em>, <em>4</em>(5), 457–464. (<a
href="https://doi.org/10.1002/evl3.188">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Despite being one of the most diverse families, scant attention has been paid to the seed dispersal system in Orchidaceae, owing to the widely accepted notion that wind dispersal is the dominant strategy. However, the indehiscent fruits, with seeds immersed in fleshy tissue, evoke the possibility of endozoochory in Apostasioideae, the earliest diverging lineage of orchids. In the present study, I investigated the seed dispersal system of ; by direct observation, time-lapse photography, and investigation of the viability of seeds passing through the digestive tract of orthopterans. This study revealed a previously undocumented seed dispersal system in ; , in which the cricket, ; , and the camel cricket, ; , consume the fruit and defecate viable seeds. Orthopterans are rarely considered seed dispersers, but the gross fruit morphology and pigmentation patterns of some ; species parallel those seen in ; , suggesting that similar seed dispersal systems could be widespread among ; species. Whether seed dispersal by orthopteran frugivores is common in Apostasioideae warrants further investigation.},
  archive      = {J_EVLETT},
  author       = {Suetsugu, Kenji},
  doi          = {10.1002/evl3.188},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {457-464},
  shortjournal = {Evol. Lett.},
  title        = {A novel seed dispersal mode of apostasia nipponica could provide some clues to the early evolution of the seed dispersal system in orchidaceae},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Hybridization and introgression between toads with different
sex chromosome systems. <em>EVLETT</em>, <em>4</em>(5), 444–456. (<a
href="https://doi.org/10.1002/evl3.191">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The growing interest in the lability of sex determination in non-model vertebrates such as amphibians and fishes has revealed high rates of sex chromosome turnovers among closely related species of the same clade. Can such lineages hybridize and admix with different sex-determining systems, or could the changes have precipitated their speciation? We addressed these questions in incipient species of toads (Bufonidae), where we identified a heterogametic transition and characterized their hybrid zone with genome-wide markers (RADseq). Adult and sibship data confirmed that the common toad ; is female heterogametic (ZW), while its sister species the spined toad ; is male heterogametic (XY). Analysis of a fine scale transect across their parapatric ranges in southeastern France unveiled a narrow tension zone (∼10 km), with asymmetric mitochondrial and nuclear admixture over hundreds of kilometers southward and northward, respectively. The geographic extent of introgression is consistent with an expansion of ; across ; ’s former ranges in Mediterranean France, as also suggested by species distribution models. However, widespread cyto-nuclear discordances (; backrosses carrying ; mtDNA) run against predictions from the dominance effects of Haldane&#39;s rule, perhaps because Y and W heterogametologs are not degenerated. Common and spined toads can thus successfully cross-breed despite fundamental differences in their sex determination mechanisms, but remain partially separated by reproductive barriers. Whether and how the interactions of their XY and ZW genes contribute to these barriers shall provide novel insights on the debated role of labile sex chromosomes in speciation.},
  archive      = {J_EVLETT},
  author       = {Dufresnes, Christophe and Litvinchuk, Spartak N and Rozenblut-Kościsty, Beata and Rodrigues, Nicolas and Perrin, Nicolas and Crochet, Pierre-André and Jeffries, Daniel L},
  doi          = {10.1002/evl3.191},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {444-456},
  shortjournal = {Evol. Lett.},
  title        = {Hybridization and introgression between toads with different sex chromosome systems},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Footprints of local adaptation span hundreds of linked genes
in the atlantic silverside genome. <em>EVLETT</em>, <em>4</em>(5),
430–443. (<a href="https://doi.org/10.1002/evl3.189">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The study of local adaptation in the presence of ongoing gene flow is the study of natural selection in action, revealing the functional genetic diversity most relevant to contemporary pressures. In addition to individual genes, genome-wide architecture can itself evolve to enable adaptation. Distributed across a steep thermal gradient along the east coast of North America, Atlantic silversides (; ) exhibit an extraordinary degree of local adaptation in a suite of traits, and the capacity for rapid adaptation from standing genetic variation, but we know little about the patterns of genomic variation across the species range that enable this remarkable adaptability. Here, we use low-coverage, whole-transcriptome sequencing of Atlantic silversides sampled along an environmental cline to show marked signatures of divergent selection across a gradient of neutral differentiation. Atlantic silversides sampled across 1371 km of the southern section of its distribution have very low genome-wide differentiation (median ; = 0.006 across 1.9 million variants), consistent with historical connectivity and observations of recent migrants. Yet almost 14,000 single nucleotide polymorphisms (SNPs) are nearly fixed (;  &gt; 0.95) for alternate alleles. Highly differentiated SNPs cluster into four tight linkage disequilibrium (LD) blocks that span hundreds of genes and several megabases. Variants in these LD blocks are disproportionately nonsynonymous and concentrated in genes enriched for multiple functions related to known adaptations in silversides, including variation in lipid storage, metabolic rate, and spawning behavior. Elevated levels of absolute divergence and demographic modeling suggest selection maintaining divergence across these blocks under gene flow. These findings represent an extreme case of heterogeneity in levels of differentiation across the genome, and highlight how gene flow shapes genomic architecture in continuous populations. Locally adapted alleles may be common features of populations distributed along environmental gradients, and will likely be key to conserving variation to enable future responses to environmental change.},
  archive      = {J_EVLETT},
  author       = {Wilder, Aryn P. and Palumbi, Stephen R. and Conover, David O. and Therkildsen, Nina Overgaard},
  doi          = {10.1002/evl3.189},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {430-443},
  shortjournal = {Evol. Lett.},
  title        = {Footprints of local adaptation span hundreds of linked genes in the atlantic silverside genome},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). How female × male and male × male interactions influence
competitive fertilization in drosophila melanogaster. <em>EVLETT</em>,
<em>4</em>(5), 416–429. (<a
href="https://doi.org/10.1002/evl3.193">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {How males and females contribute to joint reproductive success has been a long-standing question in sexual selection. Under postcopulatory sexual selection, paternity success is predicted to derive from complex interactions among females engaging in cryptic female choice and males engaging in sperm competition. Such interactions have been identified as potential sources of genetic variation in sexually selected traits but are also expected to inhibit trait diversification. To date, studies of interactions between females and competing males have focused almost exclusively on genotypes and not phenotypic variation in sexually selected traits. Here, we characterize within- and between-sex interactions in ; using isogenic lines with heritable variation in both male and female traits known to influence competitive fertilization. We confirmed, and expanded on, previously reported genotypic interactions within and between the sexes, and showed that several reproductive events, including sperm transfer, female sperm ejection, and sperm storage, were explained by two- and three-way interactions among sex-specific phenotypes. We also documented complex interactions between the lengths of competing males’ sperm and the female seminal receptacle, which are known to have experienced rapid female-male co-diversification. Our results highlight the nonindependence of sperm competition and cryptic female choice and demonstrate that complex interactions between the sexes do not limit the ability of multivariate systems to respond to directional sexual selection.},
  archive      = {J_EVLETT},
  author       = {Lüpold, Stefan and Reil, Jonathan Bradley and Manier, Mollie K. and Zeender, Valérian and Belote, John M. and Pitnick, Scott},
  doi          = {10.1002/evl3.193},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {416-429},
  shortjournal = {Evol. Lett.},
  title        = {How female × male and male × male interactions influence competitive fertilization in drosophila melanogaster},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The search for sexually antagonistic genes: Practical
insights from studies of local adaptation and statistical genomics.
<em>EVLETT</em>, <em>4</em>(5), 398–415. (<a
href="https://doi.org/10.1002/evl3.192">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sexually antagonistic (SA) genetic variation—in which alleles favored in one sex are disfavored in the other—is predicted to be common and has been documented in several animal and plant populations, yet we currently know little about its pervasiveness among species or its population genetic basis. Recent applications of genomics in studies of SA genetic variation have highlighted considerable methodological challenges to the identification and characterization of SA genes, raising questions about the feasibility of genomic approaches for inferring SA selection. The related fields of local adaptation and statistical genomics have previously dealt with similar challenges, and lessons from these disciplines can therefore help overcome current difficulties in applying genomics to study SA genetic variation. Here, we integrate theoretical and analytical concepts from local adaptation and statistical genomics research—including ; and ; statistics, genome-wide association studies, pedigree analyses, reciprocal transplant studies, and evolve-and-resequence experiments—to evaluate methods for identifying SA genes and genome-wide signals of SA genetic variation. We begin by developing theoretical models for between-sex ; and ; , including explicit null distributions for each statistic, and using them to critically evaluate putative multilocus signals of sex-specific selection in previously published datasets. We then highlight new statistics that address some of the limitations of ; and ; , along with applications of more direct approaches for characterizing SA genetic variation, which incorporate explicit fitness measurements. We finish by presenting practical guidelines for the validation and evolutionary analysis of candidate SA genes and discussing promising empirical systems for future work.},
  archive      = {J_EVLETT},
  author       = {Ruzicka, Filip and Dutoit, Ludovic and Czuppon, Peter and Jordan, Crispin Y. and Li, Xiang-Yi and Olito, Colin and Runemark, Anna and Svensson, Erik I. and Yazdi, Homa Papoli and Connallon, Tim},
  doi          = {10.1002/evl3.192},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {398-415},
  shortjournal = {Evol. Lett.},
  title        = {The search for sexually antagonistic genes: Practical insights from studies of local adaptation and statistical genomics},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020b). Issue information. <em>EVLETT</em>, <em>4</em>(5), 395–397.
(<a href="https://doi.org/10.1002/evl3.196">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_EVLETT},
  doi          = {10.1002/evl3.196},
  journal      = {Evolution Letters},
  number       = {5},
  pages        = {395-397},
  shortjournal = {Evol. Lett.},
  title        = {Issue information},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Longevity and the drift barrier: Bridging the gap between
medawar and hamilton. <em>EVLETT</em>, <em>4</em>(4), 382–393. (<a
href="https://doi.org/10.1002/evl3.173">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Most organisms have finite life spans. The maximum life span of mammals, for example, is at most some years, decades, or centuries. Why not thousands of years or more? Can we explain and predict maximum life spans theoretically, based on other traits of organisms and associated ecological constraints? Existing theory provides reasons for the prevalence of ageing, but making explicit quantitative predictions of life spans is difficult. Here, I show that there are important unappreciated differences between two backbones of the theory of senescence: Peter Medawar&#39;s verbal model, and William Hamilton&#39;s subsequent mathematical model. I construct a mathematical model corresponding more closely to Medawar&#39;s verbal description, incorporating mutations of large effect and finite population size. In this model, the drift barrier provides a standard by which the limits of natural selection on age-specific mutations can be measured. The resulting model reveals an approximate quantitative explanation for typical maximum life spans. Although maximum life span is expected to increase with population size, it does so extremely slowly, so that even the largest populations imaginable have limited ability to maintain long life spans. Extreme life spans that are observed in some organisms are explicable when indefinite growth or clonal reproduction is included in the model.},
  archive      = {J_EVLETT},
  author       = {Lehtonen, Jussi},
  doi          = {10.1002/evl3.173},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {382-393},
  shortjournal = {Evol. Lett.},
  title        = {Longevity and the drift barrier: Bridging the gap between medawar and hamilton},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Environmental variation mediates the evolution of
anticipatory parental effects. <em>EVLETT</em>, <em>4</em>(4), 371–381.
(<a href="https://doi.org/10.1002/evl3.177">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Theory maintains that when future environment is predictable, parents should adjust the phenotype of their offspring to match the anticipated environment. The plausibility of positive anticipatory parental effects is hotly debated and the experimental evidence for the evolution of such effects is currently lacking. We experimentally investigated the evolution of anticipatory maternal effects in a range of environments that differ drastically in how predictable they are. Populations of the nematode ; , adapted to 20°C, were exposed to a novel temperature (25°C) for 30 generations with either positive or zero correlation between parent and offspring environment. We found that populations evolving in novel environments that were predictable across generations evolved a positive anticipatory maternal effect, because they required maternal exposure to 25°C to achieve maximum reproduction in that temperature. In contrast, populations evolving under zero environmental correlation had lost this anticipatory maternal effect. Similar but weaker patterns were found if instead rate-sensitive population growth was used as a fitness measure. These findings demonstrate that anticipatory parental effects evolve in response to environmental change so that ill-fitting parental effects can be rapidly lost. Evolution of positive anticipatory parental effects can aid population viability in rapidly changing but predictable environments.},
  archive      = {J_EVLETT},
  author       = {Lind, Martin I. and Zwoinska, Martyna K. and Andersson, Johan and Carlsson, Hanne and Krieg, Therese and Larva, Tuuli and Maklakov, Alexei A.},
  doi          = {10.1002/evl3.177},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {371-381},
  shortjournal = {Evol. Lett.},
  title        = {Environmental variation mediates the evolution of anticipatory parental effects},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Plasticity leaves a phenotypic signature during local
adaptation. <em>EVLETT</em>, <em>4</em>(4), 360–370. (<a
href="https://doi.org/10.1002/evl3.185">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Phenotypic responses to a novel or extreme environment are initially plastic, only later to be followed by genetic change. Whether or not environmentally induced phenotypes are sufficiently recurrent and fit to leave a signature in adaptive evolution is debated. Here, we analyze multivariate data from 34 plant reciprocal transplant studies to test: (1) if plasticity is an adaptive source of developmental bias that makes locally adapted populations resemble the environmentally induced phenotypes of ancestors; and (2) if plasticity, standing phenotypic variation and genetic divergence align during local adaptation. Phenotypic variation increased marginally in foreign environments but, as predicted, the direction of ancestral plasticity was generally well aligned with the phenotypic difference between locally adapted populations, making plasticity appear to &quot;take the lead&quot; in adaptive evolution. Plastic responses were sometimes more extreme than the phenotypes of locally adapted plants, which can give the impression that plasticity and evolutionary adaptation oppose each other; however, environmentally induced and locally adapted phenotypes were rarely misaligned. Adaptive fine-tuning of phenotypes—genetic accommodation—did not fall along the main axis of standing phenotypic variation or the direction of plasticity, and local adaptation did not consistently modify the direction or magnitude of plasticity. These results suggest that plasticity is a persistent source of developmental bias that shapes how plant populations adapt to environmental change, even when plasticity does not constrain how populations respond to selection.},
  archive      = {J_EVLETT},
  author       = {Radersma, Reinder and Noble, Daniel W.A. and Uller, Tobias},
  doi          = {10.1002/evl3.185},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {360-370},
  shortjournal = {Evol. Lett.},
  title        = {Plasticity leaves a phenotypic signature during local adaptation},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Rapid local adaptation linked with phenotypic plasticity.
<em>EVLETT</em>, <em>4</em>(4), 345–359. (<a
href="https://doi.org/10.1002/evl3.176">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Models of “plasticity-first” evolution are attractive because they explain the rapid evolution of new complex adaptations. Nevertheless, it is unclear whether plasticity can facilitate rapid microevolutionary change between diverging populations. Here, we show how plasticity may have generated adaptive differences in fecundity between neighboring wild populations of burying beetles ; . These populations occupy distinct Cambridgeshire woodlands that are just 2.5 km apart and that probably originated from a common ancestral population about 1000-4000 years ago. We find that populations are divergently adapted to breed on differently sized carrion. Adaptive differences in clutch size and egg size are associated with divergence at loci connected with oogenesis. The populations differ specifically in the elevation of the reaction norm linking clutch size to carrion size (i.e., genetic accommodation), and in the likelihood that surplus offspring will be lost after hatching. We suggest that these two processes may have facilitated rapid local adaptation on a fine-grained spatial scale.},
  archive      = {J_EVLETT},
  author       = {Sun, Syuan-Jyun and Catherall, Andrew M. and Pascoal, Sonia and Jarrett, Benjamin J. M. and Miller, Sara E. and Sheehan, Michael J. and Kilner, Rebecca M.},
  doi          = {10.1002/evl3.176},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {345-359},
  shortjournal = {Evol. Lett.},
  title        = {Rapid local adaptation linked with phenotypic plasticity},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Metabolic rate, context-dependent selection, and the
competition-colonization trade-off. <em>EVLETT</em>, <em>4</em>(4),
333–344. (<a href="https://doi.org/10.1002/evl3.174">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Metabolism is linked with the pace-of-life, co-varying with survival, growth, and reproduction. Metabolic rates should therefore be under strong selection and, if heritable, become less variable over time. Yet intraspecific variation in metabolic rates is ubiquitous, even after accounting for body mass and temperature. Theory predicts variable selection maintains trait variation, but field estimates of how selection on metabolism varies are rare. We use a model marine invertebrate to estimate selection on metabolic rates in the wild under different competitive environments. Fitness landscapes varied among environments separated by a few centimeters: interspecific competition selected for higher metabolism, and a faster pace-of-life, relative to competition-free environments. Populations experience a mosaic of competitive regimes; we find metabolism mediates a competition-colonization trade-off across these regimes. Although high metabolic phenotypes possess greater competitive ability, in the absence of competitors, low metabolic phenotypes are better colonizers. Spatial heterogeneity and the variable selection on metabolic rates that it generates is likely to maintain variation in metabolic rate, despite strong selection in any single environment.},
  archive      = {J_EVLETT},
  author       = {Pettersen, Amanda K. and Hall, Matthew D. and White, Craig R. and Marshall, Dustin J.},
  doi          = {10.1002/evl3.174},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {333-344},
  shortjournal = {Evol. Lett.},
  title        = {Metabolic rate, context-dependent selection, and the competition-colonization trade-off},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The geographic mosaic of arms race coevolution is closely
matched to prey population structure. <em>EVLETT</em>, <em>4</em>(4),
317–332. (<a href="https://doi.org/10.1002/evl3.184">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Reciprocal adaptation is the hallmark of arms race coevolution. Local coadaptation between natural enemies should generate a geographic mosaic pattern where both species have roughly matched abilities across their shared range. However, mosaic variation in ecologically relevant traits can also arise from processes unrelated to reciprocal selection, such as population structure or local environmental conditions. We tested whether these alternative processes can account for trait variation in the geographic mosaic of arms race coevolution between resistant garter snakes (; ) and toxic newts (; ). We found that predator resistance and prey toxin levels are functionally matched in co-occurring populations, suggesting that mosaic variation in the armaments of both species results from the local pressures of reciprocal selection. By the same token, phenotypic and genetic variation in snake resistance deviates from neutral expectations of population genetic differentiation, showing a clear signature of adaptation to local toxin levels in newts. Contrastingly, newt toxin levels are best predicted by genetic differentiation among newt populations, and to a lesser extent, by the local environment and snake resistance. Exaggerated armaments suggest that coevolution occurs in certain hotspots, but prey population structure seems to be of particular influence on local phenotypic variation in both species throughout the geographic mosaic. Our results imply that processes other than reciprocal selection, like historical biogeography and environmental pressures, represent an important source of variation in the geographic mosaic of coevolution. Such a pattern supports the role of “trait remixing” in the geographic mosaic theory, the process by which non-adaptive forces dictate spatial variation in the interactions among species.},
  archive      = {J_EVLETT},
  author       = {Hague, Michael T. J. and Stokes, Amber N. and Feldman, Chris R. and Brodie, Edmund D.},
  doi          = {10.1002/evl3.184},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {317-332},
  shortjournal = {Evol. Lett.},
  title        = {The geographic mosaic of arms race coevolution is closely matched to prey population structure},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Loss of ecologically important genetic variation in late
generation hybrids reveals links between adaptation and speciation.
<em>EVLETT</em>, <em>4</em>(4), 302–316. (<a
href="https://doi.org/10.1002/evl3.187">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Adaptation to contrasting environments occurs when advantageous alleles accumulate in each population, but it remains largely unknown whether these same advantageous alleles create genetic incompatibilities that can cause intrinsic reproductive isolation leading to speciation. Identifying alleles that underlie both adaptation and reproductive isolation is further complicated by factors such as dominance and genetic interactions among loci, which can affect both processes differently and obscure potential links between adaptation and speciation. Here, we use a combination of field and glasshouse experiments to explore the connection between adaptation and speciation while accounting for dominance and genetic interactions. We created a hybrid population with equal contributions from four contrasting ecotypes of ; (Asteraceae), which produced hybrid genomes both before (F1 hybrid generation) and after (F4 hybrid generation) recombination among the parental ecotypes. In the glasshouse, plants in the second generation (F2 hybrid generation) showed reduced fitness as a loss of fertility. However, fertility was recovered in subsequent generations, suggesting that genetic variation underlying the fitness reduction was lost in subsequent generations. To quantify the effects of losing genetic variation at the F2 generation on the fitness of later generation hybrids, we used a reciprocal transplant to test for fitness differences between parental ecotypes, and F1 and F4 hybrids in all four parental habitats. Compared to the parental ecotypes and F1 hybrids, variance in F4 hybrid fitness was lower, and lowest in habitats that showed stronger native-ecotype advantage, suggesting that stronger natural selection for the native ecotype reduced fitness variation in the F4 hybrids. Fitness trade-offs that were present in the parental ecotypes and F1 hybrids were absent in the F4 hybrid. Together, these results suggest that the genetic variation lost after the F2 generation was likely associated with both adaptation and intrinsic reproductive isolation among ecotypes from contrasting habitats.},
  archive      = {J_EVLETT},
  author       = {Walter, Greg M. and Richards, Thomas J. and Wilkinson, Melanie J and Blows, Mark W. and Aguirre, J. David and Ortiz-Barrientos, Daniel},
  doi          = {10.1002/evl3.187},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {302-316},
  shortjournal = {Evol. Lett.},
  title        = {Loss of ecologically important genetic variation in late generation hybrids reveals links between adaptation and speciation},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Adaptation via pleiotropy and linkage: Association mapping
reveals a complex genetic architecture within the stickleback eda locus.
<em>EVLETT</em>, <em>4</em>(4), 282–301. (<a
href="https://doi.org/10.1002/evl3.175">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Genomic mapping of the loci associated with phenotypic evolution has revealed genomic “hotspots,” or regions of the genome that control multiple phenotypic traits. This clustering of loci has important implications for the speed and maintenance of adaptation and could be due to pleiotropic effects of a single mutation or tight genetic linkage of multiple causative mutations affecting different traits. The threespine stickleback (; ) is a powerful model for the study of adaptive evolution because the marine ecotype has repeatedly adapted to freshwater environments across the northern hemisphere in the last 12,000 years. Freshwater ecotypes have repeatedly fixed a 16 kilobase haplotype on chromosome IV that contains ; (; ), a gene known to affect multiple traits, including defensive armor plates, lateral line sensory hair cells, and schooling behavior. Many additional traits have previously been mapped to a larger region of chromosome IV that encompasses the ; freshwater haplotype. To identify which of these traits specifically map to this adaptive haplotype, we made crosses of rare marine fish heterozygous for the freshwater haplotype in an otherwise marine genetic background. Further, we performed fine-scale association mapping in a fully interbreeding, polymorphic population of freshwater stickleback to disentangle the effects of pleiotropy and linkage on the phenotypes affected by this haplotype. Although we find evidence that linked mutations have small effects on a few phenotypes, a small 1.4-kb region within the first intron of ; has large effects on three phenotypic traits: lateral plate count, and both the number and patterning of the posterior lateral line neuromasts. Thus, the ; haplotype is a hotspot of adaptation in stickleback due to both a small, pleiotropic region affecting multiple traits as well as multiple linked mutations affecting additional traits.},
  archive      = {J_EVLETT},
  author       = {Archambeault, Sophie L. and Bärtschi, Luis R. and Merminod, Aurélie D. and Peichel, Catherine L.},
  doi          = {10.1002/evl3.175},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {282-301},
  shortjournal = {Evol. Lett.},
  title        = {Adaptation via pleiotropy and linkage: Association mapping reveals a complex genetic architecture within the stickleback eda locus},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020c). Issue information. <em>EVLETT</em>, <em>4</em>(4), 279–281.
(<a href="https://doi.org/10.1002/evl3.190">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_EVLETT},
  doi          = {10.1002/evl3.190},
  journal      = {Evolution Letters},
  number       = {4},
  pages        = {279-281},
  shortjournal = {Evol. Lett.},
  title        = {Issue information},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Loss of consumers constrains phenotypic evolution in the
resulting food web. <em>EVLETT</em>, <em>4</em>(3), 266–277. (<a
href="https://doi.org/10.1002/evl3.170">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The loss of biodiversity is altering the structure of ecological networks; however, we are currently in a poor position to predict how these altered communities will affect the evolution of remaining populations. Theory on fitness landscapes provides a framework for predicting how selection alters the evolutionary trajectory and adaptive potential of populations, but often treats the network of interacting populations as a “black box.” Here, we integrate ecological networks and fitness landscapes to examine how changes in food-web structure shape phenotypic evolution. We conducted a field experiment that removed a guild of larval parasitoids that imposed direct and indirect selection pressures on an insect herbivore. We then measured herbivore survival as a function of three key phenotypic traits to estimate directional, quadratic, and correlational selection gradients in each treatment. We used these selection gradients to characterize the slope and curvature of the fitness landscape to understand the direct and indirect effects of consumer loss on phenotypic evolution. We found that the number of traits under directional selection increased with the removal of larval parasitoids, indicating evolution was more constrained toward a specific combination of traits. Similarly, we found that the removal of larval parasitoids altered the curvature of the fitness landscape in such a way that tended to decrease the evolvability of the traits we measured in the next generation. Our results suggest that the loss of trophic interactions can impose greater constraints on phenotypic evolution. This indicates that the simplification of ecological communities may constrain the adaptive potential of remaining populations to future environmental change.},
  archive      = {J_EVLETT},
  author       = {Barbour, Matthew A. and Greyson-Gaito, Christopher J. and Sotoodeh, Arezoo and Locke, Brendan and Bascompte, Jordi},
  doi          = {10.1002/evl3.170},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {266-277},
  shortjournal = {Evol. Lett.},
  title        = {Loss of consumers constrains phenotypic evolution in the resulting food web},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Indirect genetic effects allow escape from the inefficient
equilibrium in a coordination game. <em>EVLETT</em>, <em>4</em>(3),
257–265. (<a href="https://doi.org/10.1002/evl3.155">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Social interactions involving coordination between individuals are subject to an “evolutionary trap.” Once a suboptimal strategy has evolved, mutants playing an alternative strategy are counterselected because they fail to coordinate with the majority. This creates a detrimental situation from which evolution cannot escape, preventing the evolution of efficient collective behaviors. Here, we study this problem using evolutionary robotics simulations. We first confirm the existence of an evolutionary trap in a simple setting. We then, however, reveal that evolution can solve this problem in a more realistic setting where individuals need to coordinate with one another. In this setting, simulated robots evolve an ability to adapt plastically their behavior to one another, as this improves the efficiency of their interaction. This ability has an unintended evolutionary consequence: a genetic mutation affecting one individual&#39;s behavior also indirectly alters their partner&#39;s behavior because the two individuals influence one another. As a consequence of this indirect genetic effect, pairs of partners can virtually change strategy together with a single mutation, and the evolutionary barrier between alternative strategies disappears. This finding reveals a general principle that could play a role in nature to smoothen the transition to efficient collective behaviors in all games with multiple equilibriums.},
  archive      = {J_EVLETT},
  author       = {Bernard, Arthur and Bredeche, Nicolas and André, Jean-Baptiste},
  doi          = {10.1002/evl3.155},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {257-265},
  shortjournal = {Evol. Lett.},
  title        = {Indirect genetic effects allow escape from the inefficient equilibrium in a coordination game},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Ecological speciation in european whitefish is driven by a
large-gaped predator. <em>EVLETT</em>, <em>4</em>(3), 243–256. (<a
href="https://doi.org/10.1002/evl3.167">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Lake-dwelling fish that form species pairs/flocks characterized by body size divergence are important model systems for speciation research. Although several sources of divergent selection have been identified in these systems, their importance for driving the speciation process remains elusive. A major problem is that in retrospect, we cannot distinguish selection pressures that initiated divergence from those acting later in the process. To address this issue, we studied the initial stages of speciation in European whitefish (; ) using data from 358 populations of varying age (26–10,000 years). We find that whitefish speciation is driven by a large-growing predator, the northern pike (; ). Pike initiates divergence by causing a largely plastic differentiation into benthic giants and pelagic dwarfs: ecotypes that will subsequently develop partial reproductive isolation and heritable differences in gill raker number. Using an eco-evolutionary model, we demonstrate how pike&#39;s habitat specificity and large gape size are critical for imposing a between-habitat trade-off, causing prey to mature in a safer place or at a safer size. Thereby, we propose a novel mechanism for how predators may cause dwarf/giant speciation in lake-dwelling fish species.},
  archive      = {J_EVLETT},
  author       = {Öhlund, Gunnar and Bodin, Mats and Nilsson, Karin A. and Öhlund, Sven-Ola and Mobley, Kenyon B. and Hudson, Alan G. and Peedu, Mikael and Brännström, Åke and Bartels, Pia and Præbel, Kim and Hein, Catherine L. and Johansson, Petter and Englund, Göran},
  doi          = {10.1002/evl3.167},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {243-256},
  shortjournal = {Evol. Lett.},
  title        = {Ecological speciation in european whitefish is driven by a large-gaped predator},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). The contribution of ancient admixture to reproductive
isolation between european sea bass lineages. <em>EVLETT</em>,
<em>4</em>(3), 226–242. (<a
href="https://doi.org/10.1002/evl3.169">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Understanding how new species arise through the progressive establishment of reproductive isolation (RI) barriers between diverging populations is a major goal in Evolutionary Biology. An important result of speciation genomics studies is that genomic regions involved in RI frequently harbor anciently diverged haplotypes that predate the reconstructed history of species divergence. The possible origins of these old alleles remain much debated, as they relate to contrasting mechanisms of speciation that are not yet fully understood. In the European sea bass (; ), the genomic regions involved in RI between Atlantic and Mediterranean lineages are enriched for anciently diverged alleles of unknown origin. Here, we used haplotype-resolved whole-genome sequences to test whether divergent haplotypes could have originated from a closely related species, the spotted sea bass (; ). We found that an ancient admixture event between ; and ; is responsible for the presence of shared derived alleles that segregate at low frequencies in both lineages of ; . An exception to this was found within regions involved in RI between the two ; lineages. In those regions, archaic tracts originating from ; locally reached high frequencies or even fixation in Atlantic genomes but were almost absent in the Mediterranean. We showed that the ancient admixture event most likely occurred between ; and the ; Atlantic lineage, while Atlantic and Mediterranean ; lineages were experiencing allopatric isolation. Our results suggest that local adaptive introgression and/or the resolution of genomic conflicts provoked by ancient admixture have probably contributed to the establishment of RI between the two ; lineages.},
  archive      = {J_EVLETT},
  author       = {Duranton, Maud and Allal, François and Valière, Sophie and Bouchez, Olivier and Bonhomme, François and Gagnaire, Pierre-Alexandre},
  doi          = {10.1002/evl3.169},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {226-242},
  shortjournal = {Evol. Lett.},
  title        = {The contribution of ancient admixture to reproductive isolation between european sea bass lineages},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Predicting the strength of urban-rural clines in a mendelian
polymorphism along a latitudinal gradient. <em>EVLETT</em>,
<em>4</em>(3), 212–225. (<a
href="https://doi.org/10.1002/evl3.163">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Cities are emerging as models for addressing the fundamental question of whether populations evolve in parallel to similar environments. Here, we examine the environmental factors that drive the evolution of parallel urban-rural clines in a Mendelian trait—the cyanogenic antiherbivore defense of white clover (; ). Previous work suggested urban-rural gradients in frost and snow depth could drive the evolution of reduced hydrogen cyanide (HCN) frequencies in urban populations. Here, we sampled over 700 urban and rural clover populations across 16 cities along a latitudinal transect in eastern North America. In each population, we quantified changes in the frequency of genotypes that produce HCN, and in a subset of the cities we estimated the frequency of the alleles at the two genes (; and ; ) that epistatically interact to produce HCN. We then tested the hypothesis that cold climatic conditions are necessary for the evolution of cyanogenesis clines by comparing the strength of clines among cities located along a latitudinal gradient of winter temperature and frost exposure. Overall, half of the cities exhibited urban-rural clines in the frequency of HCN, whereby urban populations evolved lower HCN frequencies. Clines did not evolve in cities with the lowest temperatures and greatest snowfall, supporting the hypothesis that snow buffers plants against winter frost and constrains the formation of clines. By contrast, the strongest clines occurred in the warmest cities where snow and frost are rare, suggesting that alternative selective agents are maintaining clines in warmer cities. Some clines were driven by evolution at only ; , consistent with stronger and more consistent selection on this locus than on ; . Together, our results demonstrate that urban environments often select for similar phenotypes, but different selective agents and targets underlie the evolutionary response in different cities.},
  archive      = {J_EVLETT},
  author       = {Santangelo, James S. and Thompson, Ken A. and Cohan, Beata and Syed, Jibran and Ness, Rob W. and Johnson, Marc T. J.},
  doi          = {10.1002/evl3.163},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {212-225},
  shortjournal = {Evol. Lett.},
  title        = {Predicting the strength of urban-rural clines in a mendelian polymorphism along a latitudinal gradient},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Artificial selection reveals the role of transcriptional
constraints in the maintenance of life history variation.
<em>EVLETT</em>, <em>4</em>(3), 200–211. (<a
href="https://doi.org/10.1002/evl3.166">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {The trade-off between reproduction and self-maintenance is a cornerstone of life history theory, yet its proximate underpinnings are elusive. Here, we used an artificial selection approach to create replicated lines of Japanese quail (; ) that differ genetically in their reproductive investment. Whole transcriptome sequencing revealed that females from lines selected for high reproductive output show a consistent upregulation of genes associated with reproduction but a simultaneous downregulation of immune genes. Concordant phenotypic differences in immune function (i.e., specific antibody response against keyhole limpet hemocyanin) were observed between the selection lines, even in males who do not provide parental care. Our findings demonstrate the key role of obligate transcriptional constraints in the maintenance of life history variation. These constraints set fundamental limits to productivity and health in natural and domestic animal populations.},
  archive      = {J_EVLETT},
  author       = {Pick, Joel L. and Hatakeyama, Masaomi and Ihle, Kate E. and Gasparini, Julien and Haussy, Claudy and Ishishita, Satoshi and Matsuda, Yoichi and Yoshimura, Takashi and Kanaoka, Masahiro M. and Shimizu-Inatsugi, Rie and Shimizu, Kentaro K. and Tschirren, Barbara},
  doi          = {10.1002/evl3.166},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {200-211},
  shortjournal = {Evol. Lett.},
  title        = {Artificial selection reveals the role of transcriptional constraints in the maintenance of life history variation},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Morphology does not covary with predicted behavioral
correlations of the domestication syndrome in dogs. <em>EVLETT</em>,
<em>4</em>(3), 189–199. (<a
href="https://doi.org/10.1002/evl3.168">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Domesticated animals display suites of altered morphological, behavioral, and physiological traits compared to their wild ancestors, a phenomenon known as the domestication syndrome (DS). Because these alterations are observed to co-occur across a wide range of present day domesticates, the traits within the DS are assumed to covary within species and a single developmental mechanism has been hypothesized to cause the observed co-occurrence. However, due to the lack of formal testing it is currently not well-resolved if the traits within DS actually covary. Here, we test the hypothesis that the presence of the classic morphological domestication traits white pigmentation, floppy ears, and curly tails predict the strength of behavioral correlations in support of the DS in 78 dog breeds. Contrary to the expectations of covariation among DS traits, we found that morphological traits did not covary among themselves, nor did they predict the strength of behavioral correlations among dog breeds. Further, the number of morphological traits in a breed did not predict the strength of behavioral correlations. Our results thus contrast with the hypothesis that the DS arises due to a shared underlying mechanism, but more importantly, questions if the morphological traits embedded in the DS are actual domestication traits or postdomestication improvement traits. For dogs, it seems highly likely that strong selection for breed specific morphological traits only happened recently and in relation to breed formation. Present day dogs therefore have limited bearing of the initial selection pressures applied during domestication and we should reevaluate our expectations of the DS accordingly.},
  archive      = {J_EVLETT},
  author       = {Hansen Wheat, Christina and van der Bijl, Wouter and Wheat, Christopher W.},
  doi          = {10.1002/evl3.168},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {189-199},
  shortjournal = {Evol. Lett.},
  title        = {Morphology does not covary with predicted behavioral correlations of the domestication syndrome in dogs},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). On the use of the coefficient of variation to quantify and
compare trait variation. <em>EVLETT</em>, <em>4</em>(3), 180–188. (<a
href="https://doi.org/10.1002/evl3.171">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Meaningful comparison of variation in quantitative trait requires controlling for both the dimension of the varying entity and the dimension of the factor generating variation. Although the coefficient of variation (CV; standard deviation divided by the mean) is often used to measure and compare variation of quantitative traits, it only accounts for the dimension of the former, and its use for comparing variation may sometimes be inappropriate. Here, we discuss the use of the CV to compare measures of evolvability and phenotypic plasticity, two variational properties of quantitative traits. Using a dimensional analysis, we show that contrary to evolvability, phenotypic plasticity cannot be meaningfully compared across traits and environments by mean-scaling trait variation. We further emphasize the need of remaining cognizant of the dimensions of the traits and the relationship between mean and standard deviation when comparing CVs, even when the scales on which traits are expressed allow meaningful calculation of the CV.},
  archive      = {J_EVLETT},
  author       = {Pélabon, Christophe and Hilde, Christoffer H. and Einum, Sigurd and Gamelon, Marlène},
  doi          = {10.1002/evl3.171},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {180-188},
  shortjournal = {Evol. Lett.},
  title        = {On the use of the coefficient of variation to quantify and compare trait variation},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020d). Issue information. <em>EVLETT</em>, <em>4</em>(3), 177–179.
(<a href="https://doi.org/10.1002/evl3.186">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_EVLETT},
  doi          = {10.1002/evl3.186},
  journal      = {Evolution Letters},
  number       = {3},
  pages        = {177-179},
  shortjournal = {Evol. Lett.},
  title        = {Issue information},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Evolution of strategic cooperation. <em>EVLETT</em>,
<em>4</em>(2), 164–175. (<a
href="https://doi.org/10.1002/evl3.164">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Group-beneficial behaviors have presented a long-standing challenge for evolutionary theory because, although their benefits are available to all group members, their costs are borne by individuals. Consequently, an individual could benefit from “cheating” their group mates by not paying the costs while still reaping the benefits. There have been many proposed evolutionary mechanisms that could favor cooperation (and disfavor cheating) in particular circumstances. However, if cooperation is still favored in some circumstances, then we might expect evolution to favor strategic cooperation, where the level of contribution toward group-beneficial behavior is varied in response to the social context. To uncover how and why individuals should contribute toward group-beneficial behavior across social contexts, we model strategic cooperation as an evolutionary game where players can quantitatively adjust the amount they contribute toward group-beneficial behavior. We find that the evolutionarily stable strategy (ESS) predicts, unsurprisingly, that players should contribute in relation to their relatedness to the group. However, we surprisingly find that players often contribute to cooperation in such a way that their fitness is inverse to their relatedness to the group such that those that contribute to cooperation end up with the same return from group-beneficial behavior, essentially removing any potential advantage of higher relatedness. These results bring to light a paradox of group-beneficial cooperation: groups do best when they contain highly related individuals, but those with the highest relatedness to the group will often have the lowest fitness within the group.},
  archive      = {J_EVLETT},
  author       = {Madgwick, P. G. and Wolf, J. B.},
  doi          = {10.1002/evl3.164},
  journal      = {Evolution Letters},
  number       = {2},
  pages        = {164-175},
  shortjournal = {Evol. Lett.},
  title        = {Evolution of strategic cooperation},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Resource heterogeneity and the evolution of public goods
cooperation. <em>EVLETT</em>, <em>4</em>(2), 155–163. (<a
href="https://doi.org/10.1002/evl3.158">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Heterogeneity in resources is a ubiquitous feature of natural landscapes affecting many aspects of biology. However, the effect of environmental heterogeneity on the evolution of cooperation has been less well studied. Here, using a mixture of theory and experiments measuring siderophore production by the bacterium ; as a model for public goods based cooperation, we explore the effect of heterogeneity in resource availability. We show that cooperation in metapopulations that were spatially heterogeneous in terms of resources can be maintained at a higher level than in homogeneous metapopulations of the same average resource value. The results can be explained by a positive covariance between fitness of cooperators, population size, and local resource availability, which allowed cooperators to have a disproportionate advantage within the heterogeneous metapopulations. These results suggest that natural environmental variation may help to maintain cooperation.},
  archive      = {J_EVLETT},
  author       = {Stilwell, Peter and O&#39;Brien, Siobhan and Hesse, Elze and Lowe, Chris and Gardner, Andy and Buckling, Angus},
  doi          = {10.1002/evl3.158},
  journal      = {Evolution Letters},
  number       = {2},
  pages        = {155-163},
  shortjournal = {Evol. Lett.},
  title        = {Resource heterogeneity and the evolution of public goods cooperation},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Assessing biological factors affecting postspeciation
introgression. <em>EVLETT</em>, <em>4</em>(2), 137–154. (<a
href="https://doi.org/10.1002/evl3.159">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {An increasing number of phylogenomic studies have documented a clear “footprint” of postspeciation introgression among closely related species. Nonetheless, systematic genome-wide studies of factors that determine the likelihood of introgression remain rare. Here, we propose an a priori hypothesis-testing framework that uses introgression statistics—including a new metric of estimated introgression, ; —to evaluate general patterns of introgression prevalence and direction across multiple closely related species. We demonstrate this approach using whole genome sequences from 32 lineages in 11 wild tomato species to assess the effect of three factors on introgression—genetic relatedness, geographical proximity, and mating system differences—based on multiple trios within the “ABBA–BABA” test. Our analyses suggest each factor affects the prevalence of introgression, although our power to detect these is limited by the number of comparisons currently available. We find that of 14 species pairs with geographically “proximate” versus “distant” population comparisons, 13 showed evidence of introgression; in 10 of these cases, this was more prevalent between geographically closer populations. We also find modest evidence that introgression declines with increasing genetic divergence between lineages, is more prevalent between lineages that share the same mating system, and—when it does occur between mating systems—tends to involve gene flow from more inbreeding to more outbreeding lineages. Although our analysis indicates that recent postspeciation introgression is frequent in this group—detected in 15 of 17 tested trios—estimated levels of genetic exchange are modest (0.2–2.5\% of the genome), so the relative importance of hybridization in shaping the evolutionary trajectories of these species could be limited. Regardless, similar clade-wide analyses of genomic introgression would be valuable for disentangling the major ecological, reproductive, and historical determinants of postspeciation gene flow, and for assessing the relative contribution of introgression as a source of genetic variation.},
  archive      = {J_EVLETT},
  author       = {Hamlin, Jennafer A. P. and Hibbins, Mark S. and Moyle, Leonie C.},
  doi          = {10.1002/evl3.159},
  journal      = {Evolution Letters},
  number       = {2},
  pages        = {137-154},
  shortjournal = {Evol. Lett.},
  title        = {Assessing biological factors affecting postspeciation introgression},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Recent immigrants alter the quantitative genetic
architecture of paternity in song sparrows. <em>EVLETT</em>,
<em>4</em>(2), 124–136. (<a
href="https://doi.org/10.1002/evl3.162">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Quantifying additive genetic variances and cross-sex covariances in reproductive traits, and identifying processes that shape and maintain such (co)variances, is central to understanding the evolutionary dynamics of reproductive systems. Gene flow resulting from among-population dispersal could substantially alter additive genetic variances and covariances in key traits in recipient populations, thereby altering forms of sexual conflict, indirect selection, and evolutionary responses. However, the degree to which genes imported by immigrants do in fact affect quantitative genetic architectures of key reproductive traits and outcomes is rarely explicitly quantified. We applied structured quantitative genetic analyses to multiyear pedigree, pairing, and paternity data from free-living song sparrows (; ) to quantify the differences in mean breeding values for major sex-specific reproductive traits, specifically female extra-pair reproduction and male paternity loss, between recent immigrants and the previously existing population. We thereby quantify effects of natural immigration on the means, variances, and cross-sex covariance in total additive genetic values for extra-pair paternity arising within the complex socially monogamous but genetically polygynandrous reproductive system. Recent immigrants had lower mean breeding values for male paternity loss, and somewhat lower values for female extra-pair reproduction, than the local recipient population, and would therefore increase the emerging degree of reproductive fidelity of social pairings. Furthermore, immigration increased the variances in total additive genetic values for these traits, but decreased the magnitudes of the negative cross-sex genetic covariation and correlation below those evident in the existing population. Immigration thereby increased the total additive genetic variance but could decrease the magnitude of indirect selection acting on sex-specific contributions to paternity outcomes. These results demonstrate that dispersal and resulting immigration and gene flow can substantially affect quantitative genetic architectures of complex local reproductive systems, implying that comprehensive theoretical and empirical efforts to understand mating system dynamics will need to incorporate spatial population processes.},
  archive      = {J_EVLETT},
  author       = {Reid, Jane M. and Arcese, Peter},
  doi          = {10.1002/evl3.162},
  journal      = {Evolution Letters},
  number       = {2},
  pages        = {124-136},
  shortjournal = {Evol. Lett.},
  title        = {Recent immigrants alter the quantitative genetic architecture of paternity in song sparrows},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Where is the optimum? Predicting the variation of selection
along climatic gradients and the adaptive value of plasticity. A case
study on tree phenology. <em>EVLETT</em>, <em>4</em>(2), 109–123. (<a
href="https://doi.org/10.1002/evl3.160">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many theoretical models predict when genetic evolution and phenotypic plasticity allow adaptation to changing environmental conditions. These models generally assume stabilizing selection around some optimal phenotype. We however often ignore how optimal phenotypes change with the environment, which limit our understanding of the adaptive value of phenotypic plasticity. Here, we propose an approach based on our knowledge of the causal relationships between climate, adaptive traits, and fitness to further these questions. This approach relies on a sensitivity analysis of the process-based model ; , which mathematically formalizes these causal relationships, to predict fitness landscapes and optimal budburst dates along elevation gradients in three major European tree species. Variation in the overall shape of the fitness landscape and resulting directional selection gradients were found to be mainly driven by temperature variation. The optimal budburst date was delayed with elevation, while the range of dates allowing high fitness narrowed and the maximal fitness at the optimum decreased. We also found that the plasticity of the budburst date should allow tracking the spatial variation in the optimal date, but with variable mismatch depending on the species, ranging from negligible mismatch in fir, moderate in beech, to large in oak. Phenotypic plasticity would therefore be more adaptive in fir and beech than in oak. In all species, we predicted stronger directional selection for earlier budburst date at higher elevation. The weak selection on budburst date in fir should result in the evolution of negligible genetic divergence, while beech and oak would evolve counter-gradient variation, where genetic and environmental effects are in opposite directions. Our study suggests that theoretical models should consider how whole fitness landscapes change with the environment. The approach introduced here has the potential to be developed for other traits and species to explore how populations will adapt to climate change.},
  archive      = {J_EVLETT},
  author       = {Gauzere, Julie and Teuf, Bertrand and Davi, Hendrik and Chevin, Luis-Miguel and Caignard, Thomas and Leys, Bérangère and Delzon, Sylvain and Ronce, Ophélie and Chuine, Isabelle},
  doi          = {10.1002/evl3.160},
  journal      = {Evolution Letters},
  number       = {2},
  pages        = {109-123},
  shortjournal = {Evol. Lett.},
  title        = {Where is the optimum? predicting the variation of selection along climatic gradients and the adaptive value of plasticity. a case study on tree phenology},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Inference of natural selection from ancient DNA.
<em>EVLETT</em>, <em>4</em>(2), 94–108. (<a
href="https://doi.org/10.1002/evl3.165">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evolutionary processes, including selection, can be indirectly inferred based on patterns of genomic variation among contemporary populations or species. However, this often requires unrealistic assumptions of ancestral demography and selective regimes. Sequencing ancient DNA from temporally spaced samples can inform about past selection processes, as time series data allow direct quantification of population parameters collected before, during, and after genetic changes driven by selection. In this Comment and Opinion, we advocate for the inclusion of temporal sampling and the generation of paleogenomic datasets in evolutionary biology, and highlight some of the recent advances that have yet to be broadly applied by evolutionary biologists. In doing so, we consider the expected signatures of balancing, purifying, and positive selection in time series data, and detail how this can advance our understanding of the chronology and tempo of genomic change driven by selection. However, we also recognize the limitations of such data, which can suffer from postmortem damage, fragmentation, low coverage, and typically low sample size. We therefore highlight the many assumptions and considerations associated with analyzing paleogenomic data and the assumptions associated with analytical methods.},
  archive      = {J_EVLETT},
  author       = {Dehasque, Marianne and Ávila-Arcos, María C. and Díez-del-Molino, David and Fumagalli, Matteo and Guschanski, Katerina and Lorenzen, Eline D. and Malaspinas, Anna-Sapfo and Marques-Bonet, Tomas and Martin, Michael D. and Murray, Gemma G. R. and Papadopulos, Alexander S. T. and Therkildsen, Nina Overgaard and Wegmann, Daniel and Dalén, Love and Foote, Andrew D.},
  doi          = {10.1002/evl3.165},
  journal      = {Evolution Letters},
  number       = {2},
  pages        = {94-108},
  shortjournal = {Evol. Lett.},
  title        = {Inference of natural selection from ancient DNA},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020e). Issue information. <em>EVLETT</em>, <em>4</em>(2), 91–93.
(<a href="https://doi.org/10.1002/evl3.172">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_EVLETT},
  doi          = {10.1002/evl3.172},
  journal      = {Evolution Letters},
  number       = {2},
  pages        = {91-93},
  shortjournal = {Evol. Lett.},
  title        = {Issue information},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Herbivore-mediated negative frequency-dependent selection
underlies a trichome dimorphism in nature. <em>EVLETT</em>,
<em>4</em>(1), 83–90. (<a
href="https://doi.org/10.1002/evl3.157">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Negative frequency-dependent selection (NFDS) has been shown to maintain polymorphism in a diverse array of traits. The action of NFDS has been confirmed through modeling, experimental approaches, and genetic analyses. In this study, we investigated NFDS in the wild using morph-frequency changes spanning a 20-year period from over 30 dimorphic populations of ; . In these populations, plants either possess glandular (sticky) or non-glandular (velvety) trichomes, and the ratio of these morphs varies substantially among populations. Our method provided evidence that NFDS, rather than drift or migration, is the primary force maintaining this dimorphism. Most populations that were initially dimorphic remained dimorphic, and the overall mean and variance in morph frequency did not change over time. Furthermore, morph-frequency differences were not related to geographic distances. Together, these results indicate that neither directional selection, drift, or migration played a substantial role in determining morph frequencies. However, as predicted by negative frequency-dependent selection, we found that the rare morph tended to increase in frequency, leading to a negative relationship between the change in the frequency of the sticky morph and its initial frequency. In addition, we found that morph-frequency change over time was significantly correlated with the damage inflicted by two herbivores: ; and ; . The latter is a specialist on the sticky morph and damage by this herbivore was greatest when the sticky morph was common. The reverse was true for ; , such that damage increased with the frequency of the velvety morph. These findings suggest that these herbivores contribute to balancing selection on the observed trichome dimorphism.},
  archive      = {J_EVLETT},
  author       = {Goldberg, Jay K. and Lively, Curtis M. and Sternlieb, Sonya R. and Pintel, Genevieve and Hare, J. Daniel and Morrissey, Michael B. and Delph, Lynda F.},
  doi          = {10.1002/evl3.157},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {83-90},
  shortjournal = {Evol. Lett.},
  title        = {Herbivore-mediated negative frequency-dependent selection underlies a trichome dimorphism in nature},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Controlling for body size leads to inferential biases in the
biological sciences. <em>EVLETT</em>, <em>4</em>(1), 73–82. (<a
href="https://doi.org/10.1002/evl3.151">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many traits correlate with body size. Studies that seek to uncover the ecological factors that drive evolutionary responses in traits typically examine these responses relative to associated changes in body size using multiple regression analysis. However, it is not well appreciated that in the presence of strongly correlated variables, the partial (i.e., relative) regression coefficients often change sign compared to the original coefficients. Such sign reversals are difficult to interpret in a biologically meaningful way, and could lead to erroneous evolutionary inferences if the true mechanism underlying the sign reversal differed from the proposed mechanism. Here, we use simulations to demonstrate that sign reversal occurs over a wide range of parameter values common in the biological sciences. Further, as a case-in-point, we review the literature on brain size evolution; a field that explores how ecological traits relate to the evolution of relative brain size (brain size relative to body size). We find that most studies show sign reversals and thus that the inferences of many studies in this field may be inconclusive. Finally, we propose some approaches to mitigating this issue.},
  archive      = {J_EVLETT},
  author       = {Rogell, Björn and Dowling, Damian K. and Husby, Arild},
  doi          = {10.1002/evl3.151},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {73-82},
  shortjournal = {Evol. Lett.},
  title        = {Controlling for body size leads to inferential biases in the biological sciences},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Kin discrimination, negative relatedness, and how to
distinguish between selfishness and spite. <em>EVLETT</em>,
<em>4</em>(1), 65–72. (<a
href="https://doi.org/10.1002/evl3.150">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Spiteful behaviors occur when an actor harms its own fitness to inflict harm on the fitness of others. Several papers have predicted that spite can be favored in sufficiently small populations, even when the harming behavior is directed indiscriminately at others. However, it is not clear that truly spiteful behavior could be favored without the harm being directed at a subset of social partners with relatively low genetic similarity to the actor (kin discrimination, causing a negative relatedness between actor and harmed recipient). Using mathematical models, we show that (1) the evolution of spite requires kin discrimination; (2) previous models suggesting indiscriminate spite involve scenarios where the actor gains a direct feedback benefit from harming others, and so the harming is selfish rather than spiteful; (3) extreme selfishness can be favored in small populations (or, more generally, under local competition) because this is where the direct feedback benefit of harming is greatest.},
  archive      = {J_EVLETT},
  author       = {Patel, Matishalin and West, Stuart A. and Biernaskie, Jay M.},
  doi          = {10.1002/evl3.150},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {65-72},
  shortjournal = {Evol. Lett.},
  title        = {Kin discrimination, negative relatedness, and how to distinguish between selfishness and spite},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Resource-dependent evolution of female resistance responses
to sexual conflict. <em>EVLETT</em>, <em>4</em>(1), 54–64. (<a
href="https://doi.org/10.1002/evl3.153">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Sexual conflict can promote the evolution of dramatic reproductive adaptations as well as resistance to its potentially costly effects. Theory predicts that responses to sexual conflict will vary significantly with resource levels—when scant, responses should be constrained by trade-offs, when abundant, they should not. However, this can be difficult to test because the evolutionary interests of the sexes align upon short-term exposure to novel environments, swamping any selection due to sexual conflict. What is needed are investigations of populations that are well adapted to both differing levels of sexual conflict and resources. Here, we used this approach in a long-term experimental evolution study to track the evolution of female resistance to sexual conflict in the fruit fly ; . In resource-rich regimes, high-conflict females evolved resistance to continual exposure to males. There was no difference in baseline survival, consistent with the idea that responses evolving under nutritional abundance experienced no trade-offs with resistance. In the poor resource regimes, the ability of high-conflict females to evolve resistance to males was severely compromised and they also showed lower baseline survival than low-conflict females. This suggested high-conflict females traded off somatic maintenance against any limited resistance they had evolved in response to sexual conflict. Overall, these findings provide experimental support for the hypothesis that evolutionary responses to sexual conflict are critically dependent upon resource levels.},
  archive      = {J_EVLETT},
  author       = {Rostant, Wayne G. and Mason, Janet S. and de Coriolis, Jean-Charles and Chapman, Tracey},
  doi          = {10.1002/evl3.153},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {54-64},
  shortjournal = {Evol. Lett.},
  title        = {Resource-dependent evolution of female resistance responses to sexual conflict},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Sex differences in age-to-maturation relate to sexual
selection and adult sex ratios in birds. <em>EVLETT</em>, <em>4</em>(1),
44–53. (<a href="https://doi.org/10.1002/evl3.156">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Maturation (the age when organisms are physiologically capable of breeding) is one of the major life history traits that have pervasive implications for reproductive strategies, fitness, and population growth. Sex differences in maturation are common in nature, although the causes of such differences are not understood. Fisher and Lack proposed that delayed maturation in males is expected when males are under intense sexual selection, but their proposition has never been tested across a wide range of taxa. By using phylogenetic comparative analyses and the most comprehensive dataset to date, including 201 species from 59 avian families, we show that intense sexual selection on males (as indicated by polygamous mating and male-skewed sexual size dimorphism) correlates with delayed maturation. We also show that the adult sex ratio (ASR), an indicator of the social environment, is associated with sex-specific maturation because in species with a female-skewed ASR, males experience later maturation. Phylogenetic path analyses suggest that adult sex ratio drives interspecific changes in the intensity of sexual selection which, in turn, influences maturation. These results are robust to alternative phylogenetic hypotheses and to potential life-history confounds, and they provide the first comprehensive support of Fisher&#39;s and Lack&#39;s propositions. Importantly, our work suggests that both social environment and mate competition influence the evolution of a major life history trait, maturation.},
  archive      = {J_EVLETT},
  author       = {Ancona, Sergio and Liker, András and Carmona-Isunza, M. Cristina and Székely, Tamás},
  doi          = {10.1002/evl3.156},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {44-53},
  shortjournal = {Evol. Lett.},
  title        = {Sex differences in age-to-maturation relate to sexual selection and adult sex ratios in birds},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). From cryptic to colorful: Evolutionary decoupling of larval
and adult color in butterflies. <em>EVLETT</em>, <em>4</em>(1), 34–43.
(<a href="https://doi.org/10.1002/evl3.149">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Many animals undergo complete metamorphosis, where larval forms change abruptly in adulthood. Color change during ontogeny is common, but there is little understanding of evolutionary patterns in these changes. Here, we use data on larval and adult color for 246 butterfly species (61\% of all species in Australia) to test whether the evolution of color is coupled between life stages. We show that adults are more variable in color across species than caterpillars and that male adult color has lower phylogenetic signal. These results suggest that sexual selection is driving color diversity in male adult butterflies at a broad scale. Moreover, color similarities between species at the larval stage do not predict color similarities at the adult stage, indicating that color evolution is decoupled between young and adult forms. Most species transition from cryptic coloration as caterpillars to conspicuous coloration as adults, but even species with conspicuous caterpillars change to different conspicuous colors as adults. The use of high-contrast coloration is correlated with body size in caterpillars but not adults. Taken together, our results suggest a change in the relative importance of different selective pressures at different life stages, resulting in the evolutionary decoupling of coloration through ontogeny.},
  archive      = {J_EVLETT},
  author       = {Medina, Iliana and Vega-Trejo, Regina and Wallenius, Thomas and Symonds, Matthew R. E. and Stuart-Fox, Devi},
  doi          = {10.1002/evl3.149},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {34-43},
  shortjournal = {Evol. Lett.},
  title        = {From cryptic to colorful: Evolutionary decoupling of larval and adult color in butterflies},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Field cricket genome reveals the footprint of recent, abrupt
adaptation in the wild. <em>EVLETT</em>, <em>4</em>(1), 19–33. (<a
href="https://doi.org/10.1002/evl3.148">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Evolutionary adaptation is generally thought to occur through incremental mutational steps, but large mutational leaps can occur during its early stages. These are challenging to study in nature due to the difficulty of observing new genetic variants as they arise and spread, but characterizing their genomic dynamics is important for understanding factors favoring rapid adaptation. Here, we report genomic consequences of recent, adaptive song loss in a Hawaiian population of field crickets (; ). A discrete genetic variant, ; , appeared and spread approximately 15 years ago. ; erases sound-producing veins on male wings. These silent flatwing males are protected from a lethal, eavesdropping parasitoid fly. We sequenced, assembled and annotated the cricket genome, produced a linkage map, and identified a ; quantitative trait locus covering a large region of the X chromosome. Gene expression profiling showed that ; is associated with extensive genome-wide effects on embryonic gene expression. We found that flatwing male crickets express feminized chemical pheromones. This male feminizing effect, on a different sexual signaling modality, is genetically associated with the ; genotype. Our findings suggest that the early stages of evolutionary adaptation to extreme pressures can be accompanied by greater genomic and phenotypic disruption than previously appreciated, and highlight how abrupt adaptation might involve suites of traits that arise through pleiotropy or genomic hitchhiking.},
  archive      = {J_EVLETT},
  author       = {Pascoal, Sonia and Risse, Judith E. and Zhang, Xiao and Blaxter, Mark and Cezard, Timothee and Challis, Richard J. and Gharbi, Karim and Hunt, John and Kumar, Sujai and Langan, Emma and Liu, Xuan and Rayner, Jack G. and Ritchie, Michael G. and Snoek, Basten L. and Trivedi, Urmi and Bailey, Nathan W.},
  doi          = {10.1002/evl3.148},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {19-33},
  shortjournal = {Evol. Lett.},
  title        = {Field cricket genome reveals the footprint of recent, abrupt adaptation in the wild},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020). Evolutionary genomics can improve prediction of species’
responses to climate change. <em>EVLETT</em>, <em>4</em>(1), 4–18. (<a
href="https://doi.org/10.1002/evl3.154">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  abstract     = {Global climate change (GCC) increasingly threatens biodiversity through the loss of species, and the transformation of entire ecosystems. Many species are challenged by the pace of GCC because they might not be able to respond fast enough to changing biotic and abiotic conditions. Species can respond either by shifting their range, or by persisting in their local habitat. If populations persist, they can tolerate climatic changes through phenotypic plasticity, or genetically adapt to changing conditions depending on their genetic variability and census population size to allow for de novo mutations. Otherwise, populations will experience demographic collapses and species may go extinct. Current approaches to predicting species responses to GCC begin to combine ecological and evolutionary information for species distribution modelling. Including an evolutionary dimension will substantially improve species distribution projections which have not accounted for key processes such as dispersal, adaptive genetic change, demography, or species interactions. However, eco-evolutionary models require new data and methods for the estimation of a species&#39; adaptive potential, which have so far only been available for a small number of model species. To represent global biodiversity, we need to devise large-scale data collection strategies to define the ecology and evolutionary potential of a broad range of species, especially of keystone species of ecosystems. We also need standardized and replicable modelling approaches that integrate these new data to account for eco-evolutionary processes when predicting the impact of GCC on species&#39; survival. Here, we discuss different genomic approaches that can be used to investigate and predict species responses to GCC. This can serve as guidance for researchers looking for the appropriate experimental setup for their particular system. We furthermore highlight future directions for moving forward in the field and allocating available resources more effectively, to implement mitigation measures before species go extinct and ecosystems lose important functions.},
  archive      = {J_EVLETT},
  author       = {Waldvogel, Ann-Marie and Feldmeyer, Barbara and Rolshausen, Gregor and Exposito-Alonso, Moises and Rellstab, Christian and Kofler, Robert and Mock, Thomas and Schmid, Karl and Schmitt, Imke and Bataillon, Thomas and Savolainen, Outi and Bergland, Alan and Flatt, Thomas and Guillaume, Frederic and Pfenninger, Markus},
  doi          = {10.1002/evl3.154},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {4-18},
  shortjournal = {Evol. Lett.},
  title        = {Evolutionary genomics can improve prediction of species’ responses to climate change},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
<li><details>
<summary>
(2020f). Issue information. <em>EVLETT</em>, <em>4</em>(1), 1–3. (<a
href="https://doi.org/10.1002/evl3.161">www</a>)
</summary>
<textarea id="copyID" onclick="copy(this)" rows="16" cols="145">
@article{ ,
  archive      = {J_EVLETT},
  doi          = {10.1002/evl3.161},
  journal      = {Evolution Letters},
  number       = {1},
  pages        = {1-3},
  shortjournal = {Evol. Lett.},
  title        = {Issue information},
  volume       = {4},
  year         = {2020},
}
</textarea>
</details></li>
</ul>

</body>
</html>
